<h2>Description</h2>

<p>This track represents RNA-Seq data (Toung, et al. 2011) of cultured human B-cells from 20 unrelated individuals from
the <a href="https://cephb.fr/en/index.php" target="_blank">Center d'Etude du Polymorphisme Humain (CEPH)</a> 
collection (Dausset, et al. 1990). RNA-Seq is a high-throughput sequencing-based method for 
characterizing and quantifying transcriptomes. Briefly, the mRNA population of RNA by poly(A)+ 
selection was isolated, converted to a library of cDNA fragments with adaptors attached, and 
sequenced the cDNA library using Illumina 1G Genome Analyzer to a depth of approximately 40 million 
50 base-pair reads per sample. To characterize the overall gene expression landscape, all sequences 
were pooled to create an 879 million read dataset.</p>

<p>Reads were mapped to the human genome using Bowtie (Langmead, et al. 2009) and splice junctions 
were detected using Tophat (Trapnell, et al. 2009). The coverage or number of reads aligning to each
base in the genome is displayed on one track. Splice junctions are also denoted on a separate track.
</p>

<h2>Display Conventions and Configuration</h2>

<p>This track is a multi-view composite track that contains multiple data types (views). For each 
view, there are multiple subtracks that display individually on the browser. Instructions for 
configuring multi-view tracks are <a href="/goldenPath/help/multiView.html">
here</a>. The following views are in this track:</p>

<p><em>Coverage</em></p>

<ul> 
	<li>Density graph (wiggle) of coverage or reads per base.</li> 
</ul>

<p><em>Junction Reads</em></p>

<ul> 
	<li>Alignment positions of junction sites.</li>
</ul>

<h2>Methods</h2>

<p>Immortalized B cell lines for 20 European-derived individuals from the Utah pedigrees of the
<a href="https://cephb.fr/en/index.php" target="_blank">Center d'Etude du Polymorphisme Humain (CEPH)</a> 
collection were obtained from Coriell Cell Repositories. No individuals were known to be blood 
relatives, and there is no known history of major medical illness. Specifically, the individuals are
GM06985, GM07000, GM07034, GM07055, GM07056, GM07345, GM11832, GM11839, GM11992, GM11993, GM11994, 
GM12056, GM12145, GM12155, GM12716, GM12717, GM12750, GM12813, GM12872 and GM12891. The dataset 
consists of 10 men and 10 women.</p>

<p>Cells were grown to a density of 5x105 cells/mL in RPMI 1640 supplemented with 15% fetal bovine
serum, 100 units/mL penicillin-streptomycin, and 2 mM L-glutamine. Cells were harvested 24 hours
after addition of fresh medium. Total RNA was extracted from cell pellets using the RNeasy Mini-Kit
with DNase treatment (Qiagen).</p>

<p>RNA-Seq was performed as recommended by the manufacturer (Illumina). Briefly, polyA mRNA was
fragmented and first strand cDNA prepared using random hexamers. Following second strand cDNA
synthesis, end repair, addition of a single A base, adaptor ligation, agarose gel isolation of ~200
base-pair cDNA and PCR amplification of the ~200 base-pair cDNA, the samples were sequenced using
the Illumina 1G Genome Analyzer to a coverage of approximately 40 million reads per sample.</p>

<p>Across the 20 samples, 43,819,745 + 8,194,875 (mean + standard deviation) reads were obtained.
All reads from each sample were pooled to form a dataset consisting of 878,668,290 random reads.
Sequence reads were mapped to the reference human genome sequence (NCBI36.1/hg18 assembly) using
Bowtie (Langmead, et al. 2009) and Tophat (Trapnell, et al. 2009).</p>

<h2>Verification</h2>

<p>Correlation to microarray data R = 0.59</p>

<h2>Credits</h2>

<p>Cheung Lab: Jonathan M. Toung, Michael Morley, Mingyao Li, Vivian G. Cheung, Alan Bruzel, Denis
Smirnov.</p>

<p>Contact: Jonathan M. Toung.</p>

<h2>References</h2>

<p>Dausset J, Cann H, Cohen D, Lathrop M, Lalouel JM, White R.
<a href="https://www.sciencedirect.com/science/article/pii/088875439090491C" target=_blank>
Centre d'etude du polymorphisme humain (CEPH): collaborative
genetic mapping of the human genome</a>. <i>Genomics</i>. 1990 Mar;6(3):575-7.</p>

<p>Langmead B, Trapnell C, Pop M, Salzberg SL. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2009-10-3-r25"
target=_blank>
Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</a>. <i>Genome 
Biol</i>. 2009 Mar 4;10(3):R25.</p>

<p>Toung JM, Morley M, Li M, Cheung VG. <a href="https://genome.cshlp.org/content/21/6/991"
target=_blank>
RNA-sequence analysis of human B-cells.</a> 
<i>Genome Res.</i> 2011 Jun;21(6):991-8.</p>

<p>Trapnell C, Pachter L, Salzberg SL.
<a href="https://academic.oup.com/bioinformatics/article/25/9/1105/203994"
target=_blank>TopHat: discovering splice junctions with RNA-Seq</a>. <i>Bioinformatics</i>. 
2009 May 1;25(9):1105-11.</p>

