<H2>Description</H2>
<P>
This annotation shows regions detected as putative copy number polymorphisms
(CNP) and sites of detected intermediate-sized structural variation (ISV). 
The CNPs and ISVs were determined by various methods, displayed in 
individual subtracks within the annotation:</P>
<UL>
<LI>
<B>Deletions from genotype analysis (Conrad):</B> 935 deletions detected
by analysis of SNP genotypes, using the HapMap Phase I data, release 16c.1, 
CEU and YRI samples.
<LI>
<B>Deletions from haploid hybridization analysis (Hinds):</B> 100 deletions 
from haploid hybridization analysis in 24 unrelated individuals from the 
Polymorphism Discovery Resource, selected for SNP LD study.
<LI>
<B>BAC microarray analysis (Iafrate):</B> 236 putative CNP regions detected by
BAC microarray analysis in a population of 55 individuals, 16 of which had
previously-characterized chromosomal abnormalities. The group consisted of 10
Caucasians, 4 Amerindians, 2 Chinese, 2 Indo-Pakistani, 2 Sub-Saharan
African, and 35 of unknown ethnic origin.
<LI>
<B>CNP in duplication-rich regions (Locke):</B> 243 CNP regions were
identified using array CGH in the HapMap populations (269 individuals).
The study was specific to 130 putative rearrangement hotspot regions.
<LI>
<B>Deletions from genotype analysis (McCarroll):</B> 540 deletions detected
by analysis of SNP genotypes, using the HapMap Phase I data, release 16a.
<LI>
<B>SNP and BAC microarray analysis of HapMap data (Redon):</B> 1,445 copy
number variable regions found in the HapMap Phase II data.
<LI>
<B>Representational oligonucleotide microarray analysis (ROMA) (Sebat):</B> 80
putative CNP regions detected by ROMA in a population of 20 normal individuals
comprised of 1 Biaka, 1 Mbuti, 1 Druze, 1 Melanesian, 4 French, 1 Venezualan,
1 Cambodian, 1 Mayan and 9 of unknown ethnicity.
<LI>
<B>BAC microarray analysis (Sharp):</B> 140 putative CNP regions detected by
BAC microarray analysis in a population of 47 individuals comprised of 8
Chinese, 4 Japanese, 10 Czech, 2 Druze, 7 Biaka, 9 Mbuti, and 7 Amerindians.
<LI>
<B>Fosmid mapping (Tuzun):</B> 297 ISV sites detected by mapping paired-end
sequences from a human fosmid DNA library.
</UL></P>

<H2>Display Conventions and Configuration</H2>
<P>
CNP and ISV regions are indicated by solid blocks that are color-coded to 
indicate the type of variation detected:
<UL>
<LI>
<B><FONT COLOR="green">Green</FONT>:</B> gain (duplications)
<LI>
<B><FONT COLOR="red">Red</FONT>:</B> loss (deletions)
<LI>
<B><FONT COLOR="blue">Blue</FONT>:</B> gain and loss (both deletion and
duplication)
<LI>
<B>Black:</B> inversion
<LI>
<B><FONT COLOR="gray">Gray</FONT>:</B> gain or loss (unknown direction)
</UL></P>
<P>Note that display IDs are not preserved between assemblies.</P>

<H3>Conrad subtrack</H3>
<P>
The method used to identify these deletions approximates the breakpoints of
each event; therefore, a set of minimal and maximal endpoints is associated
with each deletion. Thick lines delineate the minimally deleted region; thin
lines delineate the maximally deleted region.

<H3>Sharp subtrack </H3>
<P>
On the details pages for elements in this subtrack, the table shows
value/threshold data for each individual in the population. &quot;Value&quot;
is defined as the log<sub>2</sub> ratio of fluorescence intensity of test
versus reference DNA. &quot;Threshold&quot; is defined as 2 standard
deviations from the mean log<sub>2</sub> ratio of all autosomal clones per
hybridization. The &quot;Disease Percent&quot; value reflects the percent of
the BAC that lies within a &quot;rearrangement hotspot&quot;, as defined in
Sharp <em>et al</em>. (2005).
A rearrangement hotspot is defined by the presence of flanking
intrachromosomal duplications &gt;10 kb in length with &gt;95% similarity and
separated by 50 kb - 10 Mb of intervening sequence.</P>


<H2>Methods</H2>

<H3>Conrad genotype analysis</H3>
<P>
SNPs in regions that are hemizygous for a deletion are generally miscalled as
homozygous for the allele that is present.  Hence, when a deletion is
transmitted from parent to child, the genotypes at SNPs within the deletion
region will often appear to violate the rules of Mendelian transmission.  The
authors developed a simple algorithm for scanning trio data for unusual runs of
consecutive SNPs that, in a single family, have genotype configurations
consistent with the presence of a deletion.

<H3>Hinds haploid hybridization analysis</H3>
<P>
Approximately 600 Mb of genomic DNA from 24 unrelated individuals were obtained
from the Polymorphism Discovery Resource. Haploid hybridization was used to
identify genomic intervals showing a reduced hybridization signal in comparison
to the reference assembly. PCR amplification was performed on 215 candidate
deletions. 100 deletions were selected that were unambiguously confirmed.

<H3>Iafrate BAC microarray analysis</H3>
<P>
All hybridizations were performed in duplicate incorporating a dye-reversal 
using proprietary 1 Mb GenomeChip V1.2 Human BAC Arrays consisting of 2,632 BAC 
clones (Spectral Genomics, Houston, TX). The false positive rate was estimated 
at ~1 clone per 5,264 tested. </P>
<P>
Further information is available from the 
<A HREF="http://projects.tcag.ca/variation/" TARGET=_blank>Database of Genomic
Variants</A> website.</P>

<H3>Locke analysis of duplication-rich regions</H3>
<P>
DNA samples were obtained from Coriell Cell Repositories.  The reference DNA
used for all hybridizations was from a single male of Czechoslovakian descent,
Coriell ID GM15724 (also used in the Sharp study).
<P>
A locus was considered a CNV (copy number variation) if the log ratio of
fluroescence measurements for the individuals assayed exceeded twice the
standard deviation of the autosomal clones in replicate dye-swapped 
experiments. A CNV was classified as a CNP if altered copy number was observed
in more than 1% of the 269 individuals.

<H3>McCarroll genotype analysis</H3>
<P>
A segregating deletion can leave &quot;footprints&quot; in SNP genotype data,
including apparent deviations from Mendelian inheritance, apparent deviations
from Hardy-Weinberg equilibrium and null genotypes.  Using these clues to
discover true variants is challenging, however, because the vast majority of
such observations represent technical artifacts and genotyping errors.
</P>
<P>
To determine whether a subset of &quot;failed&quot; SNP genotyping assays in
the HapMap data might reflect structural variation, the authors examined
whether such failures were physically clustered in a manner that is specific
to individuals. Consistent with this hypothesis, the rate of
Mendelian-inconsistent genotypes was elevated near other Mendelian-inconsistent
genotypes in the same individual but was unrelated to Mendelian inconsistencies
in other individuals.
</P>
<P>
The authors systematically looked for regions of the genome in which the
same failure profile appeared repeatedly at nearby markers in a manner that
was statistically unexpected based on chance.  A set of statistical thresholds
was tailored to each mode of failure, genotyping center and genotyping platform
used in the project.  The same procedure could readily apply to dense SNP data
from any platform or study.</P>

<H3>Redon analysis of HapMap data</H3>
<P>Experiments were performed with the International HapMap DNA and cell-line
collection using two technologies: comparative analysis of hybridization
intensities on Affymetric GeneChip Human Mapping 500K early access arrays
(500K EA) and comparative genomic hybridization with a Whole Genome TilePath
(WGTP) array.

<H3>Sebat ROMA</H3>
<P>
Following digestion with BglII or HindIII, genomic DNA was hybridized to a 
custom array consisting of 85,000 oligonucleotide probes. The probes were
selected to be free of common repeats and have unique homology within the
human genome. The average resolution of the array was ~35 kb; however, only
intervals in which three consecutive probes showed concordant signals were
scored as CNPs. All hybridizations were performed in duplicate incorporating a
dye-reversal, with the false positive rate estimated to be ~6%.</P>

<H3>Sharp BAC microarray analysis</H3>
<P>
All hybridizations were performed in duplicate incorporating a dye-reversal
using a custom array consisting of 2,194 end-sequence or FISH-confirmed BACs,
targeted to regions of the genome flanked by segmental duplications.
The false positive rate was estimated at ~3 clones per 4,000 tested.</P>

<H3>Tuzun fosmid mapping</H3>
<P>
Paired-end sequences from a human fosmid DNA library were mapped to the
assembly. The average resolution of this technique was ~8 kb, and included 56
sites of inversion not detectable by the array-based approaches.  However,
because of the physical constraints of fosmid insert size, this technique was
unable to detect insertions greater than 40 kb in size.</P>


<H2>Validation</H2>
<H3>Conrad genotype analysis</H3>
<P>
The authors first tested 12 predicted deletions using quantitative PCR. For all
12 deletions, DNA concentrations consistent with transmission of a deletion
from parent to child were observed.</P>
<P>To provide more extensive validation by comparative genome hybridization
(CGH), the authors designed a custom oligonucleotide microarray comprised of
380,000 probes that tile across all 134 candidate deletions identified in 9 
HapMap offspring (8 YRI and 1 CEU). The results of this CGH analysis indicate
that the majority (about 85%) of candidate deletions detected by the method are
real. 

<H3>Locke duplication-rich regions </H3>
The authors performed validation using a custom oligonucleotide array,
hybridized to 9 of the HapMap individuals. Their analysis of the validation
experiments indicated a false-negative rate of 5% and a false-positive rate of
less than 0.2%.   

<H3>McCarroll genotype analysis</H3>
<P>
Four methods of validation were used: fluorescent <em>in situ</em>
hybridization (FISH), two-color fluorescence intensity measurements, PCR
amplification and quantitative PCR.
</P>
<P>
The authors performed fluorescent <em>in situ</em> hybridization for
five candidate deletions large enough to span available FISH probes. In all
five cases, FISH assays confirmed the deletions in the predicted individuals.
</P>
<P>
The authors examined two-color allele-specific fluorescence data from SNP
genotyping assays from a data subset available at the Broad Institute, looking
for a reduction in fluorescence intensity in individuals predicted to carry a
deletion. At most SNPs in the genome, fluorescence intensity measurements
clustered into two or three discrete groups corresponding to homozygous and
hetrozygous genotypes. At 15 of 17 candidate deletion loci, fluorescence
intensity data for one or more SNPs clustered into additional groups that
corresponded to the predicted deletion genotypes.
</P>
<P>
The authors used PCR amplification to query 60 loci for which the pattern of
genotypes suggested multiple individuals with homozygous deletions.  Variants
were considered confirmed if the pattern of amplification success and failure
matched prediction across a set of 12-24 individuals. The authors confirmed 51
of 60 candidate variants by this criterion.
</P>
<P>
The authors performed quantitative PCR in all 269 HapMap DNA samples for 11
candidate deletions that overlapped the coding exons of genes and that were
discovered in many individuals. At 10/11 loci, the authors observed three
discrete clusters, identifying individuals with zero, one and two gene copies.
All 60 trios displayed Mendelian inheritance for the ten deletions, as well as
Hardy-Weinberg equilibrium in all four populations surveyed, and transmission
rates close to 50%. This suggests that the deletions behave as a stable,
heritable genetic polymorphism.
</P>

<H3>Redon analysis of HapMap data</H3>
The authors utilized numerous quality meaures, including repeated experiments
on the WGTP array for 82 individuals and on the 500K EA array for 15
individuals. The average false-positive rate per experiment was held beneath
5%. Aberrant chromosomes were removed from the analysis.    
</P>

<H2>Credits</H2>
<P>
Thanks to Lars Feuk at <A HREF="https://www.sickkids.ca/" TARGET=_blank>The
Hospital for Sick Children in Toronto</A> for providing these data in hg18
coordinates.
</P>


<H2>References</H2>
<P>
Feuk L, Carson AR, Scherer SW.
<A HREF="https://www.nature.com/articles/nrg1767"
TARGET=_blank> Structural variation in the human genome</A>.
<em>Nat Rev Genet</em>. 2006 Feb;7(2):85-97.
</P>
<P>
Conrad DF, Andrews TD, Carter NP, Hurles ME, Pritchard JK.
<A HREF="https://www.nature.com/articles/ng1697"
TARGET=_blank>A high-resolution survey of deletion polymorphism in the human
genome</A>. <em>Nat Genet.</em> 2006 Jan;38(1):75-81. </P>
<P>
Hinds DA, Kloek AP, Jen M, Chen X, Frazer KA.
<A HREF="https://www.nature.com/articles/ng1695"
TARGET=_blank>Common deletions and SNPs are in linkage disequilibrium in the
human genome</A>. <em>Nat Genet.</em> 2006 Jan;38(1):82-5. </P>
<P>
Iafrate AJ, Feuk L, Rivera MN, Listewnik ML, Donahoe PK, Qi Y, Scherer SW,
Lee C. <A HREF="https://www.nature.com/articles/ng1416"
TARGET=_blank>Detection of large-scale variation in the human genome</A>.
<em>Nat Genet.</em> 2004 Sep;36(9):949-51.</P>
<P>
Locke DP, Sharp AJ, McCarroll SA, McGrath SD, Newman TL, Cheng Z, Schwartz S,
Albertson DG, Pinkel D, Altshuler DM <em>et al</em>.
<A HREF="http://www.ajhg.org/AJHG/abstract/S0002-9297(07)63135-8"
TARGET=_blank>
Linkage disequilibrium and heritability of copy-number polymorphisms within
duplicated regions of the human genome</A>. <em>Am J Hum Genet.</em> 2006
Aug;79(2):275-90.</P>
<P>
McCarroll SA, Hadnott TN, Perry GH, Sabeti PC, Zody MC, Barrett JC, Dallaire
S, Gabriel SB, Lee C, Daly MJ <em>et al</em>.
<A HREF="https://www.nature.com/articles/ng1696"
TARGET=_blank>Common deletion polymorphisms in the human genome</A>.
<em>Nat Genet.</em> 2006 Jan;38(1):86-92.</P>
<P>
Redon R, Ishikawa S, Fitch KR, Feuk L, Perry GH, Andrews TD, Fiegler H,
Shapero MH, Carson AR, Chen W <em>et al</em>.
<A HREF="https://www.nature.com/articles/nature05329"
TARGET=_blank>
Global variation in copy number in the human genome</A>. <em>Nature.</em> 2006
Nov 23;444(7118):444-454.
<P>
Sebat J, Lakshmi B, Troge J, Alexander J, Young J, Lundin P, Maner S, Massa H,
Walker M, Chi M <em>et al</em>.
<A HREF="https://science.sciencemag.org/content/305/5683/525"
TARGET=_blank>Large-scale copy number polymorphism in the human genome</A>.
<em>Science.</em> 2004 July 23;305(5683):525-8.</P>
<P>
Sharp AJ, Locke DP, McGrath SD, Cheng Z, Bailey JA, Vallente RU, Pertz LM,
Clark RA, Schwartz S, Segraves R <em>et al</em>. <A
HREF="https://www.cell.com/ajhg/abstract/S0002-9297(07)63135-8"
TARGET=_blank>Segmental duplications and copy number variation in the human
genome</A>. <em>Am J Hum Genet.</em> 2005 Jul;77(1):78-88.</P>
<P>
Snijders AM, Nowak N, Segraves R, Blackwood S, Brown N, Conroy J, Hamilton G,
Hindle AK, Huey B, Kimura K <em>et al</em>.
<A HREF="https://www.nature.com/articles/ng754"
TARGET=_blank>Assembly of microarrays for genome-wide measurement of DNA copy
number</A>. <em>Nat Genet.</em> 2001 Nov;29(3):263-4.</P>
<P>
Tuzun E, Sharp AJ, Bailey JA, Kaul R, Morrison VA, Pertz LM, Haugen E, Hayden
H, Albertson D, Pinkel D <em>et al</em>.
<A HREF="https://www.nature.com/articles/ng1562"
TARGET=_blank>Fine-scale structural variation of the human genome</A>. 
<em>Nat Genet.</em> 2005 Jul;37(7):727-32.</P>
Nguyen DQ, Webber C, Ponting CP.
<A HREF="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0020020"
TARGET=_blank>Bias of selection on human copy-number variants</A>.
<em>PLoS Genet</em>. 2006 Feb;2(2):e20.</P>

