<H2>Description</H2>
<P>
This track displays results of the following ChIP-chip (NimbleGen) 
gamma interferon experiments on HeLa cells:
<UL>
<LI> anti-H3K4me2, no gamma interferon</LI>
<LI> anti-H3K4me2, 30 minutes after gamma interferon</LI>
<LI> anti-H3K4me3, no gamma interferon</LI>
<LI> anti-H3K4me3, 30 minutes after gamma interferon</LI>
<LI> anti-H3ac, no gamma interferon</LI>
<LI> anti-H3ac, 30 minutes after gamma interferon</LI>
<LI> anti-H4ac, no gamma interferon</LI>
<LI> anti-STAT1, 30 minutes after gamma interferon</LI>
<LI> anti-RNA Pol2 in initiation complex, no gamma interferon</LI>
<LI> anti-RNA Pol2 in initiation complex, 30 minutes after gamma interferon</LI>
</UL></P>
<P>
ENCODE region-wide location analysis of dimethylated K4 histone H3 (HK4me2 or 
diMeH3K4), trimethylated K4 histone H3 (H3K4me3
or triMeH3K4), RNA polymerase II, acetylated histone H3 (H3ac or AcH3),
acetylated histone H4 (H4ac or AcH3) and STAT1 was conducted with ChIP-chip
using chromatin extracted from HeLa cells induced for 30 minutes with
gamma interferon as well as uninduced cells. </P>

<H2>Methods</H2>
<P>
Chromatin from both induced and uninduced HeLa cells was separately
cross-linked, precipitated with different antibodies, sheared,
amplified and hybridized to an oligonucleotide tiling array produced by 
<A HREF="https://sequencing.roche.com/en/products-solutions/by-category/target-enrichment/hybridization.html" TARGET=_blank>NimbleGen Systems</A>. 
The array includes non-repetitive sequences within the 44 ENCODE
regions tiled from NCBI Build 35 (UCSC hg17) with 50-mer probes at 38
bp interval. 
<P>
For H3K4me3 and Pol2,
intensity values for biological replicate arrays were combined after
quantile normalization using 
<A HREF="http://www.r-project.org/" TARGET=_blank>R</A>.  
The averages of the quantile
normalized intensity values for each probe were then median-scaled and
Loess-normalized using R to obtain the adjusted logR-values.
<P>
For all the other markers, each replicate was Loess-normalized and
combined after intensity-based quantile normalization.
The average log ratio for each probe was derived using linear model fitting with R.  
The peak positions were identified using the 
<A TARGET=_BLANK HREF="https://laulab.ucsf.edu/data-and-software">Mpeak</A> program.
<!-- no more downloads on lab website
The log ratio for each replicate is available for download from the
<A TARGET=_BLANK HREF="http://licr-renlab.ucsd.edu/download.html">Ren Lab download page</A>.
-->
</P>

<H2>Verification</H2>
<P>
Three biological replicates were used to generate the track for each
factor at each time point with the exception of RNA Pol2 uninduced,
where only two biological replicates were used.</P>

<H2>Credits</H2>
<P>
The data for this track were generated at the 
<A HREF="http://bioinformatics-renlab.ucsd.edu/rentrac/" TARGET=_blank>Ren Lab</A>, 
Ludwig Institute for Cancer Research at UC San Diego.</P>

