<H2>Description</H2>
<P>
This track shows the transcript map of signal intensity (estimating RNA 
abundance) for the following, hybridized to the Affymetrix ENCODE 
oligonucleotide microarray:
<UL>
<LI>human neutrophil (PMN) total RNA (10 biological samples from different individuals)
<LI>human placental Poly(A)+ RNA (3 biological replicates)
<LI>total RNA from human NB4 cells (4 biological replicates), each sample
divided into three parts and treated as follows:  untreated, treated with 
retinoic acid (RA), and treated with 12-O-tetradecanoylphorbol-13 acetate (TPA)
(three out of the four original samples). Total RNA was extracted from each 
treated sample and applied to arrays in duplicate (2 technical replicates). 
</UL></P>
<P>
The human NB4 cell can be made to differentiate towards either monocytes (by
treatment with TPA) or neutrophils (by treatment with RA). See Kluger
<em>et al</em>., 2004 in the References section for more details about the
differentiation of hematopoietic cells.</P>
<P>
This array has 25-mer oligonucleotide probes tiled 
approximately every 22 bp, covering all the non-repetitive DNA sequence 
of the ENCODE regions. The transcript map is a combined signal for both 
strands of DNA. This is derived from the number of different biological 
samples indicated above, each with at least two technical replicates.</P> 
<P>
See the following NCBI Gene Expression Omnibus (GEO) accessions for details 
of experimental protocols:
<UL>
<LI> ENCODE Transcript Mapping for Human Neutrophil (PMN) Total RNA: 
<A HREF="https://www.ncbi.nlm.nih.gov/projects/geo/query/acc.cgi?acc=GSE2678"
TARGET=_blank>GSE2678</A>
<LI> ENCODE Transcript Mapping for Human Placental Poly(A)+ RNA: 
<A HREF="https://www.ncbi.nlm.nih.gov/projects/geo/query/acc.cgi?acc=GSE2671"
TARGET=_blank>GSE2671</A>
<LI> ENCODE Transcript Mapping for Total RNA from Human NB4 Cells untreated, 
treated with RA, and treated with TPA: 
<A HREF="https://www.ncbi.nlm.nih.gov/projects/geo/query/acc.cgi?acc=GSE2679"
TARGET=_blank>GSE2679</A>
</UL></P>

<H2>Display Conventions and Configuration</H2>
<P>
This annotation follows the display conventions for composite 
&quot;wiggle&quot; tracks. The subtracks within this annotation 
may be configured in a variety of ways to highlight different aspects of the 
displayed data. The graphical configuration options are shown at the top of 
the track description page, followed by a list of subtracks.  To display only 
selected subtracks, uncheck the boxes next to the tracks you wish to hide. 
For more information about the graphical configuration options, click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph
configuration help</A> link.</P>
<P>
Color differences among the subtracks are arbitrary. They provide a
visual cue for distinguishing between the different data samples.</P>

<H2>Methods</H2>
<P>

The data from biological & technical replicates were
quantile-normalized to each other and then median scaled
to 25. Using a 101 bp sliding window centered on each
oligonucleotide probe, a signal map estimating RNA
abundance was generated by computing the pseudomedian
signal of all PM-MM pairs (median of pairwise PM-MM
averages) within the window, including replicates.

<H2>Verification</H2>
<P>
Independent biological replicates (as indicated above) were generated,
and each was hybridized to at least two different arrays
(technical replicates).  Transcribed regions were then identified using a
signal threshold of 90 percentile of signal intensities, as well as a maximum
gap of 50 bp and a minimum run of 50 bp (between oligonucleotide positions).
Transcribed regions, as determined by individual biological samples, were
compared to ensure significant overlap.</P>

<H2>Credits</H2>
<P>
These data were generated and analyzed by the Yale/Affymetrix 
collaboration between the labs of Michael Snyder, Mark Gerstein and 
Sherman Weissman at Yale University and Tom Gingeras at Affymetrix.</P>

<H2>References</H2>
<P>
Bertone P, Stolc V, Royce TE, Rozowsky JS, Urban AE, Zhu X, 
Rinn JL, Tongprasit W, Samanta M, Weissman S <em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15539566&query_hl=7"
TARGET=_blank>Global identification of human transcribed sequences with 
genome tiling arrays</A>. 
<i>Science</i>. 2004 Dec 24;306(5705):2242-6.</P>
<P>
Cheng J, Kapranov P, Drenkow J, Dike S, Brubaker S, Patel S, 
Long J, Stern D, Tammana H, Helt G <em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15790807&query_hl=5"
TARGET=_blank>Transcriptional maps of 10 human chromosomes at 5-nucleotide 
resolution</A>. 
<i>Science</i>. 2005 May 20;308(5725):1149-54.</P>
<P>
Kapranov P, Cawley SE, Drenkow J, Bekiranov S, Strausberg RL, Fodor SP, 
Gingeras TR.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=11988577&query_hl=5"
TARGET=_blank>Large-scale transcriptional activity in chromosomes 21 and 
22</A>. 
<i>Science</i>. 2002 May 3;296(5569):916-9.</P>
<P>
Kluger Y, Tuck DP, Chang JT, Nakayama Y, Poddar R, Kohya N, 
Lian Z, Ben Nasr A, Halaban HR, Krause DS <em>et al</em>.
<A HREF="https://www.pnas.org/content/101/17/6508.full"
TARGET=_blank>Lineage specificity of gene expression patterns</A>. 
<i>Proc Natl Acad Sci U S A</i>. 2004 April 27;101(17):6508-13.</P>
<P>
Rinn JL, Euskirchen G, Bertone P, Martone R, Luscombe NM, 
Hartman S, Harrison PM, Nelson FK, Miller P, Gerstein M <em>et al.</em>
<A HREF="http://genesdev.cshlp.org/content/17/4/529.full"
TARGET=_blank>The transcriptional activity of human Chromosome 22</A>. 
<i>Genes Dev</i>. 2003 Feb 15;17(4):529-40.</P>
