<H2>Description</H2>
<P>
This track displays human-centric multiple sequence alignments 
and conserved elements in the 
ENCODE regions for the 36 vertebrates included in the 
<A TARGET=_BLANK HREF="http://hgdownload.soe.ucsc.edu/goldenPath/hg18/encode/MSA/DEC-2007/">December 2007 ENCODE MSA freeze</A>.
The alignments in this track were generated using the
<b>Threaded Blockset Aligner (TBA)</b>.
The conservation subtracks display conserved elements generated by two methods: <b>BinCons</b>,
a binomial-based method that calculates a conservation score in 
sliding windows with normalization for phylogenetic bias, and <b>Chai Cons</b>, 
a DNA structure-informed constraint detection algorithm that
uses hydroxyl radical cleavage patterns as a measure of DNA structure.
<P>
The multiple alignments are based on comparative sequence data generated for the ENCODE project
from NIH Intramural Sequencing Center (NISC) 
    as well as whole-genome assemblies residing at UCSC, as listed:
<P>
<BLOCKQUOTE><TABLE BORDER=1 CELLPADDING=4 BORDERCOLOR="#aaaaaa">
<TR ALIGN=left><TH>Organism</TH><TH>Species</TH><TH>Version</TH></TR>
<TR ALIGN=left><TD>Human</TD><TD><EM>Homo sapiens</EM></TD>
    <TD> UCSC <A HREF="../cgi-bin/hgGateway?db=hg18"
    TARGET=_blank>hg18</A></TD></TR>
<TR ALIGN=left><TD>Armadillo</TD><TD><EM>Dasypus novemcinctus</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Baboon</TD><TD><EM>Papio anubis</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Bat (rfbat)</TD><TD><EM>Rhinolophus ferrumequinum</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Bat (sbbat)</TD><TD><EM>Myotis lucifugus</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Cat</TD><TD><EM>Felis catus</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Chicken</TD><TD><EM>Gallus gallus</EM></TD>
    <TD> UCSC <A HREF="../cgi-bin/hgGateway?db=galGal3"
    TARGET=_blank>galGal3</A></TD></TR>
<TR ALIGN=left><TD>Chimpanzee</TD><TD><EM>Pan troglodytes</EM></TD>
    <TD> UCSC <A HREF="../cgi-bin/hgGateway?db=panTro2"
    TARGET=_blank>panTro2</A></TD></TR>
<TR ALIGN=left><TD>Colobus Monkey</TD><TD><EM>Colobus guereza</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Cow</TD><TD><EM>Bos taurus</EM></TD>
    <TD> UCSC <A HREF="../cgi-bin/hgGateway?db=bosTau3"
    TARGET=_blank>bosTau3</A></TD></TR>
<TR ALIGN=left><TD>Dog</TD><TD><EM>Canis familiaris</EM></TD>
    <TD> UCSC <A HREF="../cgi-bin/hgGateway?db=canFam2"
    TARGET=_blank>canFam2</A></TD></TR>
<TR ALIGN=left><TD>Dusky titi</TD><TD><EM>Callicebus moloch</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Elephant</TD><TD><EM>Loxodonta africana</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Flying Fox</TD><TD><EM>Pteropus vampyrus</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Galago</TD><TD><EM>Otolemur garnettii</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Gibbon</TD><TD><EM>Nomascus leucogenys leucogenys</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Guinea pig</TD><TD><EM>Cavia porcellus</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Hedgehog</TD><TD><EM>Atelerix albiventris</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Horse</TD><TD><EM>Equus caballus</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Macaque</TD><TD><EM>Macaca mulatta</EM></TD>
    <TD>UCSC <A HREF="../cgi-bin/hgGateway?db=rheMac2"
    TARGET=_blank>rheMac2</A></TD></TR>
<TR ALIGN=left><TD>Marmoset</TD><TD><EM>Callithrix jacchus</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Mouse</TD><TD><EM>Mus musculus</EM></TD>
    <TD>UCSC <A HREF="../cgi-bin/hgGateway?db=mm9"
    TARGET=_blank>mm9</A></TD></TR>
<TR ALIGN=left><TD>Mouse Lemur</TD><TD><EM>Microcebus murinus</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Opossum</TD><TD><EM>Monodelphis domestica</EM></TD>
    <TD> UCSC <A HREF="../cgi-bin/hgGateway?db=monDom4"
    TARGET=_blank>monDom4</A></TD></TR>
<TR ALIGN=left><TD>Orangutan</TD><TD><EM>Pongo abelii</EM></TD>
    <TD>  UCSC <A HREF="../cgi-bin/hgGateway?db=ponAbe2"
    TARGET=_blank>ponAbe2</A></TD></TR>
<TR ALIGN=left><TD>Owl Monkey</TD><TD><EM>Aotus nancymaae</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Platypus</TD><TD><EM>Ornithorhychus anatinus</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Rabbit</TD><TD><EM>Oryctolagus cuniculus</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Rat</TD><TD><EM>Rattus norvegicus</EM></TD>
    <TD>UCSC <A HREF="../cgi-bin/hgGateway?db=rn4"
    TARGET=_blank>rn4</A></TD></TR>
<TR ALIGN=left><TD>Rock hyrax</TD><TD><EM>Procavia capensis</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Shrew</TD><TD><EM>Sorex araneus</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Squirrel monkey</TD><TD><EM>Saimiri boliviensis boliviensis</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Squirrel</TD><TD><EM>Spermophilus tridecemlineatus</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Tenrec</TD><TD><EM>Echinops telfairi</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Tree shrew</TD><TD><EM>Tupaia belangeri</EM></TD>
    <TD> NISC</TD></TR>
<TR ALIGN=left><TD>Vervet monkey</TD><TD><EM>Chlorocebus aethiops</EM></TD>
    <TD> NISC</TD></TR>
</TABLE></BLOCKQUOTE><BR>

<H2>Display Conventions and Configuration</H2>
<P>
In full display mode, this track shows pairwise alignments
of each species aligned to the human genome.  
In dense mode, the alignments are depicted using a gray-scale
density gradient.  The checkboxes in the track configuration section allow
the exclusion of species from the pairwise display. To view detailed 
information about the alignments at a specific position, zoom the display in 
to 30,000 or fewer bases, then click on the alignment.
</P>

<H3>Gap Annotation</H3>
<P>
The <EM>Display chains between alignments</EM> configuration option
enables display of gaps between alignment blocks in the pairwise alignments in
a manner similar to the Chain track display.  The following
conventions are used:
<UL>
<LI><B>Single line:</B> no bases in the aligned species. Possibly due to a
lineage-specific insertion between the aligned blocks in the $organism genome
or a lineage-specific deletion between the aligned blocks in the aligning
species.
<LI><B>Double line:</B> aligning species has one or more unalignable bases in
the gap region. Possibly due to excessive evolutionary distance between
species or independent indels in the region between the aligned blocks in both
species.
<LI><B>Pale yellow coloring:</B> aligning species has Ns in the gap region.
Reflects uncertainty in the relationship between the DNA of both species, due
to lack of sequence in relevant portions of the aligning species.
</UL></P>

<H3>Genomic Breaks</H3>
<P>
Discontinuities in the genomic context (chromosome, scaffold or region) of the
aligned DNA in the aligning species are shown as follows:
<UL>
<LI>
<B>Vertical blue bar:</B> represents a discontinuity that persists indefinitely
on either side, <em>e.g.</em> a large region of DNA on either side of the bar
comes from a different chromosome in the aligned species due to a large scale
rearrangement.
<LI>
<B>Green square brackets:</B> enclose shorter alignments consisting of DNA from
one genomic context in the aligned species nested inside a larger chain of
alignments from a different genomic context. The alignment within the
brackets may represent a short misalignment, a lineage-specific insertion of a
transposon in the $organism genome that aligns to a paralogous copy somewhere
else in the aligned species, or other similar occurrence.
</UL></P>

<H3>Base Level</H3>
<P>
When zoomed-in to the base-level display, the track shows the base
composition of each alignment.
The numbers and symbols on the Gaps
line indicate the lengths of gaps in the $organism sequence at those
alignment positions relative to the longest non-$organism sequence.
If there is sufficient space in the display, the size of the gap is shown.
If the space is insufficient and the gap size is a multiple of 3, a
&quot;*&quot; is displayed; other gap sizes are indicated by &quot;+&quot;.</P>
<P>
Codon translation is available in base-level display mode if the
displayed region is identified as a coding segment. To display this annotation,
select the species for translation from the pull-down menu in the Codon
Translation configuration section at the top of the page. Then, select one of
the following modes:
<UL>
<LI>
<B>No codon translation:</B> the gene annotation is not used; the bases are
displayed without translation.
<LI>
<B>Use default species reading frames for translation:</B> the annotations from the genome
displayed
in the <EM>Default species for translation</EM>; pull-down menu are used to
translate all the aligned species present in the alignment.
<LI>
<B>Use reading frames for species if available, otherwise no translation:</B> codon
translation is performed only for those species where the region is
annotated as protein coding.
<LI><B>Use reading frames for species if available, otherwise use default species:</B>
codon translation is done on those species that are annotated as being protein
coding over the aligned region using species-specific annotation; the remaining
species are translated using the default species annotation.
</UL></P>
<P>
Codon translation uses the following gene tracks as the basis for
translation, depending on the species chosen.
Species listed in the row labeled &quot;None&quot; do not have
species-specific reading frames for gene translation.

<BLOCKQUOTE><TABLE BORDER=1 CELLPADDING=4 BORDERCOLOR="#aaaaaa">
<TR ALIGN=left><TD><B>Gene Track</B></TD><TD><B>Species</B></TD></TR>
<TR ALIGN=left><TD>Gencode Genes</TD><TD>human</TD></TR>
<TR ALIGN=left><TD>UCSC Genes</TD><TD>mouse</TD></TR>
<TR ALIGN=left><TD>Known Genes</TD><TD>rat</TD></TR>
<TR ALIGN=left><TD>RefSeq Genes</TD><TD>chimp</TD></TR>
<TR ALIGN=left><TD>Ensembl Genes</TD><TD>rhesus, opossum</TD></TR>
<TR ALIGN=left><TD>None</TD><TD>the remaining 30 species</TD></TR>
</TABLE>
</BLOCKQUOTE></P>

<H2>Methods</H2>

<H3>TBA</H3> 
TBA was used to align sequences in the December 2007 ENCODE sequence data 
freeze. Multiple alignments were seeded from a series of combinatorial pairwise 
blastz alignments (not referenced to any one species). The specific 
combinations were determined by the
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/hg18/encode/MSA/DEC-2007/tree_4d.tba.nh"
TARGET=_blank>species guide tree</A>. 
The resulting multiple alignments were projected onto the human reference
sequence. 
<H3>BinCons</H3> 
<P>
The <em>binCons</em> score is based on the cumulative binomial probability of
detecting the observed number of identical bases (or greater) in
sliding 25 bp windows (moving one bp at a time) between the
reference sequence and each other species, given the neutral rate
at four-fold degenerate sites. Neutral rates are calculated
separately at each targeted region. For targets with no gene annotations,
the average percent identity across all alignable sequence was instead used
to weight the individual species binomial scores; this latter
weighting scheme was found to closely match 4D weights.
Clusters of bases
that exceeded the given conservation score threshold were designated
as conserved elements.
The minimum length of a conserved element is 25
bases. Strict cutoffs were used: if even one base fell below the
conservation score threshold, it separates an element into two distinct
regions.
Regions reported here exceed a 5% False Discovery Rate 
threshold, using a window size of 7 bases.
More details on binCons can be found in Margulies <em>et. al</em>. (2003)
cited below.
<H3>Chai</H3> 
<em>Chai</em> is a DNA structure-informed evolutionary conservation algorithm that works in a 
manner analogous to the primary sequence-based <em>binCons</em>.
Instead of computing the binomial probability of observed base substitutions between species, 
<em>Chai</em> calculates the difference between DNA structural profiles as a measure of similarity.  
Single nucleotide resolution structure profiles for genomic DNA are predicted using the
algorithm described in Greenbaum <em>et. al</em> (2007), below.  
Regions reported here exceed a 5% False Discovery Rate threshold.
<H2>Credits</H2>
<P>
The TBA multiple alignments were created by Gayle McEwen & Elliott Margulies of NHGRI.  
<P>
BinCons was developed by Elliott Margulies (Margulies et al. 2003).
<P>
Chai was developed by Steve Parker & Tom Tullius (Boston University), Elliott Margulies(NHGRI) and Loren Hansen (NCBI).
</P>
<P>
The programs Blastz and TBA, which were used to generate the alignments, were
provided by Minmei Hou, Scott Schwartz and Webb Miller of the 
<A HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank>Penn State Bioinformatics 
Group</A>.</P>
<P>The phylogenetic tree is based on Murphy et al. (2001).
</P>

<H2>References</H2>
<P>
Blanchette M, Kent WJ, Riemer C, Elnitski L, Smit AF, Roskin KM,
Baertsch R, Rosenbloom K, Clawson H, Green ED, <EM>et al.</EM>
<A HREF="https://genome.cshlp.org/content/14/4/708.abstract"
TARGET=_blank>Aligning multiple genomic sequences with the threaded blockset aligner</A>.
<EM>Genome Res</EM>. 2004 Apr;14(4):708-15.
</P>
<P>
Chiaromonte F, Yap VB, Miller W.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=11928468&dopt=Abstract"
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>.
<EM>Pac Symp Biocomput</EM>. 2002;:115-26.
</P>
Greenbaum JA, Pang B, Tullius TD. 
<A HREF="https://genome.cshlp.org/content/17/6/947.full">
Construction of a genome-scale structural map at single-nucleotide resolution</A>. 
<EM>Genome Res</EM>. 2007 Jun;17(6):947-53. 
<P>
Margulies EH, Blanchette, M, NISC Comparative Sequencing Program, 
Haussler, D and Green, ED. 
<A HREF="https://genome.cshlp.org/content/13/12/2507.abstract "
TARGET=_blank>Identification and characterization of multi-species conserved sequences</A>. 
<em>Genome Res</em>. 2003 Dec;13(12): 2507-18.
</P>
<P>
Murphy WJ, Eizirik E, O'Brien SJ, Madsen O, Scally M, Douady CJ, Teeling E,
Ryder OA, Stanhope MJ, de Jong WW, Springer MS.
<A HREF="https://science.sciencemag.org/content/294/5550/2348"
TARGET=_blank>Resolution of the early placental mammal radiation using Bayesian phylogenetics</A>.
<EM>Science</EM>. 2001 Dec 14;294(5550):2348-51.
</P>
<P>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC,
Haussler D, Miller W. 
<A HREF="https://genome.cshlp.org/content/13/1/103.abstract"
TARGET=_blank>Human-mouse alignments with BLASTZ</A>.
<EM>Genome Res</EM>. 2003 Jan;13(1):103-7.

</P>
