<H2>Overview</H2>
<P>
This super-track combines related tracks of ChIP-chip data generated
by the Yale ENCODE group.
ChIP-chip, also known as genome-wide location analysis, is a technique for
isolation and identification of DNA sequences bound by specific proteins in
cells, including histones. Histone methylation and acetylation serves as a
stable genomic imprint
that regulates gene expression and other epigenetic
phenomena.  These histones are found in transcriptionally active domains
called euchromatin.</P>

<P>
These tracks contain ChIP-chip data of multiple transcription factors such as 
STAT1 and
histones in multiple cell lines such as 
<A HREF="https://www.atcc.org/ATCCAdvancedCatalogSearch/ProductDetails/tabid/452/Default.aspx?ATCCNum=CCL-2.2&Template=cellBiology" 
TARGET=_blank>HelaS3</A>
(cervix epithelial adenocarcinoma). Data are displayed as signals, p-values
and site predictions, as well as Regulatory
Factor Binding Regions (RFBR) predictions.
</P>

<H2>Credits</H2>
<P>
These data were generated and analyzed by the labs of Michael
Snyder, Mark Gerstein and Sherman Weissman at Yale University. The PCR
Amplicon arrays were manufactured by Bing Ren's lab at UCSD.</P>
<P>
The RFBR data set was made available by the Transcriptional Regulation Group 
of the
ENCODE Project Consortium. The RFBR cluster and desert tracks were generated
by Zhengdong Zhang from Mark Gerstein's group at Yale University.
</P>

<H2>References</H2>
<P>
Cawley S, Bekiranov S, Ng HH, Kapranov P, Sekinger EA, Kampa D,
Piccolboni A, Sementchenko V, Cheng J <em>et al</em>.
<A HREF="https://www.cell.com/cell/fulltext/S0092-8674(04)00127-8"
TARGET=_blank>Unbiased mapping of transcription factor binding sites along
human chromosomes 21 and 22 points to widespread regulation of noncoding
RNAs</A>. <em>Cell</em>. Feb 20;116(4):499-509.</P>
<P>
Euskirchen G, Royce TE, Bertone P, Martone R, Rinn JL, Nelson
FK, Sayward F, Luscombe NM, Miller P <em>et al</em>.
<A HREF="https://mcb.asm.org/content/24/9/3804.full?view=long&pmid=15082775"
TARGET=_blank> CREB binds to multiple loci on human chromosome 22</a>.
<em>Mol Cell Biol</em>. 2004 May;24(9):3804-14. </P>
<P>
Luscombe NM, Royce TE, Bertone P, Echols N, Horak CE, Chang
JT, Snyder M, Gerstein M.
<A HREF="https://academic.oup.com/nar/article/31/13/3477/2904250"
TARGET=_blank>ExpressYourself: A modular platform for processing and
visualizing microarray data</A>.
<em>Nucleic Acids Res</em>. 2003 Jul 1;31(13):3477-82.</P>
<P>
Martone R, Euskirchen G, Bertone P, Hartman S, Royce TE,
Luscombe NM, Rinn JL, Nelson FK, Miller P <em>et al</em>.
<A HREF="https://www.pnas.org/content/100/21/12247.full"
TARGET=_blank>Distribution of NF-kappaB-binding sites across human chromosome
22</A>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Oct 14;100(21):12247-52.</P>
<P>
Quackenbush J.
<A HREF="https://www.nature.com/articles/ng1032"
TARGET=_blank>Microarray data normalization and transformation</A>.
<em>Nat Genet</em>. 2002 Dec;32 Suppl:496-501. </P>
<P>
Efron B. <A HREF="http://www.stat.cmu.edu/~jiashun/Teaching/F08STAT756/Lectures/Efron.pdf"
TARGET=_blank>Large-Scale Simultaneous Hypothesis Testing: The Choice of a Null
Hypothesis</A>.
<em>J Am Stat Assoc</em>. 2004;99(465):96-104. </P>
<P>
Zhang ZD, Paccanaro A, Fu Y, Weissman S, Weng Z, Chang J, Snyder M,
Gerstein M.
<A HREF="https://genome.cshlp.org/content/17/6/787.full"
TARGET=_blank>Statistical analysis of the genomic distribution and
correlation of regulatory elements in the ENCODE regions</A>.
<em>Genome Res</em>. 2007 Jun;17(6):787-97. </P>

