<H2>Overview</H2>
<P>
This super-track combines related tracks of ChIP-chip data generated by
the Wadelius lab at Uppsala University, Sweden.
ChIP-chip, also known as genome-wide location analysis, is a technique for
isolation and identification of DNA sequences bound by specific proteins in
cells, including histones. Histone methylation and acetylation serves as a
stable genomic imprint
that regulates gene expression and other epigenetic
phenomena.  These histones are found in transcriptionally active domains
called euchromatin.</P>
<P>
These tracks contain ChIP-chip data for transcription factors (such as HNF-3b)
and acetylated histone H3 and H4 in cell lines including HepG2 (liver carcinoma). 
Experiments were also performed after cell treatment with Na-butyrate.</P>

<H2>Credits</H2>
<P>
These experiments were performed in the
<A HREF="https://igp.uu.se/research/genetics_genomics/wadelius/"
TARGET=_blank>Claes Wadelius lab</A>, Department of Genetics and Pathology,
Rudbeck Laboratory, Uppsala University. The statistical analysis was done at the
<A HREF="http://bioinf.icm.uu.se/" TARGET=_blank>Linnaeus Centre for
Bioinformatics</A> at Uppsala University.  Microarrays were produced at the
<A HREF="https://www.sanger.ac.uk/" TARGET=_blank>Sanger Institute</A>.
</P>

<H2>References</H2>
<P>
Ameur A, Yankovski V, Enroth S, Spjuth O, Komorowski J.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16455749"
TARGET=_blank>The LCB Data Warehouse</A>.
<em>Bioinformatics</em>. 2006 Apr 15;22(8):1024-6. </P>
<P>
Rada-Iglesias A, Wallerman O, Koch C, Ameur A, Enroth S, Clelland G, Wester K,
Wilcox S, Dovey OM, Ellis PD <em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16221759"
TARGET=_blank>Binding sites for metabolic disease related transcription factors
inferred at base pair resolution by chromatin immunoprecipitation and genomic
microarrays</A>.
<em>Hum Mol Genet</em>. 2005 Nov 15;14(22):3435-47.</P>
<P>
Smyth GK.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16646809" TARGET=_blank>
Linear models and empirical Bayes methods for assessing differential expression
in microarray experiments</A>.
<em>Stat Appl Genet Mol Biol.</em> 2004;3:Article3.</P>
<P>
Yang YH, Dudoit S, Luu P, Lin DM, Peng V, Ngai J, Speed TP.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=11842121"
TARGET=_blank>Normalization for cDNA microarray data: a robust composite method
addressing single and multiple slide systematic variation</A>.
<em>Nucleic Acids Res</em>. 2002 Feb 15;30(4):e15. </P>
