<H2>Description</H2>
<P>
The alignments in this track were generated using <A TARGET=_BLANK HREF="http://www.ebi.ac.uk/~bjp/pecan"><b>PECAN</b></a>,
a consistency-based multiple aligner.
The conservation subtracks display conservation scores and conserved elements generated by
<A TARGET=_BLANK HREF="http://mendel.stanford.edu/SidowLab/downloads/GERP/index.html"><b>GERP</b></a>.
GERP scores each column of the alignment by quantifying rejected substitutions. These scores
are then used to find regions of the alignment where high scores accumulate.
<P>
The multiple alignments are based on a set of 9-way EPO (Enredo-Pecan-Ortheus) alignments for
all high-coverage placental mammal genomes. 22 low coverage genomes (including guinea pig) were
mapped onto the alignments using the BlastZ-net alignments to the human genome. Insertions in the
low coverage genomes are ignored, i.e. low-coverage genome-specific sequence is removed from the
alignments.
<P>
<P>
The multiple alignments are based on whole-genome assemblies residing at UCSC for high coverage genomes and
Ensembl assemblies for low-coverage genomes. Ensembl adds an extra layer to the assembly in order to accommodate
gene models. The species are:
<P>
<BLOCKQUOTE><TABLE BORDER=1 CELLPADDING=4 BORDERCOLOR="#aaaaaa">
<TR ALIGN=left><TH>Organism</TH><TH>Species</TH><TH>Version</TH></TR>
<TR ALIGN=left><TD>Human</TD><TD><EM>Homo sapiens</EM></TD>
    <TD> UCSC <A HREF="../cgi-bin/hgGateway?db=hg18"
    TARGET=_blank>hg18</A></TD></TR>
<TR ALIGN=left><TD>Chimpanzee</TD><TD><EM>Pan troglodytes</EM></TD>
    <TD> UCSC <A HREF="../cgi-bin/hgGateway?db=panTro2"
    TARGET=_blank>panTro2</A></TD></TR>
<TR ALIGN=left><TD>Orangutan</TD><TD><EM>Pongo abelii</EM></TD>
    <TD>  UCSC <A HREF="../cgi-bin/hgGateway?db=ponAbe2"
    TARGET=_blank>ponAbe2</A></TD></TR>
<TR ALIGN=left><TD>Macaque</TD><TD><EM>Macaca mulatta</EM></TD>
    <TD>UCSC <A HREF="../cgi-bin/hgGateway?db=rheMac2"
    TARGET=_blank>rheMac2</A></TD></TR>
<TR ALIGN=left><TD>Mouse</TD><TD><EM>Mus musculus</EM></TD>
    <TD>UCSC <A HREF="../cgi-bin/hgGateway?db=mm9"
    TARGET=_blank>mm9</A></TD></TR>
<TR ALIGN=left><TD>Rat</TD><TD><EM>Rattus norvegicus</EM></TD>
    <TD>UCSC <A HREF="../cgi-bin/hgGateway?db=rn4"
    TARGET=_blank>rn4</A></TD></TR>
<TR ALIGN=left><TD>Dog</TD><TD><EM>Canis familiaris</EM></TD>
    <TD> UCSC <A HREF="../cgi-bin/hgGateway?db=canFam2"
    TARGET=_blank>canFam2</A></TD></TR>
<TR ALIGN=left><TD>Cow</TD><TD><EM>Bos taurus</EM></TD>
    <TD> UCSC <A HREF="../cgi-bin/hgGateway?db=bosTau3"
    TARGET=_blank>bosTau3</A></TD></TR>
<TR ALIGN=left><TD>Horse</TD><TD><EM>Equus caballus</EM></TD>
    <TD> UCSC <A HREF="../cgi-bin/hgGateway?db=equCab2"
    TARGET=_blank>equCab2</A></TD></TR>
<TR ALIGN=left><TD>Guinea pig</TD><TD><EM>Cavia porcellus</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Sloth</TD><TD><EM>Choloepus hoffmanni</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Armadillo</TD><TD><EM>Dasypus novemcinctus</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Kangaroo rat</TD><TD><EM>Dipodomys ordii</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Tenrec</TD><TD><EM>Echinops telfairi</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>European Hedgehog</TD><TD><EM>Erinaceus europaeus</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Cat</TD><TD><EM>Felis catus</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Gorilla</TD><TD><EM>Gorilla gorilla</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Elephant</TD><TD><EM>Loxodonta africana</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Mouse Lemur</TD><TD><EM>Microcebus murinus</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Bat (sbbat)</TD><TD><EM>Myotis lucifugus</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Pika</TD><TD><EM>Ochotona princeps</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Rabbit</TD><TD><EM>Oryctolagus cuniculus</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Galago</TD><TD><EM>Otolemur garnettii</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Rock hyrax</TD><TD><EM>Procavia capensis</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Flying Fox</TD><TD><EM>Pteropus vampyrus</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Shrew</TD><TD><EM>Sorex araneus</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Squirrel</TD><TD><EM>Spermophilus tridecemlineatus</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Tarsier</TD><TD><EM>Tarsius syrichta</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Tree shrew</TD><TD><EM>Tupaia belangeri</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Dolphin</TD><TD><EM>Tursiops truncatus</EM></TD>
    <TD> Ensembl</TD></TR>
<TR ALIGN=left><TD>Alpaca</TD><TD><EM>Vicugna pacos</EM></TD>
    <TD> Ensembl</TD></TR>



</TABLE></BLOCKQUOTE><BR>

<H2>Methods</H2>

<H3>PECAN</H3>
<A TARGET=_BLANK HREF="http://www.ebi.ac.uk/~bjp/pecan">PECAN</A> (v0.7), a
consistency-based multiple aligner, was used to align co-linear segments
between the 9 high-coverage genomes, as defined by
<A TARGET=_BLANK HREF="http://www.ebi.ac.uk/~jherrero/downloads/enredo">ENREDO</A> (v0.5).
The anchors for ENREDO were extracted from multiple and pairwise alignments using human,
mouse, rat, dog, cow and horse genomes. The sequences from the low-coverage genomes were
projected onto the alignments using BLASTZ-net parwise alignments between these and the human genome.
<H3>GERP</H3> 
<P>
We used <A TARGET=_BLANK HREF="http://mendel.stanford.edu/SidowLab/downloads/GERP/index.html">GERP v2.1b</a>
to score each column of the alignment. This is done by comparing the number expected with the number of
observed substitutions in every column of the alignment. These scores are then used to find regions of
the alignment whith more high conservation scores than expected by chance.
<P>

<H2>Credits</H2>
<P>
The PECAN multiple alignments were created by Kathryn Beal, Stephen Fitzgerald and Javier Herrero from the Ensembl team.
<P>
ENREDO was developed by Javier Herrero (Paten et al. 2008).
<P>
PECAN was developed by Benedict Paten (Paten et al. 2008).
<P>
GERP 2.1b was written by Eugene Davydov (Cooper et al. 2005).
</P>
<P>

<H2>References</H2>
<P>
Paten B, Herrero J, Beal K, Fitzgerald S, Birney E.
<A HREF="http://genome.cshlp.org/content/18/11/1814"
TARGET=_blank>Enredo and Pecan: genome-wide mammalian consistency-based multiple alignment with paralogs</A>.
<EM>Genome Res</EM>. 2008 Nov;18(11):1814-28.
</P>
<P>
Cooper G, Stone EA, Asimenos G, NISC Comparative Sequencing Program,
Green ED, Batzoglou S, and Sidow A.
<A HREF="http://mendel.stanford.edu/SidowLab/pdfs/2005CooperEtAlGERP.pdf"
TARGET=_blank>Distribution and intensity of constraint in mammalian genomic sequence</A>
<EM>Genome Res</EM>. 2005 Jul;15(7):901-13.
</P>
