<H2>Description</H2>
<P>
This track shows 5' cap analysis gene expression (CAGE) tags and clusters from 
FANTOM Consortium and the Riken Omics Science Center in 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=rnaExtract">RNA extracts</A> 
from different timepoints during the differentiations of human cell line THP-1.

A CAGE cluster is a region of overlapping tags with an assigned value
that represents the expression level.  

<H2>Display Conventions and Configuration</H2>
<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are 
<A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
To show only selected views, uncheck the boxes next to the ones that 
you wish to hide.
This track contains the following views:
<DL>
<DT><I>Plus</I> and <I>Minus</I> <I>Clusters</I>
<DD>These views display clusters of overlapping read mappings on the
forward and reverse genomic strands.
<DT><i>Reads</i>
<DD>The alignments view shows the individual tags (read mappings). Each tag is named with the
timepoint that it was discovered during. Different timepoints can be excluded or highlighted using
the filters. For example typing "H04" in the box and selecting "include" will only display tags from
the hour 4 timepoint.

</DL>

<H2>Methods</H2>

<P>The Clusters track was generated to help visualize the raw read
data. The goal was to assign each region of the genome a score to
indicate the likelihood of being a promoter based on CAGE data. As
noted in the Fantom Consortium et al. paper below the distribution of
reads for a particular region tends to follow a power law. For each
CAGE read the number of total reads within a 45 bp window were counted
and the number of regions of the genome with that score were
determined. Fitting a power law to that data resulted in an alpha
coefficient of 1.197 which was used to calculate the posterior
probability of a particular region of the genome. This posterior probability 
is then plotted. </P>

<P>The Reads track was generated from the alignments done by the FANTOM
Consortium. For visualization in the browser the score for the beds is
the number of reads seen multiplied by 100.</P>

<H2>Credits</H2>
<P>
These data were generated and analyzed by The FANTOM Consortium and the Riken Omics Science Center. The data files were downloaded from the Fantom Functional Annotation of the Mammalian Genome <A HREF="http://fantom.gsc.riken.jp/4/download/Tables/human/CAGE/mapping/"> download </A> area.
</P>

<H2>References</H2>
<P>
Faontom Consortium and the Riken Omics Science Center.
<A TARGET=_BLANK HREF="https://www.ncbi.nlm.nih.gov/pubmed/19377474">
The transcriptional network that controls growth arrest and differentiation in a human myeloid leukemia cell line.</A>
<EM>Nat Genetics</EM>. 2009 May; 41(5):553-62.
</P>

<P>
Kawaji H, Severin J, Lizio M, Waterhouse A, Katayama S, Irvine KM, Hume DA, 
Forrest AR, Suzuki H, Carninci P, Hayashizaki Y, Daub CO.
<A TARGET=_BLANK HREF="https://www.ncbi.nlm.nih.gov/pubmed/19374775">
The FANTOM web resource: from mammalian transcriptional landscape to its dynamic regulation.</A>
<EM>Genome Biology</EM>. 2009 Apr; 10(4):R40.
</P>



