<H2>Description</H2>
<P>
The UCSC Genes track shows gene predictions based on data
from RefSeq, Genbank, CCDS and UniProt. This is a moderately conservative set of
predictions, requiring the support of one GenBank RNA sequence plus
at least one additional line of evidence. The RefSeq RNAs are an exception to 
this, requiring no additional evidence. The track includes both protein-coding
and putative non-coding transcripts. Some of these non-coding transcripts may 
actually code for protein, but the evidence for the associated protein is weak
at best. Compared to RefSeq, this gene set has generally about 10% more 
protein-coding genes, approximately five times as many putative non-coding 
genes, and 
about twice as many splice variants.</P>

<H2>Display Conventions and Configuration</H2>

<P>
This track in general follows the display conventions for
<A HREF="../goldenPath/help/hgTracksHelp.html#GeneDisplay">gene prediction
tracks</A>.  The exons for putative noncoding genes and untranslated regions 
are represented by relatively thin blocks, while those for coding open 
reading frames are thicker.  The following color key is used:

<ul>
<li><span style="font-weight: bold;">Black</span> -- feature has a corresponding entry in the Protein
 Data Bank (PDB)</li>
<li><span style="color: #0C0C78; font-weight: bold;">Dark blue</span> -- transcript has been reviewed
 or validated by either the RefSeq, SwissProt or CCDS staff</li>
<li><span style="color: #5050A0; font-weight: bold;">Medium blue</span> -- other RefSeq transcripts</li>
<li><span style="color: #8282D2; font-weight: bold;">Light blue</span> -- non-RefSeq transcripts</li>
</ul>

</P>

<P>
This track contains an optional codon coloring
feature that allows users to quickly validate and compare gene predictions.
To display codon colors, select the <em>genomic codons</em> option from the
<em>Color track by codons</em> pull-down menu. Go to the
<A HREF="../goldenPath/help/hgCodonColoring.html">
Coloring Gene Predictions and Annotations by Codon</A> page for more
information about this feature.</P>

<H2>Methods</H2>
<P>
The UCSC Genes are built using a multi-step pipeline:  
<OL>
<LI>RefSeq and GenBank RNAs are aligned to the genome with BLAT, keeping 
only the best 
alignments for each RNA and discarding alignments of less than 98% identity.
<LI>Alignments are broken up at non-intronic gaps, with small isolated 
fragments thrown out.
<LI>A splicing graph is created for each set of overlapping alignments. This 
graph 
has an edge for each exon or intron, and a vertex for each splice site, start, 
and end.
Each RNA that contributes to an edge is kept as evidence for that edge.
Gene models from the Consensus CDS project (CCDS) are also added to the graph.
<LI>A similar splicing graph is created in the mouse, based on mouse RNA and 
ESTs. If
the mouse graph has an edge that is orthologous to an edge in the human graph, 
that is added to the evidence for the human edge.
<LI>If an edge in the splicing graph is supported by two or more human ESTs, 
it is added as evidence for the edge.
<LI>If there is an Exoniphy prediction for an exon, that is added as evidence.
<LI>The graph is traversed to generate all unique transcripts. The traversal is 
guided by the initial RNAs to avoid a combinatorial explosion in alternative 
splicing. All refSeq transcripts are output. For other multi-exon transcripts
to be output, an edge supported by at least one additional line of evidence 
beyond the RNA is required.  Single-exon genes require either two RNAs or two 
additional lines of evidence beyond the single RNA. 
<LI>Protein predictions are generated. For non-RefSeq transcripts we use the 
txCdsPredict program to determine if the transcript is protein-coding and if so,
the locations of the start and stop codons. 
The program weighs as positive evidence the length of the protein, the 
presence of a Kozak consensus sequence at the start codon, and the length of 
the orthologous predicted protein in other species.
As negative evidence it considers nonsense-mediated decay and start codons in 
any frame upstream of the predicted start codon. For RefSeq transcripts 
the RefSeq protein prediction is used directly instead of this procedure.  For CCDS proteins 
the CCDS protein is used directly.
<LI>The corresponding UniProt protein is found, if any.
<LI>The transcript is assigned a permanent "uc" accession.
</OL>

<H2>Credits</H2>
<P>
The UCSC Genes track was produced at UCSC using a computational pipeline
developed by Jim Kent, Chuck Sugnet and Mark Diekhans.  
It is based on data from NCBI
<A HREF="https://www.ncbi.nlm.nih.gov/refseq/" TARGET=_blank>RefSeq</A>,
<A HREF="https://www.uniprot.org/uniprot/" TARGET=_blank>UniProt</A> 
(including TrEMBL and TrEMBL-NEW), 
<A HREF="https://www.ncbi.nlm.nih.gov/CCDS/CcdsBrowse.cgi" TARGET=_blank>CCDS</A>, and
<A HREF="https://www.ncbi.nlm.nih.gov/genbank/"
TARGET=_blank>GenBank</A>. Our thanks to the people running these databases
and to the scientists worldwide who have made contributions to them. </P>

<H2>Data Use Restrictions</H2>
<P>
The UniProt data have the following terms of use, UniProt copyright(c) 2002 - 
2004 UniProt consortium:</P>
<P>
For non-commercial use, all databases and documents in the UniProt FTP
directory may be copied and redistributed freely, without advance
permission, provided that this copyright statement is reproduced with
each copy.</P>
<P>
For commercial use, all databases and documents in the UniProt FTP
directory except the files
<UL>
<LI>ftp://ftp.uniprot.org/pub/databases/uniprot/knowledgebase/uniprot_sprot.dat.gz
<LI>ftp://ftp.uniprot.org/pub/databases/uniprot/knowledgebase/uniprot_sprot.xml.gz
</UL>
may be copied and redistributed freely, without advance permission,
provided that this copyright statement is reproduced with each copy.
More information for commercial users can be found at the UniProt
<A HREF="https://www.uniprot.org/help/license"
TARGET=_blank>License & disclaimer</A> page.
<P>
From January 1, 2005, all databases and documents in the UniProt FTP
directory may be copied and redistributed freely by all entities,
without advance permission, provided that this copyright statement is
reproduced with each copy.</P>

<H2>References</H2>
<P>
Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J,
Wheeler DL. 
<A HREF="https://academic.oup.com/nar/article/32/suppl_1/D23/2505202" 
TARGET=_blank>GenBank: update</A>. 
<em>Nucleic Acids Res</em>. 2004 Jan 1;32:D23-6.</P>
<P>
Hsu F, Kent WJ, Clawson H, Kuhn RM, Diekhans M, Haussler D. 
<A HREF="https://academic.oup.com/bioinformatics/article/22/9/1036/200093" 
TARGET=_blank>The UCSC Known Genes</A>. 
<em>Bioinformatics</em>. 2006 May 1;22(9):1036-46.</P>
<P>
Kent WJ.
<A HREF="https://genome.cshlp.org/content/12/4/656.full" 
TARGET=_blank>BLAT - The BLAST-Like Alignment Tool</A>. 
<em>Genome Res</em>. 2002 Apr;12(4):656-64.</P>
