<H2>Description</H2>
<P>
This track shows gene predictions using the N-SCAN gene structure prediction
software provided by the Computational Genomics Lab at Washington University 
in St. Louis, MO, USA.
</P>

<H2>Methods</H2>
<P>
N-SCAN combines biological-signal modeling in the target genome sequence along
with information from a multiple-genome alignment to generate de novo gene
predictions. It extends the TWINSCAN target-informant genome pair to allow for
an arbitrary number of informant sequences as well as richer models of
sequence evolution. N-SCAN models the phylogenetic relationships between the
aligned genome sequences, context-dependent substitution rates, insertions,
and deletions.
</P>
<P>
Human N-SCAN uses mouse (mm7) as the informant
and iterative pseudogene masking. </P>

<H2>Credits</H2>
<P>
Thanks to <A HREF="https://mblab.wustl.edu/" 
TARGET=_blank>Michael Brent's Computational Genomics Group</A> at Washington 
University St. Louis for providing this data.
</P>
<p>
Special thanks for this implementation of N-SCAN to Aaron Tenney in
the Brent lab, and Robert Zimmermann, currently at Max F. Perutz
Laboratories in Vienna, Austria.
</p>


<H2>References</H2>
<P>
Gross SS, Brent MR.
<A HREF="https://www.liebertpub.com/doi/abs/10.1089/cmb.2006.13.379" TARGET=_blank>Using
multiple alignments to improve gene prediction</A>. In
<em>Proc. 9th Int'l Conf. on Research in Computational Molecular Biology</em>
(RECOMB '05):374-388 and <em>J Comput Biol</em>. 2006 Mar;13(2):379-93.
</P>
<P>
Korf I, Flicek P, Duan D, Brent MR.
<A HREF="https://academic.oup.com/bioinformatics/article/17/suppl_1/S140/261903" TARGET=_blank>Integrating genomic homology into gene structure prediction</A>.
<em>Bioinformatics</em>. 2001 Jun 1;17(90001)S140-8.</P>
<P>
van Baren MJ, Brent MR.
<A HREF="https://genome.cshlp.org/content/16/5/678"
TARGET=_blank>Iterative gene prediction and pseudogene removal improves
genome annotation</A>.
<EM>Genome Res</EM>. 2006 May;16(5):678-85.</P>
