<H2>Description</H2>
<P>
This track shows gene predictions using the N-SCAN gene structure prediction
software provided by the Computational Genomics Lab at Washington University 
in St. Louis, MO, USA.
</P>

<H2>Methods</H2>
<H3>N-SCAN</H3>
<P>
N-SCAN combines biological-signal modeling in the target genome sequence along
with information from a multiple-genome alignment to generate de novo gene
predictions. It extends the TWINSCAN target-informant genome pair to allow for
an arbitrary number of informant sequences as well as richer models of
sequence evolution. N-SCAN models the phylogenetic relationships between the
aligned genome sequences, context-dependent substitution rates, insertions,
and deletions.
</P>
<P>
Human N-SCAN uses mouse (mm7) as the informant
and iterative pseudogene masking. </P>

<H3>N-SCAN PASA-EST</H3>
<P>
N-SCAN PASA-EST combines EST alignments into N-SCAN. Similar to the conservation
sequence models in TWINSCAN, separate probability models are developed for EST
alignments to genomic sequence in exons, introns, splice sites and UTRs,
reflecting the EST alignment patterns in these regions. N-SCAN PASA-EST is more
accurate than N-SCAN while retaining the ability to discover novel genes to
which no ESTs align.
</P>
<P>
In N-SCAN PASA-EST, cDNA sequences were clustered using the PASA program
beforehand. PASA, the Program to Assemble Spliced Alignments, was created by
Brian Haas at TIGR. The algorithm assembles clusters of overlapping transcript
alignments (ESTs and full-length cDNAs) into maximal alignment assemblies,
thereby comprehensively incorporating all available transcript data and
capturing subtle splicing variations.
</P>
<P>
The PASA clusters were used as 'EST' sequences in N-SCAN PASA-EST. The
resulting gene models were updated with the input PASA clusters using the
assembly tool of the PASA pipeline. These updates consist of automatically
generated alternative splices, UTR features and sometimes merging of two gene
models. In addition, PASA assigned open reading frames to clusters that did
not overlap a gene prediction, but that did contain a full length cDNA, and
output them as 'novel genes'. Note that PASA does not use any cDNA annotation
from input but assigns the ORF itself.
</P>
<P>
No manual annotation was performed to generate any of the gene models. The
high accuracy of the set is in part due to the large number of available ESTs
and full length cDNAs.
</P>
<H2>Credits</H2>

<P>
Thanks to Michael Brent's <A HREF="https://mblab.wustl.edu/" 
TARGET=_blank>Computational Genomics Group</A> at Washington 
University St. Louis for providing these data.
</P>
<p>
Special thanks for this implementation of N-SCAN to Aaron Tenney in
the Brent lab, and Robert Zimmermann, currently at Max F. Perutz
Laboratories in Vienna, Austria.
</p>

<H2>References</H2>
<P>
Gross SS, Brent MR.
<A HREF="https://www.liebertpub.com/doi/abs/10.1089/cmb.2006.13.379" TARGET=_blank>Using
multiple alignments to improve gene prediction</A>. In
<em>Proc. 9th Int'l Conf. on Research in Computational Molecular Biology</em>
(RECOMB '05):374-388 and <em>J Comput Biol</em>. 2006 Mar;13(2):379-93.
</P>
<P>
Korf I, Flicek P, Duan D, Brent MR.
<A HREF="https://academic.oup.com/bioinformatics/article/17/suppl_1/S140/261903" TARGET=_blank>Integrating genomic homology into gene structure prediction</A>.
<em>Bioinformatics</em>. 2001 Jun 1;17(90001):S140-8.</P>
<P>
van Baren MJ, Brent MR.
<A HREF="https://genome.cshlp.org/content/16/5/678"
TARGET=_blank>Iterative gene prediction and pseudogene removal improves
genome annotation</A>.
<EM>Genome Res</EM>. 2006 May;16(5):678-85.</P>
<P>
Haas BJ, Delcher AL, Mount SM, Wortman JR, Smith RK Jr, Hannick LI, Maiti R, Ronning CM,
Rusch DB, Town CD <EM>et al</EM>.
<A HREF="https://academic.oup.com/nar/article/31/19/5654/1092988" TARGET=_blank>
Improving the <EM>Arabidopsis</EM> genome annotation using maximal transcript 
alignment assemblies.</A>
<EM>Nucleic Acids Res</EM> 2003 Oct 1;31(19):5654-66.</P>
