<H2>Description</H2>
<P>
This track displays variant base calls from the publicly released genome
sequences of several individuals:
<ul>
<li>5 Sub-Saharan African genomes sequenced by Penn State University:</li>
</ul>
<ul>
<li>!Gubi (KB1),</li>
<li>G/aq'o (NB1),</li>
<li>!Ai (MD8),</li>
<li>D#kgao (TK1),</li>
<li>Archbishop Desmond Tutu (ABTutu),</li>
</ul>
<ul>
<li>6 individuals from the 1000 Genome Project high-coverage pilot:</li>
</ul>
<ul>
<li>a CEU daughter and parents (NA12878, NA12891, NA12892) </li>
<li>a YRI daughter and parents (NA19240, NA19238, NA19239)</li>
</ul>
<ul>
<li> and independently published genomes:</li>
</ul>
<ul>
<li>Craig Venter,</li>
<li>James Watson,</li>
<li>Anonymous Yoruba individual NA18507,</li>
<li>Anonymous Han Chinese individual (YH, YanHuang Project),</li>
<li>Seong-Jim Kim (SJK),</li>
<li>Anonymous Korean individual (AK1),</li>
<li>Anonymous Irish male,</li>
<li>Gregory Lucier,</li>
<li>Stephen Quake,</li>
<li>Extinct Palaeo-Eskimo Saqqaq individual</li>
</ul>
Note: The Khoisan languages are characterized by clicks, denoting additional consonants. The ! is a palatal click, / is a dental click, and # is an alveolar click (Le Roux and White, 2004). 
</P>

<H2>Display Conventions and Configuration</H2>
<P>
Substitutions and indels are displayed as boxes.  When read frequency data are available, they
are displayed in the mouseover text (e.g. "T:8 G:3" means that 8 reads
contained a T and 3 reads contained a G at that base position), and box
colors are used to show the proportion of alleles.
In the genome browser, when viewing the forward strand of the
reference genome (the normal case), the displayed alleles are relative
to the forward strand.  When viewing the reverse strand of the
reference genome (&quot;reverse&quot; button), the displayed alleles
are reverse-complemented to match the reverse strand.  
</P>
<P>
On the details page for each variant, the alleles are given for the
forward strand of the reference genome.  Frequency and phenotype data are shown when
available.
</P>

<H2>Sources</H2>
<P>
<B>KB1, NB1, MD8, TK1, ABTutu (<A HREF="http://www.ccbb.psu.edu/" TARGET=_BLANK>Penn State</A>)</B>
<A HREF="#RefPSU">(Schuster <em>et al.</em>)</A><BR>
SNPs are from the allSNPs.txt file which can be downloaded
from <A HREF="https://usegalaxy.org/u/aun1/p/bushman">Galaxy</A>.  The indels are also 
available for download from Galaxy.
</P>

<P>
<B>CEU trio NA12878, NA12891, NA12892; YRI trio NA19240, NA19238, NA19239 
(<A HREF="https://www.internationalgenome.org/" TARGET=_BLANK>1000 Genomes Project</A>)</B>
<A HREF="#Ref1kG">(1000 Genomes)</A><BR>
The variants shown are from the 1000 Genomes Project's March 2010 
release.
The CEU variant calls were based on sequence data from the 
<A HREF="https://www.sanger.ac.uk/" TARGET=_BLANK>Wellcome Trust Sanger 
Insititute</A> and the 
<A HREF="https://www.internationalgenome.org/" 
TARGET=_BLANK>Broad Institute</A>, using the Illumina/Solexa platform.  
The YRI variant calls were based on sequence data from the
<A HREF="https://www.bcm.edu/" TARGET=_BLANK>Baylor College of Medicine 
Human Genome Sequencing Center</A> and 
<A HREF="https://www.thermofisher.com/us/en/home/brands/applied-biosystems.html" 
TARGET=_BLANK>Applied Biosystems</A>, using the SOLiD platform.  
For more information on the mapping, variant calling, filtering and 
validation, see the 
<A HREF="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/pilot_data/release/2010_03/pilot2/README_pilot2_snps"
TARGET=_BLANK>pilot 2 README</A> file.
The variant calls are available from the March 2010 release subdirectory at
<A HREF="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/release/2010_03/"
TARGET=_BLANK>EBI</A> and at
<A HREF="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/pilot_data/release/2010_03/"
TARGET=_BLANK>NCBI</A>.
</P>

<P>
<B>Craig Venter (<A HREF="https://www.jcvi.org/" TARGET=_BLANK>JCVI</A>)</B>
<A HREF="#RefVenter">(Levy <em>et al.</em>)</A><BR>
An overview is given
<A HREF="https://www.jcvi.org/about" TARGET=_BLANK>here</A>.
This subtrack contains Venter's single-base and multi-base variants 
and small (&lt; 100 bp) insertions/deletions from the file 
<A HREF="ftp://ftp.jcvi.org/pub/data/huref/HuRef.InternalHuRef-NCBI.gff"
TARGET=_BLANK>HuRef.InternalHuRef-NCBI.gff</A>,
filtered to include only Method 1 variants (where each variant was kept in its original 
form and not post-processed), and to exclude any variants that had N as an allele.
JCVI hosts a
<A HREF="https://www.jcvi.org/" TARGET=_BLANK>genome browser</A>.
</P>

<P>
<B>James Watson 
(<A HREF="http://jimwatsonsequence.cshl.edu/" TARGET=_BLANK>CSHL</A>)</B>
<A HREF="#RefWatson">(Wheeler <em>et al.</em>)</A><BR>
These single-base variants came from the file
<A HREF="ftp://ftp.hapmap.org/hapmap/jimwatsonsequence/watson_snp.gff.gz"
TARGET=_BLANK>watson_snp.gff.gz</A>.
CSHL hosts a  
<A HREF="http://jimwatsonsequence.cshl.edu/cgi-perl/gbrowse/jwsequence/" 
TARGET=_BLANK>genome browser</A>.
</P>

<P>
<B>Yoruba NA18507 
(<A HREF="https://www.illumina.com/" TARGET=_BLANK>Illumina Cambridge/Solexa</A>)</B>
<A HREF="#RefNA18507">(Bentley <em>et al.</em>)</A><BR>
Illumina released the read sequences to the 
<A HREF="https://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?study=SRP000239" 
TARGET=_BLANK>NCBI Short Read Archive</A>.  
Aakrosh Ratan in the <A HREF="http://www.bx.psu.edu/miller_lab/"
TARGET=_BLANK>Miller Lab</A> at Penn State University (PSU)
mapped the sequence reads to the reference genome and called
single-base variants and small insertions/deletions (&lt; 20 bp) using
<A HREF="http://maq.sourceforge.net/" TARGET=_BLANK>MAQ</A>.
</P>

<P>
<B>YH (<A HREF="http://yh.genomics.org.cn/" TARGET=_BLANK>YanHuang Project</A>)</B>
<A HREF="#RefYH">(Wang <em>et al.</em>)</A><BR>
The YanHuang Project released these single-base variants from the
genome of a Han Chinese individual.
The data are available from the 
<A HREF="http://yh.genomics.org.cn/" TARGET=_BLANK>YH database</A> in the file
<A HREF="http://yh.genomics.org.cn/do.downServlet?file=data/snps/yhsnp_add.gff"
TARGET=_BLANK>yhsnp_add.gff</A>.
The YanHuang Project hosts a 
<A HREF="http://yh.genomics.org.cn/mapview.jsp" TARGET=_BLANK>genome browser</A>.
</P>

<P>
<B>SJK (<A HREF="http://oia.gachon.ac.kr/eng/" 
TARGET=BLANK>GUMS</A>/<A HREF="https://www.kobic.re.kr/" TARGET=_BLANK>KOBIC)</A></B>
<A HREF="#RefSJK">(Ahn <em>et al.</em>)</A><BR>
Researchers at Gachon University of Medicine and Science (GUMS) 
and the Korean Bioinformation Center (KOBIC) 
released these single-base variants from the genome of Seong-Jin Kim.
The data are available from 
<A HREF="ftp://ftp.kobic.kr/pub/KOBIC-KoreanGenome/" TARGET=BLANK>KOBIC</A> 
in the file 
<A HREF="ftp://ftp.kobic.kr/pub/KOBIC-KoreanGenome/genetic_variations/KOREF-solexa-snp-X30_Q40d4D100.gff"
TARGET=BLANK>KOREF-solexa-snp-X30_Q40d4D100.gff</A>.
</P>

<P><B>AK1 <A HREF="http://www.gmi.ac.kr/eng/index.php">(Genomic Medicine Institute)</A></B> <A HREF="#RefAK1">(Kim et al.)</A><BR>
The variants shown are from the <A HREF="http://www.gmi.ac.kr/eng/index.php">AK1_SNP.tar.gz</A> download.
</P>
<P><B>Stephen Quake <A HREF="https://quakelab.stanford.edu/">(Stanford)</A></B> <A HREF="#refQuake">(Pushkarev et al.)</A><BR>
The variants shown are from the Trait-o-matic 
<A HREF="http://snp.med.harvard.edu/download/genotype/16">download</A>.
</P>
<P><B>Anonymous Irish male</B>
<A HREF="#refIrish">(Tong <em>et al.</em>)</A><BR/>
The SNPs shown are from the <A HREF="https://galaxyproject.org/">Galaxy</A> library,
Irish whole genome.
</P>
<P><B>Gregory Lucier</B>
<A HREF="#refLucier">(Life Technologies)</A><BR/>
The SNPs shown are from
<A HREF="http://ec2-184-73-14-243.compute-1.amazonaws.com/"
TARGET=_BLANK>Nimbus Informatics</A>.
Sequencing was done using the Life SOLiD platform.
</P>
<P><B>Palaeo-Eskimo Saqqaq individual <A HREF="https://ancientgenome.dk/">(Saqqaq Genome Project)</A></B> <A HREF="#RefSaqqaq">(Rasmussen et al.)</A><BR>
The variants shown are <A HREF="http://www.ancientgenome.dk/flatfiles/saqqaq_all.snp.gz">all the SNPs</A> found by the SNPest program, and in 
a second track the 
<A HREF="https://projects.binf.ku.dk/Palaeo-eskimo/highconfidence/">high confidence SNPs</A> from the first set.  
The allele counts are not available for these tracks but read depth is available.  The read depth was put in place of the allele counts to give a measure of 
the reliability of the call.
</P>

<H2>References</H2>
<P><A NAME="RefPSU"></A>
<B>KB1, NB1, MD8, TK1, ABTutu (Penn State)</B><BR>
Schuster SC, Miller W, Ratan A, Tomsho LP, Giardine B, Kasson LR, 
Harris RS, Petersen DC, Zhao F, Qi J, <em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/20164927"
TARGET=_BLANK>Complete Khoisan and Bantu genomes from southern Africa.</A>
<A HREF="https://www.nature.com/articles/nature08795"
TARGET=_BLANK><em>Nature.</em></A> 2010 Feb 18;463(7283):943-7.</P>

<P>
Le Roux, W., and White, A.
The voices of the San living in Southern Africa today. Cape Town: Kwla books (2004)
</P>

<P><A NAME="Ref1kG"></A>
<B>CEU trio NA12878, NA12891, NA12892; YRI trio NA19240, NA19238, NA19239 (1000 Genomes)</B><BR>
1000 Genomes Project Consortium, Durbin RM, Abecasis GR, Altshuler DL, Auton A, Brooks LD, 
Durbin RM, Gibbs RA, Hurles ME, McVean GA.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/20981092" 
TARGET=_BLANK>A map of human genome variation from population-scale sequencing.</A>
<A HREF="https://www.nature.com/articles/nature09534"
TARGET=_BLANK><em>Nature.</em></A> 2010 Oct 28;467(7319):1061-73.</P>

<P><A NAME="RefVenter"></A>
<B>Craig Venter (JCVI)</B><BR>
Levy S, Sutton G, Ng PC, Feuk L, Halpern AL, Walenz BP, Axelrod N, 
Huang J, Kirkness EF, Denisov G, <em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/17803354" 
TARGET=_BLANK>The diploid genome sequence of an individual human.</A>
<A HREF="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.0050254"
TARGET=_BLANK><em>PLoS Biol.</em></A> 2007 Sep 4;5(10):e254.</P>

<P><A NAME="RefWatson"></A>
<B>James Watson (CSHL)</B><BR>
Wheeler DA, Srinivasan M, Egholm M, Shen Y, Chen L, McGuire A, He W,
Chen YJ, Makhijani V, Roth GT, <em>et al</em>. 
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/18421352" 
TARGET=_BLANK>The complete genome of an individual by massively parallel 
DNA sequencing.</A>
<A HREF="https://www.nature.com/articles/nature06884"
TARGET=_BLANK><em>Nature.</em></A> 2008 Apr 17;452(7189):872-6.</P>

<P><A NAME="RefNA18507"></A>
<B>Yoruba NA18507 (Illumina Cambridge/Solexa)</B><BR>
Bentley DR, Balasubramanian S, Swerdlow HP, Smith GP, Milton J, Brown CG, 
Hall KP, Evers DJ, Barnes CL, Bignell HR, <em>et al.</em>
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/18987734" 
TARGET=_BLANK>Accurate whole human genome sequencing using reversible 
terminator chemistry.</A>
<A HREF="https://www.nature.com/articles/nature07517"
TARGET=_BLANK><em>Nature.</em></A> 2008 Nov 6;456(7218):53-9.</P>

<P><A NAME="RefYH"></A>
<B>YH (YanHuang Project)</B><BR>
Wang J, Wang W, Li R, Li Y, Tian G, Goodman L, Fan W, Zhang J, Li J, 
Zhang J, <em>et al.</em>
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/18987735" 
TARGET=_BLANK>The diploid genome sequence of an Asian individual.</A>
<A HREF="https://www.nature.com/articles/nature07484"
TARGET=_BLANK><em>Nature.</em></A> 2008 Nov 6;456(7218):60-5.</P>

<P><A NAME="RefSJK"></A>
<B>SJK (GUMS/KOBIC)</B><BR>
Ahn SM, Kim TH, Lee S, Kim D, Ghang H, Kim DS, Kim BC, Kim SY, Kim WY, Kim C,
<em>et al.</em>
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/19470904"
TARGET=_BLANK>The first Korean genome sequence and analysis: full genome 
sequencing for a socio-ethnic group.</A>
<A HREF="https://genome.cshlp.org/content/19/9/1622"
TARGET=_BLANK><em>Genome Res.</em></A> 2009 Sep;19(9):1622-9.</P>

<P><A NAME="RefAK1"></A>
<B>AK1 (Genomic Medicine Institute)</B><BR>
Jong-Il Kim, Young Seok Ju, Hansoo Park, Sheehyun Kim, Seonwook Lee, 
Jae-Hyuk Yi, Joann Mudge, Neil A. Miller, Dongwan Hong, Callum J. Bell,
<em>et al.</em>
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/19587683"
TARGET=_BLANK>A highly annotated whole-genome sequence of a Korean individual.</A>
<A HREF="https://www.nature.com/articles/nature08211"><em>Nature</em></A> 460, 1011-1015 (20 August 2009).</P>

<P><A NAME="refQuake"></A>
<B>Stephen Quake</B><BR>
Pushkarev D, Neff NF, Quake SR "Single-molecule Sequencing of an Individual Human Genome" <A HREF="https://www.nature.com/articles/nbt.1561"><em>Nature Biotech</em></A> 27, 847-850 (10 August 2009) doi:10.1038
<A HREF="https://web.stanford.edu/group/foundry/services/papers/Pushkarev%20Nature%20biotechnology.pdf">PDF</A></P>

<P><A NAME="refIrish"/></A>
<B>Anonymous Irish Male</B><BR>
Tong P, Prendergast JG, Lohan AJ, Farrington SM, Cronin S, Friel N, Bradley DG, Hardiman O, Evans A, Wilson JF, Loftus B.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/20822512"
TARGET=_BLANK>Sequencing and analysis of an Irish human genome.</A>
<A HREF="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2010-11-9-r91"
TARGET=_BLANK><em>Genome Biol.</em></A> 2010;11(9):R91.</P>

<P><A NAME="refLucier"/></A>
<B>Gregory Lucier</B><BR/>
Not published, data provided by Life Technologies and Nimbus Informatics.
</P>

<P><A NAME="RefSaqqaq"></A>
<B>Palaeo-Eskimo Saqqaq individual</B><BR>
Rasmussen M, Li Y, Lindgreen S, Pedersen JS, Albrechtsen A, Moltke I, Metspalu M, Metspalu E, Kivisild T, Gupta R, 
<em>et al.</em>
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/20148029">Ancient Human Genome Sequence of an Extinct Palaeo-Eskimo.</A>
<A HREF="https://www.nature.com/articles/nature08835"><em>Nature</em></A> 463, 757-762 (11 February 2010).</P>

