<html><body>

<H2>Description</H2>
<P>
This track displays genome wide localization of two transcription factors 
(USF1 and USF2) and acetylated histone H3 (H3ac) in a liver cell line (HepG2).
ChIP was performed on three biological replicates and the samples were 
hybridized to Affymetrix arrays covering the human genome at an average
resolution of 35 base pairs. This track shows average probe level intensities 
for each factor.  The companion, <em>Uppsala ChIP Sites</em> track displays 
identified positive regions for each factor.
<br><br>
The raw data for this track is available at 
<A HREF="https://www.ebi.ac.uk/arrayexpress/" TARGET=_blank>EBI ArrayExpress</A>
, as experiment E-TABM-314.
</P>
<H2>Methods</H2>
<P>
Chromatin immunoprecipitation was performed as previously described 
(Rada-Iglesias et al. 2005), but sonication conditions were optimized to 
obtain smaller fragments (approximately 300 bp), to further improve the 
resolution of our experiments. The antibodies against USF1 (H-86, sc-8983) 
and USF2 (C-20, sc-862) were from Santa Cruz biotechnology; anti-Histone 
H3 acetyl K9/14 (06-599) and normal rabbit IgG (12-370) were purchased 
from Upstate. Three completely independent biological replicates were 
performed for each antibody, obtaining the corresponding input as total 
genomic DNA reference. Hybridizations were performed using Affymetrix 
GeneChip Human Tiling 2.0R Array set (7 arrays set). Array data files for 
USF1, USF2, H3ac and IgG were normalized against corresponding Input arrays 
using Affymetrix Tiling Array Software (TAS) two-group normalization. 
An empirical Bayes algorithm was used to ensure that identified positive 
regions for USF1, USF2 and H3ac show low enrichment of IgG. 
</P>
<H2>Verification</H2>
<P>
54 regions were selected to be analyzed by qPCR using new ChIP DNA obtained 
using USF1 (C20 and H86) and USF2 antibodies. Of those 54 regions, 6 were 
negative for both USF proteins and were used to set background/cut-off 
thresholds. For the remaining 48 regions all but one was positive both 
for USF1 and USF2 (with that unique negative region being different for 
USF1 and USF2). Furthermore, there was a high correlation (R=0,79-0,85) 
between the enrichments levels determined microarray hybridizations and qPCR.
</P>
<H2>Credits</H2>
<P>
These experiments were performed in the Claes Wadelius lab 
<A HREF="https://igp.uu.se/research/genetics_genomics/wadelius/" TARGET=_blank>
Dept. of Genetics and Pathology</A>, Uppsala University, Sweden. 
Microarray hybridizations were performed at Affymetrix Inc., 
Santa Clara, USA. Data processing and statistical analysis was done at 
<A HREF="http://bioinf.icm.uu.se/" TARGET=_blank>The Linnaeus Centre for Bioinformatics</A>, Uppsala University, Sweden.
</P>
<H2>References</H2>
<P>
Rada-Iglesias A, Ameur A, Kapranov P, Enroth S, Komorowski J, Gingeras TR, 
Wadelius C. 
<A HREF="https://genome.cshlp.org/content/18/3/380" TARGET=_blank>Whole-genome
maps of USF1 and USF2 binding and histone H3 acetylation reveal 
new aspects of promoter structure and candidate genes for common human 
disorders</A>.
<EM>Genome Res.</EM> 2008 Mar;18(3):380-92.
</P>
Rada-Iglesias A, Wallerman O, Koch C, Ameur A, Enroth S, Clelland G, Wester K,
Wilcox S, Dovey OM, Ellis PD <EM>et. al.</EM>
<A HREF="https://academic.oup.com/hmg/article/14/22/3435/614346" TARGET=_blank>Binding 
sites for metabolic disease related transcription factors inferred at base 
pair resolution by chromatin immunoprecipitation and genomic microarrays</A>. 
<em>Hum Mol Genet.</em> 2005 Nov 15;14(22):3435-47.
</P>
</body></html>
