<H2>Description</H2>

<P>This track shows expression of
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=ripTgtProtein">
target RNA binding proteins (RBPs)</A>
as measured by RNA-binding protein immunoprecipitation-microarray profiling
(RIP-chip) using different
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=ripAntibody">
RIP antibodies</A>
in multiple 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType">cell lines</A>.
The <i>RBP Assoc RNA</i> view shows the genomic location of transcripts 
associated with the array probes.  
Data for this track was produced as part of the
<A TARGET=_BLANK HREF="/ENCODE/index.html">
Encyclopedia of DNA Elements (ENCODE) Project</A>,
</P>

<P>In eukaryotic organisms gene regulatory networks require an additional 
level of coordination that links transcriptional and post-transcriptional 
processes. Messenger RNAs have traditionally been viewed as passive 
molecules in the pathway from transcription to translation. However, 
it is now clear that RNA-binding proteins play a major role in 
regulating multiple mRNAs in order to facilitate gene expression patterns. 
These tracks show the associated mRNAs that co-precipitate with the 
targeted RNA-binding proteins using RIP-Chip profiling. 
</P>
<P>

<H2>Display Conventions and Configuration</H2>

<P>This track is a multi-view composite track. For each view there
are multiple subtracks that display individually in the browser.
The subtracks within this track correspond to different antibodies/target
proteins tested in different cell lines.
This track is initially released with a single view:
</P>
<DL>
<DT><i>RBP Assoc RNA</i>
<DD>The RBP Assoc RNA view shows the genomic extent of the transcripts 
associated with 
the Affymetrix Exon Array probes, shaded according to score.
</DL>
Instructions for configuring multi-view tracks are 
<A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.

<H2>Methods</H2>
<P>RBP-mRNA complexes were purified from cells grown according to the approved
<A HREF="/ENCODE/protocols/cell">ENCODE cell culture protocols </A>.
The associated messages were identified using Affymetrix 
Human Exon 1.0 ST Arrays. 
Measurements of expression at gene-level were extracted using 
Affymetrix tools, and were further processed 
to generate average fold-change and p-values for immunoprecipitation.
Enriched regions were scored in the range ~100 to ~1000, and interrogated
regions without significant signal were scored at 1.
The signal value contains the minimum log2 fold-change, the p-value
contains -log10 of the maximum p-value, and the q-value was
left at the default of -1.
</P>
<P>For additional methods detail, see 
Tenenbaum <EM>et al.</EM> 2002; Baroni <EM>et al.</EM> 2008; Penalva 
<EM>et al.</EM> 2004, below.
Details of the RIP-chip analysis methods are available
<A HREF="/ENCODE/protocols/analysis/wgEncodeSunyalbanyRnaGeneChipRbpAssocRna.txt">
here</A>.</P>

<H2>Credits</H2>

<P>These data were produced and analyzed by a collaboration between the 
<A HREF="http://sxrna.sunycnse.com/nanobio/tenenbaum/"
TARGET=_BLANK>
Tenenbaum lab</A>
at the University at Albany-SUNY, Gen*NY*Sis Center For Excellence 
in Cancer Genomics and the 
<A HREF="https://gccri.uthscsa.edu/lab/penalva-lab/" TARGET=_BLANK>
Luiz Penalva group</A>
at the Greehey Children's Cancer Research Institute, 
University of Texas Health Science Center.
</P>
<P>Contact: 
<A HREF="mailto:&#115;&#116;&#101;n&#101;&#110;&#98;&#97;u&#109;&#64;&#97;l&#98;&#97;&#110;&#121;.&#101;&#100;&#117;">
Scott Tenenbaum</A>
</P>
<!-- above address is stenenbaum at albany.edu -->

<H2>References</H2>

<P>Tenenbaum SA, Lager PJ, Carson CC, Keene JD. 
<A TARGET=_BLANK" HREF="https://www.ncbi.nlm.nih.gov/pubmed/12054896">
Ribonomics: identifying mRNA subsets in mRNP complexes using 
antibodies to RNA-binding proteins and genomic arrays. 
</A>
<EM>Methods</EM>. 2002 Feb;26(2):191-8.
</P>

<P>Baroni TE, Chittur SV, George AD, Tenenbaum SA.
<A TARGET=_BLANK" HREF="https://www.ncbi.nlm.nih.gov/pubmed/18369977">
Advances in RIP-chip analysis : RNA-binding protein 
immunoprecipitation-microarray profiling.
</A>
<EM>Methods Mol Biol</EM>. 2008;419:93-108.
</P>

<P>Penalva LO, Tenenbaum SA, Keene JD.
<A TARGET=_BLANK" HREF="https://www.ncbi.nlm.nih.gov/pubmed/14770002">
Gene expression analysis of messenger RNP complexes. 
</A>
<EM>Methods Mol Biol</EM>. 2004;257:125-34.
</P>

<P>Keene JD, Tenenbaum SA.
<A TARGET=_BLANK" HREF="https://www.ncbi.nlm.nih.gov/pubmed/12086614">
Eukaryotic mRNPs may represent posttranscriptional operons.
</A>
<EM>Mol Cell</EM>. 2002;9(6):1161-7.
</P>

<P>George AD, Tenenbaum SA.
<A TARGET=_BLANK" HREF="https://www.ncbi.nlm.nih.gov/pubmed/17114943">
MicroRNA modulation of RNA-binding protein regulatory elements.
</A>
<EM>RNA Biol</EM>. 2006;3(2):57-9. Epub 2006 Apr 1.
</P>

<H2> Data Release Policy </H2>

<P>Data users may freely use ENCODE data, but may not, without prior 
consent, submit publications that use an unpublished ENCODE dataset until 
nine months following the release of the dataset.  This date is listed in 
the <EM>Restricted Until</EM> column, above.  The full data release policy 
for ENCODE is available 
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P> 


