<h2>Description</h2>

<p>This track displays maps of chromatin state generated by the Broad/MGH
ENCODE group using ChIP-seq.  Chemical modifications (methylation, acylation)
to the histone proteins present in chromatin influence gene expression by
changing how accessible the chromatin is to transcription.
</p>

<p>The ChIP-seq method involves cross-linking histones and other DNA associated
proteins to genomic DNA within cells using formaldehyde. The cross-linked
chromatin is subsequently extracted, mechanically sheared, and
immunoprecipitated using specific antibodies. After reversal of cross-links,
the immunoprecipitated DNA is sequenced and mapped to the human reference
genome. The relative enrichment of each antibody-target (epitope) across the
genome is inferred from the density of mapped fragments.</p>

<H2>Display Conventions and Configuration</H2>

<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
ENCODE tracks typically contain one or more of the following views:
<DL>
<DT><I>Peaks</I></DT><DD> Regions of signal enrichment based on processed data
(usually normalized data from pooled replicates). ENCODE Peaks tables contain
fields for statistical significance.
Peaks for this track include a signalValue and pValue.
The signalValue represents the fold enrichment of reads across the
length of the interval, relative to random expectation.
The pValue reflects the likelihood of observing an interval of the given
length and signalValue at random. A long interval with a moderate signalValue
and a short interval with a high signalValue can therefore have the same pValue.
</DD>
<DT><I>Signal</I></DT><DD>Density graph (wiggle) of signal enrichment based on
processed data.</DD>
</DL>
Additional data that were used to generate these tracks
are located in the ENCODE <A HREF="hgTrackUi?g=wgEncodeMapability">Mappability</A> track:
<DL>
<DT><I>Alignability</I></DT><DD>
The <EM>Broad alignability</EM> track displays whether a region is made up of
mostly unique or mostly non-unique sequence. </DD>
</DL>
</P>

<h2>Methods</h2>

<P>
Cells were grown according to the approved
<A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.

<P>Chromatin immunoprecipitation was performed with each of the histone
antibodies listed above.  Isolated DNA was then end-repaired,
adapter-ligated and sequenced using Illumina Genome Analyzers.</P>

<P>Sequence reads from each IP experiment were aligned to the human reference
genome (hg18) using MAQ.  Discrete intervals of ChIP-seq fragment enrichment
were identified using a scan statistics approach, assuming a uniform
background signal.</P>

<P>More details of the experimental protocol and analysis are available
<A HREF="/ENCODE/protocols/general/wgEncodeBroadChipSeq.txt">here</A>.</P>

<h2>Release Notes</h2>

<P>
Release 3 (Mar 2010) of this track adds the HSMM cell line and
includes new experiments for H1-hESC and NHLF. No previously released data
has been replaced in this release. Update to Release 3 (Jun 2010) of this track
consists of a display change to the Signal subtracks. This update provides a
better display of the data when zoomed in to a range spanning less than 16,500
base pairs.
</P>
<P>
Release 2 did contain newer versions of previously released data, however. 
All versioned data are marked with "submittedDataVersion=V2" in the metadata, 
along with the reason for the change.
Previous versions of these files are available for download from the
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/"
 TARGET=_BLANK>FTP site</A>.
</p>
<P>
Please note that an antibody previously labeled "Pol2 (b)" is, in fact, Covance 
antibody MMS-128P with the target POLR2A.
</P>

<h2>Credits</h2>

<p>The ChIP-seq data were generated at the
 <a href="https://www.broadinstitute.org/epigenomics" title="http://www.broad.mit.edu" target="_blank">
 Broad Institute</a> and in the
 <a href="https://bernstein.dfci.harvard.edu/" title="https://bernstein.dfci.harvard.edu/" target="_blank">
 Bradley E. Bernstein lab</a> at the Massachusetts General Hospital/Harvard Medical School.
  &nbsp;&nbsp;
Contact: <A HREF="mailto:&#110;&#115;h&#111;&#114;&#101;&#115;h&#64;&#98;r&#111;&#97;&#100;.&#109;it.&#101;&#100;&#117;"
  title="&#110;&#115;h&#111;&#114;&#101;&#115;h&#64;&#98;r&#111;&#97;&#100;.&#109;it.&#101;&#100;&#117;" rel="nofollow"
  TARGET = _BLANK>Noam Shoresh</a>.
</p>
<p>Data generation and analysis was supported by funds from the NHGRI, the
Burroughs Wellcome Fund, Massachusetts General Hospital and the Broad Institute.
</p>

<h2>References</h2>

<p>
Bernstein BE, Kamal M, Lindblad-Toh K, Bekiranov S, Bailey DK, Huebert DJ, McMahon S, Karlsson EK, Kulbokas EJ 3rd, Gingeras TR <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/15680324" target="_blank">
Genomic maps and comparative analysis of histone modifications in human and mouse</a>.
<em>Cell</em>. 2005 Jan 28;120(2):169-81.
</p>

<p>
Bernstein BE, Mikkelsen TS, Xie X, Kamal M, Huebert DJ, Cuff J, Fry B, Meissner A, Wernig M, Plath K <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16630819" target="_blank">
A bivalent chromatin structure marks key developmental genes in embryonic stem cells</a>.
<em>Cell</em>. 2006 Apr 21;125(2):315-26.
</p>

<p>
Mikkelsen TS, Ku M, Jaffe DB, Issac B, Lieberman E, Giannoukos G, Alvarez P, Brockman W, Kim TK, Koche RP <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17603471" target="_blank">
Genome-wide maps of chromatin state in pluripotent and lineage-committed cells</a>.
<em>Nature</em>. 2007 Aug 2;448(7153):553-60.
</p>

<H2>Data Release Policy</H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column on the track configuration page and
the download page.  The full data release policy for ENCODE is available
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>
