<H2>Description</H2>

<P> This track depicts high throughput sequencing of long RNAs (&gt;200 nt) 
from 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=rnaExtract">
RNA samples</A> from 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=localization">
tissues or subcellular compartments</A> from 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType">ENCODE cell
lines</A>. 
The overall goal of the ENCODE project is to identify and characterize all
functional elements in the sequence of the human genome.
</P>

<H2>Display Conventions and Configuration</H2>
This track is a multi-view composite track that contains the following
<EM>views</EM>:
<DL>

<DT><i>Alignments</i>
<DD>The Alignments view shows reads mapped to the genome.  Sequences determined 
to be transcribed on the positive strand are shown in <span style="color:blue">blue</span>.  
Sequences determined to be transcribed on the negative strand are shown in 
<span style="color:orange">orange</span>.  Sequences for which the direction of 
transcription was not able to be determined are shown in <B>black</B>.

<DT><i>Raw Signals</i>
<DD>The Raw Signal views show the density of aligned tags on the plus, minus, and on both strands.
</DL>

<H2>Methods</H2>

Cells were grown according to the approved 
<A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.
<P>
<H3>Sample preparation and sequencing</H3>

<DT><i>K562 and GM12878 total cell, total RNA</i><P>
Standard Illumina Pair-end kit with the sole exception that a "tagged" random 
hexamer was used to prime the 1st strand synthesis: 5&prime;ACTGTAGGN6-3&prime;.  
The addition of this tag is what permits us to make strand assignments for the reads. 
The sequence of the tag is reported in the 5&prime; end of the read.  Asymmetric PCR 
can place the tag on either the 1st or 2nd read depending on which strand it used as 
a template. Strand assignments are made by looking for the tag at the 5&prime; end of 
either read 1 or read 2. Read 1 is physically linked to read 2. Therefore, if a tag 
is present on one end strand assignments are made for both ends.  We noted during 
analysis that the tags are generally 5&prime; truncated.  We only "strand" reads that 
contain ACTGTAGG, CTGTAGG, TGTAGG, GTAGG. Between 63-68% of reads could be stranded 
in these libraries. It is possible to cull additional stranded reads that contain 
non-templated TAGG, AGG, GG, or G sequences at their 5&prime; end. The peak in 
insert size distribution is between 200-250 nucleotides.

<DT><i>K562 cytosol, polyA+ RNA </i><P>
Oligo-dT selected poly-A+ RNA was RiboMinus-treated according to the
manufacturer's protocol (Invitrogen). The RNA was treated with tobacco alkaline
pyrophosphatase to eliminate any 5&prime; cap structures and hydrolyzed to ~200
bases via alkaline hydrolysis.  The 3&prime; end was repaired using calf
intestinal alkaline phosphatase, and poly-A polymerase was used to catalyze the
addition of Cs to the 3&prime; end.   The 5&prime; end was phosphorylated using T4
PNK, and an RNA linker was ligated onto the 5&prime; end.  Reverse transcription
was carried out using a poly-G oligo with a defined 5&prime; extension.  The
inserts were then amplified using oligos targeting the 5&prime; linker and
poly-G extension.  This cloning protocol generated stranded reads that were read
from the 5&prime; ends of the inserts.  The library was sequenced on a Solexa
platform for a total of 36 cycles; however, the reads underwent post-processing,
resulting in trimming of their 3&prime; ends. Consequently, the mapped read
lengths are variable. 
</P>

<H3>Analysis</H3>
<P>
<DT><i>K562 and GM12878 total cell, total RNA</i><P>
Tags were removed from the 5&prime; ends of the reads in accordance to their 
lengths and strand assignments made.  Subsequently, the reads were trimmed 
from their 3&prime; ends to a final length of 50 nucleotides and were mapped 
using NexAlign, a program developed by Timo Lassman, RIKEN.   We allowed up to 
2 mismatches across the entire length and only report reads that mapped to a 
single/unique locus in the assembled hg18 genome. 

<DT><i>K562 cytosol, polyA+ RNA</i><P>
Reads were mapped to the human (hg18, March 2006) assembly using Nexalign, with only
uniquely mapping (one loci), exactly matching (no mis-matches) aligned reads reported in
the processed files, as follows:
<OL>
<LI>Collect the read sequences from Illumina non-filtered output files.
</LI>
<LI>Filter out all reads that contain undefined nucleotides ('N')
</LI>
<LI>Perform iterative alignment/C-tail chopping algorithm (below). On each
alignment step, the reads are aligned to the genome with 100% identity. 
All reads that align to a single locus are withdrawn from the alignment pool and
only the reads that could not be aligned continue to the next step.
  <OL type="a">
  <LI>Align to the hg18 genome using Nexalign 1.3.3 (&copy; Timo Lassmann) without
  chopping off any nucleotides
  </LI>
  <LI>Chop off any C-blocks (until the first non-C) at the ends of the reads
  </LI>
  <LI>Align to the genome -&gt; remove and save those that align
  </LI>
  <LI>Chop off any non-Cs until the next C
  </LI>
  <LI>Chop off C-block until the next non-C
  </LI>
  <LI>Align to the genome -&gt; remove and save those that align
  </LI>
  <LI>Repeat steps d, e, and f until the reads align to the genome, or chopping 
  results in the reduction of the reads' lengths to below 16 (default), or
  there are no non-Cs left.
  </LI>
  </OL>
</LI>
</OL>
</P>
<H2>Verification</H2>

<P>Verification was done by comparison of referential data generated from 8
individual sequencing lanes (Illumina technology). 
</P>

<H2>Release Notes</H2>
This is Release 2 (Nov 2009) of this track. It includes data from additional 
experiments, and changes in formatting for the existing data described below.
<P>
The K562 cytosol alignments are exactly the same data as Release 1, but the 
alignments are now formatted in the bed14 format described below.
These data have the string submittedDataVersion="V2 - file format change" in 
their metadata and the table names are appended with the string "V2".
<P>
The data format for the alignments in this track are provided in bigBed format.  
Each record is in bed 14 format with the first 12 fields described 
<a href="/FAQ/FAQformat#format1" target="_blank">here</a>.
The final two fields are the two paired sequences, or in the case of single 
alignments, the 13th field is the sequence and the 14th field is a single N.

<H2>Credits</H2>

<DT><i>K562 cytosol, polyA+ RNA</i><P>
<P>These data were generated and analyzed by the transcriptome group at
Cold Spring Harbor Laboratories, and the Center for Genomic
Regulation (Barcelona), who are participants in the ENCODE Transcriptome Group.
</P>
<DT><i>K562 and GM12878 total cell, total RNA</i><P>
Credits: Carrie A. Davis, Jorg Drenkow, Huaien Wang, Alex Dobin and Tom Gingeras

<P>
Contacts: 
<A HREF="mailto:&#100;a&#118;&#105;s&#99;&#64;&#99;sh&#108;.&#101;&#100;&#117;">Carrie Davis</A>
<!-- above address is davisc at cshl.edu -->
 and Tom Gingeras (CSHL).
</P>

<H2> Data Release Policy </H2>

<P>Data users may freely use ENCODE data, but may not, without prior 
consent, submit publications that use an unpublished ENCODE dataset until 
nine months following the release of the dataset.  This date is listed in 
the <EM>Restricted Until</EM> column, above.  The full data release policy 
for ENCODE is available 
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P> 
