<H2>Description</H2>

<P>
This track shows enrichment of RNA sequence tags generated by high throughput 
sequencing (RNA-seq) and mapped to the human genome.  Double stranded cDNA 
was synthesized from fragmented PolyA RNA, and the PCR amplified adapter ligated 
cDNAs, 150-300 nt long, were sequenced using Illumina GA.
</p>
<P>This track shows <em>expression</em> data generated as confirmation
of the
<A HREF="hgTrackUi?g=wgEncodeYaleChIPseq">Yale TFBS</A> tracks.
</P>
<!--
<P>This track shows probable binding sites of the specified transcription
factors (TFs) in the given cell types as determined by chromatin
immunoprecipitation followed by high throughput sequencing (ChIP-Seq).
Included for each cell type is the input signal, which represents the control
condition where no antibody targeting was performed.
For each experiment (cell type vs. antibody) this track shows
a graph of enrichment for TF binding (<I>Signal</I>), along with
sites that have the greatest evidence of transcription factor binding (<I>Peaks</I>).
<P>
The sequence reads, quality scores, and alignment coordinates from
these experiments are available for download.
</P>
-->

<H2>Display Conventions and Configuration</H2>

<P>
This is a composite track that contains multiple data types
(<EM>views</EM>).  Instructions for configuring composite
tracks are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
ENCODE tracks typically contain one or more of the following views:
<DL>
<DT><I>Signal</I></DT><DD>Density graph (wiggle) of signal enrichment based 
on processed data.</DD>
<DT><I>Alignments</I></DT><DD>Mappings of short reads to the genome.  These 
are only available as downloadable files.<DD>
</DL>

<H2>Methods</H2>

<P>
Cells were grown according to the approved
<A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.
Trizol Reagent (15596-018, Life Tech) was added to the cell pellet and total RNA 
was extracted using the protocol provided by the manufacturer. PolyA RNA was 
purified using MicroPoly(A) Purist kit (AM1919, Life Tech) and  was fragmented 
using RNA Fragmentation Reagent (AM8740, Life Tech). Illumina adapters were 
ligated to double stranded cDNA which were synthesized using reagents from Life Tech 
(11917-010). PCR amplified adapter ligated cDNA (150-300 bp) was sequenced 
using Illumina GA. Sequence reads of 27-33 nt long and with 0-2 mismatches 
were mapped to the genome. 
From the mapped tags, a signal map of ChIP DNA fragments was constructed where 
the signal height is the number of overlapping fragments at each nucleotide 
position in the genome.
</p>

<H2> Credits </H2>

<P>These data were generated and analyzed by the labs of
<A HREF="http://snyderlab.stanford.edu/"
  class="external text" title="http://snyderlab.stanford.edu/"
  target="_blank">Michael Snyder</a>,
<A HREF="http://bioinfo.mbb.yale.edu/"
  class="external text"
  title="http://bioinfo.mbb.yale.edu/"
  target="_blank">Mark Gerstein</a> and
<A HREF="http://info.med.yale.edu/bcmm/SMW/SMWhome2.html"
  class="external text" title="http://info.med.yale.edu/bcmm/SMW/SMWhome2.html"
  target="_blank">Sherman Weissman</a> at Yale University;
<A HREF="http://www.genomecenter.ucdavis.edu/farnham/"
  class="external text" title="http://www.genomecenter.ucdavis.edu/farnham/"
  target="_blank">Peggy Farnham</a> at UC Davis; and
<A HREF="http://struhl.med.harvard.edu/"
  class="external text" title="http://struhl.med.harvard.edu/"
  target="_blank">Kevin Struhl</a> at Harvard.
&nbsp;&nbsp;
Contact: <A HREF="mailto:&#100;&#97;&#116;&#97;&#115;u&#98;&#109;&#105;&#115;&#115;i&#111;n&#64;&#103;&#101;r&#115;t&#101;&#105;n&#108;&#97;&#98;.&#111;&#114;&#103;"
  title="&#100;&#97;&#116;&#97;&#115;u&#98;&#109;&#105;&#115;&#115;i&#111;n&#64;&#103;&#101;r&#115;t&#101;&#105;n&#108;&#97;&#98;.&#111;&#114;&#103;" rel="nofollow"
  TARGET = _BLANK>&#100;&#97;&#116;&#97;&#115;u&#98;&#109;&#105;&#115;&#115;i&#111;n&#64;&#103;&#101;r&#115;t&#101;&#105;n&#108;&#97;&#98;.&#111;&#114;&#103;</a>.
<!-- above address is datasubmission at gersteinlab.org -->
</P>

<H2>References</H2>

<p>
Nagalakshmi U, Wang Z, Waern K, Shou C, Raha D, Gerstein M, Snyder M.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18451266" target="_blank">
The transcriptional landscape of the yeast genome defined by RNA sequencing</a>.
<em>Science</em>. 2008 Jun 6;320(5881):1344-9.
</p>

<p>
Raha D, Wang Z, Moqtaderi Z, Wu L, Zhong G, Gerstein M, Struhl K, Snyder M.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/20139302" target="_blank">
Close association of RNA polymerase II and many transcription factors with Pol III genes</a>.
<em>Proc Natl Acad Sci U S A</em>. 2010 Feb 23;107(8):3639-44.
</p>

<p>
Wu JQ, Habegger L, Noisa P, Szekely A, Qiu C, Hutchison S, Raha D, Egholm M, Lin H, Weissman S <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/20194744" target="_blank">
Dynamic transcriptomes during neural differentiation of human embryonic stem cells revealed by short, long, and paired-end sequencing</a>.
<em>Proc Natl Acad Sci U S A</em>. 2010 Mar 16;107(11):5254-9.
</p>

<H2>Data Release Policy</H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column on the track configuration page and
the download page.  The full data release policy for ENCODE is available
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>
