<H2>Description</H2>
<P>
This track shows 5' cap analysis gene expression (CAGE) tags and clusters in 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=rnaExtract">RNA extracts</A> 
from different 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=localization">
sub-cellular localizations</A> 
in multiple
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType">cell lines</A>.
A CAGE cluster is a region of overlapping tags with
an assigned value that represents the expression level.
The data in this track were produced as part of the ENCODE 
Transcriptome Project. 

<H2>Display Conventions and Configuration</H2>
<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are 
<A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.</P>

<P>To show only selected subtracks, uncheck the boxes next to the tracks that 
you wish to hide.</P>

<P>This track contains the following views:
<DL>
<DT><I>Plus</I> and <I>Minus</I> <I>Clusters</I>
<DD>These views display clusters of overlapping read mappings on the
forward and reverse genomic strands.
<DT><i>Alignments</i>
<DD>The Alignments view shows the individual tags (read mappings), with
mismatches from the genomic reference highlighted.
</DL>

<P>Color differences in subtracks are are used as a visual cue to
distinguish between the different cell types, and between annotations
on the plus and minus strand.

<H2>Methods</H2>
<P>
Cells were grown according to the approved 
<A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>
ENCODE cell culture protocols</A>.
RNA molecules longer than 200 nt 
and present in the RNA population isolated from each subcellular compartment
were fractionated into polyA+ and polyA- fractions as described in 
<A HREF="/ENCODE/protocols/general/rnaExtracts.txt" TARGET=_BLANK>these 
protocols</A>.
The CAGE tags were sequenced from the 5' ends of cap-trapped cDNAs produced
using RIKEN CAGE technology 
(Kodzius <EM>et al</EM>. 2006; Valen <EM>et al</EM>. 2009). 
To create the tag, a linker was attached to the 5'
end of polyA+ or polyA- reverse-transcribed cDNAs which were selected by cap
trapping (Carninci <EM>et al</EM>. 1996). The first 27 bp of
the cDNA were cleaved using class II restriction enzymes. A linker was then
attached to the 3' end of the cDNA. 
</P>
<P>After PCR amplification,
the tags were sequenced (36 bp single reads) using ABI 
<A
HREF="http://marketing.appliedbiosystems.com/images/Product_Microsites/Solid_Knowledge_MS/pdf/SOLiD_Dibase_Sequencing_and_Color_Space_Analysis.pdf"
TARGET=_blank>SOLiD</A> technology
(polyA- RNA from the cytosol and nucleus of K562 cell lines, and from whole cell in prostate cells)
or Illumina/Solexa GA (all other data).
Tags were mapped to the human genome
(NCBI Build36, hg18) using the program <em>nexalign</em> 
(T. Lassmann manuscript in preparation). 
SOlid CAGE sequences were mapped with up to 3 mismatches; 2 mismatches were allowed for Solexa CAGE. 
Alignments of sequences mapping 10 times or fewer were retained.
The expression level was computed as the number of reads making up the cluster,
divided by the total number of reads sequenced, times 1 million.
</P>

<H2>Release Notes</H2>
This is Release 2 of this track.  This release adds data for eight new cell-type/compartment combinations (GM12878 Nucleus, H1-hESC whole cell, HepG2 cytosol/nucleus/nucleolus, HUVEC cytosol, and NHEK cytosol/nucleus).
<H2>Credits</H2>
<P>
These data were generated and analyzed by Timo Lassmann, Phil Kapranov, 
Hazuki Takahashi, Yoshihide Hayashizaki, Carrie Davis, Tom Gingeras, and Piero Carninci.
<P>
<P>Contact: <A HREF="mailto:&#99;&#97;&#114;&#110;i&#110;&#99;i&#64;&#114;&#105;k&#101;&#110;.&#106;&#112;">
Piero Carninci</A> at <A TARGET=_BLANK HREF="http://www.osc.riken.jp/english/">
RIKEN Omics Science Center</A>
<!-- above address is carninci at riken.jp -->
</P>

<H2>References</H2>
<P>
Kodzius R, Kojima M, Nishiyori H, Nakamura M, Fukuda S, Tagami M, Sasaki D,
Imamura K, Kai C, Harbers M, <EM>et al</EM>. 
<A TARGET=_BLANK HREF="https://www.ncbi.nlm.nih.gov/pubmed/16489339">
CAGE: cap analysis of gene expression.</A>
<EM>Nat Methods</EM>. 2006 March 1; 3(3):211-222.
</P>

<P>
Valen E, Pascarella G, Chalk A, Maeda N, Kojima M, Kawazu C, Murata M, Nishiyori
H, Lazarevic D, Motti D, <EM>et al</EM>.
<A TARGET=_BLANK HREF="https://www.ncbi.nlm.nih.gov/pubmed/19074369">
Genome-wide detection and analysis of hippocampus core promoters using
DeepCAGE.</A>
<EM>Genome Res</EM>. 2009 February; 19(2):255-265.
</P>
<P>
Carninci P, Kvam C, Kitamura A, Ohsumi T, Okazaki Y, Itoh M, Kamiya M, 
Shibata K, Sasaki N, Izawa M, <EM>et al</EM>.
<A TARGET=_BLANK HREF="https://www.ncbi.nlm.nih.gov/pubmed/8938445">
High-efficiency full-length cDNA cloning by biotinylated CAP trapper.</A>
<EM>Genomics</EM>. 1996 November 1; 37(3):327-336.
</P>

<H2> Data Release Policy </H2>

<P> Data users may freely use ENCODE data, but may not, without prior 
consent, submit publications that use an unpublished ENCODE dataset until 
nine months following the release of the dataset.  This date is listed in 
the <EM>Restricted Until</EM> column, above.  The full data release policy 
for ENCODE is available 
<A HREF="/ENCODE/terms.html" TARGET=_BLANK>here</A>.</P> 
