<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <meta http-equiv="content-type"
 content="text/html; charset=ISO-8859-1">
  <title>wgEncodeRnaSeqSuper.html</title>
</head>
<body>
<h2>Description</h2>
<p>
RNA sequencing, or RNA-Seq, is a method for mapping and quantifying the
transcriptome of any organism that has a genomic DNA sequence
assembly.&nbsp; Compared to microarrays, RNA-Seq is especially
well-suited for <span style="font-style: italic;">de novo</span>
discovery of RNA splicing patterns, and for determining unequivocally
the presence or absence of lower abundance class RNAs.<br>
</p>
<p>RNA-Seq is performed by reverse-transcribing an RNA sample into
cDNA, followed by high throughput DNA sequencing.&nbsp; Most data is
produced in one of two formats: single reads, each of which comes from
one end of a randomly primed cDNA molecule; and paired-end reads, which
are obtained as pairs from both ends cDNAs resulting from random
priming. The resulting sequence reads are then informatically mapped
onto the genome sequence (<span style="font-style: italic;">Alignments</span>).&nbsp;&nbsp;
<br>
Those that don't map to the genome are mapped to known RNA splice
junctions (<span style="font-style: italic;">Splice Sites</span>). <br>
</p>
<p>These mapped reads are then counted to determine their frequency of
occurrence at known gene models. <br>
</p>
<p><br>
&nbsp;<br>
Some RNA-Seq protocols do not specify the coding strand. As a result,
there can be ambiguity at loci where both strands are transcribed. <br>
</p>
<h2>Display Conventions</h2>
This track is a multi-view composite track that contains multiple data
types (views). For each view, there are multiple subtracks that display
individually on the browser. Instructions for configuring multi-view
tracks are here. The following views are in this track:<br>
<br>
<div style="margin-left: 40px;"><span style="font-style: italic;">Signal,</span><span
 style="font-style: italic;"> Raw Signal,</span> or <span
 style="font-style: italic;">RPKM</span><br>
<div style="margin-left: 40px;">Density graph (wiggle) of signal
enrichment based on a normalized aligned read density, indicating RNA
abundance.&nbsp; In some tracks, this is divided further by strand, to
indicate the abundance of RNA transcribed on each strand.<br>
</div>
<span style="font-style: italic;">Splice Sites<br>
</span>
<div style="margin-left: 40px;">Reads mapped to the genome that do not
map in one contiguous block.&nbsp; Often, extra steps are taken to
ensure that these reads represent splice sites, such as ensuring that
they align to some sequence in a catalog of spliced RNAs,<br>
</div>
<span style="font-style: italic;">Alignments</span><br>
<div style="margin-left: 40px;">Reads mapped to the genome. For some
tracks, these reads are available for viewing.&nbsp; For others, they
cannot be viewed but are available as downloadable files.<br>
</div>
</div>
<br>
<h2>Credits<br>
</h2>
These data were generated and analyzed as part of the <a
 href="http://www.genome.gov/10005107">ENCODE</a> project, a
genome-wide consortium project with the aim of cataloging all
functional elements in the human genome.&nbsp; This effort includes
collecting a variety of data over a specific set of <a
href="/ENCODE/cellTypes.html">cell types</a>.&nbsp;
Consequently,
data
related to these tracks be available under ENCODE
tracks. &nbsp;&nbsp;
<br>
<h2>References</h2>
Morozova O, Hirst M, Marra MA.&nbsp; <a
 href="https://www.ncbi.nlm.nih.gov/pubmed/19715439"><span
 style="font-style: italic;">Applications of new sequencing
technologies for transcriptome analysis.</span></a> Annual Review of
Genomics and Human Genetics. 2009;10:135-51.<br>
<br>
Metzker ML. <a style="font-style: italic;"
 href="https://www.ncbi.nlm.nih.gov/pubmed/19997069">Sequencing
technologies - the next generation.</a> Nature Reviews: Genetics. 2010
Jan;11(1):31-46<br>
<h2>Data Release Policy</h2>
Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset
until nine months following the release of the dataset. This date is
listed in the Restricted Until column on the track configuration page
and the download page. The full data release policy for ENCODE is
available <a href="/ENCODE/terms.html">here</a>.<br>
<br>
</body>
</html>
