<H2>Description</H2>
<P>
These tracks display the level of sequence uniqueness of the reference <B>NCBI36/hg18</B>
genome assembly.  They were generated using different window sizes, and high signal
will be found in areas where the sequence is unique.

<H2>Methods</H2>
<P>
The <em>Broad alignability</em> track displays whether a region is made up of
mostly unique or mostly non-unique sequence. To generate the track, every
36-mer in the genome was marked as "unique" if the most similar 36-mer
elsewhere  in the genome have at most 2 mismatches, and as "non-unique"
otherwise. Position X in the alignable track is marked by 1 if >50% of the
bases in [X-200,X+200] are "unique" and by 0 otherwise. Every point in the
alignable track has a corresponding position in  each of the ChIP signal
tracks. The <em>Broad alignability</em> track was generated for the ENCODE
project as a tool for development of the
<A HREF="hgTrackUi?g=wgEncodeBroadChipSeq">Broad Histone</A> tracks.

<P>
The <em>Duke uniqueness</em> tracks display how unique is each sequence on the
positive strand starting at a particular base and of a particular length.
Thus, the 20 bp track reflects the uniqueness of all 20 base sequences with
the score being assigned to the first base of the sequence. Scores are
normalized to between 0 and 1 with 1 representing a completely unique
sequence and 0 representing the sequence occurs >4 times in the genome
(excluding chrN_random and alternative haplotypes).  A score of 0.5
indicates the sequence occurs exactly twice, likewise 0.33 for three times
and 0.25 for four times. The <em>Duke uniqueness</em> tracks were generated
for the ENCODE project as tools in the development of the
<A HREF="hgTrackUi?g=wgEncodeChromatinMap">Open Chromatin</A> tracks.

<P>
The <em>Duke excluded regions</em> track displays genomic regions for
which mapped sequence tags were filtered out before signal generation
and peak calling for Duke/UNC/UTA's
<A HREF="hgTrackUi?g=wgEncodeChromatinMap">Open Chromatin</A> tracks.
This track contains problematic regions for short sequence tag signal
detection (such as satellites and rRNA genes). The
<em>Duke excluded regions</em> track was generated for the ENCODE project.

<P>
The <em>Rosetta uniqueness</em> track uses sequence 'tiles' of 35 bp.
Each tile was aligned to the genome using the BWA aligner.  Tiles that align
uniquely and perfectly in hg18 receive a p-value of 1e-37, while those that
align perfectly in multiple locations receive a p-value of 0.  For each tile,
the oligo midpoint coordinate was recorded along with the -log_10 p-value:
37 (unambiguous) to 0 (ambiguous).  The <em>Rosetta uniqueness</em> track was
generated independently of the ENCODE project.

<P>
The <em>UMass uniqueness</em> track displays a uniqueness signal for each
base which represents the sum of both plus and minus strand 15-mer occurrences
of that particular 5'->3' (plus strand) sequence throughout the genome.  Scores
are normalized between 0 and 1 by calculating ( 1 / N ) where N is the number
of genome wide occurrences of the 15-mer starting at position X.  A score of 1
represents a single genome wide occurrence of that 15-mer.  A 0.5 would
represent either 2 plus strand occurrences or 1 plus and 1 minus strand
occurrence, and so on.  Ratios are rounded to 3 significant digits.  Therefore
a 0.000 would represent > 2000 occurrences.  A 0 is reserved for a given 15-mer
that is either not assembled or contains at least one N at position X.  The
<em>UMass uniqueness</em> track was generated for the ENCODE project.

<P>
The <em>CRG Alignability</em> tracks display how uniquely k-mer sequences align
to a region of the genome. To generate the data, the GEM-mappability
<A HREF="http://gemlibrary.sourceforge.net" TARGET=_BLANK>program</A> has
been employed. The method is equivalent to mapping sliding windows of k-mers
(where k has been set to 36, 40, 50, 75 or 100 nts to produce these tracks)
back to the genome using the GEM mapper aligner (up to 2 mismatches were
allowed in this case). For each window, a mapability score was computed
(S = 1/(number of matches found in the genome): S=1 means one match in the
genome, S=0.5 is two matches in the genome, and so on). The
<em>CRG Alignability</em> tracks were
generated independently of the ENCODE project, in the framework of the GEM
(GEnome Multitool) project.

<H2>Credits</H2>
<P>
The <em>Broad alignability</em> track was created by the <A HREF="mailto:&#110;&#115;h&#111;&#114;&#101;&#115;h&#64;&#98;r&#111;&#97;&#100;.&#109;it.&#101;&#100;&#117;"
title="&#110;&#115;h&#111;&#114;&#101;&#115;h&#64;&#98;r&#111;&#97;&#100;.&#109;it.&#101;&#100;&#117;" rel="nofollow" TARGET=_BLANK>Broad Institute</a>.
Data generation and analysis was supported by funds from the NHGRI (the ENCODE
project), the
Burroughs Wellcome Fund, Massachusetts General Hospital and the Broad Institute.
<P>
The <em>Duke uniqueness</em> and <em>Duke excluded regions</em> tracks were created
by <A HREF="mailto:&#116;&#115;&#102;ur&#101;&#121;&#64;&#100;&#117;&#107;&#101;.e&#100;u"
title="&#116;&#115;&#102;ur&#101;&#121;&#64;&#100;&#117;&#107;&#101;.e&#100;u" rel="nofollow" TARGET=_BLANK>Terry Furey</A>
<!-- above address is tsfurey at duke.edu -->
and Debbie Winter at Duke Univerisity's
<A HREF="https://genome.duke.edu/index.php" TARGET=_blank>Institute for Genome Sciences & Policy (IGSP)</A>;
and Stefan Graf at the
<A HREF="https://www.ebi.ac.uk/" TARGET=_blank>European Bioinformatics Insitute (EBI)</A>.
We thank NHGRI for ENCODE funding support.
<P>
The <em>Rosetta uniqueness</em> track was created by <A HREF="mailto:&#106;oh&#110;&#95;&#99;&#97;s&#116;&#108;&#101;&#64;&#109;&#101;&#114;&#99;k.&#99;&#111;&#109;">John Castle</A>,
   <!-- above address is john_castle at merck.com -->
   at
<A HREF="http://www.rii.com/" TARGET=_BLANK>Rosetta Inpharmatics (Merck)</a>,
with assistance from Melissa Cline at UCSC.
<P>
The <em>UMass uniqueness</em> track was created by <A HREF="mailto:&#98;&#114;y&#97;&#110;.l&#97;&#106;&#111;&#105;&#101;&#64;&#117;&#109;&#97;s&#115;&#109;&#101;&#100;.&#101;&#100;&#117;"
title="&#98;&#114;y&#97;&#110;.l&#97;&#106;&#111;&#105;&#101;&#64;&#117;&#109;&#97;s&#115;&#109;&#101;&#100;.&#101;&#100;&#117;" rel="nofollow" TARGET=_BLANK> Bryan Lajoie</A>
<!-- above address is bryan.lajoie at umassmed.edu -->
in <A HREF="https://www.hhmi.org/scientists/job-dekker" TARGET=_BLANK>Job Dekker's Lab</a>
at the University of Massachusetts Medical School.  Funding Support: NIH grant
HG003143 to JD.  Keck Distinguished Young Scholar Award to JD.  This track was
generated as part of the ENCODE project funded by the NHGRI.
<P>
The <em>CRG Alignability</em> track was created by Thomas Derrien and <A HREF="mailto:&#112;&#97;&#111;l&#111;.
&#114;&#105;&#98;&#101;&#99;&#97;&#64;&#103;&#109;&#97;&#105;l.
&#99;o&#109;">Paolo
Ribeca</A>
<!-- above address is paolo.ribeca at gmail.com -->
 in <A HREF="http://pasteur.crg.es/portal/page/portal/Internet/02_Research/01_Programmes/01_Bioinformatics_Genomics"
TARGET=_BLANK>Roderic Guigo's lab</A> at the Centre for Genomic
Regulation (CRG), Barcelona, Spain. Thomas Derrien was supported by funds from NHGRI for
the ENCODE project, while Paolo Ribeca was funded by a Consolider grant CDS2007-00050
from the Spanish Ministerio de Educaci&#243;n y Ciencia.
</P>

<H2> References </H2>
<P>
Derrien T, Estelle J, Marco Sola S, Knowles DG, Raineri E, Guigo R, Ribeca P.
<a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0030377"
target="_blank">Fast computation and applications of genome mappability</a>.
<em>PLoS One</em>. 2012;7(1):e30377. </P>

<H2> Data Release Policy </H2>
<P>Data users may freely use all data in this track.  ENCODE labs that
contributed annotations have exempted the data displayed here from the
ENCODE data release policy restrictions.
