<H2>Description</H2>

<P>This track is produced as part of the ENCODE Project.
This track shows DNaseI sensitivity measured genome-wide in different
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType">cell lines</A>
using the Digital DNaseI methodology (see below), and DNaseI hypersensitive
sites.
DNaseI has long been used to map general chromatin accessibility and
DNaseI hypersensitivity is a universal feature of active
<EM>cis</EM>-regulatory sequences. The use of this method has led
to the discovery of functional regulatory elements that include enhancers,
insulators, promotors, locus control regions and novel elements.
For each experiment (cell type) this track shows DNaseI hypersensitive zones
(<I>HotSpots</I>) and
hypersensitive sites (<I>Peaks</I>) based on the sequencing tag density
(<I>Signal</I>).
</P>

<H2>Display Conventions and Configuration</H2>
<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>). For each view, there are multiple subtracks that display
individually on the browser. Instructions for configuring multi-view tracks
are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
<P>
For each cell type, this track contains the following views:
<DL>
<DT><I>HotSpots</I>
<DD> DNaseI hypersensitive zones identified using the HotSpot algorithm.
<DT><I>Peaks</I>
<DD> DNaseI hypersensitive sites (DHSs) identified as signal peaks within
FDR 0.5% hypersensitive zones.
<DT><I>Raw Signal</I>
<DD>Density graph (wiggle) of signal enrichment based on aligned read density.
</DL>
<P>DNaseI sensitivity is shown as the absolute density of <EM>in vivo</EM>
cleavage sites across the genome mapped using the Digital DNaseI methodology
(see below).  Data have been normalized to 25 million reads per cell type.
<!-- TODO
DNaseI hypersensitive sites (DHSs) at False Discovery Rate (FDR) 0.5% are shown
within each density track as colored red subregions.</P>
-->

<H2>Methods</H2>

<P>
Cells were grown according to the approved
<A TARGET=_BLANK HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.
Digital DNaseI was performed by performing DNaseI digestion of intact
nuclei, isolating DNaseI 'double-hit' fragments as described in
Sabo <EM>et al.</EM> (2006), and direct sequencing of fragment
ends (which correspond to <EM>in vivo</EM> DNaseI cleavage sites)
using the Solexa platform (27 bp reads).  Uniquely mapping high-quality reads
were mapped to the genome.  DNaseI sensitivity is directly reflected in
raw tag density (<I>Signal</I>), which is shown in the track as density of tags
mapping within a 150 bp sliding window (at a 20 bp step across the genome).
DNaseI hypersensitive zones (<I>HotSpots</I>) were identified using the
HotSpot algorithm
described in Sabo <EM>et al.</EM> (2004).  0.5% false discovery
rate thresholds (FDR 0.005) were computed for each cell type by
applying the HotSpot algorithm to an equivalent number of random
uniquely mapping 27mers.  DNaseI hypersensitive sites (DHSs or <I>Peaks</I>)
were identified as signal peaks within FDR 0.5% hypersensitive
zones using a peak-finding algorithm.</P>

<H2>Verification</H2>

<P>Data were verified by sequencing biological replicates displaying
correlation coefficient &gt 0.9.  Results are extensively validated by
conventional DNaseI hypersensitivity assays using end-labeling/Southern
blotting methods. Multiple cell type Southern blotting methods are
available in 
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeUwDnaseSeq/supplemental/"
 TARGET="_blank">supplemental materials</A>.

</P>

<h2>Release Notes</h2>
<P>
This is release 5 (Oct 2011) of the UW DNaseI HS track. Southern blot validation images
have been added. The files corresponding to tables that have been revoked or replaced in <i>previous</i> releases are still available for download
from the
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeUwDnaseSeq/"
 TARGET=_BLANK>FTP site</A>.
</P>

<H2>Credits</H2>

<P>These data were generated by the UW ENCODE group. </P>
<P>Contact: <A HREF="mailto:&#115;&#117;l&#108;&#64;&#117;.w&#97;&#115;h&#105;&#110;&#103;&#116;&#111;n.&#101;&#100;&#117;">
Richard Sandstrom</A>
<!-- above address is sull at u.washington.edu -->
</P>

<H2>References</H2>

<P>Sabo PJ, Hawrylycz M, Wallace JC, Humbert R, Yu M, Shafer A, Kawamoto J,
Hall R, Mack J, Dorschner M <EM>et al</EM>.
<A TARGET=_BLANK HREF="https://www.pnas.org/content/101/48/16837">
Discovery of functional noncoding elements by digital analysis of chromatin structure</A>.
<EM>PNAS</EM>. 2004 Nov 30;101(48):16837-42.</P>

<P>Sabo PJ, Kuehn MS, Thurman R, Johnson BE, Johnson EM, Cao H, Yu M,
Rosenzweig E, Goldy J, Haydock A <EM>et al</EM>.
<A TARGET=_BLANK HREF="https://www.nature.com/articles/nmeth890">
Genome-scale mapping of DNase I sensitivity in vivo using tiling DNA microarrays</A>.
<EM>Nature Methods</EM>. 2006 Jul;3(7):511-8.</P>

<H2> Data Release Policy </H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column, above.  The full data release policy
for ENCODE is available
<A TARGET=_BLANK HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>

