<H2>Description</H2>

<P>This track shows probable binding sites of the specified transcription
factors (TFs) in the given cell types as determined by chromatin
immunoprecipitation followed by high throughput sequencing (ChIP-Seq).
Included for each cell type is the input signal, which represents the control
condition where no antibody targeting was performed.
For each experiment (cell type vs. antibody) this track shows
a graph of enrichment for TF binding (<I>Signal</I>), along with
sites that have the greatest evidence of transcription factor binding (<I>Peaks</I>).
<P>
The sequence reads, quality scores, and alignment coordinates from
these experiments are available for download.
</P>

<H2>Display Conventions and Configuration</H2>

<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
ENCODE tracks typically contain one or more of the following views:
<DL>
<DT><I>Peaks</I></DT><DD> Regions of signal enrichment based on processed data
(usually normalized data from pooled replicates). ENCODE Peaks tables contain
fields for statistical significance, including FDR
(<A HREF="/goldenPath/help/qValue.html" TARGET=_BLANK>qValue</A>).</DD>
<DT><I>Signal</I></DT><DD>Density graph (wiggle) of signal enrichment based on
processed data.</DD>
</DL>

<H2>Methods</H2>

<P>
Cells were grown according to the approved
<A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.
Further preparations were similar to those previously published
(Euskirchen <EM>et al.</EM>, 2007) with the exceptions that the cells
were unstimulated and sodium orthovanadate was omitted from the buffers.
For details on the chromatin immunoprecipitation protocol used, see
Euskirchen <EM>et al.</EM> (2007) and Rozowsky <EM>et al.</EM> (2009).
</P>
<P>
DNA recovered from the precipitated chromatin was sequenced on the Illumina (Solexa)
sequencing platform and mapped to the genome using the Eland alignment program.
ChIP-seq data was scored based on sequence reads (length ~30 bps) that align uniquely
to the human genome. From the mapped tags a signal map of ChIP DNA fragments
(average fragment length ~ 200 bp) was constructed where the signal height is the number of
overlapping fragments at each nucleotide position in the genome.
</P>
<P>
For each 1 Mb segment of each chromosome a peak height threshold was
determined by requiring a false discovery rate &lt;= 0.05 when comparing the
number of peaks above threshold as compared the number obtained from multiple
simulations of a random null background with the same number of mapped
reads (also accounting for the fraction of mapable bases
for sequence tags in that 1 Mb segment). The number of mapped tags in a putative
binding region is compared to the normalized (normalized by correlating tag
counts
in genomic 10 kb windows) number of mapped tags in the same region from an input DNA control.
Using a binomial test, only regions that have a p-value &lt;= 0.05 are considered to be
significantly enriched compared to the input DNA control.
</P>
<P>
Expression data generated as confirmation of the TFBS data
can be found in the 
<!-- <A HREF="hgTrackUi?g=wgEncodeYaleRNAseq">Yale Poly-A</A> -->
Yale Poly-A tracks (coming soon).
</p>

<H2>Release Notes</H2>

<P>
This is Release 2 (Oct 2009) of this track, which includes data from
additional experiments, and the changes described below.
</P>
<P>
A number of previously released datasets have been replaced by updated versions.
The affected database tables and files include 'V2' (or alternatively, 'Rel2')
in the name, and metadata is marked with "submittedDataVersion=V2", 
followed by the reason for replacement.
Specific changes are:
<TABLE>
<tr><th align='left' valign='top' nowrap> <i>Peaks</i></th>
<td>
The previously released "Peaks" datasets had an off-by-one start location error and did not include  
pValues and point peaks. All Peaks data now have these problems corrected.
</td></tr>

<tr><th align='left' valign='top' nowrap ><i>Signal</i></th>
<td>
The majority of the previously released "Signal" datasets contained a 
sparse representation of the signal. These have been corrected to be the continuous signal intended.
</td></tr>

<tr><th align='left' valign='top' nowrap ><i>Raw Signal</i></th>
<td>
The previously-available "Raw Signal" view of the data is no longer provided as 
the 'Signals' view provides a better measure of enrichment, and algorithms for the Raw Signal are in flux.
</td></tr>
</TABLE>
Previous versions of files are available for download from the 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeYaleChIPseq/" 
 TARGET=_BLANK>FTP site</A>
<TABLE>
<P>
This track contains experiments and data from several labs.  During review, it
was noticed that the some cell cultures were different enough between the labs
that they should be analyzed separately&mdash;using Input specific for the lab.
Specifically:

<TABLE>
<tr><th align='left' valign='top' nowrap><i>K562 vs. K562b</i></th>
<td>
The Farnham lab (UC Davis) K562 cell line variant differs from the 
Snyder (Yale) and Struhl (Harvard) labs and is now labeled: K562b. Experiments on K562b 
should be contrasted with Input (<i>UCDavis</i>).
The K562 Input previously released has been replaced by two versions 
marked K562 and K562b. The K562 version used by the Snyder and Struhl groups
directly replaces the previous version, and is versioned at V2. The K562b used by the Farnham group
is now provided as K562b.
</td></tr>

<tr><th align='left' valign='top' nowrap><i>HepG2 vs. HepG2b</i></th>
<td>
The Farnham lab  HepG2 cell line is now labeled HepG2b. 
Experiments on HepG2b should be contrasted with Input (<i>UCDavis</i>).
</td></tr>
</TABLE>

<H2>Credits</H2>

<P>These data were generated and analyzed by the labs of
<A HREF="http://www.yale.edu/snyder/"
  class="external text" title="http://www.yale.edu/snyder/"
  target="_blank">Michael Snyder</a>,
<A HREF="http://bioinfo.mbb.yale.edu/"
  class="external text"
  title="http://bioinfo.mbb.yale.edu/"
  target="_blank">Mark Gerstein</a> and
<A HREF="http://info.med.yale.edu/bcmm/SMW/SMWhome2.html"
  class="external text" title="http://info.med.yale.edu/bcmm/SMW/SMWhome2.html"
  target="_blank">Sherman Weissman</a> at Yale University;
<A HREF="http://www.genomecenter.ucdavis.edu/farnham/"
  class="external text" title="http://www.genomecenter.ucdavis.edu/farnham/"
  target="_blank">Peggy Farnham</a> at UC Davis; and
<A HREF="http://struhl.med.harvard.edu/"
  class="external text" title="http://struhl.med.harvard.edu/"
  target="_blank">Kevin Struhl</a> at Harvard.
&nbsp;&nbsp;
Contact: <A HREF="mailto:&#100;&#97;&#116;&#97;&#115;u&#98;&#109;&#105;&#115;&#115;i&#111;n&#64;&#103;&#101;r&#115;t&#101;&#105;n&#108;&#97;&#98;.&#111;&#114;&#103;"
  title="&#100;&#97;&#116;&#97;&#115;u&#98;&#109;&#105;&#115;&#115;i&#111;n&#64;&#103;&#101;r&#115;t&#101;&#105;n&#108;&#97;&#98;.&#111;&#114;&#103;" rel="nofollow"
  TARGET = _BLANK>&#100;&#97;&#116;&#97;&#115;u&#98;&#109;&#105;&#115;&#115;i&#111;n&#64;&#103;&#101;r&#115;t&#101;&#105;n&#108;&#97;&#98;.&#111;&#114;&#103;</a>.
<!-- above address is datasubmission at gersteinlab.org -->
</P>

<H2>References</H2>

<p>
Euskirchen G, Royce TE, Bertone P, Martone R, Rinn JL, Nelson FK, Sayward F, Luscombe NM, Miller P, Gerstein M <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/15082775" target="_blank">
CREB binds to multiple loci on human chromosome 22</a>.
<em>Mol Cell Biol</em>. 2004 May;24(9):3804-14.
</p>

<p>
Euskirchen GM, Rozowsky JS, Wei CL, Lee WH, Zhang ZD, Hartman S, Emanuelsson O, Stolc V, Weissman S, Gerstein MB <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17568005" target="_blank">
Mapping of transcription factor binding regions in mammalian cells by ChIP: comparison of array- and sequencing-based technologies</a>.
<em>Genome Res</em>. 2007 Jun;17(6):898-909.
</p>

<p>
Martone R, Euskirchen G, Bertone P, Hartman S, Royce TE, Luscombe NM, Rinn JL, Nelson FK, Miller P, Gerstein M <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/14527995" target="_blank">
Distribution of NF-kappaB-binding sites across human chromosome 22</a>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Oct 14;100(21):12247-52.
</p>

<p>
Robertson G, Hirst M, Bainbridge M, Bilenky M, Zhao Y, Zeng T, Euskirchen G, Bernier B, Varhol R, Delaney A <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17558387" target="_blank">
Genome-wide profiles of STAT1 DNA association using chromatin immunoprecipitation and massively parallel sequencing</a>.
<em>Nat Methods</em>. 2007 Aug;4(8):651-7.
</p>

<p>
Rozowsky J, Euskirchen G, Auerbach RK, Zhang ZD, Gibson T, Bjornson R, Carriero N, Snyder M, Gerstein MB.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/19122651" target="_blank">
PeakSeq enables systematic scoring of ChIP-seq experiments relative to controls</a>.
<em>Nat Biotechnol</em>. 2009 Jan;27(1):66-75.
</p>

<H2>Data Release Policy</H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column on the track configuration page and
the download page.  The full data release policy for ENCODE is available
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>
