<H2>Description</H2>
<P>This track is produced as part of the ENCODE Project.
This track displays maps of histone modifications genome-wide in different
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType">cell lines</A>,
using ChIP-seq high-throughput sequencing.

<H2>Display Conventions and Configuration</H2>
<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>). For each view, there are multiple subtracks that display
individually on the browser. Instructions for configuring multi-view tracks
are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
<P>
For each cell type, this track contains the following views:
<DL>
<DT><I>HotSpots</I>
<DD> ChIP-seq affinity zones identified using the HotSpot algorithm.
<DT><I>Peaks</I>
<DD> ChIP-seq affinity sites identified as signal peaks within
FDR 0.5% hypersensitive zones.
<DT><I>Raw Signal</I>
<DD>Density graph (wiggle) of signal enrichment based on aligned read density.
</DL>
<H2>Methods</H2>

<P>
Cells were grown according to the approved
<A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.
Cells were crosslinked with 1% formaldehyde, and the reaction was quenched
by the addition of glycine. Fixed cells were rinsed with PBS, lysed in nuclei
lysis buffer, and the chromatin was sheared to 200-500 bp fragments using
Fisher Dismembrator (model 500). Sheared chromatin fragments were
immunoprecipitated with specific polyclonal antibody at 4 degrees C
with gentle rotation. Antibody-chromatin complexes were washed and eluted.
The cross linking in immunoprecipitated DNA was reversed and treated with
RNase-A.  Following proteinase K treatment, the DNA fragments were purified
by phenol-chloroform-isoamyl alcohol extraction and ethanol precipitation.
20-50 ng of ChIP DNA was end-repaired, followed by the addition of adenine,
ligated to Illumina adapters and made in to Solexa library for sequencing.
<P>
The sequencing reads uniquely mapping
to the hg18 reference human genome were then scored using a scan statistic based
on the binomial distribution.  Regions of significant tag enrichment
(significance being gauged using an FDR procedure based on randomly generated
data) were then resolved into 150 bp peaks called on a continuous, sliding window
tag density representation of the data (Sabo et al., 2004).
</P>

<H2>Verification</H2>

<P>
Data were verified by sequencing biological replicates displaying correlation
coefficient >0.9.
</P>

<H2>Credits</H2>

<P>These data were generated by the UW ENCODE group. </P>
<P>Contact: <A HREF="mailto:&#115;&#117;l&#108;&#64;&#117;.w&#97;&#115;h&#105;&#110;&#103;&#116;&#111;n.&#101;&#100;&#117;">Richard Sandstrom</A>
<!-- above address is sull at u.washington.edu -->
</P>

<H2>References</H2>

<P>Sabo PJ, Hawrylycz M, Wallace JC, Humbert R, Yu M, Shafer A, Kawamoto J,
Hall R, Mack J, Dorschner MO, McArthur M, Stamatoyannopoulos JA.
<A TARGET=_BLANK HREF="https://www.pnas.org/content/101/48/16837">Discovery of functional noncoding elements by digital analysis of chromatin structure.</A>
<EM>PNAS</EM>. 2004;101:16837-16842.</P>

<H2> Data Release Policy </H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column, above.  The full data release policy
for ENCODE is available
<A TARGET=_BLANK HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>

