<H2>Description</H2>
<P>
This track displays positions of four different types of RNA in the human 
genome: 
<UL>
<LI>microRNAs from the
<A HREF="http://microrna.sanger.ac.uk/sequences/" TARGET=_blank>
miRBase</A> at the <A HREF="http://www.sanger.ac.uk/" TARGET=_blank>
Wellcome Trust Sanger Institute</A>(WTSI).
<LI>small nucleolar RNAs (C/D box and H/ACA box snoRNAs) and Cajal body-specific RNAs (scaRNAs) from the 
<A HREF="http://www-snorna.biotoul.fr/" TARGET=-blank>snoRNABase</A> maintained at the 
<A HREF="http://www-lbme.biotoul.fr/uk/index.html" TARGET=_blank>
Laboratoire de Biologie Moleculaire Eucaryote</A>
</UL></P>
<P>
C/D box and H/ACA box snoRNAs are guides for the 2'O-ribose methylation and 
the pseudouridilation, respectively, of rRNAs and snRNAs, although many of 
them have no documented target RNA.  The scaRNAs guide modifications of the
spliceosomal snRNAs transcribed by RNA polymerase II, and often contain both 
C/D and H/ACA domains.</P>

<H2>Display Conventions and Configuration</H2>
<P>
This track follows the general display conventions for 
<A HREF="../goldenPath/help/hgTracksHelp.html#GeneDisplay">gene prediction 
tracks</A>. </P>
<P>
The miRNA precursor  forms (pre-miRNA) are represented by red blocks.</P>
<P>
C/D box snoRNAs, H/ACA box snoRNAs and scaRNAs are represented by blue, 
green and magenta blocks, respectively. At a zoomed-in resolution, arrows 
superimposed on the blocks indicate the sense orientation of the snoRNAs. </P>

<H2>Methods</H2>
<P>
Precursor miRNA genomic locations from
<a href="http://microrna.sanger.ac.uk/" target="_blank">
miRBase</a>
were calculated using wublastn for sequence alignment with the requirement of
100% identity. 
The extents of the precursor sequences were not generally known and were
predicted based on base-paired hairpin structure.  The miRNA Registry is
described in Griffiths-Jones, S. (2004) and Weber, M.J. (2005) in the 
References section below.</P>
<P>
The snoRNAs and scaRNAs from the snoRNABase were aligned against the 
human genome using blat. 
</P>

<H2>Credits</H2>
Genome coordinates for this track were obtained from the miRBase sequences
<A HREF="ftp://ftp.sanger.ac.uk/pub/mirbase/sequences/CURRENT/genomes/" 
TARGET=_BLANK>FTP site</A> and from 
<A HREF="http://www-snorna.biotoul.fr/coordinates.php" TARGET=_BLANK>
snoRNABase coordinates download page.</A>
<P>

<H2>References</H2>
<P>
When making use of these data, please cite the folowing articles in addition to
the primary sources of the miRNA sequences:</P>
<P>
Griffiths-Jones S, Saini HK, van Dongen S, Enright AJ.
<A HREF="http://nar.oxfordjournals.org/cgi/content/full/36/suppl_1/D154" 
TARGET=_BLANK>miRBase: tools for microRNA genomics</A>.
<EM>Nucleic Acids Res.</EM> 2008 Jan 1;36(Database issue):D154-8.</P>
<P>
Griffiths-Jones S, Grocock RJ, van Dongen S, Bateman A, Enright AJ.
<A HREF="http://nar.oxfordjournals.org/cgi/content/abstract/34/suppl_1/D140" 
TARGET=_BLANK>miRBase: microRNA sequences, targets and gene nomenclature</A>.
<EM>Nucleic Acids Res.</EM> 2006 Jan 1;34(Database issue):D140-4.</P>
<P>
Griffiths-Jones S.
<A HREF="http://nar.oupjournals.org/cgi/content/abstract/32/suppl_1/D109"
TARGET=_blank>The microRNA Registry</A>.
<EM>Nucleic Acids Res.</EM> 2004 Jan 1;32(Database issue):D109-11.</P>
<P>
Weber MJ.
<A HREF="http://www.blackwell-synergy.com/doi/abs/10.1111/j.1432-1033.2004.04389.x"
TARGET=_blank>New human and mouse microRNA genes found by homology search</A>.
<P>
You may also want to cite The Wellcome Trust Sanger Institute 
<A HREF="http://microrna.sanger.ac.uk/"
TARGET=_blank>miRBase</A> and The Laboratoire de Biologie Moleculaire 
Eucaryote <A HREF="http://www-snorna.biotoul.fr/" TARGET=_blank>snoRNABase</A>.</P>
<P>
The following publication provides guidelines on miRNA annotation:
Ambros V. <em>et al</em>., 
<A HREF="http://www.rnajournal.org/cgi/content/abstract/9/3/277" 
TARGET=_blank>A uniform system for microRNA annotation</A>.  
<em>RNA.</em> 2003;9(3):277-9.</P>
<P>
