<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=265>
  <TR><TD ALIGN=RIGHT>
    <A HREF="https://www.ncbi.nlm.nih.gov/bioproject/12587"
    TARGET=_blank>
    <IMG SRC="../images/Macropus_eugenii.jpg" WIDTH=250 HEIGHT=325 ALT="Wallaby"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Notamacropus eugenii</em><BR> 
    </FONT>
    <FONT SIZE=-2>
      (NHGRI <A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Tammar%20Wallaby&id=4134"
              TARGET=_blank>Press Photos</A>)
    </FONT>

  </TD></TR>
</TABLE>

<P>
The Sep. 2009 <em>Notamacropus eugenii </em> (Tammar Wallaby) draft assembly 
(<A HREF="https://www.ncbi.nlm.nih.gov/bioproject/12587"
TARGET=_blank>NCBI project 12587</A> 
<A HREF="ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/vertebrate_mammalian/Notamacropus_eugenii/all_assembly_versions/GCA_000004035.1_Meug_1.1"
TARGET=_blank>GCA_000004035.1</A>)
was produced by the Tammar Wallaby Genome Sequencing Consortium,
a collaboration between the Baylor College of Medicine <a href="https://www.hgsc.bcm.edu/content/wallaby-genome-project" target="_blank">
Human Genome Sequencing Consortium</a> and the <a href="http://www.agrf.org.au/" target="_blank">
Australian Genome Research Facility</a>.
For more information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/5598/" TARGET="_blank">
Meug_1.1</A> in the NCBI Assembly database.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Wallaby genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>

    <TR><TD VALIGN=Top>GL048072</TD>
        <TD WIDTH=14>&nbsp;</TD>
        <TD VALIGN=Top>Displays all of scaffold GL192360</TD>
    </TR>
    <TR><TD VALIGN=Top>GL048072:1-400,000</TD>
        <TD WIDTH=14>&nbsp;</TD>
        <TD VALIGN=Top>Displays first 400,000 bases of scaffold GL048072</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>GL048072:30,000+2000</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Displays a region of scaffold GL048072 that spans 2000 bases, starting with position 300000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>
            EF602313; FJ238079 <BR>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Displays region between genome landmarks, such as
                      the mRNA markers EF602313 and FJ238079.
                      This syntax may also be used for other range queries,
                      such as between uniquely determined ESTs, STSs,
                      refSeqs, etc.</TD>
    </TR>
    <TR><TD VALIGN=Top>DQ304114</TD>
        <TD WIDTH=14>&nbsp;</TD>
        <TD VALIGN=Top>Displays region of mRNA with GenBank accession
                number DQ304114</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
<!--    <TR><TD VALIGN=Top>pseudogene mRNA</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR> -->
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>Bellacosa</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Bellacosa</TD>
    </TR>
    <TR><TD VALIGN=Top>Taguchi,T.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author T. Taguchi</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Taguchi T</em>, internally it
	uses the format <em>Taguchi,T.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
The genome has been sequenced to 2x coverage using ABI 3730 technology.
There are 277,711 scaffolds with a total size of 3,075,184,024 bases. The N50 size of the scaffolds is 1,537,592,012 bases with 24,533 scaffolds larger, and 253,178 scaffolds smaller. The mean scaffold size is 11,073 bases, the median scaffold size is 1,755 bases.

For more information about this assembly, see also:
<A HREF="https://www.ncbi.nlm.nih.gov/Traces/wgs/?val=ABQO00"
TARGET=_blank>WGS record at NCBI</A>.
</P>


<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/macEug2/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#wallaby">Downloads</A> page. 

The Wallaby browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#wallaby_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
