<!-- Display image at top right of page -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=315>
  <TR><TD ALIGN=RIGHT><A HREF="http://genome.wustl.edu/genomes/detail/pan-troglodytes/" TARGET=_blank>
      <IMG SRC="../images/Pan_troglodytes.jpg" WIDTH=300 HEIGHT=219 ALT="Chimpanzee"></A>
    </TD></TR>
  <TR><TD ALIGN=RIGHT>
  <FONT SIZE=-1>&quot;Clint&quot; -  <em>Pan troglodytes</em><BR>
  </FONT>
  <FONT SIZE=-2>
   Photo: <A HREF="http://www.yerkes.emory.edu/"
      TARGET=_blank>Yerkes National Primate Research Center, Emory University</A>
</FONT>
  </TD></TR>
</TABLE>

<P>
The 13 Nov. 2003 chimpanzee (<em>Pan troglodytes</em>) Arachne assembly &mdash; 
NCBI Build 1 Version 1 &mdash; was produced by the Chimpanzee Sequencing and 
Analysis Consortium.</P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the HUGO Gene Nomenclature Committee gene 
name of a human RefSeq, the accession number of an EST or mRNA,
a chromosomal coordinate range, a scaffold (supercontig)
coordinate range, or keywords from 
the GenBank description of an mRNA. The following list shows 
examples of valid position queries for the chimpanzee genome.
See the
<A HREF="../goldenPath/help/hgTracksHelp.html" TARGET=_blank>
User's Guide</A> for more information.
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TD VALIGN=Top NOWRAP><B>Request:</B><BR></TD>
       	<TD VALIGN=Top COLSPAN=2><B>&nbsp;&nbsp; Genome Browser Response:</B><br></TD>
    </TR>
    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>chr23</TD>
        <TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays all of chromosome 23</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr6:118,000,000-119,000,000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a million bases of chromosome 6, beginning at 118 million bases offset from p arm telomere</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>scaffold_1:1-1000000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays first million bases of genome positioned at scaffold 1 of the assembly</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr6:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 6 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>BRCA1</TD>
       	<TD WIDTH=14></TD>
       	<TD>Displays region of genome where human RefSeq gene BRCA1 (or features associated with BRCA1) aligns</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>AF115459</TD>
       	<TD WIDTH=14></TD>
   	<TD VALIGN=Top>Displays region of genome with mRNA with GenBank accession number AF115459</TD>
    </TR>
    <TR>
	<TD VALIGN=Top NOWRAP>348</TD>
	<TD><BR></TD>
	<TD VALIGN=Top NOWRAP>Displays the region of genome with Entrez Gene identifier 348</TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>pseudogene mRNA</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>sialic acid</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists mRNAs and RefSeqs with GenBank keywords sialic acid</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>huntington</TD>
       	<TD WIDTH=14></TD>
 	<TD VALIGN=Top>Lists mRNAs associated with Huntington's disease</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Paabo,S.</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author S. Paabo</TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although GenBank 
			requires the search format <em>Paabo S</em>, internally it uses 
			the format <em>Paabo,S.</em>.</TD>
    </TR>
</TABLE>
<P>
<HR>
<H3>Assembly details</H3>
<P>
This assembly covers about 95 percent of the genome and is based on 4X sequence 
coverage. It is composed of 361,782 contigs with an N50 length of 15.7 kb, and 
37,849 supercontigs having an N50 length of 8.6 Mb (not including gaps). The total 
contig length is 2.73 Gb, spanning 3.02 Gb. 
<P>
<A HREF="http://www.genome.gov/Pages/Research/Sequencing/SeqProposals/ChimpGenome2.pdf" TARGET=_blank>Background information</A> on the chimp genome 
sequencing project and the <A HREF="http://www.genome.gov/11509418" 
TARGET=_blank>initial news release</A> about the chimp assembly can be 
found on the NHGRI website.
For more information about the assembly process used for this draft genome, see the 
Genome Browser Assembly track details page. 
To view a chromosome correspondence table between chimp 
and human chromosomes, click 
<A HREF="/FAQ/FAQdownloads.html#download25">here</A>. 
Note: this assembly does <em>not</em> support the new 
chromosomal numbering convention in which the identifiers &quot;2a&quot; and 
&quot;2b&quot; have been assigned to the two chimp chromosomes that align to
human chromomsome 2. Instead, it retains the older numbering scheme that
assigns the identifiers 12 and 13 to these chimp chromosomes. The new convention
has been implemented in assemblies subsequent to panTro1. See the
<A HREF="../FAQ/FAQdownloads#download25">FAQ</A> for a table showing the 
correspondence between
human and chimp chromosomes.</P>
<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/panTro1/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#chimp">Downloads</A> page. 
The complete set of sequence reads is available at the 
<A HREF="https://www.ncbi.nlm.nih.gov/Traces/home/" TARGET=_blank>NCBI trace archive</A>.
The chimpanzee browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the <A HREF="../goldenPath/credits.html#chimp_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release and conditions for use of the data.
<P>
Blat searches on chrUn_random are not supported at the current time, but will 
be available soon.  
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
