<H2>Description</H2>
This track depicts gaps in the assembly.
Gaps are represented as black boxes in this track.
If the relative order and orientation of the contigs on either side
of the gap is known, it is a <em>bridged</em> gap. In this case, a white 
line is drawn through the black box representing the gap and the gap is labeled 
&quot;yes&quot;. 
</P>
<P>This assembly contains the following types of gaps:
<UL>
<LI><B>Fragment</B> - gaps between the whole genome shotgun (WGS) contigs 
of the Arachne assembly, represented by 100 <em>N</em>s .
In this assembly, all fragment gaps are bridged.
<LI><B>Contig</B> - gaps between the scaffolds (&quot;super-contigs&quot;)
constructed from the WGS contigs.  On the pseudo-chromosomes containing 
scaffolds that could not be confidently placed on a chromosome (e.g. chrN_random),
these gaps are represented by 50,000 <em>N</em>s, with the exception of
chrUn_random, on which the gaps are 10,000 <em>N</em>s. On all other chromosomes,
the gap size is based on the distance between scaffolds when aligned
to the finished human genome assembly, as described in the
Assembly track description.
<LI><B>Heterochromatin, Centromere, and Telomere</B> - gaps from large blocks of 
heterochromatin (size varies).
</UL></P>
