<H2>Description</H2>
<P>
This track displays regions of the chimpanzee draft assembly 
(panTro1) that are deleted in the human genome assembly (hg16). Only regions of 
between 80 and 12000 bases are included. The name of each deletion is a unique 
pointer to that deletion followed by an underscore and its length. A 
similar track, showing chimp deletions in the human assembly, appears in the 
human Genome Browser.</P>

<H2>Methods</H2>
<P>
The human/chimpanzee alignments were created at UCSC with blastz and blat
using a reciprocal best strategy with 
chaining and netting. The initial alignments were generated using blastz on repeatmasked 
sequence with following matrix:

<PRE>
       A    C    G    T
 A   100 -300 -150 -300
 C  -300  100 -300 -150
 G  -150 -300  100 -300
 T  -300 -150 -300  100

 O = 400, E = 30, K = 4500, L = 4500, M = 50
</PRE></P>
<P>
The overall score is the sum of the score over all pairs.</P>
<P>
The resulting alignments were processed by the axtChain program. To place 
additional chimp scaffolds that weren't initially aligned by blastz, a DNA blat 
of the unmasked sequence was performed. The resulting blat alignments were also 
chained, and then merged with the blastz-based chains produced in the previous 
step to produce "all chains", which were further processed by the chainNet and 
netSyntenic programs. Finally, a "reciprocal best" strategy was employed to 
minimize paralog fill-in for missing orthologous chimp sequence. Details of the 
alignment methods can be found in the descriptions of the Chimp Chain and Chimp Net tracks of the human genome browser.
</P>

<P>Human deletions in chimp were determined from the collection of indels implied by these alignments. The criteria for inclusion in the list of deletions were (i) within, not between, scaffolds; (ii) simple gaps only (no opposing, unmatched bases or double gaps); (iii) 80-12000 bp long; and (iv) not a missed overlap or incorrect gap size in assembly. These criteria aim to include plausible repeat insertions and exclude assembly and alignment artifacts.</P>

<H2>Credits</H2>
<P>
The chimpanzee sequence used in this track was obtained from the 13 Nov. 2003 
Arachne assembly. This sequence was provided by the National Human Genome 
Research Institute (NHGRI), the Eli & Edythe L. Broad Institute at MIT/Harvard, 
and Washington University School of Medicine.</P>
<P>
The BLASTZ program was created by Webb Miller of the 
<A HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank> Penn State Bioinformatics 
Group</A>.</P>
<P>
Jim Kent at UCSC wrote the blat program, the chaining and netting programs, and 
the scripts for displaying the alignments in this browser.</P>  
<P>
The list of mid-sized (80-12000 bp) human deletions relative to chimpanzee was 
provided by 
Tarjei Mikkelsen at MIT. The UCSC alignments of complete chimpanzee scaffolds 
to the human genome assembly were used to generate this list.</P>

<H2>References</H2>
<P>
Batzoglou S, Jaffe DB, Stanley K, Butler J, Gnerre S, Mauceli E, Berger B, Mesirov JP, Lander ES.
<A HREF="https://genome.cshlp.org/content/12/1/177.abstract" 
TARGET=_blank>ARACHNE: a whole-genome shotgun assembler</A>. 
<em>Genome Res</em>. 2002 Jan;12(1):177-89.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11779843" target="_blank">11779843</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC155255" target="_blank">PMC155255</a>
</p>

<P>
Chiaromonte F, Yap VB, Miller W.
<A HREF="http://psb.stanford.edu/psb-online/proceedings/psb02/chiaromonte.pdf"
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>. 
<em>Pac Symp Biocomput</em>. 2002:115-26.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11928468" target="_blank">11928468</a>
</p>

<P>
Jaffe DB, Butler J, Gnerre S, Mauceli E, Lindblad-Toh K, Mesirov JP, Zody MC, Lander ES.
<A HREF="https://genome.cshlp.org/content/13/1/91.abstract" 
TARGET=_blank>Whole-genome sequence assembly for mammalian genomes: Arachne 2</A>.
<em>Genome Res</em>. 2003 Jan;13(1):91-6.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12529310" target="_blank">12529310</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC430950" target="_blank">PMC430950</a>
</p>

<P>
Kent, WJ
<A HREF="https://genome.cshlp.org/content/12/4/656.abstract" 
TARGET=_blank>BLAT - the BLAST-like alignment tool</A>.
<em>Genome Res</em>. 2002 Apr;12(4):656-64.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11932250" target="_blank">11932250</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC187518" target="_blank">PMC187518</a>
</p>

<P>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC, Haussler D, Miller W.
<A HREF="https://genome.cshlp.org/content/13/1/103.abstract" 
TARGET=_blank>Human-mouse alignments with BLASTZ</A>. 
<em>Genome Res</em>. 2003 Jan;13(1):103-7.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12529312" target="_blank">12529312</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC430961" target="_blank">PMC430961</a>
</p>
