<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<HTML>
<HEAD>
	<META http-equiv=Content-Type content="text/html; charset=windows-1252">
	<META content="MSHTML 6.00.2800.1106" name=GENERATOR>
        <META name="description" content="Non-standard Join Certificates">
        <META name="keywords" content="genome, genome browser, genome assembly, UCSC, bioinformatics, certificate">
	<LINK REL="STYLESHEET" HREF="../../style/HGStyle.css">
    	<TITLE>UCSC Genome Browser: Non-standard Join Certificates: Build 34</TITLE>
</HEAD>

<BODY BGCOLOR="FFF9D2" LINK="0000CC" VLINK="#330066" ALINK="#330066">
<A NAME="TOP"></A>

<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0 WIDTH="100%">
    <TR><TH COLSPAN=2 ALIGN="left"><IMG SRC="../../images/title.jpg" ALT="UCSC Genome Bioinformatics"></TH></TR>
</TABLE>

<TR><TD COLSPAN=2 HEIGHT=40><!--#include virtual="../../cgi-bin/hgMenubar"--></TD></TR>
<!--Build 34 certificate table--------------------------------------->
<TABLE BORDER=0 WIDTH="100%"><TR><TH COLSPAN=2 ALIGN="left"><TH></TR>
<TR><TD CELLPADDING=10>
    <!--outer table is for border purposes-->
    <TABLE WIDTH="100%" BGCOLOR="#888888" BORDER="0" CELLSPACING="0" CELLPADDING="1"><TR><TD>
        <TABLE BGCOLOR="fffee8" WIDTH="100%"  BORDER="0" CELLSPACING="0" CELLPADDING="0"><TR><TD>
            <TABLE BGCOLOR="D9E4F8" BACKGROUND="../../images/hr.gif" WIDTH=100%>
                <TR><TD>
		    <FONT SIZE="4"><a NAME="TOC"></a><b>&nbsp; Non-standard Join Certificates: Human Build 34 (July 2003)</b></FONT>
                </TD></TR>
	    </TABLE>
            <TABLE BGCOLOR="fffee8" WIDTH="100%" CELLPADDING=0><TR><TH HEIGHT=10></TH></TR>
                <TR><TD WIDTH=10>&nbsp;</TD><TD>
                
<TABLE cellspacing=3 cols=9 border cellPadding=2 WIDTH="100%"
  <TR BGCOLOR=#fffee8><TH>Accession 1</TH>
  <TH>Accession 2</TH>
  <TH>Spanner</TH>
  <TH>Variation</TH>
  <TH>Variation Evidence</TH>
  <TH>Comment</TH>
  <TH>Evaluation</TH>
  <TH>Remark</TH>
  <TH>Contact</TH>
  </TR>
<TR><TD>AL606514</TD><TD>AL627107</TD><TD>&nbsp </TD><TD>sine </TD><TD>none</TD><TD>Alu. Deletions in alu repeats in both clones disrupting join. There is one BAC End which supports the join RPCI-456B9. Also looking at the actual size of the overlaps between AL627107 and AL606514, the total overlap in unsubmitted sequence is 86688bp. AL627107 also actualy overlaps AL139326 (the clone after AL606514 in the TPF) by about 50Kb, but as AL606514 was finished first, it remained in the TPF.</TD><TD>pass</TD><TD>Would be better to have BAC, fosmid end support or other, but extensive overlap from unsubmitted sequence is a good proxy.  It seems that the clone AL606514 may be unnecessary to have in TPF if AL527107 overlaps AL139326.</TD><TD><A HREF="mailto:ad1@sanger.ac.uk">ad1@sanger.ac.uk</A></TD></TR>
<TR><TD>AP002529</TD><TD>AP002530</TD><TD>AL359174</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Blunt end join submitted by JST spanned by AL359174.14 and 10 BAC, 4 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AP002530</TD><TD>AP002531</TD><TD>AL355532</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Blunt end join submitted by JST spanned by AL355532.10 and 7 BAC, 3 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AL008729</TD><TD>AL391385</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Not experimental</TD><TD>100bp overlap. Join supported by 4 BACs and 4 Fosmids. Half dovetail; misses other side by 19bp as a result of a 9bp polyA indel. AL391385.8 has (A)31; AL008729.1 has (A)22.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AL034346</TD><TD>AL499606</TD><TD>&nbsp </TD><TD>other</TD><TD>Not experimental</TD><TD>2000bp overlap. Variations clustered in the tandem repeat of the overlap. Half dovetail; misses other side by 145bp. 14 copies of 34bp repeat in AL499606.18; 25 copies of 34bp repeat in AL034346.31. Join supported by 10 BAC and 1 fosmid end pairs.</TD><TD>return</TD><TD>requires additional support eg. PCR</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AL096678</TD><TD>AL159174</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Not experimental</TD><TD>2082bp overlap. 464bp deletion in AL096678.8 which includes (CA)n and (TA)n. Join supported by 11 BACs and 6 fosmids</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AL109940</TD><TD>AL442124</TD><TD>&nbsp </TD><TD>other</TD><TD>Not experimental</TD><TD>100bp overlap +/+ orientation. 51bp tandem repeat causing assembly trouble. 20 copies in AL109940 and 40 copies in AL442124. Join supported by 5 BAC and 8 fosmid end pairs.</TD><TD>return</TD><TD>requires additional support eg pcr.</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AL357058</TD><TD>AP002528</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Preferential use of Sanger sequence (AL357058) to JST sequence (AP002528.1)</TD><TD>pass</TD><TD>This is a unique situation, with one clone contained entirely within another. This is not really a bad join, but a substitution of part of a project with another.  Pass, as discussed with GS and AJM.</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AL445250</TD><TD>AL591624</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Not experimental</TD><TD>110369bp overlap. 474 bp deletion in AL445250: 180bp (TATAA)n Simple_repeat, 200bp (TTATA)n Simple_repeat  and 61bp AT_rich  Low_complexity repeat. Join supported by 4 pairs of BAC ends and 2 pairs of fosmid ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AL591046</TD><TD>AL591073</TD><TD>&nbsp </TD><TD>other</TD><TD>Not experimental</TD><TD>101450bp overlap. Join supported by 3 pairs of BAC ends.</TD><TD>return</TD><TD>Extensive overlap, but missing information. Please describe variation</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AL591480</TD><TD>AL596449</TD><TD>&nbsp </TD><TD>other</TD><TD>Not experimental</TD><TD>32827bp overlap. 10377bp deletion in AL596449.4. Join supported by 12 BAC and 6 fosmid end pairs.</TD><TD>return</TD><TD>requires additional support eg pcr.</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AL133255</TD><TD>AL591044</TD><TD>AL596216</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Overlap of 6bp at GAATTC cloning site. Join supported by PCR product sequence AL596216.1 and 2 pairs of BAC and fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB000878</TD><TD>AC006046</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>This is the boundary between Inoko MHC class I and the class III here represented by Geraghty sequence, overlap is 18554bp</TD><TD>pass</TD><TD>probably OK, given region and extensive overlap. More BAC/fosmid end support might be useful. Fairly unique situation.</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB000878</TD><TD>AB000879</TD><TD>AP000506</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 4 BAC, 3 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB000879</TD><TD>AB000880</TD><TD>AP000507</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 2 BAC, 1 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014077</TD><TD>AB023052</TD><TD>AP000513</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 4 BAC, 8 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014077</TD><TD>AB014078</TD><TD>AP000514</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 8 BAC, 5 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014078</TD><TD>AB014079</TD><TD>AP000514</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 8 BAC, 4 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014079</TD><TD>AB014080</TD><TD>AP000514</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 8 BAC, 2 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014080</TD><TD>AB014081</TD><TD>AP000514</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 8 BAC, 4 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014081</TD><TD>AB014082</TD><TD>AP000514</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 7 BAC, 6 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014082</TD><TD>AB014083</TD><TD>AP000515</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 9 BAC, 7 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014083</TD><TD>AB014084</TD><TD>AP000515</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 9 BAC, 4 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014084</TD><TD>AB014085</TD><TD>AP000515</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 8 BAC, 5 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014085</TD><TD>AB014086</TD><TD>AP000516</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 9 BAC, 7 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014086</TD><TD>AB014087</TD><TD>AP000516</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 7 BAC, 8 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014087</TD><TD>AB014088</TD><TD>AP000516</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 8 BAC, 4 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB014088</TD><TD>AB023053</TD><TD>AP000516</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 9 BAC, 5 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB023048</TD><TD>AB023060</TD><TD>AP000510</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 4 BAC, 8 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB023048</TD><TD>AB023050</TD><TD>AP000511</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 6bp restriction site (AAGCTT). Join supported by 3 BAC, 9 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB023049</TD><TD>AB023051</TD><TD>AP000512</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 6bp restriction site (AAGCTT). Join supported by 5 BAC, 3 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB023049</TD><TD>AB023052</TD><TD>AP000513</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 4 BAC, 5 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB023050</TD><TD>AB023051</TD><TD>AP000512</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 3 BAC, 3 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB023053</TD><TD>AB023055</TD><TD>AP000517</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 8 BAC, 4 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB023056</TD><TD>AB023057</TD><TD>AP000520</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 5 BAC, 5 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB023059</TD><TD>D84394</TD><TD>AP000509</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 6bp restriction site (GAATTC). Join supported by 5 BAC, 2 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AB023059</TD><TD>AB023060</TD><TD>AP000509</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>MHC class I. Overlap of 4bp restriction site (GATC). Join supported by 3 BAC, 2 fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:ajm@sanger.ac.uk">ajm@sanger.ac.uk</A></TD></TR>
<TR><TD>AC003098</TD><TD>AC015937</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC003098 contains 22bp more of sslp at bp 93315. PAC/CIT. 2000 bp total overlap. 2 supporting fosmids</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC003976</TD><TD>AC069363</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC003976 has a 172 bp deletion in a (GAAA)n repeat.         RP11/CIT libraries 58000 bp overlap. 11 supporting fosmids.</TD><TD>pass</TD><TD>11 fosmid ends - is this overrepresentation?  Please comment.</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AB020862</TD><TD>AB020863</TD><TD>AC079187</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>6 supporting BACs, 2 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC004707</TD><TD>AC015909</TD><TD>&nbsp </TD><TD>sine </TD><TD>Blast2 and RepeatMasker</TD><TD>AC015909 contains  384 bp inserted region containing Alu and simple repeat (TGAA)n. RP11/PAC. Total 51000 bp overlap 1 supporting Bac; 11 supporting fosmids.</TD><TD>pass</TD><TD>Concern re over-representation of 11 pairs fosmid ends. Please comment.	</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC004585</TD><TD>AC018629</TD><TD>&nbsp </TD><TD>sine </TD><TD>Blast2 and RepeatMasker</TD><TD>AC004585 contains 571 bp inserted region containing Alu, FLAM_C, and simple repeat (TTCC)n. RP11/PAC Total 5500 bp overlap         3 supporting Bacs; 2 supporting fosmids.</TD><TD>pass</TD><TD>multiple variations-probably better to confirm polymorphism</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC006050</TD><TD>AC090685</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC006050 missing 312 bases within a (TATATG)n repeat.          RP11/PAC 81000 bp total overlap,6 Bacs; 5 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC006251</TD><TD>AC015922</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC006251 has a 4217 bp deletion(126811..131028).   CIT/PAC 38000 bp total overlap. 5 Bacs; 4 fosmids support join.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC007632</TD><TD>AC023394</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeatmasker and human analysis</TD><TD>TA simple repeat region is ~250bp longer in AC023394; join is supported by high % identity across the overlap and 1 fosmid and 6 BACs which link across the join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC009277</TD><TD>AC068506</TD><TD>&nbsp </TD><TD>other</TD><TD>sequence alignment; to be tested</TD><TD>duplication of 289bp in AC009277; perfect repeat; join is supported by high % identity across the overlap and 5 pairs of fosmid end seq and 6 pairs of BAC end seq which link across the join</TD><TD>return</TD><TD>Requires additional support, eg PCR</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC009285</TD><TD>AC087224</TD><TD>&nbsp </TD><TD>sslp </TD><TD>&nbsp</TD><TD>ID is 99.53 (below threshold of 99.60) in 28kb ol.   Areas of low ID are accounted for as SSLPs in simple repeat areas.  Join supported by 1 BAC and 7 fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC009455</TD><TD>AC079062</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeat masker, sequence alignment</TD><TD>TA simple repeat region is ~200bp longer in AC079062; join supported by 4 BACs which link across join.  Total overlap 41kb.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC011120</TD><TD>AC080037</TD><TD>&nbsp </TD><TD>sine </TD><TD>Blast2 and RepeatMasker</TD><TD>AC011120 contains inserted AluJo not present in AC080037. Total 142000 bp overlap 6 supporting Bacs; 19 supporting fosmids.</TD><TD>return</TD><TD>is 19 fosmid-ends overrepresentation? Please comment.</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC011825</TD><TD>AC015563</TD><TD>&nbsp </TD><TD>other</TD><TD>to be tested</TD><TD>189bp deletion in AC015563; join is supported by high % identity across the overlap and 5 pairs fosmid end seq and 4 pairs of BAC end seq which link across the join</TD><TD>return</TD><TD>Requires additional support, eg PCR</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC012213</TD><TD>AC069351</TD><TD>&nbsp </TD><TD>sine </TD><TD>Global Sequence Alignment and RepeatMasker</TD><TD>40 kb overlap. Break in alignment is caused by 310 bp AluY insertion in AC069351. 3 supporting BACs, 11 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC012238</TD><TD>AC023926</TD><TD>&nbsp </TD><TD>line </TD><TD>Global Sequence Alignment and RepeatMasker</TD><TD>33 kb overlap. Break in alignment is a result of 299 bp L1PA4 insertion in AC023926. 3 supporting BACs, 2 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC013290</TD><TD>AC022601</TD><TD>&nbsp </TD><TD>sine </TD><TD>repeatmasker</TD><TD>break in alignment caused by ~286bp insertion of AluY SINE/Alu element in AC022601; join is supported by high % identity across the overlap and 1 pair of fosmid end seq and 6 pairs of BAC end seq which link across the join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC015691</TD><TD>AC109341</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Global Sequence Alignment, RepeatMasker</TD><TD>61kb overlap, MIR and AT-rich, supported by 6 pairs of BAC ends, 4 pairs of fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC015700</TD><TD>AC100763</TD><TD>&nbsp </TD><TD>line </TD><TD>RepeatMasker, Global Sequence Alignment</TD><TD>105kb overlap, L1PA3 in AC015700`s 75649- 75828, supported by 2 pairs of BAC ends, 4 pairs of fosmid ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC015884</TD><TD>AC016292</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC015884 contains ~6 more copies of a 30bp repeat 41662 bp total overlap 3 supporting bacs, 7 supporting fosmids</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC015884</TD><TD>AC036164</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC015884 contains 1029 bp inserted region containing ~16 more copies of a 65 bp repeat.  Found at bp 316-1344 (GGGGGGGGCTGGAGTCAGAGAGACCAAGGTTGGAACCCGGC<BR>AGGTACCACGGACGGTCACACTGT) AC015884 contains 388 bp inserted region containing same repeat element as above.  Bp 1870-2257 9300 bp total overlap 3 supporting bacs, 4 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC015921</TD><TD>AC097370</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2 and RepeatMasker</TD><TD>AC015921 contains inserted region of 467 bp. 14790 total overlap. 1 supporting Bac, 4 supporting fosmids</TD><TD>return</TD><TD>requires additional support (eg pcr).</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC016888</TD><TD>AC064805</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC016888 contains 120 bp deletion in MER4A element.          49000 bp total overlap, 3 supporting Bacs; 4 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC018943</TD><TD>AC108137</TD><TD>&nbsp </TD><TD>other</TD><TD>BLAST2, Global Sequence Alignment, RepeatMasker</TD><TD>AC018943 contains 193 bp insertion that is not present in AC108137. Total overlap is 23 Kb, 5 supporting BACs, 9 supporting fosmids</TD><TD>return</TD><TD>Probably OK, but requires additional confirmation of insertion polymorphism</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC021553</TD><TD>AC100825</TD><TD>&nbsp </TD><TD>line </TD><TD>BLAST2, Global Sequence Alignment, RepeatMasker</TD><TD>AC021553 contains 231 bp of a LINE/L2 element not present in AC100825. Total overlap is 26Kb, 1 supporting BAC, 1 supporting fosmid.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC021594</TD><TD>AC068473</TD><TD>&nbsp </TD><TD>other</TD><TD>human analysis and sequence alignment</TD><TD>Insertion of a ~180bp of a 38bp repetive element in AC068473.  Join supported by 3 BACs and 10 fosmids.  Total overlap 48kb.</TD><TD>pass</TD><TD>is 12 pairs fosmid-ends overrepresentation? Comment please.</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC021760</TD><TD>AC040926</TD><TD>&nbsp </TD><TD>sslp </TD><TD>BLAST2, Global Sequence Alignment</TD><TD>overlap of 10kb. SSLP 121bp longer in AC021760. 12 supporting BACs.</TD><TD>pass</TD><TD>is 12 pairs BAC-ends overrepresentation?</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC022558</TD><TD>AC044907</TD><TD>&nbsp </TD><TD>line </TD><TD>BLAST2, Global Sequence Alignment, RepeatMasker</TD><TD>AC044907 contains 6,031 bp of a LINE/L1HS element not present in AC022558.  Total overlap is 49Kb, 5 supporting BACs, 7 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC022596</TD><TD>AC098850</TD><TD>&nbsp </TD><TD>sine </TD><TD>Blast2 and RepeatMasker</TD><TD>AC022596 contains inserted Alu not present in AC098850. Total 120000 bp overlap, 4 supporting Bacs; 18 supporting fosmids.</TD><TD>return</TD><TD>is 18 pairs fosmid-ends overrepresentation? Please comment.</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC024474</TD><TD>AC061965</TD><TD>&nbsp </TD><TD>sslp </TD><TD>BLAST2, Global Sequence Alignment, RepeatMasker</TD><TD>AC061965 contains 559 bp of C-rich, (CACCT)n, (CACCC)n elements not present in AC024474. Total overlap is 9Kb, 4 supporting BACs, 3 supporting fosmids</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC024610</TD><TD>AC078907</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC078907 missing 258 bases within a (TAG)n repeat.        54000 bp total overlap 5 supporting Bacs; 20 supporting fosmids.</TD><TD>return</TD><TD>is 20 fosmid-ends overrepresentation? Please comment.</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC025519</TD><TD>AC053479</TD><TD>&nbsp </TD><TD>sslp </TD><TD>BLAST2 and RepeatMasker</TD><TD>94 kb overlap. Break in alignment is caused by a 286 bp (TA)n insertion in AC053479. 2 supporting BACs, 3 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC025521</TD><TD>AC100832</TD><TD>&nbsp </TD><TD>sine </TD><TD>Blast2 and RepeatMasker</TD><TD>AC100832 contains 217 bp inserted region containing Alu. 33000 bp total overlap 3 supporting BACS and 1 supporting fosmid.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC025787</TD><TD>AC109635</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Global Sequence Alignment, RepeatMasker</TD><TD>116kb overlap, (TA)n (TAA)n SSLP, supported by 3 pairs of BAC ends, 4 pairs of fosmid ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC026636</TD><TD>AC104938</TD><TD>&nbsp </TD><TD>other</TD><TD>BLAST2, Global Sequence Alignment, RepeatMasker</TD><TD>There are 2 ins/del in this overlap: AC026636 contains 360 bp of a LINE/L2 element that is not present in AC104938, and AC104938 contains 133 bp of a SINE/Alu/AluJo element that is not present in AC026636. Total overlap is 36 Kb, 4 supporting BACs, 7 supporting fosmids. AC026636 is from the RP11 library, AC104938 is from the CTD library</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC026991</TD><TD>AC087367</TD><TD>&nbsp </TD><TD>sslp </TD><TD>BLAST2, Global Sequence Alignment</TD><TD>overlap of 59kb. SSLP 311bp longer in AC026991. 3 supporting BACs, 5 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC036164</TD><TD>AC087392</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC087392 contains 354bp inserted region at bp 16581. AC036164 contains 330 bp more of a (GGTTG)n repeat at bp 68475. 30896 bp total overlap. 7 supporting bacs, 4 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC036196</TD><TD>AC103858</TD><TD>&nbsp </TD><TD>other</TD><TD>BLAST2, Global Sequence Alignment, RepeatMasker</TD><TD>AC103858 has 995 bp region (containing L2 and MER elements) flanked by two AluSx elements, while in the analogous region of AC036196 there is only one AluSx element and not the L2 and MER elements. Total overlap is 47Kb, 7 supporting BACs, 2 supporting fosmids.</TD><TD>return</TD><TD>multiple complex `other` requires confirmation of polymorphism eg by PCR</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC044814</TD><TD>AC107934</TD><TD>&nbsp </TD><TD>other</TD><TD>BLAST2, Global Sequence Alignment</TD><TD>overlap of 147kb. 173bp tigger2 insertion in AC107934. 5 supporting BACs, 2 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC055854</TD><TD>AC105046</TD><TD>&nbsp </TD><TD>sslp </TD><TD>BLAST2, Global Sequence Alignment</TD><TD>overlap of 33kb. SSLP 325bp longer in AC055854. 3 supporting BACs, 6 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC083845</TD><TD>AC103760</TD><TD>&nbsp </TD><TD>line </TD><TD>Global Sequence Alignment and RepeatMasker</TD><TD>43 kb overlap. Break in alignment is caused by a 28 bp (T)n and 109 bp L1HS insertion in AC083845. 1 supporting fosmid.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC083973</TD><TD>AC107885</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 56kb. Two breaks in alignment caused by 311bp SINE insertion in AC083973 and 479bp insertion of SINE LINE SINE in AC083973.  4 supporting BACs, 5 supportings fosmids.</TD><TD>pass</TD><TD>Probably OK, but multiple problems may warrant confirmation of haplotypes</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC084125</TD><TD>AF186192</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment and RepeatMasker</TD><TD>17 kb overlap. Two breaks in the alignment. One is caused by a low complexity GC-rich region which is 967 bp longer in AF186192. The other is caused by multiple Alu and AluJ/FRAM repeats (767 bp) in AC084125 while the corresponding region in AF186192 has multiple AluSg/x repeats (567 bp). 1 supporting BAC, 4 supporting fosmids.  CTD clone and  RP4 clone libraries</TD><TD>return</TD><TD>This multiple variation is actually quite interesting, but should be confirmed as a polymorphism by pcr.</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC087269</TD><TD>AC090567</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 32kb. 166bp LTR12C insertion in AC090567. 3 supporting BACs, 6 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC087399</TD><TD>AC087675</TD><TD>&nbsp </TD><TD>other</TD><TD>sequence alignment and human analysis</TD><TD>Insertion of ~500bp made of an 80bp repetetive element in AC087399.   Insertion of  ~240bp in AC087675.   Join supported by 4 BACs and 3 fosmids.  Total overlap 97kb.</TD><TD>return</TD><TD>multiple `other` (insertions) requires additional support (eg pcr)</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC087516</TD><TD>AC103988</TD><TD>&nbsp </TD><TD>sine </TD><TD>BLAST2, Global Sequence Alignment, RepeatMasker</TD><TD>AC103988 contains 313 bp of AluSx and AluY elements not present in AC087516. Total overlap is 57Kb, 10 supporting BACs, 5 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC087675</TD><TD>AC109314</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeat masker</TD><TD>AT-rich region.  Join supported by 2 BACs and 1 fosmid.  Total overlap 42kb.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC087691</TD><TD>AC107993</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2 and RepeatMasker</TD><TD>AC087691 contains 240 bp inserted region containing MSR1 satellite elements 37000 bp total overlap 6 supporting Bacs; 2 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC090354</TD><TD>AC090396</TD><TD>&nbsp </TD><TD>other</TD><TD>sequence alignment and human analysis</TD><TD>There is a ~500bp segment of sequence that is repeated once more in AC090396 than in AC090354.  6 BACs and 5 fosmids confirm join.  Total overlap 116kb.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC090377</TD><TD>AC107990</TD><TD>&nbsp </TD><TD>sine </TD><TD>sequence alignment</TD><TD>break in alignment caused by 946bp insertion in AC107990  (contains  a AluJb and AluY), Overlap confirmed with AP001549 which does not contain the insertion. 5 pairs of fosmid end seq and 3 pairs of BAC end seq link across this join)</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC090592</TD><TD>AC100770</TD><TD>&nbsp </TD><TD>other</TD><TD>RepeatMasker, Global Sequence Alignment</TD><TD>125kb overlap, LTR33 repeated in AC090592, supported by 2 pairs of fosmid ends.  Draft AC025235 supports join.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC090639</TD><TD>AC107925</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Global Sequence Alignment</TD><TD>61kb overlap, (AT)n, AT-rich, supported by 4 pairs of BAC ends, 2 pairs of fosmid ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC090691</TD><TD>AC100797</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 57kb. Break in alignment caused by imperfect 28mer repeat ("CCCCTCGGCGAGGACAGAGGCGCGGCGG ") that is 770bps larger in AC090691. 2 supporting BACs, 6 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC090796</TD><TD>AC091561</TD><TD>&nbsp </TD><TD>sslp </TD><TD>BLAST2, Global Sequence Alignment</TD><TD>overlap of 111kb. SSLP 317bp longer in AC090796. 3 supporting BACs, 2 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC090970</TD><TD>AC090971</TD><TD>&nbsp </TD><TD>other</TD><TD>repeat masker</TD><TD>SINE/other. broken overlap caused inserted region of three Charlie1 elements with two intervening AluSx elements (Charlie1- AluSx- Charlie1- AluSx- Charlie1), 1,037 bp total, in AC090970; join is supported by high % identity across the 117 kb overlap and 12 fosmids which link across the join.</TD><TD>return</TD><TD>is 12 pairs fosmid-ends overrepresentation? Please comment.</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC091010</TD><TD>AP003480</TD><TD>&nbsp </TD><TD>sslp </TD><TD>BLAST2 and RepeatMasker</TD><TD>58 kb overlap. Break in alignment is caused by a 342 bp (TAAA)n insertion in AC091010. 7 supporting BACs, 3 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC091170</TD><TD>AC100778</TD><TD>&nbsp </TD><TD>sine </TD><TD>repeatmasker</TD><TD>~300bp AluYa5 element present in AC100778. TA simple repeat region is ~300bp longer in AC100778.   join is supported by high % identity across the overlap, 1 BAC & 5 fosmids.  Total overlap 79kb.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC079073</TD><TD>AC104042</TD><TD>&nbsp </TD><TD>sine </TD><TD>RepeatMasker, Global Sequence Alignment</TD><TD>44kb overlap, AluSx insertion in AC104042, supported by 7 pairs of BAC ends, 5 pairs of fosmid ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC079200</TD><TD>AC112673</TD><TD>&nbsp </TD><TD>line </TD><TD>BLAST2, Global Sequence Alignment</TD><TD>overlap of 18kb. 354bp LINE insertion in AC112673. 6 supporting BACs, 6 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC102802</TD><TD>AC103952</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 41kb. Two breaks in alignment caused by 4110bp insertion in AC103952 and 321bp insertion in AC102802 containing SINE. 2 supporting BACs, 1 supporting fosmid.</TD><TD>return</TD><TD>multiple, insertion probably requires additional confirmation</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC103834</TD><TD>AC103877</TD><TD>&nbsp </TD><TD>sine </TD><TD>BLAST2, Global Sequence Alignment</TD><TD>overlap of 38kb. 283bp Alu insertion in AC103834. 6 supporting BACs, 3 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC104038</TD><TD>AC107881</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Global Sequence Alignment, RepeatMasker</TD><TD>95kb overlap, SSLP (CACCC)n (CCA)n, supported by 8 pairs of BAC ends, 12 pairs of fosmid ends</TD><TD>pass</TD><TD>is 12 pairs fosmid-ends overrepresenataion?</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC104989</TD><TD>AC105029</TD><TD>&nbsp </TD><TD>sslp </TD><TD>BLAST2, Global Sequence Alignment</TD><TD>overlap of 32kb. SSLP 294bp longer in AC104989. 3 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC105110</TD><TD>AC108452</TD><TD>&nbsp </TD><TD>other</TD><TD>sequence alignment</TD><TD>inversion of ~2240bp in the overlap region; join is supported by high % identity across the overlap 10 pairs of fosmid end seq and 2 pairs of BAC end seq which link across the join</TD><TD>return</TD><TD>Requires additional support, eg PCR</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC105142</TD><TD>AC107877</TD><TD>&nbsp </TD><TD>other</TD><TD>BLAST2</TD><TD>18 kb overlap. Break in alignment is caused by a 249 bp insertion in AC107877. 8 supporting BACs and another supporting finished accession (AP004710).</TD><TD>return</TD><TD>Insertion requires futher support as being polymorphic</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC110288</TD><TD>AF067845</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment and RepeatMasker</TD><TD>other and SSLP. 26 kb overlap. Six breaks in the alignment.  1. Low complexity region is 616 bp longer in AF067845.  2.(GT)n repeat is 677 bp longer in AF067845.  3. AC110288 has an extra copy of a GGAGGAGAAACGTGGCAGGCGTGAACAGACACCACGTGAGAC repeat.  4. AF067845 has an extra copy of a CACGTGAGACAGGGAGGAGAAACGTGGCAGGCGTGAACAGA repeat.  5. AF067845 has an extra copy of a ATTTTCTGTGAGTTCCCGAGCTCTGCTCCACACCTGGGACCG<BR>GACTCGGAATTTTCTGTGAGTTCCCGAGCTCTGCTCCATACC<BR>TGGGACCGGACTCAGC repeat.  6. poly A region is 11 bp longer in AC110288.  1 supporting BAC test seq, 1 supporting draft (AC141072), 5 supporting fosmids.  CTD clone versus P1 clone.</TD><TD>return</TD><TD>Multiple `other`, requires additional support eg. PCR</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC019257</TD><TD>AC100810</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 20kb Breaks in alignment caused by imperfect 23mer repeat larger in AC100810 by 389bps "CCCCATCTGGTTAGAACGCTGCA", imperfect 37mer repeat larger in AC100810 by 480bps "CCCCAGGGGACAGCATGACAGATCCTCTCGCCGCACA", 5 supporting BACs 3 supporting fosmids</TD><TD>return</TD><TD>This probably needs additional support as to polymorphism.</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC018665</TD><TD>AC040980</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC018665 contains 43 bases more of AT repeat at 124167. 69000 bp total overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC055866</TD><TD>AC135721</TD><TD>&nbsp </TD><TD>sine </TD><TD>Blast2 and RepeatMasker</TD><TD>AC135721 contains 611 bp inserted region containing Alu. RP11/CIT libraries. 10000 bp total overlap, 4 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC073446</TD><TD>AC100756</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeat masker</TD><TD>broken overlap caused by CT_rich low complexity region (350 bp) in AC073446; join is supported by high % identity across the 162 kb overlap and 8 BAC`s and 15 fosmids which link across the join.</TD><TD>return</TD><TD>is 15 pairs fosmid-ends overrepresentation? Please comment.</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC000377</TD><TD>AC124057</TD><TD>&nbsp </TD><TD>line </TD><TD>Global Sequence Alignment, RepeatMasker</TD><TD>49kb overlap, LINE1 insertion in AC000377, supported by 7 pairs of fosmid ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC109341</TD><TD>AC131574</TD><TD>&nbsp </TD><TD>sine </TD><TD>Global Sequence Alignment</TD><TD>55kb overlap, AluSx repeats more in AC131574, supported by 6 pairs of BAC ends, 4 pairs of fosmid ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC055876</TD><TD>AC135348</TD><TD>&nbsp </TD><TD>sslp </TD><TD>BLAST2, Global Sequence Alignment, RepeatMasker</TD><TD>AC135348 contains 18 bp of (A)n that is not present in AC055876. Total overlap is 5 Kb, 14 supporting BACs, 5 supporting fosmids. AC135348 is from the RP13 library, AC055876 is from the RP11 library</TD><TD>return</TD><TD>is 15 pairs of BAC-ends overrepresentation? Please comment.</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC139149</TD><TD>AC139530</TD><TD>&nbsp </TD><TD>sine </TD><TD>sequence alignment, repeat masker</TD><TD>Insertion of  ~310bp composed of 1 AluYb8 element in AC139149.   Join supported by 10 BACs & 10 fosmid spanners.  Total overlap 42kb.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC138080</TD><TD>AP000079</TD><TD>&nbsp </TD><TD>line </TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 85kb, Break in alignment caused by LINE in 396bp segment in AC138080, 1 supporting BAC 3 supporting fosmids</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC115090</TD><TD>AC124789</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC124789 contains 339 bp more of a (TA)n repeat at bp 21617 27000 bp total overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC091172</TD><TD>AC125257</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2 and RepeatMasker</TD><TD>AC091172 contains 3932 bp deletion at 164383. 126000 bp total overlap. 3 supporting Bacs; 6 supporting fosmids.</TD><TD>return</TD><TD>big deletion---requires additional support of polymorphism (eg pcr)</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC004908</TD><TD>AC131281</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 94kb Break in alignment caused by a 3934bp segment in AC131281 containing SSLP, LINEs and an analogous 520bp segment in AC004908 containing a LINE</TD><TD>return</TD><TD>is 15 pairs of fosmid-ends overrepresentation? Multiple variations may require additional support eg. PCR</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC104389</TD><TD>AC129505</TD><TD>&nbsp </TD><TD>other</TD><TD>sequence alignment</TD><TD>778 bp deletion in AC129505, 2 pairs of fosmid end link across join</TD><TD>return</TD><TD>requires additional support eg. PCR</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC107971</TD><TD>AC130312</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Sequence Alignment, RepeatMasker</TD><TD>60kb overlap, SSLPs in centromeric region</TD><TD>return</TD><TD>requires BAC, fosmid end support or other</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC021317</TD><TD>AC115992</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC021317 contains 524 bp inserted region containing ALU at bp 36310. RP11/CIT. 21385 bp total overlap. 6 supporting bacs, 7 supporting fosmids</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC100777</TD><TD>AC116002</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeatmasker and human analysis</TD><TD>GAA simple repeat region is ~430bp longer in AC100777; join is supported by high % identity across the overlap and 5 pairs of fosmid end seq and 8 pairs of BAC end seq which link across the join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC100791</TD><TD>AC116025</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC100791 contains 353 bp more of (CAT)n repeat at bp 131892  RP11/CIT. 115000bp total overlap. 6 supporting bacs, 3 supporting fosmids</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC104419</TD><TD>AC115485</TD><TD>&nbsp </TD><TD>other</TD><TD>BLAST2, RepeatMasker</TD><TD>overlap of 27kb. Break in alignment is caused by a 3.5kb insertion (containing L2, L2, LTR60, L2) in AC115485 that is flanked by AluSx.  Analogous position in AC104419 has only AluSx. RP11 clone versus GS1 clone. 3 supporting BACs, 11 supporting fosmids, 1 supporting draft, AF215849.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC105195</TD><TD>AP004290</TD><TD>&nbsp </TD><TD>line </TD><TD>Global Sequence Alignment and RepeatMasker</TD><TD>44 kb overlap. Break in alignment is caused by a 317 bp L1MC/D insertion in AP004290. 2 supporting BACs, 2 supporting fosmids, and another supporting finished accession (AC104980).</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AP000079</TD><TD>AP000080</TD><TD>AL355484</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>3 supporting BACs, 1 supporting fosmid.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC091060</TD><TD>AC118757</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeatmasker and human analysis</TD><TD>GA-rich low complexity repeat is ~155bp longer in AC091060; join is supported by high % identity across the overlap and 5 pairs of fosmid end seq and 3 pairs of BAC end seqwhich link across the join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC104257</TD><TD>AC120041</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment and RepeatMasker</TD><TD>68 kb overlap. Two breaks in the alignment. One is caused by 1122bp L1PA4 insertion in AC104257, while the other one is caused by a 179bp (TTATA)n insertion in AC104257. 1 supporting BAC, 4 supporting fosmids. RP11 clone versus CTD clone.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC111170</TD><TD>AC111182</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2 and RepeatMasker</TD><TD>AC111170 contains 78 bp more of unresolved homopolymeric tract at 17741-17824bp 55305 bp total overlap 5 supporting bacs, 5 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC004612</TD><TD>AC123779</TD><TD>&nbsp </TD><TD>sine </TD><TD>Global Sequence Alignment and RepeatMasker</TD><TD>17 kb overlap. Break in alignment is caused by a 311 bp AluYa5 insertion in AC004612. 3 supporting BACs, 1 supporting fosmid.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC110014</TD><TD>AC120055</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeat masker and human analysis</TD><TD>TA simple repeat region is ~1kB longer in AC110014; join is supported by high % identity across the overlap and 1 pair of BAC end seq which link across the join</TD><TD>pass</TD><TD>Preferable to have additional BAC or fosmid end support</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC025018</TD><TD>AC117569</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeatmasker and human analysis</TD><TD>TA rich simple repeat region is ~500bp longer in AC117569; join is supported by 2 pairs of BAC end seq which link across the join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC110288</TD><TD>AC120035</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 29kb. Two breaks in alignment caused by imperfect 42mer repeat ("CAGCCATCCGTGGACTGGGGTAGACTGTGAGCGGTTAATC<BR>TC") that is 798bps larger in AC110288. Imperfect 45mer repeat ("CAGGTACCTGGATGGGTTCATTTGGGTGCATCACCTGGAG<BR>ACGTC") that is 225bps larger in AC110288. 1 supporting BAC, 2 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC027701</TD><TD>AC127034</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment and RepeatMasker</TD><TD>78 kb overlap. Break in alignment is caused by a 560 bp insertion of a combination of LTR`s and a SSLPs (LTR-LTR-SSLP-LTR) in AC027701. 2 supporting BACs, 2 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC104423</TD><TD>AC125437</TD><TD>&nbsp </TD><TD>sine </TD><TD>sequence alignment, repeat masker</TD><TD>Insertion of  ~320bp composed of 2 Alu elements in AC125437.   Join supported by 4 BACs and 6 fosmids.  Total overlap 45kb.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC090616</TD><TD>AC129917</TD><TD>&nbsp </TD><TD>sine </TD><TD>Blast2 and RepeatMasker</TD><TD>AC129917 contains 186 bp inserted region containing Alu.	contains AluSq not rpesent in AC129917, 75000 bp total overlap         3 supporting Bacs; 5 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC122179</TD><TD>AC127030</TD><TD>&nbsp </TD><TD>sine </TD><TD>repeatmasker, sequence alignment</TD><TD>Alu, Line insertion in AC122179</TD><TD>return</TD><TD>requires BAC, fosmid end support or other</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC107956</TD><TD>AC126389</TD><TD>&nbsp </TD><TD>sine </TD><TD>RepeatMasker, Global Sequence Alignment</TD><TD>23kb overlap, AluSq longer in AC126389 than AC107956, supported by 2 pairs of BAC ends, 2 pairs of fosmid ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC013791</TD><TD>AC131971</TD><TD>&nbsp </TD><TD>sine </TD><TD>Global Sequence Alignment</TD><TD>19kb overlap, AluSx repeat.  2 copies in AC131971, 1 copy is AC013791, 8 fosmid ends</TD><TD>pass</TD><TD>is 12 pairs fosmid-ends overrepresentation?</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC108448</TD><TD>AC131971</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Global Sequence Alignment, RepeatMasker</TD><TD>58kb overlap, GA-rich length polymorphism: 305 bp in AC108448 (near 58042-58235), 78 bp in AC131971, supported by 4 pairs of BAC ends, 8 pairs of fosmid ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC087386</TD><TD>AC126603</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeat masker</TD><TD>broken overlap caused by (CGG)n repeat (311 bp) and G-rich low complexity region (165 bp) in AC126603; join is supported by high % identity across the 27 kb overlap and 13 BAC`s and 12 fosmids which link across the join.</TD><TD>return</TD><TD>Are 13 BAC- and 12 fosmid-ends overrepresentation? Please comment</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC015988</TD><TD>AC107896</TD><TD>&nbsp </TD><TD>other</TD><TD>sequence alignment</TD><TD>Insertion of an AluYa5 repeat element in AC107896.  Insertion of ~400bp of a 80bp unknown repeat element in AC107896.  Join supported by 4 BACs and 14 fosmids.  Total overlap 112kb.</TD><TD>return</TD><TD>is 21 pairs fosmid-ends overrepresentation? Also, unknown insertion requires additional support eg. PCR to support polymorphism</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC107892</TD><TD>AC109314</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeat masker, sequence alignment</TD><TD>CCCCAGn repeat region is ~50bp longer in AC107892; join supported by 10 BACs and 8 fosmids which link across join.  Total overlap 141kb.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC026950</TD><TD>AC129915</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 18kb. Imperfect 54mer repeat larger by 1838bps in AC129915. 8 supporting BACs, 3 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC103725</TD><TD>AC131568</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 41kb. Break in alignment caused by imperfect 20mer repeat ("CAGAGGGGAACACATGGGGG") larger in AC131568 by 93bps. 4 supporting BACs, 4 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:cuomo@genome.wi.mit.edu">cuomo@genome.wi.mit.edu</A></TD></TR>
<TR><TD>AC022206</TD><TD>AC131390</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeat of GGAA in AC022206 around 164,300 bp interrupts the alignment.</TD><TD>&nbsp</TD><TD>pass</TD><TD>In future, should include BAC/fosmid end support and extent of polymorphism.</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC027130</TD><TD>AC134300</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeat of TA around 5391 bp in AC007613 interrupts the alignment.</TD><TD>&nbsp</TD><TD>pass</TD><TD>in future, please include BAC/fosmid end support and extent/size of variation. This also has a lot of overlap (89kb)) which would be useful to note.</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC127459</TD><TD>AC130450</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC010268</TD><TD>AC130454</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeat of AG around 116081 bp in AC010268 interrupts the alignment.</TD><TD>&nbsp</TD><TD>pass</TD><TD>should include BAC/fosmid end support, extensive overlap (49kb), size of polymorphic region</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC009030</TD><TD>AC130449</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeats of CT around 137400 bp and GGGAT around 78000 bp in AC130449 interrupts the alignment.</TD><TD>&nbsp</TD><TD>pass</TD><TD>should include BAC/fosmid end support, extensive overlap (49kb), size of polymorphic region</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC005591</TD><TD>AC092117</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>return</TD><TD>requires additional information. Please include BAC/fosmid end or other support. Please indicate if attempts are being made to resolve.</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC092133</TD><TD>AC135776</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeat of AAAAT around 33150 bp in AC092133.</TD><TD> OTHER EVIDENCE: Microdeletion of 538 bp at 29310 bp in AC092133 is being evaluated.</TD><TD>in process</TD><TD>complex, other-likely requires PCR confirmation of polymorphism</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC108134</TD><TD>AJ003147</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>in process</TD><TD>next round, please include BAC or fosmid end support if any.</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC098805</TD><TD>AC106820</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeats around 600-700 bp of AC098805 interrupts the alignment.</TD><TD>&nbsp</TD><TD>return</TD><TD>requires BAC, fosmid end support. Please include size of apparent polymorphism</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC093525</TD><TD>AC106820</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeat of AT, around 61 Kb of AC106820 interrupts the alignment.</TD><TD>&nbsp</TD><TD>return</TD><TD>requires BAC, fosmid end support. Please include size of apparent polymorphism</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC025277</TD><TD>AC106783</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>in process</TD><TD>please include BAC, fosmid end support</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC025280</TD><TD>AC092341</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeats at 6 locations interrupt the alignment.  Join is supported by multiple BAC and fosmid paired ends.</TD><TD>&nbsp</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC018846</TD><TD>AC092126</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC018552</TD><TD>AC092118</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental.</TD><TD>Polymorphism in simple sequence repeat of CCAT around 10500 bp in AC018552 (around 20500 bp in AC092118) interrupts the alignment.  CCAT repeat is 1.3 Kb in AC092118 vs 1.74 Kb in AC018552.  Supported by multiple BAC and fosmid paired ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC012321</TD><TD>AC092115</TD><TD>&nbsp </TD><TD>other</TD><TD>?</TD><TD>Alu deletion of 99 bp around 101645 bp in AC092115, Microdeletion of 297 bp around 111070 bp in AC092115, and deletion of 4700 bp around 3060 in AC012321 interrupt the alignment.  Clones are from different BAC libraries.  Join is supported by multiple BAC and fosmid paired ends.</TD><TD>return</TD><TD>Probably OK, but deletion should be confirmed polymorphic eg by PCR.</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC012186</TD><TD>AC092368</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>in process</TD><TD>will resolve on next punchlist</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC007334</TD><TD>AC007608</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeat of G`s and A`s around 71428 bp in AC007334 interrupts the alignment.</TD><TD>&nbsp</TD><TD>pass</TD><TD>should include size of polymorphism, BAC/fosmid end support</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC007602</TD><TD>AC020766</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental</TD><TD>Polymorphism in simple sequence repeats of GAAA and CT around 7250 bp in AC007602 interrupts the alignment.  Supported by multiple BAC and fosmid paired ends.</TD><TD>return</TD><TD>Potential overrepresentation of BAC- and fosmid-ends.  Investigate.</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC007613</TD><TD>AC007616</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeat of TA around 5391 bp in AC007613 interrupts the alignment.</TD><TD>&nbsp</TD><TD>pass</TD><TD>OK, but in future include extent of variation and BAC/fosmid end support</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC009069</TD><TD>AC009094</TD><TD>&nbsp </TD><TD>other</TD><TD>Non-experimental.</TD><TD>SSLP and other. Polymorphism in simple sequence repeat of AT around 162370 bp in AC009094 interrupt the alignment. Two Alu polymorphisms, at 106374-106678 bp in AC009069 and 184078-184381 bp in AC009094.  Supported by multiple BAC and fosmid paired ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC009088</TD><TD>AC093520</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeats at several locations interrupts the alignment.</TD><TD>&nbsp</TD><TD>pass</TD><TD>should include BAC/fosmid end support, extensive overlap (49kb), size of polymorphic region</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC009090</TD><TD>AC023825</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeats of CA, and poly-A, around 192112 bp in AC009090 interrupts the alignment.</TD><TD>&nbsp</TD><TD>pass</TD><TD>should include BAC/fosmid end support, length of polymorphism.</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC009155</TD><TD>AC026461</TD><TD>&nbsp </TD><TD>other</TD><TD>Non-experimental.</TD><TD>SSLP and other. Polymorphism in simple sequence repeat of GGAA repeats around 104053 bp in AC009155 interrupts the alignment. Polymorphism between 78050-78823 bp of AC009155 and 146099-147039 bp and microdeletion of 330 bp around 120297 of AC026461 interrupt the alignment.  Supported by multiple BAC and fosmid paired ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC004682</TD><TD>AC009087</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>return</TD><TD>certificate incomplete</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC006352</TD><TD>AC092144</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC003108</TD><TD>AC012621</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeat of TA around 53200 bp in AC012621 interrupts the alignment.</TD><TD>&nbsp</TD><TD>pass</TD><TD>Should include BAC, fosmid end support in future</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC002331</TD><TD>AC008756</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Non-experimental. Polymorphism in simple sequence repeats at several location interrupts the alignment.  Finishing quality will be evaluated.</TD><TD>&nbsp</TD><TD>in process</TD><TD>please include size of polymorphism, BAC/fosmid end support</TD><TD><A HREF="mailto:doggett@lanl.gov">doggett@lanl.gov</A></TD></TR>
<TR><TD>AC007842</TD><TD>AC011536</TD><TD>&nbsp </TD><TD>other</TD><TD>no experimental</TD><TD>Clones overlapping in a tandem repeat.  Join supported by AC139454 and also 7 other fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AC008568</TD><TD>AC010482</TD><TD>&nbsp </TD><TD>other</TD><TD>no experimental</TD><TD>123bp indel in AC010482.  Join supported by one pair of BAC ends and 2 pairs of fosmid ends.</TD><TD>return</TD><TD>This in/del require s other support, eg PCR.</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AC008659</TD><TD>AC093290</TD><TD>&nbsp </TD><TD>other</TD><TD>No experimental</TD><TD>Number of repeats varies between clones.  AC008659 is tagged in the GenBank entry as unsure # of repeat copies.  Join is supported by 11 pairs of fosmid ends.</TD><TD>return</TD><TD>Is 11 pairs of fosmid-ends overrepresentation? Also, requires additional support eg. PCR</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AC008892</TD><TD>AC016596</TD><TD>&nbsp </TD><TD>sslp </TD><TD>no experimental	</TD><TD>543bp indel in TA region.  Join supported by 1 pair of BAC ends and 1 pair of fosmid ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AC008939</TD><TD>AC020930</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>AC008939 clipped too short.  Being refinished to extend the overlap.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AC010442</TD><TD>AC106772</TD><TD>&nbsp </TD><TD>other</TD><TD>no experimental	</TD><TD>Variation between Caltech D and RP11 clones in telomeric region.  Join supported by the draft clone AC138432.  Also one other pair of BAC ends and 7 pairs of fosmid ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AC010470</TD><TD>AC022450</TD><TD>&nbsp </TD><TD>sslp </TD><TD>no experimental</TD><TD>153bp simple sequence polymorphism at the end of the clone. Join supported by 3 pairs of BAC ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AC011407</TD><TD>AC090064</TD><TD>&nbsp </TD><TD>sslp </TD><TD>no experimental</TD><TD>185bp indel in TA region. Join supported by 2 pairs of BAC ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AC027310</TD><TD>AC027316</TD><TD>&nbsp </TD><TD>other</TD><TD>no experimental</TD><TD>VNTR 29mer attcttatatatattctaagaaatatac repeated twice in AC027310 and 13 times in AC027316.  Join supported by 2 pairs of BAC ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AC093528</TD><TD>AC094099</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>AC093528 clipped too short.  Being refinished to extend the overlap.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AC114288</TD><TD>AC114307</TD><TD>&nbsp </TD><TD>sslp </TD><TD>no experimental</TD><TD>214bp indel in TA region.  Join supported by 5 pairs of BAC ends.</TD><TD>return</TD><TD>Greg`s site lists no supporting BAC or fosmid-ends. requires BAC, fosmid end support or other</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AC022093</TD><TD>AC122716</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Repeat on the end of the clones.  Another clone in progress to reinforce the join.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AC010600</TD><TD>AC020936</TD><TD>&nbsp </TD><TD>sslp </TD><TD>no experimental</TD><TD>147bp indel in TA region. Join supported by 8 pairs of BAC ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:jane@shgc.stanford.edu">jane@shgc.stanford.edu</A></TD></TR>
<TR><TD>AL160006</TD><TD>AL355990</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>overlap between position 89784-91783 in AL160006 and 1-2000 in AL355990</TD><TD>return</TD><TD>Unsure re. the nature of the problem. What type of cert. is this? If previously a blunt/short join, then this is OK, please update certificate or accession to remove. If this is a polymorphic join, please resubmit with type and length of polymorphism.</TD><TD><A HREF="mailto:kb1@sanger.ac.uk,kem@sanger.ac.uk">kb1@sanger.ac.uk, kem@sanger.ac.uk</A></TD></TR>
<TR><TD>AL139141</TD><TD>AL390718</TD><TD>&nbsp </TD><TD>other</TD><TD>Sequence alignment and manual checking</TD><TD>21000bp overlap between position 155124-176094 in AL390718 and 1-20691 in AL139141. 13bp interruption to alignment at ~164637 in AL390718. Join Supported by 6 pairs BAC ends and 9 pairs Fosmid ends.</TD><TD>pass</TD><TD>Long overlap, small variation.</TD><TD><A HREF="mailto:kem@sanger.ac.uk">kem@sanger.ac.uk</A></TD></TR>
<TR><TD>AL035411</TD><TD>AL365187</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Sequence alignment and manual checking</TD><TD>2263bp oveplap. Variation in overlap and CA repeat at ~1530bp and ~1800bp in AL035411 interupting alignment. Join supported by 3 pairs BAC ends and 7 pairs Fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:kem@sanger.ac.uk">kem@sanger.ac.uk</A></TD></TR>
<TR><TD>AC099563</TD><TD>AC114496</TD><TD>&nbsp </TD><TD>other</TD><TD>Sequence alignment and manual checking</TD><TD>93835bp overlap between 93296-186551 in AC114496 and 1-93835 in AC099563. 230bp difference in TA repeat at ~118664bp in AC114496 interrupting alignment. Join Supported by 7 pairs BAC ends and 4 pairs Fosmid ends</TD><TD>pass</TD><TD>Long overlap, small variation.</TD><TD><A HREF="mailto:kem@sanger.ac.uk">kem@sanger.ac.uk</A></TD></TR>
<TR><TD>AC099570</TD><TD>AC099571</TD><TD>&nbsp </TD><TD>other</TD><TD>Sequence alignment and manual checking</TD><TD>86042bp overlap between position 81372-157413 in AC099571 and 1-85817 in AC099570. Disruption in alignment at 87144bp in AC099571. Join Supported by 8 pairs BAC ends and 11 pairs Fosmid ends</TD><TD>pass</TD><TD>Probably OK because of long overlap, and small number of differences.</TD><TD><A HREF="mailto:kem@sanger.ac.uk">kem@sanger.ac.uk</A></TD></TR>
<TR><TD>AC093575</TD><TD>AC095030</TD><TD>&nbsp </TD><TD>other</TD><TD>Sequence alignment and manual checking</TD><TD>44080bp overlap between position 1-43867 in AC095030 and 115151-159238 in AC093575. TA rich tandem repeat showing variation and several in/dels between the two accessions. TA repeat 220bp longer in AC093575 at ~134387bp. Join Supported by 2 pairs BAC ends and 2 pairs Fosmids ends</TD><TD>return</TD><TD>probably OK because of long overlap, but multiple `other` variations need additional support of polymorphism (eg, pcr)</TD><TD><A HREF="mailto:kem@sanger.ac.uk">kem@sanger.ac.uk</A></TD></TR>
<TR><TD>AC092801</TD><TD>AC094022</TD><TD>&nbsp </TD><TD>other</TD><TD>Sequence alignment and manual checking</TD><TD>71918bp overlap between position 79465-151198 in AC092801 and 1-71918 in AC094022. Join supported by 4 pairs BAC ends and 8 pairs Fosmid ends</TD><TD>pass</TD><TD>Probably OK because of long overlap, and small number of differences.</TD><TD><A HREF="mailto:kem@sanger.ac.uk">kem@sanger.ac.uk</A></TD></TR>
<TR><TD>AC009988</TD><TD>AC026395</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>overlap between position 119060-174398 in AC009988  and 1-55109 in AC026395</TD><TD>pass</TD><TD>Large overlap, even though lack of end-to-end align.</TD><TD><A HREF="mailto:lon@sanger.ac.uk">lon@sanger.ac.uk</A></TD></TR>
<TR><TD>AL157705</TD><TD>AL392103</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Join supported by 4 pair(s) of human BAC ends, CTD-2258H15,RP11-148C9,RP11-794O16,RP11-982I3. PCR on genomic sample in process to confirm the overlap.</TD><TD>in process</TD><TD>previously returned</TD><TD><A HREF="mailto:lon@sanger.ac.uk">lon@sanger.ac.uk</A></TD></TR>
<TR><TD>AL365356</TD><TD>BX276080</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>in process</TD><TD>should drop from list next round</TD><TD><A HREF="mailto:lon@sanger.ac.uk">lon@sanger.ac.uk</A></TD></TR>
<TR><TD>AC012469</TD><TD>AL772287</TD><TD>AC016826</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Overlap of 6bp at GAATTC cloning site. Join supported by genomic sequence AC016826 RP11-570J13.  Join supported by 1 pair(s) of human BAC ends and 7 pair(s) of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:lon@sanger.ac.uk">lon@sanger.ac.uk</A></TD></TR>
<TR><TD>AC121340</TD><TD>AC140846</TD><TD>&nbsp </TD><TD>other</TD><TD>resubmisson of AC002357 in process</TD><TD>682 bp N`s are in AC002357.1.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AL590763</TD><TD>BX276092</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>New this time. In process. Problem caused by error in assembly of AL590763 (MPIMG to correct). Can be avoided temporarily by use of switchpoint in AGP.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AC074034</TD><TD>AC108685</TD><TD>AC106713</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AL133277</TD><TD>Z70689</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>E-certificate previously submitted (NCBI joins).</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AL109801</TD><TD>Z68331</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Resubmission to produce a 2000 bp overlap is in progress.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AL035609</TD><TD>AL590069</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>We have exchanged emails about this one earlier. 2000 base overlap.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AL022155</TD><TD>Z70281</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Resubmission to produce a 2000 bp overlap is in progress.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AJ271735</TD><TD>AJ271736</TD><TD>none</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Originally submitted in one piece as Y19189. Split into two segments by EMBL according to the authors (IIGB). Inclusion of Y19189 will cause problems as it is the secondary accession of each.</TD><TD>disputed</TD><TD>This is an old compound accession on chrX. Almost certainly OK, but technically not according to standard. It is supported by fosmid and BAC end links.</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AL008706</TD><TD>AL133546</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>New this time. 100 base overlap.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AC109993</TD><TD>AL645722</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>In process. Both sequences have been recently resubmitted by the respective centres (IMB, IIGB). The problem is caused by a small piece of sequence (~4 kb) that is rotated between the two accessions. One must be incorrect as both clones are from the same X chromosome. Evaluation and correction will be carried out by the two centres. Submitted also as E-certificate to NCBI joins.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AF277315</TD><TD>L44140</TD><TD>&nbsp </TD><TD>other</TD><TD>IMB resequenced 676 bp of G6PD intron 2  containing six of the divergent positions in 6 women and 11 men of Caucasian origin. In all checked chromosomes the sequences were identical to AF277315, indicating that L44140 variants represent rare alleles in th</TD><TD>resubmission of L44140 clipped back to 2000 bp overlap with AF277315 would resolve the failure to detect overlap computationally. Join supported by 1 pair BAC ends, 6 pairs fosmid ends.</TD><TD>unresolved</TD><TD>In the strictest sense, this could be seen as unresolved, since the polymorphism was not seen in the panel, but the efforts are as far as anyone could reasonably ask. Please do not resolve by clipping this back. It is preferable to have some mechanism where a user looking at this region could retrieve the certificate to see what additional measures had been taken to confirm. No further action is necessary.</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AC003659</TD><TD>AC121344</TD><TD>&nbsp </TD><TD>line </TD><TD>PCR on genomic samples in process to confirm</TD><TD>~400bp LINE element present in AC121344.5 breaks alignment; join is supported high % identity across the overlap</TD><TD>in process</TD><TD>if BAC or fosmid end support is available, PCR is not required</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AC108359</TD><TD>AC117405</TD><TD>&nbsp </TD><TD>sslp </TD><TD>PCR on genomic samples in process to confirm</TD><TD>dinucleotide repeat region is ~75bp longer in AC108359; join is supported by 5 pairs of BAC ends</TD><TD>pass</TD><TD>with BAC end support, no further action is required</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AC006062</TD><TD>AC073533</TD><TD>&nbsp </TD><TD>sslp </TD><TD>PCR on genomic sample in process to confirm the polymorphism</TD><TD>AG repeat region 350 bp longer in AC006062.4; Join supported by 1 pair of human BAC end.</TD><TD>pass</TD><TD>with BAC end support, no further action is required</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AC091810</TD><TD>AL592164</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>AC091810.2 contains 30 bp of vector sequence. Clipping and resubmission will solve this problem</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AC108359</TD><TD>AC112496</TD><TD>&nbsp </TD><TD>sslp </TD><TD>PCR on genomic samples in process to confirm</TD><TD>trinucleotide repeat region is ~765bp longer in AC112496; join is supported by high % identity across the overlap and 5 pairs of BAC ends</TD><TD>pass</TD><TD>with BAC end support, no further action is required</TD><TD><A HREF="mailto:mtr@sanger.ac.uk">mtr@sanger.ac.uk</A></TD></TR>
<TR><TD>AL591379</TD><TD>AL845321</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Pericentromeric segmental duplicated region</TD><TD>in process</TD><TD>previously E-CERTIFICATE IN PROCESS.  TPF revised in this region. Should wait to resolve through next round of punchlists. alignment does not extend to end of AL845321</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL359955</TD><TD>AL445665</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Pericentromeric segmental duplicated region</TD><TD>in process</TD><TD>previously E-CERTIFICATE IN PROCESS.  TPF revised in this region. Should wait to resolve through next round of punchlists.  alignment does not extend to end of AL445665</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL445595</TD><TD>AL512635</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Additional 73bp of sequence in AL445595;  GAGCTCGGTACCCGGGGATCTAGAGTACTGGTTGAAACCCGG<BR>ATCATGCATCAGTACGCGTGAGCTCGAATTC  represents contamination; matches cloning vector - pBACe3.6.  Resolve by re-submitting AL445595.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL163528</TD><TD>AL442637</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>AL163528 is currently not finished, which is sending  the algnment off.  Resolve by finishing AL163528</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL163528</TD><TD>AL357614</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Presence of Ns in finished clone AL357615.  Version .7 has been corrected. Join supported by 4 pair(s) of human BAC ends. Join supported by 3 pair(s) of human fosmid ends</TD><TD>in process</TD><TD>Should be resolved in next round</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL355151</TD><TD>AL627230</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Pericentromeric segmental duplicated region</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL592525</TD><TD>AL954685</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Pericentromeric segmental duplicated region</TD><TD>disputed</TD><TD>previously returned. This will not be resolved soon.</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL591438</TD><TD>AL627230</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Pericentromeric segmental duplicated region</TD><TD>in process</TD><TD>previously E-CERTIFICATE IN PROCESS.  TPF revised in this region. Should wait to resolve through next round of punchlists. no alignment found</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL590711</TD><TD>BX001034</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Pericentromeric segmental duplicated region</TD><TD>in process</TD><TD>previously E-CERTIFICATE IN PROCESS.  TPF revised in this region. Should wait to resolve through next round of punchlists. no alignment found</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL445643</TD><TD>BX001034</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Pericentromeric segmental duplicated region</TD><TD>in process</TD><TD>previously E-CERTIFICATE IN PROCESS.  TPF revised in this region. Should wait to resolve through next round of punchlists.  AL445643 is contained in BX001034</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL359955</TD><TD>AL512605</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Pericentromeric segmental duplicated region</TD><TD>in process</TD><TD>previously E-CERTIFICATE IN PROCESS.  TPF revised in this region. Should wait to resolve through next round of punchlists.  internal alignment (no dovetail)</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL591424</TD><TD>AL954642</TD><TD>&nbsp </TD><TD>other</TD><TD>In progress</TD><TD>GC rich tandem  9qtleomeric region. Indel in tandem, 22kb match then 305bp missing from AL954642.9 (RP11-885N19) from 21904. 302bp extra in tandem in AL591424.11 (RP11-424E7) from 135208-135513. Then 66kb match. Also Riethman half YAC in pipeline.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL162415</TD><TD>BX248098</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Pericentromeric segmental duplicated region  supported by 11 pair(s) of human BAC ends</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL162731</TD><TD>AL591926</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Pericentromeric segmental duplicated region. Join supported by 6 pair(s) of human BAC ends. Join supported by 15 pair(s) of human fosmid ends</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>BX005195</TD><TD>BX248112</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Pericentromeric segmental duplicated region</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL354874</TD><TD>AL391987</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Pericentromeric segmental duplicated region</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL357615</TD><TD>AL391117</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>Presence of Ns in finished clone AL357615.  Version .7 has been corrected. Join supported by 9 pair(s) of human BAC ends. Join supported by 1 pair(s) of human fosmid ends</TD><TD>in process</TD><TD>Should be resolved in next round</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AL158832</TD><TD>AL356244</TD><TD>&nbsp </TD><TD>other</TD><TD>To be tested</TD><TD>Alu. 9ptelomeric region. Indels in Alu repeats disrupting join.  Join supported by 2 pair(s) of human BAC ends. Join supported by 10 pair(s) of human fosmid ends</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sjh@sanger.ac.uk">sjh@sanger.ac.uk</A></TD></TR>
<TR><TD>AC105266</TD><TD>AC134025</TD><TD>&nbsp </TD><TD>sslp </TD><TD>cross_match, inspection in Consed</TD><TD>Alignment broken by deletions in imperfect  `ggaaa` pentanucleotide repeats.  Join supported by 6  pairs of BAC ends and 5 pairs of fosmid ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC013421</TD><TD>AC140060</TD><TD>&nbsp </TD><TD>none </TD><TD>none</TD><TD>GenBank record for AC140060 is incorrect. The  previous version is correct (gi:28460722) and will be updated</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC105765</TD><TD>AC139618</TD><TD>&nbsp </TD><TD>other</TD><TD>None</TD><TD>Gap between this pair is now filled with RP11- 158G18 (Type1) on tpf.</TD><TD>return</TD><TD>On Greg`s site says "sequence_not_yet_analyzed"; May require BAC, fosmid end support or other. Or, may resolve with next compute.</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC134998</TD><TD>AC136189</TD><TD>&nbsp </TD><TD>sslp </TD><TD>PCR on genomic samples in progress to confirm</TD><TD>tandem repeat region is ~150bp longer in AC134998</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC104448</TD><TD>AC134772</TD><TD>&nbsp </TD><TD>other</TD><TD>cross_match, inspection in Consed</TD><TD>SINE/SSLP Three breaks in alignment:  First break due to  ALU repeats, second due to dinucleotide repeat  polymorphism and third break due to ALU repeats copy  number.  Join supported by 7 pairs of fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC107304</TD><TD>AC108697</TD><TD>&nbsp </TD><TD>other</TD><TD>sequence</TD><TD>transposon sequence in AC107304 not excised; resubmission in progress</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC025572</TD><TD>AC116991</TD><TD>&nbsp </TD><TD>line </TD><TD>cross_match, RepeatMasker</TD><TD>Alignment broken by a 6Kb LINE insertion in  AC116991.2.  Join supported by 5 pairs of BAC and 12  pairs of fosmid ends</TD><TD>pass</TD><TD>is 12 pairs fosmid-ends overrepresentation?</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC063927</TD><TD>AC117435</TD><TD>&nbsp </TD><TD>other</TD><TD>PCR on genomic samples in progress to confirm</TD><TD>~255bp insertion in AC063927 breaks alignment Polymorphism; 299 bp AluY insertion in AC112646.9;</TD><TD>in process</TD><TD>requires BAC, fosmid end support or other</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC078788</TD><TD>AC117421</TD><TD>&nbsp </TD><TD>sine </TD><TD>&nbsp</TD><TD>120bp AluY insertion in AC117421.6; Join supported by 2 pairs of human BAC ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC087879</TD><TD>AC090052</TD><TD>&nbsp </TD><TD>sslp </TD><TD>&nbsp</TD><TD>GT simple repeat longer in AC087879; join is supported by 2 pairs of BAC ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC007934</TD><TD>AC128680</TD><TD>&nbsp </TD><TD>other</TD><TD>PCR on genomic sample in progress to confirm polymorphism</TD><TD>LINE and OTHER. 70 bp insertion and 520 bp L1MB6 insertion in  AC128680.6; Join supported by 2 pairs of human BAC ends.</TD><TD>return</TD><TD>`other` requires additional support, eg PCR</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC099047</TD><TD>AC131013</TD><TD>&nbsp </TD><TD>line </TD><TD>PCR on genomic samples in progress to confirm</TD><TD>~8kb LINE element present in AC131013 breaks Alignment</TD><TD>pass</TD><TD>pcr confirm not required</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC112646</TD><TD>AC130888</TD><TD>&nbsp </TD><TD>sine </TD><TD>confirmed by genomic PCR polymorphism</TD><TD>299 bp AluY insertion in AC112646.9;</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC104190</TD><TD>AC124916</TD><TD>&nbsp </TD><TD>sine </TD><TD>cross_match, inspection in Consed</TD><TD>Alignment broken by 131 bases of ALU sequences  in AC124916.3.  Join supported by 2 pairs of BAC and 3  pairs of fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC108486</TD><TD>AC109585</TD><TD>&nbsp </TD><TD>other</TD><TD>none</TD><TD>Problem with AC109585 - currently being  resubmitted.</TD><TD>pass</TD><TD>Already resolved in TPF</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC099780</TD><TD>AF152364</TD><TD>&nbsp </TD><TD>other</TD><TD>none</TD><TD>Two breaks in the alignment:  The first is due  to sequence length polymorphisms at 87223 (AC099780  coordinates) and the other is due to dinucleotide repeats  polymorphism at 95133.  Otherwise join looks OK.  Join  supported by 7 pairs of BAC and 10 pairs of fosmid ends.   PCR validation pending.</TD><TD>in process</TD><TD>OK, but encourage pcr validation</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC103587</TD><TD>AC114398</TD><TD>&nbsp </TD><TD>other</TD><TD>none</TD><TD>310-base internally duplicated region in  AC114398 between 225316 and 225625.  Another copy between  223978 and 224287.  Join supported by 4 pairs of BAC and  3 pairs of fosmid ends.  PCR validation pending.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC096917</TD><TD>AF152363</TD><TD>&nbsp </TD><TD>other</TD><TD>cross_match, inspection in Consed</TD><TD>AF152363 not finished to current standards.   Looks OK.</TD><TD>in process</TD><TD>already removed from punchlist</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC026305</TD><TD>AC068757</TD><TD>&nbsp </TD><TD>sslp </TD><TD>PCR on genomic samples in progress to confirm</TD><TD>tandem repeat region is ~225bp longer in AC026305</TD><TD>pass</TD><TD>pcr not required</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC089985</TD><TD>AC090051</TD><TD>&nbsp </TD><TD>sine </TD><TD>&nbsp</TD><TD>AluYb8 element present in AC090051 breaks alignment; join supported by 6 pairs of BAC ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC034187</TD><TD>AC068312</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 41kb. Break in alignment caused by imperfect 80mer repeat that is 876bps larger in AC034187. 4 supporting BACs, 5 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC046134</TD><TD>AC110716</TD><TD>&nbsp </TD><TD>sslp </TD><TD>join is supported by 7 pairs of human BAC ends</TD><TD>GT simple repeat region 10 bp longer in  AC110716.5</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC069261</TD><TD>AC073916</TD><TD>&nbsp </TD><TD>none </TD><TD>none</TD><TD>AC069261 is being resubmitted (see gi:22094222)</TD><TD>in process</TD><TD>will disappear</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC018842</TD><TD>AC066608</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 52kb. Break in alignment caused by a simple repeat (TA)n larger in AC066608 by 686bps. 2 supporting BACs, 6 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC021999</TD><TD>AC090955</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 98kb. Break in alignment caused by two simple repeats (CA)n and (TA)n larger in AC021999 by 152bps. 3 supporting BACs, 9 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC023481</TD><TD>AC087859</TD><TD>&nbsp </TD><TD>other</TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 55kb. Break in alignment caused by SVA repeats larger in AC087859.3 by 508bps. 1 supporting BACs, 5 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC024163</TD><TD>AC026194</TD><TD>&nbsp </TD><TD>sine </TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 88kb. Two breaks in alignment caused by two 322bp SINE insertion in AC024163. 5 supporting BACs, 5 supportings fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC006205</TD><TD>AC021054</TD><TD>&nbsp </TD><TD>sslp </TD><TD>correct sequence alignment</TD><TD>AC006205 was resubmitted, eliminating problem</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC004216</TD><TD>AC007425</TD><TD>&nbsp </TD><TD>sslp </TD><TD>&nbsp</TD><TD>repeat region ~500bp longer in AC007425; join is supported by 3 pairs of BAC ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC004801</TD><TD>AC074029</TD><TD>&nbsp </TD><TD>sslp </TD><TD>&nbsp</TD><TD>TA repeat region ~200bp longer in AC074029; join supported by 4 pairs of BAC ends</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC010370</TD><TD>AC026118</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Join supported by 4 pairs of human BAC ends</TD><TD>tandem repeat 200 bp longer in AC026118.17</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC009811</TD><TD>AC055712</TD><TD>&nbsp </TD><TD>other</TD><TD>&nbsp</TD><TD>157 bp insertion in AC055712.13; Join supported by 4 pairs of human BAC ends</TD><TD>return</TD><TD>`other` requires additional support eg pcr.</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC006512</TD><TD>U47924</TD><TD>&nbsp </TD><TD>other</TD><TD>sequence alignment</TD><TD>n`s in submission; resubmission in progress</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC011609</TD><TD>AC087428</TD><TD>&nbsp </TD><TD>sine </TD><TD>Global Sequence Alignment, RepeatMasker, Blast2</TD><TD>overlap of 60kb. Break in alignment caused by a SINE repeat larger in AC011609 by 267bps. 4 supporting BACs, 4 supporting fosmids.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu">sscherer@bcm.tmc.edu</A></TD></TR>
<TR><TD>AC010200</TD><TD>AC010209</TD><TD>&nbsp </TD><TD>other</TD><TD>sequence alignment</TD><TD>n`s in submission; resubmission in progress</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu]">sscherer@bcm.tmc.edu]</A></TD></TR>
<TR><TD>AC073244</TD><TD>AC092969</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu]">sscherer@bcm.tmc.edu]</A></TD></TR>
<TR><TD>AC093927</TD><TD>AC098971</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>AC093927 contains vector sequence on the ends; UWGC is filtering and resubmitting; join is supported by 6 pair of BAC-ends</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu]">sscherer@bcm.tmc.edu]</A></TD></TR>
<TR><TD>AC092947</TD><TD>AC130416</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu]">sscherer@bcm.tmc.edu]</A></TD></TR>
<TR><TD>AC091212</TD><TD>AC108678</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu]">sscherer@bcm.tmc.edu]</A></TD></TR>
<TR><TD>AC079602</TD><TD>Z98941</TD><TD>&nbsp </TD><TD>none </TD><TD>?</TD><TD>clone pair is currently being analyzed; join is supported by 2 pair of BAC-ends</TD><TD>in process</TD><TD>may need pcr analysis if variation is not simple</TD><TD><A HREF="mailto:sscherer@bcm.tmc.edu]">sscherer@bcm.tmc.edu]</A></TD></TR>
<TR><TD>AC087447</TD><TD>AC103854</TD><TD>AC090163</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>6 bp overlap supported by finished GenBank sequence AC090163.10 (RP11-706A13)</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AC023080</TD><TD>AC134982</TD><TD>AC126324</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>6 bp overlap supported by finished GenBank sequence AC126324.3 (RP11-358H20)</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP000807</TD><TD>AP002813</TD><TD>AP002366</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>6 bp overlap supported by finished GenBank sequence AP002366.4 (RP11-142N13)</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP001189</TD><TD>AP003119</TD><TD>AP003101</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>6 bp overlap supported by finished GenBank sequence AP003101.2 (RP11-30J7)</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP003120</TD><TD>AP006215</TD><TD>AC026168</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>6 bp overlap supported by finished GenBank sequence AC026168.3 (RP11-156E23)</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP001790</TD><TD>AP001876</TD><TD>AP000652</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>6 bp overlap supported by finished GenBank sequence AP000652.4 (CMB9-8G5)</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP000848</TD><TD>AP003781</TD><TD>AP000929</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>6 bp overlap supported by finished GenBank sequence AP000929.5 (RP11-778B17)</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP002453</TD><TD>AP003387</TD><TD>AP003384</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>6 bp overlap supported by finished GenBank sequence AP003384.2 (RP11-35N19)</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP005435</TD><TD>AP005814</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, RepeatMasker</TD><TD>SINE/LINE/SSLP. Overlap of 28 kb. Break in alignment caused by deletion of six neighboring SINE/LINE/DNA elements and a (CTTTG)n region for a total of 6014 bp found in AP005435 but not present in AP005814. Join supported by 3 pairs of human BAC ends.</TD><TD>pass</TD><TD>multiple, but fairly straightforward</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP001166</TD><TD>AP005202</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, Tandem Repeats Finder, RepeatMasker</TD><TD>Overlap of 37 kb. Break in alignment caused by near-perfect 14-mer repeat (CATGGCCCTATATG) that is 364 bp longer in AP001166. There are 34.1 copies in AP001166 and 8.1 copies in AP005202. Join supported by 2 pairs of human BAC ends and 5 pairs of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP001028</TD><TD>AP001029</TD><TD>&nbsp </TD><TD>line </TD><TD>Blast2, RepeatMasker</TD><TD>Overlap of 118 kb. Break in alignment caused by deletion of a 6228 bp LINE/L1HS element found in AP001028 but not present in AP001029. Join supported by 1 pair of human BAC ends and 11 pairs of human fosmid ends.</TD><TD>pass</TD><TD>is 11 pairs fosmid-ends overrepresentation?</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP003783</TD><TD>AP005401</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, RepeatMasker</TD><TD>LTR/SSLP. Overlap of 6.3 kb. Break in alignment caused by deletion of LTR/ERVL element and a (TA)n region for a total of 235 bp found at the end AP003783 but not present in AP005401. Join supported by 2 pairs of human BAC ends and 1 pair of human fosmid ends. The sequence will be confirmed.</TD><TD>pass</TD><TD>encourage additional confirmation in process</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP005232</TD><TD>AP005379</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, Tandem Repeats Finder</TD><TD>Overlap of 24 kb. Break in alignment caused by imperfect 15-mer repeat (ATCATCATCATCACC) that is 198 bp longer in AP005379. There are 39.7 copies in AP005379 and 16.6 copies in AP005232. No supporting BACs/fosmids, but otherwise high % identity.</TD><TD>return</TD><TD>requires BAC, fosmid end support or other</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP003037</TD><TD>AP003108</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, RepeatMasker</TD><TD>SINE/LINE. Overlap of 14 kb. Break in alignment caused by deletion of two neighboring SINE/LINE elements for a total of 338 bp found in AP003108 but not present in AP003037. Join supported by 4 pairs of human BAC ends and 5 pairs of human fosmid ends. Also confirmed by PCR.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP003395</TD><TD>AP003396</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, RepeatMasker</TD><TD>Overlap of 71 kb. Break in alignment caused by deletion of 216 bp in an SVA element in AP003396. Join supported by 2 pairs of human BAC ends and 10 pairs of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP003500</TD><TD>AP003782</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, Tandem Repeats Finder</TD><TD>Overlap of 60 kb. Break in alignment caused by imperfect 34-mer repeat (ATATATTCTACAGAATACGTATATTCTATTCTAC) that is 314 bp longer in AP003500. There are 21.3 copies in AP003782 and 30.3 copies in AP003500. Join supported by 2 pairs of human BAC ends and 7 pairs of human fosmid ends.</TD><TD>pass</TD><TD>VNTR</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP003780</TD><TD>U73634</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Overlap of 15 kb. U73634 is an old cosmid clone of questionable quality finished at University of Texas Southwestern Medical Center. The last 164 bases of U73634 do not match the corresponding region of AP003780. We are going to replace U73634 with another clone, RP11-466C23.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP000311</TD><TD>AP000312</TD><TD>none</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Short overlap is an artifact of the nested deletion method. Briefly, a library of nested deletion clones was constructed, a map was then made by using the end sequences from multiple restriction fragments, the overlaps for each clone were confirmed by sequencing, and finally a minimal set of clones was chosen for finishing. Join supported by 1 pair of human fosmid ends.</TD><TD>disputed</TD><TD>Likely to be OK. However, sequencing by nested deletions was not discussed in view of the certificates. Computational verification will not be possible without a spanning clone under current criteria.</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP000336</TD><TD>AP000337</TD><TD>none</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Short overlap is an artifact of the nested deletion method. Briefly, a library of nested deletion clones was constructed, a map was then made by using the end sequences from multiple restriction fragments, the overlaps for each clone were confirmed by sequencing, and finally a minimal set of clones was chosen for finishing. Join supported by 1 pair of human BAC ends and 2 pairs of human fosmid ends.</TD><TD>disputed</TD><TD>Likely to be OK. However, sequencing by nested deletions was not discussed in view of the certificates. Computational verification will not be possible without a spanning clone under current criteria.</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP000438</TD><TD>AP001160</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, RepeatMasker</TD><TD>SINE/LINE. Overlap of 74 kb. Break in alignment caused by deletion of two neighboring SINE/LINE elements for a total of 460 bp found in AP001160 but not present in AP000438. Join supported by 5 pairs of human BAC ends and 2 pairs of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP000689</TD><TD>AP000690</TD><TD>&nbsp </TD><TD>sine </TD><TD>Blast2, RepeatMasker</TD><TD>Overlap of 7.6 kb. Break in alignment caused by deletion of a 299 bp SINE/AluSg element found in AP000690 but not present in AP000689. Join supported by 1 pair of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP000702</TD><TD>AP000703</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Break in alignment caused by a ~26 kb deletion in AP000702 between bases 38400 and 38401 which is covered by AP000703 and AP000704; the next few hundred bases after the deletion point are chimeric, so the last 2475 bp are not used in the AGP. Join supported by 1 pair of human fosmid ends.</TD><TD>return</TD><TD>requires additional support eg. Pcr.</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP000708</TD><TD>AP003069</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, Tandem Repeats Finder</TD><TD>Overlap of 63 kb. Break in alignment caused by near perfect 125-mer repeat (CAGCACGGCCACTAGCTCTGCTCAAGAAACAAGTCTGCGTC<BR>TCCTTATTGTGCTAGTTCATTCCTCAGTTCACTACACAATCCT<BR>TCAACGTTGGCCTCTTTGCCTGTCTGCACACTGTGCCTGGT) that is 107 bp longer in AP000708. There is 1 copy in AP003069 and 2 copies in AP000708. Join supported by 2 pairs of human BAC ends and 11 pairs of human fosmid ends.</TD><TD>pass</TD><TD>VNTR; Is 11 pairs fosmid-ends overrepresentation?</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP000729</TD><TD>AP001885</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2, Tandem Repeats Finder, RepeatMasker</TD><TD>Overlap of 6.5 kb. Break in alignment caused by simple repeat (TG)n that is 12 bp longer in AP001885. There are 17 copies in AP000729 and 23 copies in AP001885. Join supported by 2 pairs of human BAC ends and 2 pairs of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP000924</TD><TD>AP001883</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Join supported by 2 pairs of human BAC ends and 2 pairs of human fosmid ends. Overlap of 60 kb. The last 43 bp of AP000924 do not match the corresponding position within AP001883, the sequence will be confirmed. There is high % identity otherwise.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP000926</TD><TD>AP003040</TD><TD>&nbsp </TD><TD>sine </TD><TD>Blast2, RepeatMasker</TD><TD>Overlap of 4 kb. Break in alignment caused by deletion of three neighboring SINE elements and an AT_rich region for a total of 1146 bp found in AP000926 but not present in AP003040. Join supported by 3 pairs of human BAC ends and 1 pair of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP001050</TD><TD>AP001051</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, Tandem Repeats Finder</TD><TD>Overlap of 27 kb. Break in alignment caused by imperfect 20-mer repeat (TGCTGAGGGAGATGAGGGCC) that is 504 bp longer in AP001051. There are 95.9 copies in AP001050 and 120.9 copies in AP001051. Join supported by 3 pairs of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP001064</TD><TD>AP001065</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, Tandem Repeats Finder</TD><TD>Overlap of 25 kb. Break in alignment caused by imperfect 41-mer repeat (CACTGTCTTGGTTGGCGTAGCCTTGTAGTAAGTTTTGACCG) that is 727 bp longer in AP001065. There are 25.9 copies in AP001064 and 40.9 copies in AP001065. Join supported by 1 pair of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP001620</TD><TD>AP001621</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, Tandem Repeats Finder</TD><TD>Overlap of 20 kb. Break in alignment caused by imperfect 50-mer repeat (CAACACCTCCATCACCAACAGAAACCAGGCGGGGCCAGCTGA<BR>CCCCAGAG) that is 235 bp longer in AP001620. There are 13.1 copies in AP001620 and 8.1 copies in AP001621. Join supported by 1 pair of human BAC ends and 5 pairs of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP001625</TD><TD>AP001626</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, Tandem Repeats Finder</TD><TD>Overlap of 39 kb. Break in alignment caused by imperfect 51-mer repeat (AGCCTCCCGCCCCGTTTACATCCACGCAGGCGTTTCCCCTT<BR>ACCTGCACCG) that is 2023 bp longer in AP001625. There are 58 copies in AP001625 and 16.4 copies in AP001626. Join supported by 1 pair of human BAC ends and 10 pairs of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP001874</TD><TD>AP003170</TD><TD>&nbsp </TD><TD>sine </TD><TD>Blast2, RepeatMasker</TD><TD>Overlap of 55 kb. Break in alignment caused by deletion of 1485 bp of SINE/Alu elements found in AP001874 but not present in AP003170. Join supported by 4 pairs of human BAC ends and 3 pairs of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP001883</TD><TD>AP001980</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, Tandem Repeats Finder</TD><TD>Overlap of 31 kb. Break in alignment caused by imperfect 17-mer repeat (AGAGTATATGCAGTGAT) that is 223 bp longer in AP001980. There are 65.2 copies in AP001883 and 79.2 copies in AP001980. Join supported by 2 pairs of human BAC ends and 1 pair of human fosmid ends. In addition, the first 10 bp of AP001980 do not match the corresponding position within AP001883, the sequence will be confirmed.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP001925</TD><TD>AP002433</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, RepeatMasker</TD><TD>SINE/LINE/SSLP. Overlap of 16 kb. Break in alignment caused by deletion of six neighboring SINE/LINE elements and a (TA)n region for a total of 1382 bp found in AP002433 but not present in AP001925. Join supported by 7 pairs of human BAC ends and 5 pairs of human fosmid ends.</TD><TD>pass</TD><TD>multiple, but fairly straightforward</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AP001994</TD><TD>AP003390</TD><TD>&nbsp </TD><TD>sine </TD><TD>Blast2, RepeatMasker</TD><TD>Overlap of 97 kb. Break in alignment caused by deletion of a 264 bp SINE/AluYa5a2 element found in AP001994 but not present in AP003390. Join supported by 6 pairs of human BAC ends and 19 pairs of human fosmid ends.</TD><TD>return</TD><TD>Probably OK, but is 19 pairs fosmid-ends overrepresentation? Please comment.</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AL109761</TD><TD>AP000432</TD><TD>&nbsp </TD><TD>sslp </TD><TD>Blast2, Tandem Repeats Finder, RepeatMasker</TD><TD>Break in alignment caused by simple repeat (T)n that is 22 bp longer in AP000432. There are 42 copies in AP000432 and 20 copies in AL109761. Join supported by excellent sequence alignment. Join supported by 2 pairs of human BAC ends and 1 pair of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AF020802</TD><TD>AP000687</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, RepeatMasker</TD><TD>Overlap of 52 kb. Break in alignment caused by partial deletion of a SVA element which is 231 bp longer in AP000687 than in AF020802. Join supported by 4 pairs of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AF045449</TD><TD>AF064857</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, Tandem Repeats Finder</TD><TD>Overlap of 12 kb. Break in alignment caused by two neighboring imperfect repeats, the first is a 59-mer (TGTTGGCCTCCTTTTCCTATTGCCTTCTCCTCAATGCCTGTG<BR>TCTGAAATTGCCATAGG) and the second is a 74-mer (ATAGGTGTTGGCCTCCTTTTCCTATTGCCTTCTCCTCTATGCC<BR>TGTGTCTGAAATTGCCCTCCACCCTTCTCTT) which results in AF045449 being 198 bp longer. There are 1.9 copies of the 51-mer and 7.1 copies of the 74-mer in AF064857. There are 3.9 copies of the 51-mer and 8.1 copies of the 74-mer in AF045449. Join supported by 2 pairs of human fosmid ends.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AF135405</TD><TD>AP001138</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>There is an 85 kb deletion in AF135405 which is covered by AP001138.</TD><TD>return</TD><TD>requires additional confirmation that deletion is a polymorphism eg by pcr.</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AF227510</TD><TD>AF260011</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, RepeatMasker</TD><TD>SINE/LINE. Overlap of 30 kb. Break in alignment caused by deletion of a 311 bp SINE/AluY element and a 217 bp LINE/L1MC1 element both found in AF227510 but not present in AF260011. Join supported by 2 pairs of human BAC ends and 2 pairs of human fosmid ends.</TD><TD>pass</TD><TD>multiple, but straightforward</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AF240627</TD><TD>AL109763</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, RepeatMasker, Tandem Repeats Finder</TD><TD>LINE/other. Overlap of 105 kb. Break in alignment caused by two differences. The first is a variation in a LINE/L1 element which is about 300 bp shorter in AF240627. The second is caused by an imperfect 50-mer repeat (GGCACGCCATTCCCTGTTACCTGCATCAGTGACTGATCAGA<BR>ATGGGGTAA) that is 250 bp longer in AF240627. There are 8.9 copies in AL109763 and 14.9 copies in AF240627. Join supported by 1 pair of human BAC ends and 7 pairs of human fosmid ends.</TD><TD>pass</TD><TD>VNTR</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AC027239</TD><TD>AP001998</TD><TD>&nbsp </TD><TD>other</TD><TD>Blast2, RepeatMasker</TD><TD>LINE/SSLP. Overlap of 68 kb. Break in alignment caused by deletion of two neighboring LINE/L1HS elements and a (A)n region for a total of 1637 bp found in AC027239 but not present in AP001998. Join supported by 10 pairs of human BAC ends. Another clone, RP11-24F19, is being sequenced to confirm this overlap.</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:taylor@gsc.riken.go.jp">taylor@gsc.riken.go.jp</A></TD></TR>
<TR><TD>AC073071</TD><TD>AC073387</TD><TD>AC073103</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Minimal overlap was finished, but AC073103 confirms overlap. Overlap also supported by BAC end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC073081</TD><TD>AC109822</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>Alu. AC073081.5 contains 312bp more data than AC109822.5, indel present, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC073094</TD><TD>AC091729</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>indel present, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join also supported by fosmid and bac end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC073216</TD><TD>AC093655</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC093655.4 contains 150 bp more than AC073216.7, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC073236</TD><TD>AF109076</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC073236.8 contains 231bp more than AF109076.1, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC068196</TD><TD>AC104076</TD><TD>&nbsp </TD><TD>sine </TD><TD>repeatmasker</TD><TD>Alu. 68409 bp overlap, AC104076.5 contains 313bp more data than AC068196.4, AC068196.4 contains 295bp more data than AC104076.5, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC058823</TD><TD>AC108159</TD><TD>&nbsp </TD><TD>other</TD><TD>repeatmasker</TD><TD>LTR/MaLR, LINE/L2, SINE/MIR, DNA/MER2_type. 44728 bp overlap, AC058823.4 contains 2196bp more data than AC108159.5, join supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC066692</TD><TD>AC079924</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>SINE/Alu, LINE/L1. AC066692.3 contains 541bp more data than AC079924.6, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC073249</TD><TD>AF104455</TD><TD>AC073249</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC073336</TD><TD>AC093726</TD><TD>AC073077</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Minimal overlap was finished, but AC073077 confirms overlap. Overlap also supported by BAC and fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC073344</TD><TD>AC093592</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>Alu. AC073344.7 contains 298bp and 307bp more data than AC093592.3, multiple indels also present, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC073879</TD><TD>AC108929</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>Alu. AC073879.7 contains 4bp more data than AC108929.4, AC108929.4 contains 1982bp more data than AC073879.7, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC074257</TD><TD>AC093583</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC074257.5 contains 1165bp more than AC093583.3, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>11 fosmid-ends - is this overrepresentation?</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC068537</TD><TD>AC110807</TD><TD>&nbsp </TD><TD>line </TD><TD>polymorphic based on genomic pcr assay</TD><TD>L2. AC110807.3 contains 218bp more data than AC068537.5, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC025561</TD><TD>AC093764</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>2084 bp overlap, indel present, join supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC025821</TD><TD>AC093803</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>SINE/Alu, LTR/ERV1, LINE/L2, LINE/L1. AC093803.3 contains 8489bp more data than AC025821.7, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC020601</TD><TD>AC093674</TD><TD>AC019304</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>another accession sequenced to confirm join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC021127</TD><TD>AC105917</TD><TD>AC025602</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>another accession sequenced to confirm join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC107464</TD><TD>AC139887</TD><TD>&nbsp </TD><TD>line </TD><TD>repeatmasker</TD><TD>L1. 4p telomere, 91687 bp overlap, AC139887.1 contains 1003bp more than AC107464.5, AC107464.5 contains 133bp more than AC139887.1, multiple indels present, supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC093877</TD><TD>AC131947</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>Alu. multiple indels present, join supported by bac and fosmid endpairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC019195</TD><TD>AL133244</TD><TD>AC019195</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC093886</TD><TD>AC131945</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>SINE/Alu, LINE/L1. AC131945.1 contains 226bp more data than AC093886.2, AC093886.2 contains 6062bp more data than AC131945.1, join supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC017084</TD><TD>AC022201</TD><TD>&nbsp </TD><TD>other</TD><TD>repeatmasker</TD><TD>DNA/MER1_type. AC017084.8 contains 14bp more data than AC022201.4, join is also supported by fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC018643</TD><TD>AC105443</TD><TD>&nbsp </TD><TD>other</TD><TD>&nbsp</TD><TD>62648bp overlap, AC105443.2 contains 15bp more on the end than AC018643.3, this is UWGC clone, we will contact them to see if this is possibly a vector issue. overlap supported by bac and fosmid end pairs</TD><TD>in process</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC018645</TD><TD>AC018648</TD><TD>&nbsp </TD><TD>sine </TD><TD>repeatmasker</TD><TD>Alu. 27084bp overlap, AC018648.5 contains 551bp more than AC018645.4</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC018678</TD><TD>AC062032</TD><TD>AC079297</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>another accession sequenced to confirm join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC018804</TD><TD>AC018865</TD><TD>&nbsp </TD><TD>other</TD><TD>in progress</TD><TD>66488 bp overlap, multiple indels present, join is also supported by bac and fosmid end pairs</TD><TD>return</TD><TD>12 bac-end and 16 fosmid-end links - is this overrepresentation? May require additional support eg. PCR</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC108938</TD><TD>AC112229</TD><TD>&nbsp </TD><TD>other</TD><TD>repeatmasker</TD><TD>LINE/L2, SINE/Alu, LTR/ERVL 88635 bp overlap, multiple indels-resulting in AC112229.4 containing 19bp more data than AC108938.3</TD><TD>return</TD><TD>requires BAC, fosmid end support or other</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC106892</TD><TD>AC110798</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>Alu. AC106892.3 contains 264bp msore data than AC110798.4, join supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC107048</TD><TD>AC108164</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>Alu. 78104 bp overlap, AC107048.3 contains 307bp, 304bp, 50bp, and 337bp more than AC108164.5, join supported by fosmid end pair, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC107214</TD><TD>AC108477</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>Alu. multiple indels present, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join supported by fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC104809</TD><TD>AC112784</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>14444 bp overlap, AC104809.5 contains 164bp more data than AC112784.4, join is also supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC104068</TD><TD>AC104795</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>Alu. AC104068.3 contains 305bp more data than AC104795.1, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC095041</TD><TD>AC108205</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>LINE/L1, DNA/MER1_type, LTR/ERVL. AC108205.5 contains 5796bp more data than AC095041.3, multiple indels also present, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC096576</TD><TD>AC107462</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC096576.3 contains 190bp more data than AC107462.5, AC107462.5 contains 1885bp more data than AC096576.3, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join supported by bac and fosmid end pairs</TD><TD>return</TD><TD>Probably OK, but 17 BAC-ends - is this overrepresentation?</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC093164</TD><TD>AC104695</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>SINE/Alu, LINE/L1. indel present, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC097108</TD><TD>AC098870</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>indel present, join supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC097526</TD><TD>AC107208</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC107208.6 contains 108bp more data than AC097526.3, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC093627</TD><TD>AC093666</TD><TD>&nbsp </TD><TD>line </TD><TD>repeatmasker</TD><TD>L1. 7p telomeric, 145213bp overlap, AC093627.4 contains 290bp and 32bp more than AC093666.4, join is also supported by bac and fosmid end pairs</TD><TD>return</TD><TD>possible over-representation of fosmid links (16). Please check and comment</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC093689</TD><TD>AC093771</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC093771.3 contains 198bp more data than AC093689.4, AC093689.4 contains 704bp more data than AC093771.3, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC093724</TD><TD>AC103564</TD><TD>&nbsp </TD><TD>other</TD><TD>	</TD><TD>AC093724.2 will be resubmitted this clone previously had a misassembly</TD><TD>in process</TD><TD>Please return certificate if not resolved; otherwise should resolve with new compute</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC093860</TD><TD>AC108060</TD><TD>AC023350</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>another accession sequenced to confirm join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC092106</TD><TD>AC114755</TD><TD>&nbsp </TD><TD>other</TD><TD>repeatmasker</TD><TD>SINE/Alu, LTR/ERV1, LINE/L1. AC092106.5 contains 6350bp more data than AC114755.4, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>This is at the edge of passing due to multiple variations.</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC092442</TD><TD>AC113615</TD><TD>AC004624</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Minimal overlap finished, but AC004624 confirms overlap. Overlap also supported by BAC and fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC092463</TD><TD>AC106045</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>LTR/MaLR, SINE/Alu. AC106045.5 contains 2432bp more data than AC092463.5, AC092463.5 contains 488bp more data than AC106045.5, join supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>12 fosmid-ends - is this overrepresentation?</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC092619</TD><TD>AC113618</TD><TD>&nbsp </TD><TD>line </TD><TD>polymorphic based on genomic pcr assay</TD><TD>L1. AC113618.5 contains 1331bp more data than AC092619.3, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>11 fosmid-ends - is this overrepresentation?</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC092628</TD><TD>AC099552</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>multiple indels, join is also supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>13 fosmid-end links - is this overrepresentation?</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC092683</TD><TD>AC099646</TD><TD>AC092683</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC092685</TD><TD>AC104073</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC092685.2 contains 4099 bp more data than AC104073.3, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC079255</TD><TD>AF093117</TD><TD>AC079255</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC079402</TD><TD>AC079769</TD><TD>&nbsp </TD><TD>other</TD><TD>in progress</TD><TD>2000 bp overlap, indel present due to simple repeat</TD><TD>return</TD><TD>requires BAC, fosmid end support or other; probably OK if this can be supplied</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC079837</TD><TD>AL133243</TD><TD>&nbsp </TD><TD>other</TD><TD>?</TD><TD>we think this might be extra vector on AL133243.1, this is GS clone, we are clipping it back manually in the agp file.  This join is supported by bac and fosmid end pairs</TD><TD>in process</TD><TD>Please return certificate when resolved; or, if extra vector, will resolve on next compute</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC006334</TD><TD>AC073330</TD><TD>&nbsp </TD><TD>sslp </TD><TD>in progress</TD><TD>AC073330.6 contains 721bp more than AC006334.3. Low complexity, simple repeats. Join is supported by both fosmid and bac end pairs</TD><TD>return</TD><TD>Greg`s site says over 20 kb difference, please investigate</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC006433</TD><TD>AC069288</TD><TD>AC069288</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC006445</TD><TD>AC020729</TD><TD>AC020729</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC006226</TD><TD>AC020729</TD><TD>AC020729</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC006230</TD><TD>AC025539</TD><TD>AC025539</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC004166</TD><TD>AC083884</TD><TD>&nbsp </TD><TD>sine </TD><TD>william`s region - will not be tested</TD><TD>Alu. AC004166.12 contains 179bp more than AC083884.6,join is also supported by bac and fosmid end pairs</TD><TD>disputed</TD><TD>12 BAC- and 27 fosmid-ends -likely to be over-representation or complications due to William`s region.  Since further work is not likely to be enlightening in the short term, this remains disputed.</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC004841</TD><TD>AC093799</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC093799.2 contains 791bp more than AC004841.2, join is also supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC004936</TD><TD>AC010991</TD><TD>AC010991</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC005015</TD><TD>AC005081</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC005015.2 contains 460bp more than AC005081.3, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC005056</TD><TD>AC005057</TD><TD>AC004956</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Minimal overlap was finished, but AC004956 confirms overlap. Overlap also supported by BAC and fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC005061</TD><TD>AC099759</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>assayed base differences in both clones as well as the 1958 indel and found both to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC005072</TD><TD>AC005103</TD><TD>AC005072</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC005086</TD><TD>AC005103</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC005103.3 contains 164 bp more than AC005086.2, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC005237</TD><TD>AC111201</TD><TD>&nbsp </TD><TD>other</TD><TD>in progress</TD><TD>1986 bp overlap, AC111201.5 contains 13bp more data than AC005237.2</TD><TD>return</TD><TD>requires BAC, fosmid end support or other</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC005326</TD><TD>AC079781</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC005326.2 contains 333bp more than AC079781.7, join is also supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC005598</TD><TD>AC108063</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeatmasker</TD><TD>1983 bp overlap, AC108063.3 contains 19bp more data than AC005598.6, join supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC005674</TD><TD>AC108199</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>Alu. 89047 bp overlap, AC108199.5 contains 232bp and 306bp more data than AC005674.11, join supported by fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC005699</TD><TD>AC025539</TD><TD>AC025539</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC003093</TD><TD>AC005084</TD><TD>AC084148</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Minimal overlap was finished, but AC084148 confirms overlap. Overlap also supported by BAC and fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC002385</TD><TD>AC092465</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC002385.1 contains 6028 bp more than AC092465.5 , bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC004027</TD><TD>AC079799</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>bp differences in the overlap were assayed and the overlap was found to be polymorphic, join supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC002088</TD><TD>AC002124</TD><TD>AC007555</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>Minimal overlap was finished, but AC007555 confirms overlap. Overlap also supported by BAC and fosmid end pairs.</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC011893</TD><TD>AC016742</TD><TD>AC012551</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>another accession sequenced to confirm join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC011744</TD><TD>AC105415</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>Alu. AC105415.2 contains 334bp more data than AC011744.8, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join supported by bac and fosmid end pairs</TD><TD>return</TD><TD>Almost certainly OK, but 16 fosmid-ends - is that overrepresentation?</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC011228</TD><TD>AC019210</TD><TD>&nbsp </TD><TD>other</TD><TD>repeatmasker</TD><TD>LTR/ERVL, LINE/L1. an indel is present in each accession at different places, 567bp more in AC011228.9, and 356 bp more in AC019210.7, supported by both fosmid and bac end pairs</TD><TD>return</TD><TD>Multiple with InDels, probably needs additional conmfirmation</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC010743</TD><TD>AL121654</TD><TD>AC010743</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC010743</TD><TD>AL121657</TD><TD>AC010743</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC010913</TD><TD>AC012366</TD><TD>AC073630</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>another accession sequenced to confirm join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC011005</TD><TD>AC083866</TD><TD>&nbsp </TD><TD>other</TD><TD>repeatmasker</TD><TD>SINE/Alu, LINE/L1. Indels present, join is also supported by bac and fosmid end pairs</TD><TD>return</TD><TD>Probably OK, but if indels are extensive requires additional support eg. PCR</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC011005</TD><TD>AC018639</TD><TD>&nbsp </TD><TD>sine </TD><TD>repeatmasker</TD><TD>Alu. AC011005.7 contains 673 bp more than AC018639.8, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC012513</TD><TD>AC069155</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC012513.12 contains 2556bp more data than AC069155.6, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC013399</TD><TD>AC079988</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>SINE/Alu, LTR/MaLR. AC079988.6 contains 298bp more data than AC013399.6, AC013399.6 contains 606bp more data than AC079988.6, join is also supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC013480</TD><TD>AC083949</TD><TD>AC013480</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC013733</TD><TD>AC073977</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>Alu. AC013733.10 contains 305bp more data than AC073977.6, join is also supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC015976</TD><TD>AC114745</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>SINE/Alu, LINE/L1, DNA/MER1_type. 117889 bp overlap, AC015976.8 contains 8425bp more data than AC114745.6, join is also supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC016692</TD><TD>AC064852</TD><TD>&nbsp </TD><TD>sine </TD><TD>repeatmasker</TD><TD>Alu. 2000 bp overlap, AC064852.4 contains 291bp more data than AC016692.9, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC016711</TD><TD>AC023430</TD><TD>AC016711</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC016759</TD><TD>AC107209</TD><TD>&nbsp </TD><TD>sine </TD><TD>polymorphic based on genomic pcr assay</TD><TD>Alu. AC107209.4 contains 316bp more data than AC016759.11, join is also supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC006840</TD><TD>AC114736</TD><TD>&nbsp </TD><TD>line </TD><TD>polymorphic based on genomic pcr assay</TD><TD>L1. AC006840.17 contains 6085bp more data than AC114736.4, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC007073</TD><TD>AC113614</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>LTR/MaLR. AC113614.4 contains 548bp more data than AC007073.2, join supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC007189</TD><TD>AC079394</TD><TD>AC079394</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC008039</TD><TD>AC018662</TD><TD>&nbsp </TD><TD>sine </TD><TD>repeatmasker</TD><TD>Alu. AC008039.1 contains 301bp more than AC018662.3, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC008122</TD><TD>AC019178</TD><TD>AC019178</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC008122</TD><TD>AC013468</TD><TD>AC013468</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC009264</TD><TD>AC009332</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeatmasker</TD><TD>AC009264.6 contains 295bp more than AC009332.6, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC009274</TD><TD>AC016831</TD><TD>&nbsp </TD><TD>line </TD><TD>repeatmasker</TD><TD>L1. 24267bp overlap, AC009274.11 contains 1064bp more than AC016831.1</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC009958</TD><TD>AC110080</TD><TD>&nbsp </TD><TD>line </TD><TD>to be tested</TD><TD>indels in both accessions, will test to confirm overlap</TD><TD>in process</TD><TD>For a simple variation, BAC or fosmid end data likely sufficient to confirm. If compound or not a recent LINE, may be preferable to confirm variation by genomic PCR.</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC010095</TD><TD>AC073415</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>SINE/Alu, LINE/L1. AC010095.6 contains 1556bp more data than AC073415.6, join is also supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC009332</TD><TD>AC108510</TD><TD>&nbsp </TD><TD>sslp </TD><TD>repeatmasker</TD><TD>AC108510.2 contains 339 bp more than AC009332.6, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC009336</TD><TD>AC016739</TD><TD>AC016739</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AL121654</TD><TD>AL133247</TD><TD>&nbsp </TD><TD>line </TD><TD>repeatmasker</TD><TD>L1. 49041 bp overlap, AL133247.1 contains 1505bp more data than AL121654.1, join supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AL121656</TD><TD>AL133244</TD><TD>&nbsp </TD><TD>sine </TD><TD>repeatmasker</TD><TD>28401 bp overlap, indel present</TD><TD>return</TD><TD>requires BAC, fosmid end support or other; if indel, may need PCR support</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AL121796</TD><TD>AL590235</TD><TD>&nbsp </TD><TD>other</TD><TD>repeatmasker</TD><TD>LINE/L1, simple_repeat. 64378 bp overlap, multiple indels present- mostly due to a dinucleotide repeat</TD><TD>return</TD><TD>requires BAC, fosmid end support or other</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC016674</TD><TD>AC092169</TD><TD>&nbsp </TD><TD>line </TD><TD>polymorphic based on genomic pcr assay</TD><TD>L1. AC016674.9 contains 6039bp more data than AC092169.4, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC018678</TD><TD>AC069333</TD><TD>AC026337</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>another accession sequenced to confirm join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC018687</TD><TD>AF250324</TD><TD>AC018687</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC097493</TD><TD>AC118279</TD><TD>AC118279</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC107397</TD><TD>AC121155</TD><TD>&nbsp </TD><TD>none </TD><TD>&nbsp</TD><TD>this was a vector clipping problem, it has been fixed with  AC107397.5</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC113607</TD><TD>AC116614</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>multiple indels, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC093898</TD><TD>AC121160</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>indel present, join supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC019341</TD><TD>AC121161</TD><TD>&nbsp </TD><TD>line </TD><TD>polymorphic based on genomic pcr assay</TD><TD>L1. AC121161.2 contains 175bp more data than AC019341.6, join supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>12 fosmid-ends - is this overrepresentation?</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC104597</TD><TD>U66061</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic</TD><TD>U66061.1 contains 10 kb more than AC104597.3, within the 10 kb is T Cell Receptor Beta Locus, NG_001333.1, join is also supported by bac and fosmid end pairs</TD><TD>return</TD><TD>17 fosmid-ends - is this overrepresentation? May require additional support eg. PCR; may be intractable due to TCR locus; if so, `disputed` or `unresolved`</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC009961</TD><TD>AC093873</TD><TD>AC087594</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>another accession finished to confirm join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC105446</TD><TD>AC118759</TD><TD>&nbsp </TD><TD>other</TD><TD>repeatmasker</TD><TD>1220 bp overlap; SINE/Alu; multiple indels in overlap, high number of substitutions, have fosmids picked in this area to confirm overlap</TD><TD>return</TD><TD>This is really an `other`; there is a SINE present, but also multiple other differences. Likely requires more than just BAC or fosmid support</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC099799</TD><TD>AC116176</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>AC116176.4 contains 189bp more data than AC099799.6, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC113607</TD><TD>AC116609</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>LINE/L1, SINE/Alu, LTR/MaLR. AC116609.6 contains 4152bp more data than AC113607.3, bp differences in the overlap were assayed and the overlap was found to be polymorphic, join is also supported by bac and fosmid end pairs</TD><TD>pass</TD><TD>13 fosmid end links---over-representation of fosmid-ends? Investigate.</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC114808</TD><TD>AC116614</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>2ptelomeric region, indel in overlap, 200 bases more in AC114808.4, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC108866</TD><TD>AC116647</TD><TD>&nbsp </TD><TD>line </TD><TD>polymorphic based on genomic pcr assay</TD><TD>L1. 2205 bp overlap, AC108866.5 contains 193bp more data than AC116647.4, join supported by fosmid end pair, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC112227</TD><TD>AC125245</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>SINE/Alu, LINE/L1. AC112227.4 contains 639bp and 81bp more than AC125245.4, join supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC112713</TD><TD>AC125333</TD><TD>&nbsp </TD><TD>other</TD><TD>in progress</TD><TD>2217 bp overlap, AC112713.3 contains 122bp more data than AC125333.4</TD><TD>return</TD><TD>requires BAC, fosmid end support or other</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC007249</TD><TD>AC007314</TD><TD>AC007314</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>no overlap was finished, but sequence from a previous accession version confirms overlap</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC009411</TD><TD>AC018742</TD><TD>AC084349</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>another accession sequenced to confirm join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC012065</TD><TD>AC012361</TD><TD>AC023124</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>another accession sequenced to confirm join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC012003</TD><TD>AC012353</TD><TD>AC034171</TD><TD>&nbsp </TD><TD>&nbsp </TD><TD>another accession finished to confirm join</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC092594</TD><TD>AC130814</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic, based on pcr assay</TD><TD>AC130814.3 contains two copies of a 19kb region and AC092594.2 only contains one copy , we have assayed this overlap and found it to be polymorphic</TD><TD>pass</TD><TD>in future, helpful to include BAC or fosmid end sequence evidence in these cases.</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC068279</TD><TD>AC133644</TD><TD>&nbsp </TD><TD>other</TD><TD>&nbsp</TD><TD>not able to pcr due to repetative nature of clone</TD><TD>unresolved</TD><TD>Assuming that no further work is possible; there is BAC and fosmid end support.</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC130223</TD><TD>AC134873</TD><TD>&nbsp </TD><TD>other</TD><TD>in progress</TD><TD>We believe that AC130223 (telomeric cosmid from Harold Riethman) is deleted, but it extends out further towards the telomere, so we want to use a portion of the sequence.   It will probably be clipped back.</TD><TD>in process</TD><TD>Please return certificate when resolved.</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC095038</TD><TD>AC118758</TD><TD>&nbsp </TD><TD>sine </TD><TD>repeatmasker</TD><TD>Alu. indel present, 2766bp overlap between AC095038.3 and AC118758.3, AC095038.3 contains 129bp more data than AC118758.3, join is supported by both fosmid and bac end pairs</TD><TD>pass</TD><TD>InDel should probably be confirmed</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC018696</TD><TD>AC116050</TD><TD>&nbsp </TD><TD>other</TD><TD>repeatmasker</TD><TD>SINE/Alu, Satellite, LTR/ERVL, LINE/L1, SINE/MIR, LTR/MaLR, LINE/L2. centromeric, multiple indels contained in overlap, entire overlap is 74kb, many fosmid and BAC links</TD><TD>return</TD><TD>clear overrepresentation of bac- and fosmid-ends. Investigate. requires additional support eg. PCR</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
<TR><TD>AC010984</TD><TD>AC133785</TD><TD>&nbsp </TD><TD>other</TD><TD>polymorphic based on genomic pcr assay</TD><TD>LINE/L2, LTR/MaLR, SINE/MIR, SINE/Alu, LINE/L1. AC133785.2 contains 8243bp more data than AC010984.7, join is also supported by bac and fosmid end pairs, bp differences in the overlap were assayed and the overlap was found to be polymorphic</TD><TD>pass</TD><TD>&nbsp</TD><TD><A HREF="mailto:tgraves@watson.wustl.edu">tgraves@watson.wustl.edu</A></TD></TR>
</TABLE>
            </TD><TD WIDTH=15></TD></TR></TABLE>
        <br></TD></TR></TABLE>
    </TD></TR></TABLE>
</TD></TR></TABLE>
</TD></TR></TABLE>

</BODY></HTML>

