<H2>Description</H2>
<P>
This track displays human-centric multiple sequence alignments in the ENCODE 
regions for the 23 vertebrates in the May 2005 ENCODE MSA freeze, based on 
comparative sequence data generated for the ENCODE project.
The alignments in this track were generated using the
Threaded Blockset Aligner (TBA).
A complete list of the vertebrates included in the May 2005 freeze may be found
at the top of the description page for this track.</P>
<P>
The Genome Browser companion tracks, TBA Cons and TBA Elements, display 
conservation scoring and conserved elements for these alignments based on 
various conservation methods.</P>

<H2>Display Conventions and Configuration</H2>
<P>
In full display mode, this track shows pairwise alignments
of each species aligned to the human genome.  
The alignments are shown in dense display mode using a gray-scale
density gradient.  The checkboxes in the track configuration section allow
the exclusion of species from the pairwise display.
<P>
When zoomed-in to the base-display level, the track shows the base
composition of each alignment. The numbers and symbols on the &quot;human
gap&quot; line indicate the lengths of gaps in the human sequence at those
alignment positions relative to the longest non-human sequence. 
If there is sufficient space in the display, the size of the gap is shown; 
if not, and if the gap size is a multiple of 3, a &quot;*&quot; is displayed, 
otherwise &quot;+&quot; is shown.  
To view detailed information about the
alignments at a specific position, zoom in the display to 30,000 or fewer 
bases, then click on the alignment.  </P>

<H2>Methods</H2>
<P>
The TBA was used to align sequences in the May 2005 ENCODE sequence data 
freeze. Multiple alignments were seeded from a series of combinatorial pairwise 
blastz alignments (not referenced to any one species). The specific 
combinations were determined by the
<A HREF= "ftp://kronos.nhgri.nih.gov/pub/outgoing/elliott/encode_tba/MAY-2005/phylo/tree.nh"
TARGET=_blank>species guide tree</A>. Additionally, a 
<A HREF="ftp://kronos.nhgri.nih.gov/pub/outgoing/elliott/tba/aux/blastz.specs"
TARGET=_blank>blastz.specs file</A>
was used to fine-tune the blastz parameters, based on the evolutionary
distance of the species being compared.
The resulting multiple alignments were projected onto the human reference
sequence. </P>

<H2>Credits</H2>
<P>
The TBA multiple alignments were created by Elliott Margulies of the 
<A HREF="https://www.genome.gov/27535200/" TARGET=_blank>Green Lab</A> at NHGRI. 
</P>
<P>
The programs Blastz and TBA, which were used to generate the alignments, were
provided by Minmei Hou, Scott Schwartz and Webb Miller of the 
<A HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank>Penn State Bioinformatics 
Group</A>.</P>
<P>The phylogenetic tree is based on Murphy et al. (2001) and general
consensus in the vertebrate phylogeny community.
</P>

<H2>References</H2>
<P>
Blanchette, M., Kent, W.J., Reimer, C., Elnitski, L., Smit, A.,
Roskin, K., Baertsch, R., Rosenbloom, K.R., Clawson, H. <em>et al.</em>
<A HREF="https://genome.cshlp.org/content/14/4/708"
TARGET=_blank>Aligning Multiple Genomic Sequences With the Threaded Blockset 
Aligner</A>.
<em>Genome Res</em> <B>14</B>, 708-15 (2004).</P>
<P>
Chiaromonte, F., Yap, V.B., and Miller, W. 
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/11928468/" 
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>.
<em>Pac Symp Biocomput</em> 2002, 115-26 (2002).</P>
<P>
Schwartz, S., Kent, W.J., Smit, A., Zhang, Z., Baertsch, R., Hardison, R., 
Haussler, D. and Miller, W.
<A HREF="https://genome.cshlp.org/content/13/1/103.abstract" 
TARGET=_blank>Human-Mouse Alignments with BLASTZ</A>.
<em>Genome Res</em> <B>13</B>(1):103-7 (2003).</P>
<P>Murphy, W.J., et al.
<A HREF="https://science.sciencemag.org/content/294/5550/2348"
TARGET=_blank>Resolution of the early placental mammal radiation using Bayesian phylogenetics</A>.  <em>Science</em> <B>294</B>(5550), 2348-51 (2001).</P>
