<H2>Description</H2>
<P>
This track shows the locations of transcriptionally active regions 
(TARs)/transcribed fragments (transfrags) for the following, hybridized to
the Affymetrix ENCODE oligonucleotide microarray:
<UL>
<LI>human neutrophil (PMN) total RNA (10 biological samples from different individuals)
<LI>human placental Poly(A)+ RNA (3 biological replicates)
<LI>total RNA from human NB4 cells (4 biological replicates), each sample
divided into three parts and treated as follows:  untreated, treated with 
retinoic acid (RA), and treated with 12-O-tetradecanoylphorbol-13 acetate (TPA)
(three out of the four original samples). Total RNA was extracted from each 
treated sample and applied to arrays in duplicate (2 technical replicates). 
</UL></P>
<P>
The human NB4 cell can be made to differentiate towards either monocytes (by
treatment with TPA) or neutrophils (by treatment with RA). See Kluger
<em>et al</em>., 2004 in the References section for more details about the
differentiation of hematopoietic cells.</P>
<P>
This array has 25-mer oligonucleotide probes tiled 
approximately every 22 bp, covering all the non-repetitive DNA sequence 
of the ENCODE regions. The transcript map is a combined signal for both 
strands of DNA. This is derived from the number of different biological 
samples indicated above, each with at least two technical replicates.</P> 
<P>
See the following NCBI GEO accessions for details of experimental protocols:
<UL>
<LI>
<A HREF="https://www.ncbi.nlm.nih.gov/projects/geo/query/acc.cgi?acc=GSE2678"
TARGET=_blank>GSE2678</A>
<LI>
<A HREF="https://www.ncbi.nlm.nih.gov/projects/geo/query/acc.cgi?acc=GSE2671"
TARGET=_blank>GSE2671</A>
<LI>
<A HREF="https://www.ncbi.nlm.nih.gov/projects/geo/query/acc.cgi?acc=GSE2679"
TARGET=_blank>GSE2679</A>
</UL></P>

<H2>Display Conventions and Configuration</H2>
<P>
TARs are represented by blocks in the graphical display. This composite 
annotation track consists of several subtracks that are listed at the top of
the track description page. To display only selected subtracks, uncheck the 
boxes next to the tracks you wish to hide.</P>
<P>
Color differences among the subtracks are arbitrary. They provide a
visual cue for distinguishing between the different data samples.</P>

<H2>Methods</H2>
<P>
The data from technical replicates were median-scaled and quantile-normalized 
to each other. Using a 101 bp sliding window centered on 
each oligonucleotide probe, a signal map estimating RNA abundance was
generated by computing the pseudomedian signal of all PM-MM pairs 
(median of pairwise PM-MM averages) within the window, including 
replicates. Biological replicate signal maps were combined by 
quantile-normalizing them between replicates and computing the median signal 
at each oligonucleotide probe location. Independent biological 
replicates (as described above) were generated, and each was hybridized
to at least two different arrays (technical replicates). Transcribed regions 
(TARs/transfrags) were then identified using a signal theshold of 90 
percentile of signal intensities, as well as a maximum gap of 50 bp and 
a minimum run of 50 bp (between oligonucleotide positions).</P>

<H2>Verification</H2>
<P>
Transcribed regions (TARs/transfrags), as determined by individual 
biological samples, were compared to ensure significant overlap.</P>

<H2>Credits</H2>
<P>
These data were generated and analyzed by the Yale/Affymetrix 
collaboration between the labs of Michael Snyder, Mark Gerstein and 
Sherman Weissman at Yale University and Tom Gingeras at Affymetrix.</P>

<H2>References</H2>
<P>
Bertone, P., Stolc, V., Royce, T.E., Rozowsky, J.S., Urban, A.E., Zhu, X., 
Rinn, J.L., Tongprasit, W., Samanta, M. <em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/15539566/"
TARGET=_blank>Global identification of human transcribed sequences with 
genome tiling arrays</A>. 
<i>Science</i> <B>306</B>(5705), 2242-6 (2004).</P>
<P>
Cheng, J., Kapranov, P., Drenkow, J., Dike, S., Brubaker, S., Patel, S., 
Long, J., Stern, D., Tammana, H. <em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/15790807/"
TARGET=_blank>Transcriptional maps of 10 human chromosomes at 5-nucleotide 
resolution</A>. 
<i>Science</i> <B>308</B>(5725), 1149-54 (2005).</P>
<P>
Kapranov, P., Cawley, S.E., Drenkow, J., Bekiranov, S., Strausberg, R.L., 
Fodor, S.P. and Gingeras, T.R.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/11988577/"
TARGET=_blank>Large-scale transcriptional activity in chromosomes 21 and 
22</A>. 
<i>Science</i> <B>296</B>(5569), 916-9 (2002).</P>
<P>
Kluger, Y., Tuck, D.P., Chang, J.T., Nakayama, Y., Poddar, R., Kohya, N., 
Lian, Z., Ben Nasr, A., Halaban, H.R. <em>et al</em>.
<A HREF="https://www.pnas.org/content/101/17/6508.full"
TARGET=_blank>Lineage specificity of gene expression patterns</A>. 
<i>Proc Natl Acad Sci U S A</i> <B>101</B>(17), 6508-13 (2004).</P>
<P>
Rinn, J.L., Euskirchen, G., Bertone, P., Martone, R., Luscombe, N.M., 
Hartman, S., Harrison, P.M., Nelson, F.K., Miller, P. <em>et al.</em>
<A HREF="http://genesdev.cshlp.org/content/17/4/529.full"
TARGET=_blank>The transcriptional activity of human Chromosome 22</A>. 
<i>Genes Dev</i> <B>17</B>(4), 529-40 (2003).</P>
