<H2>Description</H2>
<P>
This track shows the fixed alignment differences between the Chimp and Human
genomes in percent difference in 10K base windows.  The differences were
detected by using ssahaSNP (1) to align all chimp reads to the human assembly.
Uniquely placed reads were scanned for high quality single nucleotide
differences; indels were not considered.  For a 10 kb bin calculation,
the number of single nucleotide differences are summed and this is divided
by the number of aligned bases that match at the same quality thresholds
used for variation detection.  Measured in this way, the two genomes differ
on average by 1.2%.
</P><P>
The analysis for the track was prepared by Jim Mullikin. <BR>
Chimp trace data was generated by WIBR and WUGSC though an
NHGRI sequencing grant and obtained from the
<A HREF="https://www.ncbi.nlm.nih.gov/Traces/" TARGET=_blank>
NCBI Trace Archive.</A>
</P><P>
The track display was prepared by Hiram Clawson.
(<A HREF="mailto:&#104;&#105;&#114;a&#109;&#64;&#115;&#111;&#101;.
&#117;&#99;&#115;&#99;.&#101;&#100;u">
&#104;&#105;&#114;a&#109;&#64;&#115;&#111;&#101;.&#117;&#99;&#115;&#99;.
&#101;&#100;u</A>)
</P><P>
Known bugs:  The data scale display to the left of the track
does not function correctly when auto-scaling is selected.
And it only show integers even though fractional levels are requested.
</P><HR><P>
1) Ning Z, Cox AJ, Mullikin JC, SSAHA: A Fast Search Method
	for Large DNA Databases, <I> Genome Res. </I> 2001 Oct;11(10):1725-9.
</P>
