<H2>Description</H2>

<p>This track displays the 2-way regulatory potential (RP) score computed from
alignments of human (hg16, Jul. '03) and mouse (mm4, Oct. '03). A 3-way RP
score computed from alignments of human, mouse and rat is also available on
this browser.</p>

<p>RP scores compare frequencies of short alignment patterns between known
regulatory elements and neutral DNA. Preliminary results from a calibration
study investigating sensitivity and specificity of 2-way RP scores on the
hemoglobin beta gene cluster suggest the use of a threshold just above 0 for
identifying new putative regulatory elements. </p>

<p>The default viewing range for this track is from 0.00 to 0.01
(score values below the 0.00 default lower limit indicate resemblance to 
alignment patterns typical of neutral DNA, while score values above the 0.01 
default upper limit indicate very
marked resemblance to alignment patterns typical of regulatory elements in the
training set). The range of RP scores from 0.00 to 0.01 contains the prediction
threshold suggested by calibration studies, and provides an effective
visualization of the score for most genomic loci. However, the user can specify
different viewing ranges if desired. Note: Absence of a score value at a given
location indicates lack of a 2-way alignment.</p>

<p>This track may be configured in a variety of ways to highlight different
aspects of the displayed information. Click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph 
configuration help</A> link for an explanation of the configuration options.</P>

<H2>Methods</H2>

<p>The comparison employs log-ratios of transitionu probabilities from two
Markov models. Training the score entails selecting appropriate alphabet
(alignment column symbols) and order (length of the patterns = order + 1) for
the Markov models, and estimating their transition probabilities, based on
alignment data from known regulatory elements and ancestral repeats. The 2-way
RP score uses a 5-symbol alphabet and order 5. </p>

<p>In the track, score values are displayed using a system of overlapping
windows of size 100 bp along aligned portions of the human sequence Log-ratios
are added over positions in a window, and the sum normalized for length. </p>

<H2>Credits</H2>

<p>Work on RP scores is performed by members of the <a
href="http://www.ccbb.psu.edu/" target="_blank">Comparative Genomics and
Bioinformatics Center at Penn State University</a>. More information on this
research and the collection of known regulatory elements used in training the
score can be found at this site. </p>

<p>Mouse sequence data were provided by the Mouse Sequencing Consortium. The
alignment data were created in collaboration with the UCSC Genome
Bioinformatics group. </p>

<H2>References</h2>

<p>Elnitski L, Hardison R, Li J, Yang S, Kolbe D, Eswara P, O'Connor M,
Schwartz S, Miller W, Chiaromonte F. 
<A HREF="https://genome.cshlp.org/content/13/1/64.abstract" 
TARGET="_blank">Distinguishing regulatory DNA from neutral sites</A>. 
<em>Genome Res.</em> 2003 Jan;<B>13</B>(1):64-72.</p>

<p>Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison R, Haussler D,
Miller W. 
<A HREF="https://genome.cshlp.org/content/13/1/103.abstract" 
TARGET="_blank">Human-Mouse Alignments with BLASTZ</A>. 
<em>Genome Res.</em> 2003 Jan;<B>13</B>(1):103-7. </p>
