<H2>Description</H2>
<P>
The RefSeq Genes track shows known <EM>$Organism</EM> protein-coding and 
non-protein-coding genes taken from the NCBI RNA reference sequences 
collection (RefSeq), which were directly contributed to NCBI by
<a href="http://flybase.org/" target="_blank">FlyBase</a>.
The data underlying this track are updated weekly.</P>
<!--Unlike most other RefGene tracks, all of the sequences for this (dm6) 
RefGene track were deposited into GenBank by FlyBase-->

<p>
Please visit the <a href="https://www.ncbi.nlm.nih.gov/projects/RefSeq/update.cgi"
target="_blank">Feedback for Gene and Reference Sequences (RefSeq)</a> page to
make suggestions, submit additions and corrections, or ask for help concerning
RefSeq records.
</p>

<H2>Display Conventions and Configuration</H2>
<P>
This track follows the display conventions for 
<A HREF="../goldenPath/help/hgTracksHelp.html#GeneDisplay">gene prediction 
tracks</A>.
The color shading indicates the level of review the RefSeq record has 
undergone: predicted (light), provisional (medium), reviewed (dark).</P>
<P>
The item labels and display colors of features within this track can be
configured through the controls at the top of the track description page. 
This page is accessed via the small button to the left of the track's 
graphical display or through the link on the track's control menu. 
<UL>
<LI><B>Label:</B> By default, items are labeled by gene name. Click the 
appropriate Label option to display the accession name instead of the gene
name, show both the gene and accession names, or turn off the label 
completely.
<LI><B>Codon coloring:</B> This track contains an optional codon coloring 
feature that allows users to quickly validate and compare gene predictions.
To display codon colors, select the <em>genomic codons</em> option from the
<em>Color track by codons</em> pull-down menu. Click 
<A HREF="../goldenPath/help/hgCodonColoring.html">here</A> for more 
information about this feature.
<LI><B>Hide non-coding genes:</B> By default, both the protein-coding and
non-protein-coding genes are displayed.  If you wish to see only the coding
genes, click this box.
</UL></P>

<H2>Methods</H2>
<P>
RefSeq RNAs were aligned against the <em>$Organism</em> genome using blat; 
those with an alignment of less than 15% were discarded. When a single RNA 
aligned in multiple places, the alignment having the highest base identity 
was identified.  Only alignments having a base identity level within 0.1% of 
the best and at least 96% base identity with the genomic sequence were kept.
</P>


<H2>Credits</H2>
<P>
This track was produced at UCSC from RNA sequence data
generated by scientists worldwide and curated by the 
NCBI <A HREF="https://www.ncbi.nlm.nih.gov/refseq/" 
TARGET=_blank>RefSeq project</A>.  </P>

<H2>References</H2>
<p>
Kent WJ.
<a href="https://genome.cshlp.org/content/12/4/656.full" target="_blank">
BLAT--the BLAST-like alignment tool</a>.
<em>Genome Res</em>. 2002 Apr;12(4):656-64.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11932250" target="_blank">11932250</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC187518/" target="_blank">PMC187518</a>
</p>

<p>
Pruitt KD, Tatusova T, Maglott DR.
<a href="https://academic.oup.com/nar/article/33/suppl_1/D501/2505241/NCBI-Reference-Sequence-
RefSeq-a-curated-non" target="_blank">NCBI Reference Sequence (RefSeq): a curated non-redundant
sequence database of genomes, transcripts and proteins</a>.
<em>Nucleic Acids Res</em>. 2005 Jan 1;33(Database issue):D501-4.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15608248" target="_blank">15608248</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC539979/" target="_blank">PMC539979</a>
</p>
