<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=273>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://genome.wustl.edu/genomes/detail/callithrix-jacchus/"
    TARGET=_blank>
    <IMG SRC="../images/Callithrix_jacchus.jpg" WIDTH=258 HEIGHT=200 ALT="Marmoset"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Callithrix jacchus</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of NHGRI
      (<A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Marmoset%20(Callithrix%20jacchus)&id=79119" 
      TARGET=_blank>press photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The Jun. 2007 <em>Callithrix jacchus</em> draft assembly 
(WUSTL version Callithrix jacchus-2.0.2) was produced by the Washington 
University St. Louis (WUSTL) School of Medicine 
<A HREF="http://genome.wustl.edu/genomes/detail/callithrix-jacchus/"
TARGET=_blank>Genome Sequencing Center</A> in St. Louis, MO, USA
and the Baylor College of Medicine (BCM) 
<a href="https://www.hgsc.bcm.edu/content/marmoset-genome-project"
target=_blank>Human Genome Sequencing Center</a>
in Houston, TX, USA.</P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Marmoset genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
</P>
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>Contig12371</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of contig 12371</TD>
    </TR>
    <TR><TD VALIGN=Top>Contig3759:130,210-140,209</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays ten thousand bases of contig 3759</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Contig2815:171328+1500</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of Contig 2815 that spans 1500 bases, starting with position 171328 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>EF215092</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number EF215092</TD>
    </TR>
    <TR><TD VALIGN=Top>EH380513</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		EH380513</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>datson</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Datson</TD>
    </TR>
    <TR><TD VALIGN=Top>Fuchs,E.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author E. Fuchs</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Fuchs E</em>, internally it
	uses the format <em>Fuchs,E.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
The <em>C. jacchus</em> genome was sequenced to 6X coverage 
using DNA from a female marmoset provided by the Southwestern
National Primate Research Center in San Antonio, TX, USA.
DNA from a full brother of the female was used as the source
for the CHORI-259 BAC library. The combined sequence reads were 
assembled using <A HREF="http://genome.cshlp.org/content/13/9/2164.full"
TARGET=_blank>PCAP</A> and filtered for all known non-marmoset sequence 
contaminants and unscaffolded singleton contigs of less than 2 kb in length. 
This assembly differs from the previous 2.0.1 assembly (which is not featured 
in the UCSC Genome Browser) in that 586 reads containing untrimmed vector 
sequence were removed from contig ends, resulting in 34 contigs less than 2 kb 
in length.</P> 
<P>
The 2.0.2 assembly is composed of 49,724 supercontigs containing a total of 
approximately 3.02 billion bases. For more statistics and details
on the assembly process, refer to the WUSTL Callithrix_jacchus-2.0.2
<A HREF="http://genome.wustl.edu/genomes/detail/callithrix-jacchus/"
TARGET=_blank>assembly</A> page. </P>
<P>
WUSTL includes the following release notes with this 
assembly: &quot;This preliminary assembly is being made 
available to the community while the assembly continues to 
be refined.  This is a draft assembly with limited quality 
assessment and thus the data is provided &quot;as is&quot;. Once the 
assembly has been quality screened, scaffolded and 
localized by chromosome, they will be submitted to GenBank 
and the various genome browsers.&quot;</P>
<P>
Bulk downloads of the calJac1 sequence and annotations may 
be obtained from the Genome Browser 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/calJac1/">FTP server</A> or 
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#marmoset">Downloads</A> 
page. See the
<A HREF="http://genome.ucsc.edu/goldenPath/credits.html#marmoset_credits"
>data use policy</A> for conditions of use.
Please acknowledge the WUSTL School of Medicine Genome 
Sequencing Center and the BCM Human Genome Sequencing Center
in any publications that result from the use of this sequence 
assembly.</P>
<P>
We'd like to thank WUSTL and BCM for providing this assembly. The 
initial set of marmoset browser annotation tracks were 
generated by UCSC.  See the 
<A HREF="../goldenPath/credits.html#marmoset_credits">Credits</A>
page for a detailed list of the organizations and
individuals who contributed to this release.</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
