<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=185>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://www.genome.gov/dmd/index.cfm?node=Photos/Animals/Cat%20domestic" TARGET=_blank>
    <IMG SRC="../images/Felis_catus.jpg" WIDTH=170 HEIGHT=143 ALT="F. catus"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Felis catus</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
    (NHGRI <A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Cat%20domestic&id=79105" 
	        TARGET=_blank>Press Photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The March 2006 <em>Felis catus</em> draft assembly 
(Broad Release 3) was produced by the <A HREF="http://www.broadinstitute.org/"
TARGET="_blank">Broad Institute</A> of MIT/Harvard and Agencourt Bioscience 
(now a part of <a href="http://www.beckmangenomics.com" target="_blank">Beckman Coulter Genomics</a>). 

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
<EM>F. catus</EM> genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>scaffold_217569</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of scaffold 217569</TD>
    </TR>
    <TR><TD VALIGN=Top>scaffold_217569:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of scaffold 217569</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>scaffold_217569:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of scaffold 217569 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>AB030650</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number AB030650</TD>
    </TR>
    <TR><TD VALIGN=Top>DN828316</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		DN828316</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>allergen</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists allergen-related genes and mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>pseudogene mRNA</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>benson</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Benson</TD>
    </TR>
    <TR><TD VALIGN=Top>Ohno,K.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author K. Ohno</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Ohno K</em>, internally it
	uses the format <em>Ohno,K.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
The genome has been sequenced to 2X coverage and consists of 217,790
scaffolds. The total contig length for this assembly is approximately 1.6 Gb
spanning nearly 4.0 Gb (with 60.1&#37; in gaps). 
There are 749,376 contigs, with an N50 length of 2,506 bases.
There are 149,283 supercontigs, with an N50 length of 49,769 bases (not
including gaps). The N50 size is the length such that 50&#37; of the assembled
genome lies in blocks of the N50 size or longer.
</P>
<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/felCat3/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#cat">Downloads</A> page. 
These data have
<A HREF="../goldenPath/credits.html#cat_use">specific 
conditions for use</A>. 
<P>
The <EM>F. catus</EM> browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#cat_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
<!--
<P>
To access annotations of the <em>Felis Catus</em> genome made by the 
<A HREF ="http://home.ncifcrf.gov/ccr/lgd/index.asp"
TARGET=_blank>Laboratory of Genome Diversity</A> (LGC), see the
<A HREF="http://lgd.abcc.ncifcrf.gov/cgi-bin/gbrowse/cat"
TARGET=_blank>GARField</A> browser. This browser may include a slightly newer
version of the data than is available on the UCSC Genome Browser site.</P>
-->
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
