<H2>Description</H2>
<P>
This track shows known protein-coding and non-protein-coding genes 
for organisms other than $organism, taken from the NCBI RNA reference
sequences collection (RefSeq). The data underlying this track are 
updated weekly.</P>

<H2>Display Conventions and Configuration</H2>
<P>
This track follows the display conventions for 
<A HREF="../goldenPath/help/hgTracksHelp.html#GeneDisplay">gene prediction 
tracks</A>.
The color shading indicates the level of review the RefSeq record has 
undergone: predicted (light), provisional (medium), reviewed (dark).</P>
<P>
The item labels and display colors of features within this track can be
configured through the controls at the top of the track description page. 
<UL>
<LI><B>Label:</B> By default, items are labeled by gene name. Click the 
appropriate Label option to display the accession name instead of the gene
name, show both the gene and accession names, or turn off the label 
completely.
<LI><B>Codon coloring:</B> This track contains an optional codon coloring 
feature that allows users to quickly validate and compare gene predictions.
To display codon colors, select the <em>genomic codons</em> option from the
<em>Color track by codons</em> pull-down menu. Click 
<A HREF="../goldenPath/help/hgCodonColoring.html">here</A> for more 
information about this feature.
<LI><B>Hide non-coding genes:</B> By default, both the protein-coding and
non-protein-coding genes are displayed.  If you wish to see only the coding
genes, click this box.
</UL>
Note: There is a small known issue with the display of codon coloring in this
track. Of nearly 275,000 alignments, the underlying table is missing the
CDS-region exon frame for approximately 250 alignments that occur at the ends
of felCat3 scaffolds. However, the Browser assumes an exon frame of zero when
displaying these CDS regions. Please
<A HREF="mailto:&#103;&#101;&#110;o&#109;e&#64;&#115;&#111;&#101;.
&#117;&#99;s&#99;.e&#100;u">contact us</A>
<!-- above address is genome at soe.ucsc.edu -->
with any questions.
</P>

<H2>Methods</H2>
<P>
The RNAs were aligned against the $organism genome using blat; those
with an alignment of less than 15% were discarded. When a single RNA aligned 
in multiple places, the alignment having the highest base identity was 
identified.  Only alignments having a base identity level within 0.5% of 
the best and at least 25% base identity with the genomic sequence were kept.
</P>

<H2>Credits</H2>
<P>
This track was produced at UCSC from RNA sequence data
generated by scientists worldwide and curated by the 
NCBI <A HREF="https://www.ncbi.nlm.nih.gov/refseq/" 
TARGET=_blank>RefSeq project</A>.  </P>

<H2>References</H2>
<P>
Kent WJ.
<A HREF="https://genome.cshlp.org/content/12/4/656.abstract"
TARGET=_blank>BLAT - the BLAST-like alignment tool</A>.
<em>Genome Res.</em> 2002 Apr;12(4):656-64.</P>
