<H2>Description</H2>
<P>
This track shows gene predictions using the N-SCAN gene structure prediction
software provided by the Computational Genomics Lab at Washington University 
in St. Louis, MO, USA.
</P>

<H2>Methods</H2>
<P>
N-SCAN combines biological-signal modeling in the target genome sequence along
with information from a multiple-genome alignment to generate de novo gene
predictions. It extends the TWINSCAN target-informant genome pair to allow for
an arbitrary number of informant sequences as well as richer models of
sequence evolution. N-SCAN models the phylogenetic relationships between the
aligned genome sequences, context-dependent substitution rates, insertions,
and deletions.
</P>
<P>
The PASA clusters were used as 'EST' sequences in N-SCAN PASA-EST. In
addition, the xenoRefSeq track was downloaded and split in unique exon
pairs. All exon pairs that had valid splice sites were then added to the EST
track. The resulting gene models were updated with the input PASA clusters
using the assembly tool of the PASA pipeline. These updates consist of
automatically generated alternative splices, UTR features and occasional  
merging of two gene models. In addition, PASA assigned open reading frames
to clusters that did not overlap a gene prediction, but that did contain a
full length cDNA, and output them as 'novel genes'. Note that PASA does not
use any cDNA annotation from input but assigns the ORF itself.
</P>
<P>
No manual annotation was performed to generate any of the gene models.
Important note: It is possible that real genes were merged by N-SCAN.
Therefore, when looking at this track it is advisable to also open the
'Other Refseq' track. Merged genes will often overlap two separate RefSeqs.
</P>

<P>${informant}</P>

<H2>Credits</H2>
<P>
Thanks to <A HREF="https://mblab.wustl.edu/" 
TARGET=_blank>Michael Brent's Computational Genomics Group</A> at Washington 
University St. Louis for providing this data.
</P>
<p>
Special thanks for this implementation of N-SCAN to Aaron Tenney in
the Brent lab, and Robert Zimmermann, currently at Max F. Perutz
Laboratories in Vienna, Austria.
</p>

<H2>References</H2>
<p>
Gross SS, Brent MR.
<a href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2006.13.379" target="_blank">
Using multiple alignments to improve gene prediction</a>.
<em>J Comput Biol</em>. 2006 Mar;13(2):379-93.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16597247" target="_blank">16597247</a>
</p>

<p>
Korf I, Flicek P, Duan D, Brent MR.
<a href="https://academic.oup.com/bioinformatics/article/17/suppl_1/S140/261903"
target="_blank">Integrating genomic homology into gene structure prediction</a>.
<em>Bioinformatics</em>. 2001;17 Suppl 1:S140-8.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11473003" target="_blank">11473003</a>
</p>

<p>
van Baren MJ, Brent MR.
<a href="https://genome.cshlp.org/content/16/5/678.abstract" target="_blank">
Iterative gene prediction and pseudogene removal improves genome annotation</a>.
<em>Genome Res</em>. 2006 May;16(5):678-85.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16651666" target="_blank">16651666</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1457044/" target="_blank">PMC1457044</a>
</p>
