<H2>Description</H2>
This track depicts gaps in the assembly.

<P>
Gaps are represented as black boxes in this track.
If the relative order and orientation of the contigs on either side
of the gap is supported by read pair data, 
it is a <em>bridged</em> gap and a white line is drawn 
through the black box representing the gap. 
</P>
<P>This assembly contains the following principal types of gaps:
<UL>

<LI><B>clone</B> - gaps on chromosomes to represent telomere sequence at the
beginning of chromosomes</LI>
<LI><B>fragment</B> - gaps between the Whole Genome Shotgun contigs of a 
scaffold or ultracontig. These are represented by varying numbers
of <em>N</em>s in the assembly. In this context, a contig is a set of
overlapping sequence reads and a scaffold is a set of contigs ordered
and oriented during the Whole Genome Shotgun process using paired-end reads.
Contig gap sizes are usually taken from read pair data.  
</LI>


</UL></P>

<P>
See also: <A HREF="http://utgenome.org/medaka/"
	TARGET=_blank>UTGB Medaka Genome</A> at the Morishita Laboratory.
</P>

