<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=223 HEIGHT=185>
  <TR><TD ALIGN=RIGHT><A HREF="http://www.sanger.ac.uk/resources/mouse/"
    TARGET=_blank>
    <IMG SRC="../images/Mus_musculus.jpg" WIDTH=208 HEIGHT=160 ALT="Mouse"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1> <em>Mus musculus</em><BR>
    </FONT>
    <FONT SIZE=-2> (Photo courtesy of
      <A HREF="http://www.jax.org/" TARGET=_blank>The Jackson Laboratory</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The August 2005 mouse (<em>Mus musculus</em>) draft genome data was 
obtained from the Build 35 assembly by NCBI. See the Assembly Details section
below for information about the differences between Build 35 and Build 35.1.
For more information about this assembly, see
<a href="https://www.ncbi.nlm.nih.gov/assembly/5728/" target="_blank">
MGSCv35</a> in the NCBI Assembly database.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a 
sequenced genomic clone, an mRNA or EST or STS marker, or 
a cytological band, a chromosomal coordinate range, or keywords 
from the GenBank description of an mRNA. The following list 
provides examples of various types of position queries for the 
mouse genome. See the 
<A HREF="../goldenPath/help/hgTracksHelp.html" 
TARGET=_blank>User's Guide</A> for more information. 
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TD VALIGN=Top NOWRAP><B>Request:</B><BR></TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top><B>Genome Browser Response:</B><BR></TD>
    </TR>
	
    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>chr16</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays all of chromosome 16</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>chr16:1-5000000</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays first 5 million bases of chr 16</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>chr16:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 16 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>AC072048.4</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region of contig AC072048.4</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>D16Mit120</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region around STS marker DMit16120 from the MGI consensus genetic map, including 100,000 bases on each side as well</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>D16Mit120;D16Mit149</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region between STS markers D16Mit120 and D16Mit149</TD>
    </TR>

    <TR><TD VALIGN=Top><BR></TD></TR>

    <TR><TD VALIGN=Top NOWRAP>BC012683</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays alignment location of mRNA with GenBank accession BC012683</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>AW045217</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays alignment location of EST with GenBank accession AW045217</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>Ncam2</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region of genome with official MGI mouse genetic nomenclature Ncam2</TD>
    </TR>

    <TR><TD VALIGN=Top><BR></TD></TR>

    <TR><TD VALIGN=Top NOWRAP>pseudogene mRNA</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists transcribed pseudogenes but not cDNAs</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>zinc finger</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>kruppel zinc finger</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>huntington</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists candidate genes associated with Huntington's disease</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>Smith,D.</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by co-author D. Smith</TD>
    </TR>

    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although GenBank 
			requires the search format <em>Smith D</em>, internally it uses 
			the format <em>Smith,D.</em>.</TD>
    </TR>

</TABLE>
<BR>
<HR>
<H3>Assembly Details</H3>
<P>
NCBI Build 35 includes approximately 2.6 Gb of sequence, of which about
2.2 Gb is finished sequence, contained in 
chromosomes 1-19, X, Y, M (mitochondrial DNA) and Un (unmapped clone contigs).
Chromosomes 2, 4, 11 and X are finished in this assembly. 
More information about this version is available on the NCBI 
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/GCF_000001635.14/"
TARGET=_blank>Build 35 Data</A> web page. </P>
<P>
Build 35 is a fifth-generation composite assembly. In this build, chromosomes 
1,3,5,6,7,8,9,10, and 12-19 were automatically assembled. The assembly is made
up of high throughput genome sequence (HTGS) phase 3 contigs, single fragment 
HTGS phase 2 contigs and whole genome shotgun (WGS) contigs. On chromosome Y
in this assembly, only the short arm has reliable mapping data; therefore, 
most of the contigs on the Y chromosome are unlocalized. For more information 
about the assembly process, see the 
<a href="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/mouse/"
target="_blank">NCBI website</a>.</P>
<P>
<B>Notes on the UCSC mm7 version:</B> <BR><BR>
- The UCSC mm7 database contains only the reference strain C57BL/6J.<BR><BR> 
- The Build 35 assembly has a known problem: five contigs were erroneously
placed at the end of chromosome 5 (Mm5_160001_35, Mm5_160002_35, Mm5_160003_35, 
Mm5_160004_35, and Mm5_160005_35). Because the UCSC mm7 data are derived
from Build 35, this problem is also present in the UCSC browser, which 
incorrectly extends the length of chr5 by approximately 3,000,000 bp. NCBI has 
subsequently released Build 35.1, which corrects this problem. Due to the effort
involved, UCSC does not plan to upgrade the mm7 browser to the Build 35.1 
assembly.</P>
<P>
The mm7 sequence and annotation data can be downloaded from the Genome 
Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/mm7/">FTP server</A> or
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#mouse">Downloads</A> 
web page.
The mm7 annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="../goldenPath/credits.html#mouse_credits">credits</A> page
for a detailed list of the organizations and individuals who contributed 
to this release. </P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
