<H2>Description</H2>
<P>Bacterial artificial chromosomes (BACs) are a key part of many large
scale sequencing projects.  A BAC typically consists of 50-300 kb of
DNA. During the early phase of a sequencing project, it is common
to sequence a single read (approximately 500 bases) off each end of
a large number of BACs. Later in the project, these BAC end reads
can be mapped to the genome sequence.  
</P>
<P>This track shows these mappings
in cases where both ends could be mapped.  BAC end pairs can
be useful for validating the assembly over relatively long ranges.  In some
cases, BACs are useful biological reagents.  This track can also be
used for determining which BAC contains a given gene, useful information
for certain wet lab experiments.
</P>
<P>For this track, a valid pair of BAC end sequences must be
at least 15 kb but no more than 600 kb away from each other. 
The orientation of the first BAC end sequence must be &quot+&quot; and
the orientation of the second BAC end sequence must be &quot;-&quot;.</P>
</P>
<P>
The scoring scheme used for this annotation assigns 1000 to an alignment 
when the BAC end pair aligns to only one location in the genome (after 
filtering).  When a BAC end pair or clone aligns to multiple locations, the 
score is calculated as 1500/(number of alignments).</P>

<H2>Methods</H2>
<P>
The BAC end sequences in this track were obtained from Chimpanzee
BAC clone libraries RPCI-43 and PTB1.
The BAC end sequences were produced at
<A HREF="http://www.gsc.riken.go.jp/" TARGET=_blank>RIKEN Genomic 
Sciences Center</A>, in association with the Korea Research Institute of
Bioscience and Biotechnology 
(<A HREF="http://www.kribb.re.kr/eng/"
TARGET=_blank>KRIBB</A>), using the methods 
described in 
<A HREF="https://science.sciencemag.org/content/suppl/2002/01/03/295.5552.131.DC1"
TARGET=_blank>Construction and Analysis of a Human-Chimpanzee Comparative Clone 
Map</A>.
</P>
<P>The BAC end sequences were aligned to the Chimpanzee genome assembly using
the <A HREF="https://genome.cshlp.org/content/12/4/656.abstract" TARGET=_blank>blat</A> 
program developed by Jim Kent (UCSC).  The pairing of end sequences from
alignments was performed using tools developed by Terry Furey (UCSC).</P>

<H2>Credits</H2>
The BAC end sequences and pairing information were obtained from
NCBI, as processed by their BAC End pipeline, managed by Deanna Church.
Thanks to Todd Taylor at RIKEN for assistance with producing this track.
<P> 
Information about the clone, including how it
can be obtained, may be found at the 
<A HREF="https://ncbiinsights.ncbi.nlm.nih.gov/2019/05/01/clone-db-retirement/" TARGET=_blank>NCBI Clone Registry</A>.
To view the registry entry for a specific clone, open the details page for the clone and click on its name at the top of the page.
A large selection of clone libraries is distributed by the
<A HREF="https://bacpacresources.org/" TARGET=_blank> BACPAC Resources Center</A>.
</P>
