<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=315>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://genome.wustl.edu/genomes/detail/taeniopygia-guttata/"
    TARGET=_blank>
    <IMG SRC="../images/Taeniopygia_guttata.jpg" WIDTH=300 HEIGHT=300 ALT="Zebra finch"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Taeniopygia guttata</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of Dr. Art Arnold
	(<A HREF="http://www.arnoldlab.org" 
      TARGET=_blank>UCLA Department of Physiological Science</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The Jul. 2008 <em>Taeniopygia guttata</em> draft assembly 
was produced by the Genome Sequencing Center at the Washington 
University in St. Louis (WUSTL) School of Medicine. UCSC's version taeGut1
corresponds to the WUSTL v3.2.4 assembly.</P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Zebra finch genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>chrZ</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of chromosome Z</TD>
    </TR>
    <TR><TD VALIGN=Top>chr3:1-500,000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first 500,000 bases of chr 3</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr3:228,000+30000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 3 that spans 30,000 bases, starting with position 228,000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>DQ216330</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number DQ216330</TD>
    </TR>
    <TR><TD VALIGN=Top>CK308913</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		CK308913 on chr Z</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene mRNA</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>jarvis</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Jarvis</TD>
    </TR>
    <TR><TD VALIGN=Top>Wada,K.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author K. Wada</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Wada K</em>, internally it
	uses the format <em>Wada,K.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
The zebra finch DNA used for the shotgun sequencing and the BAC and cosmid
libraries was derived from a single male domesticated zebra
finch. The initial assembly was generated using PCAP with approximately 6X 
coverage. About 1.0 Gb of the 1.2-Gb genome has been ordered and oriented 
along 33 chromosomes and one linkage group. The chromosome names are based on 
their homologous chromosomes in the chicken (<em>Gallus gallus</em>). For more 
assembly details, as well as a cross reference table of zebra finch and chicken 
chromosome names, see the WUSTL 
<A HREF="http://genome.wustl.edu/genomes/detail/taeniopygia-guttata/"
TARGET=_blank>assembly page</A>.</P>
<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/taeGut1/">FTP
server</A> or <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#zebra_finch">Downloads</A> page. These data have 
<A HREF="../goldenPath/credits.html#zebra_finch_use">specific conditions for 
use</A>.
<P>
The zebra finch browser annotations were generated by  UCSC and collaborators
worldwide. See the 
<A HREF="../goldenPath/credits.html#zebra_finch_credits">Credits</A> page for a 
detailed list of the organizations and individuals who contributed to this 
release. </P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
