<H2>Description</H2>
This track depicts gaps in the draft assembly
(Oct. 2011, Fugu Genome Sequencing Consortium - FUGU5
(NCBI project 1434, GCA_000180615.2))
of the $organism genome.

<P>
Gaps are represented as black boxes in this track.
If the relative order and orientation of the scaffolds on either side
of the gap is supported by read pair data, 
it is a <em>bridged</em> gap and a white line is drawn 
through the black box representing the gap. 
</P>
<P>This assembly contains the following principal types of gaps:
<UL>

<LI><B>fragment</B> - gaps between the Whole Genome Shotgun contigs of a 
scaffold.  (In this context, a contig is a set of overlapping sequence reads.  
A scaffold is a set of contigs ordered and oriented during the 
Whole Genome Shotgun process using paired-end reads.)
These are represented by varying numbers of <em>N</em>s  in the assembly.  
Fragment gap sizes are usually taken from read pair data.</LI>
<LI><B>contig</B> - gaps between scaffolds not linked by the fingerprint 
map, but instead by marker data.  (In this context, the &quot;Contig&quot; gap 
type refers to a map contig, not a sequence contig.)</LI>
These are represented by 60 <em>N</em>s  in the assembly.
<LI><B>other</B> - gaps that UCSC has marked where <em>N</em>s are in the
assembly but were unmarked in the AGP file from the assembly definition.</LI>

</UL></P>
