<!DOCTYPE html>
<!--#set var="TITLE" value="UCSC Genome Browser: Acknowledgments" -->
<!--#set var="ROOT" value=".." -->

<!-- The following setting enables collapsible lists -->
<!--#set var="DYNAMIC" value="TRUE" -->

<!-- Relative paths to support mirror sites with non-standard GB docs install -->
<!--#include virtual="$ROOT/inc/gbPageStart.html" -->

<!-- Table of Contents -->
<h1>UCSC Genome Browser Acknowledgments</h1>
<p> 
The data and software displayed on this site are the result of a large collaborative effort among 
many individuals at UCSC and at research institutions around the world. We'd like to acknowledge the
scientists and engineers who have contributed to the UCSC Genome Browser and its data, and the 
organizations who have made this project possible through their generous funding. <p>

<p> 
<a href="#human_credits">Human</a></p>

<p>
<div class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mammals>Mammals</a>
  <div class="collapse" id="mammals">
  <ul>
    <li><a href="#alpaca_credits">Alpaca</a></li>
    <li><a href="#armadillo_credits">Armadillo</a></li>
    <li><a href="#baboon_credits">Baboon</a></li>
    <li><a href="#bison_credits">Bison</a></li>
    <li><a href="#bonobo_credits">Bonobo</a></a></li>
    <li><a href="#brown_kiwi_credits">Brown kiwi</a></li>
    <li><a href="#bushbaby_credits">Bushbaby</a></li>
    <li><a href="#cat_credits">Cat</a></li>
    <li><a href="#chimp_credits">Chimpanzee</a></li>
    <li><a href="#chinese_hamster_credits">Chinese hamster</a></li>
    <li><a href="#chinese_pangolin_credits">Chinese pangolin</a></li>
    <li><a href="#cow_credits">Cow</a></li>
    <li><a href="#dog_credits">Dog</a></li>
    <li><a href="#dolphin_credits">Dolphin</a></li>
    <li><a href="#elephant_credits">Elephant</a></li>
    <li><a href="#ferret_credits">Ferret</a></li>
    <li><a href="#gibbon_credits">Gibbon</a></li>
    <li><a href="#golden_snub-nosed_monkey_credits">Golden snub-nosed monkey</a></li>
    <li><a href="#gorilla_credits">Gorilla</a></li>
    <li><a href="#green_monkey_credits">Green monkey</a></li>
    <li><a href="#guinea_pig_credits">Guinea pig</a></li>
    <li><a href="#hawaiian_monk_seal_credits">Hawaiian monk seal</a></li>
    <li><a href="#hedgehog_credits">Hedgehog</a></li>
    <li><a href="#horse_credits">Horse</a></li>
    <li><a href="#kangaroo_rat_credits">Kangaroo rat</a></li>
    <li><a href="#microbat_credits">Little brown bat</a></li>
    <li><a href="#malayan_flying_lemur_credits">Malayan flying lemur</a></li> 
    <li><a href="#manatee_credits">Manatee</a></li>
    <li><a href="#marmoset_credits">Marmoset</a></li>
    <li><a href="#megabat_credits">Megabat</a></li>
    <li><a href="#minke_whale_credits">Minke whale</a></li> 
    <li><a href="#mouse_credits">Mouse</a></li> 
    <li><a href="#mouse_lemur_credits">Mouse lemur</a></li> 
    <li><a href="#naked_mole-rat_credits">Naked mole-rat</a></li> 
    <li><a href="#opossum_credits">Opossum</a></li> 
    <li><a href="#orangutan_credits">Orangutan</a></li> 
    <li><a href="#panda_credits">Panda</a></li> 
    <li><a href="#pig_credits">Pig</a></li> 
    <li><a href="#pika_credits">Pika</a></li> 
    <li><a href="#platypus_credits">Platypus</a></li> 
    <li><a href="#proboscis_monkey_credits">Proboscis monkey</a></li> 
    <li><a href="#rabbit_credits">Rabbit</a></li> 
    <li><a href="#rat_credits">Rat</a></li> 
    <li><a href="#rhesus_credits">Rhesus</a></li> 
    <li><a href="#rock_hyrax_credits">Rock hyrax</a></li> 
    <li><a href="#sheep_credits">Sheep</a></li> 
    <li><a href="#shrew_credits">Shrew</a></li> 
    <li><a href="#sloth_credits">Sloth</a></li> 
    <li><a href="#squirrel_credits">Squirrel</a></li> 
    <li><a href="#squirrel_monkey_credits">Squirrel monkey</a></li> 
    <li><a href="#tarsier_credits">Tarsier</a></li> 
    <li><a href="#tasmanian_devil_credits">Tasmanian devil</a></li> 
    <li><a href="#tenrec_credits">Tenrec</a></li> 
    <li><a href="#tree_shrew_credits">Tree shrew</a></li> 
    <li><a href="#wallaby_credits">Wallaby</a></li> 
    <li><a href="#white_rhinoceros_credits">White rhinoceros</a></li> 
  </ul>
  </div>
</div>

<p>
<div class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href=#vert>Other vertebrates</a>
  <div class="collapse" id="vert">
  <ul> 
    <li><a href="#african_clawed_frog_credits">African clawed frog</a></li>
    <li><a href="#american_alligator_credits">American alligator</a></li> 
    <li><a href="#atlantic_cod_credits">Atlantic cod</a></li> 
    <li><a href="#budgerigar_credits">Budgerigar</a></li> 
    <li><a href="#chicken_credits">Chicken</a></li> 
    <li><a href="#coelacanth_credits">Coelacanth</a></li> 
    <li><a href="#elephant_shark_credits">Elephant shark</a></li> 
    <li><a href="#fugu_credits">Fugu</a></li> 
    <li><a href="#garter_snake_credits">Garter snake</a></li>
    <li><a href="#golden_eagle_credits">Golden eagle</a></li>
    <li><a href="#lamprey_credits">Lamprey</a></li> 
    <li><a href="#lizard_credits">Lizard</a></li> 
    <li><a href="#medaka_credits">Medaka</a></li> 
    <li><a href="#medium_ground_finch_credits">Medium ground finch</a></li> 
    <li><a href="#nile_tilapia_credits">Nile tilapia</a></li> 
    <li><a href="#painted_turtle_credits">Painted turtle</a></li> 
    <li><a href="#stickleback_credits">Stickleback</a></li> 
    <li><a href="#tetraodon_credits">Tetraodon</a></li> 
    <li><a href="#tibetan_frog_credits">Tibetan frog</a></li> 
    <li><a href="#turkey_credits">Turkey</a></li> 
    <li><a href="#xentro_credits"><em>X. tropicalis</em></a></li> 
    <li><a href="#zebra_finch_credits">Zebra finch</a></li> 
    <li><a href="#zebrafish_credits">Zebrafish</a></li> 
  </ul> 
  </div>
</div>

<p>
<div class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href=#deuts>Deuterostomes</a>
  <div class="collapse" id="deuts"></p>
  <ul> 
    <li><a href="#ciona_credits"><em>C. intestinalis</em></a></li> 
    <li><a href="#lancelet_credits">Lancelet</a></li> 
    <li><a href="#urchin_credits"><em>S. purpuratus</em></a></li> 
  </ul> 
  </div>
</div>

<p>
<div class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href="#insects">Insects</a>
  <div class="collapse" id="insects"></p>
  <ul> 
    <li><a href="#anoGam_credits"><em>A. gambiae</em></a></li> 
    <li><a href="#apiMel_credits"><em>A. mellifera</em></a></li> 
    <li><a href="#droAna_credits"><em>D. ananassae</em></a></li> 
    <li><a href="#droEre_credits"><em>D. erecta</em></a></li> 
    <li><a href="#droGri_credits"><em>D. grimshawi</em></a></li> 
    <li><a href="#dm_credits"><em>D. melanogaster</em></a></li> 
    <li><a href="#droMoj_credits"><em>D. mojavensis</em></a></li> 
    <li><a href="#droPer_credits"><em>D. persimilis</em></a></li> 
    <li><a href="#dp_credits"><em>D. pseudoobscura</em></a></li> 
    <li><a href="#droSec_credits"><em>D. sechellia</em></a></li> 
    <li><a href="#droSim_credits"><em>D. simulans</em></a></li> 
    <li><a href="#droVir_credits"><em>D. virilis</em></a></li> 
    <li><a href="#droYak_credits"><em>D. yakuba</em></a></li> 
  </ul> 
  </div>
</div>

<p>
<div class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href="#nematodes">Nematodes</a>
  <div class="collapse" id="nematodes"></p>
  <ul> 
    <li><a href="#worm_credits"><em>C. brenneri</em></a></li> 
    <li><a href="#worm_credits"><em>C. briggsae</em></a></li> 
    <li><a href="#ce_credits"><em>C. elegans</em></a></li> 
    <li><a href="#worm_credits"><em>C. japonica</em></a></li> 
    <li><a href="#worm_credits"><em>C. remanei</em></a></li> 
    <li><a href="#worm_credits"><em>P. pacificus</em></a></li> 
  </ul> 
  </div>
</div>

<p>
<div class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href="#other">Other</a>
  <div class="collapse" id="other"></p>
  <ul> 
    <li><a href="#yeast_credits">Yeast</a></li> 
    <li><a href="#seahare_credits">California sea hare</a></li> 
  </ul> 
  </div>
</div>

<p>
<div class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href="#virus">Viruses</a>
  <div class="collapse" id="virus"></p>
  <ul> 
    <li><a href="#ebola_credits">Ebola</a></li> 
  </ul> 
  </div>
</div>

<!-- commenting out this section. It is very out of date.
<p>
<a href="#other_credits">Other scientists featured on this site</a></p>
--> 
<p>
<a href="#early_credits">Acknowledgment of early support</a></p> 
<p>
<a href="../staff.html">UCSC Genome Browser project team</a></p> 

<!-- Human Credits -->
<a name="human_credits"></a>
<h2>Human Genome Browser acknowledgments</h2>
<p> 
The UCSC Human Genome Browser is generated by the <a href="../staff.html">UCSC Genome Bioinformatics
Group</a> in collaboration with the <a href="http://www.genome.gov/10001772" 
target="_blank">International Human Genome Project</a>. The browser project is funded by grants from
the <a href="http://www.genome.gov" target="_blank">National Human Genome Research Institute</a>, 
and generous support from the <a href="http://www.hhmi.org" target="_blank">Howard Hughes Medical 
Institute</a> and the <a href="http://www.ucop.edu/california-institutes/" 
target="_blank">California Institutes for Science and Innovation</a>.</p>
<p> 
We work with collaborators at many other institutions to produce and annotate this reference 
sequence of the human genome. To view complete acknowledgments for a specific annotation, see the 
&quot;Credits&quot; section on the track's description page in the Genome Browser. For a list of 
browser-related publications, see our <a href="pubs.html">publications page</a>. 
<p> 
<strong>UCSC Human Genome Browsers:</strong> 
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=hg38" target="_blank">hg38</a> (GRCh38): Hiram Clawson, 
  Brian Raney, Robert Kuhn, Donna Karolchik, Steve Heitner </li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=hg19" target="_blank">hg19</a> (GRCh37): Hiram Clawson, 
  Brooke Rhead, Pauline Fujita, Ann Zweig, Katrina Learned, Donna Karolchik and Robert Kuhn </li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=hg18" target="_blank">hg18</a>: Engineering effort led by 
  Fan Hsu; QA effort led by Ann Zweig </li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=hg17" target="_blank">hg17</a>: Hiram Clawson, Terry 
  Furey, Heather Trumbower, Robert Kuhn, Donna Karolchik, Kate Rosenbloom, Angie Hinrichs, Rachel 
  Harte, and Jim Kent </li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=hg16" target="_blank">hg16</a>: Terry Furey, Hiram 
  Clawson, Heather Trumbower, Mark Diekhans, Robert Baertsch, Donna Karolchik, Jim Kent </li>
  <li>
  hg15 (archived): Jim Kent, Terry Furey, Matt Schwartz, Heather Trumbower, Angie Hinrichs, Fan 
  Hsu, Donna Karolchik, Jorge Garcia, Patrick Gavin, Chuck Sugnet, Yontao Lu, Mark Diekhans, Ryan 
  Weber, Robert Baertsch, Krishna Roskin, and many other students in the UCSC Genome Bioinformatics 
  group </li>
  <li>
  Earlier browsers (archived): UCSC Genome Bioinformatics staff and students </li>
</ul>

<!-- Human Data Sources -->
<h2>Human genome data sources</h2> 
<p>
Recent versions of the human genome sequence, starting with the 
<a href="../../cgi-bin/agGateway?db=hg19" target="_blank">hg19</a> (GRCh37) assembly, are provided 
by the <a href="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/" target="_blank">Genome 
Reference Consortium</a> (GRC), whose goal is to correct the small number of regions in the 
reference that are currently misrepresented, to close as many remaining gaps as possible and to 
produce alternative assemblies of structurally variant loci when necessary. </p> 
<p> 
Prior to <a href="../../cgi-bin/hgGateway?db=hg19" target="_blank">hg19</a>, the human genome 
sequence data used by the browser were generated by laboratories belonging to the 
<a href="labs.html">Human Genome Sequencing Consortium</a>. Periodic freezes of this data were 
generated by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank">NCBI</a>, and EST, mRNA, 
BACend and other types of data used for the assembly and annotation were obtained from 
<a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank">GenBank</a>, 
<a href="http://www.embl.org/" target="_blank">European Molecular Biology Lab</a> (EMBL), and 
<a href="http://www.ddbj.nig.ac.jp/" target="_blank">DNA Data Bank of Japan</a> (DDBJ). Clone maps 
were created by the individual sequencing centers. </p> 
<p> 
The original chromosome files were produced by the institutions and individuals listed in the table 
below.</p>
<p>

<!-- finished chromosome table-->
<table> 
  <tr>
    <th nowrap>Chromosome</th> 
    <th nowrap>Sequencing Centers</th> 
    <th nowrap>AGP File</th>
  </tr> 
  <tr>
    <td class='text-center'>1</td> 
    <td>The Sanger Institute, Cambridge, UK
      <br>University of Washington, Department of Genome Sciences, Seattle, WA, USA </td> 
    <td>Jane Rogers</td> 
  </tr> 
  <tr> 
    <td class='text-center'>2</td>
    <td>The Genome Institute at Washington University (WUSTL), St. Louis, MO, USA </td> 
    <td nowrap>Rick Wilson</td> 
  </tr> 
  <tr> 
    <td class='text-center'>3</td> 
    <td>Baylor College of Medicine Human Genome Sequencing Center (BCM-HGSC), Houston, TX, USA
      <br>Beijing Genomics Institute/Human Genome Center, Beijing, China 
      <br>University of Washington, Department of Genome Sciences, Seattle, WA, USA </td> 
    <td>Steve Scherer</td> 
  </tr> 
  <tr> 
    <td class='text-center'>4</td>
    <td>The Genome Institute at Washington University (WUSTL), St. Louis, MO, USA 
    <br>Stanford Human Genome Center, Palo Alto, CA, USA </td> 
    <td nowrap>Rick Wilson</td> 
  </tr> 
  <tr> 
    <td class='text-center'>5</td> 
    <td>Joint Genome Institute, U.S. Department of Energy, Walnut Creek, CA, USA 
      <br>Stanford Human Genome Center, Palo Alto, CA, USA</td> 
    <td>Jeremy Schmutz</td> 
  </tr> 
  <tr> 
    <td class='text-center'>6</td>
    <td>The Wellcome Trust Sanger Institute, Cambridge, UK</td> 
    <td>Jane Rogers</td> 
  </tr> 
  <tr> 
    <td class='text-center'>7</td> 
    <td>The Genome Institute at Washington University (WUSTL), St. Louis, MO USA 
      <br>University of Washington, Department of Genome Sciences, Seattle, WA, USA</td> 
    <td nowrap>Rick Wilson</td> 
  </tr>
  <tr> 
    <td class='text-center'>8</td>
    <td>The Broad Institute, Cambridge, MA USA 
      <br>Keio University, Tokyo, Japan 
      <br>Institute of Molecular Biology (IMB), Jena, Germany</td> 
    <td>Chad Nusbaum</td> 
  </tr> 
  <tr> 
    <td class='text-center'>9</td> 
    <td>The Wellcome Trust Sanger Institute, Cambridge, UK 
      <br>Gesellschaft fuer Biotechnologische Forschung (GBF), Braunschweig, Germany </td> 
    <td>Jane Rogers</td> 
  </tr>
  <tr> 
    <td class='text-center'>10</td>
    <td>The Wellcome Trust Sanger Institute, Cambridge, UK 
      <br>Genome Therapeutics Corporation (GTC), Waltham, MA, USA</td> 
    <td>Jane Rogers</td> 
  </tr> 
  <tr> 
    <td class='text-center'>11</td>
    <td>RIKEN Human Genome Research Group, Japan 
      <br>The Broad Institute, Cambridge, MA USA</td> 
    <td>Todd Taylor</td> 
  </tr> 
  <tr> 
    <td class='text-center'>12</td>
    <td>Baylor College of Medicine Human Genome Sequencing Center (BCM-HGSC), Houston, TX, USA</td> 
    <td>Steve Scherer</td> 
  </tr>
  <tr> 
    <td class='text-center'>13</td>
    <td>The Wellcome Trust Sanger Institute, Cambridge, UK</td> 
    <td>Jane Rogers</td> 
  </tr> 
  <tr> 
    <td class='text-center'>14</td> 
    <td>Genoscope National Sequencing Centre, Evry, France 
      <br>Institute for Systems Biology (ISB), Seattle, WA 
      <br>The Genome Institute at Washington University (WUSTL), St. Louis, MO USA 
      <br>RIKEN Human Genome Research Group, Japan </td> 
    <td>Jean Weissenbach</td> 
  </tr>
  <tr> 
    <td class='text-center'>15</td> 
    <td>The Broad Institute, Cambridge, MA USA 
      <br>Institute of Molecular Biology (IMB) 
      <br>Institute for Systems Biology (ISB), Seattle, WA</td> 
    <td>Chad Nusbaum</td> 
  </tr> 
  <tr> 
    <td class='text-center'>16</td> 
    <td>Joint Genome Institute (JGI), U.S.  Department of Energy, Walnut Creek, CA, USA 
      <br>Stanford Human Genome Center, Palo Alto, CA, USA 
      <br>The Institute for Genomic Research (TIGR), Rockville, MD, USA 
      <br>The Wellcome Trust Sanger Institute, Cambridge, UK</td> 
    <td>Jeremy Schmutz</td> 
  </tr>
  <tr> 
    <td class='text-center'>17</td> 
    <td>The Broad Institute, Cambridge, MA USA</td> 
    <td>Chad Nusbaum</td> 
  </tr>
  <tr> 
    <td class='text-center'>18</td> 
    <td>The Broad Institute, Cambridge, MA USA 
      <br>RIKEN Human Genome Research Group, Japan </td> 
    <td>Chad Nusbaum</td> 
  </tr> 
  <tr>
    <td class='text-center'>19</td> 
    <td>Joint Genome Institute (JGI), U.S. Department of Energy, Walnut Creek, CA, USA 
      <br>Stanford Human Genome Center, Palo Alto, CA, USA</td> 
    <td>Jeremy Schmutz</td> 
  </tr> 
  <tr> 
    <td class='text-center'>20</td> 
    <td>The Wellcome Trust Sanger Institute, Cambridge, UK </td> 
    <td>James Gilbert</td> 
  </tr> 
  <tr> 
    <td class='text-center'>21</td> 
    <td>RIKEN Human Genome Research Group, Japan 
      <br>Max Planck Institute for Molecular Genetics, Berlin, Germany  
      <br>Gesellschaft fuer Biotechnologische Forschung (GBF), Braunschweig, Germany 
      <br>Keio University School of Medicine, Tokyo, Japan 
      <br>Chromosome 21 Consortium</td> 
    <td nowrap>Todd Taylor</td> 
  </tr>
  <tr> 
    <td class='text-center'>22</td> 
    <td>The Wellcome Trust Sanger Institute, Cambridge, UK 
      <br>The University of Oklahoma, Norman, OK USA 
      <br>Keio University School of Medicine, Tokyo, Japan</td> 
    <td>Jane Rogers</td> 
  </tr> 
  <tr> 
    <td class='text-center'>X</td> 
    <td>The Wellcome Trust Sanger Institute, Cambridge, UK 
      <br>Baylor College of Medicine Human Genome Sequencing Center (BCM-HGSC), Houston, TX, USA 
      <br>Institute of Molecular Biology (IMB), Jena, Germany 
      <br>The Genome Institute at Washington University (WUSTL), St. Louis, MO USA 
      <br>Max Planck Institute for Molecular Genetics, Berlin, Germany</td> 
    <td>Jane Rogers</td> 
  </tr> 
  <tr> 
    <td class='text-center'>Y</td> 
    <td>The Genome Institute at Washington University (WUSTL), St. Louis, MO USA</td> 
    <td nowrap>Rick Wilson</td> 
  </tr> 
</table>

<!-- Alpaca Credits -->
<a name="alpaca_credits"></a>
<h2>Alpaca genome</h2> 
<p>
<strong>Sequencing/Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA</p>
<p> 
<strong>UCSC Alpaca Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=vicPac2" target="_blank">vicPac2</a>: Hiram Clawson and
  Luvina Guruvadoo </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=vicPac1" target="_blank">vicPac1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="alpaca_use"></a> 
<h3>Data use</h3> 
<p>
The alpaca sequence has been made freely available as part of the Mammalian Genome Project (29 
Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>. <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target=_blank>A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82. 
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The data are provided with the following understanding:</p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li>
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice.</li>  
</ul>

<!-- Armadillo Credits -->
<a name="armadillo_credits"></a>
<h2>Armadillo genome</h2> 
<p>
<strong>Sequencing/Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p>
<p> 
<strong>UCSC Armadillo Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=dasNov3" target="_blank">dasNov3</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="armadillo_use"></a> 
<h3>Data use</h3> 
<p>
The armadillo sequence has been made freely available as part of the Mammalian Genome Project (29 
Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>. <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target=_blank>A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82. 
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The data are provided with the following understanding:</p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li>
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice.</li>  
</ul>

<!-- Baboon Credits -->
<a name="baboon_credits"></a>
<h2>Baboon genome</h2> 
<p> 
<strong>Funding: </strong> <a href="http://www.genome.gov/" 
target="_blank">National Human Genome Research Institute</a> (NHGRI), USA </p> 
<p> 
<strong>Sequencing/Assembly: </strong> 
<a href="https://www.hgsc.bcm.edu/non-human-primates/baboon-genome-project" target="_blank">Baylor
College of Medicine Human Genome Sequencing Center (BCM-HGSC)</a>, Houston, TX, USA </p> 
<p> 
<strong>UCSC Baboon Genome Browser/Annotations:</strong></p> 
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=papAnu4" target="_blank">papAnu4</a>: Hiram Clawson and
  Lou Nassar </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=papAnu2" target="_blank">papAnu2</a>: Chin Li and Luvina 
  Guruvadoo </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=papHam1" target="_blank">papHam1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li>
</ul> 
<a name="baboon_use"></a>
<h3>Data use</h3> 
<p> 
The baboon sequence is made freely available for public use by the Baboon Genome Sequencing 
Consortium. Please review the BCM-HGSC <a href="https://www.hgsc.bcm.edu/bcm-hgsc-conditions-use" 
target="_blank">conditions for use</a> guidelines before using this data. </p>

<!-- Bison Credits -->
<a name="bison_credits"></a>
<h2>Bison genome</h2> 
<p>
<strong>Sequencing/Assembly:</strong> <a href="https://www.umd.edu/" target="_blank">University of
Maryland</a>, College Park, MD, USA </p> 
<p> 
<strong>UCSC Bison Genome Browser/Annotations:</strong></p>
<ul>   
  <li> 
  <a href="../../cgi-bin/hgGateway?db=bisBis1" target="_blank">bisBis1</a>: Hiram Clawson and 
  Chris Villarreal </li>
</ul> 
<a name="bison_use"></a> 
<h3>Data use</h3> 
<p>
The bison sequence is made freely available by the University of Maryland Genome Assembly Group, 
Aleksey Zimin. </p>

<!-- Bonobo Credits -->
<a name="bonobo_credits"></a>
<h2>Bonobo genome</h2>
<p>
<strong>Funding: </strong>
Italian grant <a href="http://erc.europa.eu/">Futuro in Ricerca
2010-RBFR103CE3</a> and the <a href="https://www.hhmi.org/">
Howard Hughes Medical Institute</a>
<p>

<strong>Sequencing/Assembly: </strong>
<p>
<a href="https://eichlerlab.gs.washington.edu/">Eichler Laboratory</a>,
Department of Genome Sciences, University of Washington, Seattle<br>
<a href="https://www.uniba.it/docenti/ventura-mario">Mario Ventura</a>
Universit&agrave; degli studi di Bari Aldo Moro 
<p>
<strong>UCSC Bonobo Genome Browser/Annotations:</strong></p> 
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=panpan3" target="_blank">panPan3</a>: Hiram Clawson and Lou
  Nassar</li>
</ul>
<a name="bonobo_use"></a>
<h3>Data use</h3> 
<p> 
The bonobo sequence is made freely available by the University of Bari and
University of Washington.
<hr>
<p> 
<strong>Funding: </strong> 
<a href="http://erc.europa.eu/" target="_blank">European Research Council</a> (grant 233297, TWOPAN)
and the <a href="http://www.mpg.de/" target="_blank">Max Planck Society</a></p>
<p> 
<strong>Sequencing/Assembly: </strong> <a href="http://www.eva.mpg.de/bonobo-genome/index.html" 
target="_blank">Max-Planck Institute for Evolutionary Anthropology</a>, Germany</p>
<p> 
<strong>UCSC Bonobo Genome Browser/Annotations:</strong></p> 
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=panpan2" target="_blank">panPan2</a>: Hiram Clawson and Jairo
  Navarro</li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=panPan1" target="_blank">panPan1</a>: Chin Li, Hiram Clawson, 
  and Jonathan Casper </li> 
</ul> 
<a name="bonobo_use"></a>
<h3>Data use</h3> 
<p> 
The bonobo sequence is made freely available by the Max-Planck Institute for Evolutionary 
Anthropology. The initial analysis of this dataset can be found in Pr&uuml;fer K <em>et al</em>.  
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3498939/" 
target="_blank">The bonobo genome compared with the chimpanzee and human genomes</a>. 
<em>Nature</em>. 2012 Jun 28;486(7404):527-31. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22722832" target="_blank">22722832</a></p>

<!-- Brown kiwi Credits -->
<a name="brown_kiwi_credits"></a>
<h2>Brown kiwi genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> <a href="http://www.eva.mpg.de/index.html" 
target="_blank">Max-Planck Institute for Evolutionary Anthropology</a>, Germany </p>
<p> 
<strong>UCSC Brown Kiwi Genome Browser/Annotations:</strong></p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=aptMan1" target="_blank">aptMan1</a>: Chris Eisenhart and 
  Luvina Guruvadoo </li> 
</ul> 
<a name="brown_kiwi_use"></a> 
<h3>Data use</h3> 
<p>
The brown kiwi sequence is made freely available by the Max-Planck Institute for Evolutionary 
Anthropology. The initial analysis of this dataset can be found in Le Duc D <em>et al</em>.  
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0711-4" 
target="_blank">Kiwi genome provides insights into evolution of a nocturnal lifestyle</a>. 
<em>Genome Biol</em>. 2015 Jul 23;16:147. 
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26201466" target="_blank">26201466</a> </p>

<!-- Bushbaby Credits -->
<a name="bushbaby_credits"></a>
<h2>Bushbaby genome</h2> 
<p>
<strong>Funding:</strong> <a href="http://www.genome.gov" target="_blank">National Human Genome 
Research Institute</a> (NHGRI)</p> 
<p>
<strong>Sequencing/Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA</p>
<p> 
<strong>UCSC Bushbaby Genome Browser/Annotations:</strong> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=otoGar3" target="_blank">otoGar3</a>: Hiram Clawson, Chin Li, 
  Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and Donna 
  Karolchik </li>
</ul> 
<a name="bushbaby_use"></a>
<h3>Data use</h3> 
<p>
The bushbaby sequence has been made freely available as part of the Mammalian Genome Project (29 
Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>. <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target=_blank>A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82. 
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The data are provided with the following understanding:</p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li>
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice.</li>  
</ul>

<!-- Cat Credits -->
<a name="cat_credits"></a>
<h2>Cat genome</h2> 
The UCSC felCat9 Genome Browser displays data produced by the  Genome Sequencing Center (GSC) at 
Washington University (WashU) School of Medicine:
<ul>
 <li>
  <a href="../../cgi-bin/hgGateway?db=felCat9" target="_blank">felCat9</a> browser:
  Nov. 2017 Felis_catus_9.0 assembly of the domestic cat</li>
</ul>
<p><strong>Sequencing/Assembly:</strong>
<ul>
  <li>
  <a href="https://gtac.wustl.edu/" target="_blank">Genome Sequencing Center (GSC) at Washington
  University (WashU) School of Medicine</a>, St. Louis, MO, USA </li>
</ul>
<p>

<strong>UCSC Cat Genome Browser/Annotations <a href="../../cgi-bin/hgGateway?db=felCat9" 
target="_blank">(felCat9)</a>:</strong> Hiram Clawson and Conner Powell

<hr>
<p> 
The UCSC  felCat8 and felCat5 Genome Browsers display data produced by the International Cat Genome 
Sequencing Consortium: 
<ul>
  <li>
  <a href="../../cgi-bin/hgGateway?db=felCat8" target="_blank">felCat8</a> browser:  
  Nov. 2014 Felis_catus_8.0 assembly of the domestic cat</li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=felCat5" target="_blank">felCat5</a> browser: 
   Sep. 2011 Felis_catus-6.2 assembly of the domestic cat</li>
</ul>
<p> 
<strong>Funding:</strong> <a href="http://www.genome.gov/" target="_blank">National Human Genome 
Research Institute</a> (NHGRI), Bethesda, MD, USA </p>
<p>
<strong>454 Production Sequencing: </strong> 
<a href="http://genome.wustl.edu/genomes/detail/felis-catus/" target="_blank">The Genome Institute 
at Washington University (WUSTL)</a>, St. Louis, MO, USA </p> 
<p>
<strong>BAC sequencing:</strong> The Genome Institute at WUSTL</p> 
<p>
<strong>Plasmid and fosmid sequencing:</strong> <a href="http://www.broadinstitute.org/" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA and 
<a href="http://www.beckmangenomics.com/" target="_blank">Agencourt Bioscience</a> </p>
<p>
<strong>Sequence assembly and data integration for creation of chromosomal AGP files:</strong> The 
Genome Institute at WUSTL </p>
<p>
<strong>Cat RH map:</strong> Bill Murphy, <a href="http://genetics.tamu.edu/" 
target=_blank>Texas A&M University</a>, College Station, TX, USA. </p>  
<p>
<strong>DNA source:</strong> Dr. Kristina Narfstrom, <a href="http://www.vmth.missouri.edu/" 
target=_blank>University of Missouri</a>, Columbia, MO, USA. </p> 
<p>
<strong>UCSC Cat Genome Browser/Annotations: </strong>
<ul>
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=felCat8" target="_blank">felCat8</a>:</strong> 
  Hiram Clawson, Brian Raney, and Christopher Lee </li>
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=felCat5" target="_blank">felCat5</a>:</strong> 
  Hiram Clawson, Chin Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke 
  Rhead, Greg Roe, and Donna Karolchik</li> 
</ul>
<hr>
<p> 
The UCSC <a href="../../cgi-bin/hgGateway?db=felCat4" target="_blank">felCat4</a> Genome Browser 
displays data from the Dec. 2008 catChrV17e draft assembly, based on sequence reads from six 
domestic cats and one wild cat. </p> 
<p><strong>Sequencing/Assembly:</strong> 
<ul> 
  <li>
  <a href="http://www.nisc.nih.gov/" target="_blank">NHGRI Genome Technology Branch and NIH 
  Intramural Sequencing Center</a>, Bethesda, MD, USA </li> 
  <li> 
  <a href="http://www.beckmangenomics.com/" target="_blank">Agencourt Bioscience</a>, Danvers, MA, 
  USA </li> 
  <li> 
  <a href="http://www.cancer.gov/" target="_blank">NCI Laboratory of Genomic Diversity</a>, 
  Frederick, MD, USA </li> 
  <li> 
  <a href="http://www.hillspet.com" target="_blank">Hill's Pet Nutrition, Inc.</a>, Topeka, KS, 
  USA </li> 
</ul>
<p>
<strong>UCSC Cat Genome Browser/Annotations (<a href="../../cgi-bin/hgGateway?db=felCat4" 
target="_blank">felCat4</a>):</strong> <a href="../staff.html" target="_blank">UCSC Genome 
Bioinformatics Group</a>, Santa Cruz, CA, USA - Chin Li, Hiram Clawson, Antonio Coelho, Mary 
Goldman, and Donna Karolchik </p> 
<hr>
<p> 
The UCSC <a href="../../cgi-bin/hgGateway?db=felCat3" target="_blank">felCat3</a> Genome Browser 
displays data from the March 2006 v3 draft assembly of the domestic cat (<em>Felis catus</em>).</p> 
<p>
<strong>Funding:</strong> <a href="http://www.genome.gov/" target="_blank">National Human Genome 
Research Institute</a> (NHGRI), Bethesda, MD, USA </p> 
<p>
<strong>Project Leadership:</strong> 
<a href="http://www.broadinstitute.org/science/projects/mammals-models/data-release-mammalian-genome-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA</p> 
<p>
<strong>Sequencing/Assembly:</strong> The Broad Institute - team led by by Kerstin
Lindblad-Toh</p> 
<p>
<strong>Plasmid and fosmid libraries:</strong> The Broad Institute </p>
<p>
<strong>BAC end sequencing:</strong> <a href="http://www.beckmangenomics.com/" 
target="_blank">Agencourt Bioscience</a>, Danvers, MA, USA </p> 
<p>
<strong>UCSC Cat Genome Browser/Annotations (<a href="../../cgi-bin/hgGateway?db=felCat3" 
target="_blank">felCat3</a>):</strong> Heather Trumbower, Angie Hinrichs, Mark Diekhans, Brooke 
Rhead, and Archana Thakkapallayil </p> 
<a name="cat_use"></a>
<h3>Data use</h3> 
<p> 
The catChrV17e cat assembly was initially published in Pontius JU <em>et al</em>. 
<a href="http://genome.cshlp.org/content/17/11/1675.long" target=_blank>Initial sequence and 
comparative analysis of the cat genome</a>.  <em>Genome Res</em>. 2007 Nov;17(11):1675-89. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17975172" target="_blank">17975172</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2045150/" target="_blank">PMC2045150</a></p> 
<p> 
The Broad Institute released the initial 2X (v3) assembly of the cat genome as part of the Mammalian
Genome Project (29 Mammals Project). The initial analysis of this dataset can be found in 
Lindblad-Toh K <em>et al</em>. 
<a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The cat sequence is made freely available with the following understanding: 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases.</li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged.</li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Chimp Credits -->
<a name="chimp_credits"></a>
<h2>Chimpanzee genome</h2> 
<p> 
The UCSC Chimpanzee Genome Browsers display whole-genome assembly data produced by the University of
Washington:</p>
<ul>
  <li>
  Jan. 2018 <a href="../../cgi-bin/hgGateway?db=panTro6" target="_blank">panTro6</a> browser:
  Clint_PTRv2 124x PacBio; Illumina NextSeq 500; BioNano Saphyr (two enzyme)</li>
</ul>
<p>
and draft assembly data produced by the Chimpanzee Genome Sequencing Consortium:</p>
<ul> 
  <li>
  May 2016 <a href="../../cgi-bin/hgGateway?db=panTro5" target="_blank">panTro5</a> browser: v3.0 
  6x Sanger; 55x Illumina; 9x PacBio </li> 
  <li>
  Feb. 2011 <a href="../../cgi-bin/hgGateway?db=panTro4" target="_blank">panTro4</a> browser: 
  v2.1.4 chromosome-based 6X draft assembly </li> 
  <li>
  Oct. 2010 <a href="../../cgi-bin/hgGateway?db=panTro3" target="_blank">panTro3</a> browser: 
  v2.1.3 chromosome-based 6X draft assembly </li>
  <li>
  Mar. 2006 <a href="../../cgi-bin/hgGateway?db=panTro2" target="_blank">panTro2</a> browser: 
  v2.1 chromosome-based 6X draft assembly (Oct. 2005) </li>
  <li>
  Nov. 2003 <a href="../../cgi-bin/hgGateway?db=panTro1" target="_blank">panTro1</a> browser: 
  v1.1 Arachne draft assembly </li>
</ul> 
<p>
<strong>Funding:</strong> <a href="http://www.genome.gov/" target="_blank">National Human Genome 
Research Institute</a> (NHGRI) </p>
<p>
<strong>Sequencing/Assembly (panTro6):</strong> </p>
<p>
The whole-genome assembly of a male chimpanzee, Clint (Yerkes pedigree number C0471) was done in
stages, resulting in three progressive versions:
<ul>
  <li>
  The backbone of the assembly is made up of contigs obtained by assembling the PacBio (P6-C4
  chemistry) reads using the Falcon assembler. These contigs are error-corrected using the tool
  Quiver and then further error-corrected using Pilon with 37-fold Illumina paired-end reads.</li>
  <li>
  BioNano optical maps are used to place the contigs into scaffolds, followed by chromosome-level
  scaffolding using bicolor FISH.</li>
  <li>
  The assembly is further error-corrected, particularly in regions
  that have clustered 1-2 bp indels. This is done by applying an Eichler-lab-created Freebayes-based
  indel correction method.</li>
</ul>
<p>
<strong>Sequencing/Assembly (panTro5):</strong> </p>
<ul> 
  <li> 
  BAC sequencing - McDonnell Genome Institute at Washington University School of Medicine, St 
  Louis, MO. Sequence generation. </li>
  <li> 
  Assembly and data integration for creation of chromosomal AGP files: 
  <ul> 
    <li> 
    Lukas Kuderna and Tomas Marques-Bonet, ICREA at Institut de Biologia Evolutiva, (UPF-CSIC), 
    PRBB, 08003 Barcelona, Spain, and LaDeana Hillier, McDonnell Genome Institute at Washington 
    University School of Medicine, St.  Louis, MO, USA </li> 
    <li> 
    Wes Warren, Lars Feuk (Uppsala U), Andrew Sharp (Mt Sinai), Ed Green (Dovetail), Mikkel 
    Schierup (Aarhus U), and Bojan Obradovic (Illumina).</li>
  </ul>	
</ul> 
<p>
<strong>Sequencing/Assembly (panTro2, panTro3, panTro4):</strong> </p>
<ul> 
  <li> 
  Whole genome shotgun (WGS) sequence data assembly and organization: 
  <a href="http://genome.wustl.edu/genomes/detail/pan-troglodytes/" target="_blank">The Genome 
  Institute at Washington University (WUSTL)</a>, St. Louis, MO, USA </li> 
  <li> 
  Underlying WGS data: The Genome Institute at WUSTL and 
  <a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/chimpanzee/chimpanzee-genome-project" 
  target =_blank>The Broad Institute</a>, Cambridge, MA USA </li>
  <li> 
  5-Mb region of chromosome 7 at chr7:84674857-89461887): finished by The Genome Institute at 
  WUSTL </li>
  <li> 
  Chromosome Y sequence: finished by The Genome Institute at WUSTL, with detailed mapping and 
  extensive collaboration with David Page's group at the <a href="http://wi.mit.edu/" 
  target="_blank">Whitehead Institute</a>. For more information, see: 
  <ul> 
    <li> 
    Hughes JF <em>et al</em>.  
    <a href="http://www.nature.com/nature/journal/v437/n7055/abs/nature04101.html" 
    target="_blank">Conservation of Y-linked genes during human evolution revealed by comparative 
    sequencing in chimpanzee</a>. <em>Nature</em>. 2005 Sep 1;437(7055):100-3. PMID: 16136134</li>
    <li> 
    &quot;The DNA Sequence of Chimpanzee Chromosome Y&quot;, unpublished. </li> 
  </ul> 
  <li> 
  Chromosome 21 sequence: Todd Taylor and the <a href="http://www.gsc.riken.go.jp/eng/index.html" 
  target="_blank">RIKEN Genomic Sciences Center</a>. See Watanabe <em>et al.</em> 
  <a href="http://www.nature.com/nature/journal/v429/n6990/abs/nature02564.html" 
  target="_blank">DNA sequence and comparative analysis of chimpanzee chromosome 22</a>. 
  <em>Nature</em>. 2004 May 27;429(6990):382-8. PMID: 15164055 </li>
</ul> 
<p>
<strong>Sequencing/Assembly (panTro1):</strong> Sequence assembled by teams led by Eric Lander, 
Ph.D. at The Broad Institute and Richard K. Wilson, Ph.D. at The Genome Institute at WUSTL </p> 
<p>
<strong>Alignments:</strong> LaDeana Hillier, The Genome Institute at WUSTL and The Broad 
Institute</p> 
<p>
<strong>Annotations:</strong> </p>
<ul> 
  <li> 
  The Genome Institute at WUSTL, St. Louis, MO, USA</li> 
  <li> 
  <a href="http://www.imim.cat/programesrecerca/informaticabiomedica/en_index.html" 
  target="_blank">Research Group on Biomedical Informatics of IMIM/UPF</a> (GRIB), Barcelona, 
  Spain </li> 
  <li> 
  Riken Genomic Sciences Center, Yokohama City, Japan </li>
  <li> 
  <a href="../staff.html" target="_blank">UCSC Genome Bioinformatics Group</a>, Santa Cruz, CA, 
  USA </li> 
</ul> 
<p>
<strong>UCSC Chimp Genome Browser:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=panTro6" target="_blank">panTro6</a>: Hiram Clawson and 
  Jairo Navarro</li> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=panTro5" target="_blank">panTro5</a>: Hiram Clawson and 
  Chris Villarreal </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=panTro4" target="_blank">panTro4</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=panTro3" target="_blank">panTro3</a>: Hiram Clawson, Greg 
  Roe, and Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=panTro2" target="_blank">panTro2</a>: Kate Rosenbloom, Brian
  Raney, Hiram Clawson, Ann Zweig, Archana Thakkapallayil, and Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=panTro1" target="_blank">panTro1</a>: Kate Rosenbloom, Jim 
  Kent, Hiram Clawson, Heather Trumbower, Robert Kuhn, and Donna Karolchik </li> 
</ul> 
<a name="chimp_use"></a>
<h3>Data use</h3> 
<p> 
The initial sequencing and analysis of the chimpanzee genome was published by the Chimpanzee 
Sequencing and Analysis Consortium. The chimpanzee sequence data may be freely downloaded, used in 
analyses, and repackaged in databases if the provider is properly acknowledged. Please cite this 
publication when using these data:</p> 
<p> 
The Chimpanzee Sequencing and Analysis Consortium.  
<a href="http://www.nature.com/nature/journal/v437/n7055/abs/nature04072.html" 
target="_blank">Initial sequence of the chimpanzee genome and comparison with the human genome</a>. 
<em>Nature</em>. 2005 Sep 1;437(7055):69-87.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16136131" target="_blank">16136131</a></p>

<!-- Chinese Hamster Credits -->
<a name="chinese_hamster_credits"></a>
<h2>Chinese hamster genome</h2> 
<p>
<strong>Sequencing/Assembly:</strong> <a href="https://www.eaglegenomics.com/eagle-case-studies/horizon-case-study/" 
target="_blank">Eagle Genomics Ltd</a> </p> 
<p> 
<strong>UCSC Chinese Hamster Genome Browser/Annotations:</strong> </p> 
<ul>
  <li>
  <a href="../../cgi-bin/hgGateway?db=criGriChoV2" target="_blank">criGriChoV2</a>: Hiram Clawson
   and Daniel Schmelter </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=criGri1" target="_blank">criGri1</a>: Hiram Clawson, Brian 
  Raney, and Steve Heitner </li> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=criGriChoV1" target="_blank">criGriChoV1</a>: 
Hiram Clawson, Max Haeussler, Matt Speir, and Cath Tyner </li>
</ul>
<a name="chinese_hamster_use"></a>
<h3>Data use</h3> 
<p> 
The Chinese hamster sequence data may be freely downloaded, used in analyses, and repackaged in 
databases if the provider is properly acknowledged. Please cite this publication when using these 
data:</p> 
<p> 
Lewis NE <em>et al.</em> <a href="http://www.nature.com/nbt/journal/v31/n8/full/nbt.2624.html" 
target="_blank">Genomic landscapes of Chinese hamster ovary cell lines as revealed by the 
Cricetulus griseus draft genome</a>. <em>Nat Biotechnol</em>. 2013 Aug;31(8):759-65. </p>

<!-- Chinese Pangolin Credits -->
<a name="chinese_pangolin_credits"></a>
<h2>Chinese pangolin genome</h2> 
<p> 
<strong>Funding:</strong> NIH-NHGRI grant 5U54HG00307907 to RKW, Director of The Genome Institute 
at Washington University </p> 
<p> 
<strong>DNA source:</strong> Dr. Stephen O'Brien at <a href="http://english.spbu.ru/" 
target="_blank">St. Petersburg State University</a> </p>
<p> 
<strong>Sequencing/Assembly:</strong> 
<a href="http://genome.wustl.edu/genomes/detail/Manis-pentadactyla/" target="_blank">Genome 
Institute, Washington University School of Medicine</a>, St. Louis, MO, USA </p> 
<p> 
<strong>UCSC Chinese Pangolin Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=manPen1" target="_blank">manPen1</a>: Hiram Clawson, Cath 
  Tyner </li> 
</ul> 
<a name="chinese_pangolin"></a>
<h3>Data use</h3> 
<p>
The Chinese pangolin sequence data may be freely downloaded, used in analyses, and repackaged in 
databases if the provider is properly acknowledged: Dr. Richard K. Wilson and The Genome Institute, 
Washington University School of Medicine.</p>

<!-- Cow Credits -->
<a name="cow_credits"></a>
<h2>Cow genome</h2> 
<p> 
The UCSC Cow Genome Browser displays three different assemblies of the bovine genome: one provided by 
the University of Maryland Center for Bioinformatics and Computational Biology (UMD CBCB), one 
from the Baylor College of Medicine Human Genome Sequencing Center (BCM-HGSC), and a third one
from the USDA Agricultural Research Service (USDA ARS). The assemblies 
available in the browser include: </p> 
<ul>
  <li>
  Apr. 2018 (<a href="../../cgi-bin/hgGateway?db=bosTau9" target="_blank">bosTau9</a>) assembly -
  ARS-UCD1.2 </li> 
  <li>
  Jun. 2014 (<a href="../../cgi-bin/hgGateway?db=bosTau8" target="_blank">bosTau8</a>) assembly - 
  UMD CBCB version UMD_3.1.1  </li>
  <li>
  Oct. 2011 (<a href="../../cgi-bin/hgGateway?db=bosTau7" target="_blank">bosTau7</a>) assembly - 
  Bovine Genome Sequencing Consortium (BCM-HGSC) version Btau_4.6.1 </li> 
  <li>
  Nov. 2009 (<a href="../../cgi-bin/hgGateway?db=bosTau6" target="_blank">bosTau6</a>) assembly - 
  UMD CBCB version UMD_3.1 </li>
  <li>Oct. 2007 (<a href="../../cgi-bin/hgGateway?db=bosTau4" target="_blank">bosTau4</a>) assembly 
  - BCM-HGSC version Btau_4.0 </li> 
  <li>
  Aug. 2006 (<a href="../../cgi-bin/hgGateway?db=bosTau3" target="_blank">bosTau3</a>) assembly - 
  BCM-HGSC version Btau_3.1 </li> 
  <li>Mar. 2005 (<a href="../../cgi-bin/hgGateway?db=bosTau2" target="_blank">bosTau2</a>) 
  assembly - BCM-HGSC version Btau_2.0 </li>
  <li>
  Sep. 2004 (bosTau1) assembly - BCM-HGSC version Btau_1.0 </li> 
</ul> 
<hr>
<p> 
The UMD CBCB genome assemblies are provided with the following acknowledgments: </p> 
<p> 
<strong>Genome sequence:</strong> NCBI Trace Archive </p>
<p><strong>Genome assembly:</strong> <a href="http://www.cbcb.umd.edu/" 
target="_blank">University of Maryland Center for Bioinformatics and Computational Biology</a>, 
College Park, MD, USA </p> 
<p>
<strong>UCSC Cow Genome Browser/Annotations:</strong> </p>
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=bosTau9" target="_blank">bosTau9</a>: Hiram Clawson and
  Lou Nassar </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=bosTau8" target="_blank">bosTau8</a>: Steve Heitner and 
  Luvina Guruvadoo </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=bosTau6" target="_blank">bosTau6</a>: Hiram Clawson, Chin 
  Li, Brooke Rhead, Greg Roe, Steve Heitner, and Donna Karolchik </li> 
</ul> 
<hr>
<p> 
The BCM-HGSC genome assemblies are provided with the following acknowledgments: </p> 
<p>
<strong>Funding</strong> - For a list of the many agencies that funded the cow sequencing project,
see the BCM-HGSC <a href="https://www.hgsc.bcm.edu/other-mammals/bovine-genome-project" 
target="_blank">Bovine Genome Project</a> page. </p> 
<p>
<strong>BAC library DNA: </strong> Dr. Michael MacNeil's laboratory at the 
<a href="http://www.ars.usda.gov/main/main.htm" target="_blank">USDA Agricultural Research 
Service</a>, Miles City, MT, USA </p> 
<p>
<strong>Whole genome shotgun sequence DNA: </strong> Dr. Timothy Smith's laboratory at the 
<a href="http://www.ars.usda.gov/Main/docs.htm?docid=2340" target="_blank">U.S. Meat Animal 
Research Center</a>, Clay Center, NE, USA </p> 
<p>
<strong>BAC-based fingerprint map: </strong> <a href="http://www.bcgsc.ca/" target=-blank>Genome 
Sciences Centre</a>, Vancouver, B.C. </p> 
<p>
<strong>UCSC Cow Genome Browser/Annotations:</strong> </p>
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=bosTau7" target="_blank">bosTau7</a>: Chin Li, Hiram 
  Clawson, Brian Raney and Steve Heitner </li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=bosTau4" target="_blank">bosTau4</a>: Hiram Clawson, Brian 
  Raney, and Ann Zweig </li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=bosTau3" target="_blank">bosTau3</a>: Heather Trumbower, 
  Angie Hinrichs, Kayla Smith and Donna Karolchik </li>
  <li></li>
  <a href="../../cgi-bin/hgGateway?db=bosTau2" target="_blank">bosTau2</a>: Galt Barber, Brian 
  Raney, Mark Diekhans, Jennifer Jackson, and Donna Karolchik </li> 
  <li>
  bosTau1 (archived): Heather Trumbower, Jim Kent, Hiram Clawson, Angie Hinrichs, Brian Raney, 
  Mark Diekhans, Robert Kuhn, Ali Sultan-Qurraie and Donna Karolchik </li> 
</ul> 
<a name="cow_use"></a> 
<h3>Data use</h3> 
<p>
The cow sequence data may be freely downloaded, used in analyses, and repackaged in databases if 
the provider is properly acknowledged. Please cite the appropriate publication when using these 
data.</p> 
<p> 
<strong>UMD cow assembly:</strong>
Zimin AV, Delcher AL, Florea L, Kelley DR, Schatz MC, Puiu D, Hanrahan F, Pertea G, Van Tassell 
CP, Sonstegard TS <em>et al</em>.  <a href="http://genomebiology.com/content/10/4/R42" 
target="_blank">A whole-genome assembly of the domestic cow, Bos taurus</a>.  <em>Genome Biol</em>. 2009;10(4):R42. </p> 
<p> 
<strong>Baylor cow assembly:</strong>
Bovine Genome Sequencing and Analysis Consortium. 
<a href="http://www.sciencemag.org/content/324/5926/522.abs" target="_blank">The genome sequence of 
taurine cattle: a window to ruminant biology and evolution</a>. <em>Science</em>. 
2009 Apr 24;324(5926):522-8. </p> 
<p>
For more information see the <a href="http://www.cbcb.umd.edu/" target="_blank">UMD CBCB</a> 
<!--Commenting out until link is fixed 
<a href="http://www.cbcb.umd.edu/research/bos_taurus_assembly.shtml" target="_blank">Bos taurus 
assembly</a> page
--> 
and the BCM-HGSC <a href="https://www.hgsc.bcm.edu/other-mammals/bovine-genome-project" 
target="_blank">Bovine Genome Project</a> pages.</p>

<!-- Crab-eating Macaque Credits -->
<a name="crab-eating_macaque_credits"></a>
<h2>Crab-eating macaque genome</h2> 
<p> 
<strong>Funding: </strong> <a href="http://www.genome.gov" target=_blank>National Human Genome 
Research Institute</a> (NHGRI) </p>
<p>
<strong>DNA source: </strong> Dr. Jay Kaplan, Wake Forest Primate Facility, Wake Forest, NC </p> 
<p>
<strong>Genome Sequence: </strong> 
<a href="http://genome.wustl.edu/genomes/detail/macaca-fascicularis/" target="_blank">Genome 
Institute at Washington University (WUSTL)</a>, St.  Louis, MO, USA </p> 
<p>
<strong>Sequence Assembly: </strong> Richa Agarwala, Sergey Shiryaev, NCBI and The Genome 
Institute, Washington University School of Medicine, St. Louis, MO, USA </p>
<p>
<strong>Assembly curation: </strong> LaDeana Hillier, Genome Institute, Washington University 
School of Medicine, St. Louis, MO, USA </p> 
<p>
<strong>FISH mapping data: </strong> Mariano Rocchi, Department of Biology, University of Bari, 
Bari, Italy </p> 
<p> 
<strong>UCSC Crab-eating Macaque Genome Browser/Initial Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=macFas5" target="_blank">macFas5</a>: Hiram Clawson, Brian 
  Raney and Steve Heitner </li> 
</ul> 
<a name="crab-eating_macaque_use"></a>
<h3>Data use</h3> 
<p>
The crab-eating macaque data are provided with the following understanding:</p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases.</li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Dog Credits -->
<a name="dog_credits"></a>
<h2>Dog genome</h2> 
<p><strong>canFam5 submitter:</strong> The <a target="_blank" href="https://umich.edu/">
University of Michigan</a>.</p>
<p>
<strong>Funding:</strong> The <a href="http://www.genome.gov/" target="_blank">National Human 
Genome Research Institute</a> </p> 
<p>
<strong>Project Leadership:</strong> <a href="http://www.broadinstitute.org/" target="_blank">The 
Broad Institute</a>, Cambridge, MA, USA </p> 
<p>
<strong>Sequencing/Assembly:</strong> <a href="http://www.broadinstitute.org/" target="_blank">The
Broad Institute</a> - team led by by Kerstin Lindblad-Toh, Ph.D. </p> 
<p>
<strong>Plasmid and fosmid libraries:</strong> <a href="http://www.broadinstitute.org/" 
target="_blank">The Broad Institute</a> </p> 
<p>
<strong>BAC libraries:</strong> <a href="http://www.chori.org/" target="_blank">Children's 
Hospital Oakland Research Institute</a> (CHORI), Oakland, CA, USA </p> 
<p>
<strong>BAC end sequencing:</strong> <a href="http://www.beckmangenomics.com/" 
target="_blank">Agencourt Bioscience</a>, Danvers, MA, USA </p> 
<p>
<strong>RH map:</strong> <a href="http://www.cnrs.fr/index.php" target="_blank">Centre National de
la Recherche Scientifique</a> (CNRS), France </p> 
<p>
<strong>FISH mapping:</strong> <a href="https://www.ncsu.edu/" target="_blank">North Carolina 
State University</a> (NCSU), Raleigh, NC, USA </p> 
<p>
<strong>Dog Selection:</strong> <a href="http://www.fredhutch.org/en/labs.html" 
target="_blank">Fred Hutchinson Cancer Research Center</a> (FHCRC), Seattle, WA, USA </p> 
<p>
<strong>UCSC Dog Genome Browser/Annotations:</strong></p>
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=canFam6" target="_blank">canFam6</a>: Hiram Clawson and
  Jairo Navarro</li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=canFam5" target="_blank">canFam5</a>: Hiram Clawson and
  Lou Nassar </li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=canFam4" target="_blank">canFam4</a>: Hiram Clawson and
  Lou Nassar </li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=canFam3" target="_blank">canFam3</a>: Hiram Clawson and 
  Luvina Guruvadoo </li> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=canFam2" target="_blank">canFam2</a>: Angie Hinrichs, 
  Jennifer Jackson, and Donna Karolchik </li> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=canFam1" target="_blank">canFam1</a>: Angie Hinrichs, Robert
  Kuhn and Donna Karolchik </li> 
</ul> 
<a name="dog_use"></a> 
<h3>Data use</h3> 
<p>
The dog genome sequence is made freely available by the 
<a href="http://www.broadinstitute.org/mammals/dog" target="_blank">Dog Genome Sequencing 
Project</a>. Please cite this publication when using these data:</p> 
<p> 
Lindblad-Toh K, Wade CM, Mikkelsen TS, Karlsson EK, Jaffe DB, Kamal M, Clamp M, Chang JL, 
Kulbokas EJ 3rd, Zody MC <em>et al</em>.  
<a href="http://www.nature.com/nature/journal/v438/n7069/abs/nature04338.html" 
target="_blank">Genome sequence, comparative analysis and haplotype structure of the domestic 
dog</a>. <em>Nature</em>. 2005 Dec 8;438(7069):803-19. PMID: 16341006</p>

<!-- Dolphin Credits -->
<a name="dolphin_credits"></a>
<h2>Dolphin genome</h2> 
<p> 
<strong>Sequencing: </strong> 
<a href="https://www.hgsc.bcm.edu/marine-mammals/dolphin-genome-project" target="_blank">Baylor 
College of Medicine Human Genome Sequencing Center (BCM-HGSC)</a>, Houston, TX, USA </li> 
<p> 
<strong>Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p> 
<p> 
<strong>UCSC Dolphin Genome Browser/Annotations:</strong> </p>
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=turTru2" target="_blank">turTru2</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="dolphin_use"></a> 
<h3>Data use</h3> 
<p>
The dolphin sequence has been made freely available as part of the Mammalian Genome Project (29 
Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>.  <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The dolphin data are provided with the following understanding: 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul>

<!-- Elephant Credits -->
<a name="elephant_credits"></a>
<h2>Elephant genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> 
<a href="https://www.broadinstitute.org/elephant/elephant-genome-project/" target="_blank">The 
Broad Institute</a>, Cambridge, MA, USA </p> 
<p> 
<strong>UCSC Elephant Genome Browser/Annotations:</strong> </p>
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=loxAfr3" target="_blank">loxAfr3</a>: Hiram Clawson, 
  Pauline Fujita, Vanessa Swing, Antonio Coelho, and Donna Karolchik </li> 
</ul> 
<a name="elephant_use"></a>
<h3>Data use</h3> 
<p> 
The elephant sequence is made freely available before scientific publication with the following 
understanding: </p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The center producing the data reserves the right to publish the initial large-scale analyses of 
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a 
  chromosome (that is, 30 Mb or more).  <li> Any redistribution of the data should carry this 
  notice. </li> 
</ul>

<!-- Ferret Credits -->
<a name="ferret_credits"></a>
<h2>Ferret genome</h2> 
<p> 
<strong>Funding: </strong> <a href="https://www.niaid.nih.gov/" target="_blank">National Institute
of Allergy and Infectious Diseases</a> (NIAID), Genomic Sequencing Centers for Infectious Diseases
(GSCID) program </p>
<p>
<strong>Sequencing/Assembly:</strong> Ferret Genome Sequencing Consortium and 
<a href="http://www.broadinstitute.org/" target="_blank">The Broad Institute</a>, Cambridge, MA, 
USA </p> 
<p> 
<strong>UCSC Ferret Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=musFur1" target="_blank">musFur1</a>: Hiram Clawson, Brian 
  Lee, and Brooke Rhead </li>
</ul> 
<a name="ferret_use"></a>
<h3>Data use</h3> 
<p> 
The ferret sequence data may be freely downloaded, used in analyses, and repackaged in databases if 
the provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Peng X, Alf&ouml;ldi J, Gori K, Eisfeld AJ, Tyler SR, Tisoncik-Go J, Brawand D, Law GL, Skunca N, 
Hatta M <em>et al</em>.  <a href="http://www.nature.com/nbt/journal/v32/n12/full/nbt.3079.html" 
target="_blank">The draft genome sequence of the ferret (Mustela putorius furo) facilitates study of
human respiratory disease</a>.  <em>Nat Biotechnol</em>. 2014 Dec;32(12):1250-5. </p>

<!-- Garter snake Credits -->
<a name="garter_snake_credits"></a>
<h2>Garter snake genome</h2>
<p>
<strong>Sequencing/Assembly:</strong>
<a href="http://genome.wustl.edu/" target="_blank">The Genome Institute at Washington 
University School of Medicine</a> </p>
<p>
<strong>UCSC Garter Snake Genome Browser/Annotations:</strong>
<ul>
  <li>
  <a href="../../cgi-bin/hgGateway?db=aquChr2" target="_blank">thaSir1</a>: Jonathan Casper, Conner Powell </li>
</ul>
<a name="garter_snake_use"></a>
<h3>Data use</h3>
<p>
The garter snake sequence is made freely available to the community by the Genome Sequencing Center,
Washington University School of Medicine, with the following understanding:</p>
<ul>
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li>
  <li>
  Users are free to use the data in scientific papers analyzing particular genes and regions if the
  provider of these data is properly acknowledged. </li>
  <li>
  The center producing the data reserves the right to publish the initial large-scale analyses of
  the data set, including large-scale identification of regions of evolutionary conservation and
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a
  chromosome (that is, 30 Mb or more). </li>
  <li>
  Any redistribution of the data should carry this notice. </li>
</ul>



<!-- Gibbon Credits -->
<a name="gibbon_credits"></a>
<h2>Gibbon genome</h2> 
<p> 
<strong>Funding: </strong> <a href="http://www.genome.gov" target=_blank>National Human Genome 
Research Institute</a> (NHGRI) </p>
<p>
<strong>Sequencing/Assembly:</strong> </p>
<ul> 
  <li>
  Gibbon Genome Sequencing Consortium</li>
  <li>
  <a href="http://genome.wustl.edu/genomes/detail/nomascus-leucogenys/" target="_blank">The Genome
  Institute at Washington University (WUSTL)</a>, St. Louis, MO, USA </li> 
  <li> 
  <a href="https://www.hgsc.bcm.edu/non-human-primates/gibbon-genome-project" 
  target="_blank">Baylor College of Medicine Human Genome Sequencing Center (BCM-HGSC)</a>, 
  Houston, TX, USA </li> 
</ul> 
<p>
<strong>UCSC Gibbon Genome Browser/Annotations:</strong> </p>
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=nomLeu3" target="_blank">nomLeu3</a>: Pauline Fujita and
  Hiram Clawson </li> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=nomLeu2" target="_blank">nomLeu2</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=nomLeu1" target="_blank">nomLeu1</a>: Chin Li, Brooke Rhead,
  Steve Heitner, Lauren Scott and Donna Karolchik </li> 
</ul> 
<a name="gibbon_use"></a> 
<h3>Data use</h3> 
<p>
The gibbon sequence data may be freely downloaded, used in analyses, and repackaged in databases if 
the provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Carbone L, Harris RA, Gnerre S, Veeramah KR, Lorente-Galdos B, Huddleston J, Meyer TJ, Herrero J, 
Roos C, Aken B <em>et al</em>. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249732/" 
target="_blank">Gibbon genome and the fast karyotype evolution of small apes</a>.  <em>Nature</em>. 
2014 Sep 11;513(7517):195-201. </p>

<!-- Golden eagle Credits -->
<a name="golden_eagle_credits"></a>
<h2>Golden eagle genome</h2>
<p> 
<strong>Sequencing/Assembly:</strong> 
<a href="http://genome.wustl.edu/genomes/" target="_blank">The Genome Institute at Washington 
University School of Medicine</a> </p> 
<p>
<strong>UCSC Golden eagle Genome Browser/Annotations:</strong> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=aquChr2" target="_blank">aquChr2</a>: Hiram Clawson, Jairo 
  Navarro </li>
</ul> 
<a name="golden_eagle_use"></a> 
<h3>Data use</h3> 
<p>
The golden eagle sequence is made freely available to the community by the Genome Sequencing Center,
Washington University School of Medicine, with the following understanding:</p> 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The center producing the data reserves the right to publish the initial large-scale analyses of 
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a 
  chromosome (that is, 30 Mb or more). </li>
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Golden Snub-Nosed Monkey Credits -->
<a name="golden_snub-nosed_monkey_credits"></a>
<h2>Golden snub-nosed monkey genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> Novogene </p> 
<p>
<strong>UCSC Golden Snub-Nosed Monkey Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=rhiRox1" target="_blank">rhiRox1</a>: Hiram Clawson, Luvina 
  Guruvadoo </li>
</ul> 
<a name="golden_snub-nosed_monkey_use"></a> 
<h3>Data use</h3> 
<p>
The golden snub-nosed monkey sequence data may be freely downloaded, used in analyses, and 
repackaged in databases if the provider is properly acknowledged. Please cite this publication when 
using these data:</p> 
<p> 
Zhou X, Wang B, Pan Q, Zhang J, Kumar S, Sun X, Liu Z, Pan H, Lin Y, Liu G <em>et al</em>.  
<a href="https://www.nature.com/ng/journal/v46/n12/full/ng.3137.html" target="_blank">Whole-genome 
sequencing of the snub-nosed monkey provides insights into folivory and evolutionary history</a>.  
<em>Nat Genet.</em> 2014 Dec;46(12):1303-10.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/25362486" target="_blank">25362486</a> </p>

<!-- Gorilla Credit -->
<a name="gorilla_credits"></a>
<h2>Gorilla genome</h2> 
<p>
The UCSC <a href="../../cgi-bin/hgGateway?db=gorGor6">gorGor6</a> Genome Browser displays
data from the Aug 2019 assembly of from the University of Washington. </p>
<p>
<strong>Sequencing/Assembly:</strong> <a href="https://eichlerlab.gs.washington.edu/" 
target="_blank">University of Washington</a></p>
<p>
<strong>UCSC Gorilla Genome Browser/Annotations:</strong> </p>
<ul>
  <li>
  <a href="../../cgi-bin/hgGateway?db=gorGor6" target="_blank">gorGor6</a>: Hiram Clawson and
  Daniel Schmelter </li>
</ul>
<hr>


<p> 
The UCSC <a href="../../cgi-bin/hgGateway?db=gorGor5" target="_blank">gorGor5</a> Genome Browser 
displays data from the Mar 2016 assembly of <em>Gorilla gorilla gorilla</em> produced by the 
University of Washington with the following acknowledgments: </p> 
<p>
<strong>Sequencing/Assembly:</strong> University of Washington </p>
<p>
<strong>UCSC Gorilla Genome Browser/Annotations:</strong> </p>
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=gorGor5" target="_blank">gorGor5</a>: Hiram Clawson and 
  Jairo Navarro </li> 
</ul> 
<hr>
<p> 
The UCSC <a href="../../cgi-bin/hgGateway?db=gorGor4" target="_blank">gorGor4</a> and
<a href="../../cgi-bin/hgGateway?db=gorGor3" target="_blank">gorGor3</a> Genome Browsers 
display data produced by the Wellcome Trust Sanger Institute with the following acknowledgments:</p>
<p>
<strong>Sequencing/Assembly:</strong> 
<ul> 
  <li>
  The <a href="http://www.sanger.ac.uk/resources/downloads/gorilla/" target="_blank">Wellcome 
  Trust Sanger Institute</a>, Hinxton, Cambridge, UK </li> 
  <li>
  The <a href="http://www.ebi.ac.uk/" target="_blank">European Bioinformatics Institute</a> (EBI),
  Hinxton, Cambridge, UK </li> 
</ul> 
<p>
<strong>UCSC Gorilla Genome Browser/Annotations:</strong> </p>
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=gorGor4" target="_blank">gorGor4</a>: Jonathan Casper, Brian
  Raney, Christopher Lee </li> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=gorGor3" target="_blank">gorGor3</a>: Hiram Clawson, Mark 
  Diekhans, Brian Raney, Brooke Rhead, Steve Heitner, Lauren Scott and Donna Karolchik </li> 
</ul> 
<a name="gorilla_use"></a> 
<h3>Data use</h3> 
<p>
The gorilla sequence data may be freely downloaded, used in analyses, and repackaged in databases if
the provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Scally A, Dutheil JY, Hillier LW, Jordan GE, Goodhead I, Herrero J, Hobolth A, Lappalainen T, 
Mailund T, Marques-Bonet T <em>et al</em>.  
<a href="http://www.nature.com/nature/journal/v483/n7388/abs/nature10842.html" 
target="_blank">Insights into hominid evolution from the gorilla genome sequence</a>. 
<em>Nature</em>. 2012 Mar 7;483(7388):169-75.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22398555" target="_blank">22398555</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3303130/" target="_blank">PMC3303130</a></p> 
<p> 
Additionally, this publication discusses the gorGor5 assembly:</p> 
<p> 
Gordon D, Huddleston J, Chaisson MJ, Hill CM, Kronenberg ZN, Munson KM, Malig M, Raja A, Fiddes I, 
Hillier LW <em>et al</em>.  <a href="http://science.sciencemag.org/content/352/6281/aae0344.full" 
target="_blank">Long-read sequence assembly of the gorilla genome</a>. <em>Science</em>. 
2016 Apr 1;352(6281):aae0344.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27034376" 
target="_blank">27034376</a>; PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4920363/"
target="_blank">PMC4920363</a></p> 

<!-- Green Monkey Credits -->
<a name="green_monkey_credits"></a>
<h2>Green monkey genome</h2> 
<p> 
The green monkey  assembly is provided by the Vervet Genomics Consortium with the following 
acknowledgments: </p> 
<p>
<strong>Funding: </strong> <a href="http://www.genome.gov" target=_blank>National Human Genome 
Research Institute</a> (NHGRI) </p>
<p>
<strong>DNA source: </strong> Dr. Jay Kaplan, Wake Forest Primate Facility, Wake Forest, NC, 
USA </p> 
<p>
<strong>Genome Sequence: </strong> </p>
<ul> 
  <li>
  <a href="http://genome.wustl.edu/genomes/" target="_blank">Genome Institute, Washington 
  University School of Medicine</a>, St. Louis, MO, USA </li> 
  <li> 
  <a href="https://www.mcgill.ca/humangenetics/" target="_blank">Department of Human Genetics, 
  McGill University</a>, Montreal, Canada </li> 
</ul> 
<p>
<strong>BAC Library: </strong> Dr. Pieter DeJong, <a href="http://www.chori.org/" 
target="_blank">CHORI</a>, Oakland, CA, USA </p> 
<p>
<strong>Assembly Curation: </strong> </p>
<ul> 
  <li>
  Jessica Wasserscheid, Nikoleta Juretic, Dr. Ken Dewar - 
  <a href="http://www.mcgillgenomecentre.org/" target="_blank">McGill University</a>, Montreal, 
  QC Canada </li> 
  <li>
  LaDeana Hillier - <a href="http://genome.wustl.edu/genomes/" target="_blank">Genome Institute, 
  Washington University School of Medicine</a>, St. Louis, MO, USA </li> 
</ul> 
<p>
<strong>FISH Mapping Data: </strong>  Mariano Rocchi, <a href="http://www.biologia.uniba.it/rmc/" 
target="_blank">Department of Biology, University of Bari</a>, Bari, Italy </p>
<p>
<strong>cDNA data - RNA sources: </strong> Dr.  Nelson Freimer - 
<a href="https://www.semel.ucla.edu/" target="_blank">Semel Institute for Neuroscience and Human 
Behavior, University of California Los Angeles</a>, CA, USA </p> 
<p>
<strong>UCSC Green Monkey Genome Browser/Annotations:</strong></p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=chlSab2" target="_blank">chlSab2</a>: Hiram Clawson, Brian 
  Raney, Christopher Lee </li>
</ul> 
<a name="green_monkey_use"></a> 
<h3>Data use</h3> 
<p>
The green monkey sequence is made freely available before scientific publication with the following 
understanding: </p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The center producing the data reserves the right to publish the initial large-scale analyses of   
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a chromosome
  (that is, 30 Mb or more). </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Guinea Pig Credits -->
<a name="guinea_pig_credits"></a>
<h2>Guinea pig genome</h2> 
<p> 
<strong>Funding:</strong> <a href="http://www.genome.gov/" target="_blank">National Human Genome 
Research Institute</a> </p> 
<p>
<strong>Sequencing/Assembly:</strong> <a href="http://www.broadinstitute.org/" target="_blank">The
Broad Institute</a>, Cambridge, MA, USA </p>
<!-- 
<p>
<strong>BAC End Reads:</strong> <a href="http://www.tiho-hannover.de/index_e.htm" 
target="_blank">University of Veterinary Medicine Hanover</a>, Germany, and the 
<a href="http://www.helmholtz-hzi.de/en/" target="_blank">Helmholtz Centre for Infection 
Research</a>, Braunschweig, Germany </p>
--> 
<p>
<strong>UCSC Guinea Pig Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=cavPor3" target="_blank">cavPor3</a>: Tim Dreszer, Kate 
  Rosenbloom, Hiram Clawson, Kayla Smith, Robert Kuhn, and Donna Karolchik </li> 
</ul> 
<a name="guinea_pig_use"></a>
<h3>Data use</h3> 
<p> 
The guinea pig sequence is made freely available before scientific publication with the following 
understanding: </p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The center producing the data reserves the right to publish the initial large-scale analyses of   
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a chromosome
  (that is, 30 Mb or more). </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Hawaiian monk seal Credits -->
<a name="hawaiian_monk_seal_credits"></a>
<h2>Hawaiian monk seal genome</h2>
<p>
<strong>Sequencing/Assembly: </strong>
<a href="https://www.jhu.edu/" 
target="_blank">Johns Hopkins University</a>, Baltimore, MA, USA </p>
<p>
<strong>UCSC Hawaiian monk seal Genome Browser/Annotations:</strong> </p>
<ul>
  <li>
  <a href="../../cgi-bin/hgGateway?db=neoSch1" target="_blank">neoSch1</a>: Hiram Clawson, Robert Kuhn, and Gerardo Perez</li>
</ul>
<a name="hawaiian_monk_seal_use"></a>
<h3>Data use</h3>
<p>
The Hawaiian monk seal sequence data may be freely downloaded, used in analyses, and repackaged in databases if
the provider is properly acknowledged: Alan F. Scott at Johns Hopkins University.</p>

<p>
Mohr DW, Naguib A, Weisenfeld NI, Kumar V, Shah P, Church DM, Jaffe D, Scott AF. 
 <a href="https://doi.org/10.1101/128348" 
target="_blank">Improved de Novo Genome Assembly: Linked-Read Sequencing Combined with Optical Mapping Produce a 
High Quality Mammalian Genome at Relatively Low Cost</a>. <em>BioRxiv</em>. 2017 April 18 (preprint).</p>

<!-- Hedgehog Credits -->
<a name="hedgehog_credits"></a>
<h2>Hedgehog genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a> at MIT and Harvard, Cambridge, MA, USA </p> 
<p>
<strong>DNA Samples: </strong> Dr. Neil J. Gemmell - 
<a href="http://www.otago.ac.nz/anatomy/research/r-g-d/index.html" target="_blank">Reproduction, 
Genomics and Development Group, University of Otago</a>, Dunedin, New Zealand </p> 
<p> 
<strong>UCSC Hedgehog Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=eriEur2" target="_blank">eriEur2</a>: Hiram Clawson, Brian 
  Raney and Steve Heitner </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=eriEur1" target="_blank">eriEur1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="hedgehog_use"></a>
<h3>Data use</h3> 
<p>
The hedgehog sequence has been made freely available as part of the Mammalian Genome Project (29 
Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>.  <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The hedgehog data are provided with the following understanding: 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul>

<!-- Horse Credits -->
<a name="horse_credits"></a>
<h2>Horse genome</h2> 
<p>The UCSC equCab3 Genome Browser displays data produced by the University of Louisville:
<ul>
 <li><a href="../cgi-bin/hgGateway?db=equCab3" target="_blank">equCab3</a> browser: Jan. 2018
  University of Louisville EquCab3.0
 </li>
</ul>

<p><strong>Sequencing/Assembly:</strong>
  <a href="https://louisville.edu/medicine/news/researchers-fill-gaps-in-horse-reference-genome-to-guide-new-approaches-in-fighting-disease" 
  target="_blank">University of Louisville</a>, Louisville, KY, USA 
<p>

<strong>UCSC Horse Genome Browser/Annotations:
<ul>
 <li> 
  <a href="../../cgi-bin/hgGateway?db=equCab3" 
  target="_blank">(equCab3)</a>:</strong> Hiram Clawson and Conner Powell</li>
</ul>

</p>

<hr>
<p> 
<strong>Funding:</strong> The <a href="http://www.genome.gov/" target="_blank">National Human 
Genome Research Institute</a> </p> 
<p>
<strong>Project Leadership:</strong> <a href="http://www.broadinstitute.org/" target="_blank">The 
Broad Institute</a>, Cambridge, MA, USA </p>
<p><strong>Sequencing/Assembly:</strong> <a href="http://www.broadinstitute.org/" 
target="_blank">The Broad Institute</a> </p>
<p>
<strong>BAC End Reads:</strong> <a href="http://www.tiho-hannover.de/index_e.htm" 
target="_blank">University of Veterinary Medicine Hanover</a>, Germany, and the 
<a href="http://www.helmholtz-hzi.de/en/" target="_blank">Helmholtz Centre for Infection 
Research</a>, Braunschweig, Germany </p>
<p>
<strong>UCSC Horse Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=equCab2" target="_blank">equCab2</a>: Larry Meyer, Hiram 
  Clawson, Pauline Fujita, Brooke Rhead, and Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=equCab1" target="_blank">equCab1</a>: Fan Hsu, Brooke Rhead,
  Robert Kuhn, Hiram Clawson, Angie Hinrichs, Kate Rosenbloom, and Donna Karolchik </li> 
</ul> 
<a name="horse_use"></a>
<h3>Data use</h3> 
<p> 
The horse sequence data may be freely downloaded, used in analyses, and repackaged in databases if 
the provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p>
Theodore S. Kalbfleisch, Edward S. Rice, Michael S. DePriest Jr., Brian P. Walenz, Matthew S.
Hestand <em>et al</em>. <a href="https://www.nature.com/articles/s42003-018-0199-z" 
target="_blank">Improved reference genome for the domestic horse increases assembly contiguity and
composition</a>. <em>Nature, Commincations Biology</em>. 2018 Nov 16;197:2399-3642 </p> 

<p> 
Wade CM, Giulotto E, Sigurdsson S, Zoli M, Gnerre S, Imsland F, Lear TL, Adelson DL, Bailey E, 
Bellone RR <em>et al</em>.  <a href="http://www.sciencemag.org/content/326/5954/865.abs" 
target="_blank">Genome sequence, comparative analysis, and population genetics of the domestic 
horse</a>. <em>Science</em>. 2009 Nov 6;326(5954):865-7. </p>

<!-- Kangaroo Rat Credits -->
<a name="kangaroo_rat_credits"></a>
<h2>Kangaroo rat genome</h2> 
<p> 
<strong>Sequencing: </strong> 
<a href="https://www.hgsc.bcm.edu/others-mammals/kangaroo-rat-genome-project" 
target="_blank">Baylor College of Medicine Human Genome Sequencing Center (BCM-HGSC)</a>, Houston,
TX, USA </p> 
<p>
<strong>Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p> 
<p> 
<strong>UCSC Kangaroo Rat Genome Browser/Annotations:</strong></p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=dipOrd1" target="_blank">dipOrd1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="kangaroo_rat_use"></a>
<h3>Data use</h3> 
<p> 
The kangaroo rat sequence has been made freely available as part of the Mammalian Genome Project 
(29 Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>.  <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The kangaroo rat data are provided with the following understanding: 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul>

<!-- Malayan flying lemur Credits -->
<a name="malayan_flying_lemur_credits"></a>
<h2>Malayan flying lemur genome</h2> 
<p> 
<strong>Funding: </strong> National Institutes of Health (NIH) </p>
<p>
<strong>Assembly/Sequencing: </strong> 
<a href="http://genome.wustl.edu/genomes/detail/galeopterus-variegatus/" target="_blank">The 
Genome Institute, Washington University School of Medicine</a>, St. Louis, MO, USA </p> 
<p>
<strong>DNA Source: </strong> William Murphy - <a href="http://vetmed.tamu.edu/vibs/" 
target="_blank">VIBS, Texas A&M University</a>, College Station, TX, USA </p> 
<p>
<strong>UCSC Malayan Flying Lemur Genome Browser/ Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=galVar1" target="_blank">galVar1</a>: Hiram Clawson and 
  Luvina Guruvadoo </li>
</ul> 
<a name="malayan_flying_lemur_use"></a>
<h3>Data use</h3> 
<p> 
The Malayan flying lemur sequence may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged: Richard K. Wilson and The Genome Institute,
Washington University School of Medicine. </p>

<!-- Manatee Credits -->
<a name="manatee_credits"></a>
<h2>Manatee genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p> 
<p> 
<strong>UCSC Manatee Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=triMan1" target="_blank">triMan1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="manatee_use"></a>
<h3>Data use</h3> 
<p> 
The manatee sequence data may be freely downloaded, used in analyses, and repackaged in databases if
the provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Foote AD, Liu Y, Thomas GW, Vina&rcaron; T, Alf&ouml;ldi, Deng J, Dugan S, van Elk CE, Hunter ME, 
Joshi V <em>et al</em>.  <a href="https://www.nature.com/ng/journal/v47/n3/full/ng.3198.html" 
target="_blank">Convergent evolution of the genomes of marine mammals</a>. <em>Nat Genet</em>. 
2015 Mar;47(3):272-5. PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25621460" 
target="_blank">25621460</a>; PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4644735/"
target="_blank">PMC4644735</a></p>

<!-- Marmoset Credits -->
<a name="marmoset_credits"></a>
<h2>Marmoset genome</h2>
<p>
The UCSC <a href="../../cgi-bin/hgGateway?db=calJac4">calJac4</a> Genome Browser displays data from
the May 2020 assembly of white-tufted-ear marmoset produced by the McDonnell Genome Institute at
Washington University with the following acknowledgments:</p>
<p>
<strong>Sequencing and assembly:</strong> 
<a href="http://genome.wustl.edu/" target="_blank">McDonnell Genome Institute at Washington
University</a></p>
<p>
<strong>DNA Source:</strong> Wisconsin Primate Research Center</p>
<p>
<strong>UCSC Marmoset Genome Browser/Annotations:</strong>
<ul>
  <li><a href="../../cgi-bin/hgGateway?db=calJac4">calJac4</a>: Hiram Clawson and Jairo Navarro</li>
</ul>
</p>

<hr>

<p>
<strong>Sequencing and assembly:</strong> 
<a href="http://genome.wustl.edu/genomes/detail/callithrix-jacchus/" target="_blank">The Genome 
Institute at Washington University (WUSTL)</a>, St. Louis, MO, USA and 
<a href="https://www.hgsc.bcm.edu/" target="_blank">Baylor College of Medicine Human Genome 
Sequencing Center (BCM-HGSC)</a>, Houston, TX, USA </p> 
<p> 
<strong>DNA Source:</strong> Dr. Suzette Tardif, 
<a href="http://www.txbiomed.org/primate-research-center/" target="_blank">Southwest National 
Primate Research Center</a>, San Antonio, TX, USA </p> 
<p> 
<strong>CHORI-259 BAC library:</strong> Dr. Pieter DeJong, <a href="http://www.chori.org/" 
target="_blank">Children's Hospital Oakland Research Institute</a>, Oakland, CA, USA </p> 
<p> 
<strong>UCSC Marmoset Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=calJac3" target="_blank">calJac3</a>: Hiram Clawson, Chin 
  Li, Mark Diekhans, Mary Goldman, Vanessa Swing, and Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=calJac1" target="_blank">calJac1</a>: Hiram Clawson, Robert 
  Kuhn, Pauline Fujita, Brooke Rhead, and Donna Karolchik </li> 
</ul> 
<a name="marmoset_use"></a> 
<h3>Data use</h3> 
<p> 
These data were produced by The Genome Institute at WUSTL and the BCM-HGSC and can be downloaded 
<a href="http://genome.wustl.edu/pub/organism/Primates/Callithrix_jacchus/assembly/" 
target="_blank">here</a>.</p> 
<p> 
The marmoset sequence data may be freely downloaded, used in analyses, and repackaged in databases 
if the provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Marmoset Genome Sequencing and Analysis Consortium. 
<a href="https://www.nature.com/ng/journal/v46/n8/full/ng.3042.html" target="_blank">The common 
marmoset genome provides insight into primate biology and evolution</a>. <em>Nat Genet</em>. 
2014 Aug;46(8):850-7.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25038751" 
target="_blank">25038751</a>; PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4138798/"
target="_blank">PMC4138798</a></p>

<p>
Additionally, this publication discusses the calJac4 assembly:</p>
<p>
Wang W, Liu JY, Wang HF, Yang MY, Liu QY, Ding MX.
<a href="https://pubmed.ncbi.nlm.nih.gov/25329280/" target="_blank">The complete mitochondrial
genome of white-tufted-ear marmoset, Callithrix jacchus (Primates: Callitrichinae)</a>.
<em>Mitochondrial DNA A DNA Mapp Seq Anal</em>. 2016 May;27(3):1920-1.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25329280" target="_blank">25329280</a>
doi: <a href="https://doi.org/10.3109/19401736.2014.971287"
target="_blank">10.3109/19401736.2014.971287</a>

<!-- Megabat Credits -->
<a name="megabat_credits"></a>
<h2>Megabat genome</h2> 
<p> 
<strong>Funding:</strong> <a href="http://www.genome.gov" target="_blank">National Human Genome 
Research Institute</a> (NHGRI) </p>
<p>
<strong>Sequencing/Assembly: </strong> </p>
<ul> 
  <li>
  <a href="https://www.hgsc.bcm.edu/other-mammals/megabat-genome-project" target="_blank">Baylor 
  College of Medicine Human Genome Sequencing Center (BCM-HGSC)</a>, Houston, TX, USA </li> 
  <li>
  <a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
  target="_blank">The Broad Institute</a>, Cambridge, MA, USA </li> 
</ul> 
<p> 
<strong>UCSC Megabat Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=pteVam1" target="_blank">pteVam1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik</li> 
</ul> 
<a name="megabat_use"></a>
<h3>Data use</h3> 
<p> 
The megabat sequence has been made freely available as part of the Mammalian Genome Project (29 
Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>.  <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The megabat data are provided with the following understanding: </p>
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul>

<!-- Microbat Credits -->
<a name="microbat_credits"></a>
<h2>Little brown bat genome</h2>
<p> 
<strong>Sequencing/Assembly: </strong><a href="http://www.broadinstitute.org/" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p> 
<p> 
<strong>UCSC Little brown bat Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=myoLuc2" target="_blank">myoLuc2</a>: Chin Li, Brian Raney, 
  Robert Kuhn, Steve Heitner, Brooke Rhead, Greg Roe and Donna Karolchik </li> 
</ul> 
<a name="megabat_use"></a>
<h3>Data use</h3> 
<p> 
The Little brown bat sequence has been made freely available as part of the Mammalian Genome Project (29 
Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>.  <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The Little brown bat data are provided with the following understanding: 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul> 

<!-- Minke Whale Credits -->
<a name="minke_whale_credits"></a>
<h2>Minke whale genome</h2> 
<p> 
Sequencing/Assembly: <a href="http://www.kiost.ac.kr/eng.do" target="_blank">Korea Institute
of Ocean Science and Technology</a>, Ansan, Republic of Korea </p>
<p> 
<strong>UCSC Minke Whale Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=balAcu1" target="_blank">balAcu1</a>: Hiram Clawsom and 
  Matthew Speir </li>
</ul> 
<a name="minke_whale_use"></a>
<h3>Data use</h3> 
<p> 
The minke whale genome data may be freely downloaded, used in analyses, and repackaged in databases 
if the data provider is properly acknowledged. Please cite this publication when using these 
data:</p> 
<p> 
Yim HS, Cho YS, Guang X, Kang SG, Jeong JY, Cha SS, Oh HM, Lee JH, Yang EC, Kwon KK <em>et al</em>. 
<a href="http://www.nature.com/ng/journal/v46/n1/full/ng.2835.html" target="_blank">Minke whale 
genome and aquatic adaptation in cetaceans</a>.  <em>Nat Genet</em>. 2014 Jan;46(1):88-92. 
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24270359" target="_blank">24270359</a> </p>

<!-- Mouse Credits -->
<a name="mouse_credits"></a>
<h2>Mouse genome</h2> 
<p> 
Starting with <a href="../../cgi-bin/hgGateway?db=mm10" target="_blank">mm10</a> (GRCm38), the 
mouse genome assembly is now provided by the 
<a href="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/" target="_blank">Genome 
Reference Consortium</a> (GRC). See the 
<a href="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/mouse/" target="blank">GRC Mouse 
Genome</a> web pages for acknowledgments.</p> 
<p>
<strong>UCSC Mouse Genome Browser (<a href="../../cgi-bin/hgGateway?db=mm10" 
target="_blank">mm10</a>):</strong> Hiram Clawson and Luvina Guruvadoo
<br>
<strong>UCSC Mouse Genome Browser (<a href="../../cgi-bin/hgGateway?db=mm39" 
target="_blank">mm39</a>):</strong> Hiram Clawson and Jairo Navarro </p> 
<hr>
<p> 
Prior to mm10/GRCm38, mouse assemblies were produced by the Mouse Genome Sequencing Consortium, with
these acknowledgments:</p> 
<p>
<strong>Funding:</strong> </p>
<ul> 
  <li>
  <a href="http://www.nih.gov/" target="_blank">National Institutes of Health</a></li> 
  <li>
  <a href="http://www.wellcome.ac.uk/" target="_blank">The Wellcome Trust</a> </li>
  <li>
  <a href="http://www.gsk.com/" target="_blank">GlaxoSmithKline</a> </li>
  <li>
  <a href="http://www.merck.com/" target="_blank">The Merck Genome Research Institute</a> </li>
  <li>
  <a href="http://www.affymetrix.com/estore/" target="_blank">Affymetrix</a></li> 
</ul> 
<p>
<strong>Sequencing:</strong> </p> 
<ul> 
  <li>
  <a href="http://www.broadinstitute.org/" target="_blank">The Broad Institute's Arachne 
  Group</a> </li>
  <li>
  <a href="http://www.sanger.ac.uk" target="_blank">The Wellcome Trust Sanger Institute</a> </li>
  <li>
  <a href="http://genome.wustl.edu/genomes/detail/mus-musculus/" target="_blank">The Genome 
  Institute at Washington University (WUSTL)</a> </li>
  <li>
  <a href="https://www.hgsc.bcm.edu/other-mammals/mouse-genome-project" target="_blank">Baylor 
  College of Medicine Human Genome Sequencing Center (BCM-HGSC)</a> </li>
  <li>
  <a href="http://www.ucl.ac.uk/wibr" target="_blank">Wolfson Institute for Biomedical 
  Research</a> </li> 
  <li>
  <a href="http://www.genome.ou.edu/" target="_blank">University of Oklahoma, Advanced Center for 
  Genome Technology</a> </li> 
  <li>
  <a href="http://www.har.mrc.ac.uk/" target="_blank">UK MRC Mouse Genome Centre and Mammalian 
  Genetics Unit </li>
  <li>
  <a href="http://www.nisc.nih.gov/" target="_blank">National Institutes of Health Intramural 
  Sequencing Center</a> </li> 
  <li>
  <a href="http://personalizedmedicine.partners.org/" target="_blank">Harvard Partners Genome 
  Center</a> </li>
  <li>
  <a href="http://www.cshl.edu/" target="_blank">Cold Springs Harbor Laboratory</a> </li>
</ul> 
<p>
<strong>Physical Map:</strong> </p> 
<ul> 
  <li>
  Mouse BAC libraries: <a href="http://bacpac.chori.org/" target="_blank">Children's Hospital 
  Oakland Research Institute</a> (CHORI) </li> 
  <li>
  BAC fingerprint map: <a href="http://www.bcgsc.ca/" target="_blank">Genome Sciences 
  Centre</a> </li> 
  <li>
  BAC end sequences: <a href="http://www.jcvi.org/cms/home/" target="_blank">The Institute of 
  Genome Research</a> (TIGR) </li> 
  <li>
  TPF production - Tina Graves, April Cook, Christine Burton, Darren Graham, Shannon Dugan, Donna 
  Muzny </li> 
</ul> 
<p>
<strong>Assembly:</strong> Deanna Church, Richa Agarwala, Josh Cherry, Mike DiCuccio, Hsiu-Chuan 
Chen, Paul Kitts, Victor Sapojnikov </p> 
<p>
<strong>cDNA Sequencing:</strong> </p>
<ul> 
  <li>
  <a href="http://genome.gsc.riken.go.jp/" target="_blank">Riken Institute</a> </li>
  <li>
  <a href="http://mgc.nci.nih.gov/" target="_blank">The Mammalian Gene Collection</a> </li>
</ul> 
<p>
<strong>UCSC Mouse Genome Browser:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=mm9" target="_blank">mm9</a>: Hiram Clawson, Archana 
  Thakkapallayil, Robert Kuhn and Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=mm8" target="_blank">mm8</a>: Hiram Clawson, Fan Hsu, Kayla 
  Smith, Ann Zweig, Robert Kuhn, Brooke Rhead, Archana Thakkapallayil, and Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=mm7" target="_blank">mm7</a>: Hiram Clawson, Fan Hsu, Ann 
  Zweig, Kayla Smith, Robert Kuhn and Donna Karolchik </li> 
  <li> 
  mm6 (archived): Hiram Clawson, Fan Hsu, Jennifer Jackson, Robert Kuhn, Ali Sultan-Qurraie, 
  Heather Trumbower and Donna Karolchik </li>
</ul> 
<p>
<strong>Annotations:</strong> </p>
<ul> 
  <li>
  <a href="../staff.html" target="_blank">UCSC Genome Bioinformatics Group</a> </li>
  <li>
  <a href="http://www.jax.org/" target="_blank">The Jackson Laboratory</a> </li>
  <li>
  <a href="https://www.ncbi.nlm.nih.gov/" target="_blank">NCBI</a> </li> 
  <li>
  <a href="http://www.ensembl.org/" target="_blank">Ensembl</a> </li> 
  <li>
  <a href="http://www.bx.psu.edu/miller_lab/" target="_blank">Pennsylvania State 
  University</a> </li> 
  <li>
  <a href="http://www.lbl.gov/" target="_blank">University of California Berkeley/LBNL</a> </li> 
</ul> 
<hr>
<p> 
The initial 3X coverage of the mouse genome was produced by the Mouse Sequencing Consortium, with 
the following acknowledgments: </p> 
<p>
<strong>Project Oversight:</strong> Arthur Holden and Francis Collins </p>
<p>
<strong>Funding:</strong> </p>
<ul> 
  <li>
  <a href="http://www.gsk.com/" target="_blank">GlaxoSmithKline</a> </li>
  <li>
  <a href="http://www.mgri.org/home.html" target="_blank">Merck Genome Research Institute</a></li>
  <li>
  <a href="http://www.affymetrix.com/estore/" target="_blank">Affymetrix, Inc.</a> </li>
  <li>
  <a href="http://www.wellcome.ac.uk/" target="_blank">The Wellcome Trust</a> </li>
  <li>
  <a href="http://www.nih.gov/" target="_blank">The National Institutes of Health</a> </li> 
</ul> 
<p>
<strong>Sequencing:</strong> </p>
<ul> 
  <li>
  <a href="http://www.broadinstitute.org/" target="_blank">The Broad Institute</a>, Cambridge, MA,
  USA </li>
  <li>
  <a href="http://genome.wustl.edu/genomes/detail/mus-musculus/" target="_blank">The Genome 
  Institute at Washington University (WUSTL)</a> </li>
  <li>
  <a href="http://www.sanger.ac.uk" target="_blank">The Wellcome Trust Sanger Institute</a> </li> 
</ul> 
<a name="mouse_use"></a> 
<h3>Data use</h3> 
<p>
The mouse genome data may be freely downloaded, used in analyses, and repackaged in databases if the
data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Mouse Genome Sequencing Consortium. <a href="http://www.nature.com/nature/mousegenome/" 
target="_blank">Initial sequencing and comparative analysis of the mouse genome</a>. 
<em>Nature</em>. 2002 Dec 5;420(6915):520-62.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/12466850" target="_blank">12466850</a> </p> 
<p> 
For more information about mouse assemblies provided by the GRC, see the 
<a href="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/mouse/" target="_blank">GRC mouse 
genome</a> web pages. For information about assemblies prior to mm10/GRCm38, see the Wellcome Trust 
Sanger Institute <a href="http://www.sanger.ac.uk/science/collaboration/mouse-resource-portal" 
target="_blank">Mouse Resource Portal</a>.</p>

<!-- Mouse Lemur Credits -->
<a name="mouse_lemur_credits"></a>
<h2>Mouse lemur genome</h2> 
<p> 
<strong>Sequep/Assembly: </strong> <p>
<ul> 
  <li>
  <a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
  target="_blank">The Broad Institute</a> at MIT and Harvard, Cambridge, MA, USA </li> 
  <li>
  <a href="https://www.hgsc.bcm.edu/non-human-primates/mouse-lemur-genome-project" 
  target="_blank">Baylor College of Medicine Human Genome Sequencing Center</a> (BCM-HGSC), 
  Houston, TX, USA </li> 
</ul> 
<p> 
<strong>UCSC Mouse Lemur Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=micMur2" target="_blank">micMur2</a>: Hiram Clawson, Brian 
  Raney, Christopher Lee </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=micMur1" target="_blank">micMur1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="mouse_lemur_use"></a>
<h3>Data use</h3> 
<p>
The mouse lemur sequence has been made freely available as part of the Mammalian Genome Project 
(29 Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>.  <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The mouse lemur data are provided with the following understanding: 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul>

<!-- Naked Mole-rat Credits-->
<a name="naked_mole-rat_credits"></a>
<h2>Naked mole-rat genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> <a href="http://www.genomics.cn/en/index" 
target="_blank">Beijing Genomics Institute</a>, Shenzhen, China </p>
<p> 
<strong>UCSC Naked Mole-rat Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=hetGla2" target="_blank">hetGla2</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=hetGla1" target="_blank">hetGla1</a>: Chin Li and Luvina 
  Guruvadoo </li>
</ul> 
<a name="naked_mole-rat_use"></a> 
<h3>Data use</h3> 
<p>
The naked mole-rat genome data may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite this publication when using 
these data:</p> 
<p>
Keane M, Craig T, Alf&ouml;ldi J, Berlin AM, Johnson J, Seluanov A, Gorbunova V, Di Palma F, 
Lindblad-Toh K, Church GM <em>et al</em>. 
<a href="http://bioinformatics.oxfordjournals.org/content/30/24/3558.long" target="_blank">The 
Naked Mole Rat Genome Resource: facilitating analyses of cancer and longevity-related 
adaptations</a>. <em>Bioinformatics</em>. 2014 Dec 15;30(24):3558-60. </p>

<!-- Opossum Credits -->
<a name="opossum_credits"></a>
<h2>Opossum genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://www.broadinstitute.org/mammals/opossum" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p>
<p>
<strong>FISH Mapping: </strong> <a href="https://www.ncsu.edu/" target="_blank">North Carolina 
State University</a> (NCSU), Raleigh, NC, USA </p> 
<p>
<strong>UCSC Opossum Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=monDom5" target="_blank">monDom5</a>: Andy Pohl, Hiram 
  Clawson, Brian Raney, Mark Diekhans, Brooke Rhead and Katrina Learned </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=monDom4" target="_blank">monDom4</a>: Hiram Clawson, 
  Archana Thakkapallayil, Ann Zweig, Kayla Smith and Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=monDom1" target="_blank">monDom1</a>: Hiram Clawson, Galt 
  Barber, Ali Sultan-Qurraie, Brian Raney and Donna Karolchik </li> 
</ul> 
<a name="opossum_use"></a> 
<h3>Data use</h3> 
<p>
The opossum genome data may be freely downloaded, used in analyses, and repackaged in databases if 
the data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Mikkelsen TS, Wakefield MJ, Aken B, Amemiya CT, Chang JL, Duke S, Garber M, Gentles AJ, 
Goodstadt L, Heger A <em>et al</em>. 
<a href="http://www.nature.com/nature/journal/v447/n7141/full/nature05805.html" 
target="_blank">Genome of the marsupial Monodelphis domestica reveals innovation in non-coding 
sequences</a>. <em>Nature</em>. 2007 May 10;447(7141):167-77.</p>

<!-- Orangutan Credits -->
<a name="orangutan_credits"></a>
<h2>Orangutan genome</h2>
<p> 
<strong>Sequencing:</strong> </p>
<ul> 
  <li>
  <a href="http://genome.wustl.edu/genomes/" target="_blank">The Genome 
  Institute at Washington University</a> (WUSTL), St. Louis, MO, USA </li> 
  <li> 
  <a href="https://www.hgsc.bcm.edu/non-human-primates/orangutan-genome-project" 
  target="_blank">Baylor College of Medicine Human Genome Sequencing Center</a> (BCM-HGSC), 
  Houston, TX, USA </li>
  <li> 
  <a href="https://eichlerlab.gs.washington.edu/" target="_blank">University of Washington</a>,
Seattle, WA,
  USA</li>
</ul> 
<p>
<strong>Sequence Assembly and Chromosomal Sequence/AGP Construction:</strong> The Genome Institute
at WUSTL </p>
<p>
<strong>DNA Source:</strong> 
<ul>
 <li>Dr. Greely Stones, <a href="http://www.gpz.org/" 
 target="_blank">Gladys Porter Zoo</a>, Brownsville, TX, USA</li>
 <li> Susie_PABv2, <a href="https://eichlerlab.gs.washington.edu/" target="_blank">Eichler Lab</a>,
 Seattle, WA, USA</li>
</ul>
</p> 
<p>
<strong>BAC Library:</strong> Yuko Yoshonaga in Pieter de Jong's laboratory, 
<a href="http://bacpac.chori.org" target="_blank">Children's Hospital Oakland Research 
Institute</a>, Oakland, CA, USA </p> 
<p>
<strong>Fosmid Library:</strong> The Genome Institute at WUSTL </p>
<p>
<strong>Fingerprint Map:</strong> The Genome Institute at WUSTL </p>
<p>
<strong>Cytogenetic Mapping and Human/Orangutan Breakpoint Analyses:</strong> Mariano Rocchi, 
<a href="http://www.biologia.uniba.it/DIGEMI/" target="_blank">Department of Genetics and 
Microbiology</a>, University of Bari, Bari, Italy <p> 
<p>
<strong>Fosmid End Placement against the Human Genome</strong> (used for breakpoint/inversion 
analyses during AGP construction): Lin Chen and Evan Eichler, Department of Genome Sciences, 
University of Washington, Seattle, WA, USA </p>
<p>
<strong>Funding: </strong> National Human Genome Research Institute (NHGRI), National Institutes 
of Health (NIH) </p>
<p>
<strong>UCSC Orangutan Genome Browser/Annotations:</strong> </p>
<ul>
   <li>
   <a href="../../cgi-bin/hgGateway?db=ponAbe3" target="_blank">ponAbe3</a>: Hiram Clawson, Conner
   Powell</li> 
   <li> 
   <a href="../../cgi-bin/hgGateway?db=ponAbe2" target="_blank">ponAbe2</a>: Hiram Clawson, Kayla 
   Smith, Robert Kuhn, Ann Zweig, and Donna Karolchik </li>
</ul> 
<a name="orangutan_use"></a> 
<h3>Data use</h3> 
<p>
The orangutan genome data may be freely downloaded, used in analyses, and repackaged in databases 
if the data provider is properly acknowledged. Please cite this publication when using these 
data:</p> 
<p> 
Locke DP, Hillier LW, Warren WC, Worley KC, Nazareth LV, Muzny DM, Yang SP, Wang Z, Chinwalla AT, 
Minx P <em>et al</em>. 
<a href="http://www.nature.com/nature/journal/v469/n7331/full/nature09687.html" 
target="_blank">Comparative and demographic analysis of orang-utan genomes</a>. <em>Nature</em>. 
2011 Jan 27;469(7331):529-33.
</p>
<p>

Kronenberg ZN, Fiddes IT, Gordon D, Murali S, Dutcher SK, Gage FH, Warren WC, Shendure J, Haussler
D, Schneider VA, Cao H, Ventura M, Wilson RK, Paten B, Pollen A, Eichler EE,<em> et al</em>.
<a href="http://science.sciencemag.org/content/360/6393/eaar6343.long" target="_blank">
High-resolution comparative analysis of great ape genomes.</a> <em>Science</em> 2018 Jun
8;360(6393).


 </p>

<!-- Panda Credits -->
<a name="panda_credits"></a>
<h2>Panda genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://www.genomics.cn/en/index" 
target="_blank">The Beijing Genomics Institute at Shenzhen</a> </p>
<p> 
<strong>UCSC Panda Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=ailMel1" target="_blank">ailMel1</a>: Hiram Clawson, Ann 
  Zweig, and Antonio Coelho </li> 
</ul> 
<a name="panda_use"></a> 
<h3>Data use</h3> 
<p>
The panda genome data may be freely downloaded, used in analyses, and repackaged in databases if the
data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Li R, Fan W, Tian G, Zhu H, He L, Cai J, Huang Q, Cai Q, Li B, Bai Y, Zhang Z <em>et al</em>.  
<a href="http://www.nature.com/nature/journal/v463/n7279/full/nature08696.html" 
target="_blank">The sequence and de novo assembly of the giant panda genome</a>. <em>Nature</em>. 
2010 Jan 21;463(7279):311-7.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20010809" 
target="_blank">20010809</a></p>

<!-- Pig Credits -->
<a name="pig_credits"></a>
<h2>Pig genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> </p> 
<ul> 
  <li>
  <a href="http://piggenome.org/" target="_blank">The International Swine Genome Sequencing 
  Consortium</a> </li>
  <li>
  <a href="http://www.sanger.ac.uk/resources/downloads/othervertebrates/pig.html" 
  target="_blank">Wellcome Trust Sanger Institute</a> </li>
</ul> 
<p> 
<strong>UCSC Pig Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=susScr11" target="_blank">susScr11</a>: Hiram Clawson, Cath
  Tyner </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=susScr3" target="_blank">susScr3</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg
  Roe, and  Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=susScr2" target="_blank">susScr2</a>: Hiram Clawson, Brian 
  Raney, Mark Diekhans, and Mary Goldman </li> 
</ul> 
<a name="pig_use"></a> 
<h3>Data use</h3> 
<p>
The pig genome sequence has been released by the International Swine Genome Sequencing Consortium
under the terms of the <a href="http://www.nature.com/nature/journal/v461/n7261/full/461168a.html" 
target="_blank">Toronto Statement</a>. </p> 
<p> 
The International Swine Genome Sequencing Consortium genome-wide analyses of the pig genome is 
described in:
Uenishi H, Morozumi T, Toki D, Eguchi-Ogawa T, Rund LA, Schook LB.  
<a href="http://www.biomedcentral.com/1471-2164/13/581" target=_blank">Large-scale sequencing 
based on full-length-enriched cDNA libraries in pigs: contribution to annotation of the pig genome 
draft sequence</a>. <em>BMC Genomics</em>. 2012 Nov 15;13:581.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/23150988" target="_blank">23150988</a></p> 
<p> 
The initial marker paper:
Archibald AL, Bolund L, Churcher C, Fredholm M, Groenen MA, Harlizius B, Lee KT, Milan D, Rogers J, 
Rothschild MF <em>et al</em>.  
<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-11-438" target="_blank">Pig genome sequence--analysis and publication strategy</a>.  <em>BMC Genomics</em>. 
2010 Jul 19;11:438. PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20642822" 
target="_blank">20642822</a>; PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3017778/" target="_blank">PMC3017778</a></p>

<!-- Pika Credits -->
<a name="pika_credits"></a>
<h2>Pika genome</h2> 
<p>
<strong>Sequencing/Assembly: </strong> <a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p>
<p> 
<strong>UCSC Pika Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=ochPri3" target="_blank">ochPri3</a>: Hiram Clawson, Brian 
  Raney, and Steve Heitner </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=ochPri2" target="_blank">ochPri2</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="pika_use"></a>
<h3>Data use</h3> 
<p>
The pika sequence has been made freely available as part of the Mammalian Genome Project (29 
Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>.  <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The pika data are provided with the following understanding: 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul>

<!-- Platypus Credits -->
<a name="platypus_credits"></a>
<h2>Platypus genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> 
<a href="http://genome.wustl.edu/genomes/detail/ornithorhynchus-anatinus/" target="_blank">The 
Genome Institute at Washington University (WUSTL)</a>, St. Louis, MO, USA </p> 
<p> 
<strong>UCSC Platypus Genome Browser/Annotations:</strong> </p>
<ul> 
  <li>		   
  <a href="../../cgi-bin/hgGateway?db=ornAna2" target="_blank">ornAna2</a>: Hiram Clawson, Brian 
  Raney, and Cath Tyner </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=ornAna1" target="_blank">ornAna1</a>: Angie Hinrichs, Kayla 
  Smith, Robert Kuhn, Brian Raney, and Donna Karolchik </li> 
</ul> 
<a name="platypus_use"></a>
<h3>Data use</h3> 
<p> 
The platypus genome data may be freely downloaded, used in analyses, and repackaged in databases if 
the data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Warren WC, Hillier LW, Marshall Graves JA, Birney E, Ponting CP, Grützner F, Belov K, Miller W, 
Clarke L, Chinwalla AT <em>et al</em>.  
<a href="http://www.nature.com/nature/journal/v453/n7192/full/nature06936.html" 
target="_blank">Genome analysis of the platypus reveals unique signatures of evolution</a>.  
<em>Nature</em>. 2008 May 8;453(7192):175-83.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18464734" target="_blank">18464734</a></p>

<!-- Proboscis Monkey Credits -->
<a name="proboscis_monkey_credits"></a>
<h2>Proboscis monkey genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> Proboscis Monkey Functional Genome Consortium </p>
<p>
<strong>UCSC Proboscis Monkey Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li>		   
  <a href="../../cgi-bin/hgGateway?db=nasLar1" target="_blank">nasLar1</a>: Hiram Clawson and 
  Jairo Navarro </li>
</ul> 
<a name="proboscis_monkey_use"></a> 
<h3>Data use</h3> 
<p> 
The proboscis monkey sequence is made freely available before scientific publication with the 
following understanding: </p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The center producing the data reserves the right to publish the initial large-scale analyses of   
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a chromosome
  (that is, 30 Mb or more). </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Rabbit Credits -->
<a name="rabbit_credits"></a>
<h2>Rabbit genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> 
<a href="http://www.broadinstitute.org/science/projects/mammals-models/rabbit/rabbit-genome-sequencing-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p>
<p> 
<strong>UCSC Rabbit Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=oryCun2" target="_blank">oryCun2</a>: Hiram Clawson and 
  Antonio Coelho </li>
</ul> 
<a name="rabbit_use"></a> 
<h3>Data use</h3> 
<p>
The rabbit sequence is made freely available before scientific publication with the following 
understanding: </p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The center producing the data reserves the right to publish the initial large-scale analyses of   
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a chromosome
  (that is, 30 Mb or more). </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Rat Credits -->
<a name="rat_credits"></a>
<h2>Rat genome</h2>
<h3>rn7</h3>
<p>
<strong>Sequencing/Assembly: </strong>
<a href="https://www.sanger.ac.uk/"
target="_blank">Wellcome Sanger Institute</a>, Hinxton, Cambridgeshire, UK </p>
<p>
<strong>UCSC Rat Genome Browser/Annotations:</strong> </p>
<ul>
  <li>
  <a href="../../cgi-bin/hgGateway?db=rn7" target="_blank">rn7</a>: Hiram Clawson, Lou Nassar, and
  Gerardo Perez </li>
</ul></p>
<h3>rn6 and older</h3>
<p> 
The rat genome assembly (<em>Rattus norvegicus</em>) is provided by the Rat Genome Sequencing 
Consortium with the following acknowledgments:</p> 
<p>
<strong>Project Coordination:</strong> <a href="https://www.hgsc.bcm.edu/" target="_blank">Baylor 
College of Medicine Human Genome Sequencing Center</a> (BCM-HGSC)</p> 
<p>
<strong>Funding:</strong> </p> 
<ul> 
  <li>
  <a href="http://www.genome.gov/" target="_blank">National Human Genome Research Institute</a> 
  (NHGRI) </li>
  <li>
  <a href="http://www.nhlbi.nih.gov/" target="_blank">National Heart, Lung and Blood Institute</a>
  (NHLBI) </li>
</ul> 
<p>
<strong>Sequencing:</strong> </p>
<ul> 
  <li>
  BCM-HGSC </li>
  <li>
  <a href="https://www.celera.com/" target="_blank">Celera Genomics Group of Applera 
  Corporation</a> </li> 
  <li>
  <a href="http://www.genomecorp.com/" target="_blank">Genome Therapeutics Corporation</a> </li> 
  <li>
  <a href="http://www.utah.edu/" target="_blank">University of Utah</a> </li>
</ul> 
<p>
<strong>Assembly:</strong> BCM-HGSC ATLAS group - Rui Chen, James Durbin, Paul Havlak </p> 
<p>
<strong>Fingerprint Mapping:</strong> <a href="http://www.bcgsc.ca/" target="_blank">The British 
Columbia Cancer Agency Genome Sciences Center</a> - Marco Marra, Jackie Schein </p>
<p>
<strong>BAC End Sequencing:</strong> <a href="http://www.jcvi.org/cms/home/" target="_blank">The 
Institute for Genomic Research</a> - Shaying Zhao </p>
<p>
<strong>BAC Library Construction and Distribution:</strong> <a href="http://bacpac.chori.org/" 
target="_blank">Children's Hospital Oakland Research Institute</a> (CHORI) - Pieter deJong </p> 
<p>
<strong><a href="http://rgd.mcw.edu" target="_blank">Rat Genome Database</a>:</strong> Medical 
College of Wisconsin </p>
<p>
<strong>UCSC Rat Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=rn6" target="_blank">rn6</a>: Hiram Clawson, Brian Raney and
  Matthew Speir </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=rn5" target="_blank">rn5</a>: Hiram Clawson, Brian Raney and
  Steve Heitner </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=rn4" target="_blank">rn4</a>: Angie Hinrichs, Fan Hsu, 
  Brooke Rhead, Archana Thakkapallayil, Kayla Smith, Ann Zweig, Robert Kuhn, and Donna 
  Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=rn3" target="_blank">rn3</a>: Hiram Clawson, Heather 
  Trumbower, Robert Kuhn, Yontao Lu, Terry Furey, Mark Diekhans, Robert Baertsch, Donna Karolchik,
  and Jim Kent </li> 
</ul> 
<a name="rat_use"></a>
<h3>Data use</h3> 
<p>
The rat genome data may be freely downloaded, used in analyses, and repackaged in databases if the 
data provider is properly acknowledged. Please cite these publications when using these data:</p> 
<p> 
Havlak P, Chen R, Durbin KJ, Egan A, Ren Y, Song XZ, Weinstock GM, Gibbs RA. 
<a href="http://genome.cshlp.org/content/14/4/721.long" target="_blank">The Atlas genome assembly 
system</a>. <em>Genome Res</em>. 2004 Apr;14(4):721-32.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/15060016" target="_blank">15060016</a></p> 
<p> 
Rat Genome Sequencing Project Consortium.  
<a href="http://www.nature.com/nature/journal/v428/n6982/abs/nature02426.html" 
target="_blank">Genome sequence of the Brown Norway rat yields insights into mammalian 
evolution</a>. <em>Nature</em>. 2004 Apr 1;428(6982):493-521. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/15057822" target="_blank">15057822</a></p> 
<p> 
See the Baylor <a href="https://www.hgsc.bcm.edu/bcm-hgsc-conditions-use" target="_blank">conditions
of use</a> web page for restrictions on the use of these data.</p>

<!-- Rhesus Credits -->
<a name="rhesus_credits"></a>
<h2>Rhesus genome</h2> 
<p> 
The UCSC Genome Browser displays multiple assemblies of the Rhesus macaque genome produced by 
different institutions: </p> 
<ul> 
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=rheMac10" target="_blank">rheMac10</a> Genome 
  Browser:</strong> Feb. 2019 draft assembly of the rhesus monkey, version Mmul_10, from the
  <a href="https://www.genome.wustl.edu/" 
  target="_blank">Genome Institute at Washington University School of Medicine</a>. </li>
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=rheMac8" target="_blank">rheMac8</a> Genome 
  Browser:</strong> Nov. 2015 draft assembly of the rhesus monkey, version Mmul_8.0.1, from the 
  <a href="https://www.hgsc.bcm.edu/non-human-primates/rhesus-monkey-genome-project" 
  target="_blank">Baylor College of Medicine Human Genome Sequencing Center</a>. </li> 
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=rheMac3" target="_blank">rheMac3</a> Genome 
  Browser:</strong> Oct. 2010 draft assembly of the rhesus monkey, version CR_1.0 from 
  <a href="http://www.genomics.cn/en/index" target=_blank>Beijing Genomics Institute-Shenzhen</a> 
  (BGI), Shenzhen, China </li> 
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=rheMac2" target="_blank">rheMac2</a> Genome 
  Browser:</strong> Jan. 2006 draft assembly of the rhesus monkey, version Mmul_051212 from the 
  Macaca mulatta Genome Sequencing Consortium (MMGSC) led by the 
  <a href="https://www.hgsc.bcm.edu/rhesus-monkey-genome-project" target="_blank">Baylor College of 
  Medicine Human Genome Sequencing Center (BCM-HGSC)</a>, in collaboration with the 
  <a href="http://www.jcvi.org/cms/home/" target="_blank">J. Craig Venter Science Foundation Joint 
  Technology Center</a> and <a href="http://genome.wustl.edu/genomes/detail/macaca-mulatta/" 
  target="_blank">The Genome Institute at Washington University (WUSTL)</a> </li> 
  <li> 
  <strong>rheMac1 Genome Browser:</strong> Jan. 2005 draft assembly of the rhesus monkey, version 
  Mmul_0.1 from MMGSC (see above) </li> 
</ul> 
<hr>
<p>
The Genome Institute at Washington University School of Medicine assembly is provided with the 
following acknowledgments:</p>
<p>
Macaca mulatta Sequence and Assembly Credits: Sequencing Funds - This work was supported by the 
NHGRI grant No. 5U24HG00908103, HIGH QUALITY HUMAN AND NON-HUMAN PRIMATE GENOME ASSEMBLIES, at 
the McDonnell Genome Institute, Washington University School of Medicine DNA source - Coriell 
sample id: AG07107, MMU Genome Sequencing - The McDonnell Genome Institute, Washington 
University School of Medicine. Sequence Assembly - The McDonnell Genome Institute, Washington 
University School of Medicine. Sequence assembly curation - The McDonnell Genome Institute, 
Washington University School of Medicine; The University of Washington, Seattle, WA.</p>
<hr>
<p> 
The BGI assembly is provided with the following acknowledgments: </p> 
<p> 
<strong>Funding:</strong> Southern China Center for Innovative Pharmaceuticals (SCCIP), the 
National Natural Science Foundation of China, the Shenzhen Municipal Government of China, the 
Major State Basic Research Development Program of China, the National Basic Research Program of 
China, and the National Science and Technology Major Project of Key Drug Innovation and 
Development </p> 
<p> 
<strong>Sequencing/Assembly:</strong> BGI, Shenzhen, China </p>
<hr>
<p> 
The MMGSC assembly is provided with the following acknowledgments: </p>
<p> 
<strong>Funding: </strong> <a href="http://www.genome.gov/" target="_blank">National Human Genome 
Research Institute</a> (NHGRI), USA </p> 
<p> 
<strong>Sequencing/Assembly: </strong> BCM-HGSC, Houston, TX, USA in collaboration with: </p> 
<ul> 
  <li> 
  J. Craig Venter Science Foundation Joint Technology Center, Rockville, MD, USA </li>
  <li> 
  The Genome Institute at WUSTL, St. Louis, MO, USA </li>
</ul> 
<p> 
<strong>BAC resources: </strong> <a href="http://bacpac.chori.org/rhesus250.htm" 
target="_blank">Children's Hospital Oakland Research Institute</a> (CHORI), Oakland, CA, USA</p> 
<p> 
<strong>BAC-based fingerprint map: </strong> <a href="http://www.bcgsc.ca/" target=-blank>Genome 
Sciences Centre</a>, Vancouver, B.C. </p> 
<p> 
For more information on the MMGSC rhesus macaque genome project, see the BCM HGSC 
<a href="https://www.hgsc.bcm.edu/rhesus-monkey-genome-project" target="_blank">Rhesus Monkey Genome
Project</a> web page. </p> 
<hr>
<p>
<strong>UCSC Rhesus Genome Browser/Annotations:</strong> </p>
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=rheMac10" target="_blank">rheMac10</a>: Hiram Clawson
  and Lou Nassar</li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=rheMac8" target="_blank">rheMac8</a>: Hiram Clawson, 
  Christopher Lee, and Cath Tyner </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=rheMac3" target="_blank">rheMac3</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=rheMac2" target="_blank">rheMac2</a>: Robert Baertsch, Kayla
  Smith, Ann Zweig, Robert Kuhn, and Donna Karolchik </li> 
  <li> 
  rheMac1 (archived): Robert Baertsch, Galt Barber, and Donna Karolchik </li> 
</ul> 
<a name="rhesus_use"></a>
<h3>Data use</h3> 
<p>
The Macaca mulatta Mmul_9.0 and Mmul_10 assembly sequences are made available to the community 
by the McDonnell Genome Institute, Washington University, St Louis, MO with the following 
understanding: Users are free to analyze these data if the providers of this data are properly 
cited, any whole genome analyses should follow guidelines outlined in the Toronto agreement 
on the use of this assembly.</p>

<hr>
<p>
The initial sequencing and analysis of the rhesus macaque genome produced by the MMGSC was published
in: Rhesus Macaque Genome Sequencing and Analysis Consortium. 
<a href="http://www.sciencemag.org/content/316/5822/222.long" target="_blank">Evolutionary and 
Biomedical Insights from the Rhesus Macaque Genome</a>. <em>Science</em>. 
2007 Apr 13;316(5822):222-34.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17431167" 
target="_blank">17431167</a></p> 
<p> 
The initial sequencing and analysis of the rhesus macaque genome produced by BGI was published in: 
Yan G, Zhang G, Fang X, Zhang Y, Li C, Ling F, Cooper DN, Li Q, Li Y, van Gool AJ <em>et al</em> 
<a href="http://www.nature.com/nbt/journal/v29/n11/full/nbt.1992.html" target=_blank>Genome 
sequencing and comparison of two nonhuman primate animal models, the cynomolgus and Chinese rhesus 
macaques</a>. <em>Nat Biotechnol</em>.  2011 Oct 16;29(11):1019-23.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22002653" target="_blank">22002653</a></p> 
<p> 
The rhesus macaque sequence is made freely available for public use with the following 
understanding: </p> 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases.</li>  
  <li>
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. For the rheMac1 and 
  <a href="../../cgi-bin/hgGateway?db=rheMac2" target="_blank">rheMac2</a> data, please cite the 
  BCM-HGSC web site or publications from BCM-HGSC referring to the genome sequence.</li>  
  <li>
  The BCM-HGSC and RMGSC plan to publish the assembly and genomic annotation of the dataset, 
  including large-scale identification of regions of evolutionary conservation.</li>  
  <li>
  This is in accordance with, and with the understandings in the <a href="ftLauderdale.pdf">Fort 
  Lauderdale meeting</a> discussing Community Resource Projects and the resulting 
  <a href="http://www.genome.gov/page.cfm?pageID=10506537" target="_blank">NHGRI policy 
  statement</a>. </li>
  <li>
  Any redistribution of the data should carry this notice.</li>  
</ul>

<!-- Rock Hyrax Credits -->
<a name="rock_hyrax_credits"></a>
<h2>Rock hyrax genome</h2> 
<p> 
<strong>Funding:</strong> <a href="http://www.genome.gov" target="_blank">National Human Genome 
Research Institute</a> (NHGRI) </p>
<p>
<strong>Sequencing/Assembly: </strong> </p>
<ul> 
  <li>
  <a href="https://www.hgsc.bcm.edu/hyrax-genome-project" target="_blank">Baylor College of 
  Medicine Human Genome Sequencing Center (BCM-HGSC)</a>, Houston, TX, USA </li> 
  <li>
  <a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
  target="_blank">The Broad Institute</a>, Cambridge, MA, USA </li>
</ul> 
<p> 
<strong>UCSC Rock Hyrax Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=proCap1" target="_blank">proCap1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li>
</ul> 
<a name="rock_hyrax_use"></a> 
<h3>Data use</h3> 
<p> 
The rock hyrax sequence has been made freely available as part of the 
Mammalian Genome Project (29 Mammals Project). The initial analysis of this dataset can be found in 
Lindblad-Toh K <em>et al</em>. 
<a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The rock hyrax data are provided with the following understanding: 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul>

<!--  Sheep Credits -->
<a name="sheep_credits"></a>
<h2>Sheep genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://www.sheephapmap.org/" 
target="_blank">International Sheep Genomics Consortium</a> (ISGC)</p> 
<p>
<strong>Mitochondrial Genome Sequencing:</strong> 
<a href="http://www.uni-giessen.de/cms?set_language=en" 
target="_blank">Justus-Liebig-University</a>, Germany</p> 
<p>
<strong>UCSC Sheep Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=oviAri4" target="_blank">oviAri4</a>: Hiram Clawson, Chris 
  Lee, Daniel Schmelter</li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=oviAri3" target="_blank">oviAri3</a>: Hiram Clawson, Brian 
  Raney, Matthew Speir, Brian Lee, Brooke Rhead, Steve Heitner, and Donna Karolchik</li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=oviAri1" target="_blank">oviAri1</a>: Chin Li, Greg Roe, and
  Luvina Guruvadoo</li> 
</ul> 
<a name="sheep_use"></a>
<h3>Data use</h3> 
<p> 
The sheep genome data may be freely downloaded, used in analyses, and repackaged in databases if the
data provider is properly acknowledged. Please cite these publications when using these data:</p> 
<p> 
Jiang Y, Xie M, Chen W, Talbot R, Maddox JF, Faraut T, Wu C, Muzny DM, Li Y, Zhang W <em>et al</em>.
<a href="http://www.sciencemag.org/content/344/6188/1168.abs" target="_blank">The sheep genome 
illuminates biology of the rumen and lipid metabolism</a>. <em>Science</em>. 
2014 Jun 6;344(6188):1168-73.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24904168" 
target="_blank">24904168</a></p> 
<p> 
International Sheep Genomics Consortium. 
<a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2052.2010.02100.x/abstract;jsessionid=C33A2FD64BF95FF988F729F8C53421C8.f02t03" 
target="_blank">The sheep genome reference sequence: a work in progress</a>. <em>Anim Genet</em>. 
2010 Oct;41(5):449-53.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20809919" 
target="_blank">20809919</a></p>

<!-- Shrew Credits -->
<a name="shrew_credits"></a>
<h2>Shrew genome</h2> 
<p>
<strong>Funding:</strong> <a href="http://www.genome.gov" target="_blank">National Human Genome 
Research Institute</a> (NHGRI) </p>
<p>
<strong>Sequencing/Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p>
<p> 
<strong>UCSC Shrew Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=sorAra2" target="_blank">sorAra2</a>: Hiram Clawson, Brian 
  Raney, and Steve Heitner </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=sorAra1" target="_blank">sorAra1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li>
</ul> 
<a name="shrew_use"></a>
<h3>Data use</h3> 
<p>
The shrew sequence has been made freely available as part of the 
Mammalian Genome Project (29 Mammals Project). The initial analysis of this dataset can be found in 
Lindblad-Toh K <em>et al</em>. 
<a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The shrew data are provided with the following understanding: 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul>

<!-- Sloth Credits -->
<a name="sloth_credits"></a>
<h2>Sloth genome</h2> 
<p> 
<strong>Sequencing:</strong> <a href="http://genome.wustl.edu/genomes/detail/choloepus-hoffmanni/"
target="_blank">The Genome Institute at Washington University</a> (WUSTL), St. Louis, MO, USA</p>
<p>
<strong>Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p>
<p> 
<strong>UCSC Sloth Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=choHof1" target="_blank">choHof1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="sloth_use"></a> 
<h3>Data use</h3> 
<p>
The sloth sequence has been made freely available as part of the Mammalian Genome Project (29 
Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>.  <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The sloth data are provided with the following understanding: 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul>

<!-- Squirrel Credits -->
<a name="squirrel_credits"></a>
<h2>Squirrel genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p>
<p> 
<strong>UCSC Squirrel Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=speTri2" target="_blank">speTri2</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="squirrel_use"></a>
<h3>Data use</h3> 
<p> 
The squirrel sequence has been made freely available as part of the Mammalian Genome Project (29 
Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>.  <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The squirrel data are provided with the following understanding: 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul>

<!-- Squirrel Monkey Credits -->
<a name="squirrel_monkey_credits"></a>
<h2>Squirrel monkey genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p>
<p> 
<strong>UCSC Squirrel Monkey Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=saiBol1" target="_blank">saiBol1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="squirrel_monkey_use"></a>
<h3>Data use</h3> 
<p>
The squirrel monkey sequence is made freely available before scientific publication with the 
following understanding: </p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The center producing the data reserves the right to publish the initial large-scale analyses of   
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a chromosome
  (that is, 30 Mb or more). </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Tarsier Credits -->
<a name="tarsier_credits"></a>
<h2>Tarsier genome</h2> 
<p> 
<strong>Funding:</strong> <a href="http://www.genome.gov" target="_blank">National Human Genome 
Research Institute</a> (NHGRI) </p>
<p>
<strong>Sequencing/Assembly: </strong> </p> 
<ul> 
  <li>
  <a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
  target="_blank">The Broad Institute</a>, Cambridge, MA, USA </li> 
  <li>
  <a href="http://genome.wustl.edu/genomes/detail/tarsius-syrichta/" target="_blank">Washington 
  University</a>, St Louis, MO, USA </li>
</ul> 
<p> 
<strong>UCSC Tarsier Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=tarSyr2" target="_blank">tarSyr2</a>: Hiram Clawson, Brian 
  Raney, and Matthew Speir </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=tarSyr1" target="_blank">tarSyr1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="tarsier_use"></a>
<h3>Data use</h3> 
<p> 
The tarsier sequence has been made freely available as part of the Mammalian Genome Project (29 
Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>.  <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The tarsier data are provided with the following understanding: 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul>

<!-- Tasmanian Devil Credits -->
<a name="tasmanian_devil_credits"></a>
<h2>Tasmanian devil genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://www.sanger.ac.uk/" target="_blank">Wellcome 
Trust Sanger Institute</a>, Cambridge, UK </p>
<p> 
<strong>UCSC Tasmanian Devil Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=sarHar1" target="_blank">sarHar1</a>: Chin Li and Luvina 
  Guruvadoo </li>
</ul> 
<a name="tasmanian_devil_use"></a>
<h3>Data use</h3> 
<p> 
The Tasmanian devil genome data may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite this publication when using 
these data:</p> 
<p> 
Murchison EP, Schulz-Trieglaff OB, Ning Z, Alexandrov LB, Bauer MJ, Fu B, Hims M, Ding Z, Ivakhno S,
Stewart C <em>et al</em>. <a href="http://www.cell.com/cell/abstract/S0092-8674%2812%2900081-5" 
target="_blank">Genome sequencing and analysis of the Tasmanian devil and its transmissible 
cancer</a>. <em>Cell</em>. 2012 Feb 17;148(4):780-91.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22341448" target="_blank">22341448</a></p>

<!-- Tenrec Credits -->
<a name="tenrec_credits"></a>
<h2>Tenrec genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> <a href="http://www.broadinstitute.org/" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p> 
<p> 
<strong>UCSC Tenrec Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=echTel2" target="_blank">echTel2</a>: Hiram Clawson, Brian 
  Raney and Steve Heitner </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=echTel1" target="_blank">echTel1</a>: Hiram Clawson, Brian 
  Raney and Steve Heitner </li> 
</ul> 
<a name="tenrec_use"></a> 
<h3>Data use</h3> 
<p> 
The tenrec sequence is made freely available before scientific publication with the following 
understanding: </p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The center producing the data reserves the right to publish the initial large-scale analyses of   
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a chromosome
  (that is, 30 Mb or more). </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Tree Shrew Credits -->
<a name="tree_shrew_credits"></a>
<h2>Tree shrew genome</h2> 
<p>
<strong>Funding:</strong> <a href="http://www.genome.gov" target="_blank">National Human Genome 
Research Institute</a> (NHGRI) </p>
<p>
<strong>Sequencing/Assembly: </strong>
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/29-mammals-project" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p>
<p>
<strong>UCSC Tree Shrew Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=tupBel1" target="_blank">tupBel1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="tree_shrew_use"></a>
<h3>Data use</h3> 
<p>
The tree shrew sequence has been made freely available as part of the Mammalian Genome Project (29 
Mammals Project). The initial analysis of this dataset can be found in Lindblad-Toh K 
<em>et al</em>.  <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10530.html" 
target="_blank">A high-resolution map of human evolutionary constraint using 29 mammals</a>. 
<em>Nature</em>. 2011 Oct 12;478(7370):476-82.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21993624" target="_blank">21993624</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207357/" target="_blank">PMC3207357</a></p> 
<p> 
The tree shrew data are provided with the following understanding: 
<ul> 
  <li>
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li>
</ul>

<!-- Wallaby Credits -->
<a name="wallaby_credits"></a>
<h2>Wallaby genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> The Tammar Wallaby Genome Sequencing Consortium - 
<a href="https://www.hgsc.bcm.edu/other-mammals/wallaby-genome-project" target="_blank">Baylor 
College of Medicine Human Genome Sequencing Center</a> (BCM-HGSC) and the 
<a href="http://www.agrf.org.au/" target="_blank">Australian Genome Research Facility</a> </p>
<p> 
<strong>UCSC Wallaby Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=macEug2" target="_blank">macEug2</a>: Chin Li and Luvina 
  Guruvadoo </li>
</ul> 
<a name="wallaby_use"></a>
<h3>Data use</h3> 
<p> 
The wallaby genome data may be freely downloaded, used in analyses, and repackaged in databases if 
the data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Renfree MB, Papenfuss AT, Deakin JE, Lindsay J, Heider T, Belov K, Rens W, Waters PD, Pharo EA, 
Shaw G <em>et al.</em> <a href="http://genomebiology.com/content/12/8/R81" target="_blank">Genome 
sequence of an Australian kangaroo, Macropus eugenii, provides insight into the evolution of 
mammalian reproduction and development</a>.  <em>Genome Biol</em>. 2011 Aug 29;12(8):R81.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21854559" target="_blank">21854559</a></p>

<!-- White Rhinoceros Credits -->
<a name="white_rhinoceros_credits"></a>
<h2>White rhinoceros genome</h2> 
<p>
<strong>Sequencing/Assembly: </strong> 
<a href="http://www.broadinstitute.org/science/projects/projects" target="_blank">The Broad 
Institute</a>, Cambridge, MA, USA </p> 
<p>
<strong>Data Samples: </strong> <a href="http://biology.ucsd.edu/research/faculty/oryder" 
target="_blank">Dr. Oliver Ryder</a>, San Diego Zoo Institute for Conservation Research, San 
Diego, CA </p> 
<p> 
<strong>UCSC White Rhinoceros Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=cerSim1" target="_blank">cerSim1</a>: Hiram Clawson and 
  Steve Heitner </li>
</ul> 
<a name="white_rhinoceros_use"></a>
<h3>Data use</h3> 
<p> 
The southern white rhinoceros sequence is made freely available before scientific publication with 
the following understanding: </p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The center producing the data reserves the right to publish the initial large-scale analyses of   
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a chromosome
  (that is, 30 Mb or more). </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- African clawed frog Credits -->

<a name="african_clawed_frog_credits"></a>
<h2>African clawed frog</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> 
<a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_001663975.1/" 
target="_blank"> International Xenopus Sequencing Consortium</a> </p> 
<p> 
<strong>UCSC African clawed frog Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=xenLae2" target="_blank">xenLae2</a>: Hiram Clawson and 
  Cath Tyner</li> 
</ul> 
<a name="african_clawed_frog_use"></a>
<h3>Data use</h3> 
<p>
The African clawed frog genome data may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite this publication when using 
these data:</p> 
<p>
Session AM, Uno Y, Kwon T, Chapman JA, Toyoda A, Takahashi S, Fukui A, Hikosaka A, Suzuki A, Kondo M
<em>et al</em>.
<a href="http://dx.doi.org/10.1038/nature19840" target="_blank">
Genome evolution in the allotetraploid frog Xenopus laevis</a>.
<em>Nature</em>. 2016 Oct 20;538(7625):336-343.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27762356" target="_blank">27762356</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5313049/" target="_blank">PMC5313049</a>
</p>


<!-- American Alligator Credits -->
<a name="american_alligator_credits"></a>
<h2>American alligator genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://www.crocgenomes.org/" 
target="_blank">International Crocodilian Genomes Working Group</a> </p> 
<p> 
<strong>UCSC American Alligator Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=allMis1" target="_blank">allMis1</a>: Hiram Clawson, 
  Jonathan Casper, and Brian Lee </li> 
</ul> 
<a name="American_alligator_use"></a>
<h3>Data use</h3> 
<p>
The American alligator genome data may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite this publication when using 
these data:</p> 
<p> 
St John JA, Braun EL, Isberg SR, Miles LG, Chong AY, Gongora J, Dalzell P, Moran C, Bed'hom B, 
Abzhanov A <em>et al</em>.  <a href="http://genomebiology.com/content/13/1/415" 
target="_blank">Sequencing three crocodilian genomes to illuminate the evolution of archosaurs and 
amniotes</a>. <em>Genome Biol</em>. 2012 Jan 31;13(1):415. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22293439" target="_blank">22293439</a></p>

<!-- Atlantic Cod -->
<a name="atlantic_cod_credits"></a>
<h2>Atlantic cod genome</h2> 
<p>
<strong>Funding:</strong> 
<a href="http://www.forskningsradet.no/prognett-fuge/Programme_description/1226993493179" 
target=_blank>Research Council of Norway</a> (FUGE) </p> 
<p>
<strong>Sequencing/Assembly: </strong> <a href="http://www.codgenome.no/index.php" 
target="_blank">GenoFisk Consortium</a> </p> 
<p> 
<strong>UCSC Atlantic Cod Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=gadMor1" target="_blank">gadMor1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 

<a name="atlantic_cod_use"></a>
<h3>Data use</h3> 
<p> 
The Atlantic cod genome data may be freely downloaded, used in analyses, and repackaged in databases
if the data provider is properly acknowledged. Please cite these publications when using these 
data:</p> 
<p> 
Johansen SD, Coucheron DH, Andreassen M, Karlsen BO, Furmanek T, J&oslash;rgensen TE, Emblem A, 
Breines R, Nordeide JT, Moum T <em>et al</em>. 
<a href="http://www.sciencedirect.com/science/article/pii/S1871678409000569" 
target=_blank>Large-scale sequence analyses of Atlantic cod</a>. <em>N Biotechnol</em>. 
2009 Jun;25(5):263-71.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19491044" 
target="_blank">19491044</a></p> 
<p> 
Star B, Nederbragt AJ, Jentoft S, Grimholt U, Malmstr&oslash;m M, Gregers TF, Rounge TB, Paulsen J, 
Solbakken MH, Sharma A <em>et al</em>.  
<a href="http://www.nature.com/nature/journal/v477/n7363/full/nature10342.html" 
target="_blank">The genome sequence of Atlantic cod reveals a unique immune system</a>. 
<em>Nature</em>. 2011 Aug 10;477(7363):207-10.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21832995" target="_blank">21832995</a></p>

<!-- Budgerigar Credits -->
<a name="budgerigar_credits"></a>
<h2>Budgerigar genome</h2> 
<p>
<strong>Sequencing/Assembly: </strong> <a href="http://genome.wustl.edu/genomes/" 
target="_blank">The Genome Institute at Washington University</a> (WUSTL), St. Louis, MO,
USA</p>
<p> 
<strong>UCSC Budgerigar Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=melUnd1" target="_blank">melUnd1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="budgerigar_use"></a>
<h3>Data use</h3> 
<p> 
The budgerigar sequence is made freely available before scientific publication with the following 
understanding: </p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The center producing the data reserves the right to publish the initial large-scale analyses of   
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a chromosome
  (that is, 30 Mb or more). </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Chicken Credits -->
<a name="chicken_credits"></a>
<h2>Chicken genome</h2> 
<p>
<strong>Sequencing: </strong><a href="https://www.ncbi.nlm.nih.gov/grc/chicken" target="_blank">The Genome Reference Consortium</a></p>
<ul>
  <li>
  <a href="http://genome.wustl.edu/genomes/detail/gallus-gallus/" 
  target="_blank">The Genome Institute at Washington University</a> (WUSTL), St. Louis, MO, USA
  </li>
</ul>
<p>
<strong>Physical Map:</strong> The Genome Institute at WUSTL </p> 
<p>
<strong>Genetic Mapping/Linkage Analysis: </strong>The Chicken Mapping Consortium:
</p>
<ul> 
  <li> 
  Wageningen Map based on Wageningen broilerxbroiler mapping population ("W" linkage maps; 460 
  F2 animals), coordinated by Martien Groenen, 
  <a href="http://www.wageningenur.nl/en/Expertise-Services/Chair-groups/Animal-Sciences/Animal-Breeding-and-Genetics-Group.htm" 
  target="_blank">Wageningen University</a>, Wageningen, The Netherlands </li> 
  <li> 
  East Lansing Map based on the RJFxlayer cross ("E" linkage maps; 56 BC animals), coordinated by 
  Hans Cheng, 
  <a href="https://www.ars.usda.gov/PandP/locations/cityPeopleList.cfm?modecode=50-50-00-00" 
  target="_blank">USDA-ARS</a>, East Lansing, MI, USA </li>
  <li> 
  Compton map based on layerxlayer cross ("C" linkage maps; 52 BC animals), coordinated by Nat 
  Bumstead, <a href="http://www.pirbright.ac.uk/" target="_blank">Institute for Animal 
  Health</a> </li>
  <li> 
  Consensus linkage map: Martien Groenen and Richard Crooijmans, Wageningen University </li> 
  <li>
  RH map: coordinated by Mireille Morisson and Alain Vignal, 
  <a href="https://www-lgc.toulouse.inra.fr/internet/" target="_blank">French National Institute 
  for Agricultural Research</a> (INRA), Toulouse </li> 
  <li> 
  Linkage Mapping: White Leghorn X Red Junglefowl (800 F2 animals), coordinated by Susanne Kerje, 
  Lina Jacobsson and Leif Andersson, <a href="http://www.uu.se/en/" target="_blank">Uppsala 
  University</a>, Uppsala, Sweden </li> 
  <li> 
  Clone/Marker Pairs: coordinated by Jerry Dodgson, Michigan State University, East Lansing, MI, 
  USA </li> 
  <li> 
  Marker Name/Sequence Resolution: coordinated by Hans Cheng, USDA-ARS, East Lansing, MI, USA</li>
  <li> 
  Chicken FPC browser: <a href="http://www.bioinformatics.nl/gbrowse/cgi-bin/gbrowse/ChickFPC/" 
  target="_blank">ChickFPC</a>, Martien Groenen and Richard Crooijmans 
  (<a href="../../cgi-bin/hgGateway?db=galGal3" target="_blank">galGal3</a>), Jan Aerts 
  (<a href="../../cgi-bin/hgGateway?db=galGal2" target="_blank">galGal2</a>), Wageningen 
  University </li> 
  <li> 
  Additional Mapping Data Contributions: Winston Bellott, David Page Lab, 
  <a href="http://wi.mit.edu/" target="_blank">Whitehead Institute for Biomedical Research</a> 
  (WIBR-MIT), Cambridge, MA, USA </li> 
</ul> 
<p> 
<strong>cDNA sequences:</strong> <a href="https://www.ncbi.nlm.nih.gov/" target="_blank">National 
Institutes of Health</a> and The University of Manchester 
<a href="http://www.chick.manchester.ac.uk/" target="_blank">BBSRC ChickEST Database</a> </p> 
<p> 
<strong>RJF finished clones:</strong> </p>
<ul> 
  <li> 
  <a href="http://www.nisc.nih.gov/" target="_blank">NIH Intramural Sequencing Center</a> (NISC) 
  (AC091708.2, AC094012.2, AC138565.3) <li>S. Kerje and L. Andersson, 
  <a href="http://www.uu.se/en/" target="_blank">Uppsala University</a> (AY636124.1) </li> 
  <li> 
  The Genome Institute at WUSTL (all others)</li>
</ul> 
<p> 
<strong>Assembly, Assembly/Map Integration, Golden Path Creation:</strong> The Genome Institute at
WUSTL </p> 
<p> 
<strong>SNPs and cross-referenced gene annotations:</strong> 
<a href="http://www.genomics.cn/en/index" target="_blank">The Beijing Genomics Institute 
(BGI)</a>, Beijing, China </p> 
<p> 
<strong>UCSC Chicken Genome Browser/Annotations:</strong> </p>
<ul>
  <li>
  <a href="../../cgi-bin/hgGateway?db=galGal6" target="_blank">galGal6</a>: Hiram Clawson,
  Daniel Schmelter </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=galGal5" target="_blank">galGal5</a>: Hiram Clawson, Brian 
  Raney and Matthew Speir <li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=galGal4" target="_blank">galGal4</a>: Hiram Clawson, Brian 
  Raney and Steve Heitner </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=galGal3" target="_blank">galGal3</a>: Angie Hinrichs, Kayla 
  Smith, Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=galGal2" target="_blank">galGal2</a>: Angie Hinrichs, 
  Heather Trumbower, Rachel Harte, Donna Karolchik </li> 
</ul> 
<a name="chicken_use"></a>
<h3>Data use</h3> 
<p> 
The chicken genome data may be freely downloaded, used in analyses, and repackaged in databases if 
the data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
International Chicken Genome Sequencing Consortium. 
<a href="http://www.nature.com/nature/journal/v432/n7018/full/nature03154.html" 
target="_blank">Sequence and comparative analysis of the chicken genome provide unique perspectives 
on vertebrate evolution</a>. <em>Nature</em>. 2004 Dec 9;432(7018):695-716. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/15592404" target="_blank">15592404</a></p> 
<p> 
The BGI gene annotations and SNP data are made available with the following terms of use:</p> 
<ul> 
  <li> 
  Users are free to use these data in scientific papers analyzing particular genes, if the Beijing 
  Genomics Institute (BGI) is acknowledged. </li> 
  <li> 
  BGI and its collaborators reserve the right to publish the initial analyses of these data, 
  including but not restricted to the large-scale identification of functional polymorphisms, 
  evolutionary patterns and signs of selection, correlations with known QTLs, utility of data for 
  genetic mapping purposes, etc. </li>
  <li> 
  Any redistribution of these data should carry this notice. </li>
</ul>

<!-- Coelacanth Credits -->
<a name="coelacanth_credits"></a>
<h2>Coelacanth genome</h2> 
<p>
<strong>Funding:</strong> <a href="http://www.genome.gov" target="_blank">National Human Genome 
Research Institute</a> (NHGRI) </p>
<p>
<strong>Sequencing/Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/vertebrates-invertebrates/coelacanth/coelacanth" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p> 
<p> 
<strong>UCSC Coelacanth Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=latCha1" target="_blank">latCha1</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="coelacanth_use"></a>
<h3>Data use</h3> 
<p> 
The coelacanth genome data may be freely downloaded, used in analyses, and repackaged in databases 
if the data provider is properly acknowledged. Please cite this publication when using these 
data:</p> 
<p> 
Amemiya CT, Alf&ouml;ldi J, Lee AP, Fan S, Philippe H, Maccallum I, Braasch I, Manousaki T, 
Schneider I, Rohner N <em>et al</em>. 
<a href="http://www.nature.com/nature/journal/v496/n7445/full/nature12027.html" target="_blank">The 
African coelacanth genome provides insights into tetrapod evolution</a>. <em>Nature</em>. 
2013 Apr 18;496(7445):311-6.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23598338" 
target="_blank">23598338</a></p>

<!-- Elephant Shark Credits -->
<a name="elephant_shark_credits"></a>
<h2>Elephant shark genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://www.imcb.a-star.edu.sg/php/main.php" 
target="_blank">Singapore Institute of Molecular and Cell Biology</a> </p>
<p> 
<strong>UCSC Elephant Shark Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=calMil1" target="_blank">calMil1</a>: Hiram Clawson, 
  Jonathan Casper, and Donna Karolchik </li> 
</ul> 

<a name="elephant_shark_use"></a>
<h3>Data use</h3> 
<p> 
The elephant shark genome data may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite this publication when using 
these data:</p> 
<p> 
Venkatesh B, Lee AP, Ravi V, Maurya AK, Lian MM, Swann JB, Ohta Y, Flajnik MF, Sutoh Y, Kasahara M 
<em>et al</em>. <a href="http://www.nature.com/nature/journal/v505/n7482/full/nature12826.html" 
target="_blank">Elephant shark genome provides unique insights into gnathostome evolution</a>. 
<em>Nature</em>. 2014 Jan 9;505(7482):174-9. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/24402279" target="_blank">24402279</a> </p>

<!-- Fugu Credits -->
<a name="fugu_credits"></a>
<h2>Fugu genome</h2> 
<p> 
The Fugu genome assembly (<em>Takifugu rubripes</em>) is provided by the 
<a href="http://genome.jgi-psf.org/fugu6/fugu6.info.html" target="_blank">Joint Genome Institute</a>
(JGI) as part of the International Fugu Genome Consortium, led by the JGI and the 
<a href="http://www.fugu-sg.org" target="_blank">Singapore Institute of Molecular and Cell 
Biology</a> (IMCB). 
<p> 
<strong>Sequencing/Assembly:</strong> Assembly was constructed with the JGI assembler, JAZZ, from 
paired end sequencing reads produced by JGI and the IMCB at JGI, Myriad Genetics, and Celera 
Genomics </p>
<p> 
<strong>UCSC Fugu Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=fr3" target="_blank">fr3</a>: Hiram Clawson and Greg 
  Roe</li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=fr2" target="_blank">fr2</a>: Cory McClean, Hiram Clawson, 
  Ann Zweig, and Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=fr1" target="_blank">fr1</a>: Kate Rosenbloom, Heather 
 Trumbower, Robert Kuhn, and Donna Karolchik </li> 
</ul> 
<a name="fugu_use"></a> 
<h3>Data use</h3> 
<p> 
The Fugu genome data may be freely downloaded, used in analyses, and repackaged in databases if the 
data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Kai W, Kikuchi K, Tohari S, Chew AK, Tay A, Fujiwara A, Hosoya S, Suetake H, Naruse K, Brenner S 
<em>et al</em>.  <a href="http://gbe.oxfordjournals.org/content/3/424.long" 
target="_blank">Integration of the genetic map and genome assembly of fugu facilitates insights into
distinct features of genome evolution in teleosts and mammals</a>. <em>Genome Biol Evol</em>. 
2011;3:424-42.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21551351" 
target="_blank">21551351</a></p>

<!-- Lamprey Credits -->
<a name="lamprey_credits"></a>
<h2>Lamprey genome</h2> 
<p>
<p> 
<strong>Funding:</strong> <a href="http://www.genome.gov" target="_blank">National Human Genome 
Research Institute</a> (NHGRI) </p> 
<p> 
<strong>Sequencing/Assembly:</strong> </p>
<ul> 
  <li>
  <a href="https://ukhealthcare.uky.edu/genomics-core-laboratory" target="_blank">University of
  Kentucky</a>, Lexington, KY, USA</li>
  <li>
  <a href="http://genome.wustl.edu/genomes/detail/petromyzon-marinus/" target="_blank">The 
  Genome Institute at WUSTL</a>, St. Louis, MO, USA </li>
  <li>
  Jeramiah Smith (<a href="https://bio.as.uky.edu/users/jjsmit3" target="_blank">University of 
  Kentucky</a>, <a href="http://www.gs.washington.edu/" target="_blank">University of Washington 
  Department of Genome Sciences</a>, and <a href="https://www.benaroyaresearch.org/" 
  target="_blank">Benaroya Research Institute at Virginia Mason</a>) </li> 
</ul> 
<p> 
<strong>UCSC Lamprey Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=petMar3" target="_blank">petMar3</a>: Hiram Clawson
  and Jairo Navarro </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=petMar2" target="_blank">petMar2</a>: Hiram Clawson, Brian 
  Raney, and Steve Heitner </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=petMar1" target="_blank">petMar1</a>: Hiram Clawson, Ann 
  Zweig, Pauline Fujita, and Donna Karolchik </li> 
</ul> 

<a name="lamprey_use"></a>
<h3>Data use</h3> 
<p> 
The lamprey genome data may be freely downloaded, used in analyses, and repackaged in databases if 
the data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Smith JJ, Kuraku S, Holt C, Sauka-Spengler T, Jiang N, Campbell MS, Yandell MD, Manousaki T, Meyer 
A <em>et al</em>. <a href="http://www.nature.com/ng/journal/v45/n4/full/ng.2568.html" 
target="_blank">Sequencing of the sea lamprey (Petromyzon marinus) genome provides insights into 
vertebrate evolution</a>. <em>Nat Genet</em>. 2013 Apr;45(4):415-21, 421e1-2. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/23435085" target="blank">234350859</a></p>

<!-- Lizard Credits -->
<a name="lizard_credits"></a>
<h2>Lizard genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://www.broadinstitute.org/" 
target="_blank">The Broad Institute</a>, Cambridge, MA USA </p> 
<p> 
<strong>UCSC Lizard Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=anoCar2" target="_blank">anoCar2</a>: Hiram Clawson, Brian 
  Raney, Luvina Guruvadoo, and Donna Karolchik </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=anoCar1" target="_blank">anoCar1</a>: Hiram Clawson, 
  Archana Thakkapallayil, Robert Kuhn, and Donna Karolchik </li> 
</ul> 
<a name="lizard_use"></a>
<h3>Data use</h3> 
<p> 
The lizard genome data may be freely downloaded, used in analyses, and repackaged in databases if 
the data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Alf&ouml;ldi J, Di Palma F, Grabherr M, Williams C, Kong L, Mauceli E, Russell P, Lowe CB, Glor RE, 
Jaffe JD <em>et al</em>. 
<a href="http://www.nature.com/nature/journal/v477/n7366/full/nature10390.html" target="_blank">The 
genome of the green anole lizard and a comparative analysis with birds and mammals</a>. 
<em>Nature</em>. 2011 Aug 31;477(7366):587-91.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21881562" target="_blank">21881562</a></p>

<!-- Medaka Credits -->
<a name="medaka_credits"></a>
<h2>Medaka genome</h2> 
<p> 
The medaka genome assembly (<em>Oryzias latipes</em>) is provided by Japan's 
<a href="http://www.nig.ac.jp/english/index.html" target="_blank">National Institute of Genetics</a>
(NIG) and the <a href="http://utgenome.org/medaka/" target="_blank">University of Tokyo</a>. The 
assembly was constructed using the RAMEN assembler.</p> 
<p> 
NOTE: The UCSC oryLat1 assembly was replaced by 
<a href="../../cgi-bin/hgGateway?db=oryLat2" target="_blank">oryLat2</a> in Nov. 2008 to correct a 
UCSC assembly error with chrUn in which the gap relationships between contigs within their
ultracontigs were broken. The <a href="../../cgi-bin/hgGateway?db=oryLat2" 
target="_blank">oryLat2</a> assembly is based on the same assembly as oryLat1: the v1.0 assembly 
from NIG and the University of Tokyo. </p> 
<p> 
<strong>Sequencing/Assembly: </strong> NIG and University of Tokyo, Japan </p>
<p> 
<strong>UCSC Medaka Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=oryLat2" target="_blank">oryLat2</a>: Hiram Clawson, Pauline
  Fujita, and Donna Karolchik </li>
</ul> 
<a name="medaka_use"></a> 
<h3>Data use</h3> 
<p> 
The medaka genome data may be freely downloaded, used in analyses, and repackaged in databases if 
the data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Kasahara M, Naruse K, Sasaki S, Nakatani Y, Qu W, Ahsan B, Yamada T, Nagayasu Y, Doi K, Kasai Y 
<em>et al</em>. <a href="http://www.nature.com/nature/journal/v447/n7145/full/nature05846.html" 
target="_blank">The medaka draft genome and insights into vertebrate genome evolution</a>. 
<em>Nature</em>. 2007 Jun 7;447(7145):714-9.  
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17554307" target="_blank">17554307</a></p>

<!-- Medium Ground Finch Credits -->
<a name="medium_ground_finch_credits"></a>
<h2>Medium ground finch genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> Genome 10K Project and Beijing Genomics Institute 
(BGI)</p> 
<p> 
<strong>UCSC Medium Ground Finch Genome Browser/Annotations:</strong></p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=geoFor1" target="_blank">geoFor1</a>: Hiram Clawson and 
  Greg Roe </li>
</ul> 

<a name="medium_ground_finch_use"></a> 
<h3>Data use</h3> 
<p> 
The medium ground finch  genome data may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite this publication when using 
these data:</p> 
<p>
Zhang G, Li C, Li Q, Li B, Larkin DM, Lee C5, Storz JF, Antunes A, Greenwold MJ, Meredith RW 
<em>et al</em>. <a href="http://science.sciencemag.org/content/346/6215/1311.full" 
target="_blank">Comparative genomics reveals insights into avian genome evolution and 
adaptation</a>. <em>Science</em>. 2014 Dec 12;346(6215):1311-20. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/25504712" target="_blank">25504712</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4390078/" target="_blank">PMC4390078</a></p>

<!-- Nile Tilapia Credits -->
<a name="nile_tilapia_credits"></a>
<h2>Nile tilapia genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> 
<a href="http://www.broadinstitute.org/scientific-community/science/projects/mammals-models/vertebrates-invertebrates/tilapia/tilapia-genom" 
target="_blank">The Broad Institute</a>, Cambridge, MA, USA </p> 
<p> 
<strong>UCSC Nile Tilapia Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=oreNil2" target="_blank">oreNil2</a>: Hiram Clawson, Chin 
  Li, Brian Raney, Pauline Fujita, Luvina Guruvadoo, Steve Heitner, Brooke Rhead, Greg Roe, and 
  Donna Karolchik </li> 
</ul> 
<a name="nile_tilapia_use"></a> 
<h3>Data use</h3> 
<p> 
The Nile tilapia genome data may be freely downloaded, used in analyses, and repackaged in databases
if the data provider is properly acknowledged. Please cite this publication when using these 
data:</p> 
<p>
Brawand D, Wagner CE, Li YI, Malinsky M, Keller I, Fan S, Simakov O, Ng AY, Lim ZW, Bezault E 
<em>et al</em>. <a href="https://www.nature.com/nature/journal/v513/n7518/full/nature13726.html" 
target="_blank">The genomic substrate for adaptive radiation in African cichlid fish</a>.  
<em>Nature</em>. 2014 Sep 18;513(7518):375-81.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/25186727" target="_blank">25186727</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4353498/" target="_blank">PMC4353498</a></p>

<!-- Painted Turtle Credits -->
<a name="painted_turtle_credits"></a>
<h2>Painted turtle genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> International Painted Turtle Genome Sequencing 
Consortium </p> 
<p> 
<strong>UCSC Painted Turtle Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=chrPic1" target="_blank">chrPic1</a>: Hiram Clawson, Brian 
  Raney and Steve Heitner </li> 
</ul> 
<a name="painted_turtle_use"></a> 
<h3>Data use</h3> 
<p> 
The painted turtle genome data may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite this publication when using 
these data:</p> 
<p> 
Badenhorst D, Hillier LW, Literman R, Montiel EE, Radhakrishnan S, Shen Y, Minx P, Janes DE, Warren 
WC, Edwards SV <em>et al</em>.  
<a href="https://academic.oup.com/gbe/article-lookup/doi/10.1093/gbe/evv119" 
target="_blank">Physical mapping and refinement of the painted turtle genome (Chrysemys picta) 
inform amniote genome evolution and challenge turtle-bird chromosomal conservation</a>. 
<em>Genome Biol Evol</em>. 2015 Jun 24;7(7):2038-50.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/26108489" target="_blank">26108489</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4524486/" target="_blank">PMC4524486</a></p>

<!-- Stickleback Credits -->
<a name="stickleback_credits"></a>
<h2>Stickleback genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://www.broadinstitute.org/" 
target="_blank">The Broad Institute</a>, Cambridge, MA USA </li> 
<p> 
<strong>UCSC Stickleback Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=gasAcu1" target="_blank">gasAcu1</a>: Angie Hinrichs, Hiram 
  Clawson, Kayla Smith, and Donna Karolchik </li> 
</ul> 
<a name="stickleback_use"></a>
<h3>Data use</h3> 
<p> 
The stickleback sequence is made freely available before scientific publication with the following 
understanding: </p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The center producing the data reserves the right to publish the initial large-scale analyses of   
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a chromosome
  (that is, 30 Mb or more). </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Tetraodon Credits -->
<a name="tetraodon_credits"></a>
<h2>Tetraodon genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> </p>
<ul> 
  <li> 
  <a href="http://www.genoscope.cns.fr/spip/spip.php?lang=en" target="_blank">Genoscope</a>, Evry,
  France </li> 
  <li> 
  <a href="https://www.broadinstitute.org/scientific-community/science/projects/mammals-models/vertebrates-invertebrates/other-vertebrates-and" 
  target="_blank">The Broad Institute</a>, MA, USA </li> 
</ul> 
<p> 
<strong>Project Funding:</strong> </p> 
<ul> 
  <li> 
  <a href="http://www.cng.fr/en/" target="_blank">Consortium National de Recherche en 
  Genomique</a>, France </li> 
  <li> 
  <a href="http://www.genome.gov" target="_blank">National Human Genome Research Institute</a> 
  (NHGRI), USA </li> 
</ul> 
<p> 
<strong>Initial Genome Browser Annotations:</strong> </p> 
<ul> 
  <li> 
  Genoscope - Genscan gene predictions, GAZE gene predictions </li>
  <li> 
  <a href="../staff.html">UCSC Genome Bioinformatics Group</a>, University of California, Santa 
  Cruz, CA, USA - remaining annotations (based on data from Genoscope) </li> 
</ul> 
<p> 
<strong>UCSC Tetraodon Genome Browser:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=tetNig2" target="_blank">tetNig2</a>: Hiram Clawson, Mark 
  Diekhans, Brian Raney, Ann Zweig, Mary Goldman, Vanessa Swing, Antonio Coelho, Robert Kuhn, and 
  Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=tetNig1" target="_blank">tetNig1</a>: Rachel Harte, Robert 
  Kuhn, Donna Karolchik </li> 
</ul> 
<a name="tetraodon_use"></a> 
<h3>Data use</h3> 
<p> 
The Tetraodon genome data may be freely downloaded, used in analyses, and repackaged in databases if
the data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Yue GH, Lo LC, Zhu ZY, Lin G, Feng F.  
<a href="http://www.tandfonline.com/doi/full/10.1080/10425170600700378?scroll=top&needAccess=true" 
target="_blank">The complete nucleotide sequence of the mitochondrial genome of Tetraodon 
nigroviridis</a>. <em>DNA Seq</em>. 2006 Apr;17(2):115-21. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17076253" target="_blank">17076253</a></p>

<!-- Tibetan Frog Credits -->
<a name="tibetan_frog_credits"></a>
<h2>Tibetan frog genome</h2> 
<p>
<strong>Sequencing/Assembly:</strong> <a href="http://www.genomics.cn/en/index" 
target="_blank">The Beijing Genomics Institute</a>, Shenzhen, China </p>
<p> 
<strong>UCSC Tibetan Frog Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=nanPar1" target="_blank">nanPar1</a>: Hiram Clawson and 
  Jairo Navarro </li>
</ul> 
<a name="tibetan_frog_use"></a>
<h3>Data use</h3> 
<p> 
The Tibetan frog genome data may be freely downloaded, used in analyses, and repackaged in databases
if the data provider is properly acknowledged. Please cite this publication when using these 
data:</p> 
<p>
Sun YB, Xiong ZJ, Xiang XY, Liu SP, Zhou WW, Tu XL, Zhong L, Wang L, Wu DD, Zhang BL <em>et al</em>.
<a href="http://www.pnas.org/content/112/11/E1257.long" target="_blank">Whole-genome sequence of the
Tibetan frog Nanorana parkeri and the comparative evolution of tetrapod genomes</a>. 
<em>Proc Natl Acad Sci U S A</em>. 2015 Mar 17;112(11):E1257-62. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/25733869" target="_blank">25733869</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4371989/" target="_blank">PMC4371989</a></p>

<!-- Turkey Credits -->
<a name="turkey_credits"></a>
<h2>Turkey genome</h2> 
<p> 
<strong>Sequencing/Assembly: </strong> The Turkey Genome Consortium</a> - 
<a href="http://www.cals.vt.edu/" target="_blank">Virginia Tech College of Agriculture and Life 
Sciences</a> and <a href="http://www.vbi.vt.edu/" target="_blank">Virginia Bioinformatics 
Institute</a>, Blacksburg, VA, USA </p> 
<p> 
<strong>UCSC Turkey Genome Browser/Annotations:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=melGal1" target="_blank">melGal1</a>: Chin Li, Mark 
  Diekhans, Brian Raney, Brooke Rhead, Greg Roe, Steve Heitner, and Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=melGal5" target="_blank">melGal5</a>: Hiram Clawson and 
  Chris Villarreal </li>
</ul> 
<a name="turkey_use"></a>
<h3>Data use</h3> 
<p> 
The turkey genome data may be freely downloaded, used in analyses, and repackaged in databases if 
the data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Dalloul RA, Long JA, Zimin AV, Aslam L, Beal K, Blomberg Le Ann, Bouffard P, Burt DW, Crasta O, 
Crooijmans RP <em>et al.</em> 
<a href="http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1000475" 
target="_blank">Multi-platform next-generation sequencing of the domestic turkey (Meleagris 
gallopavo): genome assembly and analysis</a>. <em>PLoS Biol</em>. 2010 Sep 7;8(9). PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/20838655" target="_blank">20838655</a></p>

<!-- X. tropicalis Credits -->
<a name="xentro_credits"></a>
<h2>X. tropicalis genome</h2>
<h3>xenTro10</h3>
<p>
<strong>Sequencing/Assembly: </strong>
<a href="https://www.berkeley.edu/"
target="_blank">University of California, Berkeley</a>, Berkeley, CA, USA </p>
<p>
<strong>UCSC <em>X. tropicalis</em> Genome Browser/Annotations:</strong> </p>
<ul>
  <li>
  <a href="../../cgi-bin/hgGateway?db=xenTro10" target="_blank">xenTro10</a>: Hiram Clawson, Lou Nassar, and
  Gerardo Perez </li>
</ul></p>
<h3>xenTro9 and older</h3> 
<p> 
<strong>Sequencing/Assembly:</strong> 
<a href="http://genome.jgi-psf.org/Xentr4/Xentr4.home.html" target="_blank">Joint Genome 
Institute</a> (JGI), Walnut Creek, CA, USA </p>
<p> 
<strong>UCSC <em>X. tropicalis</em> Genome Browser/Annotations:</strong> </p> 
<ul>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=xenTro9" target="_blank">xenTro9</a>: Hiram Clawson and
  Christopher Lee</li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=xenTro7" target="_blank">xenTro7</a>: Hiram Clawson, Brian 
  Raney, and Matthew Speir </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=xenTro3" target="_blank">xenTro3</a>: Hiram Clawson, Brian 
  Raney, Brooke Rhead, Greg Roe, Steve Heitner, and Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=xenTro2" target="_blank">xenTro2</a>: Angie Hinrichs, Kayla 
  Smith, Robert Kuhn, and Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=xenTro1" target="_blank">xenTro1</a>: Fan Hsu, Jim Kent, 
  Heather Trumbower, Hiram Clawson, Brian Raney, Galt Barber, Mark Diekhans, Angie Hinrichs, and 
  Donna Karolchik </li> 
</ul> 
<a name="xentro_use"></a>
<h3>Data use</h3> 
<p> 
The <em>X. tropicalis</em> genome data may be freely downloaded, used in analyses, and repackaged in
databases if the data provider is properly acknowledged. Please cite this publication when using 
these data:</p> 
<p> 
Hellsten U, Harland RM, Gilchrist MJ, Hendrix D, Jurka J, Kapitonov V, Ovcharenko I, Putnam NH, 
Shu S, Taher L <em>et al</em>. <a href="http://www.sciencemag.org/content/328/5978/633.abs" 
target="_blank">The genome of the Western clawed frog Xenopus tropicalis</a>. <em>Science</em>. 
2010 Apr 30;328(5978):633-6.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20431018" 
target="_blank">20431018</a></p>

<!-- Zebra Finch Credits -->
<a name="zebra_finch_credits"></a>
<h2>Zebra finch genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> 
<a href="http://genome.wustl.edu/genomes/detail/taeniopygia-guttata/" target="_blank">The Genome 
Institute at WUSTL</a>, St. Louis, MO, USA </p> 
<p> 
<strong>Zebra finch DNA:</strong> Arthur P. Arnold lab, Dept. of Physiological Science, UCLA, Los 
Angeles, CA, USA </p>
<!-- Commented out until we verify this info 
<p> 
<strong>BAC library:</strong> Barbara Blackmon, Clemson University Genomics Institute, Clemson, 
SC, USA </p>
--> 
<p> 
<strong>UCSC Zebra Finch Genome Browser/Annotation:</strong> </p>
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=taeGut2" target="_blank">taeGut2</a>: Hiram Clawson, Brian 
  Raney, and Steve Heitner </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=taeGut1" target="_blank">taeGut1</a>: Brian Raney, Kayla 
  Smith, Pauline Fujita, and Donna Karolchik </li> 
</ul> 
<a name="zebra_finch_use"></a>
<h3>Data use</h3> 
<p> 
The zebra finch genome data may be freely downloaded, used in analyses, and repackaged in databases 
if the data provider is properly acknowledged. Please cite this publication when using these 
data:</p> 
<p> 
Warren WC, Clayton DF, Ellegren H, Arnold AP, Hillier LW, K&uuml;nstner A, Searle S, White S, 
Vilella AJ, Fairley S <em>et al</em>. 
<a href="http://www.nature.com/nature/journal/v464/n7289/full/nature08819.html" 
target="_blank">The genome of a songbird</a>. <em>Nature</em>. 2010 Apr 1;464(7289):757-62.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/20360741" target="_blank">20360741</a></p>

<!-- Zebrafish Credits -->
<a name="zebrafish_credits"></a>
<h2>Zebrafish genome</h2> 
<p> 
Starting with the <a href="../../cgi-bin/hgGateway?db=danRer10" target="_blank">danRer10</a> 
(GRCz10) assembly, the zebrafish genome sequence is now provided by the 
Genome Reference Consortium (GRC). See the GRC 
<a href="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/zebrafish/" 
target="_blank">Zebrafish Genome web page</a> for acknowledgments.</p>
<p>
<strong>Most Recent UCSC Zebrafish Genome Browser (<a href="../../cgi-bin/hgGateway?db=danRer11" 
target="_blank">danRer11</a>):</strong> Christopher Eisenhart, Hiram Clawson, Christopher Lee</p>

<hr>
<p>Prior assemblies have the following acknowledgments: </p> 
<p> 
<strong>Sequencing/Assembly:</strong> 
<a href="http://www.sanger.ac.uk/resources/zebrafish/genomeproject.html" target="_blank">The 
Wellcome Trust Sanger Institute</a>, UK. </p> 
<p> 
<strong>FPC map:</strong> </p>
<ul> 
  <li> 
  The Wellcome Trust Sanger Institute, UK </li>
  <li> 
  <a href="http://www.hubrecht.eu/" target="_blank">Hubrecht Laboratory</a>, Netherlands Institute
  for Developmental Biology, The Netherlands </li> 
  <li> 
  <a href="http://www.eb.tuebingen.mpg.de/" target="_blank">Max-Planck Institute for Developmental
  Biology</a>, Germany </li>
</ul> 
<p> 
<strong>BAC Clones, BAC End Pairs: </strong> </p>
<ul> 
  <li> 
  The Wellcome Trust Sanger Institute, UK - GenBank accessions for BAC clones </li>
  <li> 
  Max-Planck Institute for Developmental Biology, Germany - BAC end pairs sequence data </li>
  <li> 
  Hubrecht Laboratory</a>, Netherlands Institute for Developmental Biology, The Netherlands - BAC 
  end pairs sequence data </li> 
</ul> 
<p> 
<strong>Annotations:</strong> </p>
<ul> 
  <li> 
  Yi Zhou, Anthony DiBiase, Leonard Zon and the Zebrafish Genome Initiative at 
  <a href="https://www.childrenshospital.org/research/labs/zon-laboratory"
  target="_blank">Boston Children's Hospital</a>, Boston, MA,
  US - collaboration on Zebrafish WZ EST clusters, BAC End Singles/Pairs, Radiation Hybrid, and 
  Human tBLASTn annotation tracks </li>
  <li> 
  <a href="http://www.genetics.wustl.edu/fish_lab/frank/cgi-bin/fish/" target="_blank">Washington 
  University in St. Louis</a>, St.  Louis, MO, US - Zebrafish WZ EST clusters data </li>
  <li> 
  <a href="http://www.ensembl.org" target="_blank">Ensembl</a>, UK - Ensembl gene predictions</li>
  <li> 
  <a href="../staff.html">UCSC Genome Bioinformatics Group</a>, University of California, Santa 
  Cruz, CA, USA - Other annotations</li> 
</ul> 
<p> 
<strong>UCSC Zebrafish Genome Browsers:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=danRer11" target="_blank">danRer11</a>: Christopher Eisenhart, 
  Hiram Clawson, Christopher Lee</li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=danRer10" target="_blank">danRer10</a>: Hiram Clawson, 
  Brian Raney, Steve Heitner </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=danRer7" target="_blank">danRer7</a>: Hiram Clawson, Mary 
  Goldman, Greg Roe, Brian Raney, Donna Karolchik  </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=danRer6" target="_blank">danRer6</a>: Galt Barber, Robert 
  Kuhn, Katrina Learned, Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=danRer5" target="_blank">danRer5</a>: Rachel Harte, Ann 
  Zweig, Donna Karolchik </li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=danRer4" target="_blank">danRer4</a>: Rachel Harte, Archana 
  Thakkapallayil, Robert Kuhn, Ann Zweig, Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=danRer3" target="_blank">danRer3</a>: Rachel Harte, Jennifer
  Jackson, Ann Zweig, Ali Sultan-Qurraie, Donna Karolchik </li> 
  <li> 
  danRer2 (archived): Rachel Harte, Mark Diekhans, Heather Trumbower, Donna Karolchik, Jennifer 
  Jackson </li> 
</ul> 
<a name="zebrafish_use"></a> 
<h3>Data use</h3> 
<p> 
The zebrafish genome data may be freely downloaded, used in analyses, and repackaged in databases if
the data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Howe K, Clark MD, Torroja CF, Torrance J, Berthelot C, Muffato M, Collins JE, Humphray S, McLaren K,
Matthews L <em>et al</em>. 
<a href="http://www.nature.com/nature/journal/v496/n7446/full/nature12111.html" 
target="_blank">The zebrafish reference genome sequence and its relationship to the human 
genome</a>.  <em>Nature</em>. 2013 Apr 25;496(7446):498-503.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/23594743" target="_blank">23594743</a></p> 
<p> 
For more information about zebrafish assemblies provided by the GRC, see the 
<a href="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/zebrafish/" target="_blank">GRC 
zebrafish</a> web pages. For information about assemblies prior to danRer10/GRCz10, see the Sanger 
Institute's  <a href="http://www.sanger.ac.uk/resources/zebrafish/genomeproject.html" 
target="_blank">Zebrafish Sequencing Project</a> web page. </p>

<!-- Ciona Credits -->
<a name="ciona_credits"></a>
<h2>C. intestinalis genome</h2> 
<p> 
<strong>Project Funding</strong>: Work performed under the auspices of the US Department of 
Energy's Office of Science, Biological and Environmental Research Program and the by the University 
of California, Lawrence Livermore National Laboratory under Contract No. W-7405-Eng-48, Lawrence 
Berkeley National Laboratory under contract No. DE-AC03-76SF00098 and Los Alamos National Laboratory
under contract No. W-7405-ENG-36.</p> 
<p> 
<strong>Project Collaborators</strong>:</p> 
<ul> 
  <li> 
  <a href="http://genome.jgi.doe.gov/" target="_blank">Joint Genome Institute</a> (JGI), Walnut 
  Creek, CA, USA </li> 
  <li> 
  <a href="http://www.zool.kyoto-u.ac.jp/zool-e.html" target="_blank">Department of Zoology, 
  Graduate School of Science, Kyoto University</a>, Kyoto, Japan </li> 
  <li> 
  <a href="http://mcb.berkeley.edu/" target="_blank">Department of Molecular and Cellular Biology, 
  University of California, Berkeley</a>, CA, USA</li> 
  <li> 
  <a href="http://www.nig.ac.jp/english/index.html" target="_blank">National Institute of 
  Genetics</a>, Mishima, Japan </li> 
  <li> 
  <a href="http://www.ornl.gov/" target="_blank">Oak Ridge National Laboratory</a>, Oak Ridge, TN, 
  USA </li>
  <li> 
  <a href="http://www.lanl.gov/" target="_blank">Los Alamos National Laboratory</a>, Los Alamos, NM,
  USA </li> 
</ul> 
<p> 
<strong>Annotations:</strong> </p> 
<ul> 
  <li> 
  Colin Dewey, University of California, Berkeley, CA, USA - SNAP Gene Predictions, using the SNAP 
  parser written by Ian Korf </li>
  <li> 
  JGI, Walnut Creek, CA, USA - JGI Gene Predictions </li> 
  <li> 
  <a href="../staff.html">UCSC Genome Bioinformatics Group</a>, University of California, Santa 
  Cruz, CA, USA - Other annotations </li> 
</ul> 
<p> 
<strong>UCSC <em>C. intestinalis</em>  Genome Browsers:</strong> </p> 
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=ci3" target="_blank">ci3</a>: Hiram Clawson and Chris 
  Villarreal</li>
  <li> 
  <a href="../../cgi-bin/hgGateway?db=ci2" target="_blank">ci2</a>: Brian Raney, Mark Diekhans, Ann 
  Zweig, Kayla Smith, Robert Kuhn, Donna Karolchik </li> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=ci1" target="_blank">ci1</a>: Brian Raney, Galt Barber, 
  Heather Trumbower, Robert Kuhn, Donna Karolchik </li> 
</ul> 
<a name="ciona_use"></a>
<h3>Data use</h3> 
<p> 
The <em>C. intestinalis</em> sequence data have been freely provided by the JGI and may be freely 
downloaded, used in analyses, and repackaged in databases if the data provider is properly 
acknowledged. Please cite this publication when using these data:</p> 
<p>
Dehal P, Satou Y, Campbell RK, Chapman J, Degnan B, De Tomaso A, Davidson B, Di Gregorio A, Gelpke 
M, Goodstein DM <em>et al</em>.  <a href="http://www.sciencemag.org/content/298/5601/2157.abtract" 
target="_blank">The draft genome of Ciona intestinalis: insights into chordate and vertebrate 
origins</a>. <em>Science</em>. 2002 Dec 13;298(5601):2157-67.  PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/12481130" target="_blank">12481130</a></p> 
<p> 
For more information about the <em>C. intestinalis</em> assembly, see the JGI 
<a href="http://genome.jgi.doe.gov/Cioin2/Cioin2.home.html" target="_blank"><em>C. intestinalis</em>
portal</a>.  </p>

<!-- Lancelet Credits -->
<a name="lancelet_credits"></a>
<h2>Lancelet genome</h2> 
<p> 
<strong>Funding:</strong> U.S. Department of Energy Office of Science, Biological and Environmental 
Research Program </p> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://genome.jgi-psf.org/Brafl1/Brafl1.home.html" 
target="_blank">JGI</a> </p> 
<p> 
<strong>UCSC Lancelet Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li> 
  <a href="../../cgi-bin/hgGateway?db=braFlo1" target="_blank">braFlo1</a>: Hiram Clawson, Ann 
  Zweig, Pauline Fujita, Donna Karolchik </li> 
</ul> 
<a name="lancelet_use"></a>
<h3>Data use</h3> 
<p> 
The lancelet genome data may be freely downloaded, used in analyses, and repackaged in databases if 
the data provider is properly acknowledged. Please cite this publication when using these data:</p> 
<p> 
Putnam NH, Butts T, Ferrier DE, Furlong RF, Hellsten U, Kawashima T, Robinson-Rechavi M, Shoguchi E,
Terry A, Yu JK <em>et al</em>. 
<a href="http://www.nature.com/nature/journal/v453/n7198/full/nature06967.html" target="_blank">The 
amphioxus genome and the evolution of the chordate karyotype</a>. <em>Nature</em>. 
2008 Jun 19;453(7198):1064-71. PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18563158" 
target="_blank">18563158</a></p>

<!-- S. purpuratus Credits-->
<a name="urchin_credits"></a>
<h2>S. purpuratus genome</h2> 
<p> 
<strong>Funding: </strong> <a href="http://www.genome.gov" target="_blank">National Human Genome 
Research Institute</a> (NHGRI)</p> 
<p>
<strong>Sequencing/Assembly: </strong> <a href="https://www.hgsc.bcm.edu/" 
target="_blank">BCM-HGSC</a>, Houston, TX, USA</p> 
<p>
<strong>UCSC <em>S. purpuratus</em> Genome Browsers/Annotations:</strong> </p> 
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=strPur2" target="_blank">strPur2</a>: Kord Kober, Heather 
  Trumbower, Rachel Harte, Kayla Smith, Donna Karolchik </li> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=strPur1" target="_blank">strPur1</a>: Andy Pohl, Kayla Smith, 
  Donna Karolchik </li> 
</ul> 
<a name="urchin_use"></a>
<h3>Data use</h3> 
<p> 
The sea urchin genome data may be freely downloaded, used in analyses, and repackaged in databases 
if the data provider is properly acknowledged. Please cite this publication when using these 
data:</p> 
<p> 
Sea Urchin Genome Sequencing Consortium.  
<a href="http://www.sciencemag.org/content/314/5801/941.long" target=_blank>The genome of the sea 
urchin Strongylocentrotus purpuratus</a>.  <em>Science</em>. 2006 Nov 10;314(5801):941-52. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17095691" target="_blank">17095691</a></p> 
<p> 
For more information on the sea urchin genome project, see the BCM-HGSC 
<a href="https://www.hgsc.bcm.edu/other-invertebrates/sea-urchin-genome-project" 
target="_blank">Sea Urchin Genome Project</a> web page.</p>

<!-- A. gambiae Credits-->
<a name="anoGam_credits"></a>
<h2>A. gambiae genome</h2> 
<p>
The MOZ2 <em>Anopheles gambiae</em> data were provided by the International Anopheles Genome Project
and downloaded from Ensembl. The data have the following acknowledgments:</p> 
<p> 
<strong>Funding:</strong> </p>
<ul>
  <li>
  <a href="http://www.niaid.nih.gov/Pages/default.aspx" target="_blank">National Institute of 
  Allergy and Infectious Diseases</a> (NIAID)</li>
  <li>
  <a href="http://www.enseignementsup-recherche.gouv.fr/" target="_blank">French Ministry of 
  Research</a> </li> 
  <li>
  <a href="http://www.who.int/en/" target="_blank">World Health Organization</a> (WHO)</li>
</ul>
<p>
Celera Genomics and Genoscope were funded respectively by <a href="http://www.nih.gov/" 
target="_blank">National Institutes of Health</a> (NIH) and the French Ministry of
Research.</p> 
<p> 
<strong>Project collaboration:</strong> The International Anopheles Genome Project - a 
collaboration between: </p> 
<ul>
  <li> 
  <a href="https://www.celera.com/" target="_blank">Celera Genomics</a> </li>
  <li>
  <a href="http://www.genoscope.cns.fr/spip/spip.php?lang=en" target="_blank">Genoscope</a> </li>
  <li> 
  <a href="http://www.nd.edu/" target="_blank">University of Notre Dame</a> </li> 
  <li> 
  <a href="http://www.ebi.ac.uk/" target="_blank">EBI</a>/<a href="http://www.sanger.ac.uk/" 
  target="_blank">Sanger Institute</a> </li>
  <li>
  <a href="http://www.embl.org/" target="_blank">EMBL</a> </li> 
  <li>
  <a href="http://www.pasteur.fr/en" target="_blank">Institut Pasteur</a> </li>
  <li>
  <a href="http://www.imbb.forth.gr/" target="_blank">The Institute of Molecular Biology and 
  Biotechnology</a> (IMBB) </li>
  <li>
  <a href="http://www.jcvi.org/cms/home/" target="_blank">The Institute for Genomic Research</a> 
  (TIGR) </li>
</ul> 
<p> 
<strong>Sequencing and Assembly:</strong> </p>
<ul>
  <li> 
  <a href="https://www.celera.com/" target="_blank">Celera Genomics</a> </li>
  <li>
  <a href="http://www.genoscope.cns.fr/spip/spip.php?lang=en" target="_blank">Genoscope</a> </li> 
</ul> 
<p> 
<strong>Data Downloads:</strong> <a href="http://metazoa.ensembl.org/Anopheles_gambiae/Info/Index" 
target="_blank">Ensembl</a> </p>
<p> 
<strong>UCSC <em>A. gambaie</em> Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=anoGam1" target="_blank">anoGam1</a>: Angie Hinrichs, Galt 
  Barber, Donna Karolchik </li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=anoGam3" target="_blank">anoGam3</a>: Hiram Clawson, Luis Nassar </li>
</ul> 
<p> 
<strong>Additional Genome Browser Annotations</strong>: </p> 
<ul>
  <li> 
  <a href="http://www.ensembl.org" target="_blank">Ensembl</a> </li>
  <li>
  <a href="https://www.celera.com/" target="_blank">Celera Genomics</a> (Ensembl Genes) </li> 
</ul>
<a name="anoGam_use"></a>
<h3>Data use</h3> 
<p> 
The <em>A. gambaie</em> sequence is made freely available by the International Anopheles Genome 
Project. Please cite the following publication when using these data: </p> 
<p> 
Holt RA, Subramanian GM, Halpern A, Sutton GG, Charlab R, Nusskern DR, Wincker P, Clark AG, Ribeiro 
JM, Wides R <em>et al</em>.  <a href="http://www.sciencemag.org/content/298/5591/129.abstract" 
target=_blank>The genome sequence of the malaria mosquito Anopheles gambiae</a>. <em>Science</em>. 
2002 Oct 4;298(5591):129-49.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12364791" 
target="_blank">12364791</a></p>

<!-- A. mellifera Credits-->
<a name="apiMel_credits"></a>
<h2>A. mellifera genome</h2> 
<p> 
<strong>Funding: </strong></p>
<ul>
  <li>
  <a href="http://www.genome.gov" target="_blank">National Human Genome Research Institute</a> 
  (NHGRI) </li>
  <li>
  <a href="http://www.usda.gov/wps/portal/usda/usdahome" target="_blank">U.S. Department of 
  Agriculture</a> (USDA) </li> 
</ul>
<p> 
<strong>Sequencing and assembly:</strong> 
<a href="https://www.hgsc.bcm.edu/arthropods/honey-bee-genome-project" target="_blank">BCM-HGSC</a>,
Houston, TX, USA </p> 
<p> 
<strong>BAC library:</strong> Pieter de Jong and Katzutoyo Osoegawa at 
<a href="http://bacpac.chori.org/" target="_blank">Children's Hospital Oakland Research 
Institute</a> (CHORI), Oakland, CA, USA </p> 
<p> 
<strong>UCSC <em>A. mellifera</em> Genome Browser/Annotations:</strong> </p> 
<ul> 
  <li>
  <a href="../../cgi-bin/hgGateway?db=apiMel2" target="_blank">apiMel2</a>: Andy Pohl, Angie 
  Hinrichs, Jennifer Jackson, Donna Karolchik </li>
  <li>
  <a href="../../cgi-bin/hgGateway?db=apiMel1" target="_blank">apiMel1</a>: Angie Hinrichs, Brian 
  Raney, Robert Kuhn, Donna Karolchik </li> 
</ul> 
<a name="apiMel_use"></a>
<h3>Data use</h3> 
<p> 
The <em>A. mellifera</em> sequence is made freely available by the Honeybee Genome Sequencing 
Consortium. Please cite the following publication when using these data: </p> 
<p> 
Honeybee Genome Sequencing Consortium. 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2048586/" target="_blank">Insights into social 
insects from the genome of the honeybee Apis mellifera</a>. <em>Nature</em>. 
2006 Oct 26;443(7114):931-49.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17073008" 
target="_blank">17073008</a></p> 
<p> 
See the Baylor <a href="https://www.hgsc.bcm.edu/bcm-hgsc-conditions-use" target="_blank">conditions
of use</a> statement for guidelines regarding the use of these data. </p>

<!-- D. ananassae Credits -->
<a name="droAna_credits"></a>
<h2>D. ananassae genome</h2> 
<p>
The droAna2 data were provided by Agencourt Bioscience. The droAna1 data were produced by the 
Institute for Genomic Research (TIGR). 
<p> 
<strong>Sequencing:</strong> <a href="http://www.beckmangenomics.com/" target="_blank">Agencourt 
Bioscience</a>, Danvers, MA, USA </p> 
<p> 
<strong>Assembly:</strong> </p>
<ul> 
  <li> 
  <a href="http://www.beckmangenomics.com/" target="_blank">Agencourt Bioscience</a>, Danvers, MA, 
  USA, using the 
  <a href="http://www.broadinstitute.org/science/programs/genome-biology/computational-rd/computational-research-and-development" 
  target="_blank">Arachne</a> assembler (droAna2)</li> 
  <li> 
  <a href="http://www.jcvi.org/cms/home/" target="_blank">TIGR</a>, Rockville, MD, USA, using an 
  assembler produced by <a href="https://www.celera.com/" target="_blank">Celera</a> (droAna1)</li> 
</ul> 
<p> 
<strong>UCSC <em>A. mellifera</em> Genome Browser/Annotations:</strong> </p> 
<ul>
  <li>
  <strong> <a href="../../cgi-bin/hgGateway?db=droAna2" target="_blank">droAna2</a>:</strong> Angie 
  Hinrichs, Brian Raney, Ann Zweig, Kayla Smith, and Donna Karolchik </li> 
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=droAna1" target="_blank">droAna1</a>:</strong> Angie 
  Hinrichs, Brian Raney, Heather Trumbower, Ali Sultan-Qurraie, and Donna Karolchik </li> 
</ul>
<a name="droAna_use"></a>
<h3>Data use</h3> 
<p> 
The <em>D. ananassae</em> sequence may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite these publications when using 
these data:</p> 
<p>
Drosophila 12 Genomes Consortium. 
<a href="https://www.nature.com/nature/journal/v450/n7167/full/nature06341.html" 
target="_blank">Evolution of genes and genomes on the Drosophila phylogeny</a>. <em>Nature</em>. 
2007 Nov 8;450(7167):203-18.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17994087" 
target="_blank">17994087</a></p> 
<p> 
Zimin AV, Smith DR, Sutton G, Yorke JA. 
<a href="https://academic.oup.com/bioinformatics/article-abstract/24/1/42/205726/Assembly-reconciliation" 
target="_blank">Assembly reconciliation</a>. <em>Bioinformatics</em>.  2008 Jan 1;24(1):42-5. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18057021" target="_blank">18057021</a></p>

<!-- D. erecta Credits -->
<a name="droEre_credits"></a>
<h2>D. erecta genome</h2> 
<p> 
<strong>Sequencing and Assembly:</strong> <a href="http://www.beckmangenomics.com/" 
target="_blank">Agencourt Bioscience</a>, Danvers, MA, USA, using the 
<a href="http://www.broadinstitute.org/science/programs/genome-biology/computational-rd/computational-research-and-development" 
target="_blank">Arachne</a> assembler </p> 
<p> 
<strong>UCSC <em>D. erecta</em> Genome Browser/Annotations: </strong> 
<ul>
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=droEre1" target="_blank">droEre1</a>:</strong> Angie 
  Hinrichs, Brian Raney, Jennifer Jackson and Donna Karolchik </li>
</ul>
<a name="droEre_use"></a>
<h3>Data use</h3> 
<p> 
The <em>D. erecta</em> sequence may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite these publications when using 
these data:</p> 
<p>
Drosophila 12 Genomes Consortium. 
<a href="https://www.nature.com/nature/journal/v450/n7167/full/nature06341.html" 
target="_blank">Evolution of genes and genomes on the Drosophila phylogeny</a>. <em>Nature</em>. 
2007 Nov 8;450(7167):203-18.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17994087" 
target="_blank">17994087</a></p> 
<p> 
Zimin AV, Smith DR, Sutton G, Yorke JA. 
<a href="https://academic.oup.com/bioinformatics/article-abstract/24/1/42/205726/Assembly-reconciliation" 
target="_blank">Assembly reconciliation</a>. <em>Bioinformatics</em>. 2008 Jan 1;24(1):42-5. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18057021" target="_blank">18057021</a></p>

<!-- D. grimshawi Credits -->
<a name="droGri_credits"></a>
<h2>D. grimshawi genome</h2> 
<p> 
<strong>Sequencing and Assembly:</strong> <a href="http://www.beckmangenomics.com/" 
target="_blank">Agencourt Bioscience</a>, Danvers, MA, USA, using the 
<a href="http://www.broadinstitute.org/science/programs/genome-biology/computational-rd/computational-research-and-development" 
target="_blank">Arachne</a> assembler </p> 
<p> 
<strong>UCSC <em>D. grimshawi</em> Genome Browser/Annotations:</strong> </p> 
<ul>
  <li>
  <strong> <a href="../../cgi-bin/hgGateway?db=droGri1" target="_blank">droGri1</a>:</strong> Angie 
  Hinrichs, Brian Raney, Jennifer Jackson, Kayla Smith and Donna Karolchik </li>
</ul>
<a name="droGri_use"></a>
<h3>Data use</h3> 
<p> 
The <em>D. grimshawi</em> sequence may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged.  Please cite these publications when using 
these data:</p> 
<p>
Drosophila 12 Genomes Consortium. 
<a href="https://www.nature.com/nature/journal/v450/n7167/full/nature06341.html" 
target="_blank">Evolution of genes and genomes on the Drosophila phylogeny</a>. <em>Nature</em>. 
2007 Nov 8;450(7167):203-18.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17994087" 
target="_blank">17994087</a></p> 
<p> 
Zimin AV, Smith DR, Sutton G, Yorke JA. 
<a href="https://academic.oup.com/bioinformatics/article-abstract/24/1/42/205726/Assembly-reconciliation" 
target="_blank">Assembly reconciliation</a>. <em>Bioinformatics</em>. 2008 Jan 1;24(1):42-5. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18057021" target="_blank">18057021</a></p>

<!-- D. melanogaster Credits -->
<a name="dm_credits"></a>
<h2>D. melanogaster genome</h2> 
<p>
The <em>D. melanogaster</em> assemblies displayed in the UCSC Genome Browser were obtained from the 
<a href="http://www.fruitfly.org/" target="_blank">Berkeley Drosophila Genome Project</a> (BDGP) 
with the following acknowledgments:</p> 
<p> 
<strong>Sequencing and Assembly:</strong></p> 
<ul> 
  <li>
  Euchromatic sequence - BDGP, Berkeley, CA, USA </li>
  <li>
  Heterochromatic sequence - <a href="http://www.fruitfly.org/" target="_blank">Drosophila 
  Heterochromatin Genome Project</a>, Lawrence Berkeley National Laboratory, Berkeley, CA, USA</li> 
</ul> 
<p>
<strong>Annotations: </strong> </p>
<ul> 
  <li>
  Euchromatic regions - <a href="http://flybase.org/" target="_blank">FlyBase</a> </li>
  <li>
  Heterochromatic regions - DHGP, Lawrence Berkeley National Laboratory, Berkeley, CA, USA </li>
  <li> 
  <a href="../staff.html">UCSC Genome Bioinformatics Group</a>, UCSC, Santa Cruz, CA, USA </li>
</ul> 
<p><strong>UCSC <em>D. melanogaster</em> Genome Browser: </strong> </p>
<ul>
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=dm6" target="_blank">dm6</a>: </strong> Hiram Clawson,
  Angie Hinrichs, and Matthew Speir </li>
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=dm3" target="_blank">dm3</a>: </strong> Angie 
  Hinrichs, Archana Thakkapallayil, Kayla Smith, and Donna Karolchik </li>
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=dm2" target="_blank">dm2</a>: </strong> Angie 
  Hinrichs, Brian Raney, Galt Barber, and Donna Karolchik </li>
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=dm1" target="_blank">dm1</a>: </strong> Angie 
  Hinrichs, Brian Raney, Heather Trumbower, Ali Sultan-Qurraie, and Donna Karolchik </li> 
</ul>
<a name="dm_use"></a>
<h3>Data use</h3> 
<p> 
For additional information about these data, including citation guidelines, see the 
<a href="http://www.fruitfly.org" target="_blank">BDGP web site</a>. </p>

<!-- D. mojavensis Credits -->
<a name="droMoj_credits"></a>
<h2>D. mojavensis genome</h2> 
<p> 
<strong>Sequencing and Assembly:</strong> <a href="http://www.beckmangenomics.com/" 
target="_blank">Agencourt Bioscience</a>, Danvers, MA, USA, using the 
<a href="http://www.broadinstitute.org/science/programs/genome-biology/computational-rd/computational-research-and-development" 
target="_blank">Arachne</a> assembler </p> 
<p> 
<strong>Funding:</strong> <a href="http://www.genome.gov" target="_blank">National Human Genome 
Research Institute</a> (NHGRI), USA </p> 
<p> 
<strong>UCSC <em>D. mojavensis</em> Genome Browser/Annotations: </strong>
<ul>
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=droMoj2" target="_blank">droMoj2</a>:</strong> Angie 
  Hinrichs, Brian Raney, Jennifer Jackson and Donna Karolchik </li> 
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=droMoj1" target="_blank">droMoj1</a>:</strong> Angie 
  Hinrichs, Brian Raney, Galt Barber and Donna Karolchik </li> 
</ul> 
<a name="droMoj_use"></a>
<h3>Data use</h3> 
<p> 
The <em>D. mojavensis</em> sequence may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite these publications when using 
these data:</p> 
<p>
Drosophila 12 Genomes Consortium. 
<a href="https://www.nature.com/nature/journal/v450/n7167/full/nature06341.html" 
target="_blank">Evolution of genes and genomes on the Drosophila phylogeny</a>. <em>Nature</em>. 
2007 Nov 8;450(7167):203-18.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17994087" 
target="_blank">17994087</a></p> 
<p> 
Zimin AV, Smith DR, Sutton G, Yorke JA. 
<a href="https://academic.oup.com/bioinformatics/article-abstract/24/1/42/205726/Assembly-reconciliation" 
target="_blank">Assembly reconciliation</a>. <em>Bioinformatics</em>. 2008 Jan 1;24(1):42-5. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18057021" target="_blank">18057021</a></p>

<!-- D. persimilis Credits -->
<a name="droPer_credits"></a>
<h2>D. persimilis genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://www.broadinstitute.org/" target="_blank">The 
Broad Institute</a>, Cambridge, MA, USA </p> 
<p> 
<strong>UCSC <em>D. persimilis</em> Genome Browser/Annotations: </strong> </p>
<ul>
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=droPer1" target="_blank">droPer1</a>:</strong> Angie 
  Hinrichs, Kayla Smith, Robert Kuhn, Jennifer Jackson and Donna Karolchik </li> 
</ul> 
<a name="droPer_use"></a>
<h3>Data use</h3> 
<p> 
The <em>D. persimilis</em> sequence may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite the following publications when
using these data:</p> 
<p>
Drosophila 12 Genomes Consortium. 
<a href="https://www.nature.com/nature/journal/v450/n7167/full/nature06341.html" 
target="_blank">Evolution of genes and genomes on the Drosophila phylogeny</a>. <em>Nature</em>. 
2007 Nov 8;450(7167):203-18.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17994087" 
target="_blank">17994087</a></p> 
<p> 
Zimin AV, Smith DR, Sutton G, Yorke JA. 
<a href="https://academic.oup.com/bioinformatics/article-abstract/24/1/42/205726/Assembly-reconciliation" 
target="_blank">Assembly reconciliation</a>. <em>Bioinformatics</em>. 2008 Jan 1;24(1):42-5. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18057021" target="_blank">18057021</a></p>

<!-- D. pseudoobscura Credits -->
<a name="dp_credits"></a>
<h2>D. pseudoobscura genome</h2> 
<p> 
<strong>Sequencing and Assembly:</strong> 
<a href="https://www.hgsc.bcm.edu/arthropods/drosophila-genome-project" 
target="_blank">BCM-HGSC</a>, Houston, TX, USA </p> 
<p> 
<strong>UCSC <em>D. pseudoobscura</em> Genome Browser/Annotations: </strong> </p>
<ul>
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=dp3" target="_blank">dp3</a>:</strong> Angie 
  Hinrichs, Brian Raney, Jennifer Jackson and Donna Karolchik </li> 
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=dp2" target="_blank">dp2</a>:</strong> Angie 
  Hinrichs, Heather Trumbower, Robert Kuhn, Donna Karolchik </li>
</ul>
<a name="dp_use"></a> 
<a name="droPse_use"></a>
<h3>Data use</h3> 
<p> 
The <em>D. pseudoobscura</em> data are made available with specific 
<a href="https://www.hgsc.bcm.edu/bcm-hgsc-conditions-use" target="_blank">conditions for use</a>. 
Please cite the following publications:</p> 
<p> 
Richards S, Liu Y, Bettencourt BR, Hradecky P, Letovsky S, Nielsen R, Thornton K, Hubisz MJ, Chen R,
Meisel RP <em>et al</em>.  <a href="http://genome.cshlp.org/content/15/1/1.long" 
target="_blank">Comparative genome sequencing of Drosophila pseudoobscura: chromosomal, gene, and 
cis-element evolution</a>.  <em>Genome Res</em>. 2005 Jan;15(1):1-18. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/15632085" target="_blank">15632085</a>; PMCID: 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC540289/" target="_blank">PMC540289</a></p> 
<p> 
Drosophila 12 Genomes Consortium. 
<a href="https://www.nature.com/nature/journal/v450/n7167/full/nature06341.html" 
target="_blank">Evolution of genes and genomes on the Drosophila phylogeny</a>. <em>Nature</em>. 
2007 Nov 8;450(7167):203-18.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17994087" 
target="_blank">17994087</a></p>

<!-- D. sechellia Credits -->
<a name="droSec_credits"></a>
<h2>D. sechellia genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://www.broadinstitute.org/" target="_blank">The 
Broad Institute</a>, Cambridge, MA USA </p>
<p> 
<strong>UCSC <em>D. sechellia</em> Genome Browser/Annotations: </strong> </p>
<ul>
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=droSec1" target="_blank">droSec1</a>:</strong> Angie 
  Hinrichs, Kayla Smith and Donna Karolchik </li>
</ul> 
<a name="droSec_use"></a>
<h3>Data use</h3> 
<p> 
The <em>D. sechellia</em> sequence may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite the following publications when
using these data:</p>
<p>
Drosophila 12 Genomes Consortium. 
<a href="https://www.nature.com/nature/journal/v450/n7167/full/nature06341.html" 
target="_blank">Evolution of genes and genomes on the Drosophila phylogeny</a>. <em>Nature</em>. 
2007 Nov 8;450(7167):203-18.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17994087" 
target="_blank">17994087</a></p> 
<p> 
Zimin AV, Smith DR, Sutton G, Yorke JA. 
<a href="https://academic.oup.com/bioinformatics/article-abstract/24/1/42/205726/Assembly-reconciliation" 
target="_blank">Assembly reconciliation</a>. <em>Bioinformatics</em>. 2008 Jan 1;24(1):42-5. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18057021" target="_blank">18057021</a></p>

<!-- D. simulans Credits -->
<a name="droSim_credits"></a>
<h2>D. simulans genome</h2> 
<p> 
<strong>Sequencing and assembly:</strong> 
<a href="http://genome.wustl.edu/genomes/detail/drosophila-simulans-white-501/" target="_blank">The 
Genome Institute at Washington University (WUSTL)</a>, St. Louis, MO, USA </p> 
<p> 
<strong>UCSC <em>D. simulans</em> Genome Browser/Annotations: </strong> </p>
<ul>
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=droSim1" target="_blank">droSim1</a></strong>: Angie 
  Hinrichs, Jennifer Jackson, Ali Sultan-Qurraie, Brian Raney and Donna Karolchik </li>
</ul>
<a name="droSim_use"></a>
<h3>Data use</h3> 
<p> 
The <em>D. simulans</em> sequence may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite the following publications when
using these data:</p> 
<p>
Drosophila 12 Genomes Consortium. 
<a href="https://www.nature.com/nature/journal/v450/n7167/full/nature06341.html" 
target="_blank">Evolution of genes and genomes on the Drosophila phylogeny</a>. <em>Nature</em>. 
2007 Nov 8;450(7167):203-18.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17994087" 
target="_blank">17994087</a></p> 

<!-- D. virilis Credits -->
<a name="droVir_credits"></a>
<h2>D. virilis genome</h2> 
<p> 
<strong>Sequencing and Assembly:</strong> <a href="http://www.beckmangenomics.com/" 
target="_blank">Agencourt Bioscience</a>, Danvers, MA, USA, using the 
<a href="http://www.broadinstitute.org/science/programs/genome-biology/computational-rd/computational-research-and-development" 
target="_blank">Arachne</a> assembler </p> 
<p> 
<strong>Funding:</strong> <a href="http://www.genome.gov" target="_blank">National Human Genome 
Research Institute</a> (NHGRI), USA </p> 
<p> 
<strong>UCSC <em>D. virilis</em> Genome Browser/Annotations: </strong> </p>
<ul>
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=droVir2" target="_blank">droVir2</a>:</strong> Angie 
  Hinrichs, Brian Raney, Jennifer Jackson and Donna Karolchik </li> 
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=droVir1" target="_blank">droVir1</a>:</strong> Angie 
  Hinrichs, Brian Raney, Heather Trumbower, Robert Kuhn, Jennifer Jackson, and Donna Karolchik</li>
</ul>
<a name="droVir_use"></a>
<h3>Data use</h3> 
<p> 
The <em>D. virilis</em> sequence may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite the following publications when
using these data:</p> 
<p>
Drosophila 12 Genomes Consortium. 
<a href="https://www.nature.com/nature/journal/v450/n7167/full/nature06341.html" 
target="_blank">Evolution of genes and genomes on the Drosophila phylogeny</a>. <em>Nature</em>. 
2007 Nov 8;450(7167):203-18.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17994087" 
target="_blank">17994087</a></p> 
<p> 
Zimin AV, Smith DR, Sutton G, Yorke JA. 
<a href="https://academic.oup.com/bioinformatics/article-abstract/24/1/42/205726/Assembly-reconciliation" 
target="_blank">Assembly reconciliation</a>. <em>Bioinformatics</em>. 2008 Jan 1;24(1):42-5. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18057021" target="_blank">18057021</a></p>

<!-- D. yakuba Credits -->
<a name="droYak_credits"></a>
<h2>D. yakuba genome</h2> 
<p> 
<strong>Funding:</strong> <a href="http://www.genome.gov/" target="_blank">National Human Genome 
Research Institute</a> (NHGRI) </p> 
<p> 
<strong>Sequencing, assembly and AGP creation:</strong> 
<a href="http://genome.wustl.edu/genomes/detail/drosophila-yakuba/" target="_blank">The Genome 
Institute at Washington University (WUSTL)</a> </p> 
<p> 
<strong>Initial Sequencing Proposal, Project Collaboration:</strong> David J. Begun and Charles H. 
Langley, <a href="http://www-eve.ucdavis.edu/" target="_blank">University of California, Davis</a>, 
CA, USA </p> 
<p> 
<strong>UCSC <em>D. yakuba</em> Genome Browser/Annotations: </strong> </p>
<ul>
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=droYak2" target="_blank">droYak2</a></strong>: Angie 
  Hinrichs, Jennifer Jackson and Donna Karolchik </li>
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=droYak1" target="_blank">droYak1</a></strong>: Angie 
  Hinrichs, Michael Chalup and Donna Karolchik </li>
</ul>
<a name="droYak_use"></a>
<h3>Data use</h3> 
<p> 
The <em>D. yakuba</em> sequence may be freely downloaded, used in analyses, and repackaged in 
databases if the data provider is properly acknowledged. Please cite the following publications when
using these data:</p> 
<p>
Drosophila 12 Genomes Consortium. 
<a href="https://www.nature.com/nature/journal/v450/n7167/full/nature06341.html" 
target="_blank">Evolution of genes and genomes on the Drosophila phylogeny</a>. <em>Nature</em>. 
2007 Nov 8;450(7167):203-18.  PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17994087" 
target="_blank">17994087</a></p> 
<p> 
Zimin AV, Smith DR, Sutton G, Yorke JA. 
<a href="https://academic.oup.com/bioinformatics/article-abstract/24/1/42/205726/Assembly-reconciliation" 
target="_blank">Assembly reconciliation</a>. <em>Bioinformatics</em>. 2008 Jan 1;24(1):42-5. PMID: 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18057021" target="_blank">18057021</a></p>

<!-- C. elegans Credits -->
<a name="ce_credits"></a>
<h2>C. elegans genome</h2> 
<a name="ce_use"></a> 
<p>
The <em>C. elegans</em> data was obtained from <a href="http://www.wormbase.org/" 
target="_blank">WormBase</a>. We'd like to thank 
<a href="http://genome.wustl.edu/" target="_blank">The Genome Institute at Washington University 
(WUSTL)</a> and the <a href="http://www.sanger.ac.uk/" target="_blank">Sanger Institute</a> for 
their collaborative work on sequencing the <em>C. elegans</em> genome, and the WormBase consortium 
for providing access to the current <em>C. elegans</em> sequence. </p>

<!-- Misc Worm Credits -->
<a name="worm_credits"></a>
<h2>Worm Genomes (other than C. elegans)</h2> 
<p>
The latest versions of the <em>C. brenneri</em>, <em>C. briggsae</em>, <em>C. japonica</em>, 
<em>C. remanei</em>, and <em>P. pacificus</em> sequences were obtained from the Genome Sequencing 
Center at <a href="http://genome.wustl.edu/genomes/category/invertebrates/" target="_blank">The 
Genome Institute at Washington University (WUSTL)</a>. The cb1 browser data were obtained from 
<a href="http://www.wormbase.org/" target="_blank">WormBase</a>.</p> 
<p> 
We'd like to thank WUSTL for providing the sequence data for these assemblies, as well as the 
Sanger Institute for their collaborative work in sequencing the initial <em>C. briggsae</em> cb1 
genome. Thanks also to the WormBase consortium for providing access to the cb1 sequence. </p> 
<a name="worm_use"></a>
<h3>Data use</h3> 
<p> 
The worm sequences are made freely available with the following understanding: </p>
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li> 
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The centers producing the data reserve the right to publish the initial large-scale analyses of 
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a 
  chromosome. </li> 
  <li> 
  This is in accordance with, and with the understandings in the <a href="ftLauderdale.pdf">Fort 
  Lauderdale meeting</a> discussing Community Resource Projects and the resulting 
  <a href="http://www.genome.gov/page.cfm?pageID=10506537" target="_blank">NHGRI policy 
  statement</a>. </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Sacchromyces cerevisiae Credits -->
<a name="yeast_credits"></a>
<h2>Yeast genome</h2> 
<p> 
The April 2011 <em>Sacchromyces cerevisiae</em> genome assembly 
(<a href="../../cgi-bin/hgGateway?db=sacCer3" target="_blank">sacCer3</a>) is based on sequence 
dated April 2011 in the <a href="http://www.yeastgenome.org/" target="_blank">Sacchromyces Genome 
Database</a> (SGD). This genome sequence was downloaded from 
<a href="ftp://ftp.ncbi.nlm.nih.gov/genbank/genomes/Eukaryotes/fungi/Saccharomyces_cerevisiae/R64-1-1/" 
target="_blank">NCBI</a> 
(genbank/genomes/Eukaryotes/fungi/Saccharomy ces_cerevisiae/SacCer_Apr2011/). The SGD gene 
annotations were downloaded from the 
<a href="http://downloads.yeastgenome.org/curation/chromosomal_feature/" target="_blank" >SGD 
downloads site</a>. The S288C strain was used in this sequencing project. </p>
<p> 
The June 2008 <em>Sacchromyces cerevisiae</em> genome assembly 
(<a href="../../cgi-bin/hgGateway?db=sacCer2" target="_blank">sacCer2</a>) is based on 
SGD sequence dated June 2008, and was obtained from the site 
http://downloads.yeastgenome.org/sequence/genomic_sequence/chromosomes/fasta/. The S288C strain was 
used in this sequencing project. </p> 
<p> 
The Oct. 2003 <em>Sacchromyces cerevisiae</em> genome assembly 
(<a href="../../cgi-bin/hgGateway?db=sacCer1" target="_blank">sacCer1</a>) is based on 
SGD sequence dated 1 Oct. 2003. The sequence, open reading frame (ORF), and gene annotations were 
downloaded from the site <a href="http://www.yeastgenome.org/download-data/sequence" 
target="_blank">http://www.yeastgenome.org/download-data/sequence</a>. The cellular localization and
protein abundance data displayed in the UCSC Gene Sorter are taken from the 
<a href="http://yeastgfp.yeastgenome.org/" target="_blank">Yeast GFP Fusion Localization 
Database</a>. </p>
<p>
<strong>UCSC <em>Sacchromyces cerevisiae</em> Genome Browser/Annotations: </strong> </p>
<ul>
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=sacCer3" target="_blank">sacCer3</a>:</strong> Hiram 
  Clawson, Greg Roe, Steve Heitner </li>
</ul>
<a name="yeast_use"></a>
<h3>Data use</h3> 
<p>
See the SGD <a href="http://www.yeastgenome.org/cache/chromosomes.shtml" target="_blank">Systematic 
Sequencing Table</a> for credit and contact information for each of the chromosomes in the assembly.
We'd like to thank Stanford University, the SGD, the University of California San Francisco (UCSF), 
Washington University in St. Louis, and The Broad Institute for providing the data and annotations 
for these browsers.</p>

<!-- California Sea Hare Credits -->
<a name="seahare_credits"></a>
<h2>California sea hare genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://www.broadinstitute.org/" target="_blank">The 
Broad Institute</a>, Cambridge, MA, USA </p> 
<p> 
<strong>UCSC California Sea Hare Genome Browser/Annotations: </strong> </p>
<ul>
  <li>
  <strong><a href="../../cgi-bin/hgGateway?db=aplCal1" target="_blank">aplCal1</a>:</strong> Galt 
  Barber, Brian Raney, Hiram Clawson, Katrina Learned and Donna Karolchik </li>
</ul> 
<a name="seahare_use"></a>
<h3>Data use</h3> 
<p> 
The California sea hare sequence is made freely available with the following understanding: </p> 
<ul> 
  <li> 
  The data may be freely downloaded, used in analyses, and repackaged in databases. </li>
  <li> 
  Users are free to use the data in scientific papers analyzing particular genes and regions if the 
  provider of these data is properly acknowledged. </li> 
  <li> 
  The center producing the data reserves the right to publish the initial large-scale analyses of 
  the data set, including large-scale identification of regions of evolutionary conservation and 
  large-scale genomic assembly. Large-scale refers to regions with size on the order of a 
  chromosome (that is, 30 Mb or more). </li> 
  <li> 
  Any redistribution of the data should carry this notice. </li> 
</ul>

<!-- Ebola Virus Credits -->
<a name="ebola_credits"></a>
<h2>Ebola virus genome</h2> 
<p> 
<strong>Sequencing/Assembly:</strong> <a href="http://www.broadinstitute.org/" target="_blank">The 
Broad Institute</a>, Cambridge, MA, USA </p> 
<p> 
<strong>UCSC Ebola virus Genome Browser/Annotations: </strong> </p>
<ul>
  <li> 
  <strong><a href="../../cgi-bin/hgGateway?db=eboVir3" target="_blank">eboVir3</a>:</strong> Hiram 
  Clawson, Brian Raney, Maximillian Haeussler, Angie Hirichs, Kate Rosenbloom, Chris Eisenhart, 
  Pauline Fujita, Matthew Speir, Donna Karolchik, Ann Zweig, David Haussler, and Jim Kent </li>
</ul>
<p>
<strong>Other Acknowledgements:</strong> Thanks to <a href="http://cbse.soe.ucsc.edu/people/berman" 
target="_blank">Phillip Berman</a> and his lab at UCSC for providing continued feedback on the Ebola
virus Genome Browser, and to the <a href="http://www.sabetilab.org/" target="_blank">Pardis Sabeti 
Lab</a> at the Broad Institute for their guidance.</p> 
<p> 
Thank you to the following institutions for proving additional annotations on the Ebola virus 
genome: </p>
<ul> 
  <li>
  <a href="https://www.ncbi.nlm.nih.gov/" target="_blank">National Center for Biotechnology 
  Information</a> (NCBI)</li> 
  <li>
  <a href="http://www.iedb.org/" target="_blank">Immune Epitope Database</a> (IEDB)</li> 
  <li>
  <a href="http://www.uniprot.org/" target="_blank">UniProt</a></li> 
  <li><a href="http://www.rcsb.org/pdb/home/home.do" target="_blank">RCSB Protein Data Bank</a> 
  (PDB)</li> 
</ul> 
<a name="ebola_use"></a>
<h3>Data use</h3> 
<p> 
The Ebola virus sequence may be freely downloaded, used in analyses, and repackaged in databases. 
The data may be freely used in scientific papers analyzing particular genes and regions if the 
provider of these data is properly acknowledged.</p>

<!-- Other Credits -->
<!-- Commenting out this section. It is very out of date, but might be a useful archive.

<a name="other_credits"></a>
    <TABLE BGCOLOR="#fffee8" WIDTH="100%" BORDER=1>
	<TR><TD>
	<TABLE BGCOLOR="#D9E4F8" WIDTH="100%">
	    <TR><TD><FONT SIZE="4"><strong>&nbsp; External Contributors
		to the UCSC Genome Browser Project</strong></FONT>
	    </TD></TR>
	</TABLE>
	<TABLE BGCOLOR="#fffee8" WIDTH="100%" CELLPADDING=0>
	    <TH HEIGHT=10></TH>
		<TR><TD COLSPAN=2></TD></TR>
	 	<TR><TD COLSPAN=2>
		    <ul>
			<li>
			<a href="http://www.affymetrix.com/estore/"
			target="_blank">Affymetrix</a>: Consensus and exemplar
			sequences for probe selection on several chips, 
			exon array data, transcriptome data, Genie Gene 
			predictions, 
			design and analysis support in the early stages of the 
			UCSC Human Genome project, including software for the 
			synthetic chromosome 22 data we used to test our 
			assembly methods.
			<li>
			<a href="http://www.genomics.agilent.com/en/home.jsp"
target="_blank">Agilent Technologies</a> 
			(Shane Giles, Peter Webb, and Dione Bailey): Agilent 
			microarray probe sets.
			<li>
			<a href="http://www.alleninstitute.org/"
target="_blank">Allen Institute 
			for Brain Science</a>: Allen Brain Atlas probes.
			<li>
			<a href="https://www.ncbi.nlm.nih.gov/genome/cyto/hbrc.shtml" target="_blank">BAC Resource
			Consortium</a>: Cytogenetic markers (FISH-mapped clones annotation track).
			<li>
			<a href="https://www.hgsc.bcm.edu/" target="_blank">Baylor College of Medicine
			Human Genome Sequencing Center (BCM-HGSC)</a>
			(Richard Gibbs, John Bouck, Kim Worley and James Durbin): Rat Genome
			sequence data, <em>D. pseudoobscura</em> sequence for alignment tracks
			with <em>D. melanogaster</em> genome.
			<li>
			<a href="http://www.fruitfly.org/" target="_blank">Berkeley Drosophila Genome
			Project</a> (BDGP): Data for <em>D. melanogaster</em> gene annotations.
			<li>
			<a href="http://www.biobase-international.com/"
			target="_blank">BioBase</a>: Transfac database data used
			in HMR Conserved Transcription Factor Binding Sites
			track.
			<li>
			<a href="http://www.bioinfor.com/" target="_blank">
			Bioinformatics Solutions, Inc.</a>
			(Ming Li and colleagues): Pattern Hunter mouse homology track.
			<li>
			<a href="http://biomarkersconsortium.org/"
			target="_blank">Biomarkers Consortium</a>: I-SPY 
			collaboration on the UCSC Cancer Genome Browser
			project.
			<li>
			<a href="http://h-invitational.jp/hinv/ahg-db/index.jsp"
			target="_blank">Biomedicinal Information Research Center 
			(BIRC)</a>, 
			National Institute of Advanced Industrial Science and 
			Technology (AIST): H-InvDB Gene Predictions.
			<li>
			<a href="http://www.bu.edu/chemistry/faculty/tullius/" 
		        target="_blank">Boston University</a> 
			(Tullius Lab, in collaboration with MIT,
			NHGRI, and the La Jolla Institute for Allergy and 
			Immunology): ORChID Predicted DNA Cleavage 
			Sites (ENCODE).
			<li>
			<a href="http://www.broadinstitute.org/"
			target="_blank">The Broad Institute</a>
			<BR>--(Eric Lander, Mike Zody, Chad Nusbaum): Plasmid end reads. Human working
			draft sequence analysis project. Special thanks to Eric 
			for inviting us to join the Hardcore Analysis Group 
			for the draft sequence project and for 
			his direction of that group. 
			<BR>--(<a href="http://bernstein.mgh.harvard.edu/"
			target="_blank">Bernstein Lab</a>): Histone modifications
			using ChIP-seq (ENCODE).
			<li>
			<a href="http://woldlab.caltech.edu/" 
			target="_blank">California Institute of Technology</a> 
			(Wold Lab): RNA-Seq data (ENCODE).
			<li>
			<a href="http://www.bcgsc.ca/"
			target="_blank">Canada's Michael Smith Genome Sciences
			Centre</a>: ORegAnno data, brain DNA methylation track 
			(ENCODE).
			<li>
			<a href="http://cedars-sinai.edu"
			target="_blank">Cedars-Sinai Medical Center</a> (CSMC):
			FISH-mapped clones annotation track. 
			<li>
			<a href="http://projects.tcag.ca/variation/" 
			target="_blank">Centre for Applied Genomics</a>: Database 
			of Genomic Variants (DGV) data.
			<li>
			<a href="http://www.cephb.fr/en/index.php"
			target="_blank">Centre d'Etude du Polymorphisme Humain</a>
			(CEPH): Data from Genotype database for SNP Population
			Allele Frequencies track.
			<li>
			<a href="http://pasteur.crg.es/portal/page/portal/Internet/02_Research/01_Programmes/01_Bioinformatics_Genomics"
			target="_blank">Centre for Genomic Regulation (CRG)</a> 
			(Guigo Lab): CRG mappability (alignability) data,
			RNA-seq data.
			<li>
			<a href="https://www-lbme.biotoul.fr/" target=blank>CNRS 
			Universit&eacute; Paul Sabatier</a>, Laboratoire de Biologie
			Mol&eacute;culaire Eucaryote, Toulouse, 
			France (Michel Weber): miRNA Data annotation track (in collaboration with Sam 
			Griffiths-Jones of The Wellcome Trust Sanger Institute).
			<li>
			<a href="http://www.cshl.org/" target=blank>Cold 
			Spring Harbor Laboratory</a> (CSHL)
			<BR>--(Stein Lab): HapMap data.
			<BR>--(<a href="http://gingeraslab.cshl.edu/" 
			target="_blank">Gingeras Lab</a> and
			<a href="http://hannonlab.cshl.edu/"
			target="_blank">Hannon Lab</a>): RNA-seq data
			tracks (ENCODE). 
			<BR>--(<a href="http://rulai.cshl.org/" 
			target="_blank">Ivo Grosse and Michael Q. Zhang</a>): 
			Predictions for First Exon Finder annotation track.
			<li>
			<a href="https://www.ncbi.nlm.nih.gov/projects/CCDS/CcdsBrowse.cgi"
			target="_blank">Consensus Coding Sequence Project</a> 
			(EBI, NCBI, WTSI, UCSC): Consensus CDS Genes.
		    	<li>
			<a href="http://compgen.bscb.cornell.edu/" 
			target="_blank">Cornell University (Siepel Lab)</a>: 
			Exoniphy.
			<li>
			<a href="http://www.genome.duke.edu/index.php" 
			target="_blank">Duke University</a>
			(Institute for Genome Sciences and Policy): 
			Open chromatin track, Human tissue Affymetrix Exon 
			expression array data (ENCODE).
			<li>
			<a href="http://www.embl.org/" target="_blank">EMBL</a> (Peter Bork): Mouse pseudogene
			predictions used for Bork Pseudogenes custom annotation track.
			<li>
			<a href="http://genetics.emory.edu/"
			target="_blank">Emory University School of Medicine</a> 
			(D.  Ledbetter,
			C. Martin): Collaboration on CGH-ICSA grant. 
			<li>
			<a href="http://www.ensembl.org/" 
			target="_blank">Ensembl</a> (Paul Flicek,  Guy Slater, 
			Imre Vastrik, Arek Kasprzyk, Michele Clamp): Ensembl 
			Gene predictions, 
			data exchange formats, 
			TPF clone map coordination, 
			exonerate mouse homology track, early help placing FISH-mapped clones. 
			<li>
			<a href="http://www.ieo.it/en/"
			target="_blank">European Institute of Oncology</a>:
			Nuclease accessible sites (ENCODE).
			<li>
			<a href="http://flybase.org/" target="_blank">FlyBase</a>: Curated Drosophila
			gene annotations.
			<li>
			<a href="http://www.fredhutch.org/en.html" target="_blank">Fred Hutchison Cancer Research Center</a> (FHCRC)
			(Barbara Trask): FISH-mapped clones annotation track.
			<li>
			<a href="http://geneontology.org/" target="_blank">Gene Ontology (GO) Consortium</a>:
			Structured annotations for Gene Sorter.
			<li>
			<a href="http://www.girinst.org/"
			target="_blank">Genetic Information Research Institute</a>
			(GIRI): Repbase repeat library, repeatmasking tools.
			<li>
			<a href="http://big.crg.cat/computational_biology_of_rna_processing" target="_blank">Genome Informatics Research Lab</a> 
			(GIRL) within Grup de Recerca en Inform&agrave;tica M&egrave;dica (GRIB)
			(Francisco C&acirc;mara Ferreira, 
			Roderic Guig&oacute;): Geneid and SGP gene prediction annotation tracks. 
			<li>
			<a href="http://www.gnf.org/" target="_blank">Genomics Institute of the Novartis
			Research Foundation</a> (GNF): Expression data using Affymetrix chips for GNF Expression
			Atlas annotation tracks.
			<li>
			<a href="http://www.gis.a-star.edu.sg/internet/site/" 
			target="_blank">Genome Institute of Singapore</a>
			(GIS): DNA Paired-end Ditag data, RNA-seq data (ENCODE).
			<li>
			<a href="http://www.genoscope.cns.fr/spip/spip.php?lang=en" target="_blank">Genoscope</a>
			(Jean Weissenbach and Olivier Jaillon): Exofish program, <em>Tetraodon nigroviridis</em> 
			sequence for Tetraodon Blat alignments annotation tracks, data for Anopheles
			Evolutionary Conserved Regions annotation track.
			<li>
			Helicos BioSciences: RNA-seq data 
			(ENCODE).
			<li>
			<a href="http://www.sickkids.ca/"
			target="_blank">Hospital for Sick Children</a> (Sick 
			Kids) (L. Feuk): Structural variation data.
			<li>
			<a href="http://research.hudsonalpha.org/Myers/"
			target="_blank">HudsonAlpha Institute for Biotechnology</a> (Myers Lab): 
		        RNA-seq data (ENCODE),
			Methyl-seq data (ENCODE), transcription factor binding 
			sites identified by ChIP-seq (ENCODE), copy number
			variation (CNV) data (ENCODE). 
			<li>
			<a href="http://www.illumina.com/" 
			target="_blank">Illumnia</a>: Probe set alignments.
			<li>
			Institute for Genomic Research (TIGR): TIGR Gene Index 
			annotation track. 
			<li>
			<a href="http://www.systemsbiology.org" target="_blank">Institute for Systems Biology</a>
			(Arian Smit): <a href="http://www.repeatmasker.org" 
			target="_blank">RepeatMasker</a> repeat identification and masking software and libraries.
			<li>
			<a href="http://www.mousephenotype.org/martsearch_ikmc_project/about"
			target="_blank">International Knockout Mouse 	
			Consortium</a>: Data for IKMC tracks.
			<li>
			Iowa State University (J.  Murray): Collaboration on 
			<a href="https://www.facebase.org/node/68"
			target="_blank">FaceBase</a> project.
			<li>
			<a href="http://www.jcvi.org/cms/home/"
			target="_blank">J. Craig Venter Institute</a>:
			Nuclease accessible sites (ENCODE).
			<li>
			<a href="http://www.informatics.jax.org/" 
			target="_blank">Jackson Laboratory</a>: 
			Mouse Genome Informatics tracks, 
			Mouse ortholog links from known genes track. 
			<li>
			<a href="http://www.genome.jp/" target="_blank">Kyoto University</a>, GenomeNet: 
			<a href="http://www.genome.jp/kegg/kegg2.html" target="_blank">Kyoto Encyclopedia of 
			Genes and Genomes</a> (KEGG) - pathway diagrams for Known Genes annotation track.
			<li>
			<a href="http://jgi.doe.gov/about-us/organization/genomic-technologies/len-pennacchio/"
			target="_blank">Lawrence Berkeley National Laboratory</a> 
			(L.A. Pennacchio): Vista HMR-Conserved
			non-coding human enhancers track.
			<li>
			<a href="http://www.lanl.gov/" target="_blank">Los Alamos National Laboratory</a> (LANL):
			FISH-mapped clones annotation track. 
		 	<li>
			<a href="http://genes.mit.edu/burgelab/" 
			target="_blank">Massachusetts Institute of Technology
			(MIT)</a> (Chris Burge): RNA-Seq data, Genscan gene 
			prediction program. 
			<li>
			<a href="http://rgd.mcw.edu/" target="_blank">Medical
			College of Wisconsin</a>: Rat Genome Database (RGD).
		 	<li>
			<a href="http://www2.mrc-lmb.cam.ac.uk/" 
			target="_blank">Medical Research Council Laboratory 
			of Molecular Biology</a>: 
			<a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/"
			target="_blank">Superfamily</a> database.
			<li>
			<a href="http://www.mountsinai.org/" target="_blank">
			Mount Sinai School of Medicine</a> 
			(<a href="http://tandem.bu.edu/benson.html" 
			target="_blank">Gary Benson</a> - now at Boston 
			University): Tandem Repeats Finder.
			<li>
			<a href="http://www.cancer.gov/"
			target="_blank">National Cancer Institute</a> (NCI), NIH: 
			<BR>--<a href="http://cancergenome.nih.gov/"
			target="_blank">The Cancer Genome Atlas</a> (TCGA).
			<BR>--<a href="http://mgc.nci.nih.gov" 
			target="_blank">Mammalian Gene Collection (MGC) 
			Project</a>. 
			<BR>--<a href="http://cgap.nci.nih.gov/" 	
			target="_blank">Cancer Genome Anatomy Project</A> (CGAP). 
			<BR>--SAGE project (in collaboration with NCBI).
			<BR>--FISH-mapped clones annotation track. 
		 	<li>
			<a href="https://www.ncbi.nlm.nih.gov/index.html" 
			target="_blank">National Center for Biotechnology
			Information</a> (NCBI), NIH 
			<BR>--<a href="https://www.ncbi.nlm.nih.gov/refseq/" 
			target="_blank">RefSeq</a>, 
			<a href="https://www.ncbi.nlm.nih.gov/gene"
			target="_blank">Entrez Gene</a>,
			<a href="https://www.ncbi.nlm.nih.gov/SNP/" 
			target="_blank">dbSNP</a>, 
			<a href="https://hapmap.ncbi.nlm.nih.gov/index.html.en"
			target="_blank">HapMap</a>,
			<a href="https://www.ncbi.nlm.nih.gov/omim"
			target="_blank">OMIM</a>, and 
			<a href="https://www.ncbi.nlm.nih.gov/pubmed"
			target="_blank">PubMed</a> databases.
			<BR>--<a href="https://www.ncbi.nlm.nih.gov/SAGE/" 
			target="_blank">Serial Analysis of
			Gene Expression Tag to Gene Mapping</a> (SAGEmap) data resource.
			<BR>--(Deanna Church): Mouse synteny track, database of 
			whole genome shotgun mouse reads and assemblies used for
			mouse tracks.
			<BR>--(Greg Schuler): GenBank freezes of human clones 
			used in the assembly, NT contigs of multiple 
			finished clones, STS maps (in earlier freezes), 
			specially-formatted versions of the finished 
			chromosomes. 
			<BR>--(Steve Sherry): SNP tracks. 
			<BR>--(Wonhee Jang): FISH-mapped clones annotation 
			track.
			<BR>--(Danielle and Jean Thierry-Mieg): Acembly Genes 
			annotation track.
			<BR>--(Lukas Wagner): 3' EST track in the July 2000 
			browser.
			<li>
			<a href="http://www.genome.gov" target="_blank">National 
			Human Genome Research Institute</a> 
			(NHGRI), NIH
			<BR>--(<a href="http://www.genome.gov/12514761"
			target="_blank">Elnitski Lab</a>): Bidirectional 
			promoters, negative regulatory elements (ENCODE).
			<BR>--(ENCODE group): Guidance for ENCODE DCC, 
			collaboration and contributions for various ENCODE 
			annotation tracks.
			<BR>--<a href="http://www.genome.gov/gwastudies/"
			target="_blank">GWAS</a>: GWAS data from the Catalog 
			of Published Genome-Wide Association Studies.
			<BR>--(NISC group): Collaboration and annotations
			for <a href="http://www.nisc.nih.gov/index.html" 
			target="_blank">NISC</a> browser, including 
			curated gene annotations, DNAse I Hypersensitive Sites 
			annotation, Multi-species Conserved Sequences (MCS) 
			tracks, Binomial_25 Conservation Scores,
			ZooSeq Blastz alignments tracks. 
			<li>
			<a href="http://www.nia.nih.gov/" 
			target="_blank">National Institute on Aging</a>, NIH:
			Data from the 
			<a href="http://geneticassociationdb.nih.gov/"
			target="_blank">Genetic Association Database</a>.
			<li>
			<a href="http://research.nki.nl/vansteensellab/"
			target="_blank">Netherlands Cancer Institute</a> (van Steensel Lab):
			Nuclear lamina associated domains tracks.
			<li>
			<a href="http://bmi.osu.edu/" target="_blank">Ohio State
			University Bioinformatics & Computational Biology Group</a> (Ramana V. Davuluri): 
			Predictions for First Exon Finder annotation track.
			<li>
			<a href="http://www.orfeomecollaboration.org/"
			target="_blank">ORFeome Collaboration</a>: ORFeome
			Collaboration human gene clones.
			<li>
			<a href="http://www.bx.psu.edu/" 
			target="_blank">Pennsylvania State University 
			Center for Comparative Genomics and 
			Bioinformatics</a>: BLASTZ, LASTZ, and multiz alignment 
			programs, Genome Alignment and Annotation Database 
			(GALA), collaboration on multiple alignment annotation 
			tracks, ENCODE annotation tracks, Personal Genome 
			Variants track, Galaxy, regulatory potential scores 
			tracks.
			<li>
			Perlegen Sciences: STS markers for 
			the Haplotype Blocks annotation track on chr 21,
			SNP data for the Affymetrix 120K SNP track, 
			TNG radiation hybrid map.
			<li>
			<a href="http://www.gsc.riken.go.jp" 
			target="_blank">RIKEN
			Genomic Sciences Center</a>: Chimpanzee BAC end sequence
			data, mouse annotations, human genome sequence 
			contributions, CAGE tags RNA subcellular localization
			data (ENCODE).
			<li>
			<a href="https://www.roswellpark.org/" 
			target="_blank">Roswell Park Cancer Institute</a> (RPCI):
			FISH-mapped clones annotation track.
			<li>
			SNP Consortium: Data for SNP annotation tracks.
			<li>
			Softberry Inc. (V. Solovyev):
			Fgenesh++ and Fgenesv+ gene predictions, TSSW promoters tracks.
			<li>
			<a href="http://www.mad-cow.org/" 
			target="_blank">Sperling Biomedical 
			Foundation</a> (Thomas Pringle): Technical consultant on
			Proteome Browser, avid Genome Browser user and the 
			source of myriad ideas for expanding its functionality, 
			early assistance and support for UCSC Genome Browser.
			<li>
			<a href="http://bejerano.stanford.edu/"
			target="_blank">Stanford University</a> 
			<BR>--(<a href="http://bejerano.stanford.edu/"
			target="_blank">Bejerano Lab</a>): GREAT tool.
			<BR>--(
			<a href="http://www.stanford.edu/group/morrinst/abi.html"
			target="_blank">Morrison Institute for Population
			and Resource Studies</a>
			Morrison Institute for Population
                        and Resource Studies, Human Genome Diversity 
			Project): Data for SNP Population Allele Frequencies 
			track.
			<BR>--<a href="http://source.stanford.edu/"
			target="_blank">SOURCE tool</a>: SOURCE links from UCSC
			Genes tracks.
			<li>
			Stanford University School of 
			Medicine (Myers Lab - now at HudsonAlpha): Data set for 
			Human Promoters custom annotation track.
			<li>
			<a href="http://switchgeargenomics.com/"
			target="_blank">SwitchGear Genomics</a> (N. Trinklein 
			and S.F. Aldred): Transcription start sites (TSSs).
			<li>
			<a href="http://www.uniprot.org/" 
			target="_blank">UniProt</a>: Protein data from UniProt 
			database.
			<li>
			<a href="http://bioinfo2.ugr.es/isochores/" 
			target="_blank">Universidad de Granada</a>, 
			Grupo de Bioinform&aacute;tica, Spain (Jose L. Oliver): 
			Isochore custom annotation track.
			<li>
			<a href="http://www.sunycnse.com/AboutUs/FacultyStaff/Faculty/ScottTenenbaum.aspx"
			target="_blank">University at Albany - SUNY</a> 
			(TenenBaum Lab): Target RNA binding protein 
			expression measured by RIP-chip (ENCODE).
			<li>
			<a href="http://qb3.berkeley.edu/ccb/"
			target="_blank">University of CA, Berkeley</a> (Lior
			Pachter, Simon Cawley, Marina Alexandersson): 
			<a href="http://bio.math.berkeley.edu/slam/mouse/" 
			target="_blank">SLAM</a> Gene predictions.
			<li>
			<a href="http://bioinformatics.ucsd.edu/"
			target="_blank">University of CA, San Diego</a> 
			<BR>--(<a href="http://www.snl.salk.edu/~geneyeo/"
			target="_blank">G. Yeo</a>, now at Salk Institute): 
			FOX2 adaptor-trimmed CLIP-seq reads track.
			<BR>--(<a href="http://bioinformatics-renlab.ucsd.edu/rentrac/"
			target="_blank">Ren Lab</a>, Ludwig Institute for Cancer
			Research): TAF1 binding in fibroblastoid cells assayed
			by ChIP-chip (ENCODE).
			<li>
			<a href="http://www.ucsf.edu/" 
			target="_blank">University of CA, San Francisco</a>:
			Collaboration on UCSC Cancer Browser, 	
			Chimera links from
			SNPs and UCSC Genes, FISH-mapped clones annotation 
			track.
			<BR>--(ENCODE group): Brain DNA methylation track 
			(ENCODE).
			<li>
			<a href="http://bme.ucsc.edu/"
			target="_blank">University of CA, Santa Cruz</a> 
			<BR>--(<a href="http://bme.ucsc.edu/people/ed"
			target="_blank">R.E. Green</a>): Neanderthal annotation 
			tracks.
			<BR>--(<a href="http://lowelab.ucsc.edu/"
			target="_blank">Lowe Lab</a>): tRNA Gene Predictions 
			using tRNAscan-SE.  
			<li>
			<a href="http://www.oncology.cam.ac.uk/"
			target="_blank">University of Cambridge, Department of
			Oncology</a>: Open chromatin track (ENCODE).
			<li>
			<a href="http://pritchardlab.stanford.edu/home.html"
			target="_blank">Stanford University</a> (Pritchard Lab):
			Collaboration on Human Genome Diversity Project tracks.
			<li>
			<a href="http://genes.uchicago.edu/contents/faculty/lieb-jason-ph-d.html"
			target="_blank">University of North Carolina, 
			Chapel Hill</a> (Lieb Lab, now at the University of Chicago): Open chromatin track 
			(ENCODE).
			<li>
			<a href="http://www.allgenes.org/" target="_blank">
			University of Pennsylvania</a> (Y. Thomas Gan):
			Database of Transcribed Sequences (DoTS) Genes custom annotation track.
			<li>
			University of Pittsburgh (M. Marazita): 
			Collaboration on 
			<a href="https://www.facebase.org/node/68"
			target="_blank">FaceBase</a> project.
			<li>
			<a href="http://www.genenetwork.org/"
			target="_blank">University of Tennessee GeneNetwork</a>: GeneNetwork 
			links from UCSC Genes annotations.
			<li>
			<a href="http://ccri.uthscsa.edu/LPenalva.asp"
			target="_blank">University of Texas Health Science
			Center</a>, Greehy Children's Cancer Research Institute 
			(L.O.F. Penalva): 
			<li>
			<a href="http://microarray.icmb.utexas.edu/research.html"
			target="_blank">University of Texas, Austin</a> (Iyer 
			Lab): Open chromatin track (ENCODE).
			<li>
			<a href="http://www.tbi.univie.ac.at/index.html" 
			target="_blank">University of Vienna</a>, Institute 
			for Theoretical Chemistry and Structural Biology, 
			Theoretical Biochemistry Group</a>: 
			Vienna RNA utility 
			<a href="http://www.tbi.univie.ac.at/RNA/"
			target="_blank">RNAplot</a> - Known Genes annotation 
			track. 
			<li>
			<a href="http://www.gs.washington.edu/"
			target="_blank">University of Washington</a>
			<BR>--(<a href="http://eichlerlab.gs.washington.edu/" 
			target="_blank">Eichler Lab</a>): Segmental duplications 
			data. 
			<BR>--(<a href="http://hgsv.washington.edu/"
			target="_blank">Human Genome Structural Variation 
			Project</a>): Variation data for HGSV tracks.
			<BR>--(ENCODE group): Digital DNase genomic footprinting
			track, DNaseI sensitivity using digital DNaseI
			methodology, DNaseI hypersensitive sites, histone
			modifications using ChIP-seq (ENCODE).
			<BR>--(<a href="http://noble.gs.washington.edu/"
			target="_blank">Noble Lab</a>): Predicted nucleosome 
			occupancy tracks.
			<BR>--(<a href="http://www.gs.washington.edu/faculty/nickerson.htm"
			target="_blank">Nickerson Lab</a>): Tajima's D SNPs track.
			<li>
			<a href="http://www.klinimm.uu.se/research/genetics_genomics/wadelius/"
			target="_blank">Uppsala University</a> 
			(Wadelius Lab): ChIP-chip data (ENCODE).
			<li>
			<a href="http://www.wustl.edu/" 
			target="_blank">Washington University in St. Louis</a> and
			<a href="http://medicine.wustl.edu/"
			target="_blank">Washington University in St. Louis
			School of Medicine</a> 
			<BR>--(Robert Waterston (now at UW), John McPhearson, Asif Chinwalla,
			Shiaw-Pyng Yang, John Wallis and LaDeana Hillier): Fingerprint-based map
			of BAC clones from the human genome and a corresponding layout of sequenced
			clones, plasmid end reads, BACend analysis, CpG islands.
			<BR>--(<a href="http://www.hhmi.org/scientists/sean-r-eddy"
			target="_blank">Sean Eddy</a>
			and Todd Lowe (now at UCSC)): RNA Genes track.
			<BR>--(<a href="http://mblab.wustl.edu/" target="_blank">Michael 
			Brent Lab</a>): N-SCAN and Twinscan Genes.
			<BR>--(ENCODE group): Brain DNA methylation track 
			(ENCODE).
			<li>
			<a href="https://www.sanger.ac.uk/about/history/hgp/" target="_blank">
			Wellcome Trust Sanger Institute</a>
			<BR>--(J. Harrow, J. Gilbert, J. Sulston, J. Rogers, 
			J. Mullikin, R. Durbin, S. Searle, I. Dunham and 
			the annotation teams): Vega Genes and Pseudogenes, 
			Sanger 20 and Sanger 22 Genes and Pseudogenes. Plasmid 
			end reads, processing of whole genome shotgun mouse 
			reads. Eponine program.
			<BR>--(S. Griffiths-Jones): miRNA Data annotation track 
			(in collaboration with 
			M. Weber of CNRS Universit&eacute; Paul Sabatier).
			<BR>--<a href="https://decipher.sanger.ac.uk/"
			target="_blank">Decipher</a>
			<BR>--<a href="http://pfam.xfam.org/"
			target="_blank">Protein Families 
			Database of Alignments and HMMs</a> (Pfam).
			<li>
			<a href="http://jura.wi.mit.edu/bio/"
			target="_blank">Whitehead Institute, Bioinformatics and
			Research Computing</a> (G. Bell): TargetScan miRNA 
			regulatory sites.
			<li>
			<a href="http://www.wormbase.org/" target="_blank">WormBase</a>
			(Lincoln Stein): Wormbase curated gene annotations for
			<em>C. elegans</em> genome.
			<li>
			<a href="http://cbb.yale.edu/"
			target="_blank">Yale University</a>: 
			<BR>--(<a href="http://cbb.yale.edu/"
			target="_blank">Snyder, Gerstein, and Weissman Labs</a>
			in collaboration with NASA Ames Research Center and 
			Eloret Corporation): 
			Transcriptionally Active Regions (TARs) data (ENCODE).
			<BR>--(<a href="http://medicine.yale.edu/lab/sestan/"
			target="_blank">Sestan Lab</a>): Exon microarray expression
			data from the late mid-fetal human brain.
		</ul></TD><TD WIDTH=15></TD></TR></TABLE>
	</TD></TR></TABLE>

	<BR>

End of commented out section -->

<!-- Early Credits -->
<a name="early_credits"></a>
<h2>Acknowledgment of early support</h2> 
<p>
During the initial stage of this project, from January 2000 to June 2001, we received no direct 
federal funding to develop the working draft other than travel support. However, we gratefully 
acknowledge the support for our personnel provided by our basic bioinformatics research grants, 
especially DOE grant DE-FG03-99ER62849 (to develop gene-finding methods), but also NSF grant 
DBI-9809007, UC BIOTECH grant 99-11, a grant from the Sloan Foundation, the David and Lucille 
Packard Foundation, NIH grant GM-52848, and the Zahler grant that provided support for Jim Kent in 
the early stages of this work. </p> 
<p> 
We are particularly thankful for the risks that Patrick Mantey (UCSC Dean of Engineering), 
James Gill (Vice Chancellor for Research), John Simpson (UCSC Provost), and M.R.C. Greenwood (UCSC 
Chancellor) were willing to take in advancing us the money in February 2000 needed to build the 
original 100-node LINUX cluster used in this work, and in the case of Dean Mantey, additional 
personnel support. Phokion Kolaitis, Chair of Computer Science, contributed as well. </p>
<p>
We thank Compaq for providing an additional 
100-CPU computational cluster in Cambridge, MA for use by the public human genome effort. We thank 
<a href="https://www.ibm.com/products/software" target="_blank">ILOG, Inc.</a> (now part of IBM)
for providing discounted licenses to its CPLEX linear programming software, which was essential for 
some of our early experiments.</p> 
<p> 
In December 2000, <a href="http://www.hhmi.org" target="_blank">HHMI</a> began providing support for
equipment and personnel. In the summer of 2001, HHMI provided the primary funding for a new cluster 
of 1000 CPUs running LINUX purchased from Rackable 
(now SGI), with additional support from 
CISI and <a href="http://www.genome.gov/" target="_blank">NHGRI</a>. Dubbed the 
UCSC KiloKluster, this equipment became the computational backbone for the initial years of the 
Genome Browser project. </p>

<!--#include virtual="$ROOT/inc/gbPageEnd.html" -->
