<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=171>
  <TR><TD ALIGN=RIGHT><A HREF="https://www.hgsc.bcm.edu/content/honey-bee-genome-project"
    TARGET=_blank>
    <IMG SRC="../images/Apis_mellifera.jpg" WIDTH=156 HEIGHT=170 ALT="Bee"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Apis mellifera</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      (NHGRI <A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Insects/Bee%20Apis%20mellifera&id=79117" 
      TARGET=_blank>Press Photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The January 2005 <em>Apis mellifera</em> genome assembly 
is based on the Amel_2.0 assembly produced by the 
Human Genome Sequencing Center (HGSC) at 
<A HREF="https://www.hgsc.bcm.edu/content/honey-bee-genome-project" 
TARGET=_blank>Baylor School of Medicine</A>.  To see a list
of the assembly's linkage groups, click 
<A HREF="/cgi-bin/hgTracks?db=apiMel2&chromInfoPage=">here</A>.
(A linkage group is a large collection of scaffolds that is somewhat
equivalent to a chromosome.)
For more information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/4388/" TARGET="_blank">
Amel_2.0</A> in the NCBI Assembly database.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by a linkage group coordinate range,
the accession number of an mRNA or EST, a gene name, or
keywords from the GenBank description of an mRNA. The following
list shows examples of valid position queries for the
<em>A. mellifera</em> genome.  Note that some position queries
(e.g. "huntington") may return matches to the mRNA records of
other species. In these cases, the mRNAs are mapped to their 
homologs in <em>A. mellifera</em>. 
See the <A HREF="../goldenPath/help/hgTracksHelp.html" 
TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TD VALIGN=Top NOWRAP><B>Request:</B><BR></TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top><B>Genome Browser Response:</B><BR></TD>
    </TR>
    <TR><TD COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top NOWRAP>Group1</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all 25,433,307 bases of linkage group  Group1</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Group10:1-50,000</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first 50,000 bases of linkage group Group10</TD>
    </TR>
    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>BI510436</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of genome aligning to EST with GenBank accession BI510436</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>AB193550</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of genome aligning to mRNA with GenBank accession AB193550</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>dock</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of genome aligning to gene with identifier dock (a <em>D. melanogaster</em> gene)</TD>
    </TR>
<!--
    <TR><TD VALIGN=Top NOWRAP>NM_057677</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of genome aligning to RefSeq accession with identifier NM_057677 (a <em>D. melanogaster</em> RefSeq entry)</TD>
    </TR>
-->
    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>preprosecapin</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists preprosecapin mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>odorant binding</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists odorant-binding protein mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>huntington</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists candidate genes associated with Huntington's disease</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>robinson</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists accessions deposited by authors named Robinson</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Robinson,G.E.</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author G.E. Robinson</TD>
    </TR>
    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although GenBank 
			requires the search format <em>Robinson GE</em>, internally it uses 
			the format <em>Robinson,G.E.</em></TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
Amel_2.0 was created using a combination of whole genome shotgun and BAC
sequencing methods.  The assembly was produced using Baylor 
HGSC's assembly engine, Atlas. Approximately 2.7 million reads were assembled, 
representing about 1.8 Gb of sequence and about 7.5x coverage of the (clonable) 
honeybee genome. 
<P>
The assembly consists of a set of contigs and scaffolds. Some scaffolds contain ordered 
and oriented sequence contigs; others contain isolated single contigs that could not be 
linked. Scaffolds are further grouped into 17 linkage groups. Contigs that could not
be placed into groups are contained in GroupUn.
The N50 size of the contigs in this assembly is 36 Kb; the N50 size of the scaffolds
is 343 Kb. The total contig length in the assembly is 225 Mb, and the total span of the
assembly is 229 Mb including gaps. 
More details about Amel_2.0 can be found 
<A HREF="ftp://ftp.hgsc.bcm.tmc.edu/Amellifera/fasta/Amel20050120-freeze/README_Amel_2.0_20050415.txt" TARGET=_BLANK>here</A>.
<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/apiMel2/">FTP server</A> or the 
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#apiMel">Downloads</A> page. 
These data are made available with 
<A HREF="https://www.hgsc.bcm.edu/content/conditions-use" 
TARGET=_BLANK>specific conditions for use</A>. 
The apiMel2 annotation tracks were generated by UCSC and collaborators worldwide. 
See the <A HREF="../goldenPath/credits.html#apiMel_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
