<H2>Description</H2>
<P>
This track shows the Amel_2.0 draft assembly of the 
<em>$organism</em> genome produced by the human Genome Sequencing Center at 
Baylor School of Medicine. Amel_2.0 was created from a 
combination of whole genome shotgun and BAC sequencing methods and was
produced using Baylor HGSC's assembly engine, Atlas. Approximately 2.7 
million reads were assembled, representing about 1.8 Gb of sequence and 
about 7.5x coverage of the (clonable) honeybee genome. 
<P>
The assembly consists of 7,655 scaffolds clustered into 17 linkage 
groups. Some scaffolds contain ordered and oriented sequence contigs; others 
contain isolated single contigs that could not be linked. 
Contigs that could not be placed into groups are contained 
in GroupUn. The N50 size of the contigs in this assembly is 36 Kb; the N50 
size of the scaffolds is 343 Kb. The total contig length in the assembly is 
225 Mb, and the total span of the assembly is 229 Mb including gaps. For 
more details about this assembly, see the Baylor Amel_2.0 
<A HREF="ftp://ftp.hgsc.bcm.tmc.edu/Amellifera/fasta/Amel20050120-freeze/README_Amel_2.0_20050415.txt"
TARGET=_blank>README</A>.
</P>
<P>In dense mode, this track depicts the path through the contigs 
(aka the golden path) used to create the assembled sequence. 
Contig boundaries are distinguished by the use of alternating gold and brown 
coloration. Where gaps
exist in the path, spaces are shown between the gold and brown
blocks.  If the relative order and orientation of the contigs
between the two blocks is known, a line is drawn to bridge the
blocks.</P>
<P>
All components within this track are of fragment type &quot;W&quot;: 
whole genome shotgun contig. 
</P>

<H2>Credits</H2>
<P>
Many thanks to the Human Genome Sequencing Center at 
<A HREF="https://www.hgsc.bcm.edu/content/honey-bee-genome-project" 
TARGET="_blank">Baylor School of Medicine</A> for providing these data.
</P>
