<h2>Description</h2>

<p>
  This track represents the gene models for the Marmoset assembly, calJac4, generated using Comparative Annotation Toolkit (CAT). 
</p>
<p>
  CAT can be found on GitHub: <a href="https://github.com/ComparativeGenomicsToolkit/Comparative-Annotation-Toolkit">  https://github.com/ComparativeGenomicsToolkit/Comparative-Annotation-Toolkit</a>
</p>

<h2>Display Conventions and Configuration</h2>

<p>
  This track follows the display conventions for gene prediction tracks. The exons for putative non-coding genes and untranslated regions are represented by relatively thin blocks, while those for coding open reading frames are thicker. Gene names are displayed in 'pack' or 'full' mode. More information about each gene can be found by clicking on the specific gene/transcript model.
</p>

<p>
The following color key is used:
</p>
<ul>
  <li> <span style="color: rgb(76,85,212)">Blue</span>: protein coding
  <li> <span style="color: rgb(85,212,76)">Green</span>: non-coding
  <li> <span style="color: rgb(135,76,212)">Purple</span>: novel predictions from augCGP/augPB
</ul>

<p>
The following metadata is available by clicking on the transcript.
</p>
<ul>
  <li><b>Gene name</b>: gene symbolic name 
  <li><b>Status of CDS start annotation</b>: is the start of the CDS fully annotated
  <li><b>Status of CDS end annotation</b>: is the end of the CDS fully annotated
  <li><b>Exon frame</b>: comma-separate list of per-exon frames, 0,1,2, or -1 if no frame for an exon
  <Li><B>Transcript ID</B>: CAT assigned unique transcript id
  <li><b>Transcript type</b>: type of transcript
  <Li><B>Gene ID</B>: CAT assigned unique gene id
  <li><b>Gene type</b>: type of gene
  <li><b>Source gene ID</b>: source GENCODE gene id
  <li><b>Source transcript ID</b>: source GENCODE transcript id
  <Li><B>Alignment ID</B>: unique alignment id for TransMap based transcripts
  <li><b>Alternative source transcripts</b>: ???
  <li><b>Paralogous alignment IDs</b>: comma separated list of alignments identified as possible paralogs for this transcript
  <li><b>Unfiltered paralogous alignment IDs</b>: paralogous alignments before filtering
  <Li><B>Collapsed Gene IDs</B>: if this gene was a part of a gene family collapse, this field reports the of identifiers of the genes collapsed together
  <Li><B>Collapsed Gene Names</B>: common names of the collapsed genes
  <li><b>Frameshifted relative to source?</b>: is the transcript frameshifted relative to source_transcript/
  <li><b>Exon support in reference annotation</b>: was this exon supported by the reference annotation?
  <li><b>Intron support in reference annotation</b>: was this intron exon supported by the reference annotation?
  <li><b>Transcript class</b>: for projection transcripts, it is maybe ortholog; for de-novo transcripts, will be one of poor_alignment, possible_paralog, putative_novel_isoform, or putative_novel
  <li><b>Transcript mode(s)</b>: Comma separated list of transcript modes
  <li><b>Valid start codon</b>: has a valid start-codon
  <li><b>Valid stop codon</b>: has a valid stop-codon
  <li><b>Proper multiple of 3 ORF</b>: is there a full valid st
  <li><b>RNA intron support</b>: RNA-Seq support introns
  <li><b>RNA exon support</b>:  RNA-Seq support exons
  <li><b>Is this transcript supported by IsoSeq?</b>: True if the transcript is supported by Iso-Seq
</ul>


<h2>Methods</h2>

<p>
  Genome annotation for the calJac4 assembly was performed using Comparative Annotation Toolkit (CAT). CAT leverages whole-genome alignments generated by Cactus to transfer annotations from one source genome to one or more target genomes. CAT also runs AUGUSTUS in both the comparative gene prediction mode and in a single-genome mode that utilizes Iso-Seq data to predict alternative isoforms. CAT then combines all of these annotation methods into a final consensus annotation set that represents orthology relationships as well as species-specific information.
</p>

<p>
  A Cactus alignment was produced with nine primate assemblies and mouse as an outgroup.
 The GENCODE v33 annotation for GRCh38 was used as a reference to map onto the calJac4 assembly. 20 IsoSeq libraries were included as evidence for the CAT annotation pipeline.
</p>

<table class="stdTbl">
  <caption>Assemblies used in Cactus alignment</caption>
  <tr>
    <td>Bonobo</td>
    <td>Pan paniscus</td>
    <td>Mhudiblu_PPA_v0</td>
    <td>GCA_013052645.1</td>
  </tr>

  <tr>
    <td>Chimp</td>
    <td>Pan troglodytes</td>
    <td>Clint_PTRv2</td>
    <td>GCF_002880755.1</td>
  </tr>

  <tr>
    <td>Gibbon</td>
    <td>Nomascus leucogenys</td>
    <td>Asia_NLE_v1</td>
    <td>GCF_006542625.1</td>
  </tr>

  <tr>
    <td>Gorilla</td>
    <td>Gorilla gorilla gorilla</td>
    <td>Kamilah_GGO_v0</td>
    <td>GCF_008122165.1</td>
  </tr>

  <tr>
    <td>Human</td>
    <td>Homo sapiens</td>
    <td>GRCh38</td>
    <td>GCA_000001405</td>
  </tr>

  <tr>
    <td>Marmoset</td>
    <td>Callithrix jacchus</td>
    <td>Callithrix_jacchus_cj1700_1.0</td>
    <td>GCA_009663435.1</td>
  </tr>

  <tr>
    <td>Orangutan</td>
    <td>Pongo abelii</td>
    <td>Susie_PABv2</td>
    <td>GCF_002880775.1</td>
  </tr>

  <tr>
    <td>Rhesus</td>
    <td>Macaca mulatta</td>
    <td>Mmul_10</td>
    <td>GCF_003339765.1</td>
  </tr>

  <tr>
    <td>Owl_monkey</td>
    <td>&#xa0</td>
    <td>&#xa0;</td>
    <td>unsubmitted</td>
  </tr>

  <tr>
    <td>Mouse</td>
    <td>Mus musculus</td>
    <td>GRCm38</td>
    <td>GCA_000001635.2</td>
  </tr>
</table>

<h2>Data Access</h2>

<p>
  For automated analysis, the genome annotation is stored in a <a href="/goldenPath/help/bigGenePred.html">bigGenePred</a>
   format file that 
can be downloaded from the download server at
<a href="http://hgdownload.soe.ucsc.edu/gbdb/$db/cat/cat-consensus-v2.bb">cat-consensus-v2.bb</a>.
Annotations can be converted to ASCII text by our tool <tt>bigBedToBed</tt>
which can be compiled from the source code or downloaded as a precompiled
binary for your system. Instructions for downloading source code and binaries can be found
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">here</a>.
The tool can also be used to obtain only features within a given range, for example:
<br> 
<tt>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/$db/cat/cat-consensus-v2.bb -chrom=chr6 -start=0 -end=1000000 stdout</tt> 
<br>
Please refer to our
<a href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing list archives</a>
for questions, or our
<a href="../FAQ/FAQdownloads.html#download36">Data Access FAQ</a>
for more information. 
</p>

<h2>Credits</h2>
<p>
The alignments were generated by, Marina Haukness, Mark Diekhans, and Ian Fiddes.
</p>

<h2>References</h2>

<p>
Armstrong J, Hickey G, Diekhans M, Fiddes IT, Novak AM, Deran A, Fang Q, Xie D, Feng S, Stiller J
<em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33177663" target="_blank">
Progressive Cactus is a multiple-genome aligner for the thousand-genome era</a>.
<em>Nature</em>. 2020 Nov;587(7833):246-251.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33177663" target="_blank">33177663</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7673649/" target="_blank">PMC7673649</a>
</p>

<p>
Fiddes IT, Armstrong J, Diekhans M, Nachtweide S, Kronenberg ZN, Underwood JG, Gordon D, Earl D,
Keane T, Eichler EE <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/29884752" target="_blank">
Comparative Annotation Toolkit (CAT)-simultaneous clade and personal genome annotation</a>.
<em>Genome Res</em>. 2018 Jul;28(7):1029-1038.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29884752" target="_blank">29884752</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6028123/" target="_blank">PMC6028123</a>
</p>

<p>
Stanke M, Steinkamp R, Waack S, Morgenstern B.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/15215400" target="_blank">
AUGUSTUS: a web server for gene finding in eukaryotes</a>.
<em>Nucleic Acids Res</em>. 2004 Jul 1;32(Web Server issue):W309-12.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15215400" target="_blank">15215400</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC441517/" target="_blank">PMC441517</a>
</p>


