<H2>Description</H2>
<P>
The ALG-1 CLIP-seq tracks show the density of reads that were mapped to
the <em>C. elegans</em> genome from independent ALG-1 CLIP-seq experiments in
larval stage 4 (L4) wildtype and ALG-1 mutant worms.  The regions
determined to be occupied by ALG-1, as indicated by the analyses
described below, are also available in the ALG-1 binding sites in larval
stage 4 worms track.
Genic regions were defined based on RNA-seq reads publicly available
from Hillier <em>et al</em>., 2009. The reannotated genic regions are displayed
as the subtrack &quot;Reannotated genic regions based on data from L3 and L4
RNAseq from Hillier <em>et al</em>&quot;.
</P><P>
ALG-1 is a <em>C. elegans</em> Argonaute protein that mediates the interaction 
of miRNAs and their mRNA targets.  CLIP-seq is a biochemical approach to
determine RNA-protein interaction loci.
</P>

<H2>Methods</H2>
<P>
Cross-linking immunoprecipitation, followed by deep sequencing
(CLIP-seq), was performed on the Argonaute protein ALG-1 in both
wild-type (WT) and ALG-1 mutant (MT) L4 worms.
</P><P>
Reads were aligned to the <em>C. elegans</em> genome (ce6) using bowtie (version
0.9.9.2) from three independent CLIP-seq experiments in both WT and
ALG-1 worms.  Reads were computationally extended 50nt in the 3'
direction to account for the expected size of PCR products that were
sequenced.  Biologically reproducible regions were determined based on
gene-specific weighting of reads in each of the 3 experiments in the two
experimental conditions (WT and MT).  Reads in each of the experimental
conditions were then collapsed, and biologically reproducible regions
were further scrutinized for significance above background using the
Poisson distribution on pre-mRNA.  Regions that passed the
Poisson-derived density cutoff and overlapped at least 25% of their
length between WT and MT experiments were eliminated, and thus ALG-1
binding sites are defined as regions that are reproducible in
independent biological experiments, contain more reads than expected by
the Poisson distribution, and do not appear in MT experiments.
</P><P>
RNA-seq reads from Hillier <em>et al</em>. in L3 and L4 worms were used to 
extend annotated genic regions into untranslated regions.
</P><P>
Detailed descriptions of these methods can be found in the online
supplementary materials for Zisoulis <em>et al</em>, 2010.
</P>

<H2>Credits</H2>
<P>
Data for this track were supplied to UCSC by the 
<A HREF="http://yeolab.ucsd.edu/yeolab/Home.html" target=_blank>Gene Yeo 
lab</A> in the Department of Cellular and Molecular Medicine, University of
California, San Diego, CA.
</P>

<H2>References</H2>
<p>
Hillier LW, Reinke V, Green P, Hirst M, Marra MA, Waterston RH.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/19181841" target="_blank">
Massively parallel sequencing of the polyadenylated transcriptome of C. elegans</a>.
<em>Genome Res</em>. 2009 Apr;19(4):657-66.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19181841" target="_blank">19181841</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2665784/" target="_blank">PMC2665784</a>
</p>

<p>
Zisoulis DG, Lovci MT, Wilbert ML, Hutt KR, Liang TY, Pasquinelli AE, Yeo GW.
<a href="https://www.nature.com/articles/nsmb.1745" target="_blank">
Comprehensive discovery of endogenous Argonaute binding sites in Caenorhabditis elegans</a>.
<em>Nat Struct Mol Biol</em>. 2010 Feb;17(2):173-9.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20062054" target="_blank">20062054</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2834287/" target="_blank">PMC2834287</a>
</p>
