<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=201>
  <TR><TD ALIGN=RIGHT><A HREF="http://www.broadinstitute.org/mammals/opossum" TARGET=_blank>
    <IMG SRC="../images/Monodelphis_domestica.jpg" WIDTH=186 HEIGHT=130 ALT="Opossum"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1> 
      <em>Monodelphis domestica</em><BR>
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of Paul Samollow,<BR>
      Southwest Foundation for Biomedical Research,<BR>San Antonio.<BR>
      (NHGRI <A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Opossum%20(short_tailed)&id=79128"
      TARGET=_blank>PressPhotos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The Oct. 2006 <em>Monodelphis domestica</em> draft assembly 
(NCBI project 12561, accession AAFR03000000)
was produced by <A HREF="http://www.broadinstitute.org/mammals/opossum"
TARGET=_blank>The Broad Institute</A>. For more information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/237648/" TARGET="_blank">
MonDom5</A> in the NCBI Assembly database.</P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Opossum genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>chrX</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of chromosome X</TD>
    </TR>
    <TR><TD VALIGN=Top>chr3:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of chr 3</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr3:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 3 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>AF049753</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number AF049753</TD>
    </TR>
    <TR><TD VALIGN=Top>EC091556</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		EC091556</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>CLOCK</TD>
        <TD WIDTH=14></TD>
        <TD>Displays a list of genomic regions showing tBLASTn alignment of peptides from human CLOCK gene</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>NM_001033969</TD>
        <TD><BR></TD>        
  	<TD VALIGN=Top>Displays the region of genome with RefSeq identifier NM_001033969, the BCRA1 gene</TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>immunoglobulin</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Lists immunoglobulin regions</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>immunoglobulin light chain kappa</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Lists only immunoglobulin light chain kappa mRNAs</TD> 
    </TR>
    <TR><TD VALIGN=Top NOWRAP>miller</TD> 
        <TD WIDTH=14></TD> 
        <TD VALIGN=Top>Lists mRNAs deposited by scientists named Miller</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Miska,K.B.</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Lists mRNAs deposited by co-author K.B. Miska</TD>
    </TR> 
    <TR><TD VALIGN=Top><br></TD></TR>

</TABLE>
    Use this last format for author queries. Although GenBank requires the 
    search format <em>Miska KB</em>, internally it uses the format 
    <em>Miska,K.B.</em>

<HR>
<H3>Assembly details</H3>
<P>
This draft of the opossum genome has approximately 6.8X coverage. The
assembly length is nearly 3.61 billion bp including gaps (3.50 billion bp 
without gaps) contained on chromosomes 1-8, X, M (mitochondrial DNA), and Un
(unmapped clone contigs).  The N50 of the scaffolds 
including gaps is 60,456 kb; the N50 without gaps is 110 kb. The N50 size 
is the length such that 50% of the assembled genome lies in blocks of the N50 
size or longer.</P>
<P>
The Broad Institute reports the following statistics for the monDom5 assembly:
<UL>
<LI> Total number of contigs in assembly: 72,811
<LI> Total number of supercontigs in assembly: 5,344
<LI> Placed contigs: 64,313
<LI> Placed supercontigs: 216
<LI> Unplaced contigs: 8,498
<LI> Total bases in chromosomes 1 thru 8 and X: 3,502,373,038
<LI> Total bases in chromosome Un: 103,241,611
<LI> Number of gap bases (==N) in chromosomes 1 thru 8 and X: 89,779,610
<LI> GC/AT ratio in chromosomes 1 thru 8 and X: 1,938,853,501/1,170,581,163 = 0.6037
<LI> GC/AT ratio in chromosome Un: 50,094,290/38,955,502 = .7776
</UL> </P>
<P>
The monDom5 sequence and annotation data can be downloaded from the 
Genome Browser 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/monDom5/">FTP server</A>
or 
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#opossum">Downloads</A> 
page. These data have <A HREF="../goldenPath/credits.html#opossum_use">specific 
conditions for use</A>.
</P>
<P>
Many thanks to The Broad Institute for the opossum assembly data.
The monDom5 annotation 
tracks were generated by UCSC and collaborators worldwide. See the 
<A HREF="../goldenPath/credits.html#opossum_credits">Credits</A> page
for a detailed list of the organizations and individuals who contributed 
to this release. </P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>finished</B>.
