<H2>Description</H2>
This track depicts gaps in the assembly.

<P>
Gaps are represented as black boxes in this track.
If the relative order and orientation of the contigs on either side
of the gap is supported by read pair data, it is a <em>bridged</em>
gap and a white line is drawn through the black box representing the gap. 
</P>
<P>This assembly contains only one type of gap:
<UL>

<LI><B>contig</B> - gaps in the sequencing of the Whole Genome Shotgun contigs.
These are undetermined bases represented by varying numbers of
<em>N</em>s  in the assembly.  Fragment gap sizes are usually taken
from read pair data.  This assembly contains 204 unique gaps covering a total
of 304 bases in the sequence.
</UL></P>

<H2>Credits</H2>
<P>
The Dec. 2008 <em>Pristionchus pacificus</em> (WUGSC 5.0.1) draft assembly was
produced by the
<A HREF="http://genome.wustl.edu/genomes/view/pristionchus_pacificus_var._california/" TARGET=_blank>
Washington University School of Medicine GSC</A> with the following
<A HREF="http://genome.wustl.edu/data/data_use_policy" TARGET=_blank>data use policy.</A>
</P>

