<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=315>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://www.cbcb.umd.edu/"
    TARGET=_blank>
    <IMG SRC="../images/Bos_taurus.jpg" WIDTH=300 HEIGHT=188 ALT="Cow"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT>
    <FONT SIZE=-1><em>Bos Taurus</em><BR>
    </FONT>
    <FONT SIZE=-2>
     Photo courtesy of Michael MacNeil, USDA<BR>
     (NHGRI <A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Cow&id=79109"
     TARGET=_blank>Press Photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The Nov. 2009 <em>Bos taurus</em> draft assembly 
(UMD_3.1, NCBI project 
<A HREF="https://www.ncbi.nlm.nih.gov/bioproject/32899"
TARGET=_blank>32899</A>, GenBank 
<A HREF="ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/vertebrate_mammalian/Bos_taurus/all_assembly_versions/GCA_000003055.3_Bos_taurus_UMD_3.1" 
TARGET=_blank>GCA_000003055.3</A>)
was produced by the Center for Bioinformatics and Computational Biology,
University of Maryland, College Park, MD.
This browser also includes the mitochondrial genome with NCBI accession 
<A HREF="https://www.ncbi.nlm.nih.gov/nuccore/NC_006853"
TARGET=_blank>NC_006853</A>. For more information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/313678/" TARGET="_blank">
Bos_taurus_UMD_3.1</A> in the NCBI Assembly database.
 
<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
bovine genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.

<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>chr28</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of chromosome 28</TD>
    </TR>
    <TR><TD VALIGN=Top>chr28:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of chromosome 28</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr28:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 28 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>BC140604</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number BC140604</TD>
    </TR>
    <TR><TD VALIGN=Top>EE384456</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		EE384456 on chr X</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>
            BC122573; BC102169 <BR>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Displays region between genome landmarks, such as
                      the mRNA markers BC122573 and BC102169.
                      This syntax may also be used for other range queries, such
                      as between uniquely determined ESTs, STSs,
                      refSeqs, etc.</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene mRNA</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zahler</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zahler</TD>
    </TR>
    <TR><TD VALIGN=Top>Evans,J.E.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author J.E. Evans</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Evans JE</em>, internally it
	uses the format <em>Evans,J.E.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
The UMD_3.1 assembly, which is based on reads downloaded from the NCBI Trace
Archive, has a coverage of 9.5X. Approximately 91% of the sequence has been 
anchored to thirty chromosomes: 1-29 and X. The remaining unanchored contigs
are contained in 3286 scaffolds (labeled chrUn_GJnnnnnn). The sequence length is
approximately 2.67 Gb, of which about 20.7 Mb are 'N' bases in gaps. The 
mitochondrial sequence (chrM) was obtained from GenBank (accession NC_006853). 
<P>
The University of Maryland Center for Bioinformatics and Computational Biology
(UMD CBCB) reports that they applied assembly improvement
techniques to produce a version that is substantially more accurate
and complete than existing alternative versions, as evaluated using independent
metrics. Based on UMD statistics, this assembly provides more genome
coverage, closes thousands of gaps, corrects many erroneous inversions,
deletions, and translocations, and fixes thousands of single-nucleotide
errors. For more information, see Zimin AV et al., 
<A HREF="http://genomebiology.com/2009/10/4/R42" TARGET=_blank>A whole-genome
assembly of the domestic cow, <em>Bos taurus</em></A>, <em>Genome
Biol</em>. 2009;10(4):R42.
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/bosTau6/">FTP
server</A> or the 
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#cow">Downloads</A> page. 
<P>
The UMD cow browser annotation tracks were generated by UCSC and collaborators 
worldwide. See the <A HREF="../goldenPath/credits.html#cow_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
