<!DOCTYPE html>
<!--#set var="TITLE" value="Genome Browser BAM Configuration" -->
<!--#set var="ROOT" value="../.." -->

<!-- Relative paths to support mirror sites with non-standard GB docs install -->
<!--#include virtual="$ROOT/inc/gbPageStart.html" -->

<a name="Topic5"></a>
<h1>Configuring BAM tracks</h1>
<p> 
Genome Browser BAM tracks may be configured in a variety of ways to highlight different aspects of 
the displayed information. The configuration options are described here and related to custom track 
settings that can alter the default appearance of the custom track. Click <a href="bam.html" 
target="_blank">here</a> for more information on BAM custom track creation.</p>
<ul>
  <li>
  <strong>Attempt to join paired end reads by name</strong>: This checkbox appears only if 
  <code>pairEndsByName</code> is included in the track settings. When checked, SAM/BAM records with 
  the same name will be joined into pairs for display, with a line drawn between them.</li>
  <li>
  <strong>Minimum alignment quality</strong>: Exclude alignments with quality less than the given 
  number. The default is 0, unless changed by the track setting <code>minAliQual</code>.</li>
  <li>
  <strong>Color track by bases</strong>: By default, mismatching bases are highlighted in the 
  display.  Change the selection to &quot;item bases&quot; to see all base values from the query 
  sequence, or &quot;OFF&quot; to ignore query sequence.</li>
  <li>
  <strong>Additional coloring modes</strong>: Other aspects of the alignments can be displayed in 
  color or grayscale. The default mode is &quot;Color by strand&quot; 
  (<code>bamColorMode=strand</code>), unless the <code>bamColorMode</code> track setting specifies 
  <code>gray</code>, <code>tag</code> or <code>off</code>.  
  <ul>
    <li>
    <strong>Color by strand</strong>: alignments on the reverse strand are colored dark red, 
    alignments on the forward strand are colored dark blue.</li>
    <li>
    <strong>Grayscale</strong>: items are shaded according to the chosen method: alignment quality, 
    base qualities, or unpaired ends. The alignment qualities of items are shaded on a scale of 0 
    (lightest) to 99 (darkest). Base qualities are shaded on a scale of 0 (lightest) to 40 
    (darkest). When &quot;unpaired ends&quot; is selected, items that were paired in sequencing but 
    whose mate was not mapped are colored gray, while singletons and properly paired items are 
    black. Alignment quality is the default (<code>bamGrayMode=aliQual</code>) unless 
    <code>bamGrayMode</code> track setting is <code>baseQual</code> or <code>unpaired</code>.</li>
    <li>
    <strong>Use R,G,B colors specified in user-defined tag</strong>: SAM/BAM may include 
    user-defined tags, whose names begin with X, Y or Z and include one other letter or number. The 
    user-defined tag named here specifies red, green and blue (RGB) intensities	as a 
    zero-terminated string (tag type <code>Z</code>) containing comma-separated triples of numbers 
    from 0-255. For example, if a SAM/BAM record includes the tag <code>YC:Z:255,0,0</code>, then 
    the item is colored red;  <code>YC:Z:0,0,255</code> makes the item blue. By default, the tag is 
    &quot;YC&quot; unless changed using the track setting <code>bamColorTag</code>.</li>
    <li>
    <strong>No additional coloring</strong></li>
  </ul>
  <li>
  <strong>Display data as a density graph</strong>: This feature enables the BAM data to be 
  displayed as a bar graph where the height is proportional to the number of reads mapped to each 
  genomic position. Through dynamic calculation of items in the current window, this feature plots 
  a line similar to a wiggle graph that can be customized with a number of 
  <a href="hgWiggleTrackHelp.html" target="_blank">graph-based configuration</a> options such as 
  drawing indicator lines, smoothing plots, adjusting graph height and vertical range, and switching
  from bars to points. Please note that the feature is best displayed with <em>Display mode</em> set
  to <code>full</code> and that the default <em>Data view scaling</em> is <code>auto-Scale
  to data view</code>. Also, please note that when set to display as a density graph, other BAM 
  display options described on this page, such as coloring for strand, alignment quality and base     qualities, are not applied.</li>
</ul>
<p>
When you have finished making your configuration changes, click the <em>Submit</em> button to return
to the annotation track display page.</p>

<!--#include virtual="$ROOT/inc/gbPageEnd.html" -->
