<!DOCTYPE html>
<!--#set var="TITLE" value="Genome Browser Codon Coloring" -->
<!--#set var="ROOT" value="../.." -->

<!-- Relative paths to support mirror sites with non-standard GB docs install -->
<!--#include virtual="$ROOT/inc/gbPageStart.html" -->

<h1>Codon and Base Coloring for Alignment Tracks</h1> 
<p> 
The Genome Browser's codon coloring feature allows users to quickly validate and compare alignment 
tracks, including mRNAs. To turn on codon coloring, select the desired option from the <em>Color 
track by codons</em> pull-down menu.</p>
<p> 
<strong>Color Options</strong></p> 
<ul> 
  <li>
  <em>genomic codons</em> - codons are labeled and colored according to the genomic sequence</li> 
  <li>
  <em>mRNA codons</em> - codons are labeled and colored according to the aligned sequence</li> 
  <li><em>nonsynonymous mRNA codons</em> - codons are labeled and colored according to the aligned 
  sequence. Only nonsynonymous codons are labeled. Conservative and non-conservative substitutions 
  are defined using the BLOSUM62 score matrix. Conservative substitutions (yellow) are defined as 
  positive BLOSUM scores, while non-conservative substitutions (red) are defined as negative 
  scores.</li> 
  <li>
  <em>mRNA bases</em> - bases are labeled. </li> 
  <li>
  <em>different mRNA bases</em> - only those bases that differ from the genomic sequence are labeled
  and colored.</li> 
</ul>
<p> 
<strong>Color Legend</strong><p> 
<table border="1"> 
  <tr>
    <th>&nbsp;</th>
    <th>Genomic/mRNA codons</th> 
    <th>Nonsynonymous mRNA codons</th> 
    <th>mRNA bases/different mRNA bases</th></tr> 
  <tr>
    <td><font color="FF0000"><strong>red</strong></font></td> 
    <td>&quot;*&quot; stop codons</td> <td>non-conservative codon substitutions</td> 
    <td>base substitutions</td></tr> 
  <tr> 
    <td><font color="E0E000"><strong>yellow</strong></font></td> 
    <td>&nbsp;</td> 
    <td>conservative codon substitutions</td> 
    <td>&nbsp;</td></tr> 
  <tr> 
    <td><font color="00DBDB"><strong>cyan</strong></font></td> 
    <td>spliced or truncated partial codons</td> 
    <td>spliced or truncated partial codons</td> 
    <td>&nbsp;</td></tr> 
  <tr> 
    <td><font color="00E000"><strong>green</strong></font></td> 
    <td>start codons incl. methionine</td> 
    <td>&nbsp;</td> 
    <td>&nbsp;</td></tr> 
  <tr> 
    <td><strong>black</strong></td> 
    <td>&quot;X&quot; truncated codons</td> 
    <td>&nbsp;</td> 
    <td>&nbsp;</td></tr> 
</table>
<p> 
<strong>Details</strong></p> 
<p>
The codon reading frame is defined by the GenBank CDS annotation of the aligned sequence. Each codon
is colored and labeled in the direction of transcription. A sequence without a CDS will not be 
colored; these can be non-protein coding RNAs or an mRNA where a CDS annotation was not provided in 
the original data.</p>
<p> 
The mRNA bases/different mRNA bases display options are  colored and labeled in the direction of the
genomic sequence. Note that it is possible to show the base complement, and thus change the base 
labeling, by clicking the arrow to the left of the base display or by clicking the
&quot;reverse&quot; button.</p>
<p> 
When zoomed out past the base level, the browser will choose one color to represent many bases. The 
priority of display, from most important to least important, is: different mRNA base/nonsynonymous 
codon coloring (if enabled), and then <a href="hgIndelDisplay.html">alignment coloring</a> (if 
enabled). The browser will not display genomic/mRNA codon coloring when viewing large regions of the
genome. To view labeling, the track must be zoomed to within 3 times the base level.</p>
<p> 
For information about alignment insertion/deletion display options, click 
<a href="hgIndelDisplay.html">here</a>.</p> 

<h2>References</h2> 
<p>
Henikoff S and Henikoff JG. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC50453/" 
target="_blank">Amino acid substitution matrices from protein blocks</a>. <em>PNAS</em>. 
1992;89(22):10915-10919.</p>

<!--#include virtual="$ROOT/inc/gbPageEnd.html" -->
