<!DOCTYPE html>
<!--#set var="TITLE" value="Genome Browser hic Tracks" -->
<!--#set var="ROOT" value="../.." -->

<!-- Relative paths to support mirror sites with non-standard GB docs install -->
<!--#include virtual="$ROOT/inc/gbPageStart.html" -->

<!-- hic Track Help Page -->

<h1>Configuring hic tracks</h1>
<p>
Genome Browser hic tracks display heatmaps of chromatin folding data and may be configured
in several ways to highlight different aspects of the data. By default, hic tracks will use
auto-scale options for resolution and score coloring.
</p>
<p>
The following section describes configuration settings available to hic tracks. For more information
on setting up and uploading hic data, click the link on <a href="hic.html" target="_blank">
hic custom track creation</a>.
</p>
<h2>Configuring the display mode</h2>
<p>
There are three display methods available for Hi-C heatmaps: square, triangle, and arc.
<p class='text-center'>
  <img src="../../images/hicDrawModes.png" alt="hic draw modes"> 
</p>
<p>
Square mode provides a traditional Hi-C display in which chromosome positions are mapped along the
top-left-to-bottom-right diagonal. Interaction values are plotted symetrically on both sides of 
that diagonal to form a square. The upper-left corner of the square corresponds to the left-most 
position of the window in view, while the bottom-right corner corresponds to the right-most 
position of the window.
</p><p>
The color shade at any point within the square shows the proximity score for two genomic regions:
the region where a vertical line drawn from that point intersects with the diagonal, and the region
where a horizontal line from that point intersects with the diagonal. A point directly on the
diagonal shows the score for how proximal a region is to itself (scores on the diagonal are usually
quite high unless no data are available). A point at the extreme bottom left of the square shows the
score for how proximal the left-most position within the window is to the right-most position within
the window.
</p><p>
In triangle mode, the display is quite similar to square except that only the top half of the square
is drawn (eliminating the redundancy), and the image is rotated so that the diagonal of the square
now lies on the horizontal axis. This display consumes less vertical space in the image, although it
may be more difficult to ascertain exactly which positions correspond to a point within the
triangle.
</p><p>
In arc mode, simple arcs are drawn between the centers of interacting regions. The color of each arc
corresponds to the proximity score. Self-interactions are not displayed.
</p>
<p>
<strong>Score normalization settings</strong>
</p><p>
Score values for this type of display correspond to how close two genomic regions are in 3D space. A
high score indicates more links were formed between them in the experiment, which suggests that the
regions are near to each other. A low score suggests that the regions are farther apart. High scores
are displayed with a more intense color value; low scores are displayed in paler shades.
</p><p>
There are four score values available in this display: NONE, VC, VC_SQRT, and KR. NONE provides raw,
un-normalized counts for the number of interactions between regions. VC, or Vanilla Coverage,
normalization (Lieberman-Aiden <em>et al</em>., 2009) and the VC_SQRT variant normalize these count
values based on the overall count values for each of the two interacting regions. Knight-Ruiz, or
KR, matrix balancing (Knight and Ruiz, 2013) provides an alternative normalization method where the
row and column sums of the contact matrix equal 1.
</p><p>
Color intensity in the heatmap goes up to indicate higher scores, but eventually saturates at a
maximum beyond which all scores share the same color intensity. The value of this maximum score for
saturation can be set manually by un-checking the &quot;Auto-scale&quot; box. When the
&quot;Auto-scale&quot; box is checked, it automatically sets the saturation maximum to be double 
(2x) the median score in the current display window.
</p><p>
<strong>Resolution settings</strong>
</p><p>
The resolution for each track is measured in base pairs and represents the size of the bins into
which proximity data are gathered. The list of available resolutions ranges from 1kb to 10Mb. There
is also an &quot;Auto&quot; setting, which attempts to use the coarsest resolution that still
displays at least 500 bins in the current window.
</p>
<p>
When you have finished making your configuration changes, click the <button>Submit</button> button 
to return to the annotation track display page.</p>
<h2>References</h2>
<p>
Knight P, Ruiz D.
<a href="https://doi.org/10.1093/imanum/drs019" target="_blank">
A fast algorithm for matrix balancing</a>.
<em>IMA J Numer Anal</em>. 2013 Jul;33(3):1029-1047.
</p><p>
Lieberman-Aiden E, van Berkum NL, Williams L, Imakaev M, Ragoczy T, Telling A, Amit I, Lajoie BR,
Sabo PJ, Dorschner MO <em>et al</em>.
<a href="http://www.sciencemag.org/cgi/pmidlookup?view=long&amp;pmid=19815776" target="_blank">
Comprehensive mapping of long-range interactions reveals folding principles of the human genome</a>.
<em>Science</em>. 2009 Oct 9;326(5950):289-93.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19815776" target="_blank">19815776</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2858594/" target="_blank">PMC2858594</a>

<!--#include virtual="$ROOT/inc/gbPageEnd.html" -->
