<!DOCTYPE html>
<!--#set var="TITLE" value="Genome Browser Indel Display" -->
<!--#set var="ROOT" value="../.." -->

<!-- Relative paths to support mirror sites with non-standard GB docs install -->
<!--#include virtual="$ROOT/inc/gbPageStart.html" -->

<h1>Alignment Insertion/Deletion Display Options</h1>
<p> 
The Genome Browser offers several ways to highlight gaps in alignments of query sequences (usually 
transcripts) to the genome. Gaps result from sequences in the genome or query (or both) that cannot 
be aligned.</p>
<p> 
<strong>Legend</strong></p>
<ul> 
  <li>
  double horizontal line (=): both the genome and query have unalignable sequence between regions of
  aligned sequence, a double-sided insertion. If this option is not selected, it will display as a 
  single horizontal line.</li> 
  <li>
  <font color="#FFA70F">orange lines</font> or <font color="#9214FF">purple lines</font>: 
  unalignable query sequence, orange for the middle of a sequence and purple for the beginning or 
  end.</li> 
  <li>
  <font color="#00E000">green lines</font>: poly-A tail or poly-T head that does not align to the 
  genome.</li> 
</ul> 
<p>
<strong>Details</strong></p> 
<p>
When zoomed out past the base level, the browser chooses one color to represent many bases. The 
priority of display, from most important to least important, is: 
<a href="hgCodonColoringMrna.html">different mRNA base/nonsynonymous codon coloring</a> (if enabled)
<em>or </em> <a href="hgBaseLabel.html">different item bases</a> (if enabled), unalignable query 
sequence (orange or purple), an insertion in both genome and query (double horizontal line), and a 
poly-A tail (green). The browser will not display <a href="hgCodonColoringMrna.html">genomic/mRNA 
codon coloring</a> when viewing large regions of the genome.</p>
<p> 
<strong>Interpretation of Display</strong></p> 
<p>
Gaps are usually due to a deletion or insertion in one or both sequences, or, infrequently, a 
problem in the sequencing. Often, the genome will have a large &quot;insertion&quot; relative to a 
query (single horizontal line) that is actually an intron. Double-sided insertions (double 
horizontal line) are unusual and may indicate an assembly error, sequencing error, or 
polymorphism.</p>
<p> 
Unalignable query sequence in the middle of a query (orange) implies extra bases in the transcript 
sequence or missing bases in the genome sequence. Insertions at the beginning or end of a query 
(purple), implies a partial alignment of the query. For instance, a very short sequence next to 
large intron gap may be incorrectly aligned. Unalignable query sequence may also be due to 
polymorphisms.</p>
<p> 
Poly-T heads result from queries that are the reverse complement of the genomic sequence. Poly-A 
tails and poly-T heads (green) of mRNAs usually can not be aligned to the genome; this is a special 
case of an unalignable query sequence.</p>
<p> 
For information about mRNA codon and base coloring, click 
<a href="hgCodonColoringMrna.html">here</a>. For information about EST base coloring, click 
<a href="hgBaseLabel.html">here</a>.</p>

<!--#include virtual="$ROOT/inc/gbPageEnd.html" -->
