<!DOCTYPE html>
<!--#set var="TITLE" value="Genome Browser Wiggle Tracks" -->
<!--#set var="ROOT" value="../.." -->

<!-- Relative paths to support mirror sites with non-standard GB docs install -->
<!--#include virtual="$ROOT/inc/gbPageStart.html" -->

<!-- Graph-Based Track Help Page -->
<h1>Configuring graph-based tracks</h1> 
<p> 
Genome Browser graph-based tracks may be configured in a variety of ways to highlight different 
aspects of the displayed information. Some track types, such as BAM tracks and certain gene tracks, 
offer dynamic calculation of items in the current window to display a density graph where the height
is proportional to the number of items mapped to each genomic position. The following information 
explains each of the available configuration options.</p>
 
<h3>Display settings</h3>
<p>
<img src="../../images/2conf_enco.png" width="800px" ><br>
<br>
<strong>Type of graph</strong>: The default &quot;Bar&quot; setting depicts the graph data using 
color-filled bars. To view the data as a series of points or lines, select the 
&quot;Points&quot; setting.</p>  
<p>
<strong>Track height (boxed in blue)</strong>: To change the default display height of the graph in
pixels, type in a value from within the indicated range.</p>  

<p> 
<strong>Data view scaling (boxed in red)</strong>: The default &quot;Use vertical viewing range setting&quot; 
option displays the data using the parameters specified in the Vertical viewing range setting. To 
configure the graph to automatically scale to a range defined by the minimum and maximum data 
points in the current view, select the &quot;Auto-scale to data view&quot; option. To keep the 
y=0 value in view at all times when Auto-scale is selected, set &quot;Always include 
zero&quot; to &quot;ON&quot;.</p>
<ul>
  <li>When viewing signal tracks within a composite group use
  &quot;group auto-scale&quot; to enable having all tracks scaled against the one track
  in the group that has the highest maximum data points in the current view.
  For example, below is a side-by-side image of two views of the same data from a selection
  of cell lines within a composite of related RNA-seq experiments. On the left is the
  original &quot;auto-scale to data view&quot; setting, where each track is auto-scaled
  to appear at each track's highest value. And on the right is the new &quot;group
  auto-scale&quot; setting for the same RNA-seq data where all tracks are scaled against
  the one track in the region that has the highest value (67215 for IMR9 cell TAP + 1).<br>
  <img src="../../images/groupAutoScale.png" width='600' alt="Auto-scale comparison"><br>
  Click this <a href="/cgi-bin/hgTracks?hgS_doLoadUrl=submit&hgS_loadUrlName=http://genome.ucsc.edu/goldenPath/help/examples/sessions/session.autoScale"
  target="_blank">link</a> to interact with the original &quot;auto-scale to data view&quot on
  the left, and to contrast the results with the new &quot;group auto-scale&quot; click this
  <a href="/cgi-bin/hgTracks?hgS_doLoadUrl=submit&hgS_loadUrlName=http://genome.ucsc.edu/goldenPath/help/examples/sessions/session.groupAutoScale"
  target="_blank">link</a>.
  <li>Note that the &quot;group auto-scale&quot setting is meant to be turned on and off at the
  composite group track set level, but you can toggle individual tracks. For instance,
  click the second &quot;group auto-scale&quot; session
  <a href="/cgi-bin/hgTracks?hgS_doLoadUrl=submit&hgS_loadUrlName=http://genome.ucsc.edu/goldenPath/help/examples/sessions/session.groupAutoScale"
  target="_blank">link</a> that has all tracks adjusted to
  the highest value (67215 for IMR9 cell TAP + 1). If you right-click that individual highest track
  and instead of configuring the track set, select to configure only the subtrack IMR9 cell TAP + 1
  from &quot;group auto-scale&quot; to &quot;auto-scale to data view&quot and select OK
  and then refresh your browser window (or click one of the &quot;refresh&quot; buttons on the
  screen) you will see all the tracks adjust to the next-highest track within all tracks
  still tagged to display &quot;group auto-scale&quot (39320 for IMR9 cell + 1).</li>
</ul>
<p> 
<strong>Vertical viewing range (boxed in red)</strong>: The min and max values specify the vertical portion of 
the graph that is displayed (default range is 30-70). These numbers can be used to set data 
threshold indicators. For example, to display only those GC values greater than 50 percent, set 
the min value to &quot;50&quot;.</p> 
<p> 
<strong>Transform function</strong>: Transforms the data points by the function selected in the 
drop-down menu. Usually the default setting is &quot;None&quot;.</p> 
<p> 
<strong>Windowing function</strong>: When a  view is too large to show individual data values, the
values must be combined to produce a plot point. This option specifies the combining function to 
be used (default is &quot;Mean&quot;):</p> 
<ul> 
  <li> 
  &quot;Mean+whiskers&quot; - displays the mean in a dark shade, one standard deviation around the
  mean in a medium shade, and the maximum/minimum in a light shade. For bar graphs only the mean, 
  the mean plus a standard deviation, and the max are visible. This mode is not available if the 
  Overlay method is stacked.</li>  
  <li> 
  &quot;Maximum&quot; - displays the maximum value of all the points to be combined.</li> 
  <li> 
  &quot;Mean&quot; - displays the mean.</li> 
  <li> 
  &quot;Minimum&quot; - displays the minimum of all the points to be combined.</li> 
</ul> 
<p> 
<strong>Smoothing window</strong>: When set to a numerical value, this option determines the size,
in pixels, of a smoothing window to be passed over the plot to smooth the edges of the bars or 
lines. This is equivalent to a trend line calculation on the graph. The default setting is 
&quot;OFF&quot;.</p>  
<p> 
<strong>Negate values</strong>: When checked, all values in the wiggle are negated, meaning that 
positive values become negative and vice-versa. This is useful for wiggles representing 
transcription or other activities on the minus strand. Be aware that wiggles with negative values
are drawn in <strong>altColor</strong>  not <strong>color</strong> as positive values
are. The below image shows ENCODE RNA-seq data around two genes on different strands, SIRT1 and 
HERC4, with a minus signal track using <strong>Negate values</strong> to flip the wiggle display to
emphasize that HERC4 is expressed on the minus strand. This image also displays the signal graphed
in points and a smoothing window of 16 pixels.<br>
<img src="../../images/wig_negate-1.png" width="800px" >
</p> 
<p> 
<strong>Draw y indicator lines (boxed in orange)</strong>:</p> 
<ul> 
  <li> 
  <strong>at y= 0.0</strong>: Select the &quot;ON&quot; setting to display a line marking the 0.0 
  position on the graph (default is &quot;OFF&quot;).</li> 
  <li> 
  <strong>at y= </strong>: Select the  &quot;ON&quot; setting to display a line on the graph at 
  the specified numerical value (defaults are &quot;0&quot; and &quot;OFF&quot;). This line can be
  used to mark a significant threshold on the graph. For example, in the image below, 
  <strong>y=3</strong>.<br>
   <img src="../../images/wiggle_y-1.png" width="800px" >
   </li> 
</ul> 
<p> 
When you have finished making your configuration changes, click the <em>Apply</em> button to preview
your changes, or click the <em>Submit</em> button to return
to the annotation track display page.</p> 

<a name="TRACK_CONT"></a>
<h3>Annotation track display modes</h3>
<p>
Each annotation track within the window may have up to five display modes:
<p>
<dl class="gbsNoBullet">
  <dt>Dense</dt> 
  <dd>The track is displayed with all features collapsed into a single line.
  The darker the line color the greater the wiggle value at that location.<br>
  <img src="../../images/wiggle_dense-1.png" alt="Wiggle in dense display" width="800"> 
  </dd>
  <dt>Squish</dt> 
  <dd>The track is displayed with all the features collapsed into a single
  line, much like the dense display mode with greater compression.<br>
  <img src="../../images/wiggle_squish-1.png" width="800px">
  </dd>
  <dt>Pack</dt> 
  <dd>The track displays the wiggle value associated with each annotation feature creating a 
  histogram-like image, much like the full display mode with greater compression.<br>
  <img src="../../images/wiggle_pack-1.png" width="800px" >
  </dd>
  <dt>Full</dt> 
  <dd>The track displays the wiggle value associated with each annotation feature creating a 
  histogram-like image. <br>
  <img src="../../images/wiggle_full-1.png" width="800px">
  </dd>
  <dt>Hide</dt> 
  <dd>The track is not displayed at all. To hide all the annotation tracks, click
  the <em>hide all</em> button.</dd>
</dl>

<h3>Overlay method</h3> 
<p>
Note that not all graph-based tracks include the Overlay options.</p>

<dl class="gbsNoBullet">
  <dt>Transparent</dt>
  <dd>This setting displays the colored 
  transparent graphs of multiple subtracks overlayed in the same vertical space.<br>
  <img src="../../images/trans_encode-1.png" width="800px" height="100px">
  </dd>
  <dt>Solid</dt> 
  <dd>This setting displays the colored opaque graphs of multiple subtracks 
  overlayed in the same vertical space.<br>
  <img src="../../images/encode_solid-1.png" width="800px" height="100px">  
  </dd>
  <dt>Stacked</dt>
  <dd>This setting displays each graph stacked on top of each other where the
  high point of the graph is the sum of all the values.<br>
  <img src="../../images/encode_stack-1.png" width="800px" height="100px">
  </dd>
  <dt>None</dt> 
  <dd>This setting displays each graph in its own vertical space. <br>
  <img src="../../images/encode_none-1.png" width="800px" height="130px">
  </dd>
</dl> 



<!--#include virtual="$ROOT/inc/gbPageEnd.html" -->
