<h2>Description</h2>
<p>
This track shows all occurrences of a selected short motif within the displayed position range of 
the assembly sequence. It is useful for finding oligonucleotides, restriction sites, or other 
recurring short sequences within the assembly. In full display mode, each motif occurrence is 
labeled by the strand on which the match is located, followed by the starting coordinate of the 
match. In cases where the input motif sequence is identical to its reverse complement, only the 
match on the &quot;+&quot; strand is shown.</p> 
<p>
The track may be configured to search for any short sequence of 2 - 30 bases in length. Sequences 
may include <a href="/goldenPath/help/iupac.html" target="_blank">IUPAC ambiguity codes</a>. To change the motif, 
open the track's description page (by clicking the track control label or the mini-button to the 
left of the track), then type a new sequence into the text box.</p>
<p>
To see how to create a bed file of the short match data see this mailing list question
<a href="https://groups.google.com/a/soe.ucsc.edu/d/msg/genome/zlvaV8Wzeaw/xPwdY7mQ0f8J"
target="_blank">here</a>.</p>

<h2>Credits</h2>
<p>
This track was generated by 
<a href="mailto:&#107;&#101;&#110;t&#64;&#115;&#111;e.&#117;&#99;&#115;&#99;.&#101;&#100;&#117;">Jim Kent</a> 
of the UCSC Genome Bioinformatics Group.</p>

