<H2>Description</H2>
<P>
This track shows conservation based the Multiple Sequence Alignment (MSA) 
results of VAX003 sub-type B HIV-1 gp120 nucleotide sequences.
</P>

<H2>Methods</H2>
<h3>Conservation</h3>
<P>At each genomic position of the MSA, the counts of different bases 
(A, T, C, G, "-") were tallied and the frequency distribution calculated.  
The most frequent base at each genomic position was used as the 
conservation measure.
</P>
<h3>MSA method</h3>
<P>
Katoh <EM>et al.</EM> reported modifications of the MAFFT program 
which were used in these multiple alignments.  These modifications included 
two local (L-INS-i and E-INS-i) and one global (G-INS-i) iterative 
refinement algorithms.
MAFFT also incorporates a BLOSUM62 scoring matrix   
(an empirical matrix calculated from large datasets representing extreme protein 
family diversity) and assumes a fixed frequency for each AA for building MSAs; 
however, the estimated relative rates of changes in this matrix may be too 
general to fit datasets of specific gene families or high substitution rates, 
such as gp120. Hence, here we performed multiple MAFFT alignments using both 
the BLOSUM62 and gp120-specific scorings matrixes and/or fixed and variable 
AA frequencies estimated for the data at hand. Gene-specific empirical matrices 
and AA frequencies based on the VAX003 and LAGB-VAX003 datasets were generated 
via the program 
<A HREF="http://matrixgen.sourceforge.net" TARGET=_blank>MATRIXGEN</A>.
</P>
<P>The quality of the MSA was then compared using the maximum likelihood 
(ML) tree building approach in PhyML (Guindon and Gascuel 2003) under the 
GTR+G+I model. ML trees showing the highest likelihood scores were indicative 
of better alignments. G-INS-i + BLOSUM62 + AA specific frequencies generated 
the trees with the best likelihood scores. 
</P>
<P>Uncertainty in our MSAs was assessed using GBlocks v0.91b (Castresana 2000). 
Ambiguous blocks were selected and removed according to the following settings: 
<UL>
 <LI>minimum number of sequences for a conserved position = 517</LI>
 <LI>minimum number of sequences for a flank position = 877</LI>
 <LI>maximum number of contiguous non-conserved positions = 8</LI>
 <LI>minimum length of a block = 10</LI>
 <LI>allowed gap positions = all</LI>
</UL>
These settings generated a conserved MSA of 1,401 sites.

<H2>Credits</H2>
<P>
The MSA of VAX003 HIV-1 sequences was performed by Keith A. Crandall at 
<A HREF="http://www.genoma-llc.com" TARGET=_BLANK>Genoma LLC</A>.
</P>

<H2>References</H2>
<P>
Castresana J. <A HREF="http://mbe.oxfordjournals.org/cgi/content/abstract/17/4/540" TARGET=_blank>Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis</A>. 
<EM>Molecular Biology and Evolution</EM>. 2000 April;17(4):540-52.
</P>
<P>
Guindon S, Gascuel O.
<A HREF="https://www.ncbi.nlm.nih.gov/sites/entrez?cmd=Retrieve&db=PubMed&list_uids=14530136&dopt=Citation" TARGET=_blank>A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</A>.
<EM>Syst Biol</EM>. 2003;52(5):696-704.
</P>
<P>
Katoh K, Kuma K, Toh H, Miyata T. 
<A HREF="http://nar.oxfordjournals.org/cgi/content/abstract/33/2/511" TARGET=_blank>MAFFT version 5: improvement in accuracy of multiple sequence alignment</A>
. <EM>Nucleic Acids Res</EM>. 2005;33(2):511-18.
</P>
