<!-- Display image in righthand corner -->
<!--
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=185>
  <TR><TD ALIGN=RIGHT>
    <A HREF="*** link to sequencing center's project page"
    TARGET=_blank>
    <IMG SRC="../images/Meloidogyne_hapla.jpg" WIDTH=*** HEIGHT=*** ALT="Root knot"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Meloidogyne hapla</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of ***
      (<A HREF="*** link to source" 
      TARGET=_blank>***source</A>)
    </FONT>
  </TD></TR>
</TABLE>
-->

<P>
The Sep. 2008 <em>Meloidogyne hapla</em> draft assembly 
was produced by
<A HREF="http://www.pngg.org/cbnp/index.php"
TARGET=_blank>The Center for the Biology of Nematode Parasitism</A>,
N.C. State, sequencing center.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
M. hapla genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>MhA1_Contig127:1-100,000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first hundred thousand bases of contig MhA1_Contig127</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>MhA1_Contig127:100,000+2,000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of contig MhA1_Contig127 that spans 2000 bases, starting with position 100,000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>AF117339</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number AF117339</TD>
    </TR>
    <TR><TD VALIGN=Top>BM883020</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		BM883020</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zahler</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named zahler</TD>
    </TR>
    <TR><TD VALIGN=Top>Taguchi,T.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author T. Taguchi</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Taguchi T</em>, internally it
	uses the format <em>Taguchi,T.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
See also, ABLG01 WGS
<A HREF="https://www.ncbi.nlm.nih.gov/Traces/wgs/?val=ABLG01"
TARGET=_blank>information.</A>
</P>
<P>
This draft assembly is composed of 3,452 contigs for a total size of
53,017,507 nucleotides.  The largest contig, MhA1_Contig88, is
360,446 nucleotides.  One half of the sequence, 26,508,753 bases,
is contained in 372 contigs, all of at least 37,608 bases (N50).
There are no gaps in these contigs.
</P>
<P>
The included mitochondrial sequence is from the genbank accession
<A HREF="https://www.ncbi.nlm.nih.gov/nuccore/195972419"
TARGET=_blank>NC_010383.2</A>.
</P>

<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/melHap1/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#melHap">Downloads</A> page. 

The Meloidogyne hapla browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#worm_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
