<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=215>
  <TR><TD ALIGN=RIGHT><A HREF="http://www.genoscope.cns.fr/spip/Tetraodon-nigroviridis-a-fish-with.html" TARGET=_blank>
    <IMG SRC="../images/Tetraodon_nigroviridis.jpg" WIDTH=200 HEIGHT=96
      ALT="Tetraodon"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Tetraodon nigroviridis</em><BR> 
    </FONT>
    <FONT SIZE=-2> (Photo courtesy of
      <A HREF="http://www.genoscope.cns.fr/spip/" TARGET=_blank>Genoscope</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The <em>Tetraodon nigroviridis</em> V7 assembly (February 2004)
was provided by 
<A HREF="http://www.genoscope.cns.fr/spip/" TARGET=_blank>Genoscope</A>,
Evry, France.  The assembly is the result of a collaboration
between Genoscope and the
<A HREF="https://www.broadinstitute.org/scientific-community/science/projects/mammals-models/vertebrates-invertebrates/other-vertebrates-and"
TARGET=_blank>Eli and Edythe L. Broad Institute</A> of MIT and
Harvard, MA, USA.  The sequence data, which were assembled using
the Arachne program, were generated by both institutes. The project
was supported by the Consortium National de Recherche en Genomique
and the National Human Genome Research Institute (NHGRI).
For more information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/228438/" TARGET="_blank">
ASM18073v1</A> in the NCBI Assembly database.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of
an mRNA or EST, a chromosome range, or keywords from 
the GenBank description of an mRNA. The following list provides 
examples of various types of position queries for the Tetraodon
genome. Note that some position queries (e.g. "huntington") may
return matches to the mRNA records of other species. In these
cases, the mRNAs are mapped to their homologs in Tetraodon.
See the <A HREF="../goldenPath/help/hgTracksHelp.html"
TARGET=_blank>User Guide</A> for more information. 
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
<TR><TD VALIGN=Top NOWRAP><B>Request:</B><br></TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top><B>Genome Browser Response:</B><br></TD></TR>
	
<TR><TD VALIGN=Top><br></TD></TR>
	
<TR><TD VALIGN=Top NOWRAP>chr1</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays all of chromosome 1</TD></TR>
<TR><TD VALIGN=Top NOWRAP>chr1:1-200000</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays first two hundred thousand bases of chromosome 1</TD></TR>
    <TR><TD VALIGN=Top NOWRAP>chr1:100000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chromosome 1 that spans 2000 bases, starting with position 100000 </TD>
    </TR>
<TR><TD VALIGN=Top><br></TD></TR>

<TR><TD VALIGN=Top NOWRAP>CR731623</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region containing Tetraodon mRNA with GenBank accession number CR731623</TD></TR>

<TR><TD VALIGN=Top NOWRAP>BU572495</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region containing Tetraodon EST with GenBank accession BU572495
</TD></TR>

<TR><TD VALIGN=Top><br></TD></TR>

<TR><TD VALIGN=Top NOWRAP>IL11</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs related to IL11 (interleukin-11)</TD>
</TR>


<TR><TD VALIGN=Top NOWRAP>pseudogene mRNA</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists transcribed pseudogenes but not cDNAs, in GenBank</TD></TR>
<TR><TD VALIGN=Top NOWRAP>T-cell receptor</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs for T-cell receptor genes in GenBank</TD></TR>
<TR><TD VALIGN=Top NOWRAP>kinase</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists many kinase mRNAs</TD></TR>
<TR><TD VALIGN=Top NOWRAP>tyrosine kinase</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists only tyrosine kinases</TD></TR>
<TR><TD VALIGN=Top NOWRAP>breast cancer</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs associated with breast cancer</TD></TR>

<TR><TD VALIGN=Top><br></TD></TR>

<TR><TD VALIGN=Top NOWRAP>abkevich</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Abkevich</TD></TR>
<TR><TD VALIGN=Top NOWRAP>Lutfalla,G.</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by co-author G. Lutfalla</TD></TR>

<TR><TD VALIGN=Top><br></TD></TR>
	
<TR><TD COLSPAN="3" > Use this last format for author queries. Although 
GenBank requires the search format <em>Lutfalla G</em>, internally it uses
the format <em>Lutfalla,G.</em>
</TD></TR>
</TABLE>

<P>
<HR>
<H3>Assembly details</H3>
<P>
This assembly was constructed using the whole genome shotgun (WGS) approach, 
resulting in a sequence coverage of about 7.9X. The assembly contains 45,609 
contigs and 25,773 scaffolds generated by the Arachne program and covers more
than 90% of the genome. Additional linking data were used to build ultracontigs 
and to organize the assembly into chromosomes.  Genoscope estimates the size of 
the Tetraodon genome to be about 385 Mb.
</P>
<P>
Downloads of the Tetraodon data and annotations may be obtained from the UCSC
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/tetNig1/" TARGET=_blank>FTP 
site</A> or <A HREF=http://hgdownload.soe.ucsc.edu/downloads.html#tetraodon>
Downloads page</A>. 
These data have been freely provided by Genoscope before publication for use
in the UCSC Genome Browser with 
<A HREF="../goldenPath/credits.html#tetraodon_use">specific conditions for use</A>.
</P>
<P>
Many thanks to Genoscope and the Broad Institute of MIT and Harvard for this
genome assembly. The tetNig1 annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="../goldenPath/credits.html#tetraodon_credits">credits</A> page
for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
