<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=315 HEIGHT=215>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://panda.genomics.org.cn/page/panda/index.jsp"
    TARGET=_blank>
    <IMG SRC="../images/Ailuropoda_melanoleuca.jpg" WIDTH=300 HEIGHT=200 ALT="Panda"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Ailuropoda melanoleuca</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of Jeff Kubina<BR>
      (<A HREF="http://commons.wikimedia.org/wiki/File:Giant_Panda_2004-03-2.jpg" 
      TARGET=_blank>Wikimedia Commons</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The Dec. 2009 <em>Ailuropoda melanoleuca</em> draft assembly 
(BGI-Shenzhen AilMel 1.0 Dec. 2009) was produced by the
Beijing Genomics Institute at Shenzhen
<A HREF="http://www.genomics.cn/en/index" TARGET=_blank>
(BGI-Shenzhen)</A> sequencing center. For more information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/7268/" TARGET="_blank">
AilMel_1.0</A> in the NCBI Assembly database.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Panda genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>GL192360</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of scaffold GL192360</TD>
    </TR>
    <TR><TD VALIGN=Top>GL192370.1:1-1,000,000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of scaffold GL192370</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>GL192379.1:1,000,000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of scaffold GL192379.1 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>DQ852340</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number DQ852340</TD>
    </TR>
    <TR><TD VALIGN=Top>AF325709</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		AF325709 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>liao</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Liao</TD>
    </TR>
    <TR><TD VALIGN=Top>Huang,D.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author D. Huang</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Huang D</em>, internally it
	uses the format <em>Huang,D.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
The genome has been sequenced to 94% coverage.
See also:
<A HREF="http://www.nature.com/nature/journal/v463/n7279/full/nature08696.html"
TARGET=_blank>The sequence and de novo assembly of the giant panda genome</A>. <I>Nature.</I> 2010 January 21;463:311-317.
</P>
<P>
There are 200,592 WGS fragments for a total of 2,245,296,026 bases,
with 119,126 fragment gaps in a combined total of 54,196,184 bases
of <em>N</em>s.  All these gaps are bridged gaps.  The mitochondrial
sequence, accession <A HREF="https://www.ncbi.nlm.nih.gov/nuccore/NC_009492.1"
TARGET=_blank>NC_009492</A> is included as <em>chrM</em>.
</P>

<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/ailMel1/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#panda">Downloads</A> page. 

The Panda browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#panda_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
