<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=365>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://genome.jgi-psf.org/Brafl1/Brafl1.info.html"
    TARGET=_blank>
    <IMG SRC="../images/Branchiostoma_floridae.jpg" WIDTH=350 HEIGHT=121 ALT="Lancelet"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Branchiostoma lanceolatum</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of 
      <A HREF="http://www.obs-banyuls.fr/en/index.php"
      TARGET=_blank>Michael Fuentes and H&eacute;ctor Escriv&agrave;,
		Observatoire Oc&eacute;anologique de Banyuls sur mer</A>. All rights reserved.
    </FONT>
  </TD></TR>
</TABLE>

<P>
The Mar. 2006 <em>Branchiostoma floridae</em> draft assembly 
(JGI v.1.0 (March 2006)) was produced by the
<A HREF="http://genome.jgi-psf.org/Brafl1/Brafl1.info.html" TARGET=_blank>
Joint Genome Institute</A>.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Lancelet genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>chrM</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of chromosome M</TD>
    </TR>
    <TR><TD VALIGN=Top>chrUn:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of chr Un</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chrUn:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr Un that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>AF303216</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number AF303216</TD>
    </TR>
    <TR><TD VALIGN=Top>AI391435</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		AI391435</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zhang</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zhang</TD>
    </TR>
    <TR><TD VALIGN=Top>Taguchi,T.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author T. Taguchi</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Taguchi T</em>, internally it
	uses the format <em>Taguchi,T.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
The genome of Branchiostoma floridae is estimated to be
approximately 575 Mb contained in 19 pairs of chromosomes,
and is being sequenced to approximately 8.1 X depth.
</P>
<P>
v.1.0 (March 2006): Approximately 6.5 Million shotgun reads were initially
assembled using JAZZ. A high allelic polymorphism rate of 5-10% allowed the
two haplotypes to be assembled separately at approximately 75% of genomic loci.
There are a total of 3,032 scaffolds, with a total length of 923 Mb composed
of 81,073 contigs. Half of the assembly is contained in 174 scaffolds, all at
least 1.6 Mb in length. The length-weighted mean contig size (L50) is 26 kb.
</P>
The lancelet 
mitochondrial sequence is also available as the virtual chromosome 
&quot;chrM&quot;. (GenBank: NC_000834.1 GI:5881414)
</P>

<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/braFlo1/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#lancelet">Downloads</A> page. 

These data have
<A HREF="../goldenPath/credits.html#lancelet_use">specific conditions for use</A>. 

The Lancelet browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#lancelet_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
