<H2>Description</H2>
<P>
Bacterial artificial chromosomes (BACs) are a key part of many 
large-scale sequencing projects.  A BAC typically consists of 50 - 300 kb of
DNA.  During the early phase of a sequencing project, it is common
to sequence a single read (approximately 500 bases) off each end of
a large number of BACs.  Later on in the project, these BAC end reads
can be mapped to the genome sequence. </P>
<P>
This track shows these mappings
in cases where both ends could be mapped.  These BAC end pairs can
be useful for validating the assembly over relatively long ranges.  In some
cases, the BACs are useful biological reagents.  This track can also be
used for determining which BAC contains a given gene, useful information
for certain wet lab experiments.</P>
<P>
The <A HREF="https://bacpacresources.org/dromel98.htm" TARGET=_blank>RPCI-98</A> 
and 
<A HREF="https://bacpacresources.org/libraries.php" TARGET=_blank>DrosBAC</A> 
libraries, individual clones, and hybridization filters are available 
from the 
<A HREF="https://bacpacresources.org/" TARGET=_blank>BACPAC 
Resources Center (BPRC)</A> at Children's Hospital Oakland Research 
Institute (CHORI). 
Individual clones from the RPCI-98 library are named BACR01A01 - 
BACR48H12 (96-well format; R stands for EcoRI).  
Individual clones from the DrosBAC library are named BACN01A01 - 
BACN47H12 and BACH48A01 - BACH61H12 (N stands for NdeII; H stands for HinDIII).
</P>
<P>
In order to be included in this track, a 
valid pair of BAC end sequence alignments must be
at least 25 kb but no more than 500 kb away from each other. 
The orientation of the first BAC end sequence must be &quot;+&quot; and
the orientation of the second BAC end sequence must be &quot;-&quot;.</P>
<P>
The scoring scheme used for this annotation assigns 1000 to an alignment 
when the BAC end pair aligns to only one location in the genome (after 
filtering).  When a BAC end pair or clone aligns to multiple locations, the 
score is calculated as 1500/(number of alignments).</P>

<H2>Methods</H2>
<P>
BAC end sequences were downloaded from 
<a href="https://jacob.cea.fr/drf/ifrancoisjacob/english/Pages/Departments/Genoscope.aspx"
target="_blank">Genoscope</a> 
(<a href="https://jacob.cea.fr/drf/ifrancoisjacob/english/Pages/Departments/Genoscope/Genoscope_s-bioinformatics-resources.aspx"
target="_blank">https://jacob.cea.fr/drf/ifrancoisjacob/english/Pages/Departments/Genoscope/Genoscope_s-bioinformatics-resources.aspx</a>) 
and then placed on the assembled sequence using Jim Kent's 
<a href="https://genome.cshlp.org/content/12/4/656.abstract"
target="_blank">blat</a> program.
Terry Furey's pslPairs program was used to identify paired end alignments.
</P>

<H2>Credits</H2>
<P>
The <A HREF="https://bacpacresources.org/dromel98.htm" TARGET=_blank>RPCI-98</A> 
BAC library was produced by BACPAC Resources, then at
Roswell Park Cancer Institute and now at 
<A HREF="https://bacpacresources.org/" TARGET=_blank>CHORI</A>,
in collaboration with the 
<A HREF="https://www.fruitfly.org/" 
TARGET=_blank>Berkeley Drosophila Genome Project</A>.
The 
<A HREF="https://bacpacresources.org/libraries.php" TARGET=_blank>DrosBAC</A> 
library was made by Alain Billaud at 
<A HREF="https://cephb.fr/" TARGET=_blank>CEPH 
(Centre d'Etude du Polymorphisme Humain)</A> in a
collaboration with the European Drosophila Genome Project co-ordinated
by D. Glover.
Thanks to <a href="https://jacob.cea.fr/drf/ifrancoisjacob/english/Pages/Departments/Genoscope.aspx"
target="_blank">Genoscope</A> for providing the BAC end sequence files.</P>
