<H2>Description</H2>
<P>
This track shows the draft assembly of the $organism genome.  

Whole-genome shotgun reads were assembled into contigs.  When possible, 
contigs were grouped into scaffolds (also known as &quot;supercontigs&quot;).
The order, orientation and gap sizes between contigs within a scaffold are
based on paired-end read evidence. </P>
<P>
Locations of contigs and scaffolds were deduced from runs of Ns in the
assembled sequence.  A run of Ns of more than 50000 bases
was assumed to be a gap between scaffolds, and a run of Ns between
20 and 50000 was assumed to be a gap
between contigs.</P>
<P>
In dense mode, this track depicts the contigs that make up the 
currently viewed scaffold. 
Contig boundaries are distinguished by the use of alternating gold and brown 
coloration. Where gaps
exist between contigs, spaces are shown between the gold and brown
blocks.  The relative order and orientation of the contigs
within a scaffold is always known; therefore, a line is drawn in the graphical
display to bridge the blocks.</P>
<P>
All components within this track are of fragment type &quot;D&quot;: 
Draft. </P>
