<H2>Description</H2>
<P>
This track depicts gaps in the draft assembly
(Nov. 2009, SGSC Sscrofa9.2 (NCBI project 10718, GCA_000003025.2))
of the $organism genome.

The pig genome assembly is a hybrid of BAC end-sequencing to
anchor contigs combined with whole-genome shotgun (WGS) libraries.
See also: <A HREF="https://www.sanger.ac.uk/resources/downloads/othervertebrates/pig.html"
TARGET=_blank>Porcine Genome Sequencing Project</A> description.
</P>

<P>
In dense mode, this track depicts the contigs that make up the 
currently viewed chromosome.  Contig boundaries are distinguished by
the use of alternating gold and brown coloration.  Where gaps
exist between contigs, spaces are shown between the gold and brown
blocks.  When the relative order and orientation of the contigs
within a chromosome are known a line is drawn in the graphical display
to bridge the blocks.</P>

<P>
Components in this track are of types:
<UL>
<LI>&quot;A&quot;: Active finishing</LI>
<LI>&quot;D&quot;: Draft sequence</LI>
<LI>&quot;F&quot;: Finished sequence</LI>
</UL>
See also: <A HREF="https://www.ncbi.nlm.nih.gov/assembly/agp/AGP_Specification/"
TARGET=_blank>AGP specification</A>.
</P>
