<h2>Description and display conventions</h2>

<p>
<b>NumtS</b> (Nuclear mitochondrial sequences) are mitochondrial fragments inserted 
in nuclear genomic sequences. The most credited hypothesis concerning their generation 
suggests that in presence of mutagenic agents, or under stress conditions, fragments of 
mtDNA escape from mitochondria, reach the nucleus and insert into chromosomes during 
break repair; although NumtS can also derive from duplication of genomic fragments. 
NumtS may be a cause of contamination during mtDNA sequencing and hence frequent false 
low heteroplasmic evidences have been reported. The Bioinformatics group chaired 
by M. Attimonelli (University of Bari, Italy) has produced the RPNumtS (Reference Pig 
NumtS) compilation annotating  Pig assembled NumtS. To allow the scientific community to 
access the compilation and to perform genomics comparative analyses inclusive of the 
NumtS data, the group has designed the Pig NumtS tracks described below.
</p>

<p>
The NumtS tracks show nuclear and mitochondrial regions, based on the High Score Pairs 
(HSPs) obtained by aligning the mitochondrial reference genome (NC_012095) with the 
susScr2 assembly of the pig genome.
</p>
 
<ol>
<li><b>"NumtS (Nuclear mitochondrial Sequences)" Track</b>
<p>
The "NumtS mitochondrial sequences" track shows the mapping of the HSPs returned by 
BlastN on the nuclear genome. The shading of the items reflect the similarity returned 
by BlastN, and the direction of the arrows is concordant with the strand of the 
alignment. For every item, a link pointing to the mitochondrial mapping is provided, 
thus allowing easier cross referencing between the NumtS subtracks.
</p>
</li>

<li><b>"NumtS assembled" Track</b>
<p>
The "NumtS assembled" track shows items obtained by assembling HSPs annotated in the 
"NumtS" track fulfilling the following conditions:
<ul>
<li>The orientation of their alignments must be concordant.</li>
<li>The distance between them must be less than 2 kb, on the mitochondrial genome 
as well as on the nuclear genome.</li>
</ul>
<p>
Exceptions for the second condition arise when a long repetitive element is present between two HSPs.
</p>
</li>
 

<li><b>"NumtS on mitochondrion" Track</b>
<p>
The "NumtS on mitochondrion" track shows the mapping of the HSPs on the mitochondrial 
genome. The shading of the items reflects the similarity returned by BlastN, and the 
direction of the arrows is concordant with the strand of the alignment. 
For every, a link pointing to the nuclear mapping is provided.
</p>
 
<li><b>"Pig NumtS on mitochondrion SNP" Track</b>
<p>
The "Pig NumtS SNP" shows the mapping of the HSPs on the mitochondrial genome, with 
the SNPs which fall within, derived from comparison with the susScr2 assembly. No shading is provided here.
</p>
</ol> 

<h2>Methods</h2>
<p>
NumtS mappings were obtained by running Blast2seq (program: BlastN) between each chromosome 
of the Pig Genome (susScr2 assembly) and the Pig mitochondrial reference sequence (AC: NC_012095), 
fixing the e-value threshold to 1e-03. The assembling of the HSPs was performed with spreadsheet 
interpolation and manual inspection. BED format is used for the first three annotation tracks, 
while for the last one the SAM/BAM format is preferred.
</p>
 
<h2>Credits</h2>
<p>
These data were provided by Francesco Maria Calabrese, Dario Leonardo Balacco, Domenico Simone 
and Marcella Attimonelli from the Department of of Biosciences, Biotechnology and Pharmacological 
Sciences (University of Bari, Italy).
Manual inspection and format details have been carried out by Francesco Maria Calabrese, 
Dario Leonardo Balacco and Domenico Simone.
</p>

<h2>References</h2>

<p>
Calabrese FM, Simone D, Attimonelli M.
<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S4-S15" target="_blank">
Primates and mouse NumtS in the UCSC Genome Browser</a>.
<em>BMC Bioinformatics</em>. 2012 Mar 28;13 Suppl 4:S15.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22536961" target="_blank">22536961</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3314570" target="_blank">PMC3314570</a>
</p>

<p>
Lascaro D, Castellana S, Gasparre G, Romeo G, Saccone C, Attimonelli M.
<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-9-267" target="_blank">
The RHNumtS compilation: features and bioinformatics approaches to locate and quantify Human
NumtS</a>.
<em>BMC Genomics</em>. 2008 Jun 3;9:267.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18522722" target="_blank">18522722</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2447851" target="_blank">PMC2447851</a>
</p>

<p>
Simone D, Calabrese FM, Lang M, Gasparre G, Attimonelli M.
<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-12-517" target="_blank">
The reference human nuclear mitochondrial sequences compilation validated and implemented on the
UCSC genome browser</a>.
<em>BMC Genomics</em>. 2011 Oct 20;12:517.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22013967" target="_blank">22013967</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3228558" target="_blank">PMC3228558</a>
</p>
