<H2>Description and Methods </H2>

<p>H3K4me1 and H3K4me3 binding at 24 hours post fertilization in the zebrafish embryo.</p>

<P>Methylation of lysine 4 on histone H3 can serve as a landmark of cis regulatory elements in the
genome.  In particular, tri-methylation of lysine 4 (H3K4me3) usually marks promoters of active
genes, while mono-methylation (H3K4me1) can be more widely distributed and associates with multiple
regulatory elements, including enhancers and insulators.  Given the increasing importance and value
of the zebrafish as a model system, an initial assay of H3K4me1 and H3K4me3 binding in the zebrafish
embryo at 24 hours post fertilization was undertaken, a time at which most of the organs systems have
begun to develop.  These data provide a valuable resource for preliminary identification of cis
regulatory features in the zebrafish embryonic genome.</p>

<H2>Display Conventions and Configuration</H2>

<ul>
	<li><b>Signal</b> - Density coverage of sequence tags for a given sample. </li>
	<li><b>Peaks</b> - Genome coordinates of highest tag density within a hotspot; identified using
MACS.</li>
	<li><b>Hotspots</b> - Regions of enrichment for binding over input alone (p&lt;1x10-5); identified
using MACS.</li>
</ul>


<H2>Methods</H2>

<p>For each ChIP, 200 wild-type zebrafish embryos staged at 24 hours post fertilization (hpf) were 
dissociated into single cell suspensions. Cells were cross-linked with a 1% formaldehyde/PBS
solution for 10 minutes at room temperature.  After adding glycine, the samples were centrifuged and
pellets resuspended in lysis buffer. The samples were sonicated using a Microsonicator (Cole and
Palmer Instruments) to a size distribution between 200-1,000 bp. Immunoprecipitation was performed
using anti-H3K4me1 (ab8895, Abcam) or anti-H3K4me3 antibody (ab8580, Abcam) overnight at 4&deg;C. 
Both antibody-treated and untreated samples were incubated with protein A agarose beads for one hour
at 4&deg;C and subsequently washed.  Immunoprecipitated DNAs were purified using a PCR purification
kit. DNA, from both antibody and input samples, was polished and A-tailed followed by ligation to
Illumina adapters.  Adaptor-ligated fragments were gel-purified and amplified by PCR, followed by a
final gel purification.  Single end 36 or 72 bp reads from adaptor-ligated libraries were obtained
by deep sequencing on Illumina Genome Analyzers operated by the UMass Center for AIDS Research Deep
Sequencing Core.</p>

<P>Sequence tags were mapped to the zebrafish genome (Zv7) using Bowtie, and only tags with unique
genomic coordinates and less than 2 bp of mismatch were included in subsequent analyses.  The
"Signal" track was generated by extending each uniquely mapped sequence tag to 250 bp in length,
followed by coverage calculation for each position in a 250 bp moving window.  To minimize file size
of these annotation tracks, only genomic positions with at least 10 sequence tags were used to
generate wiggle files.  Regions of H3K4me1 or H3K4me3 enrichment over input (referred to hereafter
as "hotspots") and "peaks" of binding within these enriched regions were determined using the
Model-based Analysis of ChIP-Seq algorithm (MACS, version 1.4; (Zhang, et al. 2008) with the default
settings and the following modified parameters: -f BED  -w --single-wig --gsize 1700000000 --tsize
36 --bw 150).</p>


<H2>Verification</H2>

<p>Several of the identified peaks have been functionally validated using in vivo reporter assays in
zebrafish.</p>

<H2>Credits</H2>

<p>These data were generated by <a href="https://www.umassmed.edu/Lawson-Lab"
target="_blank">the Lawson Lab</a>. Aaron Aday generated the ChIP-Seq libraries.
Computational analysis was performed by Abirami Lakshmanan and Dr. Julie Zhu.</p>

<P>Contact: Nathan Lawson (Nathan.lawson@um<span style="display: none;">-null-</span>assmed.edu)</p>

<H2>References</H2>
<p>
Aday AW, Zhu LJ, Lakshmanan A, Wang J, Lawson ND.
<a href="https://www.sciencedirect.com/science/article/pii/S0012160611001564" target="_blank">
Identification of cis regulatory features in the embryonic zebrafish genome through large-scale
profiling of H3K4me1 and H3K4me3 binding sites</a>.
<em>Dev Biol</em>. 2011 Sep 15;357(2):450-62.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21435340" target="_blank">21435340</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3273848/" target="_blank">PMC3273848</a>
</p>

<p>
Zhang Y, Liu T, Meyer CA, Eeckhoute J, Johnson DS, Bernstein BE, Nusbaum C, Myers RM, Brown M, Li W
<em>et al</em>.
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2008-9-9-r137" target="_blank">
Model-based analysis of ChIP-Seq (MACS)</a>.
<em>Genome Biol</em>. 2008;9(9):R137.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18798982" target="_blank">18798982</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2592715/" target="_blank">PMC2592715</a>
</p>
