<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=185>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://genome.jgi-psf.org/Nemve1/Nemve1.home.html"
    TARGET=_blank>
    <IMG SRC="../images/Nematostella_vectensis.jpg" WIDTH=*** HEIGHT=*** ALT="Starlet sea anemone"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Nematostella vectensis</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of Mansi Srivastava
      (<A HREF="http://cigbrowser.berkeley.edu/index.php?option=com_content&task=view&id=32&Itemid=20" 
      TARGET=_blank>CIG UC Berkeley</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The Jun. 2007 <em>Nematostella vectensis</em> draft assembly 
is based on sequence version Nemve1 assembled and sequenced by the
<A HREF="http://genome.jgi-psf.org/Nemve1/Nemve1.home.html">US DOE Joint Genome Institute(JGI)</A>.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Starlet sea anemone genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>scaffold10</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of scaffold10</TD>
    </TR>
    <TR><TD VALIGN=Top>scaffold3:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of scaffold 3</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>scaffold3:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of scaffold 3 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>BX950328</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number BX950328</TD>
    </TR>
    <TR><TD VALIGN=Top>BU335953</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		BU335953 on scaffold 3</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene mRNA</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zhang</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zhang</TD>
    </TR>
    <TR><TD VALIGN=Top>Taguchi,T.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author T. Taguchi</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Taguchi T</em>, internally it
	uses the format <em>Taguchi,T.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
Nematostella vectensis genome assembly 1.0 constructed with the JGI assembler, Jazz, includes approximately 7.8X WGS in small insert end-sequence coverage. After trimming for vector and quality, and excluding short/redundant scaffolds, there were 10,942 assembled scaffolds, containing 2,817,779 reads, and 357 Mbp of sequence. Further exclusion of contaminant and mis-assembled scaffolds reduced this to 10,804 scaffolds, with a total length of 356 Mbp. Roughly half of the genome is contained in 181 scaffolds all at least 473 Kb in length (N50).
</P>


<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/nemVec1/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#starlet_sea anemone">Downloads</A> page. 

These data have
<A HREF="http://genome.jgi-psf.org/Nemve1/Nemve1.download.html">
specific conditions for use</A>. The genome assembly was presented in:
<p>
Putnam NH, Srivastava M, Hellsten U, Dirks B, Chapman J, Salamov A, Terry A, Shapiro H, Lindquist E, Kapitonov VV, Jurka J, Genikhovich G, Grigoriev IV, Lucas SM, Steele RE, Finnerty JR, Technau U, Martindale MQ, Rokhsar DS. (2007) <a href="http://dx.doi.org/10.1126/science.1139158"><em>Sea anemone genome reveals ancestral eumetazoan gener epertoire and genomic organization</em></a>. Science. 317, 86-94.
</p>

<p>
The Starlet sea anemone browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#starlet_sea anemone_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
