<!--Introduction---------------------------------------------->
<A NAME=begin></A>
      <TABLE WIDTH="100%" BGCOLOR="#888888" BORDER="0" CELLSPACING="0" CELLPADDING="1">
        <TR><TD>
        <TABLE BGCOLOR="#fffee8" WIDTH="100%"  BORDER="0" CELLSPACING="0" CELLPADDING="0">
          <TR><TD>
           <TABLE BGCOLOR="#D9E4F8" BACKGROUND="../../images/hr.gif" WIDTH="100%"><TR><TD>
             <A NAME="About"></A><FONT SIZE="4"><B>&nbsp; Introductory Session
  &nbsp;&nbsp;&nbsp;   &bull; &nbsp;&nbsp;&nbsp   W  10-21-2009 &nbsp; &bull; &nbsp;
  6 PM</B></FONT>
            </TD></TR></TABLE>

            <TABLE BGCOLOR="#fffee8" WIDTH="100%" CELLPADDING=3>
              <TR><TH HEIGHT=3></TH></TR>
              <TR><TD WIDTH=10></TD><TD>
              <P>

<P>
This session will focus on navigation and how to use the Genome Browser
to get information about genes.
</P>

<OL>
  <LI>Open a Genome Browser in a separate window by visiting
  <A HREF = "http://genome.ucsc.edu" TARGET = _BLANK>genome.ucsc.edu</A>.
  If you are using a windows-based computer, you can toggle back and forth between this
  page the browser window page using Alt-Tab.
  On a Mac: use Command (apple)-backtick (cmd-`).
  </LI>

  <LI>Click the <TT>Genome Browser</TT> link in either the top bar or the left bar.
  </LI>

  <LI>In case you have used the browser before, let's all start at the same place.
  <BR>Click:  <TT>Click here to reset the browser user interface settings to
  their defaults</TT>.
  </LI>

  <LI>Use the assembly pulldown to select the Mar 2006 human assembly (hg18).
  </LI>

  <LI>Hit the <TT>[Submit]</TT> button to go to the hg18 browser.
  </LI>

  <LI>Here is a <A HREF = "emoryData.html" TARGET = _BLANK>Clinical Report</A>
   from a microarray test on a patient, which we will use in this exercise.
   We will investigate this report to see what we can learn about the
   microdeletion region.
  </LI>

  <LI> Back at the Browser window, turn off all tracks using the
   <TT>hide all</TT> button.
  </LI>

  <LI> Beneath the now blank image, find the UCSC Genes track in the Gene and Gene
  Prediction track group.  Use the pull-down to set this track
  to <TT>pack</TT>. Hit any of the <TT>refresh</TT> buttons.
  </LI>

  <LI>Type or copy/paste <TT>22q11.21</TT> into the <TT>position/search</TT> box.
   Hit the <TT>jump</TT> button.<BR>
   The image will redraw at that chromosome
   band: 4.2 Mb. The Chromosome Band track will automatically open.
  </LI>

  <LI>Type or copy/paste <TT>chr22:17086001-19835417</TT> into the
   <TT>position/search</TT> box.  Hit the <TT>jump</TT> button.<BR>
   The image will open to that location (2.75 Mb).
  </LI>

  <LI>Type or copy/paste <TT>192430</TT> into the Position box. The Genome
  Browser returns a search results page with all items matching that
  number.  Choose the first item in the list: <TT>192430 at
  chr22:18124226-18134855</TT>.
   The OMIM Gene track is on, and OMIM Gene 192430 is highlighted.
  </LI>

  <LI>Press the <TT>zoom out 10x</TT> button. (106 Kb).
  </LI>

  <LI>Click the label that says <TT>192430</TT> or the blue box to the
   right &mdash; this box
   shows the location of the actual gene annotation.<BR>
   This will take you to the details page for that gene.
  </LI>

  <LI>Click this link: <TT>OMIM Database: 192430</TT><BR>
   This will take you to the OMIM website for that gene.<BR>
   Close the OMIM window.
  </LI>

  <LI>Click this link to go to the UCSC Genes details page:
    <BR><TT>UCSC Canonical Gene uc002zqa.1: Homo sapiens T-box 1 (TBX1),
    transcript variant C, mRNA</TT>.
  <BR>Close this window to return to the main browser window.
  </LI>

  <LI>Click the <TT>Genome Browser</TT> link in blue bar at top of window
   (better than the back button, and goes to the same place.)
  <BR><B>At this point, your browser should look like this:</B><BR>
<A HREF =
"http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&hgS_otherUserName=Example&hgS_otherUserSessionName=hg18_ashg2009_1"
TARGET = _BLANK><IMG SRC = "screen01.gif" WIDTH = 600 HEIGHT = 141 ></A>
<BR>
   <B>If it doesn't, you can click
  <A HREF = "http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&hgS_otherUserName=Example&hgS_otherUserSessionName=hg18_ashg2009_1" TARGET = _BLANK>here</A>
  to catch up.</B>
  </LI>

<HR>
<BR>&nbsp;

  <LI>To the far left of the data tracks (browser image with white background)
  are a series of vertical gray and blue boxes (mini-buttons).
  Click on the little blue mini-button to the left of the OMIM track in the
  Browser image.<BR>
  This will take you to the track configuration page.
  </LI>

  <LI>Click on the radio button: <TT>OMIM gene or syndrome</TT>.
  Press <TT>[Submit]</TT> button.<BR>
  The OMIM gene names will appear in the image, where available.
  </LI>

  <LI>Type or copy/paste <TT>DiGeorge</TT> into the <TT>position/search</TT> box.
  <BR> Click the third item in the search results list: <TT>DGCR8 (uc002zri.1) at
  chr22:18447834-18479400 - DiGeorge syndrome critical region gene 8</TT>.
  </LI>
  <LI><TT>Zoom out 10x</TT> (twice).
  </LI>

  <LI>Click the gray mini-button to the left of the UCSC Genes track.<BR>
  This will take you to the UCSC Genes track configuration page.

  <LI>Uncheck the <TT>splice variants</TT> checkbox. Press <TT>[Submit]</TT>.
  One variant per gene unclutters the screen.
  <BR>Note that you will sometimes lose highlighting if you have chosen a different
  variant.
  </LI>

  <LI>Scroll down the screen to the Track Controls below the image.
  <BR>Click the <B>+</B> sign in the blue bar, <TT>Phenotype and Disease
  Associations</TT>.
  </LI>

  <LI>Turn on the GAD View track (Genetic Association Studies of Complex
   Diseases and Disorders) to <TT>pack</TT> with the pull-down.  Hit
   <TT>refresh</TT>.
  <BR><B>At this point, your browser should look like this:</B><BR>
<A HREF = "http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&hgS_otherUserName=Example&hgS_otherUserSessionName=hg18_ashg2009_2"
  TARGET = _BLANK><IMG SRC = "screen02.gif" WIDTH = 600 HEIGHT = 318 ></A>
  <BR>
  <B>If it doesn't, you can click
  <A HREF = "http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&hgS_otherUserName=Example&hgS_otherUserSessionName=hg18_ashg2009_2"
  TARGET = _BLANK>here</A> to catch up.</B>
  </LI>

<HR>
<BR>&nbsp;

  <LI>Click the top-most mini-button on the left side of the image.  This will
   take you to the track controls for the Base Position track.
  </LI>

  <LI>Check the two boxes:  Display <TT>assembly</TT> and <TT>position</TT> 
   at the bottom of the configuration section.  Then press <TT>[Submit]</TT>.
  </LI>

  <LI>Use the drag-n-zoom feature to zoom into the region around the TBX1 gene
  in the GAD track:
  <BR>Position your mouse near the <TT>6</TT> of 2006 in the assembly name in the Base
  Position track at the very top of the browser image.  Click and hold, then
  drag to the right to the first <TT>2</TT> of chr22:
  in the position part at the top of the image.  Note that both the coordinates in
  the <TT>position/search</TT> box and the size of the region change as you drag.
  <BR>Release the mouse.  The image will resize to the coordinates you have just
  requested.
  </LI>

  <LI>Click on the red box in the GAD track labeled <TT>TBX1</TT>.  This will
   take you to the GAD View track details page.
  </LI>

  <LI>Click into any of the links at the top of the page for access to relevant
   biomedical literature:
   <BR>- Genetic Association Database: TBX1
   <BR>- CDC HuGE Published Literature: TBX1
   <BR>Close these extra windows.
  </LI>

  <LI>Click the <TT>Genome Browser</TT> link in blue bar at top of window to
   return to the main browser page.
  </LI>

  <LI>Users of the Genome Browser often find that their gene of interest
  maps to the "bottom" strand of the reference assembly.  An example of
  such a gene is the GNBL1 gene at the right side of screen.  This orientation
  is indicated by the left-pointing carets in the intron regions.
  <BR>Click the <TT>reverse</TT> button below the browser image to view the image
  in the opposite direction.
  <BR>Click it again to return to "normal" direction.
  </LI>

  <LI>Scroll down the page to the stack of blue bars in the Track Controls and
  open the <TT>Variation and Repeats</TT> section with the <B>+</B> sign.
  </LI>

  <LI>Turn on the SNP130 track (Simple Nucleotide Polymorphisms from dbSNP) to
  <TT>pack</TT> using the pull-down. Then click <TT>refresh</TT>.
  </LI>

  <LI>Locate the group of OMIM annotations in the middle of the browser image
  (three blue bars and one red, stacked together).
  <BR>Use drag-n-zoom to bracket these annotations.
  Note that the SNP track now shows individual SNPs.
  </LI>


  <BR><B>At this point, your browser should look like this:</B><BR>
<A HREF = "http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&hgS_otherUserName=Example&hgS_otherUserSessionName=hg18_ashg2009_3"
  TARGET = _BLANK><IMG SRC = "screen03.gif" WIDTH = 600 HEIGHT = 368 ></A>
  <BR>
   <B>If it doesn't, you can click
  <A HREF = "http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&hgS_otherUserName=Example&hgS_otherUserSessionName=hg18_ashg2009_3"
  TARGET = _BLANK>here</A> to catch up.</B>

<HR>
<BR>&nbsp;

  <LI>Notice the large exon near the center of the screen with two green
  SNPs in it:  (one is rs72646952).
  (Read about the color scheme in the usual place: Configuration page
  via the mini-button).
  Bracket that exon with the drag-n-zoom and fill the screen with it.
  <BR>Notice the small two-headed arrows on the right and left edges of the
  image (in the TBX1 gene introns).
  <BR>Hover your mouse over them to see which exon is next (off-screen).
  </LI>

  <LI>Click the double-arrow at the right side.  Your window size remains the
   same, but exon 4 is now in view.  Note the red SNP.  It is a non-synonomous
   (missense) SNP in exon 4.
  </LI>

  <LI>Click directly on the red SNP (<TT>rs28939675</TT>) and see the details
  about it, including:

   <PRE>
 Coding annotations by dbSNP:
 NM_005992: missense F (TTC) --> Y (TAC)

 UCSC's predicted function relative to selected gene tracks:
 UCSC Genes	TBX1 (uc002zqc.1)	missense F (TTC) --> Y (TAC) </PRE>
  </LI>
  
  <LI>Return using <TT>Gene Browser</TT> link.
  </LI>

  <LI>Fill the screen with just this exon using drag-n-zoom.  Note the
   amino acids appear at the top of the display when zoomed closely enough.
  </LI>

  <LI>In the track controls of the <TT>Genes and Gene Prediction Tracks</TT>
   group, turn on the <TT>Transmap</TT> track to <TT>show</TT>.
   Note that several organisms have mRNAs with amino acid variation
   in this exon (mouse, <EM>Xenopus tropicalis</EM>, zebrafish).
  </LI>


  <LI>Click into one of these, then click on the long link in the middle of
  the page that looks similar to this (works for the human mRNAs track, too):

  <PRE>
 SIZE IDENTITY CHROMOSOME  STRAND    START     END              QUERY      START  END  TOTAL
--------------------------------------------------------------------------------------------
 1317   72.0%         22     +  18128367  18134272       NM_183339.1-1.1   361  1693  2369 </PRE>
  </LI>

  <LI> Click the "<TT>together</TT>" link at the bottom of the list in the
  frame on the left to see something similar to:

  <PRE>
 Side by Side Alignment

 00000361 atgatttcagcaatatcaagcccgtggctgacgcagctgtcccatttttg 00000410
 >>>>>>>> ||||| || ||  | || ||||||||||| |||||||| || ||||| || >>>>>>>>
 18128367 atgatctccgccgtgtccagcccgtggctcacgcagctctcgcatttctg 18128416 </PRE>
  </LI>

  <LI>Click your browser's <TT>back</TT> button until you are back at the
  browser image (probably three times).
  </LI>

  <LI>Click the <TT>mini-button</TT> to the left of the lower of the three
  TransMap subtracks.
  </LI>

  <LI>Unclick the <TT>accession</TT> checkbox (leaving only the <TT>common
   name</TT> box checked.  <TT>[Submit]</TT>.
  <BR>Now it is easier to read the species names of the orthologs.

  <BR><B>At this point, your browser should look like this:</B><BR>
<A HREF = "http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&hgS_otherUserName=Example&hgS_otherUserSessionName=hg18_ashg2009_4"
  TARGET = _BLANK><IMG SRC = "screen04.gif" WIDTH = 600 HEIGHT = 291 ></A>
  <BR>
  <B>If it doesn't, you can click
  <A HREF = "http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&hgS_otherUserName=Example&hgS_otherUserSessionName=hg18_ashg2009_4"
  TARGET = _BLANK>here</A> to see where we ended up.</B>
  </LI>

  <LI>Just for fun, click on the double-headed right-arrow to see the next exon.
  Amino acid diffrences that strongly affect the polarity are indicated with a
  stronger color.
  </LI>
</OL>


		    </P>
                    </TD><TD WIDTH=15>
                </TD></TR></TABLE>
            <BR></TD></TR></TABLE>
        </TD></TR></TABLE>
