<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=315>
  <TR><TD ALIGN=RIGHT><A HREF="http://genome.wustl.edu/genomes/detail/pan-troglodytes/" TARGET=_blank>
      <IMG SRC="../images/Pan_troglodytes.jpg" WIDTH=300 HEIGHT=219 ALT="Chimp"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1>&quot;Clint&quot; - <em>Pan troglodytes</em><BR>
    </FONT>
    <FONT SIZE=-2> 
    Photo: <A HREF="http://www.yerkes.emory.edu/"
      TARGET=_blank>Yerkes National Primate Research Center, Emory University</A>
    </FONT>
  </TD></TR>
</TABLE>

<P>
The October 2010 chimpanzee (<em>Pan troglodytes</em>) browser displays
data from the 6X whole genome shotgun draft assembly
(CGSC 2.1.3, GCA_000001515.3) produced by the 
<A HREF="http://genome.wustl.edu/genomes/detail/pan-troglodytes/"
TARGET=_BLANK>Chimpanzee Sequencing and Analysis Consortium</A>.
This version was assembled and organized by the 
<A HREF="http://genome.wustl.edu/genomes/detail/pan-troglodytes/"
TARGET=_blank>Genome Center at Washington University</A> from data 
generated at the Washington University St. Louis School of Medicine and the 
Broad Institute. For more information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/202458/" TARGET="_blank">
Pan_troglodytes-2.1.3</A> in the NCBI Assembly database.</P>

<P>
The v2.* assemblies use a chromosomal numbering scheme that reflects
orthology between the human and chimpanzee chromosomes. For details, see 
the Assembly details section below and the Genome Browser 
<A HREF="../FAQ/FAQdownloads#download25">FAQ</A>. 
</P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the HUGO Gene Nomenclature Committee gene 
name of a human RefSeq, the accession number of an EST or mRNA,
a chromosomal coordinate range, or keywords from 
the GenBank description of an mRNA. The following list shows 
examples of valid position queries for the chimpanzee genome.
See the
<A HREF="../goldenPath/help/hgTracksHelp.html" TARGET=_blank>
User's Guide</A> for more information.
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TD VALIGN=Top NOWRAP><B>Request:</B><BR></TD>
       	<TD VALIGN=Top COLSPAN=2><B>&nbsp;&nbsp; Genome Browser Response:</B><br></TD>
    </TR>
    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>chr22</TD>
        <TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays all of chromosome 22</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr2a:11,250,001-12,250,000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a million bases of chromosome 2a, beginning at 
base 11,250,001. Note that <BR>
	chromosome 2 in this assembly has been split into two parts: 2a
and 2b.</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr2a:11,250,001+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 2a that spans 2000 bases, starting at position 11,250,001</TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>BRCA1</TD>
       	<TD WIDTH=14></TD>
       	<TD>Displays a list of genomic regions where human RefSeq gene BRCA1 (or features associated with BRCA1) aligns</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>AF115459</TD>
       	<TD WIDTH=14></TD>
   	<TD VALIGN=Top>Displays region of genome with mRNA with GenBank accession number AF115459</TD>
    </TR>
    <TR>
	<TD VALIGN=Top NOWRAP>348</TD>
	<TD><BR></TD>
	<TD VALIGN=Top NOWRAP>Displays the region of genome with Entrez Gene identifier 348</TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>pseudogene mRNA</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>sialic acid</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists mRNAs and RefSeqs with GenBank keywords sialic acid </TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>huntington</TD>
       	<TD WIDTH=14></TD>
 	<TD VALIGN=Top>Lists mRNAs associated with Huntington's disease</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Paabo,S.</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author S. Paabo</TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although GenBank 
			requires the search format <em>Paabo S</em>, internally it uses 
			the format <em>Paabo,S.</em>.</TD>
    </TR>
</TABLE>
<P>
<HR>
<H3>Assembly details</H3>
<P>
This assembly covers about 97 percent of the genome and is based on 6X sequence 
coverage. It comprises 192,898 contigs with an N50 length of 44 kb and 33,990 
supercontigs with an N50 length of 8.4 Mb. Improvements introduced with the 
2.1.3 assembly include the addition of over 300,000 finishing reads and the 
merging in of 640 finished BACS. Approximately 49,000 additional merges were 
made in this assembly as compared with the 2.1 assembly.
<P>
The whole genome shotgun data were derived primarily from the donor Clint, a 
captive-born male chimpanzee from the Yerkes Primate Research Center in Atlanta,
GA, USA. The sequence data were assembled and organized by the Genome Center at 
Washington University from underlying whole genome shotgun data generated at 
the Washington University School of Medicine and the Broad Institute. For 
detailed information about the assembly process, see the sequencing and analysis
discussion in the 
<A HREF="https://www.ncbi.nlm.nih.gov/nuccore/AACZ00000000" 
TARGET=_blank>GenBank accession record</A>.  To read more about the
chimpanzee genome sequencing program, see the Washington University 
<A HREF="http://genome.wustl.edu/genomes/detail/pan-troglodytes/"
TARGET=_blank><em>Pan troglodytes</em></A> web page and the National Institutes 
of Health NIH News <A HREF="http://www.genome.gov/15515096" 
TARGET=_blank>summary of the chimpanzee analysis paper</A>.</P>
<P>
The chromosomal numbering scheme reflects a standard that preserves 
orthology with human chromosomes.  Proposed by E.H. McConkey in 2004, 
the numbering convention was subsequently endorsed by the 
International Chimpanzee Sequencing and Analysis Consortium.
This standard assigns the identifiers &quot;2a&quot; and &quot;2b&quot; to the 
two chimp chromosomes that fused in the human genome to form chromosome 2. Note 
that
the genome assembly shown in the Nov. 2003 panTro1 Genome Browser retains the
older numbering scheme in which these chromosomes are numbered 12 and 13. To 
view a table showing the correspondence between human and chimp chromosomes, 
see the <A HREF="../FAQ/FAQdownloads#download25">FAQ</A>.</P> 
<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/panTro3/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#chimp">Downloads</A> page. 
The Chimp browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#chimp_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to this release. 
</P>
<P>
<HR>
<H3>References</H3>
<P>
The Chimpanzee Sequencing and Analysis Consortium. 
<A HREF="http://www.nature.com/nature/journal/v437/n7055/full/nature04072.html" TARGET=_blank>
Initial sequence of the chimpanzee genome and comparison with the human genome</a>.
<em>Nature</em>. 2005 Sep 1;437(7055):69-87.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16136131" target="_blank">16136131</a>
</p>

<p>
Huang X, Yang SP, Chinwalla AT, Hillier LW, Minx P, Mardis ER, Wilson RK.
<a href="http://nar.oxfordjournals.org/content/34/1/201.abstract" target="_blank">
Application of a superword array in genome assembly</a>.
<em>Nucleic Acids Res</em>. 2006;34(1):201-5.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16397298" target="_blank">16397298</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1325203/" target="_blank">PMC1325203</a>
</p>

<p>
Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D.
<a href="http://www.pnas.org/content/100/20/11484.abstract" target="_blank">
Evolution's cauldron: duplication, deletion, and rearrangement in the mouse and human genomes</a>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Sep 30;100(20):11484-9.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14500911" target="_blank">14500911</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC208784/" target="_blank">PMC208784</a>
</p>

<p>
McConkey EH.
<a href="http://www.karger.com/Article/Abstract/78022" target="_blank">
Orthologous numbering of great ape and human chromosomes is essential for comparative genomics</a>.
<em>Cytogenet Genome Res</em>. 2004;105(1):157-8.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15218271" target="_blank">15218271</a>
</p>

<p>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC, Haussler D, Miller W.
<a href="http://genome.cshlp.org/content/13/1/103.abstract" target="_blank">
Human-mouse alignments with BLASTZ</a>.
<em>Genome Res</em>. 2003 Jan;13(1):103-7.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12529312" target="_blank">12529312</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC430961/" target="_blank">PMC430961</a>
</p>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
