<H2>Description</H2>
<P>
This track depicts the gaps in the assembly
(Oct. 2010, CGSC 2.1.3 (GCA_000001515.3)) of the $organism genome
provided by the Chimpanzee Genome Sequencing Consortium.<BR>
These gaps -- with the exception of intractable heterochromatic
gaps -- will be closed during the finishing process. 
</P><P>
Gaps are represented as black boxes in this track.
If the relative order and orientation of the contigs on either side
of the gap is known, it is a <em>bridged</em> gap and a white line is drawn 
through the black box representing the gap. 
</P>
<P>This assembly contains the following principal types of gaps:
<UL>
<LI><B>Fragment</B> - gaps between the contigs of a draft clone.
(In this context, a contig is a set of overlapping sequence reads.)
These may or may not be bridged.
<LI><B>Contig</B> - non-bridged gaps between map contigs.
<LI><B>Centromere</B> - non-bridged gaps from centromeres.
<LI><B>Telomere</B> - non-bridged gaps from telomeres.
<LI><B>heterochromatin</B> - non-bridged gaps from large blocks of
	heterochromatin.
<LI><B>Short Arm</B> - non-bridged long gaps on the short arm of the chromosome. 
<LI><B>Clone</B> - gaps between WGS contig sequences, all indicated as
	bridged gaps, and found only on chromosomes chr7, chr21 and chrY
<LI><B>Other</B> - unknown sequence marked as 'N' nucleotides, not mentioned
	as a gap in the AGP files, all indicated as bridged gaps
</UL></P>
