<H2>Description</H2>
<P>
This track shows genes predicted by Twinscan downloaded from 
<A HREF="http://flybase.org/" TARGET=_BLANK>FlyBase</A>
(<A HREF="ftp://ftp.flybase.net/genomes/Drosophila_pseudoobscura/dpse_r10_20041216/"
TARGET=_BLANK><EM>D. pseudoobscura</EM> version 1.0</A>).  
The Twinscan program predicts genes in a manner similar to Genscan, except 
that Twinscan takes advantage of genome comparisons to improve gene prediction
accuracy. More information and a web server can be found at
<A HREF="https://mblab.wustl.edu/" TARGET=_blank>
https://mblab.wustl.edu/</A>. </P>

<H2>Display Conventions and Configuration</H2>
<P>
This track follows the display conventions for
<A HREF="../goldenPath/help/hgTracksHelp.html#GeneDisplay">gene prediction
tracks</A>. </P>
<P>
The track description page offers the following filter and configuration
options:
<UL>
<LI><B>Color track by codons:</B> Select the <em>genomic codons</em> option
to color and label each codon in a zoomed-in display to facilitate validation
and comparison of gene predictions. Click the 
<A HREF="../goldenPath/help/hgCodonColoring.html">Help on codon coloring</A>
link on the track description page for more information about this feature.
</UL></P>

<H2>Methods</H2>
<P>
The Twinscan algorithm is described in Korf, <em>et al.</em>, 2001, in the
References section below.</P>

<H2>Credits</H2>
<P>
Thanks to <A HREF="https://mblab.wustl.edu/"
TARGET=_blank>Michael Brent's Computational Genomics Group</A> at Washington
University St. Louis for providing these data.</P>

<H2>References</H2>
<p>
Korf I, Flicek P, Duan D, Brent MR.
<a href="https://academic.oup.com/bioinformatics/article/17/suppl_1/S140/261903" target="_blank">
Integrating genomic homology into gene structure prediction</a>.
<em>Bioinformatics</em>. 2001;17 Suppl 1:S140-8.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11473003" target="_blank">11473003</a>
</p>
