<h2>Description</h2>

This track depicts gaps in the draft assembly (Oct. 2011, Btau_4.6.1
(NCBI project 12555, accession GCA_000003205.4)) of the $organism genome.

<p>
Gaps are represented as black boxes in this track. If the relative order and
orientation of the contigs on either side of the gap are supported by read pair data,
it is a <em>bridged</em> gap and a white line is drawn through the black box
representing the gap.
</p>

<p>
This assembly contains the following principal types of gaps:
<ul>

<li><b>Fragment</b> - Gaps between the Whole Genome Shotgun contigs of a
supercontig.  (In this context, a contig is a set of overlapping sequence reads.
A supercontig is a set of contigs ordered and oriented during the
Whole Genome Shotgun process using paired-end reads.)
These are represented by varying numbers of <em>N</em>s in the assembly.
Fragment gap sizes are usually taken from read pair data.</li>
<li><b>Clone</b> - Gaps between supercontigs linked by the fingerprint map.
In general, these are represented by 10,000 <em>N</em>s  in the assembly.</li>
<li><b>Contig</b> - Gaps between supercontigs not linked by the fingerprint
map, but instead by marker data.  (In this context, the &quot;Contig&quot; gap
type refers to a map contig, not a sequence contig.)
In general, these are represented by 10,000 <em>N</em>s  in the assembly for all
chromosomes except chrUn (concatenation of unplaced supercontigs), where
gaps of 1,000 <em>N</em>s  are used.  Gaps of other sizes were used when mRNA or
other data suggested possible but not confirmed links between supercontigs.</li>
<li><b>Other</b> - Gaps of varying size marked with Ns in the sequence that were
not annotated in the AGP assembly definition file.</li>
<li><b>Telomere</b> - Gaps for telomeres were included when they could be
reasonably localized.  These are represented by 10,000 <em>N</em>s in
the assembly.</li>
</ul>
</p>

<p>
See also: <a href="https://www.ncbi.nlm.nih.gov/assembly/agp/AGP_Specification/"
target="_blank">AGP specification</a>
</p>
