<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=185>
  <TR><TD ALIGN=RIGHT><A HREF="http://genome.wustl.edu/genomes/detail/gallus-gallus/"
    TARGET=_blank>
    <IMG SRC="../images/Gallus_gallus.jpg" WIDTH=170 HEIGHT=136 ALT="Chicken"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Gallus gallus</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of Bill Payne,<BR>Michigan State University<BR>
      (NHGRI <A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Red%20Jungle%20Fowl&id=2120" 
      TARGET=_blank>Press Photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The February 2004 chicken (<em>Gallus gallus</em>) v1.0 draft
assembly was produced by the Genome Sequencing Center at the
Washington University School of Medicine in St. Louis.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic supercontig, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
chicken genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>chrZ</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of chromosome Z</TD>
    </TR>
    <TR><TD VALIGN=Top>chr3:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of chr 3</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr3:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 3 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top>Contig44</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of assembly for the supercontig
		with ID contig44</TD>
    </TR>
    <TR><TD VALIGN=Top>Contig44.48</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of assembly for the contig with
		ID contig44.48</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>BX950328</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number BX950328</TD>
    </TR>
    <TR><TD VALIGN=Top>BU335953</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		BU335953 on chr Z</TD>
    </TR>
    <TR><TD VALIGN=Top>chr13_3.145</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of genome containing Genscan
		gene prediction chr13_3.145</TD>
    </TR> 
    <TR><TD VALIGN=Top>CWNT-8C</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of genome with gene symbol CWNT-8C</TD>
    </TR>
    <TR>
	<TD VALIGN=Top>NM_204735</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays the region of genome with RefSeq identifier
		NM_204735</TD>
    </TR>
    <TR><TD VALIGN=Top>374180</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays the region of genome with Entrez Gene
		identifier 374180</TD>
    </TR>
    <TR><TD VALIGN=Top>ENSGALT00000023622</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays the region of genome with Ensembl identifier 
	ENSGALT00000023622. NOTE: Due to space constraints, the Genome Browser 
	displays abridged Ensembl identifiers, but the full ID name must be
	used in searches. For example, the ENSGALT00000023622 identifier
	displays as ENSGAL23622 in the Genome Browser.</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene mRNA</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zhang</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zhang</TD>
    </TR>
    <TR><TD VALIGN=Top>Taguchi,T.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author T. Taguchi</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Taguchi T</em>, internally it
	uses the format <em>Taguchi,T.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
The genome has been sequenced to 6.63X coverage.
Approximately 88% of the sequence has been anchored to chromosomes, 
which include autosomes 1-24, 26-28, and 32, and sex chromosomes W and Z. 
(In contrast to mammals, the female chicken is heterogametic (ZW) and the 
male is homogametic (ZZ).) 
The remaining unanchored contigs that could be localized to a chromosome 
have been concatenated into the virtual 
chromosomes &quot;chr*_random&quot;, separated by gaps of 10,000 bp. 
Unanchored contigs that could not be localized to a chromosome 
have been concatenated into the virtual 
chromosome &quot;chrUn&quot;, separated by gaps of 1,000 bp in order 
to reduce the total size of chrUn. 
The chicken 
mitochondrial sequence is also available as the virtual chromosome 
&quot;chrM&quot;. 
</P>

<P>
WARNING: This release of the G. gallus chrW contains some sequence
that is not specific to chromosome W.  A large portion of the
chromosome was assigned to W based on the presence of W-specific
repeats which, as it has now been shown, are not W-specific.  Thus, the
only portions of chrW which should currently be considered specific to
W are:
</P>
<UL>
               <LI>chrW:1-195,831</LI>
               <LI>chrW:4,895,452-4,916,845</LI>
               <LI>chrW_random (all)</LI>
</UL>

<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/galGal2/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#chicken">Downloads</A> page. These data
have <A HREF="../goldenPath/credits.html#chicken_use">specific conditions for use</A>. 
The chicken browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#chicken_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
