<!-- Display image at top right of page -->
<table border="0" align="right" width="167">
  <tr><td align="right"><a href="http://www.genome.gov/17515860" target="_blank">
    <img src="../images/Canis_familiaris.jpg" width=152 height=200 alt="Dog"></a>
  </td></tr>
  <tr><td align="right"> 
    <font size="-1"> "Tasha" -  <em>Canis lupus familiaris</em><br>
    </font>
    <font size="-2"> (NHGRI <a href="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Dog&id=79106" 
      target="_blank">Press Photos</a>) 
    </font>
  </td></tr>
</table>

<p>
The May 2005 dog (<em>Canis familiaris</em>) whole genome shotgun
(WGS) assembly v2.0 was sequenced and assembled by the
<a href="http://www.broadinstitute.org/mammals/dog" target="_blank">
Broad Institute</a> of MIT/Harvard and Agencourt Bioscience (now part of
<a href="http://www.beckmangenomics.com">Beckman Coulter Genomics</a>).
For more information about this assembly, see
<a href="https://www.ncbi.nlm.nih.gov/assembly/237548/" target="_blank">
CanFam2.0</a> in the NCBI Assembly database.</p>

<h3>Sample Position Queries</h3>

<p>
A genome position can be specified by the accession number of a
sequenced genomic clone, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA.
The following list provides examples 
of various types of position queries for the dog genome. See the 
<a href="../goldenPath/help/hgTracksHelp.html" 
target="_blank">User's Guide</a> for more information. 
<p>

<table border="0" cellpadding="0" cellspacing="0">
    <tr><td valign="top" nowrap><b>Request:</b><br></td>
	<td width="14">&nbsp;</td>
	<td valign="top"><b>Genome Browser Response:</b><br></td>
    </tr>
	
    <tr><td valign="top"><br></td></tr>
    <tr><td valign="top" nowrap>chr16</td>
	<td width="14">&nbsp;</td>
	<td valign="top">Displays all of chromosome 16</td>
    </tr>

    <tr><td valign="top" nowrap>chr16:1-5000000</td>
	<td width="14">&nbsp;</td>
	<td valign="top">Displays first 5 million bases of chr 16</td>
    </tr>

    <tr><td valign="top" nowrap>chr16:1000000+2000</td>
       	<td width="14"></td>
       	<td valign="top">Displays a region of chr 16 that spans 2000 bases, starting with position 1000000 </td>
    </tr>

    <tr><td valign="top" nowrap>AY572227</td>
	<td width="14">&nbsp;</td>
	<td valign="top">Displays location of mRNA with GenBank accession AY572227</td>
    </tr>
    <tr><td valign="top" nowrap>BM538765</td>
	<td width="14">&nbsp;</td>
	<td valign="top">Displays location of EST with GenBank accession BM538765</td>
    </tr>
    <tr><td valign="top" nowrap>chr6_8.29</td>
	<td width="14">&nbsp;</td>
	<td valign="top">Displays location of Genscan gene prediction with identifier chr6_8.29</td>
    </tr>

    <tr><td valign="top"><br></td></tr>
    <tr><td valign="top" nowrap>pseudogene mRNA</td>
	<td width="14">&nbsp;</td>
	<td valign="top">Lists transcribed pseudogenes but not cDNAs</td>
    </tr>

    <tr><td valign="top" nowrap>zinc finger</td>
	<td width="14">&nbsp;</td>
	<td valign="top">Lists zinc finger mRNAs</td>
    </tr>

    <tr><td valign="top" nowrap>breast cancer</td>
	<td width="14">&nbsp;</td>
	<td valign="top">Lists mRNAs associated with breast cancer susceptibility proteins</td>
    </tr>

    <tr><td valign="top" nowrap>evans</td>
	<td width="14">&nbsp;</td>
	<td valign="top">Lists mRNAs deposited by scientist named Evans</td>
    </tr>
    <tr><td valign="top" nowrap>Murphy,K.E.</td>
	<td width="14">&nbsp;</td>
	<td valign="top">Lists mRNAs deposited by co-author K.E. Murphy</td>
    </tr>

    <tr><td valign="top"><br></td></tr>
    <tr><td colspan="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Murphy KE</em>, internally
	it uses the format <em>Murphy,K.E.</em>.</td>
    </tr>
</table>
</p>

<h3>Assembly Details</h3>

<p>
The dog genome contains approximately 2.5 billion base pairs. 
This sequence is based on 7.6X coverage of the dog genome, assuming a WGS
assembly size of 2.4 Gb. 
The boxer breed was selected for the initial sequencing effort based on the
lower variation rate in its genome relative to other breeds. For more
information about this assembly, see the Broad Institute
<a href="http://www.broadinstitute.org/mammals/dog" target="_blank">Dog Genome
Sequencing Project</a> web page.
</p>

<p>
The canFam2 sequence and annotation data can be downloaded from the 
UCSC Genome Browser 
<a href="ftp://hgdownload.soe.ucsc.edu/goldenPath/canFam2/">FTP server</a>
or 
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#dog">downloads</a> 
page.
</p>

<p>
Many thanks to the Broad Institute of MIT/Harvard, NHGRI, and the many
institutions who contributed to the sequencing and mapping effort for this
release. The canFam2 annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<a href="/goldenPath/credits.html#dog_credits">credits</a> page
for a detailed list of the organizations and individuals who contributed 
to this release. 
</p>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
