<H3>Description</H3>

<P>This column shows the expression of genes during 
the cell cycle of the budding yeast <em>S. cerevisiae</em>.  
<P>
Cells from a synchronous yeast culture were collected at 17 time points taken
at 10 minute intervals, covering nearly 2 full cell cycles.
By default, the column shows the ratio of gene expression at a specific
cell cycle phase to expression of the gene overall. A gene that is more 
highly expressed in a cell cycle phase is 
colored red, and a less expressed gene is shown in green. 
The values are colored on a logarithmic scale. This coloring is standard, 
but is the opposite of what an inexperienced user 
might expect: in this case, red means go and green means stop!  
Black indicates that a gene is neither over- nor under-expressed in the cell
cycle phase. Uncolored boxes (white on most browsers) represent missing data.  

<H3>Column Configuration</H3>
Several column configuration options are available on the Configuration page, 
which can be accessed via the <em>configure</em> button in the top menu.
<UL>
<LI>
<B>Absolute expression values:</B> The display can be configured to show 
absolute expression values rather than ratio values. To toggle the display, 
select the <em>absolute</em> option from the <em>values</em> pulldown menu. 
Absolute expression values
are shown in shades of gray, with higher expression levels shown in lighter 
shades. As with ratio values, absolute values are colored on a logarithmic 
scale.</LI>
<LI>
<B>Color scheme:</B> Color-blind users can switch the coloring scheme from red/green to 
yellow/blue via the <em>Expression ratio colors</em> pulldown menu at the top
of the Configuration page.</LI>
<LI>
<B>Level of cell cycle detail:</B> The <em>tissues</em> pulldown menu
controls the amount of cell cycle data displayed. By default, only the 
median values for selected cell cycle phases are shown. To display the value of 
all individual experimental replicates of all phases, 
select the <em>all replicates</em> option. </LI>
<LI>
<B>Brightness:</B> To increase or decrease the brightness of the expression colors,
edit the number in the column's <em>brightness</em> text box. Values greater 
than 1.0 increase the brightness, while those less than 1.0 dim the color. </LI>
</UL>

<H3>Methods</H3>
The data in this column were collected as described in the paper
Cho RJ <em>et al</em>. (1998), 
"<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=9702192&dopt=Abstract" TARGET=_blank>A genome-wide 
transcriptional analysis of the mitotic cell cycle</A>", 
<em>Mol Cell.</em> 1998 Jul;2(1):65-73.

<P>To associate particular genes with expression data we use a fairly simple-minded
scheme.  First we align the cDNA sequences around which the probes were designed to
the genome.  Then for each gene we pick the cDNA sequence with the alignment that most
overlaps the gene as the source for expression data for that gene.  Since many of the cDNA 
sequences and probes designed for the cDNA sequences just cover part of the gene, in most
cases different splice varients of a gene will end up mapped to the same cDNA sequence. 
</P>


<H3>Credits</H3>
Many thanks to 
Cho <em>et al</em>. for making the data available.
