<h2>Description</h2>

<p> This track shows the primers for the
<a href="https://artic.network/" target="_blank">ARTIC network</a> SARS-CoV-2 sequencing protocol,
Version 4.1 (January 7, 2022), which restores the functionality of some primers against
the Omicron variant.
</p>

<h2>Display Conventions and Configuration</h2>

<p>
Genomic locations of primers are highlighted. A click on them shows the primer pool.</p>

<h2>Methods</h2>
<p>
Artic Network primer sequences were downloaded from
<a href="https://github.com/artic-network/artic-ncov2019/tree/master/primer_schemes/nCoV-2019/V4.1/"
 target=_blank>Github</a> (file
<a href="https://github.com/artic-network/artic-ncov2019/tree/master/primer_schemes/nCoV-2019/V4.1/SARS-CoV-2.primer.bed"
target="_blank">SARS-CoV-2.primer.bed</a>) and converted to bigBed.
</p>

<h2>Data Access</h2>
<p>
The raw data can be explored interactively with the
<a href="../goldenPath/help/api.html">REST API</a>,
<a href="hgTables">Table Browser</a>, or combined with other datasets in the
<a href="hgIntegrator">Data Integrator</a> tool.
For automated analysis, the genome annotation is stored in
a bigBed file that can be downloaded from
<a href="http://hgdownload.soe.ucsc.edu/gbdb/wuhCor1/bbi/articV4.1.bb">the download server</a>.</p>
<p>
Annotations can
be converted from binary to ASCII text by our command-line tool bigBedToBed.
Instructions for downloading this command can be found on our
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">utilities page</a>.
The tool can also be used to obtain features within a given range without downloading the file,
for example:</p>
<code>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/wuhCor1/bbi/articV4.1.bb -chrom=NC_045512v2 -start=0 -end=29902 stdout</code>
<p>
Please refer to our
<a href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing list archives</a>
for questions, or our
<a href="../../FAQ/FAQdownloads.html#download36">Data Access FAQ</a>
for more information.</p>
