<h2>Description</h2>

<p> This track shows potential contact residues for ligands, inferred from
structures in the PDB database by Alastair Fyfe, UCSC.
</p>

<h2>Display Conventions and Configuration</h2>

<p>
Genomic locations of contact residues are highlighted with thick blocks, 
that look identical to exons. Contact residues of the same PDB structure
are connected by thin intron lines.
</p>
<p>
To display the 3D structure viewer with these contact residues highlighted,
follow the outlink at the top of the details page of any feature.
</p>

<h2>Methods</h2>
<p>
PDB SEQRES protein sequences were aligned to the genome with tblastn.
Ligands were determined manually, all amino
acids closer than 3.5 Angstroms were obtained as described below.
The positions of these close amino acids on the genome were determined 
using the tblastn alignment and are highlighted on the genome with exon blocks. 
</p>
<p>
The find nearby amino acids, inter-atom distances were calculated with the libraries
<a target=_blank href="https://gemmi.readthedocs.io/en/latest/">GEMMI</a> and 
<a target=_blank href="http://www.ysbl.york.ac.uk/~cowtan/clipper/doc/">Clipper</a> 
at a threshold of 3.5 <span>&#8491;</span>. Distances between atoms 
whose proximity is due to crystal packing or symmetry are listed but cannot be
inspected in the viewer. These contacts are flagged by a check in the "Symm?"
column. They include atoms in adjacent unit cells and atoms brought into
proximity by application of a symmetry transformation to the asymmetric unit
(ASU), but not atoms related by non-crystallographic symmetry (NCS).<br><br>For
each amino acid in the contact table the "Codon" column gives the codon's
nucleotide values and position in the 
<a target=_blank href="https://www.ncbi.nlm.nih.gov/nuccore/NC_045512">NC_045512.2</a> reference
sequence. Amino acids that could not be aligned to the reference, for example
insertions or deletions engineered to facilitate protein expression, are
flagged  by "-".</p>

<h2>Data Access</h2>
<p>
The raw data can be explored interactively with the
<a href="hgTables">Table Browser</a> or combined with other datasets in the
<a href="hgIntegrator">Data Integrator</a> tool.
For automated analysis, the genome annotation is stored in
a bigBed file that can be downloaded from
<a href="http://hgdownload.soe.ucsc.edu/gbdb/wuhCor1/bbi/contacts.bb">the download server</a>.</p>
<p>
Annotations can
be converted from binary to ASCII text by our command-line tool bigBedToBed.
Instructions for downloading this command can be found on our
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">utilities page</a>.
The tool can also be used to obtain features within a given range without downloading the file,
for example:</p>
<code>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/wuhCor1/bbi/contacts.bb -chrom=NC_045512v2 -start=0 -end=29902 stdout</code>
<p>
Please refer to our
<a href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing list archives</a>
for questions, or our
<a href="../../FAQ/FAQdownloads.html#download36">Data Access FAQ</a>
for more information.</p>
<h2>Credits</h2>
Track created by Alastair Fyfe, UCSC, mentored by David Haussler.
