<H2>Description</H2>
<p>This track shows deletions that have been found in the sequences uploaded to the GISAID database as of June 6, 2020.
Three confidence levels of deletion calls are shown: </p>
<ul>
<li><span style="color: rgb(170,170,170)"><b>deletions found in at least 1 
GISAID sequence</b></span></li>
<li> <span style="color: rgb(85,85,85)"><b>deletions found in at least 2 GISAID
sequences</b></span></li>
<li><span style="color: rgb(0,0,0)"><b>deletions found in at least 2 GISAID sequences that
were able to be validated with raw reads.</b></span></li></ul>

<H2>Methods</H2>
<p>
We accessed all GISAID SARS-CoV-2 sequences on June 6, 2020. We filtered to
high coverage reads encompassing the entire SARS-CoV-2 genome (&gt;=29000 bps),
leaving 12,403 sequences.
We aligned the reads using MAFFT.</p>

<H2>Verification</H2>
<p>We validated several deletions with the raw reads from NCBI's SRA Run browser. 
Additionally, NYU Langone Health provided us with the aligned reads for many of 
their sequences.</p>

<h2>Data Access</h2>
<p>
The raw data can be explored interactively with the
<a href="hgTables">Table Browser</a>, combined with other datasets in the
<a href="hgIntegrator">Data Integrator</a> tool, 
or downloaded directly as &quot;microdel.txt.gz&quot; from
<a href="http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/database">the download server</a>.
Please refer to our
<a href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing list archives</a>
for questions, or our
<a href="../../FAQ/FAQdownloads.html#download36">Data Access FAQ</a>
for more information.</p>

<H2>Credits</H2>
<p>
We thank all of the labs that submitted their sequences to the GISAID database. 
The full acknowledgement table can be found at <A 
HREF="https://github.com/briannachrisman/SARS-CoV-2_Microdeletions/blob/master/acknowledgments.pdf" 
TARGET=ucscSchema title='Open table schema in new window'>
https://github.com/briannachrisman/SARS-CoV-2_Microdeletions/blob/master/acknowledgments.pdf</A>.
We thank the public health laboratories VIDRL and MDU-PHL at The Peter Doherty Institute for 
Infection and Immunity for providing over 1000 high quality raw reads to NCBI. 
Thank you NYU Langone SARS-CoV2 Sequencing Team's Matthew T Maurano, Matija Snuderl, and 
Adriana Heguy for providing many of their raw reads.
</p>


<H2>References</H2>
<p>Chrisman, Brianna Sierra, Kelley Paskov, Nate Stockham, Kevin Tabatabaei, Jae-Yoon Jung, Peter Washington, Maya Varma, Min Woo Sun, Sepideh Maleki, and Dennis P. Wall. "Indels in SARS-CoV-2 occur at template-switching hotspots." BioData Mining 14, no. 1 (2021): 1-16. <A HREF="https://doi.org/10.1186/s13040-021-00251-0" TARGET=_blank title='Open table schema in new window'>https://doi.org/10.1186/s13040-021-00251-0</A>
<P>
