<h2>Description</h2>
<p>
<a href="https://nextstrain.org/ncov" target="_blank">Nextstrain.org</a> displays
data about mutations that have occurred in the current 2019/2020 outbreak of SARS-CoV-2.
Nextstrain has a powerful user interface for viewing the time stamped phylogenetic tree
that it infers from the patterns of mutations in sequences worldwide.
Nextstrain maintains an ongoing pipeline that continuously obtains SARS-CoV-2 genome sequences
and metadata from <a href="https://gisaid.org" target="_blank">GISAID</a>,
aligns them against the reference genome
(<a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_045512.2" target="_blank">NC_045512.2</a>),
and infers a phylogenetic tree.
</p>
<p>
This track shows the alternate allele frequency of each mutation reported by Nextstrain as a
bar graph with the height indicating the frequency.
(The <em>Nextstrain Mutations</em> track offers a more detailed display of the mutations,
breaking up the vertical bars according to the order of virus genome samples in the
phylogenetic tree.)
</p>

<h2>Methods</h2>
<p>Nextstrain downloads SARS-CoV-2 genomes from
  <a href="https://gisaid.org/" target="_blank">GISAID</a>
  as they are submitted by labs worldwide.
  The sequences are processed by an
  <a href="https://github.com/nextstrain/ncov" target="_blank">automated pipeline</a>
  and annotations are written to a data file
  that UCSC downloads and extracts annotations for display.</p>

<h2>Data Access</h2>
<p>
You can download the bigBed file underlying this track (nextstrainSamples*.bigWig) from our
<a href="https://hgdownload.soe.ucsc.edu/gbdb/wuhCor1/nextstrain/"
target="_blank">Download Server</a>. The data can be explored interactively with the
<a href="../cgi-bin/hgTables">Table Browser</a>
or the <a href="../cgi-bin/hgIntegrator">Data Integrator</a>. The data can be
accessed from scripts through our <a href="https://api.genome.ucsc.edu">API</a>.
<pre>
http://api.genome.ucsc.edu/getData/track?genome=wuhCor1;track=nextstrainFreqB4;chrom=NC_045512v2;start=100;end=7875
</pre>
Command-line extraction can be accomplished using an example like the following command:
<pre>
bigWigToWig -udcDir=. -chrom=NC_045512v2 -start=100 end=7875 http://hgdownload.soe.ucsc.edu/gbdb/wuhCor1/nextstrain/nextstrainSamples.bigWig myOutput.wig
</pre></p>
<p>
Please refer to our <a href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome"
target="_blank">mailing list archives</a> for questions, or our
<a href="../FAQ/FAQdownloads.html#download36" target="_blank">Data Access FAQ</a> for more
information.</p>

<h2>Data usage policy</h2>
The data presented here is intended to rapidly disseminate analysis of
important pathogens. Unpublished data is included with permission of the data
generators, and does not impact their right to publish. Please contact the
respective authors (available via the
<a href="https://github.com/nextstrain/ncov/blob/master/data/metadata.tsv"
   target="_blank">Nextstrain metadata.tsv file</a>)
if you intend to carry out further research using their data.
Derived data, such as phylogenies, can be downloaded from
<a href="https://nextstrain.org/ncov" target="_blank">nextstrain.org</a>
(see &quot;DOWNLOAD DATA&quot; link at bottom of page) -
please contact the relevant authors where appropriate.

<h2>Credits</h2>
<p>Thanks to
  <a href="https://nextstrain.org/ncov" target="_blank">nextstrain.org</a> for
  sharing its analysis of genomes collected by
  <a href="https://gisaid.org/" target="_blank">GISAID EpiCoV TM</a>,
  and to researchers worldwide for sharing their SARS-CoV-2 genome sequences.
</p>

<h2>References</h2>
<p>
Hadfield J, Megill C, Bell SM, Huddleston J, Potter B, Callender C, Sagulenko P, Bedford T, Neher
RA.
<a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/bty407"
    target="_blank">
    Nextstrain: real-time tracking of pathogen evolution</a>.
<em>Bioinformatics</em>. 2018 Dec 1;34(23):4121-4123.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29790939" target="_blank">29790939</a>; PMC: <a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6247931/" target="_blank">PMC6247931</a>
</p>
<p>
Sagulenko P, Puller V, Neher RA.
<a href="https://academic.oup.com/ve/article-lookup/doi/10.1093/ve/vex042" target="_blank">
TreeTime: Maximum-likelihood phylodynamic analysis</a>.
<em>Virus Evol</em>. 2018 Jan;4(1):vex042.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29340210" target="_blank">29340210</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5758920/" target="_blank">PMC5758920</a>
</p>
