<h2>Description</h2>
<p>
<a href="https://nextstrain.org/ncov/global" target=_blank>Nextstrain.org</a> displays
data about mutations in the SARS-CoV-2 RNA and protein sequences that have
occurred in different samples of the virus during the current 2019-2021 outbreak.
Nextstrain has a powerful user interface for viewing the evolutionary tree
that it infers from the patterns of mutations in sequences worldwide, but
does not offer a detailed plot of mutations along the genome
that can be correlated with other molecular information,
so we have processed their data into this track to display the mutations
called by Nextstrain for each sample that Nextstrain has obtained from
<a href="https://gisaid.org" target=_blank>GISAID</a>.
</p>
<p>
Click on the vertical column in the display for any position in the
SARS-CoV-2 genome to see more details about the mutation(s) that occur
at that position, including protein change (if applicable; protein
changes use gene names in the <em>Nextstrain Genes</em> track), number of
samples with the mutation, list giving the nucleotide (allele) for that
position in each GISAID sample, etc.
</p>
<p>
Nextstrain identifies certain clades within the phylogenetic tree
according to
<a href="https://github.com/nextstrain/ncov/blob/master/defaults/clades.tsv" target="_blank">a set of defining mutations</a>.
The <a href="hgTrackUi?db=wuhCor1&g=nextstrainClade"><em>Nextstrain Clades</em></a>
track provides more information about these clades
and serves as a useful color key for the clade colors in the phylogenetic tree display.
</p>
<p>
This track is composed of several subtracks so that different subsets of mutations may be viewed:
  <ul>
    <li><b>Recurrent Bi-allelic</b>: This is the only subtrack displayed by default.
      It is limited to mutations that have been observed in at least two samples, and
      excludes positions at which more than one alternate allele has been observed in
      more than one sample.</li>
    <li><b>All</b>: All mutations found in all samples.</li>
    <li><b><em>&lt;Clade&gt;</em> Mutations</b>: All mutations found in samples belonging to
      <em>&lt;Clade&gt;</em>, which is one of Nextstrain's clades (19A, 19B, 20A, etc.)</li>
  </ul>
</p>

<h2>Display Conventions</h2>
<p>
In &quot;dense&quot; mode, a vertical line is drawn at each position where there is a mutation.
In &quot;pack&quot; mode, the display shows a plot of all samples' mutations, with samples
ordered using Nextstrain's phylogenetic tree in order to highlight patterns of linkage.</p>
<p>
Each sample is placed in a horizontal row of pixels; when the number of
samples exceeds the number of vertical pixels for the track, multiple
samples fall in the same pixel row and pixels are averaged across samples.</p>
<p>
Each mutation is a vertical bar at its position in the SARS-CoV-2 genome
with white (invisible) representing the reference allele
and black representing the non-reference allele(s).
Tick marks are drawn at the top and bottom of each mutation's vertical bar
to make the bar more visible when most alleles are reference alleles.
Insertions and deletions are not shown as these are removed from the data
by Nextstrain.
</p>
<p>
The phylogenetic tree for the samples built by Nextstrain is depicted
in the left column of the display.
Mousing over this will show the GISAID identifiers for the different samples.
When the vertical height of the track is set sufficiently high
(10 pixels per sample with the default font),
sample names are drawn to the right of the tree; however, with thousands of
samples in the Nextstrain tree, and a maximum track height of 2500 pixels,
the full Nextstrain tree is too large for sample names to be displayed.
In the track controls, the user can choose to display subtracks containing
the phylogenetic trees and mutations for individual clades.
Some clades have few enough samples that they can be made tall enough to
display sample names.
Branches of the phylogenetic tree are colored by clade using the same
color scheme as
<a href="https://nextstrain.org/ncov/global?c=clade_membership" target=_blank>nextstrain.org</a>.
</p>

<h2>Methods</h2>
<p>Nextstrain downloads SARS-CoV-2 genomes from
  <a href="https://gisaid.org/" target=_blank>GISAID</a>
  as they are submitted by labs worldwide, and downsamples to a subset of several thousand
  sequences in order to provide an interactive display.
  The selected subset of GISAID sequences is processed by an
  <a href="https://github.com/nextstrain/ncov" target=_blank>automated pipeline</a>,
  producing an annotated phylogenetic tree data structure underlying the Nextstrain display;
  UCSC downloads the results and extracts annotations for display.</p>

<h2>Data Access</h2>
<p>
SARS-CoV-2 mutations displayed by Nextstrain are derived from a subset of
<a href="https://www.gisaid.org" target="_blank">GISAID</a> sequences, and the GISAID
<a href="https://www.gisaid.org/registration/terms-of-use/" target="_blank">Terms and Conditions</a>
prohibit the redistribution of GISAID-derived data. They also require that the submitters of all
sequences be acknowledged when the mutations are used.
<a href="https://nextstrain.org/ncov/global" target="_blank">Nextstrain.org</a> offers
phylogenetic trees, author credits and other files:
scroll to the bottom of the page and click &quot;DOWNLOAD DATA&quot;, and a dialog with
download options appears.
</p>

<p>
All GISAID SARS-CoV-2 genome sequences and metadata are available for download from
<a href="https://gisaid.org" target="_blank">GISAID</a> EpiCoV&trade; by registered users.
We have a program faToVcf that can extract VCF from a multi-sequence FASTA alignment such as the
&quot;msa_<em>date</em>&quot;
download file from GISAID.  faToVcf is available for Linux and MacOSX on the download server:
<a href="https://hgdownload.soe.ucsc.edu/admin/exe/"
target="_blank">https://hgdownload.soe.ucsc.edu/admin/exe</a>.
It requires at least 4GB of memory to process the complete msa_<em>date</em> file.
Here are some steps to get started using faToVcf:</p>
<ul>
  <li> This command enables faToVcf to be run as a program (otherwise the command would say
&quot;Permission denied&quot;):
<pre>chmod a+x faToVcf</pre>

  <li> This command shows basic usage instructions and describes the options:
<pre>./faToVcf</pre>

  <li> This command converts msa fasta to VCF without per-sample genotype columns
    (substitute correct date for &quot;0925&quot; in filenames):
<pre>
./faToVcf -includeRef \
    -ref='hCoV-19/Wuhan/Hu-1/2019|EPI_ISL_402125|2019-12-31|Asia' \
    -vcfChrom=NC_045512.2 \
    -noGenotypes \
    msa_0925.fasta msa_0925.sites.vcf
</pre>
</ul>

<h2>Credits</h2>
<p>This work is made possible by the open sharing of genetic data by research
groups from all over the world. We gratefully acknowledge their contributions.
Special thanks to
  <a href="https://nextstrain.org/ncov" target=_blank>nextstrain.org</a> for
  sharing its analysis of genomes collected by
  <a href="https://gisaid.org/" target=_blank>GISAID</a>.
</p>

<h2>Data usage policy</h2>
<p>
The data presented here is intended to rapidly disseminate analysis of
important pathogens. Unpublished data is included with permission of the data
generators, and does not impact their right to publish. Please contact the
respective authors
if you intend to carry out further research using their data.
Author contact info is available via
<a href="https://nextstrain.org/ncov" target=_blank>nextstrain.org</a>:
scroll to the bottom of the page, click "DOWNLOAD DATA" and click
"ALL METADATA (TSV)" in the resulting dialog.
</p>

<h2>References</h2>
<p>
Hadfield J, Megill C, Bell SM, Huddleston J, Potter B, Callender C, Sagulenko P, Bedford T, Neher
RA.
<a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/bty407"
    target="_blank">
    Nextstrain: real-time tracking of pathogen evolution</a>.
<em>Bioinformatics</em>. 2018 Dec 1;34(23):4121-4123.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29790939" target="_blank">29790939</a>; PMC: <a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6247931/" target="_blank">PMC6247931</a>
</p>
<p>
Sagulenko P, Puller V, Neher RA.
<a href="https://academic.oup.com/ve/article-lookup/doi/10.1093/ve/vex042" target="_blank">
TreeTime: Maximum-likelihood phylodynamic analysis</a>.
<em>Virus Evol</em>. 2018 Jan;4(1):vex042.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29340210" target="_blank">29340210</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5758920/" target="_blank">PMC5758920</a>
</p>
<p>
Nguyen LT, Schmidt HA, von Haeseler A, Minh BQ.
<a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msu300" target="_blank">
IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood
phylogenies</a>.
<em>Mol Biol Evol</em>. 2015 Jan;32(1):268-74.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25371430" target="_blank">25371430</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4271533/" target="_blank">PMC4271533</a>
</p>
