<h2>Description</h2>
<p>
<a href="https://nextstrain.org/ncov" target=_blank>Nextstrain.org</a> displays
data about mutations that occur in the current 2019/2020 outbreak.
Nextstrain has a powerful user interface for viewing the timestamped molecular phylogeny tree
that it infers from the patterns of mutations in sequences worldwide using the
<a href="https://github.com/neherlab/treetime" target=_blank>TreeTime</a> algorithm.
</p>
<p>
Nextstrain defines clades named by year and letter, anticipating that SARS-CoV-2 will
become a seasonal virus in the coming years.  For more information about the rationale
and properties of current clades, see
<a href="https://github.com/nextstrain/ncov/blob/master/docs/src/reference/naming_clades.md"
   target=_blank>https://github.com/nextstrain/ncov/blob/master/docs/src/reference/naming_clades.md</a>.
</p>
<p>
The <em>Nextstrain Mutations</em> track contains all mutations shown on Nextstrain,
of which this track's mutations are a very small subset.
</p>

<h2>Methods</h2>
<p>Nextstrain downloads SARS-CoV-2 genomes from
  <a href="https://gisaid.org/" target=_blank>GISAID</a>
  as they are submitted by labs worldwide.
  Nextstrain identifies mutations that define clades of interest and puts them in the file
  <a href="https://github.com/nextstrain/ncov/blob/master/config/clades.tsv"
     target=_blank>clades.tsv</a>.
  The genome sequences and metadata including clades.tsv are processed by an
  <a href="https://github.com/nextstrain/ncov" target=_blank>automated pipeline</a>
  and annotations are written to a data file
  that UCSC downloads and extracts annotations for display.</p>

<h2>Data Access</h2>
<p>You can download the bigBed file underlying this track (nextstrainClade.bb) from our 
<a href="https://hgdownload.soe.ucsc.edu/gbdb/wuhCor1/nextstrain/" 
target=_blank>Download Server</a>. The data can be explored interactively with the 
<a href="../cgi-bin/hgTables">Table Browser</a>
or the <a href="../cgi-bin/hgIntegrator">Data Integrator</a>. The data can also be
accessed from scripts through our <a href="https://api.genome.ucsc.edu">API</a>.</p>

<h2>Credits</h2>
<p>Thanks to
  <a href="https://nextstrain.org/ncov" target=_blank>nextstrain.org</a> for
  sharing its analysis of genomes collected by
  <a href="https://gisaid.org/" target=_blank>GISAID</a>,
  and to researchers worldwide for sharing their SARS-CoV-2 genome sequences.
</p>

<h2>References</h2>
<p>
Hadfield J, Megill C, Bell SM, Huddleston J, Potter B, Callender C, Sagulenko P, Bedford T, Neher
RA.
<a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/bty407"
    target="_blank">
    Nextstrain: real-time tracking of pathogen evolution</a>.
<em>Bioinformatics</em>. 2018 Dec 1;34(23):4121-4123.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29790939" target="_blank">29790939</a>; PMC: <a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6247931/" target="_blank">PMC6247931</a>
</p>
