<h2>Description</h2>
<p>These tracks show curated SARS-CoV-2 protein-coding genes conserved within the Sarbecovirus subgenus as determined using PhyloCSF [1], FRESCo [2], and other comparative genomics methods, consistent with experimental evidence in SARS-CoV-2. Ambiguous gene names were resolved according to the recommendations in <a href="https://www.sciencedirect.com/science/article/pii/S0042682221000532" title="Jungreis, I., Nelson, C. W., Ardern, Z., Finkel, Y., Krogan, N. J., Sato, K., ... &amp; Kellis, M. (2021). Conflicting and ambiguous names of overlapping ORFs in the SARS-CoV-2 genome: A homology-based resolution. Virology 558, 145-151.">[3]</a>. For a complete description of the evidence, see <a href="https://www.nature.com/articles/s41467-021-22905-7" title="Jungreis I, Sealfon R, Kellis M (2021). SARS-CoV-2 gene content and COVID-19 mutation impact by comparing 44 Sarbecovirus genomes. Nature Communications 12(1), 1-20.">[4]</a>.</p> For a complete description of the evidence, see [4].</p>
<ul>
<li>
      The <b>PhyloCSF Genes</b> track shows the conserved protein-coding genes, namely ORF1a, ORF1ab, S, ORF3a, ORF3c, E, M, ORF6, ORF7a, ORF7b, ORF8, N, and ORF9b.
      <p> Notes:</p>
      <ul>
          <li>
          ORF3c is a 41 codon ORF overlapping ORF3a in a different frame with coordinates 25457-25582; it has also been referred to as ORF3h, ORF3a*, and 3a.iORF1.
          </li>
          <li>
              ORF9b is a 97 codon ORF overlapping N in a different frame with coordinates 28284-28577; it has also been referred to as  ORF9a.
          </li>
      </ul>
</li>

<li>
      The <b>PhyloCSF Rejected Genes</b> track shows other gene candidates that have been proposed that do not show the signature of conserved protein-coding genes or persuasive experimental evidence of function <a href="https://www.nature.com/articles/s41467-021-22905-7" title="Jungreis I, Sealfon R, Kellis M (2021). SARS-CoV-2 gene content and COVID-19 mutation impact by comparing 44 Sarbecovirus genomes. Nature Communications 12(1), 1-20.">[4]</a>, and are thus unlikely to be actual protein-coding genes, namely ORF2b, ORF3d, ORF3d-2, ORF3b, ORF9c, and ORF10.
      <p> Notes:</p>
      <ul>
          <li>
              ORF2b is a 39 codon ORF with coordinates 21744-21860 overlapping the spike protein in a different frame; it has also been referred to as S.iORF1.
          </li>
          <li>
              ORF3d is a 57 codon ORF with coordinates 25524-25697 overlapping ORF3a in a different frame; it has also been referred to as ORF3b.
          </li>
          <li>
              ORF3d-2 is a 33 codon ORF with coordinates 25596-25697 that is a subset of ORF3d starting at a downstream in-frame AUG codon; it has also been referred to as 3a.iORF2.
          </li>
          <li>
              ORF3b is the 22 codon ortholog of the 5' end of SARS-CoV ORF3b with coordinates 25814-25882, ending at an in-frame stop codon that is not present in SARS-CoV.
          </li>
          <li>
              ORF9c is a 73 codon ORF overlapping N in a different frame with coordinates 28734-28955; it has also been referred to as ORF9b and ORF14.
          </li>
      </ul>
  </li>
</ul>

<h2>Data Access</h2>
<p>
The raw data can be explored interactively with the
<a href="hgTables">Table Browser</a> or combined with other datasets in the
<a href="hgIntegrator">Data Integrator</a> tool.
For automated analysis, the genome annotation is stored in
a bigBed file that can be downloaded from
<a href="http://hgdownload.soe.ucsc.edu/gbdb/wuhCor1/bbi/contacts.bb">the download server</a>.</p>
<p>
Annotations can
be converted from binary to ASCII text by our command-line tool bigBedToBed.
Instructions for downloading this command can be found on our
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">utilities page</a>.
The tool can also be used to obtain features within a given range without downloading the file,
for example:</p>
<code>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/wuhCor1/bbi/phyloGenes/PhyloCSFgenes.bb -chrom=NC_045512v2 -start=0 -end=29902 stdout</code>
<p>
Please refer to our
<a href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing list archives</a>
for questions, or our
<a href="../../FAQ/FAQdownloads.html#download36">Data Access FAQ</a>
for more information.</p>


<h2>Methods</h2>
<p>See [4]. Note that the data was updated in June 2021: ORF14 was renamed to ORF9c, ORF2b and ORF3d-2 were added.</p>

<h2>Credits</h2>
<p>Questions should be directed to <a href="mailto:iljungr@csail.mit.edu"><span>Irwin Jungreis</span></a>.</p>
<p>If you use the SARS-CoV-2 PhyloCSF Genes Track Hub, please cite Jungreis et al. 2021 [4].</p>
<h2>References</h2>

<p><span>[1] </span>Lin MF, Jungreis I, and Kellis M (2011). PhyloCSF: a comparative genomics method to distinguish protein-coding and non-coding regions. <a href="https://academic.oup.com/bioinformatics/article/27/13/i275/178183"><span><i>Bioinformatics</i></span><span> 27:i275-i282</span></a> (ISMB/ECCB 2011).</p>

<p><span>[2] </span>Sealfon RS, Lin MF, Jungreis I, Wolf MY, Kellis M, Sabeti PC (2015). FRESCo: finding regions of excess synonymous constraint in diverse viruses. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4376164/"><span><i>Genome Biol. doi: 10.1186/s13059-015-0603-7</i></span></a>.</p>

<p><span>[3] </span>Jungreis, I., Nelson, C. W., Ardern, Z., Finkel, Y., Krogan, N. J., Sato, K., ... &amp; Kellis, M. (2021). Conflicting and ambiguous names of overlapping ORFs in the SARS-CoV-2 genome: A homology-based resolution. <i>Virology</i> 558, 145-151. <a href="https://www.sciencedirect.com/science/article/pii/S0042682221000532"><span>doi.org/10.1016/j.virol.2021.02.013</span></a></p>

<p><span>[4] </span>Jungreis I, Sealfon R, Kellis M (2021). SARS-CoV-2 gene content and COVID-19 mutation impact by comparing 44 <i>Sarbecovirus</i> genomes. <i>Nature Communications</i> 12(1), 1-20. <a href="https://www.nature.com/articles/s41467-021-22905-7"><span>doi:10.1038/s41467-021-22905-7</span></a></p>
