<h2>Description</h2>
<p>
This track shows the locations of those primers in detection kits that match the
reference sequence. The primers were copied from 
<a target="_blank" href="https://docs.google.com/spreadsheets/d/17eucK_mSAGgYvxVCKJkbbYaogr2ofbhgW2f9W7K3oOA/edit#gid=636736390">
a spreadsheet</a> created by the project
<a target="_blank" href="https://sites.google.com/view/opencovid19/home">
OpenCovid19</a>.
The initial version of the track used the FASTA file from <a href="https://tomeraltman.net/2020/04/05/all-public-primers.html" target=_blank">Design Flaws in COVID-19 Primers from Multiple International Labs</a>.
</p>

<p>
Most are RT-qPCR primer sets, sequencing primers have the prefix Seq1- or Seq2-. RT-qPCR sets consists of one forward, one reverse and one internal probe, as indicated by the names.</p>

<p>
As expected, the three control primers were not found at all: US-CDC-Control_RP-P, US-CDC-Control_RP-R, US-CDC-Control_RP-F.
</p>

<p>Here is a quick overview of the origin of the primers, please see the website and spreadsheet linked above for more details:</p>
<table class="stdTbl">
    </tr>
        <th>
            Prefix
        </th>
        <th>
            Institution
        </th>
        <th>
            Overview
        </th>
        <th>
            Manual
        </th>
    </tr>

    <tr>
        <td>CN-CDC-</td>
        <td>China CDC</td>
        <td></td>
        <td></td>
    </tr>

    <tr>
        <td>NIID-</td>
        <td>Nat. Inst. of Infect. Dis., Japan</td>
        <td></td>
        <td></td>
    </tr>

    <tr>
        <td>WH-</td>
        <td>National Inst. of Health (Thailand)</td>
        <td></td>
        <td></td>
    </tr>

    <tr>
        <td>HKU-</td>
        <td>The University of Hong King</td>
        <td>Detects N gene and Orf1b. Not specific for SARS-Cov2, but other Sarbecovirus species are not in circulation</td>
        <td><a href="https://www.who.int/docs/default-source/coronaviruse/peiris-protocol-16-1-20.pdf?sfvrsn=af1aac73_4" target=_blank>WHO Peiris Protocol</a></td>
    </tr>

    <tr>
        <td>US-CDC-</td>
        <td>CDC, USA</td>
        <td>Three reactions, target: N gene. One primer/probe set detects all
            betacoronaviruses, two sets are specific for SARS-CoV-2. All 3 assays must be positive 
        </td>
        <td><a href="https://www.fda.gov/media/134922/download" target=_blank>Instructions</a></td>
    </tr>

    <tr>
        <td>EU-Drosten</td>
        <td>Drosten Lab, Charite Berlin, Germany</td>
        <td>Set 1: run E and RdRp primers (designed for SARS-CoV, SARS-CoV-2, and bat-associated
        betacoronaviruses), if Set 1 is positive, use SARS-Cov-2 specific detection primer</td>
    <td><a href="https://www.who.int/docs/default-source/coronaviruse/protocol-v2-1.pdf?sfvrsn=a9ef618c_2" target=_blank>Instructions</a></td>
    </tr>

</table>

<p>
The sequences and the identifiers for these primers were obtained from the following sources, among others:

<ul>
    <li><a href="https://www.cdc.gov/coronavirus/2019-ncov/lab/rt-pcr-panel-primer-probes.html">CDC Laboratory Guidance</a>
    <li><a href="https://www.biorxiv.org/content/10.1101/2020.02.25.964775v1">Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2</a>
    <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6988269/">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</a>
</ul>
More details can be found in the spreadsheet linked above.


<h2>Methods</h2>
The primers were mapped with the following command:</br><tt>blat ../../wuhCor1.2bit primers.fa stdout -stepSize=3 -tileSize=6  -minScore=10 -oneOff=1 -noHead -fine | pslReps stdin stdout /dev/null -minNearTopSize=10 -minCover=0.8 -nohead > primers.psl</tt>


<h2>Data Access</h2>
<p>
You can download the PSL file underlying this track (primers) from our
<a href="http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/database/" 
target=_blank>Download Server</a>.
The data can be explored interactively with the <a href="../cgi-bin/hgTables">Table Browser</a>
or the <a href="../cgi-bin/hgIntegrator">Data Integrator</a>. The data can also be
accessed from scripts through our <a href="https://api.genome.ucsc.edu">API</a>.</p>

<h2>Credits</h2>
<p>
This data annotation track was made by Maximilian Haeussler, with assistance from Daniel Schmelter.
Fasta data collected by Tomer Altman (Biome Bioinformatics), prepared by Jason Fernandes (UCSC), updated by Darach Miller (Stanford) and the OpenCovid19 project.</p>
