<H2>Description</H2>
<p>This track shows data from Anna Pyle's lab at Yale University describing the RNA secondary 
structure of the SARS-CoV-2 genome. The authors performed experimental
measurements using SHAPE (selective 2-hydroxyl acylation) as well as in-silico analysis.
</p>

<p>These data are described in depth in 
<a target="_blank" href="https://www.biorxiv.org/content/10.1101/2020.07.06.190660v1.">de Cesaris Araujo Tavares <em>et al</em></a> 
and <a target="_blank" href="https://www.biorxiv.org/content/10.1101/2020.07.10.197079v1.full.pdf">Huston, Wan, <em>et al</em></a>.</p>

<H2>Display Conventions and Configuration</H2>

<p>Two tracks are available:</p>
<p><b>SHAPE reactivity:</b> A low SHAPE score (&lt;0.4) indicates that the nucleotide is
not accessible (folded between complementary regions - base-paired). A mid 
(0.4&lt;SHAPE&lt;0.85) or high SHAPE reactivity
(&gt;0.85) indicates the nucleotide is flexible (single-stranded). A score of -999
means no data for that nucleotide was recovered experimentally. For
visualization purposes the minimum display value has been set to -0.5 and the
max has been set to 4.</p>

<p><b>Full Length Shannon Entropy:</b> A genome-wide Shannon entropy profile derived from 
base pairing probabilities was computed using SuperFold with in vivo SHAPE reactivity as 
constraints . A low Shannon entropy (near 0) is evidence for well-determined RNA conformation.
</p>

<p>Note that the authors provide further data (including secondary structure 
predictions) at their github repository.</p>

<H2>Methods</H2>
<p>For Shannon entropy, comSuperFold was used with default settings to predict
secondary structure for the full-length SARS-CoV-2 RNA genome.</p>
<p>For
SHAPE_reactivity, VeroE6 cells were infected with 105 PFU of SARS133 CoV-2
isolate USA-WA1/2020. See the <a target=_blank
href="https://www.biorxiv.org/content/10.1101/2020.07.10.197079v1.full.pdf">preprint</a>
for further details.  
wigToBigWig was used to convert wiggles from the author's github repository to bigWig files after filtering out headers.
</p>

<p>The Github repository with all raw data files can be found here: <a target="_blank" 
href="https://github.com/pylelab/SARS-CoV-2_SHAPE_MaP_structure">https://github.com/pylelab/SARS-CoV-2_SHAPE_MaP_structure</a>.</p>

<H2>References</H2>

<p>
Huston NC, Wan H, Araujo Tavares RC, Wilen C, Pyle AM.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32676598" target="_blank">
    Comprehensive in-vivo secondary structure of the SARS-CoV-2 genome reveals novel regulatory motifs
    and mechanisms</a>.
<em>bioRxiv</em>. 2020 Jul 10;.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32676598" target="_blank">32676598</a>; PMC: <a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7359520/" target="_blank">PMC7359520</a>
</p>
<p>
Araujo Tavares RC, Mahadeshwar G, Pyle AM.
<a href="https://www.biorxiv.org/content/10.1101/2020.07.06.190660v1." target="_blank">
    The global and local distribution of RNA structure throughout the SARS-CoV-2 genome</a>.
<em>bioRxiv</em>. 2020 Jul 7;.
doi: <a href="https://doi.org/10.1101/2020.07.06.190660" target="_blank">190660</a>
</p>
