<h2>Description</h2> 
<p>
The SARS-CoV-2 spike protein, which binds the virus to host cells, is a key target of vaccine 
development to combat COVID-19, and mutations in this protein have potential to change infectivity 
and response to disease treatments as well as vaccine efficacy.
As of February 2021 more than a dozen mutations in this protein have been detected via sequencing 
worldwide, and specific mutations have been identified to be associated with
viruses that are significantly more transmissible.</p>
<p>
This track presents amino acid mutations identified in the SARS-CoV2 Spike protein, based on the community annotation at 
<a target="_blank" href="https://covariants.org">CoVariants.org</a>,
supplemented by the
<a target="_blank" href="https://en.wikipedia.org/wiki/Variants_of_SARS-CoV-2">
Variants of SARS-CoV-2</a> Wikipedia page.</p>

<p>
Mutations in this track (with nicknames): H69-, D80Y, S98F, A222V, N439K (Nick), L452R, Y453F, S477N, E484 (Eeek), N501 (Nelly), D614G (Doug), A626S, P681H (Pooh), A701B, V1122</p>
<p>
Information provided for each mutation, if available, includes:
<ul>
<li>Date first sequenced</li>
<li>Notes of clinical significance</li>
<li>Notes regarding geographic incidence</li>
<li>Link to charts and tables presenting geographic distribution</li>
<li>Link to Nextstrain build</li>
<li>Links to relevant publications</li>
</ul>
</p>


<h2>Display Conventions</h2>
<p>
The track items are colored as follows:

<table cellpadding="2">
<tr><td>Red</td><td style='background-color:#FF0000; padding-left: 20px;'></td><td>Identified as strong antibody escape by Bloom Lab RBD-mutation screen</td></tr>
<tr><td>Purple</td><td style='background-color:#800080; padding-left: 20px;'></td><td>ACE2 receptor binding region (RBD)</td></tr>
<tr><td>Blue</td><td style='background-color:#0000FF; padding-left: 20px;'></td><td>Other</td></tr>
</table>

<!--
<table cellpadding="2">
<tr><td>Red</td><td style='background-color:#FF0000; padding-left: 20px;'></td><td>Most recent (Fall/Winter 2020-2021)</td></tr>
<tr><td>Purple</td><td style='background-color:#800080; padding-left: 20px;'></td><td>Summer 2020</td></tr>
<tr><td>Blue</td><td style='background-color:#0000FF; padding-left: 20px;'></td><td>Oldest (Winter/Spring 2020)</td></tr>
</table>
-->

<p>
The mutation name, e.g. N501 as well as alternative identifiers (e.g. N501Y, 501Y) or nickname 
if present, can be typed in to the browser position box to navigate the browser to the mutation 
position and highlight the mutation in the browser window.</p>

<h2>Release Notes</h2>
<p>
This track was updated to include the <em>L453R</em> mutation in the B.1.429 variant (first 
identified in California), as displayed in the 
<a target="_blank" href="../cgi-bin/hgTrackUi/db=wuhCor1&g=variantMuts">Variants of Concern</a> 
track.
The color scheme was also changed in this track update.
</p>

<h2>Data Access</h2>
<p>
The raw data can be explored interactively with the <A HREF="../../cgi-bin/hgTables"TARGET=_blank>Table Browser</a>, or <A HREF="../../cgi-bin/hgIntegrator"TARGET=_blank>Data Integrator</a>.
For automated analysis, the genome annotation can be downloaded from the
<A HREF="http://hgdownload.soe.ucsc.edu/gbdb/$db/spikeMuts/"
TARGET=_blank>downloads server</a>.
Data files for earlier versions of this track can be downloaded from our 
<a href="http://hgdownload.soe.ucsc.edu/goldenPath/archive/$db/spikeMuts">archive download server</a>.
Please refer to our <A HREF="https://groups.google.com/a/soe.ucsc.edu/g/genome"TARGET=_blank>mailing list archives</a>
for questions, or our <A HREF="../FAQ/FAQdownloads.html#download36"TARGET=_blank>Data Access FAQ</a> for more information.
</p>

<h2>Credits</h2>
<p>
Thanks to <a target="_blank" href="https://neherlab.org/emma-hodcroft.html">Emma B. Hodcroft</a>, Institute of Social and Preventive Medicine, University of Bern, Switzerland
for leading and maintaining the community annotation resource on which this track is largely based.<p>

<h2>References</h2>
<p>
<a target="_blank" href="https://github.com/hodcroftlab/covariants#covariants-sars-cov-2-mutations-and-variants-of-interest">CoVariants: SARS-CoV-2 Mutations and Variants of Interest</a>
Emma B. Hodcroft, Institute of Social and Preventive Medicine, University of Bern, Bern, Switzerland
</p>
<p>
<a target="_blank" href="https://en.wikipedia.org/wiki/Variants_of_SARS-CoV-2">Variants of SARS-CoV-2</a>, Wikipedia
</p>



<!-- Redmine Track #26782 -->
