<h2>Description</h2>
<p>
This track displays amino acid and nucleotide mutations in SARS-CoV-2 variants
as defined in December 2021 by <a target='_blank' 
href='https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/'>
the World Health Organization (WHO)</a>. Note that the 
<a href="https://www.cdc.gov/coronavirus/2019-ncov/variants/variant-info.html" 
target="_blank">Center for Disesase Control (CDC) classification</a> of SARS-CoV-2 
variants is slightly different than the WHO.</p>

<p>
Mutations in this track were identified from viral sequences from 
<a href="https://gisaid.org" target=_blank>GISAID</a>.
Variant incidence and geographic distribution information is available from links to the 
<a target="_blank" href="https://Outbreak.info">Outbreak.info</a> web resource
on the mutation details pages.</p>

<p><ul>
<li><b>Variants of Concern</b> (VOC) have evidence for increased transmissibility, 
virulence, and/or decreased diagnostic, therapeutic, or vaccine efficacy.</li>
<li><b>Variants of Interest</b> (VOI) contain mutations suspected or confirmed 
to cause a change in transmissibility, virulence, or diagnostic / therapeutic / 
vaccine efficacy, plus evidence of significant community transmission, 
a cluster of cases, or detection in multiple countries.</li>
<li><b>Variants under Monitoring</b> (VUM) include variants with unclear 
epidemiological impact. This track includes only the four VUMs which were previously 
identified as Variants of Interest, now reclassified at this lower level of concern.</li>
</ul>
</p>

<p>
The related track
<a target="_blank" href="../cgi-bin/hgTrackUi?db=wuhCor1&g=lineageB_1_1_7_US"> B.1.1.7 in USA</a>
displays a phylogenetic tree of the first B.1.1.7 (Alpha) variant sequences collected 
in the United States.
</p>

<p>BV-BRC has a similar <a href="https://www.bv-brc.org/view/VariantLineage/#view_tab=overview"
target=_blank>list of variants of concern and their mutations</a>, but has added 
representative sequences.
</p>


<h2>Display Conventions</h2>
<p>
Track colors are based on
<a target="_blank" href="https://nextstrain.org/ncov/gisaid/global">Nextstrain.org</a>
clade coloring:
</p>

<img style="margin: 8px; border: 1.5px solid; width: 801px; height: 547" 
        src="../images/nextstrain.2021-12-02.png">

<p>
The Greek-letter names assigned by the World Health Organization (WHO) are listed 
in this table, along with lineage and clade designations:

<table style="border-spacing: 20px 2px;">
  <tr>
    <th>Color</th><th>WHO label</th>
        <th>Pangolin lineage</th><th>Nextstrain clade</th><th>GISAID clade</th>
        <th>First detected</th><th>Date VOC/VOI</th><th>Type</th>
    </tr>

  <tr>
    <td><span style="background-color: #3F4CCB;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></td>
    <td>Alpha  </td>
    <td>B.1.1.7</td>
    <td>20I (V1)</td>
    <td>GRY</td>
    <td>Sep 2020, United Kingdom</td>
    <td>18-Dec-2020</td>
    <td>VOC</td>
  </tr>

  <tr>
    <td><span style="background-color: #492AB5;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></td>
    <td>Beta   </td>
    <td>B.1.351</td>
    <td>20H (V2)</td>
    <td>GH/501Y.V2</td>
    <td>May 2020, South Africa</td>
    <td>18-Dec-2020</td>
    <td>VOC</td>
  </tr>

  <tr>
    <td><span style="background-color: #4271CE;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></td>
    <td>Gamma</td>
    <td>P.1</td>
    <td>20J (V3)</td>
    <td>GR/501Y.V3</td>
    <td>Nov 2020, Brazil</td>
    <td>11-Jan-2021</td>
    <td>VOC</td>
  </tr>

  <tr>
    <td><span style="background-color: #4C8FC0;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></td>
    <td>Delta</td>
    <td>B.1.617.2</td>
    <td>21A</td>
    <td>GK/478K.V1</td>
    <td>Oct 2020, India</td>
    <td>11-May-2021</td>
    <td>VOC</td>
  </tr>

  <tr>
    <td><span style="background-color: #DB2823;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></td>
    <td>Omicron</td>
    <td>B.1.1.529</td>
    <td>21K</td>
    <td>GR/484A</td>
    <td>Nov 2020, South Africa</td>
    <td>26-Nov-2021</td>
    <td>VOC</td>
  </tr>

  <tr>
    <td><span style="background-color: #E68033;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></td>
    <td>Lambda</td>
    <td>C.37</td>
    <td>21G</td>
    <td>GR/452Q.V1</td>
    <td>Dec 2020, Peru</td>
    <td>14-Jun-2021</td>
    <td>VOI</td>
  </tr>
  <tr>
    <td><span style="background-color: #E2562B;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></td>
    <td>Mu</td>
    <td>B.1.621</td>
    <td>21H</td>
    <td>GH</td>
    <td>Jan 2021, Colombia</td>
    <td>30-Aug-2021</td>
    <td>VOI</td>
  </tr>

  <tr>
    <td><span style="background-color: #A0BE59;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></td>
    <td>Epsilon (former)</td>
    <td>B.1.429</td>
    <td>21C</td>
    <td>GH/452R.V1</td>
    <td>Mar 2020, USA</td>
    <td>06-Jul-2021</td>
    <td>VUM</td>
  </tr>

  <tr>
    <td><span style="background-color: #BBBC49;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></td>
    <td>Eta (former)</td>
    <td>B.1.525</td>
    <td>21D</td>
    <td>G/484K.V3</td>
    <td>Dec 2020</td>
    <td>20-Sep-2021</td>
    <td>VUM</td>
  </tr>

  <tr>
    <td><span style="background-color: #E19F3A;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></td>
    <td>Iota (former)</td>
    <td>B.1.526</td>
    <td>21F</td>
    <td>GH/253G.V1</td>
    <td>Nov 2020, USA</td>
    <td>20-Sep-2021</td>
    <td>VUM</td>
  </tr>

  <tr>
    <td><span style="background-color: #85BA6F;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></td>
    <td>Kappa (former)</td>
    <td>B.1.617.1</td>
    <td>21B</td>
    <td>G/452R.V3</td>
    <td>Oct 2020, India</td>
    <td>20-Sep-2021</td>
    <td>VUM</td>
  </tr>

</table>
</p>
<p>
Mutations in the amino acid track are named with the format: 
<pre>
        [Reference amino acid][1-based coordinate in peptide][Alternate amino acid]. E.g., L452R
</pre>  
<p>
Mutations in the nucleotide track are named with the format: 
<pre>
        [Reference nucleotide][1-based coordinate in genome][Alternate nucleotide]. E.g., T22918G
</pre>
Insertions and deletions in both tracks are named: 
<pre>
        [del/ins]_[1-based genomic coordinate of first affected nucleotide].  E.g., del_21991
</pre>
</p>

<h2>Methods</h2>
<p>
For each virus variant, SARS-CoV-2 genome sequences containing all characteristic
mutations of the lineage were downloaded from 
<a target="_blank" href="https://www.gisaid.org/hcov19-variants/">GISAID</a>
 using the lineage search feature
(restricting to complete, high-coverage genomes, and restricting to earliest sample
collection dates when there were too many results for the download limit of 10,000
sequences per query).
</p>

<p>
Sequences were aligned to the
<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_045512.2">
SARS-CoV-2 reference genome</a>
using the
<a href="https://github.com/roblanf/sarscov2phylo/blob/master/scripts/global_profile_alignment.sh" target="_blank">
global_profile_alignment.sh</a>
script from the
<a href="https://github.com/roblanf/sarscov2phylo/" target=_blank>sarscov2phylo</a> repository.
Single-nucleotide substitutions were extracted from the alignment using the UCSC tool
<em>faToVcf</em>
(available on <a target="_blank" 
href="https://hgdownload.soe.ucsc.edu/admin/exe/">the UCSC download server</a>
or from <a target="_blank" href="https://anaconda.org/bioconda/ucsc-fatovcf">bioconda</a>;
also requires the
<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_045512.2">
SARS-CoV-2 reference sequence</a>).
Single-nucleotide substitutions present at a frequency
of at least 0.95 (.70 for Delta, .80 for Omicron) were retained while all others are discarded.
</p>
<p>
For indel detection, the 
<a target="_blank" href="https://github.com/lh3/minimap2">Minimap2</a> 
suite of tools was used as follows:
</p>
<pre>
        minimap2 --cs [Reference Sequence] [Set of Unaligned Sequences] | paftools.js call -L 10000 -
</pre>
<p>
Indels present at a frequency of at least 0.85 (.50 for Delta, .70 for Omicron) were retained.
Less stringent cutoffs were applied to Delta and Omicron
variant sequences due to low quality of early sequences.</p>
<p>
The results were then combined and formatted by
<a target="blank" 
href="https://github.com/nickeener/sarscov2lineages/blob/main/scripts/lineageVariants.py">lineageVariants.py</a>. 
The entire pipeline was run using
<a target="blank" href="https://github.com/nickeener/sarscov2lineages/blob/main/lineageVariants.sh">
lineageVariants.sh</a>.
<p>

<h2>Data Access</h2>
<p>
You can download the bigBed data files for this track from the
<a href="https://hgdownload.soe.ucsc.edu/gbdb/wuhCor1/variantMuts/"
target=_blank>UCSC Download Server</a>.
The data can be explored interactively with the
<a href="hgTables">Table Browser</a>
or the <a href="hgIntegrator">Data Integrator</a>. The data can be
accessed from scripts through our <a href="https://api.genome.ucsc.edu">API</a>.
For complete genome Fasta sequences of variants of concern, please visit the following 
third-party page:
</p>
<ul>
<li><a href="https://www.bv-brc.org/view/VariantLineage/" target="_blank">SARS-CoV-2 Variants
and Lineages of Concern</a> from the Bacterial and Viral Bioinformatics Resource 
Center</a>.</li>
</ul>

<h2>Release Notes</h2>
<p>
Version 2 of this track adds one new Variant of Concern (Delta), two new Variants of 
Interest (Lambda, Mu), and three named variants previously VOI, now designated as less 
concerning Variants under Monitoring (Eta, Iota, Kappa). 
The track labels of all variants have been updated to include WHO labels. Track colors 
reflect Nextstrain conventions at the time of track data update (September 10, 2021).
</p>
<p>
Omicron was added December 2, 2021.
</p>

<h2>Credits</h2>
<p>
This work is made possible by the open sharing of genetic data by research
groups from all over the world. We gratefully acknowledge their contributions.
We thank 
<a target="_blank" href="https://www.robertlanfear.com/">Rob Lanfear</a> 
at the Australia National University for developing and maintaining the
<a href="https://github.com/roblanf/sarscov2phylo/" target=_blank>sarscov2phylo</a> web resource.
We also thank 
the <a href="https://sulab.org/" rel="noreferrer" target="_blank">Su</a>, 
<a href="https://wulab.io/" rel="noreferrer" target="_blank">Wu</a>, 
and <a href="https://andersen-lab.com/" rel="noreferrer" 
target="_blank">Andersen</a> labs at Scripps Research for creating the 
<a target="_blank" href="https://Outbreak.info">Outbreak.info</a> resource.
The lineageVariants scripts were developed and run at UCSC by Nick Keener,
Kate Rosenbloom and Angie Hinrichs.
</p>

<h2>References</h2>
<p>
Rambaut A, Holmes EC, O'Toole &#193;, Hill V, McCrone JT, Ruis C, du Plessis L, Pybus OG.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32669681" target="_blank">
A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</a>.
<em>Nat Microbiol</em>. 2020 Nov;5(11):1403-1407.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32669681" target="_blank">32669681</a>
</p>
<p>
Rambaut A, Loman N, Pybus O, Barclay W, Barrett J,
Carabelli A, Connor T, Peacock T, Robertson DL, Volz E, <em>et al.</em>
<a href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563"
   target=_blank>Preliminary genomic characterization of an emergent SARS-CoV-2 lineage in the UK
  defined by a novel set of spike mutations</a>.
<em>Virological</em>.  2020 Dec 18.
</p>

<p>
Volz E, Mishra S, Chand M, Barrett JC, Johnson E,
Geidelberg L, Hinsley WR, Laydon DJ, Dabrera G, O'Toole &#193;, <em>et al.</em>
<a href="https://virological.org/t/transmission-of-sars-cov-2-lineage-b-1-1-7-in-england-insights-from-linking-epidemiological-and-genetic-data/576" target=_blank>
Transmission of SARS-CoV-2 Lineage B.1.1.7 in England: Insights from linking 
epidemiological and genetic data</a>.
<em>Virological</em>.  2020 Dec 31.</p>

<p>
Tegally <em>et al</em>, December 21, 2020.
<a target="_blank" href="https://www.medrxiv.org/content/10.1101/2020.12.21.20248640v1">
Emergence and rapid spread of a new severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) lineage with multiple spike mutations in South Africa</a> medRxiv preprint.
Zhang <em> al</em>, January 20, 2021.
<a target="_blank" href="https://www.medrxiv.org/content/10.1101/2021.01.18.21249786v1">
Emergence of a novel SARS-CoV-2 strain in Southern California, USA</a> 
medRxiv preprint.
</p>

<p>
Voloch <em>et al</em>, December 26, 2020. 
<a target="_blank" href="https://www.medrxiv.org/content/10.1101/2020.12.23.20248598v1">
Genomic characterization of a novel SARS-CoV-2 lineage from Rio de Janeiro, Brazil</a> medRxiv preprint.
</p>

<p>
Lanfear, Rob (2020). A global phylogeny of SARS-CoV-2 sequences from GISAID. Zenodo DOI:
<a href="https://zenodo.org/record/4289383" target=_blank>10.5281/zenodo.3958883</a>
</p>

<p>
Li Heng
<a href="https://www.ncbi.nlm.nih.gov/pubmed/29750242" target="_blank">
Minimap2: pairwise alignment for nucleotide sequences</a>.
<em>Bioinformatics</em>. 2018 Sep 15;34(18):3094-3100.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29750242" target="_blank">29750242</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6137996/" target="_blank">PMC6137996</a>
</p>

<p>
Gangavarapu, Karthik; Alkuzweny, Manar; Cano, Marco; Haag, Emily; Latif, Alaa Abdel; 
Mullen, Julia L.; Rush, Benjamin; Tsueng, Ginger; Zhou, Jerry; Andersen, Kristian G.; 
Wu, Chunlei; Su, Andrew I.; Hughes, Laura D. 
<a target="_blank" href="https://outbreak.info/">Outbreak.info</a>
</p>
