<H3>Description</H3>
<P>This column shows the ratio of expression of each gene in a particular phase
of the worm life cycle to the expression of the gene in mixed wild-type N2
cultures.  These experiments were replicated several times.  This track
shows just a representative replication.</P>
<P>
A gene that is more highly 
expressed in a life cycle phase is colored red, and a less expressed gene is 
shown in green. The values are colored on a logarithmic scale. This coloring is 
standard, but is the opposite of what an inexperienced user 
might expect: in this case, red means go and green means stop!  
Black indicates that a gene is neither over- nor under-expressed in a life cycle
phase. Uncolored boxes (white on most browsers) represent missing data.  

<H3>Column Configuration</H3>
Column configuration options are available on the Configuration page, 
which can be accessed via the <em>configure</em> button in the top menu.
<UL>
<LI>
<B>Color scheme:</B> Color-blind users can switch the coloring scheme from red/green to 
yellow/blue via the <em>Expression ratio colors</em> pulldown menu at the top
of the Configuration page.</LI>
<LI>
<B>Brightness:</B> To increase or decrease the brightness of the expression colors,
edit the number in the column's <em>brightness</em> text box. Values greater 
than 1.0 increase the brightness, while those less than 1.0 dim the color. </LI>
</UL>

<H3>Methods</H3>
The data in this column were collected as described in the paper
Jiang M, et.al. (2001), 
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=11134517&dopt=Abstract" TARGET=_blank>"Genome-wide analysis of 
developmental and sex-regulated gene expression profiles in 
Caenorhabditis elegans"</A>, <em>PNAS</em> 98(1):218-23. 
We retrieved the data from the
<A HREF="http://genome-www5.stanford.edu/" TARGET=_blank>Stanford Microarray 
Database</A>.
<P>To associate particular genes with expression data we use a fairly simple-minded
scheme.  First we align the cDNA sequences around which the probes were designed to
the genome.  Then for each gene we pick the cDNA sequence with the alignment that most
overlaps the gene as the source for expression data for that gene.  Since many of the cDNA 
sequences and probes designed for the cDNA sequences just cover part of the gene, in most
cases different splice varients of a gene will end up mapped to the same cDNA sequence. 
</P>

<H3>Credits</H3>
Many thanks to the <A HREF="http://cmgm.stanford.edu/~kimlab/" TARGET=_blank>Kim 
Lab</A> for making the data available.

