<H2>Description</H2>
<P>
This track shows syntenous (corresponding) regions between $organism and 
rat chromosomes. The Nov. 2002 (rn1) assembly of the rat genome 
was used in this annotation.</P>

<H2>Methods</H2>
<P>
We passed a 100k non-overlapping window over the genome and, using the 
blastz best-in-rat genome alignments, looked for high-scoring regions with at 
least 40% of the bases aligning with the same region in rat. 100k 
segments were joined together if they agreed in direction and were within 500 kb 
of each other in the $organism genome and within 4 Mb of each other in the 
rat.  Gaps were joined between syntenic anchors if the bases between two 
flanking regions agreed with synteny (direction and rat location). 
Finally, we extended the syntenic block to include those areas.</P>

<H2>Credits</H2>
<P>
Contact <A HREF="mailto:&#98;&#97;&#101;&#114;&#116;s&#99;h&#64;&#99;&#115;&#101;.&#117;&#99;s&#99;.e&#100;&#117;">Robert 
Baertsch</A> at UCSC for more information about this track.</P>

<H2>References</H2>
<P>
Schwartz, S., Kent, W.J., Smit, A., Zhang, Z., Baertsch, R., Hardison, R., 
Haussler, D., and Miller, W.
<A HREF="http://www.genome.org/cgi/content/abstract/13/1/103" 
TARGET=_blank>Human-Mouse Alignments with BLASTZ</A>. 
<em>Genome Res.</em> <B>13</B>(1), 103-7 (2003).</P>

