<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=315>
  <TR><TD ALIGN=RIGHT>
    <A HREF="https://www.ncbi.nlm.nih.gov/Traces/wgs/?val=CABD02"
    TARGET=_blank>
    <IMG SRC="../images/Gorilla_gorilla_gorilla.jpg" WIDTH=300 HEIGHT=300 ALT="Gorilla"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Gorilla gorilla gorilla</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Public Domain Photograph
    </FONT>
  </TD></TR>
</TABLE>

<P>
The May 2011 <em>Gorilla gorilla gorilla</em> draft assembly 
is produced by the
<A HREF="http://www.sanger.ac.uk/resources/downloads/gorilla/"
TARGET=_blank>Wellcome Trust Sanger Institute</A>
(<A HREF="https://www.ncbi.nlm.nih.gov/bioproject/31265"
TARGET=_blank>NCBI project 31265</A>, GCA_000151905.1). For more
information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/309168/" TARGET="_blank">
gorGor3.1</A> in the NCBI Assembly database.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Gorilla genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>chrM</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of chromosome M</TD>
    </TR>
    <TR><TD VALIGN=Top>chr3:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of chr 3</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr3:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 3 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>NM_201653</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>This search finds the non-Gorilla RefSeq accessioned item with name: NM_201653</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>NM_201653; NM_029604</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Display the region between these two non-Gorilla RefSeq accessioned items</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>BX950328</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number BX950328</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene mRNA</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zahler</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zahler</TD>
    </TR>
    <TR><TD VALIGN=Top>Taguchi,T.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author T. Taguchi</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Taguchi T</em>, internally it
	uses the format <em>Taguchi,T.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
See also: <A HREF="https://www.ncbi.nlm.nih.gov/Traces/wgs/?val=CABD02"
TARGET=_blank>WGS CABD02</A><BR>
Assembly involved several phases, starting with a de novo assembly of the
initial data using the ABySS and Phusion assemblers. Contigs from this
'seed' assembly were then grown by assembling and attaching Solexa read
pairs from the initial data. To improve long-range structure, supercontig
construction was guided by placing (where possible) seed contigs in
accordance with their homologous locations on the human genome, breaking
supercontigs wherever a potential break between human and gorilla was
inferred in an alignment of all the Solexa data to human.
</P><P>
This draft assembly contains all the assembled supercontigs plus a
mitochondrial sequence (chrM - NC_011120).
</P><P>
Two types of sequencing were used in this assembly:
<UL>
<LI> 2.1x WGS capillary sequenced read pairs.
<LI> 35x Solexa sequenced read pairs with insert sizes 150 bp and 450 bp.
</UL>
All sequence was derived from DNA sampled from a single female Western
Lowland gorilla (Gorilla gorilla gorilla), Kamilah. 
</P>

<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/gorGor3/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#gorilla">Downloads</A> page. 

These data have
<A HREF="http://www.sanger.ac.uk/datasharing/"
>specific conditions for use</A>. 

The Gorilla browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#gorilla_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
