<H2>Description</H2>
<P>
This track depicts gaps in the May 2011 <em>Gorilla gorilla gorilla</em>
draft assembly produced by the
<A HREF="https://www.sanger.ac.uk/resources/downloads/gorilla/"
TARGET=_blank>Wellcome Trust Sanger Institute</A>
(<A HREF="https://www.ncbi.nlm.nih.gov/bioproject/31265"
TARGET=_blank>NCBI bioproject 31265</A>, GCA_000151905.1).

Many of these gaps &mdash; with the exception of intractable
heterochromatic gaps &mdash; may be closed during the finishing process.
</P>

<P>
Gaps are represented as black boxes in this track.
If the relative order and orientation of the contigs on either side
of the gap is supported by read pair data, 
it is a <em>bridged</em> gap and a white line is drawn 
through the black box representing the gap. 
</P>
<P>This assembly contains the following principal types of gaps:

<UL>
<LI><B>Other</B> - Gaps of varying size marked with <em>N</em>s in the
sequence that were not annotated in the AGP assembly definition file.</LI>
<LI><B>Fragment</B> - Gaps between the Whole Genome Shotgun contigs of a 
supercontig. (In this context, a contig is a set of overlapping sequence reads.
A supercontig is a set of contigs ordered and oriented during the 
Whole Genome Shotgun process using paired-end reads.)
These are represented by varying numbers of <em>N</em>s in the assembly.</LI>
<LI><B>Contig</B> - Gaps of unknown size between contigs, all at a default
size of 100 bases marked as <em>N</em>s in the assembly.</LI>
</UL></P>

<P>
See also: <A HREF="https://www.ncbi.nlm.nih.gov/Traces/wgs/?val=CABD02"
TARGET=_blank>WGS CABD02</A>
</P>
