<!DOCTYPE html>
<!--#set var="TITLE" value="CIRM Stem Cell Hub at UCSC: Stem Cell Experiment Standards" -->

<!--#include virtual="setRoot.html" -->

<!--#include virtual="${ROOT}inc/header.html" -->

<div class="cirm-page-body">
  <div class="cirm-page-title back-orange">
     <h4 class="cirm-page-title-text">Stem Cell Experiment 'Standards'</h4>
  </div>
  <a name="experiments"></a>
  <div class="row no-gutters">
    <div class="col-md-12">
      <div class="cirm-page-title back-dark-blue text-center">
        <h5 class="cirm-page-title-text">Experiments</h5>
      </div>
    </div>
  </div>
  <div class="row">
    <div class="col-md-12 text-center">
      <!-- Image Map Generated by http://www.image-map.net/ -->
      <img src="../images/Experiments.png" usemap="#experiments-map">

      <map name="experiments-map">
        <!-- Identify Drug Targets -->
        <area target=""
        alt="Stem cells can be used to identify drug targets and test potential therapeutics."
        title="Stem cells can be used to identify drug targets and test potential therapeutics."
        href="" coords="98,261,79" shape="circle">
        <!-- Toxicity Screen -->
        <area target=""
        alt="New drugs may be tested on stem cells for safety and effectiveness, prior to testing new drugs in human trials."
        title="New drugs may be tested on stem cells for safety and effectiveness, prior to testing new drugs in human trials."
        href="" coords="97,533,79" shape="circle">
        <!-- Tissue/Cell Generation -->
        <area target=""
        alt="Stem cells may be utilized to generate healthy cells to replace diseased cells."
        title="Stem cells may be utilized to generate healthy cells to replace diseased cells."
        href="" coords="388,689,81" shape="circle">
        <!-- Personalized Medicine -->
        <area target=""
        alt="iPS cells that are derived from patients can be used to examine individualized responses to drugs and therapies."
        title="iPS cells that are derived from patients can be used to examine individualized responses to drugs and therapies."
        href="" coords="677,533,80" shape="circle">
        <!-- Developmental Studies -->
        <area target=""
        alt="By studying stem cell differentiation we can deepen our understanding of prevention and treatment of birth defects."
        title="By studying stem cell differentiation we can deepen our understanding of prevention and treatment of birth defects."
        href="" coords="668,261,80" shape="circle">
        <!-- Disease Models -->
        <area target=""
        alt="Stem cells can be used to generate disease-specific tissues. This allows researchers to better understand disease states and aid the drug development process."
        title="Stem cells can be used to generate disease-specific tissues. This allows researchers to better understand disease states and aid the drug development process."
        href="" coords="387,96,79" shape="circle">
      </map>
    </div>
  </div>

  <a name="samples"></a>
  <div class="row">
    <div class="col-md-12">
      <div class="cirm-page-title back-dark-blue text-center">
        <h5 class="cirm-page-title-text">Samples</h5>
      </div>
    </div>
  </div>
  <div class="row">
    <div class="col-md-12 text-center">
      <!-- Image Map Generated by http://www.image-map.net/ -->
      <img src="images/SampleDiagram.png" usemap="#sample-diagram">

      <map name="sample-diagram">
        <!-- Stem Cell -->
        <area target="_blank"
        alt="Stem Cell" title="Stem Cell"
        href="https://www.ebi.ac.uk/ols/ontologies/clo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000034"
        coords="52,185,37" shape="circle">
        <!-- Transformed -->
        <area target="_blank" alt="Transformed" title="Transformed"
        href="https://www.ebi.ac.uk/ols/ontologies/ero/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FERO_0002001"
        coords="63,365,48" shape="circle">
        <!-- Somatic -->
        <area target="_blank" alt="Somatic" title="Somatic"
        href="https://www.ebi.ac.uk/ols/ontologies/clo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0002371"
        coords="202,416,34" shape="circle">
        <!-- Cancer -->
        <area target="_blank" alt="Cancer" title="Cancer"
        href="https://www.ebi.ac.uk/ols/ontologies/clo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBI_0001906"
        coords="332,332,37" shape="circle">
        <!-- Purchased Biosample -->
        <area target="" alt="Purchased Biosample" title="Purchased Biosample"
        href=""
        coords="199,237,53" shape="circle">
        <!-- Healthy Tissue -->
        <area target="" alt="Healthy Tissue" title="Healthy Tissue"
        href=""
        coords="524,368,37" shape="circle">
        <!-- Diseased Tissue -->
        <area target="" alt="Diseased Tissue" title="Diseased Tissue"
        href=""
        coords="713,262,49" shape="circle">
        <!-- Primary Cell Line -->
        <area target="" alt="Primary Cell Line" title="Primary Cell Line"
        href=""
        coords="554,236,53" shape="circle">
        <!-- Sample -->
        <area target="" alt="Sample" title="Sample"
        href=""
        coords="377,81,68" shape="circle">
      </map>
    </div>
  </div>

  <a name="samples"></a>
  <div class="row">
    <div class="col-md-12">
      <div class="cirm-page-title back-dark-blue text-center">
        <h5 class="cirm-page-title-text">Sample Processing</h5>
      </div>
    </div>
  </div>
  <div class="row">
    <div class="col-md-12 text-center">
      <!-- Image Map Generated by http://www.image-map.net/ -->
      <img src="images/SampleProcessing.png" usemap="#sample-processing-map">

      <map name="sample-processing-map">
        <!-- Sample Processing -->
        <area target="" alt="Sample Processing" title="Sample Processing"
        href=""
        coords="277,5,498,65" shape="rect">
        <!-- Stem Cell Differentiation -->
        <area target=""
        alt="Stem cell differentiation involves the changing of a cell to a more specialized cell type, involving a switch from proliferation to specialization. This involves a succession of alterations in cell morphology, membrane potential, metabolic activity and responsiveness to certain signals. Differentiation leads to the commitment of a cell to developmental lineages and the acquisition of specific functions of committed cells depending upon the tissue in which they will finally reside. Stem cell differentiation is tightly regulated by signaling pathways and modifications in gene expression."
        title="Stem cell differentiation involves the changing of a cell to a more specialized cell type, involving a switch from proliferation to specialization. This involves a succession of alterations in cell morphology, membrane potential, metabolic activity and responsiveness to certain signals. Differentiation leads to the commitment of a cell to developmental lineages and the acquisition of specific functions of committed cells depending upon the tissue in which they will finally reside. Stem cell differentiation is tightly regulated by signaling pathways and modifications in gene expression."
        href="https://www.tocris.com/cell-biology/stem-cell-differentiation"
        coords="10,158,145,226" shape="rect">
        <!-- Mesoderm -->
        <area target="_blank" alt="Mesoderm" title="Mesoderm"
        href="https://www.ebi.ac.uk/ols/ontologies/ncro/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCRO_0000113"
        coords="103,292,222,353" shape="rect">
        <!-- Heart Muscle -->
        <area target="" alt="Heart Muscle" title="Heart Muscle"
        href=""
        coords="113,387,212,438" shape="rect">
        <!-- Red Blood Cell -->
        <area target="" alt="Red Blood Cell" title="Red Blood Cell"
        href=""
        coords="238,387,345,439" shape="rect">
        <!-- Ectoderm -->
        <area target="_blank" alt="Ectoderm" title="Ectoderm"
        href="https://www.ebi.ac.uk/ols/ontologies/ncro/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCRO_0000111"
        coords="105,464,221,519" shape="rect">
        <!-- Skin -->
        <area target="" alt="Skin" title="Skin"
        href=""
        coords="114,550,211,590" shape="rect">
        <!-- Neuron -->
        <area target="" alt="Neuron" title="Neuron"
        href=""
        coords="237,550,317,591" shape="rect">
        <!-- Endoderm -->
        <area target="_blank" alt="Endoderm" title="Endoderm"
        href="https://www.ebi.ac.uk/ols/ontologies/ncro/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCRO_0000112"
        coords="103,619,221,676" shape="rect">
        <!-- Lung -->
        <area target="" alt="Lung" title="Lung"
        href=""
        coords="114,703,210,743" shape="rect">
        <!-- Pancreas -->
        <area target="" alt="Pancreas" title="Pancreas"
        href=""
        coords="238,703,326,743" shape="rect">
        <!-- Gene Modification -->
        <area target="_blank" alt="Gene Modification" title="Gene Modification"
        href="https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C126104"
        coords="210,158,337,227" shape="rect">
        <!-- Organoids -->
        <area target=""
        alt="Organoids grown from stem cells - cells that can divide indefinitely and produce different types of cells as part of their progeny. Scientists have learned how to create the right environment for the stem cells so they can follow their own genetic instructions to self-organize, forming tiny structures that resemble miniature organs composed of many cell types. Organoids can range in size from less than the width of a hair to five millimeters."
        title="Organoids grow from stem cellsâ€”cells that can divide indefinitely and produce different types of cells as part of their progeny. Scientists have learned how to create the right environment for the stem cells so they can follow their own genetic instructions to self-organize, forming tiny structures that resemble miniature organs composed of many cell types. Organoids can range in size from less than the width of a hair to five millimeters."
        href="https://hsci.harvard.edu/organoids"
        coords="429,159,537,227" shape="rect">
        <!-- Pluripotent Stem Cell -->
        <area target=""
        alt="Organoids derived from ESCs and iPSCs undergo directed differentiation towards the desired germ lineage, eventually generating floating spheroids that are subsequently embedded in ECM to initiate organoid culture."
        title="Organoids derived from ESCs and iPSCs undergo directed differentiation towards the desired germ lineage, eventually generating floating spheroids that are subsequently embedded in ECM to initiate organoid culture."
        href="https://www.nature.com/articles/ncb3312"
        coords="363,263,472,335" shape="rect">
        <!-- Primary Tissue -->
        <area target=""
        alt="Organoids can be generated from primary tissue that is dissociated into functional sub-tissue units containing stem cells. These are further digested into single cells and FACS sorted to enrich for stem cells. Both the functional units and single stem cells can give rise to organoids under the appropriate culture conditions."
        title="Organoids can be generated from primary tissue that is dissociated into functional sub-tissue units containing stem cells. These are further digested into single cells and FACS sorted to enrich for stem cells. Both the functional units and single stem cells can give rise to organoids under the appropriate culture conditions."
        href="https://www.nature.com/articles/ncb3312"
        coords="495,264,604,334" shape="rect">
        <!-- iPSC Reprogramming -->
        <area target="" alt="IPSC Reprogramming" title="IPSC Reprogramming"
        href=""
        coords="574,158,767,227" shape="rect">
        <!-- Integrating Method -->
        <area target=""
        alt="Overall, integrating viral vectors efficiently and reliably generate iPSCs, but they have several safety concerns. First, they require the use of potentially harmful viral particles that express oncogenes, such as Myc. Viral vectors also have the largest genomic footprint of iPSC generating methods due to the risk of insertional mutagenesis. Random integration also creates heterogeneous iPSC cell lines, which can complicate comparisons made between lines. Incomplete silencing of transgenes is a concern as well, and reactivation of Myc or other oncogenes after differentiation has been linked to tumor formation in iPSC-derived and iPSC-transplanted mice. Cre-deletable or Tet-inducible lentiviruses address some of these concerns, but overall integrating viral systems currently lack the safety required for translational use."
        title="Overall, integrating viral vectors efficiently and reliably generate iPSCs, but they have several safety concerns. First, they require the use of potentially harmful viral particles that express oncogenes, such as Myc. Viral vectors also have the largest genomic footprint of iPSC generating methods due to the risk of insertional mutagenesis. Random integration also creates heterogeneous iPSC cell lines, which can complicate comparisons made between lines. Incomplete silencing of transgenes is a concern as well, and reactivation of Myc or other oncogenes after differentiation has been linked to tumor formation in iPSC-derived and iPSC-transplanted mice. Cre-deletable or Tet-inducible lentiviruses address some of these concerns, but overall integrating viral systems currently lack the safety required for translational use."
        href="https://blog.addgene.org/delivery-methods-for-generating-ipscs"
        coords="574,158,767,227" shape="rect">
        <!-- Conditional -->
        <area target=""
        alt="Doxycycline-inducible lentivirus are capable of expressing transcription factors (Oct4, Klf4, Sox2 or c-Myc) under the control of the doxycycline (Dox) inducible tetO operator when transduced into cells."
        title="Doxycycline-inducible lentivirus are capable of expressing transcription factors (Oct4, Klf4, Sox2 or c-Myc) under the control of the doxycycline (Dox) inducible tetO operator when transduced into cells."
        href="https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma/Bulletin/1/72397-stemgent-brochure-final.pdf"
        coords="352,502,451,550" shape="rect">
        <!-- Silenced -->
        <area target=""
        alt="Integrating viral vectors that allow delivery of genes into the genome of divinding cells and are usually silenced in mature cells to prevent tumor formation caused by continuous upregulation of transduced oncogenes. Ex. Retrovirus, Lentivirus"
        title="Integrating viral vectors that allow delivery of genes into the genome of divinding cells and are usually silenced in mature cells to prevent tumor formation caused by continuous upregulation of transduced oncogenes. Ex. Retrovirus, Lentivirus"
        href="https://www.sciencedirect.com/science/article/pii/B9780128018880000205"
        coords="458,501,536,551" shape="rect">
        <!-- Removable -->
        <area target=""
        alt="The integrated transgene is removed from the donor genome following completion of the reprogramming process. Ex: LoxP, piggyBac"
        title="The integrated transgene is removed from the donor genome following completion of the reprogramming process. Ex: LoxP, piggyBac"
        href="https://www.sciencedirect.com/science/article/pii/B9780128018880000205"
        coords="545,503,634,551" shape="rect">
        <!-- Non-Integrating Method -->
        <area target=""
        alt="Non-integrating methods have a smaller genetic footprint compared to integrating approaches. These methods eliminate the risk of insertional mutagenesis, the presence of a genetic scar, and incomplete silencing of transgenes. Overall, non-integrating approaches are safer than integrating methods, with RNA and protein delivery considered the safest since there's minimal risk of lingering expression of reprogramming factors."
        title="Non-integrating methods have a smaller genetic footprint compared to integrating approaches. These methods eliminate the risk of insertional mutagenesis, the presence of a genetic scar, and incomplete silencing of transgenes. Overall, non-integrating approaches are safer than integrating methods, with RNA and protein delivery considered the safest since there's minimal risk of lingering expression of reprogramming factors."
        href="https://blog.addgene.org/delivery-methods-for-generating-ipscs"
        coords="486,597,643,668" shape="rect">
        <!-- DNA -->
        <area target="" alt="DNA" title="DNA"
        href=""
        coords="468,742,546,789" shape="rect">
        <!-- Minicircle -->
        <area target=""
        alt="Minicircles are like mini-plasmids. They contain only a eukaryotic promoter and the cDNA(s) to be expressed and they don't integrate or replicate. Their small size leads to higher transfection efficiencies and they tend to express for longer periods of time than traditional plasmids due to lower activation of DNA-silencing mechanisms."
        title="Minicircles are like mini-plasmids. They contain only a eukaryotic promoter and the cDNA(s) to be expressed and they don't integrate or replicate. Their small size leads to higher transfection efficiencies and they tend to express for longer periods of time than traditional plasmids due to lower activation of DNA-silencing mechanisms."
        href="https://blog.addgene.org/delivery-methods-for-generating-ipscs"
        coords="86,828,163,876" shape="rect">
        <!-- Episomal -->
        <area target=""
        alt="Non-integrating reprogramming method that generates transgene-free and vector-free cells."
        title="Non-integrating reprogramming method that generates transgene-free and vector-free cells."
        href="https://blog.addgene.org/delivery-methods-for-generating-ipscs"
        coords="180,828,261,876" shape="rect">
        <!-- Plasmids -->
        <area target=""
        alt="Generating iPSCs with plasmid-based expression requires serial transfection of 1 or 2 plasmids that express the reprogramming factors of interest. The advantages of this method are that it's relatively simple to implement and doesn't require time-consuming production of virus. In theory this is a non-integrating approach; however, in practice integration can occur."
        title="Generating iPSCs with plasmid-based expression requires serial transfection of 1 or 2 plasmids that express the reprogramming factors of interest. The advantages of this method are that it's relatively simple to implement and doesn't require time-consuming production of virus. In theory this is a non-integrating approach; however, in practice integration can occur."
        href="https://blog.addgene.org/delivery-methods-for-generating-ipscs"
        coords="276,827,355,877" shape="rect">
        <!-- Sendai Virus -->
        <area target=""
        alt="Non-integrating reprogramming method that generates transgene-free and vector-free cells. Sendai virus is a single stranded, negative sense RNA virus. It's a member of the Paramyxoviridae family of viruses, which also includes measles and mumps. Sendai transduces a wide range of cell types and replicates in the cytoplasm independent of the cell cycle. A challenge of using Sendai is that since it's replication competent, it's difficult to eliminate the virus from all cells, even after many passages."
        title="Non-integrating reprogramming method that generates transgene-free and vector-free cells. Sendai virus is a single stranded, negative sense RNA virus. It's a member of the Paramyxoviridae family of viruses, which also includes measles and mumps. Sendai transduces a wide range of cell types and replicates in the cytoplasm independent of the cell cycle. A challenge of using Sendai is that since it's replication competent, it's difficult to eliminate the virus from all cells, even after many passages."
        href="https://blog.addgene.org/delivery-methods-for-generating-ipscs"
        coords="372,828,450,877" shape="rect">
        <!-- Adenovirus -->
        <area target=""
        alt="Non-integrating reprogramming method that generates transgene-free and vector-free cells.  Adenoviral vectors infect dividing and nondividing cells and have an ~8kb packaging capacity. With this packaging capacity, reprogramming factors can be delivered either as a single polycistronic transgene or with four different adenoviruses, each expressing one factor. These vectors don't integrate into the genome and are instead lost by dilution via cell division. A drawback to this approach is it has lower levels of efficiency at generating iPSCs, usually several orders of magnitude lower than retroviruses; however, because they are less likely to cause insertional mutagenesis, adenoviral vectors are considered a safer way to express reprogramming factors for therapeutic applications."
        title="Non-integrating reprogramming method that generates transgene-free and vector-free cells.  Adenoviral vectors infect dividing and nondividing cells and have an ~8kb packaging capacity. With this packaging capacity, reprogramming factors can be delivered either as a single polycistronic transgene or with four different adenoviruses, each expressing one factor. These vectors don't integrate into the genome and are instead lost by dilution via cell division. A drawback to this approach is it has lower levels of efficiency at generating iPSCs, usually several orders of magnitude lower than retroviruses; however, because they are less likely to cause insertional mutagenesis, adenoviral vectors are considered a safer way to express reprogramming factors for therapeutic applications."
        href="https://blog.addgene.org/delivery-methods-for-generating-ipscs"
        coords="467,829,547,877" shape="rect">
        <!-- Non-DNA -->
        <area target="" alt="Non-DNA" title="Non-DNA"
        href=""
        coords="450,934,547,981" shape="rect">
        <!-- miRNA -->
        <area target=""
        alt="This protocol requires four serial transfections of the miRNA reprogramming cocktail every 48 hr. The efficiency of miRNA-mediated reprogramming is approximately ~0.002-0.01%, depending on the cell type. Similar to other non-DNA-mediated reprogramming strategies, the miRNA technique requires no screening to exclude exogenous DNA sequences that integrate into iPSC clones, but the requirement of multiple miRNAs and multiple transfections renders the strategy more complicated for reprogramming."
        title="This protocol requires four serial transfections of the miRNA reprogramming cocktail every 48 hr. The efficiency of miRNA-mediated reprogramming is approximately ~0.002-0.01%, depending on the cell type. Similar to other non-DNA-mediated reprogramming strategies, the miRNA technique requires no screening to exclude exogenous DNA sequences that integrate into iPSC clones, but the requirement of multiple miRNAs and multiple transfections renders the strategy more complicated for reprogramming."
        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4176696/"
        coords="170,1028,250,1076" shape="rect">
        <!-- Small Molecules -->
        <area target=""
        alt="Chemical approaches can be widely applied to manipulate cell fates and states, including pluripotent reprogramming, directed differentiation, and lineage reprogramming."
        title="Chemical approaches can be widely applied to manipulate cell fates and states, including pluripotent reprogramming, directed differentiation, and lineage reprogramming."
        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5413596/"
        coords="267,1029,345,1076" shape="rect">
        <!-- mRNAs -->
        <area target=""
        alt="This method uses in vitro transcription (IVT) reactions utilizing PCR-amplified templates that encode the OSKM or OSKM plus Lin28 reprogramming factors."
        title="This method uses in vitro transcription (IVT) reactions utilizing PCR-amplified templates that encode the OSKM or OSKM plus Lin28 reprogramming factors."
        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4176696/"
        coords="363,1029,432,1076" shape="rect">
        <!-- Recombinant Proteins -->
        <area target=""
        alt="The delivery of reprogramming factors as proteins represents an alternative way to avoid the introduction of exogenous transgene into donor cells."
        title="The delivery of reprogramming factors as proteins represents an alternative way to avoid the introduction of exogenous transgene into donor cells."
        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4176696/"
        coords="449,1029,547,1077" shape="rect">
      </map>
    </div>
  </div>

  <a name="samples"></a>
  <div class="row">
    <div class="col-md-12">
      <div class="cirm-page-title back-dark-blue text-center">
        <h5 class="cirm-page-title-text">Assays</h5>
      </div>
    </div>
  </div>
  <div class="row">
    <div class="col-md-12 text-center">
      <!-- Image Map Generated by http://www.image-map.net/ -->
      <img src="images/Assays-3.png" usemap="#assay-img-map">

      <map name="assay-img-map">
        <!-- Transcriptomics -->
        <area target="" alt="Transcriptomics" title="Transcriptomics"
        href=""
        coords="5,133,141,201" shape="rect">
        <!-- RNA-seq -->
        <area target="" alt="RNA-seq" title="RNA-seq"
        href=""
        coords="25,362,121,412" shape="rect">
        <!-- Bulk RNA -->
        <area target="_blank" alt="Bulk RNA"
        title="Bulk RNA" href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0008896"
        coords="23,467,123,518" shape="rect">
        <!-- Single Cell RNA -->
        <area target="_blank" alt="Single Cell RNA" title="Single Cell RNA"
        href="https://www.ebi.ac.uk/ols/ontologies/obi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBI_0002631"
        coords="216,468,313,516" shape="rect">
        <!-- Fluidigm/C1 -->
        <area target="_blank" alt="Fluidigm/C1" title="Fluidigm/C1"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0010047"
        coords="7,677,103,725" shape="rect">
        <!-- 10X -->
        <area target="_blank" alt="10X" title="10X"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0008995"
        coords="110,678,209,726" shape="rect">
        <!-- Drop-seq -->
        <area target="_blank" alt="Drop-seq" title="Drop-seq"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0008722"
        coords="214,678,311,725" shape="rect">
        <!-- sc-ATAC-seq -->
        <area target="_blank" alt="sc-ATAC-seq" title="sc-ATAC-seq"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0008925"
        coords="319,678,417,725" shape="rect">
        <!-- Smart-seq -->
        <area target="_blank" alt="Smart-seq" title="Smart-seq"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0008930"
        coords="425,678,521,727" shape="rect">
        <!-- Frac-seq -->
        <area target="_blank" alt="Frac-seq" title="Frac-seq"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0010035"
        coords="139,362,236,411" shape="rect">
        <!-- Capture-seq -->
        <area target="_blank"
        alt="CaptureSeq is a targeted RNA sequencing method that is able to provide higher sequencing coverage for selected regions of the genome. This method follows the TruSeq RNA sample preparation protocol, in which mRNA is first isolated from total RNA by poly(A) selection and then fragmented. Double-stranded cDNA copies of the fragments are generated using reverse transcriptase and then ligated to p5 and p7 adapters. Next, these cDNA library fragments are amplified by the polymerase chain reaction (PCR). To increase specificity, custom capture probes are hybridized to the cDNA and bound to an array while other transcripts are washed away prior to PCR amplification. This process leaves the targeted fragments that are ready for sequencing."
        title="CaptureSeq is a targeted RNA sequencing method that is able to provide higher sequencing coverage for selected regions of the genome. This method follows the TruSeq RNA sample preparation protocol, in which mRNA is first isolated from total RNA by poly(A) selection and then fragmented. Double-stranded cDNA copies of the fragments are generated using reverse transcriptase and then ligated to p5 and p7 adapters. Next, these cDNA library fragments are amplified by the polymerase chain reaction (PCR). To increase specificity, custom capture probes are hybridized to the cDNA and bound to an array while other transcripts are washed away prior to PCR amplification. This process leaves the targeted fragments that are ready for sequencing."
        href="https://sapac.illumina.com/science/sequencing-method-explorer/kits-and-arrays/captureseq.html"
        coords="253,362,350,410" shape="rect">
        <!-- Genomics -->
        <area target="" alt="Genomics" title="Genomics"
        href=""
        coords="205,133,342,203" shape="rect">
        <!-- WGS -->
        <area target="_blank" alt="WGS" title="WGS"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0003744"
        coords="168,259,264,307" shape="rect">
        <!-- Amplicon-seq -->
        <area target="_blank" alt="Amplicon-seq" title="Amplicon-seq"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0003747"
        coords="283,257,370,306" shape="rect">
        <!-- Other -->
        <area target="" alt="Other" title="Other" href="" coords="436,133,570,202" shape="rect">
        <!-- GUIDE-seq -->
        <area target="_blank" alt="GUIDE-seq" title="GUIDE-seq"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0008760"
        coords="455,257,551,307" shape="rect">
        <!-- Epigenomics -->
        <area target="" alt="Epigenomics" title="Epigenomics"
        href=""
        coords="636,134,770,202" shape="rect">
        <!-- Methylome -->
        <area target="" alt="Methylome" title="Methylome"
        href=""
        coords="426,363,522,411" shape="rect">
        <!-- WGBS -->
        <area target="_blank" alt="WGBS" title="WGBS"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0008985"
        coords="369,467,465,516" shape="rect">
        <!-- RRBS -->
        <area target="_blank" alt="RRBS" title="RRBS"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0008900"
        coords="483,468,579,517" shape="rect">
        <!-- Chromatin Modeling -->
        <area target="" alt="Chromatin Modeling" title="Chromatin Modeling"
        href=""
        coords="539,362,636,411" shape="rect">
        <!-- ATAC-seq -->
        <area target="_blank" alt="ATAC-seq" title="ATAC-seq"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0007045"
        coords="482,572,579,621" shape="rect">
        <!-- Hi-C -->
        <area target="_blank" alt="Hi-C" title="Hi-C"
        href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0007693"
        coords="597,574,693,620" shape="rect">
        <!-- Histone Modification -->
        <area target="" alt="Histone Modification" title="Histone Modification"
        href=""
        coords="655,362,750,411" shape="rect">
        <!-- Transcription factor ChIP-seq -->
        <area target=""
        alt="Chromatin immunoprecipitation with expectation of narrow peaks as most transcription factors."
        title="Chromatin immunoprecipitation with expectation of narrow peaks as most transcription factors." href="" coords="532,677,647,725" shape="rect">
        <!-- Histone ChIP-seq -->
        <area target=""
        alt="Chromatin immunoprecipitation with expectation of broad peaks as most modified histones."
        title="Chromatin immunoprecipitation with expectation of broad peaks as most modified histones."
        href="" coords="654,678,752,725" shape="rect">
        <!-- Assays -->
        <area target="" alt="Assays" title="Assays"
        href=""
        coords="310,9,464,87" shape="rect">
      </map>
    </div>
  </div>
</div>

<!--#include virtual="${ROOT}inc/footer.html" -->
