<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=185>
  <TR><TD ALIGN=RIGHT><A HREF="http://genome.wustl.edu/genomes/detail/gallus-gallus/"
    TARGET=_blank>
    <IMG SRC="../images/Gallus_gallus.jpg" WIDTH=170 HEIGHT=136 ALT="Chicken"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Gallus gallus</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of Bill Payne,<BR>Michigan State University<BR>
      (NHGRI <A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Red%20Jungle%20Fowl&id=2120" 
      TARGET=_blank>Press Photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The May 2006 chicken (<em>Gallus gallus</em>) v2.1 draft
assembly was produced by the
<A HREF="http://genome.wustl.edu/genomes/detail/gallus-gallus/" TARGET="_blank">
Genome Sequencing Center</A> at the Washington University School of
Medicine in St. Louis, MO, USA (WUSTL). For more information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/238058/" TARGET="_blank">
Gallus_gallus-2.1</A> in the NCBI Assembly database.</P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic supercontig, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
chicken genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>chrZ</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of chromosome Z</TD>
    </TR>
    <TR><TD VALIGN=Top>chr3:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of chr 3</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr3:23000+4000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 3 that spans 4000 bases, starting with position 23,000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>BX950328</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number BX950328</TD>
    </TR>
    <TR><TD VALIGN=Top>BU335953</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		BU335953 on chr Z</TD>
    </TR>
    <TR><TD VALIGN=Top>chr27_1.383</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of genome containing Genscan
		gene prediction chr27_1.383</TD>
    </TR> 
    <TR><TD VALIGN=Top>CWNT-8C</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of genome with gene symbol CWNT-8C</TD>
    </TR>
    <TR>
	<TD VALIGN=Top>NM_204735</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays the region of genome with RefSeq identifier
		NM_204735</TD>
    </TR>
    <TR><TD VALIGN=Top>374180</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays the region of genome with Entrez Gene
		identifier 374180</TD>
    </TR>
<!--
    <TR><TD VALIGN=Top>ENSGALT00000023622</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays the region of genome with Ensembl identifier 
	ENSGALT00000023622. NOTE: Due to space constraints, the Genome Browser 
	displays abridged Ensembl identifiers, but the full ID name must be
	used in searches. For example, the ENSGALT00000023622 identifier
	displays as ENSGAL23622 in the Genome Browser.</TD>
    </TR>
-->
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene mRNA</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zhang</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zhang</TD>
    </TR>
    <TR><TD VALIGN=Top>Taguchi,T.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author T. Taguchi</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Taguchi T</em>, internally it
	uses the format <em>Taguchi,T.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
In this assembly, 198,000 additional reads covering all contig ends and regions 
of low quality have been added to the original assembly's 6.6X coverage.
Approximately 95% of the sequence has been anchored to chromosomes, 
which include autosomes 1-24, 26-28, and 32, and sex chromosomes W and Z. 
(In contrast to mammals, the female chicken is heterogametic (ZW) and the 
male is homogametic (ZZ).) 
The remaining unanchored contigs that could be localized to a chromosome 
have been concatenated into the virtual 
chromosomes &quot;chr*_random&quot;, separated by gaps of 10,000 bp. 
Unanchored contigs that could not be localized to a chromosome 
have been concatenated into the virtual 
chromosome &quot;chrUn_random&quot;, separated by gaps of 100 bp to reduce the 
total size of chrUn_random. 
The chicken 
mitochondrial sequence is also available as the virtual chromosome 
&quot;chrM&quot;. 
</P>
<P>
This assembly has the following statistics.
<UL>
<LI>ultracontigs: N50 = 15.5 Mb (n=19); longest ultracontig = 80.3 Mb (chr3)
<LI>supercontigs: N50 = 11.1 Mb (n=26); longest supercontig = 52 Mb (chr2)  
<LI>contigs: N50 = 45 kb (n=5863); longest contig = 625 kb
</UL>
In this context, a contig is defined
as a contiguous stretch of DNA, a supercontig is a set of ordered/oriented 
contigs linked by read-pairing data, and an ultracontig is an 
ordered/oriented list of supercontigs. The N50 size is the length such that 
50% of the assembled genome lies in blocks of the N50 size or longer.
</P>
<P>
Although centromere positions are indicated on this assembly, little is known 
of their exact sequence.  The centromeres of
18 chromosomes were tentatively localized based on FISH hybridization
using BAC clones, genetic markers flanking the centromeres in
coordination with mapping gaps in the physical map, repetitive
sequence content, and analysis of proximity to the constrictions of
the mitotic metaphase chromosomes. For more information on the process
used to create the chromosomal sequences and assign centromere locations, see 
the WUSTL <em>Gallus gallus</em> 
<A HREF="http://genome.wustl.edu/genomes/detail/gallus-gallus/" TARGET=_blank>assembly release
notes</A>.</P>
<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/galGal3/">FTP server</A> or 
the 
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#chicken">Downloads</A> 
page. These data have 
<A HREF="../goldenPath/credits.html#chicken_use">specific conditions for 
use</A>. 
The chicken browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#chicken_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
