<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=265 HEIGHT=196>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://www.broadinstitute.org/science/projects/mammals-models/vertebrates-invertebrates/aplysia/aplysia-genome-sequencing-project"
    TARGET=_blank>
    <IMG SRC="../images/Aplysia_californica.jpg" WIDTH=250 HEIGHT=181 ALT="Sea Hare"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Aplysia californica</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of Genny Anderson.
    </FONT>
  </TD></TR>
</TABLE>

<P>
The Sept. 2008 <em>Aplysia californica</em> draft assembly 
(Broad Institute version Aplcal2.0) was produced by the
<A HREF="http://www.broadinstitute.org" TARGET=_blank>Broad Institute</A>,
Cambridge, MA, USA. For more information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/3458/" TARGET="_blank">
Aplcal2.0</A> in the NCBI Assembly database.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
sea hare genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>scaffold_0</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of scaffold 0</TD>
    </TR>
    <TR><TD VALIGN=Top>scaffold_0:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of scaffold 0</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>scaffold_0:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of scaffold 0 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>AF372526</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number AF372526</TD>
    </TR>
    <TR><TD VALIGN=Top>EB241582</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		EB241582 on scaffold 0</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zhang</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zhang</TD>
    </TR>
    <TR><TD VALIGN=Top>Taguchi,T.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author T. Taguchi</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Taguchi T</em>, internally it
	uses the format <em>Taguchi,T.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
The sea hare whole genome shotgun assembly was produced by the Broad
Institute using the Arachne assembler. The genome has been sequenced to 9.86x coverage. Approximately 74% of the sequence has been anchored to 8766 supercontigs
ranging in size from 1.7 Mb down to 5 Kb. 13.9 Mb reads created 9.5 Gb of base 
calls, 7.0 Gb of which were incorporated into assembly supercontigs resulting
in a total genome assembly size of approximately 712 Mb.
The mitochondrial sequence is also available as the virtual chromosome 
&quot;chrM&quot;. 
For more information on the assembly, see the Broad Institute
<A HREF="http://www.broadinstitute.org/science/projects/mammals-models/vertebrates-invertebrates/aplysia/aplysia-genome-sequencing-project"
TARGET=_blank>Aplysia Genome Project</A> page.
</P>
<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/aplCal1/">FTP
server</A> or the 
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#seahare">Downloads</A> 
page. 
These data have <A HREF="../goldenPath/credits.html#seahare_use">specific 
conditions for use</A>. 
</P>
<P>
The sea hare browser annotation tracks were generated by UCSC and 
its collaborators. See the 
<A HREF="/goldenPath/credits.html#seahare_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
