<H2>Description</H2>
<P>
This track shows the draft assembly of the $organism genome.  
This version was assembled from 2.05 million whole genome shotgun (WGS) reads. 
The Phusion (Mullikin, J.) assembler was used to create contigs based on 
overlap information; supercontigs were then created using read 
pair information to cross gaps. These supercontigs were assembled into mapped 
ultracontigs based on FPC fingerprint mapping with material from previously 
finished clones used to bridge gaps. The sequence data was used to choose 
primers along major ultracontigs, from which a genetic map was constructed 
yielding six chromosomes. The genetic mapping data, along with data from 
1:1 elegans:briggsae orthologs, were used to refine ultracontigs and to order 
and orient the sequences along the <em>C. briggsae</em> chromosomes. </P>
<P>
In dense mode, this track depicts the contigs that make up the 
currently viewed scaffold. 
Contig boundaries are distinguished by the use of alternating gold and brown 
coloration. Where gaps
exist between contigs, spaces are shown between the gold and brown
blocks.  The relative order and orientation of the contigs
within a scaffold is always known; therefore, a line is drawn in the graphical
display to bridge the blocks.</P>
<P>
All components within this track are of fragment type &quot;W&quot;: 
whole genome shotgun contig. </P>

