<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=297>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://genome.wustl.edu/genomes/detail/pongo-abelii/"
    TARGET=_blank ALIGN=RIGHT >
    <IMG SRC="../images/Pongo_pygmaeus_abelii.jpg" WIDTH=282 HEIGHT=200 ALT="Orangutan"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Pongo pygmaeus abelii</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of NHGRI
      (<A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Primates&id=79121" 
      TARGET=_blank>press photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
<B>UCSC Genome Browser assembly ID:</B> ponAbe2<BR>
<B>Sequencing/Assembly provider ID:</B> 
<A HREF="http://genome.wustl.edu/genomes/detail/pongo-abelii/" 
TARGET=_blank>WUSTL Pongo_abelii-2.0.2</A><BR>
<B>Assembly date:</B> Jul. 2007<BR>
<B>NCBI Genome ID:</B> <A HREF="https://www.ncbi.nlm.nih.gov/genome/325"
TARGET="_blank">325</A> (Pongo abelii)<BR>
<B>NCBI Assembly ID:</B> <A HREF="https://www.ncbi.nlm.nih.gov/genome/assembly/6678/"
TARGET="_blank">6678</A> (P_pygmaeus_2.0.2)<BR>
<B>NCBI BioProject ID:</B></I> <A HREF="https://www.ncbi.nlm.nih.gov/bioproject/20869"
TARGET="_blank">20869</A>
</P>

<HR>
<P>
<B>Search the assembly:</B>
<UL>
  <LI>
  <B>By position or search term: </B> Use the &quot;position or search term&quot;
  box to find areas of the genome associated with many different attributes, such
  as a specific chromosomal coordinate range; mRNA, EST, or STS marker names; or
  keywords from the GenBank description of an mRNA.
  <A HREF="../goldenPath/help/query.html">More information</A>, including sample
  queries.
  <LI>
  <B>By gene name: </B> Type a gene name into the &quot;search term&quot; box,
  choose your gene from the drop-down list, then press &quot;submit&quot; to go
  directly to the assembly location associated with that gene.
  <A HREF="../goldenPath/help/geneSearchBox.html">More information</A>.
  <LI>
  <B>By track type: </B> Click the &quot;track search&quot; button
  to find Genome Browser tracks that match specific selection criteria.
  <A HREF="../goldenPath/help/trackSearch.html">More information</A>.
</UL>
</P>
<HR>
<P>
<B>Download sequence and annotation data:</B>
<UL>
  <LI><A HREF="../goldenPath/help/ftp.html">Using rsync</A> (recommended)
  <LI><A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/ponAbe2/">Using FTP</A>
  <LI><A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#orangutan">Using HTTP</A>
  <LI><A HREF="../goldenPath/credits.html#orangutan_use">Data use conditions and
restrictions</A>
  <LI><A HREF="../goldenPath/credits.html#orangutan_credits">Acknowledgments</A>
</UL>
</P>
<HR>

<H3>Assembly details</H3>
<P>
The <em>Pongo pygmaeus abelii</em> assembly (6X coverage) consists of whole 
genome shotgun data from primary donor-derived reads obtained from 
&quot;Susie&quot;, a female Sumatran orangutan housed at the Gladys Porter Zoo 
in Brownsville, TX.
The combined sequence reads were assembled using the PCAP software (Huang, 
2003) and filtered for all known non-orangutan sequence 
contaminants using stringent parameters derived by eliminating detectable
global mis-assemblies larger than 50 kb. Mis-assemblies (interchromosomal 
cross-overs) were determined by aligning the orangutan genome against the 
human genome. 
Sequences were obtained from plasmids, fosmids, and BAC-end
sequences from CHORI-276, also obtained from &quot;Susie&quot;. A fingerprint
map with a target of 12X clone coverage is currently in progress.
</P>
<P>
The assembly data were aligned against the human genome at UCSC
(B. Raney) using BLASTZ (Schwartz, 2003) to align and score
non-repetitive orangutan regions against repeat-masked human
sequence. The alignment chains differentiated between orthologous and
paralogous alignments (Kent, 2003), and only &quot;reciprocal best&quot;
alignments were retained in the final alignment set. The orangutan AGP files
were generated from these alignments in a manner similar to that
already described (The Chimpanzee Genome Sequencing and Analysis
Consortium, 2005). Documented inversions based primarily on FISH data
(Rocchi, personal communication) as well as inversions suggested by the
assembly and supported by additional mapping data (e.g. fosmid end
sequences against the human assembly, (Chen and Eichler, personal
communication)) were introduced. The alignments to human chromosome 2 were
separated into organutan chromosomes 2A and 2B.  Finally, 78
finished BAC (CHORI-273) clones were integrated into the final
chromosomal sequences.
</P>
<P>
Of the 3.09 Gb of total sequence, 3.08 Gb are ordered and oriented along
the chromosomes. Gap sizes between supercontigs were estimated based on
their size in human, with a maximum gap size of 30 kb allowed.
Click the &quot;Sequences&quot; link above for a list of the chromosomes
available in this assembly. The mitochondrial sequence is also available as 
the virtual chromosome &quot;chrM&quot;. For more information about this
assembly, see the WUSTL 
<A HREF="http://genome.wustl.edu/genomes/detail/pongo-abelii/"
TARGET=_blank><em>Pongo abelii</em></A> web page.</P>
</P>

<HR>
<H3>GenBank pipeline details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.

<HR>
<H3>References</H3>
<P>
Locke DP, Hillier LW, Warren WC, Worley KC, Nazareth LV, Muzny DM, Yang SP, Wang Z,
Chinwalla AT, Minx P <em>et al</em>. <A HREF="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3060778/"
target="_blank">Comparative and demographic analysis of orang-utan genomes</a>.
<em>Nature</em>. 2011 Jan 27;469(7331):529-33. 
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20010809" target="_blank">20010809</a>
</P>
