<H3>Description</H3>

<P>This column shows the expression of genes in various phases 
of the fly life cycle.  The experiments were replicated two times each.  
<P>
By default, the column shows the ratio of gene expression at a specific
life cycle phase to expression of the gene in mixed egg-to-adult
cultures. A gene that is more highly expressed in a life cycle phase is 
colored red, and a less expressed gene is shown in green. 
The values are colored on a logarithmic scale. This coloring is standard, 
but is the opposite of what an inexperienced user 
might expect: in this case, red means go and green means stop!  
Black indicates that a gene is neither over- nor under-expressed in the life
cycle phase. Uncolored boxes (white on most browsers) represent missing data.  

<H3>Column Configuration</H3>
Several column configuration options are available on the Configuration page, 
which can be accessed via the <em>configure</em> button in the top menu.
<UL>
<LI>
<B>Absolute expression values:</B> The display can be configured to show 
absolute expression values rather than ratio values. To toggle the display, 
select the <em>absolute</em> option from the <em>values</em> pulldown menu. 
Absolute expression values
are shown in shades of gray, with higher expression levels shown in lighter 
shades. As with ratio values, absolute values are colored on a logarithmic 
scale.</LI>
<LI>
<B>Color scheme:</B> Color-blind users can switch the coloring scheme from red/green to 
yellow/blue via the <em>Expression ratio colors</em> pulldown menu at the top
of the Configuration page.</LI>
<LI>
<B>Level of expression detail:</B> The <em>phases</em> pulldown menu
controls the amount of life cycle data displayed. By default, only the 
median values for selected life cycle phases are shown. To display the value of 
all individual experimental replicates of all phases, 
select the <em>all replicates</em> option. To view a single median value for each 
phase, choose the <em>median of replicates</em> option.</LI>
<LI>
<B>Brightness:</B> To increase or decrease the brightness of the expression colors,
edit the number in the column's <em>brightness</em> text box. Values greater 
than 1.0 increase the brightness, while those less than 1.0 dim the color. </LI>
</UL>

<H3>Methods</H3>
The data in this column were collected as described in the paper
Arbeitman MN, <em>et al</em>. (2002). 
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=12351791&dopt=Abstract" TARGET=_blank>
"Gene expression during the life cycle of Drosophila melanogaster"</A>. 
<em>Science</em> 2002 Sep 27;297(5590):2270-5.
We retrieved the data from the
<A HREF="http://genome-www5.stanford.edu/" TARGET=_blank>Stanford Microarray 
Database</A>.
<P>To associate particular genes with expression data we use a fairly simple-minded
scheme.  First we align the cDNA sequences around which the probes were designed to
the genome.  Then for each gene we pick the cDNA sequence with the alignment that most
overlaps the gene as the source for expression data for that gene.  Since many of the cDNA 
sequences and probes designed for the cDNA sequences just cover part of the gene, in most
cases different splice varients of a gene will end up mapped to the same cDNA sequence. 
</P>


<H3>Credits</H3>
Many thanks to 
<A HREF="http://genome.med.yale.edu/Lifecycle/" TARGET=_blank>
Arbeitman <em>et al</em>.</A> for making the data available.
