<H2>Description</H2>
<P>
This track shows best-in-genome  
<em>$organism</em>/<em>$o_organism</em> alignments based on the 
<em>$organism</em>/<em>$o_organism</em> alignment net, <em>i.e.</em> the 
best chain for every part of the <em>$organism</em> genome.  
The <em>$o_organism</em> sequence is from 
<A HREF="http://www.wormbase.org/" TARGET=_blank>WormBase</A>
Version cb25.agp8 ($o_date ) assembly.</P>

<H2>Display Conventions and Configuration</H2>
<P>
The track displays alignment scores and, when zoomed in to the base level, 
the alignment itself. 
In the graphical display, the shade (in dense display mode) 
or height (in full mode) of a vertical line indicates the score 
of the best chained and netted alignment at that location.  
Clicking on the track brings up detailed information about the 
blastz alignments (taken from the axt-formatted output) in the 
currently displayed range of positions.  </P>

<H2>Methods</H2>
<P>
Chains were derived from blastz alignments, using the methods
described on the chain tracks description pages, and sorted with the 
highest-scoring chains in the genome ranked first. The program
chainNet was then used to place the chains one at a time, trimming them as 
necessary to fit into sections not already covered by a higher-scoring chain. 
During this process, a natural hierarchy emerged in which a chain that filled 
a gap in a higher-scoring chain was placed underneath that chain. The program 
netSyntenic was used to fill in information about the relationship between 
higher- and lower-level chains, such as whether a lower-level
chain was syntenic or inverted relative to the higher-level chain. 
The program netClass was then used to fill in how much of the gaps and chains 
contained <em>N</em>s (sequencing gaps) in one or both species and how much
was filled with transposons inserted before and after the two organisms 
diverged.  The program <em>netToAxt</em> translates the net back 
to axt.  <em>netToAxt</em> is run with -maxGap=300.  

<H2>Credits</H2>
<P>
The chainNet, netSyntenic, and netClass programs were
developed at the University of California
Santa Cruz by Jim Kent.</P>
<P>
Blastz was developed at <A HREF="http://bio.cse.psu.edu"
TARGET=_blank>Pennsylvania State University</A> by
Minmei Hou, Scott Schwartz, Zheng Zhang, and Webb Miller with advice from
Ross Hardison.</P>
<P>
Lineage-specific repeats were identified by Arian Smit and his program 
<A HREF="http://www.repeatmasker.org" TARGET=_blank>RepeatMasker</A>.</P>
<P>
The browser display and database storage of the nets were made
by Robert Baertsch and Jim Kent.</P>

<H2>References</H2>
<P>
Kent, W.J., Baertsch, R., Hinrichs, A., Miller, W., and Haussler, D.
<A HREF="http://www.pnas.org/cgi/content/abstract/1932072100v1"
TARGET=_blank>Evolution's cauldron: Duplication, deletion, and rearrangement
in the mouse and human genomes</A>.
<em>Proc Natl Acad Sci USA</em> <B>100</B>(20), 11484-11489 (2003).</P>
<P>
Schwartz, S., Kent, W.J., Smit, A., Zhang, Z., Baertsch, R., Hardison, R.,
Haussler, D., and Miller, W.
<A HREF="http://www.genome.org/cgi/content/abstract/13/1/103"
TARGET=_blank>Human-Mouse Alignments with BLASTZ</A>.
<em>Genome Res.</em> <B>13</B>(1), 103-7 (2003).</P>
