<!-- Display image at top right of page -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=315>
  <TR><TD ALIGN=RIGHT><A HREF="https://www.hgsc.bcm.edu/content/rhesus-monkey-genome-project" TARGET=_blank>
      <IMG SRC="../images/Rhesus_macaque.jpg" WIDTH=300 HEIGHT=225 ALT="Rhesus Macaque"></A>
    </TD></TR>
  <TR><TD ALIGN=RIGHT> 
  <FONT SIZE=-1> <em>Macaca mulatta</em><BR> 
  </FONT>
  <FONT SIZE=-2>
    Photo courtesy of 
    <A HREF = "http://www1.wfubmc.edu/pathresearch/faculty/kaplan.htm" 
    TARGET = _BLANK>J.R. Kaplan</A><BR>
</FONT>
  </TD></TR>
</TABLE>

<P>
The Jan. 2005 Rhesus monkey or Rhesus macaque (<em>Macaca mulatta</em>) preliminary assembly, 
<P>
Mmul_0.1, was obtained from the Baylor College of Medicine Human Genome 
Sequencing Center (BCM HGSC). The sequencing and assembly of the 
<em>Macaca mulatta</em> genome is a project of the Macaque Genome Sequencing 
Consortium led by the Baylor College of Medicine Human Genome Sequencing 
Center, in collaboration with the J. Craig Venter Institute Joint Technology 
Center, and the Genome Sequencing Center at Washington University School of 
Medicine, St. Louis. </P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced mRNA or EST, a scaffold coordinate range, a gene identifier,
or keywords from the GenBank description of an mRNA.
The following list shows examples of valid position queries for
the rhesus genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
TARGET=_blank>User's Guide</A> for more information. 
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TD VALIGN=Top NOWRAP><B>Request:</B><BR></TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top><B>Genome Browser Response:</B><BR></TD>
    </TR>
    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>SCAFFOLD80001</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of SCAFFOLD80001 (on an unmapped genome assembly)</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>SCAFFOLD544:26,374-29,857</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays a region of SCAFFOLD544:26,374-29,857 with a cluster of Spliced ESTs</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>SCAFFOLD544:26,374+2000</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays a region of SCAFFOLD544 that spans 2000 bases, starting with position 26,374 </TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>

    <TR><TD VALIGN=Top NOWRAP>CN647194</TD>
       	<TD WIDTH=14></TD>
      	<TD VALIGN=Top>Displays EST with GenBank accession CN647194</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>AF518412</TD>
       	<TD WIDTH=14></TD>
      	<TD VALIGN=Top>Displays mRNA with GenBank accession AF518412</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>PRNP</TD>
    	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays human and chimp RefSeq gene PRNP aligned to SCAFFOLD5559</TD>
    </TR>

    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>leptin</TD>
    	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists mRNAs associated with leptin receptors </TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>pseudogene mRNA</TD>
    	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>Scinicariello,F.</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author F. Scinicariello</TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although GenBank 
			requires the search format <em>Scinicariello F</em>, internally it uses 
			the format <em>Scinicariello,F.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
Mmul_0.1 is a preliminary assembly using whole genome shotgun (WGS) reads 
from small and medium insert clones. The following assembly information was
taken from the BCM HGSC <A HREF="ftp://ftp.hgsc.bcm.tmc.edu/pub/data/Rmacaque/fasta/README_Mmul.txt" 
TARGET=_blank>README</A> that accompanies the Mmul_0.1 download data.</P>
<P>
This release was produced by assembling whole genome shotgun
reads with the Atlas genome assembly system at BCM HGSC. Several WGS 
libraries, with inserts of 2-4 kb and 10 kb, were used to produce the data. 
About 14.58 million reads were used in the assembly, representing about 
12.6 Gb of sequence and about 4.6x coverage of the (clonable) macaque
genome. Highly-repeated sequences and sequences from BAC clones were
omitted from this assembly and will be placed in a subsequent version
of the draft sequence.</P>
<P>
The Atlas assembler produces a set of contigs and scaffolds. Scaffolds 
include sequence contigs that can be ordered and
oriented with respect to each other, as well as isolated contigs that
could not be linked (single contig scaffolds or singletons). Reads
that clustered into groups of two or less were not assembled, and 
highly-repetitive reads that clustered into groups of more than 18 were 
assembled separately.  The N50 of the contigs is 10.5 kb; the N50 of the 
scaffolds is 79.7 kb. The total length of all contigs is 2.69 Gb, or 2.81 Gb
when the gaps between contigs are included.</P>
<P>
This is a draft sequence and may contain errors; therefore, users should 
exercise caution. Typical errors in draft genome sequences include
misassemblies of repeated sequences, collapses of repeated regions,
and unmerged overlaps (e.g. due to polymorphisms) creating artificial
duplications. However base accuracy in contigs is usually very high with 
most errors near the ends of contigs. </P>
<P>
To review information on the quality of this assembly, see the
<A HREF="ftp://ftp.hgsc.bcm.tmc.edu/pub/data/Rmacaque/fasta/README_Mmul.txt" 
TARGET=_blank>Mmul_0.1 README</A>. For general information about the rhesus 
monkey sequencing project, see the Baylor
<A HREF="https://www.hgsc.bcm.edu/content/rhesus-monkey-genome-project"
TARGET=_blank>Rhesus Monkey Project</A> web page.</P>
<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/rheMac1/">FTP
server</A> or the 
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#rhesus">Downloads</A> 
page. These data have specific 
<A HREF="https://www.hgsc.bcm.edu/content/conditions-use" TARGET=_blank>conditions for 
use</A>. The rheMac1 annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="../goldenPath/credits.html#rhesus_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to this release. </P>
