<H2>Description</H2>
<P>
This track summarizes the alternative splicing seen in the mRNA and
EST tracks. The blocks represent exons and the lines are the possible
splice junctions.  When the track is in full mode
and the resolution is approximately gene level the exons are laid out
such that alternative exons don't overlap each other and are easier to
see.  The more ESTs and mRNAs that contain that exon or splice
junction the darker they are drawn.
<P>
To help filter out the noise present in the EST libraries both
exons and splice junctions were filtered. ESTs and mRNAs were removed if 
the first and last exon were not within the transcription start and 
end sites for the RefSeq sequence from that region.  Only those exons and
splice junctions which have an orthologous exon or splice junction
present in the mouse transcriptome, or are present three or more
times, were kept. This process is similar to that presented in 
C.W. Sugnet, W.J. Kent, M. Ares Jr., and D. Haussler.
<A HREF="http://www-smi.stanford.edu/projects/helix/psb04/sugnet.pdf"
TARGET=_blank>Transcriptome and Genome Conservation of Alternative Splicing 
Events in Humans and Mice</A>. 

<H2>Methods</H2>
<P>
The outline for generating this track is:
<UL>

    <LI> mRNAs and ESTs were aligned to the genomic sequence using <a
    href="../cgi-bin/hgBlat">blat</a>. A near-best-in-genome filter was
    applied where only alignments with 97% identity over 90% of the
    transcript and with a score no more than 0.5% lower than the best
    score were kept.</LI>

    <LI> Genomic sequence was retrieved and used to orient ESTs and mRNAs
    using consensus splice sites, GT-&gt;AG, and the less common
    GC-&gt;AG.</LI>

    <LI> Alignments were clustered together by sequence overlap in exons. As
    new splice sites were discovered, they were entered into the graphs
    as vertices, and the exons, introns, and splice junctions
    connecting them were recorded as edges. Each graph was considered to
    be a single locus, although they may be fragments of an actual
    gene structure. The supporting mRNA and EST accessions for each
    edge were also stored.</LI>

    <LI> Truncated transcripts were extended by overlap with other
    transcripts to the next consensus splice site.  This avoided
    keeping vertexes in the graph that were not true splice sites.</LI>
</UL>
<H2>See Also:</H2>
<P>
Please note
the <A HREF="/goldenPath/$db/altGraphX/frame.html" TARGET=_blank>Frames based index view</A>
of all the alt-splice sites.
</P>
