<H2>Description</H2>
<H3>Mononucleosome control track</H3>
The control data was produced from the lightly micrococcal 
nuclease-digested paired-end sequenced genomic DNA fragments in the 
size range of 400-900 base pairs. Each feature corresponds to one end of the 
paired-end fragment with thick portion representing the sequenced 25 
base pairs of that end. Features span 147 base pairs (displayed equivalently 
to mononucleosome data).

<H3>Mononucleosome sense strand reads</H3>
Mononucleosomal fragments were obtained by micrococcal nuclease 
digestion of C. elegans lysates, DNA extraction, and sequencing of 
nucleosome-bound DNA fragments. Features represent inferred 147 base pair 
cores from fragments that mapped to the sense strand of the reference 
genome. The thick portion of each feature (50 base pairs) represents the 
sequenced part of the nucleosome-bound fragment.

<H3>Mononucleosome antisense strand reads</H3>
Mononucleosomal fragments were obtained by micrococcal nuclease 
digestion of C. elegans lysates, DNA extraction and sequencing of 
nucleosome-bound DNA fragments. Features represent inferred 147 base pair 
cores from fragments that mapped to the antisense strands. The thick 
portion of each feature (50 base pairs) represents the sequenced part of the 
core.

<H2>Display Conventions and Configuration</H2>
<P>In each of the sub-tracks, sequences starting at the same base pair were 
collapsed and color-coded from the lightest to the darkest color in 
five shades according to the following category-based scale:

<blockquote><table border="1" bordercolor="#aaaaaa" cellpadding="4">
<tbody><tr align="left">
<td><b>Number of read instances</b></td><td><b>Color shade</b></td></tr>
<tr align="left"><td>1</td><td>lightest</td></tr>
<tr align="left"><td>2</td><td>light</td></tr>
<tr align="left"><td>3-5</td><td>medium</td></tr>
<tr align="left"><td>6-10</td><td>dark</td></tr>
<tr align="left"><td>more than 10</td><td>darkest</td></tr>
</tbody></table></blockquote>

<P>Hold your mouse over individual items in each track to see the number 
of read instances starting at that base pair.</P>


<H2>Credits</H2>
The data for this track is supplied by the
<A HREF="https://med.stanford.edu/profiles/Arend_Sidow" TARGET=_blank>Sidow Lab</A>
and the
<A HREF="https://sites.stanford.edu/firelab/" TARGET=_blank>Fire Lab</A>
at the <A HREF="https://med.stanford.edu/"
TARGET=_blank>Stanford School of Medicine</A>.<BR>
Track display advice provided by
<A HREF="mailto:&#104;&#105;&#114;a&#109;&#64;&#115;&#111;&#101;
.&#117;&#99;&#115;&#99;.&#101;&#100;u">Hiram Clawson</A>,
UCSC Genome Browser Engineering.

<H2>References</H2>
<P>
Valouev A, Ichikawa J, Tonthat T, Stuart J, Ranade S, Peckham H, Zeng K, Malek JA, Costa G, McKernan
K <em>et al</em>.
<A HREF="https://genome.cshlp.org/content/18/7/1051.short"
TARGET=_blank>A high-resolution, nucleosome position map of
C. elegans reveals a lack of universal
sequence-dictated positioning</A>.
<em>Genome Res</em>. 2008 Jul;18(7):1051-63.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18477713" target="_blank">18477713</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2493394" target="_blank">PMC2493394</a>
</p>

<P>
Supported oligo ligation detection (SOLiD) sequencing technology from
<A HREF="https://www.thermofisher.com/us/en/home/life-science/sequencing/next-generation-sequencing.html"
TARGET=_blank>Applied Biosystems</A>.
</P>
