<h2>Description</h2>

<p>
Most proteins are composed of one or more conserved functional regions called
domains. This track shows the high-quality, manually curated
<a href="http://pfam.xfam.org" target="_blank">Pfam-A</a>
domains found in transcripts from the NCBI Genes track.
</p>

<h2>Display Conventions and Configuration</h2>

<p>
This track follows the display conventions for
<a href="../goldenPath/help/hgTracksHelp.html#GeneDisplay">gene tracks</a>.
</p>

<h2>Methods</h2>

<p>
The amino acid sequences from the NCBI Genes 
are submitted to the set of Pfam-A HMMs, which annotate regions within the
predicted peptide that are recognizable as Pfam protein domains. These regions
are then mapped to the transcripts themselves using the
<a href="http://hgdownload.soe.ucsc.edu/admin/exe/" target="_blank">pslMap utility</a>.
</p>

<h2>Credits</h2>

<p>
pslMap was written by Mark Diekhans at UCSC.
</p>

<h2>References</h2>
<p>
Finn RD, Mistry J, Tate J, Coggill P, Heger A, Pollington JE, Gavin OL, Gunasekaran P, Ceric G,
Forslund K <em>et al</em>.
<a href="http://nar.oxfordjournals.org/content/38/suppl_1/D211.long" target="_blank">
The Pfam protein families database</a>.
<em>Nucleic Acids Res</em>. 2010 Jan;38(Database issue):D211-22.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19920124" target="_blank">19920124</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2808889/" target="_blank">PMC2808889</a>
</p>
