<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=315>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://www.sanger.ac.uk/Info/Statistics/"
    TARGET=_blank>
    <IMG SRC="../images/Gorilla_gorilla_gorilla.jpg" WIDTH=300 HEIGHT=300 ALT="Gorilla"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Gorilla gorilla gorilla</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Public Domain Photograph
    </FONT>
  </TD></TR>
</TABLE>

<P>
The Oct. 2008 <em>Gorilla gorilla gorilla</em> draft assembly 
(Sanger Institute genome_31265) was produced by the
<A HREF="http://www.sanger.ac.uk/Info/Statistics/"
TARGET=_blank>Sanger Institute</A> sequencing center.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Gorilla genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>chrZ</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of chromosome Z</TD>
    </TR>
    <TR><TD VALIGN=Top>chr3:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of chr 3</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr3:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 3 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>BX950328</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number BX950328</TD>
    </TR>
    <TR><TD VALIGN=Top>BU335953</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		BU335953 on chr Z</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene mRNA</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zhang</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zhang</TD>
    </TR>
    <TR><TD VALIGN=Top>Taguchi,T.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author T. Taguchi</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Taguchi T</em>, internally it
	uses the format <em>Taguchi,T.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
This is <A HREF="https://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=nuccore&id=211774530"
TARGET=_blank>NCBI Genome Project 31265</A> with accession numbers specified
by LOCUS accession record CABD01000000
</P>
<P>
This draft assembly is a 2x ABI based Phusion assembly which has been improved
with 20x Solexa short read sequence.  The short reads were aligned to
the ABI assembly using Maq and where evidence exists from the
Solexa reads that the ABI base call is wrong it has been corrected.
Regions of the ABI assembly that had no Solexa coverage were
assumed to be non-Gorilla sequence and removed.  Large gaps in the
Supercontigs are inferred by Phusion from the distance between
placed fosmid end reads.
</P>
<P>
In this initial assembly, there are 217,735 identified gaps, comprising
a total of 265,263,406 bases.  There is an additional 16,910,764 bases
marked with <em>N</em>s outside of identified gaps.  The total base
count of AGCT nucleotides is 2,043,974,660 bases, plus the
identified gaps and additional <em>N</em>s add up to 2,326,148,830 bases
of sequence in 609,861 Supercontigs.  The Supercontig N50 size is 11,283 bases,
with 45,589 Supercontigs larger than that comprising one half of the
sequence (1,163,074,415 bases)
</P>
The Gorilla 
mitochondrial sequence is also available as the virtual chromosome 
&quot;chrM&quot;. (NCBI accession NC_011120) 
</P>


<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/gorGor1/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#gorilla">Downloads</A> page. 

These data have
<A HREF="../goldenPath/credits.html#gorilla_use">specific 
conditions for use</A>. 

The Gorilla browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#gorilla_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
