<H2>Description</H2>
This track depicts gaps in the assembly.

<P>
Gaps are represented as black boxes in this track.
If the relative order and orientation of the contigs on either side
of the gap is supported by read pair data, 
it is a <em>bridged</em> gap and a white line is drawn 
through the black box representing the gap. 
</P>
<P>This assembly contains the following principal types of gaps:
<UL>

<LI><B>Fragment</B> - gaps between the Whole Genome Shotgun contigs of a 
supercontig.  (In this context, a contig is a set of overlapping sequence reads.
A supercontig is a set of contigs ordered and oriented during the 
Whole Genome Shotgun process using paired-end reads.)
These are represented by varying numbers of <em>N</em>s  in the assembly.  
Fragment gap sizes are usually taken from read pair data.  
</LI>

</UL></P>
<P>
In this initial assembly, there are 217,735 identified gaps, comprising
a total of 265,263,406 bases.  There is an additional 16,910,764 bases
marked with <em>N</em>s outside of identified gaps.  The total base
count of AGCT nucleotides is 2,043,974,660 bases, plus the
identified gaps and additional <em>N</em>s add up to 2,326,148,830 bases
of sequence in 609,861 Supercontigs.  The Supercontig N50 size is 11,283 bases,
with 45,589 Supercontigs larger than that comprising one half of the
sequence (1,163,074,415 bases)
</P>
