<H2>Description</H2>
<P>
This track shows the draft assembly of the $organism genome.  

Whole-genome shotgun reads were assembled into contigs.  When possible, 
contigs were grouped into Supercontigs.
The order, orientation and gap sizes between contigs within a Supercontig are
based on paired-end read evidence. </P>
<P>
In dense mode, this track depicts the contigs that make up the 
currently viewed Supercontig. 
Contig boundaries are distinguished by the use of alternating gold and brown 
coloration. Where gaps
exist between contigs, spaces are shown between the gold and brown
blocks.  The relative order and orientation of the contigs
within a scaffold is always known; therefore, a line is drawn in the graphical
display to bridge the blocks.</P>
<P>
All components within this track are of fragment type &quot;W&quot;: 
Whole Genome Shotgun contig. </P>

<P>
In this initial assembly, there are 217,735 identified gaps, comprising
a total of 265,263,406 bases.  There is an additional 16,910,764 bases
marked with <em>N</em>s outside of identified gaps.  The total base
count of AGCT nucleotides is 2,043,974,660 bases, plus the
identified gaps and additional <em>N</em>s add up to 2,326,148,830 bases
of sequence in 609,861 Supercontigs.  The Supercontig N50 size is 11,283 bases,
with 45,589 Supercontigs larger than that comprising one half of the
sequence (1,163,074,415 bases)
</P>
