<H3>Description</H3>
<P>
This column shows microarray gene expression data from the Gladstone Institute.
</P>
<P>
By default, the column shows the ratio of expression of a gene in a specific 
set of tissues to the expression of the gene overall. A gene that is more 
highly expressed in a tissue is colored red, and a less expressed gene is 
shown in green. The values are colored on a logarithmic scale. This coloring 
is standard, but is the opposite of what an inexperienced user 
might expect: in this case, red means go and green means stop!  
Black indicates that a gene is neither over- nor under-expressed in the tissue. 
Uncolored boxes (white on most browsers) represent missing data.  </P>

<H3>Column Configuration</H3>
<P>
Several column configuration options are available on the Configuration page, 
which can be accessed via the <em>configure</em> button in the top menu.
<UL>
<LI>
<B>Absolute expression values:</B> The display can be configured to show 
absolute expression values rather than ratio values. To toggle the display, 
select the <em>absolute</em> option from the <em>values</em> pulldown menu. 
Absolute expression values
are shown in shades of gray, with higher expression levels shown in lighter 
shades. As with ratio values, absolute values are colored on a logarithmic 
scale.</LI>
<LI>
<B>Color scheme:</B> Color-blind users can switch the coloring scheme from 
red/green to yellow/blue via the <em>Expression ratio colors</em> pulldown 
menu at the top of the Configuration page.</LI>
<LI>
<B>Level of tissue detail:</B> The <em>tissue</em> pulldown menu
controls the amount and type of tissue data displayed. By default, only a few 
selected tissues are shown.  To display the value of all 
individual experimental replicates of all tissues, 
select the <em>all replicates</em> option.</LI>
<LI>
<B>Brightness:</B> To increase or decrease the brightness of the expression 
colors, edit the number in the column's <em>brightness</em> text box. Values 
greater than 1.0 increase the brightness, while those less than 1.0 dim the 
color. </LI>
</UL></P>

<H3>Methods</H3>
<P>
For detailed information about the experiments, see
Abeyta MJ <em>et al.</em> 
<A HREF="http://hmg.oxfordjournals.org/cgi/content/abstract/13/6/601" 
TARGET=_blank>Unique gene expression signatures of independently-derived 
human embryonic stem cell lines</A>. 
<em>Hum. Mol. Gen.</em> 2004 Mar 15;13(6):601-8.</P>

<P>When calculating expression ratios, the overall expression level in the
denominator was calculated by first taking the median of replicates for
each tissue and then taking the median of these medians. </P>
<P>
To associate particular genes with expression data, a fairly 
simple-minded scheme was used.  First, the cDNA sequences around which 
the probes were designed were aligned to the genome.  Then, for each gene, the 
cDNA sequence with the alignment that most overlaps the gene was selected as 
the source for expression data for that gene.  Because
many of the cDNA sequences and probes designed for the cDNA sequences cover
only part of the gene, in most cases different splice varients of a gene 
will end up mapped to the same cDNA sequence. </P>

<H3>Credits</H3>
<P>
The data for this column were kindly provided by the
<A HREF="http://www.gladstone.ucsf.edu/gladstone/php/?sitename=conklin" 
TARGET=_blank>Bruce Conklin lab</A>
at the <A HREF="http://www.gladstone.ucsf.edu/" TARGET=_blank>Gladstone 
Institute at UCSF</A>.</P>

