<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=185>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://www.broadinstitute.org/models/anole"
    TARGET=_blank>
    <IMG SRC="../images/Anolis_carolinensis.jpg" WIDTH=168 HEIGHT=272 ALT="Lizard"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Anolis carolinensis</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of
      (<A HREF="http://www.steevithak.com/"
      TARGET=_blank>R. Steven Rainwater</A> all rights reserved.)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The May 2010 <em>Anolis carolinensis</em> draft assembly 
(Broad version AnoCar2.0, accession #GCA_000090745.1) was
produced by the
<A HREF="http://www.broadinstitute.org/models/anole"
TARGET="_blank">Broad Institute</A>, Cambridge, MA.
For more information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/305698/" TARGET="_blank">
AnoCar2.0</A> in the NCBI Assembly database.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
lizard genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>chrLGh</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of microchromosome chrLGh</TD>
    </TR>
    <TR><TD VALIGN=Top>chrLGh:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of microchromosome chrLGh</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chrLGh:100000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of microchromosome chrLGh that spans 2000 bases, starting with position 100000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>AF223224</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number AF223224</TD>
    </TR>
    <TR><TD VALIGN=Top>AF508304</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays human mRNA with GenBank accession
		AF508304 aligning to anoCar2 assembly</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>androgen</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Anolis carolinensis androgen receptor mRNA, partial
cds</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zahler</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zahler</TD>
    </TR>
    <TR><TD VALIGN=Top>Kitaguchi,T.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author T. Kitaguchi</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Kitaguchi T</em>, internally it
	uses the format <em>Kitaguchi,T.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
The lizard genome has been sequenced to 7.1X coverage.
The draft sequence was assembled into the chromosomes
chr1, chr2, chr3, chr4, chr5, and chr6, as well as several microchromosome 
linkage groups that have not yet been assigned numbers: 
chrLGa, chrLGb, chrLGc, chrLGd, chrLGf, chrLGg, chrLGh. Names will be
assigned to several of these microchromosomes in future releases; for
example, chrLGc=chr7, chrLGd=chr8, chrLGa=chr9, and chrLGb=chrX.
This assembly also contains 6,443 scaffolds with 717,481,773 bases of sequence 
that have not been localized to chromosome positions. The largest scaffold, 
chrUn_GL343193, comprises 14,352,415 bases. The scaffolds have an N50 size of 
1,224,583; half of the additional sequence is contained in 154 scaffolds of at 
least this size.  The total sequence size including chromosomes and
scaffolds is 1,799,143,587 bases.  The Genome Browser also displays the 
mitochondrion (chrM) sequence 
<A HREF="https://www.ncbi.nlm.nih.gov/nuccore/NC_010972"
TARGET=_blank>NC_010972</A> from GenBank.
For more information on the AnoCar2.0 assembly, see the 
<A HREF="https://www.ncbi.nlm.nih.gov/Traces/wgs/?val=AAWZ02"
TARGET=_blank>assembly record</A> in GenBank.
</P>
<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/anoCar2/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#lizard">Downloads</A> page. 
These data have <A HREF="../goldenPath/credits.html#lizard_use">specific 
conditions for use</A>. 
The lizard browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#lizard_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
