<H2>Description</H2>
This track depicts gaps in the draft assembly
(May 2010, Broad Institute of MIT and Harvard AnoCar 2.0 (GCA_000090745.1))
of the $organism genome.  <A HREF="https://www.ncbi.nlm.nih.gov/Traces/wgs/?val=AAWZ02"
TARGET=_blank>(WGS assembly information)</A>

Many of these gaps &mdash; with the
exception of intractable heterochromatic gaps &mdash; may be closed during the
finishing process.

<P>
Gaps are represented as black boxes in this track.
If the relative order and orientation of the contigs on either side
of the gap is supported by read pair data, 
it is a <em>bridged</em> gap and a white line is drawn 
through the black box representing the gap. 
</P>
<P>This assembly contains the following principal types of gaps:
<UL>
<LI><B>Fragment</B> - gaps between the Whole Genome Shotgun contigs of a 
supercontig. These are represented by varying numbers of <em>N</em>s in the 
assembly. In this context, a contig is a set of overlapping sequence reads and  
a supercontig is a set of contigs ordered and oriented during the 
Whole Genome Shotgun process using paired-end reads. Fragment gap sizes 
are usually taken from read pair data.  
</LI>

<LI><B>Clone</B> - gaps between supercontigs linked by the physical map.  
In general, these are represented by 1,000 <em>N</em>s in the assembly.<BR>

Clone gaps of 3,000,000 have been placed at the centromeres
of chromosomes 1-6 and LGa.
</LI>

</UL></P>
