<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=185>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://www.sanger.ac.uk/resources/zebrafish/genomeproject.html"
    TARGET=_blank>
    <IMG SRC="../images/Danio_rerio.jpg" WIDTH=300 HEIGHT=98 ALT="Zebrafish"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Danio rerio</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of NHGRI
      (<A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Zebrafish&id=79130" 
      TARGET=_blank>Press Photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The Jul. 2010 zebrafish (<em>Danio rerio</em>) Zv9 assembly (GCA_000002035.2)
was produced by 
<A HREF="http://www.sanger.ac.uk/" TARGET=_blank>The Wellcome Trust Sanger 
Institute</A>, UK. For more information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/238918/" TARGET="_blank">
Zv9</A> in the NCBI Assembly database.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
zebrafish genome.  Note that some position queries (e.g. "huntington") may 
return matches to the mRNA records of other species. In these cases, the 
mRNAs are mapped to their homologs in zebrafish. See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
<TR><TD VALIGN=Top NOWRAP><B>Request:</B><br></TD>
	<TD VALIGN=Top COLSPAN=2><B>&nbsp;&nbsp; Genome Browser Response:</B><br></TD></TR>
	
<TR><TD VALIGN=Top><br></TD></TR>
	
<TR><TD VALIGN=Top NOWRAP>chr1</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays all of chromosome 1</TD></TR>
<TR><TD VALIGN=Top NOWRAP>chr1:1-200000</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays first two hundred thousand bases of chromosome 1</TD></TR>
    <TR><TD VALIGN=Top NOWRAP>chr1:100000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 1 that spans 2000 bases, starting with position 100000 </TD>
    </TR>
<TR><TD VALIGN=Top><br></TD></TR>

<TR><TD VALIGN=Top NOWRAP>U30710</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region containing zebrafish mRNA with GenBank accession number U30710</TD></TR>

<TR><TD VALIGN=Top NOWRAP>AA658622</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region containing zebrafish EST with GenBank accession AA658622
</TD></TR>
<TR><TD VALIGN=Top NOWRAP>ENSDART00000025573</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region containing Ensembl gene prediction transcript ENSDART00000025573
</TD></TR>
<TR><TD VALIGN=Top><br></TD></TR>

<TR><TD VALIGN=Top NOWRAP>p53</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs related to the p53 tumor suppressor</TD></TR> 
</TD></TR>
<TR><TD VALIGN=Top NOWRAP>pseudogene mRNA</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists transcribed pseudogenes but not cDNAs, in GenBank</TD></TR>
<TR><TD VALIGN=Top NOWRAP>homeobox caudal</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes in GenBank</TD></TR>
<TR><TD VALIGN=Top NOWRAP>zinc finger</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD></TR>
<TR><TD VALIGN=Top NOWRAP>kruppel zinc finger</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD></TR>
<TR><TD VALIGN=Top NOWRAP>huntington</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs associated with Huntington's disease</TD></TR>

<TR><TD VALIGN=Top><br></TD></TR>

<TR><TD VALIGN=Top NOWRAP>porter</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientists named Porter</TD></TR>
<TR><TD VALIGN=Top NOWRAP>Amsterdam,A.</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by co-author A. Amsterdam</TD></TR>

<TR><TD VALIGN=Top><br></TD></TR>
	
<TR><TD COLSPAN="3" > Use this last format for author queries. Although 
GenBank requires the search format <em>Amsterdam A</em>, internally it uses 
the format <EM>Amsterdam,A.</EM>

<HR>
<H3>Assembly details</H3>
<P>
The Zv9 assembly comprises a sequence length of 1.4 Gb in 26 chromosomes and 
1,107
scaffolds. This assembly is based on a clone path 
sorted with the high-density meiotic map SATMAP (Clark et al., in
preparation). The data freeze was taken on 1 April 2010. The remaining gaps 
were filled with sequence from WGS31, a combined Illumina and capillary 
assembly. The assembly integration process involved sequence alignemnts as well 
as cDNA, marker and BAC/Fosmid end sequence placements.
For more details about the Zv9 assembly, see the Sanger Institute 
page for the 
<A HREF="http://www.sanger.ac.uk/resources/zebrafish/genomeproject.html"
TARGET=_blank><em>Danio rerio</em> Sequencing Project</A>. </P>
<P>
In this assembly, scaffolds that are based on clone contigs or could be 
associated with chromosome placements through marker information are named 
Zv9_scaffold<em>n</em>. WGS contigs that could not be placed on chromosomes 
are named Zv9_NA<em>n</em>. Linkage group 
numbers were translated directly into chromosome numbers (e.g. linkage 
group 1 = chromosome 1). The zebrafish mitochondrial sequence is also available 
as the virtual chromosome, chrM (GenBank accession NC_002333.2)</P>
<P>
The Sanger Institute notes that this assembly release is still preliminary. 
The regions of the assembly covered by WGS contigs are of lower quality. 
</P>
<P>
Downloads of the zebrafish data and annotations can be obtained from the UCSC
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/danRer7/" 
TARGET=_blank>FTP site</A> or 
<A HREF=http://hgdownload.soe.ucsc.edu/downloads.html#zebrafish>Downloads page</A>. 
This data set has 
<A HREF="http://www.sanger.ac.uk/legal/"
TARGET=_blank>specific conditions for use</A>.  The danRer7 annotation tracks 
were generated by UCSC and collaborators worldwide. See the 
<A HREF="../goldenPath/credits.html#zebrafish_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to this release. </P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
