<H2>Description</H2>
<P>This track shows the Zebrafish Zv3 (Nov. 2003)  assembly 
provided by the 
<A HREF="http://www.sanger.ac.uk/Projects/D_rerio/Zv3_assembly_information.shtml" 
TARGET=_blank>The Wellcome Trust Sanger Institute</A>. 
The assembly has a sequence coverage of about 5.7X and contains 58,339 scaffolds 
(supercontigs) totaling 1.5 billion base pairs.
</P>
<P>In dense mode, this track depicts the path through the draft and
finished clones (aka the golden path) used to create the assembled sequence.
Clone boundaries are distinguished by the use of alternating gold and brown
coloration. Where gaps
exist in the path, spaces are shown between the gold and brown
blocks.  If the relative order and orientation of the contigs
between the two blocks is known, a line is drawn to bridge the
blocks.</P>
<P>
The Genome Browser depicts the zebrafish genome as 25 chromosomes consisting of
whole genome shotgun (WGS) supercontigs that were mapped to a fingerprinted
contig (FPC) and were from a known chromosome. There are also 3 unordered virtual 
chromosomes: 
<UL>
<LI><B>chrNA</B> - WGS contigs that could not be related to any FPC contig
<LI><B>chrFinished</B> - finished clones that could not be place into WGS 
supercontigs
<LI><B>chrUn</B> - WGS supercontigs that mapped to FPC contigs, but the 
chromosome is unknown
</UL>
The virtual chromosomes contain 1000 bp 
scaffold gaps that are shown in the Gap track annotation.
<P>
All components within this track are of fragment type &quot;W&quot;
(WGS contig) except for those on chrFinished and chrM, which are of 
type &quot;F&quot; (Finished).
</P>
<H2>Methods</H2>
<P>
This assembly was constructed using the assembler, Phusion, to cluster reads. 
Phrap was then used for cluster assembly and consensus generation. 
For the clone-based mapping and finishing, clones from different libraries were
fingerprinted by digestion with the HindIII restriction enzyme.  From the 
information produced in this way, overlapping clones were linked into 
fingerprinted contigs (FPCs). Next, clones from a tiling path through the FPC 
contigs were selected for high quality sequencing. The resulting 
sequence was submitted to EMBL/GenBank. 
<P>
The supercontigs tied to the FPC map create the assembly shown in this track. 
1.083 Gigabases or 74% of the sequence 
could be tied to the FPC map. The finished clone sequence was then 
analyzed via a pipeline that included repeatmasking, <em>ab initio</em> gene 
prediction and blast searches against all protein, EST and cDNA sequences that 
were available. Results from this analysis were used to manually annotate clones 
with gene structures, descriptions and poly-A features. At this point, the clone 
was submitted to EMBL/GenBank again and can be browsed in 
<A HREF="http://vega.sanger.ac.uk/Danio_rerio/" TARGET=_blank>Vega</A>.
</P>
<H2>Credits</H2>
<P>
The Zv3 Zebrafish assembly was produced by The Wellcome Trust Sanger Institute,
in collaboration with the Max Planck Institute for Developmental Biology, 
the Netherlands Institute for Developmental Biology (Hubrecht Laboratory),
and Yi Zhou, Anthony DiBiase and Leonard Zon from the Boston Children's 
Hospital. 
