<H2>Description</H2>
<P>
This track shows alignments between $organism radiation hybrid (RH) map 
sequences (consisting of ESTs and other genetic markers) and the genome. Many 
of these markers are not genetically polymorphic and therefore cannot be 
placed on genetic maps. The markers with RH map positions are useful for 
locating genes lying in genomic regions with mutations and in the 
positional cloning of zebrafish genetic mutants that exhibit interesting 
phenotypes. </P>
<P>
8707 RH map sequences were aligned, consisting of:
<UL> 
<LI>2835 clone ends from Dr. Robert Geisler at the 
<A HREF="http://www.eb.tuebingen.mpg.de/" TARGET=_blank>Max Planck Institute 
(MPI) for Developmental Biology</A>, Germany and the 
<A HREF="http://www.niob.knaw.nl/" TARGET=_blank>Hubrecht Laboratory</A> in 
the Netherlands
<LI>2015 ESTs from <A HREF = "https://www.ncbi.nlm.nih.gov/" 
TARGET=_blank>NCBI</A>
<LI>171 RH shotgun sequences from Dr. Robert Geisler at the MPI and the 
<A HREF="http://zon.tchlab.org" 
TARGET=_blank>Zebrafish Genome Initiative</A> at Childrens Hospital, Boston
<LI>3686 WZ ESTs from 
<A HREF="http://www.genetics.wustl.edu/fish_lab/frank/cgi-bin/fish/" 
TARGET=_blank>WashU-Zebrafish Genome Resources</A>
</UL></P>
<P>
The majority of these sequences are single-read sequences 
in the range of about 40 - 1300 bp. The ESTs usually
represent fragments of transcribed genes.  

<H2>Display Conventions and Configuration</H2>
<P>
This track follows the display conventions for 
<A HREF="../goldenPath/help/hgTracksHelp.html#PSLDisplay" 
TARGET=_blank>PSL alignment tracks</A>. 
The strand information (+/-) indicates the
direction of the match between the EST and the matching
genomic sequence.  It bears no relationship to the direction
of transcription of the RNA with which it might be associated.</P>

<H2>Methods</H2>
<P>
To generate this track, $organism RH map sequences were aligned against the 
genome using blat. Only alignments with a base identity level within 0.1% of 
the best and at least 96% base identity with the genomic sequence were kept. 
The following pslReps parameters were used: <em>-nearTop=0.0001 -minAli=0.96
-minCover=0.40</em>. </P>

<H2>Credits</H2>
<P>
The $Organism RH Map track was produced by UCSC in collaboration with the 
Zebrafish Genome Initiative at Childrens Hospital, Boston, from sequence data
obtained from the individuals and institutions mentioned in the Description 
section. </P>
