<H2>Description</H2>
<P>
This track shows AceView gene models constructed from
mRNA, EST and genomic evidence by Danielle and Jean Thierry-Mieg
and Vahan Simonyan using the 
<A HREF="https://www.ncbi.nlm.nih.gov/IEB/Research/Acembly/index.html"
TARGET=_blank>Acembly</A> program.</P> 

<H2>Display Conventions and Configuration</H2>
<P>
This track follows the display conventions for 
<A HREF="../goldenPath/help/hgTracksHelp.html#GeneDisplay">gene prediction 
tracks</A>. Gene models that fall into the &quot;main&quot; prediction class
are displayed in <font color="#80007D">purple</font>; &quot;putative&quot; 
genes are displayed in <font color="#C8007D" >pink</font>.</P>
<P>
The track description page offers the following filter and configuration
options:
<UL>
<LI><B>Gene Class filter:</B> Select the <em>main</em> or <em>putative</em>
option to filter the display by prediction class. 
<LI><B>Color track by codons:</B> Select the <em>genomic codons</em> option
to color and label each codon in a zoomed-in display to facilitate validation 
and comparison of gene predictions. Click the &quot;Codon coloring help&quot; 
link on the track description page for more information about this feature.
</UL>
</P>

<H2>Methods</H2>
<P>
AceView attempts to find the best alignment of each mRNA/EST against the
genome, and clusters the alignments into the least possible number of
alternatively spliced transcripts. The reconstructed transcripts are then
clustered into genes by simple transitive contact. To see the evidence that 
supports each transcript, click the &quot;Outside Link&quot; on an individual 
transcript's details page to access the NCBI AceView web site. </P>
<P>
Each AceView transcript model has a gene cluster designation
(alternate name) that is categorized into a prediction class
of either <font color="#80007D" >main</font> or 
<font color="#C8007D" >putative</font>.</P>
<P>
<b>Prediction Class:</b> <font color="#80007D" >main</font> 
<BR> Class of genes that includes the protein coding genes (defined
here by CDS &gt; 100 amino acids) and all genes with at least one
well-defined standard intron, i.e., an intron with a GT-AG or GC-AG
boundary, supported by at least one clone matching exactly, with
no ambiguous bases, and the 8 bases on either side of the intron 
identical to the genome. Genes with a CDS smaller than 100 amino acids are
included in this class if they meet one of the following conditions: they 
have a NCBI RefSeq sequence (NM_#) or an OMIM identifier, or they encode a 
protein with BlastP homology (&lt; 1e-3) to a cDNA-supported nematode AceView 
protein.</P>
<P>
<b>Prediction Class:</b> <font color="#C8007D" >putative</font>
<BR>Class of genes that have no standard intron and do not
encode CDS of more than 100 amino acids, yet may be sufficiently useful to 
justify not disregarding them completely. Putative genes may be of two
types: either those supported by more than six cDNA clones or those that
encode a putative protein with an interesting annotation. Examples include
a PFAM motif, a BlastP hit to a species other than itself (&lt; 1e-3), 
a transmembrane domain or other rare and meaningful domains
identified by Psort2, or a highly probable localization in a cell
compartment (excluding cytoplasm and nucleus).</P>

<H2>Credits</H2>
<P>
Thanks to <A HREF="mailto:mieg@ncbi.nlm.nih.gov">Danielle and Jean 
Thierry-Mieg</A> at NIH for providing this track.</P>

<H2>References</H2>
<P>
Thierry-Mieg D, Thierry-Mieg J. 
<A HREF="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2006-7-s1-s12" 
TARGET=_blank>AceView: a comprehensive cDNA-supported gene and transcripts 
annotation</A>.
<em>Genome Biol</em>. 2006;<B>7</B> Suppl 1:S12.1-14.</P>
