<H2>Description</H2>
<P>
This track displays blastz alignments of the $o_organism assembly 
($o_db, $o_date) to the $organism genome, filtered by axtBest and 
subsetAxt with very stringent constraints as described below.</P>

<H2>Display Conventions and Configuration</H2>
<P>
This track follows the display conventions for 
<A HREF="../goldenPath/help/hgTracksHelp.html#PSLDisplay">PSL alignment
tracks</A>. 
Each item in the display is identified by the chromosome, strand, and 
location of the match (in thousands). </P>
<P>
The track description page offers the following filter and configuration
options:
<UL>
<LI><B>Color track</B>: To display the chromosome color track, click the
<em>on</em> button next to &quot;Color track based on chromosome&quot;.
When the color track is activated, each of the items within the annotation
track will be colored to show the chromosome in the aligning genome
assembly from which the alignment originated.</LI>
<LI><B>Chromosome filter</B>: To display only alignments from a specific
chromomsome in the aligning assembly, type the chromosome number (in the
form <I>chrN</I>) in the text box to the right of &quot;Filter by
chromosome&quot;. For example, to display alignments from chromosome 6,
type &quot;chr6&quot;.
</LI>
</UL>
</P>
<P>
When you have finished configuring the filter, click the <I>Submit</I>
button to return to the annotation tracks page.</P>

<H2>Methods</H2>
<P>
For blastz, 12 of 19 seeds were used and then scored using:
<PRE>
      A     C     G     T
A    91  -114   -31  -123
C  -114   100  -125   -31
G   -31  -125   100  -114
T  -123   -31  -114    91

O = 400, E = 30, K = 3000, L = 3000, M = 50
</PRE></P>
<P>
A second pass was made at reduced stringency (7mer seeds and
MSP threshold of K=2200) to attempt to fill in gaps of up to about 10K bp.
Lineage-specific repeats were abridged during this alignment.</P>
AxtBest was used to select only the best alignment for any given region
of the genome.  SubsetAxt was then run on axtBest-filtered alignments 
with this matrix:
<PRE>
      A     C     G     T
A   100  -200  -100  -200
C  -200   100  -200  -100
G  -100  -200   100  -200
T  -200  -100  -200   100
</PRE>
with a gap open penalty of 2000 and a gap extension penalty of 50. 
The minimum score threshold was 3400.</P>

<H2>Credits</H2>
<P>
The alignments are contributed by Scott Schwartz from the 
<A HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank>Penn State Bioinformatics 
Group</A>. The best in genome filtering is done by UCSC's 
axtBest and subsetAxt programs. Mouse sequence data were provided by the 
Mouse Genome Sequencing Consortium. </P>

<H2>References</H2>
<P>
Chiaromonte, F., Yap, V.B., Miller, W.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=11928468&dopt=Abstract"
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>.
<em>Pac Symp Biocomput 2002</em>, 115-26 (2002).</P>
<P>
Schwartz, S., Kent, W.J., Smit, A., Zhang, Z., Baertsch, R., Hardison, R.,
Haussler, D., and Miller, W.
<A HREF="http://www.genome.org/cgi/content/abstract/13/1/103"
TARGET=_blank>Human-mouse alignments with BLASTZ</A>.
<em>Genome Res.</em> <B>13</B>(1), 103-107 (2003).</P>

