<H2>Description</H2>
<P>
This track shows a discretized version of genome-wide DNaseI
hypersensitive (HS) sites as determined in human CD4+ T-cells.  DNaseI
HS sites identify regions of open chromatin believed to be largely 
free of nucleosomes and have been shown to accurately identify the
locations of genetic regulatory elements, including promoters,
enhancers, silencers, insulators, and locus control regions.  DNaseI
HS sites are believed to differ in degree of openness.  Scores shown
in this annotation reflect an estimate of this openness.
</P>

<H2>Display Conventions and Configuration</H2>
<P>
DNaseI HS sites are represented as gray or black boxes.  The
intensity of the box reflects the HS score and represents the
estimated degree of hypersensitivity of the site.
</p>

<H2>Methods</H2>
<P>
DNaseI HS were determined for human CD4+ T-cells using
DNase-sequencing and DNase-chip.  Over 12.5 million uniquely aligned
DNase sequence tags were generated using Illumina (Solexa) and 454
Life Sciences (Roche) sequencers from the same biological sample.
Each base was assigned a score using Parzen windows kernal density
estimation.  The Nimblegen 38-array whole genome platform was used to
generate data from 2 biological replicates.  Ratio scores for the two
were averaged.  These data were combined by re-scaling each using
Z-scores and then summing.  This annotation reflects those contiguous
regions with combined scores above zero.  Each record contains the maximum
base pair score within each identified region.

</P>

<H2>Credits</H2>
<P>
This annotation was created by Alan Boyle, Terry Furey, and Greg
Crawford at Duke University's <A HREF="https://genome.duke.edu/"
TARGET=_blank>Institute for Genome Sciences & Policy (IGSP)</A>. </P>

<H2>References</H2>
<P>
Boyle AP, Davis S, Shulha HP, Meltzer P, Margulies EH, Weng Z, Furey TS,
Crawford GE.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/18243105?ordinalpos=1&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum"
TARGET=_blank>High-resolution mapping and characterization of open chromatin
across the genome</A>. <em>Cell</em>. 2008 Jan 25;132(2):311-22.</P>

<p>
Crawford GE, Davis S, Scacheri PC, Renaud G, Halawi MJ, Erdos MR,
Green R, Meltzer PS, Wolfsberg TG, Collins FS. <A
HREF="https://academic.oup.com/nar/article/34/12/e84/2375632"
TARGET=_blank>DNase-chip: a high-resolution method to identify DNase I
hypersensitive sites using tiled microarrays</a>. <em>Nat
Methods</em>. 2006 Jul;3(7):503-9.
</p>

<P>
Crawford GE, Holt IE, Whittle J, Webb BD, Tai D, Davis S, Margulies
EH, Chen Y, Bernat JA, Ginsburg D, Zhou D, Luo S, Vasicek TJ, Daly MJ,
Wolfsberg TG, Collins FS.  <A
HREF="https://genome.cshlp.org/content/16/1/123.full"
TARGET=_blank>Genome-wide mapping of DNase hypersensitive sites using
massively parallel signature sequencing (MPSS)</a>.  <em>Genome
Res.</em> 2006 Jan;16(1):123-31. 
</p>
