<H2>Description</H2>
<P>
This track displays multi-species conserved sequences (MCSs)
derived from phastCons, binCons, 
<A HREF="http://mendel.stanford.edu/sidowlab/downloads/GERP/index.html"
TARGET=_blank>GERP</A> (Genomic Evolutionary Rate Profiling),
and SCONE conservation scoring
of Threaded Blockset Aligner (TBA) multiple sequence alignments in the 
ENCODE regions. </P>
<P>
The multiple sequence alignments may be viewed in the TBA Alignment
track. Another related track, TBA Cons, shows the conservation scoring. 
The descriptions accompanying these tracks detail
the methods used to create the alignments and conservation scoring.</P>

<H2>Display Conventions and Configuration</H2>
<P>
The locations of conserved elements are indicated by blocks in the graphical
display. This composite annotation track consists of several subtracks that
show conserved elements derived by the various methods listed above.
To view only selected subtracks, uncheck the boxes next to the tracks
you wish to hide. </P>
<P>
The display may also be filtered to show only those items
with unnormalized scores that meet or exceed a certain threshold. To set a
threshold, type the minimum score into the text box at the top of the 
description page.</P>
<P>
Display characteristics specific to certain subtracks are described in the
respective Methods sections below.</P>
 
<H2>Methods</H2>
<P>

<H3>PhastCons-based Elements</H3>
<P>
The predicted MCSs are segments of the alignment that are likely to
have been &quot;generated&quot; by the conserved state of the phylo-HMM,
<em>i.e</em>., maximal segments in which the maximum-likelihood (Viterbi)
path remains in the conserved state.</P>  

<H3>BinCons-based Elements</H3>
<P>
The binCons score is based on the cumulative binomial probability of 
detecting the observed number of identical bases (or greater) in 
sliding 25 bp windows (moving one bp at a time) between the 
reference sequence and each other species, given the neutral rate
at four-fold degenerate sites. Neutral rates are calculated
separately at each targeted region. For targets with no gene annotations,
the average percent identity across all alignable sequence was instead used
to weight the individual species binomial scores; this latter
weighting scheme was found to closely match 4D weights.</P>
<P>
The negative log of these p-values was then averaged across all 
human-referenced pairwise combinations, and the highest-scoring overlapping 
25 bp window for each base was the resulting score. This track shows the 
plotting of a ranked percentile score normalized between 0 and 1 across all 
ENCODE regions, such that the top 5% most conserved sequence across all ENCODE
regions have a score of 0.95 or greater, the top 10% have a score of 0.9 or 
greater, and so on.</P>
<P>
For each ENCODE target, a conservation score threshold was picked to match
the number of conserved bases predicted by phastCons, an alternative method
for measuring conservation. This latter method has been found slightly more
reliable for predicting the expected fraction of conserved sequence 
in each target. Clusters of bases
that exceeded the given conservation score threshold were designated 
as MCSs. The minimum length of an MCS is 25
bases. Strict cutoffs were used: if even one base fell below the
conservation score threshold, it separated an MCS into two distinct
regions.</P>
<P>
More details on binCons can be found in Margulies <em>et. al</em>. (2003)
cited below.</P>

<H3>GERP-based Elements</H3>
<P>
GERP elements are scored according to the inferred intensity
of purifying selection
and are measured as &quot;rejected substitutions&quot; (RSs). RSs capture the
magnitude of difference between the number of &quot;observed&quot; substitutions
(estimated using maximum likelihood) and the number that would be
&quot;expected&quot; under a neutral model of evolution.   
The RS is displayed as part of the item name.
Items with higher RSs are displayed in a darker shade of blue. The score shown 
on the details page, which has been scaled by 300 for display purposes, is 
generally not as accurate as the RS count that is part of the item name.</P>
<P>
&quot;Constrained elements&quot; are identified as those groups
of consecutive human bases that have an observed rate of evolution that is
smaller than the expected rate.  These groups of columns are merged if they
are less than a few nucleotides apart and are scored according to the sum of
the site-by-site difference between observed and expected rates (RS).</P>
<P>
Permutations of the actual alignments were analyzed, and the &quot;constrained
elements&quot; identified in these permuted alignments were treated as
&quot;false positives&quot;.  Subsequently, an RS threshold was picked such
that the total length of &quot;false positive&quot;  constrained elements
(identified in the permuted alignments) was less than 5% of the length of
constrained elements identified in the actual alignment.
Thus, all annotated constrained elements are significant at better
than 95% confidence, and the total fraction of the ENCODE regions
annotated as constrained is 5-7%.</P>

<H3>SCONE-based Elements</H3>
<P>
SCONE provides p-value scores per base. Constrained elements are
defined based on SCONE site-specific scores as follows. An additive
score is first defined as the sum of (-log p + log t) along an
interval, where p is the SCONE score and t is some threshold value for
conservation. This additive score may be treated as a random walk;
elements are defined as the intervals between local minima and maxima
along this walk. Subsequently, a cutoff is set for the additive scores
for each element. This cutoff is chosen such that the elements scoring
above the cutoff for a random sequence of scores draw from a uniform
distribution [0,1] with threshold t = 0.25 will cover no more than
0.25% of the sequence.</P>

<H2>Credits</H2>
<P>
PhastCons was developed by 
<A HREF="http://siepellab.labsites.cshl.edu/" TARGET=_BLANK>
Adam Siepel</A>, Cold Spring Harbor Laboratory, while at the
<A HREF="https://ucscgenomics.soe.ucsc.edu/team/"
TARGET=_blank>Haussler Lab</A> at UCSC.</P>
<P>
BinCons was developed by Elliott Margulies of NHGRI, while at the
<A HREF="https://www.genome.gov/27535200/" TARGET=_blank>Eric Green lab</A>.
BinCons and phastCons MCS data were contributed by Elliott Margulies,
with assistance from Adam Siepel of UCSC.</P>
<P>
GERP was developed primarily by Greg Cooper in the lab of
<A HREF="http://mendel.stanford.edu/sidowlab/" TARGET=_blank>Arend Sidow</A>
at Stanford University (Depts of Pathology and Genetics), in close collaboration
with Eric Stone (Biostatistics, NC State), and George Asimenos and
Eugene Davydov in the lab of
<A HREF="https://www.serafimb.org/people.html/" TARGET=_blank>Serafim Batzoglou</A>
(Dept. of Computer Science, Stanford).</P>
<P>
SCONE was developed by Saurabh Asthana in the lab of <a
href="http://genetics.bwh.harvard.edu/genetics/labs/Sunyaev/index.html" TARGET=_blank>Shamil
Sunyaev</a> at Harvard Medical School and Brigham &amp; Women's Hospital
(Department of Medicine/Division of Genetics).
<P>
TBA was provided by Minmei Hou, Scott Schwartz and Webb Miller of the 
<A HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank>Penn State Bioinformatics 
Group</A>.</P>

<H2>References</H2> 
<P>
See the TBA Alignment and TBA Cons tracks for references.</P>
