<H2>Description</H2>
<P>
This track displays DNaseI-hypersensitive sites identified
using two methods (Dnase-chip and MPSS sequencing) 
in the  GM06990 lymphoblastoid cell line
and in non-activated and activated CD4+ T-cells.
DNaseI-hypersensitive sites are
associated with all types of gene regulatory regions, 
including promoters, enhancers, silencers, and locus control regions.
CD4+ T-cells, also known as helper or inducer T cells, are
involved in generating an immune response.  CD4+ T-cells are also
one of the primary targets of the HIV virus.</P>

<H2>Display Conventions and Configuration</H2>
<P>
Within the subtracks, the gray and black blocks 
(which appear as vertical lines when the display is zoomed-out) 
represent probable hypersensitive sites. The darker the blocks, the 
more likely the site is to be hypersensitive.</P>
<P>
To display only selected subtracks, uncheck the boxes next to the tracks you 
wish to hide. The display may also be filtered to show only those items
with unnormalized scores that meet or exceed a certain threshhold. To set a
threshhold, type the minimum score into the text box at the top of the 
description page.</P>

<H2>Methods</H2>
<H3> Dnase-Chip </H3>
<P>
DNaseI hypersensitive sites were isolated using a method called
DNase-chip (Crawford <em>et al</em>., unpublished).  Briefly, DNaseI digested
ends from intact chromatin were captured using three different DNase
concentrations as well as three biological replicates.  This material
was amplified, labeled, and hybridized to NimbleGen ENCODE tiled
microarrays.  Each DNase-chip region has a unique identifier, visible
when the track is displayed in full or packed mode. The first part of
the identifier describes the cell type from which the data was derived.
The last number of each identifier indicates the percent of
regions that represent valid DNase hypersensitive sites.</P>

<H3> MPSS Sequencing </H3>
<P>
Primary human CD4+ T cells were activated by incubation with anti-CD3
and anti-CD28 antibodies for 24 hours. DNaseI-hypersensitive sites
were cloned from the cells before and after activation,
and sequenced using massively parallel signature sequencing 
(Brenner <em>et al</em>., 2000; Crawford <em>et al</em>., 2006). 
Only those clusters of multiple DNaseI library sequences that map within 500 
bases of each other are displayed.
Each cluster has a unique identifier, visible when the track is displayed
in full or packed mode. The last digit of each identifier represents the 
number of sequences that map within that particular cluster. The sequence number
is also reflected in the score, <em>e.g.</em> a cluster of two sequences scores 
500, three sequences scores 750 and four or more sequences scores 1000.  </P>

<H2>Verification</H2>
<H3> Dnase-Chip </H3>
<P>
Real-time PCR assay was used to validate a randomly selected subset
of DNase-chip regions. Approximately 20-30% of regions detected 
in only a single DNase concentration are valid. These clusters are shown
in light gray. 50-80% of regions detected in 2 out of 3 DNase
concentrations are valid, indicated by dark gray. 90% of regions
detected in all 3 DNase concentrations are valid, shown in black.</P>
<P>
This data set includes elements for all 44 ENCODE regions.</P>

<H3> MPSS Sequencing </H3>
<P>
Real-time PCR assay was used to verify valid
DNaseI-hypersensitive sites.  Approximately 50% of
clusters of two sequences are valid.  These clusters are shown
in light gray. 80% of clusters of three sequences are valid, indicated by
dark gray. 100% of clusters of four or more
sequences are valid, shown in black.</P>
<P>
This data set includes confirmed elements for 35 of the 44
ENCODE regions.
It is estimated that these data identify
10-20% of all hypersensitive sites within CD4+ T cells. Further
sequencing will be required to identify additional sites.</P>

<H2>Credits</H2>
<P>
These data were produced at the 
<A HREF="http://www.genome.gov/10000351" TARGET=_blank>Collins Lab</A> 
at NHGRI.  Thanks to Gregory E. Crawford and Francis S. Collins for supplying 
the information for this track.</P>

<H2>References</H2>
<P>
<P>
Brenner S, Johnson M, Bridgham J, Golda G, Lloyd DH, Johnson D, Luo S, 
McCurdy S, Foy M, Ewan M <em>et al</em>.
<A HREF="http://www.nature.com/cgi-taf/DynaPage.taf?file=/nbt/journal/v18/n6/abs/nbt0600_630.html&dynoptions=doi1097883799" 
TARGET=_blank>Gene expression analysis by massively parallel signature 
sequencing (MPSS) on microbead arrays</A>.
<em>Nat. Biotechnol.</em> 2000 Jun;18(6):597-8.
<P>
Crawford GE, Holt IE, Mullikin JC, Tai D, Blakesley R, Bouffard G, Young A, 
Masiello C, Green ED, Wolfsberg TG <em>et al</em>.
<A HREF="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=14732688" 
TARGET=_blank>Identifying gene regulatory elements by genome-wide recovery of 
DNase hypersensitive sites</A>.
<em>Proc. Natl. Acad. Sci. USA</em>. 2004 Jan 27;101(4):992-7.</P>
<P>
Crawford GE, Holt IE, Whittle J, Webb BD, Tai D, Davis S, Margulies EH, Chen Y, 
Bernat JA, Ginsburg D <em>et al</em>.
<A HREF="http://www.genome.org/cgi/content/full/16/1/123"
TARGET=_blank>Genome-wide mapping of DNase hypersensitive 
sites using massively parallel signature sequencing (MPSS)</A>.
<em>Genome Res.</em> 2006 Jan;16(1):123-31.
(See also NHGRI's
<A HREF="http://research.nhgri.nih.gov/DNaseHS/May2005/"
TARGET=_blank>data site</A> for the project.)</P>
<P>
McArthur M, Gerum S, Stamatoyannopoulos G.
<A HREF="http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6WK7-457CXYS-4M&_coverDate=10%2F12%2F2001&_alid=210456595&_rdoc=1&_fmt=&_orig=search&_qd=1&_cdi=6899&_sort=d&view=c&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=5a2dd2b59e368e190942669df47d72c8"
TARGET=_blank>Quantification of DNaseI-sensitivity by real-time PCR: 
quantitative analysis of DNaseI-hypersensitivity of the mouse beta-globin 
LCR</A>.
<em>J. Mol. Biol.</em> 2001 Oct 12;313(1):27-34.</P>
