<H2>Description</H2>
<P>
The consensus elements in this track were generated by the ENCODE Multi-Species
Analysis group from the nine different combinations of three conservation 
algorithms (phastCons, binCons, and GERP) and three sequence alignment 
methods (TBA, MLAGAN and MAVID) applied to the ENCODE region sequences 
of 
<A HREF="../images/phylo/species28.gif" TARGET=_BLANK>
28 vertebrate species</A>
    as defined in the 
<A TARGET=_BLANK HREF="http://hgdownload.soe.ucsc.edu/goldenPath/hg17/encode/alignments/SEP-2005/">September 2005 ENCODE MSA sequence freeze</A>
and the
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/hg17/encode/alignments/SEP-2005/phylo/tree_4d.tba.v2.nh" TARGET=_BLANK> MSA species guide tree</A>.</P>
<P>
Three different stringencies were used. 
The <em>loose</em> set of constrained 
sequences represent bases identified as being constrained by any 
conservation algorithm on any alignment. The <em>moderate</em> set of 
constrained sequences is derived from bases shown to be constrained by
at least two of the three conservation algorithms on at least two of 
the three alignments. Finally, the <em>strict</em> set of constrained 
sequences represent only those bases that were constrained using all 
three conservation programs on all three multi-sequence alignments.</P>

<H2>Display Conventions and Configuration</H2>
<P>
The locations of constrained elements are indicated by blocks in the graphical 
display.  To show only selected subtracks within this annotation, 
uncheck the boxes next to the tracks you wish to hide. </P>

<H2>Methods</H2>
<P>
See the description pages for the TBA Elements,
MLAGAN Elements and
MAVID Elements 
for additional information about methods used to generate these data.</P>

<H2>Verification</H2>
<P>
See the description pages for the TBA Elements, MLAGAN Elements
and MAVID Elements
for information about verification techniques used to generate these data.</P> 
<H2>Credits</H2>
<P>
The strict, moderate, and loose data shown in these subtracks were 
contributed by Elliott Margulies of NHGRI representing the ENCODE Multi-Species
Analysis group.</P>
<P>
<H4> Conservation Scoring </H4>
PhastCons was developed by 
<A TARGET=_BLANK HREF="http://siepellab.labsites.cshl.edu/">
Adam Siepel</A>,
Cold Spring Harbor Laboratory, while at the
<A HREF="https://ucscgenomics.soe.ucsc.edu/team/"
TARGET=_blank>Haussler Lab</A> at UCSC.</P>
<P>
BinCons was developed by Elliott Margulies, while at the
<A HREF="https://www.genome.gov/27535200/" TARGET=_blank>Eric Green lab</A>.
<P>
<A HREF="http://mendel.stanford.edu/sidowlab/downloads/GERP/index.html"
TARGET=_blank>GERP</A>
was developed primarily by Greg Cooper in the lab of
<A HREF="http://mendel.stanford.edu/sidowlab/" TARGET=_blank>Arend Sidow</A>
at Stanford University (Depts of Pathology and Genetics), in close collaboration
with Eric Stone (Biostatistics, NC State), and George Asimenos and
Eugene Davydov in the lab of
<A HREF="https://www.serafimb.org/people.html/" TARGET=_blank>Serafim Batzoglou</A>
(Dept. of Computer Science, Stanford).</P>
<P>
<H4> Sequence Alignment </H4>
TBA and Blastz were developed by Minmei Hou, Scott Schwartz and Webb Miller of the 
<A HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank>Penn State Bioinformatics 
Group</A>.
<P>
<A HREF="http://lagan.stanford.edu/lagan_web/index.shtml" TARGET=_blank>MLAGAN</A>,
Shuffle-LAGAN, and SuperMap were written by Mike Brudno 
while at the Batzoglou lab.
<A HREF="http://www.drive5.com/muscle/" TARGET=_BLANK>
MUSCLE</A>
was authored by Bob Edgar.
<A HREF="https://www.advbiocomp.com/blast.html" TARGET=_blank>AB-BLAST</A> 
was provided by the 
<A HREF="https://wustl.edu/" TARGET=_blank>Gish lab</A>
at the School of Medicine, University of Washington in St. Louis.</P>
<P>
<A TARGET=_BLANK HREF="https://www.biostat.wisc.edu/~cdewey/mercator/">
Mercator</A>
was written by Colin Dewey and Lior Pachter at the
<A TARGET=_BLANK HREF="http://pachterlab.github.io/group.html">
Pachter Lab Comparative Genomics Group</A> at UC Berkeley.
<A TARGET=_BLANK HREF="pachterlab.github.io/software.html">
MAVID</A>
was authored by Nicholas Bray and Lior Pachter.</P>
<P>
The phylogenetic tree is based on 
<A HREF="https://www.nature.com/articles/35054550"
TARGET=_BLANK>
Murphy et al. (2001)</A>
and general consensus in the vertebrate phylogeny community.
</P>
<H2>References</H2>
<P>
Blanchette, M., Kent, W.J., Reimer, C., Elnitski, L., Smit, A.,
Roskin, K., Baertsch, R., Rosenbloom, K.R., Clawson, H. <em>et al.</em>
<A HREF="https://genome.cshlp.org/content/14/4/708"
TARGET=_blank>Aligning Multiple Genomic Sequences With the Threaded Blockset 
Aligner</A>.
<em>Genome Res</em> <B>14</B>(4), 708-15 (2004).</P>
<P>
Bray, N. and Pachter, L. 
<A HREF="https://genome.cshlp.org/content/14/4/693.abstract"
TARGET=_blank>MAVID: Constrained Ancestral Alignment of Multiple Sequences</A>.
<em>Genome Res</em> <B>14</B>(4), 693-99 (2004).</P>
<P>
Brudno, M., Do, C., Cooper, G., Kim, M.F., Davydov, E., NISC Comparative 
Sequencing Program, Green, E.D., Sidow, A. and Batzoglou, S. 
<A HREF="https://genome.cshlp.org/content/13/4/721.abstract"
TARGET=_blank>LAGAN and Multi-LAGAN: efficient tools for large-scale multiple 
alignment of genomic DNA</A>.
<em>Genome Res</em> <B>13</B>(4), 721-31 (2003).</P>
<P>
Brudno, M., Malde, S., Poliakov, A., Do, C., Couronne, O., Dubchak, I. and 
Batzoglou, S.
<A HREF="https://academic.oup.com/bioinformatics/article/19/suppl_1/i54/227687"
TARGET=_blank>Global alignment: finding rearrangements during alignment</A>.
<em>Bioinformatics</em> <B>19</B>(Suppl. 1), i54-i62 (2003).</P>
<P>
Burge, C. and Karlin, S. 
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=9149143"
TARGET=_blank>Prediction of complete gene structures in human genomic DNA</A>.
<em>J Mol Biol</em> <B>268</B>(1), 78-94 (1997).</P>
<P>
Chiaromonte, F., Yap, V.B., and Miller, W. 
<A HREF="http://psb.stanford.edu/psb-online/proceedings/psb02/chiaromonte.pdf" 
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>.
<em>Pac Symp Biocomput</em>, 115-26 (2002).</P>
<P>
Cooper, G.M., Stone, E.A., Asimenos, G., NISC Comparative Sequencing Program,
Green, E.D., Batzoglou, S. and Sidow, A.
<A HREF="https://genome.cshlp.org/content/15/7/901.full"
TARGET=_blank>Distribution and intensity of constraint in mammalian genomic 
sequence</A>.
<em>Genome Res</em> <B>15</B>(7), 901-13 (2005).</P>
<P>
Dewey, C.N. and Pachter, L. 
<A HREF="https://www.biostat.wisc.edu/~cdewey/mercator/"
TARGET=_blank>Mercator: multiple whole-genome orthology  map construction</A>.
<em>In preparation</em>. </P>
Edgar, R.C. 
<A HREF="https://academic.oup.com/nar/article/32/5/1792/2380623"
TARGET=_blank>MUSCLE: multiple sequence alignment with high
accuracy and high throughput</A>.
<em>Nucl Acids Res</em> <B>32</B>(5), 1792-97 (2004).</P>
<P>
Kent, W.J. 
<A HREF="https://genome.cshlp.org/content/12/4/656.full"
TARGET=_blank>BLAT-the BLAST-like alignment tool</A>.
<em>Genome Res</em> <B>12</B>(4), 656-664 (2002).</P>
<P>
Kurtz, S., Phillippy, A., Delcher, A.L., Smoot, M., Shumway, M., Antonescu, C. 
and Salzberg, S.L. 
<A HREF="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2004-5-2-r12"
TARGET=_blank>Versatile and open software for comparing large genomes</A>.
<em>Genome Biol</em> <B>5</B>(2), R12 (2004). </P>
<P>
Margulies, E.H., Blanchette, M., NISC Comparative Sequencing Program, 
Haussler, D. and Green, E.D. 
<A HREF="https://genome.cshlp.org/content/13/12/2507"
TARGET=_blank>Identification and characterization of multi-species conserved 
sequences</A>. 
<em>Genome Res</em> <B>13</B>(12), 2507-18 (2003). </P>
<P>
Murphy, W.J., et al.
<A HREF="https://science.sciencemag.org/content/294/5550/2348"
TARGET=_blank>Resolution of the early placental mammal radiation using Bayesian phylogenetics</A>.  <em>Science</em> <B>294</B>(5550), 2348-51 (2001).</P>
<P>
Schwartz, S., Kent, W.J., Smit, A., Zhang, Z., Baertsch, R., Hardison, R., 
Haussler, D. and Miller, W.
<A HREF="https://genome.cshlp.org/content/13/1/103.abstract" 
TARGET=_blank>Human-Mouse Alignments with BLASTZ</A>.
<em>Genome Res</em> <B>13</B>(1), 103-7 (2003).</P>
<P>
Siepel, A., Bejerano, G., Pedersen, J.S., Hinrichs, A., Hou, M.,
Rosenbloom, K., Clawson, H., Spieth, J., Hillier, L.W. <em>et al.</em>
<A HREF="https://genome.cshlp.org/content/15/8/1034"
TARGET=_blank>Evolutionarily conserved elements in vertebrate,
insect, worm, and yeast genomes</A>.  
<em>Genome Res</em> <B>15</B>(8), 1034-50 (2005).</P>

