<H2>Description</H2>
<P>
This track displays human-centric multiple sequence alignments in the 
ENCODE regions for the 28 vertebrates included in the 
<A TARGET=_BLANK HREF="http://hgdownload.soe.ucsc.edu/goldenPath/hg17/encode/alignments/SEP-2005/">September 2005 ENCODE MSA freeze</A>,
based on comparative sequence data generated for the ENCODE project
as well as whole-genome assemblies residing at UCSC, as listed:
<UL>
<LI>human (May 2004, hg17)
    <LI>armadillo (NISC and May 2005 Broad Assisted Assembly v 1.0)
    <LI>baboon (NISC)
    <LI>chicken (Feb 2004, galGal2)
    <LI>chimp (Nov 2003, panTro1)
    <LI>colobus_monkey (NISC)
    <LI>cow (BCM)
    <LI>dog (July 2004, canFam1)
    <LI>dusky_titi (NISC)
    <LI>elephant (NISC and May 2005 Broad Assisted Assembly v 1.0)
    <LI>fugu (Aug 2002, fr1)
    <LI>galago (NISC)
    <LI>hedgehog (NISC)
    <LI>macaque (Jan 2005, rheMac1)
    <LI>marmoset (NISC)
    <LI>monodelphis (Oct 2004, monDom1)
    <LI>mouse (Mar 2005, mm6)
    <LI>mouse_lemur (NISC)
    <LI>owl_monkey (NISC)
    <LI>platypus (NISC and Aug 2005 Mullikin Phusion Assembly of WUGSC Traces)
    <LI>rabbit (NISC and May 2005 Broad Assisted Assembly v 1.0)
    <LI>rat (June 2003, rn3)
    <LI>rfbat (NISC)
    <LI>shrew (NISC and Sep 2005 Mullikin Phusion Assembly of Broad Traces)
    <LI>tenrec (Apr 2005 Mullikin Phusion Assembly of Broad Traces)
    <LI>tetraodon (Feb 2004, tetNig1)
    <LI>xenopus (Oct 2004, xenTro1)
    <LI>zebrafish (June 2004, danRer2)
</UL>
The alignments in this track were generated using the
Threaded Blockset Aligner (TBA).
The Genome Browser companion tracks, TBA Cons and TBA Elements, display 
conservation scoring and conserved elements for these alignments based on 
various conservation methods.</P>

<H2>Display Conventions and Configuration</H2>
<P>
In full display mode, this track shows pairwise alignments
of each species aligned to the human genome.  
In dense mode, the alignments are depicted using a gray-scale
density gradient.  The checkboxes in the track configuration section allow
the exclusion of species from the pairwise display.
<P>
When zoomed-in to the base-display level, the track shows the base
composition of each alignment. The numbers and symbols on the 
&quot;Gaps&quot; line indicate the lengths of gaps in the human sequence at those
alignment positions relative to the longest non-human sequence. If there is
sufficient space in the display, the size of the gap is shown; if not, and if
the gap size is a multiple of 3, a &quot;*&quot; is displayed, 
otherwise &quot;+&quot; is shown.
To view detailed information about the
alignments at a specific position, zoom in the display to 30,000 or fewer 
bases, then click on the alignment.  </P>

<H2>Methods</H2>
<P>
The TBA was used to align sequences in the September 2005 ENCODE sequence data 
freeze. Multiple alignments were seeded from a series of combinatorial pairwise 
blastz alignments (not referenced to any one species). The specific 
combinations were determined by the
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/hg17/encode/alignments/SEP-2005/phylo/tree_4d.tba.v2.nh"
TARGET=_blank>species guide tree</A>. 
The resulting multiple alignments were projected onto the human reference
sequence. </P>

<H2>Credits</H2>
<P>
The TBA multiple alignments were created by Elliott Margulies of NHGRI,
while at the <A HREF="https://www.genome.gov/27535200/" TARGET=_blank>Green Lab</A>.
</P>
<P>
The programs Blastz and TBA, which were used to generate the alignments, were
provided by Minmei Hou, Scott Schwartz and Webb Miller of the 
<A HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank>Penn State Bioinformatics 
Group</A>.</P>
<P>The phylogenetic tree is based on Murphy <em>et al.</em> (2001).
</P>

<H2>References</H2>
<P>
Blanchette M, Kent WJ, Reimer C, Elnitski L, Smit A,
Roskin K, Baertsch R, Rosenbloom KR, Clawson H <em>et al.</em>
<A HREF="https://genome.cshlp.org/content/14/4/708.full"
TARGET=_blank>Aligning Multiple Genomic Sequences With the Threaded Blockset 
Aligner</A>.
<em>Genome Res</em>. 2004;14(4):708-15.</P>
<P>
Chiaromonte F, Yap VB, Miller W. 
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=11928468&dopt=Abstract" 
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>.
<em>Pac Symp Biocomput</em>. 2002;115-26.</P>
<P>
Murphy WJ, Eizirik E, O'Brien SJ, Madsen O, Scally M, Douady CJ, Teeling E,
Ryder OA, Stanhope MJ, de Jong WW <em>et al.</em>
<A HREF="https://science.sciencemag.org/content/294/5550/2348"
TARGET=_blank>Resolution of the early placental mammal radiation using Bayesian phylogenetics</A>.
<em>Science</em>. 2001;294(5550):2348-51.</P>
<P>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison R, Haussler D, Miller W.
<A HREF="https://genome.cshlp.org/content/13/1/103.full" 
TARGET=_blank>Human-Mouse Alignments with BLASTZ</A>.
<em>Genome Res</em>. 2003;13(1):103-7.</P>
