<H2>Description</H2>
<P>
This track displays different measurements of conservation based on 
the multiple sequence alignments of ENCODE regions generated by the
Threaded Blockset Aligner (TBA) and shown in the TBA Alignment track.
The conservation scoring used to create this track was generated by
three programs: 
<UL>
<LI>phastCons (phylogenetic hidden-Markov model method)
<LI><A HREF="http://mendel.stanford.edu/sidowlab/downloads/GERP/index.html"
TARGET=_blank>GERP</A> (Genomic Evolutionary Rate Profiling)
<LI>SCONE (from Harvard Genetics)
</UL>
A related 
track, TBA Elements, shows multi-species conserved sequences (MCSs) based on
the conservation measurements displayed in this track.</P>
<P>
For details on the conservation scores generated by each program, refer to the 
individual Methods subsections.</P>

<H2>Display Conventions and Configuration</H2>
<P>
The subtracks within this composite annotation track may be configured in a 
variety of ways to highlight different aspects of the 
displayed data. The graphical configuration options 
are shown at the top of the track description page, followed by a list of 
subtracks.  A subtrack may be hidden from view by checking the box to the
left of the track name in the list.  For more information about the 
graphical configuration options, click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph
configuration help</A> link.</P>
<P>
Color differences among the subtracks are arbitrary; they provide a
visual cue for distinguishing the different gene prediction methods. See the
Methods section for display information specific to each subtrack.</P>

<H2>Methods</H2>
<P>
The methods used to create the TBA alignments in the ENCODE
regions are described in the TBA Alignment track description.</P>

<H3>PhastCons</H3>
<P>
The phastCons program predicts conserved elements and produces base-by-base
conservation scores using a two-state phylogenetic hidden Markov model.
The model consists of a state for conserved regions and a 
state for nonconserved regions, each of which is associated with a 
phylogenetic model.  These two models are identical
except that the branch lengths of the conserved phylogeny are 
multiplied by a scaling parameter <em>rho</em> (0 &lt; <em>rho</em> &lt; 1).</P>
<P>
For determining the conservation for the ENCODE 
alignments,  the nonconserved model was estimated 
from four-fold degenerate coding sites within the ENCODE regions using 
the program phyloFit. The parameter <em>rho</em> was then estimated by 
maximum likelihood, conditional on the nonconserved model, using the EM 
algorithm implemented in phastCons.  Parameter estimation was based on 
a single large alignment, constructed by concatenating the 
alignments for all conserved regions.</P>
<P>
PhastCons was run with the options <TT>--expected-lengths 15</TT> and
<TT>--target-coverage 0.01</TT> to obtain the desired level of 
&quot;smoothing&quot; and a final coverage by conserved elements of 5%.</P>
<P>
The conservation score at each base is the posterior probability that the
base was generated by the conserved state of the phylo-HMM.  It can
be interpreted as the probability that the base is in a conserved
element, given the assumptions of the model and the estimated parameters.
Scores range from 0 to 1, with higher scores corresponding to
higher levels of conservation.  </P>
<P>
More details on phastCons can be found in Siepel <em>et. al.</em> (2005)
cited below.</P>

<H3>GERP</H3>
<P>
The GERP score is the expected substitution rate minus the observed substitution
rate at a particular human base. Scores are estimated on a column-by-column 
basis using multiple sequence alignments of mammalian genomic DNA.
The scores are both positive and negative, with negative values (<em>i.e.</em> 
observed &gt; expected) corresponding to neutral or unconstrained sites and 
positive values (<em>i.e.</em> observed &lt; expected) corresponding to 
constrained or slowly evolving sites. 
The expected and observed rates are both calculated on a phylogenic tree using 
the same fixed topology. 
The branch lengths of the expected tree are based on the average substitutions 
at neutral sites. 
The branch lengths of the observed tree, which is calculated separately for each
human base, are based on the substitutions seen at the column of the multiple 
alignment at that base. 
Species that have gaps at a particular column are not considered in the scoring 
for that column.</P>
<P>
Higher scores correspond to human
bases in alignment columns with higher degrees of similarity, <em>i.e.</em>
bases that have evolved slowly, some of which have been under purifying 
selection. The opposite holds true for swiftly evolving (low similarity) 
columns. </P>
<P>
Scores are deterministic, given a maximum-likelihood model of
nucleotide substitution, species topology, neutral tree, and alignment.</P>

<H3>SCONE</H3>
<P>
SCONE is a probabilistic measure of purifying selection expressed as a
p-value that a given position evolves neutrally. It has a model of
evolution that considers both sequence-contextual effects on
substitution rates and insertion/deletion events. This model may be
used to compute the probability of any transitional event along a
lineage.</P>
<P>
The score is computed for any column in a multiple sequence alignment
by first parsimoniously inferring the evolutionary history of the
site, using a given phylogenetic tree with known branch-lengths.
Subsequently, transition probabilities are computed for each branch in
the tree. A heuristic score is computed using the formula:
<BLOCKQUOTE>
S = ln(product(all i in M)/product(all j in C))
</BLOCKQUOTE>
where M and C are the set of all branches in the tree that contain mutations and the set of all branches in 
the tree that do not contain mutations, respectively. This heuristic score
serves to effectively partition sites according to the influence of
purifying selection on the site.</P>
<P>
This heuristic score is used to compute a p-value by comparing it
against the expected distribution of neutral scores as determined by
Monte-Carlo simulation. Forward simulation of evolution is performed
along the phylogenetic tree using the SCONE model of mutation events, and
the above heuristic score is computed for a simulated tree. Repeated
simulation produces a distribution of scores that reflects the
neutral expected distribution. A p-value score may be computed by
counting the fraction of simulated heuristic scores that fall below
the heuristic score for the site.</P>

<H2>Credits</H2>
<P>
PhastCons was developed by 
<A HREF="http://siepellab.labsites.cshl.edu/" TARGET=_BLANK>
Adam Siepel</A>, Cold Spring Harbor Laboratory, while at the
<A HREF="https://ucscgenomics.soe.ucsc.edu/team/"
TARGET=_blank>Haussler lab</A> at UCSC.</P>
<P>
GERP was developed primarily by Greg Cooper in the lab of
<A HREF="http://mendel.stanford.edu/sidowlab/" TARGET=_blank>Arend Sidow</A>
at Stanford University
(Depts of Pathology and Genetics), in close collaboration with
Eric Stone (Biostatistics, NC State), and George Asimenos and
Eugene Davydov in the lab of
<A HREF="https://www.serafimb.org/people.html/" TARGET=_blank>Serafim Batzoglou</A>
(Dept. of Computer Science, Stanford).</P>
<P>
SCONE was developed by Saurabh Asthana in the lab of <a
href=http://genetics.bwh.harvard.edu/genetics/labs/Sunyaev/index.html>Shamil
Sunyaev</a> at Harvard Medical School and Brigham &amp; Women's Hospital
(Department of Medicine/Division of Genetics).</P>
<P>
TBA was provided by Minmei Hou, Scott Schwartz and Webb Miller of the 
<A HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank>Penn State Bioinformatics 
Group</A>.</P>
<P>
The GERP data for this track was generated by Greg Cooper.
The PhastCons data was generated by Elliott Margulies, 
with assistance from Adam Siepel. 
The SCONE data was generated by Saurabh Asthana.</P>

<H2>References</H2>
<P>
Asthana S, Roytberg M, Stamatoyannopoulos J, Sunyaev S.
<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0030254"
target="_blank">Analysis of Sequence Conservation at Nucleotide Resolution</a>.
<em>PLoS Comput. Biol.</em> 2007 Dec 28:3(12):e254.</P>
<P>
Blanchette M, Kent WJ, Reimer C, Elnitski L, Smit A,
Roskin K, Baertsch R, Rosenbloom KR, Clawson H, Green ED, <em>et al.</em>
<A HREF="https://genome.cshlp.org/content/14/4/708"
TARGET=_blank>Aligning Multiple Genomic Sequences With the Threaded Blockset 
Aligner</A>.
<em>Genome Res.</em> 2004 Apr:14(4):708-15.</P>
<P>
Cooper GM, Stone EA, Asimenos G, NISC Comparative Sequencing Program,
Green ED, Batzoglou , Sidow A.
<A HREF="https://genome.cshlp.org/content/15/7/901.full"
TARGET=_blank>Distribution and intensity of constraint in mammalian genomic 
sequence</A>.
<em>Genome Res.</em> 2005 Jul;15(7):901-13. Epub 2005 Jun 17.</P>
<P>
Margulies EH, Blanchette M, NISC Comparative Sequencing Program, 
Haussler D, Green ED. 
<A HREF="https://genome.cshlp.org/content/13/12/2507"
TARGET=_blank>Identification and characterization of multi-species conserved 
sequences</A>. 
<em>Genome Res.</em> 2003 Dec;13(12):2507-18. </P>
<P>
Siepel A, Bejerano G, Pedersen JS, Hinrichs A, Hou M,
Rosenbloom K, Clawson H, Spieth J, Hillier LW, Richards S, <em>et al.</em>
<A HREF="https://genome.cshlp.org/content/15/8/1034"
TARGET=_blank>Evolutionarily conserved elements in vertebrate,
insect, worm, and yeast genomes</A>.  
<em>Genome Res.</em> 2005 Aug;15(8):1034-50. Epub 2005 Jul 15.</P>
