<H2>Description</H2>
<P>
ENCODE region-wide location analysis of H3 and H4 histones
was conducted employing ChIP-chip using chromatin extracted from 
<A HREF="http://ccr.coriell.org/Sections/Search/Sample_Detail.aspx?Ref=GM06990&PgId=166"
	TARGET=_blank>GM06990</A> (lymphoblastoid), 
   <A href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=789258&dopt=Abstract"
	TARGET=_blank>K562</A> (myeloid leukemia-derived),
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=13286432&query_hl=3" TARGET=_BLANK>
HeLaS3</A> (cervix carcinoma), 
<A HREF="http://www.biotech.ist.unige.it/cldb/cl1670.html" TARGET=_BLANK>
HFL-1</A> (embryonic lung fibroblast),
<A HREF="http://www.biotech.ist.unige.it/cldb/cl3531.html" TARGET=_BLANK>
MOLT-4</A> (lymphoblastic leukemia), and
PTR8 cells.
Experiments were conducted with antibodies to the following histones:
<A HREF="http://www.abcam.com/index.html?datasheet=8895" 
    TARGET=_blank>H3K4me1</A>,
<A HREF="http://www.abcam.com/index.html?datasheet=7766" 
    TARGET=_blank>H3K4me2</A>,
<A HREF="http://www.abcam.com/index.html?datasheet=8580"
    TARGET=_blank>H3K4me3</A>,
<A HREF="http://www.upstate.com/browse/productdetail.asp?ProductId=07-523"
    TARGET=_blank>H3K9me3</A>,
<A HREF="http://www.upstate.com/browse/productdetail.asp?ProductId=05-851"
    TARGET=_blank>H3K27me3</A>,
<A HREF="http://www.abcam.com/index.html?datasheet=9050"
    TARGET=_blank>H3K36me3</A>,
<A HREF="http://www.abcam.com/index.html?datasheet=2621"
    TARGET=_blank>H3K79me3</A>,
<A HREF="http://www.upstate.com/browse/productdetail.q.productid.e.06-599"
    TARGET=_blank>H3ac</A>,
<A HREF="http://www.upstate.com/browse/productdetail.q.productid.e.06-866"
    TARGET=_blank>H4ac</A>, and
<A HREF="http://www.scbt.com/product.php?datasheet=15914"
    TARGET=_blank>CTCF</A>.
</P>

<P>
Histone methylation and acetylation serves as a stable genomic imprint
that regulates gene expression and other epigenetic 
phenomena.  These histones are found in transcriptionally active domains 
called euchromatin.</P>

<H2>Display Conventions and Configuration</H2>
<P>
This annotation follows the display conventions for composite 
&quot;wiggle&quot; tracks. The subtracks within this annotation 
may be configured in a variety of ways to highlight different aspects of the 
displayed data. The graphical configuration options are shown at the top of 
the track description page, followed by a list of subtracks.  To display only 
selected subtracks, uncheck the boxes next to the tracks you wish to hide. 
For more information about the graphical configuration options, click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph
configuration help</A> link.</P>

<H2>Methods</H2>
<P>
Chromatin from the cell line was cross-linked with 1% formaldehyde,
precipitated with antibody binding to the histone, and sheared and hybridized to
the 
<A HREF="ftp://ftp.sanger.ac.uk/pub/encode/microarrays/ENCODE3.1.1.hg16.adf"
    TARGET=_BLANK>
Sanger ENCODE3.1.1</A>
DNA microarray.
DNA was not amplified prior to hybridization.</P>
<P>
The raw and transformed data files reflect fold enrichment over background,
averaged over six replicates.</P>

<H2>Verification</H2>
<P>                   
There are six replicates: two technical replicates (immunoprecipitations)
for each of the three biological replicates (cell cultures).</P>
<P>
Raw and transformed (averaged) data can be downloaded from the Wellcome
Trust Sanger Institute via the 
<A HREF="http://www.sanger.ac.uk/PostGenomics/encode/data-access.shtml"
    TARGET=_BLANK>
ENCODE data access web site</A> or the
<A HREF="ftp://ftp.sanger.ac.uk/pub/encode" TARGET=_BLANK>
ENCODE FTP site</A>.
</P>

<H2>Credits</H2>
<P>
The data for this track were generated by the 
<A HREF="http://www.sanger.ac.uk/PostGenomics/encode/investigators.shtml"
TARGET=_blank>ENCODE investigators</A> at the 
<A HREF="http://www.sanger.ac.uk/" TARGET=_blank>Wellcome Trust Sanger 
Institute</a>, Hinxton, UK.</P>
