<H2>Description</H2>
<P>
This track displays human-centric multiple sequence alignments in the 
ENCODE regions for the 28 vertebrates included in the 
<A TARGET=_BLANK HREF="http://hgdownload.soe.ucsc.edu/goldenPath/hg17/encode/alignments/SEP-2005/">September 2005 ENCODE MSA freeze</A>,
based on comparative sequence data generated for the ENCODE project
as well as whole-genome assemblies residing at UCSC, as listed:
<UL>
    <LI>human (May 2004, hg17)
    <LI>chimp (Nov 2003, panTro1)
    <LI>colobus_monkey (NISC)
    <LI>baboon (NISC)
    <LI>macaque (Jan 2005, rheMac1)
    <LI>dusky_titi (NISC)
    <LI>owl_monkey (NISC)
    <LI>marmoset (NISC)
    <LI>mouse_lemur (NISC)
    <LI>galago (NISC)
    <LI>rat (June 2003, rn3)
    <LI>mouse (Mar 2005, mm6)
    <LI>rabbit (NISC and May 2005 Broad Assisted Assembly v 1.0)
    <LI>cow (BCM)
    <LI>dog (July 2004, canFam1)
    <LI>rfbat (NISC)
    <LI>hedgehog (NISC)
    <LI>shrew (NISC and Sep 2005 Mullikin Phusion Assembly of Broad Traces)
    <LI>armadillo (NISC and May 2005 Broad Assisted Assembly v 1.0)
    <LI>elephant (NISC and May 2005 Broad Assisted Assembly v 1.0)
    <LI>tenrec (Apr 2005 Mullikin Phusion Assembly of Broad Traces)
    <LI>monodelphis (Oct 2004, monDom1)
    <LI>platypus (NISC and Aug 2005 Mullikin Phusion Assembly of WUGSC Traces)
    <LI>chicken (Feb 2004, galGal2)
    <LI>xenopus (Oct 2004, xenTro1)
    <LI>tetraodon (Feb 2004, tetNig1)
    <LI>fugu (Aug 2002, fr1)
    <LI>zebrafish (June 2004, danRer2)
</UL>
</P>
<P>
The alignments in this track were generated using the 
<A HREF="http://lagan.stanford.edu/lagan_web/index.shtml" TARGET=_blank>LAGAN</A>
Alignment Toolkit. The Genome Browser companion tracks, MLAGAN Cons and MLAGAN Elements, 
display conservation scoring and conserved elements for these alignments based 
on various conservation methods.</P>

<H2>Display Conventions and Configuration</H2>
<P>
In full display mode, this track shows pairwise alignments
of each species aligned to the human genome.  
In dense mode, the alignments are depicted using a gray-scale
density gradient.  The checkboxes in the track configuration section allow
the exclusion of species from the pairwise display.</P>
<P>
When zoomed-in to the base-display level, the track shows the base
composition of each alignment. The numbers and symbols on the &quot;Gaps&quot; 
line indicate the lengths of gaps in the human sequence at those
alignment positions relative to the longest non-human sequence. If there is
sufficient space in the display, the size of the gap is shown; if not, and if
the gap size is a multiple of 3, a &quot;*&quot; is displayed, 
otherwise &quot;+&quot; is shown.
To view detailed information about the
alignments at a specific position, zoom in the display to 30,000 or fewer
bases, then click on the alignment.  </P>

<H2>Methods</H2>
<P>
MLAGAN alignments were produced by a pipeline specifically designed for
ENCODE. First, <A HREF="https://www.advbiocomp.com/blast.html" TARGET=_blank>AB-BLAST</A> was 
used to find local similarities (anchors) 
between the human sequence and the sequence of every other species. Then, 
Shuffle-LAGAN was used to calculate the highest-scoring
human-monotonic 
chain of these local similarities (according to a scoring scheme that 
penalized evolutionary rearrangements), and &mdash; with the help of a utility 
called SuperMap &mdash; produce a map of orthologous segments, in increasing 
human coordinates. This map was used to undo the genomic rearrangements of the 
other sequence and convert it to a form that was directly alignable to the
human sequence. The new humanized sequences, together with the human 
sequence, were then multiply aligned using 
<A HREF="http://lagan.stanford.edu/lagan_web/index.shtml" TARGET=_blank>MLAGAN</A>.
The resulting alignments were subsequently refined using 
<A HREF="http://www.drive5.com/muscle/" TARGET=_blank>MUSCLE</A>, which 
processed small non-overlapping alignment windows and realigned them in an
iterative fashion, keeping the refined alignment if it had a better 
sum-of-pairs score than the original.  Finally, a pairwise refinement
round was performed, during which the pieces that had very low identity (in the 
induced pairwise alignments between human and each species) were removed 
from the alignment.</P>

<H2>Credits</H2>
<P>
The MLAGAN alignments were generated by George Asimenos from Stanford's ENCODE group.</P>
<P>
Shuffle-LAGAN, SuperMap and MLAGAN were written by Mike Brudno.</P>
<P>
MUSCLE was authored by Bob Edgar.</P>
<P>
WU-BLAST was provided by
the <A HREF="https://wustl.edu/" TARGET=_blank>Gish lab</A>
at the School of Medicine, University of Washington in St. Louis.</P>
<P>The phylogenetic tree is based on Murphy <em>et al.</em> (2001).

<H2>References</H2>
<P>
Brudno M, Do C, Cooper G, Kim MF, Davydov E, Green ED, Sidow A, Batzoglou S.
<A HREF="https://genome.cshlp.org/content/13/4/721.full"
TARGET=_blank>LAGAN and Multi-LAGAN: efficient tools for large-scale multiple 
alignment of genomic DNA</A>.
<em>Genome Res</em>. 2003;13(4):721-31.</P>
<P>
Brudno M, Malde S, Poliakov A, Do C, Couronne O, Dubchak I, Batzoglou S.
<A HREF="https://academic.oup.com/bioinformatics/article/19/suppl_1/i54/227687"
TARGET=_blank>Global alignment: finding rearrangements during alignment</A>.
<em>Bioinformatics</em>. 2003;19(Suppl. 1):i54-i62.</P>
<P>
Edgar RC. 
<A HREF="https://academic.oup.com/nar/article/32/5/1792/2380623"
TARGET=_blank>MUSCLE: multiple sequence alignment with high
accuracy and high throughput</A>.
<em>Nucl Acids Res</em>. 2004;32(5):1792-7.</P>
<P>
Murphy WJ, Eizirik E, O'Brien SJ, Madsen O, Scally M, Douady CJ, Teeling E,
Ryder OA, Stanhope MJ, de Jong WW <em>et al.</em>
<A HREF="https://science.sciencemag.org/content/294/5550/2348"
TARGET=_blank>Resolution of the early placental mammal radiation using Bayesian phylogenetics</A>.
<em>Science</em>. 2001;294(5550):2348-51.</P>
