<H2>Description</H2>
<P>
This track displays human-centric multiple sequence alignments in the 
ENCODE regions for the 28 vertebrates included in the 
<A TARGET=_BLANK HREF="http://hgdownload.soe.ucsc.edu/goldenPath/hg17/encode/alignments/SEP-2005/">September 2005 ENCODE MSA freeze</A>,
based on comparative sequence data generated for the ENCODE project
as well as whole-genome assemblies residing at UCSC, as listed:
<UL>
<LI>human (May 2004, hg17)
    <LI>armadillo (NISC and May 2005 Broad Assisted Assembly v 1.0)
    <LI>baboon (NISC)
    <LI>chicken (Feb 2004, galGal2)
    <LI>chimp (Nov 2003, panTro1)
    <LI>colobus_monkey (NISC)
    <LI>cow (BCM)
    <LI>dog (July 2004, canFam1)
    <LI>dusky_titi (NISC)
    <LI>elephant (NISC and May 2005 Broad Assisted Assembly v 1.0)
    <LI>fugu (Aug 2002, fr1)
    <LI>galago (NISC)
    <LI>hedgehog (NISC)
    <LI>macaque (Jan 2005, rheMac1)
    <LI>marmoset (NISC)
    <LI>monodelphis (Oct 2004, monDom1)
    <LI>mouse (Mar 2005, mm6)
    <LI>mouse_lemur (NISC)
    <LI>owl_monkey (NISC)
    <LI>platypus (NISC and Aug 2005 Mullikin Phusion Assembly of WUGSC Traces)
    <LI>rabbit (NISC and May 2005 Broad Assisted Assembly v 1.0)
    <LI>rat (June 2003, rn3)
    <LI>rfbat (NISC)
    <LI>shrew (NISC and Sep 2005 Mullikin Phusion Assembly of Broad Traces)
    <LI>tenrec (Apr 2005 Mullikin Phusion Assembly of Broad Traces)
    <LI>tetraodon (Feb 2004, tetNig1)
    <LI>xenopus (Oct 2004, xenTro1)
    <LI>zebrafish (June 2004, danRer2)
</UL></P>
<P>
The alignments in this track were generated using the 
<A TARGET=_BLANK HREF="https://www.biostat.wisc.edu/~cdewey/mercator/">
Mercator</A> 
orthology mapping program and the
<A TARGET=_BLANK HREF="pachterlab.github.io/software.html">
MAVID</A>
multiple global alignment program.
The Genome Browser companion tracks, MAVID Cons and MAVID Elements, 
display conservation scoring and conserved elements for these alignments based 
on various conservation methods.</P>

<H2>Display Conventions and Configuration</H2>
<P>
In full display mode, this track shows pairwise alignments
of each species aligned to the human genome.  
In dense mode, the alignments are depicted using a gray-scale
density gradient.  The checkboxes in the track configuration section allow
the exclusion of species from the pairwise display.</P>
<P>
When zoomed-in to the base-display level, the track shows the base
composition of each alignment. The numbers and symbols on the 
&quot;Gaps&quot; line indicate the lengths of gaps in the human sequence at 
those alignment positions relative to the longest non-human sequence. If there is
sufficient space in the display, the size of the gap is shown; if not, and if
the gap size is a multiple of 3, a &quot;*&quot; is displayed,
otherwise &quot;+&quot; is shown.
To view detailed information about the
alignments at a specific position, zoom in the display to 30,000 or fewer
bases, then click on the alignment.  </P>

<H2>Methods</H2>
<P>
Mercator was first used to identify the colinear and orthologous  
segments in the sequences given for each ENCODE region.  
Input to Mercator was generated by using Genscan to predict genes in all  
sequences, Blat to compare predicted coding exons, and MUMmer to identify 
non-coding exact matches between all 
pairs of sequences. The output of Mercator was a small-scale one-to-one 
orthology map for each ENCODE region, as well as a
set of alignment constraints based on matched landmarks (<em>e.g.</em>, exons
and long non-coding exact matches). </P> 
<P>
MAVID was then used to construct a global
multiple alignment of each colinear orthologous segment set   
specified in the orthology map.  As part of its input, MAVID used a
phylogenetic tree determined from alignments of four-fold degenerate
sites in the ENCODE regions.</P>

<H2>Credits</H2>
<P>
Generation of the MAVID alignments was engineered by Colin Dewey at the 
<A TARGET=_BLANK HREF="http://pachterlab.github.io/group.html">
Pachter Lab Comparative Genomics Group</A> at UC Berkeley.</P>
<P>
Mercator was written by Colin Dewey and Lior Pachter.</P>
<P>
MAVID was authored by Nicholas Bray and Lior Pachter.</P>
<P>The phylogenetic tree is based on Murphy et al. (2001).</P>

<H2>References</H2>
<P>
Bray, N. and Pachter, L. 
<A HREF="https://genome.cshlp.org/content/14/4/693.abstract"
TARGET=_blank>MAVID: Constrained Ancestral Alignment of Multiple Sequences</A>.
<em>Genome Res</em> <B>14</B>(4), 693-699 (2004).</P>
<P>
Burge, C. and Karlin, S. 
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=9149143"
TARGET=_blank>Prediction of complete gene structures in human genomic DNA</A>.
<em>J Mol Biol</em> <B>268</B>(1), 78-94 (1997).</P>
<P>
Dewey, C.N. and Pachter, L. 
<A HREF="https://www.biostat.wisc.edu/~cdewey/mercator/"
TARGET=_blank>Mercator: multiple whole-genome orthology  map construction</A>.
<em>In preparation</em>. </P>
<P>
Kent, W.J. 
<A HREF="https://genome.cshlp.org/content/12/4/656.full"
TARGET=_blank>BLAT-the BLAST-like alignment tool</A>.
<em>Genome Res</em> <B>12</B>(4), 656-664 (2002).</P>
<P>
Kurtz, S., Phillippy, A., Delcher, A.L., Smoot, M., Shumway, M., Antonescu, C. 
and Salzberg, S.L. 
<A HREF="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2004-5-2-r12"
TARGET=_blank>Versatile and open software for comparing large genomes</A>.
<em>Genome Biol</em> <B>5</B>(2), R12 (2004). </P>
<P>
Murphy, W.J., et al.
<A HREF="https://science.sciencemag.org/content/294/5550/2348"
TARGET=_blank>Resolution of the early placental mammal radiation using Bayesian phylogenetics</A>.  <em>Science</em> <B>294</B>(5550), 2348-51 (2001).</P>
