<H2>Description</H2>
<P>
The exoniphy program identifies evolutionarily conserved protein-coding
exons in a multiple alignment using a phylogenetic hidden Markov
model (phylo-HMM), a kind of statistical model that simultaneously
describes exon structure and exon evolution.  This track shows exoniphy
predictions for the human May 2004 (hg17), mouse Aug. 2005 (mm7), rat
Jun. 2003 (rn3), and dog May 2005 (canFam2) genomes, as aligned by the
multiz program.  For this track, only alignments on the &quot;syntenic net&quot;
between human and each other species were considered. 
</P>

<H2>Methods</H2>
<P>
For a description of exoniphy, see Siepel <em>et al.</em> (2004).
Multiz is described in Blanchette <em>et al.</em> (2004).
The alignment chaining methods behind the &quot;syntenic net&quot; are 
described in Kent <em>et al.</em> (2003).</P>

<H2>References</H2>
<P>
Blanchette, M. <em>et al.</em> 
<A HREF="https://genome.cshlp.org/content/14/4/708.abstract"
TARGET=_blank>Aligning multiple genomic sequences with the threaded
blockset aligner</A>. 
<em>Genome Res.</em> <B>14</B>, 708-175 (2004).</P>
<P>
Kent, W.J. <em>et al.</em> 
<A HREF="https://www.pnas.org/content/100/20/11484"
TARGET=_blank>Evolution's cauldron: duplication, deletion, and
rearrangement in the mouse and human genomes</A>.
<em>P. Natl. Acad. Sci. USA</em> <B>100</B>(20), 11484-11489 (2003).</P>
<P>
Siepel, A. and Haussler, D. 
<A HREF="http://compgen.cshl.edu/~acs/recomb2004.pdf"
TARGET=_blank>Computational identification of evolutionarily conserved
exons</A>. <em>RECOMB '04</em> (2004).</P>
