<H2>Description</H2> 
<P> 
<A HREF="https://www.ncbi.nlm.nih.gov/variation/news/NCBI_retiring_HapMap/"
TARGET=_blank>The HapMap Project</A>
identified a set of approximately four million
common SNPs, and genotyped these SNPs in four populations.
The intent is that this data can be used as a reference for future studies
of human disease. This track displays the genotype counts and allele
frequencies of those SNPs. The data displayed are from release 21a (HapMap 
Phase II), based on 
<A HREF="https://www.ncbi.nlm.nih.gov/SNP/index.html" TARGET=_blank>dbSNP</A>
build 125.   This track also provides orthologous alleles from chimp and 
macaque.
</P>
<P>
The HapMap data are from these four human populations:
<UL>
<LI>Yoruba in Ibadan, Nigeria (YRI)</LI>
<LI>Japanese in Tokyo, Japan (JPT)</LI>
<LI>Han Chinese in Beijing, China (CHB)</LI>
<LI>CEPH (Utah residents with ancestry from northern and western Europe) (CEU)</LI>
</UL>
Each of the populations is displayed in a separate subtrack.
</P>
<P>
The CEU and YRI data are comprised of 90 individuals in parent-child trios. The UCSC display
removes the data for the children, leaving 60 individuals in each population.
The CHB and JPT data are comprised of 45 individuals. Over 12% of HapMap SNPs
are available for only a subset (1-3) of the populations. When 
available, the CHB and JPT SNPs were assayed in a minimum of 18 individuals,
with over 97% of SNPs assayed in 45 or more individuals. The minimums for CEU
and YRI are 26 and 24 respectively, with over 94% of SNPs assayed in 55 or more
individuals.
</P>
<P>
The HapMap assays provide biallelic results. Over 99.9% of HapMap SNPs are
included in dbSNP build125 as biallelic; approximately 3,000 are more complex.
Two-thirds of the HapMap SNPs are transitions: one-third are A/G, one-third
are C/T.
</P>
<P>
The orthologous alleles in chimp (panTro2) and macaque (rheMac2) 
were derived using 
<A HREF="http://genome.soe.ucsc.edu/goldenPath/help/hgTracksHelp.html#Convert">liftOver</A>.
Chimp alleles are available for over 96% 
of the human HapMap SNPs; macaque alleles are available for 88%.
</P>
<P>
15% of HapMap SNPs are monomorphic in all individuals in all populations.
Within single populations, 21.5% of the SNPs are monomorphic in YRI
and 38% of the SNPs are monomorphic in JPT individuals.
</P>
<P>
Approximately 20% of HapMap SNPs have a different major allele in different
populations.
</P>
<P>
No two HapMap SNPs occupy the same position. Aside from seven SNPs from the 
pseudo
autosomal region of chrX, no rsIds are included more than once. No HapMap SNPs
occur on chrM or on "random" chromosomes.
</P>

<H2>Display Conventions and Configuration</H2>
<P>
<B>Note: calculation of heterozygosity has changed since this version of 
the track.</B>
In this track, <em>expected</em> heterozygosity is calculated 
as follows: the allele counts from all populations are summed 
(not normalized for population size)
and used to determine overall major and minor allele frequencies.  
Assuming Hardy-Weinberg equilibrium, overall expected heterozygosity
is calculated as two times the product of major and minor allele
frequencies 
(see <A HREF="https://www.ncbi.nlm.nih.gov/books/NBK21425/box/A2510/"
TARGET=_BLANK><em>Modern Genetic Analysis</em>, section 17-2</A>).
[In the HapMap SNPs track in the Mar. 2006 (hg18) assembly,
<em>observed</em> heterozygosity is calculated as follows: each population's
heterozygosity is computed as the proportion of heterozygous individuals in 
the population.  The population heterozygosities are averaged to determine the 
overall observed heterozygosity.]
</P>
<P>
The human SNPs are displayed in gray using a color gradient based on minor allele
frequency. The higher the minor allele frequency, the darker the display. 
By definition, the maximum minor allele frequency is 50%. 
When zoomed to base level, the major allele is displayed for each population. 
</P>
<P>
Reversing the <A HREF="hgTrackUi?g=ruler">base position track</A> 
will cause the HapMap display to reverse as
well. This is the recommended configuration for SNPs on the negative strand.
</P>
<P>
The orthologous alleles from chimp and macaque are displayed in brown using a color 
gradient based on quality score.
Quality scores range from 0 to 100 representing low to high quality. For 
orthologous alleles, the higher the quality, the darker the display. Quality 
scores are not available for chimp chromosomes chr21 and chrY; these were set to
98, consistent with the panTro2 browser quality track.
</P>
<P>
Filters are provided for the data attributes described above. Additionally,
a filter is provided for heterozgosity over all populations. The measure of
heterozygosity used is 2pq (from Hardy-Weinberg equilibrium). 
Filters are applied to all six subtracks.  This is true, even if a subtrack 
is not displayed.
<P>
Notes on orthologous allele filters:
<UL>
<LI>If the major allele is different between populations, no overall major
allele for human is determined, thus the &quot;matching&quot; filters for
orthologous alleles do not apply to these SNPs.</LI>
<LI>If a SNP is monomorphic in all populations, the minor allele is not
verified in the HapMap dataset.  In these cases, the filter to match
orthologous alleles to the minor human allele will yield no results.</LI>
</UL>
</P>

<H2>Credits</H2>
<P>
This track is based on <A HREF="https://www.ncbi.nlm.nih.gov/variation/news/NCBI_retiring_HapMap/"
TARGET=_BLANK>International HapMap Project</A> release 21a data, provided by the <A
HREF="https://www.ncbi.nlm.nih.gov/variation/news/NCBI_retiring_HapMap/"
TARGET=_BLANK>HapMap Data Coordination Center</A>. 
</P>

<H2>References</H2>

<H3>HapMap Project</H3>
<P>
The International HapMap Consortium. 
<A HREF="https://www.nature.com/articles/nature06258"
TARGET=_BLANK>A second generation human haplotype map of over 3.1 million SNPs</A>.
<em>Nature</em>. 2007 Oct 18;449(7164):851-61.</P>
<P>
The International HapMap Consortium.
<A HREF="https://www.nature.com/articles/nature04226"
TARGET=_blank>A haplotype map of the human genome</A>.
<em>Nature</em>. 2005 Oct 27;437(7063):1299-320.</P> 
<P>
The International HapMap Consortium.
<A HREF="https://www.nature.com/articles/nature02168"
TARGET=_blank>The International HapMap Project</A>.
<em>Nature</em>. 2003 Dec 18;426(6968):789-96.</P>

<H3>HapMap Data Coordination Center</H3>
<P>
Thorisson GA, Smith AV, Krishnan L, Stein LD.
<A HREF="https://genome.cshlp.org/content/15/11/1592.abstract"
TARGET=_blank>The International HapMap Project Web site</A>.
<em>Genome Res</em>. 2005 Nov;15(11):1592-3.</P>

<H3>A Sampling of HapMap Literature</H3>
<P>
Gibson J, Morton NE, Collins A.
<A HREF="https://academic.oup.com/hmg/article/15/5/789/596723"
TARGET=_blank>Extended tracts of homozygosity in outbred human populations</A>.
<em>Hum Mol Genet</em>. 2006 Mar 1; 15(5):789-95.
</P>
<P>
Redon R, Ishikawa S, Fitch KR, Feuk L, Perry GH, Andrews TD, Fiegler H, Shapero
MH, Carson AR, Chen W <em>et al</em>.
<A HREF="https://www.nature.com/articles/nature05329"
TARGET=_blank>Global variation in copy number in the human genome</A>.
<em>Nature</em>. 2006 Nov 23;444(7118):444-454.
<P>
Spielman RS, Bastone LA, Burdick JT, Morley M, Ewens WJ, Cheung VG.
<A HREF="https://www.nature.com/articles/ng1955"
TARGET=_blank>Common genetic variants account for differences in gene
expression among ethnic groups</A>. <em>Nature Genet</em>. 2007
Feb;39(2):226-31.
</P>
<P>
Tenesa A, Navarro P, Hayes BJ, Duffy DL, Clarke GM, Goddard ME, Visscher PM.
<A HREF="https://genome.cshlp.org/content/17/4/520.abstract"
TARGET=_blank>Recent human effective population size estimated from linkage
disequilibrium</A>. <em>Genome Res</em>. 2007 Apr;17(4):520-6.
</P>
Voight BF, Kudaravalli S, Wen X, Pritchard JK.
<A HREF="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.0040072"
TARGET=_blank>A Map of Recent Positive Selection in the Human Genome</A>.
<em>PLoS Biol</em>. 2006 Mar;4(3):e72.</P>
<P>
Weir BS, Cardon LR, Anderson AD, Nielsen DM, Hill WG.
<A HREF="https://genome.cshlp.org/content/15/11/1468.abstract" TARGET=_blank>
Measures of human population structure show heterogeneity among genomic
regions</A>. <em>Genome Res</em>. 2005 Nov;15(11):1468-76.
</P>

<H3>Data Source</H3>
<P>
The source for this track are the genotypes_chr*_*_r21a_nr.txt.gz files from
<BR>
http://www.hapmap.org/downloads/genotypes/2007-01/rs_strand/non-redundant.
</P>
