<H2>Description</H2>
<P>
The potential coding fragment track shows genomic fragments that are likely to
be protein-coding genes. When compared with the homologous known gene in the 
same genome, the potential coding fragment preserves key features a 
protein-coding gene should have, such as open reading frame, splicing sites, 
and gene structure.  The structure of a potential coding fragment shown in the
browser is based on the gene structure of its homologous known gene. </P>

<H2>Methods</H2>
<P> 
A set of pre-aligned human known genes was mapped across the human 
genome through the Human Blastz Self Alignment. This method, called the 
homologous mapping method, identifies fragments that are homologs of known 
genes. We used known genes from the Consensus CDS (CCDS), RefSeq, and MGC gene
sets, which are currently the most reliable collections of genes. These genes 
have been aligned to the human genome, and their locations in the genome are 
available in the UCSC Genome Browser.  
</P>
<P> 
We compared each homologous fragment with its known gene and 
collected a set of features, such as sequence identity, dN/dS ratio, 
splicing sites, and number of premature stop codons, to determine whether the 
homologous fragment was a gene.
These homologous fragments belong to either coding genes or pseudogenes. We 
took homologous fragments overlapping known genes as  
positive samples and those overlapping known pseudogenes as negative samples,
then used these samples to train Support Vector Machines (SVMs) to 
separate coding fragments from pseudo fragments. The trained SVMs were 
used to classify homologous fragments into potential coding elements or 
potential pseudo elements. Finally, a heuristic filter was used to correct 
some misclassified fragments and to determine whether the homologous fragment 
was likely to be a coding gene. </P>

<H2>References</H2>
<P>
Kent, W.J., Baertsch, R., Hinrichs, A., Miller, W., and Haussler, D.
<A HREF="http://www.pnas.org/cgi/content/abstract/1932072100v1" 
TARGET=_blank>Evolution's cauldron: 
Duplication, deletion, and rearrangement in the mouse and human genomes</A>. 
<I>Proc Natl Acad Sci USA</I> <B>100</B>(20), 11484-11489 (2003).</P>
<P>
Pruitt, K.D., Tatusova, T., and Maglott D.R.
<A HREF="http://nar.oupjournals.org/cgi/content/full/33/suppl_1/D501" 
TARGET=_blank>NCBI Reference Sequence (RefSeq): a curated non-redundant 
sequence database of genomes, transcripts and proteins</A>. 
<I>Nucl. Acids Res.</I> <B>33</B>(1), D501-D504 (2005). </P>
<P>
Schwartz, S., Kent, W.J., Smit, A., Zhang, Z., Baertsch, R., Hardison, R., 
Haussler, D., and Miller, W.
<A HREF="http://www.genome.org/cgi/content/abstract/13/1/103" TARGET=_blank>Human-Mouse 
Alignments with BLASTZ</A>. <em>Genome Res.</em> <B>13</B>(1), 103-7 (2003).
</P>

<H2>Credits</H2>
<P>
The homologous mapping method and this browser track were developed by 
<A HREF="mailto:&#121;&#116;&#108;&#117;@s&#111;e.&#117;&#99;sc.&#101;&#100;&#117;">Yontao Lu</A> at UCSC.</P>
