<H2>Description</H2>
<P>
This track shows a measure of evolutionary conservation in 17 vertebrates, 
including mammalian, amphibian, bird, and fish species, 
based on a phylogenetic hidden Markov model, phastCons 
(Siepel <em>et al.</em>, 2005).
Multiz alignments of the following assemblies were used to generate this
track: 
<UL>
<LI>$organism ($date, $db)
<LI>chimp (Nov 2003, panTro1)
<LI>macaque (Jan 2006, rheMac2)
<LI>mouse (May 2004, mm7)
<LI>rat (Jun 2003, rn3)
<LI>rabbit (May 2005, oryCun1)
<LI>dog (May 2005, canFam2)
<LI>cow (Mar 2005, bosTau2)
<LI>armadillo (May 2005, dasNov1)
<LI>elephant (May 2005, loxAfr1)
<LI>tenrec (Jul 2005, echTel1)
<LI>opossum (Jun 2005, monDom2)
<LI>chicken (Feb 2004, galGal2)
<LI>frog (Oct 2004, xenTro1)
<LI>zebrafish (May 2005, danRer3)
<LI>Tetraodon (Feb 2004, tetNig1)
<LI>Fugu (Aug 2002, fr1)
</UL></P>

<H2>Display Conventions and Configuration</H2>
<P>
In full and pack display modes, conservation scores are displayed as
a &quot;wiggle&quot; (histogram), where the height reflects the 
size of the score. Pairwise alignments of each 
species to the $organism genome are displayed below as
a grayscale density plot (in pack mode) or as a &quot;wiggle&quot;
(in full mode) that indicates alignment quality.
In dense display mode, conservation is shown in grayscale using
darker values to indicate higher levels of overall conservation 
as scored by phastCons. </P>
<P>
The conservation wiggle can be configured in a variety of ways to 
highlight different aspects of the displayed information. 
Click the <A HREF="../goldenPath/help/hgWiggleTrackHelp.html" 
TARGET=_blank>Graph configuration help</A> link for an explanation 
of the configuration options.</P>
<P>
Checkboxes in the track configuration section allow excluding
species from the pairwise display; however, this does not remove them
from the conservation score display.
To view detailed information about the alignments at a specific
position, zoom in the display to 30,000 or fewer bases, then click on
the alignment.</P>

<H3>Gap Annotation</H3>
<P>
The &quot;Display chains between alignments&quot; configuration option 
enables display of gaps between alignment blocks in the pairwise alignments in 
a manner similar to the Chain track display.  The following
conventions are used:
<UL>
<LI><B>Single line:</B> No bases in the aligned species. Possibly due to a
lineage-specific insertion between the aligned blocks in the $organism genome
or a lineage-specific deletion between the aligned blocks in the aligning
species.
<LI><B>Double line:</B> Aligning species has one or more unalignable bases in
the gap region. Possibly due to excessive evolutionary distance between 
species or independent indels in the region between the aligned blocks in both
species. 
<LI><B>Pale yellow coloring:</B> Aligning species has Ns in the gap region.
Reflects uncertainty in the relationship between the DNA of both species, due
to lack of sequence in relevant portions of the aligning species. 
</UL></P>

<H3>Genomic Breaks</H3>
<P>
Discontinuities in the genomic context (chromosome, scaffold or region) of the
aligned DNA in the aligning species are shown as follows: 
<UL>
<LI>
<B>Vertical blue bar:</B> Represents a discontinuity that persists indefinitely
on either side, <em>e.g.</em> a large region of DNA on either side of the bar
comes from a different chromosome in the aligned species due to a large scale
rearrangement. 
<LI>
<B>Green square brackets:</B> Enclose shorter alignments consisting of DNA from
one genomic context in the aligned species nested inside a larger chain of
alignments from a different genomic context. The alignment within the
brackets may represent a short misalignment, a lineage-specific insertion of a
transposon in the $organism genome that aligns to a paralogous copy somewhere
else in the aligned species, or other similar occurrence.
</UL></P>

<H3>Base Level</H3>
<P>
When zoomed-in to the base-level display, the track shows the base 
composition of each alignment. 
The numbers and symbols on the Gaps
line indicate the lengths of gaps in the $organism sequence at those 
alignment positions relative to the longest non-$organism sequence. 
If there is sufficient space in the display, the size of the gap is shown; 
if not, and if the gap size is a multiple of 3, a &quot;*&quot; is displayed, 
otherwise &quot;+&quot; is shown.</P>
<P>
Codon translation is available in base-level display mode if the
displayed region is identified as a coding segment. To display this annotation,
select the species for translation from the pull-down menu in the Codon
Translation configuration section at the top of the page. Then, select one of
the following modes:
<UL>
<LI> 
<B>No codon translation:</B> The gene annotation is not used; the bases are
displayed without translation. 
<LI>
<B>Use default species reading frames for translation:</B> The annotations from the genome
displayed 
in the <em>Default species to establish reading frame</em> pull-down menu are used to
translate all the aligned species present in the alignment. 
<LI>
<B>Use reading frames for species if available, otherwise no translation:</B> Codon
translation is performed only for those species where the region is
annotated as protein coding.
<LI><B>Use reading frames for species if available, otherwise use default species:</B>
Codon translation is done on those species that are annotated as being protein
coding over the aligned region using species-specific annotation; the remaining
species are translated using the default species annotation. 
</UL></P>
<P>
Codon translation uses the following gene tracks as the basis for
translation, depending on the species chosen:

<BLOCKQUOTE><TABLE BORDER=1 CELLPADDING=4 BORDERCOLOR="#aaaaaa">
<TR ALIGN=left><TD><B>Gene Track</B></TD><TD><B>Species</B></TD></TR>
<TR ALIGN=left><TD>Known Genes</TD><TD>human, mouse, rat</TD></TR>
<TR ALIGN=left><TD>RefSeq Genes</TD><TD>chicken</TD></TR>
<TR ALIGN=left><TD>MGC Genes</TD><TD>X. tropicalis</TD></TR>
<TR ALIGN=left><TD>Ensembl Genes</TD><TD>Fugu, chimp</TD></TR>
<TR ALIGN=left><TD>mRNAs</TD><TD>rhesus, rabbit, dog, cow, zebrafish</TD></TR>
<TR ALIGN=left><TD>not translated</TD><TD>armadillo, elephant, tenrec, opossum, Tetraodon</TD></TR>
</TABLE></BLOCKQUOTE>
</P>

<H2>Methods</H2>
<P> 
Best-in-genome pairwise alignments were generated for each species 
using blastz, followed by chaining and netting.  The pairwise alignments
were then multiply aligned using multiz, following the ordering of the 
species tree diagrammed above.
The resulting multiple alignments were then assigned conservation scores by 
phastCons,
using a tree model with branch lengths derived from the ENCODE project 
Multi-Species Sequence Analysis group, September 2005 tree model.
This tree was generated from TBA alignments over 23 vertebrate species and is
based on 4D sites.</P>
<P>
The phastCons program computes conservation scores based on a phylo-HMM, a
type of probabilistic model that describes both the process of DNA
substitution at each site in a genome and the way this process changes from
one site to the next (Felsenstein and Churchill 1996, Yang 1995, Siepel and
Haussler 2005).  PhastCons uses a two-state phylo-HMM, with a state for
conserved regions and a state for non-conserved regions.  The value plotted
at each site is the posterior probability that the corresponding alignment
column was "generated" by the conserved state of the phylo-HMM.  These
scores reflect the phylogeny (including branch lengths) of the species in
question, a continuous-time Markov model of the nucleotide substitution
process, and a tendency for conservation levels to be autocorrelated along
the genome (i.e., to be similar at adjacent sites).  The general reversible
(REV) substitution model was used.  Note that, unlike many
conservation-scoring programs, phastCons does not rely on a sliding window
of fixed size, so short highly-conserved regions and long moderately
conserved regions can both obtain high scores.  More information about
phastCons can be found in Siepel et al. (2005).</P> 
<P> 
PhastCons currently treats alignment gaps as missing data, which
sometimes has the effect of producing undesirably high conservation scores
in gappy regions of the alignment.  We are looking at several possible ways
of improving the handling of alignment gaps.</P>

<H2>Credits</H2>
<P>
This track was created using the following programs:
<UL>
<LI> Alignment tools: blastz and multiz by Minmei Hou, Scott Schwartz and Webb 
Miller of the <A HREF="http://www.bx.psu.edu/miller_lab/" 
TARGET=_blank>Penn State Bioinformatics Group</A>
<LI> Chaining and Netting:  axtChain, chainNet by Jim Kent at UCSC
<LI> Conservation scoring: PhastCons, phyloFit, tree_doctor, msa_view by 
<A HREF="http://siepellab.labsites.cshl.edu/"
TARGET=_blank>Adam Siepel</A> while at UCSC, now at Cold Spring Harbor Laboratory
<LI> MAF Annotation tools: mafAddIRows by Brian Raney, UCSC; 
genePredToMafFrames by Mark Diekhans, UCSC
<LI> Tree image generator: phyloPng by Galt Barber, UCSC
<LI> Conservation track display: Kate Rosenbloom, Hiram Clawson (wiggle 
display), and Brian Raney (gap annotation and codon framing) at UCSC
</UL>
</P>
<P>The phylogenetic tree is based on Murphy et al. (2001) and general 
consensus in the vertebrate phylogeny community.
</P>

<H2>References</H2>

<H3>Phylo-HMMs and phastCons</H3>
<P>
Felsenstein, J. and Churchill, G.A. 
<A HREF="https://academic.oup.com/mbe/article/13/1/93/1055515"
TARGET=_blank>A hidden Markov model approach to 
variation among sites in rate of evolution</A>.
<em>Mol Biol Evol</em> <B>13</B>, 93-104 (1996). </P>
<P>
Siepel, A. and Haussler, D. Phylogenetic hidden Markov models.
In R. Nielsen, ed., <em>Statistical Methods in Molecular Evolution</em>,
pp. 325-351, Springer, New York (2005). </P>
<P>
Siepel, A., Bejerano, G., Pedersen, J.S., Hinrichs, A., Hou, M., Rosenbloom, 
K., Clawson, H., Spieth, J., Hillier, L.W., Richards, S., Weinstock, G.M., 
Wilson, R. K., Gibbs, R.A., Kent, W.J., Miller, W., and Haussler, D. 
<A HREF="https://genome.cshlp.org/content/15/8/1034"
TARGET=_blank>Evolutionarily conserved elements in vertebrate, insect, worm, 
and yeast genomes</A>.
<em>Genome Res.</em> <B>15</B>, 1034-1050 (2005).</P>
<P>
Yang, Z. 
<A HREF="https://www.genetics.org/content/139/2/993" 
TARGET=_blank>A space-time process model for the evolution of DNA
sequences</A>.  <em>Genetics</em> <B>139</B>, 993-1005 (1995). </P>

<H3>Chain/Net:</H3>
<P>
Kent, W.J., Baertsch, R., Hinrichs, A., Miller, W., and Haussler, D. 
<A HREF="https://www.pnas.org/content/100/20/11484"
TARGET=_blank>Evolution's cauldron: 
Duplication, deletion, and rearrangement in the mouse and human genomes</A>. 
<I>Proc Natl Acad Sci USA</I> <B>100</B>(20), 11484-11489 (2003).

<H3>Multiz:</H3>
<P>
Blanchette, M., Kent, W.J., Riemer, C., Elnitski, .L, Smit, A.F.A., Roskin, 
K.M., Baertsch, R., Rosenbloom, K., Clawson, H., Green, E.D., Haussler, D., 
Miller, W. 
<A HREF="https://genome.cshlp.org/content/14/4/708.abstract"
TARGET=_blank>Aligning Multiple Genomic Sequences with the Threaded Blockset Aligner</A>.
<em>Genome Res.</em> <B>14</B>(4), 708-15 (2004).

<H3>Blastz:</H3>
<P>
Chiaromonte, F., Yap, V.B., and Miller, W. 
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=11928468&dopt=Abstract" 
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>. 
<em>Pac Symp Biocomput</em> <B>2002</B>, 115-26 (2002). </P>
<P>
Schwartz, S., Kent, W.J., Smit, A., Zhang, Z., Baertsch, R., Hardison, R., 
Haussler, D., and Miller, W. 
<A HREF="https://genome.cshlp.org/content/13/1/103.abstract"
TARGET=_blank>Human-Mouse Alignments with BLASTZ</A>. 
<em>Genome Res.</em> <B>13</B>(1), 103-7 (2003).</P>

<H3>Phylogenetic Tree:</H3>
<P>
Murphy, W.J., et al.
<A HREF="https://science.sciencemag.org/content/294/5550/2348"
TARGET=_blank>Resolution of the early placental mammal radiation using Bayesian phylogenetics</A>.
<em>Science</em> <B>294</B>(5550), 2348-51 (2001).</P>
