<H2>Description</H2> 
<P> 
Linkage disequilibrium (LD) is the association of alleles on
chromosomes.  It measures the difference between the observed allele
frequency for a two locus allele as compared to its expected
frequency, which is the product of the two single allele frequencies.
When LD is low, the two loci tend to be inherited in a nearly random
manner.</P>
<P>
This track shows three different measures of linkage disequilibrium
&mdash; D', r<sup>2</sup>, and LOD (log odds) &mdash; between
pairs of SNPs in the displayed region.  LD is useful for
understanding the associations between genetic variants
and can be helpful in selecting SNPs for genotyping. </P>
<P>
By default, LOD values are displayed in full mode.  Each diagonal
represents a different SNP with each diamond representing a pairwise
comparison between two SNPs.  Shades are used to indicate linkage
disequilibrium between the pair of SNPs, with darker shades indicating
stronger LD.  For the LOD values, additional colors are used in some
cases:
<UL>
  <LI><I>Shades of pink/red</I> diamonds are drawn when the statistical
      significance is high (LOD >= 2). </LI>
  <LI><I>Light blue</I> diamonds indicate high D' values (>0.99) with
      low statistical significance (LOD &lt; 2). </LI>
  <LI><I>White</I> diamonds indicate pairwise D' values less than 1
      with no statistically significant evidence of LD (LOD &lt; 2).</LI>
</UL>
<table summary="Standard Color Scheme" border="1">
<colgroup><col><col><col></colgroup>
<tbody>
<tr><td align="center">&nbsp;</td><td align="center">D' &lt; 1</td><td align="center">D' = 1</td></tr>
<tr><td align="center">LOD &ge; 2</td><td align="center">shades of pink/red</td><td align="center">bright red</td></tr>
<tr><td align="center">LOD &lt; 2</td><td align="center">white</td><td align="center">blue</td></tr>
</tbody>
</table>
</P>

<H2>Methods</H2>
<P>
Genomic sequences for numerous samples were derived from cloned PCR
products. The single chromosome sequences thus derived were analyzed
by Phred/Phrap/PolyBayes to determine SNPs. Only SNPs that appeared at
high quality read locations in at least two samples were passed to
Haploview for analysis. For each sample, the single sequenced chromosome
was duplicated to present a virtual homozygote to Haploview.
</P>
<P>
Haploview uses a two marker EM (ignoring missing data) to estimate the
maximum-likelihood values of the four gamete frequencies, from which
the D', LOD, and r<sup>2</sup> calculations derive.  Haplotype phase
is inferred using a standard EM algorithm with a partition-ligation
approach for blocks with greater than 10 markers.</P>

<H2>Display Conventions and Configuration</H2>
<P>
<UL>
 <LI>
  Display Mode<BR>
  <UL>
   <LI><i>Full</i> mode shows the pairwise LD values in a
        Haploview-style mountain plot. </LI>
   <LI><i>Dense</i> mode shows the pairwise LD values in a single line
       for each population, where the intensity at each position is
       the average of all of the LD values between the SNP at that
       position and all other SNPs within 250 kb.</LI>
  </UL>
 </LI>
 <LI>
  LD Values: measures of linkage disequilibrium<BR>
  <UL>
   <LI><i>r<sup>2</sup></i> displays the raw r<sup>2</sup> value, or
       the square of the correlation coefficient for a given marker
       pair.  SNPs that have not been separated by recombination have
       r<sup>2</sup> = 1; in this case, these two markers are said to
       be redundant for genotyping, but may have different functional
       effects.  Lower r<sup>2</sup> values show a lower degree of LD,
       indicating that some recombination has occurred in this
       population.  See Hill and Robertson (1966) for details.
       </LI>
   <LI><i>D'</i> displays the raw D' value, which is the normalized
       covariance for a given marker pair.  A D' value of 1 (complete
       LD) indicates that two SNPs have not been separated by
       recombination, while lower values indicate evidence of
       recombination in the history of the sample.  Only D' values
       near 1 are a reliable measure of LD; lower values are difficult
       to interpret as the magnitude of D' depends strongly on sample
       size.  See Lewontin (1988) for more details.</LI>
   <LI><i>LOD</i> displays the log odds score for linkage
       disequilibrium between a given marker pair, and is shown by
       default.</LI>
  </UL>
 </LI>
 <LI>
  Track Geometry<BR>
  <UL>
   <LI><i>Trim to triangle</i> shows the standard mountain plot
       (default); turning this option off will show LD values with
       SNPs outside the window.</LI>
   <LI><i>Invert</i>ing makes it easier to visually compare two
       adjacent populations.</LI>
  </UL>
 </LI>
 <LI>Colors<BR>
  <UL>
   <LI><i>LD Values</i> can be drawn in a variety of colors, with red
       as default.  The intensity of the color is proportional to the
       strength of the LD measure chosen above.  </LI>
   <LI><i>Outline</i>s can be drawn in contrasting colors or turned
       off.  Outlines are automatically suppressed when the window is
       larger than 100,000 bp.</LI>
  </UL>
 </LI>
</UL>
</P>

<H2>Credits</H2>
<P>
This track was created by 
<A HREF="mailto:daryl@soe.ucsc.edu">Daryl Thomas</A> 
at UCSC 
following the display style from 
<A HREF="http://www.broad.mit.edu/mpg/haploview/" TARGET=_blank>Haploview</A>.
</P>

<H2>References</H2>

<H3>Phred/Phrap/Consed</H3>
<P>
Ewing, B., Hillier, L., Wendl, M., Green, P.
<A href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=9521921"> Basecalling of automated sequencer traces using phred. I. Accuracy assessment</A>.
<em>Genome Research</em> <B>8</B>:175-185 (1998).
</p><p>
Ewing, B., Green, P.
<A href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=9521922"> Basecalling of automated sequencer traces using phred. II. Error probabilities</A>. 
<em>Genome Research</em> <B>8</B>:186-194 (1998).
</p><p>
Gordon, D., Abajian, C., Green, P.
<A href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9521923&amp;dopt=Abstract">
Consed: a graphical tool for sequence finishing</A>.
<em>Genome Research</em> <B>8</B>:195-202 (1998).
</P>

<H3>PolyBayes</H3>
<P>
Marth, G.T. <I>et al.</I> <A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=10581034&itool=iconabstr&query_hl=4&itool=pubmed_DocSum">
A general approach to single-nucleotide polymorphism discovery</A>.
<em>Nature Genetics</em> <B>23</B>:452-456 (1999).
</P>

<H3>Haploview</H3>
<P>
Barrett, J.C., Fry, B., Maller, J. and Daly, M.J. 
<A HREF="http://bioinformatics.oxfordjournals.org/cgi/content/full/bth457?ijkey=54SiAdNbKzbNg&keytype=ref" 
TARGET=_blank> Haploview: analysis and visualization of LD and haplotype 
maps</A>. 
<em>Bioinformatics</em> <B>21</B>(2), 263-5 (2005).</P>

<H3>General references on Linkage Disequilibrium</H3>
<P>
Lewontin, R.C.
<A HREF="http://www.genetics.org/cgi/reprint/120/3/849"
TARGET=_blank> On measures of gametic disequilibrium</A>.
<em>Genetics</em> <B>120</B>, 849-52 (1988).</P>
<P>
Hill, W. G. and Robertson, A. The effect of linkage on limits to artificial selection. <em>Genet. Res.</em>, <b>8</b>:269-294 (1966).
</P>
