<H2>Description</H2>
<P>
Genes in metazoa are controlled by a complex array of cis-regulatory
elements that include core and distal promoters, enhancers, insulators,
silencers etc (Levine and Tjian, 2003). A unifying feature of such elements
active in cell nuclei is a chromatin-based epigenetic signature known as a
nuclease hypersensitive site (Elgin, 1988; Gross and Garrard, 1988; Wolffe,
1998). This track presents the results of a collaboration between Sangamo
BioSciences, Inc. and the European Institute of Oncology to isolate such
regulatory elements from human CD34+ hematopoietic stem cells (Urnov <em>et
al.</em>, submitted). This effort made use of a method developed at Sangamo
BioSciences to isolate such nuclease hypersensitive sites from living cells
with minimal, if any, contamination from bulk DNA (Urnov <em>et al</em>., 
submitted; US patent pending).

<H2>Display Conventions</H2>
<P>
The track annotates the location of 3,314 cis-regulatory elements in human
CD34+ cells in the human genome in the form of 40 bp tags. Note the method
identifies a specific position in chromatin that is hypersensitive to
nuclease, but does not map the boundaries of the regulatory element per se.
A conservative estimate of element size would be that occupied by one
nucleosome, <em>i.e.</em>, 180 - 200 bp surrounding the tag, although there is
precedent in the literature for nuclease hypersensitive sites that span
more than the length of one nucleosome (Turner, 2001; Wolffe, 1998).

<H2>Methods</H2>
<P>
CD34+ cells (enriched in hematopoietic stem cells) were prepared from
healthy donors following guidelines established by the Ethics Committee of
the European Institute of Oncology (IEO), Milan. Mobilization of CD34+
cells to the peripheral blood was stimulated by G-CSF treatment according
to standard procedures. After mobilization, donors were subjected to
leukophoresis, and <10% of the sample was used in the experiment. CD34+
cells were purified using a magnetic positive selection procedure
(&quot;EASYSEP&quot;; Stemcell, Vancouver, Canada). Purity of separation was
evaluated by FACS after staining with an anti-Human CD34 FITC-conjugate
antibody (Stemcell). Upon purification, the cell cycle status of the CD34+
cells was monitored by propidium iodide staining and FACS analysis. G0/G1
cells varied from approximately 90% to >95% of the total cells. Cells were
immediately used for the isolation of cis-regulatory DNA elements using the
nuclease hypersensitive site isolation protocol developed at Sangamo (Urnov
<em>et al.</em>, submitted).

<H2>Verification</H2>
<P>
In collaboration with scientists at the J. Craig Venter Institute and the
European Institute of Oncology, the method was initially validated on human
tissue culture cells by examining the colocalization of DNA fragments
isolated from cells with experimentally determined nuclease hypersensitive
sites in chromatin as mapped by indirect end-labeling and Southern blotting
(Nedospasov and Georgiev, 1980; Wu, 1980). Nineteen out of nineteen
randomly chosen clones from those libraries represented bona fide DNAse I
hypersensitive sites in chromatin (Urnov <em>et al.</em>, submitted). These data
confirmed that the method yields very high-content libraries of active cis-
regulatory DNA elements, supporting its application to human CD34+ cells.
Analysis of libraries of cis-regulatory elements prepared using this method
from CD34+ cells showed that 50 out of 55 randomly chosen clones &mdash; 91% &mdash;
coincided with DNAse I hypersensitive sites (Urnov <em>et al.</em>, submitted).

<H2>Credits</H2>
<P>
The library of regulatory DNA elements from human CD34+ cells was prepared,
sequenced, and validated by Saverio Minucci and colleagues at the 
<A HREF="http://www.ieo.it/english/Pages/Default.aspx" TARGET=_BLANK>
European Institute of Oncology</A>, 
using a method developed by Fyodor Urnov, Alan Wolffe, and colleagues at 
<A HREF="http://www.sangamo.com/index.html" TARGET=_BLANK>
Sangamo BioSciences</A>, and validated in collaboration with Sam Levy and 
colleagues (<A HREF="https://www.jcvi.org/"
TARGET=_BLANK>J. Craig Venter Institute</A>).

<H2>References</H2>
<P>
Elgin SC. 
<A HREF="https://www.jbc.org/article/S0021-9258(19)77625-X/fulltext" TARGET=_blank>The
formation and function of DNase I hypersensitive sites in the process of gene 
activation</A>.
<I>J Biol Chem</I>. 1988 Dec 25;263(36):19259-62.</P>
<P>
Gross DS, Garrard WT. 
<A HREF="https://www.annualreviews.org/doi/pdf/10.1146/annurev.bi.57.070188.001111" TARGET=_blank>Nuclease hypersensitive sites in chromatin</A>. 
<em>Ann Rev Biochem</em>. 1988;57:159-197. 
<P>
Levine M, Tjian R. 
<A HREF="https://www.nature.com/articles/nature01763" TARGET=_BLANK >Transcription regulation and animal diversity</A>. 
<I>Nature</I>. 2003 Jul 10;424(6945):147-51.</P>
<P>
Nedospasov S, Georgiev G. 
<A HREF="https://www.sciencedirect.com/science/article/pii/0006291X80903666"
target=_BLANK>Non-random cleavage of SV40 DNA in the compact minichromosome and free in solution by
micrococcal nuclease</A>.  
<I>Biochem Biophys Res Commun</I>. 1980 Jan 29;92(2):532-9. </P>
<P>
Turner BM. <I>Chromatin and Gene Regulation: Mechanisms in Epigenetics</I>.
Blackwell Publishers, Oxford. 2001.</P>
<P>
Urnov FD, Minucci S, Levy S <em>et al</em>. 
Genome-wide chromatin-based isolation of active <i>cis</i>-regulatory DNA elements from human cells. <I>Submitted</I>.</P>
<P>
Wolffe AP. 
<I>Chromatin Structure and Function</I>. 
Academic Press, San Diego, CA. 1998.</P>
<P>
Wu C. 
<A HREF="https://www.nature.com/articles/286854a0" target=_BLANK>The
5' ends of <i>Drosophila </i>heat shock genes in chromatin are hypersensitive to
DNase I</A>. 
<I>Nature</I>. 1980 Aug 28;286(5776):854-60. </P>
