<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=290 HEIGHT=290>
  <TR><TD ALIGN=RIGHT>
    <A HREF="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/"
	TARGET=_blank>
    <IMG SRC="../images/human.jpg" WIDTH=275 HEIGHT=275 ALT="human"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT>
    <FONT SIZE=-1> <EM>Homo sapiens</EM><BR>
    </FONT>
    <FONT SIZE=-2> (Graphic courtesy of 
      <A HREF="http://www.cbse.ucsc.edu/" TARGET=_blank>CBSE</A>)
    </FONT>
  </TD></TR>
</TABLE>


<P>
The February 2009 human reference sequence (GRCh37) was produced by the
<A HREF="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/"
TARGET=_blank>Genome Reference Consortium</A>. For more information about
this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/2758/" TARGET="_blank">
GRCh37</A> in the NCBI Assembly database.</P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic clone, an mRNA or EST or STS marker,
a chromosomal coordinate range, or keywords from 
the GenBank description of an mRNA. The following list shows 
examples of valid position queries for the human genome.
See the
<A HREF="../goldenPath/help/hgTracksHelp.html" TARGET=_blank>
User's Guide</A> for more information.
<P>

<TABLE  border=0 CELLPADDING=1 CELLSPACING=0>
    <TR><TD VALIGN=Top NOWRAP><B>Request:</B><BR></TD>
       	<TD VALIGN=Top COLSPAN=2><B>&nbsp;&nbsp; Genome Browser Response:</B><br></TD>
    </TR>

    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>chr7</TD>
        <TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays all of chromosome 7</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>chrUn_gl000212</TD>
        <TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays all of the unplaced contig gl000212</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>20p13</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays region for band p13 on chr 20</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>chr3:1-1000000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays first million bases of chr 3, counting from p-arm telomere</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>chr3:1000000+2000 </TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr3 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>

    <TR><TD VALIGN=Top NOWRAP> 
            RH18061;RH80175 <BR> 
            15q11;15q13 <BR>
            rs1042522;rs1800370</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays region between genome landmarks, such as
                      the STS markers RH18061 and RH80175,
                      or chromosome bands 15q11 to 15q13,
                      or SNPs rs1042522 and rs1800370.
                      This syntax may also be used for other range queries, such
		      as between uniquely determined ESTs, mRNAs,
		      refSeqs, etc.</TD>
    </TR>

    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>D16S3046</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays region around STS marker D16S3046 from the Genethon/Marshfield maps.
                      Includes 100,000 bases on each side as well.</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>AA205474</TD>
       	<TD WIDTH=14></TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession AA205474 in BRCA1 cancer 
                      gene on chr 17</TD>
    </TR>
<!-- 
    <TR><TD VALIGN=Top NOWRAP>ctgchr7_ctg</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays region of the clone contig ctgchr7_ctg 
                       (set "Map contigs" track to "dense" and refresh to see contigs)</TD>
    </TR> 
-->
    <TR><TD VALIGN=Top NOWRAP>AC008101</TD>
    	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays region of clone with GenBank accession AC008101</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>AF083811</TD>
       	<TD WIDTH=14></TD>
   	<TD VALIGN=Top>Displays region of mRNA with GenBank accession number AF083811</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>PRNP</TD>
       	<TD WIDTH=14></TD>
       	<TD>Displays region of genome with HUGO Gene Nomenclature Committee identifier PRNP</TD>
    </TR>
    <TR>
	<TD VALIGN=Top NOWRAP>NM_017414</TD>
	<TD><BR></TD>
	<TD VALIGN=Top>Displays the region of genome with RefSeq identifier NM_017414</TD>
    </TR>
    <TR>
	<TD VALIGN=Top NOWRAP>NP_059110</TD> 
	<TD><BR></TD>
 	<TD VALIGN=Top NOWRAP> Displays the region of genome with protein accession number NP_059110</TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>pseudogene mRNA</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>homeobox caudal</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>zinc finger</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>kruppel zinc finger</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>huntington</TD>
       	<TD WIDTH=14></TD>
 	<TD VALIGN=Top>Lists candidate genes associated with Huntington's disease</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>zahler</TD>
       	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zahler</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Evans,J.E.</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author J.E.  Evans</TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although GenBank 
			requires the search format <em>Evans JE</em>, internally it uses 
			the format <em>Evans,J.E.</em>.</TD>
    </TR>
</TABLE>
<P>
<HR>
<H3>Assembly Details</H3>
<P>
The GRCh37 build reference 
sequence is considered to be &quot;finished&quot;, a technical term indicating
that the sequence is highly accurate (with fewer than one error per 10,000
bases) and highly contiguous (with the only remaining gaps corresponding 
to regions whose sequence cannot be reliably resolved with current
technology). Future work on the reference sequence will focus on improving 
accuracy and reducing gaps in the sequence. Statistics for the GRCh37 build
assembly can be found on the NCBI 
<A HREF="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/human/data/?build=37"
TARGET=_blank>Build 37.1 Statistics</A> web page. For a glossary of assembly-related
terms, please see the GRC
<A HREF="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/info/definitions.shtml"
TARGET="_blank">Assembly Terminology</A> page.</P>
<P>
<B>Note on chrM</B><BR>
Since the release of the UCSC hg19 assembly, the <EM>Homo sapiens</EM>
mitochondrion sequence used as &quot;chrM&quot; in the Genome Browser,
<A HREF="https://www.ncbi.nlm.nih.gov/nuccore/17981852" TARGET=_blank>NC_001807</A>,
has been replaced in GenBank with the record
<A HREF="https://www.ncbi.nlm.nih.gov/nuccore/251831106" TARGET=_blank>NC_012920</A>,
the <A HREF="http://mitomap.org/bin/view.pl/MITOMAP/HumanMitoSeq" TARGET=_blank>Revised Cambridge
Reference Sequence (rCRS)</A>.
We have not replaced the chrM sequence in the hg19 Genome Browser.
However, in March 2020 we added the rCRS sequence to hg19 separately as &quot;chrMT&quot;.
In the latest human assembly release (Dec. 2013, GRCh38/hg38), &quot;chrM&quot; is the rCRS
and NC_001807 is not included.
<P>
<B>Chromosome naming scheme</B><BR>
In addition to the &quot;regular&quot; chromosomes, the hg19 browser contains
9 alternative haplotype sequences, 39 unplaced contigs, and 20 unlocalized contigs
from the initial GRCh37 assembly release.
In March 2020 we added patch sequences from GRC patch release 13 (GRCh37.p13):
73 "novel" patches (alternative haplotype sequences) and 131 "fix" patches
(corrected sequences).
For unlocalized contigs, the contig name is appended to the regular chromosome
name, as in <TT>chr1_gl000191_random</TT>.  If the chromosome is unknown, the
contig is represented with the name &quot;chrUn&quot; followed by the contig
identifier, as in <TT>chrUn_gl000211</TT>.  Note that the chrUn contigs are no
longer placed in a single, artificial chromosome as they have been in previous UCSC
assemblies.
The 9 alternative haplotypes released with the original assembly are shown in the
table below. For subsequent alternative haplotype patches the regular chromosome
name is followed by the contig accession and then &quot;_alt&quot;, for example 
<TT>chr5_gl339449_alt.</TT> Fix sequences introduced by patches are named in a similar 
fashion but with &quot;_fix&quot; instead of &quot;_alt&quot;, e.g. <TT>chrX_jh806590_fix</TT>. 
Additional information on alt and fix sequences can be found in our 
<A HREF="../FAQ/FAQdownloads.html#downloadFix">FAQ</A>. See the 
<A HREF="../cgi-bin/hgTracks?db=hg19&chromInfoPage=">sequences</A> page for a complete list of 
hg19 sequences.
<P>
The nine haplotype sequences in the initial GRCh37 release are named:
<BR>
<TABLE BORDER=1 CELLPADDING = 3><TR><TH>name</TH><TH>accession</TH><TH>UCSC chr name</TH></TR>
<TR><TD ALIGN=LEFT>HSCHR6_MHC_APD_CTG1</TD><TD>GL000250.1</TD><TD ALIGN=LEFT>chr6_apd_hap1</TD></TR>
<TR><TD ALIGN=LEFT>HSCHR6_MHC_COX_CTG1</TD><TD>GL000251.1</TD><TD ALIGN=LEFT>chr6_cox_hap2</TD></TR>
<TR><TD ALIGN=LEFT>HSCHR6_MHC_DBB_CTG1</TD><TD>GL000252.1</TD><TD ALIGN=LEFT>chr6_dbb_hap3</TD></TR>
<TR><TD ALIGN=LEFT>HSCHR6_MHC_MANN_CTG1</TD><TD>GL000253.1</TD><TD ALIGN=LEFT>chr6_mann_hap4</TD></TR>
<TR><TD ALIGN=LEFT>HSCHR6_MHC_MCF_CTG1</TD><TD>GL000254.1</TD><TD ALIGN=LEFT>chr6_mcf_hap5</TD></TR>
<TR><TD ALIGN=LEFT>HSCHR6_MHC_QBL_CTG1</TD><TD>GL000255.1</TD><TD ALIGN=LEFT>chr6_qbl_hap6</TD></TR>
<TR><TD ALIGN=LEFT>HSCHR6_MHC_SSTO_CTG1</TD><TD>GL000256.1</TD><TD ALIGN=LEFT>chr6_ssto_hap7</TD></TR>
<TR><TD ALIGN=LEFT>HSCHR4_1_CTG9</TD><TD>GL000257.1</TD><TD ALIGN=LEFT>chr4_ctg9_hap1</TD></TR>
<TR><TD ALIGN=LEFT>HSCHR17_1_CTG5</TD><TD>GL000258.1</TD><TD ALIGN=LEFT>chr17_ctg5_hap1</TD></TR>
</TABLE>
</P>
<P>
See the 
<A HREF="http://www.ucl.ac.uk/cancer/medical-genomics/mhc"
TARGET=_blank>Wellcome Trust Sanger Institute MHC Haplotype 
Project</A> web site for additional information on the 
chr6 alternate haplotype assemblies.</P>
<P>
The Y chromosome in this assembly contains two pseudoautosomal regions (PARs) 
that were taken from the corresponding regions in the X chromosome and are 
exact duplicates:
<BLOCKQUOTE>
chrY:10001-2649520 and chrY:59034050-59363566
<BR>chrX:60001-2699520 and chrX:154931044-155260560
</BLOCKQUOTE>
<P>
For further information on GRCh37 build see the NCBI 
<A HREF="https://www.ncbi.nlm.nih.gov/genome/guide/human/release_notes.html"
TARGET=_blank>GRCh37 release notes</A>.</P>
<P>
Bulk downloads of the sequence and annotation data are available via the Genome Browser
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/hg19">FTP server</A> 
or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#human">Downloads</A>
page. We recommend that you use
<A HREF="../goldenPath/help/ftp.html">rsync</A> for downloading 
large or multiple files. </P>
<P>
The hg19 annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="../goldenPath/credits.html#human_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to this release. </P>
<P>
<!--Commented out til we get these
<HR>
<H3>Statistical information</H3>
<UL>
<LI><A HREF="../goldenPath/certificates/">Non-Standard Join Certificates</A>
<LI><A HREF="../goldenPath/stats.html#hg19">Summary Statistics</A>
</UL>
-->
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>finished</B>.
