<H2>Description</H2>
<P>
This track displays copy number variants (CNVs), insertions/deletions (InDels),
inversions and inversion breakpoints annotated by the
<A HREF="http://dgv.tcag.ca/dgv/app/home"
TARGET=_BLANK>Database of Genomic Variants</A> (DGV), which
contains genomic variations observed in healthy individuals.
DGV focuses on structural variation, defined as
genomic alterations that involve segments of DNA that are larger than
1000 bp.  Insertions/deletions of 50 bp or larger are also included.
</P>

<H2>Display Conventions</H2>
<P>
This track contains three subtracks:
<p>
<ul>
<li>Structural Variant Regions: annotations that have been generated from one or more reported
structural variants at the same location.
</li>
<li>Supporting Structural Variants: the sample-level reported structural variants.
</li>
<li>Gold Standard Variants: curated variants from a selected number of studies in DGV.
</ul>
<P>
Color is used in these subtracks to indicate the type of variation:
<UL>
 <LI><B><span style="color:#C000C0;">Inversions</span></B> and
     <B><span style="color:#C000C0;">inversion breakpoints</span></B> are purple.
 </LI>

 <LI>CNVs and InDels are blue if there is a
   <B><span style="color:#0000C0;">gain in size</span></B> relative to the reference.
 </LI>

 <LI>CNVs and InDels are red if there is a
   <B><span style="color:#C00000;">loss in size</span></B> relative to the reference.
 </LI>

 <LI>CNVs and InDels are brown if there are reports of
   <B><span style="color:#8B4513;">both a loss and a gain in size</span></B>
   relative to the reference.
 </LI>
</UL>
</P>
<p>
The DGV Gold Standard subtrack utilizes a boxplot-like
display to represent the merging of records as explained in the <em>Methods</em>
section below. In this track, the middle box (where applicable), represents
the high confidence location of the CNV, while the thin lines and end boxes
represent the possible range of the CNV.
</p>

<P>
Clicking on a variant leads to a page with detailed information about the variant, 
such as the study reference and PubMed abstract link, the study's method and any
genes overlapping the variant. Also listed, if available, are the sequencing or array platform
used for the study, a sample cohort description, sample size, sample ID(s) in which
the variant was observed, observed gains and observed losses.
If the particular variant is a merged variant, links to genome browser views of 
the supporting variants are listed. If the particular variant is a supporting variant,
a link to the genome browser view of its merged variant is displayed.
A link to DGV's Variant Details page for each variant is also provided.
</P>
<P>
For most variants, DGV uses <a href="http://dgv.tcag.ca/dgv/data-model/entd.html#Variant"
target="_blank">accessions</a> from peer archives of structural variation
(<A HREF="https://www.ncbi.nlm.nih.gov/dbvar/" TARGET=_BLANK>dbVar</A>
at NCBI or <A HREF="https://www.ebi.ac.uk/dgva" TARGET=_BLANK>DGVa</A> at EBI).
These accessions begin with either &quot;essv&quot;,
&quot;esv&quot;, &quot;nssv&quot;, or &quot;nsv&quot;, followed by a number.
Variant submissions processed by EBI begin with &quot;e&quot;
and those processed by NCBI begin with &quot;n&quot;.
</P>
<P>
Accessions with ssv are for variant calls on a particular sample, and if they
are copy number variants, they generally indicate whether the change is a gain
or loss. 
In a few studies the ssv represents the variant called by a single
algorithm. If multiple algorithms were used, overlapping ssv's from
the same individual would be combined to generate a sample level
sv. 
</P>
<P>
If there are many samples analyzed in a study, and if there are many
samples which have the same variant, there will be multiple ssv's with
the same start and end coordinates.
These sample level variants are then merged and combined to form a
representative variant that highlights the common variant found in
that study. The result is called a structural variant (sv) record.
Accessions with sv are for regions asserted by submitters to contain
structural variants, and often span ssv elements for both losses and
gains.  dbVar and DGVa do not record numbers of losses and gains
encompassed within sv regions.
</P>
<P>
DGV merges clusters of variants that share at least 70% reciprocal
overlap in size/location, and assigns an accession beginning with
&quot;dgv&quot;, followed by an internal variant serial number,
followed by an abbreviated study id.  For example,
the first merged variant from the Shaikh et al. 2009 study (study
accession=nstd21) would be dgv1n21. The second merged variant would be
dgv2n21 and so forth.
Since in this case there is an additional level of clustering,
it is possible for an &quot;sv&quot; variant to be both a merged
variant and a supporting variant.
</P>
<P>
For most sv and dgv variants, DGV displays the total number of
sample-level gains and/or losses at the bottom of their variant detail
page. Since each ssv variant is for one sample, its total is 1.
</P>

<H2>Methods</H2>
<P>
Published structural variants are imported from peer archives
<A HREF="https://www.ncbi.nlm.nih.gov/dbvar/" TARGET=_BLANK>dbVar</A> and
<A HREF="https://www.ebi.ac.uk/dgva" TARGET=_BLANK>DGVa</A>.
DGV then applies <a href="http://dgv.tcag.ca/dgv/app/faq#q4"
target="_blank">quality filters</a> and merges overlapping variants.
</P>
<P>
For data sets where the variation calls are reported at a
sample-by-sample level, DGV merges calls with similar boundaries
across the sample
set. Only variants of the same type (i.e. CNVs, Indels, inversions)
are merged, and gains and losses are merged separately.
Sample level calls that overlap by &ge; 70% are merged in this
process.
</P>

<p>
The initial criteria for the Gold Standard set require that a variant is found
in at least two different studies and found in at least two different samples. After
filtering out low-quality variants, the remaining variants are clustered according
to 50% minimum overlap, and then merged into a single record. Gains and losses are merged 
separately.
</p>

<p>
The highest ranking variant in the cluster defines the inner box, while 
the outer lines define the maximum possible start and stop coordinates of the CNV.
In this way, the inner box forms a high-confidence CNV location and the thin connecting 
lines indicate confidence intervals for the location of CNV.
</p>

<H2>Credits</H2>
<P>
Thanks to the <A HREF="http://dgv.tcag.ca/dgv/app/home"
TARGET=_BLANK>Database of Genomic Variants</A> for providing these data.
In citing the Database of Genomic Variants please refer to MacDonald
<em>et al</em>.
</P>

<H2>References</H2>
<p>
Iafrate AJ, Feuk L, Rivera MN, Listewnik ML, Donahoe PK, Qi Y, Scherer SW, Lee C.
<a href="https://www.nature.com/articles/ng1416" target="_blank">
Detection of large-scale variation in the human genome</a>.
<em>Nat Genet</em>. 2004 Sep;36(9):949-51.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15286789" target="_blank">15286789</a>
</p>

<p>
MacDonald JR, Ziman R, Yuen RK, Feuk L, Scherer SW.
<a href="https://academic.oup.com/nar/article/42/D1/D986/1068860/The-Database-of-Genomic-Variants-
a-curated" target="_blank">
The Database of Genomic Variants: a curated collection of structural variation in the human
genome</a>.
<em>Nucleic Acids Res</em>. 2014 Jan;42(Database issue):D986-92.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24174537" target="_blank">24174537</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3965079/" target="_blank">PMC3965079</a>
</p>

<p>
Zhang J, Feuk L, Duggan GE, Khaja R, Scherer SW.
<a href="https://www.karger.com/Article/FullText/95916" target="_blank">
Development of bioinformatics resources for display and analysis of copy number and other structural
variants in the human genome</a>.
<em>Cytogenet Genome Res</em>. 2006;115(3-4):205-14.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17124402" target="_blank">17124402</a>
</p>

