<H2>Description</H2>
<P>
This track shows over 8.75 million single nucleotide variants (SNVs)
and over 600,000 insertions or deletions (indels)
as well as exome variant calling regions
in 60,706 unrelated individuals sequenced as part of various population genetic 
and disease-specific 
<A HREF="http://exac.broadinstitute.org/about" TARGET=_BLANK>studies</A>
collected by the
<A HREF="http://exac.broadinstitute.org/" TARGET=_BLANK>Exome Aggregation Consortium (ExAC)</A>,
release 0.3.
Raw data from all studies have been reprocessed through a unified pipeline and jointly
variant-called to increase consistency across projects.
For more information on the processing pipeline and population annotations,
see the release files
<A HREF="ftp://ftp.broadinstitute.org/pub/ExAC_release/release0.3/README.release0.3" TARGET=_BLANK>
README.release0.3</A>
and
<A HREF="ftp://ftp.broadinstitute.org/pub/ExAC_release/release0.3/README.population_annotations" TARGET=_BLANK>
README.population_annotations</A>
respectively.
</P>

<H2>Display Conventions</H2>
<P>
For the variants subtrack,
in &quot;dense&quot; mode, a vertical line is drawn at the position of each variant.
In &quot;pack&quot; mode, ref and alt alleles are displayed to the left of a vertical
line with colored portions corresponding to allele counts.  Hovering the mouse pointer
over a variant pops up a display of alleles and counts.
</P>

<h2>Data Access</h2>

<p>
The raw data can be explored interactively with the 
<a href="../cgi-bin/hgTables">Table Browser</a>, or the
<a href="../cgi-bin/hgIntegrator">Data Integrator</a>.
For automated analysis, the genome annotation is stored in a bigBed file that can be downloaded from the
<a href="http://hgdownload.soe.ucsc.edu/gbdb/$db/ExAC/" target="_blank">download server</a>.
The underlying data files for this track are called
<tt>ExAC.r0.3.sites.vep.hg19.vcf.gz</tt> and <tt>exacCallingRegions.bb</tt>. 
Individual regions or the whole genome annotation can be obtained using our tool <tt>bigBedToBed</tt> 
which can be compiled from the source code or downloaded as a precompiled binary
for your system. Instructions for downloading source code and binaries can be found
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">here</a>. 
The tool can also be used to obtain only features within a given range, for example:<br> 
<tt>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/$db/ExAC/exacCallingRegions.bb -chrom=chr6 -start=0 -end=1000000 stdout</tt> 
<br>
Please refer to our <a href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing list archives</a>
for questions, or our <a href="../FAQ/FAQdownloads.html#download36">Data Access FAQ</a> for more information. 
</p>

<H2>Credits</H2>
<P>
Thanks to the
<A HREF="http://exac.broadinstitute.org/about" TARGET=_BLANK>Exome Aggregation Consortium</A>
for making these data available in advance of publication.  The data are released under a
Fort Lauderdale Agreement; please email
<A HREF="mailto:&#101;x&#111;&#109;&#101;&#99;&#111;&#110;&#115;o&#114;&#116;&#105;&#117;&#109;&#64;&#103;&#109;&#97;&#105;&#108;.
c&#111;m">
&#101;x&#111;&#109;&#101;&#99;&#111;&#110;&#115;o&#114;&#116;&#105;&#117;&#109;&#64;&#103;&#109;&#97;&#105;&#108;.c&#111;m</A>
<!-- above address is exomeconsortium at gmail.com -->
with any further questions and please reference the 2016 paper.
</P>

<H2>References</H2>

<p>
Exome Aggregation Consortium (ExAC), Cambridge, MA
(<A HREF="http://exac.broadinstitute.org" TARGET=_BLANK>http://exac.broadinstitute.org</A>),
release 0.3, downloaded Mar. 30, 2015.
</p>

<p>
Lek M, Karczewski KJ, Minikel EV, Samocha KE, Banks E, Fennell T, O'Donnell-Luria AH, Ware JS, Hill
AJ, Cummings BB <em>et al</em>.
<a href="https://www.nature.com/articles/nature19057" target="_blank">
Analysis of protein-coding genetic variation in 60,706 humans</a>.
<em>Nature</em>. 2016 Aug 17;536(7616):285-91.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27535533" target="_blank">27535533</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5018207/" target="_blank">PMC5018207</a>
</p>
