<H2>Affymetrix Arrays</H2>

<H4>Affymetrix Genome-Wide Human SNP Array 6.0 and SV</H4>
The SNP Array 6.0 includes more than 906,600 single nucleotide polymorphisms (SNPs) 
and more than 946,000 probes for the detection of copy number variation. 
The SNPs include the 482,000 SNPs from the 5.0 Array (unbiased selection).
In addition, 424,000 new SNPs were chosen in the following areas:
<UL>
 <LI>Tag SNPs</LI>
 <LI>SNPs from chromosomes X and Y</LI>
 <LI>Mitochondrial SNPs</LI>
 <LI>New SNPs added to dbSNP</LI>
 <LI>SNPs in recombination hotspots</LI>
</UL>
<P>
The structural variation copy number (SV) probes include 202,000 probes 
targeting 5,677 known CNV regions
from the Toronto Database of Genomic Variants.  The additional 744,000 probes 
are evenly spaced throughout the genome.
</p>

<H4>Affymetrix Genome-Wide Human SNP Array 5.0</H4>
The SNP Array 5.0 is a single microarray featuring all single nucleotide 
polymorphisms (SNPs) from the original two-chip Mapping 500K Array Set, as 
well as 420,000 additional non-polymorphic probes that can measure other 
genetic differences, such as copy number variation.

<H4>Affymetrix 500K (250K Nsp and 250K Sty)</H4>
This annotation displays the SNPs available for genotyping with the 
GeneChip Human Mapping 500K Array Set from Affymetrix.  It is comprised of
two arrays: Nsp and Sty, which contain approximately 262,000 and 238,000 SNPs,
respectively.

<H4>Affymetrix CytoScan HD</H4>
The CytoScan High-Density (HD) Array provides whole-genome coverage with 
enriched coverage of all constitutional and cancer-related genes on a 
single array. The complete CytoScan array includes:

<blockquote>
<table class="descTbl" border=1 cellspacing=1 cellpadding=3>
  <tbody>
  <tr style="background-color: #d9f8e4">
	<td>Genome build</td>
	<td>hg19</td>
  </tr>
  <tr style="background-color: #d9e4ff">
	<td>Total number of copy-number markers</td>
	<td>2,696,550</td>
  </tr>
  <tr style="background-color: #d9f8e4">
	<td>Total number of non-polymorphic markers</td>
	<td>1,953,246</td>
  </tr>
  <tr style="background-color: #d9e4ff">
	<td>Number of SNP markers</td>
	<td>743,304</td>
  </tr>
  <tr style="background-color: #d9f8e4">
	<td>SNP markers with &gt;99% genotype accuracy</td>
	<td>749,157</td>
  </tr>
  <tr style="background-color: #d9e4ff">
	<td>Autosomal markers</td>
	<td>2,491,915</td>
  </tr>
  <tr style="background-color: #d9f8e4">
	<td>Pseudoautosomal markers</td>
	<td>4,624</td>
  </tr>
  <tr style="background-color: #d9e4ff">
	<td>Intragenic markers</td>
	<td>1,410,535</td>
  </tr>
  <tr style="background-color: #d9f8e4">
	<td>Intergenic markers</td>
	<td>1,286,015</td>
  </tr>
  </tbody>
</table>
</blockquote>

<P>
Probes are colored 
<strong style="color:#327D32">green</strong> for CNV probesets and
<strong style="color:#880088">purple</strong> for SNP probesets. 
Some SNP probesets are also considered informative for CNV detection.
</P>

<H2>Agilent Arrays</H2>
<p>
Non-SNP subtracks are colored in alternating shades of
<strong style="color:#0D8529">green</strong> and
<strong style="color: #E68620">orange</strong>
to highlight track boundaries. SNP subtracks
are colored blue with SNP probes colored
<strong style="color: #1317D6">dark blue</strong> and CGH probes
colored
<strong style="color: #55BEE0">light blue</strong>.
</p>

<p>
The track consists of the following subtracks:
</p>
The arrays listed in this track are probes from the
<a href="https://www.genomics.agilent.com/article.jsp?pageId=75"
target="_blank">Agilent Catalog Oligonucleotide Microarrays</a>.  The following table summarizes the different Agilent arrays:
</P>
<table class="descTbl" border=1 cellspacing=1 cellpadding=3>
<thead>
<tr style="background-color: #1616D1; color: #fff;"> 
	<th>Description</th>
	<th>AMADID</th>
	<th>Samples per Slide</th>
	<th>Biological Features</th>
	<th>Median Probe Spacing</th>
	<th>Gene-biased</th>
	<th>SNPs</th>
</tr>
</thead>
<tbody>
<tr style="background-color: #d9f8e4">
	<td>SurePrint G3 Human CGH Microarray 1x1M</td>
	<td>021529</td>
	<td>1</td>
	<td>969,033</td>
	<td>2.1 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9e4ff">
	<td>SurePrint G3 Human High-Resolution Microarray 1x1M</td>
	<td>023642</td>
	<td>1</td>
	<td>989,214</td>
	<td>2.6 kb</td>
	<td>No (evenly tiled)</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9f8e4">
	<td>SurePrint G3 Human CGH+SNP Microarray 2x400K</td>
	<td>028081</td>
	<td>2</td>
	<td>411,434</td>
	<td>7 kb</td>
	<td>Yes</td>
	<td>65,000</td>
</tr>
<tr style="background-color: #d9e4ff">
	<td>SurePrint G3 Human CGH Microarray 2x400K</td>
	<td>021850</td>
	<td>2</td>
	<td>415,914</td>
	<td>5.3 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9f8e4">
	<td>SurePrint G3 Human Cancer CGH+SNP Microarray 4x180K</td>
	<td>030587</td>
	<td>4</td>
	<td>172,595</td>
	<td>25 kb</td>
	<td>Yes (cancer genes)</td>
	<td>65,000</td>
</tr>
<tr style="background-color: #d9e4ff">
	<td>SurePrint G3 Human CGH+SNP Microarray 4x180K</td>
	<td>029830</td>
	<td>4</td>
	<td>172,595</td>
	<td>25 kb</td>
	<td>No (ISCA regions)</td>
	<td>65,000</td>
</tr>
<tr style="background-color: #d9f8e4">
	<td>SurePrint G3 Human CGH Microarray 4x180K</td>
	<td>022060</td>
	<td>4</td>
	<td>174,675</td>
	<td>13 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9e4ff">
	<td>SurePrint G3 Human CGH Microarray 8x60K</td>
	<td>021924</td>
	<td>8</td>
	<td>59,175</td>
	<td>41 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9f8e4">
	<td>SurePrint HD Human CGH Microarray 1x244K</td>
	<td>014693</td>
	<td>1</td>
	<td>238,331</td>
	<td>8.9 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9e4ff">
	<td>SurePrint HD Human CGH Microarray 2x105K</td>
	<td>014698</td>
	<td>2</td>
	<td>100,034</td>
	<td>22 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9f8e4">
	<td>SurePrint HD Human CGH Microarray 4x44K</td>
	<td>014950</td>
	<td>4</td>
	<td>43,143</td>
	<td>43 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
</tbody>
</table>

<H2>Illumina Arrays</H2>

<H4>Illumina HumanHap650Y</H4>
This annotation displays the SNPs available for genotyping with Illumina's
HumanHap650Y Genotyping BeadChip.  The HumanHap650Y contains over 650,000 markers,
extending the HumanHap550 by adding 100,000 additional Yoruba-specific tag
SNPs.  On average, there is 1 SNP every 5.3 kb, 6.2 kb and 5.4 kb across
the genome in the CEU, CHB+JPT and YRI populations, respectively.
The HumanHap650Y was derived from release 21 of the 
<a href="https://hapmap.ncbi.nlm.nih.gov/" target="_blank">
International HapMap Project</a>.

<H4>Illumina HumanHap550</H4>
This annotation displays the SNPs available for genotyping with Illumina's
HumanHap550 Genotyping BeadChip.  The HumanHap550 contains over 550,000 markers,
the majority of which are tag SNPs 
derived from release 20 of the 
<a href="https://hapmap.ncbi.nlm.nih.gov/" target="_blank">
International HapMap Project</a>.  In addition,
approximately 7800 non-synonymous SNPs, a higher density of tag SNPs in
the MHC region, over 150 mitochondrial SNPs and over 4000
SNPs from regions with copy number polymorphism were included.   
In the CEU population, an r-squared threshold of 0.8 was used
for common SNPs in genes, within 10 kb of genes or in evolutionarily
conserved regions.  For all other regions, an r-squared threshold of 0.7 was used.
On average, there is 1 SNP every 5.5 kb, 6.5 kb and 6.2 kb across the genome in 
the CEU, CHB+JPT and YRI populations, respectively.

<H4>Illumina HumanHap300</H4>
This annotation displays the SNPs available for genotyping with Illumina's
HumanHap300 Genotyping BeadChip.  The HumanHap300 contains over 317,000 tagSNP markers
derived from Phase I of the 
<a href="https://hapmap.ncbi.nlm.nih.gov/" target="_blank">
International HapMap Project</a>.  In addition,
approximately 7300 non-synonymous SNPs and a higher density of tag SNPs in
the MHC region were included.   On average, there is 1 SNP every 9 kb across
the genome and median spacing is 5 kb.

<H4>Illumina Human1M-Duo</H4>
This annotation displays the SNPs available for genotyping with Illumina's
Human1M-Duo Genotyping BeadChip.  The Human1M-Duo contains more than 1,100,000 tagSNP markers
and a set of ~60,000 additional CNV-targeted markers. 
The median spacing is 1.5kb (mean - 2.4 kb).

<H4>Illumina HumanOmni1-Quad v1</H4>
The HumanOmni1-Quad BeadChip consists of 1,140,419 markers in a 4-sample
format. The whole-genome content provides high
genomic coverage rates of 93%, 92%, and 76% at r2 &gt; 0.8 for the CEU,
CHB+JPT, and YRI populations, respectively. High density markers with a
median spacing of 1.2 kb ensure the highest level of resolution for CNV
and breakpoint identification.
 
The content has been derived from the 1,000 Genomes Project,
all three HapMap phases, and recently published studies, including 
new coding variants identified by the 1000 Genomes Project and
markers chosen in high-value regions of the genome: ABO blood
typing SNPs, cSNPs, disease-associated SNPs, eSNPs, SNPs in mRNA splice
sites, ADME genes, AIMs, HLA complexes, indels, introns, MHC regions,
miRNA binding sites, mitochondrial DNA, PAR, promoter regions, and
Y-chromosome.

<H4>Illumina Human660W-Quad v1</H4>
The Human660W-Quad BeadChip consists of 657,366 markers in a 4-sample
format. The Human660W-Quad BeadChip provides 87%, 85%, and 56% coverage
of CEU, CHB+JPT, and YRI populations at r2 &gt; 0.8. For 
CNV and cytogenetic analysis, the dense backbone content is combined
with an additional ~100,000 markers that target observed common CNVs.

<H4>HumanCytoSNP-12 v2.1</H4>

The 301,232 markers on the HumanCytoSNP-12 represents a complete 12-sample panel of 
genome-wide SNPs for a uniform backbone and additional markers targeting all regions of 
known cytogenetic importance. Backbone markers provide genome-wide marker spacing of 10kb. 
This is supplemented with dense coverage (at 6 kb spacing average) of ~250 genomic regions 
commonly studied in cytogenetics labs and targeted coverage in ~400 additional genes, 
subtelomeric regions, pericentromeric regions, and sex chromosomes. 
An efficiency-optimized tagging strategy provides a panel for GWAS 
(70% coverage in CEU at r2 &gt; 0.8) in the highest throughput and most cost-effective 
whole-genome DNA Analysis BeadChip.

<h4>Illumina Global Diversity Array</h4>
<p>
The
<a href="https://www.illumina.com/products/by-type/microarray-kits/infinium-global-diversity.html"
target="_blank">Global Diversity Array-8</a> v1.0 BeadChip includes coverage of the ACMG 59-gene
clinical research variants and multi-ethnic, genome-wide content. The GDA is the commercial version
of the array chosen by the <a href="https://allofus.nih.gov/" target="_blank">All of Us</a> Research
Program and is designed to capture a wider range of the world's populationsthan traditional
microarrays.</p>

<H2>Methods</H2>
<P>
Position, strand, and polymorphism data were obtained from Affymetrix and 
supplemented with links to corresponding dbSNP rsIDs based on a positional
lookup into 
<A HREF="https://www.ncbi.nlm.nih.gov/SNP/" TARGET=_blank>
dbSNP</A>. The Affy 6.0 Array is based on dbSNP build 127; the Affy 5.0 Array 
is based on dbSNP build 126. The Affy 500K Array is based on dbSNP build 125 
and was translated from hg17 by UCSC using rsID lookup.  
In fewer than 2% of the cases, a dbSNP rsID was
not present in dbSNP at the Affymetrix array position. 
Reference allele information was retrieved from the UCSC database based on dbSNP position
and strand data. 
</P>
<P>
Illumina data were supplied as rsIDs and position based on dbSNP build 126. 
Strand, polymorphism and reference allele information was retrieved from the UCSC database 
based on rsID and position.
The Illumina arrays are comprised of probes for 4 of the possible single-base substitutions:
A/C, A/G, C/T and G/T.  A/T and C/G probes will be available in future arrays.
</P>
<P>
For Illumina Human1M-Duo, the position, strand, polymorphism and reference allele information was 
retrieved from the snp129 table of UCSC database if the marker ID can be found in dbSNP 129, 
otherwise the information is retrieved from the data provided by Illumina.
</P>
<P>
For Illumina HumanOmni1-Quad, Human660W-Quad, and HumanCytoSNP-12, 
the position, strand, polymorphism and reference allele information was 
retrieved from the snp130 table of UCSC database if the marker ID can be found in dbSNP 130, 
otherwise the information is retrieved from the data provided by Illumina.
</P>
<P>
Agilent's oligonucleotide CGH (Comparative Genomic Hybridization) platform enables the 
study of genome-wide DNA copy number changes at a high resolution. The CGH probes on Agilent 
aCGH microarrays are 60-mer oligonucleotides synthesized in situ using Agilent's inkjet 
SurePrint technology. The probes represented on the Agilent CGH microarrays have been 
selected using algorithms developed specifically for the CGH application, assuring optimal 
performance of these probes in detecting DNA copy number changes. 
</P>
<P>
The Agilent SurePrint G3 CGH+SNP microarrays are designed for high quality human DNA 
copy-number profiling combined with the simultaneous detection of copy-neutral 
aberrations, such as lack or loss of heterozygosity (LOH) and uniparental disomy (UPD). 
Identification of LOH/UPD is enabled by the presence of a set of SNP probes on the CGH+SNP 
microarrays resulting in ~5-10 Mb resolution for copy neutral LOH/UPD detection across the 
entire genome.
</P>
<P>
The Agilent catalog CGH and CGH+SNP microarrays are printed on 1 in. x 3 in. glass slides and are 
available in several formats. The human catalog SurePrint G3 microarrays formats are the 1x1M 
(gene-biased or evenly tiled), 2x400K (CGH-only or CGH+SNP), 4x180K (CGH-only or CGH+SNP), 
and 8x60K. The legacy human catalog SurePrint HD microarrays are the 1x244K, 2x105K, and 
4x44K. 
</P>
<p>
Non-SNP subtracks are colored in alternating shades of 
green and orange to highlight track boundaries. SNP subtracks 
are colored blue with SNP probes colored dark blue and 
CGH probes colored light blue. The track consists of the 
following subtracks:
</P>

<H2>References</H2>
<P>
More information on Agilent Oligonucleotide Microarrays can be found on 
<a href="https://www.genomics.agilent.com/article.jsp?pageId=75"
target="_blank">Agilent's website</a>.
</P>
<p>
Barrett MT, Scheffer A, Ben-Dor A, Sampas N, Lipson D, Kincaid R, Tsang P, Curry B, Baird K, Meltzer
PS <em>et al</em>.
<a href="https://www.pnas.org/content/101/51/17765" target="_blank">
Comparative genomic hybridization using oligonucleotide microarrays and total genomic DNA</a>.
<em>Proc Natl Acad Sci U S A</em>. 2004 Dec 21;101(51):17765-70.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15591353" target="_blank">15591353</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC535426/" target="_blank">PMC535426</a>
</p>

<P>
More information on the Affymetrix arrays is available at these sites:
<UL>
<LI>
Genome-Wide Human 
<A HREF="https://www.thermofisher.com/order/catalog/product/901182"
target="_blank">SNP Array 6.0</A>.
</LI>
<LI>
Genome-Wide Human 
<A HREF="https://www.thermofisher.com/order/catalog/product/901182" target="_blank">
SNP Array 5.0</A>.
</LI>
<LI>
Mapping 
<A HREF="http://www.affymetrix.com/support/technical/byproduct.affx?product=500k" TARGET=_blank>
500K</A> Array Set.
</LI>
</UL>

More information on the Illumina arrays is available at these sites:
<UL>
<LI>
<a href="https://support.illumina.com/array/array_kits/human1m-duo_dna_analysis_kit.html" target="_blank">
Human1M-Duo</a> Genotyping BeadChip.
</LI>
<LI>
<a href="https://support.illumina.com/array/array_kits/humanomni1-quad_beadchip_kit.html" target="_blank">
HumanOmni1-Quad v1</a> Genotyping BeadChip.
</LI>
<LI>
<a href="https://support.illumina.com/array/array_kits/human660w-quad_dna_analysis_kit.html" target="_blank">
Human660W-Quad v1</a> Genotyping BeadChip.
</LI>

<LI>
<A HREF="https://www.illumina.com/products/by-type/clinical-research-products/human-cytosnp-12.html" TARGET=_blank>
HumanCytoSNP-12 v2</A> Genotyping BeadChip.
</LI>
</UL>
</P>

<H2>Credits</H2>
<P>
Thanks to Shane Giles, Peter Webb, and Anniek De Witte from Agilent
Technologies, Venu Valmeekam from Affymetrix, and Luana Galver and
Jennifer L. Stone from Illumina for
providing these data.
</P>
