<h2>Description</h2>

<p>
This track helps call out sections of the genome that often cause problems for bioinformaticians.
The 12 subtracks identify genomic regions known to cause analysis artifacts for common sequencing
downstream computations, such as alignment, variant calling, or peak calling. The underlying data was
imported from the
 <a href="https://www.ncbi.nlm.nih.gov/variation/tools/get-rm/" target=_blank>NCBI GeT-RM</a>, the 
<a href="https://www.nist.gov/programs-projects/genome-bottle" target=_blank>Genome-in-a-Bottle</a>,
and Anshul Kundaje's <a href="https://github.com/Boyle-Lab/Blacklist/blob/master/lists/hg19-blacklist-README.pdf" target=_blank>ENCODE Blacklist</a> projects.
</p>

<p>
The only exception is the <b>UCSC Unusual Regions</b> subtrack, which contains annotations of 
<a href="../cgi-bin/hgTracks?db=hg19&chromInfoPage=">fixed sequences, alternate haplotypes,
 unplaced contigs, pseudo-autosomal regions, and mitochondria</a> 
that can yield alignments with low-quality mapping scores and discordant read pairs. This data set
was manually curated, based on the <a href="../cgi-bin/hgGateway">Genome Browser's assembly</a>
description, the <a href="https://genome.ucsc.edu/FAQ/FAQdownloads.html">FAQs</a> about assembly,
and the <a href="../cgi-bin/hgTrackUi?db=hg19&g=refSeqComposite">NCBI RefSeq "other" annotations</a>
track data.
</p>

<p>
The <b>ENCODE Blacklist</b> subtrack contains a comprehensive set of regions which are troublesome
for high-throughput Next-Generation Sequencing (NGS) aligners. These regions tend to have a very
high ratio of multi-mapping to unique mapping reads and high variance in mappability due to
repetitive elements such as satellite, centromeric and telomeric repeats. 
</p>

<p>
The <b>Genome-In-A-Bottle</b> (GIAB) track set contains defined regions where it is difficult to
make a confident call, due to low coverage, systematic sequencing errors, and local alignment
problems. These regions were identified from sequencing data generated by multiple technologies.
</p>

<p>
The <b>NCBI GeT-RM</b>, Genetic Testing Reference Materials, track set contains highly homologous
gene- and exon-level regions difficult
or impossible to analyze with standard Sanger or short-read NGS approaches and are relevant to
current clinical testing. 

<h2>Display Conventions and Configuration</h2>

<p>
Each track contains a set of regions of varying length with no special configuration options. 
The <em>UCSC Unusual Regions</em> track has a mouse-over description, all other tracks have at most
a name field, which can be shown in pack mode. The tracks are usually kept in dense mode.
</p>

<p>
The <em>Hide empty subtracks</em> control hides subtracks with no data in the browser window. Changing the browser window by zooming or scrolling may result in the display of a different selection of tracks.
</p>

<H2>Data access</H2>
<p>
The raw data can be explored interactively with the <a href="../cgi-bin/hgTables">Table Browser</a>
or the <a href="../cgi-bin/hgIntegrator">Data Integrator</a>.

<p>
For automated download and analysis, the genome annotation is stored in bigBed files that
can be downloaded from
<a href="http://hgdownload.soe.ucsc.edu/gbdb/$db/bbi/problematic/" target="_blank">our download server</a>.
Individual
regions or the whole genome annotation can be obtained using our tool <tt>bigBedToBed</tt>
which can be compiled from the source code or downloaded as a precompiled
binary for your system. Instructions for downloading source code and binaries can be found
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">here</a>.
The tool
can also be used to obtain only features within a given range, e.g. 
<tt>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/hg19/bbi/problematic/deadZone.bb -chrom=chr21 -start=0 -end=100000000 stdout</tt></p>
</p>

<p>
<h2>Methods</h2>

<p>
Files were downloaded from the respective databases and converted to bigBed format.
The procedure is documented in our
<a href="https://github.com/ucscGenomeBrowser/kent/blob/master/src/hg/makeDb/doc/hg19.txt"
target="_blank">hg19 makeDoc file</a> (search problematic).
</p>

<h2>Credits</h2>
<p>
Thanks to Anna Benet-Pages, Max Haeussler, Angie Hinrichs, and Daniel Schmelter
at the UCSC Genome Browser for planning, building, and testing these tracks. The
underlying data comes from the 
<a href="https://github.com/Boyle-Lab/Blacklist/blob/master/lists/hg19-blacklist-README.pdf" target=_blank>ENCODE Blacklist</a>, the
<a target=_blank href="https://www.ncbi.nlm.nih.gov/variation/tools/get-rm/">GeT-RM</a>,
and the
<a href="https://www.nist.gov/programs-projects/genome-bottle" target=_blank>Genome-in-a-Bottle</a>
projects.
</p>

<h2>References</h2>
<p>
Amemiya HM, Kundaje A, Boyle AP.
<a href="https://www.nature.com/articles/s41598-019-45839-z" target="_blank">
The ENCODE Blacklist: Identification of Problematic Regions of the Genome</a>.
<em>Sci Rep</em>. 2019 Jun 27;9(1):9354.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31249361" target="_blank">31249361</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6597582/" target="_blank">PMC6597582</a>
</p><p>
Zook JM, Chapman B, Wang J, Mittelman D, Hofmann O, Hide W, Salit M.
<a href="https://dx.doi.org/10.1038/nbt.2835" target="_blank">
Integrating human sequence data sets provides a resource of benchmark SNP and indel genotype
calls</a>.
<em>Nat Biotechnol</em>. 2014 Mar;32(3):246-51.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24531798" target="_blank">24531798</a>
</p>
<p>
Mandelker D, Schmidt RJ, Ankala A, McDonald Gibson K, Bowser M, Sharma H, Duffy E, Hegde M, Santani
A, Lebo M <em>et al</em>.
<a href="https://www.nature.com/articles/gim201658" target="_blank">
Navigating highly homologous genes in a molecular diagnostic setting: a resource for clinical next-
generation sequencing</a>.
<em>Genet Med</em>. 2016 Dec;18(12):1282-1289.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27228465" target="_blank">27228465</a>
</p>


