<H2>Description</H2>
<P>
This track displays maps of histone H3K4me3 in human brain,
identified by chromatin immunoprecipitation followed by high-throughput
sequencing (ChIP-Seq).
Measurements were made in neuronal and non-neuronal nuclei collected from
prefrontal cortex (PFC) of 11 individuals ranging in age from 0.5 to 69 years.</P>
<P>
ChIP-Seq begins by using formaldehyde to cross-link
histones and other DNA-associated proteins to genomic DNA. The cross-linked
chromatin is subsequently extracted, mechanically sheared, and
immunoprecipitated using specific antibodies. After reversal of cross-links,
the immunoprecipitated DNA is sequenced and mapped to the human reference
genome. The relative enrichment of each antibody-target (epitope) across
the genome is inferred from the density of mapped fragments.</P>
<P>

<H2>Methods</H2>
<P>
Human prefrontal cortex samples used in this study were obtained from the
Brain and Tissue Bank for Developmental Disorders, University of Maryland
and a brain bank at the University of California, Irvine.  Nuclei extraction,
chromatin immunoprecipitation and sequencing were carried out as described
in Cheung I, et al. (2010). Sequencing was performed on an Illumina
Genome Analyzer (GA II).</P>
<P>
The length of sequence reads was 36 bp. Mapping was performed using 
Bowtie (version 0.11.3, 
Langmead B, Trapnell C, Pop M, Salzberg SL (2009)) allowing up to one
mismatch to map all sequence reads to the gender appropriate human genome hg18 (NCBI36), and
67-87% of the reads in the neuronal samples mapped to one unique
location in the genome. The mapped reads were analyzed using the MACS software
package (version 1.3.5, Zhang Y, et al. (2008)) to identify peaks, with bw = 230 bp, as defined
experimentally by PCR, tSize = 36 bp, and other parameters set at default.
The scored peaks were then lifted to the hg19 (GRCh37) assembly.  
Signal tracks were generated directly from a remapping of the reads to hg19.
</P>

<H2>Verification</H2>
<P>
Experimental testing was performed on a subset of the peaks that were 
positioned more than
10 kb from annotated genes for RNA expression in PFC neurons by qRT-PCR and
in situ hybridization. These results suggest that H3K4me3 mapping can serve as a
guide to uncover potentially hundreds of unannotated novel and cell-specific
transcripts in the brain.</P>

<H2>Credits</H2>
<P>
Chromatin immunoprecipitation experiments were carried out by Iris Cheung,
Yan Jiang and Schahram Akbarian; analyses were performed by Hennady Shulha,
Jie Wang and Zhiping Weng at the University of Massachusetts Medical School.</P>

<H2>References</H2>
<P>
Cheung I, Shulha HP, Jiang Y, Matevossian A, Wang J, Weng Z, Akbarian S.
<A HREF="https://www.pnas.org/content/107/19/8824">Developmental regulation and individual differences of neuronal H3K4me3 epigenomes in the prefrontal cortex</A>.
<i>Proc Natl Acad Sci U S A</i>. 2010 May 11;107(19):8824-9.</P>
<P>
Langmead B, Trapnell C, Pop M, Salzberg SL.
<A HREF="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2009-10-3-r25#abs">
Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</A>. 
<I>Genome Biol</I>. 2009;10(3):R25.</P>
<P>
Zhang Y, Liu T, Meyer CA, Eeckhoute J, Johnson DS, Bernstein BE, Nusbaum C, Myers RM, Brown M, Li W, Liu XS.
<A HREF="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2008-9-9-r137#abs">
Model-based analysis of ChIP-Seq (MACS)</A>. 
<I>Genome Biol</I>. 2008;9(9):R137.</P>

