<H2>Description</H2>

<P>This track is produced as part of the ENCODE Project. 
Describe the project here.
A major goal of the ENCODE project is the complete characterization and understanding of the regulation and dynamics of the human transcriptome.
High-throughput sequencing of reverse-transcribed RNA molecules (RNA-seq) has revolutionized work in this area by providing a genome-wide digital readout of transcription at base-pair resolution.
However, such measurements are typically performed on population consisting of a large number of cells thus the readout is the average of the transcriptome profiles of individual cells.
This masks potentially important cell-to-cell variation in transcript abundance, allele expression bias, dynamic response to external stimuli, etc. In order to adress these issues,
we have generated single-cell RNA-seq measurements for invidual cells from ENCODE cell lines.
</P>
 

<H2>Display Conventions and Configuration</H2>
<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are <A href="/goldenPath/help/multiView.html" target="_BLANK">here</A>.
The following views are in this track:
</P><DL>
<DT><I>Raw Signal</I>
</DT><DD>Density graph (wiggle) of read coverage.
</DD></DL></DT>

<H2>Downloadable Files</H2>

<P>The following files can be found on the <A target="_BLANK" href="/goldenPath/hg19/encodeDCC/wgEncodeCaltechSCRnaSeq/">downloads page</A>.
</P>

<P>
<H3>Raw Reads, Alignments and Coverage Tracks:</H3>
<DD><b>*.fastq</b> - raw sequence files in fastq format with phred33 quality scores<br></DD>
<DD><b>*.BigWig</b> - read density files (<I>RawSignal</I> view, BigWig format; "plus" and "minus" versions provided for stranded data)<br></DD>
<DD><b>*.bam</b> - all alignments in <A target="_BLANK" href="http://samtools.sourceforge.net/SAM1.pdf">SAM/BAM format</A> (<I>Alignments</I> view)<br></DD>

<H3>Expression Estimates and Transcript Models (Cufflinks): </H3>
<DD><b>*.genes.FPKM</b> - expression level estimates for GENCODE CRCh37.v7 genes in FPKM (Fragments Per Kilobase per Million fragments)<br></DD>
</DT>

<H2>Methods</H2>

<H3>Experimental Procedures</H3>

<P> 
Cells were grown according to the approved <A href="/ENCODE/protocols/cell" target="_BLANK">ENCODE cell culture protocols</A> 
Single GM12878 cells from suspension culture are aspirated with a micropipette.  The cell is deposited into a cell lysis/dT priming solution under visual observation to confirm the presence of a single cell.  The lysate is then frozen on dry ice and stored at -80C until processing.  After denaturation at 70C, first strand reverse transcription is performed with a template switching enzyme (Clontech).  18 cycles of PCR are used to amplify the cDNA into a double stranded library.  The amplified library is fragmented using transposome-mediated “tagmentation” (Epicentre), which simultaneously fragments and attaches Illumina sequencing primer sequences to the cDNA.  The fragmented cDNA (average size 300 bp) is amplified for an additional 9 cycles using primers which incorporate bridge PCR sequences and multiplex bar codes.  The final product is cleaned up using SPRI beads and submitted for bridge PCR cluster formation on an Illumina flow cell. 
</P>
<H3>Data Processing and Analysis</H3>
 

<H2>Verification</H2>


<H2>Credits</H2>

<P>Wold Group: Ali Mortazavi, Brian Williams, Georgi Marinov, Diane Trout, Brandon King, Ken McCue, Lorian Schaeffer.</P>
<P>Myers Group: Norma Neff, Florencia Pauli, Fan Zhang, Tim Reddy, Rami Rauch, Chris Partridge.</P> 
<P>Illumina gene expression group: Gary Schroth, Shujun Luo, Eric Vermaas.</P>
<P>TopHat/Cufflinks development: Cole Trapnell, Lior Pachter, Steven Salzberg .</P>

<P>
Contacts: 
<A HREF="mailto:&#103;e&#111;r&#103;&#105;&#64;&#99;a&#108;&#116;&#101;&#99;&#104;.
e&#100;&#117;">
Georgi Marinov</A> (data coordination/informatics/experimental).
<!-- above address is georgi at caltech.edu -->
 
<A HREF="mailto:&#100;&#105;&#97;&#110;e&#64;&#99;&#97;l&#116;&#101;&#99;&#104;.
&#101;d&#117;">
Diane Trout</A> (informatics) and
<!-- above address is diane at caltech.edu -->
<A HREF="mailto:&#98;&#97;&#119;&#105;&#108;l&#105;_&#57;&#49;&#49;&#50;&#53;&#64;&#121;&#97;h&#111;&#111;.
&#99;o&#109;">
Brian Williams</A> (experimental).
<!-- above address is bawilli_91125 at yahoo.com -->

</P>

<H2>References</H2>
EXAMPLE
<P>Mortazavi A, Williams BA, McCue K, Schaeffer L, and Wold BJ. 
<A target="_BLANK" href="https://www.ncbi.nlm.nih.gov/pubmed/18516045">Mapping and quantifying mammalian transcriptomes by RNA-seq</A>.
<EM>Nature Methods</EM>. 2008 Jul; 5(7):621-628.</P> 


<H2> Data Release Policy </H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column, above.  The full data release policy
for ENCODE is available
<A href="/ENCODE/terms.html" target="_BLANK">here</A>.</P>
