<h2>Description</h2>
<p>
The GENCODE Genes track (version 14, October 2012) shows high-quality manual
annotations merged with evidence-based automated annotations across the entire
human genome generated by the
<a href="https://www.gencodegenes.org/" target="_blank">GENCODE project</a>.
The GENCODE gene set presents a full merge
between HAVANA manual annotation process and Ensembl automatic annotation pipeline.
Priority is given to the manually curated HAVANA annotation using predicted
Ensembl annotations when there are no corresponding manual annotations.
The annotation was carried out on genome assembly GRCh37 (hg19).
</p>
<p>
As of GENCODE Version 11, Ensembl and GENCODE have converged.  The gene
annotations in the GENCODE comprehensive set are the same as the corresponding
Ensembl release.  UCSC will continue to provide a separate Ensembl track on
Human in the same format as the Ensembl tracks on other organisms.
</p>
<p>
<B>NOTE</B>: Due to the UCSC Genome Browser using the NC_001807 mitochondrial
genome sequence (chrM) and GENCODE annotating the NC_012920 mitochondrial
sequence, the GENCODE mitochondrial annotations have been lifted to NC_001807
coordinates in the UCSC Genome Browser.  The original annotations with
NC_012920 coordinates are available for download in the GENCODE GTF files.
</p>

<h2> Display Conventions and Configuration </h2>
<P>
This track is a multi-view composite track that contains differing data sets
(<EM>views</EM>).  Instructions for configuring multi-view tracks are
<A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
To show only selected subtracks, uncheck the boxes next to the tracks that
you wish to hide.</P>
<b>Views</b> available on this track are:
<dl>
    <dt><i>Genes</i></dt>
    <dd> The gene annotations in this view are divided into three subtracks:</dd>
</dl>
<ul>
  <li><em>GENCODE Basic set</em> is a subset of the <em>Comprehensive set</em>. 
    The selection criteria are described in the <a href="#basicSetSelection">methods section</a>.</li>
  <li><em>GENCODE Comprehensive set</em> contains all GENCODE coding and non-coding transcript annotations,
    including polymorphic pseudogenes.  This includes both manual and
    automatic annotations.  This is a super-set of the <em>Basic set</em>.</li>
  <li><em>GENCODE Pseudogenes</em> include all annotations except polymorphic pseudogenes.</li>
</ul>
    
<dl>
    <dt><i>2-way</i></dt> 
</dl>
<ul>
    <li><em>GENCODE 2-way Pseudogenes</em> contains pseudogenes predicted by both the Yale
        Pseudopipe and UCSC Retrofinder pipelines. 
        The set was derived by looking for 50 base pairs
        of overlap between pseudogenes derived from both sets based on their 
        chromosomal coordinates.  When multiple Pseudopipe
        predictions map to a single Retrofinder prediction, only one match is kept
        for the 2-way consensus set.
    </li>
</ul>

<dl>
    <dt><i>PolyA</i></dt>
</dl>
<ul>
<li><em>GENCODE PolyA</em> contains polyA signals and sites manually annotated on
    the genome based on transcribed evidence (ESTs and cDNAs) of 3' end of
    transcripts containing at least 3 A's not matching the genome.</li>
</ul>


<p><b>Filtering</b> is available for the items in the GENCODE Basic, Comprehensive and Pseudogene tracks
using the following criteria:</p>
<ul>
  <li> Transcript class: filter by the basic biological function of a transcript
    annotation
    <ul>
     <li> All - don't filter by transcript class</li>
     <li> coding - display protein coding transcripts, including polymorphic pseudogenes</li>
     <li> nonCoding - display non-protein coding transcripts</li>
     <li> pseudo - display pseudogene transcript annotations</li>
     <li> problem - display problem transcripts (Biotypes of <em>retained_intron</em>, <em>TEC</em>, or <em>disrupted_domain</em>)
   </ul>
  </li>

  <li> Transcript Annotation Method: filter by the method used to create the annotation
   <ul>
     <li> All - don't filter by transcript class</li>
     <li> manual - display manually created annotations, including those that are 
       also created automatically</li>
     <li> automatic - display automatically created annotations, including those that are 
       also created manually</li>
     <li> manual_only - display manually created annotations that were
       not annotated by the automatic method</li>
     <li> automatic_only - display automatically created annotations that were
       not annotated by the manual method</li>
   </ul>
   </li>
  <li> Transcript Biotype: filter transcripts by
       <a href="https://www.gencodegenes.org/gencode_biotypes.html" target="_blank">biotype</a></li>
</ul>

<p><b>Coloring</b> for the gene annotations is based on the annotation type: </p>
<ul>
  <li><font color="#0c0c78"><b>coding</b></font> 
  <li><font color="#006400"><b>non-coding</b></font> 
  <li><font color="#ff33ff"><b>pseudogene</b></font> 
  <li><font color="#fe0000"><b>problem</b></font>
  <li><font color="#ff33ff"><b>all 2-way pseudogenes</b></font>
  <li><font color="#000000"><b>all polyA annotations</b></font>
</ul>

<h2>Methods</h2>

<p>
The GENCODE project aims to annotate all evidence-based gene features on the 
human reference sequence with high accuracy by integrating 
computational approaches (including comparative methods), manual
annotation and targeted experimental verification. This goal includes identifying 
all protein-coding loci with associated alternative variants, non-coding
loci which have transcript evidence, and pseudogenes. 
For a detailed description of the methods and references used, see
Harrow <em>et al.</em> (2006).
</p>

<p>
<b><a name="basicSetSelection">GENCODE <em>Basic Set</em> selection:</a></b>
The GENCODE <em>Basic Set</em> is intended to provide a simplified subset of
the GENCODE transcript annotations that will be useful to the majority of
users. The goal was to have a high-quality basic set that also covered all loci.  
Selection of GENCODE annotations for inclusion in the <em>basic set</em>
was determined independently for the coding and non-coding transcripts at each
gene locus.
</p>
<ul>
  <li> Criteria for selection of coding transcripts (including polymorphic pseudogenes) at a given
       locus:
    <ul>
      <li> All full-length coding transcripts (except problem transcripts or transcripts that are
           nonsense-mediated decay) were included in the basic set.</li>
      <li> If there were no transcripts meeting the above criteria, then the partial coding
           transcript with the largest CDS was included in the basic set (excluding problem transcripts).</li>
    </ul>
  </li>
  <li> Criteria for selection of non-coding transcripts at a given locus:
    <ul>
      <li> All full-length non-coding transcripts (except problem transcripts)
           with a well characterized biotype (see below) were included in the
           basic set.</li>
      <li> If there were no transcripts meeting the above criteria, then the largest non-coding
           transcript was included in the basic set (excluding problem transcripts).</li>
    </ul>
  </li>
  <li> It no transcripts were included by either the above criteria, the longest
    problem transcript is included.
  </li>
</ul>

<P>
<b>Non-coding transcript categorization:</b> 
Non-coding transcripts are categorized using
their <a href="https://www.gencodegenes.org/gencode_biotypes.html" target="_blank">biotype</a>
and the following criteria:
</p>
<ul>
  <li> well characterized: <em>antisense, Mt_rRNA, Mt_tRNA, miRNA, rRNA, snRNA, snoRNA</em></li>
  <li> poorly characterized: <em>3prime_overlapping_ncrna, lincRNA, misc_RNA, non_coding, processed_transcript, sense_intronic, sense_overlapping</em></li>
</ul>

<p>
<b><a name="tsl">Transcription Support Level (TSL):</a></b>
It is important that users understand how to assess transcript annotations
that they see in GENCODE. While some transcript models have a high level of
support through the full length of their exon structure, there are also
transcripts that are poorly supported and that should be considered
speculative. The Transcription Support Level (TSL) is a method to highlight the
well-supported and poorly-supported transcript models for users. The method
relies on the primary data that can support full-length transcript
structure: mRNA and EST alignments supplied by UCSC and Ensembl.</p>

<p>The mRNA and EST alignments are compared to the GENCODE transcripts and the
transcripts are scored according to how well the alignment matches over its
full length. 
The GENCODE TSL provides a consistent method of evaluating the
level of support that a GENCODE transcript annotation is
actually expressed in humans.  Human transcript sequences from the 
<a href="https://www.insdc.org/"target="_blank">International Nucleotide
Sequence Database Collaboration</a> (GenBank, ENA, and DDBJ) are used as
the evidence for this analysis.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/15713233" target="_blank">
Exonerate</a> RNA alignments from Ensembl,
BLAT RNA and EST alignments from the UCSC Genome Browser Database are used in
the analysis. Erroneous transcripts and libraries identified in lists
maintained by the Ensembl, UCSC, HAVANA and RefSeq groups are flagged as
suspect.  GENCODE annotations for protein-coding and non-protein-coding
transcripts are compared with the evidence alignments.</p>

<p>Annotations in the MHC region and other immunological genes are not
evaluated, as automatic alignments tend to be very problematic. 
Methods for evaluating single-exon genes are still being developed and 
they are not included
in the current analysis.  Multi-exon GENCODE annotations are evaluated using
the criteria that all introns are supported by an evidence alignment and the
evidence alignment does not indicate that there are unannotated exons. Small
insertions and deletions in evidence alignments are assumed to be due to
polymorphisms and not considered as differing from the annotations. All
intron boundaries must match exactly. The transcript start and end locations
are allowed to differ.</p>

<p>The following categories are assigned to each of the evaluated annotations:</p>

<ul>
  <li> <a name="tsl1"><b>tsl1</b></a> - all splice junctions of the transcript are supported by
    at least one non-suspect mRNA
  <li> <a name="tsl2"><b>tsl2</b></a> - the best supporting mRNA is flagged as suspect or the support is from multiple ESTs</li>
  <li> <a name="tsl3"><b>tsl3</b></a> - the only support is from a single EST</li>
  <li> <a name="tsl4"><b>tsl4</b></a> - the best supporting EST is flagged as suspect</li>
  <li> <a name="tsl5"><b>tsl5</b></a> - no single transcript supports the model structure</li>
  <li> <a name="tslNA"><b>tslNA</b></a> - the transcript was not analyzed for one of the following reasons:
    <ul>
      <li> pseudogene annotation, including transcribed pseudogenes
      <li> human leukocyte antigen (HLA) transcript
      <li> immunoglobin gene transcript
      <li> T-cell receptor transcript
      <li> single-exon transcript (will be included in a future version)
    </ul>
  </li>
</ul>

<h2>Downloads</h2>
GENCODE GTF files are available from the
<a href="https://www.gencodegenes.org/releases/14.html" target="_blank">GENCODE release 14</a> site.

<h2>Verification</h2>

<P>
Selected transcript models are verified experimentally by RT-PCR amplification followed by sequencing.
Those experiments can be found at GEO:</P>
<ul>
 <!-- <li> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE30619" target="_blank"><b>GSE30619:[E-MTAB-612]</b></a> - Batch I is based on annotation from July 2008 (without pseudogenes).</li>
  <li> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25711" target="_blank"><b>GSE25711:[E-MTAB-407]</b></a> - Batch II is based on annotation from April 2009. </li>
  <li> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE30612" target="_blank"><b>GSE30612:[E-MTAB-533]</b></a> - Batch III is verifying RGASP models for c.elegans and human.</li> -->
  <li> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE34797" target="_blank"><b>GSE34797:[E-MTAB-684]</b></a> - Batch IV is based on chromosome 3, 4 and 5 annotations from GENCODE 4 (January 2010).</li>
  <li> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE34820" target="_blank"><b>GSE34820:[E-MTAB-737]</b></a> - Batch V is based on annotations from GENCODE 6 (November 2010).</li>
  <li> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE34821" target="_blank"><b>GSE34821:[E-MTAB-831]</b></a> - Batch VI is based on annotations from GENCODE 6 (November 2010) as well as transcript models predicted by the Ensembl Genebuild group based on the Illumina Human BodyMap 2.0 data. </li>
</ul>
<P> See Harrow <em>et al.</em> (2006) for information on verification
techniques.
</P>

<h2>Release Notes</h2>
<p>
This GENCODE version 14 corresponds to Ensembl 69 from October 2012
and Vega 49 from September 2012.</p> 
<!--
GENCODE/Ensembl release information: https://www.gencodegenes.org/releases/
Vega release information: http://vega.sanger.ac.uk/index.html
 -->
<p>See also: <a href="https://www.gencodegenes.org/" target="_blank">The GENCODE Project</a>
</p>

<h2>Credits</h2>
<p> This GENCODE release is the result of a collaborative effort among
the following laboratories: (contact: <A HREF="mailto:&#103;&#101;n&#99;o&#100;&#101;&#64;&#115;an&#103;&#101;&#114;.&#97;&#99;.u&#107;">
GENCODE at the Sanger Institute</A>)
<!-- above address is gencode at sanger.ac.uk -->
</p>

<table style="text-align: left; width: 100%;" border="1" cellpadding="2" cellspacing="2">
  <tbody>
    <tr>
      <td style="vertical-align: top; text-align: left;"><b>Lab/Institution</b></td>
      <td style="vertical-align: top; text-align: left;"><b>Contributors</b></td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;"><a href="https://www.sanger.ac.uk/people/faculty/previous-faculty/professor-tim-hubbard" target="_blank">GENCODE Principal Investigator</a></td>
      <td style="vertical-align: top;">Tim Hubbard</td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;"><a href="https://www.sanger.ac.uk/science/groups/vertebrate-annotation" target="_blank">HAVANA manual annotation group</a>,
          Wellcome Trust Sanger Insitute (WTSI), Hinxton, UK</td>
      <td style="vertical-align: top;">Adam Frankish, Jose Manuel Gonzalez, Mike Kay, Alexandra Bignell, 
        Gloria Despacio-Reyes, Garaub Mukherjee, Gary Sanders, Veronika Boychenko, Jennifer Harrow</td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;"><a href="https://www.crg.eu/roderic_guigo" target="_blank">Genome Bioinformatics Lab (CRG)</a>,
          Barcelona, Spain</td>
      <td style="vertical-align: top;">Thomas Derrien, Tyler Alioto, Andrea Tanzer, Roderic Guig&oacute;</td>
    </tr>
    <tr>
      <td><a href="http://genome.ucsc.edu/" target="_blank">Genome Bioinformatics, University of California Santa Cruz</a> (UCSC), USA</td>
      <td>Rachel Harte, Mark Diekhans, Robert Baertsch, David Haussler</td>
    </tr>
    <tr>
      <td><a href="https://mblab.wustl.edu/" target="_blank">Computational Genomics Lab, Washington University</a> St. Louis (WUSTL), USA</td>
      <td>Jeltje van Baren, Charlie Comstock, David Lu, Michael Brent</td>
    </tr>
    <tr>
      <td><a href="https://www.csail.mit.edu/" target="_blank">Computer Science and Artificial Intelligence Lab</a>,
          <a href="https://www.broadinstitute.org/" target="_blank">Broad Institute of MIT and Harvard</a>, USA</td>
      <td>Mike Lin, Manolis Kellis</td>
    </tr>
    <tr>
      <td><a href="https://medicine.yale.edu/bbs/computational/" target="_blank">Computational Biology and Bioinformatics, Yale University</a> (Yale), USA</td>
      <td>Philip Cayting, Suganthi Balasubramanian, Baikang Pei, Cristina Sisu, Mark Gerstein</td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;"><a href="https://www.unil.ch/cig/home.html" target="_blank">Center for Integrative Genomics</a>,
          University of Lausanne, Switzerland</td>
      <td style="vertical-align: top;"> Cedric Howald, Alexandre Reymond</td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;"><a href="http://www.ensembl.org/info/genome/genebuild/" target="_blank">Ensembl Genebuild group</a>,
          Wellcome Trust Sanger Insitute (WTSI), Hinxton, UK</td>
      <td style="vertical-align: top;">Steve Searle, Bronwen Aken, Amonida Zadissa, Daniel Barrell
      </td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;"><a href="https://www.cnio.es/ing/grupos/plantillas/presentacion.asp?pag=726" target="_blank">
Structural Computational Biology Group</a>, Centro Nacional de Investigaciones Oncologicas (CNIO), Madrid, Spain</td>
      <td>Jos&eacute; Manuel Rodr&iacute;guez, Michael Tress, Alfonso Valencia</td>
    </tr>
  </tbody>
</table>

<h2>References</h2>

<p>
Flicek P, Amode MR, Barrell D, Beal K, Brent S, Chen Y, Clapham P, Coates G, Fairley S, Fitzgerald S <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21045057" target="_blank">
Ensembl 2011</a>.
<em>Nucleic Acids Res</em>. 2011 Jan;39(Database issue):D800-6.
</p>

<p>
Harrow J, Denoeud F, Frankish A, Reymond A, Chen CK, Chrast J, Lagarde J, Gilbert JG, Storey R, Swarbreck D <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16925838" target="_blank">
GENCODE: producing a reference annotation for ENCODE</a>.
<em>Genome Biol</em>. 2006;7 Suppl 1:S4.1-9.
</p>

<h2>Data Release Policy</h2>
<p>GENCODE data are available for use without restrictions.</p>
