<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <meta http-equiv="content-type"
 content="text/html; charset=ISO-8859-1">
  <title>wgEncodeProtGenoSuper.html</title>
</head>
<body>

<h2>Description</h2>
<p>This super track shows proteogenomic mappings of peptides from <a
 href="https://www.genome.gov/10005107">ENCODE</a> cell
types mapped to the human genome which represent fragments of translated proteins 
from the respective cell lines.  Proteogenomic methods differ from conventional 
mass-spec proteomic methods. Conventional mass-spec proteomic methods identify 
peptides by comparing them to peptides produced from known proteins. In contrast, 
proteogenomic methods compare the peptides to all peptides that might theoretically 
be produced by the six possible translation frames of the genome (three on the 
forward strand plus three on the reverse strand) to identify the genomic region 
from which the peptides were produced. Study of proteogenomic data offers insights 
on numerous regulatory mechanisms, including: translation, pre-mRNA splicing and 
transcript diversity, nonsense-mediated decay, and transcription of novel protein-coding 
genes.  Two tracks UNC/BSU ProtGenc and UNC/BSU ProtGeno that differ in database 
search scheme are presented here.  UNC/BSU ProtGenc are the combined results from 
searching against Hg19 and GENCODE of peptides of both regular and post-translational 
modifications.  UNC/BSU ProtGeno only contains the database search results of Hg19 
of regular peptides.<br>
</p>

<h2>Display Conventions and Configuration</h2>
<p>
Most ENCODE tracks contain multiple subtracks corresponding to
multiple experimental conditions. If a track contains a large
number of subtracks, only some subtracks will be displayed by default.
The user can select which subtracks are displayed via the display
controls
on the track details pages.
</p>

<h2>Credits</h2>
<p>
These data were generated and analyzed as part of the <a
 href="https://www.genome.gov/10005107">ENCODE</a> project, a
genome-wide consortium project with the aim of cataloging all
functional elements in the human genome. This effort includes
collecting a variety of data across related experimental conditions to
facilitate integrative analysis. Consequently, additional ENCODE tracks
may contain data that is relevant to the data in these tracks.
</p>

<h2>References</h2>

<p>
Giddings MC, Shah AA, Gesteland R, Moore B.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/12518051" target="_blank">
Genome-based peptide fingerprint scanning</a>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Jan 7;100(1):20-5.
</p>

<p>
Jaffe JD, Berg HC, Church GM.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/14730672" target="_blank">
Proteogenomic mapping as a complementary method to perform genome annotation</a>.
<em>Proteomics</em>. 2004 Jan;4(1):59-77.
</p>


<h2>Data Release Policy</h2>
<p>
Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset
until nine months following the release of the dataset. This date is
listed in the <span style="font-style: italic;">Restricted Until</span> 
column on the track configuration page and the download page. The full 
data release policy for ENCODE is available 
<a href="/ENCODE/terms.html">here</a>.
</p>
</body>
</html>
