<H2>Description</H2>
<p>
This release of the GENCODE Genes track (Version 3c, <B>October 2009</B>) 
shows high-quality 
manual annotations in the ENCODE regions generated by the
<a href="http://www.sanger.ac.uk/resources/databases/encode" target="_blank">GENCODE project</a>.
<p>
Version 3 of the GENCODE gene set presents a full merge between HAVANA and 
ENSEMBL, giving priority to the manually curated Havana objects and using 
ENSEMBL objects where they are different or fall into un-annotated regions.
The annotation was carried out on genome assembly GRCh37 (hg19), features are 
projected back to NCBI36 (hg18) where possible.
GENCODE 3c is a small update of version 3b (July 09 freeze) mainly for 
chromosomes 3 &amp; 4 for which the latest annotation was held back and QC'ed 
again to be used in the RNASeq Genome Annotation Assessment Project. Statistics 
about this release can be found 
<a href="http://www.sanger.ac.uk/PostGenomics/encode/stats.html" target="_blank">here</a>.

<H2> Display Conventions and Configuration </H2>

The annotations are divided into separate tracks based on
source/confidence. The GENCODE project recommends that the
annotations from level 1 &amp; 2 be used as the reference gene annotation,
level 3 was added to fill gaps for methods that analyze the entire
genome and require a full set.
</p>

<ul>
  <li> Level 1: <b>validated</b> </li>
</ul>

<dl>
  <dd>At this time only pseudogene loci, that were predicted by the
analysis-pipelines from YALE, UCSC as well as by HAVANA manual annotation from
WTSI. </dd>
</dl>

<ul>
  <li> Level 2: <b>manual annotation</b> </li>
</ul>

<dl>
  <dd>HAVANA manual annotation from WTSI.<BR>
     The following regions are considered "fully annotated" and contain
level 2 annotation from HAVANA only, although they will still be updated:
     chromosomes 1, 2, 6, 9, 10, 13, 20, 21, 22, X, Y, ENCODE pilot
regions, chr11:2353995-3878750.</dd>
<!--  <dd>HAVANA manual annotation from WTSI. </dd> -->
</dl>

<ul>
  <li> Level 3: <b>automated annotation</b> </li>
</ul>

<dl>
  <dd>ENSEMBL loci in regions where no HAVANA annotation can be found. </dd>
</dl>

</p><p>
<B>NOTE</B>: The release cycles for GENCODE, Havana and Ensembl differ. Users
are cautioned to compare release dates to determine which annotation is most
current.

</p><p>
The gene annotations are colored based on the HAVANA annotation type and
the confidence level. See the table below for the color key, as well as more
detail about the transcript and feature types.
</P>

<P>
<table border="1" bordercolor="#aaaaaa" cellpadding="4">
  <tbody>
    <tr>
      <th align="left">Class</th>
      <th align="left">Color</th>
      <th align="left">Description</th>
      <th align="left">Transcript Types (see <a href="http://vega.sanger.ac.uk/info/about/gene_and_transcript_types.html" target="_blank">Vega Transcript Types</a>)</th>
    </tr>
<tr><td>Validated_coding                <td><B><font color="#993300">Dark Orange  <!-- 153,51,0    --></font></B><td>Level 1 Validated:<BR>coding regions                   <td>protein_coding</tr>
<tr><td>Validated_processed             <td><B><font color="#FE9A2E">Light Orange <!-- 254,154,46  --></font></B><td>Level 1 Validated:<BR>processed                        <td>processed_transcript</tr>
<tr><td>Validated_processed_pseudogene  <td><B><font color="#990066">Dark Pink    <!-- 153,0,102   --></font></B><td>Level 1 Validated:<BR>processed pseudogenes            <td>processed_pseudogene, processed_transcript, transcribed_processed_pseudogene</tr>
<tr><td>Validated_unprocessed_pseudogene<td><B><font color="#AA11AA">Medium Pink  <!-- 170,17,170  --></font></B><td>Level 1 Validated:<BR>unprocessed pseudogenes          <td>transcribed_unprocessed_pseudogene, unprocessed_pseudogene</tr>
<tr><td>Validated_pseudogene            <td><B><font color="#FF33FF">Light Pink   <!-- 255,51,255  --></font></B><td>Level 1 Validated:<BR>pseudogenes                      <td>IG_pseudogene, polymorphic_pseudogene, pseudogene, retrotransposed, unitary_pseudogene</tr>

<tr><td>Havana_coding                   <td><B><font color="#993300">Dark Orange  <!-- 153,51,0    --></font></B><td>Level 2 Manual annotation:<BR>coding                   <td>IG_C_gene,IG_D_gene,IG_J_gene,IG_V_gene,protein_coding</tr>
<tr><td>Havana_nonsense                 <td><B><font color="#DF7401">Medium Orange<!-- 223,117,1   --></font></B><td>Level 2 Manual annotation:<BR>nonsense                 <td>nonsense_mediated_decay</tr>
<tr><td>Havana_non_coding               <td><B><font color="#FE9A2E">Light Orange <!-- 254,154,46  --></font></B><td>Level 2 Manual annotation:<BR>non-coding               <td>ambiguous_orf, antisense, non_coding, processed_transcript, retained_intron</tr>
<tr><td>Havana_polyA                    <td><B><font color="#000000">Black        <!-- 0,0,0       --></font></B><td>Level 2 Manual annotation:<BR>polyA                    <td>polyA_signal, polyA_site, pseudo_polyA</tr>
<tr><td>Havana_processed_pseudogene     <td><B><font color="#990066">Dark Pink    <!-- 153,0,102   --></font></B><td>Level 2 Manual annotation:<BR>processed pseudogene     <td>processed_pseudogene, transcribed_processed_pseudogene</tr>
<tr><td>Havana_unprocessed_pseudogene   <td><B><font color="#AA11AA">Medium Pink  <!-- 170,17,170  --></font></B><td>Level 2 Manual annotation:<BR>unprocessed pseudogene   <td>transcribed_unprocessed_pseudogene, unprocessed_pseudogene</tr>
<tr><td>Havana_pseudogene               <td><B><font color="#FF33FF">Light Pink   <!-- 255,51,255  --></font></B><td>Level 2 Manual annotation:<BR>pseudogene               <td>IG_pseudogene, TR_pseudogene, polymorphic_pseudogene, pseudogene, retrotransposed, unitary_pseudogene</tr>
<tr><td>Havana_TEC                      <td><B><font color="#666666">Grey         <!-- 102,102,102 --></font></B><td>Level 2 Manual annotation:<BR>TEC                      <td>TEC, artifact</tr>

<tr><td>Ensembl_coding                  <td><B><font color="#AA0000">Dark Red     <!-- 170,0,0     --></font></B><td>Level 3 Automated annotation:<BR>coding                <td>IG_C_gene, IG_D_gene, IG_J_gene, IG_V_gene, protein_coding</tr>
<tr><td>Ensembl_non_coding              <td><B><font color="#FE9A2E">Light Orange <!-- 254,154,46  --></font></B><td>Level 3 Automated annotation:<BR>non-coding            <td>antisense, non_coding, processed_transcript, retained_intron</tr>
<tr><td>Ensembl_pseudogene              <td><B><font color="#990066">Dark Pink    <!-- 153,0,102   --></font></B><td>Level 3 Automated annotation:<BR>pseudogene            <td>IG_pseudogene, miRNA_pseudogene, misc_RNA_pseudogene, pseudogene, retrotransposed, unitary_pseudogene</tr>
<tr><td>Ensembl_processed_pseudogene    <td><B><font color="#AA11AA">Medium Pink  <!-- 170,17,170  --></font></B><td>Level 3 Automated annotation:<BR>processed pseudogene  <td>processed_pseudogene</tr>
<tr><td>Ensembl_unprocessed_pseudogene  <td><B><font color="#FF33FF">Light Pink   <!-- 255,51,255  --></font></B><td>Level 3 Automated annotation:<BR>unprocessed pseudogene<td>unprocessed_pseudogene</tr>
<tr><td>Ensembl_RNA                     <td><B><font color="#FF9999">Light Red    <!-- 255,153,153 --></font></B><td>Level 3 Automated annotation:<BR>RNA transcripts       <td>Mt_rRNA, Mt_tRNA, Mt_tRNA_pseudogene, miRNA, misc_RNA, rRNA, rRNA_pseudogene, scRNA_pseudogene, snRNA, snRNA_pseudogene, snoRNA, snoRNA_pseudogene, tRNA_pseudogene, tRNAscan</tr>

<tr><td>2way_consensus_pseudogene       <td><B><font color="#5522AA">Dark Purple  <!-- 85,34,170   --></font></B><td>Level 3 Automated annotation:<BR>pseudogenes           <td>pseudogenes</tr>

<!-- Other colors: color="#006600">Dark Green color="#00CC00">Medium Green color="#00FF00">Light Green -->
  </tbody>
</table>
</P>

<P>
This track uses filtering by category to select subsets of transcripts and has
additional advanced features.  Help with these features can be found
<A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.

<h2>Methods</h2>

<P>
We aim to annotate all evidence-based gene features at high accuracy on
the human reference sequence. This includes identifying all
protein-coding loci with associated alternative variants, non-coding
loci which have transcript evidence, and pseudogenes. We integrate
computational approaches (including comparative methods), manual
annotation and targeted experimental verification.
</P>
<P>For a detailed description of the methods and references used, see
Harrow <em>et al</em> (2006).
</P>

<h2>Verification</h2>

<P> See Harrow <em>et al.</em> (2006) for information on verification
techniques.
</P>

<h2>FTP Downloads</h2>
<P> The data can be downloaded at the GENCODE ftp site <A HREF=ftp://ftp.sanger.ac.uk/pub/gencode/release_3c/ target="_blank">here</A>.

<h2>Credits</h2>

<p> This GENCODE release is the result of a collaborative effort among
the following laboratories: (contact: <A HREF="mailto:&#103;&#101;n&#99;o&#100;&#101;&#64;&#115;an&#103;&#101;&#114;.&#97;&#99;.u&#107;">
GENCODE at the Sanger Institute</A>.
<!-- above address is gencode at sanger.ac.uk -->)
</p>
<P>
<table style="text-align: left; width: 100%;" border="1" bordercolor="#aaaaaa" cellpadding="2" cellspacing="2">
  <tbody>
    <tr>
      <td style="vertical-align: top; text-align: left;"> <b>Lab/Institution</b><br>
      </td>
      <td style="vertical-align: top; text-align: left;"><b>Contributors</b><br>
      </td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;"><a href="http://www.sanger.ac.uk/HGP/havana/havana.shtml" target="_blank">HAVANA annotation group</a>,
          Wellcome Trust Sanger Insitute (WTSI), Hinxton, UK</td>
      <td style="vertical-align: top;">Adam Frankish, James Gilbert, Jennifer Harrow,
Felix Kokocinski, Stephen Trevanion, Tim Hubbard (GENCODE Principal Investigator)<br>
      </td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;"><a href="http://genome.imim.es/" target="_blank">Genome Bioinformatics Lab (CRG)</a>,
          Barcelona, Spain</td>
      <td style="vertical-align: top;">Thomas Derrien, Tyler Alioto, Andrea Tanzer, Roderic Guig&oacute;</td>
    </tr>
    <tr>
      <td><a href="http://genome.ucsc.edu/" target="_blank">Genome Bioinformatics, University of California Santa Cruz</a> (UCSC), USA</td>
      <td>Rachel Harte, Mark Diekhans, Robert Baertsch, David Haussler</td>
    </tr>
    <tr>
      <td><a href="http://mblab.wustl.edu/" target="_blank">Comp. Genomics Lab, Washington University</a> St. Louis (WUSTL), USA</td>
      <td>Jeltje van Baren, Charlie Comstock, David Lu, Michael Brent</td>
    </tr>
    <tr>
      <td><a href="http://www.csail.mit.edu/" target="_blank">Computer Science and Artificial Intelligence Lab</a>,
          <a href="http://www.broad.mit.edu/" target="_blank">Broad Institute of MIT and Harvard</a>, USA</td>
      <td>Mike Lin, Manolis Kellis</td>
    </tr>
    <tr>
      <td><a href="http://bioinfo.mbb.yale.edu/" target="_blank">Bioinformatics, Yale University</a> (Yale), USA</td>
      <td>Philip Cayting, Mark Gerstein</td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;"><a href="http://www.unil.ch/cig" target="_blank">Center for Integrative Genomics</a>,
          University of Lausanne, Switzerland</td>
      <td style="vertical-align: top;"> Cedric Howald, Alexandre Reymond</td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;"><a href="http://www.ensembl.org/info/docs/genebuild/genome_annotation.html" target="_blank">ENSEMBL genebuild group</a>,
          Wellcome Trust Sanger Insitute (WTSI), Hinxton, UK</td>
      <td style="vertical-align: top;">Bronwen Aken, Julio Fernandez Banet, Stephen Searle<br>
      </td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;"><a href="http://www.cnio.es/ing/grupos/plantillas/presentacion.asp?pag=726" target="_blank">
Structural Computational Biology Group</a>, Centro Natcional de Investigaciones Oncologicas (CNIO), Madrid, Spain</td>
      <td>Manuel Rodr&iacute;guez Jos&eacute;, Jan-Jaap Wesselink, Michael Tress, Alfonso Valencia</td>
    </tr>
  </tbody>
</table>

</P>

<h2>References</h2>

<P> Coffey AJ, Kokocinski F, Calafato MS, Scott CE, Palta P, Drury E, Joyce CJ, Leproust EM, Harrow J, Hunt S, <em>et al</em>.
<A HREF="http://www.nature.com/ejhg/journal/v19/n7/abs/ejhg201128a.html" TARGET="_BLANK">The GENCODE exome: sequencing the complete human exome</A>.
<em>European Journal of Human Genetics</em>. March 2011;19 827-831. [Epub ahead of print]
</P>

<P> Harrow J, Denoeud F, Frankish A, Reymond A, Chen CK, Chrast J, Lagarde J, Gilbert JG, Storey R, Swarbreck D <em>et al</em>.
<a href="http://genomebiology.com/2006/7/S1/S4" target="_blank">GENCODE: producing a reference annotation
for ENCODE</a>. <em>Genome Biol</em>. 2006;7 Suppl
1:S4.1-9.</P>

<H2> Data Release Policy </H2>
<p>GENCODE data are available for use without restrictions.
The full data release policy for ENCODE is available
<a HREF="../ENCODE/terms.html" TARGET=_BLANK>here</a>.</p>
