<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <meta http-equiv="content-type"
 content="text/html; charset=ISO-8859-1">
  <title>wgEncodeTfBindingSuper.html</title>
</head>
<body>

<h2>Description</h2>

<p>
Transcription is regulated through the binding of transcription factor
proteins to specific <em>cis</em>-level regulatory sites in the DNA.
The nature of this regulation depends on the transcription factor.
For example, some proteins activate transcription by recruiting RNA
polymerase, some repress transcription by suppressing this
recruitment, and others insulate proximal regions from the activity of
nearby transcriptional activators or repressors.
A key characteristic of each transcription factor protein is its DNA
binding domain.  Each DNA binding domain recognizes and interacts with
DNA that matches a specific nucleotide pattern, or motif.  These
motifs tend to be short and degenerate, so even when the DNA binding
motif is known, one cannot generally predict where a given
transcription factor may bind.  In general, transcription factor
binding is determined experimentally.
<p>

These tracks contain transcription factor binding sites determined by
ChIP-seq.  This process involves fragmenting DNA, selecting the
fragments of DNA that are bound by a certain transcription factor, and
sequencing those DNA fragments.  This generally yields a large library
of DNA sequences, including some that were bound by the transcription
factor directly, some that were bound indirectly via interactions with
other molecules, and some false positives (such as cases of
nonspecific binding).  With the appropriate analysis methods, ChIP-seq can be 
a valuable approach for elucidating transcription factor binding and 
<em>cis</em>-level regulation.

<br>

<h2>Display Conventions</h2>

These tracks are multi-view composite tracks that contains multiple
data types (views). Each view within each track
has separate display controls, as described <a
 href="/goldenPath/help/multiView.html">here</a>.
Most ENCODE tracks contain multiple subtracks, corresponding to
multiple experimental conditions. If a track contains a large
number of subtracks, only some subtracks will be displayed by default.
The user can select which subtracks are displayed via the display controls
on the track details pages.

<h2>Credits<br></h2>

These data were generated and analyzed as part of the <a
 href="https://www.genome.gov/10005107">ENCODE</a> project, a
genome-wide consortium project with the aim of cataloging all
functional elements in the human genome. This effort includes
collecting a variety of data across related experimental conditions, to
facilitate integrative analysis. Consequently, additional ENCODE tracks may 
contain data that is relevant to the data in these tracks. 

<h2>References</h2>

<p>
Euskirchen GM, Rozowsky JS, Wei CL, Lee WH, Zhang ZD, Hartman S, Emanuelsson O, Stolc V, Weissman S, Gerstein MB <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17568005" target="_blank">
Mapping of transcription factor binding regions in mammalian cells by ChIP: comparison of array- and sequencing-based technologies</a>.
<em>Genome Res</em>. 2007 Jun;17(6):898-909.
</p>

<p>
Hudson ME, Snyder M.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17191608" target="_blank">
High-throughput methods of regulatory element discovery</a>.
<em>Biotechniques</em>. 2006 Dec;41(6):673, 675, 677 passim.
</p>

<h2>Data Use Policy</h2>
<p>
<b>
External data users may freely download, analyze and publish results based on
any ENCODE data without restrictions as soon as they are released</b>.
The full policy is available here:
<a target="_blank" href="https://www.genome.gov/Pages/Research/ENCODE/ENCODE_Data_Use_Policy_for_External_Users_03-07-14.pdf">ENCODE Data Use Policy for External Users</a>

</body>
</html>
