<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <meta http-equiv="content-type"
 content="text/html; charset=ISO-8859-1">
  <title>wgEncodeExonArraySuper.html</title>
</head>
<body>
<h2>Description</h2>
<p>
These tracks display microarray data.  Samples are hybridized to the
<a href="https://www.thermofisher.com/order/catalog/product/900651" target="_blank">Affymetrix Human
Exon 1.0</a> GeneChip. In contrast to traditional microarrays, which are designed to measure
overall gene expression, the Affymetrix exon arrays contain separate
probesets for each exon. This allows large-scale measurement of
alternative splicing, alternative polyadenylation, and alternative
promoter usage. Additionally, analysis of the constitutive exons
(those included in all transcripts) offer accurate measurement of
overall gene expression levels. Further information on the
design and content of exon arrays can be found
<a href="http://tools.thermofisher.com/content/sfs/brochures/exon_array_design_technote.pdf"
target="_blank">here</a>.
<br>
</p>
<h2>Display Conventions</h2>
<p>Items on these tracks pertain to Affymetrix exon array probesets.
For
information regarding specific microarray probesets, turn on the <a
 href="/cgi-bin/hgTrackUi?db=hg19&g=affyExonArray">Affy Exon Array</a> track, 
 which can be found in the &quot;Expression&quot; track group. These tracks
are multi-view composite tracks that contains multiple
data types (views). Each view within each track
has separate display controls, as described <a
 href="/goldenPath/help/multiView.html">here</a>.
Most ENCODE tracks contain multiple subtracks, corresponding to
multiple experimental conditions. If a track contains a large
number of subtracks, only some subtracks will be displayed by default.
The user can select which subtracks are displayed via the display controls
on the track details pages.</p>
<h2>Credits<br>
</h2>
<p>These data were generated and analyzed as part of the <a
 href="https://www.genome.gov/10005107">ENCODE</a> project, a
genome-wide consortium project with the aim of cataloging all
functional elements in the human genome. This effort includes
collecting a variety of data across related experimental conditions, to
facilitate integrative analysis. Consequently, additional ENCODE tracks may 
contain data that is relevant to the data in these tracks.</p>
<h2>References</h2>
<p>
Bemmo A, Benovoy D, Kwan T, Gaffney DJ, Jensen RV, Majewski J. <a
 href="https://www.ncbi.nlm.nih.gov/pubmed/18990248"><span
 style="font-style: italic;">Gene expression and isoform variation
analysis using Affymetrix Exon Arrays.</span></a> BMC Genomics. 2008
Nov 7;9:529.</p>
<p>Gardina PJ, Clark TA, Shimada B, Staples MK, Yang Q, Veitch J,
Schweitzer A, Awad T, Sugnet C, Dee S, Davies C, Williams A, Turpaz Y. <span
 style="font-style: italic;"><a
 href="https://www.ncbi.nlm.nih.gov/pubmed/17192196">Alternative
splicing and differential gene expression in colon cancer detected by a
whole genome exon array.</a> </span>BMC Genomics. 2006 Dec 27;7:325.</p>
<h2>Data Release Policy</h2>
<p>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset
until nine months following the release of the dataset. This date is
listed in the Restricted Until column on the track configuration page
and the download page. The full data release policy for ENCODE is
available <a href="/ENCODE/terms.html">here</a>.</p>

</body>
</html>
