<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=306>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://genome.wustl.edu/genomes/detail/pristionchus-pacificus-var-california/"
    TARGET=_blank>
    <IMG SRC="../images/Pristionchus_pacificus.jpg" WIDTH=261 HEIGHT=291 ALT="P. pacificus"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Pristionchus pacificus</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Scanning electron micrograph courtesy of
	Jürgen Berger, and Ralf J. Sommer
      (<A HREF="http://www.eb.tuebingen.mpg.de/research/departments/evolutionary-biology"
      TARGET=_blank>Max Planck Institute for Developmental Biology</A>,
	All Rights Reserved.)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The Feb. 2007 <em>Pristionchus pacificus</em> (5.0) draft assembly was
produced by the 
<A HREF="http://genome.wustl.edu/genomes/detail/pristionchus-pacificus-var-california/"
TARGET=_blank>Genome Sequencing Center</A> at Washington University in St. Louis
School of Medicine.</P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
<em>P. pacificus</em> genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>chrUn:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of chrUn</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chrUn:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chrUn that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>DQ270213</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number DQ270213</TD>
    </TR>
    <TR><TD VALIGN=Top>BQ088015</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		BQ088015</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zhang</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zhang</TD>
    </TR>
    <TR><TD VALIGN=Top>Taguchi,T.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author T. Taguchi</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Taguchi T</em>, internally it
	uses the format <em>Taguchi,T.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
This genome has been sequenced to 8.92X coverage.  There are 5,106 scaffolds
comprising 169,747,139 bases, constructed from 18,752 contigs comprising
145,948,246 bases. The UCSC browser displays the 5,106 scaffolds
as a single artifical chromosome, chrUn, of size 174,852,139 bases with an
artificial gap of 1000 between each scaffold.  The largest scaffold (Contig0)
is 2,658,817 bases.  One half of the sequence, 84,873,569 bases, is
contained in 64 scaffolds, all of at least 823,357 bases (N50).
</P>
<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/priPac1/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#priPac">Downloads</A> page. </P>
<P>
The <em>P. pacificus</em> browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="../goldenPath/credits.html#worm_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
