<H2>Description</H2>
This track depicts gaps in the assembly.

<P>
Gaps are represented as black boxes in this track.
If the relative order and orientation of the contigs on either side
of the gap is supported by read pair data, 
it is a <em>bridged</em> gap and a white line is drawn 
through the black box representing the gap. 
</P>
<P>This assembly contains a single type of gap:
<UL>

<LI><B>contig</B> - gaps between the Whole Genome Shotgun contigs of a 
supercontig.  (In this context, a contig is a set of overlapping sequence
reads.  A supercontig is a set of contigs ordered and oriented during the 
Whole Genome Shotgun process using paired-end reads.)
These are represented by varying numbers of <em>N</em>s  in the assembly.  
Contig gap sizes are usually taken from read pair data.  
</UL></P>

<H2>Credits</H2>
<P>
See also: <A HREF="https://www.ncbi.nlm.nih.gov/Traces/wgs/?val=ABLE03"
TARGET=_blank>ABLE03 WGS</A> information.
</P>
<P>
The Sep. 2010 <em>Caenorhabditis japonica</em> draft assembly was produced by 
the Genome Sequencing Center at the 
<A HREF="http://genome.wustl.edu/genomes/view/caenorhabditis_japonica/"
TARGET=_blank>Washington University in St. Louis (WUSTL) School of Medicine</A>.
The UCSC caeJap4 version corresponds to WUSTL version 7.0.1,
WormBase WS220 with the following
<A HREF="http://genome.wustl.edu/data/data_use_policy" TARGET=_blank>data usage policy.</A>
</P>
