<h2>Description</h2>

<p>
This track shows alignments of alternate locus (also known as &quot;alternate haplotype&quot;)
reference sequences to main chromosome sequences in the reference genome assembly.
Some loci in the genome are highly variable, with sets of variants that tend
to segregate into distinct haplotypes.
Only one haplotype can be included in a reference assembly chromosome sequence.
Instead of providing a separate complete chromosome sequence for each haplotype,
which could cause confusion with divergent chromosome coordinates and
ambiguity about which sequence is the official reference, the
<a href="https://www.ncbi.nlm.nih.gov/grc" target=_blank>Genome Reference Consortium
(GRC)</a> adds alternate locus sequences, ranging from tens of thousands of bases
up to low millions of bases in size, to represent the distinct haplotypes.  
</p>

<h2>Display Conventions and Configuration</h2>

<p>
This track follows the display conventions for
<a href="../goldenPath/help/hgTracksHelp.html#PSLDisplay" target="_blank">
PSL alignment tracks</a>.
Mismatching bases are highlighted in red.
Several types of alignment gap may also be colored;
for more information, see
<a href="../goldenPath/help/hgIndelDisplay.html" target="_blank">
Alignment Insertion/Deletion Display Options</a>.

</p>

<h2>Credits</h2>
<p>
The alignments were provided by NCBI as GFF files and translated into the PSL
representation for browser display by UCSC.
</p>
