<h2>Description</h2>

<p>
These tracks represent the experimentally validated promoters generated by 
the <a href="https://epd.epfl.ch/" target="_blank">Eukaryotic Promoter Database</a>.
</p>

<h2>Display Conventions and Configuration</h2> 

<p>
Each item in the track is a representation of the promoter sequence identified by EPD. The
&quot;thin&quot; part of the element represents the 49 bp upstream of the annotated transcription
start site (TSS) whereas the &quot;thick&quot; part represents the TSS plus 10 bp downstream. The
relative position of the thick and thin parts define the orientation of the promoter.</p>
<p>
Note that the EPD team has created a <a href="hgHubConnect">public track hub</a> containing
promoter and supporting annotations for human, mouse, and other vertebrate and model organism
genomes.</p>

<h2>Methods</h2>
<p>
Briefly, gene transcript coordinates were obtained from multiple sources (HGNC, GENCODE, Ensembl,
RefSeq) and validated using data from CAGE and RAMPAGE experimental studies obtained from FANTOM 5,
UCSC, and ENCODE. Peak calling, clustering and filtering based on relative expression were applied
to identify the most expressed promoters and those present in the largest number of samples.</p>
<p>
For the methodology and principles used by EPD to predict TSSs, refer to Dreos <i>et al</i>.
(2013) in the References section below. A more detailed description of how this data was
generated can be found at the following links:

<ul>
  <li>
    Human promoter pipelines: 
    <a target="_blank"
    href="https://epd.epfl.ch/epdnew/documents/Hs_epdnew_006_pipeline.php">coding</a>,
    <a target="_blank"
    href="https://epd.epfl.ch/epdnew/documents/HsNC_epdnew_001_pipeline.php">non-coding</a>
  </li>
  <li>
    Mouse promoter pipelines:
    <a target="_blank"
    href="https://epd.epfl.ch/epdnew/documents/Mm_epdnew_003_pipeline.php">coding</a>,
    <a target="_blank"
    href="https://epd.epfl.ch/epdnew/documents/MmNC_epdnew_001_pipeline.php">non-coding</a>
  </li>
</ul>
</p>

<h2>Credits</h2>

<p>
Data was generated by the EPD team at the 
<a target="_blank" href="https://www.sib.swiss/">Swiss Institute of Bioinformatics</a>. 
For inquiries, contact the EPD team using this <a target="_blank"
href="https://epd.epfl.ch/webmail.php">on-line form</a> 
or email 
<A HREF="mailto:&#112;&#104;&#105;&#108;&#105;&#112;&#112;.
&#98;u&#99;h&#101;r&#64;&#101;&#112;fl.
&#99;&#104;">
&#112;&#104;&#105;&#108;&#105;&#112;&#112;.
&#98;u&#99;h&#101;r&#64;&#101;&#112;fl.
&#99;&#104;</A>
<!-- above address is philipp.bucher at epfl.ch -->
.
</p>

<h2>References</h2>

<p>
Dreos R, Ambrosini G, Perier RC, Bucher P.
<a href="https://nar.oxfordjournals.org/content/41/D1/D157.full" target="_blank">
EPD and EPDnew, high-quality promoter resources in the
next-generation sequencing era</a>. <em>Nucleic Acids
Res</em>. 2013 Jan 1;41(D1):D157-64. PMID: <a
href="https://www.ncbi.nlm.nih.gov/pubmed/23193273"
target="_blank">23193273</a>.
</p>

