<H2>Description</H2>
<P>
This track shows a measure of evolutionary conservation in $organism, rat, human, 
dog, cow, opossum, chicken, frog, zebrafish, and Tetraodon based on a phylogenetic hidden Markov model (phastCons).
Multiz alignments of the following assemblies were used to generate this
annotation: 
<UL>
<LI>$organism ($date, $db)
<LI>rat (Jun. 2003, rn3)
<LI>human (May 2004, hg17)
<LI>dog (Jul. 2004, canFam1)
<LI>cow (Sep. 2004, bosTau1)
<LI>opossum (Oct. 2004, monDom1)
<LI>chicken (Feb. 2004, galGal2)
<LI>frog (Oct. 2004, xenTro1)
<LI>zebrafish (Jun. 2004, danRer2)
<LI>Tetraodon (Feb. 2004, tetNig1)
</UL></P>

<H2>Display Conventions and Configuration</H2> <P>
<P>
In full display mode, this track shows the overall conservation score across 
all species, as well as pairwise alignments 
of rat, human, dog, cow, opossum, chicken, zebrafish and Tetraodon, each 
aligned to the $organism genome.  The pairwise alignments are
shown in dense display mode using a grayscale 
density gradient.  The checkboxes in the track configuration section allow
the exclusion of species from the pairwise display; however, this does not
remove them from the conservation score display.  </P>
<P>
When zoomed-in to the base-display level, the track shows the base 
composition of each alignment. The numbers and symbols on the Gaps
line indicate the lengths of gaps in the $organism sequence at those 
alignment positions relative to the longest non-$organism sequence. 
If there is sufficient space in the display, the size of the gap is shown; 
if not, and if the gap size is a multiple of 3, a &quot;*&quot; is displayed, 
otherwise &quot;+&quot; is shown. 
To view detailed information about the alignments at a specific position,
zoom in the display to 30,000 or fewer bases, then click on the alignment.</P>
<P>
This track may be configured in a variety of ways to highlight different 
aspects of the displayed information. Click 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>here</A>
for an explanation of the configuration options.</P>

<H2>Methods</H2>
<P> 
Best-in-genome blastz pairwise alignments 
were multiply aligned using multiz, beginning with $organism-rat alignments
and subsequently adding in the other species, as diagrammed above.
The resulting multiple alignments were then assigned 
conservation scores by phastCons.</P>
<P>
The phastCons program computes conservation scores based on a phylo-HMM, a
type of probabilistic model that describes both the process of DNA
substitution at each site in a genome and the way this process changes from
one site to the next (Felsenstein and Churchill 1996, Yang 1995, Siepel and
Haussler 2005).  PhastCons uses a two-state phylo-HMM, with a state for
conserved regions and a state for non-conserved regions.  The value plotted
at each site is the posterior probability that the corresponding alignment
column was "generated" by the conserved state of the phylo-HMM.  These
scores reflect the phylogeny (including branch lengths) of the species in
question, a continuous-time Markov model of the nucleotide substitution
process, and a tendency for conservation levels to be autocorrelated along
the genome (i.e., to be similar at adjacent sites).  The general reversible
(REV) substitution model was used.  Note that, unlike many
conservation-scoring programs, phastCons does not rely on a sliding window
of fixed size, so short highly-conserved regions and long moderately
conserved regions can both obtain high scores.  More information about
phastCons can be found in Siepel et al. (2005).</P> 
<P> 
PhastCons currently treats alignment gaps as missing data, which
sometimes has the effect of producing undesirably high conservation scores
in gappy regions of the alignment.  We are looking at several possible ways
of improving the handling of alignment gaps.</P>

<H2>Credits</H2>
<P>
This track was created at UCSC using the following programs:
<UL>
<LI>
Blastz and multiz by Minmei Hou, Scott Schwartz and Webb Miller of the 
<A HREF="http://bio.cse.psu.edu/" TARGET=_blank>Penn State Bioinformatics 
Group</A>. 
<LI>
AxtBest, axtChain, chainNet, netSyntenic, and netClass 
by Jim Kent at UCSC. 
<LI> PhastCons by Adam Siepel at Cornell University. 
<LI>"Wiggle track" plotting software by Hiram Clawson at UCSC.
</UL>
</P>
<P>The phylogenetic tree is based on Murphy et al. (2001) and general
consensus in the vertebrate phylogeny community.
</P>

<H2>References</H2>

<H3>Phylo-HMMs and phastCons</H3>
<P>
Felsenstein J and Churchill GA (1996).  
<A HREF="http://mbe.oxfordjournals.org/cgi/content/abstract/13/1/93"
TARGET=_blank>A hidden Markov model approach to 
variation among sites in rate of evolution</A>.
<em>Mol Biol Evol</em> 13:93-104. </P>
<P>
Siepel A and Haussler D (2005). Phylogenetic hidden Markov models.
In R. Nielsen, ed., <em>Statistical Methods in Molecular Evolution</em>,
pp. 325-351, Springer, New York. </P>
<P>
Siepel, A., Bejerano, G., Pedersen, J.S., Hinrichs, A., Hou, M., Rosenbloom, 
K., Clawson, H., Spieth, J., Hillier, L.W., Richards, S., Weinstock, G.M., 
Wilson, R. K., Gibbs, R.A., Kent, W.J., Miller, W., and Haussler, D. 
<A HREF="http://www.genome.org/cgi/doi/10.1101/gr.3715005"
TARGET=_blank>Evolutionarily conserved elements in vertebrate, insect, worm, 
and yeast genomes</A>.
<em>Genome Res.</em> <B>15</B>, 1034-1050 (2005).</P>
<P>
Yang Z (1995).  
<A HREF="http://www.genetics.org/cgi/content/abstract/139/2/993" 
TARGET=_blank>A space-time process model for the evolution of DNA
sequences</A>.  <em>Genetics</em>, 139:993-1005. </P>

<H3>Chain/Net</H3>
<P>
Kent, W.J., Baertsch, R., Hinrichs, A., Miller, W., and Haussler, D.
<A HREF="http://www.pnas.org/cgi/content/abstract/1932072100v1"
TARGET=_blank>Evolution's cauldron: Duplication, deletion, and rearrangement
in the mouse and human genomes</A>.
<em>Proc Natl Acad Sci USA</em> <B>100</B>(20), 11484-11489 (2003).</P>

<H3>Multiz</H3>
<P>
Blanchette, M., Kent, W.J., Riemer, C., Elnitski, L., Smit, A.F.A.,
Roskin, K.M., Baertsch, R., Rosenbloom, K., Clawson, H., Green, E.D.,
Haussler, D., Miller, W.
<A HREF="http://www.genome.org/cgi/content/abstract/14/4/708"
TARGET=_blank>Aligning multiple genomic sequences with the threaded blockset
aligner</A>.
<em>Genome Res.</em> <B>14</B>(4), 708-15 (2004).</P>

<H3>Blastz</H3>
<P>
Chiaromonte, F., Yap, V.B., Miller, W.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=11928468&dopt=Abstract"
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>.
<em>Pac Symp Biocomput 2002</em>, 115-26 (2002).</P>
<P>
Schwartz, S., Kent, W.J., Smit, A., Zhang, Z., Baertsch, R., Hardison, R.,
Haussler, D., and Miller, W.
<A HREF="http://www.genome.org/cgi/content/abstract/13/1/103"
TARGET=_blank>Human-Mouse Alignments with BLASTZ</A>.
<em>Genome Res.</em> <B>13</B>(1), 103-7 (2003).</P>

<H3>Phylogenetic Tree</H3>
<P>
Murphy, W.J., et al.
<A HREF="http://www.sciencemag.org/cgi/content/abstract/294/5550/2348"
TARGET=_blank>Resolution of the early placental mammal radiation using Bayesian phylogenetics</A>.
<em>Science</em> <B>294</B>(5550), 2348-51 (2001).</P>
