<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=215>
  <TR><TD ALIGN=RIGHT><A HREF="http://www.sanger.ac.uk/resources/zebrafish/genomeproject.html"
    TARGET=_blank>
    <IMG SRC="../images/Danio_rerio.jpg" WIDTH=200 HEIGHT=65 ALT="Zebrafish"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Danio rerio</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      (NHGRI <A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Zebrafish&id=79130" 
      TARGET=_blank>Press Photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The May 2005 zebrafish (<em>Danio rerio</em>) Zv5 assembly was produced by 
The Wellcome Trust Sanger Institute in collaboration with the Max Planck 
Institute for Developmental Biology in Tuebingen, Germany, and the Netherlands
Institute for Developmental Biology (Hubrecht Laboratory), Utrecht, The 
Netherlands. 

<H3>Sample position queries</P></H3>
<P>
A genome position can be specified by the accession number of
an mRNA or EST, a scaffold range,
<!-- an Ensembl transcript ID, -->
or keywords from 
the GenBank description of an mRNA. The following list provides 
examples of various types of position queries for the zebrafish genome. Note 
that some position queries (e.g. "huntington") may return matches to the mRNA 
records of other species. In these cases, the mRNAs are mapped to their 
homologs in zebrafish.  See the 
<A HREF="../goldenPath/help/hgTracksHelp.html" TARGET=_blank>User Guide</A> for more help. 
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
<TR><TD VALIGN=Top NOWRAP><B>Request:</B><br></TD>
	<TD VALIGN=Top COLSPAN=2><B>&nbsp;&nbsp; Genome Browser Response:</B><br></TD></TR>
	
<TR><TD VALIGN=Top><br></TD></TR>
	
<TR><TD VALIGN=Top NOWRAP>chr1</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays all of chromosome 1</TD></TR>
<TR><TD VALIGN=Top NOWRAP>chr1:1-200000</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays first two hundred thousand bases of chromosome 1</TD></TR>
    <TR><TD VALIGN=Top NOWRAP>chr1:100000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 1 that spans 2000 bases, starting with position 100000 </TD>
    </TR>
<TR><TD VALIGN=Top><br></TD></TR>

<TR><TD VALIGN=Top NOWRAP>U30710</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region containing zebrafish mRNA with GenBank accession number U30710</TD></TR>

<TR><TD VALIGN=Top NOWRAP>AA658622</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region containing zebrafish EST with GenBank accession AA658622
</TD></TR>

<!--Commented out until implemented
<TR><TD VALIGN=Top NOWRAP>ENSDART00000049940</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region containing Ensembl gene prediction transcript ENSDART00000049940
</TD></TR>
-->

<TR><TD VALIGN=Top><br></TD></TR>

<TR><TD VALIGN=Top NOWRAP>p53</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs related to the p53 tumor suppressor</TD></TR> 
</TD></TR>


<TR><TD VALIGN=Top NOWRAP>pseudogene mRNA</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists transcribed pseudogenes but not cDNAs, in GenBank</TD></TR>
<TR><TD VALIGN=Top NOWRAP>homeobox caudal</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes in GenBank</TD></TR>
<TR><TD VALIGN=Top NOWRAP>zinc finger</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD></TR>
<TR><TD VALIGN=Top NOWRAP>kruppel zinc finger</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD></TR>
<TR><TD VALIGN=Top NOWRAP>huntington</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs associated with Huntington's disease</TD></TR>

<TR><TD VALIGN=Top><br></TD></TR>

<TR><TD VALIGN=Top NOWRAP>porter</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientists named Porter</TD></TR>
<TR><TD VALIGN=Top NOWRAP>Amsterdam,A.</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by co-author A. Amsterdam</TD></TR>

<TR><TD VALIGN=Top><br></TD></TR>
	
<TR><TD COLSPAN="3" > Use this last format for author queries. Although 
GenBank requires the search format <em>Amsterdam A</em>, internally it uses
the format <em>Amsterdam,A.</em>
</TABLE>
<P>
<HR>
<H3>Assembly details</H3>
<P>
The Zv5 assembly consists of 1,630,306,866 bp in 16,214 scaffolds 
(N50 = 1,116,981 bp) with a sequence coverage of approximately 6.5-7x. 
The assembly has been tied to the FPC map (data freeze 15th February, 2005) 
and contains 699 Mb from 4,519 sequenced clones.
<P>
This assembly was constructed using two different strategies. In the first, 
a whole genome shotgun (WGS) approach was used.  DNA from 1000 five-day-old 
Tuebingen embryos was used to generate plasmid and fosmid libraries. The 
libraries were sequenced and the resulting traces were assembled with the 
Sanger Institute assembler, Phusion. The second approach used traditional 
clone mapping and sequencing techniques to produce a higher quality genome 
sequence. BAC libraries were selected and fingerprinted to generate a 
fingerprint contig (FPC) map. From this map, a tiling path was calculated 
that covered the genome sequence clone-by-clone. Clones from this path were 
selected for high quality sequencing and then pieced together to form the 
genome sequence. 
<P>
74% of the sequence (1,200,129,620 bp) is in FPC contigs and is tied to 
chromosomes 1-25. These contigs provided a template for placing the unfinished 
sequence. The remaining sequence was filled with WGS contigs using a 
combination of sequence alignment and BAC end positions as well as features
such as zebrafish cDNAs and markers. 183,993,739 bp of sequence is in 265 
scaffolds tied to unplaced FPC contigs and there are 246,183,507 bp in 
14,676 NA scaffolds. The WGS assembly consists of 20,541,433 reads totaling 
14,160,626,498 bp and this includes an additional set of 6,882,050 reads from 
a new library generated from a single Tuebingen double haploid zebrafish. Reads were clustered using Phusion, and Phrap was used to assemble clusters and for 
consensus generation. The resulting 247,928 contigs have an N50 size of 
20,629 bp. From these, 105,987 scaffolds or supercontigs were created with an 
N50 size of 687,451 bp.
<P>
The Sanger Institute notes there is high level of misassembly present in 
this release due to the large amount of polymorphism in the DNA source. 
Regions where the physical map is incomplete or contains gaps have sequence
with a higher number of misassemblies. Highly variable regions within the 
genome posed assembly difficulties, most likely because the sequences 
originated from different haplotypes. This also results in assembly dropouts
and duplications.

For more information about this assembly, see the Sanger Institute 
page for the <A HREF="http://www.sanger.ac.uk/resources/zebrafish/genomeproject.html"
TARGET=_blank><em>Danio rerio</em> Sequencing Project</A>. 
<P>
Downloads of the Zebrafish data and annotations can be obtained from the UCSC
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/danRer3/" TARGET=_blank>FTP 
site</A> or 
<A HREF=http://hgdownload.soe.ucsc.edu/downloads.html#zebrafish>Downloads 
page</A>. This data set has <A HREF="http://www.sanger.ac.uk/legal/" TARGET=_blank>specific conditions for use</A>.
The danRer1 annotation tracks were generated by UCSC and collaborators worldwide. 
Special thanks to the Zebrafish Genome Initiative at Children's Hospital in 
Boston, MA, USA for their collaboration on this release.
See the <A HREF="../goldenPath/credits.html#zebrafish_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
