<H2>Description</H2>
<P>This track shows duplicated regions of the genome.
The percent identity of the duplicon is reflected in
the level of gray in the graphic display.  In the
details page the 'score' is ten times the percent identity
(excluding gaps).</P>

<H2>Methods</H2>
Blastz and axtChain were run on the $organism genome
vs. itself to create gapped alignments.  The chains were
broken up at gaps of 10,000 bases or more.  After this
break up chains spanning less than 5,000 bases or where
more than 50% of the bases were in gaps were discarded.
This procedure avoids including old paralogous genes that
are under selection in the set of duplicons.

<H2>Credits</H2>
This track was created by Jim Kent.
