<H2>Description</H2>
<P>
This track displays multiz alignments of the Feb. 2003 mouse draft
assembly and the rat Jan. 2003 assembly to the $organism genome.</P>

<H2>Methods</H2>
<P>
Multiz is a multiple alignment program that takes
<A HREF="http://bio.cse.psu.edu/dist/blastz.pdf" TARGET=_blank>blastz</A> 
/axtBest alignments as input.  It uses the same scoring
matrix as blastz between pairs of sequences:  
<PRE>
      A     C     G     T
A    91  -114   -31  -123
C  -114   100  -125   -31
G   -31  -125   100  -114
T  -123   -31  -114    91

O = 400, E = 30, K = 3000, L = 3000, M = 50
</PRE>
</P><P>
The overall score is the sum of the score between all pairs.  </P>

<H2>Credits</H2>
<P>
<em>Multiz</em> was created by Minmei Hou and Webb Miller from the
<A HREF="http://bio.cse.psu.edu/" TARGET=_blank>
Penn State Bioinformatics Group</A>. Jim Kent wrote
<A HREF="http://www.genome.org/cgi/content/full/13/1/103" TARGET=_blank>axtBest</A>
and the scripts to run multiz genome-wide and to display
the alignments in this browser.  
Mouse sequence data 
are provided by the Mouse Genome Sequencing Consortium. 
Rat sequence data are provided by the Rat Sequencing 
Consortium.</P>

<H2>References</H2>
<P><A HREF="http://bio.cse.psu.edu/dist/blastz.pdf" TARGET=_blank>Human-Mouse Alignments with BLASTZ</A>. Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison R, Haussler D, and Miller W.  <em>Genome Research</em> 2003 Jan;13(1):103-7.</P>

<P><A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=11928468&dopt=Abstract" TARGET=_blank>Scoring pairwise genomic sequence alignments</A>. Chiaromonte F, Yap VB, Miller W. <em>Pac Symp Biocomput</em> 2002;:115-26.</P>

