<H2>Description</H2>
This track depicts gaps in the draft assembly (Sep. 2011, Broad CanFam3.1 (GCA_000002285.2))
of the $organism genome.

<P>
Gaps are represented as black boxes in this track.
If the relative order and orientation of the contigs on either side
of the gap is supported by read pair data, 
it is a <em>bridged</em> gap and a white line is drawn 
through the black box representing the gap. 
</P>
<P>This assembly contains the following principal types of gaps:

<UL>
<LI><B>Short Arm</B> - 100 bases of <em>N</em>s marked at the beginning of
each chromosome</LI>
<LI><B>Fragment</B> - gaps between the Whole Genome Shotgun contigs of a 
supercontig. These are represented by varying numbers of <em>N</em>s in the 
assembly. In this context, a contig is a set of overlapping sequence reads and  
a supercontig is a set of contigs ordered and oriented during the 
Whole Genome Shotgun process using paired-end reads. Fragment gap sizes 
are usually taken from read pair data.</LI>
<LI><B>Clone</B> - gaps between supercontigs linked by the physical map.  
In general, these are represented by 1,000 <em>N</em>s  in the assembly.</LI>
<LI><B>Other</B> - sequences of <em>N</em>s in the assembly that were not
annotated in the AGP file as gap.  Added by UCSC.</LI>

</UL></P>

