<H2>Description</H2>

The Structural variations track shows duplications (DUP), deletions (DEL),
insertions (INS) and inversions (INV) which were identified from 27 domestic
dogs of 19 breeds.

<H2>Methods</H2>

The structural variations (SV) were called from 10x sequencing data.
10x reads were aligned to the CanFam4 reference by long ranger.
Multiple algorithms were used to scan the SVs in different types and sizes.
Individual SVs were QC filtered and merged as a nonredundant dataset.

<H2>Credits</H2>

The structural variation track was produced as a part of GSD_1.0/CanFam4
reference assembly project by the Lindblad-Toh group of comparative genomics
at Uppsala University. Please cite the paper if you use the data for your
research.

<H2>References</H2>

Wang, C.; Wallerman, O.; Arendt, M.-L.; Sundstr&ouml;m, E.; Karlsson, &Aring;.;
Nordin, J.; M&auml;kel&auml;inen, S.; Pielberg, G. R.; Hanson, J.;
Ohlsson, &Aring;.; Saellstr&ouml;m, S.; R&ouml;nnberg, H.; Ljungvall, I.;
H&auml;ggstr&ouml;m, J.; Bergstr&ouml;m, T. F.;
Hedhammar, &Aring;.; Meadows, J. R. S.; Lindblad-Toh, K.
<BR>
<A HREF="https://doi.org/10.1101/2020.07.02.185108" target=_blank>
A New Long-Read Dog Assembly Uncovers Thousands of Exons and Functional
Elements Missing in the Previous Reference</A>.
<A HREF="https://www.biorxiv.org/content/10.1101/2020.07.02.185108v1" target=_blank>
bioRxiv 2020, 2020.07.02.185108</A>.
