<H2>Description</H2>
<P>
This track shows the draft assembly of the $organism genome.  
This release was assembled from whole genome shotgun (WGS) reads using the 
Atlas genome assembly system. Several WGS libraries, with inserts of 2 - 4 kb 
and 4 - 6 kb, were used to produce the data. Approximately 23 million reads 
were assembled, representing about 17.7 Gb of sequence and 6.2x coverage of the 
(clonable) bovine genome. Highly repeated sequences and BAC clones sequences 
were omitted from this assembly; these will be placed in a subsequent version 
of the draft sequence. </P>
<P>
The assembly contains chromosomes 1-29, X, M, and Bin0, 
as well as 98058 scaffolds. Bin0 is an artificial chromosome consisting of 
contigs of 3 or fewer reads that were not incorporated into scaffolds. Baylor 
Chr30 is labeled chrX in the UCSC Genome Browser. Baylor artificial chromosome 
ChrUn was split into scaffolds for display in the browser. chrM was obtained 
from NCBI (gi=12800). </P>
<P>
In dense mode, this track depicts the contigs that make up the 
currently viewed chromosome or scaffold. 
Contig boundaries are distinguished by the use of alternating gold and brown 
coloration. Where gaps
exist between contigs, spaces are shown between the gold and brown
blocks.  The relative order and orientation of the contigs
within a chromosome or scaffold is always known; therefore, a line is drawn 
in the graphical display to bridge the blocks.</P>
<P>
All components within this track are of fragment type &quot;W&quot;
(whole genome shotgun contig). </P>



