<H2>Description</H2>
<P>
This track shows the draft assembly of the $organism genome.  

Whole-genome shotgun reads were assembled into contigs.  When possible, 
contigs were grouped into scaffolds (also known as &quot;supercontigs&quot;).
The order, orientation and gap sizes between contigs within a scaffold are
based on paired-end read evidence. </P>
<P>
In dense mode, this track depicts the contigs that make up the 
currently viewed scaffold. 
Contig boundaries are distinguished by the use of alternating gold and brown 
coloration. Where gaps
exist between contigs, spaces are shown between the gold and brown
blocks.  The relative order and orientation of the contigs
within a scaffold is always known; therefore, a line is drawn in the graphical
display to bridge the blocks.</P>
<P>
All components within this track are of fragment type &quot;W&quot;: 
Whole Genome Shotgun contig. </P>

<H3>Assembly details</H3>
<P>
The genome has been sequenced to 6.3X coverage.  See also:
<A HREF="http://genome.wustl.edu/genomes/detail/caenorhabditis-japonica/"
TARGET=_blank>WUSTL assembly page</A>
</P>
<P>
There are 35,273 contigs in the assembly containing 129,347,181 nucleotides.
The mean size of these contigs is 3,667 bases, with an N50 measurement of
6,365 contigs of size 5,781 bases or larger containing half of the sequence.
</P>
<P>
The smaller contigs have been assembled into 4,657 supercontigs
named Contig0 through Contig10115 containing 151,722,573 nucleotides.
The mean size of these supercontigs is 32,579 bases, with an N50 measurement
of 596 supercontigs of size 68,505 bases or larger containing half of the
sequence.
</P>
<P>
The UCSC genome browser combines the supercontigs together as an
artifical chromosome chrUn with 1,000 base gaps between these supercontigs.
This chrUn sequence is 156,378,573 bases, with the artifical 1,000 base
gaps covering 4,656,000 bases.
</P>
