<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=253 HEIGHT=215>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://www.uky.edu/Ag/Horsemap/"
    TARGET=_blank>
    <IMG SRC="../images/Equus_caballus.jpg" WIDTH=238 HEIGHT=200 ALT="Horse"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT>"Twilight" - <FONT SIZE=-1><em>Equus caballus</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of NHGRI
      (<A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Horse&id=79086" 
      TARGET=_blank>press photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The Jan. 2007 <em>Equus caballus</em> draft assembly EquCab1 (UCSC version
equCab1) was produced by <A HREF="http://www.broadinstitute.org/"
TARGET=_blank>The Broad Institute</A>.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Horse genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>chr13</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of chromosome 13</TD>
    </TR>
    <TR><TD VALIGN=Top>chr13:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of chromosome 13</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr13:1000000+20000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 13 that spans 20,000 bases, starting with position 1,000,000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>AF506972</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number AF506972</TD>
    </TR>
    <TR><TD VALIGN=Top>BM734875</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		BM734875 on chr chr11</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene mRNA</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>smith</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Smith</TD>
    </TR>
    <TR><TD VALIGN=Top>Collins,M.E.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author M.E. Collins</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Collins ME</em>, internally it
	uses the format <em>Collins,M.E.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
The genome has been sequenced to 6.8X coverage.
Approximately 84% of the sequence has been anchored to chromosomes, 
which include autosomes 1-31 and the sex chromosome X.
Unanchored contigs that could not be localized to a chromosome 
have been concatenated into the virtual 
chromosome &quot;chrUn&quot; separated by gaps of 1,000 bp.
The mitochondrial sequence is also available in the Genome Browser as the 
virtual chromosome &quot;chrM&quot;. For more details about the assembly, see
the Broad Institute <A HREF="http://www.broadinstitute.org/mammals/horse"
TARGET=_blank>Horse Genome Project</A> page.
</P>
<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/equCab1/">FTP
server</A> or <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#horse">Downloads</A> page. 
These data have
<A HREF="../goldenPath/credits.html#horse_use">specific 
conditions for use</A>.
The Horse browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#horse_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
