<h2>Description</h2>
<p>
This track displays data from <a
href="https://pubmed.ncbi.nlm.nih.gov/31097668/"
target="_blank"><em>Single-cell genomics identifies cell type-specific
molecular changes in autism</em></a>. Single-nucleus RNA sequencing (snRNA-seq)
was performed on post-mortem cortical tissue samples from patients with autism
spectrum disorder (ASD) as well as control donors. A total of 17 cell clusters
were identified using known cell type markers found in Velmeshev <em>et
al</em>., 2019.</p>

<p>
This track collection contains five bar chart tracks of RNA expression in the human
cerebral cortex where cells are grouped by cell type 
(<a href="hgTrackUi?g=cortexVelmeshevCellType">Cortex Cells</a>), diagnosis
(<a href="hgTrackUi?g=cortexVelmeshevDiagnosis">Cortex Diagnosis</a>), donor 
(<a href="hgTrackUi?g=cortexVelmeshevDonor">Cortex Donor</a>), sample 
(<a href="hgTrackUi?g=cortexVelmeshevSample">Cortex Sample</a>), and sex
(<a href="hgTrackUi?g=cortexVelmeshevSex">Cortex Sex</a>). 
The default track displayed is <a href="hgTrackUi?g=cortexVelmeshevCellType">Cortex Cells</a>.</p>

<h2>Display Conventions</h2>
<p>
The cell types are colored by which class they belong to according to the following table.</p>

<p>
<table cellpadding='2'>
  <thead><tr>
    <th style="border-bottom: 2px solid;">Color</th>
    <th style="border-bottom: 2px solid;">Cell classification</th>
  </tr></thead>
  <tr><td style='background-color: #F0C000;'></td><td>neural</td></tr>
  <tr><td style="background-color: #F01111;"></td><td>immune</td></tr>
  <tr><td style="background-color: #00C000;"></td><td>endothelial</td></tr>
  <tr><td style="background-color: #80D000;"></td><td>glia</td></tr>
</table>
</p>

<p>
Cells that fall into multiple classes will be colored by blending the colors associated
with those classes. The colors will be purest in the 
<a href="hgTrackUi?g=cortexVelmeshevCellType">Cortex Cells</a> subtrack, where
the bars represent relatively pure cell types. They can give an overview of the
cell composition within other categories in other subtracks as well.</p>

<h2>Method</h2>
<p>
Healthy cortical samples were taken from 16 controls (ages 4-22) without 
neurological disorders and 15 ASD patients (ages 7-21). A total of 41 post-mortem
tissue samples were obtained from both the prefrontal cortex (PFC) and anterior
cingulate cortex (ACC). When present, subcortical white matter was removed
prior to collection from cortical samples containing all layers of cortical
grey matter. ASD and control samples were matched for sex and age and processed
together to minimize batch effects. Nuclei were isolated from brain tissue
using a glass dounce homogenizer in lysis buffer and then filtered twice
through a 30 <span>&#181;</span>m cell strainer. Next, samples were processed
using 10x Genomics 3' library kit and the resulting single-nucleus libraries
were pooled together and sequenced on an Illumina NovaSeq 6000. This process
generated 104,559 single-nuclei gene expression profiles in total. </p>

<p>
The cell/gene matrix and cell-level metadata was downloaded from the <a
href="http://cells.ucsc.edu/?ds=autism" target="_blank">UCSC Cell Browser</a>. The
UCSC command line utility matrixClusterColumns, matrixToBarChart, and bedToBigBed
were used to transform these into a bar chart format bigBed file that can be
visualized. The coloring was done by defining colors for the broad level cell
classes and then using another UCSC utility, hcaColorCells, to interpolate the
colors across all cell types. The UCSC utilities can be found on 
<a href="https://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads"
target="_blank">our download server</a>.</p>

<h2>Data Access</h2>
<p>
The raw <a href="/goldenPath/help/barChart.html">bar chart</a> data can be
explored interactively with the <a href="../cgi-bin/hgTables">Table
Browser</a>, or the <a href="../cgi-bin/hgIntegrator">Data Integrator</a>. For
automated analysis, the data may be queried from our <a
href="/goldenPath/help/api.html">REST API</a>. Please refer to our <a
href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing
list archives</a> for questions, or our <a
href="../FAQ/FAQdownloads.html#download36">Data Access FAQ</a> for more
information.</p>


<h2>Credit</h2>
<p>
Thanks to Dmitry Velmeshev and to the many authors who worked on
producing and publishing this data set. The data were integrated into the UCSC
Genome Browser by Jim Kent and Brittney Wick then reviewed by by Daniel Schmelter. 
The UCSC work was paid for by the Chan Zuckerberg Initiative.</p>

<h2>References</h2>
<p>
Velmeshev D, Schirmer L, Jung D, Haeussler M, Perez Y, Mayer S, Bhaduri A, Goyal N, Rowitch DH,
Kriegstein AR.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31097668" target="_blank">
Single-cell genomics identifies cell type-specific molecular changes in autism</a>.
<em>Science</em>. 2019 May 17;364(6441):685-689.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31097668" target="_blank">31097668</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7678724/" target="_blank">PMC7678724</a>
</p>
