<h2>Description</h2>
<p>
This track displays data from <a href="https://pubmed.ncbi.nlm.nih.gov/31604275/" 
target="_blank"><em>Spatiotemporal immune zonation of the human kidney</em></a>. 
Droplet-based single-cell RNA sequencing (scRNA-seq) was used to profile 40,268 
mature human kidney cells. After principal component analysis, identified clusters 
were manually curated into four major cellular compartments using canonical markers 
as found in Stewart <em>et al</em>., 2019: endothelial, immune, fibroblast, and epithelium.

<p>
This track collection contains six bar chart tracks of RNA expression in the
human kidney where cells are grouped by merged cell type 
(<a href="hgTrackUi?g=kidneyStewartCellType">Kidney Cells</a>), broad cell type 
(<a href="hgTrackUi?g=kidneyStewartBroadCellType">Kidney Broad CT</a>), detailed cell type 
(<a href="hgTrackUi?g=kidneyStewartDetailedCellType">Kidney Details</a>), compartment
(<a href="hgTrackUi?g=kidneyStewartCompartment">Kidney Compartment</a>), experiment 
(<a href="hgTrackUi?g=kidneyStewartExperiment">Kidney Experiment</a>), and project 
(<a href="hgTrackUi?g=kidneyStewartProject">Kidney Project</a>).
The default track displayed is 
<a href="hgTrackUi?g=kidneyStewartCellType">Kidney Cells</a>.</p>

<h2>Display Conventions</h2>
<p>
The cell types are colored by which class they belong to according to the following table.</p>

<p>
<table cellpadding='2'>
  <thead><tr>
    <th style="border-bottom: 2px solid;">Color</th>
    <th style="border-bottom: 2px solid;">Cell classification</th>
  </tr></thead>
  <tr><td style="background-color: #B05020;"></td><td>fibroblast</td></tr>
  <tr><td style="background-color: #F01111;"></td><td>immune</td></tr>
  <tr><td style="background-color: #0000FF;"></td><td>kidney specific</td></tr>
  <tr><td style="background-color: #0099C0;"></td><td>epithelial</td></tr>
  <tr><td style="background-color: #00C000;"></td><td>endothelial</td></tr>
</table>
</p>

<p>
Cells that fall into multiple classes will be colored by blending the colors associated
with those classes.</p>

<h2>Data Access</h2>
<p>
The raw <a href="/goldenPath/help/barChart.html">bar chart</a> data can be
explored interactively with the <a href="../cgi-bin/hgTables">Table
Browser</a>, or the <a href="../cgi-bin/hgIntegrator">Data Integrator</a>. For
automated analysis, the data may be queried from our <a
href="/goldenPath/help/api.html">REST API</a>. Please refer to our <a
href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing
list archives</a> for questions, or our <a
href="../FAQ/FAQdownloads.html#download36">Data Access FAQ</a> for more
information.</p>

<h2>Method</h2>
<p>
14 mature healthy human kidney samples were obtained from individuals (ages
1-72) that either underwent tumor nephrectomy (n=10) or from kidneys donated
for transplantation (n=4) but were unsuitable for use. Kidney tissues from
tumor nephrectomies were collected from unaffected areas estimated to be
corticomedullary. Samples were enzymatically dissociated and enriched for live
cells (experiment set 1) or enriched for leukocytes with a density gradient and
then for live cells (experiment set 2). Single cell libraries were prepared
using 10x Genomics 3' v2 kit and sequenced on an Illumina HiSeq4000.</p>

<p>
The cell/gene matrix and cell-level metadata was downloaded from the <a
href="http://cells.ucsc.edu/?ds=kidney-atlas" target="_blank">UCSC Cell Browser</a>. 
The UCSC command line utility matrixClusterColumns, matrixToBarChart, and bedToBigBed
were used to transform these into a bar chart format bigBed file that can be
visualized. The coloring was done by defining colors for the broad level cell
classes and then using another UCSC utility, hcaColorCells, to interpolate the
colors across all cell types. The UCSC utilities can be found on
 <a href="https://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads"
target="_blank">our download server</a>.</p>

<h2>Data Access</h2>
<p>
The raw <a href="/goldenPath/help/barChart.html">bar chart</a> data can be
explored interactively with the <a href="../cgi-bin/hgTables">Table
Browser</a> or the <a href="../cgi-bin/hgIntegrator">Data Integrator</a>. For
automated analysis, the data may be queried from our <a
href="/goldenPath/help/api.html">REST API</a>. Please refer to our <a
href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing
list archives</a> for questions, or our <a
href="../FAQ/FAQdownloads.html#download36">Data Access FAQ</a> for more
information.</p>

<h2>Credit</h2>
<p>
Thanks to Benjamin J Stewart, John R Ferdinand, and to the many authors who worked on
producing and publishing this data set. The data were integrated into the UCSC
Genome Browser by Jim Kent and Brittney Wick then reviewed by Daniel Schmelter. The 
UCSC work was paid for by the Chan Zuckerberg Initiative.</p>

<h2>References</h2>
<p>
Stewart BJ, Ferdinand JR, Young MD, Mitchell TJ, Loudon KW, Riding AM, Richoz N, Frazer GL,
Staniforth JUL, Vieira Braga FA <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31604275" target="_blank">
Spatiotemporal immune zonation of the human kidney</a>.
<em>Science</em>. 2019 Sep 27;365(6460):1461-1466.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31604275" target="_blank">31604275</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7343525/" target="_blank">PMC7343525</a>
</p>

