<h2>Description</h2>
<p>
This track displays data from <a
href="https://pubmed.ncbi.nlm.nih.gov/32624006/" target="_blank"><em>A
reference single-cell transcriptomic atlas of human skeletal muscle tissue
reveals bifurcated muscle stem cell populations</em></a>. Muscle tissue was
analyzed using single-cell RNA-sequencing (scRNA-seq) and subsequent clustering
distinguished 16 muscle-resident cell types based on their identified marker
genes found in De Micheli <em>et al</em>., 2020. Muscle samples were from
surgically discarded tissue taken from a wide variety of anatomical sites.</p>

<p>
This track collection contains two bar chart tracks of RNA expression in the
human muscle where cells are grouped by cell type 
(<a href="hgTrackUi?g=muscleDeMicheliCellType">Muscle Cells</a>) or biosample
(<a href="hgTrackUi?g=muscleDeMicheliSample">Muscle Sample</a>). 
The default track displayed is 
<a href="hgTrackUi?g=muscleDeMicheliCellType">Muscle Cells</a>.</p>

<h2>Display Conventions</h2>
<p>
The cell types are colored by which class they belong to according to the following table.</p>

<p>
<table cellpadding='2'>
  <thead><tr>
    <th style="border-bottom: 2px solid;">Color</th>
    <th style="border-bottom: 2px solid;">Cell classification</th>
  </tr></thead>
  <tr><td style="background-color: #707070;"></td><td>stem cell</td></tr>
  <tr><td style="background-color: #FF8838;"></td><td>adipose</td></tr>
  <tr><td style="background-color: #B05020;"></td><td>fibroblast</td></tr>
  <tr><td style="background-color: #F01111;"></td><td>immune</td></tr>
  <tr><td style="background-color: #C000C0;"></td><td>muscle</td></tr>
  <tr><td style="background-color: #00C000;"></td><td>endothelial</td></tr>
</table>
</p>

<p>
Cells that fall into multiple classes will be colored by blending the colors associated
with those classes. The colors will be purest in the 
<a href="hgTrackUi?g=muscleDeMicheliCellType">Muscle Cells</a> subtrack, where
the bars represent relatively pure cell types. They can give an overview of the
cell composition within other categories in other subtracks as well. Note that the 
<a href="hgTrackUi?g=muscleDeMicheliSample">Muscle Sample</a> subtrack is colored based on 
colors provided from Figure 1 from De Micheli <em>et al</em>., 2020.</p>

<!--#insert file="$track.insert.html"-->

<h2>Method</h2>
<p>
Muscle samples were taken from 10 healthy donors of ages ranging from 41-81
years old from different sections of the face (F), trunk (T), and leg (L).
Excessive fat and connective tissue were removed from the muscle samples prior
to enzymatic dissociation. Next, libraries were prepared using the 10x Genomics
3' v2 or v3 library kit and sequenced on the Illumina NextSeq 500. This
resulted in libraries with 200-250 million reads which were processed using Cell
Ranger version 3.1. In total, over 22,000 RNA transcriptomic profiles were
generated from all of the samples after quality control filtering. The single
cell transcriptomes from all 10 datasets were integrated using a scRNA-seq
integration method called Scanorama as described in the reference below.

<p>The cell/gene matrix and cell-level metadata was downloaded from the 
<a href="http://cells.ucsc.edu/?ds=muscle-cell-atlas" target="_blank">UCSC Cell Browser</a>.
The UCSC command line utility matrixClusterColumns, matrixToBarChart, and bedToBigBed were used
to transform these into a bar chart format bigBed file that can be visualized. The coloring 
was done by defining colors for the broad level cell classes and then using another UCSC utility,
hcaColorCells, to interpolate the colors across all cell types. The UCSC utilities can be found on
<a href="https://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads"
target="_blank">our download server</a>.</p>

<h2>Data Access</h2>
<p>
The raw <a href="/goldenPath/help/barChart.html">bar chart</a> data can be
explored interactively with the <a href="../cgi-bin/hgTables">Table
Browser</a> or the <a href="../cgi-bin/hgIntegrator">Data Integrator</a>. For
automated analysis, the data may be queried from our <a
href="/goldenPath/help/api.html">REST API</a>. Please refer to our <a
href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing
list archives</a> for questions, or our <a
href="../FAQ/FAQdownloads.html#download36">Data Access FAQ</a> for more
information.</p>
  
<h2>Credit</h2>
<p>
Thanks to Andrea De Micheli of the Cosgrove Laboratory at Cornell University
and to the many authors who worked on producing and publishing this data set. The
data were integrated into the UCSC Genome Browser by Jim Kent and Brittney Wick 
then reviewed Luis Nassar. The UCSC work was paid for by the Chan Zuckerberg Initiative.</p>

<h2>References</h2>
<p>
De Micheli AJ, Spector JA, Elemento O, Cosgrove BD.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32624006" target="_blank">
A reference single-cell transcriptomic atlas of human skeletal muscle tissue reveals bifurcated
muscle stem cell populations</a>.
<em>Skelet Muscle</em>. 2020 Jul 6;10(1):19.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32624006" target="_blank">32624006</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7336639/" target="_blank">PMC7336639</a></p>
