<H2>Description</H2>
<H2>Display Conventions</H2>
<P>
In Full and Pack display modes, expression for each transcript is represented by a colored bargraph,
where the height of each bar represents the median expression level across all samples for a 
cell type, and the bar color indicates the cell type.
<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<img border='1' src="../images/gtex/gtexGeneTcap.png"><br>
The bargraph display has the same width and cell type order for all genes.
Mouse hover over a bar will show the cell type and median expression level.
The Squish display mode draws a rectangle for each gene, colored to indicate the cell type
with highest expression level if it contributes more than 10% to the overall expression
(and colored black if no cell type predominates).
In Dense mode, the darkness of the grayscale rectangle displayed for the gene reflects the total
median expression level across all cell types.</p>
<P>
Click-through on a graph displays a boxplot of expression level quartiles with outliers, 
per cell type.
The track configuration page provides controls to limit the genes and cell types displayed,
and to select raw or log transformed expression level display.</P>

The groups were identified using the T-SNE image below.  
<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<img border='1' src="../images/nowakowskiTSNE.png"><br>

<H2>Methods</H2>
<p>
Average expression values for each cluster were calculated and 
used to generate a bed6+5 file that is the base of the track.  This was done using the UCSC
tool expMatrixToBarchartBed.  The bed track was then converted to a bigBed file using the 
UCSC tool bedToBigBed.  
</p>
<H2>Credits</H2>
<p>
The data was produced by the Pollen and Kriegstein labs.  Alex Pollen, Tom Nowakowski,
and Aparma Bhaduri were key contributors.  

The track was produced by Chris Eisenhart and Max Haeussler at the UCSC genome 
browser group. 

</p>
<H2>References</H2>
