<H2>Description</H2>
<P>
This track displays single cell data from 12 papers covering 14 organs.  Cells are grouped 
together by organ and cell type.  The cell types are based on annotations published alongside
the papers.  These were curated at UCSC to as much as possible use the same cell type 
terminanologies across papers and organs.  In some cases we merged together small populations
of cells annotated as distict but related types into a single type so as to have enough cells 
to call gene expression levels accurate.

The gene expression levels are normalized so that that total level of expression for all genes in a
single cell or cell type adds up to one million. 
</P>

<H2>Display Conventions and Configuration</H2>
<P>
</P>
<H2>Methods</H2>
<P>
Each organ (or tissue) was integreted and curated into the Genome Browser indiviually.  (Make table of contributing tracks here).   All components were normalized to be in parts per million using
the matrixNormalize command available from UCSC.  Metadata was cleaned up using the tabToTabDir 
tool.  The major clean ups were unpacking abbreviations, replacing jargon with standard English, 
choosing shorted terms to shorten long labels, labeling outliers, etc. Before integration we 
invited the original data producers as well as local biologists and informaticions to view the
data.
</P>
<H2>Credits</H2>
Many thanks to the data contributing labs for sharing their high quality research.  
Thanks to the Cell Browser team including Matt Speir and Max Haeussler, for their work
in integratinging these sets into the cell browser.  In most cases their efforts were
ahead of our own, and we could leverage their work making the job much easier.  With the
Genome Browser group Jim Kent did initial wrangling,  Brittney Wick did substantial data
cleanup and coordination with the labs.  Jonathan Spear ...  
<P>
</P>

<H2>References</H2>
<p>
</p>
