<H2>Description</H2>
<P>
The <a href="https://sites.google.com/ucsc.edu/t2tworkinggroup" target=_blank>T2T assembly</a>
is a high-quality human genome assembly published in 2020/2021.
DNA material was obtained from a cell line (CHM13) taken from tissue
that forms when sperm fertilizes a non-viable egg that lacks a nucleus, also
known as complete hydatidiform mole. Having only a single
copy of every chromosome (haploid) makes genome assembly easier. According 
to its preprint, the "T2T-CHM13 assembly includes gapless assemblies for all 22 autosomes plus
Chromosome X, corrects numerous errors, and introduces nearly 200 million bp of
novel sequence containing 2,226 paralogous gene copies, 115 of which are
predicted to be protein coding."
</P>

<P>
This track includes subtracks that show alignments of the human hg38 primary assembly to the
T2T CHM13 V1.1 assembly. There is one subtrack with all possible "chain" alignments and two
filtered, single-coverage subtracks with the best alignment per range, in two slightly different
display formats (chain and net).  The filtered alignments are typically used
for annotation coordinate conversions from hg38 to T2T, using the UCSC
"liftOver" tool.</P>

<P>
In the context of the UCSC alignment tracks, "chains" are longer alignments
built by merging ("chaining") several shorter lastz alignment anchors. The result is 
a gap scoring system that allows longer gaps than traditional affine gap
scoring systems. It can also tolerate gaps in both assemblies. These
&quot;double-sided&quot; gaps can be caused by local inversions and overlapping
deletions in both species. 
</P>

<P>
The chain track displays boxes joined together by either single or
double lines. The boxes represent aligning regions.
Single lines indicate gaps that are largely due to a deletion in 
one assembly or an insertion in the other assembly.
Double lines represent more complex gaps that involve substantial
sequence in both species. This may result from inversions, overlapping
deletions, an abundance of local mutation, or an unsequenced gap in one
species.  In cases where multiple chains align over a particular region of
the other genome, the chains with single-lined gaps are often 
due to processed pseudogenes, while chains with double-lined gaps are more 
often due to paralogs and unprocessed pseudogenes.</P> 
<P>

<P>
In the "pack" and "full" display
modes, the individual feature names indicate the chromosome, strand, and
location (in thousands) of the match for each matching alignment.</P>

<H3>Net Track</H3>
<P>
The net track shows the best chain for 
every part of the other assembly, just like the "liftOver" chain track,
but with slightly different display options.
</P>

<H2>Display Conventions and Configuration</H2>
<H3>Chain Track</H3>
<P>By default, the chains to chromosome-based assemblies are colored
based on which chromosome they map to in the aligning organism. To turn
off the coloring, check the &quot;off&quot; button next to: Color
track based on chromosome.</P>
<P>
To display only the chains of one chromosome in the aligning
organism, enter the name of that chromosome (e.g. chr4) in box next to: 
Filter by chromosome.</P>

<H3>Net Track</H3>
<P>
In full display mode, the top-level (level 1)
chains are the largest, highest-scoring chains that
span this region.  In many cases gaps exist in the
top-level chain.  When possible, these are filled in by
other chains that are displayed at level 2.  The gaps in 
level 2 chains may be filled by level 3 chains and so
forth. </P>
<P>
In the graphical display, the boxes represent ungapped 
alignments; the lines represent gaps.  Click
on a box to view detailed information about the chain
as a whole; click on a line to display information
about the gap.  The detailed information is useful in determining
the cause of the gap or, for lower level chains, the genomic
rearrangement. </P> 
<P> 
Individual items in the display are categorized as one of four types
(other than gap):</P>
<P><UL>
<LI><B>Top</B> - the best, longest match. Displayed on level 1.
<LI><B>Syn</B> - line-ups on the same chromosome as the gap in the level above
it.
<LI><B>Inv</B> - a line-up on the same chromosome as the gap above it, but in 
the opposite orientation.
<LI><B>NonSyn</B> - a match to a chromosome different from the gap in the 
level above.
</UL></P>

<H2>Methods</H2>
<H3>Chain track</H3>

GRCh38 (hg38) primary assembly sequences were aligned with the
T2T V1.1 assembly using LASTZ.  The primary assembly is
chromosomes 1-22, X, Y, and the mitochondria.

<P>
The following lastz matrix was used<BR>for the alignments
</P>

<TABLE BORDER=1 CELLPADDING=4 BORDERCOLOR="#aaaaaa">
<TR ALIGN=right><TD>&nbsp;</TD><TD><B>A</B></TD><TD><B>C</B></TD><TD><B>G</B></TD><TD><B>T</B></TD></TR>
<TR ALIGN=right><TD><B>A<B></TD><TD>90</TD><TD>-330</TD><TD>-236</TD><TD>-356</TD></TR>
<TR ALIGN=right><TD><B>C<B></TD><TD>-330</TD><TD>100</TD><TD>-318</TD><TD>-236</TD></TR>

<TR ALIGN=right><TD><B>G<B></TD><TD>-236</TD><TD>-318</TD><TD>100</TD><TD>-330</TD></TR>
<TR ALIGN=right><TD><B>T<B></TD><TD>-356</TD><TD>-236</TD><TD>-330</TD><TD>90</TD></TR>
</TABLE>

Chains scoring below a minimum
score of 5000 were discarded; the remaining chains
are displayed in this track.  The linear gap matrix used with axtChain:<BR>
&quot;loose&quot;.

<H3>Net track</H3>
<P>
Chains were derived from lastz alignments, using the methods
described on the chain tracks description pages, and sorted with the 
highest-scoring chains in the genome ranked first. The program
chainNet was then used to place the chains one at a time, trimming them as 
necessary to fit into sections not already covered by a higher-scoring chain. 
During this process, a natural hierarchy emerged in which a chain that filled 
a gap in a higher-scoring chain was placed underneath that chain. The program 
netSyntenic was used to fill in information about the relationship between 
higher- and lower-level chains, such as whether a lower-level
chain was syntenic or inverted relative to the higher-level chain. 
</P>

<H2>Data access</H2>
The chain files can be downloaded from <a target=_blank href="http://t2t.gi.ucsc.edu/chm13/hub/t2t-chm13-v1.1/downloads/">
    http://t2t.gi.ucsc.edu/chm13/hub/t2t-chm13-v1.1/downloads/</a>

<H2>Credits</H2>
This track was created by Mark Diekhans from the T2T 1.1 assembly using software 
that is part of the Genome Browser toolkit.
<P>
Lastz (previously known as blastz) was developed at
<A HREF="http://www.bx.psu.edu/miller_lab/" 
TARGET=_blank>Pennsylvania State University</A> by 
Minmei Hou, Scott Schwartz, Zheng Zhang, and Webb Miller with advice from
Ross Hardison.</P>
<P>
Lineage-specific repeats were identified by Arian Smit and his 
<A HREF="http://www.repeatmasker.org" TARGET=_blank>RepeatMasker</A>
program.</P>
<P>
The axtChain program was developed at the University of California at 
Santa Cruz by Jim Kent with advice from Webb Miller and David Haussler.</P>
<P>
The browser display and database storage of the chains and nets were created
by Robert Baertsch and Jim Kent.</P>
<P>
The chainNet and netSyntenic programs were
developed at the University of California
Santa Cruz by Jim Kent.</P>
<P>

<H2>References</H2>
On the T2T assembly:


<p>
Nurk S.
<em>et al</em>.
<a href="https://www.biorxiv.org/content/10.1101/2021.05.26.445798v1" target="_blank">
    The complete sequence of a human genome
</a>
<em>Biorxiv</em>. May 2021
</p>

<p>
<a href="https://github.com/marbl/CHM13" target="_blank">
  T2T CHM13 Github page</a>
</p>

<p>
Logsdon GA, Vollger MR, Hsieh P, Mao Y, Liskovykh MA, Koren S, Nurk S, Mercuri L, Dishuck PC, Rhie A
<em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33828295" target="_blank">
    The structure, function and evolution of a complete human chromosome 8</a>.
<em>Nature</em>. 2021 May;593(7857):101-107.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33828295" target="_blank">33828295</a>; PMC: <a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8099727/" target="_blank">PMC8099727</a>
</p>

<p>
Miga KH, Koren S, Rhie A, Vollger MR, Gershman A, Bzikadze A, Brooks S, Howe E, Porubsky D, Logsdon
GA <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32663838" target="_blank">
    Telomere-to-telomere assembly of a complete human X chromosome</a>.
<em>Nature</em>. 2020 Sep;585(7823):79-84.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32663838" target="_blank">32663838</a>; PMC: <a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7484160/" target="_blank">PMC7484160</a>
</p>

On the Genome Browser alignment pipeline:

<P>
Chiaromonte F, Yap VB, Miller W.
<A HREF="http://psb.stanford.edu/psb-online/proceedings/psb02/chiaromonte.pdf"
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>.
<em>Pac Symp Biocomput</em>. 2002:115-26.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11928468" target="_blank">11928468</a>
</p>

<P>
Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D.
<A HREF="https://www.pnas.org/content/100/20/11484"
TARGET=_blank>Evolution's cauldron:
duplication, deletion, and rearrangement in the mouse and human genomes</A>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Sep 30;100(20):11484-9.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14500911" target="_blank">14500911</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC208784/" target="_blank">PMC208784</a>
</p>

<P>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC,
Haussler D, Miller W.
<A HREF="https://genome.cshlp.org/content/13/1/103.abstract"
TARGET=_blank>Human-mouse alignments with BLASTZ</A>.
<em>Genome Res</em>. 2003 Jan;13(1):103-7.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12529312" target="_blank">12529312</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC430961/" target="_blank">PMC430961</a>
</p>
