<h2>Description</h2>
<p>
This track shows approximately 73 million single nucleotide variants (SNVs) and
5 million short insertions/deletions (indels)
produced by the
  <a href="https://www.internationalgenome.org/" target="_blank">International
Genome Sample Resource (IGSR)</a> from sequence data generated by the
<a href="https://www.internationalgenome.org/about" target="_blank">1000 Genomes Project</a>
in its Phase 3 sequencing of 2,504 genomes from 16 populations worldwide.</p>
<p>
Variants were called on the autosomes (chromosomes 1 through 22) and on the
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2435358/"
target=_blank>Pseudo-Autosomal Regions (PARs)</a> of chromosome X.
Therefore this track has no annotations on alternate haplotype sequences, fix patches,
chromosome Y, or the non-PAR portion (the majority) of chromosome X.
</p>
<p>
The variant genotypes have been phased
(i.e., the two alleles of each diploid genotype have been assigned to two
<a href="https://en.wikipedia.org/wiki/Haplotype" target="_blank">haplotypes</a>,
one inherited from each parent).
This extra information enables a clustering of independent haplotypes
by local similarity for display.
</p>

<h2>Display Conventions</h2>
<p>
<!-- An image with arrows & numbers sure would help here! and on the controls page -->
<!-- Also, this will apply to any VCF track w/phased genotypes, so probably should -->
<!-- be moved into an include file. -->
In &quot;dense&quot; mode, a vertical line is drawn at the position of each
variant.
In &quot;pack&quot; mode, since these variants have been phased, the
display shows a clustering of haplotypes in the viewed range, sorted
by similarity of alleles weighted by proximity to a central variant.
The clustering view can highlight local patterns of linkage.</p>
<p>
In the clustering display, each sample's phased diploid genotype is split
into two independent haplotypes.
Each haplotype is placed in a horizontal row of pixels; when the number of
haplotypes exceeds the number of vertical pixels for the track, multiple
haplotypes fall in the same pixel row and pixels are averaged across haplotypes.</p>
<p>
Each variant is a vertical bar with white (invisible) representing the reference allele
and black representing the non-reference allele(s).
Tick marks are drawn at the top and bottom of each variant's vertical bar
to make the bar more visible when most alleles are reference alleles.
The vertical bar for the central variant used in clustering is outlined in purple.
In order to avoid long compute times, the range of alleles used in clustering
may be limited; alleles used in clustering have purple tick marks at the
top and bottom.</p>
<p>
The clustering tree is displayed to the left of the main image.
It does not represent relatedness of individuals; it simply shows the arrangement
of local haplotypes by similarity.  When a rightmost branch is purple, it means
that all haplotypes in that branch are identical, at least within the range of
variants used in clustering.
</p>

<h2>Methods</h2>
<p>
The genomes of 2,504 individuals were sequenced using both whole-genome sequencing
(mean depth = 7.4x) and targeted exome sequencing (mean depth = 65.7x).
Sequence reads were aligned to the reference genome using alt-aware BWA-MEM
(Zheng-Bradley <em>et al.</em>).
Variant discovery and quality control were performed as described in
(Lowy-Gallego <em>et al.</em>).


See also:
<ul>
<li><a href="https://www.internationalgenome.org/analysis"
    target="_blank">1000 Genomes Project - Analysis overview</a></li>
<li><a href="https://www.internationalgenome.org/faq"
       target="_blank">IGSR/1000 Genomes Frequently Asked Questions (FAQ)</a></li>
</ul>
</p>

<h2>Data Access</h2>
<p>
VCF files were downloaded from
<a href="http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000_genomes_project/release/20190312_biallelic_SNV_and_INDEL/"
target="_blank">EBI</a>
and are also available for download from
<a href="http://hgdownload.soe.ucsc.edu/gbdb/hg38/1000Genomes/">UCSC</a>.
</p>

<h2>Credits</h2>
<p>
Thanks to the
  <a href="https://www.internationalgenome.org/" target="_blank">International Genome Sample
Resource (IGSR)</a>
for making these variant calls freely available.
</p>

<h2>References</h2>

<p>
Zheng-Bradley X, Streeter I, Fairley S, Richardson D, Clarke L, Flicek P, 1000 Genomes Project
Consortium.
<a href="https://academic.oup.com/gigascience/article-lookup/doi/10.1093/gigascience/gix038"
target="_blank">
Alignment of 1000 Genomes Project reads to reference assembly GRCh38</a>.
<em>Gigascience</em>. 2017 Jul 1;6(7):1-8.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28531267" target="_blank">28531267</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5522380/" target="_blank">PMC5522380</a>
</p>

<p>
Fairley S, Lowy-Gallego E, Perry E, Flicek P.
<a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkz836" target="_blank">
The International Genome Sample Resource (IGSR) collection of open human genomic variation
resources</a>.
<em>Nucleic Acids Res</em>. 2019 Oct 4.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31584097" target="_blank">31584097</a>
</p>

<p>
Lowy-Gallego E, Fairley S, Zheng-Bradley X, Ruffier M, Clarke L, Flicek P,
1000 Genomes Project Consortium.
<a href="https://wellcomeopenresearch.org/articles/4-50" target=_blank>
Variant calling on the GRCh38 assembly with the data from phase three of the 1000 Genomes Project [version 1; peer review: 2 not approved]</a>.
<em>Wellcome Open Research</em>. 2019 Mar. 11.
</p>

<p>
1000 Genomes Project Consortium, Auton A, Brooks LD, Durbin RM, Garrison EP, Kang HM, Korbel JO,
Marchini JL, McCarthy S, McVean GA <em>et al</em>.
<a href="https://www.nature.com/articles/nature15393" target="_blank">
A global reference for human genetic variation</a>.
<em>Nature</em>. 2015 Oct 1;526(7571):68-74.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26432245" target="_blank">26432245</a>
</p>
