<h2>Description</h2>
<p>
This track shows approximately 4.5 million single nucleotide variants (SNVs) and
0.6 million short insertions/deletions (indels) from 7 different parent/child trios as
produced by the
<a href="https://www.internationalgenome.org/" target="_blank">International
Genome Sample Resource (IGSR)</a>, from sequence data generated by the
<a href="https://www.internationalgenome.org/about" target="_blank">1000 Genomes Project</a>
in its Phase 3 sequencing of 2,504 genomes from 16 populations worldwide.</p>
<p>
Variants were called on the autosomes (chromosomes 1 through 22) and on the
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2435358/"
target=_blank>Pseudo-Autosomal Regions (PARs)</a> of chromosome X.
Therefore this track has no annotations on alternate haplotype sequences, fix patches,
chromosome Y, or the non-PAR portion (the majority) of chromosome X.
</p>
<p>
The variant genotypes have been phased (i.e., the two alleles of each diploid genotype
have been assigned to two
<a href="https://en.wikipedia.org/wiki/Haplotype" target="_blank">haplotypes</a>,
one inherited from each parent). This information allows us to illustrate which
haplotypes in the child have been inherited from which parent.
</p>

<p>Trios from six different populations are available, including:
<ul>
  <li>YRI - Yoruban from Idaban, Nigeria</li>
  <li>KHV - Kinh in Ho Chi Minh City, Vietnam</li>
  <li>PUR - Puerto Ricans from Puerto Rico</li>
  <li>CEU - CEPH Utah</li>
  <li>CHS - Southern Han Chinese</li>
  <li>MXL - Mexican Ancestry from Los Angeles</li>
</ul>
</p>

<h2>Display Conventions and Configuration</h2>
<p>
This track illustrates the vcfPhasedTrio track type, where two lines, one for each chromosome
in the diploid genome, is drawn per sample in the underlying VCF. Variants in the window
are then drawn on the haplotype line corresponding to which haplotype they belong to, such that
variants on the same line were likely inherited together. The sorting routine is the same as
what is used to draw the haplotype sorted display in the non-trio 1000 Genomes track, and is
described <a href="../goldenPath/help/hgVcfTrackHelp.html">here</a>.
</p>

<p>
The child haplotypes are drawn in the center of each group, flanked above and below by
parent haplotypes, and variants are sorted to show the transmitted alleles:
<pre>
parent 1 untransmitted haploytpe 
parent 1 transmitted haplotype
child haplotype inherited from parent 1
child haplotype inherited from parent 2
parent 2 transmitted haplotype
parent 2 untransmitted haploytpe 
</pre>

<p>
Track configuration options include:
<ul>
<li>Showing the child haplotypes below the parent(s)</li>
<li>Toggling the haplotype labels with mother/father/child or VCF sample IDs</li>
<li>Hiding the parent samples</li>
</ul>
</p>

<p>
Allele coloring options include:
<ul>
<li>No shading - the default option</li>
<li>Shading by functional effect of the variant relative to NCBI RefSeq Curated Transcripts:
    <ul><li>reference alleles invisible</li>
        <li>alternate alleles in red for non-synonymous</li>
        <li>alternate alleles in green for synonymous</li>
        <li>alternate alleles in blue for UTR/noncoding</li>
        <li>alternate alleles in black otherwise</li>
    </ul>
</li>
<li>Child de novo alleles in red - all alternate alleles black except for cases where the child has
an allele not present in either parent</li>
<li>Child alleles that are &quot;inconsistent&quot; with phasing in red - all alternate alleles black except for cases where the &quot;inherited&quot; child allele does not match the &quot;transmitted&quot; parent allele. Note that as the genomic location changes, and thus the alleles present to use for sorting change, whether an allele is marked as inconsistent can change as well. Because all the variants present in the window are considered a haplotype, what haplotypes are considered &quot;inherited&quot; and &quot;transmitted&quot; varies as the viewing location changes</li>
</ul>
</p>

<p>
From the subtrack configure menu, there is the option to manually rearrange 
the family order for each trio by dragging haplotypes. 
</p>

<p>
Clicking on a variant takes one to a details page with the standard VCF details, including
INFO column annotations, the REF and ALT alleles, and the genotypes from all three samples.
</p>

<h2>Methods</h2>
<p>
The genomes of 2,504 individuals were sequenced using both whole-genome sequencing
(mean depth = 7.4x) and targeted exome sequencing (mean depth = 65.7x).
Sequence reads were aligned to the reference genome using alt-aware BWA-MEM
(Zheng-Bradley <em>et al.</em>).
Variant discovery and quality control were performed as described in
Lowy-Gallego <em>et al.</em></p>
<p>
See also:
<ul>
<li><a href="https://www.internationalgenome.org/analysis"
    target="_blank">1000 Genomes Project - Analysis overview</a></li>
<li><a href="https://www.internationalgenome.org/faq"
       target="_blank">IGSR/1000 Genomes Frequently Asked Questions (FAQ)</a></li>
</ul>
</p>

<h3>UCSC Methods</h3>
<p>
Trio samples were extracted out of both the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000_genomes_project/release/20190312_biallelic_SNV_and_INDEL/">main 1000 Genomes set</a>, and the
<a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000_genomes_project/release/20190312_biallelic_SNV_and_INDEL/supporting/related_samples/">related samples</a> using the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/integrated_call_samples_v2.20130502.ALL.ped">pedigree information</a> from 1000
Genomes. Variants that were homozygous reference across all three samples were removed.
</p>

<h2>Data Access</h2>
<p>
Trio VCFs are available for download from
<a href="http://hgdownload.soe.ucsc.edu/gbdb/hg38/1000Genomes/trio/">our download server</a>.
</p>

<h2>Credits</h2>
<p>
Thanks to the
  <a href="https://www.internationalgenome.org/" target="_blank">International Genome Sample
Resource (IGSR)</a>
for making these variant calls freely available.
</p>

<h2>References</h2>

<p>
Zheng-Bradley X, Streeter I, Fairley S, Richardson D, Clarke L, Flicek P, 1000 Genomes Project
Consortium.
<a href="https://academic.oup.com/gigascience/article-lookup/doi/10.1093/gigascience/gix038"
target="_blank">
Alignment of 1000 Genomes Project reads to reference assembly GRCh38</a>.
<em>Gigascience</em>. 2017 Jul 1;6(7):1-8.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28531267" target="_blank">28531267</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5522380/" target="_blank">PMC5522380</a>
</p>

<p>
Fairley S, Lowy-Gallego E, Perry E, Flicek P.
<a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkz836" target="_blank">
The International Genome Sample Resource (IGSR) collection of open human genomic variation
resources</a>.
<em>Nucleic Acids Res</em>. 2019 Oct 4.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31584097" target="_blank">31584097</a>
</p>

<p>
Lowy-Gallego E, Fairley S, Zheng-Bradley X, Ruffier M, Clarke L, Flicek P,
1000 Genomes Project Consortium.
<a href="https://wellcomeopenresearch.org/articles/4-50" target=_blank>
Variant calling on the GRCh38 assembly with the data from phase three of the 1000 Genomes Project [version 1; peer review: 2 not approved]</a>.
<em>Wellcome Open Research</em>. 2019 Mar. 11.
</p>

<p>
1000 Genomes Project Consortium, Auton A, Brooks LD, Durbin RM, Garrison EP, Kang HM, Korbel JO,
Marchini JL, McCarthy S, McVean GA <em>et al</em>.
<a href="https://www.nature.com/articles/nature15393" target="_blank">
A global reference for human genetic variation</a>.
<em>Nature</em>. 2015 Oct 1;526(7571):68-74.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26432245" target="_blank">26432245</a>
</p>
