<H2>Description</H2>
<P>
This track shows the Zebrafish Zv7 (July 2007) assembly provided by 
<A HREF="https://www.sanger.ac.uk/resources/zebrafish/genomeproject.html"
TARGET=_blank>The Wellcome Trust Sanger Institute</A>. 
It contains scaffolds (supercontigs) totaling 1,440,582,308 bp. 
The assembly has been tied to the fingerprint contig (FPC) map 
(data freeze April 11, 2007), which provides a tiling path of sequenced clones.
1.02 Gb of sequence from 7,823 sequenced clones (7,139 finished and 684 
unfinished) were used as a scaffold for the assembly. Gaps were filled with 
contigs from a whole genome shotgun (WGS) assembly of 5.5x coverage comprised
of reads from a library created from a single Tuebingen doubled haploid 
zebrafish. Approximately 1.28 Gbp (89%) of the resulting integrated assembly 
were placed on chromosomes 1-25, including estimated gap sizes and 100 bp gaps 
inserted betweeen scaffolds. The complete sequence of the mitochondrion genome,
which is shown as chrM in the Genome Browser, was obtained from GenBank.  </P>
<P>
The entire assembly of 5,036 scaffolds includes the 26 chromosomes and 5,010 
unplaced scaffolds that fall into two groups:
<UL>
<LI><B>Zv7_NAXX</B> - WGS contigs that could not be related to any FPC contig 
and could not be be placed on a chromosome (4,844 unmapped scaffolds)
<LI><B>Zv7_scaffoldXXXX</B> - sequences based on FPC contigs or linked to 
chromosomes via a marker (166 unmapped scaffolds)
</UL>
The unplaced scaffolds contain 100 bp gaps that are shown in the 
Gap annotation. 
<P>
All components within this track are of fragment type &quot;W&quot;
(WGS contig) except for chrM, which is type &quot;F&quot; (Finished).
<P>
In dense mode, this track depicts the path through the draft and
finished clones (aka the golden path) used to create the assembled sequence.
Clone boundaries are distinguished by the use of alternating gold and brown
coloration. Where gaps exist in the path, spaces are shown between the gold 
and brown blocks.  If the relative order and orientation of the scaffolds
between the two blocks is known, a line is drawn to bridge the
blocks.</P>
<P>For detailed information on the methods used to produce this assembly, see 
the Wellcome Trust Sanger Institute 
<A HREF="ftp://ftp.sanger.ac.uk/pub/zfish/assembly/Zv7/README2"
TARGET=_blank><em>Danio rerio</em> Sequencing Project</A> website.

<H2>Credits</H2>
<P>
The Zv7 Zebrafish assembly was produced by The Wellcome Trust Sanger Institute,
in collaboration with the Max Planck Institute for Developmental Biology, 
the Netherlands Institute for Developmental Biology (Hubrecht Laboratory),
and Yi Zhou, Anthony DiBiase and Leonard Zon from the Boston Children's 
Hospital.</P> 
