<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
<HEAD>
	<META HTTP-EQUIV="Content-Type" CONTENT="text/html;CHARSET=iso-8859-1">
	<META name="description" content="ENCODE Project at UCSC - About ENCODE Data">
	<META name="keywords" content="ENCODE, project, ENCODE project, genome, genome browser, human genome, Blat, UCSC, bioinformatics, gene prediction, SNP, EST, mRNA, NHGRI, NISC">
	<TITLE>About ENCODE Data at UCSC</TITLE>
	<LINK REL="STYLESHEET" HREF="../style/HGStyle.css">
</HEAD>

<BODY BGCOLOR="FFF9D2" LINK="0000CC" VLINK="#330066" ALINK="#6600FF">
<A NAME="TOP"></A>

<!-- Header Images and links -->
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0 WIDTH="100%">
  <TR>
    <TH COLSPAN=2 ALIGN="left">&nbsp;&nbsp;
      <A HREF="http://www.genome.gov/10005107" TARGET=_BLANK><IMG SRC="../images/ENCODE_scaleup_logo.png" height=50 ALT="ENCODE Project at NHGRI"></A>
      &nbsp;&nbsp;
          <A HREF="/ENCODE/"><IMG SRC="../images/encodeAtUcsc.png" ALT="ENCODE Data Coordination Center at UCSC"></A>
    </TH>
  </TR>

  <TR><TD COLSPAN=2 HEIGHT=40><!--#include virtual="../inc/encode.topbar.html"--></TD></TR>

    <TR HEIGHT="100%" VALIGN="top"><TD VALIGN="top">
        <!--outer table is for border purposes-->
        <TABLE WIDTH="100%" BGCOLOR="#888888" BORDER="0" CELLSPACING="0" CELLPADDING="1">
            <TR><TD>	
            <TABLE BGCOLOR="fffee8" WIDTH="100%"  BORDER="0" CELLSPACING="0" CELLPADDING="0">
                <TR><TD>	

	        <TABLE BGCOLOR="fffee8" WIDTH="100%" CELLPADDING=0>
                    <TR><TH HEIGHT=3></TH></TR>
                    <TR><TD WIDTH=10></TD><TD>
&nbsp;<br>
ENCODE investigators employ a variety of assays and methods to identify functional elements.
The discovery and annotation of gene elements is accomplished primarily by sequencing RNA from a
diverse range of sources, comparative genomics, integrative
bioinformatic methods, and human curation. Regulatory elements are
typically investigated through DNA hypersensitivity assays, assays of
DNA methylation, and chromatin immunoprecipitation (ChIP) of proteins
that interact with DNA, including modified histones and transcription
factors, followed by sequencing (ChIP-Seq).</p>
<p>
<table><tr>
<td><img border=1 src="../images/EncodeDatatypes2013.png" alt="" height="430"></td>
</tr><tr>
<td><font size=-1>Credits: Darryl Leja (NHGRI), Ian Dunham (EBI), Michael Pazin (NHGRI)</font></td>
</tr></table></p>
<p>
To visualise human ENCODE data at UCSC, open the
<a href="../cgi-bin/hgGateway?org=human" target="_blank">Genome Browser</a>, 
select the February 2009 assembly (GRCh37/hg19) or the March 2006 assembly (NCBI36/hg18) 
of the human genome, and go to your region of interest.
The bulk of
the ENCODE data can be found in the <em>Expression</em> and <em>Regulation</em>
track groups, with a few in the <em>Mapping</em>, <em>Genes</em>, and
                              <em>Variation</em> groups. Although most
participating research groups have provided several tracks, generally
only selected data from each research group are displayed by default.
Click the hyperlinked name of a particular track to display a page
containing configuration options and details about the methods used to
generate the data. See the Genome Browser <a
 href="../goldenPath/help/hgTracksHelp.html" target="_blank">User's Guide</a> for
further information about displaying tracks and navigating in the
Genome Browser.</p>
<p>
Data from the earlier ENCODE project
pilot phase, which covered approximately 1% of the genome,&nbsp; are
available on the March 2006 (NCBI36/hg18),
May 2004 (NCBI35/hg17), and July 2003 (NCBI34/hg16) human genome assemblies. 
The <a href="pilot.html">ENCODE Pilot Project</a>
web pages provide convenient browser access to these regions.

		    </P>

		    </TD><TD WIDTH=15>
                </TD></TR></TABLE>
	    <BR></TD></TR></TABLE>
        </TD></TR></TABLE>
	<BR>
    </TD></TR></TABLE>
</BODY></HTML>

