<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
    <head>
        <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
        <title>Software and Resources for Analyzing ENCODE data</title>
        <link rel="stylesheet" href="/style/HGStyle.css" type="text/css">
        <link rel="stylesheet" href="/style/encodeProject.css" type="text/css">
    </head>

    <body>
        <!--#include virtual="/inc/encodeProjectSoftware.topbar.html"-->
        <!--#include virtual="/inc/encodeProject.newportal.html"-->

        <div class="encodeHeader">
            <a href="http://www.genome.gov/10005107"><img src="/images/ENCODE_scaleup_logo.png" alt="ENCODE Project at NHGRI" title="ENCODE Project at NHGRI"></a>
            <span class="txt">Software and Resources for Analyzing ENCODE data</span>
        </div>
       <div class="wrapper">
                <div class="bar"><h4 class="title">Tools for analyzing ENCODE data</h4></div>
                <div class="content">
                      <ul>
                        <p><li>
                        <a class="thick" href="http://www.broadinstitute.org/mammals/haploreg/haploreg.php" target="_blank">HaploReg</a><br>
                        Explores annotations of the noncoding genome at variants on haplotype blocks,
                        such as candidate regulatory SNPs at disease-associated loci. Under Set
                        Options tab, set Browse ENCODE button to &quot;on&quot; and select an LD threshold
                        and reference population. Under Build Query Tab, enter a SNP (rsXXXXX),
                        a set of SNPs, a genomic region, or select a GWAS from the drop down menu.
                        HaploReg returns SNPs in LD with query SNPs, their frequency in 4 populations 
                        from 1000 Genomes Phase1, and also tells you what evidence ENCODE has found for 
                        regulatory protein binding (mouse over to see the protein names), chromatin
                        structure (mouse over to see the cell types with DNase hypersensitivity), 
                        the chromatin state of the region (the chromatin  state can predict an enhancer or promoter),
                        and putative transcription factor binding motifs that are altered by the variant.  Clicking on 
                        the SNP name hyperlink reveals further details, including cell type metadata and the mechanism of 
                        disruption/creation of TF binding regulatory motifs (showing the PWM matched and its alignment to
                        the local sequence context). SNPs are also intersected with cross-species conserved elements,
                        chromatin states from the Roadmap Epigenomics Consortium, and lead eQTLs from the GTEx Project browser.
                        </p>

                        <p>
                        Ward LD, Kellis M.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/22064851" target="_blank">
                        HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants</a>.
                        <em>Nucleic Acids Res</em>. 2012 Jan;40(Database issue):D930-4.
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://regulome.stanford.edu/" target="_blank">RegulomeDB</a><br>
                        Identifies DNA features and regulatory elements in non-coding regions of the human genome. One can
                        enter dbSNP IDs, BED files, VCF files, or GFF3 files.  A score is returned assessing the evidence
                        for regulatory potential.  Clicking on the score reveals the data supporting the inference, by data
                        type and cell type.  One can also click on hyperlinks to see the SNP or the region in the UCSC
                        browser, ENSEMBL browser, and dbSNP.
                        </p>

                        <p>
                        Boyle AP, Hong EL, Hariharan M, Cheng Y, Schaub MA, Kasowski M, Karczewski KJ, Park J, Hitz BC, Weng
                        S <em>et al</em>.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/22955989" target="_blank">
                        Annotation of functional variation in personal genomes using RegulomeDB</a>.
                        <em>Genome Res</em>. 2012 Sep;22(9):1790-7.
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://piq.csail.mit.edu/" target="_blank">PIQ: Protein Interaction Quantification</a><br>
                        PIQ is a computational method that models the magnitude and shape of
                        genome-wide DNase profiles to facilitate the identification of
                        transcription factor (TF) binding sites.   The input of PIQ is one
                        or more DNase-seq experiments, the genome sequence of the organism
                        assayed and a list of motifs represented as position weight matrices (PWMs)
                        that describe candidate TF binding sites.  PIQ uses machine learning methods
                        to normalize input DNase-seq data and then predicts TF binding by detecting
                        both the shape and magnitude of DNase profiles specific to each TF. The
                        output of PIQ is the probability of occupancy for each candidate binding
                        site in the genome, along with aggregate TF-specific scores (e.g. metrics
                        for TF-specific chromatin opening). 
                        </p>

                        <p>
                        Sherwood RI, Hashimoto T, O'Donnell CW, Lewis S, Barkal AA, van Hoff JP, Karun V, Jaakkola T,
                        Gifford DK.
                        <a href="http://dx.doi.org/10.1038/nbt.2798" target="_blank">
                        Discovery of directional and nondirectional pioneer transcription factors by modeling DNase profile
                        magnitude and shape</a>.
                        <em>Nat Biotechnol</em>. 2014 Feb;32(2):171-8.
                        PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24441470" target="_blank">24441470</a>; PMC: <a
                        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3951735" target="_blank">PMC3951735</a>
                        </p>
                        </li></p>


                        <p><li>
                        <a class="thick" href="http://www.factorbook.org/" target="_blank">Factorbook</a><br>
                        A wiki-style resource that organizes all the information associated with each transcription factor (TF),
                        including the ChIP-seq peaks, discovered motifs, TF-TF interactions, and the chromatin features
                        (histone modification patterns, DNase I cleavage, and nucleosome positioning) around the ChIP-seq
                        peaks. Will be updated as the project proceeds. The Factorbook display of this information is
                        transcription factor anchored and dynamic.
                        </p>

                        <p>
                        Wang J, Zhuang J, Iyer S, Lin X, Whitfield TW, Greven MC, Pierce BG, Dong X, Kundaje A, Cheng Y
                        <em>et al</em>.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/22955990" target="_blank">
                        Sequence features and chromatin structure around the genomic regions bound by 119 human
                        transcription factors</a>.
                        <em>Genome Res</em>. 2012 Sep;22(9):1798-812.
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://compbio.mit.edu/encode-motifs" target="_blank">ENCODE-motifs</a><br>
                        A database that uncovers the molecular basis of TF binding in the human genome based on regulatory
                        motif analysis of all Transcription Factors (TFs) grouped by family. This allows browsing of all known
                        motifs for each factor, curated from TRANSFAC, Jaspar, and Protein Binding Microarray (PBM) experiments,
                        and their enrichment and instances within corresponding TF binding experiments. It also provides a list
                        of novel regulatory motifs discovered by systematic application of several motif discovery tools
                        (including MEME, MDscan, Weeder, AlignACE) and evaluated based on their enrichment relative to control
                        motifs within TF-bound regions. ENCODE-motifs also provides a genome-wide map of regulatory motif
                        instances in the human genome for both known and novel motifs.
                        </p>

                        <p>
                        Kheradpour P, Kellis M.
                        <a href="http://nar.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=24335146" target="_blank">
                        Systematic discovery and characterization of regulatory motifs in ENCODE TF binding experiments</a>.
                        <em>Nucleic Acids Res</em>. 2014 Mar 1;42(5):2976-87.
                        PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24335146" target="_blank">24335146</a>; PMC: <a
                        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3950668" target="_blank">PMC3950668</a>
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://sparkinsight.org" target="_blank">Spark</a><br>
                        Spark is an interactive pattern discovery and visualization approach designed with epigenomic data
                        in mind. Spark can reveal both known and novel epigentic signatures.
                        </li> </p>
                        
                        <p>
                        Nielsen CB, Younesy H, O'Geen H, Xu X, Jackson AR, Milosavljevic A, Wang T, Costello JF, Hirst M,
                        Farnham PJ, Jones SJ.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/22960372" target="_blank">
                          Spark: a navigational paradigm for genomic data exploration</a>.
                        <em>Genome Res</em>. 2012 Nov;22(11):2262-9. PMID: 22960372; PMCID: PMC3483555 
                        </p>

                        <p><li>
                        <a class="thick" href="http://dnase.genome.duke.edu/" target="_blank">Regulatory Elements Database</a><br>
                        Using an intuitive interface, you can 1) identify DNaseI-hypersensitive sites (DHS) within a
                        genomic region of interest, 2) predict the target gene for DHS of interest, 3) predict the
                        DHS that regulate a gene of interest, 4) identify clusters of similarly regulated DHS, that may
                        have related function, 5) identify enriched motifs for transcription factors that may bind
                        in these similarly regulated DHS, and 6) identify DHS that contain a DNA sequence motif for a
                        transcription factor of interest.  The Regulatory Elements Database provides access to roughly
                        2.8 million DNaseI-hypersensitive sites and their signal in 112 human samples, as well as
                        Affymetrix microarray expression data for the same cell-types.
                        </li> </p>

                        <p>
                        Sheffield NC, Thurman RE, Song L, Safi A, Stamatoyannopoulos JA, Lenhard B, Crawford GE, Furey TS.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/23482648" target="_blank">
                        Patterns of regulatory activity across diverse human cell types predict tissue identity,
                        transcription factor binding, and long-range interactions</a>.
                        <em>Genome Res</em>. 2013 May;23(5):777-88. PMID: 23482648; PMCID: PMC3638134
                        </p>
          
                        <p><li>
                        <a class="thick">Tutorial on functional annotation of non-coding sequences</a><br>
                        Briefly describes how to display the basic ENCODE tracks, using the UCSC genome browser, at
                        SNPs identified from GWAS studies.
                        </p>

                        <p>
                        Mortlock DP, Pregizer S.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/22241655" target="_blank">
                        Identifying functional annotation for noncoding genomic sequences</a>.
                        <em>Curr Protoc Hum Genet</em>. 2012 Jan;Chapter 1:Unit1.10.
                        </li></p>
                        </ul>
            <p>
            See also the 
            <a target=_blank href="usageResources.html">ENCODE Education & Outreach</a> page.</p>

               </div><!--end content-->
       </div><!--end wrapper--> 
       <p class="date">Updated 24 Mar 2014</p>
    </body>
</html>
