<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
<HEAD>
	<META HTTP-EQUIV="Content-Type" CONTENT="text/html;CHARSET=iso-8859-1">
	<META name="description" content="ENCODE Project at UCSC - Pilot Phase Contributors">
	<META name="keywords" content="ENCODE, project, ENCODE project, genome, genome browser, human genome, Blat, UCSC, bioinformatics, gene prediction, SNP, EST, mRNA, NHGRI, NISC">
	<TITLE>ENCODE Project at UCSC: Pilot Phase Contributors</TITLE>
	<LINK REL="STYLESHEET" HREF="../style/HGStyle.css">
</HEAD>

<BODY BGCOLOR="FFF9D2" LINK="0000CC" VLINK="#330066" ALINK="#6600FF">
<A NAME="TOP"></A>

<!-- Header Images and links -->
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0 WIDTH="100%">
  <TR>
    <TH COLSPAN=2 ALIGN="left">&nbsp;&nbsp;
      <A HREF="http://www.genome.gov/10005107" TARGET=_BLANK><IMG SRC="/images/encodelogo.gif" height=50 ALT="ENCODE Project at NHGRI"></A>
      &nbsp;&nbsp;
        <A HREF="/ENCODE/"><IMG SRC="../images/encodePilot.jpg" ALT="ENCODE Pilot Project at UCSC"></A>
    </TH>
  </TR>

  <TR><TD COLSPAN=2 HEIGHT=40><!--#include virtual="../inc/encode.topbar_pilot.html"--></TD></TR>

    <TR HEIGHT="100%" VALIGN="top"><TD VALIGN="top">
        <!--outer table is for border purposes-->
        <TABLE WIDTH="100%" BGCOLOR="#888888" BORDER="0" CELLSPACING="0" CELLPADDING="1">
            <TR><TD>	
            <TABLE BGCOLOR="fffee8" WIDTH="100%"  BORDER="0" CELLSPACING="0" CELLPADDING="0">
                <TR><TD>	

	        <TABLE BGCOLOR="D9E4F8" BACKGROUND="../images/hr.gif" WIDTH="100%"><TR><TD>
	            <A NAME="Existing"></A><FONT SIZE="4"><B>&nbsp; 
		    ENCODE pilot phase data contributors</B></FONT></TD>
	          </TR></TABLE>
	
	        <TABLE BGCOLOR="fffee8" WIDTH="100%" CELLPADDING=0>
                    <TR><TH HEIGHT=3></TH></TR>
                    <TR><TD WIDTH=10></TD><TD>
		    <P>
		    These researchers received NHGRI funds for the
		    large-scale application of existing technologies
		    to determine functional elements across all
		    ENCODE target regions.
		    <UL>
                    <LI>
		    Ian Dunham, David Vetrie, and Nigel Carter,
                    <A HREF="http://www.sanger.ac.uk/PostGenomics/encode/"
		    TARGET=_blank>The Wellcome Trust Sanger Institute</A>, 
		    Hinxton, U.K.
		    <BR>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    Detecting human functional sequences with microarrays
		    </LI>
                    <LI>
                    <A HREF="http://mexico.bioch.virginia.edu/"
		    TARGET=_blank>Anindya Dutta</A>, University of Virginia, 
		    Charlottesville
		    <BR>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    Mapping replication elements on human chromosomes
		    </LI>
                    <LI>
		    Thomas Gingeras,  
		    <A HREF="http://www.affymetrix.com/" 
		    TARGET=_blank>Affymetrix, Inc.</A>, Santa Clara, California
		    <BR>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    <A HREF="http://www.affymetrix.com/transcriptome/index.affx"
		    TARGET=_blank>Mapping sites of transcription and 
		    regulation</A>
		    </LI>
                    <LI>
		    <A HREF="http://www1.imim.es/~rguigo/" 
		    TARGET=_blank>Roderic Guigo</A>,  
                    <A HREF="http://genome.imim.es/" 
		    TARGET=_blank>Municipal Institute of Medical Research</A>, 
		    Barcelona, Spain
		    <BR>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    <A HREF="http://genome.imim.es/gencode/" 
		    TARGET=_blank>Encyclopedia of genes and gene variants</A>
		    </LI>
                    <LI>
		    <A HREF="http://www-shgc.stanford.edu/myerslab/" 
		    TARGET=_blank>Richard Myers</A>, Stanford University, Palo 
		    Alto, California 
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    <A HREF="http://www-shgc.stanford.edu/genetics/encode.html" 
		    TARGET=_blank>The Stanford ENCODE Project</A>
                    <LI>
                    <A HREF="http://licr-renlab.ucsd.edu"
		    TARGET=_blank>Bing Ren</A>,  
                    <A HREF="http://ludwig.ucsd.edu/" 
		    TARGET=_blank>Ludwig Institute for Cancer Research</A>, 
		    University of California, San Diego
		    <BR>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    Mapping transcriptional regulatory elements in human DNA
		    </LI>
                    <LI>
                    <A HREF="http://www.yale.edu/snyder/"
		    TARGET=_blank>Michael Snyder</A>,  Yale University, New 
		    Haven, Connecticut
		    <BR>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    Transcription and regulatory elements in ENCODE regions
		    </LI>
		    <LI>
                    <A HREF="http://www.gs.washington.edu/faculty/stam.htm"
		    TARGET=_blank>George Stamatoyannopoulos</A>, 
                    University of Washington, Seattle 
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    Identification of functional DNA elements by HSqPCR
		    </LI>
		    </UL></P>
		    </TD><TD WIDTH=15>
                </TD></TR></TABLE>
	    <BR></TD></TR></TABLE>
        </TD></TR></TABLE>
	<BR>

        <!--outer table is for border purposes-->
        <TABLE WIDTH="100%" BGCOLOR="#888888" BORDER="0" CELLSPACING="0" CELLPADDING="1">
            <TR><TD>	
            <TABLE BGCOLOR="fffee8" WIDTH="100%"  BORDER="0" CELLSPACING="0" CELLPADDING="0">
                <TR><TD>	

	        <TABLE BGCOLOR="D9E4F8" BACKGROUND="../images/hr.gif" WIDTH="100%"><TR><TD>
	            <A NAME="Existing"></A><FONT SIZE="4"><B>&nbsp; 
		    Groups participating in the ENCODE Technology Development Phase
		    </B></FONT>
	        </TD></TR></TABLE>
	
	        <TABLE BGCOLOR="fffee8" WIDTH="100%" CELLPADDING=0>
                    <TR><TH HEIGHT=3></TH></TR>
                    <TR><TD WIDTH=10></TD><TD>
		    <P>
		    These researchers received NHGRI funds to improve or 
		    develop new technologies for finding functional elements in 
		    genomic DNA, and will employ them across all ENCODE target 
		    regions. Groups that are also participating in the ENCODE 
		    Pilot Phase are marked with a star (*).
		    <UL>
                    <LI>
                    <A HREF="http://www.umassmed.edu/pgfe/faculty/Dekker.cfm?start=0&"
		    TARGET=_blank>Job Dekker</A>, University of Massachusetts 
		    Medical School, Worcester
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    Structural annotation of the human genome
                    </LI>
                    <LI>
                    <A HREF="http://cmm.ucsd.edu/fu/"
		    TARGET=_blank>Xiang-Dong Fu</A>, University of California, 
		    San Diego
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    A novel chIP-chip technology for ENCODE
		    </LI>
                    <LI>
		    Roland Green *,  
                    <A HREF="http://www.nimblegen.com/technology/index.html"
		    TARGET=_blank>Nimblegen Systems, Inc.</A>, Madison, 
		    Wisconsin
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    Discovery of binding sites for transcription factors
		    </LI>
                    <LI>
		    <A HREF="http://genetics.mgh.harvard.edu/PublicWeb/faculty/kingston/index.html" 
		    TARGET=_blank>Robert Kingston</A>,
		    <A HREF="http://genetics.mgh.harvard.edu/PublicWeb/" 
		    TARGET=_blank>Massachusetts General Hospital</A>, Boston
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    Long-range, high-resolution mapping of chromatin
		    </LI>
                    <LI>
		    Mark McCormick,  
                    <A HREF="http://www.nimblegen.com/technology/index.html"
		    TARGET=_blank>Nimblegen Systems, Inc.</A>, Madison, 
		    Wisconsin
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    DNA array-based exon detection and linkage mapping
		    </LI>
		    <LI> 
                    <A HREF="http://zlab.bu.edu/zhiping/" 
		    TARGET=_blank>Zhiping Weng</A> *,  Boston University
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    Alternative promoter usage in tissue-specific gene expression
		    </LI>
		    </UL></P>
	            </TD><TD WIDTH=15>
                </TD></TR></TABLE>

	        <TABLE BGCOLOR="fffee8" WIDTH="100%" CELLPADDING=0>
                    <TR><TH HEIGHT=3></TH></TR>
                    <TR><TD WIDTH=10></TD><TD>
		    <P>
                    Second round of technology development grantees (September 
		    2004):
		    <UL>
                    <LI>
                    <A HREF="http://www-biology.ucsd.edu/faculty/ecker.html" 
		    TARGET=_blank>Joseph Ecker</A>, Salk Institute
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    Genome Wide Analysis of DNA Methylation
		    </LI>
                    <LI>
                    <A HREF="http://www.biosci.utexas.edu/mgm/people/faculty/profiles/iyer.htm" 
		    TARGET=_blank>Vishwanath Iyer</A>, University of Texas, 
		    Austin
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    STAGE and FAIRE for Regulatory Element Identification
		    </LI>
                    <LI>
                    Madaiah Puttaraju,  
		    Intronn, Inc.
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    Use of RNA trans-splicing to identify splice sites
		    </LI>
                    <LI>
                    <A HREF="http://www.gis.a-star.edu.sg/internet/site/investigators.php?f=cv&user_id=19" 
		    TARGET=_blank>Yijun Ruan</A>,  
                    <A HREF="http://www.gis.a-star.edu.sg/internet/site" 
		    TARGET=_blank>Genome Institute of Singapore</A>
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    Ditag technologies for complete transcriptome annotation
		    </LI>
                    <LI>
                    <A HREF="http://www.albany.edu/cancergenomics/faculty/stenenbaum/stenenbaum.html" 
		    TARGET=_blank>Scott Tenenbaum</A>, SUNY Albany
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    Identifying functional regulatory elements in RNA
		    </LI>
                    <LI>
                    <A HREF="http://www.bu.edu/chemistry/faculty/profile/tullius.html" 
		    TARGET=_blank>Tom Tullius</A>, Boston University
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    Structure of genomic DNA at single-nucleotide resolution
		    </LI>
		    </UL></P>
	            </TD><TD WIDTH=15>
                </TD></TR></TABLE>
	    <BR></TD></TR></TABLE>
        </TD></TR></TABLE>
	<BR>

        <!--outer table is for border purposes-->
        <TABLE WIDTH="100%" BGCOLOR="#888888" BORDER="0" CELLSPACING="0" CELLPADDING="1">
            <TR><TD>	
            <TABLE BGCOLOR="fffee8" WIDTH="100%"  BORDER="0" CELLSPACING="0" CELLPADDING="0">
                <TR><TD>	

	        <TABLE BGCOLOR="D9E4F8" BACKGROUND="../images/hr.gif" WIDTH="100%"><TR><TD>
	            <A NAME="Other"></A><FONT SIZE="4"><B>&nbsp; 
		    Other Groups
		    </B></FONT>
	        </TD></TR></TABLE>
	
	        <TABLE BGCOLOR="fffee8" WIDTH="100%" CELLPADDING=0>
                    <TR><TH HEIGHT=3></TH></TR>
                    <TR><TD WIDTH=10></TD><TD>
		    <P>
		    These researchers provided infrastructure development 
		    to support ENCODE-related research.
		    <UL>
                    <LI>
		    <A HREF="http://www.genome.gov/10000736" 
		    TARGET=_blank>Andreas D. Baxevanis</A>,  
		    <A HREF="http://www.genome.gov/10000608" 
		    TARGET=_blank>NHGRI Division of Intramural Research</A>
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    Coordination of the database for other data types
		    </LI>
                    <LI>
		    Pieter De Jong, <A HREF="http://www.chori.org/" 
		    TARGET=_blank>Children's Hospital Oakland Research 
		    Institute</A>
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    <A HREF="http://bacpac.chori.org/" TARGET=_blank>Clone 
		    resources</A> to support comparative sequencing.
		    </LI>
		    <LI> 
                    <A HREF="http://www.genome.gov/10000452" 
		    TARGET=_blank>Eric Green</A>, 
		    <A HREF="http://www.nisc.nih.gov/" 
		    TARGET=_blank>NIH Intramural Sequencing Center</A>
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    <A HREF="http://www.nisc.nih.gov/open_page.cgi?path=/projects/encode/index.cgi"
                    TARGET=_blank>Comparative sequencing</A>
		    </LI>
                    <LI>
                    <A HREF="http://www.bmb.psu.edu/faculty/hardison/hardison.html" 
		    TARGET=_blank>Ross Hardison</A>,  
                    <A HREF="http://www.bx.psu.edu/miller_lab/" 
		    TARGET=_blank>Webb Miller</A>,  
		    <A HREF="http://www.bx.psu.edu/" 
		    TARGET=_blank>Pennsylvania State University</A>
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    Computational tools for comparative genomics; collaboration with UCSC
                    </LI>
                    <LI>
                    <A HREF="http://www.soe.ucsc.edu/~kent/"
		    TARGET=_blank>Jim Kent</A>, 
                    <A HREF="http://www.cbse.ucsc.edu/people/haussler"
		    TARGET=_blank>David Haussler</A>, Kate Rosenbloom, Daryl Thomas, Ting 
		    Wang, and the  
		    <A HREF="../staff.html"
		    TARGET=_blank>UCSC Genome Bioinformatics staff</A>
		    <BR>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
		    Coordination of the Genome Browser website and the database 
		    for all sequence-related data
		    </LI>
                    <LI>
                    Kerstin Lindblad-Toh,  Michele Clamp,
                    <A HREF="http://www.broad.mit.edu/" TARGET=_BLANK>Broad 
		    Institute</A>
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    Computational tools for comparative genomics and prediction 
		    of cis-regulatory elements
                    </LI>
                    <LI>
                    <A HREF="http://www.bcgsc.ca/about/faculty/person?pid=mmarra" 	
		    TARGET=_blank>Marco Marra</A>,  
                    British Columbia Cancer Agency Genome Sciences Center
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    Generation of maps and contigs for ENCODE BACs
                    </LI>
                    <LI>
                    Greg Schuler,  
                    <A HREF="https://www.ncbi.nlm.nih.gov/" 
		    TARGET=_blank>National Center for Biotechnology Information 
		    (NCBI)</A>
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    Coordination of ENCODE data with NCBI
                    </LI>
		    </UL></P>
	            </TD><TD WIDTH=15>
                </TD></TR></TABLE>

	        <TABLE BGCOLOR="fffee8" WIDTH="100%" CELLPADDING=0>
                    <TR><TH HEIGHT=3></TH></TR>
                    <TR><TD WIDTH=10></TD><TD>
		    <P>
                    Additional ENCODE pilot phase participants:
		    <UL>
                    <LI>
                    Greg Crawford,
                    <A HREF="http://www.genome.gov/10000779" TARGET=_blank>
		    Francis Collins</A>, 
                    NHGRI Intramural Program
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    Identification of DNAse hypersensitive sites
                    </LI>
                    <LI>
                    <A HREF="http://www.bcgsc.ca/about/faculty/person?pid=sjones" TARGET=_BLANK>
                    Steven Jones</A>, 
                    <A HREF="http://www.bcgsc.ca/about/" 
		    TARGET=_BLANK>British Columbia Cancer Agency Genome Sciences
		    Center</A>
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    Computational tools for comparative genomics,
                    visualization of genome,
                    features, and prediction of cis-regulatory elements
                    </LI>
                    <LI>
                    <A HREF="http://www.cbcb.umd.edu/~salzberg/"
                    TARGET=_BLANK>Steven Salzberg</A>,
                    <A HREF="http://www.cbcb.umd.edu/" TARGET=_BLANK>
                    Center for Bioinformatics and Computational Biology</A>,
                    University of Maryland
		    <BR> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
                    Computational tools for comparative genomics
                    </LI>
		    </UL></P>
	            </TD><TD WIDTH=15>
                </TD></TR></TABLE>

	    <BR></TD></TR></TABLE>
        </TD></TR></TABLE>
        <BR>
    </TD></TR></TABLE>

</BODY></HTML>

