<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
    <head>
        <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
        <title>Software Tools Used to Create the ENCODE Resource</title>
        <link rel="stylesheet" href="/style/HGStyle.css" type="text/css">
        <link rel="stylesheet" href="/style/encodeProject.css" type="text/css">
    </head>

    <body>
        <!--#include virtual="/inc/encodeProjectSoftware.topbar.html"-->

        <div class="encodeHeader">
            <a href="http://www.genome.gov/10005107"><img src="/images/ENCODE_scaleup_logo.png" alt="ENCODE Project at NHGRI" title="ENCODE Project at NHGRI"></a>
            <span class="txt">Software Tools Used to Create the ENCODE Resource</span>
        </div>

       <div class="wrapper">
                <div class="bar"><h4 class="title">Contents</h4></div>
                <div class="content">

                <p>
                <ul>
                  <li><a href="#features">Tools for identifying features from ENCODE assays</a></li>
                  <li><a href="#analysis">Integrative analysis across data types</a></li>
                  <li><a href="#metrics">Quality Metrics for ENCODE data</a></li>
                  <li><a href="#other">Other Tools</a></li>
                </ul>
                </p>
                <p class="date">Updated 21 March 2014</p>

               </div><!--end content-->
       </div><!--end wrapper-->

       <div class="wrapper">
                <div class="bar"><h4 class="title"><a name="features">Tools for identifying features from ENCODE assays</a></h4></div>
                <div class="content">
                        <h4>ChIP-seq Peak Callers</h4>

                      <ul>

                        <p><li>
                        <a class="thick" href="http://compbio.med.harvard.edu/Supplements/ChIP-seq/" target="_blank">SPP</a><br>
                        A ChIP-seq peak calling algorithm, implemented as an R package, that accounts for the offset
                        in forward-strand and reverse-strand reads to improve resolution, compares enrichment in
                        signal to background or control experiments, and can also estimate whether the available number
                        of reads is sufficient to achieve saturation, meaning that additional reads would not allow
                        identification of additional peaks. SPP will be used in the ENCODE 3 uniform peak calling pipeline.
                        </p>
                        <p>
                        Kharchenko PV, Tolstorukov MY, Park PJ.
                        <a href="http://dx.doi.org/10.1038/nbt.1508" target="_blank">
                        Design and analysis of ChIP-seq experiments for DNA-binding proteins</a>.
                        <em>Nat Biotechnol</em>. 2008 Dec;26(12):1351-9.
                        PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19029915" target="_blank">19029915</a>; PMC: <a
                        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2597701" target="_blank">PMC2597701</a>
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://sysbio.mit.edu/gem/" target="_blank">GEM</a><br>
                        GEM is a Java software package for analyzing genome wide ChIP-seq/ChIP-exo data.
                        GEM can decompose single observed peaks into multiple binding events, determine
                        binding event location at high spatial resolution, and discover explanatory DNA
                        sequence motifs with an integrated model of ChIP reads and proximal DNA sequences.
                        GEM is able to process single-end or paired-end data and can be run in single-condition
                        mode or multi-condition mode.  GEM will be used in the ENCODE 3 uniform peak calling pipeline.
                        </p>
                        <p>
                        Guo Y, Mahony S, Gifford DK.
                        <a href="http://dx.plos.org/10.1371/journal.pcbi.1002638" target="_blank">
                        High resolution genome wide binding event finding and motif discovery reveals transcription factor
                        spatial binding constraints</a>.
                        <em>PLoS Comput Biol</em>. 2012;8(8):e1002638.
                        PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22912568" target="_blank">22912568</a>; PMC: <a
                        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3415389" target="_blank">PMC3415389</a>
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://info.gersteinlab.org/PeakSeq" target="_blank">PeakSeq</a><br>
                        Identifies enriched regions in ChIP-seq type experiments and explicitly compares signal
                        experiments to control experiments. PeakSeq will be used in the ENCODE 3 uniform peak calling
                        pipeline.
                        </p>
                        <p>
                        Rozowsky J, Euskirchen G, Auerbach RK, Zhang ZD, Gibson T, Bjornson R, Carriero N, Snyder M,
                        Gerstein MB.
                        <a href="http://dx.doi.org/10.1038/nbt.1518" target="_blank">
                        PeakSeq enables systematic scoring of ChIP-seq experiments relative to controls</a>.
                        <em>Nat Biotechnol</em>. 2009 Jan;27(1):66-75.
                        PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19122651" target="_blank">19122651</a>; PMC: <a
                        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2924752" target="_blank">PMC2924752</a>
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://liulab.dfci.harvard.edu/MACS/" target="_blank">MACS</a><br>
                        A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset
                        in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson
                        distribution to effectively capture local biases in the genome.  MACS 1.4 was used in
                        the ENCODE 2 uniform peak calling pipeline.
                        </p>
                        <p>
                        Feng J, Liu T, Zhang Y.
                        <a href="http://dx.doi.org/10.1002/0471250953.bi0214s34" target="_blank">
                        Using MACS to identify peaks from ChIP-Seq data</a>.
                        <em>Curr Protoc Bioinformatics</em>. 2011 Jun;Chapter 2:Unit 2.14.
                        PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21633945" target="_blank">21633945</a>; PMC: <a
                        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3120977" target="_blank">PMC3120977</a>
                        </p>
                        <p>
                        Zhang Y, Liu T, Meyer CA, Eeckhoute J, Johnson DS, Bernstein BE, Nusbaum C, Myers RM, Brown M, Li W
                        <em>et al</em>.
                        <a href="http://genomebiology.com/content/9/9/R137" target="_blank">
                        Model-based analysis of ChIP-Seq (MACS)</a>.
                        <em>Genome Biol</em>. 2008;9(9):R137.
                        PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18798982" target="_blank">18798982</a>; PMC: <a
                        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2592715" target="_blank">PMC2592715</a>
                        </p>
                        </li></p>

                      </ul>

                        <hr>
                        <h4>RNA-seq Analysis</h4>

                      <ul>
                        <p><li>
                        <a class="thick" href="https://github.com/alexdobin/STAR/releases" target="_blank">STAR</a><br>
                        STAR (Spliced Transcript Alignment to a Reference) aligns short and long
                        RNA-seq reads to a reference genome using uncompressed suffix arrays.
                        </p>
                        <p>
                        Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, Batut P, Chaisson M, Gingeras TR.
                        <a href="http://bioinformatics.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=23104886"
                        target="_blank">
                        STAR: ultrafast universal RNA-seq aligner</a>.
                        <em>Bioinformatics</em>. 2013 Jan 1;29(1):15-21.
                        PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23104886" target="_blank">23104886</a>; PMC: <a
                        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905" target="_blank">PMC3530905</a>
                        </p>
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://sammeth.net/confluence/display/FLUX/Home" target="_blank">Flux Capacitor</a><br>
                        A program to estimate the frequencies of annotated transcripts (GTF format)
                        from an RNA-Seq experiment, solving a linear program inferred from the
                        observed read mappings (BAM format). There are options for single, stranded,
                        and/or paired-end reads.
                        </p>
                        <p>
                        Montgomery SB, Sammeth M, Gutierrez-Arcelus M, Lach RP, Ingle C, Nisbett J, Guigo R, Dermitzakis ET.
                        <a href="http://dx.doi.org/10.1038/nature08903" target="_blank">
                        Transcriptome genetics using second generation sequencing in a Caucasian population</a>.
                        <em>Nature</em>. 2010 Apr 1;464(7289):773-7.
                        PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20220756" target="_blank">20220756</a>; PMC: <a
                        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3836232" target="_blank">PMC3836232</a>
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://info.gersteinlab.org/FusionSeq" target="_blank">FusionSeq</a><br>
                        A computational framework that detects chimeric transcripts from paired-end RNA-seq experiments.
                        </p>

                        <p>
                        Sboner A, Habegger L, Pflueger D, Terry S, Chen DZ, Rozowsky JS, Tewari AK, Kitabayashi N, Moss BJ, Chee MS <em>et al</em>.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/20964841" target="_blank">
                        FusionSeq: a modular framework for finding gene fusions by analyzing paired-end RNA-sequencing data</a>.
                        <em>Genome Biol</em>. 2010;11(10):R104.
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://archive.gersteinlab.org/proj/rnaseq/IQSeq/" target="_blank">IQSeq</a><br>
                        Uses RNA-seq data for isoform quantification.
                        </p>

                        <p>
                        Du J, Leng J, Habegger L, Sboner A, McDermott D, Gerstein M.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/22238592" target="_blank">
                        IQSeq: integrated isoform quantification analysis based on next-generation sequencing</a>.
                        <em>PLoS One</em>. 2012;7(1):e29175.
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://info.gersteinlab.org/RSEQtools" target="_blank">RSEQtools</a><br>
                        Processes, quantifies, and annotates gene expression data from RNA-seq experiments. Utilizes
                        Mapped Read Format (MRF) for secure and efficient analysis.
                        </p>

                        <p>
                        Habegger L, Sboner A, Gianoulis TA, Rozowsky J, Agarwal A, Snyder M, Gerstein M.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/21134889" target="_blank">
                        RSEQtools: a modular framework to analyze RNA-Seq data using compact, anonymized data summaries</a>.
                        <em>Bioinformatics</em>. 2011 Jan 15;27(2):281-3.
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://www.broadinstitute.org/software/scripture/" target="_blank">Scripture</a><br>
                        Reconstructs transcriptomes, relying solely on RNA-seq reads and an assembled genome to build a
                        transcriptome ab initio. The statistical methods to estimate read coverage significance are also
                        applicable to other sequencing data. Scripture also has modules for ChIP-seq peak calling.
                        </p>

                        <p>
                        Guttman M, Garber M, Levin JZ, Donaghey J, Robinson J, Adiconis X, Fan L, Koziol MJ, Gnirke A, Nusbaum C <em>et al</em>.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/20436462" target="_blank">
                        Ab initio reconstruction of cell type-specific transcriptomes in mouse reveals the conserved multi-exonic structure of lincRNAs</a>.
                        <em>Nat Biotechnol</em>. 2010 May;28(5):503-10.
                        </li></p>
                      </ul>

                        <hr>
                        <h4>Other</h4>

                      <ul>
                        <p><li>
                        <a class="thick" href="http://alleleseq.gersteinlab.org/" target="_blank">AlleleSeq</a><br>
                        Quantifies the allele-specific binding and expression at the SNP sites using
                        the RNA-seq, ChIP-seq datasets, and diploid genome sequences.
                        </p>

                        <p>
                        Rozowsky J, Abyzov A, Wang J, Alves P, Raha D, Harmanci A, Leng J, Bjornson R, Kong Y, Kitabayashi N <em>et al</em>.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/21811232" target="_blank">
                        AlleleSeq: analysis of allele-specific expression and binding in a network framework</a>.
                        <em>Mol Syst Biol</em>. 2011 Aug 2;7:522.
                        </li></p>

                        <p><li>
                        <a name="hotspot" class="thick" href="http://www.uwencode.org/software/hotspot" target="_blank">Hotspot</a><br>
                        Identifies regions of local enrichment, including peaks, in genomic short-read sequence data.
                        Uses the binomial distribution with a local background model to automatically correct for
                        broad-scale regional differences in tag levels.  It is applicable to a wide variety of epigenetic
                        profiling assays, including ChIP-seq and DNase-seq.  Hotspot forms the basis of the
                        <a href="#spot">SPOT</a> data quality metric (described below).
                        </p>

                        <p>
                        John S, Sabo PJ, Thurman RE, Sung MH, Biddie SC, Johnson TA, Hager GL, Stamatoyannopoulos JA.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/21258342" target="_blank">
                        Chromatin accessibility pre-determines glucocorticoid receptor binding patterns</a>.
                        <em>Nat Genet</em>. 2011 Mar;43(3):264-8.
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://archive.gersteinlab.org/proj/incrna/" target="_blank">incRNA</a><br>
                        A computational framework that identifies structured RNAs by combining a variety of gene expression
                        data and several sequence-based metrics.
                        </p>

                        <p>
                        Lu ZJ, Yip KY, Wang G, Shou C, Hillier LW, Khurana E, Agarwal A, Auerbach R, Rozowsky J, Cheng C <em>et al</em>.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/21177971" target="_blank">
                        Prediction and characterization of noncoding RNAs in <em>C. elegans</em> by integrating conservation, secondary structure, and high-throughput sequencing and array data</a>.
                        <em>Genome Res</em>. 2011 Feb;21(2):276-85.
                        </li></p>

                        <p><li>
                        <a class="thick">Irreproducible Discovery Rate (IDR)</a><br>
                        Measures consistency between replicates in high-throughput experiments.  The core IDR R package can
                        be downloaded from the IDR <a href="http://cran.r-project.org/web/packages/idr/index.html" target="_blank">
                        download page</a>. The version of IDR code used for processing ENCODE ChIP-seq data and the
                        commands / manual for running IDR on ChIP-seq data can be found
                        at the <a href="https://sites.google.com/site/anshulkundaje/projects/idr" target="_blank">
                        IDR: Reproducibility and automatic thresholding of ChIP-seq data</a> page.
                        </p>

                        <p>
                        Kundaje A, Li Q, Brown J, Rozowsky J, Harmanci A, Wilder S, Gerstein M, Dunham I, Birney E, Batzoglou S <em>et al</em>.
                        Reproducibility measures for automatic threshold selection and quality control in ChIP-seq datasets. (In Preparation)
                        </p>

                        <p>
                        Li Q, Brown JB, Huang H, Bickel PJ.
                        <em>Annals of Applied Statistics</em>. 5(3):1752-1779.
                        </li></p>

                        <p><li>
                        <a class="thick" href="https://sites.google.com/site/anshulkundaje/projects/wiggler" target="_blank">Wiggler</a><br>
                        Produces normalized genome-wide signal coverage tracks from raw read alignment files.
                        Allows pooling of replicate datasets while allowing for replicate and data-type specific read
                        shifting and smoothing parameters. It can be used to generate signal density maps for ChIP-seq,
                        DNase-seq, FAIRE-seq and MNase-seq data. Wiggler also implicitly models variability in
                        mappability to appropriately normalize signal density and distinguish missing data from true zero
                        signal. Wiggler software is available for download from the align2rawsignal
                        <a href="http://code.google.com/p/align2rawsignal/" target="_blank">
                        project page</a>.
                        </p>

                        <p>
                        Hoffman MM, Ernst J, Wilder SP, Kundaje A, Harris RS, Libbrecht M, Giardine B, Ellenbogen PM, Bilmes
                        JA, Birney E <em>et al</em>.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/23221638" target="_blank">
                        Integrative annotation of chromatin elements from ENCODE data</a>.
                        <em>Nucleic Acids Res</em>. 2013 Jan;41(2):827-41.
                        PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23221638" target="_blank">23221638</a>; PMC: <a
                        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3553955" target="_blank">PMC3553955</a>
                        </p>

                        </li></p>
                      </ul>
               </div><!--end content-->
       </div><!--end wrapper-->

       <div class="wrapper">
                <div class="bar"><h4 class="title"><a name="analysis">Integrative analysis across data types</a></h4></div>
                <div class="content">
                        <h4>Genome Segmentation</h4>

                      <ul>
                        <p><li>
                        <a class="thick" href="http://compbio.mit.edu/ChromHMM/" target="_blank">ChromHMM</a><br>
                        Learns and characterizes chromatin states.
                        </p>

                        <p>
                        Ernst J, Kellis M.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/22373907" target="_blank">
                        ChromHMM: automating chromatin-state discovery and characterization</a>.
                        <em>Nat Methods</em>. 2012 Feb 28;9(3):215-6.
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://noble.gs.washington.edu/proj/segtools/" target="_blank">Segtools</a><br>
                        A Python package that analyzes genomic segmentations. The software efficiently calculates
                        a variety of summary statistics and produces corresponding publication quality visualizations. The
                        overall goal of Segtools is to provide a bird's-eye view of complex genomic data sets, allowing
                        researchers to easily generate and confirm hypotheses.
                        </p>

                        <p>
                        Buske OJ, Hoffman MM, Ponts N, Le Roch KG, Noble WS.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/22029426" target="_blank">
                        Exploratory analysis of genomic segmentations with Segtools</a>.
                        <em>BMC Bioinformatics</em>. 2011 Oct 26;12:415.
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://noble.gs.washington.edu/proj/segway/" target="_blank">Segway</a><br>
                        Uses a machine learning method to analyze multiple tracks of functional genomics data,
                        searching for recurring patterns.  The software automatically partitions the genome into
                        non-overlapping segments and assigns each segment a label.  The resulting annotation provides a
                        human-interpretable summary of the functional landscape of the genome, yielding hypotheses about
                        novel instances or classes of functional elements.
                        </p>

                        <p>
                        Hoffman MM, Buske OJ, Wang J, Weng Z, Bilmes JA, Noble WS.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/22426492" target="_blank">
                        Unsupervised pattern discovery in human chromatin structure through genomic segmentation</a>.
                        <em>Nat Methods</em>. 2012 Mar 18;9(5):473-6.
                        </li></p>
                      </ul>

                        <hr>
                        <h4>Other</h4>

                      <ul>
                        <p><li>
                        <a class="thick" href="http://act.gersteinlab.org/" target="_blank">ACT</a><br>
                        Performs aggregation and correlation analyses of genomic tracks.
                        </p>

                        <p>
                        Jee J, Rozowsky J, Yip KY, Lochovsky L, Bjornson R, Zhong G, Zhang Z, Fu Y, Wang J, Weng Z <em>et al</em>.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/21349863" target="_blank">
                        ACT: aggregation and correlation toolbox for analyses of genome tracks</a>.
                        <em>Bioinformatics</em>. 2011 Apr 15;27(8):1152-4.
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://code.google.com/p/cagt/" target="_blank">CAGT (Clustering AGgregation Tool)</a><br>
                        Deciphers the heterogeneity and diversity of profiles of functional signals (e.g.,
                        chromatin mark ChIP-seq signal) centered at a collection of sites (e.g., TSSs or TF
                        binding sites) in a genome. Rather than averaging the profiles over all the anchor
                        sites (traditional aggregation plots), CAGT accounts for the inherent heterogeneity
                        in signal magnitude, shape and orientation. CAGT partitions the set of anchor sites into
                        compact clusters such that each cluster represents anchor points that show similar patterns
                        of the functional signal profiles with different clusters having distinct patterns. The
                        different groups of patterns are often enriched for distinct biological functions.
                        </p>

                        <p><li>
                        <a class="thick" href="http://archive.gersteinlab.org/proj/tftarget/" target="_blank">TIP</a><br>
                        Predicts the targets of transcription factors using the data from ChIP-seq experiments.
                        </p>

                        <p>
                        Cheng C, Min R, Gerstein M.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/22039215" target="_blank">
                        TIP: a probabilistic method for identifying transcription factor target genes from ChIP-seq binding profiles</a>.
                        <em>Bioinformatics</em>. 2011 Dec 1;27(23):3221-7.
                        </li></p>
                      </ul>
               </div><!--end content-->
       </div><!--end wrapper-->

       <div class="wrapper">
                <div class="bar"><h4 class="title"><a name="metrics">Quality Metrics for ENCODE data</a></h4></div>
                <div class="content">
                      <ul>
                        <p><li>
                        <a class="thick">NSC, RSC, PBC</a><br>
                        The NSC (Normalized strand cross-correlation) and RSC (relative strand cross-correlation) metrics
                        use cross-correlation of stranded read density profiles to measure enrichment independently of peak
                        calling. The PBC (PCR bottleneck coefficient) is an approximate measure of library complexity. PBC is
                        the ratio of (non-redundant, uniquely mappable reads)/(uniquely mappable reads). The code for computing
                        RSC and NSC can be downloaded from the phantompeakqualtools
                        <a href="http://code.google.com/p/phantompeakqualtools/" target="_blank">
                        project page</a>.
                        </p>

                        <p>
                        Kundaje A, Jung LY, Kharchenko P, Wold B, Sidow A, Batzoglou S, Park P.
                        Assessment of ChIP-seq data quality using cross-correlation analysis. (Submitted)
                        </li></p>

                        <p><li>
                        <a name="spot" class="thick">SPOT (Signal Portion of Tags)</a><br>
                        Measures signal-to-noise in genome-wide epigenetic profiling assays by calculating the fraction
                        of reads that fall in tag-enriched regions (see the <a href="#hotspot">Hotspot</a> program,
                        described above) from a sample of 5 million reads.  The SPOT methodology can be generalized to
                        use any peak-finder.  A publication of SPOT and a more complete description are in preparation.
                        </li></p>
                      </ul>
               </div><!--end content-->
       </div><!--end wrapper-->

       <div class="wrapper">
                <div class="bar"><h4 class="title"><a name="other">Other Tools</a></h4></div>
                <div class="content">
                      <ul>
                        <p><li>
                        <a class="thick" href="http://code.google.com/p/bedops/" target="_blank">BEDOPS</a><br>
                        Performs common genomic analysis tasks and offers improved flexibility, scalability
                        and execution time characteristics over previously published packages. The suite
                        includes a utility to compress large inputs into a lossless format that can provide
                        greater space savings and faster data extractions than alternatives.
                        </p>

                        <p>
                        Neph S, Kuehn MS, Reynolds AP, Haugen E, Thurman RE, Johnson AK, Rynes E, Maurano MT, Vierstra J,
                        Thomas S <em>et al</em>.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/22576172" target="_blank">
                        BEDOPS: high-performance genomic feature operations</a>.
                        <em>Bioinformatics</em>. 2012 Jul 15;28(14):1919-20.
                        </li></p>

                        <p><li>
                        <a class="thick" href="http://noble.gs.washington.edu/proj/genomedata/" target="_blank">Genomedata</a><br>
                        Efficiently stores multiple tracks of numeric data anchored to a genome. The format allows fast
                        random access to hundreds of gigabytes of data, while retaining a small disk space footprint.
                        Utilities have also been developed to load data into this format. A reference implementation in
                        Python and C components is available under the GNU General Public License.
                        </p>

                        <p>
                        Hoffman MM, Buske OJ, Noble WS.
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/20435580" target="_blank">
                        The Genomedata format for storing large-scale functional genomics data</a>.
                        <em>Bioinformatics</em>. 2010 Jun 1;26(11):1458-9.
                        </li></p>
                      </ul>
               </div><!--end content-->
       </div><!--end wrapper-->
    </body>
</html>
