<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<html>
<head>
  <title>Random Regions for 1% Project</title>
</head>
  <body>
                                               
<h1>Stratified Random Picks</h1>
                        To make the stratified picks the genome is divided
 into   the   top   20%,   middle  30%, and bottom 50% along two axis - gene
 density   and   nontranscribed     conservation.   Then three random picks
 are taken   from   each strata, and   a  fourth pick in  the strata that
are underrepresented     in the manual picks.      One additional  backup
pick is made in each strata    in case there is an   unforeseen  technical
 problem with a region.  The   backup pick is parenthesized   below.<br>
                   <br>

                   The left coordinate is the June genomic position for the 
 feature     while    <br>
                   the right coordinate is the November genomic position
for   the   feature.<br>
                                            
<h2>consNonTx 0% - 50%, gene 0% - 50% (1 manual)</h2>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr13:28500001-29000000">June:
          chr13:28500001-29000000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr13:24500016-25000015">Nov:
          chr13:24500016-25000015</a>  consNonTx  2.8%, gene  0.5%<br>

                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr2:51700001-52200000">June: 
         chr2:51700001-52200000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr2:51837455-52337454">Nov: 
         chr2:51837455-52337454</a>   consNonTx  3.8%, gene  0.0%<br>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr4:119000001-119500000">June: 
         chr4:119000001-119500000</a> &nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr4:118527386-119027385">Nov: 
         chr4:118527386-119027385</a>   consNonTx  3.9%, gene  0.0%<br>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr10:54300001-54800000">June: 
         chr10:54300001-54800000</a> &nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr10:54489120-54989119">Nov: 
         chr10:54489120-54989119</a>   consNonTx  2.8%, gene  1.2%<br>

                        (<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr5:15900001-16400000">June: 
         chr5:15900001-16400000</a> &nbsp; &nbsp;<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr5:16187472-16687471">Nov: 
         chr5:16187472-16687471</a>   consNonTx  5.1%, gene  1.7%)<br>
                                               
<h2>consNonTx 0% - 50%, gene 50% - 80% (4 manual)</h2>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr2:115500001-116000000">June: 
         chr2:115500001-116000000</a> &nbsp; &nbsp;<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr2:116215329-116715328"> 
         Nov: chr2:116215329-116715328</a>   consNonTx  6.2%, gene  2.3%<br>

                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr18:61100001-61600000">June: 
         chr18:61100001-61600000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr18:61234622-61734621">Nov: 
         chr18:61234622-61734621</a>   consNonTx  3.4%, gene  3.4%<br>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr12:40500001-41000000">June: 
         chr12:40500001-41000000</a> &nbsp; &nbsp;<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr12:40239443-40739442">Nov: 
  chr12:40239443-40739442</a>          consNonTx  1.7%, gene  3.1%<br>
                        (<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr2:196700001-197200000">June: 
         chr2:196700001-197200000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr12:40239443-40739442">Nov: 
         chr2:197214044-197714043</a>   consNonTx  5.4%, gene  3.3%)<br>

                                               
<h2>consNonTx 0% - 50%, gene 80% - 100% (11 manual)</h2>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr2:232500001-233000000">June: 
         chr2:232500001-233000000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr2:233173598-233673597">Nov: 
         chr2:233173598-233673597</a>   consNonTx  1.3%, gene  4.6%<br>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr13:111900001-112400000">June: 
         chr13:111900001-112400000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr13:107927238-108427237">Nov: 
         chr13:107927238-108427237</a>   consNonTx  1.1%, gene  5.5%<br>

                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr21:36900001-37400000">June: 
         chr21:36900001-37400000</a> &nbsp; &nbsp;<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr21:36983033-37483032">Nov: 
         chr21:36983033-37483032</a>   consNonTx  2.3%, gene  5.2%<br>
                        (<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr4:47800001-48300000">June: 
         chr4:47800001-48300000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr4:48032776-48532775">Nov: 
         chr4:48032776-48532775</a>   consNonTx  1.9%, gene  4.4%)<br>

                                               
<h2>consNonTx 50% - 80%, gene 0% - 50% (2 manual)</h2>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr16:25300001-25800000">June: 
         chr16:25300001-25800000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr16:25969826-26469825">Nov: 
         chr16:25969826-26469825</a>   consNonTx  9.7%, gene  0.5%<br>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr5:141800001-142300000">June: 
         chr5:141800001-142300000</a> &nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr5:142482586-142982585">&nbsp;Nov: 
         chr5:142482586-142982585</a>   consNonTx  6.7%, gene  1.7%<br>

                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr18:25400001-25900000">June: 
         chr18:25400001-25900000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr18:25196197-25696196">Nov: 
         chr18:25196197-25696196</a>   consNonTx  7.4%, gene  0.9%<br>
                        (<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr4:124800001-125300000">June: 
         chr4:124800001-125300000</a> &nbsp; &nbsp;<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr4:124166677-124666676">Nov: 
         chr4:124166677-124666676</a>   consNonTx  6.3%, gene  0.9%)<br>

                                               
<h2>consNonTx 50% - 80%, gene 50% - 80% (4 manual)</h2>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr5:56000001-56500000">June: 
         chr5:56000001-56500000</a> &nbsp; &nbsp;<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr5:57392856-57892855">Nov: 
         chr5:57392856-57892855</a> consNonTx   7.9%, gene  2.2%<br>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr6:131800001-132300000">June: 
         chr6:131800001-132300000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr6:132023965-132523964">Nov: 
         chr6:132023965-132523964</a> consNonTx   6.9%, gene  2.1%<br>

                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr6:73700001-74200000">June: 
         chr6:73700001-74200000</a> &nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr6:73699933-74199932">&nbsp;Nov: 
         chr6:73699933-74199932</a> consNonTx   6.4%, gene  3.6%<br>
                        (<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr4:53700001-54200000">June: 
         chr4:53700001-54200000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr4:53859184-54359183">Nov: 
         chr4:53859184-54359183</a> consNonTx   9.0%, gene  2.1%)<br>

                                               
<h2>consNonTx 50% - 80%, gene 80% - 100% (3 manual)</h2>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr1:149000001-149500000">June: 
         chr1:149000001-149500000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr1:146905332-147405331">Nov: 
         chr1:146905332-147405331</a>  consNonTx 10.2%, gene  8.4%<br>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr9:122800001-123300000">June: 
         chr9:122800001-123300000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr9:123331831-123831830">Nov: 
         chr9:123331831-123831830</a>  consNonTx  8.3%, gene  5.9%<br>

                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr15:39100001-39600000">June: 
         chr15:39100001-39600000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr15:36628619-37128618">Nov: 
        chr15:36628619-37128618</a> consNonTx  9.7%, gene 10.6%<br>
                        (<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr17:33400001-33900000">June: 
         chr17:33400001-33900000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr17:35665792-36165791">Nov: 
         chr17:35665792-36165791</a>  consNonTx  7.7%, gene  6.1%)<br>

                                               
<h2>consNonTx 80% - 100%, gene 0% - 50% (3 manual)</h2>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr14:51200001-51700000">June: 
         chr14:51200001-51700000</a> &nbsp;&nbsp;	<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr14:47673341-48173340">Nov: 
        chr14:47673341-48173340</a> consNonTx 14.9%, gene  0.1%<br>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr11:133100001-133600000">June: 
         chr11:133100001-133600000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr11:132612235-133112234">Nov: 
         chr11:132612235-133112234</a>  consNonTx 13.5%, gene  0.3%<br>

                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr16:52600001-53100000">June: 
         chr16:52600001-53100000</a> &nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr16:62362206-62862205">Nov: 
         chr16:62362206-62862205</a>  consNonTx 15.4%, gene  0.0%<br>
                        (<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chrX:41900001-42400000">June: 
         chrX:41900001-42400000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chrX:42149253-42649252">Nov: 
         chrX:42149253-42649252</a>  consNonTx 13.4%, gene  0.7%)<br>

                                               
<h2>consNonTx 80% - 100%, gene 50% - 80% (1 manual)</h2>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr8:117800001-118300000">June: 
         chr8:117800001-118300000</a> &nbsp; &nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr8:118874200-119374199">Nov: 
         chr8:118874200-119374199</a>  consNonTx 11.4%, gene  3.2%<br>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr14:96900001-97400000">June: 
         chr14:96900001-97400000</a> &nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr14:93204045-93704044">Nov: 
         chr14:93204045-93704044</a>  consNonTx 15.9%, gene  2.9%<br>

                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chrX:117500001-118000000">June: 
         chrX:117500001-118000000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chrX:119675382-120175381">Nov: 
         chrX:119675382-120175381</a>  consNonTx 10.7%, gene  2.0%<br>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr6:108100001-108600000">June: 
         chr6:108100001-108600000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr6:108287568-108787567">Nov: 
         &nbsp;chr6:108287568-108787567</a>	consNonTx   18.6%, gene  2.3%<br>

                                               
<h2>consNonTx 80% - 100%, gene 80% - 100% (1 manual) </h2>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr2:218300001-218800000">June: 
         chr2:218300001-218800000</a> &nbsp;&nbsp; <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr2:218998720-219498719">Nov: 
         chr2:218998720-219498719</a>  consNonTx 13.3%, gene  9.1%<br>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr11:66700001-67200000">June: 
         chr11:66700001-67200000</a>	&nbsp; &nbsp;<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr11:65865884-66365883">Nov: 
         chr11:65865884-66365883</a>  consNonTx 13.4%, gene  9.0%<br>

                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr20:33600001-34100000">June: 
         chr20:33600001-34100000</a> &nbsp; &nbsp;<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr20:33559944-34059943">Nov: 
         chr20:33559944-34059943</a>	consNonTx   11.5%, gene  9.2%<br>
                        <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr6:41300001-41800000">June: 
         chr6:41300001-41800000</a>	&nbsp; &nbsp;<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr6:41294331-41794330">Nov: 
         chr6:41294331-41794330</a>  consNonTx 15.2%, gene  4.8%<br>
                        (<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr9:124300001-124800000">June: 
         chr9:124300001-124800000</a>	&nbsp; &nbsp;<a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr9:124831831-125331830">Nov: 
         chr9:124831831-125331830</a> consNonTx  11.4%, gene  5.4%)<br>

                                                
<h1>Stratification of Manual Picks</h1>
               Here is the noncoding conservation and gene density of non-overlapping 
         500 kb regions in the manual picks.   The boundaries between strata 
   are:                             
<pre><tt>low 50%  middle 30%  high 20%<br>         ------------------------------<br>gene      0.0-1.9%  1.9-4.2%   4.2-100%<br>consNotTx 0.0-6.3%  6.3-10.6% 10.6-100%<br></tt><tt><br>CFTR </tt><br>June: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:114288355-116165780"><tt><tt>chr7:114288355-</tt></tt><tt>116165780</tt></a>   Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:114288155-116165580">chr7:114288155-116165580</a><br><tt>  </tt><tt><br>Interleukin_Cluster <br></tt>June: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr5:130778557-131778556"><tt>chr5:130778557-</tt><tt>131778556</tt></a><tt> </tt>  Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr5:131703638-132703637">chr5:131703638-132703637</a>  <br><tt>   </tt><tt><br>Apo_Cluster <br></tt>June: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr11:118810001-119310000"><tt><tt>chr11:118810001-119310000</tt></tt></a>   Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr11:117969240-118469239">chr11:117969240-118469239</a><br><tt>   </tt><tt><br>Chr22 </tt><br>June: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr22:28500001-30200000">chr22:28500001-30200000</a><tt> </tt>  Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr22:28500001-30200000">chr22:28500001-30200000</a><br><tt>  </tt><tt><br>Chr21 <br></tt>June: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr21:30323762-32019746"><tt><tt>chr21:30323762-</tt></tt><tt>32019746</tt></a>   Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr21:30406794-32102778">chr21:30406794-32102778</a><br><tt>  </tt><tt></tt><br><tt>ChrX </tt><tt><br></tt>June: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chrX:147250001-148500000"><tt>chrX:147250001-</tt><tt>148500000</tt></a>   Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chrX:149572309-150846234">chrX:149572309-150846234</a><br><tt>  </tt><tt><br>Chr19 </tt><tt><br></tt>June: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr19:55200001-56200000"><tt>chr19:55200001-56200000</tt></a><tt> </tt>  Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr19:54724484-55728861">chr19:54724484-55728861</a> <br><tt>   </tt><tt><br>Alpha_Globin <br></tt>June: <tt><a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr16:79138-579137">chr16:79138-579137</a></tt>   Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr16:10001-510000">chr16:10001-510000</a><br><tt>    </tt><tt><br>Beta_Globin </tt><tt><br></tt>June: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr11:5550000-6549999"><tt>chr11:5550000-6549999</tt></a><tt>   </tt>Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr11:5076527-6078118">chr11:5076527-6078118</a><br><tt>  <br>HOXA_cluster <br></tt>June: <tt><a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:26600001-27100000">chr7:26600001-27100000</a>   </tt>Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:26599801-27099800">chr7:26599801-27099800</a><br><tt><br>IGF2/H19 <br></tt>June: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr11:300001-900000">chr11:300001-900000</a>   Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr11:1941933-2547980">chr11:1941933-2547980</a> PROBLEMATIC REGION - SUBSTANTIALLY REARRANGED BETWEEN JUNE AND NOVEMBER BUILDS<br><tt><br>FOXP2 </tt><br>June: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:112410791-113410790">chr7:112410791-113410790</a>   Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:112410791-113410790">chr7:112410791-113410790</a><br><tt>   </tt><tt><br></tt></pre>

                                                
<h1>Semi-Manual Picks</h1>
                         Here's the stratification of the other zoo-seq regions.
    I  recommend      picking  7q21.13 and 7q31.33 to round things out. 
                         
<pre><tt><br>7q21.13 <tt><br>June: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:88319137-89433560">chr7:88319137-89433560</a></tt>  Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:88318937-89433360">chr7:88318937-89433360</a><tt><br>  </tt><tt><br>7q21.3 <tt><br></tt></tt><tt><tt>June:</tt></tt><tt><tt> <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:91589227-92559635">chr7:91589227-92559635</a></tt>  Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:91589027-92559435">chr7:91589027-92559435</a><br><tt> </tt><tt><br>7q21.3 <tt><br></tt></tt><tt><tt>June:</tt></tt><tt><tt> <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:93650712-94868826">chr7:93650712-94868826</a></tt>Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:93650512-94868626">chr7:93650512-94868626</a><br><tt>  </tt><tt><br>7q31.33 <tt><br></tt></tt><tt><tt>June:</tt></tt><tt><tt> <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:124556444-125719632">chr7:124556444-125719632</a></tt>   Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:124556244-125719432">chr7:124556244-125719432</a><br><tt>   </tt><tt><br>7q32.1 <br></tt><tt><tt>June:</tt></tt><tt> <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:126427707-127330661">chr7:126427707-127330661</a></tt><tt> </tt>  Nov: <a
 href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:126427507-127330461">chr7:126427507-127330461</a><tt></tt></tt></tt></tt></tt></pre>

         <tt><tt><tt><tt>                               </tt></tt></tt></tt>
         
<h1><tt><tt><tt><tt>Methods</tt></tt></tt></tt></h1>
         <tt><tt><tt><tt>                              </tt></tt></tt></tt> 
        
<p><tt><tt><tt><tt>Gene density is defined as percentage of bases covered
    either by Ensembl        genes, or human mRNA best blat alignments in
the    UCSC browser database.        </tt></tt></tt></tt></p>
         <tt><tt><tt><tt>                               </tt></tt></tt></tt>
         
<p><tt><tt><tt><tt>Nontranscribed transcription was measured by a fairly
elaborate process.         125 base non-overlapping subwindows were taken
inside of the 500,000    base    windows.   Subwindows with less than 75%
of their bases in a mouse    alignment    were thrown out.   For the remaining
subwindows the percentage    with at least   80% base identity is used as
the conservation score.  To   get the nontranscribed   conservation score
the mouse alignments in regions   corresponding to Ensembl   genes, all genbank
mRNA blastz alignments, Fgenesh++   gene predictions, twinScan   gene predictions,
 spliced EST alignments, and  repeats  were thrown out.</tt></tt></tt></tt></p>

         <tt><tt><tt><tt>                  <br>
                       <br>
                      <br>
                     <br>
                     <br>
                   <br>
                <br>
               <br>

                   </tt></tt></tt></tt>  <br>
         <br>
  <br>
 <br>
</body>
</html>

