<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html
	PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US"><head><title>Random Regions for 1% Project</title>
</head><body vlink="#885500" text="#002244" bgcolor="#ffeedd" link="#aa0000">
<h1>Stratified Random Picks</h1>

<p>
To make the stratified picks the genome is divided into the top 20%,
middle 30%, and bottom 50% along two axis - gene density and
nontranscribed conservation. Then three random picks are taken from each
strata, and a fourth pick in the strata that are underrepresented in the
manual picks. One additional backup pick is made in each strata in case
there is an unforeseen technical problem with a region. The backup pick
is last entry in each table section.</p>

<TABLE BORDER=1><TR><TH colspan=4>consNonTx 0% - 50%, gene 0% - 50% (1 manual)</TH></TR>
<TR><TH>June 2002</TH><TH> November 2002</TH><TH> April 2003 </TH><TH> stats </TH></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr13:28500001-29000000">
chr13:28500001-29000000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr13:24500016-25000015">
chr13:24500016-25000015 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr13:29450016-29950015">
chr13:29450016-29950015 </A> </TD>
<TD> consNonTx 2.8%, gene 0.5% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr2:51700001-52200000">
chr2:51700001-52200000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr2:51837455-52337454">
chr2:51837455-52337454 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr2:51616414-52116413">
chr2:51616414-52116413 </A> </TD>
<TD> consNonTx 3.8%, gene 0.0% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr4:119000001-119500000">
chr4:119000001-119500000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr4:118527386-119027385">
chr4:118527386-119027385 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr4:118639860-119139859">
chr4:118639860-119139859 </A> </TD>
<TD> consNonTx 3.9%, gene 0.0% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr10:54300001-54800000">
chr10:54300001-54800000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr10:54489120-54989119">
chr10:54489120-54989119 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr10:55376221-55876220">
chr10:55376221-55876220 </A> </TD>
<TD> consNonTx 2.8%, gene 1.2% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr5:15900001-16400000">
chr5:15900001-16400000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr5:16187472-16687471">
chr5:16187472-16687471 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr5:15942554-16442553">
chr5:15942554-16442553 </A> </TD>
<TD> consNonTx 5.1%, gene 1.7% </TD></TR>
</TABLE>
<TABLE BORDER=1><TR><TH colspan=4>consNonTx 0% - 50%, gene 50% - 80% (4 manual)</TH></TR>
<TR><TH>June 2002</TH><TH> November 2002</TH><TH> April 2003 </TH><TH> stats </TH></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr2:115500001-116000000">
chr2:115500001-116000000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr2:116215329-116715328">
chr2:116215329-116715328 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr2:118201388-118701387">
chr2:118201388-118701387 </A> </TD>
<TD> consNonTx 6.2%, gene 2.3% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr18:61100001-61600000">
chr18:61100001-61600000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr18:61234622-61734621">
chr18:61234622-61734621 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr18:61046295-61546294">
chr18:61046295-61546294 </A> </TD>
<TD> consNonTx 3.4%, gene 3.4% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr12:40500001-41000000">
chr12:40500001-41000000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr12:40239443-40739442">
chr12:40239443-40739442 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr12:40056957-40556956">
chr12:40056957-40556956 </A> </TD>
<TD> consNonTx 1.7%, gene 3.1% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr2:196700001-197200000">
chr2:196700001-197200000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr2:197214044-197714043">
chr2:197214044-197714043 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr2:198465410-198965409">
chr2:198465410-198965409 </A> </TD>
<TD> consNonTx 5.4%, gene 3.3% </TD></TR>
</TABLE>
<TABLE BORDER=1><TR><TH colspan=4>consNonTx 0% - 50%, gene 80% - 100% (11 manual)</TH></TR>
<TR><TH>June 2002</TH><TH> November 2002</TH><TH> April 2003 </TH><TH> stats </TH></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr2:232500001-233000000">
chr2:232500001-233000000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr2:233173598-233673597">
chr2:233173598-233673597 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr2:234508167-235008166">
chr2:234508167-235008166 </A> </TD>
<TD> consNonTx 1.3%, gene 4.6% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr13:111900001-112400000">
chr13:111900001-112400000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr13:107927238-108427237">
chr13:107927238-108427237 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr13:112376702-112876701">
chr13:112376702-112876701 </A> </TD>
<TD> consNonTx 1.1%, gene 5.5% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr21:36900001-37400000">
chr21:36900001-37400000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr21:36983033-37483032">
chr21:36983033-37483032 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr21:39242992-39742991">
chr21:39242992-39742991 </A> </TD>
<TD> consNonTx 2.3%, gene 5.2% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr4:47800001-48300000">
chr4:47800001-48300000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr4:48032776-48532775">
chr4:48032776-48532775 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr4:47573442-48073441">
chr4:47573442-48073441 </A> </TD>
<TD> consNonTx 1.9%, gene 4.4% </TD></TR>
</TABLE>
<TABLE BORDER=1><TR><TH colspan=4>consNonTx 50% - 80%, gene 0% - 50% (2 manual)</TH></TR>
<TR><TH>June 2002</TH><TH> November 2002</TH><TH> April 2003 </TH><TH> stats </TH></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr16:25300001-25800000">
chr16:25300001-25800000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr16:25969826-26469825">
chr16:25969826-26469825 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr16:25800363-26300362">
chr16:25800363-26300362 </A> </TD>
<TD> consNonTx 9.7%, gene 0.5% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr5:141800001-142300000">
chr5:141800001-142300000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr5:142482586-142982585">
chr5:142482586-142982585 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr5:141883116-142383115">
chr5:141883116-142383115 </A> </TD>
<TD> consNonTx 6.7%, gene 1.7% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr18:25400001-25900000">
chr18:25400001-25900000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr18:25196197-25696196">
chr18:25196197-25696196 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr18:25353226-25853225">
chr18:25353226-25853225 </A> </TD>
<TD> consNonTx 7.4%, gene 0.9% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr4:124800001-125300000">
chr4:124800001-125300000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr4:124166677-124666676">
chr4:124166677-124666676 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr4:124280349-124780348">
chr4:124280349-124780348 </A> </TD>
<TD> consNonTx 6.3%, gene 0.9% </TD></TR>
</TABLE>
<TABLE BORDER=1><TR><TH colspan=4>consNonTx 50% - 80%, gene 50% - 80% (4 manual)</TH></TR>
<TR><TH>June 2002</TH><TH> November 2002</TH><TH> April 2003 </TH><TH> stats </TH></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr5:56000001-56500000">
chr5:56000001-56500000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr5:57392856-57892855">
chr5:57392856-57892855 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr5:55805775-56305774">
chr5:55805775-56305774 </A> </TD>
<TD> consNonTx 7.9%, gene 2.2% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr6:131800001-132300000">
chr6:131800001-132300000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr6:132023965-132523964">
chr6:132023965-132523964 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr6:132111977-132611976">
chr6:132111977-132611976 </A> </TD>
<TD> consNonTx 6.9%, gene 2.1% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr6:73700001-74200000">
chr6:73700001-74200000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr6:73699933-74199932">
chr6:73699933-74199932 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr6:73683390-74183389">
chr6:73683390-74183389 </A> </TD>
<TD> consNonTx 6.4%, gene 3.6% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr4:53700001-54200000">
chr4:53700001-54200000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr4:53859184-54359183">
chr4:53859184-54359183 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr4:53728692-54228691">
chr4:53728692-54228691 </A> </TD>
<TD> consNonTx 9.0%, gene 2.1% </TD></TR>
</TABLE>
<TABLE BORDER=1><TR><TH colspan=4>consNonTx 50% - 80%, gene 80% - 100% (3 manual)</TH></TR>
<TR><TH>June 2002</TH><TH> November 2002</TH><TH> April 2003 </TH><TH> stats </TH></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr1:149000001-149500000">
chr1:149000001-149500000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr1:146905332-147405331">
chr1:146905332-147405331 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr1:147933156-148433155">
chr1:147933156-148433155 </A> </TD>
<TD> consNonTx 10.2%, gene 8.4% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr9:122800001-123300000">
chr9:122800001-123300000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr9:123331831-123831830">
chr9:123331831-123831830 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr9:125138972-125638971">
chr9:125138972-125638971 </A> </TD>
<TD> consNonTx 8.3%, gene 5.9% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr15:39100001-39600000">
chr15:39100001-39600000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr15:36628619-37128618">
chr15:36628619-37128618 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr15:41311935-41810934">
chr15:41311935-41810934 </A> <BR> (manually placed) </TD>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr17:33400001-33900000">
chr17:33400001-33900000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr17:35665792-36165791">
chr17:35665792-36165791 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr17:33478638-33978637">
chr17:33478638-33978637 </A> </TD>
<TD> consNonTx 7.7%, gene 6.1% </TD></TR>
</TABLE>
<TABLE BORDER=1><TR><TH colspan=4>consNonTx 80% - 100%, gene 0% - 50% (3 manual)</TH></TR>
<TR><TH>June 2002</TH><TH> November 2002</TH><TH> April 2003 </TH><TH> stats </TH></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr14:51200001-51700000">
chr14:51200001-51700000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr14:47673341-48173340">
chr14:47673341-48173340 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr14:51867364-52367363">
chr14:51867364-52367363 </A> </TD>
<TD> consNonTx 14.9%, gene 0.1% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr11:133100001-133600000">
chr11:133100001-133600000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr11:132612235-133112234">
chr11:132612235-133112234 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr11:131133068-131633067">
chr11:131133068-131633067 </A> </TD>
<TD> consNonTx 13.5%, gene 0.3% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr16:52600001-53100000">
chr16:52600001-53100000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr16:62362206-62862205">
chr16:62362206-62862205 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr16:62010885-62510884">
chr16:62010885-62510884 </A> </TD>
<TD> consNonTx 15.4%, gene 0.0% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chrX:41900001-42400000">
chrX:41900001-42400000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chrX:42149253-42649252">
chrX:42149253-42649252 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chrX:42714870-43214869">
chrX:42714870-43214869 </A> </TD>
<TD> consNonTx 13.4%, gene 0.7% </TD></TR>
</TABLE>
<TABLE BORDER=1><TR><TH colspan=4>consNonTx 80% - 100%, gene 50% - 80% (1 manual)</TH></TR>
<TR><TH>June 2002</TH><TH> November 2002</TH><TH> April 2003 </TH><TH> stats </TH></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr8:117800001-118300000">
chr8:117800001-118300000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr8:118874200-119374199">
chr8:118874200-119374199 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr8:118481838-118981837">
chr8:118481838-118981837 </A> </TD>
<TD> consNonTx 11.4%, gene 3.2% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr14:96900001-97400000">
chr14:96900001-97400000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr14:93204045-93704044">
chr14:93204045-93704044 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr14:97378512-97878511">
chr14:97378512-97878511 </A> </TD>
<TD> consNonTx 15.9%, gene 2.9% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chrX:117500001-118000000">
chrX:117500001-118000000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chrX:119675382-120175381">
chrX:119675382-120175381 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chrX:120734591-121234590">
chrX:120734591-121234590 </A> </TD>
<TD> consNonTx 10.7%, gene 2.0% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr6:108100001-108600000">
chr6:108100001-108600000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr6:108287568-108787567">
chr6:108287568-108787567 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr6:108264834-108764833">
chr6:108264834-108764833 </A> </TD>
<TD> consNonTx 18.6%, gene 2.3% </TD></TR>
</TABLE>
<TABLE BORDER=1><TR><TH colspan=4>consNonTx 80% - 100%, gene 80% - 100% (1 manual)</TH></TR>
<TR><TH>June 2002</TH><TH> November 2002</TH><TH> April 2003 </TH><TH> stats </TH></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr2:218300001-218800000">
chr2:218300001-218800000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr2:218998720-219498719">
chr2:218998720-219498719 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr2:220241365-220741364">
chr2:220241365-220741364 </A> </TD>
<TD> consNonTx 13.3%, gene 9.1% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr11:66700001-67200000">
chr11:66700001-67200000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr11:65865884-66365883">
chr11:65865884-66365883 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr11:64434365-64934364">
chr11:64434365-64934364 </A> </TD>
<TD> consNonTx 13.4%, gene 9.0% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr20:33600001-34100000">
chr20:33600001-34100000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr20:33559944-34059943">
chr20:33559944-34059943 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr20:34509944-35009943">
chr20:34509944-35009943 </A> </TD>
<TD> consNonTx 11.5%, gene 9.2% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr6:41300001-41800000">
chr6:41300001-41800000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr6:41294331-41794330">
chr6:41294331-41794330 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr6:41299332-41799331">
chr6:41299332-41799331 </A> </TD>
<TD> consNonTx 15.2%, gene 4.8% </TD></TR>
<TR><TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr9:124300001-124800000">
chr9:124300001-124800000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr9:124831831-125331830">
chr9:124831831-125331830 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr9:126638972-127138971">
chr9:126638972-127138971 </A> </TD>
<TD> consNonTx 11.4%, gene 5.4% </TD></TR>
</TABLE>
<H1>Stratification of Manual Picks</H1>
<p>Here is the noncoding conservation and gene density
of non-overlapping 500 kb regions in the manual picks.   The
boundaries between strata are:</p>
<pre><tt>          low 50%  middle 30%  high 20%<br>
         ------------------------------<br>
gene      0.0-1.9%  1.9-4.2%   4.2-100%<br>
consNotTx 0.0-6.3%  6.3-10.6% 10.6-100%<br></tt></pre>
<TABLE BORDER=1><TR><TH>June 2002</TH><TH> November 2002 </TH><TH> April 2003 </TH><TH> Interest </TH> </TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:114288355-116165780">
chr7:114288355-116165780 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:114288155-116165580">
chr7:114288155-116165580 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr7:115351222-117228647">
chr7:115351222-117228647 </A> </TD>
<TD> CFTR 3 </TD><TD> &nbsp; </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr5:130778557-131778556">
chr5:130778557-131778556 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr5:131703638-132703637">
chr5:131703638-132703637 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr5:131287278-132287277">
chr5:131287278-132287277 </A> </TD>
<TD> Interleukin_Cluster 3 </TD><TD> &nbsp; </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr11:118810001-119310000">
chr11:118810001-119310000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr11:117969240-118469239">
chr11:117969240-118469239 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr11:116491019-116991018">
chr11:116491019-116991018 </A> </TD>
<TD> Apo_Cluster 3 </TD><TD> &nbsp; </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr22:28500001-30200000">
chr22:28500001-30200000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr22:28500001-30200000">
chr22:28500001-30200000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr22:30128508-31828507">
chr22:30128508-31828507 </A> </TD>
<TD> Chr22 3 </TD><TD> &nbsp; </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr21:30323762-32019746">
chr21:30323762-32019746 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr21:30406794-32102778">
chr21:30406794-32102778 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr21:32666762-34362746">
chr21:32666762-34362746 </A> </TD>
<TD> Chr21 3 </TD><TD> &nbsp; </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chrX:147250001-148500000">
chrX:147250001-148500000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chrX:149572309-150846234">
chrX:149572309-150846234 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chrX:150700001-151950000">
chrX:150700001-151950000 </A> <BR> (manually placed) </TD>
<TD> ChrX 3 </TD><TD> &nbsp; </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr19:55200001-56200000">
chr19:55200001-56200000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr19:54724484-55728861">
chr19:54724484-55728861 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr19:59007794-60008669">
chr19:59007794-60008669 </A> <BR> (manually placed) </TD>
<TD> Chr19 3 </TD><TD> &nbsp; </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr16:79138-579137">
chr16:79138-579137 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr16:10001-510000">
chr16:10001-510000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr16:1-500000">
chr16:1-500000 </A> <BR> (manually placed) </TD>
<TD> Alpha_Globin 3 </TD><TD> &nbsp; </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr11:5550000-6549999">
chr11:5550000-6549999 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr11:5076527-6078118">
chr11:5076527-6078118 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr11:4733457-5735048">
chr11:4733457-5735048 </A> </TD>
<TD> Beta_Globin 3 </TD><TD> &nbsp; </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:26600001-27100000">
chr7:26600001-27100000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:26599801-27099800">
chr7:26599801-27099800 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr7:26665793-27165792">
chr7:26665793-27165792 </A> </TD>
<TD> HOXA_cluster 3 </TD><TD> &nbsp; </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr11:300001-900000">
chr11:300001-900000 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr11:1941933-2547980">
chr11:1941933-2547980 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr11:1702703-2308750">
chr11:1702703-2308750 </A> <BR> (manually placed) </TD>
<TD> IGF2/H19 4 </TD><TD>PROBLEMATIC REGION - SUBSTANTIALLY REARRANGED BETWEEN JUNE AND NOVEMBER BUILDS]</TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:112410791-113410790">
chr7:112410791-113410790 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:112410791-113410790">
chr7:112410791-113410790 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr7:113473834-114473833">
chr7:113473834-114473833 </A> </TD>
<TD> FOXP2 3 </TD><TD> &nbsp; </TD></TR>
</TR></TABLE>
<H1>Semi-Manual Picks</H1>
<p>Here's the stratification of the other zoo-seq regions. I recommend
picking 7q21.13 and 7q31.33 to round things out.</p>
<TABLE BORDER=1><TR><TH>June 2002</TH><TH> November 2002 </TH><TH> April 2003 </TH><TH> Chrom band </TH></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:88319137-89433560">
chr7:88319137-89433560 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:88318937-89433360">
chr7:88318937-89433360 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr7:89381916-90496339">
chr7:89381916-90496339 </A> </TD>
<TD> 7q21.13 3 </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:91589227-92559635">
chr7:91589227-92559635 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:91589027-92559435">
chr7:91589027-92559435 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr7:92652026-93622434">
chr7:92652026-93622434 </A> </TD>
<TD> 7q21.3 3 </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:93650712-94868826">
chr7:93650712-94868826 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:93650512-94868626">
chr7:93650512-94868626 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr7:94713518-95931632">
chr7:94713518-95931632 </A> </TD>
<TD> 7q21.3 3 </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:124556444-125719632">
chr7:124556444-125719632 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:124556244-125719432">
chr7:124556244-125719432 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr7:125619343-126782531">
chr7:125619343-126782531 </A> </TD>
<TD> 7q31.33 3 </TD></TR>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg12&amp;position=chr7:126427707-127330661">
chr7:126427707-127330661 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg13&amp;position=chr7:126427507-127330461">
chr7:126427507-127330461 </A> </TD>
<TD>
<A href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg15&amp;position=chr7:127490599-128393553">
chr7:127490599-128393553 </A> </TD>
<TD> 7q32.1 3 </TD></TR>
</TR></TABLE>
<H1> Methods </H1>
<p>Gene density is defined as percentage of bases covered either by Ensembl
genes, or human mRNA best blat alignments in the UCSC browser database.</p>

<p>Nontranscribed transcription was measured by a fairly elaborate process.
125 base non-overlapping subwindows were taken inside of the 500,000
base windows. Subwindows with less than 75% of their bases in a mouse
alignment were thrown out. For the remaining subwindows the percentage
with at least 80% base identity is used as the conservation score. To
get the nontranscribed conservation score the mouse alignments in
regions corresponding to Ensembl genes, all genbank mRNA blastz
alignments, Fgenesh++ gene predictions, twinScan gene predictions,
spliced EST alignments, and repeats were thrown out.</p>
<HR><CENTER> See also:
<A HREF="picks_2003-03-03.html"> Previous version of this WEB page data </A>
</CENTER>
<HR><CENTER>
Please report any problems on this page to Hiram Clawson at:
<A HREF="mailto:&#104;&#105;&#114;&#97;&#109;&#64;&#115;&#111;&#101;.&#117;&#99;sc.e&#100;u?subject=Request&#32;on&#32;picks.html">
&#104;&#105;&#114;&#97;&#109;&#64;&#115;&#111;&#101;.&#117;&#99;sc.e&#100;u</A>
<BR><!--<A HREF="http://www.cbse.ucsc.edu/Genome/">-->
The Human Genome Project at UCSC <!--</A>--><BR>
</CENTER><HR><CENTER>This page created: 2003-05-29
</CENTER>
</body></html>
