<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html><head>
<meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
<!-- -->
</head><body><p><strong><font size="4">Small RNA Transfrags from CSHL</font></strong></p>
<p>Small RNA reads from Cold Spring Harbor Lab (CSHL) were assembled 
into transfrags by merging overlapping reads.  In order to minimize 
ambiguity from reads that have the potential to map to multiple genomic 
loci, only the uniquely mapping reads were used to generate transfrags.  
The BED6+ format files are based on, but not generated directly by, the 
"intervals-to-contigs" Galaxy tool written by Assaf Gordon (<a href="mailto:gordon@cshl.edu">gordon@cshl.edu</a>) in the Hannon lab at CSHL. Below is a description of the columns in this format, and how each column is calculated.</p>
<p><strong>Output Columns</strong></p>
<p>(Bed-style transfrag information)</p>
<ul class="simple">
<li><ol class="first arabic">
<li>chromosome</li>
</ol>
</li>
<li><ol class="first arabic" start="2">
<li>transfrag's start coordinate</li>
</ol>
</li>
<li><ol class="first arabic" start="3">
<li>transfrag's end coordinate</li>
</ol>
</li>
<li><ol class="first arabic" start="4">
<li>Contig's name.  The numeral in the name indicates rank in terms of abundance within this dataset.</li>
</ol>
</li>
<li><ol class="first arabic" start="5">
<li>Score (0 to 1000).  Scores are calculated thusly: 1000*[# reads in transfrag]/[# reads in most abundant transfrag in this dataset]</li>
</ol>
</li>
<li><ol class="first arabic" start="6">
<li>Strand (orientation, + or -)</li>
</ol>
</li>
</ul>
<p>(Additional Sequences Information)</p>
<ul class="simple">
<li><ol class="first arabic" start="7">
<li>transfrag's length (number of covered bases = end - start)</li>
</ol>
</li>
<li><ol class="first arabic" start="8">
<li>number of unique sequences in this transfrag</li>
</ol>
</li>
<li><ol class="first arabic" start="9">
<li>total reads count in this transfrag</li>
</ol>
</li>
<li><ol class="first arabic" start="10">
<li>minimum sequence-count value</li>
</ol>
</li>
<li><ol class="first arabic" start="11">
<li>maximum sequence-count value</li>
</ol>
</li>
<li><ol class="first arabic" start="12">
<li>average seqeunce-count value</li>
</ol>
</li>
<li><ol class="first arabic" start="13">
<li>first-quartile sequence-count value</li>
</ol>
</li>
<li><ol class="first arabic" start="14">
<li>median sequence-count value</li>
</ol>
</li>
<li><ol class="first arabic" start="15">
<li>third-quartile sequence-count value</li>
</ol>
</li>
</ul>
<p>(Additional Reads Information)</p>
<ul class="simple">
<li><ol class="first arabic" start="16">
<li>minimum reads-count value</li>
</ol>
</li>
<li><ol class="first arabic" start="17">
<li>maximum reads-count value</li>
</ol>
</li>
<li><ol class="first arabic" start="18">
<li>average seqeunce-count value</li>
</ol>
</li>
<li><ol class="first arabic" start="19">
<li>first-quartile reads-count value</li>
</ol>
</li>
<li><ol class="first arabic" start="20">
<li>median reads-count value</li>
</ol>
</li>
<li><ol class="first arabic" start="21">
<li>third-quartile reads-count value</li>
</ol>
</li>
</ul>
<p>(Additional Intervals Information)</p>
<ul class="simple">
<li><ol class="first arabic" start="22">
<li>number of regions in this transfrag (each region has different value 
for sequence-count and reads-count)</li>
</ol>
</li>
<li><ol class="first arabic" start="23">
<li>starting coordinates of significant regions in this transfrag (see 
example below)</li>
</ol>
</li>
<li><ol class="first arabic" start="24">
<li>length (in bases) of each significant regions</li>
</ol>
</li>
<li><ol class="first arabic" start="25">
<li>sequence-count for each significant region</li>
</ol>
</li>
<li><ol class="first arabic" start="26">
<li>reads-count for each significant region</li>
</ol>
</li>
<li><ol class="first arabic" start="27">
<li>Integrated reads-count sum (inner-product of columns 22 and 24)</li>
</ol>
</li>
</ul>
<hr class="docutils">
<p><strong>Concrete Example</strong></p>
<p>Assume the following intervals over an imaginary chromosome <em>chr1</em>:</p>
<pre class="literal-block">chr1        100     132     4
chr1        110     142     3
chr1        130     160     7
chr1        170     201     3
chr1        190     225     1
</pre>
<p>Plotting these intervals:</p>
<img alt="./static/images/intervals_to_transfrags.png" src="smallRNA_transfrags_description_files/intervals_to_transfrags.png">
<p>These intervals cover two transfraguous regions (marked in red): 100-160
 and 170-225.</p>
<p>The output file will contain two lines (one for each transfrag):</p>
<pre class="literal-block">chr1      100     160     transfrag-1     1000     +     60      3       14      1       3       1.6     1       2       2       4       14      7.35    7       7       7       5       100,110,130,133,143     10,20,3,10,17   1,2,3,2,1       4,7,14,10,7     441
chr1      170     225     transfrag-2     286      +     55      2       4       1       2       1.21818 1       1       1       1       4       2.38182 1       3       3       3       170,190,202             20,12,23        1,2,1           3,4,1   131
</pre>
<p>The rest of the explanation will focus on the first transfrag only:</p>
<p>(transfrag information):</p>
<ul class="simple">
<li>transfrag is on chromosome <em>chr1</em> (column 1)</li>
<li>transfrag has <em>sense</em> orientation (column 6) - assumed so 
beacause no orientation information was found.</li>
<li>transfrag starts at coordinate 100 (column 2)</li>
<li>transfrag ends at coordinate 160 (column 3)</li>
<li>transfrag's name is transfrag-1 (column 4).  It is the most abundant transfrag in the sample, hence the rank score of 1.</li>
<li>transfrag has a score of 1000 (column 5) - it is the most abundant 
congtig on this chromosome.  The second transfrag would have a score of 
1000*(4/14) = 286.</li>
<li>transfrag has <em>sense</em> orientation (column 6) - assumed so 
beacause no orientation information was found.</li>
<li>transfrag covers 60 bases (column 7)</li>
<li>transfrag has 3 sequences (column 8)</li>
<li>transfrag has 14 reads (column 9)</li>
</ul>
<p>(sequence-count information):</p>
<ul class="simple">
<li>minimum sequence-count is 1 (only one interval covers coordinates 
100 to 110) (column 10)</li>
<li>maximum sequence-count is 3 (three intervals are covering 
coordinates 130 to 132) (column 11)</li>
<li>average sequence-count value is 1.6 ( 60 bases are covered, with 
coverage sum = 10x1 + 20x2 + 3x3 + 10x2 + 17x1 = 96. 96/60=1.6 ) (column
 12)</li>
<li>first-quartile sequence-count value is 1 ( There are 27 bases 
covered with value 1, 30 bases covered with value 2, and three bases 
covered with value 3) (column 13)</li>
<li>median sequence-count value is 2 (column 14)</li>
<li>third-quartile sequence-count value is 2 ( column 15 )</li>
</ul>
<p>(reads-count information):</p>
<ul class="simple">
<li>minimum reads-count is 4 (coordinates 100 to 110 are covered by the 
lowest number of reads = 4) (column 16)</li>
<li>maximum reads-count is 14 (three intervals, whose reads-count sum is
 14, are covering coordinates 130 to 132) (column 17)</li>
<li>average reads-count value is 7.35 ( 60 bases are covered, with 
coverage sum = 10x4 + 20x7 + 3x14 + 10x10 + 17x7 = 441. 441/60=7.35 ) 
(column 18)</li>
<li>first-quartile reads-count value is 7 ( 10 bases coverged with 4, 37
 covered with 7, 10 covered with 10, 3 covered with 14 ) (column 19)</li>
<li>median reads-count value is 7 (column 20)</li>
<li>third-quartile reads-count value is 7 ( column 21 )</li>
</ul>
<p>(significant regions information):</p>
<ul class="simple">
<li>This transfrag has five significant regions (column 22).</li>
<li><em>Significant</em> coordinates are ones in which the 
sequence-count and reads-count change. In this examples, the coordinates
 are 100,110,130,133,143. Look at the plot to better understand how 
these coodinates are determined. (column 23).</li>
<li>Each significant region covers a varied number of bases. Example: 
the region which starts at 100 covers 10 bases. the region which starts 
at 110 covers 20 bases. the region which starts at 130 covers 3 bases, 
etc. (column 24)</li>
<li>Sequence-Count value for each significant region . Example: the 
region which starts at 100 has sequence-count=1. (column 25)</li>
<li>Reads-Count value for each significant region . Example: the region 
which starts at 100 has reads-count=4. (column 26)</li>
<li>Integrated reads-count sum - (Inner product of column 22 and column 
26) = 10*4 + 20*7 + 3*14 + 10*10 + 17*7 = 441</li>
</ul>


                    
        
<script type="text/javascript">
   $( function() {
       $( 'li > ul' ).each( function( i ) {
           if ( $( this )[0].className == 'toolParameterExpandableCollapsable' )
           {
               var parent_li = $( this ).parent( 'li' );
               var sub_ul = $( this ).remove();
               parent_li.find( 'span' ).wrapInner( '<a/>' ).find( 'a' ).click( function() {
                 sub_ul.toggle();
                 $( this )[0].innerHTML = ( sub_ul[0].style.display=='none' ) ? '[+]' : '[-]';
               });
               parent_li.append( sub_ul );
           }
       });
       $( 'ul ul' ).each( function(i) {
           if ( $( this )[0].className == 'toolParameterExpandableCollapsable' && this.attributes.getNamedItem( 'default_state' ).value == 'collapsed' )
           {
               $( this ).hide();
           }
       });
   });

$( function() {
    $("div.checkUncheckAllPlaceholder").each( function( i ) {
        $( this )[0].innerHTML = '<a class="action-button" onclick="checkUncheckAll( \'' + this.attributes.getNamedItem( 'checkbox_name' ).value + '\', 1 );"><span>Select All</span></a> <a class="action-button" onclick="checkUncheckAll( \'' + this.attributes.getNamedItem( 'checkbox_name' ).value + '\', 0 );"><span>Unselect All</span></a>';
    });
});

</script>

</body></html>

