<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=201>
  <TR><TD ALIGN=RIGHT><A HREF="http://www.genome.gov/11510687" TARGET=_blank>
    <IMG SRC="../images/Monodelphis_domestica.jpg" WIDTH=186 HEIGHT=130 ALT="Opossum"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1> 
      <em>Monodelphis domestica</em><BR>
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of Paul Samollow,<BR>
      Southwest Foundation for Biomedical Research,<BR>San Antonio.<BR>
      (NHGRI <A HREF="http://www.genome.gov/dmd/index.cfm?node=Photos/Animals/Opossum%20(short_tailed)" TARGET=_blank>Press 
      Photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The June 2005 opossum (<em>Monodelphis domestica</em>) V3 preliminary assembly 
was produced by <A HREF="http://www.broad.mit.edu/mammals/opossum/" TARGET=_blank>The 
Broad Institute</A>.</P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by a scaffold coordinate range, an mRNA 
or EST, or keywords from the GenBank description of an mRNA.
The following list provides examples of various types of position queries 
for the opossum genome. See the 
<A HREF="../goldenPath/help/hgTracksHelp.html" 
TARGET=_blank>User's Guide</A> for more information. 
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TD VALIGN=Top NOWRAP><B>Request:</B><BR></TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top><B>Genome Browser Response:</B><BR></TD>
    </TR>
	
    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>scaffold_1802</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays all of scaffold #1802</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>scaffold_6606:63,250-84,705</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays bases 63250 through 84705 of scaffold #6606</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>scaffold_1802:100000+2000</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays a region of scaffold_1802 that spans 2000 bases, starting with position 100000 </TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>AF049761</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region of mRNA with GenBank accession number AF049761</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>CLOCK</TD>
        <TD WIDTH=14></TD>
        <TD>Displays region of genome showing tBLASTn alignment of peptides from human CLOCK gene</TD>
    </TR>
<!--
    <TR><TD VALIGN=Top NOWRAP>AA205474</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Displays region of EST with GenBank accession AA205474 in BRCA1 cancer gene on chr 17</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>PRNP</TD>
        <TD WIDTH=14></TD>
        <TD>Displays region of genome with HUGO Gene Nomenclature Committee identifier PRNP</TD>
    </TR>
        <TD VALIGN=Top NOWRAP>NM_017414</TD>
        <TD><BR></TD>        
  	<TD VALIGN=Top>Displays the region of genome with RefSeq identifier NM_017414</TD>
    </TR>
-->
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>immunoglobulin</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Lists immunoglobulin regions</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>immunoglobulin light chain kappa</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Lists only immunoglobulin light chain kappa mRNAs</TD> 
    </TR>
    <TR><TD VALIGN=Top NOWRAP>miller</TD> 
        <TD WIDTH=14></TD> 
        <TD VALIGN=Top>Lists mRNAs deposited by scientists named Miller</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Miska,K.B.</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Lists mRNAs deposited by co-author K.B. Miska</TD>
    </TR> 
    <TR><TD VALIGN=Top><br></TD></TR>
<!--
    <TR><TD COLSPAN="3"> 
    </TD></TR>
-->
</TABLE>
    Use this last format for author queries. Although GenBank requires the 
    search format <em>Miska KB</em>, internally it uses the format 
    <em>Miska,K.B.</em>

<H3>Assembly Details</H3>
<P>
The Broad Institute reports the following statistics on the monDom2 assembly:
<UL>
<LI> Placed reads: 33,507,069
<LI> Number of contigs: 65,946
<LI> Total bases in contigs: 3,496,445,653
<LI> Number of scaffolds: 5,077
<LI> Total bases in scaffolds: 3,575,239,585
<LI> Number of gap bases in scaffolds between contigs: 78,793,932
<LI> GC/AT ratio: 1,322,125,364/2,174,320,289 = 0.6081
<LI> Scaffold names: scaffold_0 - scaffold_5076
<LI> Largest scaffold: scaffold_0 at 199,827,570 bases
<LI> Smallest scaffold: scaffold_5076 at 2,002 bases
</UL>
</P>
<P>
The monDom2 sequence and annotation data can be downloaded from the 
UCSC Genome Browser 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/monDom2/">FTP server</A>
or 
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#opossum">downloads</A> 
page. These data have <A HREF="../goldenPath/credits.html#opossum_use">specific 
conditions for use</A>.
</P>
<P>
Many thanks to The Broad Institute for the opossum assembly data.
The monDom2 annotation 
tracks were generated by UCSC and collaborators worldwide. See the 
<A HREF="../goldenPath/credits.html#opossum_credits">credits</A> page
for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
