<html>
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
    <title>ENCODE Data Matrix</title>
    <link rel="stylesheet" href="/style/HGStyle.css" type="text/css">
    <link rel="stylesheet" href="/style/encodeProject.css" type="text/css">
    <link rel="stylesheet" href="/style/encodeDataMatrix.css" type="text/css">
    <script src="/js/jquery.js" type="text/javascript"></script>
    <script src="/js/jquery.floatheader.js" type="text/javascript"></script>
    <script src="/js/jquery.rotateTableCellContent.js" type="text/javascript"></script>
    <script src="/js/jquery.cookie.js" type="text/javascript"></script>
    <script src="/js/utils.js" type="text/javascript"></script>
    <script src="/js/encodeProject.js" type="text/javascript"></script>
    <script src="/js/encodeMatrix.js" type="text/javascript"></script>
</head>

<body>
    <!-- Standard ENCODE top menu bar -->
    <!--#include virtual="/inc/encodeProject.topbar.html"-->

    <!-- Standard ENCODE page header -->
    <div class="encodeHeader">
        <a href="/ENCODE/index.html"><img src="/images/gbLogoOnly.png" alt="ENCODE Data at UCSC" title="ENCODE Data at UCSC"></a>
        <span id="pageHeader" class="txt">ENCODE Data Matrix</span>
        <span id="assemblyLabel"></span>
        <span class='encodeHeaderDates'>2007 - 2012</span>
        <span id="spinner"></span>
        <span id="expMatrixHelpIcon" title="Click for help" class="helpLauncher">Overview...</span>
    </div>
    <div id="expMatrixHelpText" class="helpText wrapper">
    <div class="bar"><h4 class="title">About the ENCODE Experiment Matrix</h4></div>
    <div class="content">
<p>
The ENCODE Experiment Matrix is a set of web pages that visually summarize the types of data
produced by the ENCODE project during the first production phase (September 2007 to July 2012).
<em>For ENCODE data generated after 2012, see the ENCODE Portal</em>, 
<a target=_blank href="https://www.encodeproject.org"> encodeproject.org</a>.</p>
<p>
The data summarized here is all hosted at UCSC as browser tracks and downloadable files.
The grid on the main Experiment Matrix page shows the number of experiments for each cell 
type/assay pairing.
The 
<a href='javascript:void(window.open(encodeMatrix.pageFor("chipMatrix", encodeMatrix_organism)));'>
<strong>ChIP-seq Experiment Matrix</strong></a>
page provides a more detailed view of the chromatin immunoprecipitation experiment subset, showing
experiments by cell type and antibody target.
Cell types annotated as normal vs. cancer are marked with a colored dot.
The companion
<a href='javascript:void(window.open(encodeMatrix.pageFor("dataSummary", encodeMatrix_organism)));'>
<strong>Experiment Summary</strong></a>
page lists the number of experiments by assay type alone and may include annotations that are
cell-type independent (annotations on the reference genome). 
</p>

<p>
An ENCODE experiment is defined as a biochemical assay and follow-on data analyses performed on a
single cell type by a single lab.  Data from an experiment is typically displayed in multiple
browser tracks that offer different views of the data (e.g. enrichment signal graph, peak calls)
and is available for download in multiple file formats (e.g. sequence alignments in BAM format, 
signal graph in bigWig format) that support different analysis methods. 
Data for multiple replicates are included in a single experiment.</p>
<p>
To display or download ENCODE data, select either the <em>tracks</em> or <em>files</em> option 
in the
<img style='border: 1px solid gray;' height='17' width='164' src='/images/encode/dataMatrixSearchPanel.png'>
panel, then click the appropriate box in the matrix grid (or row in the summary tables). 
This will display a search window (via the UCSC <em>Track Search</em> or <em>File Search</em> 
tool) that lists the full set of browser tracks or downloadable files for the selected experiments.
This window also provides options to configure the browser view or refine the list of 
downloadable files.</p>
<p>
These pages are best viewed in Firefox, Chrome, or Safari browsers.</p>

    <span id='dataMatrixXIcon' title='Click to hide help' class="xIcon">&times;</span>
    </div><!-- end content -->
    </div><!-- end helpText -->
</div><!-- end wrapper -->

    <!-- a consider using caption tag, since colspan=0 doesn't work cross-browser -->
    <table id="matrixTable"> 
        <thead>
        <tr id="headerLabelRow"><td></td>
            <td id="elementHeaderLabel" class="axisType" colspan=100 title="Click to view information about all assays">
                <a target="cvWindow" href="/cgi-bin/hgEncodeVocab?ra=encode/cv.ra&type=dataType">Assays</a></td></tr>

        <tr id="columnHeaders">
            <!-- Radiobuttons to select Files or Tracks, created from lib -->
            <td>
            <div id="searchTypePanel" class="panel"></div>
            <div align='bottom' id="cellHeaderLabel" class="axisType" title="Click to view information about all cell types">
                <!-- TODO: configurable for mouse -->
                <a target="cvWindow" href="/cgi-bin/hgEncodeVocab?ra=encode/cv.ra&type=Cell+Line&organism=human">Cell Types</div>
            </td>
        </tr>
        </thead>

        <!-- Main content:  grid.  Script populates, based on data from the server -->
        <tbody></tbody>
    </table>

</body>
</html>
