<!-- Display image in righthand corner -->
<!--
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=185>
  <TR><TD ALIGN=RIGHT>
    <A HREF="*** link to sequencing center's project page"
    TARGET=_blank>
    <IMG SRC="../images/Brugia_malayi.jpg" WIDTH=*** HEIGHT=*** ALT="B. malayi"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Brugia malayi</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of ***
      (<A HREF="*** link to source" 
      TARGET=_blank>***source</A>)
    </FONT>
  </TD></TR>
</TABLE>
-->

<P>
The Sep. 2007 <em>Brugia malayi</em> draft assembly is produced by
TIGR and the Sanger Institute.

See also: <em>Draft genome of the filarial nematode parasite Brugia malayi.</em>,
<A HREF="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2613796/"
TARGET=_blank>Science, 2007 Sep 21; 317(5845):1756-60</A>

and the AAQA01 WGS
<A HREF="https://www.ncbi.nlm.nih.gov/Traces/wgs/?val=AAQA01"
TARGET=_blank>information.</A>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
B. malayi genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>Bmal_supercontig14973:1-100,000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first hundred thousand bases of supercontig Bmal_supercontig14973</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Bmal_supercontig14973:100,000+2,000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of supercontig Bmal_supercontig14973 that spans 2000 bases, starting with position 100,000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>DQ270213</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number DQ270213</TD>
    </TR>
    <TR><TD VALIGN=Top>AW257639</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		AW257639</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>colbert</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Colbert</TD>
    </TR>
    <TR><TD VALIGN=Top>Taguchi,T.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author T. Taguchi</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Taguchi T</em>, internally it
	uses the format <em>Taguchi,T.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
This draft assembly is composed of 27,211 supercontigs for a total size of
95,828,100 nucleotides.  The largest supercontig, Bmal_supercontig14972, is
6,534,162 nucleotides.  One half of the sequence, 47,914,050 bases,
is contained in 253 supercontigs, all of at least 37,841 bases (N50).
There are 59,379 sequences of gap in these supercontigs, for a total
of 6,592,564 nucleotides indicated in the sequence by &quot;N&quot;.
</P>


<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/bruMal1/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#bruMal">Downloads</A> page. 

The B. malayi browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#worm_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
