<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=315>
  <TR><TD ALIGN=RIGHT>
    <A HREF="https://www.hgsc.bcm.edu/content/bovine-genome-project"
    TARGET=_blank>
    <IMG SRC="../images/Bos_taurus.jpg" WIDTH=300 HEIGHT=188 ALT="Cow"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Bos Taurus</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
     Photo courtesy of Michael MacNeil, USDA<BR>
     (NHGRI <A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Cow&id=79109"
     TARGET=_blank>Press Photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The April 2009 <em>Bos Taurus</em> draft assembly 
(Btau_4.2) submitted by 
<A HREF="http://genomes.arc.georgetown.edu/drupal/bovine/?q=bovine_genome_consortium"
TARGET=_blank>Bovine Genome Sequencing Consortium</A>
was produced by the 
<A HREF="https://www.hgsc.bcm.edu/content/bovine-genome-project"
TARGET=_blank>Baylor College of Medicine Human Genome Sequencing Center</A>.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Cow genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>chr28</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of chromosome 28</TD>
    </TR>
    <TR><TD VALIGN=Top>chr28:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of chromosome 28</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr28:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 28 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>BC140604</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number BC140604</TD>
    </TR>
    <TR><TD VALIGN=Top>EE384456</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of EST with GenBank accession
		EE384456 on chr X</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene mRNA</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zhang</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zhang</TD>
    </TR>
    <TR><TD VALIGN=Top>Moore,S.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author S. Moore</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Moore S</em>, internally it
	uses the format <em>Moore,S.</em>.</TD>
    </TR>
</TABLE>

<HR>
<H3>Assembly details</H3>
<P>
The Btau_4.2 release substituted available high quality finished BAC sequences 
for the corresponding draft assembly scaffolds from the Btau_4.0 assembly.  The 
description of Btau_4.0 is given below:
</P>
<P>
The Btau_4.0 release was produced by the Atlas genome assembly system at
Baylor College of Medicine Human Genome Sequencing Center. The  sequencing 
strategy combined BAC shotgun reads with whole genome shotgun reads from 
small insert libraries as well as BAC end sequences. The assembly contains 
chromosomes 1-29 and X as well as 11869 scaffolds (named chrUn.004.*). 
The mitochondrial sequence (chrM) was obtained from Genbank
accession GI:60101824.
</P>
<P>
The Btau_4.0 assembly (7.1x coverage) was tested against available 
bovine sequence data sets 
(EST sequences and finished BAC sequences) for extent of coverage 
(completeness). When assembled contigs were tested, over 95% of the sequences 
in these data sets were found to be represented, indicating that the shotgun 
libraries used to sequence the genome were comprehensive. Of the 1.04 million 
EST sequences 95.0% were contained in the assembled contigs. Assuming the ESTs 
are uniformly distributed throughout the genome, the estimated genome size is 
2.73 Gb/95% = 2.87 Gb. 

For detailed information on the sequencing and assembly techniques,
see the <A HREF="ftp://ftp.hgsc.bcm.tmc.edu/pub/data/Btaurus/fasta/Btau20070913-freeze/README.Btau20070913.txt"
TARGET=_blank>Btau_4.0 Readme</A> and  
<A HREF="ftp://ftp.hgsc.bcm.tmc.edu/pub/data/Btaurus/fasta/Btau20080815/README.Btau20080815.txt"
TARGET=_blank>Btau_4.2 Readme</A>
filethat accompanies these two  releases and the Baylor 
<A HREF="https://www.hgsc.bcm.edu/content/bovine-genome-project"
TARGET=_blank>Bovine Genome Project</A> web page. </P>
<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/bosTau4/">FTP
server</A> or the 
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#cow">Downloads</A> 
page. These data have specific
<A HREF="https://www.hgsc.bcm.edu/content/conditions-use"
TARGET=_blank>conditions for use</A>.
The bosTau4 annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="../goldenPath/credits.html#cow_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to this release. </P>
