<H2>Description</H2>
<P>

This track identifies predicted Alternative Conserved Exons (human-mouse 
conservation), as predicted by ACEScan.  These are exons that are 
present in some transcripts, but skipped by alternative splicing in 
other transcripts in both human and mouse.   Alternate use of skipped  
exons has important consequences during gene expression and in disease.

<H2>Methods</H2>
<P>

Putative alternative conserved exons on mRNAs were identified using a 
machine-learning algorithm, Regularized Least-Squares Classification. 
Characteristics of known exons that have been skipped in both human 
and mouse mRNAs were determined by considering factors such as
exon and intron length, splice-site strength, sequence conservation, 
and region-specific oligonucleotide composition.  </P>

<P>A training set was made by comparing known exons that are skipped 
in some transcripts to exons never skipped.
These characteristics were then applied to the whole genome to predict
skipped exons in other transcripts.  This track displays exons with 
positive ACEScan scores.</P>

<P>For further details of the method used to generate this annotation, 
please refer to Yeo <I>et al.</I> (2005).  

<H2>Credits</H2>
<P>

Thanks to Gene Yeo at the Crick-Jacobs Center, Salk Institute and 
Christopher Burge, MIT, for providing this annotation.  For additional 
information on ACEscan predictions please contact 
&#103;&#101;n&#101;&#121;&#101;o&#64;&#115;al&#107;.&#101;&#100;&#117;
<!-- above address is geneyeo at salk dot edu -->
 or  
&#99;&#98;&#117;&#114;ge&#64;&#109;i&#116;.e&#100;&#117;.
<!-- above address is cburge at mit dot edu. -->

<H2>References</H2>
<P>

Yeo GW, Van Nostrand E, Holste D, Poggio T, Burge CB (2005), 
<A HREF = "https://www.pnas.org/content/102/8/2850" 
TARGET = _BLANK>
Identification and analysis of alternative splicing events conserved 
in human and mouse.</A>  
<I>Proc Natl Acad Sci U S A.</I> 2005 
Feb 22;102(8):2850-5.

</P>
