<H2>Description</H2>
<P>
This track displays transcriptome data from tiling GeneChips produced by 
<A HREF="https://www.thermofisher.com/us/en/home/life-science/microarray-analysis.html" TARGET="_blank">Affymetrix</A>. For the
complete non-repetitive portion of the human genome, more than 256 million
probe pairs were tiled every 5 bp in non-repeat-masked areas and hybridized to
whole-cell short RNA (&lt;200 nucleotides) from two different cell lines. 
Note that the female cell lines HeLa, SK-N-AS, and U87MG do not contain data 
for chrY. Data were collected using a strand-specific methodology that allows 
separate measurements for top strand (sense) and bottom strand (antisense) RNA 
signals. For experimental details and results, see Kapranov <em>et al</em>. in 
the References section below. </P>
<P>
This track shows the coordinates of transcribed fragments (transfrags) 
representing short RNAs. A separate track -- &quot;Affy 
Tx sRNA signals&quot; -- contains signals (PM-MM values) for each probe pair 
plotted against its genomic coordinates (see Kapranov <em>et al</em>. in the
References section below). </P>

<H2>Display Conventions and Configuration</H2>
<P>
This annotation follows the display conventions for composite tracks. The
subtracks within this annotation may be configured in a variety of ways to
highlight different aspects of the displayed data. The graphical configuration
options are shown at the top of the track description page, followed by a list
of subtracks. For more information about the graphical configuration options,
click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html">Graph configuration help</A>
link. To display only selected subtracks, uncheck the
boxes next to the tracks you wish to hide. 

<H2>Methods</H2>
<P>
Genomic regions corresponding to the sRNA transcripts were mapped for the plus
or the minus strands of the genome and for HepG2 and HeLa cell lines
respectively. The following analysis steps were performed: 
<OL>
<LI>
Replicate arrays were quantile-normalized and the median intensity (using both 
PM and MM intensities) of each array was scaled to a target value of 20.
<LI>
The expression level was estimated for each mapped probe position without
application of a smoothing window. These data are displayed in the track 
&quot;Affy Tx sRNA Sig&quot;. This was followed by generation of 
transfrags as follows: 
<UL>
<LI>
probes with intensities corresponding to the 98th percentile of
all the PM-MM probe intensities were identified.
<LI>
transfrags were determined by connecting adjacent positive probes with a 
maximum allowable gap of four nucleotides and a minimum run of seven nucleotides
(at least two consecutive positive probes with no negative probes between), and 
must be called positive reproducibly in both biological replicas.
<LI>
to further prioritize the transfrag maps, the intensities of all
transfrags were calculated based on the composite signals of the two
biological replicas, and transfrags corresponding to the top 25 quartile were
used for further analysis.
</UL>
<LI>
In addition, several successive steps were performed to avoid potentially 
cross-hybridizing sequences and/or those that mapped to potentially non-unique 
regions. Transfrags were discarded if they overlapped pseudogenes or harbored 
low-complexity or repetitive sequences. The resulting maps are believed to 
represent a very conservative view of the sRNA population in the cell.
</OL>

<H2>Credits</H2>
<P>
Data generation and analysis were performed by the transcriptome group at
Affymetrix with assistance from colleagues at the University of 
Leipzig, Fraunhofer Institute, and University of Vienna: P. Kapranov, J. Cheng,
S. Dike, D.A. Nix, R. Duttagupta, A.T. Willingham, P.F. Stadler, J. Hertel, 
J. Hackermüller, I.L. Hofacker, I. Bell, E. Cheung, J. Drenkow, E. Dumais, 
S. Patel, G. Helt, M. Ganesh, S. Ghosh, A. Piccolboni, V. Sementchenko, 
H. Tammana, T.R. Gingeras.</P>
<P>
Questions or comments about this annotation?  Email 
<A HREF="mailto:&#103;&#101;&#110;om&#101;&#64;&#115;&#111;&#101;.u&#99;&#115;c.&#101;&#100;&#117;">&#103;&#101;&#110;om&#101;&#64;&#115;&#111;&#101;.u&#99;&#115;c.&#101;&#100;&#117;</A>.
<!-- above address is genome at soe.ucsc.edu -->

<H2>References</H2>
<P>
Kapranov P, Cheng J, Dike S, Nix DA, Duttagupta R, Willingham AT, Stadler PF, Hertel J, Hackermüller J, Hofacker IL, <em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=17510325&amp;ordinalpos=1&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum"
TARGET="_blank">RNA maps reveal new RNA classes and a possible function for
pervasive transcription</A>. 
<em>Science</em>. 2007 Jun 8;316(5830):1484-8. </P>
