<H2>Description</H2>
<P>
This track displays transcriptome data from tiling GeneChips produced by 
<A HREF="https://www.thermofisher.com/us/en/home/life-science/microarray-analysis.html" target="_blank">Affymetrix</A>. For the 
complete non-repetitive portion of the human genome, more than 256 million 
probe pairs were tiled every 5 bp in non-repeat-masked areas and hybridized to 
cytosolic polyA+ long RNA (&gt;200 nucleotides) from 8 different cell lines. 
Note that the female cell lines HeLa, SK-N-AS, and U87MG do not contain data 
for chrY. For HeLa and HepG2, 
samples were also produced for nuclear polyA+ RNA in addition to cytosolic 
polyA+ RNA. For experimental details and results, see Kapranov <em>et al</em>. 
in the References section below. </P>
<P>
This track contains the signal level estimated 
for each mapped probe position after hybridization with long RNA samples.
A separate track -- &quot;Affy Tx lRNA Reg &quot; -- contains transcribed 
fragments (transfrags) representing long RNAs (see Kapranov <em>et al</em>. in 
the References section below). While the raw data are based on perfect match 
minus mismatch (PM - MM) probe values and may contain negative values, the track
has a minimum value of zero for visualization purposes. Likewise, the probes 
with high signal values are cut off at the intensity of 150 by default and will 
appear to have similar magnitude.</P>

<H2>Display Conventions and Configuration</H2>
<P>
This annotation follows the display conventions for composite tracks. The
subtracks within this annotation may be configured in a variety of ways to
highlight different aspects of the displayed data. The graphical configuration
options are shown at the top of the track description page, followed by a list
of subtracks. For more information about the graphical configuration options,
click the <A HREF="../goldenPath/help/hgWiggleTrackHelp.html">Graph
configuration help</A> link. To display only selected subtracks, uncheck the
boxes next to the tracks you wish to hide. </P>

<H2>Methods</H2>
<P>
For each data point, probes within 30 bp on either side were used to improve
the estimate of expression level for a particular probe. This helped to smooth
the data and produce a more robust estimate of the transcription level at a
particular genomic location. The following analysis steps were performed: 
<OL>
<LI>
Replicate arrays were quantile-normalized and the median intensity (using both 
PM and MM intensities) of each array was scaled to a target value of 50. 
<LI>
The expression level was estimated for each mapped probe position by
collecting all the probe pairs that fell within a window of ± 30
bp, calculating all non-redundant pairwise averages of PM - MM values
of all probe pairs in the window, and taking the median of all resulting 
pairwise averages. 
<LI>
The resulting signal value is the Hodges-Lehmann estimator associated with the
Wilcoxon signed-rank statistic of the PM - MM values that lie within ± 30 bp of
the sliding window centered at every genomic coordinate.
<LI>
Transfrags in the track labeled &quot;Affy Tx lRNA Reg&quot; were
determined by connecting adjacent positive probes with a maximum allowable gap
of 11 nucleotides and a minimum run of 49 nucleotides (at least 11 probes with
at most one negative probe between two positive probes). Transfrags were 
discarded if they overlapped pseudogenes or harbored low-complexity or 
repetitive sequences.
</OL></P>

<H2>Credits</H2>
<P>
Data generation and analysis were performed by the transcriptome group at
Affymetrix with assistance from colleagues at the University of 
Leipzig, Fraunhofer Institute, and University of Vienna: P. Kapranov, J. Cheng,
S. Dike, D.A. Nix, R. Duttagupta, A.T. Willingham, P.F. Stadler, J. Hertel, 
J. Hackermüller, I.L. Hofacker, I. Bell, E. Cheung, J. Drenkow, E. Dumais, 
S. Patel, G. Helt, M. Ganesh, S. Ghosh, A. Piccolboni, V. Sementchenko, 
H. Tammana, T.R. Gingeras.</P>
<P>
Questions or comments about this annotation?  Email 
<A HREF="mailto:&#103;&#101;&#110;om&#101;&#64;&#115;&#111;&#101;.u&#99;&#115;c.&#101;&#100;&#117;">&#103;&#101;&#110;om&#101;&#64;&#115;&#111;&#101;.u&#99;&#115;c.&#101;&#100;&#117;</A>.
<!-- above address is genome at soe.ucsc.edu -->

<H2>References</H2>
<P>
Kapranov P, Cheng J, Dike S, Nix DA, Duttagupta R, Willingham AT, Stadler PF, Hertel J, Hackermüller J, Hofacker IL, <em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=17510325&amp;ordinalpos=1&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum"
TARGET="_blank">RNA maps reveal new RNA classes and a possible function for
pervasive transcription</A>. 
<em>Science</em>. 2007 Jun 8;316(5830):1484-8. </P>
