<H2>Description</H2>
<P>
This track contains tBLASTn alignments of the peptides
from the predicted and known genes identified in the <em>D. melanogaster</em>
FlyBase as of 24 July 2004.</P>

<H2>Methods</H2>
<P>
First, the predicted proteins from the <em>D. melanogaster</em> FlyBase track 
were aligned with the <em>D. melanogaster</em> genome using the blat program 
to discover exon boundaries.  
Next, the amino acid sequences that make up each exon were aligned with the 
$organism sequence using the tBLASTn program.
Finally, the putative $organism exons were chained together using an 
organism-specific maximum gap size but no gap penalty. The single best exon 
chains extending over more than 60% of the query protein were included. Exon 
chains that extended over 60% of the query and matched at least 60% of the 
protein's amino acids were also included.</P>

<H2>Credits</H2> 
<P>
tBLASTn is part of the NCBI Blast tool set. For more information on Blast, see
Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&list_uids=2231712&dopt=Abstract"
TARGET=_blank>Basic local alignment search tool</A>.
<em>J Mol Biol.</em> 1990 Oct 5;215(3):403-410.</P>
<P>
<A HREF="http://www.genome.org/cgi/content/abstract/12/4/656" 
TARGET=_blank>Blat</A> was written by Jim Kent. The remaining utilities 
required to produce this track were written by Jim Kent or Brian Raney.</P>
