<H2>Description</H2>
<p>
This track set shows <b>GWAS meta-analyses</b> from the 
<a target=_blank href="https://www.covid19hg.org/">
<b>COVID-19 Host Genetics Initiative (HGI)</b></a>: 
a collaborative effort to facilitate 
the generation, analysis and sharing of COVID-19 host genetics research.
The COVID-19 HGI organizes meta-analyses across multiple studies contributed by 
<a target="_blank" href="https://www.covid19hg.org/partners/">partners world-wide</a>
to identify the genetic determinants of SARS-CoV-2 infection susceptibility and disease severity 
and outcomes. Moreover, the COVID-19 HGI also aims to provide a platform for study partners to 
share analytical results in the form of summary statistics and/or individual level data where 
possible.
</p>

<p>
The specific phenotypes studied by the COVID-19 HGI are those that benefit from maximal sample 
size: primary analysis on disease severity. Two meta-analyses are represented in this track:
</p>

<ul>
<li><b>ANA_C2_V2</b>: covid vs. population (6696 cases from 18 studies)</li>
<li><b>ANA_B2_V2</b>: hospitalized covid vs. population (3199 cases from 8 studies)</li>
</ul>

<H2>Display Conventions</H2>
<p>
Displayed items are colored by <b>GWAS effect</b>:  red for positive, blue for negative. 
The height of the item reflects the <b>effect size</b>. The effect size, defined as the 
contribution of a SNP to the genetic variance of the trait, was measured as beta coefficient 
(beta). The higher the absolute value of the beta coefficient, the stronger the effect.
The color saturation indicates <b>statistical significance</b>: p-values smaller than 1e-5
are brightly colored (bright red
<span style='background-color: #ff0000;'>&nbsp;&nbsp;</span>
, bright blue
<span style='background-color: #0000ff;'>&nbsp;&nbsp;</span>
),
those with less significance (p >= 1e-5) are paler (light red
<span style='background-color: #ffa0a0;'>&nbsp;&nbsp;</span>
, light blue
<span style='background-color: #a0a0ff;'>&nbsp;&nbsp;</span>
). For better visualization of the data, only SNPs with p-values smaller than 1e-3 are 
displayed by default. 
</p>

<p>
Each track has separate display controls and data can be filtered according to the
<b>number of studies</b>, <b>minimum -log10 p-value</b>, and the
<b>effect size (beta coefficient)</b>, using the track <b>Configure</b> options.
</p>

<p>
<b>Mouseover</b> on items shows the rs ID (or chrom:pos if none assigned), both the non-effect 
and effect alleles, the effect size (beta coefficient), the p-value, and the number of 
studies.
Additional information on each variant can be found on the details page by clicking on the item.
</p>  

<H2>Methods</H2>
<p>
COVID-19 Host Genetics Initiative (HGI) GWAS meta-analysis round 3 (July 2020) results were used 
in this study. Each participating study partner submitted GWAS summary statistics for up to four 
of the <a target=_blank href="https://www.covid19hg.org/results/">COVID-19 phenotype definitions</a>.
</p>
<p>
Data were generated from genome-wide SNP array and whole exome and genome
sequencing, leveraging the impact of both common and rare variants. The statistical analysis
performed takes into account differences between sex, ancestry, and date of sample collection. 
Alleles were harmonized across studies and reported allele frequencies are based on gnomAD 
version 3.0 reference data. Most study partners used the <b>SAIGE GWAS</b> pipeline in order 
to generate summary statistics used for the COVID-19 HGI meta-analysis. The summary statistics 
of individual studies were manually examined for inflation, 
deflation, and excessive number of false positives. Qualifying summary statistics were filtered for 
<b>INFO > 0.6</b> and <b>MAF > 0.0001</b> prior to meta-analyzing the entirety of the data. 
The meta-analysis was done using inverse variance weighting of effects method, accounting for 
strand differences and allele flips in the individual studies. 
</p>
<p>
The meta-analysis results of variants appearing in at least three studies (analysis C2) or two 
studies (all other analyses) were made publicly available.
The meta-analysis software and workflow are available <a target=_blank 
href="https://github.com/covid19-hg/META_ANALYSIS">here</a>. More information about the 
prospective studies, processing pipeline, results and data sharing can be found 
<a target=_blank href="https://www.covid19hg.org/about/">here</a>.
</p>


<H2>Data Access</H2>
<p>
The data underlying these tracks and summary statistics results are publicly available in 
<a target=_blank href="https://www.covid19hg.org/results">COVID19-hg Release 3 (June 2020)</a>.
The raw data can be explored interactively with the <a target="_blank" href="../cgi-bin/hgTables">
Table Browser</a>, or the <a target="_blank" href="../cgi-bin/hgIntegrator">Data Integrator</a>. 
Please refer to
our <a href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome"
target="_blank">mailing list archives</a> for questions, or our <a target="_blank"
href="../FAQ/FAQdownloads.html#download36">Data Access FAQ</a> for more information.
</p>

<H2>Credits</H2>
<p>
Thanks to the COVID-19 Host Genetics Initiative contributors and project leads for making these 
data available, and in particular to Rachel Liao, Juha Karjalainen, and Kumar Veerapen at the 
Broad Institute for their review and input during browser track development.
</p>

<H2>References</H2>

<p>
COVID-19 Host Genetics Initiative.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32404885" target="_blank">
The COVID-19 Host Genetics Initiative, a global initiative to elucidate the role of host genetic
factors in susceptibility and severity of the SARS-CoV-2 virus pandemic</a>.
<em>Eur J Hum Genet</em>. 2020 Jun;28(6):715-718.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32404885" target="_blank">32404885</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7220587/" target="_blank">PMC7220587</a>
</p>



