<H2>Description</H2>
<P>
This track displays copy number variants (CNVs), insertions/deletions (InDels),
inversions and inversion breakpoints annotated by the 
<A HREF="http://dgvbeta.tcag.ca/dgv/app/home" 
TARGET=_BLANK>Database of Genomic Variants</A> (DGV), which
contains genomic variations observed in healthy individuals.  
DGV focuses on structural variation, defined as 
genomic alterations that involve segments of DNA that are larger than
1000 bp.  Insertions/deletions of 100 bp or larger are also included.
</P>

<H2>Display Conventions</H2>
<P>
Color is used to indicate the type of variation.  
<BR>Please note that variants now link to DGV's new browser as of February 2013.
<UL>
 <LI><B><span style="color:#C000C0;">inversions</span></B> and
     <B><span style="color:#C000C0;">inversion breakpoints</span></B> are purple.  
 </LI>

 <LI>CNVs and InDels are blue if there is a 
   <B><span style="color:#0000C0;">gain in size</span></B> relative to the reference.
 </LI>

 <LI>CNVs and InDels are red if there is a 
   <B><span style="color:#C00000;">loss in size</span></B> relative to the reference.
 </LI>

 <LI>CNVs and InDels are brown if there are reports of
   <B><span style="color:#8B4513;">both a loss and a gain in size</span></B> 
   relative to the reference.
 </LI>

 <LI>CNVs and InDels are black if there is an
   <B><span style="color:#010000;">unknown</span></B>
   change relative to the reference.
 </LI>
</UL>
</P>
<P>
Variants are displayed with accession numbers with the following
<A HREF="http://dgvbeta.tcag.ca/dgv/data-model/entd.html#Variant"
TARGET=_BLANK>DGV nomenclature</A>. When possible, DGV uses accessions
from peer archives of structural variation (<A HREF="https://www.ncbi.nlm.nih.gov/dbvar/" TARGET=_BLANK>dbVar</A>
at NCBI or <A HREF="https://www.ebi.ac.uk/dgva/" TARGET=_BLANK>DGVa</A> at EBI).
These accessions begin with either &quot;essv&quot;,
&quot;esv&quot;, &quot;nssv&quot;, or &quot;nsv&quot;, followed by a number.
Variant submissions processed by EBI begin with &quot;e&quot;
and those processed by NCBI begin with &quot;n&quot;.
</P>
<P>
Accessions with ssv are for variant calls on a particular sample, and if they
are copy number variants, they generally indicate whether the change is a gain
or loss. Accessions with sv are for regions asserted by submitters to contain
structural variants, and often span ssv elements for both losses and gains.
dbVar and DGVa do not record numbers of losses and gains encompassed within
sv regions.
</P>
<P>
DGV merges clusters of variants that share at least 70% reciprocal
overlap in size/location, and provides an sv-like record with an accession that
begins with &quot;dgv_&quot;. For most sv and dgv variants, DGV displays the total number
of gains and/or losses at the bottom of their variant detail page. Since each ssv variant
is for one sample, its total is 1.
</P>

<H2>Methods</H2>
<P>
DGV collects these variants by ongoing manual curation of the literature.  
A brief description of the method and sample used for a particular 
variant is included on the details page, along with a link to the 
PubMed abstract for the study from which the variants were collected.  
</P>
<P>
For data sets where the variation calls are reported at a 
sample-by-sample level, DGV merges calls with similar boundaries 
across the sample
set. Only variants of the same type (i.e. CNVs, Indels, inversions)
are merged, and gains and losses are merged separately.  In addition,
if several different platforms/approaches are used within the same
study, these datasets are merged separately. Sample level calls that
overlap by &ge; 70% are merged in this process.
</P>

<H2>Credits</H2>
<P>
Thanks to the <A HREF="http://dgvbeta.tcag.ca/dgv/app/home" 
TARGET=_BLANK>Database of Genomic Variants</A> for providing these data.
In citing the Database of Genomic Variants please refer to Iafrate 
<em>et al.</em>.  
</P>

<H2>References</H2>
<P>
Iafrate AJ, Feuk L, Rivera MN, Listewnik ML, Donahoe PK, Qi Y,
Scherer SW, Lee C.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/15286789" 
TARGET=_BLANK>Detection of large-scale variation in the human genome</A>.
<em>Nat Genet.</em> 2004 Sep;36(9):949-51. PMID:15286789.</P>
<P>
Zhang J, Feuk L, Duggan GE, Khaja R, Scherer SW. 
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/17124402" 
TARGET=_BLANK>Development of bioinformatics resources for display and
analysis of copy number and other structural variants in the human
genome</A>.
<em>Cytogenet Genome Res.</em> 2006;115(3-4):205-14. PMID:17124402.</P>

