<H2>Description</H2>
<P>
This track displays copy number variants (CNVs), insertions/deletions (InDels),
inversions and inversion breakpoints annotated by the
<A HREF="http://projects.tcag.ca/variation/"
TARGET=_BLANK>Database of Genomic Variants</A> (DGV), which
contains genomic variations observed in healthy individuals.
DGV focuses on structural variation, defined as
genomic alterations that involve segments of DNA that are larger than
1000 bp.  Insertions/deletions of 100 bp or larger are also included.
</P>

<H2>Display Conventions</H2>
<P>
Color is used to indicate the type of variation.
<UL>
 <LI><B><FONT COLOR = "C000C0">inversions</FONT></B> and
     <B><FONT COLOR = "C000C0">inversion breakpoints</FONT></B> are purple.
 </LI>

 <LI>CNVs and InDels are red if there is a
   <B><FONT COLOR = "C00000">gain in size</FONT></B> relative to the reference.
 </LI>

 <LI>CNVs and InDels are blue if there is a
   <B><FONT COLOR = "0000C0">loss in size</FONT></B> relative to the reference.
 </LI>

 <LI>CNVs and InDels are green if there are reports of
   <B><FONT COLOR = "00C000">both a loss and a gain in size</FONT></B>
   relative to the reference.
 </LI>
</UL>

</P>

<H2>Methods</H2>
<P>
DGV collects these variants by ongoing manual curation of the literature.
A brief description of the method and sample used for a particular
variant is included on the details page, along with a link to the
PubMed abstract for the study from which the variants were collected.
</P>
<P>
For data sets where the variation calls are reported at a
sample-by-sample level, DGV merges calls with similar boundaries
across the sample
set. Only variants of the same type (i.e. CNVs, Indels, inversions)
are merged, and gains and losses are merged separately.  In addition,
if several different platforms/approaches are used within the same
study, these datasets are merged separately. Sample level calls that
overlap by &ge; 70% are merged in this process.
</P>

<H2>Credits</H2>
<P>
Thanks to the <A HREF="http://projects.tcag.ca/variation/"
TARGET=_BLANK>Database of Genomic Variants</A> for providing these data.
In citing the Database of Genomic Variants please refer to Iafrate
<em>et al.</em>.
</P>

<H2>References</H2>
<P>
Iafrate AJ, Feuk L, Rivera MN, Listewnik ML, Donahoe PK, Qi Y,
Scherer SW, Lee C.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15286789"
TARGET=_BLANK>Detection of large-scale variation in the human genome</A>.
<em>Nat Genet.</em> 2004 Sep;36(9):949-51.</P>
<P>
Zhang J, Feuk L, Duggan GE, Khaja R, Scherer SW.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17124402"
TARGET=_BLANK>Development of bioinformatics resources for display and
analysis of copy number and other structural variants in the human
genome</A>.
</em>Cytogenet Genome Res.</em> 2006;115(3-4):205-14</P>

