<H2>Description</H2>
<P>
Genes in metazoa are controlled by a complex array of cis-regulatory elements 
that include core and distal promoters, enhancers, insulators, silencers, etc. 
(Levine and Tjian, 2003). In living cells, functionally active cis-regulatory 
elements bear a unifying feature, which is a chromatin-based epigenetic 
signature known as nuclease hypersensitivity (Elgin, 1988; Gross and Garrard, 
1988; Wolffe, 1998). This track presents the results of a collaboration 
between J. Craig Venter Institute (JCVI, Rockville MD) and the European 
Institute of Oncology (Milan, Italy) to isolate nuclease accessible sites 
(NAS) from primary human CD34+ hematopoietic stem and progenitor cells, and 
from CD34- cells, maturating myeloid cells generated by in vitro 
differentiation of CD34+ cells (Gargiulo et al., submitted). This effort made 
use of a method (originally developed at Sangamo BioSciences, Richmond, CA) to 
isolate such NAS from living cells using restriction enzymes (RE), leading to 
minimal, if any, contamination from bulk DNA. High throughput 454 sequencing 
was then used to generate NAS libraries in CD34+ and CD34- cells: this 
technology has been named "NA-Seq" (Gargiulo et al., submitted).
<H2>Display Conventions</H2>
<P>
The track annotates the location of NAS in the genome of human CD34+ and CD34- 
cells in the form of tags, generated by NA-Seq and obtained by merging NAS 
within 600 bp. Note that the method identifies a specific position in chromatin 
that is sensitive to nucleases, but does not map the boundaries of a 
regulatory element per se. A conservative estimate of element size would be 
the space occupied by one nucleosome, i.e., 180 - 200 bp surrounding the tag, 
although there is precedent in the literature for nuclease hypersensitive 
sites that span more than the length of one nucleosome (Turner, 2001; Wolffe, 
1998; Boyle, 2008).
<H2>Methods</H2>
<P>
CD34+ cells (enriched in hematopoietic stem and progenitor cells) were 
prepared from healthy donors following guidelines established by the Ethics 
Committee of the European Institute of Oncology (IEO), Milan. Mobilization of 
CD34+ cells to the peripheral blood was stimulated by G-CSF treatment 
according to standard procedures. After mobilization, donors were subjected to 
leukaphereses, and &lt;10% of the sample was used in the experiment. CD34+ 
cells were purified using a magnetic positive selection procedure ("EASYSEP"; 
Stemcell, Vancouver, Canada). Purity of separation was evaluated by FACS 
after staining with an anti-Human CD34 FITC-conjugate antibody (Stemcell). 
Upon purification, the cell cycle status of the CD34+ cells was monitored by 
propidium iodide staining and FACS analysis. G0/G1 cells varied from 
approximately 90% to &gt;95% of the total cells. Cells were immediately used 
for the isolation of NAS using the nuclease hypersensitive site isolation 
protocol (Gargiulo et al., submitted).
<H2>Verification</H2>
<P>
The method was initially validated on human tissue culture cells by examining 
the colocalization of DNA fragments isolated from cells with experimentally 
determined nuclease hypersensitive sites in chromatin as mapped by indirect 
end-labeling and Southern blotting (Nedospasov and Georgiev, 1980; Wu, 1980). 
Nineteen out of nineteen randomly chosen clones from those libraries 
represented bona fide DNAse I hypersensitive sites in chromatin (Fyodor Urnov, 
unpublished results). These data confirmed that the method yields very 
high-content libraries of active cis-regulatory DNA elements, supporting its 
application to human CD34+ cells. In collaboration with scientists at the J. 
Craig Venter Institute and the European Institute of Oncology, libraries of 
NAS were prepared using this method in HT 454 sequencing from CD34+ and CD34- 
cells, and showed that 41 out of 51 randomly chosen clones - &gt;80% - 
coincided with DNAse I hypersensitive sites (Gargiulo et al., submitted).
<H2>Credits</H2>
<P>
The library of Nuclease Accessible sites (NAS) from human CD34+/CD34- cells 
was prepared and validated by Saverio Minucci and colleagues at the European 
Institute of Oncology. Sequencing was performed by Sam Levy and colleagues 
(J. Craig Venter Institute). This method was initially developed and validated 
by Fyodor Urnov, Alan Wolffe, and colleagues at Sangamo BioSciences, Inc. 
<H2>References</H2>
<p>
Boyle AP, Davis S, Shulha HP, Meltzer P, Margulies EH, Weng Z, Furey TS, Crawford GE.
<a href="https://www.sciencedirect.com/science/article/pii/S0092867407016133" target="_blank">
High-resolution mapping and characterization of open chromatin across the genome</a>.
<em>Cell</em>. 2008 Jan 25;132(2):311-22.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18243105" target="_blank">18243105</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2669738/" target="_blank">PMC2669738</a>
</p>

<p>
Elgin SC.
<a href="https://www.jbc.org/article/S0021-9258(19)77625-X/fulltext" target="_blank">
The formation and function of DNase I hypersensitive sites in the process of gene activation</a>.
<em>J Biol Chem</em>. 1988 Dec 25;263(36):19259-62.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3198625" target="_blank">3198625</a>
</p>

<P>
Gargiulo G, Levy S, et al. A Global Analysis of chromatin Accessibility and 
Dynamics during Hematopoietic Differentiation. <EM>Submitted</EM>.</P>

<p>
Gross DS, Garrard WT.
<a href="https://www.annualreviews.org/doi/abs/10.1146/annurev.bi.57.070188.001111" target="_blank">
Nuclease hypersensitive sites in chromatin</a>.
<em>Annu Rev Biochem</em>. 1988;57:159-97.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3052270" target="_blank">3052270</a>
</p>

<p>
Levine M, Tjian R.
<a href="https://www.nature.com/articles/nature01763" target="_blank">
Transcription regulation and animal diversity</a>.
<em>Nature</em>. 2003 Jul 10;424(6945):147-51.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12853946" target="_blank">12853946</a>
</p>

<p>
Nedospasov SA, Georgiev GP.
<a href="https://www.sciencedirect.com/science/article/pii/0006291X80903666" target="_blank">
Non-random cleavage of SV40 DNA in the compact minichromosome and free in solution by micrococcal
nuclease</a>.
<em>Biochem Biophys Res Commun</em>. 1980 Jan 29;92(2):532-9.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6243943" target="_blank">6243943</a>
</p>

<P>
Turner BM. 
<A HREF="https://onlinelibrary.wiley.com/doi/book/10.1002/9780470750629" target=_BLANK>
Chromatin and Gene Regulation: Mechanisms in Epigenetics</A>.
Blackwell Science Ltd., Oxford. 2001.</P>

<P>
Wolffe AP. Chromatin: Structure and Function. 
Academic Press, San Diego, CA. 1998.</P>

<p>
Wu C.
<a href="https://www.nature.com/articles/286854a0" target="_blank">
The 5' ends of Drosophila heat shock genes in chromatin are hypersensitive to DNase I</a>.
<em>Nature</em>. 1980 Aug 28;286(5776):854-60.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6774262" target="_blank">6774262</a>
</p>
