<H2>Description</H2>
<P>
This track shows regions that co-precipitate with antibodies against
each of ten factors in all ENCODE regions, in retinoic-acid stimulated
HL-60 cells harvested after 0, 2, 8, and 32 hours.  Clustered sites are shown in
separate subtracks for each of the ten antibodies:
<UL>
<LI>Brg1 - Brahma-related Gene 1</LI>
<LI>CEBPe - CCAAT-enhancer binding protein-epsilon</LI>
<LI>CTCF - CCTC binding factor</LI>
<LI>H3K27me3 (H3K27T) - Histone H3 tri-methylated lysine 27</LI>
<LI>H4Kac4 (HisH4) - Histone H4 tetra-acetylated lysine</LI>
<LI>P300 - E1A-binding protein, 300-KD</LI>
<LI>PU1 - Spleen focus forming virus proviral integration oncogene</LI>
<LI>Pol2 - RNA Polymerase II (8WG16 ab against pre-initiation complex form)</LI>
<LI>RARA (RARecA) - Retinoic Acid Receptor-Alpha</LI>
<LI>SIRT1 - Sirtuin-1</LI>
</UL>
Retinoic acid-stimulated HL-60 cells were harvested  and
whole cell extracts (control) were made. An antibody was used to
immunoprecipitate bound chromatin fragments (treatment). DNA was
purified from these samples and hybridized to Affymetrix ENCODE
oligonucleotide tiling arrays, which have 25-mer probes tiled
every 22 bp on average in the non-repetitive ENCODE regions.</P>

<H2>Display Conventions and Configuration</H2>
<P>
The subtracks within this composite annotation track
may be configured in a variety of ways to highlight different aspects of the 
displayed data. The graphical configuration options for the subtracks 
are shown at the top of the track description page, followed by a list of 
subtracks.  
For more information about the graphical configuration options, click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph
configuration help</A> link.</P>
<P>
Color differences among the subtracks are arbitrary. They provide a
visual cue for finding the same antibody in different timepoint tracks.</P>

<H2>Methods</H2>
<P>
The data from replicate arrays were quantile-normalized (Bolstad <em>et
al</em>., 2003) and all arrays were scaled to a median array intensity of
22. Within a sliding 1001 bp window centered on each probe, a signal
estimator S = ln[max(PM - MM, 1)] (where PM is perfect match and MM is
mismatch) was computed for each biological replicate treatment- and
all replicate control-probe pairs. An estimate of the significance of
the enrichment of treatment signal for each replicate over control
signal in each window was given by the P-value computed using the
Wilcoxon Rank Sum test over each biological replicate treatment and
all control signal estimates in that window.  The median of the log
transformed P-value (-10 log<SUB>10</SUB> P) across processed replicate 
data is displayed.</P>
<P>
Several independent biological replicates (four each for Brg1, CEBPe,
CTCF, PU1, and SIRT1; five each for H3K27me3, H4Kac4, P300, Pol2 and
RARA) were generated and hybridized
to duplicate arrays (two technical replicates). Reproducible enriched
regions were generated from the signal by first applying a cutoff of
20 to the log transformed P-values, a maxGap and minRun of 500 and 0
basepairs respectively, to each biological replicate.  Since each
region or site may be comprised of more than one probe, a median
based on the distribution of log transformed P-values was computed per
site for each of the respective replicates. These seed sites were then
ranked individually within each of the replicates. If a site was
absent in a replicate, the maximum or worst rank of the distribution
was assigned to it. </P>
<P>
The following three values were computed for each
site by combining data from all biological replicates: 
<UL>
<LI>average of all ranks computed among biological replicates
<LI>sum of all pairwise differences in these ranks computed among biological
replicates
<LI>a combined P-value, using a chi square distribution,
across all replicates
</UL></P>
<P>
The final sites were selected when all of the
above three metrics were relatively low, where &quot;low&quot; corresponds to 
the top 25 percentile of the distribution. </P>

<H2>Verification</H2>
<P>
Using the P-values from the biological replicates, all pairwise
rank correlation coefficients were computed among biological
replicates.  Data sets showing both consistent pairwise correlation
coefficients and at least weak positive correlation across all pairs
were considered reproducible. </P>

<H2>Credits</H2>
<P>
These data were generated and analyzed by the Gingeras/Struhl
collaboration with the Tom Gingeras group at 
<A HREF="https://www.thermofisher.com/us/en/home/life-science/microarray-analysis.html" TARGET=_blank>Affymetrix</A> and 
<A HREF="https://struhl.med.harvard.edu/"
TARGET=_blank> Kevin Struhl</A>'s group at Harvard Medical School. </P>

<H2>References</H2>
<P>
Please see the <A 
HREF="https://www.thermofisher.com/us/en/home/life-science/microarray-analysis/microarray-analysis-instruments-software-services/microarray-analysis-software/affymetrix-transcriptome-analysis-console-software.html"
TARGET=_blank>
Affymetrix Transcriptome</A> site for a project overview and
additional references to Affymetrix tiling array publications.</P>
<P>
Bolstad, B. M., Irizarry, R. A., Astrand, M., and Speed, T. P. 
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/12538238" TARGET=_blank>A comparison 
of normalization methods for high density oligonucleotide array data based 
on variance and bias</A>. 
<I>Bioinformatics</I> <B>19</B>(2), 185-193 (2003).</P>
<P>
Cawley, S., Bekiranov, S., Ng, H. H., Kapranov, P., Sekinger,
E. A., Kampa, D., Piccolboni, A., Sementchenko, V., Cheng, J.,
Williams, A. J., <I>et al</I>. 
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=14980218" TARGET=_blank>Unbiased mapping of 
transcription factor binding sites along human chromosomes 21 and 22 points 
to widespread regulation of noncoding RNAs</A>. 
<I>Cell</I> <B>116</B>(4), 499-509 (2004).</P>
