<H2>Description</H2>
<P>
This track shows regions that co-precipitate with antibodies against
each of 4 factors in all ENCODE regions, in retinoic-acid stimulated
<A HREF="https://www.atcc.org/ATCCAdvancedCatalogSearch/ProductDetails/tabid/452/Default.aspx?ATCCNum=CCL-240&Template=cellBiology">
HL-60</A>
(leukemia) cells harvested after 0, 2, 8, and 32 hours, and in a fifth
factor tested in 
<A HREF="https://www.atcc.org/ATCCAdvancedCatalogSearch/ProductDetails/tabid/452/Default.aspx?ATCCNum=HTB-33&Template=cellBiology">
ME-180</A> cervical carcinoma cells.
Median of the transformed P-value (-10 log[10] P) across
processed replicate data is displayed as 
separate subtracks for each antibody:
<UL>
<LI>H4Kac4 (HisH4) - Histone H4 tetra-acetylated lysine</LI>
<LI>H3K9K14ac2 (H3K9K14D) - Histone H3 K9 K14 Di-Acetylated</LI>
<LI>Pol2 - RNA Polymerase II (8WG16 ab against pre-initiation complex form)</LI>
<LI>p63_ActD - p63, in actinomycin-D treated ME-180 cells </LI>
<LI>p63_mActD - p63 in untreated ME-180 cells</LI>
</UL>
Retinoic acid-stimulated HL-60 cells and ME-180 cells (actinomycin-D treated
or untreated) were harvested and
whole cell extracts (control) were made. An antibody was used to
immunoprecipitate bound chromatin fragments (treatment). DNA was
purified from these samples and hybridized to Affymetrix ENCODE
oligonucleotide tiling arrays, which have 25-mer probes tiled
every 22 bp on average in the non-repetitive ENCODE regions.</P>
<P>
Only the median of the transformed P-value (-10 log[10] P) is displayed; 
data for all biological replicates 
can be downloaded from Affymetrix in 
<A HREF="http://transcriptome.affymetrix.com/download/ENCODE/affy_june1/wig/ChIP/" 
TARGET=_BLANK>wiggle</A>,
<A HREF="http://transcriptome.affymetrix.com/download/ENCODE/affy_june1/cel_data/ChIP/" 
TARGET=_BLANK>cel</A>, and
<A HREF="http://transcriptome.affymetrix.com/download/ENCODE/affy_june1/soft/ChIP/" 
TARGET=_BLANK>soft</A> formats.</P>

<H2>Display Conventions and Configuration</H2>
<P>
The subtracks within this composite annotation track
may be configured in a variety of ways to highlight different aspects of the 
displayed data. The graphical configuration options for the subtracks 
are shown at the top of the track description page, followed by a list of 
subtracks.  
For more information about the graphical configuration options, click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph
configuration help</A> link.</P>
<P>
Color differences among the subtracks are arbitrary. They provide a
visual cue for finding the same antibody in different timepoint tracks.</P>

<H2>Methods</H2>
<P>
The data from replicate arrays were quantile-normalized (Bolstad <em>et
al</em>., 2003) and all arrays were scaled to a median array intensity of
22. Within a sliding 1001 bp window centered on each probe, a signal
estimator S = ln[max(PM - MM, 1)] (where PM is perfect match and MM is
mismatch) was computed for each biological replicate treatment- and
all replicate control-probe pairs. An estimate of the significance of
the enrichment of treatment signal for each replicate over control
signal in each window was given by the P-value computed using the
Wilcoxon Rank Sum test over each biological replicate treatment and
all control signal estimates in that window.  The median of the 
transformed P-value (-10 log[10] P) across processed replicate data is
displayed.</P>

<H2>Verification</H2>
<P>
Using the P-values from the biological replicates, all pairwise
rank correlation coefficients were computed among biological
replicates.  Data sets showing both consistent pairwise correlation
coefficients and at least weak positive correlation across all pairs
were considered reproducible. </P>

<H2>Credits</H2>
<P>
These data were generated and analyzed by the Gingeras/Struhl
collaboration with the Tom Gingeras group at 
<A HREF="https://www.thermofisher.com/us/en/home/life-science/microarray-analysis.html" TARGET=_blank>Affymetrix</A> and 
<A HREF="https://struhl.med.harvard.edu/"
TARGET=_blank> Kevin Struhl</A>'s group at Harvard Medical School. </P>

<H2>References</H2>
<P>
Please see the <A 
HREF="https://www.thermofisher.com/us/en/home/life-science/microarray-analysis/microarray-analysis-instruments-software-services/microarray-analysis-software/affymetrix-transcriptome-analysis-console-software.html"
TARGET=_blank>
Affymetrix Transcriptome</A> site for a project overview and
additional references to Affymetrix tiling array publications.</P>
<P>
Bolstad, B. M., Irizarry, R. A., Astrand, M., and Speed, T. P. 
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/12538238" TARGET=_blank>A comparison 
of normalization methods for high density oligonucleotide array data based 
on variance and bias</A>. 
<I>Bioinformatics</I> <B>19</B>(2), 185-193 (2003).</P>
<P>
Cawley, S., Bekiranov, S., Ng, H. H., Kapranov, P., Sekinger,
E. A., Kampa, D., Piccolboni, A., Sementchenko, V., Cheng, J.,
Williams, A. J., <I>et al</I>. 
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=14980218" TARGET=_blank>Unbiased mapping of 
transcription factor binding sites along human chromosomes 21 and 22 points 
to widespread regulation of noncoding RNAs</A>. 
<I>Cell</I> <B>116</B>(4), 499-509 (2004).</P>
<P>
Yang A, Zhu Z, Kapranov P, McKeon F, Church GM, Gingeras TR, Struhl K.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17188034" TARGET=_blank>
Relationships between p63 binding, DNA sequence, transcription
activity, and biological function in human cells</A>. 
<I>Mol. Cell.</I> <B>24</B>(4), 593-602 (2006). </P>

