<H2>Description</H2>
<P>
Formaldehyde-Assisted Isolation of Regulatory Elements (FAIRE) is a procedure 
used to isolate chromatin that is resistant to the formation of protein-DNA 
cross-links.  These tracks display FAIRE data from 2091 fibroblast cells 
hybridized to high-resolution <A HREF="https://sequencing.roche.com/en/products-solutions/by-category/target-enrichment/hybridization.html"
TARGET=_blank>NimbleGen</A> arrays that tile the ENCODE regions. The four 
datasets, in practical terms, can be thought of as independent replicates. 
However, because they were part of a series of experiments aimed at optimizing 
cross-linking conditions in human cells, the data represent different 
cross-linking times (1, 2, 4, and 7 minutes). Although the individual 
replicates are not displayed, the replicate data and also the signal averages 
and the peaks for the averages can be 
<A TARGET=BLANK HREF="http://hgdownload.soe.ucsc.edu/goldenPath/hg17/encode/datafiles/UncFaire">downloaded</A>.</P>

<H2>Display Conventions and Configuration</H2>
<P>
The FAIRE data are represented by three subtracks. One subtrack shows the 
average normalized log<sub>2</sub> ratios for the tiled probes; the other two 
subtracks 
display peaks.  The peaks in one set were determined using <em>PeakFinder</em> 
software supplied by NimbleGen.  A false positive rate (FPR) was estimated for 
the peak 
set using a permutation-based method. All peaks had an FPR of &lt; 0.01. The
peaks in the other set (Apr. 2006 update) were identified by ChIPOTle, a 
peak-finding algorithm that uses a sliding window to identify statistically
significant signals that comprise a peak.  A null distribution was determined 
by reflecting the negative data, which is presumed to be noise, about zero and 
a Gaussian distribution was fitted to it.  Windows were considered 
significant with a p-value &lt; 1e-25, after using the Benjamini-Hochberg 
correction for multiple tests.</P>
<P>
This annotation follows the display conventions for composite 
tracks. The subtracks within this annotation 
may be configured in a variety of ways to highlight different aspects of the 
displayed data. The graphical configuration options are shown at the top of 
the track description page, followed by a list of subtracks.  To display only 
one subtrack, uncheck the box next to the track you wish to hide. 
For more information about the graphical configuration options, click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph
configuration help</A> link. Note that the graphical configuration options are 
available only for the Signal subtrack; the Peaks subtracks are fixed. </P>

<H2>Methods</H2>
<P>
To perform FAIRE, proteins were cross-linked to DNA using 
1% formaldehyde solution, the complex was sheared using sonication, and a 
phenol/chloroform extraction was performed to remove DNA fragments 
crosslinked to protein.  The DNA recovered in the aqueous phase was 
fluorescently-labeled and hybridized to a microarray along with 
fluorescently-labeled genomic DNA as a control.  Ratios were scaled by 
subtracting the Tukey Bi-weight mean for the log-ratio values from each 
log-ratio value, as recomended by NimbleGen.  Results in yeast were 
consistent with enrichment for nucleosome-depleted regions of the genome. 
Therefore, the method may have utility as a positive selection for genomic 
regions with properties normally detected by assays like DNAse 
hypersensitivity. </P>

<H2>Verification</H2>
<P>
The data were verified using PCR with primers designed to promoters enriched 
with FAIRE and downstream coding regions. </P> 

<H2>Credits</H2>
<P>
Cell culture, fixing, and DNA amplification were performed by Jonghwan Kim in 
the <A HREF="http://microarray.icmb.utexas.edu/" TARGET=_blank>Vishy Iyer 
lab</A> at the University of Texas, Austin.  FAIRE was done by Paul Giresi in 
the <A HREF="http://www.bio.unc.edu/faculty/lieb/labpages/default.shtml"
TARGET=_blank>Jason Lieb lab</A> at the University of North Carolina at 
Chapel Hill. Paul Giresi of <A HREF="https://sequencing.roche.com/en/products-solutions/by-category/target-enrichment/hybridization.html"
TARGET=_blank>NimbleGen</A> did the sample labeling and hybridization 
with the help of Mike Singer and Roland Green.  Nan Jiang at NimbleGen supplied 
the Software used for the permutation analysis.</P>

<H2>References</H2>
<P>
Buck, M.J., Nobel, A.B., and Lieb, J.D. 
<A HREF="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2005-6-11-r97" TARGET=_blank>ChIPOTle: a 
user-friendly tool for the analysis of ChIP-chip data</A>. <em>Genome Biol.</em>
<B>6</B>(11), R97 (2005).</P>
<P>
Nagy, P.L., Cleary, M.L., Brown, P.O., and Lieb, J.L. 
<A HREF="https://www.pnas.org/content/100/11/6364.full"
TARGET=_blank>Genomewide demarcation of RNA polymerase II transcription units 
revealed by physical fractionation of chromatin</A>. 
<em>PNAS</em> <B>100</B>(11), 6364-9 (2003).</P>
