<H2>Description</H2>
<P>
ENCODE region-wide location analysis of histones H3 and H4 with antibodies
H3K4me2, H3K4me3, H3ac, H4ac and STAT1
was conducted with ChIP-chip, using chromatin extracted from HeLa
cells induced for 30 min with interferon-gamma as well as uninduced cells.
The H3K4me2, H3K4me, and H3ac forms of histone H3 and the H4ac form of 
histone H4 are associated with up-regulation of gene expression. STAT1
(signal transducer and activator of transcription) binds to DNA and activates
transcription in response to various cytokines, including interferon-gamma.
</P>

<H2>Display Conventions and Configuration</H2>
<P>
This annotation follows the display conventions for composite 
&quot;wiggle&quot; tracks. The subtracks within this annotation 
may be configured in a variety of ways to highlight different aspects of the 
displayed data. The graphical configuration options are shown at the top of 
the track description page, followed by a list of subtracks.  To display only 
selected subtracks, uncheck the boxes next to the tracks you wish to hide. 
For more information about the graphical configuration options, click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph
configuration help</A> link.</P>

<H2>Methods</H2>
<P>
Chromatin from both induced and uninduced cells was separately cross-linked,
precipitated with the antibodies, sheared, amplified and hybridized
to a PCR DNA tiling array produced at the Ren Lab at UC San Diego. The array
was composed of 24,537 non-repetitive sequences within the 44 ENCODE regions. </P>
<P>
Each state had three or more biological replicates. Each experiment
was loess-normalized using R. The P-value and R-value were calculated using 
the modified single array error model (Li, Z. <em>et al</em>., 2003). The 
P-value and R-value were then derived from the weighted average results of the
replicates. </P>
<P>
The displayed values were scaled to 0 - 16, corresponding to negative log base
10 of the P-value.</P>

<H2>Verification</H2>
<P>
There were three biological replicates for each of the two experiments.</P>
<P>
The 
<A HREF=" https://www.ncbi.nlm.nih.gov/projects/geo/query/acc.cgi?acc=GPL1454&view=quick"
TARGET=_blank>array platform</A>, the 
<A HREF="https://www.ncbi.nlm.nih.gov/projects/geo/query/acc.cgi?acc=GSE2730&view=quick"
TARGET=_blank>raw and normalized data</A> for each experiment, and the
<A HREF="ftp://ftp.ncbi.nih.gov/pub/geo/data/geo/raw_data/series/GSE1778"
TARGET=_blank>image files</A> have all been deposited at the NCBI 
<A HREF="https://www.ncbi.nlm.nih.gov/geo/" TARGET=_blank>GEO Microarray 
Database</A> (pending approval).</P>

<H2>Credits</H2>
<P>
The data for this track were generated at the 
<A HREF="http://licr-renlab.ucsd.edu" TARGET=_blank>Ren Lab</A>, Ludwig 
Institute for Cancer Research at UC San Diego. </P>

<H2>References</H2>
<P>
Kim, T., Barrera, L.O., Qu, C., van Calcar, S., Trinklein, N.,
Cooper, S., Luna, R., Glass, C.K., Rosenfeld, M.G.,
Myers, R., Ren, B.
<A TARGET=_BLANK HREF="http://www.genome.org/cgi/content/abstract/15/6/830">
Direct isolation and identification of promoters in the human genome.</A>
<em>Genome Research</em> <B>15</B>,830-839 (2005).</P>
<P>
Li, Z., Van Calcar, S., Qu, C., Cavenee, W.K., Zhang, M.Z., and Ren, B. 
<A HREF="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=12808131" 
TARGET=_blank>A global transcriptional regulatory role for c-Myc in Burkitt's 
lymphoma cells</A>.
<em>Proc. Natl. Acad. Sci.</em> <B>100</B>(14), 8164-8169 (2003).</P>
<P>
Ren, B., Robert, F., Wyrick, J. W., Aparicio, O., Jennings, E.
G., Simon, I., Zeitlinger, J., Schreiber, J., Hannett, N., Kanin, E.,
Volkert , T. L., Wilson, C., Bell, S. P. and Young, R. A. 
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11125145" 
TARGET=_blank>Genome-wide location and function of DNA-associated proteins</A>
<em>Science</em> <B>290</B>(5500), 2306-2309 (2000).</P>
