<H2>Description</H2>
<P>
This track shows depth sequencing coverage for the four HapMap populations in 
the ten ENCODE regions that have been resequenced for variation. The data for
each population is shown in a separate subtrack:
<UL>
<LI>HapMap Allele Frequencies (CEU): Utah residents with ancestry from northern 
and western Europe
<LI>HapMap Allele Frequencies (CHB): Han Chinese in Beijing, China
<LI>HapMap Allele Frequencies (JPT): Japanese in Tokyo, Japan
<LI>HapMap Allele Frequencies (YRI): Yoruba in Ibadan, Nigeria
</UL></P>
<P>
The ENCODE regions targeted in this annotation include: 
<UL>
<LI>ENr112 (chr2)
<LI>ENr131 (chr2)
<LI>ENr113 (chr4)
<LI>ENm010 (chr7)
<LI>ENm013 (chr7)
<LI>ENm014 (chr7)
<LI>ENr321 (chr8)
<LI>ENr232 (chr9)
<LI>ENr123 (chr12)
<LI>ENr213 (chr18)
</UL></P>

<H2>Display Conventions and Configuration</H2>
<P>
The subtracks within this annotation 
may be configured in a variety of ways to highlight different aspects of the
displayed data. The graphical configuration options are shown at the top of
the track description page, followed by a list of subtracks.  To display only
selected subtracks, uncheck the boxes next to the tracks you wish to hide. 
For more information about the graphical configuration options, click the
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph
configuration help</A> link.</P>

<H2>Methods</H2>
<P>
Each data value represents the number of sequencing traces that covered the 
nucleotide.
See the <A HREF="https://www.ncbi.nlm.nih.gov/variation/news/NCBI_retiring_HapMap/" TARGET=_blank>International HapMap 
Project</A> website for information about how these data were collected and
analyzed. </P>

<H2>Credits</H2>
<P>
These data were obtained from HapMap public release 16c.1. Thanks to the
International HapMap Project for making this information available.</P>
