<H2>Overview</H2>
<P>
This super-track combines related tracks from the
ENCODE Gene Annotation Assessment Project (EGASP)
<A HREF="https://genome.crg.es/gencode/workshop2005.html" TARGET=_BLANK>
2005 Gene Prediction Workshop</A>. 
The goal of the workshop was to evaluate automatic methods for
gene annotation of the human genome, with a focus on protein-coding genes.
Predictions were evaluated in terms of their ability to
reproduce the high-quality manually assisted GENCODE gene annotations 
and to predict novel transcripts.
<P>
The <B>EGASP Full</B> track shows gene predictions covering all
44 ENCODE regions submitted before the GENCODE annotations were released.
The <B>EGASP Partial</B> track shows gene predictions that cover
some of the ENCODE regions, submitted before the GENCODE release.
The <B>EGASP Update</B> track shows gene predictions that cover
all ENCODE regions, submitted after the GENCODE release.
<P>
These annotations were originally produced using the hg17 assembly. </P>
<P>
The following gene predictions are included:
<UL>
<LI>
<A HREF="http://hollywood.mit.edu/acescan/" TARGET=_blank>ACEScan</A></LI>
<LI>
<A HREF="https://www.ncbi.nlm.nih.gov/ieb/research/acembly/"
TARGET=_blank>AceView</A></LI>
<LI>
DOGFISH-C</LI>
<LI>
<A HREF="http://www.ensembl.org" TARGET=_blank>Ensembl</A></LI>
<LI>
<A HREF="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2006-7-s1-s7"
TARGET=_blank>Exogean</A></LI>
<LI>
<A HREF="http://compbio.fmph.uniba.sk/exonhunter/"
TARGET=_blank>ExonHunter</A></LI>
<LI>
<A HREF="http://linux1.softberry.com/berry.phtml" TARGET=_blank>Fgenesh Pseudogenes</A></LI>
<LI>
<A HREF="http://linux1.softberry.com/berry.phtml" TARGET=_blank>Fgenesh++</A></LI>
<LI>
<A HREF="https://genome.crg.es/software/geneid/index.html"
TARGET=_blank>GeneID-U12</A> </LI>
<LI>
<A HREF="http://opal.biology.gatech.edu/GeneMark/"
TARGET=_blank>GeneMark</A></LI>
<LI>
<A HREF="http://www.genezilla.org/" TARGET=_blank>GeneZilla</A></LI>
<LI>
<A HREF="http://www.cbcb.umd.edu/software/jigsaw/" TARGET=_blank>JIGSAW</A></LI>
<LI>
<A HREF="https://mblab.wustl.edu/software/" TARGET=_blank>Pairagon/N-SCAN</A></LI>
<LI>
SAGA</LI>
<LI>
<A HREF="https://genome.crg.es/software/sgp2/index.html"
TARGET=_blank>SGP2-U12</A></LI>
<LI>
SPIDA</LI>
<LI>
<A HREF="https://mblab.wustl.edu/" TARGET=_blank>Twinscan-MARS</A></LI>
<LI>
<A HREF="http://www.pseudogenes.org" TARGET=_blank>Yale pseudogenes</A></LI>
</UL>
</P>

<H2>Credits</H2>
<P>
Click <A HREF="https://genome.crg.es/gencode/participants.html"
TARGET=_blank>here</A> for a complete list of people who participated in the
GENCODE project. </P>
<P>
The following individuals and institutions provided the data for the subtracks
in this annotation:
<UL>
<LI>
<B>AceView:</B> Danielle and Jean Thierry-Mieg,
<A HREF="https://www.ncbi.nlm.nih.gov/" TARGET=_blank>NCBI</A>, National
Institutes of Health.
<LI>
<B>DOGFISH-C:</B> David Carter, Informatics Dept.,
<A HREF="https://www.sanger.ac.uk/" TARGET=_blank>Wellcome Trust Sanger
Institute</A>.
<LI>
<B>Ensembl:</B> Stephen Searle, Wellcome Trust Sanger Institute (joint
<A HREF="https://www.sanger.ac.uk/"
TARGET=_blank>Sanger</A>/<A HREF="https://www.ebi.ac.uk/"
TARGET=_blank>EBI</A> project).
<LI>
<B>Exogean:</B> Sarah Djebali, Dyogen Lab,
<A HREF="https://www.ens.psl.eu/?lang=en" TARGET=_blank>Ecole Normale
Supérieure</A> (Paris, France).
<LI>
<B>ExonHunter:</B> Tomas Vinar, <A HREF="https://cs.uwaterloo.ca/research/research-areas/bioinformatics"
TARGET=_blank>Waterloo Bioinformatics</A>, School of Computer Science,
University of Waterloo.
<LI>
<B>Fgenesh, Fgenesh++:</B> Victor Solovyev,
<A HREF="https://www.royalholloway.ac.uk/computerscience/home.aspx"
TARGET=_blank>Department of Computer Science</A>,
Royal Holloway, London University.
<LI>
<B>GeneID-U12, SGP2-U12:</B> Tyler Alioto,
Grup de Recerca en Informàtica Biomèdica
(<A HREF="http://grib.imim.es" TARGET=_blank>GRIB</A>) at
the Institut Municipal d'Investigació Mèdica (IMIM), Barcelona.
<LI>
<B>GeneMark:</B> Mark Borodovsky, Alex Lomsadze and Alexander Lukashin,
<A HREF="https://biosciences.gatech.edu/" TARGET=_blank>Department of
Biology</A>, Georgia Institute of Technology.
<LI>
<B>JIGSAW:</B>  Jonathan Allen,  Steven Salzberg group, The Institute for
Genomic Research (<A HREF="https://www.jcvi.org/" TARGET=_blank>TIGR</A>)
and the Center for Bioinformatics and Computational Biology
(<A HREF="http://www.cbcb.umd.edu" TARGET=_blank>CBCB</A>) at the
University of Maryland, College Park.
<LI>
<B>Pairagon/N-SCAN:</B> Randall Brown, <A HREF="https://mblab.wustl.edu/"
TARGET=_blank>Laboratory for Computational Genomics</A>, Washington University
in St. Louis.
<LI>
<B>SPIDA:</B> Damian Keefe, Birney Group, <A HREF="https://www.ebi.ac.uk/"
TARGET=_blank>EMBL-EBI</A>.
<LI>
<B>Twinscan:</B> Paul Flicek, Brent Lab,
<A HREF="https://mblab.wustl.edu/" TARGET=_blank>Washington University
in St. Louis</A>.
<LI>
<B>ACEScan:</B> Gene Yeo, Crick-Jacobs Center for Computational Biology,
<A HREF="https://www.salk.edu/" TARGET=_blank>Salk Institute</A>.
<LI>
<B>Augustus:</B> Mario Stanke, <A HREF="http://gobics.de/department/"
TARGET=_blank>Department of Bioinformatics</A>, University of Göttingen,
Germany.
<LI>
<B>GeneZilla:</B> William Majoros, Dept. of Bioinformatics, The Institute for
Genomic Research (<A HREF="https://www.jcvi.org/" TARGET=_blank>TIGR</A>).
<LI>
<B>SAGA:</B> Sourav Chatterji, Lior Pachter lab,
<A HREF="https://pachterlab.github.io/group.html" TARGET=_blank>Department of Mathematics</A>, U.C. Berkeley.
</UL></P>

<H2>References</H2>
<P>
Ashurst JL, Chen CK, Gilbert JG, Jekosch K, Keenan S, Meidl P, Searle SM,
Stalker J, Storey R, Trevanion S <em>et al</em>.
<A HREF="https://academic.oup.com/nar/article/33/suppl_1/D459/2505459"
TARGET=_blank>The Vertebrate Genome Annotation (Vega) database</A>.
<em>Nucleic Acids Res</em>. 2005 Jan 1;33(Database issue):D459-65. </P>
<P>
Guigo R, Dermitzakis ET, Agarwal P, Ponting CP, Parra G, Reymond A, Abril JF,
Keibler E, Lyle R, Ucla C <em>et al</em>.
<A HREF="https://www.pnas.org/content/100/3/1140.full"
TARGET=_blank>Comparison of mouse and human genomes followed by experimental
verification yields an estimated 1,019 additional genes</A>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Feb 4;100(3):1140-5.
</P>
<P>
Mouse Genome Sequencing Consortium.
<A HREF="https://www.nature.com/articles/nature01262"
TARGET=_blank>Initial sequencing and comparative analysis of the mouse
genome</A>. <em>Nature</em>. 2002 Dec 5;420(6915):520-62. </P>
<P>
Reymond A, Marigo V, Yaylaoglu MB, Leoni A, Ucla C, Scamuffa N, Caccioppoli C, Dermitzakis ET, Lyle R, Banfi S <em>et al</em>.
<A HREF="https://www.nature.com/articles/nature01178"
TARGET=_blank>Human chromosome 21 gene expression atlas in the mouse</A>.
<em>Nature</em>. 2002 Dec 5;420(6915):582-6. </P>
<P>
Reymond A, Camargo AA, Deutsch S, Stevenson BJ, Parmigiani RB, Ucla C, Bettoni F, Rossier C, Lyle R, Guipponi M <em>et al</em>.
<A HREF="https://www.sciencedirect.com/science/article/pii/S0888754302967819"
TARGET=_blank>Nineteen additional unpredicted transcripts from human
chromosome 21</A>. <em>Genomics</em>. 2002 Jun;79(6):824-32.</P>
<P>
Chatterji S, Pachter L.
<A HREF="https://dl.acm.org/citation.cfm?id=974639&jmp=cit&coll=GUIDE&dl=GUIDE"
TARGET=_blank>Multiple organism gene finding by collapsed Gibbs sampling</A>.
<em>J Comput Biol</em>. 2005 Jul-Aug;12(6):599-608. </P>
<P>
Siepel A, Haussler D.
<A HREF="https://dl.acm.org/citation.cfm?id=974638&coll=GUIDE&dl=GUIDE"
TARGET=_blank>Computational identification of evolutionarily conserved
exons</A>.
<em>Proc. 8th Int'l Conf. on Research in Computational Molecular Biology</em>.
2004;177-186.</P>

<H3>Augustus</H3>
<P>
Stanke M, Waack S.
<A HREF="https://academic.oup.com/bioinformatics/article/19/suppl_2/ii215/180603"
TARGET=_blank>Gene prediction with a hidden Markov model and a new intron
submodel</A>.
<em>Bioinformatics</em>. 2003;19(Suppl. 2):ii215-ii225.</P>
<P>
Stanke M, Steinkamp R, Waack S, Morgenstern B.
<A HREF="https://academic.oup.com/nar/article/32/suppl_2/W309/1040489"
TARGET=_blank>AUGUSTUS: a web server for gene finding in eukaryotes</A>.
<em>Nucleic Acids Res</em>. 2004 Jul 1;32(Web Server issue):W309-12.</P>

<H3>FGenesh++</H3>
<P>
Solovyev VV.
&quot;Statistical approaches in Eukaryotic gene prediction&quot;.
In <em>Handbook of Statistical Genetics</em> (eds. Balding D <em>et al</em>.)
(John Wiley & Sons, Inc., 2001). p. 83-127.</P>

<H3>GeneID</H3>
<P>
Blanco E, Parra G, Guigó R.
&quot;Using geneid to identify genes&quot;.
In <em>Current Protocols in Bioinformatics</em>, Unit 4.3. (eds. Baxevanis AD.)
(John Wiley & Sons, Inc., 2002).</P>
<P>
Guigó R.
<A HREF="https://www.liebertpub.com/doi/abs/10.1089/cmb.1998.5.681"
TARGET=_blank>Assembling genes from predicted exons in linear time with
dynamic programming</A>.
<em>J Comput Biol</em>. 1998 Winter;5(4):681-702. </P>
<P>
Guigó R, Knudsen S, Drake N, Smith T.
<A HREF="https://www.sciencedirect.com/science/article/pii/002228369290130C"
TARGET=_blank>Prediction of gene structure</A>.
<em>J Mol Biol</em>. 1992 Jul 5;226(1):141-57.</P>
<P>
Parra G, Blanco E, Guigó R.
<A HREF="https://genome.cshlp.org/content/10/4/511.full"
TARGET=_blank>GeneID in <em>Drosophila</em></A>.
<em>Genome Res</em>. 2000 Apr;10(4):511-5. </P>

<H3>JIGSAW</H3>
<P>
Allen JE, Pertea M, Salzberg SL.
<A HREF="https://genome.cshlp.org/content/14/1/142.full"
TARGET=_blank>Computational gene prediction using multiple sources of
evidence</A>.
<em>Genome Res</em>. 2004 Jan;14(1):142-8.</P>
<P>
Allen JE, Salzberg SL.
<A HREF="https://academic.oup.com/bioinformatics/article/21/18/3596/202486"
TARGET=_blank>JIGSAW: integration of multiple sources of evidence for gene
prediction</A>.
<em>Bioinformatics</em>. 2005 Sep 15;21(18):3596-603.</P>

<H3>SGP2</H3>
<P>
Guigó R, Dermitzakis ET, Agarwal P, Ponting CP, Parra G,
Reymond A, Abril JF, Keibler E, Lyle R, Ucla C <em>et al</em>.
<A HREF="https://www.pnas.org/content/100/3/1140.full"
TARGET=_blank>Comparison of mouse and human genomes followed by experimental
verification yields an estimated 1,019 additional genes</A>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Feb 4;100(3):1140-5.</P>
<P>
Parra G, Agarwal P, Abril JF, Wiehe T, Fickett JW, Guigó R.
<A HREF="https://genome.cshlp.org/content/13/1/108.full"
TARGET=_blank>Comparative gene prediction in human and mouse</A>.
<em>Genome Res</em>. 2003 Jan;13(1):108-17. </P>

