<H2>Description</H2>
<P>
This track shows deletion/insertion polymorphisms (DIPs). In packed and full 
modes, the sequence variation is shown to the left of the DIP. 
The naming convention &quot;-/<em>sequence</em>&quot; is used for deletions; 
&quot;<em>sequence</em>/-&quot; is used for insertions.  The details
page shows the name of the trace used to define the polymorphism, the
quality score, and the strand on which the trace aligns to the reference
sequence.</P>
<P>
The quality score reflects the minimum PHRED quality value over 
the entire range of the DIP within the trace, plus 5 flanking bases. 
PHRED quality scores are 
expressed as log probabilities using the formula: 
<PRE>  Q = -10 * log<sub>10</sub>(P<sub>e</sub>)
</PRE>
where  P<sub>e</sub> is the estimated probability of an 
error at that base. PHRED quality scores typically vary from 0 to 40, where 0 
indicates complete uncertainty about the base and 40 implies odds of 10,000 
to 1 that the base is correct. Sometimes a PHRED value of 50 or higher is 
used to denote finished sequence.  A color gradient is used to distinguish
quality scores in the browser display: brighter shading indicates higher 
scores.</P>
<P>
The &quot;Trace Pos&quot; value on the details page indicates the 3' position 
of the DIP within the trace. The alleles are 
reported relative to the &quot;+&quot; strand of the reference sequence; 
however, the trace may actually align to the &quot;-&quot; strand. 
When viewing the chromatogram using the URL provided, 
if the trace aligned to the &quot;-&quot; strand, the DIP bases in the trace 
will be the reverse compliment of the variant allele given.</P>

<H2>Methods</H2>
<P>
All human trace data from NCBI's trace archive were aligned to hg17 with 
ssahaSNP, followed by ssahaDIP post-processing to detect deletion/insertion 
polymorphisms. DIPs within ENCODE regions were extracted.</P>

<H2>Verification</H2>
<P>
For verification, 500k traces from the mouse whole genome shotgun (WGS) 
<A HREF="ftp://ftp.ncbi.nih.gov/pub/TraceDB/mus_musculus/fasta.mus_musculus.045.gz"
TARGET=_blank>sequencing effort</A> were compared to mm6 using ssahaSNP and 
ssahaDIP. Because mm6 and these traces are from the same mouse strain, 
C57BL/6J, the DIP rate should be very low. Applying a quality threshold of 
Q23, the detected DIP rate was one DIP per 140k Neighborhood Quality Standard 
(NQS) bases.  This level was ten-fold lower than the SNP rate for the same 
data set using ssahaSNP, which has been validated as having a 5% false positive rate. 
The detected DIP rate for human traces against hg17 is one DIP per 12k NQS 
bases, indicating a false positive rate of 12k/140k, or about 8%.</P>
<P>
Further validation experiments are in progress.</P>

<H2>Credits</H2>
<P>
All analyses were performed by Jim Mullikin using ssahaSNP and ssahaDIP. 
The trace data were contributed to the trace archive by many sequencing 
centers.</P>

<H2>References</H2>
<P>
Ning Z, Cox AJ, Mullikin JC. 
<A HREF="https://genome.cshlp.org/content/11/10/1725"
TARGET=_blank>SSAHA: A fast search method for large DNA databases</A>.
<em>Genome Res.</em> 2001 Oct;11(10):1725-9. </P> 
<P>
The International SNP Map Working Group. 
<A HREF="https://www.nature.com/articles/35057149"
TARGET=_blank>A map of human genome sequence variation containing 
1.4 million single nucleotide polymorphisms</A>.
<em>Nature.</em> 2001 Feb 15;409(6822):928-33.</P>
