<H2>Description</H2>
<P>
This track shows updated versions of gene predictions submitted for the 
ENCODE Gene Annotation Assessment Project 
(<A HREF="https://genome.crg.es/gencode/workshop2005.html"
TARGET=_blank>EGASP</A>) Gene Prediction Workshop 2005.
The following gene predictions are included:
<UL>
<LI>
<A HREF="http://augustus.gobics.de/" TARGET=_blank>Augustus</A></LI>
<LI> Exogean</LI>
<LI>
<A HREF="http://www.softberry.com/berry.phtml?topic=index&group=programs&subgroup=gfind"
TARGET=_blank>FGenesh++</A></LI>
<LI>
<A HREF="https://genome.crg.es/software/geneid/index.html" 
TARGET=_blank>GeneID</A>-U12 </LI>
<LI>
<A HREF="http://www.cbcb.umd.edu/software/jigsaw/" TARGET=_blank>Jigsaw</A></LI>
<LI>
<A HREF="https://genome.crg.es/software/sgp2/index.html" 
TARGET=_blank>SGP2</A>-U12</LI>
<LI>
<A HREF="http://www.pseudogenes.org" TARGET=_blank>Yale pseudogenes</A></LI>
</UL>
The original EGASP submissions are displayed in the companion tracks, 
EGASP Full and EGASP Partial.</P>

<H2>Display Conventions and Configuration</H2>
<P>
Data for each gene prediction method within this composite annotation track 
are displayed in separate subtracks. See the top of the track description page 
for a complete list of the subtracks available for this annotation. To display
only selected subtracks, uncheck the boxes next to the tracks you wish to
hide.  
<P>
The individual subtracks within this annotation follow the display conventions 
for <A HREF="../goldenPath/help/hgTracksHelp.html#GeneDisplay">gene prediction
tracks</A>. Display characteristics specific to individual subtracks are 
described in the Methods section. The track description page offers the option 
to color and label codons in a zoomed-in display of the subtracks to facilitate 
validation and comparison of gene predictions. To enable this feature, select 
the <em>genomic codons</em> option from the &quot;Color track by codons&quot;
menu. Click the
<A HREF="../goldenPath/help/hgCodonColoring.html">Help on codon coloring</A>
link for more information about this feature. </P>
<P>
Color differences among the subtracks are arbitrary. They provide a
visual cue for distinguishing the different gene prediction methods.</P>

<H2>Methods</H2>

<H3>Augustus</H3>
<P>
Augustus uses a generalized hidden Markov model (GHMM) that models 
coding and non-coding sequence, splice sites, the branch point region, 
the translation start and end, and the lengths of exons and introns. 
This version has been trained on a set of 1284 human genes.
The track contains four sets of predictions: <em>ab initio</em>,
EST and protein-based, mouse homology-based, and those using
EST/protein and mouse homology evidence as additional input to Augustus
for the predictions.</P>
<P>
The EST and protein evidence was generated by aligning sequences from the dbEST 
and nr databases to the ENCODE region using wublastn and wublastx.
The resulting alignments were used to generate hints about putative splice 
sites, exons, coding regions, introns, translation start and 
translation stop.</P>
<P>
The mouse homology evidence was generated by aligning pairs of human and
mouse genomic sequences using the program 
<A HREF="http://dialign.gobics.de/chaos-dialign-submission"
TARGET=_blank>DIALIGN</A>. Regions conserved at the peptide level were used to 
generate hints about coding regions.</P>

<H3>Exogean</H3>
<P>
Exogean produces alternative transcripts by combining mRNA and cross-species 
sequence alignments using heuristic rules. The program implements a generic 
framework based on directed acyclic colored multigraphs (DACMs). In Exogean, 
DACM nodes represent biological objects (mRNA or protein HSPs/transcripts) and
multiple edges between nodes represent known relationships between these 
objects derived from human expertise. Exogean DACMs are succesively built and 
reduced, leading to increasingly complex objects. This process
enables the production of alternative transcripts from initial HSPs.</P>

<H3>FGenesh++</H3>
<P>
FGenesh++ predictions are based on hidden Markov models and protein similarity to
the NR database.  For more information, see the reference below.

<H3>GeneID-U12</H3>
<P>
The GeneID program predicts genes in anonymous genomic sequences 
designed with a hierarchical structure.
In the first step, splice sites, start and stop codons are predicted and scored 
along the sequence using position weight arrays (PWAs).
Next, exons are built from the sites. Exons are scored as the sum of the scores 
of the defining sites plus the the log-likelihood ratio of a Markov model for 
coding DNA.
Finally, the gene structure is assembled from the set of predicted exons, 
maximizing the sum of the scores of the assembled exons.
The modified version of GeneID used to generate the predictions in this track 
incorporates models for U12-dependent splice signals in addition to U2 splice 
signals.</P>
<P>
The GeneID subtrack shows all GeneID genes. Only U12 introns
and their flanking exons are displayed in the GeneID U12 subtrack.
Exons flanking predicted U12-dependent introns are assigned a type
attribute reflecting their splice sites, displayed on
the details page of the GeneID U12 subtrack as the &quot;Alternate Name&quot; 
of the item composed of the intron plus flanking exons.</P>

<H3>Jigsaw</H3>
<P>
Jigsaw is a gene prediction program that determines genes based on 
target genomic sequence and output from a gene structure annotation database.
Data downloaded from UCSC's annotation database is 
used as input and includes the following tracks of evidence:
Known Genes, Ensembl, RefSeq, GeneID, Genscan, SGP, Twinscan, Human mRNAs,
TIGR Gene Index, UniGene, Most Conserved Elements and Non-human RefSeq Genes.
<A HREF="http://ccb.jhu.edu/software/glimmerhmm/"
TARGET=_blank>GlimmerHMM</A> and <A HREF="http://genezilla.org"
TARGET=_blank>GeneZilla</A>, two open source <em>ab initio</em> gene-finding 
programs based on GHMMs, are also used.</P>

<H3>SGP2-U12</H3>
<P>
To predict genes in a genomic query, SGP2 combines GeneID predictions with 
tblastx comparisons of the genomic query against other genomic sequences.
This modified version of SGP2 uses models for U12-dependent splice signals 
in addition to U2 splice signals. The reference genomic sequence for this data 
set is the Oct. 2004 release of mouse sequence syntenic to ENCODE regions.</P>
<P>
The SGP2 and SGP2 U12 tracks follow the same display conventions as the 
GeneID and GeneID U12 subtracks described above.</P>

<H3>Yale Pseudogenes</H3>
<P>
For this analysis, pseudogenes were defined as genomic sequences similar 
to known human genes and with various disablements (premature stop codons or
frameshifts) in their &quot;putative&quot; protein-coding regions.</P>
<P>
The protein sequences of known human genes (as annotated by ENSEMBL) were used
to search for similar nongenic sequences in ENCODE regions.  The matching
sequences were assessed as disabled copies of genes based on the occurrences of
premature stop codons or frameshifts.  The intron-exon structure of the
functional gene was further used to infer whether a pseudogene was duplicated
or processed (a duplicated pseudogene keeps the intron-exon structure of its
parent functional gene). Small pseudogene sequences were labeled as fragments or
other types.</P>
<P>
All pseudogenes in this track were manually curated.
In the browser, the track details page shows the pseudogene type.</P>

<H2>Credits</H2>
<P>
Augustus was written by Mario Stanke at the
<A HREF="http://gobics.de/department/" TARGET=_blank>Department of 
Bioinformatics</A> of the University of Göttingen in Germany.</P>
<P>
Exogean was developed by Sarah Djebali and Hugues Roest Crollius from the
Dyogen Lab, <A HREF="https://www.ens.psl.eu/" TARGET=_blank>Ecole 
Normale Supérieure</A> (Paris, France) and Franck Delaplace
from the Laboratoire de Méthodes Informatiques 
(<A HREF="https://www.sigles.net/sigle/lami-laboratoire-des-methodes-informatiques" TARGET=_blank>LaMI</A>), (Evry, 
France).</P>
<P>
The FGenesh++ gene predictions were provided by Victor Solovyev of
<A HREF="http://www.softberry.com/" TARGET=_blank>Softberry Inc.</A>
<P>
The GeneID-U12 and SGP2-U12 programs were developed by the
Grup de Recerca en Informàtica Biomèdica 
(<A HREF="http://grib.imim.es" TARGET=_blank>GRIB</A>) at 
the Institut Municipal d'Investigació Mèdica (IMIM) in Barcelona.
The version of GeneID on which GeneID-U12 is based (geneid_v1.2) was written by 
Enrique Blanco and Roderic Guigó.
The parameter files were constructed by Genis Parra and Francisco Camara.
Additional contributions were made by Josep F. Abril, Moises Burset and Xavier 
Messeguer. Modifications to GeneID that allow for the prediction of 
U12-dependent splice sites and incorporation of U12 introns into gene models 
were made by Tyler Alioto.</P>
<P>
Jigsaw was developed at The Institute for Genomic Research 
(<A HREF="https://www.jcvi.org/" TARGET=_blank>TIGR</A>)
by Jonathan Allen and Steven Salzberg,
with computational gene-finder contributions from Mihaela Pertea and William 
Majoros.  Continued maintenance and development of Jigsaw will
be provided by the Salzberg group at the Center for Bioinformatics 
and Computational Biology 
(<A HREF="http://www.cbcb.umd.edu" TARGET=_blank>CBCB</A>) at the 
University of Maryland, College Park.</P>
<P>
The Yale Pseudogenes were generated by the pseudogene annotation group of 
<A HREF="http://bioinfo.mbb.yale.edu/" TARGET=_blank>Mark Gerstein</A> at Yale 
University.</P>

<H2>References</H2>

<H3>Augustus</H3>
<P>
Stanke, M. 
<A HREF="https://ediss.uni-goettingen.de/handle/11858/00-1735-0000-0006-B3F8-4"
TARGET=_blank>Gene prediction with a hidden Markov model</A>.
<em>Ph.D. thesis</em>, Universit&auml;t G&ouml;ttingen, Germany (2004).</P>
<P>
Stanke, M. and Waack, S. 
<A HREF="https://academic.oup.com/bioinformatics/article/19/suppl_2/ii215/180603"
TARGET=_blank>Gene prediction with a hidden Markov model and a new intron 
submodel</A>.
<em>Bioinformatics</em>, <B>19</B>(Suppl. 2), ii215-ii225 (2003).</P>
<P>
Stanke, M., Steinkamp, R., Waack, S. and Morgenstern, B. 
<A HREF="https://academic.oup.com/nar/article/32/suppl_2/W309/1040489"
TARGET=_blank>AUGUSTUS: a web server for gene finding in eukaryotes</A>.
<em>Nucl. Acids Res.</em>, <B>32</B>, W309-W312 (2004).</P>

<H3>FGenesh++</H3>
<P>
Solovyev V.V. 
&quot;Statistical approaches in Eukaryotic gene prediction&quot;.
In <em>Handbook of Statistical Genetics</em> (eds. Balding D. et al.)
(John Wiley & Sons, Inc., 2001). p. 83-127.</P>

<H3>GeneID</H3>
<P>
Blanco, E., Parra, G.  and Guigó, R. 
&quot;Using geneid to identify genes&quot;. 
In <em>Current Protocols in Bioinformatics</em>, Unit 4.3. (ed. Baxevanis, A.D.)
(John Wiley & Sons, Inc., 2002).</P>
<P>
Guigó, R. 
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=10072084&dopt=Abstract"
TARGET=_blank>Assembling genes from predicted exons in linear time with 
dynamic programming</A>. 
J Comput Biol. <B>5</B>(4), 681-702 (1998).</P>
<P>
Guigó, R., Knudsen, S., Drake, N. and Smith, T.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=1619647&query_hl=5"
TARGET=_blank>Prediction of gene structure</A>. 
J Mol Biol. <B>226</B>(1), 141-57 (1992).</P>
<P>
Parra, G., Blanco, E. and Guigó, R. 
<A HREF="https://genome.cshlp.org/content/10/4/511.full"
TARGET=_blank>GeneID in <em>Drosophila</em></A>. 
<em>Genome Research</em> <B>10</B>(4), 511-515 (2000).</P>

<H3>Jigsaw</H3>
<P>
Allen, J.E., Pertea, M.  and Salzberg, S.L.
<A HREF="https://genome.cshlp.org/content/14/1/142.full"
TARGET=_blank>Computational gene prediction using multiple sources of 
evidence</A>. 
<em>Genome Res.</em>, <B>14</B>(1), 142-8 (2004). </P> 
<P>
Allen, J.E. and Salzberg, S.L.
<A HREF="https://academic.oup.com/bioinformatics/article/21/18/3596/202486"
TARGET=_blank>JIGSAW: integration of multiple sources of evidence for gene 
prediction</A>.
<em>Bioinformatics</em> <B>21</B>(18), 3596-3603 (2005).</P>

<H3>SGP2</H3>
<P>
Guigó, R., Dermitzakis, E.T., Agarwal, P., Ponting, C.P., Parra, G., 
Reymond, A., Abril, J.F., Keibler, E., Lyle, R., Ucla, C. <em>et al</em>. 
<A HREF="https://www.pnas.org/content/100/3/1140.full"
TARGET=_blank>Comparison of mouse and human genomes followed by experimental 
verification yields an estimated 1,019 additional genes</A>. 
<em>Proc Natl Acad Sci U S A</em> <B>100</B>(3), 1140-5 (2003).</P>
<P>
Parra, G., Agarwal, P., Abril, J.F., Wiehe, T., Fickett, J.W. and Guigó, R. 
<A HREF="https://genome.cshlp.org/content/13/1/108.full"
TARGET=_blank>Comparative gene prediction in human and mouse</A>. 
<em>Genome Res.</em> <B>13</B>(1), 108-17 (2003). </P>
