<H2>Description</H2>
<P>
The Gencode Genes track (v3.1, March 2007) shows high-quality manual
annotations in the ENCODE regions generated by the
<A HREF="https://genome.crg.es/gencode/" TARGET="_blank">GENCODE project</a>.
</P>
<P>
The gene annotations are colored based on the HAVANA annotation type. See the 
table below for the color key, as well as more detail about the transcript
and feature types.  The Gencode project recommends that the annotations
with known and validated transcripts; i.e., the types <em>Known</em>
and <em>Novel_CDS</em> (which are colored
dark green in the track
display) be used as the reference gene annotation.
</P>
<P>The v3.1 release includes the following updates and enhancements to v2.2 
(Oct. 2005):
<UL>
  <LI>Apart from the usual additions and corrections, 69 loci (consisting of 132
  transcripts) were re-annotated based on Rapid Amplification of cDNA Ends   
  (RACE), array, and sequencing
  analyses performed within the Affymetrix/GENCODE collaboration
  (see the Methods section below, also Denoeud <em>et al</em>., 2007 and The 
  ENCODE Project Consortium, 2007).
  <LI>The polymorphic gene type was added.
  <LI>PolyA features were added.
  <LI>A bug affecting frames of CDSs with missing start or stop codons was 
  fixed.
  <LI>The experimental validation data contained in the Gencode
  Introns track from the previous release were integrated into the
  Gencode Genes track by annotators using the Human and Vertebrate Analysis and 
  Annotation manual curation process (HAVANA). 
</UL>
</P>
<P>
<TABLE border="1" bordercolor="#aaaaaa" cellpadding="4">
  <tbody>
    <tr>
      <th align="left">Type</th>
      <th align="left">Color</th>
      <th align="left">Description</th>
    </tr>
    <tr>
      <td>Known</td>
      <td><font color="#215b30">dark green</font></td>
      <td><b>Known protein-coding genes</b> (<em>i.e.</em>, referenced
	in <A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=gene"
	TARGET=_blank>Entrez Gene</A>)</td>
    </tr>
    <tr>
      <td>Novel_CDS</td>
      <td><font color="#215b30">dark green</font></td>
      <td><b>Have an open reading frame</b> (ORF) and are identical, or
	have homology, to cDNAs or proteins but do not fall into the above
	category. These can be known in the sense that they are represented by
	mRNA sequences in the public databases, but they are <b>not yet 
	represented
	in Entrez Gene</b> or have not received an official gene name. They can 
	also
	be novel in that they are not yet represented by an mRNA sequence in
	human.</td>
    </tr>
    <tr>
      <td>Novel_transcript</td>
      <td><font color="#54bc00">light green</font></td>
      <td>Similar to Novel_CDS; however, <b>cannot be assigned an unambigous 
	ORF</b>.</td>
    </tr>
    <tr>
      <td>Putative</td>
      <td><font color="#54bc00">light green</font></td>
      <td>Have identical, or have homology to <b>spliced ESTs</b>, but
	are <b>devoid of significant ORF and polyA features</b>. These are
	generally short (two or three exon) genes or gene fragments.</td>
    </tr>
    <tr>
      <td>TEC</td>
      <td><font color="#54bc00">light green</font></td>
      <td>(<b><u>T</u>o <u>E</u>xperimentally <u>C</u>onfirm</b>)
	Single-exon objects (supported by multiple unspliced ESTs with polyA
	sites and signals). </td>
    </tr>
    <tr>
      <td>Polymorphic</td>
      <td><font color="#a020f0">purple</font></td>
      <td> Have <b>functional transcripts in one haplotype</b> and <b>"pseudo"
	(non-functional) transcripts in another</b>.</td>
    </tr>
    <tr>
      <td>Processed_pseudogene</td>
      <td><font color="#005bbf"> blue</font></td>
      <td><b>Pseudogenes that lack introns</b> and are thought to arise
	from <b>reverse transcription</b> of mRNA followed by reinsertion of
	DNA into the genome. </td>
    </tr>
    <tr>
      <td>Unprocessed_pseudogene</td>
      <td><font color="#005bbf"> blue</font></td>
      <td><b>Pseudogenes that can contain introns</b>, as they are
	produced by <b>gene duplication</b>.</td>
    </tr>
    <tr>
      <td>Artifact</td>
      <td><font color="#636863">grey</font></td>
      <td><b>Transcript evidence and/or its translation equivocal</b>.
	Usually these arise from high-throughput cDNA sequencing projects that
	submit automatic annotation, sometimes resulting in erroneous CDSs in
	what turns out to be, for example, 3' UTRs. In addition HAVANA has
	extended this category to include cDNAs with non-canonical splice sites
	due to deletion/sequencing errors.</td>
    </tr>
    <tr>
      <td>PolyA_signal</td>
      <td><font color="#5e2605">brown</font></td>
      <td> <b>Polyadenylation signal</b> </td>
    </tr>
    <tr>
      <td>PolyA_site</td>
      <td style="color: rgb(255, 102, 0);">orange</td>
      <td> <b>Polyadenylation site</b> </td>
    </tr>
    <tr>
      <td>Pseudo_polyA</td>
      <td style="color: rgb(255, 153, 255);">pink</td>
      <td> <b>"Pseudo"-polyadenylation signal</b> detected in the sequence
	of a <b>processed pseudogene</b>.<br>
        <B>Warning: </B>
        Pseudo_polyA features and processed_pseudogenes
        generally don't overlap. The reason is that pseudogene annotations are
        based solely on protein evidence, whereas pseudo_polyA signals are
        identified from transcript evidence; as they are found at the end of
        the 3' UTR, they can lie several kb downstream of the 3' end of the
        pseudogene. </td>
    </tr>
  </tbody>
</TABLE>
</P>
<P>
The current full set of GENCODE annotations is available for download 
<A TARGET=_blank 
HREF="ftp://genome.imim.es/pub/projects/gencode/data/havana-encode/current/">here</A>.<BR>
</P>

<H2>Methods</H2>
<P>
For a detailed description of the methods and references used, see Harrow 
<em>et al</em>., 2006 and Denoeud <em>et al</em>., 2007. 

<H3>5' RACE/array experiments</H3>
<P>
A combination of 5&#8217; RACE and 
high-density tiling microarrays were used to empirically annotate 5&#8217; 
transcription start sites (TSSs) and internal exons of all 410 annotated
protein-coding loci across the 44 ENCODE regions (Oct. 2005 GENCODE
freeze). The 5&#8217; RACE reactions were performed with oligonucleotides
mapping to a coding exon common to most of the transcripts of a protein-coding 
gene locus annotated by GENCODE (Oct. 2005 freeze) on polyA+ RNA
from twelve adult human tissues (brain, heart, kidney, spleen, liver,
colon, small intestine, muscle, lung, stomach, testis, placenta) and
three cell lines 
(<A HREF="https://www.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=GM06990&PgId=166"
        TARGET=_blank>GM06990</A> (lymphoblastoid),
<A HREF="https://www.atcc.org/ATCCAdvancedCatalogSearch/ProductDetails/tabid/452/Default.aspx?ATCCNum=CCL-240&Template=cellBiology" 
TARGET=_blank>HL60</A> (acute promyelocytic leukemia) and
<A HREF="https://www.atcc.org/ATCCAdvancedCatalogSearch/ProductDetails/tabid/452/Default.aspx?ATCCNum=CCL-2.2&Template=cellBiology" 
TARGET=_blank>HeLaS3</A> (cervix carcinoma)).</P> 
<P>
The RACE reactions were then hybridized to 20 nucleotide-resolution
Affymetrix tiling arrays covering the non-repeated regions of the 44
ENCODE regions. The resulting &quot;RACEfrags&quot;
-- array-detected fragments of RACE products -- were assessed for
novelty by comparing their genome coordinates to those of
GENCODE-annotated exons. Connectivity between novel RACEfrags and their
respective index exon were further investigated by RT-PCR, cloning and
sequencing. The resulting cDNA sequences (deposited in GenBank under
accession numbers DQ655905-DQ656069 and
EF070113-EF070122) were then fed into the HAVANA annotation pipeline as
mRNA evidence (see &quot;HAVANA manual annotations&quot; below).

<H3>HAVANA manual annotations</H3>
<P>
The <A HREF="https://www.sanger.ac.uk/science/projects/manual-annotation" TARGET=_blank>HAVANA</A>
process was used to produce these annotations.
</P>
<P>
Before the manual annotation process begins, an automated analysis pipeline
for similarity searches and <em>ab initio</em> predictions is run
on a computer farm and stored in an Ensembl MySQL
database using a modified Ensembl analysis pipeline system. All
searches and prediction algorithms, except CpG island prediction (see
cpgreport in the EMBOSS application suite), are run on repeat-masked
sequence. RepeatMasker is used to mask interspersed repeats, followed by Tandem
repeats finder to mask tandem repeats.
<P>
Nucleotide sequence databases are searched with wuBLASTN, and
significant hits are re-aligned to the unmasked genomic sequence using
est2genome.
The UniProt protein database is searched with wuBLASTX, and the
accession numbers of significant hits are found in the Pfam
database. The hidden Markov models for Pfam protein domains are aligned
against the genomic sequence using Genewise to provide annotation of
protein domains. </P>
<P>
Several <em>ab initio</em> prediction algorithms are also run:
Genescan and Fgenesh for genes, tRNAscan to find tRNAgenes and Eponine
TSS to predict transcription start sites.</P>
<P>
Once the automated analysis is complete, the annotator uses a Perl/Tk
based graphical interface, &quot;otterlace&quot;, developed in-house at 
the Wellcome Trust Sanger Institute to edit annotation data held in a 
separate MySQL database system. The interface displays a rich,
interactive graphical view of the genomic region, showing features such as 
database matches, gene predictions, and transcripts created by the
annotators. Gapped alignments of nucleotide and protein blast hits to
the genomic sequence are viewed and explored using the &quot;Blixem&quot;
alignment viewer. </P>
<P>
Additionally, the &quot;Dotter&quot; dot plot tool is used to show the
pairwise alignments of unmasked sequence, thus revealing the location
of exons that are occasionally missed by the automated blast searches
because of their small size and/or match to repeat-masked sequence. </P>
<P>
The interface provides a number of tools that the annotator uses to
build genes and edit annotations: adding transcripts, exon coordinates,
translation regions, gene names and descriptions, remarks and
polyadenlyation signals and sites.</P>

<H2>Verification</H2>
<P>
See Harrow <em>et al</em>., 2006 for information on verification techniques.
</P>

<H2>Credits</H2>
<P>
This GENCODE release is the result of a collaborative effort among
the following laboratories:
<BR><BR>

<TABLE style="text-align: left; width: 100%;" border="1" bordercolor="#aaaaaa" 
cellpadding="2" cellspacing="2">
  <tbody>
    <tr>
      <td style="vertical-align: top; text-align: left;">
	<B>Lab/Institution</B><BR>
      </td>
      <td style="vertical-align: top; text-align: left;"><B>Contributors</B><BR>
      </td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;">HAVANA annotation 
	group, Wellcome Trust Sanger Insitute, Hinxton, UK</td>
      <td style="vertical-align: top;">Adam Frankish, Jonathan Mudge, James 
	Gilbert, Tim Hubbard, Jennifer Harrow<br>
      </td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;">Genome Bioinformatics
	 Lab CRG, Barcelona, Spain</td>
      <td style="vertical-align: top;">France Denoeud, Julien Lagarde, Sylvain 
	Foissac, Robert Castelo, Roderic Guig&oacute; (GENCODE Principal 
	Investigator)</td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;">Department of 
	Genetic Medicine and Development, University of Geneva, Switzerland</td>
      <td style="vertical-align: top;">Catherine Ucla, Carine Wyss,
	Caroline Manzano, Colette Rossier, Stylianos E. Antonorakis</td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;">Center for 
	Integrative Genomics, University of Lausanne, Switzerland</td>
      <td style="vertical-align: top;">Jacqueline Chrast, Charlotte N.
	Henrichsen, Alexandre Reymond</td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: left;">Affymetrix, Inc., 
	Santa Clara, CA, USA</td>
      <td style="vertical-align: top;">Philipp Kapranov, Thomas R. Gingeras</td>
    </tr>
  </tbody>
</TABLE>

<h2>References</h2>

<p>
Denoeud F, Kapranov P, Ucla C, Frankish A, Castelo R, Drenkow J, Lagarde J, Alioto T, Manzano C,
Chrast J <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17567994" target="_blank">
Prominent use of distal 5' transcription start sites and discovery of a large number of additional
exons in ENCODE regions</a>.
<em>Genome Res</em>. 2007 Jun;17(6):746-59.
</p>

<p>
Harrow J, Denoeud F, Frankish A, Reymond A, Chen CK, Chrast J, Lagarde J, Gilbert JG, Storey R,
Swarbreck D <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16925838" target="_blank">
GENCODE: producing a reference annotation for ENCODE</a>.
<em>Genome Biol</em>. 2006;7 Suppl 1:S4.1-9.
</p>

<p>
ENCODE Project Consortium, Birney E, Stamatoyannopoulos JA, Dutta A, Guig&#243; R, Gingeras TR,
Margulies EH, Weng Z, Snyder M, Dermitzakis ET <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17571346" target="_blank">
Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot
project</a>.
<em>Nature</em>. 2007 Jun 14;447(7146):799-816.
</p>
