<H2>Description</H2>
<P>
This track shows gene predictions submitted for the ENCODE Gene Annotation 
Assessment Project 
(<A HREF="https://genome.crg.es/gencode/workshop2005.html"
TARGET=_blank>EGASP</A>) Gene Prediction Workshop 2005 that cover only
a partial set of the 44 ENCODE regions.  The partial set excludes
the 13 ENCODE regions for which high-quality annotations were released in late
2004.
The following gene predictions are included:
<UL>
<LI>
<A HREF="http://hollywood.mit.edu/acescan/" TARGET=_blank>ACEScan</A></LI>
<LI>
<A HREF="http://augustus.gobics.de/" TARGET=_blank>Augustus</A></LI>
<LI>
<A HREF="http://www.genezilla.org/" TARGET=_blank>GeneZilla</A></LI>
<LI>
SAGA</LI>
</UL>
The EGASP Full companion track shows original gene prediction submissions for 
the full set of 44 ENCODE regions using Gene Prediction algorithms other than 
those used here; the EGASP Update track shows updated versions
of some of the submitted predictions.</P>

<H2>Display Conventions and Configuration</H2>
<P>
Data for each gene prediction method within this composite annotation track 
is displayed in a separate subtrack. See the top of the track description page 
for a complete list of the subtracks available for this annotation. To display
only selected subtracks, uncheck the boxes next to the tracks you wish to
hide.  
<P>
The individual subtracks within this annotation follow the display conventions 
for <A HREF="../goldenPath/help/hgTracksHelp.html#GeneDisplay">gene prediction
tracks</A>. The track description page offers the option 
to color and label codons in a zoomed-in display of the subtracks to facilitate 
validation and comparison of gene predictions. To enable this feature, select 
the <em>genomic codons</em> option from the &quot;Color track by codons&quot;
menu. Click the
<A HREF="../goldenPath/help/hgCodonColoring.html">Help on codon coloring</A>
link for more information about this feature. </P>
<P>
Color differences among the subtracks are arbitrary. They provide a
visual cue for distinguishing the different gene prediction methods.</P>

<H2>Methods</H2>

<H3>ACEScan</H3>
<P>
ACEScan (Alternative Conserved Exons Scan) 
indicates alternative splicing that is evolutionarily conserved in human and 
mouse/rat. The Conserved Alternative Exon Predictions subtrack shows
predicted alternative conserved exons. The Unconserved Alternative and 
Constitutive Exon Predictions subtrack shows exons that 
are predicted to be constitutive or may have species-specific alternative 
splicing. <P>

<H3>Augustus</H3>
<P>
Augustus uses a generalized hidden Markov model (GHMM) that
models coding and non-coding sequence, splice sites, the branch point region, 
translation start and end, and lengths of exons and introns. The track 
contains four different sets of predictions. <em>Ab initio</em>
single genome predictions are based solely on the input sequence. EST and
protein evidence predictions were generated using AGRIPPA hints based on 
alignments of human sequence from the dbEST and nr databases. Mouse homology 
gene predictions were produced using mouse genomic sequence only; BLAST, CHAOS, 
DIALIGN were used to generate the hints for Augustus. The combined 
EST/protein evidence and mouse homology gene predictions were created using 
human sequence from the dbEST and nr databases and mouse genomic sequence to 
generate hints for Augustus.
Additional predictions and methods for this subtrack are available in the
EGASP Updates track.</P>

<H3>GeneZilla</H3>
<P>
GeneZilla is a program for the computational prediction of protein-coding genes 
in eukaryotic DNA, based on the generalized hidden Markov model (GHMM) 
framework. These predictions were generated using GeneZilla and 
<A HREF="http://www.bioinformatics.org/project/?group_id=485"
TARGET=_blank>IsoScan</A>, which uses a four-state hidden Markov model to 
predict isochores (regions of homogeneous G+C content) in genomic DNA.</P>

<H3>SAGA</H3>
<P>
SAGA is an <em>ab initio</em> multiple-species gene-finding program based on the
Gibbs sampling-based method described in Chatterji <em>et al.</em> (2004). In 
addition to sampling parameters, SAGA also uses a phyloHMM based model to 
boost the scores, similar to the method described in Siepel <em>et al.</em>
(2004).</P>

<H2>Credits</H2>
<P>
The gene prediction data sets were submitted by the following individuals and 
institutions:
<UL>
<LI>
<B>ACEScan:</B> Gene Yeo, Crick-Jacobs Center for Computational Biology, 
<A HREF="https://www.salk.edu/" TARGET=_blank>Salk Institute</A>.
<LI>
<B>Augustus:</B> Mario Stanke, <A HREF="http://gobics.de/department/"
TARGET=_blank>Department of Bioinformatics</A>, University of Göttingen, 
Germany.
<LI>
<B>GeneZilla:</B> William Majoros, Dept. of Bioinformatics, The Institute for 
Genomic Research (<A HREF="https://www.jcvi.org/" TARGET=_blank>TIGR</A>).
<LI>
<B>SAGA:</B> Sourav Chatterji, Lior Pachter lab, 
<A HREF="https://pachterlab.github.io/group.html" TARGET=_blank>Department of Mathematics</A>, U.C. Berkeley.
</UL></P>

<H2>References</H2>
<P>
Chatterji, S. and Pachter, L. 
<A HREF="https://dl.acm.org/citation.cfm?id=974639&jmp=cit&coll=GUIDE&dl=GUIDE#CIT"
TARGET=_blank>Multiple organism gene finding by collapsed Gibbs sampling</A>. 
<em>Proc. 8th Int'l Conf. on Research in Computational Molecular Biology</em>, 
187-193 (2004).</P>
<P>
Siepel, A. and Haussler, D. 
<A HREF="https://dl.acm.org/citation.cfm?id=974638&coll=GUIDE&dl=GUIDE"
TARGET=_blank>Computational identification of evolutionarily conserved 
exons</A>. 
<em>Proc. 8th Int'l Conf. on Research in Computational Molecular Biology</em>, 
177-186 (2004).</P>
