<H2>Description</H2>
<P>
This track shows recombination rates measured in
centiMorgans/Megabase in ten ENCODE regions that have been resequenced: 
<UL>
<LI>ENr112 (chr2)
<LI>ENr131 (chr2)
<LI>ENr113 (chr4)
<LI>ENm010 (chr7)
<LI>ENm013 (chr7)
<LI>ENm014 (chr7)
<LI>ENr321 (chr8)
<LI>ENr232 (chr9)
<LI>ENr123 (chr12)
<LI>ENr213 (chr18)
</UL></P>
<P>
Observations from sperm studies (Jeffreys <em>et al</em>., 2001) and 
patterns of genetic variation (McVean <em>et al</em>., 2004; Crawford 
<em>et al</em>., 2004) show
that recombination rates in the human genome vary extensively over kilobase 
scales and that much recombination occurs in recombination hotspots. This
provides an explanation for the apparent block-like structure of linkage 
disequlibrium (Daly <em>et al</em>., 2001; Gabriel <em>et al</em>., 2002).</P>
<P>
Fine-scale recombination rate estimates provide a new route to
understanding the molecular mechanisms underlying human recombination.
A better understanding of the genomic landscape of human recombination
rate variation would facilitate the efficient design and analysis of
disease association studies and greatly improve inferences from
polymorphism data about selection and human demographic history.</P>

<H2>Display Conventions and Configuration</H2>
<P>
This annotation track may be configured in a variety of ways to highlight 
different aspects of the displayed data. The graphical configuration options 
are shown at the top of the track description page. 
For more information, click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph
configuration help</A> link.</P>

<H2>Methods</H2>
<P>
Fine-scale recombination rates are estimated using the reversible-jump
Markov chain Monte Carlo method (McVean <em>et al</em>., 2004).  This
approach explores the posterior distribution of fine-scale recombination
rate profiles, where the state-space considered is the distribution of
piece-wise constant recombination maps.  The Markov chain explores the
distribution of both the number and location of change-points, in addition
to the rates for each segment.  A prior is set on the number of
change-points that increases the smoothing effect of trans-dimensional
MCMC, which is necessary because of the composite-likelihood scheme
employed.</P>
<P>
This method is implemented in the package 
<A HREF="http://www.stats.ox.ac.uk/~mcvean/LDhat/" TARGET=_blank>LDhat</A>, 
which includes full details of installation and implementation.  </P>
<P>
For the ENCODE regions, a block-penalty of 5 was used (calibrated by simulation 
and comparison to data from sperm-typing studies).  Each region was
analyzed as a single run with 10,000,000 iterations, sampling every 5000th
iteration and discarding the first third of all samples as burn-in.  The
mean posterior rate for each SNP interval is the value reported.  Because of 
the non-independence of the composite likelihood scheme,
the quantiles of the sampling distribution do not reflect true uncertainty
and are therefore not given.</P>
<P>
Estimates were generated separately from each of the four ENCODE resequencing 
populations, and then combined to give a single figure.  Differences between 
populations are not significant.</P>

<H2>Validation</H2>
<P>
This approach has been validated in three ways: by extensive
simulation studies and by comparisons with independent estimates of
recombination rates, both over large scales from the genetic map and
over fine scales from sperm analysis.  Full details of validation can be 
found in McVean <em>et al</em>. (2004) and Winckler <em>et al</em>. (2005).</P>

<H2>Credits</H2>
<P>
The data is based on <A HREF="https://www.ncbi.nlm.nih.gov/variation/news/NCBI_retiring_HapMap/" TARGET=_blank>HapMap</A> 
release 16.  The recombination rates were ascertained by Gil McVean from the
<A HREF="http://mathgen.stats.ox.ac.uk/"
TARGET=_blank>Mathematical Genetics Group</A> at the University of Oxford.</P>

<H2>References</H2>
<P>
Crawford, D.C., Bhangale, T., Li, N., Hellenthal, G., Rieder, M.J., 
Nickerson, D.A. and Stephens, M.
<A HREF="https://www.nature.com/articles/ng1376"
TARGET=_blank>Evidence for substantial fine-scale variation in recombination 
rates across the human genome</A>.
<em>Nat Genet.</em> <B>36</B>(7), 700-6 (2004).</P>
<P>
Daly, M.J., Rioux, J.D., Schaffner, S.F., Hudson, T.J. and  Lander, E.S.
<A HREF="https://www.nature.com/articles/ng1001-229"
TARGET=_blank>High-resolution haplotype structure in the human genome</A>.
<em>Nat Genet.</em> <B>29</B>(2), 229-32 (2001). </P>
<P>
Gabriel, S.B., Schaffner, S.F., Nguyen, H., Moore, J.M., Roy, J., Blumenstiel, 
B., Higgins, J., DeFelice, M., Lochner, A., Faggart, M. <em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=12029063&query_hl=3"
TARGET=_blank>The structure of haplotype blocks in the human genome</A>.
<em>Science</em> <B>296</B>(5576), 2225-9 (2002).</P>
<P>
Jeffreys, A.J,. Kauppi, L. and Neumann, R.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&cmd=Retrieve&db=PubMed&list_uids=11586303&dopt=Abstract"
TARGET=_blank>Intensely punctate meiotic recombination in the class II region 
of the major histocompatibility complex</A>.
<em>Nat Genet.</em> <B>29</B>(2), 217-22 (2001).</P>
<P>
McVean, G.A., Myers, S.R., Hunt, S., Deloukas, P., Bentley, D.R. and Donnelly, 
P.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15105499&query_hl=9"
TARGET=_blank>The fine-scale structure of recombination rate variation in the 
human genome</A>.
<em>Science</em> <B>304</B>(5670), 581-4 (2004).</P>
<P>
Winckler, W., Myers, S.R., Richter, D.J., Onofrio, R.C., McDonald, G.J., 
Bontrop, R.E., McVean, G.A., Gabriel, S.B., Reich, D., Donnelly, P. 
<em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15705809&query_hl=9"
TARGET=_blank>Comparison of fine-scale recombination rates in humans and 
chimpanzees</A>.
<em>Science</em> <B>308</B>(5718), 107-11 (2005).</P>

