<H2>Description</H2>
<P>
The subtracks within this annotation show replication origins identified using 
the nascent strand method (Ori-NS), the bubble trapping method (Ori-Bubble) 
and the TR<SUB>50</SUB> local minima method (Ori-TR50). 
Tracks are available for <A HREF=
"https://www.atcc.org/ATCCAdvancedCatalogSearch/ProductDetails/tabid/452/Default.aspx?ATCCNum=CCL-2&Template=cellBiology"
TARGET=_blank>HeLa</A> cells (cervix carcinoma) for all methods and 
<A HREF="https://www.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=GM06990&PgId=166"
 TARGET=_blank>GM06990</A> cells (lymphoblastoid) for Ori-NS.

<H2>Display Conventions and Configuration</H2>
<P>
This annotation follows the display conventions for composite 
tracks. To show only selected subtracks within this annotation, 
uncheck the boxes next to the tracks you wish to hide.
</P>

<H2>Nascent Strand Method (Ori-NS)</H2>
<H3>Description</H3>
<P>
ENCODE region-wide mapping of replication origins was performed.  
Origin-centered nascent-strands purified from HeLa and 
GM06990 cell lines were hybridized to Affymetrix 
ENCODE tiling arrays.

<H3>Methods</H3>
<P>
Cells in their exponential stage of growth were labeled, in culture, with 
bromodeoxyuridine (BrdU) for 30 mins. DNA was then isolated from the cells. 
Nascent strands of 0.5-2.5 kb synthesized with incorporation of BrdU, 
representing the replication origins, were purified using a sucrose 
gradient followed by immunoprecipitation with BrdU antibody (Giacca 
<em>et al</em>., 1997). The purified nascent strands were amplified and 
then hybridized to Affymetrix ENCODE tiling arrays, which have 25-mer 
probes tiled every 22 bp, on average, in the non-repetitive sequence of the 
ENCODE regions. As an experimental control, genomic DNA was hybridized to 
arrays independently.</P> 
<P>
Replication origins were identified by estimating the 
significance of the enrichment of nascent strands DNA (treatment) signal over 
genomic DNA (control) signal in a sliding window of 1000 bp. An estimate of 
significance in the window was calculated by computing the p-value using the 
Wilcoxon Rank-Sum test over all three biological replicates and control signal 
estimates in that window. The origins (Ori-NS) represented in the subtrack are 
the genomic regions that showed a signal enrichment pValue <= 0.001.

<H3>Verification</H3>
<P>
The origin mapping experiments were completed for three biological sets.

<H3>Credits</H3>
<P>
Data generation and analysis for the subtracks using the Ori-NS method were 
performed by the DNA replication group in the Dutta Lab at the University of 
Virginia: Neerja Karnani, Christopher Taylor, Ankit Malhotra, Gabe Robins 
and Anindya Dutta. 
</P>
Christopher Taylor and Neerja Karnani prepared the data for presentation in 
the UCSC Genome Browser. 

<H3>References</H3>
<P>
Giacca M, Pelizon C, Falaschi A. <A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=9441856&query_hl=7" 
TARGET=_blank>Mapping replication origins by quantifying relative abundance 
of nascent DNA strands using competitive polymerase chain reaction</A>. 
<em>Methods</em>. 1997;13(3):301-12.
</P>

<H2>Bubble Trapping Method (Ori-Bubble)</H2>
<H3>Description</H3>
<P>
ENCODE region-wide mapping of replication origins in HeLa 
cells was performed by the bubble trapping method. Replication origins were
identified by hybridization to Affymetrix ENCODE tiling arrays. 

<H3>Methods</H3>
<P>
The bubble trapping method works on the principle that circular plasmids can be
trapped in gelling agarose followed by the application of electrical current
for a prolonged period of time (see Mesner <em>et al</em>. 2006 for more 
details). Entrapment occurs by an apparent physical linkage of the circular 
DNA with the agarose matrix. The circular bubble component of the DNA 
replication intermediates was therefore enriched by agarose trapping. After 
recovery from the agarose gel, a library of the entrapped DNA was formed by DNA cloning. Subsequently, DNA from the library was labeled and hybridized to 
Affymetrix ENCODE tiling arrays, which have 25-mer probes tiled every 22 bp 
on average in the non-repetitive ENCODE regions. As an experimental control, 
genomic DNA was hybridized to arrays independently.</P>
<P>
Replication origins were identified by estimating the significance of the 
enrichment of the bubble-trapped DNA (treatment) signal over genomic DNA 
(control) signal in a sliding window of 10,000 bp. An estimate of significance 
in the window was calculated by computing the p-value using the Wilcoxon 
Rank-Sum test over all three biological replicates and the control signal 
estimates in that window. The origins (Ori-Bubble) hence represented in the 
UCSC browser track are the genomic regions that showed a signal enrichment 
pValue <= 0.001.
</P>

<H3>Verification</H3>
<P>
The origin mapping experiments were completed for two biological sets. 
</P>

<H3>Credits</H3>
<P>
Data generation and analysis for the subtrack using the Ori-bubble method 
were performed by the DNA replication group in the Dutta Lab and Hamlin Lab 
at the University of Virginia: Neerja Karnani, Larry Mesner, Christopher 
Taylor, Ankit Malhotra, Gabe Robins, Anindya Dutta and Joyce Hamlin.</P> 
<P>
Neerja Karnani and Christopher Taylor prepared the data for presentation in the UCSC Genome Browser.</P>

<H3>References</H3>
<P>
Mesner LD, Crawford EL, Hamlin JL. <A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16507369&query_hl=4" TARGET=_blank>Isolating apparently pure libraries of replication origins 
from complex genomes</A>. <em>Mol Cell</em>. 2006 Mar 3;21(5):719-26.
</P>

<H2>TR<SUB>50</SUB> local minima method (Ori-TR50)</H2>
<H3>Description</H3>
<P>
ENCODE region-wide mapping of replication origins in HeLa 
cells was performed by the TR<SUB>50</SUB> local minima method. Replication 
origins were identified by hybridization to Affymetrix ENCODE tiling arrays.
</P>
 
<H3>Methods</H3>
<P>
The experimental strategy adopted to map this profile involved isolation of 
replication products from HeLa cells synchronized at the G1-S boundary by 
thymidine-aphidicolin double block. Cells released from the block were labeled 
with BrdU at every two-hour interval of the 10 hours of S-phase. Subsequently,
DNA was isolated from the cells. The heavy-light (H/L) DNA representing the 
pool of DNA replicated during each two-hour labeling period was separated from 
the unlabeled DNA by double cesium chloride density gradient centrifugation. 
The purified H/L DNA was then hybridized to a high-density genome-tiling
Affymetrix array comprised of all unique probes within the ENCODE regions.</P>
<P>
The time of replication of 50% (TR<SUB>50</SUB>) of each microarray probe was 
calculated by accumulating the sum over the five time points and linearly 
interpolating the time when 50% was reached.  Each probe was also classified 
as showing temporally specific replication (all alleles replicating 
together within a two-hour window) or temporally non-specific replication 
(at least one allele replicating apart from the others by at least a two 
hour difference).</P>
<P>
The TR<SUB>50</SUB> data for the temporally specific probes was then 
smoothed within a 60 kb window using lowess smoothing. Local minima (within 
a 30 kb window) on the smoothed TR<SUB>50</SUB> curve were identified which 
had at least 30 probes in the window on both sides of the minimum to locate 
possible origins of replication. A confidence value was calculated for each 
site as the average difference from the value of the local minimum of all 
TR<SUB>50</SUB> values falling into the 30 kb window.</P>

<H3>Verification</H3>
<P>
The replication experiments were completed for two biological sets and a 
technical replicate in the HeLa adherent cell line.</P>

<H3>Credits</H3>
<P>
Data generation and analysis for the subtrack using the Ori-TR50 method 
were performed by the DNA replication group in the Dutta Lab at the 
University of Virginia: Neerja Karnani, Christopher Taylor, Hakkyun Kim, 
Louis Lim, Ankit Malhotra, Gabe Robins and Anindya Dutta.</P>
<P>
Neerja Karnani and Christopher Taylor prepared the data for presentation in the UCSC Genome Browser.</P>

<H3>References</H3>
<P>
Jeon Y, Bekiranov S, Karnani N, Kapranov P, Ghosh S, MacAlpine D, Lee C, 
Hwang DS, Gingeras TR, Dutta A.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15845769&query_hl=1"
TARGET=_blank>Temporal profile of replication of human chromosomes</A>.
<em>Proc Natl Acad Sci U S A</em>. 2005 May 3;102(18):6419-24.
</P>
