<H2>Description</H2>
This track shows the location of sites showing transcription (transfrags) for 
chromosomes 21 and 22 for 5 cell lines and 11 tissues. The 5 cell lines used 
were: GM06990, HepG2, K562, HeLaS3 and Tert-BJ; the 11 tissues used were: 
cerebellum, brain frontal lobe, hippocampus, hypothalamus, fetal spleen, fetal 
kidney, fetal thymus, ovary, placenta, prostate and testis. Purified cytosolic 
polyA+ RNA from GM06990, HepG2 and Tert-BJ cell lines, as well as purified 
polyA+ RNA from whole-cell extracts of the remaining cell lines and tissues, 
were hybridized to Affymetrix Chromosome 21_22_v2 oligonucleotide tiling 
arrays, which have 25-mer probes spaced on average every 17 bp (center-center 
of each 25mer) in the non-repetitive regions of human chromosomes 21 and 22. 
Clustered sites are shown in separate subtracks for each cell and tissue types. 
<P> 
Data for all biological replicates can be 
<A HREF="http://transcriptome.affymetrix.com/download/Affy_ENCODE_extension/"
TARGET = _BLANK >downloaded</A> from Affymetrix in wig, BED, and cel formats.

<H2>Display Conventions and Configuration</H2>
The subtracks within this composite annotation track may be configured
in a variety of ways to highlight different aspects of the displayed
data. The graphical configuration options for the subtracks are shown
at the top of the track description page, followed by a list of
subtracks. To show only selected subtracks, uncheck the boxes next to
the tracks that you wish to hide.
 
<H2>Methods</H2>
The data from replicate arrays were quantile-normalized (Bolstad <EM>et al</EM>., 2003) 
and all arrays were scaled to a median array intensity of 330. Using two 
different approaches: i) no sliding window ii) sliding 51-bp window centered on 
each probe, an estimate of RNA abundance (signal) was computed by calculating 
the median of all pairwise average PM-MM values, where PM is a perfect match 
and MM is a mismatch. Both Kapranov <EM>et al</EM>. (2002) and Cawley <EM>et al</EM>. 
(2004) are good references for the experimental methods. The latter also describes the 
analytical methods.
<P> 
<H3>Verification</H3>
Single biological replicates were generated and hybridized to duplicate arrays 
(two technical replicates). Transcribed regions (see the Affy RNA Signal track)
were generated from the composite signal track by merging genomic positions to 
which probes are mapped. This merging was based on a 5% false positive rate 
cutoff in negative bacterial controls, a maximum gap (MaxGap) of 25 basepairs 
and minimum run (MinRun) of 25 basepairs. 
 
<H2>Credits</H2>
These data were generated and analyzed by the collaboration of the
following groups: the Tom Gingeras group at <A HREF="https://www.thermofisher.com/us/en/home/life-science/microarray-analysis.html"
TARGET = _BLANK>Affymetrix</a>, Roderic Guigo group at Centre de Regulacio Genomica, 
Alexandre Reymond group at the University of Lausanne and Stylianos Antonarakis 
group at the University of Geneva. 

<H2>References</H2>
Please see the <A HREF="https://www.thermofisher.com/us/en/home/life-science/microarray-analysis/microarray-analysis-instruments-software-services/microarray-analysis-software/affymetrix-transcriptome-analysis-console-software.html"
TARGET = _BLANK>
Affymetrix Transcriptome</a> site for a project overview and additional
references to Affymetrix tiling array publications.
<P>
Bolstad BM, Irizarry RA, Astrand M, Speed TP.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/12538238"
TARGET = _BLANK>
A comparison of normalization methods for high density oligonucleotide
array data based on variance and bias</a>.
<EM>Bioinformatics.</EM> 2003 Jan 22;19(2):185-93.
<P>
Cawley S, Bekiranov S, Ng HH, Kapranov P, Sekinger EA,
Kampa D, Piccolboni A, Sementchenko V, Cheng J, Williams  AJ <em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=14980218" TARGET = _BLANK>Unbiased mapping of transcription factor 
binding sites along human chromosomes 21 and 22 points to widespread 
regulation of noncoding RNAs</a>. <em>Cell.</em> 2004 Feb 20;116(4):499-509.
<P>
Kapranov P, Cawley SE, Drenkow J, Bekiranov S, Strausberg RL, Fodor SP,
Gingeras TR. <A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=11988577"
TARGET = _BLANK>
Large-scale transcriptional activity in chromosomes 21 and 22</a>.
<EM>Science.</EM> 2002 May 3;296(5569):916-9.
