<h2>Description</h2>
<p>
This track represents a comprehensive set of human transcription factor binding sites based on 
ChIP-seq experiments generated by production groups in the ENCODE Consortium between 
February 2011 and November 2018.</p>
<p>
Transcription factors (TFs) are proteins that bind to DNA and interact with RNA polymerases to
regulate gene expression.  Some TFs contain a DNA binding domain and can bind directly to 
specific short DNA sequences ('motifs');
others bind to DNA indirectly through interactions with TFs containing a DNA binding domain.
High-throughput antibody capture and sequencing methods (e.g. chromatin immunoprecipitation
followed by sequencing, or 'ChIP-seq') can be used to identify regions of
TF binding genome-wide.  These regions are commonly called ChIP-seq peaks.</p>

The related
<i>Transcription Factor ChIP-seq Clusters</i> tracks 
(<a target="_blank" href="../cgi-bin/hgTrackUi?db=hg19&g=encRegTfbsClustered">hg19</a>,
<a target="_blank" href="../cgi-bin/hgTrackUi?db=hg38&g=encRegTfbsClustered">hg38</a>)
provide summary views of this data.
</p>  
<p>

<h2>Display and File Conventions and Configuration</h2>
<p>
The display for this track shows site location with the point-source of the peak marked with a 
colored vertical bar and the level of enrichment at the site indicated by the darkness of the item.
The subtracks are colored by UCSC ENCODE 2 cell type color conventions on the hg19 assembly, 
and by similarity of cell types in DNaseI hypersensitivity assays (as in the
<a target="_blank" href="../cgi-bin/hgTrackUi?db=hg38&g=wgEncodeRegDnaseWig">DNase Signal</a>)
track in the hg38 assembly.<p>

The display can be filtered to higher valued items, using the 
<em>Score range:</em> configuration item.
The score values were computed at UCSC based on signal values assigned by the ENCODE
pipeline.
The input signal values were multiplied by a normalization factor calculated as the ratio
of the maximum score value (1000) to the signal value at 1 standard deviation from the mean,
with values exceeding 1000 capped at 1000. This has the effect of distributing scores up to 
mean + 1std across the score range, but assigning all above to the maximum score.

<h2>Methods</h2>
<p>
The ChIP-seq peaks in this track were
generated by the
the <a target="_blank" href="https://www.encodeproject.org/chip-seq/transcription_factor/">ENCODE Transcription Factor ChIP-seq Processing Pipeline</a>.
Methods documentation and full metadata for each track can be found at the 
<a target="_blank" href="https://www.encodeproject.org">ENCODE project portal</a>, using
The ENCODE file accession (ENCFF*)</em> listed in the track label.
</p>

<h2>Credits</h2>
<p>
Thanks to the ENCODE Consortium, the ENCODE ChIP-seq production laboratories, and the
ENCODE Data Coordination Center for generating and processing the datasets used here.
Special thanks to Henry Pratt, Jill Moore, Michael Purcaro, and Zhiping Weng, PI, at the 
<a target="_blank" href="https://www.umassmed.edu/zlab/">ENCODE Data Analysis Center
(ZLab at UMass Medical Center)</a> for providing the peak datasets, metadata,
and guidance developing this track. Please check the
<a target="_blank" href="../cgi-bin/hgHubConnect?hubSearchTerms=ZLab">ZLab ENCODE Public Hubs</a>
for the most updated data.
</p>

<h2>References</h2>

<p>ENCODE Project Consortium.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21526222" title="https://www.ncbi.nlm.nih.gov/pubmed/21526222"  rel="nofollow" TARGET="_BLANK">
A user's guide to the encyclopedia of DNA elements (ENCODE)</a>.
<em>PLoS Biol</em>. 2011 Apr;9(4):e1001046. PMID: 21526222; PMCID: PMC3079585
</p>

<p>ENCODE Project Consortium.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22955616" title="https://www.ncbi.nlm.nih.gov/pubmed/22955616"  rel="nofollow" TARGET="_BLANK">
An integrated encyclopedia of DNA elements in the human genome</a>.
<em>Nature</em>. 2012 Sep 6;489(7414):57-74. PMID: 22955616; PMCID: PMC3439153
</p>
<p>
Sloan CA, Chan ET, Davidson JM, Malladi VS, Strattan JS, Hitz BC, Gabdank I, Narayanan AK, Ho M, Lee
BT <em>et al</em>.
<a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkv1160" target="_blank">
ENCODE data at the ENCODE portal</a>.
<em>Nucleic Acids Res</em>. 2016 Jan 4;44(D1):D726-32.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26527727" target="_blank">26527727</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4702836/" target="_blank">PMC4702836</a>
</p>
<p>
Gerstein MB, Kundaje A, Hariharan M, Landt SG, Yan KK, Cheng C, Mu XJ, Khurana E, Rozowsky J,
Alexander R <em>et al</em>.
<a href="https://www.nature.com/articles/nature11245" target="_blank">
Architecture of the human regulatory network derived from ENCODE data</a>.
<em>Nature</em>. 2012 Sep 6;489(7414):91-100.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22955619" target="_blank">22955619</a>
</p>
<p>
Wang J, Zhuang J, Iyer S, Lin X, Whitfield TW, Greven MC, Pierce BG, Dong X, Kundaje A, Cheng Y
<em>et al</em>.
<a href="https://genome.cshlp.org/content/22/9/1798.long" target="_blank">
Sequence features and chromatin structure around the genomic regions bound by 119 human
transcription factors</a>.
<em>Genome Res</em>. 2012 Sep;22(9):1798-812.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22955990" target="_blank">22955990</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3431495/" target="_blank">PMC3431495</a>
</p>
<p>
Wang J, Zhuang J, Iyer S, Lin XY, Greven MC, Kim BH, Moore J, Pierce BG, Dong X, Virgil D <em>et
al</em>.
<a href="https://academic.oup.com/nar/article/41/D1/D171/1069417" target="_blank">
Factorbook.org: a Wiki-based database for transcription factor-binding data generated by the ENCODE
consortium</a>.
<em>Nucleic Acids Res</em>. 2013 Jan;41(Database issue):D171-6.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23203885" target="_blank">23203885</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531197/" target="_blank">PMC3531197</a>
</p>

<H2> Data Use Policy </H2>
<P> <B>Users may freely download, analyze and publish results based on any ENCODE data without 
restrictions.</B>
Researchers using unpublished ENCODE data are encouraged to contact the data producers to discuss possible coordinated publications; however, this is optional. </p>
<p>
<B><I>Users of ENCODE datasets are requested to cite the ENCODE Consortium and ENCODE 
production laboratory(s) that generated the datasets used, as described in
<A target="_blank" href="https://www.encodeproject.org/help/citing-encode/">Citing ENCODE</A>.</B></I></p>

