<h2>Description</h2>
<p>
This supertrack is a collection of gene prediction tracks and is composed of the following tracks:
</p>
<dl>
    <dt><b>AUGUSTUS</b></dt>
      <dd>
      shows <i>ab initio</i> predictions from the program
      <a href="http://bioinf.uni-greifswald.de/augustus/" target="_blank">AUGUSTUS</a>
      (version 3.1). The predictions are based on the genome sequence alone.</dd>
    <dt><b>Geneid Genes</b></dt>
      <dd>
      shows gene predictions from the
      <a href="https://genome.crg.cat/software/geneid/index.html" target="">geneid</a>
      program. Geneid is a program to predict genes in anonymous genomic sequences designed with a
      hierarchical structure.</dd>
    <dt><b>Genscan Genes</b></dt>
      <dd>
      shows predictions from the
      <a href="http://hollywood.mit.edu/GENSCAN.html" target="_blank">Genscan</a>
      program. The predictions are based on transcriptional, translational and donor/acceptor
      splicing signals as well as the length and compositional distributions of exons, introns and
      intergenic regions.</dd>
    <dt><b>SGP Genes</b></dt>
      <dd>
      shows gene predictions from the
      <a href="https://genome.crg.cat/services/sgp2/" target="_blank">SGP2</a> homology-based gene
      prediction program. To predict genes in a genomic query, SGP2 combines geneid predictions with
      tblastx comparisons of the genome of the target species against genomic sequences of other
      species (reference genomes) deemed to be at an appropriate evolutionary distance from the
      target.</dd>
    <dt><b>SIB Genes</b></dt>
      <dd>
      a transcript-based set of gene predictions based on data from RefSeq and
      EMBL/GenBank. The track includes both protein-coding and non-coding transcripts. The coding
      regions are predicted using
      <a href="http://estscan.sourceforge.net/" target="_blank">ESTScan</a>.</dd>
</dl>
<p>
More information about display conventions, methods, credits, and references can be found on each
subtrack's description page.</p>
