<H2>Description</H2>
<P>
This track displays the probes from the 
<A HREF="http://www.genomics.agilent.com/CollectionOverview.aspx?PageType=Application&SubPageType=ApplicationOverview&PageID=75" 
TARGET=_blank>Agilent Catalog Oligonucleotide Microarrays</A>.
</P>
<P>
Agilent's oligonucleotide CGH (Comparative Genomic Hybridization) platform enables the 
study of genome-wide DNA copy number changes at a high resolution. The CGH probes on Agilent 
aCGH microarrays are 60-mer oligonucleotides synthesized in situ using Agilent's inkjet 
SurePrint technology. The probes represented on the Agilent CGH microarrays have been 
selected using algorithms developed specifically for the CGH application, assuring optimal 
performance of these probes in detecting DNA copy number changes. 
</P>
<P>
The Agilent SurePrint G3 CGH+SNP microarrays are designed for high quality human DNA 
copy-number profiling combined with the simultaneous detection of copy-neutral 
aberrations, such as lack or loss of heterozygosity (LOH) and uniparental disomy (UPD). 
Identification of LOH/UPD is enabled by the presence of a set of SNP probes on the CGH+SNP 
microarrays resulting in ~5-10 Mb resolution for copy neutral LOH/UPD detection across the 
entire genome.
</P>
<P>
The Agilent catalog CGH and CGH+SNP microarrays are printed on 1 in. x 3 in. glass slides and are 
available in several formats. The human catalog SurePrint G3 microarrays formats are the 1x1M 
(gene-biased or evenly tiled), 2x400K (CGH-only or CGH+SNP), 4x180K (CGH-only or CGH+SNP), 
and 8x60K. The legacy human catalog SurePrint HD microarrays are the 1x244K, 2x105K, and 
4x44K. 
</P>
<p>
Non-SNP subtracks are colored in alternating shades of 
green and orange to highlight track boundaries. SNP subtracks 
are colored blue with SNP probes colored dark blue and 
CGH probes colored light blue. The track consists of the 
following subtracks:
</P>


<table class="descTbl" border=1 cellspacing=1 cellpadding=3>
<thead>
<tr style="background-color: #1616D1; color: #fff;"> 
	<th>Description</th>
	<th>AMADID</th>
	<th>Samples per Slide</th>
	<th>Biological Features</th>
	<th>Median Probe Spacing</th>
	<th>Gene-biased</th>
	<th>SNPs</th>
</tr>
</thead>
<tbody>
<tr style="background-color: #d9f8e4">
	<td>SurePrint G3 Human CGH Microarray 1x1M</td>
	<td>021529</td>
	<td>1</td>
	<td>969,033</td>
	<td>2.1 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9e4ff">
	<td>SurePrint G3 Human High-Resolution Microarray 1x1M</td>
	<td>023642</td>
	<td>1</td>
	<td>989,214</td>
	<td>2.6 kb</td>
	<td>No (evenly tiled)</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9f8e4">
	<td>SurePrint G3 Human CGH+SNP Microarray 2x400K</td>
	<td>028081</td>
	<td>2</td>
	<td>411,434</td>
	<td>7 kb</td>
	<td>Yes</td>
	<td>65,000</td>
</tr>
<tr style="background-color: #d9e4ff">
	<td>SurePrint G3 Human CGH Microarray 2x400K</td>
	<td>021850</td>
	<td>2</td>
	<td>415,914</td>
	<td>5.3 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9f8e4">
	<td>SurePrint G3 Human Cancer CGH+SNP Microarray 4x180K</td>
	<td>030587</td>
	<td>4</td>
	<td>172,595</td>
	<td>25 kb</td>
	<td>Yes (cancer genes)</td>
	<td>65,000</td>
</tr>
<tr style="background-color: #d9e4ff">
	<td>SurePrint G3 Human CGH+SNP Microarray 4x180K</td>
	<td>029830</td>
	<td>4</td>
	<td>172,595</td>
	<td>25 kb</td>
	<td>No (ISCA regions)</td>
	<td>65,000</td>
</tr>
<tr style="background-color: #d9f8e4">
	<td>SurePrint G3 Human CGH Microarray 4x180K</td>
	<td>022060</td>
	<td>4</td>
	<td>174,675</td>
	<td>13 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9e4ff">
	<td>SurePrint G3 Human CGH Microarray 8x60K</td>
	<td>021924</td>
	<td>8</td>
	<td>59,175</td>
	<td>41 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9f8e4">
	<td>SurePrint HD Human CGH Microarray 1x244K</td>
	<td>014693</td>
	<td>1</td>
	<td>238,331</td>
	<td>8.9 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9e4ff">
	<td>SurePrint HD Human CGH Microarray 2x105K</td>
	<td>014698</td>
	<td>2</td>
	<td>100,034</td>
	<td>22 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
<tr style="background-color: #d9f8e4">
	<td>SurePrint HD Human CGH Microarray 4x44K</td>
	<td>014950</td>
	<td>4</td>
	<td>43,143</td>
	<td>43 kb</td>
	<td>Yes</td>
	<td>0</td>
</tr>
</tbody>
</table>

<H2>Illumina Arrays</H2>

<H4>Illumina HumanHap650Y</H4>
This annotation displays the SNPs available for genotyping with Illumina's
HumanHap650Y Genotyping BeadChip.  The HumanHap650Y contains over 650,000 markers,
extending the HumanHap550 by adding 100,000 additional Yoruba-specific tag
SNPs.  On average, there is 1 SNP every 5.3 kb, 6.2 kb and 5.4 kb across
the genome in the CEU, CHB+JPT and YRI populations, respectively.
The HumanHap650Y was derived from release 21 of the 
<A HREF="http://www.hapmap.org/" TARGET=_blank>
International HapMap Project</A>.

<H4>Illumina HumanHap550</H4>
This annotation displays the SNPs available for genotyping with Illumina's
HumanHap550 Genotyping BeadChip.  The HumanHap550 contains over 550,000 markers,
the majority of which are tag SNPs 
derived from release 20 of the 
<A HREF="http://www.hapmap.org/" TARGET=_blank>
International HapMap Project</A>.  In addition,
approximately 7800 non-synonymous SNPs, a higher density of tag SNPs in
the MHC region, over 150 mitochondrial SNPs and over 4000
SNPs from regions with copy number polymorphism were included.   
In the CEU population, an r-squared threshold of 0.8 was used
for common SNPs in genes, within 10 kb of genes or in evolutionarily
conserved regions.  For all other regions, an r-squared threshold of 0.7 was used.
On average, there is 1 SNP every 5.5 kb, 6.5 kb and 6.2 kb across the genome in 
the CEU, CHB+JPT and YRI populations, respectively.

<H4>Illumina HumanHap300</H4>
This annotation displays the SNPs available for genotyping with Illumina's
HumanHap300 Genotyping BeadChip.  The HumanHap300 contains over 317,000 tagSNP markers
derived from Phase I of the 
<A HREF="http://www.hapmap.org/" TARGET=_blank>
International HapMap Project</A>.  In addition,
approximately 7300 non-synonymous SNPs and a higher density of tag SNPs in
the MHC region were included.   On average, there is 1 SNP every 9 kb across
the genome and median spacing is 5 kb.

<H4>Illumina Human1M-Duo</H4>
This annotation displays the SNPs available for genotyping with Illumina's
Human1M-Duo Genotyping BeadChip.  The Human1M-Duo contains more than 1,100,000 tagSNP markers
and a set of ~60,000 additional CNV-targeted markers. 
The median spacing is 1.5kb (mean - 2.4 kb).

<H4>Illumina HumanOmni1-Quad v1</H4>
The HumanOmni1-Quad BeadChip consists of 1,140,419 markers in a 4-sample
format. The whole-genome content provides high
genomic coverage rates of 93%, 92%, and 76% at r2 > 0.8 for the CEU,
CHB+JPT, and YRI populations, respectively. High density markers with a
median spacing of 1.2 kb ensure the highest level of resolution for CNV
and breakpoint identification.
 
The content has been derived from the 1,000 Genomes Project,
all three HapMap phases, and recently published studies, including 
new coding variants identified by the 1000 Genomes Project and
markers chosen in high-value regions of the genome: ABO blood
typing SNPs, cSNPs, disease-associated SNPs, eSNPs, SNPs in mRNA splice
sites, ADME genes, AIMs, HLA complexes, indels, introns, MHC regions,
miRNA binding sites, mitochondrial DNA, PAR, promoter regions, and
Y-chromosome.

<H4>Illumina Human660W-Quad v1</H4>
The Human660W-Quad BeadChip consists of 657,366 markers in a 4-sample
format. The Human660W-Quad BeadChip provides 87%, 85%, and 56% coverage
of CEU, CHB+JPT, and YRI populations at r2 > 0.8. For 
CNV and cytogenetic analysis, the dense backbone content is combined
with an additional ~100,000 markers that target observed common CNVs.

<H4>HumanCytoSNP-12 v2.1</H4>

The 301,232 markers on the HumanCytoSNP-12 represents a complete 12-sample panel of 
genome-wide SNPs for a uniform backbone and additional markers targeting all regions of 
known cytogenetic importance. Backbone markers provide genome-wide marker spacing of 10kb. 
This is supplemented with dense coverage (at 6 kb spacing average) of ~250 genomic regions 
commonly studied in cytogenetics labs and targeted coverage in ~400 additional genes, 
subtelomeric regions, pericentromeric regions, and sex chromosomes. 
An efficiency-optimized tagging strategy provides a panel for GWAS 
(70% coverage in CEU at r2 > 0.8) in the highest throughput and most cost-effective 
whole-genome DNA Analysis BeadChip.

<H2>References</H2>
<P>
More information on Agilent Oligonucleotide Microarrays can be found on 
<A HREF="http://www.genomics.agilent.com/collectionoverview.aspx?pagetype=application&subpagetype=applicationoverview&pageid=75"
TARGET=_blank>Agilent's website</A>.
</P>
<P>
Barrett MT, Scheffer A, Ben-Dor A, Sampas N, Lipson D, Kincaid R, Tsang P, 
Curry B, Baird K, Meltzer PS <EM>et al.</EM>
<A HREF="http://www.pnas.org/cgi/content/abstract/101/51/17765"
TARGET=_blank>Comparative genomic hybridization using oligonucleotide 
microarrays and total genomic DNA.</A> 
<EM>PNAS</EM>. 2004 Dec 21;101(51):17765-70.
</P>

<H2>Credits</H2>
<P>
Thanks to Shane Giles, Peter Webb, and Anniek De Witte from Agilent
Technologies, Venu Valmeekam from Affymetrix, and Luana Galver and
Jennifer L. Stone from Illumina for
providing these data.
</P>
