<h2>Description</h2>
<p>
The tracks listed here contain data from
<a target="_blank" href="https://www.nist.gov/programs-projects/genome-bottle"><b>The Genome in a
Bottle Consortium (GIAB)</b></a>, an open, public consortium hosted by 
<a target="_blank" href="https://www.nist.gov/">NIST</a>. The priority of GIAB is to develop 
reference standards, reference methods, and reference data by authoritative characterization of 
human genomes for use in <b>benchmarking</b>, including analytical validation and technology 
development that will support translation of whole human genome sequencing to clinical practice. The
sole purpose of this work is to provide validated variants and regions to enable technology and 
bioinformatics developers to benchmark and optimize their detection methods.
</p>
<p> 
The <b>Ashkenazim and the Chinese Trio tracks</b> show benchmark <b>SNV calls</b> from two 
son/father/mother trios of Ashkenazi Jewish and Han Chinese ancestry from the 
<a target="_blank" href="https://www.personalgenomes.org/us">Personal Genome Project</a>, 
consented for commercial redistribution.
</p>
<p>
The <b>Genome In a Bottle Structural Variants track</b> shows benchmark <b>SV calls</b> (nssv) 
and <b>variant regions</b> (nsv) (5,262 insertions and 4,095 deletions, &gt 50 bp, in 2.51 Gb of 
the genome) from the son (HG002/NA24385) from the Ashkenazi Jewish trio.
</p>
<p>
Samples are disseminated as National Institute of Standards and Technology (NIST)
<a target="_blank"
href="https://www.coriell.org/1/NIGMS/Collections/NIST-Reference-Materials">Reference Materials</a>.
</p>
<h2>Display Conventions and Configuration</h2>
These tracks are multi-view composite tracks that contain multiple data types (views). Each view 
within a track has separate display controls, as described 
<a target="_blank" href="../goldenPath/help/multiView.html">here</a>.
</p>
<p>
Unlike a regular genome browser track, the <b>Ashkenazim and the Chinese Trio</b> tracks display 
the genome variants of each individual as two <b>haplotypes</b>; SNPs, small insertions and deletions
are mapped to each haplotype based on the <b>phasing</b> information of the VCF file. The
<em>haplotype</em> 1 and the <em>haplotype 2</em> are displayed as two separate black lanes for the
browser window region. Each variant is drawn as a vertical dash. <b>Homozygous variants</b> will
show two identical dashes on both haplotype lanes. <b>Phased heterozygous variants</b> are placed on
one of the haplotype lanes and <b>unphased heterozygous variants</b> are displayed in the area
between the two haplotype lanes.
</p>
<p>
Predicted de novo variants and variants that are inconsistent with phasing in the trio son can be 
colored in <b><font color="red">red</font></b> using the track Configuration options.
</p>

<h2>Data Access</h2>
The raw data can be explored interactively with the <a target="_blank" href="../cgi-bin/hgTables">
Table Browser</a>, or the <a target="_blank" href="../cgi-bin/hgIntegrator">Data Integrator</a>. For
automated analysis, the data may be queried from our <a target="_blank"
href="/goldenPath/help/api.html">REST API</a>.
</p>
<p>
Benchmark VCF and BED files for small variants are available for GRCh37 and GRCh38 under each
genome at <a target="_blank"
href="ftp://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/">NCBI FTP</a> site. 
Structural variants are available for GRCh37 at dbVAR 
<a target="_blank" href+"https://www.ncbi.nlm.nih.gov/sites/dbvarapp/studies/nstd175/">nst175</a>.
</p>
 
<h2>References</h2>

<p>
Zook JM, McDaniel J, Olson ND, Wagner J, Parikh H, Heaton H, Irvine SA, Trigg L, Truty R, McLean CY
<em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30936564" target="_blank">
An open resource for accurately benchmarking small variant and reference calls</a>.
<em>Nat Biotechnol</em>. 2019 May;37(5):561-566.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30936564" target="_blank">30936564</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6500473/" target="_blank">PMC6500473</a>
</p>

<p>
Zook JM, Hansen NF, Olson ND, Chapman L, Mullikin JC, Xiao C, Sherry S, Koren S, Phillippy AM,
Boutros PC <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32541955" target="_blank">
A robust benchmark for detection of germline large deletions and insertions</a>.
<em>Nat Biotechnol</em>. 2020 Jun 15;.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32541955" target="_blank">32541955</a>
</p>

