<H2>Description</H2> 
<P> 
<A HREF="https://hapmap.ncbi.nlm.nih.gov/" TARGET=_blank>The HapMap Project</A>
identified a set of approximately four million
common SNPs, and genotyped these SNPs in four populations in Phase II of the 
project.  In Phase III, it genotyped approximately 1.4 to 1.5 million SNPs 
in eleven populations.  This track shows the combined data from Phases II and III.
The intent is that this data can be used as a reference for future studies
of human disease. This track displays the genotype counts and allele
frequencies of those SNPs, and (when available) shows orthologous alleles 
from the chimp and macaque reference genome assemblies.
</P>
<P>
The four million HapMap Phase II SNPs were genotyped on individuals 
from these four human populations:
<UL>
<LI><A HREF="https://www.coriell.org/0/Sections/Collections/NHGRI/Yoruba.aspx?PgId=128&coll=HG"
TARGET=_BLANK>Yoruba in Ibadan, Nigeria (YRI)</A></LI>
<LI><A HREF=https://www.coriell.org/0/Sections/Collections/NHGRI/Japanese.aspx?PgId=359&coll=HG""
TARGET=_BLANK>Japanese in Tokyo, Japan (JPT)</A></LI>
<LI><A HREF="https://www.coriell.org/0/Sections/Collections/NHGRI/HanChinese.aspx?PgId=432&coll=HG"
TARGET=_BLANK>Han Chinese in Beijing, China (CHB)</A></LI>
<LI><A HREF="https://cephb.fr/en/index.php"
TARGET=_BLANK>CEPH (Utah residents with ancestry from northern and western Europe) (CEU)</A></LI>
</UL>
Phase III expanded to eleven populations: the four above, plus the following:
<UL>
<LI><A HREF="https://www.coriell.org/0/Sections/Collections/NHGRI/AASWusa.aspx?PgId=365&coll=HG"
TARGET=_BLANK>African Ancestry in SouthWestern United States (ASW)</A></LI>
<LI><A HREF="https://www.coriell.org/0/Sections/Collections/NHGRI/DenverChinese.aspx?PgId=361&coll=HG"
TARGET=_BLANK>Chinese Ancestry in Metropolitan Denver, CO, US (CHD)</A></LI>
<LI><A HREF="https://www.coriell.org/0/Sections/Collections/NHGRI/Gujarati.aspx?PgId=362&coll=HG"
TARGET=_BLANK>Gujarati Indians in Houston, TX (GIH)</A></LI>
<LI><A HREF="https://www.coriell.org/0/Sections/Collections/NHGRI/Luhya.aspx?PgId=360&coll=HG"
TARGET=_BLANK>Luhya in Webuye, Kenya (LWK)</A></LI>
<LI><A HREF="https://www.coriell.org/0/Sections/Collections/NHGRI/Mexican.aspx?PgId=364&coll=HG"
TARGET=_BLANK>Mexican Ancestery in Los Angeles, CA, US (MEX)</A></LI>
<LI><A HREF="https://www.coriell.org/0/Sections/Collections/NHGRI/Maasai.aspx?PgId=539&coll=HG"
TARGET=_BLANK>Masai in Kinyawa, Kenya (MKK)</A></LI>
<LI><A HREF="https://www.coriell.org/0/Sections/Collections/NHGRI/Toscani.aspx?PgId=363&coll=HG"
TARGET=_BLANK>Toscani in Italia (TSI)</A></LI>
</UL>
Each of the populations is displayed in a separate subtrack.
</P>
<P>
The HapMap assays provide biallelic results. Over 99.8% of HapMap SNPs are
described as biallelic in 
<A HREF="https://www.ncbi.nlm.nih.gov/SNP/" TARGET=_BLANK>dbSNP</A> build 129; 
approximately 6,800 are described as more complex types (in-del, mixed, etc).
70% of the HapMap SNPs are transitions: 35% are A/G, 35% are C/T.
</P>
<P>
The orthologous alleles in chimp (panTro2) and macaque (rheMac2) 
were derived using 
<A HREF="http://genome.soe.ucsc.edu/goldenPath/help/hgTracksHelp.html#Convert">liftOver</A>.
</P>
<P>
No two HapMap SNPs occupy the same position. Aside from 430 SNPs from the 
pseudoautosomal region of chrX and chrY, no SNP is mapped to more than one 
location in the reference genome.  
No HapMap SNPs occur on &quot;random&quot; chromosomes (concatenations
of unordered and unoriented contigs).
</P>

<H2>Display Conventions and Configuration</H2>
<P>
<B>Note: calculation of heterozygosity has changed since the Phase II (rel22)
version of this track.</B>  
<em>Observed</em> heterozygosity is calculated as follows: each population's
heterozygosity is computed as the proportion of heterozygous individuals in 
the population.  The population heterozygosities are averaged to determine the 
overall observed heterozygosity.  
[For Phase II genotypes, <em>expected</em> heterozygosity was calculated 
as follows: the allele counts from all populations were summed 
(not normalized for population size)
and used to determine overall major and minor allele frequencies.  
Assuming Hardy-Weinberg equilibrium, overall expected heterozygosity
was calculated as two times the product of major and minor allele
frequencies 
(see <A HREF="https://www.ncbi.nlm.nih.gov/books/NBK21425/box/A2510/"
TARGET=_BLANK><em>Modern Genetic Analysis</em>, section 17-2</A>).]
</P>
<P>
The human SNPs are displayed in gray using a color gradient based on minor allele
frequency. The higher the minor allele frequency, the darker the display. 
By definition, the maximum minor allele frequency is 50%. 
When zoomed to base level, the major allele is displayed for each population. 
</P>
<P>
The orthologous alleles from chimp and macaque are displayed in brown using a color 
gradient based on quality score.
Quality scores range from 0 to 100 representing low to high quality. For 
orthologous alleles, the higher the quality, the darker the display. Quality 
scores are not available for chimp chromosomes chr21 and chrY; these were set to
98, consistent with the panTro2 browser quality track.
</P>
<P>
Filters are provided for the data attributes described above. Additionally,
a filter is provided for observed heterozgosity (average of all populations'
observed heterozygosities).
Filters are applied to all subtracks, even if a subtrack is not displayed.
<P>
Notes on orthologous allele filters:
<UL>
<LI>If a SNP's major allele is different between populations, no overall major
allele for human is determined, thus the &quot;matches major human allele&quot;
and &quot;matches minor human allele&quot; filters for
orthologous alleles do not apply.</LI>
<LI>If a SNP is monomorphic in all populations, the minor allele is not
verified in the HapMap dataset.  In these cases, the filter to match
orthologous alleles to the minor human allele will yield no results.</LI>
</UL>
</P>

<H2>Credits</H2>
<P>
This track is based on <A HREF="https://hapmap.ncbi.nlm.nih.gov/" TARGET=_BLANK>
International HapMap Project</A> release 27 data, provided by the <A
HREF="https://hapmap.ncbi.nlm.nih.gov/downloads/" TARGET=_BLANK>HapMap Data Coordination
Center</A>. 
</P>

<H2>References</H2>

<H3>HapMap Project</H3>
<P>
The International HapMap Consortium. 
<A HREF="https://www.nature.com/articles/nature06258"
TARGET=_BLANK>A second generation human haplotype map of over 3.1 million SNPs</A>.
<em>Nature</em>. 2007 Oct 18;449(7164):851-61.</P>
<P>
The International HapMap Consortium.
<A HREF=https://www.nature.com/articles/nature04226"
TARGET=_blank>A haplotype map of the human genome</A>.
<em>Nature</em>. 2005 Oct 27;437(7063):1299-320.</P> 
<P>
The International HapMap Consortium.
<A HREF="https://www.nature.com/articles/nature02168"
TARGET=_blank>The International HapMap Project</A>.
<em>Nature</em>. 2003 Dec 18;426(6968):789-96.</P>

<H3>HapMap Data Coordination Center</H3>
<P>
Thorisson GA, Smith AV, Krishnan L, Stein LD.
<A HREF="https://genome.cshlp.org/content/15/11/1592.full"
TARGET=_blank>The International HapMap Project Web site</A>.
<em>Genome Res</em>. 2005 Nov;15(11):1592-3.</P>

<H3>A Sampling of HapMap Literature</H3>
<P>
Gibson J, Morton NE, Collins A.
<A HREF="https://academic.oup.com/hmg/article/15/5/789/596723"
TARGET=_blank>Extended tracts of homozygosity in outbred human populations</A>.
<em>Hum Mol Genet</em>. 2006 Mar 1; 15(5):789-95.
</P>
<P>
Redon R, Ishikawa S, Fitch KR, Feuk L, Perry GH, Andrews TD, Fiegler H, Shapero
MH, Carson AR, Chen W <em>et al</em>.
<A HREF="https://www.nature.com/articles/nature05329"
TARGET=_blank>Global variation in copy number in the human genome</A>.
<em>Nature</em>. 2006 Nov 23;444(7118):444-454.
<P>
Spielman RS, Bastone LA, Burdick JT, Morley M, Ewens WJ, Cheung VG.
<A HREF="https://www.nature.com/articles/ng1955"
TARGET=_blank>Common genetic variants account for differences in gene
expression among ethnic groups</A>. <em>Nature Genet</em>. 2007
Feb;39(2):226-31.
</P>
<P>
Tenesa A, Navarro P, Hayes BJ, Duffy DL, Clarke GM, Goddard ME, Visscher PM.
<A HREF="https://genome.cshlp.org/content/17/4/520.full"
TARGET=_blank>Recent human effective population size estimated from linkage
disequilibrium</A>. <em>Genome Res</em>. 2007 Apr;17(4):520-6.
</P>
Voight BF, Kudaravalli S, Wen X, Pritchard JK.
<A HREF="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.0040072"
TARGET=_blank>A Map of Recent Positive Selection in the Human Genome</A>.
<em>PLoS Biol</em>. 2006 Mar;4(3):e72.</P>
<P>
Weir BS, Cardon LR, Anderson AD, Nielsen DM, Hill WG.
<A HREF="https://genome.cshlp.org/content/15/11/1468.full"
TARGET=_blank>Measures of human population structure show heterogeneity among genomic
regions</A>. <em>Genome Res</em>. 2005 Nov;15(11):1468-76.
</P>

<H3>Data Source</H3>
<P>
The genotypes_chr*_*_r27_nr.b36_fwd.txt.gz files from the
<A HREF="ftp://ftp.ncbi.nlm.nih.gov/hapmap/genotypes/2009-02_phaseII+III/forward/non-redundant/"
TARGET=_BLANK>HapMap FTP site</A> were processed to make this track.
</P>
