<H2>Description</H2> 
<P> 

The HapMap project identified and genotyped a set of SNPs to be used
as a reference for future studies of human disease.   
This track displays the genotype counts and allele frequencies of those SNPs.
<P>
There are 4 populations: CEU, CHB, JPT and YRI.
The CEU and YRI data is comprised of 90 individuals in trios.   
The UCSC display removes the data for the children, leaving 60 individuals in each population.
The CHB and JPT data is comprised of 45 individuals.
Each of the 4 populations is displayed in a separate subtrack.
Over 12% of HapMap SNPs are available for only a subset (1-3) of the populations.
</P>
The HapMap assays provided biallelic results.   Over 99.9% of HapMap SNPs are known to dbSNP build125 as
biallelic; approximately 3000 are more complex.   Two-thirds of the HapMap SNPs are 
transitions; one-third are A/G, one-third are C/T.
<P>
Orthologous alleles in chimp (panTro2) and macaque (rheMac2) are also provided.
These were derived using liftOver.   Chimp alleles are available for over 96% of the human
HapMap SNPs.   Macaque alleles are available for 88% of the human SNPs.
<P>
15% of HapMap SNPs were monomorphic in all individuals in all populations.
Within single populations, monomorphic ranged from 21.5% of the YRI individuals to
38% of the JPT individuals.
<P>
Approximately 20% of HapMap SNPs have a different major allele in different populations.
<P>
No two HapMap SNPs occupy the same position.   
Aside from 7 SNPs from the pseudo autosomal region of chrX, no 
rsIds are included more than once.



<H2>Display Conventions and Configuration</H2>
<P>
The human SNPs are displayed using a color gradient based on minor allele frequency.
The higher the minor allele frequency, the darker the display.   
The maximum minor allele frequency is of course 50%.
When zoomed to base level, the major allele is displayed for the human SNPs.
<P>
The ortho alleles are displayed using a color gradient based on quality score.
Quality scores range from 0 to 100 representing low to high quality.   For ortho
alleles, the higher the quality, the darker the display.
<P>

Filters are provided for the data attributes described above.
Additionally, a filter is provided for heterozgosity over all populations.
The measure of heterozygosity used is 2pq.
Filters are applied to all 6 subtracks.
<H2>Credits</H2>
<P>
This track is based on release 21a from the HapMap Data Coordination Center.  

<H2>References</H2>

<H3>HapMap Project</H3>
<P>
The International HapMap Consortium.
<A HREF="http://www.hapmap.org/downloads/presentations/Nature_HapMap_phaseI.pdf" 
TARGET=_blank> A haplotype map of the human genome</A>.
<em>Nature</em> <B>437</B>, 1299-1320 (2005).</P> 
<P>
The International HapMap Consortium.
<A HREF="http://www.hapmap.org/downloads/nature02168.pdf" 
TARGET=_blank> The International HapMap Project</A>.
<em>Nature</em> <B>426</B>, 789-96 (2003).</P>

<H3>HapMap Data Coordination Center</H3>
<P>
Thorisson, G.A., Smith, A.V., Krishnan, L. and Stein, L.D. 
<A HREF="http://www.hapmap.org/downloads/presentations/Genome_research_art10.pdf" 
TARGET=_blank> The International HapMap Project Web site</A>.
<em>Genome Res</em> <B>15</B>, 1591-3 (2005).</P> 

<H3>Data Source</H3>
<P>
sftp://www.hapmap.org/downloads/genotypes/2007-01/rs_strand/non-redundant/genotypes_chr*_*_r21a_nr.txt.gz
</P>
