<H2>Description</H2>
<P>
This track shows a 3-SNP moving average of 
<em>p(1-p)</em> where <em>p</em> is the major allele frequency
(i.e. half of the expected heterozygosity) 
on seven continents,
from SNPs genotyped in 53 populations worldwide by the 
<A HREF="http://www.stanford.edu/group/morrinst/hgdp.html" TARGET=_BLANK>Human 
Genome Diversity Project</A> in collaboration with the
<A HREF="https://cephb.fr/en/cephdb/" TARGET=_BLANK>Centre d'Etude 
du Polymorphisme Humain</A> (HGDP-CEPH).
This track and several others are available from the 
<A HREF="http://hgdp.uchicago.edu/" TARGET=_BLANK>HGDP Selection Browser</A>.
</P>

<H2>Methods</H2>
<P>
Samples collected by the HGDP-CEPH from 1,043 individuals from around the
world were genotyped for 657,000 SNPs at
<A HREF="https://hagsc.org/hgdp/" TARGET=_BLANK>Stanford</A>.
The 53 populations were divided into seven continental groups: Africa,
Middle East, Europe, South Asia, East Asia, Oceania and the Americas.
Allele frequencies were used to calculate <em>p(1-p)</em> 
for each SNP, and then a
3-SNP average was computed for each SNP and its two neighboring SNPs.</P>

<P>
The associated analysis tracks HGDP FST, HGP iHS, and HGDP XP-EHH
(Pickrell <em>et al.</em>) did not make use of all African populations, but
instead used only the Bantu populations because a more closely related
group was desired for comparison with other continental groups.
For this track, separate subtracks show the expected heterozygosity 
of all African populations and of only Bantu populations.
</P>

<H2>Credits</H2>
<P>
Thanks to the HGDP-CEPH and Joe Pickrell in the 
<A HREF="http://web.stanford.edu/group/pritchardlab/home.html"TARGET=_BLANK>Pritchard
lab</A> at the University of Chicago for providing these data.
</P>


<H2>References</H2>
<P>
Pickrell JK, Coop G, Novembre J, Kudaravalli S, Li J, Absher D,
Srinivasan BS, Barsh GS, Myers RM, Feldman MW, Pritchard JK.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/19307593?dopt=Abstract"
TARGET=_BLANK>Signals of recent positive selection in a worldwide sample of
human populations</A>. <em>Genome Res.</em> 2009 May;19(5):826-37.</P>

<P>
Li JZ, Absher DM, Tang H, Southwick AM, Casto AM, Ramachandran S,
Cann HM, Barsh GS, Feldman M, Cavalli-Sforza LL, Myers RM.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/18292342?dopt=Abstract"
TARGET=_BLANK>Worldwide human relationships inferred from genome-wide
patterns of variation</A>. <em>Science</em>. 2008 Feb 22;319(5866):1100-4.</P>

<P>
Cann HM, de Toma C, Cazes L, Legrand MF, Morel V, Piouffre L, Bodmer
J, Bodmer WF, Bonne-Tamir B, Cambon-Thomsen A <em>et al.</em>
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/11954565?dopt=Abstract" 
TARGET=_BLANK>A human genome diversity cell line panel</A>.
<em>Science</em>. 2002 Apr 12;296(5566):261-2.</P>

