<H2>Description</H2>
<P>
In this track, the value shown for each SNP is -log<sub>10</sub> of the
fraction of SNPs with a more extreme F<sub>ST</sub> value than that SNP.


Relative F<sub>ST</sub> 
(also known as the <A HREF="https://en.wikipedia.org/wiki/Fixation_index"
TARGET=_BLANK>Fixation index</A>) values were calculated
from SNPs genotyped in 53 populations worldwide by the 
<A HREF="https://morrisoninstitute.stanford.edu/" TARGET=_BLANK>Human 
Genome Diversity Project</A> in collaboration with the
<A HREF="https://cephb.fr/en/index.php" TARGET=_BLANK>Centre d'Etude 
du Polymorphisme Humain</A> (HGDP-CEPH).
This track and several others are available from the 
<A HREF="http://hgdp.uchicago.edu/" TARGET=_BLANK>HGDP Selection Browser</A>.
</P>
<P>
From <A HREF="https://en.wikipedia.org/wiki/Fixation_index"
TARGET=_BLANK>Wikipedia</A>:
<blockquote>
Fixation index (F<sub>ST</sub>) is a measure of population differentiation based
on genetic polymorphism data, such as Single nucleotide polymorphisms
(SNPs) or microsatellites. It is a special case of 
<A HREF="https://en.wikipedia.org/wiki/F-statistics"
TARGET=_BLANK>F-statistics</A>, the
concept developed in the 1920s by Sewall Wright.  This statistic
compares the genetic variability within and between populations and is
frequently used in the field of population genetics.
</blockquote>
From <A HREF="http://www.uwyo.edu/dbmcd/popecol/maylects/popgengloss.html"
TARGET=_BLANK>http://www.uwyo.edu/dbmcd/popecol/Maylects/PopGenGloss.html</A>:
<blockquote>
F<sub>ST</sub> is the proportion of the total genetic variance
contained in a subpopulation (the S subscript) relative to the total
genetic variance (the T subscript). Values can range from 0 to 1. High
F<sub>ST</sub> implies a considerable degree of differentiation among
populations.  
</blockquote>
</P>

<H2>Methods</H2>
<P>
Samples collected by the HGDP-CEPH from 1,043 individuals from around the
world were genotyped for 657,000 SNPs at
<A HREF="https://hagsc.org/hgdp/" TARGET=_BLANK>Stanford</A>.
The 53 populations were divided into seven continental groups: Africa,
Middle East, Europe, South Asia, East Asia, Oceania and the Americas.
F<sub>ST</sub> was computed for all SNPs, and then each SNP's place in 
the empirical F<sub>ST</sub> distribution was used to derive the scores 
shown in this track, -log<sub>10</sub> of the fraction of SNPs with a 
more extreme F<sub>ST</sub> value than that SNP.
</P>

<H2>Credits</H2>
<P>
Thanks to the HGDP-CEPH and Joe Pickrell in the 
<A HREF="http://web.stanford.edu/group/pritchardlab/home.html"TARGET=_BLANK>Pritchard
lab</A> at the University of Chicago for providing these data.
</P>


<H2>References</H2>
<P>
Pickrell JK, Coop G, Novembre J, Kudaravalli S, Li J, Absher D,
Srinivasan BS, Barsh GS, Myers RM, Feldman MW, Pritchard JK.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/19307593?dopt=Abstract"
TARGET=_BLANK>Signals of recent positive selection in a worldwide sample of
human populations</A>. <em>Genome Res.</em> 2009 May;19(5):826-37.</P>

<P>
Li JZ, Absher DM, Tang H, Southwick AM, Casto AM, Ramachandran S,
Cann HM, Barsh GS, Feldman M, Cavalli-Sforza LL, Myers RM.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/18292342?dopt=Abstract"
TARGET=_BLANK>Worldwide human relationships inferred from genome-wide
patterns of variation</A>. <em>Science</em>. 2008 Feb 22;319(5866):1100-4.</P>

<P>
Cann HM, de Toma C, Cazes L, Legrand MF, Morel V, Piouffre L, Bodmer
J, Bodmer WF, Bonne-Tamir B, Cambon-Thomsen A <em>et al.</em>
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/11954565?dopt=Abstract"
TARGET=_BLANK>A human genome diversity cell line panel</A>.
<em>Science</em>. 2002 Apr 12;296(5566):261-2.</P>

