<H2>Description</H2>
<P>
This track shows per-continent integrated haplotype score (iHS, 
Voight <em>et al.</em>), a measure of very recent positive selection.
Scores were calculated using SNPs genotyped in 53 populations worldwide by the 
<A HREF="https://morrisoninstitute.stanford.edu/" TARGET=_BLANK>Human 
Genome Diversity Project</A> in collaboration with the
<A HREF="https://cephb.fr/en/index.php" TARGET=_BLANK>Centre d'Etude 
du Polymorphisme Humain</A> (HGDP-CEPH).
This track and several others are available from the 
<A HREF="http://hgdp.uchicago.edu/" TARGET=_BLANK>HGDP Selection Browser</A>.
</P>

<H2>Methods</H2>
<P>
Samples collected by the HGDP-CEPH from 1,043 individuals from around the
world were genotyped for 657,000 SNPs at
<A HREF="https://hagsc.org/hgdp/" TARGET=_BLANK>Stanford</A>.
The 53 populations were divided into seven continental groups: Africa
(Bantu populations only),
Middle East, Europe, South Asia, East Asia, Oceania and the Americas.
Bantu populations in Africa were chosen instead of all African populations
because a more closely related group was desired for comparison with 
other continental groups.
iHS was then calculated for each population group using the program ihs
(<A HREF="http://hgdp.uchicago.edu/Software/ihs.tar" 
TARGET=_BLANK>source code available</A>) and then normalizing the 
resulting unstandardized iHS scores in derived allele frequency bins
as described in (Voight <em>et al.</em>).
Per-SNP iHS scores were smoothed in windows of 31 SNPs, centered on 
each SNP.  The final score is -log<sub>10</sub> of the proportion of 
smoothed scores higher than each SNP's smoothed score.
</P>

<H2>Credits</H2>
<P>
Thanks to the HGDP-CEPH and Joe Pickrell in the 
<A HREF="http://web.stanford.edu/group/pritchardlab/home.html"TARGET=_BLANK>Pritchard
lab</A> at the University of Chicago for providing these data.
</P>


<H2>References</H2>
<P>Voight BF, Kudaravalli S, Wen X, Pritchard JK.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/16494531?dopt=Abstract" 
TARGET=_BLANK>A map of recent positive selection in the human genome</A>.
<em>PLoS Biol</em>. 2006 Mar;4(3):e72.</P>

<P>
Pickrell JK, Coop G, Novembre J, Kudaravalli S, Li J, Absher D,
Srinivasan BS, Barsh GS, Myers RM, Feldman MW, Pritchard JK.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/19307593?dopt=Abstract"
TARGET=_BLANK>Signals of recent positive selection in a worldwide sample of
human populations</A>. <em>Genome Res.</em> 2009 May;19(5):826-37.</P>

<P>
Li JZ, Absher DM, Tang H, Southwick AM, Casto AM, Ramachandran S,
Cann HM, Barsh GS, Feldman M, Cavalli-Sforza LL, Myers RM.
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/18292342?dopt=Abstract"
TARGET=_BLANK>Worldwide human relationships inferred from genome-wide
patterns of variation</A>. <em>Science</em>. 2008 Feb 22;319(5866):1100-4.</P>

<P>
Cann HM, de Toma C, Cazes L, Legrand MF, Morel V, Piouffre L, Bodmer
J, Bodmer WF, Bonne-Tamir B, Cambon-Thomsen A <em>et al.</em>
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/11954565?dopt=Abstract" 
TARGET=_BLANK>A human genome diversity cell line panel</A>.
<em>Science</em>. 2002 Apr 12;296(5566):261-2.</P>
