<br><br>
<div class="warn-note" style="border: 2px solid #9e5900; padding: 5px 20px; background-color: #ffe9cc;">
<p><b>The ClinGen CNVs track is no longer being updated. These data, along with updates,
can be found in the <a target="_blank" href="/cgi-bin/hgTrackUi?g=clinvar">
ClinVar Copy Number Variants (ClinVar CNVs) track</a>.</p>
<p>
See our <a target="_blank" href="/goldenPath/newsarch.html#093020b">
news archive</a> for more information.</p></b>
</div><br>

<H2>Description</H2>
<P>
<div class="warn-note" style="border: 2px solid #9e5900; padding: 5px 20px; background-color: #ffe9cc;">
<p><span style="font-weight: bold; color: #c70000;">NOTE:</span><br>
 <B>These data are for research purposes only.  While the ClinGen data are
 open to the public, users seeking information about a personal medical or
 genetic condition are urged to consult with a qualified physician for
 diagnosis and for answers to personal medical questions.
 </P>
 
 <P>UCSC presents these data for use by qualified professionals, and even
 such professionals should use caution in interpreting the significance of 
 information found here.  No single data point should be taken at face 
 value and such data should always be used in conjunction with as much 
 corroborating data as possible.  No treatment protocols should be 
 developed or patient advice given on the basis of these data without 
 careful consideration of all possible sources of information.
 </P>

 <P>No attempt to identify individual patients should
 be undertaken.  No one is authorized to attempt to identify patients 
 by any means.
 </P>
 </B>
</div>

</P>

<P>  
The <a href="https://www.clinicalgenome.org/" target="_blank">Clinical Genome Resource</a> (ClinGen)
is a National Institutes of Health (NIH)-funded program dedicated to building a genomic
knowledge base to improve patient care. 
This will be accomplished by harnessing the data from both research efforts and clinical genetic
testing, and using it to propel expert and machine-driven curation activities. 
By facilitating collaboration within the genomics community,
we will all better understand the relationship between genomic variation and human health. 
ClinGen will work closely with the National
Center for Biotechnology Information (NCBI) of the National Library of Medicine (NLM), 
which will distribute this information through its
ClinVar database.
</P>

<p>
The ClinGen dataset displays clinical microarray data <b>submitted</b> to dbGaP/dbVar at NCBI
by ClinGen member laboratories (dbVar study
<a href="https://www.ncbi.nlm.nih.gov/dbvar/studies/nstd37/" target="_blank">nstd37</a>),
as well as clinical data reported in Kaminsky <i>et al.</i>, 2011 (dbVar study
<a href="https://www.ncbi.nlm.nih.gov/dbvar/studies/nstd101" target="_blank">ntsd101</a>)
(see reference below). This track shows copy number variants (CNVs) found in patients referred
for genetic testing for indications such as intellectual disability, developmental delay,
autism and congenital anomalies. Additionally, the ClinGen &quot;<b>Curated</b> Pathogenic&quot; and
&quot;<b>Curated</b> Benign&quot; tracks represent genes/genomic regions reviewed for dosage sensitivity
in an evidence-based manner by the ClinGen Structural Variation Working Group (dbVar study
<a href="https://www.ncbi.nlm.nih.gov/dbvar/studies/nstd45/" target="_blank">nstd45</a>).
</p>

<P>The CNVs in this study have been reviewed for their clinical significance by
the submitting ClinGen laboratory. Some of the deletions and duplications in the track
have been reported as causative for a phenotype by the submitting clinical 
laboratory; this information was based on current knowledge at the time of submission.
However, it should be noted that phenotype information is often vague and imprecise and
should be used with caution.  While all samples were submitted because of a phenotype in 
a patient, only 15% of patients had variants determined to be causal, 
and most patients will have additional variants that are not causal.
</P>

<P>CNVs are separated into subtracks and are labeled as:
<ul>
  <li>Pathogenic</li>
  <li>Uncertain: Likely Pathogenic</li>
  <li>Uncertain</li>
  <li>Uncertain: Likely Benign</li>
  <li>Benign</li>
</ul>
The user should be aware that some of the data were submitted using a 3-class
system, with the two &quot;Likely&quot; categories omitted.  
</P>

<P>  Two subtracks, &quot;Path Gain&quot; and &quot;Path Loss&quot;, are aggregate tracks
showing graphically the accumulated level of gains and losses in the 
Pathogenic subtrack across the genome.  Similarly, &quot;Benign Gain&quot; and
&quot;Benign Loss&quot; show the accumulated level of gains and losses in the
Benign subtrack.  These tracks are collectively called &quot;<b>Coverage</b>&quot;
tracks.
</P>

<P>  Many samples have multiple variants, not all of which are causative 
of the phenotype.  The CNVs in these samples have been decoupled, so it is not
possible to connect multiple imbalances as coming from a single patient.
It is therefore not possible to identify individuals via their genotype.  
</P>


<H2>Methods and Color Convention</H2>
<P>
The samples were analyzed by arrays from patients referred for 
cytogenetic testing due to clinical phenotypes.  Samples were analyzed with a 
probe spacing of 20-75 kb. The minimum CNV breakpoints are shown; if available,
the maximum CNV breakpoints are provided in the details page, but are not shown 
graphically on the Browser image.
</P>

<P>Data were submitted to 
<A HREF="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000205.v2.p1"
TARGET="_BLANK">dbGaP</A> at NCBI and thence decoupled as described into
<A HREF="https://www.ncbi.nlm.nih.gov/dbvar/studies/nstd37/"
TARGET="_BLANK">dbVar</A> for unrestricted release.
</P>

<P>
The entries are colored <B><FONT COLOR = RED>red for loss</FONT></B>  and 
<B><FONT COLOR = BLUE>blue for gain</FONT></B>.  The names of items use the 
ClinVar convention of appending "_inheritance" indicating the mechanism of 
inheritance, if known: "_pat, _mat, _dnovo, _unk" as paternal, maternal, 
de novo and unknown, respectively. 
</P>

<H2>Verification</H2>
<P>
Most data were validated by the submitting laboratory using various methods, 
including FISH, G-banded karyotype, MLPA and qPCR.
</P>

<H2>Credits</H2>
<P>
Thank you to ClinGen and NCBI for technical coordination and consultation, and to
the UCSC Genome Browser staff for engineering the track display.
</P>

<H2>References</H2>
<p>
Miller DT, Adam MP, Aradhya S, Biesecker LG, Brothman AR, Carter NP, Church DM, Crolla JA, Eichler
EE, Epstein CJ <em>et al</em>.
<a href="https://www.sciencedirect.com/science/article/pii/S0002929710002089" target="_blank">
Consensus statement: chromosomal microarray is a first-tier clinical diagnostic test for individuals
with developmental disabilities or congenital anomalies</a>.
<em>Am J Hum Genet</em>. 2010 May 14;86(5):749-64.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20466091" target="_blank">20466091</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2869000/" target="_blank">PMC2869000</a>
</p>

<p>
Kaminsky EB, Kaul V, Paschall J, Church DM, Bunke B, Kunig D, Moreno-De-Luca D, Moreno-De-Luca A,
Mulle JG, Warren ST <em>et al</em>.
<a href="https://www.nature.com/articles/gim2011130" target="_blank">
An evidence-based approach to establish the functional and clinical significance of copy number
variants in intellectual and developmental disabilities</a>.
<em>Genet Med</em>. 2011 Sep;13(9):777-84.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21844811" target="_blank">21844811</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3661946/" target="_blank">PMC3661946</a>
</p>
