<H2>Overview</H2>

<div class="figure" style="  float: right;  width: 30%; min-width: 450px; border: 
  thin silver solid ; margin: 0.5em; padding: 0.5em">
  <p><img src="../images/laminB1Super.gif" alt="Nuclear Lamina and Chromosomal Organization"  >
  <p style="text-align: center;">
Model of chromosome organization in interphase, summarizing the main results
presented in this paper. Large, discrete chromosomal domains are dynamically
associated (double arrows) with the nuclear lamina, and demarcated by putative
insulator elements that include CTCF binding sites, promoters that are oriented
away from the lamina, and CpG islands (Fig. S1, Guelen <EM>et al.</EM>, 2008).
</div>

<P>
The architecture of human chromosomes in interphase nuclei is
still largely unknown. Microscopy studies have indicated that specific
regions of chromosomes are located in close proximity to the
nuclear lamina (NL, a dense fibrillar network associated with the inner face 
of the nuclear envelope). 
This has led to the idea that certain genomic elements may be attached to the 
NL, which may contribute to
the spatial organization of chromosomes inside the nucleus.
This track represents a high-resolution map of genome-NL interactions in human 
Tig3 lung fibroblasts, 
as determined by the DamID technique. 
</P>

<h3> NKI LaminB1 track </h3>
<P>
The LaminB1 track shows a high resolution
map of the interaction sites of the entire genome with
Lamin B1, (a key NL component) in human fibroblasts.
This map shows that genome-lamina interactions occur through more than 1,300 
sharply defined large domains 0.1-10 megabases in size. 
Microscopy evidence indicates that most of these domains are preferentially 
located at nuclear periphery. 
These lamina associated domains (LADs) are characterized by low gene-expression
levels,
indicating that LADs represent a repressive chromatin environment. 
The borders of LADs are demarcated by the insulator
protein CTCF, by promoters that are oriented away from
LADs, or by CpG islands, suggesting possible mechanisms of
LAD confinement. 
Taken together, these results demonstrate that
the human genome is divided into large, discrete domains that are
units of chromosome organization within the nucleus (see Guelen <EM>et al.</EM>, 
2008).
</P>

<h3> NKI LADs track </h3>
<P> The LADs track shows Lamina Associated Domains, or LADs, based on a 
genome-wide DamID profile of LaminB1 (above). 
For the definition of LADs, the full-genome lamin B1 DamID data set was
binarized by setting tiling array probes with positive DamID log ratios to 1 and
otherwise to 21. Next, a two-step algorithm was used to identify LADs. First,
sharp transitions were identified with a sliding edge filter, which calculates the
difference in average binary values in two windows of 99 neighbouring probes
immediately left and right of a queried probe. The cutoff for this difference was
chosen such that the number of edges detected in randomly permuted data sets
was less than 5% of the number of edges detected in the original lamin B1 data
set. Second, pairs of adjacent 'left' and 'right' edges were identified that together
enclosed a region of arbitrary size with at least 70% of the enclosed probes
reporting a positive log2 ratio. A total of 1,344 regions fulfilled these criteria
and were termed LADs. In 20 randomly permuted data sets, fewer than 13
domains were identified by the same criteria. Note that there are also
lamin-B1-positive domains flanked by one or two gradual or irregular transitions.
Because it is difficult to define the borders of such domains precisely, these 
'fuzzy' domains are not analyzed here. 
(see Guelen <EM>et al.</EM>, 2008).
</P>

<H2>Display Conventions and Configuration</H2>

<P> The LaminB1 wiggle track values range from -6.602 to 5.678 and were 
normalized so have a median of 0 and standard deviation of 1.037.  The 
default vertical viewing range for the wiggle track was chosen from -2 
to 2 because  this is roughly +/- 2 standard deviations.
</P>

<P> For an example region see genomic location: 
<a href="/cgi-bin/hgTracks?db=hg18&amp;position=chr4%3A35000001-45000000&amp;laminB1Super=show&laminB1Lads=full&laminB1=full">chr4:35,000,001-45,000,000</a> (Fig 1, Guelen <EM>et al.</EM>, 2008).
</P>


<H2>Methods</H2> 

<P> The DamID technique was applied to generate a high-resolution map of NL 
interactions for the entire human genome. 
DamID is based on targeted adenine methylation of DNA sequences that interact 
in vivo with a protein of interest. 
</P>

<P> DamID was performed with lentiviral transduction as described
(Guelen <EM>et al.</EM>, 2008). In short, a fusion protein consisting
of Escherichia coli DNA adenine methyltransferase (Dam) fused to human
LaminB1 was introduced into cultured Tig3 human lung fibroblasts. 
Dam methylates adenines in the sequence GATC, a mark absent in most eukaryotes. 
Here, the LaminB1-Dam fusion protein incorporates
in the nuclear lamina, as verified by immunofluorescence
staining. Hence, the sequences near the nuclear lamina are marked with
a unique methylation tag. The adenine methylation pattern was detected with 
genomic tiling arrays. 
Unfused Dam was used as a
reference (http://research.nki.nl/vansteensellab/DamID.htm). The data
shown are the log2-ratio of LaminB1-Dam fusion protein over Dam-only.
</P>

<P> Sample labelling and hybridizations were performed by NimbleGen
Inc., on a set of 8 custom-designed oligonucleotide arrays, with a median
probe spacing of ~750 bp. All probes recognize unique (non-repetitive) sequences. 
The raw data was log2 transformed and loess
normalized. Between array median/scale normalization was based on 6979
probes common to all arrays. Replicate arrays were averaged and the
full data set normalized to genome-wide median.
</P>

<H2>Verification</H2>

<P> The data are based on two independent biological replicates. 
Fluorescence in situ hybridization microscopy confirmed 
that most of the LaminB1 associated regions are preferentially located at 
the nuclear periphery. 
The array platform, the <A HREF="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE8854"
TARGET = _BLANK>raw 
and normalized data</A> have been deposited at the NCBI Gene Expression Omnibus 
(GEO) (https://www.ncbi.nlm.nih.gov/geo/) under accession number GSE8854.
</P>

<H2>Credits</H2>

<P> The data for this track were generated by Lars Guelen, Ludo Pagie,
and Bas van Steensel at the <a href="https://chromatingenomics.org/"
target="_blank">Van Steensel Lab</a>, Netherlands Cancer Institute.
</P>

<H2>References</H2>
<p>
Guelen L, Pagie L, Brasset E, Meuleman W, Faza MB, Talhout W, Eussen BH, de Klein A, Wessels L, de
Laat W <em>et al</em>.
<a href="https://www.nature.com/articles/nature06947" target="_blank">
Domain organization of human chromosomes revealed by mapping of nuclear lamina interactions</a>.
<em>Nature</em>. 2008 Jun 12;453(7197):948-51.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18463634" target="_blank">18463634</a>
</p>
