<h2>Description</h2>

<div class="warn-note" style="border: 2px solid #9e5900; padding: 5px 20px; background-color: #ffe9cc;">
<p><span style="font-weight: bold; color: #c70000;">NOTE:</span><br> 
LOVD is intended for use primarily by physicians and other
professionals concerned with genetic disorders, by genetics researchers, and
by advanced students in science and medicine. While the LOVD database is
open to the public, users seeking information about a personal medical or
genetic condition are urged to consult with a qualified physician for
diagnosis and for answers to personal questions.  Further, please be
sure to visit the <a href = "http://www.lovd.nl/3.0/home"
target="_blank">LOVD web site</a> for the very latest, as they are continually 
updating data.</p>

<p><span style="font-weight: bold; color: #c70000;">DOWNLOADS:</span><br>
LOVD databases are owned by their respective curators
and are not available for download or mirroring 
by any third party without their permission.  Batch queries on this track are only available via the
UCSC Beacon API (see below). See also the
<a href= "http://lovd.nl/2.0/index_list.php" target="_blank">LOVD web site</A>
for a list of database installations and the respective curators. </p></div>

<p>
This track shows the genomic positions of all <b>public</b> entries in public
installations of the Leiden Open Variation Database system (<a
href="http://www.lovd.nl/3.0/home" target="_blank">LOVD</a>) and the effect of the 
variant, if annotated.  
UCSC does not host any further information.  To get details on a variant
(bibliographic reference, phenotype, disease, patient, etc.), follow the
&quot;Link to LOVD&quot; to the central server at Leiden, which will then redirect you
to the details page on the particular LOVD server reporting this variant.
</p>

<p>
LOVD is a flexible, freely-available tool for gene-centered collection and
display of DNA variations.  It is not a database itself, but rather a platform
where curators store and analyze data.  While the LOVD team and the biggest LOVD
sites are run at the Leiden University Medical Center, LOVD installations and their
curators are spread over the whole world. Most LOVD databases report at least 
some of their content back to Leiden to allow global cross-database search, which
is, among others, exported to this UCSC Genome Browser track every month.
</p>
<p>A few LOVD databases are entirely missing from this track. Reasons include configuration issues and 
intentionally blocked data search. During the last check in June 2015, the following databases did not export any variants:
http://www.ucl.ac.uk/ldlr/Current/,
http://brca.iarc.fr/LOVD/,
http://bioinf4.fcm.unicamp.br/LOVD/,
http://vitigene.igib.res.in,
http://fishmap2.igib.res.in/mirlovd/,
http://www.medgen.mcgill.ca/nndi/,
http://sysbio.org.cn/,
http://genome.igib.res.in/PGx/,
http://www-huge.uni-regensburg.de/BEST1_database/,
http://genome.igib.res.in/tblovd/,
https://ab-openlab.csir.res.in/mitolsdb/,
http://genome.igib.res.in/snolovd/,
http://www.genomed.org/lovd2/,
http://skingenetics.igib.res.in/,
http://mitodyn.org/. Curators who want to share data in their database so it is present in this track can find more details in the <a href="http://www.lovd.nl/3.0/faq#FAQ18">LOVD FAQ</a></p>


<h2>Batch queries</h2>
<p>The LOVD data is not available for download or for batch queries in the Table Browser. 
However, it is available for programmatic access via the <a href="https://ga4gh.org/#/beacon">Global
Alliance Beacon API</a>, a web service that accepts queries in the form
(genome, chromosome, position, allele) and returns "true" or "false" depending on whether there
is information about this allele in the database. For more details see our 
<a href="hgBeacon">Beacon Server</a>. </p>
<p>To find all LOVD databases that contain variants of a given gene, you can get a list of databases by constructing a url in the format geneSymbol.lovd.nl, for example, <a href="https://grenada.lumc.nl/LSDB_list/lsdbs/TP53">tp53.lovd.nl</a>. You can then use the LOVD API to retrieve more detailed information from a particular database. See the <a href="http://www.lovd.nl/3.0/faq#FAQ15">LOVD FAQ</a>. 
</p>
<h2>Display Conventions and Configuration</h2>

<p>
Genomic locations of LOVD variation entries are labeled with the gene symbol
and the description of the mutation according to <a
href="http://www.hgvs.org/mutnomen/" target="_blank">Human Gene Variation Society
standards</a>. For instance, the label AGRN:c.172G&gt;A means that the cDNA of AGRN is
mutated from G to A at position 172.
</p>

<p>
Since October 2017, the functional effect for variants is shown on the
details page, if annotated. The possible
values are: notClassified, functionAffected, notThisDisease, notAnyDisease,
functionProbablyAffected, functionProbablyNotAffected, functionNotAffected
unknown. LOVD does not use the term "pathogenic", please see the <a
href="http://varnomen.hgvs.org/bg-material/basics/">HGVS Terminology page</a> for more details.
</p>

<p>
All other information is shown on the respective LOVD variation page, accessible via the
&quot;Link to LOVD&quot; above.
</p>

<h2>Methods</h2>

<p>
The mappings displayed in this track were provided by LOVD.
</p>

<h2>Credits</h2>

<p>
Thanks to the LOVD team, Ivo Fokkema, Peter Taschner, Johan den Dunnen, and all LOVD curators who gave permission to 
show their data.
</p>

<h2>References</h2>

<p>
Fokkema IF, Taschner PE, Schaafsma GC, Celli J, Laros JF, den Dunnen JT.
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/humu.21438" target="_blank">
LOVD v.2.0: the next generation in gene variant databases</a>.
<em>Hum Mutat</em>. 2011 May;32(5):557-63.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21520333" target="_blank">21520333</a>
</p>
