<h2>Description</h2>
<p>
This track shows most variants found in the full text of scientific publications gathered by
<a target=_blank href="https://www.genomenon.com/">Genomenon Mastermind</a>. Mastermind
uses a software that searches for disease-gene-variant associations in the 
scientific literature. The genome browser track shows only if a
variant has been indexed by the search engine.
</p>

<p>
To get details on a variant (bibliographic references, disease, etc)
click it and follow the &quot;Protein change and link to details&quot; at the top
of the details page. Mouse over an item to show the gene and amino acid change and the 
scores MMCNT1, MMCNT2 and MMCNT3, explained below.
</p>

<p>
Genomenon Mastermind Genomic Search Engine is a commercial database of variants
likely to be mentioned in full text scientific articles. A limited number of
queries per week is free for healthcare professionals and researchers, if they register on the
<a href="https://mastermind.genomenon.com/users/sign_up/" target=_"blank">signup
page</a> page. Advanced features require a license for the
<a href="https://www.genomenon.com/mastermind-pro-upgrade/" target="_blank">Mastermind Professional Edition</a>, 
which contains the same content but allows more comprehensive searches.
</p>

<h2>Display Conventions and Configuration</h2>

<p>
Genomic locations of variants are labeled with the nucleotide change.
Hover over the features to see the gene, the amino acid change and the scores MMCNT1, MMCNT2 and 
MMCNT3, described below. All other information is shown on the respective Mastermind variant detail
page, accessible via the &quot;Protein change and link to details&quot; at the top of the details page. The
features are colored based on their evidence:
</p>

<p>As suggested by Genomenom, we added a filter on all variants, so the data are not exactly identical 
to their website. We skip 
variants with more than one nucleotide and a MMCNT of 0 and where the variant is not an indel. 
This means that for longer variants, only variants are shown that are explicitly
mentioned in the papers. This makes the data more specific.
</p>

<p>
<table>
  <thead>
  <tr>
    <th style="border-bottom: 2px solid #6678B1;">Color</th>
    <th style="border-bottom: 2px solid #6678B1;">Level of support</th>
  </tr>
  </thead>
  <tr>
    <th bgcolor="#0C0C78"></th>
    <th align="left">High: at least one paper mentions this exact cDNA change</th>
  </tr>
  <tr>
    <th bgcolor="#5050A0"></th>
    <th align="left">Medium: at least two papers mention a variant that leads to the same amino acid change</th>
  </tr>
  <tr>
    <th bgcolor="#8282D2"></th>
    <th align="left">Low: only a single paper mentions a variant that leads to the same amino acid change</th>
  </tr>
</table>
</p>

<p>
The three numbers that are shown on the mouse-over and the details page have the following meaning (MM=Mastermind):
<ul>
    <li><b>MMCNT1</b>: cDNA-level exact matches. This is the number of articles that mention the variant at the nucleotide level in either the title/abstract or the full-text.
    <li><b>MMCNT2</b>: cDNA-level possible matches. This is the number of articles with nucleotide-level matches (from 1) plus articles with protein-level matches in which the publication did not specify the cDNA-level change, meaning they could be referring to this nucleotide-level variant but there is insufficient data in these articles to determine conclusively.
    <li><b>MMCNT3</b>: This is the number of articles citing any variant resulting in the same biological effect as this variant. This includes the articles from MMCNT1 and MMCNT2 plus articles with alternative cDNA-level variants that result in the same protein effect.
</ul>
On the track settings page one can filter on these scores under the display mode section by entering
a minimum number of articles for each kind of evidence.

</p>

<H2>Data access</H2>
<p>
The raw data can be explored interactively with the <a href="../cgi-bin/hgTables">Table Browser</a>
or the <a href="../cgi-bin/hgIntegrator">Data Integrator</a>. The data can be accessed from scripts through our 
<a href="https://api.genome.ucsc.edu">API</a>, the track name is "mastermind".

<p>
For automated download and analysis, the genome annotation is stored in a bigBed file that
can be downloaded from
<a href="http://hgdownload.soe.ucsc.edu/gbdb/$db/bbi/" target="_blank">our download server</a>.
The file for this track is called <tt>mastermind.bb</tt>. Individual
regions or the whole genome annotation can be obtained using our tool <tt>bigBedToBed</tt>
which can be compiled from the source code or downloaded as a precompiled
binary for your system. Instructions for downloading source code and binaries can be found
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">here</a>.
The tool
can also be used to obtain only features within a given range, e.g. 
<tt>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/hg19/bbi/mastermind.bb -chrom=chr21 -start=0 -end=100000000 stdout</tt></p>
</p>

<p>
Previous versions of this track can be found on our <a href="http://hgdownload.soe.ucsc.edu/goldenPath/archive/$db/mastermind">archive download server</a>.
</p>

<h2>Methods</h2>

<p>The Mastermind Cited Variants file was <a target=_blank href="https://www.genomenon.com/cvr/">downloaded</a>,
converted to BED format with scripts that are available in our 
<a target=_blank href="https://github.com/ucscGenomeBrowser/kent/tree/master/src/hg/utils/otto/mastermind">Git
repository</a> and converted to a bigBed file with the UCSC genome browser tool
bedToBigBed.</p>

<p>This track is automatically updated two weeks after every Mastermind CVR release, which happens every three months.</p>


<h2>Credits</h2>

<p>
Thanks to Mark Kiel, Steve Schwartz and Clayton Wheeler from Genomenon for making these data available.
</p>

<!-- <h2>References</h2> -->

