<H2>Description</H2>
<P>
The Modern Human Seq track shows human sequence reads of five individuals mapped
to the human genome. The purpose of this track is to put the divergence of the
Neandertal genomes into perspective with regard to present-day humans.</P>

<H2>Display Conventions and Configuration</H2>
<P>
The sequence reads (query sequences) from each of the five individuals are
contained in separate subtracks. Use the checkboxes to select which
individuals will be displayed in the browser. Click and drag the
sample name to reorder the subtracks. The order in which the subtracks appear in
the subtrack list will be the order in which they display in the browser.</P>
<P>
The query sequences in the SAM/BAM alignment representation 
are normalized to the + strand of the reference genome
(see the <A HREF="http://samtools.github.io/hts-specs/SAMv1.pdf#page=4"
TARGET=_blank>SAM Format Specification</A>
for more information on the SAM/BAM file format). If a query sequence was
originally the reverse of what has been stored and aligned, it will have the
following
<A HREF="http://samtools.github.io/hts-specs/SAMv1.pdf#page=5"
TARGET=_blank>flag</A>:
<PRE>
(0x10) Read is on '-' strand.
</PRE></P>
<P>
BAM/SAM alignment representations also have <A HREF="http://samtools.github.io/hts-specs/SAMv1.pdf#page=6"
TARGET=_blank>tags</A>. Some tags are predefined and others (those beginning
with X, Y or Z) are defined by the aligner or data submitter.
The following is a list of the tags associated with this track. For this 
track, those starting with X are specific to the 
<A HREF="http://bio-bwa.sourceforge.net/"
TARGET=_blank>Burrows-Wheeler Aligner (BWA)</A>.
<UL>
<LI><A HREF="http://bio-bwa.sourceforge.net/bwa.shtml"
TARGET=_blank><B>XT</B></A>: Type: Unique/Repeat/N/Mate-sw</LI>
<LI><A HREF="http://samtools.github.io/hts-specs/SAMv1.pdf#page=7"
TARGET=_blank><B>NM</B></A>: Number of nucleotide differences (i.e. edit distance to the reference sequence)</LI>
<LI><A HREF="http://samtools.github.io/hts-specs/SAMv1.pdf#page=7"
TARGET=_blank><B>SM</B></A>: Mapping quality if the read is mapped as a single read rather than as a read pair</LI>
<LI><A HREF="http://samtools.github.io/hts-specs/SAMv1.pdf#page=7"
TARGET=_blank><B>AM</B></A>: Smaller single-end mapping quality of the two reads in a pair</LI>
<LI><A HREF="http://bio-bwa.sourceforge.net/bwa.shtml"
TARGET=_blank><B>X0</B></A>: Number of best hits</LI>
<LI><A HREF="http://bio-bwa.sourceforge.net/bwa.shtml"
TARGET=_blank><B>X1</B></A>: Number of suboptimal hits found by BWA</LI>
<LI><A HREF="http://bio-bwa.sourceforge.net/bwa.shtml"
TARGET=_blank><B>XM</B></A>: Number of mismatches in the alignment</LI>
<LI><A HREF="http://bio-bwa.sourceforge.net/bwa.shtml"
TARGET=_blank><B>XO</B></A>: Number of gap opens</LI>
<LI><A HREF="http://bio-bwa.sourceforge.net/bwa.shtml"
TARGET=_blank><B>XG</B></A>: Number of gap extentions</LI>
<LI><A HREF="http://samtools.github.io/hts-specs/SAMv1.pdf#page=7"
TARGET=_blank><B>MD</B></A>: String for mismatching positions in the format of [0-9]+(([ACGTN]|\^[ACGTN]+)[0-9]+)*</LI>
       </UL>
        </LI>
</UL>
</P>
<P>
The item labels and display colors of features within this track can be
configured through the controls at the top of the track description page.
</P>
<UL>
<LI><B>Display Read Names</B>: By default, read names are not displayed. To 
display the read names, selected the check box next to "Display read names".
</LI>

<LI><B>Attempt to join paired end reads by name</B>: When checked (default),
reads with the same name will be joined into pairs for display,
with a line drawn between them.</LI>

<LI><B>Minimum alignment quality</B>: Excludes alignments with quality less than
the given number. The default is 0.</LI>

<LI><B>Color track by bases</B>: By default, mismatching bases are highlighted
in the display. Change the selection to &quot;item bases&quot; to see all base
values from the query sequence, or &quot;OFF&quot; to ignore query sequence.
Click <A HREF="/goldenPath/help/hgBaseLabel.html">here</A> for additional 
information.</LI>

<LI><B>Alignment Gap/Insertion Display Options</B>: Click
<A HREF="/goldenPath/help/hgIndelDisplay.html">here</A> for help with 
these options.

<LI><B>Additional coloring modes</B>: Other aspects of the alignments can be
displayed in color or grayscale.</LI>
    <UL>
    <LI><B>Color by strand</B>: Alignments on the reverse strand are colored 
    dark red, alignments on the forward strand are colored dark blue.</LI>

    <LI><B>Grayscale</B>: Items are shaded according to the chosen method:
    alignment quality, base qualities, or unpaired ends.
    The alignment qualities of individual items are shaded on a scale of 0 
    (lightest) to 99 (darkest).
    Base qualities are shaded on a scale of 0 (lightest) to 40 (darkest).
    When &quot;unpaired ends&quot; is selected, items that were paired in 
    sequencing but whose mate was not mapped are colored gray, while singletons 
    and properly paired items are black. Alignment quality is the default.</LI>
    </UL>
    </LI>
</UL>
</P>

<H2>Methods</H2>
<P>
The genomes of a San individual from Southern Africa (HGDP01029), a Yoruba
individual from West Africa (HGDP00927), a Han Chinese individual (HGDP00778), 
an individual from Papua New Guinea (HGDP00542), and a French individual
(HGDP00521) from Western Europe were sequenced to 4- to 6-fold coverage on
the Illumina GAII platform. These sequences were aligned to the human
reference genome (NCBI36/hg18) using the <A HREF="http://bio-bwa.sourceforge.net/"
TARGET=_blank>Burrows-Wheeler Aligner (BWA)</A>. Reads with an alignment
quality of less than 30 were not included in these data. Those with an alignment
quality greater than or equal to 30 were analyzed using a similar approach to
that used for the Neandertal data.
</P>

<H2>Credits</H2>
<P>
This track was produced at UCSC using data generated by
<A HREF="mailto:&#101;&#100;&#64;&#115;&#111;e.&#117;&#99;s&#99;.&#101;&#100;&#117;"
title="&#101;&#100;&#64;&#115;&#111;e.&#117;&#99;s&#99;.&#101;&#100;&#117;" rel="nofollow"
TARGET=_BLANK>Ed Green</A>.
</P>

<H2>References</H2>
<P>
Green RE, Krause J, Briggs AW, Maricic T, Stenzel U, Kircher M, Patterson N,
Li H, Zhai W, Fritz MH  <em>et al.</em>
<A HREF="https://science.sciencemag.org/content/328/5979/710"
TARGET=_BLANK>A Draft Sequence of the Neandertal Genome</A>.
<em>Science</em>. 2010 7 May;328(5979):710-22.
</P>
<P>
Li H, Durbin R. <A HREF="https://www.ncbi.nlm.nih.gov/pubmed/19451168"
TARGET=_blank>Fast and accurate short read alignment with Burrows-Wheeler 
Transform</A>.  <EM> Bioinformatics</EM>. 2009 Jul 15;25(14):1754-60.</P>
