<H2>Description</H2>
<P>
This track shows 13 regions of the human genome in which there is
considerably more haplotype diversity among non-African genomes than
within African genomes.  A prediction of Neandertal-to-modern human
gene flow is that these deeply divergent haplotypes which exist only in
non-African populations entered the human gene pool from Neandertals.
Of the 12 candidate gene flow regions with tag SNP data, there are 10
regions in which Neandertals match the deep haplotype clade unique to
non-Africans (out of Africa, OOA) instead of the cosmopolitan
haplotype clade shared by Africans and non-Africans (cosmopolitan,
COS).
</P>
<P>
The table below was copied from Table 5, &quot;Non-African haplotypes 
match Neandertal at an unexpected rate&quot;, from Green <em>et al.</em>:
<BLOCKQUOTE><TABLE BORDER=1 CELLPADDING=4 BORDERCOLOR="#aaaaaa">
<TR><TD>Region</TD><TD>Genomic Size</TD><TD>S<sub>T</sub></TD>
<TD>Average<BR>Frequency<BR>in OOA</TD><TD>AM</TD><TD>DM</TD><TD>AN</TD><TD>DN</TD>
<TD>Qualitative<BR>Assessment</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr1:168,110,001-168,220,000&ntOoaHaplo=pack">chr1:168,110,001-168,220,000</A></TD><TD>110,000</TD><TD>2.9</TD><TD>6.3%</TD><TD>5</TD><TD>10</TD><TD>1</TD><TD>0</TD><TD>OOA</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr1:223,760,001-223,910,000&ntOoaHaplo=pack">chr1:223,760,001-223,910,000</A></TD><TD>150,000</TD><TD>2.8</TD><TD>6.3%</TD><TD>1</TD><TD>4</TD><TD>0</TD><TD>0</TD><TD>OOA</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr4:171,180,001-171,280,000&ntOoaHaplo=pack">chr4:171,180,001-171,280,000</A></TD><TD>100,000</TD><TD>1.9</TD><TD>5.2%</TD><TD>1</TD><TD>2</TD><TD>0</TD><TD>0</TD><TD>OOA</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr5:28,950,001-29,070,000&ntOoaHaplo=pack">chr5:28,950,001-29,070,000</A></TD><TD>120,000</TD><TD>3.8</TD><TD>3.1%</TD><TD>16</TD><TD>16</TD><TD>6</TD><TD>0</TD><TD>OOA</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr6:66,160,001-66,260,000&ntOoaHaplo=pack">chr6:66,160,001-66,260,000</A></TD><TD>100,000</TD><TD>5.7</TD><TD>28.1%</TD><TD>6</TD><TD>6</TD><TD>0</TD><TD>0</TD><TD>OOA</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr9:32,940,001-33,040,000&ntOoaHaplo=pack">chr9:32,940,001-33,040,000</A></TD><TD>100,000</TD><TD>2.8</TD><TD>4.2%</TD><TD>7</TD><TD>14</TD><TD>0</TD><TD>0</TD><TD>OOA</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr10:4,820,001-4,920,000&ntOoaHaplo=pack">chr10:4,820,001-4,920,000</A></TD><TD>100,000</TD><TD>2.6</TD><TD>9.4%</TD><TD>9</TD><TD>5</TD><TD>0</TD><TD>0</TD><TD>OOA</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr10:38,000,001-38,160,000&ntOoaHaplo=pack">chr10:38,000,001-38,160,000</A></TD><TD>160,000</TD><TD>3.5</TD><TD>8.3%</TD><TD>5</TD><TD>9</TD><TD>2</TD><TD>0</TD><TD>OOA</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr10:69,630,001-69,740,000&ntOoaHaplo=pack">chr10:69,630,001-69,740,000</A></TD><TD>110,000</TD><TD>4.2</TD><TD>19.8%</TD><TD>2</TD><TD>2</TD><TD>0</TD><TD>1</TD><TD>OOA</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr15:45,250,001-45,350,000&ntOoaHaplo=pack">chr15:45,250,001-45,350,000</A></TD><TD>100,000</TD><TD>2.5</TD><TD>1.1%</TD><TD>5</TD><TD>6</TD><TD>1</TD><TD>0</TD><TD>OOA</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr17:35,500,001-35,600,000&ntOoaHaplo=pack">chr17:35,500,001-35,600,000</A></TD><TD>100,000</TD><TD>2.9</TD><TD>(no tags)</TD><TD>n/a</TD><TD>n/a</TD><TD>n/a</TD><TD>n/a</TD><TD>n/a</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr20:20,030,001-20,140,000&ntOoaHaplo=pack">chr20:20,030,001-20,140,000</A></TD><TD>110,000</TD><TD>5.1</TD><TD>64.6%</TD><TD>0</TD><TD>0</TD><TD>10</TD><TD>5</TD><TD>COS</TD></TR>
<TR><TD><A HREF="../cgi-bin/hgTracks?db=$db&position=chr22:30,690,001-30,820,000&ntOoaHaplo=pack">chr22:30,690,001-30,820,000</A></TD><TD>130,000</TD><TD>3.5</TD><TD>4.2%</TD><TD>0</TD><TD>2</TD><TD>5</TD><TD>2</TD><TD>COS</TD></TR>
</TABLE>
<BR>
S<sub>T</sub> = estimated ratio of OOA/African gene tree depth.<BR>
Average Frequency in OOA = average (across tag SNPs in the region) of the population frequency in the 48 OOA individuals of the OOA-only allele for each tag SNP.<BR>
AM = Neandertal has ancestral allele and matches OOA-specific clade.<BR>
DM = Neandertal has derived allele and matches OOA-specific clade.<BR>
AN = Neandertal has ancestral allele and does not match OOA-specific clade.<BR>
DN = Neandertal has derived allele and does not match OOA-specific clade.
</P></BLOCKQUOTE>

<H2>Display Conventions and Configuration</H2>
<P>
A region is colored green if its qualitative assessment is OOA, blue
if COS, and gray if unknown (no tag SNPs in region).
</P>

<H2>Methods</H2>
<P>
Green <em>et al.</em> used 1,263,750 Perlegen Class A SNPs, identified
in 71 individuals of diverse ancestry (see Hinds <em>et al.</em>), to
identify 13 candidate gene flow regions (Supplemental Online Materials
Text 17).
24 individuals of European ancestry and 24 individuals of Han Chinese
ancestry were used to represent the non-African population, and the
remaining 23 individuals, of African American ancestry, were used to
represent the African population.
</P>
<P>
From the 1,263,750 Perlegen Class A SNPs, they identified 166 tag
SNPs that separate (see below) 12 of the haplotype clades in
non-Africans (OOA) from the cosmopolitan haplotype clades shared between
Africans and non-Africans (COS) and for which they had data from the
Neandertals. Of the 13 regions, one had no tag SNPs so could not be
assessed, two were COS, and 10 were OOA (see final column Table 1).
</P>
<P>
Overall, the Neandertals match the deep clade unique to non-Africans
(OOA) at 133 of the 166 tag SNPs. They assessed the rate at which
Neandertal matches each of these clades by further subdividing the 133
tag SNPs based on their ancestral or derived status in
Neandertal and whether they matched the OOA-specific clade or not.
Candidate regions were qualitatively assessed to be OOA matches for 
Neandertal when the proportion of tag SNPs matching the OOA-specific 
clade is much more than 50%.
</P>

<H2>Credits</H2>
<P>
This track was produced at UCSC using data generated by
<A HREF="mailto:&#101;&#100;&#64;&#115;&#111;e.&#117;&#99;s&#99;.&#101;&#100;&#117;"
title="&#101;&#100;&#64;&#115;&#111;e.&#117;&#99;s&#99;.&#101;&#100;&#117;" rel="nofollow"
TARGET=_BLANK>Ed Green</a>.
</P>

<H2>References</H2>
<p>
Green RE, Krause J, Briggs AW, Maricic T, Stenzel U, Kircher M, Patterson N, Li H, Zhai W, Fritz MH
<em>et al</em>.
<a href="https://science.sciencemag.org/content/328/5979/710" target="_blank">
A draft sequence of the Neandertal genome</a>.
<em>Science</em>. 2010 May 7;328(5979):710-22.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20448178" target="_blank">20448178</a>
</p>

<p>
Hinds DA, Stuve LL, Nilsen GB, Halperin E, Eskin E, Ballinger DG, Frazer KA, Cox DR.
<a href="https://science.sciencemag.org/content/307/5712/1072" target="_blank">
Whole-genome patterns of common DNA variation in three human populations</a>.
<em>Science</em>. 2005 Feb 18;307(5712):1072-9.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15718463" target="_blank">15718463</a>
</p>
