<H2>Description</H2>

<!--#insert file="omimDescriptionText.html"-->

<P>
This track shows the <B>cytogenetic locations</B> of phenotype entries in the Online Mendelian
Inheritance in Man (<A HREF="https://www.omim.org/" TARGET=_blank>OMIM</A>) database for which
the gene is unknown.
</P>

<H2>Display Conventions and Configuration</H2>

<P>Cytogenetic locations of OMIM entries are displayed as solid
blocks.  The entries are colored according to the OMIM phenotype map key of associated disorders:

<UL>
<LI><B><FONT COLOR="#AAC4AA">Lighter Green</FONT></B> for phenotype map key 1 OMIM records
-  the disorder has been placed on the map based on its association with
a gene, but the underlying defect is not known.
<LI><B><FONT COLOR="#669666">Light Green</FONT></B> for phenotype map key 2 OMIM records
- the disorder has been placed on the map by linkage; no mutation has
been found.
<LI><B><FONT COLOR="#005000">Dark Green</FONT></B> for phenotype map key 3 OMIM records
- the molecular basis for the disorder is known; a mutation has been
found in the gene.
<LI><B><FONT COLOR="#69329B">Purple</FONT></B> for phenotype map key 4 OMIM records
- a contiguous gene deletion or duplication syndrome; multiple genes
are deleted or duplicated causing the phenotype.
</UL>
<P>Gene symbols and disease information, when available, are displayed on the details pages.
</P>
<P>The descriptions of OMIM entries are shown on the main browser display when Full display
mode is chosen.  In Pack mode, the descriptions are shown when mousing over each entry. Items
displayed can be filtered according to phenotype map key on the track controls page.
</P>

<H2>Methods</H2>
<P>
This track was constructed as follows: 
<UL>
<LI>The data file <TT>genemap.txt</TT> from OMIM was loaded into the MySQL table
<TT>omimGeneMap</TT>.
<LI>Entries in <TT>genemap.txt</TT> having disorder info were parsed and loaded into the
<TT>omimPhenotype</TT> table. The phenotype map keys (the numbers (1)(2)(3)(4) from the
disorder columns) were placed into a separate field.
<LI>The cytogenetic location data (from the location column in <TT>omimGeneMap</TT>) were
parsed and converted into genomic start and end positions based on the <TT>cytoBand</TT> table.
These genomic positions, together with the corresponding OMIM IDs, were loaded into the
<TT>omimLocation</TT> table.
<LI>All entries with no associated phenotype map key and all OMIM gene entries as reported in the
&quot;OMIM Genes&quot; track were then excluded from the <TT>omimLocation</TT> table.
</UL>

<h2>Data Access</h2>
<p>
Because OMIM has only allowed Data queries within individual chromosomes, no download files are
available from the Genome Browser. Full genome datasets can be downloaded directly from the
<a href="https://omim.org/downloads/" target="_blank">OMIM Downloads page</a>.
All genome-wide downloads are freely available from OMIM after registration.</p>
<p>
If you need the OMIM data in exactly the format of the UCSC Genome Browser,
for example if you are running a UCSC Genome Browser local installation (a partial &quot;mirror&quot;),
please create a user account on omim.org and contact OMIM via
<a href="https://omim.org/contact" target="_blank">https://omim.org/contact</a>. Send them your OMIM
account name and request access to the UCSC Genome Browser 'entitlement'. They will
then grant you access to a MySQL/MariaDB data dump that contains all UCSC
Genome Browser OMIM tables.</p>
<p>
UCSC offers queries within chromosomes from
<a href="hgTables" target=_blank>Table Browser</a> that include a variety
of filtering options and cross-referencing other datasets using our
<a href="hgIntegrator" target=_blank>Data Integrator</a> tool.
UCSC also has an <a href="../goldenPath/help/api.html" target=_blank>API</a>
that can be used to retrieve data in JSON format from a particular chromosome range.</p>
<p>
Please refer to our searchable
<A HREF="https://groups.google.com/a/soe.ucsc.edu/forum/?hl=en&fromgroups#!search/download+snps"
target=_blank>mailing list archives</a>
for more questions and example queries, or our
<a HREF="../FAQ/FAQdownloads.html#download36" target=_blank>Data Access FAQ</a>
for more information.</p>

<H2>Credits</H2>
<P>
Thanks to OMIM and NCBI for the use of their data. This track was constructed by Fan Hsu,
Robert Kuhn, and Brooke Rhead of the UCSC Genome Bioinformatics Group.</P>

<H2>References</H2>
<p>
Amberger J, Bocchini CA, Scott AF, Hamosh A.
<a href="https://academic.oup.com/nar/article/37/suppl_1/D793/1003813" target="_blank">
McKusick's Online Mendelian Inheritance in Man (OMIM)</a>.
<em>Nucleic Acids Res</em>. 2009 Jan;37(Database issue):D793-6.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18842627" target="_blank">18842627</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2686440/" target="_blank">PMC2686440</a>
</p>

<p>
Hamosh A, Scott AF, Amberger JS, Bocchini CA, McKusick VA.
<a href="https://academic.oup.com/nar/article/33/suppl_1/D514/2505259" target="_blank">
Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic
disorders</a>.
<em>Nucleic Acids Res</em>. 2005 Jan 1;33(Database issue):D514-7.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15608251" target="_blank">15608251</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC539987/" target="_blank">PMC539987</a>
</p>
