<H2>Description</H2>
<P>
This track shows the location of recombination hotspots detected from
patterns of genetic variation. 
It is based on the HapMap Phase I data, release 16a, and Perlegen data (Hinds <em>et al.</em>, 2005).
</P>
<P>
Observations from sperm studies (Jeffreys <em>et al</em>., 2001) and
patterns of genetic variation (McVean <em>et al</em>., 2004; Crawford
<em>et al</em>., 2004) show that recombination rates in the human
genome vary extensively over kilobase scales and that much
recombination occurs in recombination hotspots. This provides an
explanation for the apparent block-like structure of linkage
disequilibrium (Daly <em>et al</em>., 2001; Gabriel <em>et al</em>.,
2002).
</P>
<P>
Recombination hotspot estimates provide a new route to
understanding the molecular mechanisms underlying human recombination.
A better understanding of the genomic landscape of human recombination
hotspots would facilitate the efficient design and analysis of
disease association studies and greatly improve inferences from
polymorphism data about selection and human demographic history.
</P>

<H2>Methods</H2>
<P>
Recombination hotspots are identified using the likelihood-ratio test
described in McVean <I>et al</I>. (2004) and Winckler <I>et al</I>. (2005), 
referred to as LDhot.  For successive intervals of 200 kb, the maximum
likelihood of a model with a constant recombination rate is compared
to the maximum likelihood of a model in which the central 2 kb is a
recombination hotspot (likelihoods are approximated by the composite
likelihood method of Hudson 2001).  The observed difference in log
composite likelihood is compared against the null distribution, which
is obtained by simulations.  Simulations are matched for sample size,
SNP density, background recombination rate and an approximation to the
ascertainment scheme (a panel of 12 individuals with a Poisson number
of chromosomes, mean 1, sampled from this panel, using a single hit
ascertainment scheme for dbSNP and resequencing of 16 individuals for
the ten HapMap ENCODE regions).  Evidence for a hotspot was assessed in
each analysis panel separately (YRI, CEU and combined CHB+JPT), and
p-values were combined such that a hotspot requires that two of the
three populations show some evidence of a hotspot (p &lt; 0.05) and at
least one population showed stronger evidence for a hotspot
(p &lt; 0.01). Hotspot centers were estimated at those locations where
distinct recombination rate estimate peaks occurred with at least a factor 
of two separation between peaks, within the low p-value intervals.
</P>

<H2>Validation</H2>
<P>
This approach has been validated in three ways:  Over large scales from 
the genetic map, both by extensive simulation studies and by comparisons with 
independent estimates of recombination rates, and over fine scales from sperm 
analysis.  Full details of validation can be 
found in McVean <em>et al</em>. (2004) and Winckler <em>et al</em>. (2005).
</P>

<H2>Credits</H2>
<P>
The HapMap data are based on <A HREF="https://www.ncbi.nlm.nih.gov/variation/news/NCBI_retiring_HapMap/" TARGET=_blank>HapMap</A> 
release 16a; the Perlegen data are from Hinds <em>et al</em>. (2005).  
The recombination hotspots were ascertained by Simon Myers from the
<A HREF="http://mathgen.stats.ox.ac.uk/"
TARGET=_blank>Mathematical Genetics Group</A> at the University of Oxford.
</P>

<H2>References</H2>
<P>
Crawford, D.C., Bhangale, T., Li, N., Hellenthal, G., Rieder, M.J., 
Nickerson, D.A. and Stephens, M.
<A HREF="https://www.nature.com/articles/ng1376"
TARGET=_blank>Evidence for substantial fine-scale variation in recombination 
rates across the human genome</A>.
<em>Nat Genet.</em> <B>36</B>(7), 700-6 (2004).
</P>
<P>
Daly, M.J., Rioux, J.D., Schaffner, S.F., Hudson, T.J. and  Lander, E.S.
<A HREF="https://www.nature.com/articles/ng1001-229"
TARGET=_blank>High-resolution haplotype structure in the human genome</A>.
<em>Nat Genet.</em> <B>29</B>(2), 229-32 (2001).
</P>
<P>
Gabriel, S.B., Schaffner, S.F., Nguyen, H., Moore, J.M., Roy, J., Blumenstiel, 
B., Higgins, J., DeFelice, M., Lochner, A., Faggart, M. <em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=12029063&query_hl=3"
TARGET=_blank>The structure of haplotype blocks in the human genome</A>.
<em>Science</em> <B>296</B>(5576), 2225-9 (2002).
</P>
<P>
Hudson, R. R. Two-locus sampling distributions and their application.  <I>Genetics</I> <B>159</B>(4):1805-1817 (2001).
</P>
<P>
Hinds, D.A., Stuve, L.L., Nilsen, G.B., Halperin, E., Eskin, E., Ballinger, D.G., Frazer, K.A., Cox, D.R.
<A HREF="https://science.sciencemag.org/content/307/5712/1072"
TARGET=_blank>Whole-Genome Patterns of Common DNA Variation in Three Human Populations</A>.
<em>Science</em> <B>307</B>(5712), 1072-1079 (2005).
</P>
<P>
Jeffreys, A.J,. Kauppi, L. and Neumann, R.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&cmd=Retrieve&db=PubMed&list_uids=11586303&dopt=Abstract"
TARGET=_blank>Intensely punctate meiotic recombination in the class II region 
of the major histocompatibility complex</A>.
<em>Nat Genet.</em> <B>29</B>(2), 217-22 (2001).
</P>
<P>
McVean, G.A., Myers, S.R., Hunt, S., Deloukas, P., Bentley, D.R. and Donnelly, 
P.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15105499&query_hl=9"
TARGET=_blank>The fine-scale structure of recombination rate variation in the 
human genome</A>.
<em>Science</em> <B>304</B>(5670), 581-4 (2004).
</P>
<P>
Winckler, W., Myers, S.R., Richter, D.J., Onofrio, R.C., McDonald, G.J., 
Bontrop, R.E., McVean, G.A., Gabriel, S.B., Reich, D., Donnelly, P. 
<em>et al</em>.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15705809&query_hl=9"
TARGET=_blank>Comparison of fine-scale recombination rates in humans and 
chimpanzees</A>.
<em>Science</em> <B>308</B>(5718), 107-11 (2005).
</P>
