<H2>Description</H2>
<P>
ChIP-seq is a new approach for investigating entire transcription factor-DNA interactomes.
ChIP-seq is comprised of chromatin immunoprecipitation (ChIP) followed by single-molecule-based
sequencing (seq).  ChIP-seq avoids the complications of array hybridization.
<P>
This track shows 25mer sequence reads obtained via a ChIP-seq protocol 
from an NRSF/REST-enriched ChIP sample and a companion control sample
of the same fixed chromatin without immuno-enrichment.   These sequence
reads were later analyzed using a peak locator (data to be available in the future)
to identify NRSF-positive binding events.
<P>
Chromatin immunoprecipitation was applied to NRSF/REST loci identified by a
PSFM (position specific frequency matrix) computational screen.   NRSF/REST
is a zinc finger repressor that negatively regulates many neuronal genes in
stem and progenitor cells and in nonneuronal cell types such as the
Jurkat T cell line used for this track.
<P>
NRSF (neuron-restrictive silencer factor), also known as REST (repressor element-1 
silencing transcription factor), was chosen because prior studies provide
a large set of target genes.  In addition, the DNA motif bound by NRSF 
(known as NRSE/RE1) is long (21 bp) and well-specified, and there is a high-quality 
monoclonal antibody that recognizes NRSF efficiently in ChIP experiments.

<H2>Methods</H2>
<P>
A Jurkat human T lymphoblast cell line from 
<A HREF="http://www.atcc.org/" TARGET=_blank>ATCC</A>
was cultured according to standard protocols.
Chromatin immunoprecipitation was performed as described in Mortazavi et al. (2006) using a
custom monoclonal antibody.  
Libraries were prepared from the ChIP DNA using the ligation mediated PCR Solexa protocol.  
The Solexa library construction protocol was modified to include a PCR preamplification following 
linker ligation and preceding gel electrophoresis.  Reducing the size and narrowing the
size range of DNAs collected from gel purification is intended to improve positional resolution of
ChIP-seq.
<P>
<A HREF="http://woldlab.caltech.edu/html/cistematic" TARGET=_blank>Cistematic</A>
(Wold lab) was used to perform motif-oriented analysis.  The NRSE2 PSFM was used to
identify canonical NRSE sites.   These sites were used to compare and call the distances from
peaks of ChIP-seq read-tag distributions at each location.
Reads which mapped to multiple genomic locations are not included.
Up to two mismatches were allowed.

<H2>Verification</H2>
<P>
These data represent a pool of 4 replicate immunoprecipitations.  
<P>
NRSF-binding sites previously identified by qPCR or transfection assays (74 loci) plus a set of known negatives
(139 loci) were used to estimate a sensitivity of 87% and a specificity of 97% for clusters of 13 tags 
separated by no more than 100 bp.
<P>
771 computationally high-scoring NRSE motifs were to assess the precision of ChIP-seq site location. 
754 sites were found in the ChIP-seq experiments, and the center of a 21-bp NRSE motif was within 50 base pairs
of the called ChIPSeq peak for 94% of these.
<P>
NRSF binding in or near promoters is expected to be correlated with low levels of transcription.
Labeled cRNA was hybridized to Illumina Sentrix RefSeq8 whole-genome gene expression microarrays.
Illumina BeadStudio software was used to extract and normalize the data (rank-invariant method).
A database of transcription start sites from 
<A HREF="http://www.switchdb.com/" TARGET=_blank>SwitchGear Genomics</A>
was used to obtain high-confidence
promoter predictions.   The 230 transcripts occurring near ChIP-seq peaks had a median expression 
intensity of 6.8, while the full set of 20,589 transcripts had a median expression intensity of 23.6.
The difference of medians was significant (P = 1 x 10-11) by the Mann-Whitney test.

<H2>Display conventions</H2>

Sequence reads that align to the forward strand are displayed in green; reads that align to the reverse
strand are displayed in red.

<H2>Credits</H2>
<P>
<A HREF="http://www-shgc.stanford.edu/myerslab" TARGET=_blank>
Myers Group</A>: David Johnson, Betsy Anton, Loan Nguyen, Cat Medina, Richard Myers.
<P>
<A HREF="http://woldlab.caltech.edu/html/woldlab" TARGET=_blank>
Wold Group</A>: Barbara Wold, Ali Mortazavi, Kenneth McCue.
<P>
<A HREF="http://www.illumina.com" TARGET=_blank>Solexa/Illumina</A> Sequencing: Gary Schroth.

<H2>References</H2>
<P>
Fields S.
<A HREF="http://www.sciencemag.org/cgi/content/summary/316/5830/1441" TARGET=_blank>
Site-Seeing by Sequencing.</A>
<em>Science</em> 2007 June;<B>316</B>:1441-1442.
<P>
Johnson DS, Mortazavi A, Myers RM, Wold B.
<A HREF="http://www.sciencemag.org/cgi/content/abstract/316/5830/1497" TARGET=_blank>
Genome-Wide Mapping of in Vivo Protein-DNA Interactions.</A>
<em>Science</em> 2007 June;<B>316</B>:1497-1502.
<P>
Mortazavi A, Thompson EC, Garcia ST, Myers RM, Wold B. 
<A HREF="http://www.genome.org/cgi/content/abstract/16/10/1208" TARGET=_blank>
Comparative genomics modeling of the NRSF/REST repressor network: 
from single conserved sites to genome-wide repertoire.</A>
<em>Genome Res. </em>2006 Oct;<B>16</B>(10):1208-21. 
<P>Schoenherr CJ, Paquette AJ, Anderson DJ.
<A HREF="http://www.pnas.org/cgi/content/abstract/93/18/9881" TARGET=_blank>
Identification of potential target genes for the neuron-restrictive silencer factor.</A>
<em>Proc. Natl. Acad. Sci.</em>1996;<B>93</B>:9881-9886.
<H2>Related Work</H2>
<P>
Mikkelsen TS et.al.
<A HREF="http://www.nature.com/nature/journal/vaop/ncurrent/abs/nature06008.html" TARGET=_blank>
Genome-wide maps of chromatin state in pluripotent and lineage-committed cells.</A>
<em>Nature</em> 5 July 2007; <B>448</B>(7149).
