<H2>Description</H2>
<P>
This track displays positions of four different types of RNA in the human 
genome: 
<UL>
<LI>microRNAs from the
<A HREF="http://www.sanger.ac.uk/Software/Rfam/mirna/" TARGET=_blank>
miRNA Registry</A> at the <A HREF="http://www.sanger.ac.uk/" TARGET=_blank>
Wellcome Trust Sanger Institute</A>
<LI>small nucleolar RNAs (C/D box and H/ACA box snoRNAs) and Cajal body-specific RNAs (scaRNAs) from the 
<A HREF="http://www-snorna.biotoul.fr/" TARGET=-blank>snoRNABase</A> maintained at the 
<A HREF="http://www-lbme.biotoul.fr/uk/index.html" TARGET=_blank>Laboratoire de Biologie 
Moléculaire Eucaryote</A>
</UL></P>
<P>
C/D box and H/ACA box snoRNAs are guides for the 2'O-ribose methylation and 
the pseudouridilation, respectively, of rRNAs and snRNAs, although many of 
them have no documented target RNA.  The scaRNAs guide modifications of the
spliceosomal snRNAs transcribed by RNA polymerase II, and often contain both 
C/D and H/ACA domains.</P>

<H2>Display Conventions and Configuration</H2>
<P>
This track follows the general display conventions for 
<A HREF="../goldenPath/help/hgTracksHelp.html#GeneDisplay">gene prediction 
tracks</A>. </P>
<P>
The miRNA precursor  forms (pre-miRNA) are represented by red blocks.</P>
<P>
C/D box snoRNAs, H/ACA box snoRNAs and scaRNAs are represented by blue, green and 
magenta blocks, respectively. At a zoomed-in resolution, arrows superimposed 
on the blocks indicate the sense orientation of the snoRNAs. </P>

<H2>Methods</H2>
<P>
Mature and precursor miRNAs from the miRNA Registry were aligned against the 
genome using blat.
The extents of the precursor sequences were not generally known and were
predicted based on base-paired hairpin structure.  The miRNA Registry is
described in Griffiths-Jones, S. (2004) and Weber, M.J. (2005) in the 
References section below.</P>
<P>
The snoRNAs and scaRNAs from the snoRNABase were aligned against the 
human genome using blat. 
</P>

<H2>Credits</H2>
<P>
The miRNA annotation was contributed by Michel Weber of 
<A HREF="http://www-lbme.biotoul.fr/uk/index.html" TARGET=_blank>Laboratoire de Biologie 
Moléculaire Eucaryote, CNRS Université Paul Sabatier </A> (UMR5099, Toulouse, 
France) and Sam Griffiths-Jones of <A HREF="http://www.sanger.ac.uk" 
TARGET=_blank>The Wellcome Trust Sanger Institute</A> (Cambridge, UK).</P> 
<P>
The snoRNA annotations were contributed by Michel Weber and 
Laurent Lestrade of the 
<A HREF="http://www.iefg.biotoul.fr/" TARGET=_blank>Institut d'Exploration 
Fonctionnelle des Génomes</A> (IFR109, Toulouse, France). </P>
<P> 
<A HREF="mailto:&#102;a&#110;&#104;s&#117;&#64;&#115;&#111;&#101;.&#117;c&#115;&#99;.&#101;&#100;&#117;">Fan Hsu</A> from the UCSC Genome Bioinformatics 
Group created the combined annotation track.</P>

<H2>References</H2>
<P>
When making use of these data, please cite: 
<P>
Griffiths-Jones S. 
<A HREF="http://nar.oxfordjournals.org/cgi/content/abstract/32/suppl_1/D109" 
TARGET=_blank>The microRNA Registry</A>.
<em>Nucl. Acids Res.</em> 2004 Jan 1;32(D):D109-11.</P>
<P>
Weber MJ. 
<A HREF="http://www.blackwell-synergy.com/doi/abs/10.1111/j.1432-1033.2004.04389.x"
TARGET=_blank>New human and mouse microRNA genes found by homology search</A>.
<em>Febs J.</em> 2005 Jan;272(1):59-73.</P>
<P>
You may also want to cite The Wellcome Trust Sanger Institute 
<A HREF="http://www.sanger.ac.uk/Software/Rfam/mirna/"
TARGET=_blank>miRNA Registry</A> and The Laboratoire de Biologie Moleculaire 
Eucaryote <A HREF="http://www-snorna.biotoul.fr/" TARGET=_blank>snoRNA 
database</A>.</P>
<P>
The following publication provides guidelines on miRNA annotation:
Ambros V. <em>et al</em>., 
<A HREF="http://www.rnajournal.org/cgi/content/abstract/9/3/277" 
TARGET=_blank>A uniform system for microRNA annotation</A>.  
<em>RNA.</em> 2003;9(3):277-9.</P>
<P>
For more information on blat, see 
Kent WJ.
<A HREF="http://www.genome.org/cgi/content/abstract/12/4/656" 
TARGET=_blank>BLAT - the BLAST-like alignment tool</A>.
<em>Genome Res.</em> 2002;12(4):656-664.</P>
