<H2>Description</H2>
<P>
This track shows the locations of transcriptionally active regions 
(TARs)/transcribed fragments (transfrags) hybridized to an oligonucleotide 
microarray with a design based on human assembly hg13 (NCBI Build 31)
(Bertone <em>et al</em>., 2004).</P>

<H2>Methods</H2>
<P>
Microarrays were designed using sequence from the human hg13 assembly. The
genome sequence was screened for repetitive elements and low-complexity DNA
using RepeatMasker in the sensitive mode. Additional low-complexity filtering
was performed using the NSEG (segment sequence(s) by local complexity) 
program using a minimum segment length of 21 nucleotides to determine 
low complexity segments of lowest probability. After filtering, 1.5 Gb of 
nonrepetitive DNA remained and microarray probes were chosen using the NASA 
Oligonucleotide Probe Selection Algorithm (NOPSA).</P>
<P>
NOPSA is designed to find the optimal probes for hybridization. A
database of the frequency of every 18-mer in the genome is created using a
hash algorithm. Chaining was used to resolve collisions. Average frequencies
of 36-mers in the genome were determined from the frequencies of each 18-mer
subsequence in the 36-mer and its reverse complement. 36-mer oligonucleotides
with a frequency equal to one are selected as potential probes for the
microarray (from supporting online material for Stolc <em>et al</em>., 2004) 
</P>
<P>
This resulted in probe selection based on several criteria:
<UL>
<LI>Every 36-mer in the genome is unique. 
<LI>Sequences that could form a loop with a stem of &gt; 7 bp were excluded. 
<LI>Factors such as sequence length, extent of complementarity and base 
composition were also considered. 
</UL></P>
<P>A total of 51,874,388 36-mer 
oligonucleotide probes were selected from both the sense and antisense strands
at an average resolution of 46 bp to cover the non-repetitive sequence from
the whole genome. Probes were spaced every 10 nucleotides on average. The 
probes were synthesized via maskless photolithography at a feature density of 
approximately 390,000 probes per slide.</P>
<P>
Biological samples that were hybridized to the arrays consisted of 
triple-selected human liver poly(A)+ RNA pooled from several individuals 
(supplied by Ambion). One biological replicate was carried out.</P>
<P>
See this NCBI 
<A HREF="https://www.ncbi.nlm.nih.gov/projects/geo/query/acc.cgi?acc=GSE1904"
TARGET=_blank>GEO accession</A> for details of experimental protocols.</P>
<P>
The TARs identified for hg13 (NCBI Build 31) were mapped to this
assembly using Blat.  The program pslCDnaFilter was used to filter 
alignments using the parameters
<TT>-minId=0.96</TT>,<TT> -minCover=0.25</TT>,
<TT> -localNearBest=0.001</TT>,<TT>-minQSize=20</TT>,
<TT> -minNonRepSize=16</TT>,<TT> -ignoreNs</TT>,<TT> -bestOverlap</TT>.</P>

<H2>Display Conventions</H2>
<P>
TARs are represented by blocks in the graphical display. The numeric part of 
the ID displayed when the track has pack or full visibility is the ID used
by the Yale Database for Active Regions with Tools 
(<A HREF="http://papers.gersteinlab.org/papers/dart/" TARGET=_blank>DART</A>). A link to 
this database is provided on the details page for each TAR.</P>

<H2>Data Analysis</H2>
<P>
Two groups of TARs were identified: Normal and Poly(A)-associated.</P>

<H3>Normal TARs:</H3> 
<P>
Clusters of transcription units were identified that consisted of at least 
five consectutive probes with fluorescence intensities in the top 
90th intensity percentile and with genomic coordinates within a 250-nt 
window. After collecting these regions genome-wide, their locations were 
compared to those of annotated components of genes. As a result, a
total of 13,889 transcription units, ranging in size from 209 to 3,438 
nucleotides, 
were identified. Under the null hypothesis of zero transcription, only 400
were expected to be found. Of those regions identified, one-third (4,931) 
correspond to previously annotated exons while the other 8,958 are new 
transcribed sequences that are referred to as TARs.</P>

<H3>Poly(A)-associated TARs:</H3> 
<P>
Another set of criteria was used to find TARs in which the probe hybridization 
intensities were correlated with the presence of a polyadenylation signal 3' 
to the TAR.  Transcription units are five consecutive probes with fluoroscence 
intensities in the top 80th intensity percentile and in a window of 250 
nucleotides.
The 3' region also must contain or be close to a polyadenylation signal.
Transcription units with an associated polyadenylation signal of 
&quot;AATAAA&quot; were assigned to a type I group, while those with 
&quot;ATTAAA&quot; were type II. Only 100 of 
these should occur at random in the genome under the null hypothesis of zero
transcription. The majority (1,991) were found to be within annotated exons, 
and 952 were located more than 10 kb from an annotated gene. A total of 1,371 
type I and 674 type II poly(A) sequences were identified within exons of 
known genes. 1,289 (94%) of type I and 607 (90%) of type II instances were 
found to be in the 3' exon of the gene.</P>

<H2>Verification</H2>
<P>
The TARs were validated using RT-PCR on human liver poly(A)+ RNA. Forty-eight 
poly(A)-associated and 48 non-poly(A)-associated TARs were investigated. 
In 94% (90/96) of cases, the PCR products were found to be of the expected 
size in a single-pass assay.</P>

<H2>Credits</H2>
<P>
These data were generated and analyzed by a collaboration between the labs of 
Michael Snyder, 
<A HREF="http://bioinfo.mbb.yale.edu/" TARGET=_blank>Mark Gerstein</A>, 
and Sherman Weissman at Yale University and with 
NASA Ames Research Center (Moffett Field, California) and Eloret Corporation 
(Sunnyvale, California).</P>

<H2>References</H2>
<P>
Bertone P, Stolc V, Royce TE, Rozowsky JS, Urban AE, Zhu X, Rinn JL, Tongprasit W, Samanta M, Weissman S <em>et al</em>.
<A HREF="https://science.sciencemag.org/content/306/5705/2242"
TARGET=_blank>Global identification of human transcribed sequences with 
genome tiling arrays</A>. 
<em>Science</em>. 2004 Dec 24;306(5705):2242-6.</P>
<P>
Stolc V, Gauhar Z, Mason C, Halasz G, van Batenburg MF, Rifkin SA, Hua S, 
Herreman T, Tongprasit W, Barbano PE <em>et al</em>. 
<A HREF="https://science.sciencemag.org/content/306/5696/655" 
TARGET=_blank>A gene expression map for the euchromatic genome of <em>Drosophilamelanogaster</em></A>.
<em>Science</em>. 2004 Oct 22;306(5696):655-60.</P>
