<H2>Description</H2>
<P>
This track summarizes alternative splicing shown in the mRNA and
EST tracks. The blocks represent exons; lines indicate possible
splice junctions. The graphical display is drawn such that no exons
overlap, making alternative events easier to view when the track
is in full display mode and the resolution is set to approximately gene-level.
</P>
<P>
To help reduce the noise present in the EST libraries, 
exons and splice junctions are filtered based on orthologous human
transcripts and the frequency with which an exon or intron appears in mouse
transcript libraries. Only those exons and splice junctions that have
an orthologous exon or splice junction in the human 
transcriptome or are present three or more times in the mouse transcriptome are 
kept. Transcripts labeled as mRNA in GenBank are weighted more heavily, 
reflecting their typically higher quality. This process is similar
to that presented in Sugnet, C.W. <em>et al.</em>,
<A HREF="http://helix-web.stanford.edu/psb04/sugnet.pdf"
TARGET=_blank>Transcriptome and genome conservation of alternative splicing 
events in humans and mice</A>. 
<em>Pacific Symposium on Biocomputing (PSB) 2004 Online Proceedings</em>. </P>

<H2>Methods</H2>
<P>
The splicing graphs for each genome were generated separately
from their native EST and mRNA transcripts using the following
process: 
<UL>
<LI>
The mRNAs and ESTs were aligned to the genomic sequence using 
<A HREF="../cgi-bin/hgBlat?db=$db">blat</A>. A near-best-in-genome filter was
applied such that only alignments with 97% identity over 90% of the
transcript and with a score no more than 0.5% lower than the best
score were kept.
<LI>
The ESTs and mRNAs were oriented by examining the splice
sites used in the genomic sequence. Only consensus splice sites,
GT->AG and the less common GC->AG, were used to orient the
transcripts.
<LI>
Alignments were clustered together by sequence overlap in
exons. As new splice sites were discovered, they were entered into
the graphs as vertices; the exons, introns, and splice
junctions connecting them were recorded as edges. Each graph was
considered to be a single locus, although they may be fragments of
an actual gene structure. The supporting mRNA and EST accessions
for each edge were also stored.
<LI> 
Truncated transcripts were extended by overlap with other
transcripts to the next consensus splice site. This prevented
the retention of vertices in the graph that were not true splice sites.
</UL></P>
<P>
After the splicing graphs were constructed independently for both
human and mouse, they were mapped to each other using the entire set of
genome human net alignments (viewable on the browser as
the Human Net track). Only those exons and splice junctions that were common 
to both or occurred three or more times in the mouse transcript were kept 
in the splicing graph. When counting the number of times an
exon or splice junction was included in the mouse transcripts, those
designated as mRNA were weighted more heavily than those designated as 
EST.</P>

<H2>References</H2>
<P>
For more information on the human net alignments, see 
Kent, W.J., Baertsch, R., Hinrichs, A., Miller, W., and Haussler, D.
<A HREF="http://www.pnas.org/cgi/content/abstract/1932072100v1" 
TARGET=_blank>Evolution's cauldron: 
Duplication, deletion, and rearrangement in the mouse and human genomes</A>. 
<I>Proc Natl Acad Sci USA</I> <B>100</B>(20), 11484-11489 (2003).</P>

<H2>Credits</H2>
<P>
This annotation was generated by <A HREF="mailto:sugnet@soe.ucsc.edu">Chuck 
Sugnet</A> of the UCSC Genome Bioinformatics Group.</P>
