<H2>Description</H2>
<P>
This track shows the number of 5' cap analysis gene expression (CAGE) tags 
that map to the genome on the &quot;plus&quot; and &quot;minus&quot; strands at 
a specific location.  For clarity, only the first 5' nucleotide in the tag 
(relative to the transcript direction) is considered. Areas in which many tags 
map to the same region may indicate a significant transcription start site.
The number of tags should be proportional to the expression rate in the 
originating tissues.</P>

<H2>Display Conventions and Configuration</H2>
<P>
The position of the first 5' nucleotide in the tag is represented by a solid
block. The height of the block indicates the number of 5' CAGE tag starts 
that map at that location.</P>
<P>
This composite annotation track contains two subtracks that 
may be configured in a variety of ways to highlight different aspects of the 
displayed data. The graphical configuration options are shown at the top of 
the track description page, followed by a list of subtracks.  
For more information about the graphical configuration options, click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph
configuration help</A> link. To display only selected subtracks, uncheck the 
boxes next to the tracks you wish to hide. </P>

<H2>Methods</H2>

<P>
To create the tag, a linker was
attached to the 5' end of full-length cDNAs which had been selected by cap
trapping. The first 20 bp of the cDNA were cleaved using the class II
restriction enzyme, MmeI, followed by PCR amplification. Concatamers
of the resulting 32 bp tags were then formed for more efficient sequencing.
A total of 7,151,511 mapped CAGE tags from 145 cDNA libraries, corresponding to 
22 distinct tissues were produced. 
The tags were mapped to the mm5 assembly and lifted to other
assemblies using UCSC's liftOver tool.
For more information on
CAGE, see Shiraki <em>et al.</em> (2003) and Carninci <EM>et al</EM> (2005)
below.  The mapping methodology employed in this annotation will be 
described in upcoming publications.</P>

<H2>Credits</H2>
<P>
These data were contributed by the Functional Annotation of Mouse
(<A HREF="https://fantom.gsc.riken.jp/" TARGET=_blank>FANTOM</A>)
Consortium, RIKEN Genome Science Laboratory and
<A HREF="http://genome.gsc.riken.go.jp/home.html" TARGET=_blank>
RIKEN Genome Exploration Research Group</A>
(Genome Network Project Core Group).  </P>

<P>
<B>FANTOM Consortium:</B> P. Carninci, T. Kasukawa, S. Katayama, Gough, 
M. Frith, N. Maeda, R. Oyama, T. Ravasi, B. Lenhard, C. Wells, R. 
Kodzius, K. Shimokawa, V. B. Bajic, S. E. Brenner, S. Batalov, A. R. R. 
Forrest, M. Zavolan, M. J. Davis, L. G. Wilming, V. Aidinis, J. Allen, 
A. Ambesi-Impiombato, R. Apweiler, R. N. Aturaliya, T. L. Bailey, M. 
Bansal, K. W. Beisel, T. Bersano, H. Bono,  A. M. Chalk, K. P. Chiu, V. 
Choudhary, A. Christoffels, D. R. Clutterbuck, M. L. Crowe, E. Dalla, 
B. P. Dalrymple, B. de Bono, G. Della Gatta, D. di Bernardo, T. Down, 
P. Engstrom, M. Fagiolini, G. Faulkner, C. F. Fletcher, T. Fukushima, 
M. Furuno, S. Futaki, M. Gariboldi, P. Georgii-Hemming, T. R. Gingeras, 
T. Gojobori, R. E. Green, S. Gustincich, M. Harbers, V. Harokopos, Y. 
Hayashi, S. Henning, T. K. Hensch, N. Hirokawa, D. Hill, L. Huminiecki, 
M. Iacono, K. Ikeo, A. Iwama, T. Ishikawa, M. Jakt, A. Kanapin, M. 
Katoh, Y. Kawasawa, J. Kelso, H. Kitamura,  H. Kitano, G. Kollias, S. 
P. T. Krishnan, A.F. Kruger, K. Kummerfeld, I. V. Kurochkin, 
L. F. Lareau, L. Lipovich, J. Liu, S. Liuni, S. McWilliam, M. Madan 
Babu, M. Madera, L. Marchionni, H. Matsuda, S. Matsuzawa, H. Miki, F. 
Mignone, S. Miyake, K. Morris, S. Mottagui-Tabar, N. Mulder, N. Nakano, 
H. Nakauchi, P. Ng, R. Nilsson, S. Nishiguchi, S. Nishikawa, F. Nori, 
O. Ohara, Y. Okazaki, V. Orlando, K. C. Pang, W. J. Pavan, G. Pavesi, 
G. Pesole, N. Petrovsky, S. Piazza, W. Qu, J. Reed, J. F. Reid, B. Z. 
Ring, M. Ringwald, B. Rost, Y. Ruan, S. Salzberg, A. Sandelin, C. 
Schneider, C. Schoenbach, K. Sekiguchi, C. A. M. Semple, S. Seno, 
L. Sessa, Y. Sheng, Y. Shibata, H. Shimada, K. Shimada, B. Sinclair, S. 
Sperling, E. Stupka, K. Sugiura, R. Sultana, Y. Takenaka, K. Taki, K. 
Tammoja, S. L. Tan, S. Tang, M. S. Taylor, J. Tegner, S. A. Teichmann, 
H. R. Ueda, E. van Nimwegene, R. Verardo, C. L. Wei, K. Yagi, H. 
Yamanishi, E. Zabarovsky, S. Zhu, A. Zimmer, W. Hide, C. Bult, S. M. 
Grimmond, R. D. Teasdale, E. T. Liu, V. Brusic, J. Quackenbush, C. 
Wahlestedt, J. Mattick, D. Hume. </P>
<P>
<B>RIKEN Genome Exploration Research Group:</B> C. Kai, D. Sasaki, Y. 
Tomaru, S. Fukuda, M. Kanamori-Katayama, M. Suzuki, J. Aoki, T. 
Arakawa, J. Iida, K. Imamura, M. Itoh, T. Kato,  H. Kawaji, N. 
Kawagashira, T. Kawashima, M. Kojima, S. Kondo, H. Konno, K. Nakano, N. 
Ninomiya, T. Nishio, M. Okada, C. Plessy, K. Shibata, T. Shiraki, S. 
Suzuki, M. Tagami, K Waki, A. Watahiki, Y. Okamura-Oho, H. Suzuki, J. 
Kawai. </P>
<P>
<B>General Organizer</B>: Y. Hayashizaki

<H2>References</H2>
<p>
Carninci P, Kasukawa T, Katayama S, Gough J, Frith MC, Maeda N, Oyama R, Ravasi T, Lenhard B, Wells
C <em>et al</em>.
<a href="https://science.sciencemag.org/content/309/5740/1559" target="_blank">
The transcriptional landscape of the mammalian genome</a>.
<em>Science</em>. 2005 Sep 2;309(5740):1559-63.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16141072" target="_blank">16141072</a>
</p>

<p>
Shiraki T, Kondo S, Katayama S, Waki K, Kasukawa T, Kawaji H, Kodzius R, Watahiki A, Nakamura M,
Arakawa T <em>et al</em>.
<a href="https://www.pnas.org/content/100/26/15776" target="_blank">
Cap analysis gene expression for high-throughput analysis of transcriptional starting point and
identification of promoter usage</a>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Dec 23;100(26):15776-81.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14663149" target="_blank">14663149</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC307644/" target="_blank">PMC307644</a>
</p>
