<H2>Description</H2>
This track depicts gaps in the draft assembly (Jul. 2011, HetGla_1.0 
(NCBI project <A HREF="https://www.ncbi.nlm.nih.gov/bioproject/68323"
TARGET=_blank>NCBI project 68323</A>, <A HREF="ftp://ftp.ncbi.nlm.nih.gov/genbank/genomes/Eukaryotes/vertebrates_mammals/Heterocephalus_glaber/HetGla_1.0/"
TARGET=_blank>GCA_00000230455.1</A>)).

Many of these gaps &mdash; with the
exception of intractable heterochromatic gaps &mdash; may be closed during the
finishing process.

<P>
Gaps are represented as black boxes in this track.
If the relative order and orientation of the contigs on either side
of the gap is supported by read pair data,
it is a <em>bridged</em> gap and a white line is drawn
through the black box representing the gap.
</P>
<P>This assembly contains the following principal types of gaps:
<UL>
<LI><B>Fragment</B> - gaps between the Whole Genome Shotgun contigs of a
supercontig. These are represented by varying numbers of <em>N</em>s in the
assembly. In this context, a contig is a set of overlapping sequence reads and
a supercontig is a set of contigs ordered and oriented during the
Whole Genome Shotgun process using paired-end reads. Fragment gap sizes
are usually taken from read pair data.
</LI>

</UL></P>




