<H2>Methods</H2>
<P>
Blood samples were collected from 78 patients at Kenema Government Hospital
in Sierra Leone.  For details of RNA preservation, PCR, human RNA depletion,
library construction and sequencing, see Supplemental Materials and Methods
of Gire <em>et al.</em>
</P>
<P>
Gire <em>et al.</em> analyzed the 78 Sierra Leone patient sequences together with
3 sequences from the 2014 outbreak in Guinea (Baize <em>et al.</em>;
suspected sequencing errors were masked, see Supplemental Materials and Methods of
Gire <em>et al.</em>), for a total of 81 sequences from 2014.
In addition, some analyses included 20 sequences from past outbreaks of Zaire Ebola
virus, 1976-2008, for a total of 101 sequences.  Sequence variants were extracted
directly from multiple sequence alignments of the group of 101 sequences (1976-2014).
A custom release of
<A HREF="http://snpeff.sourceforge.net/" TARGET="_BLANK">SnpEff</A>
(v4.0, build 2014-07-01, to support ribosomal slippage in transcription of GP gene)
was used to predict functional effect of variants on genes (noncoding, synonymous
or missense).
</P>
