<H2>Description</H2>
<P>
This track shows multiple alignments of 30 vertebrate
species and measurements of evolutionary conservation using
two methods (<em>phastCons</em> and <em>phyloP</em>) from the
<A HREF="http://compgen.cshl.edu/phast/" target=_BLANK>
PHAST package</A>, for
all species (vertebrate) and two subsets (Euarchontoglires and placental mammal).
The multiple alignments were generated using multiz and
other tools in the UCSC/<A HREF="http://www.bx.psu.edu/miller_lab/" 
TARGET=_blank>Penn State Bioinformatics</A>
comparative genomics alignment pipeline.
Conserved elements identified by phastCons are also displayed in
this track.
</P>
<P>
The species are divided into three different groups.
The Euarchontoglires subset (10 species plus mouse),
the placental mammal subset (19 species plus mouse),
and all 30 vertebrate species together.
These three measurements produce the same results in 
regions where only Euarchontoglires appear in the alignment. 
For other regions, the non-Euarchontoglires species can either 
boost the scores (if conserved) or decrease them (if non-conserved).
The placental mammal conservation helps to identify sequences that are under 
different evolutionary pressures in mammals and non-mammal vertebrates.
</P>
<P>
<UL>
<LI> Euarchontoglires subset: mouse(mm9), rat(rn4), Guinea Pig(cavPor2),
Rabbit(oryCun1), Human(hg18), Chimp(panTro2), Orangutan(ponAbe2),
Rhesus(rheMac2), Marmoset(calJac1), Bushbaby(otoGar1), Tree Shrew(tupBel1)
<LI> Placental mammal subset: the Euarchontoglires above plus:
Shrew(sorAra1), Hedgehog(eriEur1), Dog(canFam2), Cat(felCat3),
Horse(equCab1), Cow(bosTau3), Armadillo(dasNov1), Elephant(loxAfr1),
Tenrec(echTel1)
<LI> Vertebrates: the placentals and Euarchontoglires above plus:
Opossum(monDom4), Platypus(ornAna1), Chicken(galGal3), Lizard(anoCar1),
Frog(xenTro2), Tetraodon(tetNig1), Fugu(fr2), Stickleback(gasAcu1),
Medaka(oryLat1), Zebrafish(danRer5)
</UL>
</P>
<P>
PhastCons (which has been used in previous Conservation tracks) is a hidden
Markov model-based method that estimates the probability that each
nucleotide belongs to a conserved element, based on the multiple alignment.
It considers not just each individual alignment column, but also its
flanking columns.  By contrast, phyloP separately measures conservation at
individual columns, ignoring the effects of their neighbors.  As a
consequence, the phyloP plots have a less smooth appearance than the
phastCons plots, with more "texture" at individual sites.  The two methods
have different strengths and weaknesses.  PhastCons is sensitive to "runs"
of conserved sites, and is therefore effective for picking out conserved
elements.  PhyloP, on the other hand, is more appropriate for evaluating
signatures of selection at particular nucleotides or classes of nucleotides
(e.g., third codon positions, or first positions of miRNA target sites).
</P>
<P>
Another important difference is that phyloP can measure acceleration
(faster evolution than expected under neutral drift) as well as
conservation (slower than expected evolution).  In the phyloP plots, sites
predicted to be conserved are assigned positive scores (and shown in blue),
while sites predicted to be fast-evolving are assigned negative scores (and
shown in red).  The absolute values of the scores represent -log p-values
under a null hypothesis of neutral evolution.  The phastCons scores, by
contrast, represent probabilities of negative selection and range between 0
and 1.
</P>
<P>
Both phastCons and phyloP treat alignment gaps and unaligned nucleotides as
missing data, and both were run with the same parameters for each
species set (vertebrates, placental mammals, and Euarchontoglires).
Thus, in regions in which only Euarchontoglires appear in the alignment, all three
sets of scores will be the same, but in regions in which additional species
are available, the mammalian and/or vertebrate scores may differ from the
Euarchontoglires scores.  The alternative
plots help to identify sequences that are under different evolutionary
pressures in, say, Euarchontoglires and non-Euarchontoglires, or mammals and non-mammals.
</P>

<P>
The multiple alignments were generated using multiz and 
other tools in the UCSC/<A HREF="http://www.bx.psu.edu/miller_lab/" 
TARGET=_blank>Penn State Bioinformatics</A>
comparative genomics alignment pipeline.
The conservation measurements were created using the phastCons package from
<A HREF="http://siepellab.labsites.cshl.edu/" TARGET=_blank>
Adam Siepel</A> at Cold Spring Harbor Laboratory. </P>
<P>
Details of the alignment parameters are noted in the genomewiki
<A HREF="http://genomewiki.ucsc.edu/index.php/Mm9_multiple_alignment"
TARGET=_blank>Mm9 multiple alignment</A> page.
</P>
<P>
The species aligned for this track include the reptile, amphibian, 
bird, and fish clades, as well as marsupial, monotreme (platypus), 
and placental mammals. Compared to the previous 17-vertebrate alignment,
this track includes 13 new species and 4 species with updated
sequence assemblies (<B>Table 1</B>). The new species consist of seven 
high-coverage (5-8.5X) assemblies (orangutan, marmoset, horse, platypus, 
lizard, and two teleost fish: stickleback and medaka)
and six low-coverage (2X) genome assemblies from mammalian species selected for 
sampling by NHGRI (bushbaby, tree shrew, guinea pig, 
hedgehog, common shrew, and cat).
The cow, chicken, fugu, and zebrafish assemblies in this 
track have been updated from those used in the previous 17-species alignment.
</P>
<P>
UCSC has repeatmasked and aligned the low-coverage genome assemblies, and
provides the sequence for download; however, we do not construct
genome browsers for them. Missing sequence in the low-coverage assemblies is
highlighted in the track display by regions of yellow when zoomed out
and Ns displayed at base level (see <EM>Gap Annotation</EM>, below). </P>
<P>
<BLOCKQUOTE><TABLE BORDER=1 CELLPADDING=4 BORDERCOLOR="#aaaaaa">
<TR ALIGN=left><TH>Organism</TH><TH>Species</TH><TH>Release date</TH><TH>UCSC version</TH></TR>
<TR ALIGN=left><TD>Mouse</TD><TD>Mus musculus</TD><TD>
    Jul 2007</TD><TD> <A HREF="../cgi-bin/hgGateway?db=mm9"
    TARGET=_blank>mm9</A></TD></TR>
<TR ALIGN=left><TD>Armadillo</TD><TD>Dasypus novemcinctus</TD><TD>May 2005</TD>
    <TD><A HREF="ftp://hgdownload.soe.ucsc.edu/gbdb/dasNov1/"
    TARGET=_blank>dasNov1</A>*</TD></TR>
<TR ALIGN=left><TD>Bushbaby</TD><TD>Otolemur garnetti</TD><TD>Dec 2006</TD>
    <TD><A HREF="ftp://hgdownload.soe.ucsc.edu/gbdb/otoGar1/"
    TARGET=_blank>otoGar1</A>*</TD></TR>
<TR ALIGN=left><TD>Cat</TD><TD>Felis catus</TD><TD>
    Mar 2006</TD><TD> <A HREF="../cgi-bin/hgGateway?db=felCat3"
    TARGET=_blank>felCat3</A></TD></TR>
<TR ALIGN=left><TD>Chicken</TD><TD>Gallus gallus</TD><TD>
    May 2006</TD><TD> <A HREF="../cgi-bin/hgGateway?db=galGal3"
    TARGET=_blank>galGal3</A></TD></TR>
<TR ALIGN=left><TD>Chimpanzee</TD><TD>Pan troglodytes</TD><TD>
    Mar 2006</TD><TD> <A HREF="../cgi-bin/hgGateway?db=panTro2"
    TARGET=_blank>panTro2</A></TD></TR>
<TR ALIGN=left><TD>Cow</TD><TD>Bos taurus</TD><TD>
    Aug 2006</TD><TD> <A HREF="../cgi-bin/hgGateway?db=bosTau3"
    TARGET=_blank>bosTau3</A></TD></TR>
<TR ALIGN=left><TD>Dog</TD><TD>Canis familiaris</TD><TD>
    May 2005</TD><TD> <A HREF="../cgi-bin/hgGateway?db=canFam2"
    TARGET=_blank>canFam2</A></TD></TR>
<TR ALIGN=left><TD>Elephant</TD><TD>Loxodonta africana</TD><TD>May 2005</TD>
    <TD><A HREF="ftp://hgdownload.soe.ucsc.edu/gbdb/loxAfr1/"
    TARGET=_blank>loxAfr1</A>*</TD></TR>
<TR ALIGN=left><TD>Frog</TD><TD>Xenopus tropicalis</TD><TD>
    Aug 2005</TD><TD> <A HREF="../cgi-bin/hgGateway?db=xenTro2"
    TARGET=_blank>xenTro2</A></TD></TR>
<TR ALIGN=left><TD>Fugu</TD><TD>Takifugu rubripes</TD><TD>
    Oct 2004</TD><TD> <A HREF="../cgi-bin/hgGateway?db=fr2"
    TARGET=_blank>fr2</A></TD></TR>
<TR ALIGN=left><TD>Guinea pig</TD><TD>Cavia porcellus</TD><TD>Oct 2005</TD>
    <TD><A HREF="ftp://hgdownload.soe.ucsc.edu/gbdb/cavPor2/"
    TARGET=_blank>cavPor2</A>*</TD></TR>
<TR ALIGN=left><TD>Hedgehog</TD><TD>Erinaceus europaeus</TD><TD>June 2006</TD>
    <TD><A HREF="ftp://hgdownload.soe.ucsc.edu/gbdb/eriEur1/"
    TARGET=_blank>eriEur1</A>*</TD></TR>
<TR ALIGN=left><TD>Horse</TD><TD>Equus caballus</TD><TD>
    Jan 2007</TD><TD> <A HREF="../cgi-bin/hgGateway?db=equCab1"
    TARGET=_blank>equCab1</A></TD></TR>
<TR ALIGN=left><TD>Human</TD><TD>Homo sapiens</TD><TD>
    Mar 2006</TD><TD> <A HREF="../cgi-bin/hgGateway?db=hg18"
    TARGET=_blank>hg18</A></TD></TR>
<TR ALIGN=left><TD>Lizard</TD><TD>Anolis carolinensis</TD><TD>
    Feb 2007</TD><TD> <A HREF="../cgi-bin/hgGateway?db=anoCar1"
    TARGET=_blank>anoCar1</A></TD></TR>
<TR ALIGN=left><TD>Marmoset</TD><TD>Callithrix jacchus</TD><TD>June 2007</TD>
    <TD><A HREF="../cgi-bin/hgGateway?db=calJac1"
    TARGET=_blank>calJac1</A></TD></TR>
<TR ALIGN=left><TD>Medaka</TD><TD>Oryzias latipes</TD><TD>
    Apr 2006</TD><TD><A HREF="ftp://hgdownload.soe.ucsc.edu/gbdb/oryLat1/"
    TARGET=_blank>oryLat1</A>*</TD></TR>
<TR ALIGN=left><TD>Opossum</TD><TD>Monodelphis domestica</TD><TD>
    Jan 2006</TD><TD> <A HREF="../cgi-bin/hgGateway?db=monDom4"
    TARGET=_blank>monDom4</A></TD></TR>
<TR ALIGN=left><TD>Orangutan</TD><TD>Pongo pygmaeus abelii</TD><TD>
    July 2007</TD><TD><A HREF="../cgi-bin/hgGateway?db=ponAbe2"
    TARGET=_blank>ponAbe2</A></TD></TR>
<TR ALIGN=left><TD>Platypus</TD><TD>Ornithorhychus anatinus</TD><TD>
    Mar 2007</TD><TD> <A HREF="../cgi-bin/hgGateway?db=ornAna1"
    TARGET=_blank>ornAna1</A></TD></TR>
<TR ALIGN=left><TD>Rabbit</TD><TD>Oryctolagus cuniculus</TD><TD>May 2005</TD>
    <TD><A HREF="ftp://hgdownload.soe.ucsc.edu/gbdb/oryCun1/"
    TARGET=_blank>oryCun1</A>*</TD></TR>
<TR ALIGN=left><TD>Rat</TD><TD>Rattus norvegicus</TD><TD>
    Nov 2004</TD><TD> <A HREF="../cgi-bin/hgGateway?db=rn4"
    TARGET=_blank>rn4</A></TD></TR>
<TR ALIGN=left><TD>Rhesus</TD><TD>Macaca mulatta</TD><TD>
    Jan 2006</TD><TD> <A HREF="../cgi-bin/hgGateway?db=rheMac2"
    TARGET=_blank>rheMac2</A></TD></TR>
<TR ALIGN=left><TD>Shrew</TD><TD>Sorex araneus</TD><TD>June 2006</TD>
    <TD><A HREF="ftp://hgdownload.soe.ucsc.edu/gbdb/sorAra1/"
    TARGET=_blank>sorAra1</A>*</TD></TR>
<TR ALIGN=left><TD>Stickleback</TD><TD>Gasterosteus aculeatus</TD><TD>
    Feb 2006</TD><TD> <A HREF="../cgi-bin/hgGateway?db=gasAcu1"
    TARGET=_blank>gasAcu1</A></TD></TR>
<TR ALIGN=left><TD>Tenrec</TD><TD>Echinops telfairi</TD><TD>July 2005</TD>
    <TD><A HREF="ftp://hgdownload.soe.ucsc.edu/gbdb/echTel1/"
    TARGET=_blank>echTel1</A>*</TD></TR>
<TR ALIGN=left><TD>Tetraodon</TD><TD>Tetraodon nigroviridis</TD><TD>
    Feb 2004</TD><TD> <A HREF="../cgi-bin/hgGateway?db=tetNig1"
    TARGET=_blank>tetNig1</A></TD></TR>
<TR ALIGN=left><TD>Tree shrew</TD><TD>Tupaia belangeri</TD><TD>Dec 2006</TD>
    <TD><A HREF="ftp://hgdownload.soe.ucsc.edu/gbdb/tupBel1/"
    TARGET=_blank>tupBel1</A>*</TD></TR>
<TR ALIGN=left><TD>Zebrafish</TD><TD>Danio rerio</TD><TD>
    July 2007</TD><TD> <A HREF="../cgi-bin/hgGateway?db=danRer5"
    TARGET=_blank>danRer5</A></TD></TR>
</TABLE><BR>
<B>Table 1.</B> <EM>Genome assemblies included in the 30-way Conservation 
track.</EM>
<BR>* Data download only, browser not available.
</BLOCKQUOTE></P>

Downloads for data in this track are available:
<UL>
<LI>
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/mm9/multiz30way/">Multiz alignments</A> (MAF format), and phylogenetic trees
<LI>
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/mm9/phyloP30way/">PhyloP conservation</A> (WIG format)
<LI>
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/mm9/phastCons30way/">PhastCons conservation</A> (WIG format)
</UL>

<H2>Display Conventions and Configuration</H2>
<P>
In full and pack display modes, conservation scores are displayed as
<EM>wiggle tracks</EM> (histograms) in which the height reflects the 
size of the score. 
The conservation wiggles can be configured in a variety of ways to 
highlight different aspects of the displayed information. 
Click the <A HREF="../goldenPath/help/hgWiggleTrackHelp.html" 
TARGET=_blank>Graph configuration help</A> link for an explanation 
of the configuration options.</P>
<P>
Pairwise alignments of each species to the $organism genome are 
displayed below the conservation histogram as a grayscale density plot (in 
pack mode) or as a wiggle (in full mode) that indicates alignment quality.
In dense display mode, conservation is shown in grayscale using
darker values to indicate higher levels of overall conservation 
as scored by phastCons. </P>
<P>
Checkboxes on the track configuration page allow selection of the
species to include in the pairwise display.  
Configuration buttons are available to select all of the species (<EM>Set 
all</EM>), deselect all of the species (<EM>Clear all</EM>), or 
use the default settings (<EM>Set defaults</EM>).
By default, the following 8 species are included in the pairwise display:
rat, human, orangutan, dog, horse, opossum, chicken, and stickleback.
Note that excluding species from the pairwise display does not alter the
the conservation score display.</P>
<P>
To view detailed information about the alignments at a specific
position, zoom the display in to 30,000 or fewer bases, then click on
the alignment.</P>

<H3>Gap Annotation</H3>
<P>
The <EM>Display chains between alignments</EM> configuration option 
enables display of gaps between alignment blocks in the pairwise alignments in 
a manner similar to the Chain track display.  The following
conventions are used:
<UL>
<LI><B>Single line:</B> No bases in the aligned species. Possibly due to a
lineage-specific insertion between the aligned blocks in the $organism genome
or a lineage-specific deletion between the aligned blocks in the aligning
species.
<LI><B>Double line:</B> Aligning species has one or more unalignable bases in
the gap region. Possibly due to excessive evolutionary distance between 
species or independent indels in the region between the aligned blocks in both
species. 
<LI><B>Pale yellow coloring:</B> Aligning species has Ns in the gap region.
Reflects uncertainty in the relationship between the DNA of both species, due
to lack of sequence in relevant portions of the aligning species. 
</UL></P>

<H3>Genomic Breaks</H3>
<P>
Discontinuities in the genomic context (chromosome, scaffold or region) of the
aligned DNA in the aligning species are shown as follows: 
<UL>
<LI>
<B>Vertical blue bar:</B> Represents a discontinuity that persists indefinitely
on either side, <em>e.g.</em> a large region of DNA on either side of the bar
comes from a different chromosome in the aligned species due to a large scale
rearrangement. 
<LI>
<B>Green square brackets:</B> Enclose shorter alignments consisting of DNA from
one genomic context in the aligned species nested inside a larger chain of
alignments from a different genomic context. The alignment within the
brackets may represent a short misalignment, a lineage-specific insertion of a
transposon in the $organism genome that aligns to a paralogous copy somewhere
else in the aligned species, or other similar occurrence.
</UL></P>

<H3>Base Level</H3>
<P>
When zoomed-in to the base-level display, the track shows the base 
composition of each alignment. 
The numbers and symbols on the Gaps
line indicate the lengths of gaps in the $organism sequence at those 
alignment positions relative to the longest non-$organism sequence. 
If there is sufficient space in the display, the size of the gap is shown. 
If the space is insufficient and the gap size is a multiple of 3, a 
&quot;*&quot; is displayed; other gap sizes are indicated by &quot;+&quot;.</P>
<P>
Codon translation is available in base-level display mode if the
displayed region is identified as a coding segment. To display this annotation,
select the species for translation from the pull-down menu in the Codon
Translation configuration section at the top of the page. Then, select one of
the following modes:
<UL>
<LI> 
<B>No codon translation:</B> The gene annotation is not used; the bases are
displayed without translation. 
<LI>
<B>Use default species reading frames for translation:</B> The annotations from the genome
displayed 
in the <EM>Default species to establish reading frame</EM> pull-down menu are used to
translate all the aligned species present in the alignment. 
<LI>
<B>Use reading frames for species if available, otherwise no translation:</B> Codon
translation is performed only for those species where the region is
annotated as protein coding.
<LI><B>Use reading frames for species if available, otherwise use default species:</B>
Codon translation is done on those species that are annotated as being protein
coding over the aligned region using species-specific annotation; the remaining
species are translated using the default species annotation. 
</UL></P>
<P>
Codon translation uses the following gene tracks as the basis for
translation, depending on the species chosen (<B>Table 2</B>).  
Species listed in the row labeled &quot;None&quot; do not have 
species-specific reading frames for gene translation.

<BLOCKQUOTE><TABLE BORDER=1 CELLPADDING=4 BORDERCOLOR="#aaaaaa">
<TR ALIGN=left><TD><B>Gene Track</B></TD><TD><B>Species</B></TD></TR>
<TR ALIGN=left><TD>Known Genes</TD><TD>human, mouse</TD></TR>
<TR ALIGN=left><TD>Ensembl Genes</TD><TD>rat, rhesus, chimp, dog, opossum, platypus,
zebrafish, fugu, stickleback, medaka</TD></TR>
<TR ALIGN=left><TD>RefSeq Genes</TD><TD>cow, frog</TD></TR>
<TR ALIGN=left><TD>mRNAs</TD><TD>orangutan, elephant, rabbit, cat, horse,
chicken, lizard, armadillo, tetraodon</TD></TR>
<TR ALIGN=left><TD>None</TD><TD>marmoset, bushbaby, tree shrew, guinea pig,
shrew, hedgehog, tenrec</TD></TR>
</TABLE>
<B>Table 2.</B> <EM>Gene tracks used for codon translation.</EM>
</BLOCKQUOTE></P>

<H2>Methods</H2>
<P> 
Pairwise alignments with the mouse genome were generated for 
each species using lastz from repeat-masked genomic sequence. 
Lineage-specific repeats were removed prior to alignment, then reinserted.
Pairwise alignments were then linked into chains using a dynamic programming
algorithm that finds maximally scoring chains of gapless subsections
of the alignments organized in a kd-tree.
The scoring matrix and parameters for pairwise alignment and chaining
were tuned for each species based on phylogenetic distance from the reference.
High-scoring chains were then placed along the genome, with
gaps filled by lower-scoring chains, to produce an alignment net.
For more information about the chaining and netting process and 
parameters for each species, see the description pages for the Chain and Net 
tracks.</P>
<P>
An additional filtering step was introduced in the generation of the 30-way
conservation track to reduce the number of paralogs and pseudogenes from the 
high-quality assemblies and the suspect alignments from the low-quality 
assemblies:
the pairwise alignments of high-quality mammalian 
sequences (placental and marsupial) were filtered based on synteny; 
those for 2X mammalian genomes were filtered to retain only 
alignments of best quality in both the target and query (&quot;reciprocal 
best&quot;).</P>
<P>
The resulting best-in-genome pairwise alignments
were progressively aligned using multiz/autoMZ, 
following the tree topology diagrammed above, to produce multiple alignments.
The multiple alignments were post-processed to
add annotations indicating alignment gaps, genomic breaks,
and base quality of the component sequences.
The annotated multiple alignments, in MAF format, are available for
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/mm9/multiz30way"
TARGET=_blank>bulk download</A>.
An alignment summary table containing an entry for each
alignment block in each species was generated to improve
track display performance at large scales.
Framing tables were constructed to enable
visualization of codons in the multiple alignment display.</P>

<h3> Phylogenetic Tree Model</h3>
<P>
Both <em>phastCons</em> and <em>phyloP</em> are phylogenetic methods that
rely on a tree model containing the tree topology,
branch lengths representing evolutionary distance at neutrally
evolving sites, the background distribution of nucleotides, and a substitution
rate matrix.  The 
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/mm9/phastCons30way/vertebrate.mod"
TARGET=_blank>vertebrate tree model</A> for this track was
generated using the <em>phyloFit</em> program from the PHAST package  
(REV model, EM algorithm, medium precision) using multiple alignments of 
4-fold degenerate sites extracted from the 30-way alignment
(msa_view).  The 4d sites were derived from the 
<A HREF="https://genome.crg.es/gencode/" TARGET=_blank>Oct 2005 Gencode 
Reference Gene set</A>,
which was filtered to select single-coverage long transcripts. The
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/mm9/phastCons30way/placental.mod" 
TARGET=_blank>placental mammal tree model</A> and the
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/mm9/phastCons30way/euarchontoglires.mod" 
TARGET=_blank>Euarchontoglires tree model</A> were extracted from the vertebrate model.
The phastCons parameters were
tuned to produce 5% conserved elements in the genome for the vertebrate
conservation measurement.  This parameter set (expected-length=45, 
target-coverage=.3, rho=.31) was then used to generate the placental
mammal and Euarchontoglires conservation scoring.</P>
<P>
The phastCons program computes conservation scores based on a phylo-HMM, a
type of probabilistic model that describes both the process of DNA
substitution at each site in a genome and the way this process changes from
one site to the next (Felsenstein and Churchill 1996, Yang 1995, Siepel and
Haussler 2005).  PhastCons uses a two-state phylo-HMM, with a state for
conserved regions and a state for non-conserved regions.  The value plotted
at each site is the posterior probability that the corresponding alignment
column was "generated" by the conserved state of the phylo-HMM.  These
scores reflect the phylogeny (including branch lengths) of the species in
question, a continuous-time Markov model of the nucleotide substitution
process, and a tendency for conservation levels to be autocorrelated along
the genome (i.e., to be similar at adjacent sites).  The general reversible
(REV) substitution model was used.  Unlike many conservation-scoring programs, 
note that phastCons does not rely on a sliding window
of fixed size; therefore, short highly-conserved regions and long moderately
conserved regions can both obtain high scores.  More information about
phastCons can be found in Siepel <EM>et al</EM>. 2005.</P> 
<P> 
PhastCons currently treats alignment gaps as missing data, which
sometimes has the effect of producing undesirably high conservation scores
in gappy regions of the alignment.  We are looking at several possible ways
of improving the handling of alignment gaps.</P>

<h3> PhyloP Conservation </h3>
<P>
The phyloP program supports several different methods for computing
p-values of conservation or acceleration, for individual nucleotides or
larger elements (<a href="http://compgen.cshl.edu/phast/" target="_blank">
http://compgen.cshl.edu/phast/</a>).  Here it was used
to produce separate scores at each base (--wig-scores option), considering
all branches of the phylogeny rather than a particular subtree or lineage
(i.e., the --subtree option was not used).  The scores were computed by
performing a likelihood ratio test at each alignment column (--method LRT),
and scores for both conservation and acceleration were produced (--mode
CONACC).
</P>
<h3> Conserved Elements </h3>
<P>
The conserved elements were predicted by running phastCons with the
--viterbi option.  The predicted elements are segments of the alignment
that are likely to have been "generated" by the conserved state of the
phylo-HMM. Each element is assigned a log-odds score equal to its log
probability under the conserved model minus its log probability under the
non-conserved model. The "score" field associated with this track contains
transformed log-odds scores, taking values between 0 and 1000. (The scores
are transformed using a monotonic function of the form a * log(x) + b.) The
raw log odds scores are retained in the "name" field and can be seen on the
details page or in the browser when the track's display mode is set to
"pack" or "full".
</P>

<H2>Credits</H2>
<P> This track was created using the following programs:
<UL>
<LI> Alignment tools: lastz (formerly blastz) and multiz by Minmei Hou, Scott Schwartz and Webb 
Miller of the <A HREF="http://www.bx.psu.edu/miller_lab/" 
TARGET=_blank>Penn State Bioinformatics Group</A>
<LI> Chaining and Netting:  axtChain, chainNet by Jim Kent at UCSC
<LI> Conservation scoring: phastCons, phyloP, phyloFit, tree_doctor, msa_view
and other programs in PHAST by 
<A HREF="http://siepellab.labsites.cshl.edu/"
TARGET=_blank>Adam Siepel</A> at Cold Spring Harbor Laboratory (original development
done at the Haussler lab at UCSC).
<LI> MAF Annotation tools: mafAddIRows by Brian Raney, UCSC; mafAddQRows
by Richard Burhans, Penn State; genePredToMafFrames by Mark Diekhans, UCSC
<LI> Tree image generator: phyloPng by Galt Barber, UCSC
<LI> Conservation track display: Kate Rosenbloom, Hiram Clawson (wiggle 
display), and Brian Raney (gap annotation and codon framing) at UCSC
</UL>
</P>
<P>The phylogenetic tree is based on Murphy <EM>et al</EM>. (2001) and general 
consensus in the vertebrate phylogeny community as of March 2007.
</P>

<H2>References</H2>

<H3>Phylo-HMMs, phastCons, and phyloP:</H3>
<p>
Felsenstein J, Churchill GA.
<a href="https://academic.oup.com/mbe/article/13/1/93/1055515"
target="_blank">A Hidden Markov Model approach to
variation among sites in rate of evolution</a>.
<em>Mol Biol Evol</em>. 1996 Jan;13(1):93-104.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8583911" target="_blank">8583911</a>
</p>

<p>
Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K,
Clawson H, Spieth J, Hillier LW, Richards S, <em>et al.</em>
<a href="https://genome.cshlp.org/content/15/8/1034"
target="_blank">Evolutionarily conserved elements in vertebrate, insect, worm,
and yeast genomes</a>.
<em>Genome Res</em>. 2005 Aug;15(8):1034-50.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16024819" target="_blank">16024819</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1182216/" target="_blank">PMC1182216</a>
</p>

<p>
Siepel A, Haussler D.
<a href="http://compgen.cshl.edu/~acs/phylohmm.pdf"
target="_blank">Phylogenetic Hidden Markov Models</a>.
In: Nielsen R, editor. Statistical Methods in Molecular Evolution.
New York: Springer; 2005. pp. 325-351.
</p>

<p>
Yang Z.
<a href="https://www.genetics.org/content/139/2/993"
target="_blank">A space-time process model for the evolution of DNA
sequences</a>.
<em>Genetics</em>. 1995 Feb;139(2):993-1005.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/7713447" target="_blank">7713447</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1206396/" target="_blank">PMC1206396</a>
</p>

<H3>Chain/Net:</H3>
<p>
Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D.
<a href="https://www.pnas.org/content/100/20/11484"
target="_blank">Evolution's cauldron:
duplication, deletion, and rearrangement in the mouse and human genomes</a>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Sep 30;100(20):11484-9.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14500911" target="_blank">14500911</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC208784/" target="_blank">PMC208784</a>
</p>

<H3>Multiz:</H3>
<p>
Blanchette M, Kent WJ, Riemer C, Elnitski L, Smit AF, Roskin KM,
Baertsch R, Rosenbloom K, Clawson H, Green ED, <em>et al.</em>
<a href="https://genome.cshlp.org/content/14/4/708.abstract"
target="_blank">Aligning multiple genomic sequences with the threaded blockset aligner</a>.
<em>Genome Res</em>. 2004 Apr;14(4):708-15.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15060014" target="_blank">15060014</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC383317/" target="_blank">PMC383317</a>
</p>

<H3>Lastz (formerly Blastz):</H3>
<p>
Chiaromonte F, Yap VB, Miller W.
<a href="http://psb.stanford.edu/psb-online/proceedings/psb02/chiaromonte.pdf"
target="_blank">Scoring pairwise genomic sequence alignments</a>.
<em>Pac Symp Biocomput</em>. 2002:115-26.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11928468" target="_blank">11928468</a>
</p>

<p>
Harris RS.
<a href="http://www.bx.psu.edu/~rsharris/rsharris_phd_thesis_2007.pdf"
target="_blank">Improved pairwise alignment of genomic DNA</a>.
<em>Ph.D. Thesis</em>. Pennsylvania State University, USA. 2007.
</p>

<p>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC,
Haussler D, Miller W.
<a href="https://genome.cshlp.org/content/13/1/103.abstract"
target="_blank">Human-mouse alignments with BLASTZ</a>.
<em>Genome Res</em>. 2003 Jan;13(1):103-7.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12529312" target="_blank">12529312</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC430961/" target="_blank">PMC430961</a>
</p>

<H3>Phylogenetic Tree:</H3>
<p>
Murphy WJ, Eizirik E, O'Brien SJ, Madsen O, Scally M, Douady CJ, Teeling E,
Ryder OA, Stanhope MJ, de Jong WW, Springer MS.
<a href="https://science.sciencemag.org/content/294/5550/2348"
target="_blank">Resolution of the early placental mammal radiation using Bayesian phylogenetics</a>.
<em>Science</em>. 2001 Dec 14;294(5550):2348-51.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11743200" target="_blank">11743200</a>
</p>
