<H2>Overview</H2>

<div class="figure" style="float: right;  width: 439px; border: thin silver solid ; margin: 0 20px
20px 20px; padding: 20px 10px 10px 10px; text-align: center; font-size: 0.9em; background-color:
#e9e9e9;"> <img src="../images/laminB1Mm9.png" alt="Nuclear Lamina and Chromosomal Organization in
Mouse Cells"> <p>Model of chromosome organization in interphase, summarizing the main results
presented in this paper. Large, discrete chromosomal domains are dynamically associated with the
nuclear lamina (NL), in a manner that is dependent on the cell type (Fig. 7, Peric-Hupkes, et al.
2010).</p> </div>

<P>The three-dimensional organization of chromosomes within the nucleus and its dynamics during
differentiation are largely unknown. To visualize this process in molecular detail, high-resolution
maps of genome-nuclear lamina interactions during subsequent differentiation of mouse embryonic stem
cells were generated via lineage-committed neural precursor (or, neural progenitor) cells into
terminally differentiated astrocytes. In addition, genome-nuclear lamina interactions for mouse
embryonic fibroblasts were profiled.</P>

<P>This revealed that a basal chromosome architecture present in embryonic stem cells is
cumulatively altered at hundreds of sites during lineage commitment and subsequent terminal
differentiation. This remodeling involves both individual transcription units and multi-gene
regions, and affects many genes that determine cellular identity. Often, genes that move away from
the lamina are concomitantly activated; many others however remain inactive yet become unlocked for
activation in a next differentiation step. These results suggest that lamina-genome interactions are
widely involved in the control of gene expression programs during lineage commitment and terminal
differentiation.</P>

<h3> NKI mouse LaminB1 ESC track </h3> <P> The mouse LaminB1 ESC track shows a high resolution map
of the interaction sites of the entire genome with Lamin B1 (a key NL component) in mouse embryonic
stem cells. </P>

<h3> NKI mouse LaminB1 NPC track </h3> <P> The mouse LaminB1 NPC track shows a high resolution map
of the interaction sites of the entire genome with Lamin B1 (a key NL component) in mouse neural
progenitor cells. </P>

<h3> NKI mouse LaminB1 AC track </h3> <P> The mouse LaminB1 AC track shows a high resolution map of
the interaction sites of the entire genome with Lamin B1 (a key NL component) in mouse astrocytes.
</P>

<h3> NKI mouse LaminB1 MEF track </h3> <P> The mouse LaminB1 MEF track shows a high resolution map
of the interaction sites of the entire genome with Lamin B1 (a key NL component) in mouse embryonic
fibroblasts. </P>


<H2>Display Conventions and Configuration</H2>

<P>The LaminB1 wiggle tracks values range from -6.00 to 4.93. The default vertical viewing range for
the wiggle track was chosen from -1.5 to 1.5 because this is roughly +/- 1.5  standard deviations.
</P>

<P> For an example region see genomic location: <a
href="/cgi-bin/hgTracks?clade=mammal&org=Mouse&db=mm9&position=chr14:92,000,000-96,000,000&refGene=
pack&laminB1Super=show&laminB1_ES=full&laminB1_AC=full&laminB1_NP=full&laminB1_EF=full">chr14:92,000
,000-96,000,000</a> (Fig 3A, Peric-Hupkes, Meuleman <EM>et al.</EM>, 2010). </P>


<H2>Methods</H2>

<P>The DamID technique was applied to generate high-resolution maps of NL interactions for the
entire mouse genome. DamID is based on targeted adenine methylation of DNA sequences that interact
in vivo with a protein of interest. </P>




<P>DamID was performed as described (Peric-Hupkes, et al. 2010). In short, a fusion protein
consisting of Escherichia coli DNA adenine methyltransferase (Dam) fused to mouse LaminB1 was
introduced into cultured cells. Dam methylates adenines in the sequence GATC, a mark absent in most
eukaryotes. Here, the LaminB1-Dam fusion protein incorporates in the nuclear lamina, as verified by
immunofluorescence staining. Hence, the sequences near the nuclear lamina are marked with a unique
methylation tag. The adenine methylation pattern was detected with genomic tiling arrays. <a
href="https://chromatingenomics.org/damid/">Unfused Dam</a> was used as a reference. The
data shown are the log2-ratio of LaminB1-Dam fusion protein over Dam-only.</P>

<P> Sample labelling and hybridizations were performed as described (Peric-Hupkes, et al. 2010), on
a custom-designed Nimblegen HD2 array, with a median probe spacing of ~1kbp. All probes recognize
unique (non-repetitive) sequences. The raw data was log2 transformed and loess normalized, followed
by quantile normalization across the single channel data of all hybridizations. Replicate arrays
were averaged. </P>


<H2>Verification</H2>

<P> The data are based on two independent biological replicates for each cell type, performed on
separate days. Fluorescence in situ hybridization microscopy confirmed that most of the LaminB1
associated regions are preferentially located at the nuclear periphery. The array platform, the <A
HREF="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17051" TARGET="_BLANK">raw and normalized
data</A> have been deposited at the <a href="https://www.ncbi.nlm.nih.gov/geo/">NCBI Gene Expression
Omnibus (GEO)</a> under accession number <a
href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17051" TARGET="_BLANK">GSE17051</A>. </P>


<H2>Credits</H2>

<P> The data for this track were generated by Daan Peric-Hupkes, Wouter Meuleman and Bas van
Steensel at the <A HREF="https://chromatingenomics.org/" TARGET=_blank>Van Steensel Lab</A>,
Netherlands Cancer Institute. </P>


<H2>References</H2>
<p>
Peric-Hupkes D, Meuleman W, Pagie L, Bruggeman SW, Solovei I, Brugman W, Gr&#228;f S, Flicek P,
Kerkhoven RM, van Lohuizen M <em>et al</em>.
<a href="https://www.sciencedirect.com/science/article/pii/S1097276510003217" target="_blank">
Molecular maps of the reorganization of genome-nuclear lamina interactions during
differentiation</a>.
<em>Mol Cell</em>. 2010 May 28;38(4):603-13.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20513434" target="_blank">20513434</a>
</p>
