<H2>Description</H2> 
<P> 
This track shows predictions of conserved elements produced by the phastCons
program based on a whole-genome alignment of vertebrates, the
placental mammal subset of species in the alignment, and the
euarchontoglire subset of species in the alignment.  
They are based on a phylogenetic hidden Markov model
(phylo-HMM), a type of probabilistic model that describes both the process
of DNA substitution at each site in a genome and the way this process
changes from one site to the next. </P>

<P>
Details of the alignment parameters are noted in the genomewiki
<A HREF="http://genomewiki.ucsc.edu/index.php/Mm9_multiple_alignment"
TARGET=_blank>Mm9 multiple alignment</A> page.
</P>

<H2>Subsets</H2>
<P>
<UL>
<LI> Euarchontoglire subset: mouse(mm9), rat(rn4), Guinea Pig(cavPor2),
Rabbit(oryCun1), Human(hg18), Chimp(panTro2), Orangutan(ponAbe2),
Rhesus(rheMac2), Marmoset(calJac1), Bushbaby(otoGar1), Tree Shrew(tupBel1)
<LI> Placental mammal subset: the euarchontoglires above plus:
Shrew(sorAra1), Hedgehog(eriEur1), Dog(canFam2), Cat(felCat3),
Horse(equCab1), Cow(bosTau3), Armadillo(dasNov1), Elephant(loxAfr1),
Tenrec(echTel1)
<LI> Vertebrates: the placentals and euarchontoglires above plus:
Opossum(monDom4), Platypus(ornAna1), Chicken(galGal3), Lizard(anoCar1),
Frog(xenTro2), Tetraodon(tetNig1), Fugu(fr2), Stickleback(gasAcu1),
Medaka(oryLat1), Zebrafish(danRer5)
</UL>
</P>

<H2>Methods</H2>
<P>
Best-in-genome pairwise alignments were generated for
each species using blastz, followed by chaining and netting.  A multiple
alignment was then constructed from these pairwise alignments using multiz.
Predictions of conserved elements were then obtained by running phastCons
on the multiple alignments with the --most-conserved option.
For more details see the track description for the Conservation track.</P>
<P>
PhastCons constructs a two-state phylo-HMM with a state for conserved
regions and a state for non-conserved regions.  The two states share a
single phylogenetic model, except that the branch lengths of the tree
associated with the conserved state are multiplied by a constant scaling
factor <em>rho</em> (0 <= <em>rho</em> <= 1).  The free parameters of the
phylo-HMM, including the scaling factor <em>rho</em>, are estimated from
the data by maximum likelihood using an EM algorithm.  This procedure is
subject to certain constraints on the "coverage" of the genome by conserved
elements and the "smoothness" of the conservation scores.  Details can be
found in Siepel et al. (2005).</P>
<P>
The predicted conserved elements are segments of the alignment that are
likely to have been "generated" by the conserved state of the phylo-HMM.
Each element is assigned a log-odds score equal to its log probability
under the conserved model minus its log probability under the non-conserved
model.  The "score" field associated with this track contains transformed
log-odds scores, taking values between 0 and 1000.  (The scores are
transformed using a monotonic function of the form a * log(x) + b.)  The
raw log odds scores are retained in the "name" field and can be seen on the
details page or in the browser when the track's display mode is set to
"pack" or "full".</P>

<H2>Credits</H2>
<P>
This track was created at UCSC using the following programs:
<UL>
<LI>
Blastz and multiz by Minmei Hou, Scott Schwartz and Webb Miller of the 
<A HREF="http://www.bx.psu.edu/" TARGET=_blank>Penn State Bioinformatics 
Group</A>. 
<LI>
AxtBest, axtChain, chainNet, netSyntenic, and netClass
by Jim Kent at UCSC. 
<LI>PhastCons by Adam Siepel at Cornell University. 
</UL>
</P>

<H2>References</H2>

<H3>PhastCons</H3>
<P>
Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K,
Clawson H, Spieth J, Hillier LW, Richards S, <EM>et al.</EM>
<A HREF="http://www.genome.org/cgi/doi/10.1101/gr.3715005"
TARGET=_blank>Evolutionarily conserved elements in vertebrate, insect, worm,
and yeast genomes</A>.
<EM>Genome Res</EM>. 2005 Aug;15(8):1034-50.

<H3>Chain/Net</H3>
<P>
Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D.
<A HREF="http://www.pnas.org/cgi/content/abstract/1932072100v1"
TARGET=_blank>Evolution's cauldron:
duplication, deletion, and rearrangement in the mouse and human genomes</A>.
<EM>Proc Natl Acad Sci U S A</EM>. 2003 Sep 30;100(20):11484-9.

<H3>Multiz</H3>
<P>
Blanchette M, Kent WJ, Riemer C, Elnitski L, Smit AF, Roskin KM,
Baertsch R, Rosenbloom K, Clawson H, Green ED, <EM>et al.</EM>
<A HREF="http://www.genome.org/cgi/content/abstract/14/4/708"
TARGET=_blank>Aligning multiple genomic sequences with the threaded blockset
aligner</A>.
<EM>Genome Res</EM>. 2004 Apr;14(4):708-15.

<H3>Blastz</H3>
<P>
Chiaromonte F, Yap VB, Miller W.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=11928468&dopt=Abstract"
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>.
<EM>Pac Symp Biocomput</EM>. 2002;:115-26.

<P>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC,
Haussler D, Miller W.
<A HREF="http://www.genome.org/cgi/content/abstract/13/1/103"
TARGET=_blank>Human-mouse alignments with BLASTZ</A>.
<EM>Genome Res</EM>. 2003 Jan;13(1):103-7.
