<H2>Description</H2>

<P>
This track shows a comprehensive survey of cis-regulatory elements
in the mouse genome by using ChIP-seq &#40;Robertson <EM> et al.</EM>, 2007&#41; to identify
transcription factor binding sites and chromatin modification
profiles in many mouse &#40;C57Bl/6&#41; tissues and primary cells, 
including bone marrow, cerebellum, cortex, heart, kidney,
liver, lung, spleen, mouse embryonic fibroblast cells 
&#40;MEFs&#41; and embryonic stem &#40;ES&#41; cells.
</P> 
<P>
In specific, the Ren lab examined RNA polymerase II &#40;PolII&#41;, 
co-activator protein p300, the insulator protein CTCF, and 
two chromatin modification marks H3K4me3 and H3K4me1 due 
to their demonstrated utilities in identifying promoters, enhancers 
and insulator elements &#40;Barski <EM>et al.</EM>, 2007; Blow <EM>et al.</EM>, 2010; 
Heintzman <EM>et al.</EM>, 2009; Kim <EM>et al.</EM>, 2007; Kim <EM> et al.</EM>,
2005a; Visel <EM>et al.</EM>, 2009&#41;.
Enrichment of H3K4me3 or PolII signals is a 
strong indicator of active promoter, while the 
presence of p300 or H3K4me1 outside of promoter 
regions has been used as a mark for enhancers. 
CTCF binding sites are considered as a mark for 
potential insulator elements.  For each transcription 
factor or chromatin mark in each tissue, ChIP-seq was
carried out with at least two biological 
replicates. Each experiment produced 20-30 
million monoclonal, uniquely mapped tags.
</P>

<H2>Display Conventions and Configuration</H2>

<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
This track contains the following views:
<DL>
<DT><I>Peaks</I></DT><DD> Regions of signal enrichment based on processed data
(normalized data from pooled replicates). Intensity is represented in
grayscale,  darker shading shows higher intensity (a solid vertical line
	in the peak region represents the the point with the highest signal).
</DD>
<DT><I>Signal</I></DT><DD>Density graph (wiggle) of signal enrichment based on
processed data.</DD>
</DL>
</P>

<H2>Methods</H2>

<P>
Cells were grown according to the approved
<A HREF="/ENCODE/protocols/cell/mouse" TARGET=_BLANK>ENCODE cell culture protocols</A>.
</P>

<P><DT><B><I>Enrichment and Library Preparation</I></B></DT>

Chromatin immunoprecipitation was performed according to 
<A HREF="http://bioinformatics-renlab.ucsd.edu/RenLabChipProtocolV1.pdf" 
TARGET=_BLANK>Ren Lab ChIP Protocol</A>.
</P>
<P>
Library construction was performed according to 
<A HREF="http://bioinformatics-renlab.ucsd.edu/RenLabLibraryProtocolV1.pdf"
TARGET=_BLANK>Ren Lab Library Protocol</A>.
</P>

<P><DT><B><I>Sequencing and Analysis</I></B></DT>
Samples were sequenced on Illumina Genome Analyzer II
Genome Analyzer IIx, and HiSeq 2000 platforms
for 36 cycles.  Image analysis, base calling and 
alignment to the mouse genome version mm9 were performed 
using Illumina&#39;s RTA and Genome Analyzer Pipeline software. 
Alignment to the mouse genome was performed using ELAND
or Bowtie (Langmead <EM>et al.</EM>, 2009) with 
a seed length of 25 and allowing up to two mismatches. 
Only the sequences that mapped to one location were used 
for further analysis.  Of those sequences, clonal reads,
defined as having the same start position on the same 
strand, were discarded.  BED and wig files were
created using custom perl scripts.
</P>

<H2>Credits</H2>

<P>
These data were generated and analyzed in
<A HREF="http://licr-renlab.ucsd.edu/" TARGET=_BLANK>Bing Ren&#39;s laboratory</A>
at the Ludwig Institute for Cancer Research.
</P>
<P>
Contact: <A HREF="mailto:&#121;&#55;&#115;h&#101;n&#64;&#117;&#99;&#115;&#100;.
&#101;&#100;u">Yin Shen</A>
<!-- above address is y7shen at ucsd.edu -->
</P>

<H2>References</H2>

<p>
Barski A, Cuddapah S, Cui K, Roh TY, Schones DE, Wang Z, Wei G, Chepelev I, Zhao K.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17512414" target="_blank">
High-resolution profiling of histone methylations in the human genome</a>.
<em>Cell</em>. 2007 May 18;129(4):823-37.
</p>

<p>
Blow MJ, McCulley DJ, Li Z, Zhang T, Akiyama JA, Holt A, Plajzer-Frick I, Shoukry M, Wright C, Chen F <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/20729851" target="_blank">
ChIP-Seq identification of weakly conserved heart enhancers</a>.
<em>Nat Genet</em>. 2010 Sep;42(9):806-10.
</p>

<p>
Heintzman ND, Hon GC, Hawkins RD, Kheradpour P, Stark A, Harp LF, Ye Z, Lee LK, Stuart RK, Ching CW <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/19295514" target="_blank">
Histone modifications at human enhancers reflect global cell-type-specific gene expression</a>.
<em>Nature</em>. 2009 May 7;459(7243):108-12.
</p>

<p>
Kim TH, Abdullaev ZK, Smith AD, Ching KA, Loukinov DI, Green RD, Zhang MQ, Lobanenkov VV, Ren B.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17382889" target="_blank">
Analysis of the vertebrate insulator protein CTCF-binding sites in the human genome</a>.
<em>Cell</em>. 2007 Mar 23;128(6):1231-45.
</p>

<p>
Kim TH, Barrera LO, Qu C, Van Calcar S, Trinklein ND, Cooper SJ, Luna RM, Glass CK, Rosenfeld MG, Myers RM <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/15899964" target="_blank">
Direct isolation and identification of promoters in the human genome</a>.
<em>Genome Res</em>. 2005 Jun;15(6):830-9.
</p>

<p>
Langmead B, Trapnell C, Pop M, Salzberg SL.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/19261174" target="_blank">
Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</a>.
<em>Genome Biol</em>. 2009;10(3):R25.
</p>

<p>
Robertson G, Hirst M, Bainbridge M, Bilenky M, Zhao Y, Zeng T, Euskirchen G, Bernier B, Varhol R, Delaney A <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17558387" target="_blank">
Genome-wide profiles of STAT1 DNA association using chromatin immunoprecipitation and massively parallel sequencing</a>.
<em>Nat Methods</em>. 2007 Aug;4(8):651-7.
</p>

<p>
Visel A, Blow MJ, Li Z, Zhang T, Akiyama JA, Holt A, Plajzer-Frick I, Shoukry M, Wright C, Chen F <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/19212405" target="_blank">
ChIP-seq accurately predicts tissue-specific activity of enhancers</a>.
<em>Nature</em>. 2009 Feb 12;457(7231):854-8.
</p>

<H2>Data Release Policy</H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column on the track configuration page and
the download page.  The full data release policy for ENCODE is available
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>
