<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=315>
  <TR><TD ALIGN=RIGHT><A HREF="http://www.broadinstitute.org/science/projects/mammals-models/brown-bat/brown-bat" TARGET=_blank>
      <IMG SRC="../images/Myotis_lucifugus.jpg" WIDTH=300 HEIGHT=219 ALT="Little Brown Bat"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT>
    <FONT SIZE=-1><em>Myotis lucifugus</em><BR>
    </FONT>
    <FONT SIZE=-2>
    Photo credit: <A HREF="http://animaldiversity.org/site/accounts/pictures/Myotis_lucifugus.html" TARGET=_blank>Phil Myers, Museum of Zoology, University of Michigan</A>. All rights reserved.<BR>
    Courtesy of
    <A HREF = "http://www.nsf.gov/"
    TARGET = _BLANK>National Science Foundation</A><BR>
    </FONT>
  </TD></TR>
</TABLE>



<P>
The Jul. 2010 <em>Myotis lucifugus</em> draft assembly 
(Broad Institute (NCBI Project 
<A
HREF="https://www.ncbi.nlm.nih.gov/bioproject/16951"
TARGET=_blank>16951</A>,
<A
HREF="ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/vertebrate_mammalian/Myotis_lucifugus/all_assembly_versions/GCA_000147115.1_Myoluc2.0" 
TARGET=_blank>GCA_000147115.1</A>)) 
produced by the 
<A
HREF="http://www.broadinstitute.org/science/projects/mammals-models/brown-bat/brown-bat"
TARGET=_blank>Little Brown Bat Genome Project</A> at 
<A
HREF="http://www.broadinstitute.org/" 
TARGET=_blank>Broad Institute of MIT and Harvard</A>.
For more information about this assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/169908/" TARGET="_blank">
Myoluc2.0</A> in the NCBI Assembly database.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Little brown bat genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>GL429775</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of scaffold GL429775</TD>
    </TR>
    <TR><TD VALIGN=Top>GL429775:1-1000000</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first million bases of GL429775</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>GL429775:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of scaffold GL429775 that spans 2000 bases, starting with position 1000000 </TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>EU044718</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number EU044718</TD>
    </TR>
    <TR><TD VALIGN=Top>AK131176</TD>
        <TD WIDTH=14>&nbsp;</TD>
      	<TD VALIGN=Top>Displays region of Other mRNA with GenBank accession
		AK131176</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>
            AY057101; AY728075 <BR>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Displays region between genome landmarks, such as
                      the mRNA markers AY057101 and AY728075.
                      This syntax may also be used for other range queries, such
                      as between uniquely determined ESTs, STSs,
                      refSeqs, etc.</TD>
    </TR>
    </TR>
    <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>zahler</TD>
        <TD WIDTH=14>&nbsp;</TD>
        <TD VALIGN=Top>Lists mRNAs deposited by scientist named Zahler</TD>
    </TR>
    <TR><TD VALIGN=Top>Evans,J.E.</TD>
        <TD WIDTH=14>&nbsp;</TD>
        <TD VALIGN=Top>Lists mRNAs deposited by co-author J.E. Evans</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
        GenBank requires the search format <em>Evans JE</em>, internally it
        uses the format <em>Evans,J.E.</em>.</TD>
    </TR>

</TABLE>

<HR>
<H3>Assembly details</H3>
<P>

The genome has been sequenced to 2X coverage. There are 3407 scaffolds (named  
GLnnnnnn) with 8247 unplaced contigs (named AAPEnnnnnnnn) in the assembly.
The sequence length is 2,034,575,300 bases. There are 68,155,432 'N' bases in gaps, leaving 1,966,419,868 ACGT bases.


<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/myoLuc2/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#microbat">Downloads</A> page. 

The Little brown bat browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#microbat_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
