<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=201>
  <TR><TD ALIGN=RIGHT><A HREF="http://www.broadinstitute.org/mammals/opossum" TARGET=_blank>
    <IMG SRC="../images/Monodelphis_domestica.jpg" WIDTH=186 HEIGHT=130 ALT="Opossum"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1> 
      <em>Monodelphis domestica</em><BR>
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of Paul Samollow,<BR>
      Southwest Foundation for Biomedical Research,<BR>San Antonio.<BR>
      (NHGRI <A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Opossum%20(short_tailed)&id=79128"
       TARGET=_blank>Press Photos</A>)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The January 2006 opossum (<em>Monodelphis domestica</em>) draft assembly 
was produced by <A HREF="http://www.broadinstitute.org/mammals/opossum"
TARGET=_blank>The Broad Institute</A>.</P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by a scaffold coordinate range, an mRNA 
or EST, or keywords from the GenBank description of an mRNA.
The following list provides examples of various types of position queries 
for the opossum genome. See the 
<A HREF="../goldenPath/help/hgTracksHelp.html" 
TARGET=_blank>User's Guide</A> for more information. 
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TD VALIGN=Top NOWRAP><B>Request:</B><BR></TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top><B>Genome Browser Response:</B><BR></TD>
    </TR>
	
    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>chrX</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays all of chromosome X</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr2:1-1000000</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays first 1,000,000 bases of chr 2</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>chr2:1000000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of chr 2 that spans 2000 bases, starting with position 1,000,000 </TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>AF049761</TD>
	<TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Displays region of mRNA with GenBank accession number AF049761</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>CLOCK</TD>
        <TD WIDTH=14></TD>
        <TD>Displays a list of genomic regions showing tBLASTn alignment of peptides from human CLOCK gene</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>EC091554</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Displays region of EST with GenBank accession EC091554</TD>
    </TR>
        <TD VALIGN=Top NOWRAP>NM_001033969</TD>
        <TD><BR></TD>        
  	<TD VALIGN=Top>Displays the region of genome with RefSeq identifier NM_001033969, the BCRA1 gene</TD>
    </TR>
    <TR><TD VALIGN=Top><br></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>immunoglobulin</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Lists immunoglobulin regions</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>immunoglobulin light chain kappa</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Lists only immunoglobulin light chain kappa mRNAs</TD> 
    </TR>
    <TR><TD VALIGN=Top NOWRAP>miller</TD> 
        <TD WIDTH=14></TD> 
        <TD VALIGN=Top>Lists mRNAs deposited by scientists named Miller</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Miska,K.B.</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Lists mRNAs deposited by co-author K.B. Miska</TD>
    </TR> 
    <TR><TD VALIGN=Top><br></TD></TR>
<!--
    <TR><TD COLSPAN="3"> 
    </TD></TR>
-->
</TABLE>
    Use this last format for author queries. Although GenBank requires the 
    search format <em>Miska KB</em>, internally it uses the format 
    <em>Miska,K.B.</em>

<H3>Assembly Details</H3>
<P>
This draft of the opossum genome has approximately 6.5X coverage. The
assembly length is nearly 3.57 billion bp including gaps (3.50 billion bp 
without gaps) contained on chromosomes 1-8, X, and Un. The N50 of the scaffolds 
including gaps is 59,775,228 bp; the N50 without gaps is 107,990. The N50 size 
is the length such that 50% of the assembled genome lies in blocks of the N50 
size or longer.</P>
<P>
The Broad Institute reports the following statistics for the monDom4 assembly:
<UL>
<LI> Total number of contigs in assembly: 72,811
<LI> Total number of supercontigs in assembly: 5,344
<LI> Placed contigs: 62,871
<LI> Placed supercontigs: 207
<LI> Unplaced contigs: 9,940
<LI> Total bases in chromosomes 1 thru 8 and X: 3,431,385,331
<LI> Total bases in chromosome Un: 174,229,318
<LI> Number of gap bases (==N) in chromosomes 1 thru 8 and X: 88,330,281
<LI> GC/AT ratio in chromosomes 1 thru 8 and X: 1,258,733,983/2,084,321,067 = 0.6039
<LI> GC/AT ratio in chromosome Un: 65,676,956/92,911,214 = .7069
</UL> </P>
<P>
The monDom4 sequence and annotation data can be downloaded from the 
Genome Browser 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/monDom4/">FTP server</A>
or 
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#opossum">Downloads</A> 
page. These data have <A HREF="../goldenPath/credits.html#opossum_use">specific 
conditions for use</A>.
</P>
<P>
Many thanks to The Broad Institute for the opossum assembly data.
The monDom4 annotation 
tracks were generated by UCSC and collaborators worldwide. See the 
<A HREF="../goldenPath/credits.html#opossum_credits">Credits</A> page
for a detailed list of the organizations and individuals who contributed 
to this release. </P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
