<H2>Description</H2>
<P>
This track shows multiple alignments to the $organism/NC_002549.1
reference strain, 160 virus sequences, composed of 158 ebola virus
sequences and two Marburg virus sequences.
The multiple alignments were generated using multiz and
other tools in the UCSC/<A HREF="http://www.bx.psu.edu/miller_lab/"
TARGET=_blank>Penn State Bioinformatics</A>
comparative genomics alignment pipeline.
</P>

<H2>Methods</H2>
<P>
Pairwise alignments with the reference sequence were generated for
each sequence using lastz version 1.03.52.
Parameters used for each lastz alignment:
<pre>
# hsp_threshold      = 2200
# gapped_threshold   = 4000 = L
# x_drop             = 910
# y_drop             = 3400 = Y
# gap_open_penalty   = 400
# gap_extend_penalty = 30
#        A    C    G    T
#   A   91  -90  -25 -100
#   C  -90  100 -100  -25
#   G  -25 -100  100  -90
#   T -100  -25  -90   91
# seed=1110100110010101111 w/transition
# step=1
</pre>
Pairwise alignments were then linked into chains using a dynamic programming
algorithm that finds maximally scoring chains of gapless subsections
of the alignments organized in a kd-tree.  Parameters used in
the chaining (axtChain) step: -minScore=10 -linearGap=loose
</P>
<P>
High-scoring chains were then placed along the genome, with
gaps filled by lower-scoring chains, to produce an alignment net.
</P>
<P>
The multiple alignment was constructed from the resulting best-in-genome
pairwise alignments progressively aligned using multiz/autoMZ,
following a simple binary tree phylogeny:
<pre>
(((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((
(((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((
(NC_002549v1 AF086833v2) AY142960v1) KC242791v1) KC242801v1) AF272001v1) 
AF499101v1) EU224440v2) KC242799v1) KC242796v1) AY354458v1) JQ352763v1) 
KC242792v1) KC242793v1) KC242795v1) KC242797v1) KC242794v1) KC242798v1) 
KC242786v1) KC242787v1) KC242789v1) KC242784v1) KC242785v1) KC242790v1) 
KC242788v1) KC242800v1) HQ613403v1) HQ613402v1) KJ660347v2) KJ660346v2) 
KJ660348v2) KM034562v1) KM034555v1) KM034557v1) KM034554v1) KM233056v1) 
KM233113v1) KM233070v1) KM233053v1) KM233063v1) KM233050v1) KM233069v1) 
KM233109v1) KM233110v1) KM233057v1) KM034560v1) KM233097v1) KM233039v1) 
KM233096v1) KM233099v1) KM233072v1) KM233054v1) KM233051v1) KM233098v1) 
KM233103v1) KM233049v1) KM233080v1) KM233106v1) KM233092v1) KM233102v1) 
KM233052v1) KM233062v1) KM233114v1) KM034556v1) KM034559v1) KM233075v1) 
KM233100v1) KM034551v1) KM233081v1) KM233094v1) KM233116v1) KM034553v1) 
KM233065v1) KM233068v1) KM233084v1) KM233040v1) KM233047v1) KM233112v1) 
KM233104v1) KM233045v1) KM233088v1) KM034558v1) KM034561v1) KM233105v1) 
KM233037v1) KM233108v1) KM233043v1) KM233089v1) KM233073v1) KM233078v1) 
KM233038v1) KM233044v1) KM233035v1) KM233042v1) KM233067v1) KM233071v1) 
KM233077v1) KM034550v1) KM233061v1) KM233074v1) KM233041v1) KM233085v1) 
KM233091v1) KM233055v1) KM233093v1) KM233107v1) KM233066v1) KM233101v1) 
KM233036v1) KM233079v1) KM233046v1) KM233095v1) KM233087v1) KM233058v1) 
KM233115v1) KM233086v1) KM233048v1) KM034549v1) KM233076v1) KM233082v1) 
KM233064v1) KM233090v1) KM233059v1) KM233083v1) KM034552v1) KM233111v1) 
KM233060v1) KM233117v1) KM233118v1) KM034563v1) AF522874v1) NC_004161v1) 
AY769362v1) FJ621583v1) JX477166v1) JX477165v1) AB050936v1) FJ621585v1) 
FJ621584v1) JN638998v1) AY729654v1) NC_006432v1) KC545390v1) KC545391v1) 
KC545389v1) KC545392v1) KC589025v1) KC242783v2) FJ217161v1) NC_014373v1) 
KC545394v1) KC545395v1) KC545393v1) KC545396v1) FJ217162v1) NC_014372v1) 
EU338380v1) FJ968794v1) NC_024781v1) NC_001608v3)
</pre>
<pre>
(((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((
(((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((
(NC_002549v1_1976 AF086833v2_1976) Mayinga_2002) Bonduni_1977) deRoover_1976)
Mayinga_1976) Mouse_Mayinga_2002) GuineaPig_Mayinga_2007) 13709Kikwit_1995)
13625Kikwit_1995) Zaire_1995) Kikwit_1995) Gabon_1994) 1Eko_1996)
1Mbie_Gabon_1996) 1Oba_Gabon_1996) 2Nza_1996) 1Ikot_Gabon_1996) Luebo1_2007)
Luebo23_2007) Luebo4_2007) Luebo9_2007) Luebo0_2007) Luebo5_2007) Luebo43_2007)
Ilembe_2002) M-M_2007) 034-KS_2008) Guinea_Gueckedou-C07_2014)
Guinea_Kissidougou-C15_2014) Guinea_Gueckedou-C05_2014) G3686v1_2014)
G3676v2_2014) G3677v2_2014) G3676v1_2014) G3735v1_2014) G3856v1_2014)
G3770v2_2014) G3724_2014) G3764_2014) G3713v2_2014) G3770v1_2014) G3846_2014)
G3848_2014) G3735v2_2014) G3682v1_2014) G3823_2014) EM112_2014) G3822_2014)
G3825v2_2014) G3782_2014) G3729_2014) G3713v3_2014) G3825v1_2014) G3831_2014)
G3707_2014) G3799_2014) G3840_2014) G3818_2014) G3829_2014) G3713v4_2014)
G3758_2014) G3856v3_2014) G3677v1_2014) G3680v1_2014) G3788_2014) G3826_2014)
EM096_2014) G3800_2014) G3820_2014) NM042v1_2014) G3670v1_2014) G3769v1_2014)
G3769v4_2014) G3807_2014) EM113_2014) EM124v3_2014) G3851_2014) G3834_2014)
EM124v1_2014) G3810v2_2014) G3679v1_2014) G3683v1_2014) G3838_2014) EM110_2014)
G3845_2014) EM120_2014) G3814_2014) G3786_2014) G3796_2014) EM111_2014)
EM121_2014) EM104_2014) EM119_2014) G3769v3_2014) G3771_2014) G3795_2014)
EM095_2014) G3752_2014) G3787_2014) EM115_2014) G3808_2014) G3817_2014)
G3734v1_2014) G3819_2014) G3841_2014) G3769v2_2014) G3827_2014) EM106_2014)
G3798_2014) EM124v2_2014) G3821_2014) G3810v1_2014) G3750v1_2014) G3857_2014)
G3809_2014) EM124v4_2014) EM095B_2014) G3789v1_2014) G3805v1_2014)
G3765v2_2014) G3816_2014) G3750v2_2014) G3805v2_2014) EM098_2014) G3850_2014)
G3750v3_2014) NM042v2_2014) NM042v3_2014) G3687v1_2014) Reston_PA_1990)
Pennsylvania_1990) reconstructReston_2008) Reston08-A_2008)
Alice_TX_USA_MkCQ8167_1996) Reston09-A_2009) Reston_1996) Reston08-E_2008)
Reston08-C_2008) Nakisamata_2011) Gulu_Uganda_2000) Gulu_2000) EboSud-603_2012)
EboSud-609_2012) EboSud-602_2012) EboSud-682_2012) EboSud-639_2012) Maleo_1979)
Bundibugyo_Uganda_2007) Bundibugyo_2007) EboBund-120_2012) EboBund-122_2012)
EboBund-112_2012) EboBund-14_2012) Cote_dIvoire_CIEBOV_1994) Cote_dIvoire_1994)
Yambio_2004) Boniface_1976) Marburg_KitumCave_Kenya_1987)
Marburg_MtElgon_Musoke_Kenya_1980)
</pre>
Framing tables from the genes were constructed to enable
visualization of codons in the multiple alignment display.</P>

<H2>Credits</H2>
<P> This track was created using the following programs:
<UL>
<LI> Alignment tools: blastz and multiz by Minmei Hou, Scott Schwartz and Webb
Miller of the <A HREF="http://www.bx.psu.edu/miller_lab/"
TARGET=_blank>Penn State Bioinformatics Group</A>
<LI> Chaining and Netting:  axtChain, chainNet by Jim Kent at UCSC
<LI> Conservation scoring: phastCons, phyloP, phyloFit, tree_doctor, msa_view and
other programs in PHAST by
<A HREF="http://siepellab.labsites.cshl.edu/"
TARGET=_blank>Adam Siepel</A> at Cold Spring Harbor Laboratory (original development
done at the Haussler lab at UCSC).
<LI> MAF Annotation tools: mafAddIRows by Brian Raney, UCSC; mafAddQRows
by Richard Burhans, Penn State; genePredToMafFrames by Mark Diekhans, UCSC
<LI> Tree image generator: phyloPng by Galt Barber, UCSC
<LI> Conservation track display: Kate Rosenbloom, Hiram Clawson (wiggle
display), and Brian Raney (gap annotation and codon framing) at UCSC
</UL>
</P>

<h2>References</h2>

<h3>Phylo-HMMs, phastCons, and phyloP:</h3>

<p>
Stephen K. Gire, Augustine Goba, Kristian G. Andersen, ...  Robert F. Garry, S. Humarr Khan, and Pardis C. Sabeti
<a href="http://www.sciencemag.org/content/345/6202/1369.abstract"
target="blank">Genomic surveillance elucidates Ebola virus origin
and transmission during the 2014 outbreak</a>
<em>Science</em> 12 September 2014: 345 (6202), 1369-1372.<br>
Published online 28 August 2014 [DOI:10.1126/science.1259657] 
</p>

<p>
Pollard KS, Hubisz MJ, Rosenbloom KR, Siepel A.
<a href="http://genome.cshlp.org/content/20/1/110.long" target="_blank">
Detection of nonneutral substitution rates on mammalian phylogenies</a>.
<em>Genome Res</em>. 2010 Jan;20(1):110-21.
</p>

<p>
Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K,
Clawson H, Spieth J, Hillier LW, Richards S, <em>et al</em>.
<a href="http://genome.cshlp.org/content/15/8/1034" target="_blank">
Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes</a>.
<em>Genome Res</em>. 2005 Aug;15(8):1034-50.
</p>

<h3>Chain/Net:</h3>

<p>
Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D.
<a href="http://www.pnas.org/content/100/20/11484.abstract" target="_blank">
Evolution's cauldron: duplication, deletion, and rearrangement in the mouse and human genomes</a>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Sep 30;100(20):11484-9.
</p>

<h3>Multiz:</h3>

<p>
Blanchette M, Kent WJ, Riemer C, Elnitski L, Smit AF, Roskin KM,
Baertsch R, Rosenbloom K, Clawson H, Green ED, <em>et al</em>.
<a href="http://genome.cshlp.org/content/14/4/708.abstract" target="_blank">
Aligning multiple genomic sequences with the threaded blockset aligner</a>.
<em>Genome Res</em>. 2004 Apr;14(4):708-15.
</p>

<p>
Harris RS.
<a href="http://www.bx.psu.edu/~rsharris/rsharris_phd_thesis_2007.pdf"
target="_blank">Improved pairwise alignment of genomic DNA</a>.
<em>Ph.D. Thesis</em>. Pennsylvania State University, USA. 2007.
</p>

<h3>Blastz:</h3>

<p>
Chiaromonte F, Yap VB, Miller W.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/11928468" target="_blank">
Scoring pairwise genomic sequence alignments</a>.
<em>Pac Symp Biocomput</em>. 2002;:115-26.
</p>

<p>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC,
Haussler D, Miller W.
<a href="http://genome.cshlp.org/content/13/1/103.abstract" target="_blank">
Human-mouse alignments with BLASTZ</a>.
<em>Genome Res</em>. 2003 Jan;13(1):103-7.
</p>
