<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=315 HEIGHT=138>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://www.fugu-sg.org"
    TARGET=_blank><IMG SRC="../images/Takifugu_rubripes.jpg"
	WIDTH=300 HEIGHT=123 ALT="Fugu"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Takifugu rubripes </em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of
(<A HREF="http://www.imcb.a-star.edu.sg/php/venkatesh.php"
      TARGET=_blank>Byrappa Venkatesh, all rights reserved.</A>)
      <P>
    </FONT>
  </TD></TR>
</TABLE>

<P>
The <em> Takifugu rubripes</em> v.3.0 (Aug. 2002) whole genome shotgun
assembly was provided by the
<A HREF="http://genome.jgi-psf.org/fugu6/fugu6.info.html"
TARGET=_blank>US DOE Joint Genome Institute</A> (JGI)
as part of the International Fugu Genome Consortium, led by JGI and the 
<A HREF="http://www.fugu-sg.org"
TARGET=_blank>Singapore Institute of Molecular and Cell Biology</A> (IMCB). 

<H3>Sample position queries</P></H3>
<P>
A genome position can be specified by the accession number of
an mRNA or EST, a scaffold range, an Ensembl transcript ID, or keywords from 
the GenBank description of an mRNA. The following list provides 
examples of various types of position queries for the Fugu genome. Note that
some position queries (e.g. "huntington") may return matches to the mRNA records
of other species. In these cases, the mRNAs are mapped to their homologs in Fugu.
See the 
<A HREF="../goldenPath/help/hgTracksHelp.html" TARGET=_blank>User Guide</A> for more help. 
<P>

<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
<TR><TD VALIGN=Top NOWRAP><B>Request:</B><br></TD>
	<TD VALIGN=Top COLSPAN=2><B>&nbsp;&nbsp; Genome Browser Response:</B><br></TD></TR>
	
<TR><TD VALIGN=Top><br></TD></TR>
	
<TR><TD VALIGN=Top NOWRAP>scaffold_407</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays all of scaffold 407</TD></TR>
<TR><TD VALIGN=Top NOWRAP>scaffold_1:1-200000</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays first two hundred thousand bases of scaffold 1</TD></TR>
    <TR><TD VALIGN=Top NOWRAP>scaffold_1:100000+2000</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of scaffold 1 that spans 2000 bases, starting with position 100000 </TD>
    </TR>
<TR><TD VALIGN=Top><br></TD></TR>

<TR><TD VALIGN=Top NOWRAP>AF083811</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays region containing fugu mRNA with GenBank accession number AJ310911</TD></TR>

<TR><TD VALIGN=Top NOWRAP>BU805870</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays region containing fugu EST with GenBank accession BU805870
</TD></TR>

<TR><TD VALIGN=Top NOWRAP>SINFRUT00000127009</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays region containing Ensembl gene prediction transcript SINFRUT00000127009
</TD></TR>

<TR><TD VALIGN=Top><br></TD></TR>

<TR><TD VALIGN=Top NOWRAP>p53</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists mRNAs related to the p53 tumor suppressor
</TD></TR>


<TR><TD VALIGN=Top NOWRAP>pseudogene mRNA</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists transcribed pseudogenes but not cDNAs, in GenBank</TD></TR>
<TR><TD VALIGN=Top NOWRAP>homeobox caudal</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes in GenBank</TD></TR>
<TR><TD VALIGN=Top NOWRAP>zinc finger</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD></TR>
<TR><TD VALIGN=Top NOWRAP>kruppel zinc finger</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD></TR>
<TR><TD VALIGN=Top NOWRAP>huntington</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists mRNAs associated with Huntington's disease</TD></TR>

<TR><TD VALIGN=Top><br></TD></TR>

<TR><TD VALIGN=Top NOWRAP>zahler</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Zahler</TD></TR>
<TR><TD VALIGN=Top NOWRAP>Evans,J.E.</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists mRNAs deposited by co-author J.E.  Evans</TD></TR>

<TR><TD VALIGN=Top><br></TD></TR>
	
<TR><TD COLSPAN="3" > Use this last format for author queries. Although GenBank 
requires the search format <em>Evans JE</em>, internally it uses the
format <em>Evans,J.E.</em></TD></TR>
</TABLE>
<P>
<HR>
<H3>Assembly details</H3>
This assembly was constructed with the JGI
assembler, JAZZ, from paired end sequencing reads produced by JGI and IMCB
at JGI, Myriad 
Genetics, and Celera Genomics, resulting in a sequence coverage of 5.7X. All 
reads are plasmid, cosmid, or BAC end sequences, with the predominant coverage
derived from 2 Kb insert plasmids. This assembly contains 20,379
scaffolds totaling 319 million base pairs. The largest 679 scaffolds
total 160 million base pairs. 
Note: IMCB and Ensembl refer to this release of the Fugu genome as Version 2 rather than 
Version 3, because it is the second annotated release.
</P>

<P>
Downloads of the Fugu data and annotations may be obtained from the UCSC
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/fr1/"
TARGET=_blank>FTP site</A> or
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#fugu"
TARGET=_blank>Downloads</A> page.  These data have been freely
provided by the JGI for use in the UCSC Genome Browser.
</P>

<P>
Note: the genus name Fugu was recently changed to Takifugu.
GenBank files for this organism are named with the Takifugu genus,
but NCBI includes Fugu as a synonym in the database for searches.
Both genus names are currently in use.
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
