<H2>Description</H2> <P>
<P>
This track shows a measure of evolutionary conservation in <I>C. elegans</I> and
<I>C. briggsae</I> based on a phylogenetic hidden Markov model (phastCons).
Blastz alignments of the March 2004 assembly of <I>C. elegans</I> (ce2) and the July
2002 assembly of <I>C. briggsae</I> (cb1) were used to generate this
annotation.</P>
<P>
In full display mode, this track shows the phylogenetic conservation score
as well as a summary of the pairwise alignments (shown in dense display
mode using a grayscale density gradient).  The checkboxes in the track
configuration section allow the exclusion of species from the pairwise
display; however, this does not remove them from the conservation score
display.  </P>
<P>
When zoomed-in to the base-display level, the track shows the base
composition of each alignment.  The numbers and symbols on the
&quot;Gaps&quot; line indicate the lengths of gaps in the <I>C. elegans</I>
sequence at those alignment positions relative to <I>C. briggsae</I>. 
If there is sufficient space in the display, the size of the gap is shown; 
if not, and if the gap size is a multiple of 3, a &quot;*&quot; is displayed, 
otherwise &quot;+&quot; is shown.  To view
detailed information about the alignments at a specific position, zoom in
the display to 30,000 or fewer bases, then click on the alignment.  </P>
<P>
This track may be configured in a variety of ways to highlight different aspects 
of the displayed information. Click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph 
configuration help</A> link for an explanation of the configuration options.</P>

<H2>Methods</H2>
<P> 
Best-in-genome pairwise alignments were generated using blastz, followed by
chaining and netting.  These alignments were then assigned conservation
scores by phastCons.</P>
<P>
The phastCons program computes conservation scores based on a phylo-HMM, a
type of probabilistic model that describes both the process of DNA
substitution at each site in a genome and the way this process changes from
one site to the next (Felsenstein and Churchill 1996, Yang 1995, Siepel and
Haussler 2005).  PhastCons uses a two-state phylo-HMM, with a state for
conserved regions and a state for non-conserved regions.  The value plotted
at each site is the posterior probability that the corresponding alignment
column was "generated" by the conserved state of the phylo-HMM.  These
scores reflect the phylogeny (including branch lengths) of the species in
question, a continuous-time Markov model of the nucleotide substitution
process, and a tendency for conservation levels to be autocorrelated along
the genome (i.e., to be similar at adjacent sites).  The general reversible
(REV) substitution model was used.  Note that, unlike many
conservation-scoring programs, phastCons does not rely on a sliding window
of fixed size, so short highly-conserved regions and long moderately
conserved regions can both obtain high scores.  More information about
phastCons can be found in Siepel et al. (2005).</P> 
<P> 
PhastCons currently treats alignment gaps as missing data, which
sometimes has the effect of producing undesirably high conservation scores
in gappy regions of the alignment.  We are looking at several possible ways
of improving the handling of alignment gaps.</P>

<H2>Credits</H2>
<P>
This track was created at UCSC using the following programs:
<UL>
<LI>
Blastz by Scott Schwartz and Webb Miller of the 
<A HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank>Penn State Bioinformatics 
Group</A>. 
<LI>
AxtBest, axtChain, chainNet, netSyntenic, and netClass 
by Jim Kent at UCSC. 
<LI> PhastCons by Adam Siepel at UCSC. 
<LI>"Wiggle track" plotting software by Hiram Clawson at UCSC.
</UL>
</P>

<H2>References</H2>

<H3>Phylo-HMM</H3>
<P>
Felsenstein J, Churchill GA.  
<A HREF="https://academic.oup.com/mbe/article/13/1/93/1055515/A-Hidden-Markov-Model-approach-to-variation-among"
TARGET=_blank>A hidden Markov model approach to 
variation among sites in rate of evolution</A>.
<em>Mol Biol Evol</em>. 1996 Jan;13(1):93-104.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8583911" target="_blank">8583911</a>
</p>

<p>
Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K, Clawson H, Spieth J, Hillier
LW, Richards S <em>et al</em>.
<a href="https://genome.cshlp.org/content/15/8/1034"  target="_blank">
Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes</a>.
<em>Genome Res</em>. 2005 Aug;15(8):1034-50.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16024819" target="_blank">16024819</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1182216" target="_blank">PMC1182216</a>
</p>

<P>
Siepel A, Haussler D.
<A HREF="http://compgen.cshl.edu/~acs/phylohmm.pdf"
TARGET=_blank>Phylogenetic Hidden Markov Models</A>.
In: Nielsen R, editor. Statistical Methods in Molecular Evolution. 
New York: Springer; 2005. pp. 325-351.
</P>

<P>
Yang Z.  
<A HREF="https://www.genetics.org/content/139/2/993"
TARGET=_blank>A space-time process model for the evolution of DNA sequences</A>.
<em>Genetics</em>. 1995 Feb;139(2):993-1005.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/7713447" target="_blank">7713447</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1206396" target="_blank">PMC1206396</a>
</p>

<H3>Chain/Net</H3>
<P>
Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D.
<A HREF="https://www.pnas.org/content/100/20/11484"
TARGET=_blank>Evolution's cauldron: 
Duplication, deletion, and rearrangement in the mouse and human genomes</A>. 
<em>Proc Natl Acad Sci U S A</em>. 2003 Sep 30;100(20):11484-9.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14500911" target="_blank">14500911</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC208784" target="_blank">PMC208784</a>
</p>

<H3>Blastz</H3>
<P>
Chiaromonte F, Yap VB, Miller W. 
<A HREF="http://psb.stanford.edu/psb-online/proceedings/psb02/chiaromonte.pdf"
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>. 
<em>Pac Symp Biocomput</em>. 2002:115-26.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11928468" target="_blank">11928468</a>
</p>

<P>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC, Haussler D, Miller W.
<A HREF="https://genome.cshlp.org/content/13/1/103.abstract"
 TARGET=_blank>Human-Mouse Alignments with BLASTZ</A>.
<em>Genome Res</em>. 2003 Jan;13(1):103-7.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12529312" target="_blank">12529312</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC430961" target="_blank">PMC430961</a>
</p>
