<H2>Description</H2>
<P>
This track shows microRNA target sites in 3' UTRs as predicted by PicTar,
based on the 3' UTRs annotated by WormBase and on UCSC's 3' UTRs predictions
(see below). </P>

<H2>Methods</H2>
<P>
The original PicTar algorithm was published in Krek <EM>et al.</EM>,
2005. The annotations displayed in this track are updated predictions as
published in Lall <EM>et al.</EM>, 2006.</P>
<P>
PicTar is a hidden Markov model
that assigns probabilities to 3' UTR subsequences as a binding site for a
microRNA, considers all possible ways the 3' UTR could be bound by microRNAs,
and then uses a maximum likelihood method to compute the optimal likelihood
under which the 3' UTR could be explained by microRNAs and background.
The score is this likelihood divided by background, <EM>i.e.</EM>, the local
base composition of each 3' UTR is taken into account. To fit the
track conventions of the UCSC browser (integers), all scores were scaled by
the maximum score of all microRNA 3'-UTR scores observed. Note that the PicTar
algorithm
scores any 3' UTR that has at least one aligned conserved predicted binding
site for a microRNA, but then incorporates all possible binding sites into
the score, even if they appear to be non-conserved. Because the score for
a 3' UTR is a &quot;phylo&quot; average over all orthologous 3' UTRs used,
&quot;scattered&quot;
sites that appear in many species may boost the score, and individual sites
shown in the display may not be aligned and conserved in all species under
consideration. </P>
<P>
For the nematode target predictions, three species were used:
<EM>C. elegans</EM>, <EM>C. briggsae</EM> and <EM>C. remanei</EM>. Because 3' 
UTRs are rarely annotated in WormBase or RefSeq, UCSC &quot;guessed&quot; the
3' UTR sequences for all transcripts without 3' UTRs annotations by extending 
the CDS 500 bases downstream of the stop codon, and stopping when another
CDS was hit.
</P>
<P>
For a detailed analysis of signal-to-noise ratios and sensitivity,
please refer to Lall <EM>et al.</EM>, 2006. </P>

<H2>Credits</H2>
<P>
Thanks to the Dominic Grün, Yi-Lu Wang, and Nikolaus Rajewsky for providing
this annotation.  More detailed information about individual
predictions, including links to other databases, can be found on the
<A HREF="https://pictar.mdc-berlin.de/" TARGET=_blank>PicTar</A> website,
a project of the
<A HREF="https://www.mdc-berlin.de/1151037/en/research/research_teams/systems_biology_of_gene_regulatory_elements"
TARGET=_blank>Rajewsky lab</A> while at the New York University
<A HREF="https://as.nyu.edu/biology.html"
TARGET=_blank>Center for Comparative Functional Genomics</A>. </P>

<H2>References</H2>
<P>
Krek A, Gr&#252;n D, Poy MN, Wolf R, Rosenberg L, Epstein EJ, MacMenamin P, da Piedade I, Gunsalus
KC, Stoffel M <em>et al</em>.
<A HREF="https://www.nature.com/articles/ng1536"
TARGET=_blank>Combinatorial microRNA target predictions</A>.
<em>Nat Genet</em>. 2005 May;37(5):495-500.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15806104" target="_blank">15806104</a>
</p>

<P>
Murphy WJ, Eizirik E, O'Brien SJ, Madsen O, Scally M, Douady CJ, Teeling E, Ryder OA, Stanhope MJ,
de Jong WW <em>et al</em>.
<A HREF="https://www.cell.com/current-biology/abstract/S0960-9822(06)01059-1"
TARGET=_blank>A genome-wide map of conserved microRNA targets in <EM>C.
elegans</EM></A>.
<em>Curr Biol</em>. 2006 Mar 7;16(5):460-71.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16458514" target="_blank">16458514</a>
</p>
