<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=210 HEIGHT=250>
  <TR><TD ALIGN=RIGHT><A HREF="https://www.hgsc.bcm.edu/content/sea-urchin-genome-project" 
    TARGET=_blank>
    <IMG SRC="../images/S_purpuratus.jpg" WIDTH=200 HEIGHT=188 ALT="Purple Sea Urchin"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1> <em>Strongylocentrotus purpuratus</em><BR>
    </FONT>
    <FONT SIZE=-2> (Photo courtesy of Steven Murray, CSU Fullerton)
    </FONT>
  </TD></TR>
</TABLE>

<P>
The April 2005 release of the Purple Sea Urchin genome (<em>Strongylocentrotus 
purpuratus</em>) was produced by Baylor College of Medicine's Human Genome 
Sequencing Center (BCM HGSC) and corresponds to their Spur_0.5 whole genome 
shotgun assembly.  </P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by a linkage group coordinate range,
the accession number of an mRNA or EST, a gene name, or
keywords from the GenBank description of an mRNA. The following
list shows examples of valid position queries for the
<em>S. purpuratus</em> genome.  Note that some position queries
(e.g. "brachyury") may return matches to the mRNA records of
other species. In these cases, the mRNAs are mapped to their 
homologs in <em>S. purpuratus</em>. 
See the <A HREF="../goldenPath/help/hgTracksHelp.html" 
TARGET=_blank>User's Guide</A> for more information.
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TD VALIGN=Top NOWRAP><B>Request:</B><BR></TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top><B>Genome Browser Response:</B><BR></TD>
    </TR>
    <TR><TD COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top NOWRAP>Scaffold5728</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all 24,891 bases of Scaffold5728</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Scaffold49105:1-10,000</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays first 10,000 bases of Scaffold49105</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>AY044637</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Searches for regions of genome aligning to mRNA with GenBank accession AY044637</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>cdk4</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Searches for regions of genome aligning to cdk4 gene</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>cyclin dependent</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists cyclin-dependent protein mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>davidson</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists accessions deposited by authors named Davidson</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Davidson,E.H.</TD>
       	<TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author E.H. Davidson</TD>
    </TR>
    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although GenBank 
			requires the search format <em>Davidson EH</em>, internally it uses 
			the format <em>Davidson,E.H.</em></TD>
    </TR>
</TABLE>

<P>
<HR>
<H3>Assembly Details</H3>
<P>
This release was produced by assembling whole genome shotgun
reads using the Atlas genome assembly system at the BCM HGSC.
Several whole genome shotgun libraries, with
inserts of 2-6 kb, were used to produce the data. About 7 million
reads were assembled, representing about 800 Mb of sequence and about
6x coverage of the (clonable) sea urchin genome. Highly repeated
sequences were assembled separately into reptigs and merged into the
genome assembly.  Sequences from BAC clones were omitted from this
assembly and will be placed in a subsequent version of the draft
sequence.  </P>
<P>
The total length of all contigs greater than 1 Kb is 768 Mb (668 Mb of
unique contigs and 100 Mb of reptigs).  After 109,344 BAC end
sequences were mapped and used for scaffolding, and the gaps between
contigs in scaffolds are included, the total span of the assembly is
1.13 Gb. This length is 240 Mb larger than the estimated genome size
due to the preliminary nature of the scaffolding.</P>
<P>
This is a draft sequence and may contain errors; therefore, users should 
exercise caution. Typical errors in draft genome sequences include misassemblies
of repeated sequences, collapses of repeated regions, and unmerged overlaps 
(e.g. due to polymorphisms) creating artificial duplications. However, base 
accuracy in contigs (contiguous blocks of sequence) is usually very high with
most errors near the ends of contigs.</P>
<P>
More assembly details are in the 
<A HREF="ftp://ftp.hgsc.bcm.tmc.edu/pub/data/Spurpuratus/fasta/Spur20050415/READMEApril2005.txt" 
TARGET=_blank>Spur_0.5 README file</A> and on the BCM HGSC
<A HREF="https://www.hgsc.bcm.edu/content/sea-urchin-genome-project" 
TARGET=_blank>Sea Urchin Genome Project</A> web page.</P>
<P>
Bulk downloads of the sequence and annotation data are available via the Genome 
Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/strPur1/">FTP
server</A> or the 
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#urchin">Downloads</A> 
page.  These data have specific 
<A HREF="https://www.hgsc.bcm.edu/content/conditions-use" 
TARGET=_blank>conditions for use</A>. The strPur1 annotation tracks were 
generated by UCSC and collaborators worldwide. See the 
<A HREF="../goldenPath/credits.html#urchin_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to this release. </P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>low-coverage</B>.
