<H2> Description </H2>
<P>
This track shows RNA secondary structure predictions made with the
EvoFold program, a comparative method that exploits the evolutionary signal
of genomic multiple-sequence alignments for identifying conserved
functional RNA structures. </P>

<H2> Display Conventions and Configuration </H2>
<P>
Track elements are labeled using the convention <em>ID_strand_score</em>.
At the zoomed-out level, secondary structure prediction regions are
indicated by blocks, with the stem-pairing regions shown in a darker shade 
than unpaired regions. Arrows indicate the predicted strand.
When zoomed in to the base level, the specific secondary structure predictions 
are shown in parenthesis format. The confidence score for each position is
indicated in grayscale, with darker shades corresponding to higher scores.
<P>
The details page for each track element shows the predicted secondary structure 
(labeled <em>SS anno</em>), together with details of the multiple species 
alignments at that location. Substitutions relative to the human sequence are 
color-coded according to their compatibility with the predicted secondary 
structure (see the color legend on the details page). Each prediction is 
assigned an overall score and a sequence of position-specific scores. The 
overall score measures evidence for any functional RNA structures in the given 
region, while the position-specific scores (0 - 9) measure the confidence of 
the base-specific annotations. Base-pairing positions are annotated 
with the same pair symbol. The <em>offsets</em> are provided to ease
visual navigation of the alignment in terms of the human sequence. The offset
is calculated (in units of ten) from the start position of the element on 
the positive strand or from the end position when on the negative strand.</P>
<P>
The graphical display may be filtered to show only those track elements 
with unnormalized scores that meet or exceed a certain threshhold. To set a 
threshhold, type the minimum score into the text box at the top of the 
description page.</P>

<H2>Methods</H2>
<P>
Evofold makes use of phylogenetic
stochastic context-free grammars (phylo-SCFGs), which are combined
probabilistic models of RNA secondary structure and primary sequence
evolution. The predictions consist of both a specific RNA secondary
structure and an overall score. The overall score is essentially a
log-odd score phylo-SCFG modeling the constrained evolution of
stem-pairing regions and one which only models unpaired regions.</P>
<P>
The predictions for this track were based on the conserved elements of
an 8-way vertebrate alignment of the human, chimpanzee, mouse, rat,
dog, chicken, zebrafish, and Fugu assemblies. </P>

<H2>Credits</H2> 
<P>
The EvoFold program and browser track were developed by 
<A HREF="mailto:jsp@soe.ucsc.edu">Jakob Skou Pedersen</A> of the UCSC
Genome Bioinformatics Group.</P>

<H2>References</H2>
<P>
Knudsen, B. and J.J. Hein.  
<A HREF="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/15/6/446"
TARGET=_blank>RNA secondary structure prediction using stochastic context-free 
grammars and evolutionary history</A>.
<em>Bioinformatics</em> <B>15</B>(6), 446-54 (1999).</P>
<P>
Pedersen, J.S., Bejerano, G. and Haussler, D. Identification and
classification of conserved RNA secondary structures in the human
genome. <em>(In preparation)</em>.
<P>
Pedersen, J.S., Meyer, I.M., Forsberg, R., Simmonds, P. and Hein, J. 
<A HREF="http://nar.oxfordjournals.org/cgi/content/full/32/16/4925"
TARGET=_blank>A comparative method for finding and folding RNA
secondary structures within protein-coding regions</A>. 
<em> Nucl Acids Res</em> <B>32</B>, 4925-36 (2004).</P>
<P>
Siepel, A., Bejerano, G., Pedersen, J.S., Hinrichs, A., Hou, M.,
Rosenbloom, K., Clawson, H., Spieth, J., Hillier, L. W., Richards, S.
<em>et al</em>.
Evolutionarily conserved elements in vertebrate, insect, worm, and yeast 
genomes. 
<em>Genome Res.</em> <em>(In press)</em>.</P>
