<H2> Description</H2>
<P>
This track displays translated blat alignments of mRNA for all Drosophila 
species from GenBank.</P>

<H2>Display Conventions and Configuration</H2>
<P>
This track follows the display conventions for 
<A HREF="../goldenPath/help/hgTracksHelp.html#PSLDisplay" 
TARGET=_blank>PSL alignment tracks</A>. In dense display mode, the items that
are more darkly shaded indicate matches of better quality. </P>
<P>
The strand information (+/-) for this track is in two parts.  The
first + indicates the orientation of the query sequence whose
translated protein produced the match (here always 5' to 3', hence +).
The second + or - indicates the orientation of the matching 
translated genomic sequence. Because the two orientations of a DNA 
sequence give different predicted protein sequences, there are four 
combinations.  ++ is not the same as --, nor is +- the same as -+.</P>

<H2>Methods</H2>
<P>
mRNAs for all Drosophila species were aligned against the genome using blat
in translated mode. When a single mRNA aligned in multiple places, only
the best alignments also having at least 25% sequence identity were kept.

<H2>Credits</H2>
<P>
The all-Drosphila mRNA track was produced at UCSC from mRNA sequence data
submitted to the international public sequence databases by 
scientists worldwide.</P>

<H2>References</H2>
<P>
Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, 
Wheeler DL. 
<A HREF="http://nar.oupjournals.org/cgi/content/abstract/32/suppl_1/D23" 
TARGET=_blank>GenBank: update</A>. <em>Nucleic Acids Res.</em> 
2004 Jan 1;32(Database issue):D23-6.</P>
<P>
Kent WJ.
<A HREF="http://www.genome.org/cgi/content/abstract/12/4/656" 
TARGET=_blank>BLAT - the BLAST-like alignment tool</A>.
<em>Genome Res.</em> 2002 Apr;12(4):656-64.</P>
