<H2>Description</H2>
<P>
This track shows alignments between the expressed
sequence tags (ESTs) in GenBank of all Drosophila species and the genome.
ESTs are single-read sequences, typically about 500 bases in length, that 
usually represent fragments of transcribed genes.</P>

<H2>Display Conventions and Configuration</H2>
<P>
This track follows the display conventions for 
<A HREF="../goldenPath/help/hgTracksHelp.html#PSLDisplay" 
TARGET=_blank>PSL alignment tracks</A>. In dense display mode, the items that
are more darkly shaded indicate matches of better quality. </P>
<P>
The strand information (+/-) for this track is in two parts.  The
first + or - indicates the orientation of the query sequence whose
translated protein produced the match.  The second + or - indicates the
orientation of the matching translated genomic sequence.  Because the two
orientations of a DNA sequence give different predicted protein sequences,
there are four combinations.  ++ is not the same as --, nor is +- the same
as -+.</P>

<H2>Methods</H2>
<P>
To make an EST, RNA is isolated from cells and reverse
transcribed into cDNA.  Typically, the cDNA is cloned
into a plasmid vector and a read is taken from the 5'
and/or 3' primer.  For most &mdash; but not all &mdash; ESTs, the
reverse transcription is primed by an oligo-dT, which
hybridizes with the poly-A tail of mature mRNA. The
reverse transcriptase may or may not make it to the 5'
end of the mRNA,  which may or may not be degraded.</P>
<P>
In general, the 3' ESTs mark the end of transcription
reasonably well, but the 5' ESTs may end at any point
within the transcript.  Some of the newer cap-selected
libraries cover transcription start reasonably well. Before the 
cap-selection techniques
emerged, some projects used random rather than poly-A
priming in an attempt to retrieve sequence distant from the
3' end.  These projects were successful at this, but as
a side effect also deposited sequences from unprocessed
mRNA and perhaps even genomic sequences into the EST databases.
Even outside of the random-primed projects, there is a
degree of non-mRNA contamination. Because of this, a
single unspliced EST should be viewed with considerable
skepticism.  </P>
<P>
To generate this track, ESTs for all Drosophila species from GenBank were 
aligned against the genome using blat.
Note that the maximum intron length
allowed by blat is 750,000 bases, which may eliminate some ESTs with very 
long introns that might otherwise align. When a single 
EST aligned in multiple places, the alignment having the 
highest base identity was found. Only alignments having
a base identity level within 1% of the best and at least 93% base identity 
to the genomic sequence were kept.</P>

<H2>Credits</H2>
<P>
The all-Drosphila EST track is produced at UCSC from EST sequence data
submitted to the international public sequence databases by 
scientists worldwide.</P>

<H2>References</H2>
<P>
Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, 
Wheeler DL. 
<A HREF="http://nar.oupjournals.org/cgi/content/abstract/32/suppl_1/D23" 
TARGET=_blank>GenBank: update</A>. <em>Nucleic Acids Res.</em> 
2004 Jan 1;32(Database issue):D23-6.</P>
<P>
Kent WJ.
<A HREF="http://www.genome.org/cgi/content/abstract/12/4/656" 
TARGET=_blank>BLAT - the BLAST-like alignment tool</A>.
<em>Genome Res.</em> 2002 Apr;12(4):656-64.</P>


