<H2>Description</H2>
<P>
These tracks contain BAC end read alignments produced by the BLAT
alignment algorithm (Kent 2002). The BAC end reads were derived
from sequencing the <A HREF="https://bacpacresources.org/library.php?id=166"
TARGET=_blank>CHORI-259 library</A>.
</P>

<H2>Methods</H2>
<P>
Source BAC end sequences were obtained from sequencing the
<A HREF="https://bacpacresources.org/library.php?id=166"
TARGET=_blank>CHORI-259 library</A>.
For alignments BAC end sequences were aligned to the marmoset assembly
using BLAT with parameters -minIdentity 92 -fastMap minScore 100.
To ensure unique identifiers for each alignment, only the top match
is considered for each read and the match is verified by the
Washington University-provided "reads.placed" (374 Mb) file from the
<A HREF="ftp://ftp.ncbi.nlm.nih.gov/genbank/genomes/Eukaryotes/vertebrates_mammals/Callithrix_jacchus/Callithrix_jacchus-3.2/"
TARGET=_blank>Callithrix jacchus v3.2</A> assembly.
</P>

<H2>Credits</H2>
<P>
This track was produced by Vivek Appadurai and Pat Minx at
<A HREF="https://profiles.wustl.edu/en/publications/the-common-marmoset-genome-provides-insight-into-primate-biology-"
TARGET=_blank>Washington University School of Medicine</A> from data submitted
to the NCBI sequence read archive. 
</P>

<H2>References</H2>
<p>
Kent WJ.
<a href="https://genome.cshlp.org/content/12/4/656.abstract" target="_blank">
BLAT--the BLAST-like alignment tool</a>.
<em>Genome Res</em>. 2002 Apr;12(4):656-64.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11932250" target="_blank">11932250</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC187518/" target="_blank">PMC187518</a>
</p>
