<H2>Description</H2>
This track depicts gaps in the assembly.
<P>
Gaps are represented as black boxes in this track.
If the relative order and orientation of the contigs on either side
of the gap is supported by read pair data, 
it is a <em>bridged</em> gap and a white line is drawn 
through the black box representing the gap. 
</P>
<P>This assembly contains the following principal types of gaps:
<UL>

<LI><B>Fragment</B> - gaps between the Whole Genome Shotgun contigs of a 
supercontig.  (In this context, a contig is a set of overlapping sequence
reads.  A supercontig is a set of contigs ordered and oriented during the 
Whole Genome Shotgun process using paired-end reads.)
These are represented by varying numbers of <em>N</em>s  in the assembly.  
Fragment gap sizes are usually taken from read pair data.  
</LI>
<LI><B>Clone</B> - gaps between supercontigs linked by the fingerprint map.  
These are represented by 10,000 <em>N</em>s  in the assembly.
</LI>
<LI><B>Contig</B> - gaps between supercontigs not linked by the fingerprint 
map, but instead by marker data.  (In this context, the &quot;Contig&quot; gap 
type refers to a map contig, not a sequence contig.)  These are represented
by 100 <em>N</em>s in the assembly. 
</LI>
<LI><B>Centromere</B> - gaps for centromeres were included when they could be 
reasonably localized.  These are represented by 1,000,000 <em>N</em>s in
the assembly. 
</LI>
<LI><B>Telomere</B> - gaps for telomeres were included when they could be 
reasonably localized.  These are represented by 10,000 <em>N</em>s in
the assembly. 
</LI>

</UL></P>
