<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=205>
  <TR><TD ALIGN=RIGHT>
    <A HREF="https://www.ncbi.nlm.nih.gov/bioproject/13975"
    TARGET=_blank>
    <IMG SRC="../images/Nomascus_leucogenys.jpg" WIDTH=190 HEIGHT=269 ALT="Gibbon"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Nomascus leucogenys</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of NHGRI
      (<A HREF="http://www.genome.gov/dmd/img.cfm?node=Photos/Animals/Primates&id=79115"
      TARGET=_blank>press photos</A>)
    </FONT>
  </TD></TR>
</TABLE>
<P>
The Jan. 2010 <em>Nomascus leucogenys</em> (Northern White-Cheeked Gibbon) draft assembly
(<A HREF="https://www.ncbi.nlm.nih.gov/bioproject/13975"
TARGET=_blank>NCBI project 13975</A>, GCA_000146795.1) 
was produced by Gibbon Genome Sequencing Consortium 
(the <A HREF="http://genome.wustl.edu/genomes/"
TARGET=_blank>Genome Sequencing Center</A> 
at Washington University
and Baylor College of Medicine  
<A HREF="https://www.hgsc.bcm.edu/content/gibbon-genome-project"
TARGET=_blank>Human Genome Sequencing Center</A>). For more information about this
assembly, see
<A HREF="https://www.ncbi.nlm.nih.gov/assembly/266948/" TARGET="_blank">
Nleu1.0</A> in the NCBI Assembly database.


<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a
sequenced genomic region, an mRNA or EST, a chromosomal coordinate
range, or keywords from the GenBank description of an mRNA. The
following list shows examples of valid position queries for the
Marmoset genome.  See the
<A HREF="../goldenPath/help/hgTracksHelp.html"
    TARGET=_blank>User's Guide</A> for more information.
</P>
<P>
<TABLE border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TH ALIGN=LEFT>Request:</TH>
        <TH WIDTH=14>&nbsp;</TH>
       	<TH ALIGN=LEFT>Genome Browser Response:</TH></TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD VALIGN=Top>GL398881</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays all of scaffold GL398881</TD>
    </TR>
    <TR><TD VALIGN=Top>GL397282:130,210-140,209</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays ten thousand bases of scaffold GL397282</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>GL397282:171328+1500</TD>
       	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of scaffold GL397282 that spans 1500 bases, starting with position 171328 </TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>NM_201653</TD>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>This search finds the non-Gibbon RefSeq accessioned item with name: NM_201653</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>
            BC122573; BC102169 <BR>
        <TD WIDTH=14></TD>
        <TD VALIGN=Top>Displays region between genome landmarks, such as
                      the mRNA markers BC122573 and BC102169.
                      This syntax may also be used for other range queries,
                      such as between uniquely determined ESTs, STSs,
                      refSeqs, etc.</TD>
    </TR>
    <TR><TD VALIGN=Top>BC140604</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Displays region of mRNA with GenBank accession
		number BC140604</TD>
    </TR>
   <TR><TD VALIGN=Top COLSPAN=3>&nbsp;<br></TD></TR>
    <TR><TD VALIGN=Top>pseudogene mRNA</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists transcribed pseudogenes, but not cDNAs</TD>
    </TR>
   <TR><TD VALIGN=Top>homeobox caudal</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs for caudal homeobox genes</TD>
    </TR>
    <TR><TD VALIGN=Top>zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists many zinc finger mRNAs</TD>
    </TR>
    <TR><TD VALIGN=Top>kruppel zinc finger</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists only kruppel-like zinc fingers</TD>
    </TR>
    <TR><TD VALIGN=Top>datson</TD>
        <TD WIDTH=14>&nbsp;</TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientist named Datson</TD>
    </TR>
    <TR><TD VALIGN=Top>Fuchs,E.</TD>
        <TD WIDTH=14>&nbsp;</TD>
       	<TD VALIGN=Top>Lists mRNAs deposited by co-author E. Fuchs</TD>
    </TR>
    <TR><TD VALIGN=Top COLSPAN=3>&nbsp;</TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although
	GenBank requires the search format <em>Fuchs E</em>, internally it
	uses the format <em>Fuchs,E.</em>.</TD>
    </TR>
</TABLE>


<HR>
<H3>Assembly details</H3>
<P>
The genome has been sequenced to 5.6x coverage using Sanger methods.
There are 17,968 scaffolds with a total size of 2,936,035,333 bases. The N50 size of the scaffolds is 22,692,035 bases with 41 scaffolds larger, and 17,927 scaffolds smaller. The mean scaffold size is 163,404 bases, the median scaffold size is 4,849 bases.
 
For more information about this assembly, see also:
<A HREF="https://www.ncbi.nlm.nih.gov/Traces/wgs/?val=ADFV00"
TARGET=_blank>WGS record at NCBI</A>.
</P>


<P>
Bulk downloads of the sequence and annotation data are available via 
the Genome Browser <A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/nomLeu1/">FTP
server</A> or the <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#gibbon">Downloads</A> page. 

The Gibbon browser annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="/goldenPath/credits.html#gibbon_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to the success of this release. 
</P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
