<H2>Description</H2>
This track depicts gaps in the Jan. 2010 <em>Nomascus leucogenys</em>
(Northern White-Cheeked Gibbon) draft assembly
(<A HREF="https://www.ncbi.nlm.nih.gov/bioproject/13975"
TARGET="_blank">NCBI project 13975</A>, GCA_000146795.1)
produced by the Gibbon Genome Sequencing Consortium
(the <A HREF="https://www.genome.wustl.edu/services/sequencing/" TARGET=_blank>Genome Sequencing
Center</A> at Washington University and Baylor College of Medicine
<A HREF="https://www.hgsc.bcm.edu/content/gibbon-genome-project"
TARGET="_blank">Human Genome Sequencing Center</A>).

<P>
Gaps are represented as black boxes in this track. If the relative order
and orientation of the contigs on either side of the gap is supported by
read pair data, it is a <em>bridged</em> gap and a white line is drawn 
through the black box representing the gap.</P>

<P>This assembly contains the following principal types of gaps:
<UL>

<LI><B>Fragment</B> - Gaps between the Whole Genome Shotgun contigs of a 
supercontig. These are represented by varying numbers of <em>N</em>s in the 
assembly. In this context, a contig is a set of overlapping sequence reads and  
a supercontig is a set of contigs ordered and oriented during the 
Whole Genome Shotgun process using paired-end reads. Fragment gap sizes 
are usually taken from read pair data.  
</LI>


</UL></P>


