<!-- Display image in righthand corner -->
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=230 HEIGHT=285>
  <TR><TD ALIGN=RIGHT>
    <A HREF="http://genome.jgi-psf.org/Xentr4/Xentr4.home.html"
    TARGET=_blank>
    <IMG SRC="../images/Xenopus_tropicalis.jpg" WIDTH=207 HEIGHT=260 ALT="Frog"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Xenopus tropicalis</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      Photo courtesy of UNC Chapel Hill,
      (<A HREF="http://www.unc.edu/"
      TARGET=_blank>Chris Showell</A>, all rights reserved.)
      <P>
    </FONT>
  </TD></TR>
</TABLE>

<P>
The August 2005 frog (<em>Xenopus tropicalis</em>) whole genome shotgun 
(WGS) assembly version 4.1 was sequenced and assembled by the 
DOE Joint Genome Institute (JGI).
</P>

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of a sequenced 
genomic clone, an mRNA or EST,
a chromosomal coordinate range, or keywords from 
the GenBank description of an mRNA. The following list provides examples 
of various types of position queries for the frog genome. See the 
<A HREF="../goldenPath/help/hgTracksHelp.html" 
TARGET=_blank>User's Guide</A> for more information. 
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TD VALIGN=Top NOWRAP><B>Request:</B><BR></TD>
	<TD VALIGN=Top COLSPAN=2><B>&nbsp;&nbsp; Genome Browser Response:</B><BR></TD>
    </TR>
	
    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>scaffold_3</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays all of scaffold 3</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>scaffold_3:1-2000000</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays first 2 million bases of scaffold_3</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>scaffold_3:100000+2000</TD>
	<TD WIDTH=14></TD>
       	<TD VALIGN=Top>Displays a region of scaffold_3 that spans 2000 bases, starting with position 100000 </TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>CR848332</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays location of mRNA with GenBank accession CR848332</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>AL680330</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays location of EST with GenBank accession AL680330</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>scaffold_1001.7</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays location of Genscan gene prediction with identifier scaffold_1001.7</TD>
    </TR>

    <TR><TD VALIGN=Top><BR></TD></TR>

    <TR><TD VALIGN=Top NOWRAP>dazl</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists dazl mRNAs</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>zinc finger</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists zinc finger mRNAs</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>asashima</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists mRNAs deposited by scientists named Asashima</TD>
    </TR>
    <TR><TD VALIGN=Top NOWRAP>Papalopulu,N.</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists mRNAs deposited by co-author N. Papalopulu</TD>
    </TR>

    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD COLSPAN="3">Use this last format for author queries. 
			Although GenBank requires the search format 
			<em>Papalopulu N</em>, internally it uses 
			the format <em>Papalopulu,N.</em>.</TD>
    </TR>
</TABLE>

<H3>Assembly Details</H3>
<P>
Assembly version 4.1--which was constructed from WGS reads using the JGI 
assembler, Jazz--contains 19,501 scaffolds with an average coverage of 7.65X. 
Roughly half the genome is contained in 272 scaffolds, each of at least 1.56  
Mb in length. In this release, some scaffolds showing homology to a known 
prokaryotic contaminant as well as non-cellular or vector contamination have 
been removed by the JGI and placed in a separate directory. The <em>X. 
tropicalis</em> assembly will be improved over the coming year by additional 
sequencing of large insert clones, targeted gap closure, and the incorporation 
of physical and genetic mapping information as it becomes available.
For more information about this assembly, see the JGI
<A HREF="http://genome.jgi-psf.org/Xentr4/Xentr4.home.html" 
TARGET=_blank><em>X. tropicalis</em> website</A>.</P>
<P>
The xenTro2 sequence and annotation data can be downloaded from the 
UCSC Genome Browser 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/xenTro2/">FTP server</A>
or 
<A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#xentro">downloads</A> 
page. These data have 
<A HREF="http://genome.jgi-psf.org/Xentr4/Xentr4.download.html" 
TARGET=_blank>specific conditions for use</A>.</P>
<P>
Many thanks to the JGI and the other
institutions who contributed to the sequencing and mapping effort for this
release. The xenTro2 annotation tracks were generated by UCSC and 
collaborators worldwide. See the 
<A HREF="../goldenPath/credits.html#xentro_credits">credits</A> page
for a detailed list of the organizations and individuals who contributed 
to this release. </P>
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
