<H2>Description</H2>
<P>
This track shows the NCBI clone end mappings from the
<A HREF="https://ncbiinsights.ncbi.nlm.nih.gov/2019/05/01/clone-db-retirement/"
TARGET=_blank>NCBI Clone DB</A> database.</P>
<P>
Bacterial artificial chromosomes (BACs) are a key part of many
large-scale sequencing projects.  A BAC typically consists of 50 - 300 kb of
DNA.  During the early phase of a sequencing project, it is common
to sequence a single read (approximately 500 bases) off each end of
a large number of BACs.  Later on in the project, these BAC end reads
can be mapped to the genome sequence.</P>
<P>
These BAC end pairs can be useful for validating the assembly over
relatively long ranges.  In some cases, the BACs are useful biological
reagents.  This track can also be used for determining which BAC
contains a given gene, useful information for certain wet lab experiments.</P>
<P>
The scoring scheme used for this annotation assigns 1000 to an alignment
when the BAC end pair aligns to only one location in the genome (after
filtering).  When a BAC end pair or clone aligns to multiple locations, the
score is calculated as 1500/(number of alignments).</P>

<h2>Display Conventions and Configuration</h2>

<p>
Items in this track are colored according to their strand orientation. Blue indicates alignment to the forward strand, 
and green indicates alignment to the negative strand.
</p> 

<H2>Methods</H2>
<P>
The mappings of these BAC end sequences are taken directly from the
NCBI Clone DB FTP site
<A HREF="ftp://ftp.ncbi.nih.gov/repository/clone/reports/Rattus_norvegicus/"
TARGET=_blank>ftp.ncbi.nih.gov/repository/clone/reports/Rattus_norvegicus/</A>
*.GCF_000001895.5.102.*.gff files.</P>
<P>
UCSC filtered the NCBI Clone DB mapped ends to drop ends that mapped to a
region that was three times longer than the median size of the clones in
the library.</P>
<P>
Click through on displayed items to the Clone DB database information,
including
<A HREF="https://ncbiinsights.ncbi.nlm.nih.gov/2019/05/01/clone-db-retirement/"
TARGET=_blank>Clone DB distributor</A> references.</P>
<TABLE BORDER=1>

<TR><TH COLSPAN=7>clone information from NCBI Clone DB and UCSC mapping statistics</TH>
<TR><TH>library<BR>name</TH>
    <TH>total<BR>clones</TH>
    <TH>total&nbsp;end<BR>sequences</TH>
    <TH>NCBI&nbsp;mapped<BR>ends</TH>
    <TH>UCSC&nbsp;filtered<BR>ends</TH>
    <TH>UCSC<BR>dropped</TH>
    <TH>per-cent<BR>dropped</TH>
</TR>
<TR><TH ALIGN=LEFT><A HREF="https://ncbiinsights.ncbi.nlm.nih.gov/2019/05/01/clone-db-retirement/" TARGET="_blank">RNB1</A></TH><TD ALIGN=RIGHT>168,885</TD><TD ALIGN=RIGHT>307,263</TD><TD ALIGN=RIGHT>134,215</TD><TD ALIGN=RIGHT>124,987</TD><TD ALIGN=RIGHT>9,228</TD><TD ALIGN=RIGHT>6.88</TD></TR>
<TR><TH ALIGN=LEFT><A HREF="https://ncbiinsights.ncbi.nlm.nih.gov/2019/05/01/clone-db-retirement/" TARGET="_blank">RNB2</A></TH><TD ALIGN=RIGHT>115,060</TD><TD ALIGN=RIGHT>331,272</TD><TD ALIGN=RIGHT>96,829</TD><TD ALIGN=RIGHT>89,108</TD><TD ALIGN=RIGHT>7,721</TD><TD ALIGN=RIGHT>7.97</TD></TR>
<TR><TH ALIGN=LEFT><A HREF="https://ncbiinsights.ncbi.nlm.nih.gov/2019/05/01/clone-db-retirement/" TARGET="_blank">CH230</A></TH><TD ALIGN=RIGHT>192,416</TD><TD ALIGN=RIGHT>227,486</TD><TD ALIGN=RIGHT>83,816</TD><TD ALIGN=RIGHT>78,516</TD><TD ALIGN=RIGHT>5,300</TD><TD ALIGN=RIGHT>6.32</TD></TR>
<TR><TH ALIGN=LEFT><A HREF="https://ncbiinsights.ncbi.nlm.nih.gov/2019/05/01/clone-db-retirement/" TARGET="_blank">RP32</A></TH><TD ALIGN=RIGHT>2,883</TD><TD ALIGN=RIGHT>294</TD><TD ALIGN=RIGHT>106</TD><TD ALIGN=RIGHT>105</TD><TD ALIGN=RIGHT>1</TD><TD ALIGN=RIGHT>0.94</TD></TR>
</TABLE>
</P>

<H2>Credits</H2>
<P>
Additional information about the clone, including how it
can be obtained, may be found at the
<A HREF="https://ncbiinsights.ncbi.nlm.nih.gov/2019/05/01/clone-db-retirement/"
TARGET=_blank>NCBI Clone Registry</A>. To view the registry entry for a
specific clone, open the details page for the clone and click on its name at
the top of the page.</P>
