<style>
#histoneTable  th {
    border-bottom: 2px solid #6678b1;
}
#histoneTable  td {
    padding: 0 10px 0 10px;
}
.colorPatch {
    width: 10px; 
}
</style>

<h2>Description</h2>
<p>
The ENCODE project has established an epigenomic resource for mammalian development,
profiling a diverse panel of mouse tissues at eight developmental stages from 10.5 days post
conception until birth.</p>  
<div><img border="1" src="../images/renMouseDev.png" style="margin-left: 100px;"></div>
<p>
This track set presents the results of a comprehensive study of chromatin
state across these developmental stages, based on 1,128 ChiP-seq assays of 8 histone modifications 
in 12 tissues, performed by the
<a target="_blank" href="http://renlab.sdsc.edu/renlab_website//">laboratory of Bing Ren</a>
as part of the ENCODE Consortium, phase 3.</p>
<p>
Histone modifications are post-translational chemical modifications (e.g. methylation, 
acetylation) of histone proteins present in chromatin. Histone modifications are core 
components of a cell's epigenome, and influence gene expression by modulating the activity of 
DNA sequences. </p>

<h2> Display Conventions and Configuration </h2>
<p>
This track is a multi-view composite track that contains two data types
(<em>views</em>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are <a href="/goldenPath/help/multiView.html" target=_blank>here</a>.
The views in this track are:
<dl>
<dt><em>Peaks</em></dt><dd> Regions of statistically significant signal
enrichment</DD>
<dt><em>Signal</em></dt><dd> Density graph of signal enrichment.</dd>
</DL>
<p>Peaks displayed in this track set are replicated peaks called with the 
<a target="_blank" href="https://www.encodeproject.org/software/idr/">
Irreproducible Discovery Rate</a> (IDR) framework</a> 
These peaks are identical to those in the &quot;replicated peaks&quot; files 
on the ENCODE data portal.  
Signals displayed in this track correspond to the &quot;fold change 
over control&quot; files from merged replicates, reflecting the fold enrichment of ChIP signal over 
the input control. Data for individual replicates are available through the 
<a target="_blank" href="https://www.encodeproject.org/">
ENCODE portal</a>.
</p>
<p>
Tracks are colored by histone modification. The table below list shows the assigned track color
and the genomic region type and activity level characteristic of each modification.
<table id="histoneTable" style="margin-left: 100px";>
<thead>
    <tr><th>Modification</th><th>&nbsp;</th><th colspan=2>Color</th><th>Regions</th><th>Activity Level</th></tr>
</thead>
<tr><td>  H3K4me3  </td><td></td>
        <td class="colorPatch" style='background-color:darkblue;'></td><td> dark blue</td>
            <td>  promoters </td><td>  active, poised  </td></tr>
<tr><td>  H3K4me2  </td><td></td>
        <td class="colorPatch" style='background-color:royalblue;'></td><td> royal blue</td>
            <td>  promoters, enhancers  </td><td>  active, poised  </td></tr>
<tr><td>  H3K4me1  </td><td></td><td 
        class="colorPatch" style='background-color:lightblue;'></td><td> light blue </td>
            <td>  enhancers  </td><td>  active, poised  </td></tr>
<tr><td>  H3K27ac  </td><td></td>
        <td class="colorPatch" style='background-color:darkgreen;'></td><td> dark green</td>
            <td>  promoters, enhancers  </td><td>  active  </td></tr>
<tr><td>  H3K9ac  </td><td></td>
        <td class="colorPatch" style='background-color:lightgreen;'></td><td> light green</td>
            <td>  promoters, enhancers  </td><td>  active  </td></tr>
<tr><td>  H3K36me3 </td><td></td>
        <td class="colorPatch" style='background-color:gold;'></td><td> gold</td>
            <td>  exons</td><td>  active  </td></tr>
<tr><td>  H3K27me3  </td><td></td>
        <td class="colorPatch" style='background-color:darkred;'></td><td> dark red</td>
            <td>  introns, intergenic  </td><td>  repressed  </td></tr>
<tr><td>  H3K9me3  </td><td></td>
        <td class="colorPatch" style='background-color:red;'></td><td> light red</td>
            <td>  heterochromatin  </td><td>  repressed  </td></tr>
</table>

</p>

<h2>Methods</h2>

<h3>ChIP-seq data generation</h3>
ChIP-seq experiments for all marks and tissues from E11.5 - P0 were performed 
as described in Shen et. al 2012 (see References below). 
The ChIP-seq protocol was modified slightly for all E10.5 
experiments due to the low amount of input (&quot;micro&quot; ChIP-seq). 
Detailed protocols for both standard and micro ChIP-seq are available from the ENCODE 
project portal (see Data Access section, below).

<ul>
<li><i>Standard ChIP-seq (E11.5-P0)</i>

<a target="_blank" href="https://www.encodeproject.org/documents/3125496b-c833-4414-bf5f-84dd633eb30d/@@download/attachment/Ren_Tissue_Fixation_and_Sonication_v060614.pdf">
Tissue fixation & sonication</a>, 
<a target="_blank" href="https://www.encodeproject.org/documents/89795b31-e65a-42ca-9d7b-d75196f6f4b3/@@download/attachment/Ren%20Lab%20ENCODE%20Chromatin%20Immunoprecipitation%20Protocol_V2.pdf">
Immunoprecipitation</a>, 
<a target="_blank" href="https://www.encodeproject.org/documents/4f73fbc3-956e-47ae-aa2d-41a7df552c81/@@download/attachment/Ren_ChIP_Library_Preparation_v060614.pdf">
Library preparation</a>.

<li><i>Micro-ChIP-seq (E10.5)</i>

<a target="_blank" href="https://www.encodeproject.org/documents/1fcaab50-6ca0-4778-88cb5f6b85170d21/@@download/attachment/Ren%20Lab%20ENCODE%20Tissue%20Fixation%20and%20Sonication%20Protocol%20MicroChIP.pdf">
Tissue fixation & sonication</a>, 
<a target="_blank" href="https://www.encodeproject.org/documents/18580e80-0907-4258-a412-46bcc37bd040/@@download/attachment/Ren%20Lab%20ENCODE%20Chromatin%20Immunoprecipitation%20Protocol%20MicroChIP.pdf">
Immunoprecipitation & Library preparation</a>.
</ul>

<h3>ChIP-seq antibodies</h3>
A full list of antibodies used by the Ren Lab for ENCODE can be found
<a target="_blank" href="https://www.encodeproject.org/search/?type=AntibodyLot&status=released&characterizations.lab.title=Bing+Ren%2C+UCSD">
here</a>. The antibody catalog and lot numbers are available for each experiment on the 
<a target="_blank" href="https://www.encodeproject.org/"</a>
ENCODE portal</a>.


<h3>ChIP-seq data processing</h3> 

ChIP-seq data were analyzed using a software pipeline implemented by the ENCODE Data 
Coordinating Center (DCC) for the ENCODE Consortium. Each step of the pipeline corresponds 
to a script written in the Python programming language that assembles the input files, runs 
external programs, and calculates quality-control metrics. The ENCODE histone ChIP-seq pipeline 
is among the collection of ENCODE Uniform Processing Pipelines that can be found
<a target="_blank" href="https://github.com/ENCODE-DCC/chip-seq-pipeline">
here</a>.

<h3>ChIP-seq peak calling</h3>

Replicated peaks were called using an approach based on 
<a target="_blank" href="https://www.encodeproject.org/software/idr/">
Irreproducible Discovery Rate</a>
to ensure an adequate sampling of noise for subsequent replicate comparisons. 
Briefly, peaks were initially called at a relaxed p-value threshold of 1 x 10-2.  
Such relaxed peak sets were generated for each biological replicate, for the replicates pooled, 
and for pooled pseudoreplicates of each true replicate (each pseudoreplicate consists of half 
the reads sampled without replacement). 
Peaks from the pooled replicate set were retained in the replicated peak set if they overlapped 
by at least half their length (in bases) peaks from both biological replicates.  
Additionally, peaks that overlapped both pooled pseudoreplicates were added to the replicated 
peak set.  
In this way very strong biological replicates "rescue" peaks that were only marginal in a second replicate.

<h2>Data Access</h2>

<p>Data from this and all phases of ENCODE are publicly available through the 
<a target="_blank" href="https://www.encodeproject.org/"</a>
ENCODE portal</a>.
The specific experiments included in this track set are listed
<a target="_blank" href="ettps://www.encodeproject.org/search/?type=Experiment&status=released&assay_slims=DNA+binding&award.rfa=ENCODE3&lab.title=Bing+Ren%2C+UCSD&limit=all">
here</a>.
</p>

<h2>Credits</h2>

Thanks to David Gorkin and Yanxiao Zhang at the Ren lab (UCSD/Ludwig Institute for Cancer Research) 
and Iros Barozzi of the <a target="_blank" href="https://biosciences.lbl.gov/divisions/egsb/">Environmental Genomics and Systems Biology Division</a>
at the Lawrence Berkeley National Laboratory for providing this data and assisting with track development at UCSC.</p>

<h2>References</h2>
<p>
Gorkin et al. An atlas of dynamic chromatin landscapes in the developing mouse fetus.
Nature, In Press. (pre-print: doi: 
<a target="_blank" href="https://doi.org/10.1101/166652">https://doi.org/10.1101/166652</a>)</p>

<p>
Sloan CA, Chan ET, Davidson JM, Malladi VS, Strattan JS, Hitz BC, Gabdank I, Narayanan AK, Ho M, Lee
BT <em>et al</em>.
<a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkv1160" target="_blank">
ENCODE data at the ENCODE portal</a>.
<em>Nucleic Acids Res</em>. 2016 Jan 4;44(D1):D726-32.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26527727" target="_blank">26527727</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4702836/" target="_blank">PMC4702836</a>
</p>

Shen Y, Yue F, McCleary DF, Ye Z, Edsall L, Kuan S, Wagner U, Dixon J, Lee L, Lobanenkov VV <em>et
al</em>.
<a href="https://doi.org/10.1038/nature11243" target="_blank">
A map of the cis-regulatory sequences in the mouse genome</a>.
<em>Nature</em>. 2012 Aug 2;488(7409):116-20.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22763441" target="_blank">22763441</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4041622/" target="_blank">PMC4041622</a>
</p>


