<h2>Description</h2>

The ENCODE project has established an epigenomic resource for mammalian development, 
profiling a diverse panel of mouse tissues at eight developmental stages from 10.5 days 
post conception until birth. </p>
<div><img border="1" src="../images/renMouseDev.png" style="margin-left: 100px;"></div>
<p>
This track set presents the results of a comprehensive study of chromatin accessibility 
across these developmental stages, based on 122 ATAC-seq assays in 12 tissues performed
by the
<a target="_blank" href="http://renlab.sdsc.edu/renlab_website//">laboratory of Bing Ren</a>
as part of the ENCODE Consortium, phase 3.</p>

<p>
ATAC-seq profiles &quot;open&quot; or &quot;accessible&quot; chromatin that is relatively devoid 
of packaging nucleosome particles. 
This chromatin accesibility is characteristic of active and poised cis regulatory elements,
and makes the underlying DNA more amenable to the binding of transcription factors,
which can function in turn to recruit co-factors and influence transcriptional activity.</p>

<h2>Display Conventions and Configuration</h2>
<p>
This track is a multi-view composite track that contains two data types (<em>views</em>). 
For each view, there are multiple subtracks that display individually on the browser. 
Instructions for configuring multi-view tracks are
<a href="/goldenPath/help/multiView.html" target=_blank>here</a>.
The views in this track are:
<dl>
<dt><em>Peaks</em></dt><dd> Regions of statistically significant signal
enrichment</DD>
<dt><em>Signal</em></dt><dd> Density graph of signal enrichment.</dd> 
</dl>
<p>
Peaks displayed in this track set are called on data merged between two biological replicates.
Signals displayed in this track correspond to the &quot;fold change over background&quot; 
files from merged replicates.
Data for individual replicates are available through the ENCODE portal and the
<a target="_blank" href="http://renlab.sdsc.edu/renlab_website//download/encode3-mouse-histone-atac/">Ren Lab Website</a>.</p>

<h2>Methods</h2>
<p>
<h3>ATAC-seq data generation</h3>
<p>
ATAC-seq was performed as described in Buenrostro et al. 2013 (see References below), 
with modifications to optimize for frozen tissue. 
The complete ATAC-seq protocol is available via the ENCODE data portal 
<a target="_blank" href="https://www.encodeproject.org/documents/4a2fc974-f021-4f85-ba7a-bd401fe682d1/@@download/attachment/RenLab_ATACseq_protocol_20170130.pdf">here</a>.</p>

<h3>ATAC-seq data processing</h3>
<p>
ATAC-seq reads were trimmed with a custom adapter script and mapped to mm10 using 
bowtie version 2.2.6 and samtools version 1.2 to eliminate PCR duplicates. 
MACS2 version 2.1.1.20160309 was used for generating signal tracks and peak calling 
with the following parameters: 
<pre>
-nomodel -shift 37 -ext 73 -pval 1e-2 -B -SPMR -call-summits
</pre>
</p>
<p>
To produce a set of "replicated" ATAC-seq peaks for analysis, the peak calling steps above 
were performed for each pair of replicates independently as well as a pooled set of data 
from both replicates. 
The intersectBed tool from the bedtools v2.27.1 suite was used to identify a set of 
replicated peaks, defined as the subset of peaks from the pooled set that are
present independently in both replicate peak call sets.</p>

<h2>Data Access</h2>
<p>
Experiments included in this track set are available from the ENCODE portal
<a target="_blank" href="https://www.encodeproject.org/search/?type=Experiment&status=released&assay_slims=DNA+accessibility&assay_title=ATAC-seq&award.rfa=ENCODE3&lab.title=Bing+Ren%2C+UCSD&limit=all">here</a>.</p>

<h2>Credits</h2>
<p>
Thanks to David Gorkin, Yuan Zhao and Yanxiao Zhang at the Ren lab (UCSD/Ludwig Institute for
Cancer Research) for providing this data and assisting with track development at UCSC.</p>

<h2>References</h2>
<p>
Buenrostro JD, Giresi PG, Zaba LC, Chang HY, Greenleaf WJ.
<a href="https://www.nature.com/articles/nmeth.2688" target="_blank">
Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin,
DNA-binding proteins and nucleosome position</a>.
<em>Nat Methods</em>. 2013 Dec;10(12):1213-8.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24097267" target="_blank">24097267</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3959825/" target="_blank">PMC3959825</a>
</p>
<p>
Gorkin et al. An atlas of dynamic chromatin landscapes in the developing mouse fetus.
Nature, In Press. (pre-print: doi: 
<a target="_blank" href="https://doi.org/10.1101/166652">https://doi.org/10.1101/166652</a>)</p>
<p>
Sloan CA, Chan ET, Davidson JM, Malladi VS, Strattan JS, Hitz BC, Gabdank I, Narayanan AK, Ho M, Lee
BT <em>et al</em>.
<a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkv1160" target="_blank">
ENCODE data at the ENCODE portal</a>.
<em>Nucleic Acids Res</em>. 2016 Jan 4;44(D1):D726-32.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26527727" target="_blank">26527727</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4702836/" target="_blank">PMC4702836</a>
</p>

