<H2>Description</H2>
This track depicts gaps in the assembly
(Dec. 2011, Genome Reference Consortium Mouse Build 38 (GCA_000001635.2))
of the $organism genome.  These gaps - with the
exception of intractable centromere gaps - will be closed during the
finishing process. 
</P><P>
Gaps are represented as black boxes in this track.
If the relative order and orientation of the contigs on either side
of the gap is known, it is a <em>bridged</em> gap and a white line is drawn 
through the black box representing the gap. 
</P>
<P>This assembly contains the following principal types of gaps:
<UL>
(In this context, a contig is a set of overlapping sequence reads.)
<LI><B>Clone</B> - gaps between clones (114 gaps).
<LI><B>Contig</B> - gaps between map contigs, various sizes (104 gaps).
<LI><B>Telomere</B> - 42 gaps for telomeres (100,000 <em>N</em>s)
<LI><B>Centromere</B> - 20 gaps for centromeres (size: 2,890,000 <em>N</em>s)
<LI><B>Short_arm</B> - 21 gaps for the short arm (10,000 <em>N</em>s)
	at base positions 100,001-110,000 of each chromosome.
<LI><B>other</B> - sequence of <em>N</em>s in the assembly that were not
   marked as gaps in the AGP assembly definition file, various sizes (384 gaps).
<LI><B>Fragment</B> - a single gap of 31 bases in chrX_GL456233_random.
</UL></P>
