
<h2>Description</h2>
<p>This track shows the alignment of the sequencing reads to the genome. BAM files were provided
by the data submitters, one or two files per cell. All BAM files for the cells of a cluster were merged
and are shown here together. The readgroup (RG) BAM tag indicates the original cell.</p>

<h2>Display Conventions and Configuration</h2>
<p>Alignable regions are shown in back, unalignable regions between two alignable ones shown with thin lines. For configuration options, see <a href="https://genome.ucsc.edu/goldenpath/help/hgBamTrackHelp.html">the BAM tracks help page</a></p>

<h2>Methods</h2>
<p>The alignments were used as submitted. They were merged with "samtools merge" into a single BAM file per cluster.</p>

<h2>Data Access</h2>
<p>The merged BAM files can be downloaded from the <a href='http:'>track hub directory</a>.</p>

<h2>References</h2>
<p>
<p>
The Tabula Muris Consortium, Stephen R. Quake, Tony Wyss-Coray, Spyros DarmanisThe Tabula Muris Consortium, Stephen R. Quake, Tony Wyss-Coray, Spyros Darmanis:<br>
<a href="https://doi.org/10.1101/237446" target="_blank">
Single-cell transcriptomic characterization of 20 organs and tissues from individual mice creates a Tabula Muris</a>.
<em>bioRxiv March 2018</em>, manuscript 237446.
</p>

</p>