<h2>Description</h2>

<p>
Retrotransposition is a process involving the copying of DNA by a group of
enzymes that have the ability to reverse transcribe spliced mRNAs, and the
insertion of these processed mRNAs back into the genome resulting
in single-exon copies of genes and sometime chimeric genes. Retrogenes are
mostly non-functional pseudogenes but some are functional genes that have
acquired a promoter from a neighboring gene, or transcribed pseudogenes, and
some are anti-sense transcripts that may impede mRNA translation.
</p>

<h2>Methods</h2>

<p>
All mRNAs of a species from GenBank were aligned to the genome using
<a href="http://www.bx.psu.edu/~rsharris/lastz/" target="_blank">lastz</a>
(Miller lab, Pennsylvania State University). mRNAs that aligned twice in the genome
(once with introns and once without introns) were initially screened. Next, a series
of features were scored to determine candidates for retrotransposition events. 
These features included position and length of the polyA tail, percent coverage of the 
retrogene alignment to the parent, degree of synteny with human, coverage of repetitive 
elements, number of exons that can still be aligned to the retrogene, number of putative 
introns removed at the retrogene locus and degree of divergence from the parent gene.
Retrogenes were classified using a threshold score function that is a linear combination 
of this set of features.
Retrogenes in the final set were selected using a score threshold based on a ROC plot
against the <a href="http://vega.archive.ensembl.org/index.html" target="_blank">Vega</a> annotated
pseudogenes.
</p>

<h3>Retrogene Statistics table:</h3>

<ul>
  <li><b>Expression of Retrogene:</b> The following values are possible where
  those that are not expressed are classed as <em>pseudogene</em> or
  <em>mrna</em>:</li>
  <ul>
    <li><em>pseudogene</em> indicates that the parent gene has been annotated
    by one of NCBI's RefSeq, UCSC Genes or Mammalian Gene Collection (MGC).</li>
    <li><em>mrna</em> indicates that the parent gene is a spliced mrna that
    has no annotation in NCBI's RefSeq, UCSC Genes or Mammalian Gene Collection
    (MGC). Therefore, the retrogene is a product of a potentially non-annotated
    parent gene and is a putative pseudogene of that putative parent gene.</li>
    <li><em>expressed weak</em> indicates that there is a mRNA overlapping
    the retrogene, indicating possible transcription. <em>noOrf</em> indicates
    that an ORF was not identified by BESTORF.</li>
    <li><em>expressed</em> indicates that there is a medium level of mRNAs/ESTs
    mapping to the retrogene locus, indicating possible transcription.</li>
    <li><em>expressed strong</em> indicates that there is a mRNA overlapping
    the retrogene, and at least five spliced ESTs indicating probable transcription.
    <em>noOrf</em> indicates that an ORF was not identified by BESTORF.</li>
    <li><em>expressed shuffle</em> indicates that the retrogene was inserted into
    a pre-existing annotated gene.</li>
  </ul>
  <li><b>Score:</b> Weighted sum of features (mentioned above) of the potential retrogene.</li>
  <li><b>Percent Gene Alignment Coverage (Bases Matching Parent):</b> Shows
  the percentage of the parent gene aligning to this region.</li>
  <li><b>Intron Count:</b> Number of introns is the number of gaps in
  the alignment between the parent mRNA and the genome where gaps are &gt;80 bp and
  the ratio of the mRNA alignment gap to the genome alignment gap is less than
  30% after removing repeats.</li>
  <li><b>Gap Count:</b> Numer of gaps in the alignment of between the parent
  mRNA and the genome after removing repeats. Gaps are not counted if the gap on
  the mRNA side of the alignment is a similar size to the gap in the genome
  alignment.</li>
  <li><b>BESTORF Score:</b>
  <a href="http://linux1.softberry.com/berry.phtml?topic=bestorf&group=help&subgroup=gfind"
  target="_blank">BESTORF</a> (written by Victor Solovyev) predicts potential open reading
  frames (ORFs) in mRNAs/ESTs with very high accuracy using a Markov chain model of coding
  regions and a probabilistic model of translation start codon potential. The score
  threshold for finding an ORF is 50 (Jim Kent, personal communication).</li>
</ul>

<h3><a name=orthology>Break in Orthology table:</a></h3>

<p>
Retrogenes inserted into the genome since the mouse/human divergence show a break
in the human genome syntenic net alignments to the mouse genome. A break in orthology score is 
calculated and weighted before contributing to the final retrogene score. The break in orthology score
ranges from 0-130 and it represents the portion of the genome that is missing in each species relative
to the reference genome (mouse mm10) at the retrogene locus as defined by syntenic
alignment nets. If the score is 0, there is orthologous DNA and no break in ortholog with the other species; this 
could be an ancient retrogene; duplicated pseudogenes may also score low because they are often generated 
via large segmental duplication events so the size of the pseudogene is small relative to the size of the 
inserted duplicated sequence. Scores greater than 100 represent cases where the retrogene alignment has no 
flanking alignment resulting from an ancient insertion or other complex rearrangement.
</p>
<p>
Breaks in orthology with human and dog tend to be due to genomic
insertions in the rodent lineage so sequence gaps are not treated as orthology breaks. 
Relative orthology of human/mouse and dog/mouse nets are used to avoid false positives due to deletions 
in the human genome. Since older retrogenes will not show a break in orthology, this feature is 
weighted lower than other features when scoring putative retrogenes.
</p>

<h2>Credits</h2>

<p>
The RetroFinder program and browser track were developed by
<a href="mailto:&#98;a&#101;r&#116;&#115;&#99;&#104;&#64;&#115;&#111;&#101;.
&#117;&#99;sc.&#101;&#100;&#117;">Robert Baertsch</a> at UCSC.
</p>
<!-- above address is baertsch at soe.ucsc.edu -->

<h2>References</h2>

<p>
Baertsch R, Diekhans M, Kent WJ, Haussler D, Brosius J.
<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-9-466" target="_blank">
Retrocopy contributions to the evolution of the human genome</a>.
<em>BMC Genomics</em>. 2008 Oct 8;9:466.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18842134" target="_blank">18842134</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2584115" target="_blank">PMC2584115</a>
</p>

<p>
Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D.
<a href="https://www.pnas.org/content/100/20/11484" target="_blank">
Evolution's cauldron: duplication, deletion, and rearrangement in the mouse and human genomes</a>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Sep 30;100(20):11484-9.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14500911" target="_blank">14500911</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC208784" target="_blank">PMC208784</a>
</p>

<p>
Pei B, Sisu C, Frankish A, Howald C, Habegger L, Mu XJ, Harte R, Balasubramanian S, Tanzer A,
Diekhans M <em>et al</em>.
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2012-13-9-r51" target="_blank">
The GENCODE pseudogene resource</a>.
<em>Genome Biol</em>. 2012 Sep 26;13(9):R51.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22951037" target="_blank">22951037</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3491395" target="_blank">PMC3491395</a>
</p>

<p>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC, Haussler D, Miller W.
<a href="https://genome.cshlp.org/content/13/1/103.abstract" target="_blank">
Human-mouse alignments with BLASTZ</a>.
<em>Genome Res</em>. 2003 Jan;13(1):103-7.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12529312" target="_blank">12529312</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC430961" target="_blank">PMC430961</a>
</p>

<p>
Zheng D, Frankish A, Baertsch R, Kapranov P, Reymond A, Choo SW, Lu Y, Denoeud F, Antonarakis SE,
Snyder M <em>et al</em>.
<a href="https://genome.cshlp.org/content/17/6/839.full" target="_blank">
Pseudogenes in the ENCODE regions: consensus annotation, analysis of transcription, and
evolution</a>.
<em>Genome Res</em>. 2007 Jun;17(6):839-51.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17568002" target="_blank">17568002</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1891343" target="_blank">PMC1891343</a>
</p>
