<P>
The <em>Drosophila yakuba</em> Release 2.0 (Nov. 2005) draft genome sequence 
was produced by the 
<A HREF="http://genome.wustl.edu/"
TARGET=_blank>Genome Sequencing Center</A> at the Washington University 
(WUSTL) School of Medicine in St. Louis.

<H3>Sample position queries</H3>
<P>
A genome position can be specified by the accession number of an
mRNA or EST, a chromosomal range, or keywords
from the GenBank description of an mRNA. The following list provides
examples of valid position queries for the Drosophila
genome. Note that some position queries (e.g. &quot;huntington&quot;) may return
matches to the mRNA records of other species. In these cases, the
mRNAs are mapped to their homologs in Drosophila.  See the 
<A HREF="../goldenPath/help/hgTracksHelp.html" TARGET=_blank>
User's Guide</A> for more information.
<P>
<TABLE  border=0 CELLPADDING=0 CELLSPACING=0>
    <TR><TD VALIGN=Top NOWRAP><B>Request:</B><BR></TD>
	<TD VALIGN=Top COLSPAN=2><B>&nbsp;&nbsp; Genome Browser Response:</B><BR></TD>
    </TR>
	
    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>chr2L</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays all of chromosome 2L (see the Assembly Details
	section for a list of chromosomes present in this assembly)</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>chrYh:1-75000</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays first 75,000 bases of chromosome Yh (h = heterochromatin random reads)</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>AF532011</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays region aligning with <em>D. yakuba</em> mRNA with GenBank accession number AF532011</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>BQ705701</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Displays region aligning with <em>D. yakuba</em> EST with GenBank accession BQ705701</TD>
    </TR>

    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD VALIGN=Top NOWRAP>p53</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists mRNAs related to the p53 tumor suppressor</TD>
    </TR> 

    <TR><TD VALIGN=Top NOWRAP>zinc finger</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists zinc finger mRNAs</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>huntington</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists mRNAs associated with Huntington's disease</TD>
    </TR>

    <TR><TD VALIGN=Top NOWRAP>Tautz,D.</TD>
	<TD WIDTH=14></TD>
	<TD VALIGN=Top>Lists mRNAs deposited by co-author D. Tautz</TD>
    </TR>

    <TR><TD VALIGN=Top><BR></TD></TR>
    <TR><TD COLSPAN="3"> Use this last format for author queries. Although GenBank 
			requires the search format <em>Tautz D</em>, internally it uses 
			the format <em>Tautz,D.</em>.</TD>
    </TR>
</TABLE>
<P>
<HR>
<H3>Assembly details</H3>
<P>
The Release 2.0 assembly includes both raw whole genome shotgun (WGS) data and
data from two rounds of automated, directed read selection, which improved
the sequence quality and narrowed or (in some cases) closed gaps.
The raw WGS assembly data consists of 8,362 supercontigs larger than 1 kb, 
with a total size of approximately 161 Mb and an N50 length of about 2.5 Mb. 
The total number of contigs greater than 1 kb is 11,806, with a total size of
approximately 161 Mb and an N50 length of 119 kb. </P>
<P>
The data were
intially assembled using the whole-genome assembly program, PCAP 
(<A HREF="http://genome.cshlp.org/content/13/9/2164.long"
TARGET=_BLANK>Huang, <em>et al.</em>, 2003</A>).
To create chromosomal assignments and ordering along the
chromosomes, WUSTL first identified regions of the <em>D. yakuba</em> assembly
that uniquely aligned with the <em>D. melanogaster</em> genome. The <em>D. 
yakuba</em> contigs were initially ordered by their position along the
assigned <em>D. melanogaster</em> chromosomes (or segments for the
heterochromatin). Alignment of all <em>D. melanogaster</em> mRNAs against the
<em>D. yakuba</em> assembly were used to confirm these DNA-based alignments.
WUSTL then introduced the inversions indicated by the <em>D. yakuba</em> sequence
assembly to create the <em>D. yakuba</em> chromosomal files. The number of
inversions required and the resulting banding patterns are in good
agreement with the expected number and patterns, based on polytene
chromosome banding data from 
<A HREF="https://www.ncbi.nlm.nih.gov/pubmed/6967"
TARGET=_BLANK>Lemeunier and Ashburner</A> (1976).</P>
<P>
The sequence is distributed among the following chromosomes: 2L, 2R, 
2h, 3L, 3R, 3h, 4, X, Xh, Yh, M (mitochondria) and U. Sequence that
could not be reliably ordered within a chromosome <em>N</em> has been placed on
chr<em>N</em>_random. Heterochromatin that has been localized to a specific
chromosome can be found in chr<em>N</em>h or Chr<em>N</em>h_random. </P>
<P>
For more information about the <em>D. yakuba</em> assembly and statistics, see 
the WUSTL Genome Sequencing Center 
<A HREF="http://genome.wustl.edu/genomes/detail/drosophila-yakuba/"
TARGET=_blank><em>Drosophila yakuba</em></A> web page.
<P>
Downloads of the droYak2 data and annotations may be obtained from the 
UCSC Genome Browser 
<A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/droYak2/" TARGET=_blank>FTP server</A>
or <A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#droYak">Downloads</A> page. 
The droYak2 annotation tracks were generated by UCSC and 
collaborators worldwide. Thanks to the Genome Sequencing Center, WUSTL School
of Medicine for providing the sequence of this genome.  See the 
<A HREF="../goldenPath/credits.html#droYak_credits">Credits</A> 
page for a detailed list of the organizations and individuals who contributed 
to this release. 
<HR>
<H3>GenBank Pipeline Details</H3>
For the purposes of the GenBank alignment pipeline, this assembly is considered to be: <B>well-ordered</B>.
