<h2>Description</h2>

<p>
The RefSeq mRNAs gene track for the <em>$organism ($date)</em>
genome assembly displays translated blat alignments of vertebrate and
invertebrate mRNA in
<a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank"> GenBank</a>.
</p>

<h2>Track statistics summary</h2>
<p>
<b>Total genome size: </b>2,654,624,157 (not counting gaps)<br>
<b>Gene count: </b>22,442<br>
<b>Bases in genes: </b>838,462,469 (txStart to txEnd)<br>
<b>Genes percent genome coverage: </b>% 31.585<br>
<b>Bases in exons: </b>53,564,706<br>
<b>Exons percent genome coverage: </b>% 2.018<br>
</p>

<h2>Search tips</h2>
<p>
Please note, the name searching system is not completely case insensitive.
When in doubt, enter search names in all lower case to find gene names.
</p>

<h2>Methods</h2>

<p>
The mRNAs were aligned against the <em>$organism ($date)</em> genome using
translated blat.  When a single mRNA aligned in multiple places, the alignment
having the highest base identity was found.  Only those alignments having a base
identity level within 1% of the best and at least 25% base identity with the
genomic sequence were kept.
</p>

<p>
Specifically, the translated blat command is:
<pre>
blat -noHead -q=rnax -t=dnax -mask=lower target.fa query.fa target.query.psl

where target.fa is one of the chromosome sequence of the genome assembly,
and the query.fa is the mRNAs from RefSeq
</pre>
The resulting PSL outputs are filtered:
<pre>
pslCDnaFilter -minId=0.35 -minCover=0.25  -globalNearBest=0.0100 -minQSize=20 \
  -ignoreIntrons -repsAsMatch -ignoreNs -bestOverlap \
    all.results.psl mm39.xenoRefGene.psl
</pre>
The filtered mm39.xenoRefGene.psl is converted to
<a href='http://genome.ucsc.edu/FAQ/FAQformat.html#format9'
target=_blank>genePred data</a> to display for this track.
</p>

<H2>Credits</h2>

<p>
The mRNA track was produced at UCSC from mRNA sequence data
submitted to the international public sequence databases by
scientists worldwide.
</p>

<h2>References</h2>
<p>
Benson DA, Cavanaugh M, Clark K, Karsch-Mizrachi I, Lipman DJ, Ostell J, Sayers EW.
<a href="https://academic.oup.com/nar/article/41/D1/D36/1068219/GenBank" target="_blank">
GenBank</a>.
<em>Nucleic Acids Res</em>. 2013 Jan;41(Database issue):D36-42.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23193287" target="_blank">23193287</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531190/" target="_blank">PMC3531190</a>
</p>

<P>
Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL.
<A HREF="https://academic.oup.com/nar/article/32/suppl_1/D23/2505202/GenBank-update"
TARGET=_blank>GenBank: update</A>.
<em>Nucleic Acids Res</em>. 2004 Jan 1;32(Database issue):D23-6.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14681350" target="_blank">14681350</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC308779/" target="_blank">PMC308779</a>
</p>

<P>
Kent WJ.
<A HREF="https://genome.cshlp.org/content/12/4/656.abstract"
TARGET=_blank>BLAT - the BLAST-like alignment tool</A>.
<em>Genome Res</em>. 2002 Apr;12(4):656-64.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11932250" target="_blank">11932250</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC187518/" target="_blank">PMC187518</a>
</p>

